The following text has been accessed from https://en.wikipedia.org/wiki/Regulation_of_gene_expression at Thu Aug 8 23:59:20 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Regulation of gene expression ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
"Gene modulation" redirects here. For information on therapeutic regulation of
gene expression, see therapeutic_gene_modulation.
For vocabulary, see Glossary_of_gene_expression_terms.
Regulation of gene expression by a hormone receptor
Diagram showing at which stages in the DNA-mRNA-protein pathway expression can
be controlled
Regulation of gene expression, or gene regulation,[1] includes a wide range of
mechanisms that are used by cells to increase or decrease the production of
specific gene_products (protein or RNA). Sophisticated programs of gene
expression are widely observed in biology, for example to trigger developmental
pathways, respond to environmental stimuli, or adapt to new food sources.
Virtually any step of gene expression can be modulated, from transcriptional
initiation, to RNA_processing, and to the post-translational_modification of a
protein. Often, one gene regulator controls another, and so on, in a gene
regulatory_network.
Gene regulation is essential for viruses, prokaryotes and eukaryotes as it
increases the versatility and adaptability of an organism by allowing the cell
to express protein when needed. Although as early as 1951, Barbara_McClintock
showed interaction between two genetic loci, Activator (Ac) and Dissociator
(Ds), in the color formation of maize seeds, the first discovery of a gene
regulation system is widely considered to be the identification in 1961 of the
lac_operon, discovered by FranÃ§ois_Jacob and Jacques_Monod, in which some
enzymes involved in lactose metabolism are expressed by E._coli only in the
presence of lactose and absence of glucose.
In multicellular organisms, gene regulation drives cellular_differentiation and
morphogenesis in the embryo, leading to the creation of different cell types
that possess different gene expression profiles from the same genome sequence.
Although this does not explain how gene regulation originated, evolutionary
biologists include it as a partial explanation of how evolution works at a
molecular_level, and it is central to the science of evolutionary_developmental
biology ("evo-devo").
⁰
***** Contents *****
    * 1_Regulated_stages_of_gene_expression
    * 2_Modification_of_DNA
          o 2.1_Structural
          o 2.2_Chemical
    * 3_Regulation_of_transcription
    * 4_Regulation_of_transcription_in_cancer
    * 5_Regulation_of_transcription_in_addiction
    * 6_Post-transcriptional_regulation
    * 7_Three_prime_untranslated_regions_and_microRNAs
    * 8_Regulation_of_translation
    * 9_Examples_of_gene_regulation
          o 9.1_Developmental_biology
    * 10_Circuitry
          o 10.1_Up-regulation_and_down-regulation
          o 10.2_Inducible_vs._repressible_systems
          o 10.3_Theoretical_circuits
    * 11_Study_methods
    * 12_See_also
    * 13_Notes_and_references
    * 14_Bibliography
    * 15_External_links
***** Regulated stages of gene expression[edit] *****
Any step of gene expression may be modulated, from the DNA-RNA transcription
step to post-translational_modification of a protein. The following is a list
of stages where gene expression is regulated, the most extensively utilised
point is Transcription Initiation:
    * Chromatin_domains
    * Transcription
    * Post-transcriptional_modification
    * RNA transport
    * Translation
    * mRNA_degradation
***** Modification of DNA[edit] *****
In eukaryotes, the accessibility of large regions of DNA can depend on its
chromatin structure, which can be altered as a result of histone modifications
directed by DNA_methylation, ncRNA, or DNA-binding_protein. Hence these
modifications may up or down regulate the expression of a gene. Some of these
modifications that regulate gene expression are inheritable and are referred to
as epigenetic_regulation.
**** Structural[edit] ****
Transcription of DNA is dictated by its structure. In general, the density of
its packing is indicative of the frequency of transcription. Octameric protein
complexes called nucleosomes are responsible for the amount of supercoiling of
DNA, and these complexes can be temporarily modified by processes such as
phosphorylation or more permanently modified by processes such as methylation.
Such modifications are considered to be responsible for more or less permanent
changes in gene expression levels.[2]
**** Chemical[edit] ****
Methylation_of_DNA is a common method of gene silencing. DNA is typically
methylated by methyltransferase enzymes on cytosine nucleotides in a CpG
dinucleotide sequence (also called "CpG_islands" when densely clustered).
Analysis of the pattern of methylation in a given region of DNA (which can be a
promoter) can be achieved through a method called bisulfite mapping. Methylated
cytosine residues are unchanged by the treatment, whereas unmethylated ones are
changed to uracil. The differences are analyzed by DNA sequencing or by methods
developed to quantify SNPs, such as Pyrosequencing (Biotage) or MassArray
(Sequenom), measuring the relative amounts of C/T at the CG dinucleotide.
Abnormal methylation patterns are thought to be involved in oncogenesis.[3]
Histone_acetylation is also an important process in transcription. Histone
acetyltransferase enzymes (HATs) such as CREB-binding_protein also dissociate
the DNA from the histone complex, allowing transcription to proceed. Often, DNA
methylation and histone deacetylation work together in gene_silencing. The
combination of the two seems to be a signal for DNA to be packed more densely,
lowering gene expression.[citation_needed]
***** Regulation of transcription[edit] *****
Main article: Transcriptional_regulation
1: RNA Polymerase, 2: Repressor, 3: Promoter, 4: Operator, 5: Lactose, 6: lacZ,
7: lacY, 8: lacA. Top: The gene is essentially turned off. There is no lactose
to inhibit the repressor, so the repressor binds to the operator, which
obstructs the RNA polymerase from binding to the promoter and making lactase.
Bottom: The gene is turned on. Lactose is inhibiting the repressor, allowing
the RNA polymerase to bind with the promoter, and express the genes, which
synthesize lactase. Eventually, the lactase will digest all of the lactose,
until there is none to bind to the repressor. The repressor will then bind to
the operator, stopping the manufacture of lactase.
Regulation of transcription thus controls when transcription occurs and how
much RNA is created. Transcription of a gene by RNA_polymerase can be regulated
by several mechanisms. Specificity_factors alter the specificity of RNA
polymerase for a given promoter or set of promoters, making it more or less
likely to bind to them (i.e., sigma_factors used in prokaryotic_transcription).
Repressors bind to the Operator, coding sequences on the DNA strand that are
close to or overlapping the promoter region, impeding RNA polymerase's progress
along the strand, thus impeding the expression of the gene.The image to the
right demonstrates regulation by a repressor in the lac operon. General
transcription_factors position RNA polymerase at the start of a protein-coding
sequence and then release the polymerase to transcribe the mRNA. Activators
enhance the interaction between RNA polymerase and a particular promoter,
encouraging the expression of the gene. Activators do this by increasing the
attraction of RNA polymerase for the promoter, through interactions with
subunits of the RNA polymerase or indirectly by changing the structure of the
DNA. Enhancers are sites on the DNA helix that are bound by activators in order
to loop the DNA bringing a specific promoter to the initiation complex.
Enhancers are much more common in eukaryotes than prokaryotes, where only a few
examples exist (to date).[4] Silencers are regions of DNA sequences that, when
bound by particular transcription factors, can silence expression of the gene.
***** Regulation of transcription in cancer[edit] *****
Main article: Regulation_of_transcription_in_cancer
In vertebrates, the majority of gene promoters contain a CpG_island with
numerous CpG_sites.[5] When many of a gene's promoter CpG sites are methylated
the gene becomes silenced.[6] Colorectal cancers typically have 3 to 6 driver
mutations and 33 to 66 hitchhiker or passenger mutations.[7] However,
transcriptional silencing may be of more importance than mutation in causing
progression to cancer. For example, in colorectal cancers about 600 to 800
genes are transcriptionally silenced by CpG island methylation (see regulation
of_transcription_in_cancer). Transcriptional repression in cancer can also
occur by other epigenetic mechanisms, such as altered expression of microRNAs.
[8] In breast cancer, transcriptional repression of BRCA1 may occur more
frequently by over-expressed microRNA-182 than by hypermethylation of the BRCA1
promoter (see Low_expression_of_BRCA1_in_breast_and_ovarian_cancers).
***** Regulation of transcription in addiction[edit] *****
One of the cardinal features of addiction is its persistence. The persistent
behavioral changes appear to be due to long-lasting changes, resulting from
epigenetic alterations affecting gene expression, within particular regions of
the brain.[9] Drugs of abuse cause three types of epigenetic alteration in the
brain. These are (1) histone acetylations and histone_methylations, (2) DNA
methylation at CpG_sites, and (3) epigenetic downregulation_or_upregulation of
microRNAs.[9][10] (See Epigenetics_of_cocaine_addiction for some details.)
Chronic nicotine intake in mice alters brain cell epigenetic control of gene
expression through acetylation_of_histones. This increases expression in the
brain of the protein FosB, important in addiction.[11] Cigarette addiction was
also studied in about 16,000 humans, including never smokers, current smokers,
and those who had quit smoking for up to 30 years.[12] In blood cells, more
than 18,000 CpG_sites (of the roughly 450,000 analyzed CpG sites in the genome)
had frequently altered methylation among current smokers. These CpG sites
occurred in over 7,000 genes, or roughly a third of known human genes. The
majority of the differentially methylated CpG_sites returned to the level of
never-smokers within five years of smoking cessation. However, 2,568 CpGs among
942 genes remained differentially methylated in former versus never smokers.
Such remaining epigenetic changes can be viewed as âmolecular scarsâ[10]
that may affect gene expression.
In rodent models, drugs of abuse, including cocaine,[13] methampheamine,[14]
[15] alcohol[16] and tobacco smoke products,[17] all cause DNA damage in the
brain. During repair of DNA damages some individual repair events can alter the
methylation of DNA and/or the acetylations or methylations of histones at the
sites of damage, and thus can contribute to leaving an epigenetic scar on
chromatin.[18]
Such epigenetic scars likely contribute to the persistent epigenetic changes
found in addiction.
***** Post-transcriptional regulation[edit] *****
Main article: Post-transcriptional_regulation
After the DNA is transcribed and mRNA is formed, there must be some sort of
regulation on how much the mRNA is translated into proteins. Cells do this by
modulating the capping, splicing, addition of a Poly(A) Tail, the sequence-
specific nuclear export rates, and, in several contexts, sequestration of the
RNA transcript. These processes occur in eukaryotes but not in prokaryotes.
This modulation is a result of a protein or transcript that, in turn, is
regulated and may have an affinity for certain sequences.
***** Three prime untranslated regions and microRNAs[edit] *****
Main article: Three_prime_untranslated_region
Main article: MicroRNA
Three_prime_untranslated_regions (3'-UTRs) of messenger_RNAs (mRNAs) often
contain regulatory sequences that post-transcriptionally influence gene
expression.[19] Such 3'-UTRs often contain both binding sites for microRNAs
(miRNAs) as well as for regulatory proteins. By binding to specific sites
within the 3'-UTR, miRNAs can decrease gene expression of various mRNAs by
either inhibiting translation or directly causing degradation of the
transcript. The 3'-UTR also may have silencer regions that bind repressor
proteins that inhibit the expression of a mRNA.
The 3'-UTR often contains miRNA_response_elements_(MREs). MREs are sequences to
which miRNAs bind. These are prevalent motifs within 3'-UTRs. Among all
regulatory motifs within the 3'-UTRs (e.g. including silencer regions), MREs
make up about half of the motifs.
As of 2014, the miRBase web site,[20] an archive of miRNA sequences and
annotations, listed 28,645 entries in 233 biologic species. Of these, 1,881
miRNAs were in annotated human miRNA loci. miRNAs were predicted to have an
average of about four hundred target mRNAs (affecting expression of several
hundred genes).[21] Freidman et al.[21] estimate that >45,000 miRNA target
sites within human mRNA 3'-UTRs are conserved above background levels, and >60%
of human protein-coding genes have been under selective pressure to maintain
pairing to miRNAs.
Direct experiments show that a single miRNA can reduce the stability of
hundreds of unique mRNAs.[22] Other experiments show that a single miRNA may
repress the production of hundreds of proteins, but that this repression often
is relatively mild (less than 2-fold).[23][24]
The effects of miRNA dysregulation of gene expression seem to be important in
cancer.[25] For instance, in gastrointestinal cancers, a 2015 paper identified
nine miRNAs as epigenetically altered and effective in down-regulating DNA
repair enzymes.[26]
The effects of miRNA dysregulation of gene expression also seem to be important
in neuropsychiatric disorders, such as schizophrenia, bipolar_disorder, major
depressive_disorder, Parkinson's_disease, Alzheimer's_disease and autism
spectrum disorders.[27][28][29]
***** Regulation of translation[edit] *****
Main article: Translational_regulation
The translation of mRNA can also be controlled by a number of mechanisms,
mostly at the level of initiation. Recruitment of the small ribosomal subunit
can indeed be modulated by mRNA secondary structure, antisense RNA binding, or
protein binding. In both prokaryotes and eukaryotes, a large number of RNA
binding proteins exist, which often are directed to their target sequence by
the secondary structure of the transcript, which may change depending on
certain conditions, such as temperature or presence of a ligand (aptamer). Some
transcripts act as ribozymes and self-regulate their expression.
***** Examples of gene regulation[edit] *****
    * Enzyme_induction is a process in which a molecule (e.g., a drug) induces
      (i.e., initiates or enhances) the expression of an enzyme.
    * The induction of heat_shock_proteins in the fruit fly Drosophila
      melanogaster.
    * The Lac_operon is an interesting example of how gene expression can be
      regulated.
    * Viruses, despite having only a few genes, possess mechanisms to regulate
      their gene expression, typically into an early and late phase, using
      collinear systems regulated by anti-terminators (lambda_phage) or
      splicing modulators (HIV).
    * Gal4 is a transcriptional activator that controls the expression of GAL1,
      GAL7, and GAL10 (all of which code for the metabolic of galactose in
      yeast). The GAL4/UAS_system has been used in a variety of organisms
      across various phyla to study gene expression.[30]
**** Developmental biology[edit] ****
Main article: Evolutionary_developmental_biology
A large number of studied regulatory systems come from developmental_biology.
Examples include:
    * The colinearity of the Hox_gene cluster with their nested antero-
      posterior patterning
    * Pattern generation of the hand (digits - interdigits): the gradient of
      sonic_hedgehog (secreted inducing factor) from the zone_of_polarizing
      activity in the limb, which creates a gradient of active Gli3, which
      activates Gremlin, which inhibits BMPs also secreted in the limb, results
      in the formation of an alternating pattern of activity as a result of
      this reaction-diffusion_system.
    * Somitogenesis is the creation of segments (somites) from a uniform tissue
      (Pre-somitic Mesoderm). They are formed sequentially from anterior to
      posterior. This is achieved in amniotes possibly by means of two opposing
      gradients, Retinoic acid in the anterior (wavefront) and Wnt and Fgf in
      the posterior, coupled to an oscillating pattern (segmentation clock)
      composed of FGF + Notch and Wnt in antiphase.[31]
    * Sex determination in the soma of a Drosophila requires the sensing of the
      ratio of autosomal genes to sex_chromosome-encoded genes, which results
      in the production of sexless splicing factor in females, resulting in the
      female isoform of doublesex.[32]
***** Circuitry[edit] *****
Main article: Gene_regulatory_network
**** Up-regulation and down-regulation[edit] ****
Up-regulation is a process that occurs within a cell triggered by a signal
(originating internal or external to the cell), which results in increased
expression of one or more genes and as a result the protein(s) encoded by those
genes. Conversely, down-regulation is a process resulting in decreased gene and
corresponding protein expression.
    * Up-regulation occurs, for example, when a cell is deficient in some kind
      of receptor. In this case, more receptor protein is synthesized and
      transported to the membrane of the cell and, thus, the sensitivity of the
      cell is brought back to normal, reestablishing homeostasis.
    * Down-regulation occurs, for example, when a cell is overstimulated by a
      neurotransmitter, hormone, or drug for a prolonged period of time, and
      the expression of the receptor protein is decreased in order to protect
      the cell (see also tachyphylaxis).
**** Inducible vs. repressible systems[edit] ****
Gene Regulation can be summarized by the response of the respective system:
    * Inducible systems - An inducible system is off unless there is the
      presence of some molecule (called an inducer) that allows for gene
      expression. The molecule is said to "induce expression". The manner by
      which this happens is dependent on the control mechanisms as well as
      differences between prokaryotic and eukaryotic cells.
    * Repressible systems - A repressible system is on except in the presence
      of some molecule (called a corepressor) that suppresses gene expression.
      The molecule is said to "repress expression". The manner by which this
      happens is dependent on the control mechanisms as well as differences
      between prokaryotic and eukaryotic cells.
The GAL4/UAS_system is an example of both an inducible and repressible system.
Gal4 binds an upstream activation sequence (UAS) to activate the transcription
of the GAL1/GAL7/GAL10 cassette. On the other hand, a MIG1 response to the
presence of glucose can inhibit GAL4 and therefore stop the expression of the
GAL1/GAL7/GAL10 cassette.[33]
**** Theoretical circuits[edit] ****
    * Repressor/Inducer: an activation of a sensor results in the change of
      expression of a gene
    * negative feedback: the gene product downregulates its own production
      directly or indirectly, which can result in
          o keeping transcript levels constant/proportional to a factor
          o inhibition of run-away reactions when coupled with a positive
            feedback loop
          o creating an oscillator by taking advantage in the time delay of
            transcription and translation, given that the mRNA and protein
            half-life is shorter
    * positive feedback: the gene product upregulates its own production
      directly or indirectly, which can result in
          o signal amplification
          o bistable switches when two genes inhibit each other and both have
            positive feedback
          o pattern generation
***** Study methods[edit] *****
For DNA and RNA methods, see nucleic_acid_methods.
For protein methods, see protein_methods.
In general, most experiments investigating differential expression used whole
cell extracts of RNA, called steady-state levels, to determine which genes
changed and by how much. These are, however, not informative of where the
regulation has occurred and may mask conflicting regulatory processes (see
post-transcriptional_regulation), but it is still the most commonly analysed
(quantitative_PCR and DNA_microarray).
When studying gene expression, there are several methods to look at the various
stages. In eukaryotes these include:
    * The local chromatin environment of the region can be determined by ChIP-
      chip analysis by pulling down RNA_Polymerase_II, Histone_3 modifications,
      Trithorax-group_protein, Polycomb-group_protein, or any other DNA-binding
      element to which a good antibody is available.
    * Epistatic interactions can be investigated by synthetic_genetic_array
      analysis
    * Due to post-transcriptional regulation, transcription rates and total RNA
      levels differ significantly. To measure the transcription rates nuclear
      run-on assays can be done and newer high-throughput methods are being
      developed, using thiol labelling instead of radioactivity.[34]
    * Only 5% of the RNA polymerised in the nucleus exits,[35] and not only
      introns, abortive_products, and non-sense transcripts are degradated.
      Therefore, the differences in nuclear and cytoplasmic levels can be see
      by separating the two fractions by gentle lysis.[36]
    * Alternative splicing can be analysed with a splicing array or with a
      tiling array (see DNA_microarray).
    * All in_vivo RNA is complexed as RNPs. The quantity of transcripts bound
      to specific protein can be also analysed by RIP-Chip. For example, DCP2
      will give an indication of sequestered protein; ribosome-bound gives and
      indication of transcripts active in transcription (although a more dated
      method, called polysome fractionation, is still popular in some labs)
    * Protein levels can be analysed by Mass_spectrometry, which can be
      compared only to quantitative_PCR data, as microarray data is relative
      and not absolute.
    * RNA and protein degradation rates are measured by means of transcription
      inhibitors (actinomycin_D or Î±-amanitin) or translation inhibitors
      (Cycloheximide), respectively.
***** See also[edit] *****
    * Artificial_transcription_factors (small molecules that mimic
      transcription factor protein)
    * Cellular_model
    * Conserved_non-coding_DNA_sequence
    * Enhancer_(genetics)
    * Gene_structure
    * Spatiotemporal_gene_expression
***** Notes and references[edit] *****
   1. ^Reference, Genetics Home. "Can_genes_be_turned_on_and_off_in_cells?".
      Genetics Home Reference.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Bell JT, Pai AA, Pickrell JK, Gaffney DJ, Pique-Regi R, Degner JF, Gilad
      Y, Pritchard JK (2011). "DNA_methylation_patterns_associate_with_genetic
      and_gene_expression_variation_in_HapMap_cell_lines". Genome Biology. 12
      (1): R10. doi:10.1186/gb-2011-12-1-r10. PMC 3091299. PMID 21251332.
   4. ^Vertino PM, Spillare EA, Harris CC, Baylin SB (Apr 1993). "Altered
      chromosomal_methylation_patterns_accompany_oncogene-induced
      transformation_of_human_bronchial_epithelial_cells" (PDF). Cancer
      Research. 53 (7): 1684â9. PMID 8453642.
   5. ^Austin S, Dixon R (Jun 1992). "The_prokaryotic_enhancer_binding_protein
      NTRC_has_an_ATPase_activity_which_is_phosphorylation_and_DNA_dependent".
      The EMBO Journal. 11 (6): 2219â28. doi:10.1002/j.1460-
      2075.1992.tb05281.x. PMC 556689. PMID 1534752.
   6. ^Saxonov S, Berg P, Brutlag DL (2006). "A_genome-wide_analysis_of_CpG
      dinucleotides_in_the_human_genome_distinguishes_two_distinct_classes_of
      promoters". Proc. Natl. Acad. Sci. U.S.A. 103 (5): 1412â7. Bibcode:
      2006PNAS..103.1412S. doi:10.1073/pnas.0510310103. PMC 1345710.
      PMID 16432200.
   7. ^Bird A (2002). "DNA methylation patterns and epigenetic memory". Genes
      Dev. 16 (1): 6â21. doi:10.1101/gad.947102. PMID 11782440.
   8. ^Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW
      (2013). "Cancer_genome_landscapes". Science. 339 (6127): 1546â58.
      Bibcode:2013Sci...339.1546V. doi:10.1126/science.1235122. PMC 3749880.
      PMID 23539594.
   9. ^Tessitore A, Cicciarelli G, Del Vecchio F, Gaggiano A, Verzella D,
      Fischietti M, Vecchiotti D, Capece D, Zazzeroni F, Alesse E (2014).
      "MicroRNAs_in_the_DNA_Damage/Repair_Network_and_Cancer". Int J Genom.
      2014: 1â10. doi:10.1155/2014/820248. PMC 3926391. PMID 24616890.
  10. ^ a bNestler EJ (January 2014). "Epigenetic_mechanisms_of_drug
      addiction". Neuropharmacology. 76 Pt B: 259â68. doi:10.1016/
      j.neuropharm.2013.04.004. PMC 3766384. PMID 23643695.
  11. ^ a bRobison AJ, Nestler EJ (October 2011). "Transcriptional_and
      epigenetic_mechanisms_of_addiction". Nat. Rev. Neurosci. 12 (11):
      623â37. doi:10.1038/nrn3111. PMC 3272277. PMID 21989194.
  12. ^Levine A, Huang Y, Drisaldi B, Griffin EA, Pollak DD, Xu S, Yin D,
      Schaffran C, Kandel DB, Kandel ER (November 2011). "Molecular_mechanism
      for_a_gateway_drug:_epigenetic_changes_initiated_by_nicotine_prime_gene
      expression_by_cocaine". Sci Transl Med. 3 (107): 107ra109. doi:10.1126/
      scitranslmed.3003062. PMC 4042673. PMID 22049069.
  13. ^Joehanes R, Just AC, Marioni RE, Pilling LC, Reynolds LM, Mandaviya PR,
      et al. (October 2016). "Epigenetic_Signatures_of_Cigarette_Smoking". Circ
      Cardiovasc Genet. 9 (5): 436â447. doi:10.1161/CIRCGENETICS.116.001506.
      PMC 5267325. PMID 27651444.
  14. ^de Souza MF, GonÃ§ales TA, Steinmetz A, Moura DJ, Saffi J, Gomez R,
      Barros HM (April 2014). "Cocaine induces DNA damage in distinct brain
      areas of female rats under different hormonal conditions". Clin. Exp.
      Pharmacol. Physiol. 41 (4): 265â9. doi:10.1111/1440-1681.12218.
      PMID 24552452.
  15. ^Johnson Z, Venters J, Guarraci FA, Zewail-Foote M (June 2015).
      "Methamphetamine induces DNA damage in specific regions of the female rat
      brain". Clin. Exp. Pharmacol. Physiol. 42 (6): 570â5. doi:10.1111/1440-
      1681.12404. PMID 25867833.
  16. ^Tokunaga I, Ishigami A, Kubo S, Gotohda T, Kitamura O (August 2008).
      "The peroxidative DNA damage and apoptosis in methamphetamine-treated rat
      brain". J. Med. Invest. 55 (3â4): 241â5. doi:10.2152/jmi.55.241.
      PMID 18797138.
  17. ^Rulten SL, Hodder E, Ripley TL, Stephens DN, Mayne LV (July 2008).
      "Alcohol induces DNA damage and the Fanconi anemia D2 protein implicating
      FANCD2 in the DNA damage response pathways in brain". Alcohol. Clin. Exp.
      Res. 32 (7): 1186â96. doi:10.1111/j.1530-0277.2008.00673.x.
      PMID 18482162.
  18. ^Adhami N, Chen Y, Martins-Green M (October 2017). "Biomarkers of disease
      can be detected in mice as early as 4 weeks after initiation of exposure
      to third-hand smoke levels equivalent to those found in homes of
      smokers". Clin. Sci. 131 (19): 2409â2426. doi:10.1042/CS20171053.
      PMID 28912356.
  19. ^Dabin J, Fortuny A, Polo SE (June 2016). "Epigenome_Maintenance_in
      Response_to_DNA_Damage". Mol. Cell. 62 (5): 712â27. doi:10.1016/
      j.molcel.2016.04.006. PMC 5476208. PMID 27259203.
  20. ^Ogorodnikov A, Kargapolva Y, Danckwardt S (2016). "Processing_and
      transcriptome_expansion_at_the_mRNA_3â²_end_in_health_and_disease:
      finding_the_right_end". Eur J Physiol. 468 (6): 993â1012. doi:10.1007/
      s00424-016-1828-3. PMC 4893057. PMID 27220521.
  21. ^ miRBase.org
  22. ^ a bFriedman RC, Farh KK, Burge CB, Bartel DP (Jan 2009). "Most
      mammalian_mRNAs_are_conserved_targets_of_microRNAs". Genome Research. 19
      (1): 92â105. doi:10.1101/gr.082701.108. PMC 2612969. PMID 18955434.
  23. ^Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, Castle J,
      Bartel DP, Linsley PS, Johnson JM (Feb 2005). "Microarray analysis shows
      that some microRNAs downregulate large numbers of target mRNAs". Nature.
      433 (7027): 769â73. Bibcode:2005Natur.433..769L. doi:10.1038/
      nature03315. PMID 15685193.
  24. ^Selbach M, SchwanhÃ¤usser B, Thierfelder N, Fang Z, Khanin R, Rajewsky N
      (Sep 2008). "Widespread changes in protein synthesis induced by
      microRNAs". Nature. 455 (7209): 58â63. Bibcode:2008Natur.455...58S.
      doi:10.1038/nature07228. PMID 18668040.
  25. ^Baek D, VillÃ©n J, Shin C, Camargo FD, Gygi SP, Bartel DP (Sep 2008).
      "The_impact_of_microRNAs_on_protein_output". Nature. 455 (7209): 64â71.
      Bibcode:2008Natur.455...64B. doi:10.1038/nature07242. PMC 2745094.
      PMID 18668037.
  26. ^Palmero EI, de Campos SG, Campos M, de Souza NC, Guerreiro ID, Carvalho
      AL, Marques MM (Jul 2011). "Mechanisms_and_role_of_microRNA_deregulation
      in_cancer_onset_and_progression". Genetics and Molecular Biology. 34 (3):
      363â70. doi:10.1590/S1415-47572011000300001. PMC 3168173.
      PMID 21931505.
  27. ^Bernstein C, Bernstein H (May 2015). "Epigenetic_reduction_of_DNA_repair
      in_progression_to_gastrointestinal_cancer". World Journal of
      Gastrointestinal Oncology. 7 (5): 30â46. doi:10.4251/wjgo.v7.i5.30.
      PMC 4434036. PMID 25987950.
  28. ^Maffioletti E, Tardito D, Gennarelli M, Bocchio-Chiavetto L (2014).
      "Micro_spies_from_the_brain_to_the_periphery:_new_clues_from_studies_on
      microRNAs_in_neuropsychiatric_disorders". Frontiers in Cellular
      Neuroscience. 8: 75. doi:10.3389/fncel.2014.00075. PMC 3949217.
      PMID 24653674.
  29. ^Mellios N, Sur M (2012). "The_Emerging_Role_of_microRNAs_in
      Schizophrenia_and_Autism_Spectrum_Disorders". Frontiers in Psychiatry. 3:
      39. doi:10.3389/fpsyt.2012.00039. PMC 3336189. PMID 22539927.
  30. ^Geaghan M, Cairns MJ (Aug 2015). "MicroRNA and Posttranscriptional
      Dysregulation in Psychiatry". Biological Psychiatry. 78 (4): 231â9.
      doi:10.1016/j.biopsych.2014.12.009. PMID 25636176.
  31. ^Barnett JA (2004). "A history of research on yeasts 7: enzymic
      adaptation and regulation". Yeast. 21 (9): 703â746. doi:10.1002/
      yea.1113. PMID 15282797.
  32. ^DequÃ©ant ML, PourquiÃ© O (May 2008). "Segmental patterning of the
      vertebrate embryonic axis". Nat Rev Genet. 9 (5): 370â82. doi:10.1038/
      nrg2320. PMID 18414404.
  33. ^ Gilbert SF (2003). Developmental biology, 7th ed., Sunderland, Mass:
      Sinauer Associates, 65â6.
  34. ISBN 0-87893-258-5.
  35. ^Nehlin JO, Carlberg M, Ronne H (1991). "Control_of_yeast_GAL_genes_by
      MIG1_repressor:_a_transcriptional_cascade_in_the_glucose_response". EMBO
      J. 10 (11): 3373â7. doi:10.1002/j.1460-2075.1991.tb04901.x. PMC 453065.
      PMID 1915298.
  36. ^Cheadle C, Fan J, Cho-Chung YS, Werner T, Ray J, Do L, Gorospe M, Becker
      KG (2005). "Control_of_gene_expression_during_T_cell_activation:
      alternate_regulation_of_mRNA_transcription_and_mRNA_stability". BMC
      Genomics. 6: 75. doi:10.1186/1471-2164-6-75. PMC 1156890. PMID 15907206.
  37. ^Jackson DA, Pombo A, Iborra F (Feb 2000). "The_balance_sheet_for
      transcription:_an_analysis_of_nuclear_RNA_metabolism_in_mammalian_cells".
      FASEB Journal. 14 (2): 242â54. doi:10.1096/fasebj.14.2.242.
      PMID 10657981.
  38. ^Schwanekamp JA, Sartor MA, Karyala S, Halbleib D, Medvedovic M,
      Tomlinson CR (2006). "Genome-wide analyses show that nuclear and
      cytoplasmic RNA levels are differentially affected by dioxin". Biochimica
      et Biophysica Acta. 1759 (8â9): 388â402. doi:10.1016/
      j.bbaexp.2006.07.005. PMID 16962184.
***** Bibliography[edit] *****
    * Latchman, David S. (2005). Gene_regulation:_a_eukaryotic_perspective.
      Psychology Press. ISBN 978-0-415-36510-9.
***** External links[edit] *****
    * Plant_Transcription_Factor_Database_and_Plant_Transcriptional_Regulation
      Data_and_Analysis_Platform
    * Regulation_of_Gene_Expression_(MeSH) at the US National Library of
      Medicine Medical_Subject_Headings (MeSH)
    * ChIPBase An open database for decoding the transcriptional regulatory
      networks of non-coding RNAs and protein-coding genes from ChIP-seq data.
    * v
    * t
    * e
Gene_expression
                       * Genetic_code
                       * Central_dogma
Introduction                 o DNA â RNA â Protein
to_genetics            * Special_transfers
                             o RNAâRNA
                             o RNAâDNA
                             o ProteinâProtein
                   Types                  * Bacterial
                                          * Eukaryotic
                                          * Transcription_factor
                   Key elements           * RNA_polymerase
                                          * Promoter
Transcription                             * Precursor_mRNA_(pre-mRNA /_hnRNA)
                                          * 5'_capping
                   Post-transcription     * Splicing
                                          * Polyadenylation
                                          * Histone_acetylation_and
                                            deacetylation
                   Types            * Prokaryotic
                                    * Eukaryotic
Translation                         * Ribosome
                   Key elements     * Transfer_RNA_(tRNA)
                                    * Ribosome-nascent_chain_complex_(RNC)
                                    * Post-translational_modification
                       * Epigenetic
                             o imprinting
                       * Transcriptional
                             o Gene_regulatory_network
                             o cis-regulatory_element
                       * lac_operon
Regulation             * Post-transcriptional
                             o sequestration_(P-bodies)
                             o alternative_splicing
                             o microRNA
                       * Translational
                       * Post-translational
                             o reversible
                             o irreversible
Influential people     * FranÃ§ois_Jacob
                       * Jacques_Monod
    * [Portal] Portal
    * v
    * t
    * e
The development of phenotype
                                 * Genotypeâphenotype_distinction
                                 * Norms_of_reaction
                                 * Geneâenvironment_interaction
                                 * Geneâenvironment_correlation
Key concepts                     * Operon
                                 * Heritability
                                 * Quantitative_genetics
                                 * Heterochrony
                                       o Neoteny
                                 * Heterotopy
                                 * Canalisation
                                 * Genetic_assimilation
                                 * Dominance
                                 * Epistasis
Genetic_architecture             * Fitness_landscape/evolutionary_landscape
                                 * Pleiotropy
                                 * Plasticity
                                 * Polygenic_inheritance
                                 * Transgressive_segregation
                                 * Sequence_space
                                 * Epigenetics
                                 * Maternal_effect
Non-genetic influences           * Genomic_imprinting
                                 * Dual_inheritance_theory
                                 * Polyphenism
                                 * Developmental_biology
                                 * Morphogenesis
                                       o Eyespot
Developmental architecture             o Pattern_formation
                                 * Segmentation
                                 * Metamerism
                                 * Modularity
                                 * Evolvability
Evolution of genetic systems     * Mutational_robustness
                                 * Neutral_networks
                                 * Evolution_of_sexual_reproduction
                                          * Regulation of gene expression
                                          * Gene_regulatory_network
                                          * Developmental-genetic_toolkit
                             Systems      * Evolutionary_developmental_biology
                                          * Homeobox
                                          * Hedgehog_signaling_pathway
                                          * Notch_signaling_pathway
                                          * Homeotic_gene
Control of development                    * Hox_gene
                                          * Pax_genes
                                                o eyeless_gene
                                          * Distal-less
                             Elements     * Engrailed
                                          * cis-regulatory_element
                                          * Ligand
                                          * Morphogen
                                          * Cell_surface_receptor
                                          * Transcription_factor
                                 * C._H._Waddington
                                 * Richard_Lewontin
                                 * FranÃ§ois_Jacob + Jacques_Monod
                                       o Lac_operon
Influential figures              * Eric_F._Wieschaus
                                 * Christiane_NÃ¼sslein-Volhard
                                 * William_McGinnis
                                 * Mike_Levine
                                 * Sean_B._Carroll
                                       o Endless_Forms_Most_Beautiful
Debates                          * Nature_versus_nurture
                                 * Morphogenetic_field
Index_of_evolutionary_biology_articles
    * v
    * t
    * e
Molecular_biology
    * Computational_biology
    * Developmental_biology
    * Functional_biology/medicine
                              * History_of_molecular_biology
            Central_dogma     * DNA_replication (DNA)
                              * Transcription (RNA)
                              * Translation (protein)
                              *     o Genetic
                                    o Heredity
                              * Promoter
                                    o Pribnow_box
                                    o TATA_box
                              * Operon
                                    o gal_operon
                                    o lac_operon
Overview    Element                 o trp_operon
                              * Intron
                              * Exon
                              * Terminator
                              * Enhancer
                              * Repressor
                                    o lac_repressor
                                    o trp_repressor
                              * Silencer
                              * Histone_methylation
                              * Cell_biology
            Linked life       * Biochemistry
                              * Computational_biology
                              * Genetics
                                * Mitosis
                                * Cell_signalling
            Concepts            * Post-transcriptional_modification
                                * Post-translational_modification
                                * Dry_lab / Wet_lab
                                * Cell_culture
                                * Model_organisms (such as C57BL/6_mice)
                                * Methods
Engineering                           o Nucleic_acid
            Techniques                o Protein
                                * Fluorescence, Pigment & Radioactivity
                                  DNA_microarray
                                  Mass_spectrometry
                                  Lab-on-a-chip
                                * Epigenetic
            Gene regulation     * Genetic
                                * Post-transcriptional
                                * Post-translational_regulation
    * Glossary
    * [Category]Molecular_biology
    * [Portal]Molecular_and_cellular_biology
    * [WikiProject] WikiProject
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Regulation_of_gene_expression&oldid=906704467"
Categories:
    * Gene_expression
    * DNA
    * RNA
    * Posttranslational_modification
    * Evolutionary_developmental_biology
Hidden categories:
    * CS1:_long_volume_value
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_January_2011
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * Nederlands
    * æ¥æ¬èª
    * PortuguÃªs
    * SlovenÅ¡Äina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Svenska
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
Edit_links
    * This page was last edited on 17 July 2019, at 17:07 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
