The following text has been accessed from https://en.wikipedia.org/wiki/Molecular_evolution at Fri Aug 9 04:03:15 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Molecular evolution ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
The process of change in the sequence composition of cellular molecules across
generations
 This article needs additional citations for verification. Please help improve_this
 article by adding_citations_to_reliable_sources. Unsourced material may be challenged
 and removed.
 Find sources: "Molecular_evolution" â news Â· newspapers Â· books Â· scholar Â·
 JSTOR (March 2018)(Learn_how_and_when_to_remove_this_template_message)
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
Molecular evolution is the process of change in the sequence composition of
cellular molecules such as DNA, RNA, and proteins across generations. The field
of molecular evolution uses principles of evolutionary_biology and population
genetics to explain patterns in these changes. Major topics in molecular
evolution concern the rates and impacts of single nucleotide changes, neutral
evolution vs. natural_selection, origins of new genes, the genetic nature of
complex_traits, the genetic basis of speciation, evolution of development, and
ways that evolutionary forces influence genomic and phenotypic changes.
⁰
***** Contents *****
    * 1_History
    * 2_Forces_in_molecular_evolution
          o 2.1_Mutation
          o 2.2_Recombination
                # 2.2.1_Gene_conversion
          o 2.3_Genetic_drift
          o 2.4_Selection
    * 3_Genome_architecture
          o 3.1_Genome_size
          o 3.2_Repetitive_elements
          o 3.3_Chromosome_number_and_organization
          o 3.4_Gene_content_and_distribution
          o 3.5_Organelles
    * 4_Origins_of_new_genes
    * 5_In_vitro_molecular_evolution_experiments
    * 6_Molecular_phylogenetics
    * 7_The_driving_forces_of_evolution
    * 8_Protein_evolution
          o 8.1_Relation_to_nucleic_acid_evolution
    * 9_Discordance_with_morphological_evolution
    * 10_Journals_and_societies
    * 11_See_also
    * 12_References
    * 13_Further_reading
***** History[edit] *****
Main article: History_of_molecular_evolution
The history_of_molecular_evolution starts in the early 20th century with
comparative biochemistry, and the use of "fingerprinting" methods such as
immune assays, gel_electrophoresis and paper_chromatography in the 1950s to
explore homologous_proteins.[1][2] The field of molecular evolution came into
its own in the 1960s and 1970s, following the rise of molecular_biology. The
advent of protein_sequencing allowed molecular biologists to create phylogenies
based on sequence comparison, and to use the differences between homologous
sequences as a molecular_clock to estimate the time since the last_universal
common_ancestor.[1] In the late 1960s, the neutral_theory_of_molecular
evolution provided a theoretical basis for the molecular clock,[3] though both
the clock and the neutral theory were controversial, since most evolutionary
biologists held strongly to panselectionism, with natural_selection as the only
important cause of evolutionary change. After the 1970s, nucleic acid
sequencing allowed molecular evolution to reach beyond proteins to highly
conserved ribosomal_RNA sequences, the foundation of a reconceptualization of
the early history_of_life.[1]
***** Forces in molecular evolution[edit] *****
The content and structure of a genome is the product of the molecular and
population genetic forces which act upon that genome. Novel genetic variants
will arise through mutation and will spread and be maintained in populations
due to genetic_drift or natural_selection.
**** Mutation[edit] ****
Main article: Mutation
This hedgehog has no pigmentation due to a mutation.
Mutations are permanent, transmissible changes to the genetic_material (DNA or
RNA) of a cell or virus. Mutations result from errors in DNA_replication during
cell_division and by exposure to radiation, chemicals, and other environmental
stressors, or viruses and transposable_elements. Most mutations that occur are
single_nucleotide_polymorphisms which modify single bases of the DNA sequence,
resulting in point_mutations. Other types of mutations modify larger segments
of DNA and can cause duplications, insertions, deletions, inversions, and
translocations.
Most organisms display a strong bias in the types of mutations that occur with
strong influence in GC-content. Transitions (A â G or C â T) are more
common than transversions (purine (adenine or guanine)) â pyrimidine
(cytosine or thymine, or in RNA, uracil))[4] and are less likely to alter amino
acid sequences of proteins.
Mutations are stochastic and typically occur randomly across genes. Mutation
rates for single nucleotide sites for most organisms are very low, roughly
10â9 to 10â8 per site per generation, though some viruses have higher
mutation rates on the order of 10â6 per site per generation. Among these
mutations, some will be neutral or beneficial and will remain in the genome
unless lost via genetic_drift, and others will be detrimental and will be
eliminated from the genome by natural_selection.
Because mutations are extremely rare, they accumulate very slowly across
generations. While the number of mutations which appears in any single
generation may vary, over very long time periods they will appear to accumulate
at a regular pace. Using the mutation rate per generation and the number of
nucleotide differences between two sequences, divergence times can be estimated
effectively via the molecular_clock.
**** Recombination[edit] ****
Further information: Genetic_recombination
Recombination involves the breakage and rejoining of two chromosomes (M and F)
to produce two re-arranged chromosomes (C1 and C2).
Recombination is a process that results in genetic exchange between chromosomes
or chromosomal regions. Recombination counteracts physical linkage between
adjacent genes, thereby reducing genetic_hitchhiking. The resulting independent
inheritance of genes results in more efficient selection, meaning that regions
with higher recombination will harbor fewer detrimental mutations, more
selectively favored variants, and fewer errors in replication and repair.
Recombination can also generate particular types of mutations if chromosomes
are misaligned.
*** Gene conversion[edit] ***
Gene_conversion is a type of recombination that is the product of DNA_repair
where nucleotide damage is corrected using an homologous genomic region as a
template. Damaged bases are first excised, the damaged strand is then aligned
with an undamaged homolog, and DNA synthesis repairs the excised region using
the undamaged strand as a guide. Gene conversion is often responsible for
homogenizing sequences of duplicate genes over long time periods, reducing
nucleotide divergence.
**** Genetic drift[edit] ****
Genetic_drift is the change of allele frequencies from one generation to the
next due to stochastic effects of random_sampling in finite populations. Some
existing variants have no effect on fitness and may increase or decrease in
frequency simply due to chance. "Nearly neutral" variants whose selection
coefficient is close to a threshold value of 1 / the effective_population_size
will also be affected by chance as well as by selection and mutation. Many
genomic features have been ascribed to accumulation of nearly neutral
detrimental mutations as a result of small effective population sizes.[5] With
a smaller effective population size, a larger variety of mutations will behave
as if they are neutral due to inefficiency of selection.
**** Selection[edit] ****
Selection occurs when organisms with greater fitness, i.e. greater ability to
survive or reproduce, are favored in subsequent generations, thereby increasing
the instance of underlying genetic variants in a population. Selection can be
the product of natural selection, artificial selection, or sexual selection.
Natural_selection is any selective process that occurs due to the fitness of an
organism to its environment. In contrast sexual_selection is a product of mate
choice and can favor the spread of genetic variants which act counter to
natural selection but increase desirability to the opposite sex or increase
mating success. Artificial_selection, also known as selective breeding, is
imposed by an outside entity, typically humans, in order to increase the
frequency of desired traits.
The principles of population genetics apply similarly to all types of
selection, though in fact each may produce distinct effects due to clustering
of genes with different functions in different parts of the genome, or due to
different properties of genes in particular functional classes. For instance,
sexual selection could be more likely to affect molecular evolution of the sex
chromosomes due to clustering of sex specific genes on the X, Y, Z or W.
Selection can operate at the gene level at the expense of organismal fitness,
resulting in a selective advantage for selfish_genetic_elements in spite of a
host cost. Examples of such selfish elements include transposable elements,
meiotic drivers, killer X chromosomes, selfish mitochondria, and self-
propagating introns. (See Intragenomic_conflict.)
***** Genome architecture[edit] *****
Main article: Genome_evolution
**** Genome size[edit] ****
Genome size is influenced by the amount of repetitive DNA as well as number of
genes in an organism. The C-value_paradox refers to the lack of correlation
between organism 'complexity' and genome size. Explanations for the so-called
paradox are two-fold. First, repetitive genetic elements can comprise large
portions of the genome for many organisms, thereby inflating DNA content of the
haploid genome. Secondly, the number of genes is not necessarily indicative of
the number of developmental stages or tissue types in an organism. An organism
with few developmental stages or tissue types may have large numbers of genes
that influence non-developmental phenotypes, inflating gene content relative to
developmental gene families.
Neutral explanations for genome size suggest that when population sizes are
small, many mutations become nearly neutral. Hence, in small populations
repetitive content and other 'junk'_DNA can accumulate without placing the
organism at a competitive disadvantage. There is little evidence to suggest
that genome size is under strong widespread selection in multicellular
eukaryotes. Genome size, independent of gene content, correlates poorly with
most physiological traits and many eukaryotes, including mammals, harbor very
large amounts of repetitive DNA.
However, birds likely have experienced strong selection for reduced genome
size, in response to changing energetic needs for flight. Birds, unlike humans,
produce nucleated red blood cells, and larger nuclei lead to lower levels of
oxygen transport. Bird metabolism is far higher than that of mammals, due
largely to flight, and oxygen needs are high. Hence, most birds have small,
compact genomes with few repetitive elements. Indirect evidence suggests that
non-avian theropod dinosaur ancestors of modern birds [6] also had reduced
genome sizes, consistent with endothermy and high energetic needs for running
speed. Many bacteria have also experienced selection for small genome size, as
time of replication and energy consumption are so tightly correlated with
fitness.
**** Repetitive elements[edit] ****
Transposable_elements are self-replicating, selfish genetic elements which are
capable of proliferating within host genomes. Many transposable elements are
related to viruses, and share several proteins in common.
**** Chromosome number and organization[edit] ****
The number_of_chromosomes in an organism's genome also does not necessarily
correlate with the amount of DNA in its genome. The ant Myrmecia pilosula has
only a single pair of chromosomes[7] whereas the Adders-tongue fern
Ophioglossum reticulatum has up to 1260 chromosomes.[8] Cilliate genomes house
each gene in individual chromosomes, resulting in a genome which is not
physically linked. Reduced linkage through creation of additional chromosomes
should effectively increase the efficiency of selection.
Changes in chromosome number can play a key role in speciation, as differing
chromosome numbers can serve as a barrier to reproduction in hybrids. Human
chromosome_2 was created from a fusion of two chimpanzee chromosomes and still
contains central telomeres as well as a vestigial second centromere.
Polyploidy, especially allopolyploidy, which occurs often in plants, can also
result in reproductive incompatibilities with parental species. Agrodiatus blue
butterflies have diverse chromosome numbers ranging from n=10 to n=134 and
additionally have one of the highest rates of speciation identified to date.[9]
**** Gene content and distribution[edit] ****
Different organisms house different numbers of genes within their genomes as
well as different patterns in the distribution of genes throughout the genome.
Some organisms, such as most bacteria, Drosophila, and Arabidopsis have
particularly compact genomes with little repetitive content or non-coding DNA.
Other organisms, like mammals or maize, have large amounts of repetitive DNA,
long introns, and substantial spacing between different genes. The content and
distribution of genes within the genome can influence the rate at which certain
types of mutations occur and can influence the subsequent evolution of
different species. Genes with longer introns are more likely to recombine due
to increased physical distance over the coding sequence. As such, long introns
may facilitate ectopic_recombination, and result in higher rates of new gene
formation.
**** Organelles[edit] ****
In addition to the nuclear genome, endosymbiont organelles contain their own
genetic material typically as circular plasmids. Mitochondrial and chloroplast
DNA varies across taxa, but membrane-bound proteins, especially electron
transport chain constituents are most often encoded in the organelle.
Chloroplasts and mitochondria are maternally inherited in most species, as the
organelles must pass through the egg. In a rare departure, some species of
mussels are known to inherit mitochondria from father to son.
***** Origins of new genes[edit] *****
New genes arise from several different genetic mechanisms including gene
duplication, de novo origination, retrotransposition, chimeric gene formation,
recruitment of non-coding sequence, and gene truncation.
Gene_duplication initially leads to redundancy. However, duplicated gene
sequences can mutate to develop new functions or specialize so that the new
gene performs a subset of the original ancestral functions. In addition to
duplicating whole genes, sometimes only a domain or part of a protein is
duplicated so that the resulting gene is an elongated version of the parental
gene.
Retrotransposition creates new genes by copying mRNA to DNA and inserting it
into the genome. Retrogenes often insert into new genomic locations, and often
develop new expression patterns and functions.
Chimeric_genes form when duplication, deletion, or incomplete
retrotransposition combine portions of two different coding sequences to
produce a novel gene sequence. Chimeras often cause regulatory changes and can
shuffle protein domains to produce novel adaptive functions.
De_novo_gene_birth can also give rise to new genes from previously non-coding
DNA.[10] For instance, Levine and colleagues reported the origin of five new
genes in the D. melanogaster genome from noncoding DNA.[11][12] Similar de novo
origin of genes has been also shown in other organisms such as yeast,[13] rice
[14] and humans.[15] De novo genes may evolve from transcripts that are already
expressed at low levels.[16] Mutation of a stop_codon to a regular codon or a
frameshift may cause an extended protein that includes a previously non-coding
sequence. The formation of novel genes from scratch typically can not occur
within genomic regions of high gene density. The essential events for de novo
formation of genes is recombination/mutation which includes insertions,
deletions, and inversions. These events are tolerated if the consequence of
these genetic events does not interfere in cellular activities. Most genomes
comprise prophages wherein genetic modifications do not, in general, affect the
host genome propagation. Hence, there is higher probability of genetic
modifications, in regions such as prophages, which is proportional to the
probability of de novo formation of genes.[17]
De novo evolution of genes can also be simulated in the laboratory. For
example, semi-random gene sequences can be selected for specific functions.[18]
More specifically, they selected sequences from a library that could complement
a gene_deletion in E._coli. The deleted gene encodes ferric enterobactin
esterase (Fes), which releases iron from an iron chelator, enterobactin. While
Fes is a 400 amino_acid protein, the newly selected gene was only 100 amino
acids in length and unrelated in sequence to Fes.[18]
***** In vitro molecular evolution experiments[edit] *****
Principles of molecular evolution have also been discovered, and others
elucidated and tested using experimentation involving amplification, variation
and selection of rapidly proliferating and genetically varying molecular
species outside cells. Since the pioneering work of Sol Spiegelmann in 1967
[ref], involving RNA that replicates itself with the aid of an enzyme extracted
from the QÃ virus [ref], several groups (such as Kramers [ref] and Biebricher/
Luce/Eigen [ref]) studied mini and micro variants of this RNA in the 1970s and
1980s that replicate on the timescale of seconds to a minute, allowing hundreds
of generations with large population sizes (e.g. 10^14 sequences) to be
followed in a single day of experimentation. The chemical kinetic elucidation
of the detailed mechanism of replication [ref, ref] meant that this type of
system was the first molecular evolution system that could be fully
characterised on the basis of physical chemical kinetics, later allowing the
first models of the genotype to phenotype map based on sequence dependent RNA
folding and refolding to be produced [ref, ref]. Subject to maintaining the
function of the multicomponent QÃ enzyme, chemical conditions could be varied
significantly, in order to study the influence of changing environments and
selection pressures [ref]. Experiments with in vitro RNA quasi species included
the characterisation of the error threshold for information in molecular
evolution [ref], the discovery of de novo evolution [ref] leading to diverse
replicating RNA species and the discovery of spatial travelling waves as ideal
molecular evolution reactors [ref, ref]. Later experiments employed novel
combinations of enzymes to elucidate novel aspects of interacting molecular
evolution involving population dependent fitness, including work with
artificially designed molecular predator prey and cooperative systems of
multiple RNA and DNA [ref, ref]. Special evolution reactors were designed for
these studies, starting with serial transfer machines, flow reactors such as
cell-stat machines, capillary reactors, and microreactors including line flow
reactors and gel slice reactors. These studies were accompanied by theoretical
developments and simulations involving RNA folding and replication kinetics
that elucidated the importance of the correlation structure between distance in
sequence space and fitness changes [ref], including the role of neutral
networks and structural ensembles in evolutionary optimisation.
***** Molecular phylogenetics[edit] *****
Main articles: Molecular_systematics and Phylogenetics
Molecular systematics is the product of the traditional fields of systematics
and molecular_genetics. It uses DNA, RNA, or protein sequences to resolve
questions in systematics, i.e. about their correct scientific_classification or
taxonomy from the point of view of evolutionary_biology.
Molecular systematics has been made possible by the availability of techniques
for DNA_sequencing, which allow the determination of the exact sequence of
nucleotides or bases in either DNA or RNA. At present it is still a long and
expensive process to sequence the entire genome of an organism, and this has
been done for only a few species. However, it is quite feasible to determine
the sequence of a defined area of a particular chromosome. Typical molecular
systematic analyses require the sequencing of around 1000 base_pairs.
***** The driving forces of evolution[edit] *****
Main articles: Neutral_theory_of_molecular_evolution, Modern_synthesis_(20th
century), and Mutationism
Depending on the relative importance assigned to the various forces of
evolution, three perspectives provide evolutionary explanations for molecular
evolution.[19][20]
Selectionist hypotheses argue that selection is the driving force of molecular
evolution. While acknowledging that many mutations are neutral, selectionists
attribute changes in the frequencies of neutral alleles to linkage
disequilibrium with other loci that are under selection, rather than to random
genetic_drift.[21] Biases in codon_usage are usually explained with reference
to the ability of even weak_selection to shape molecular evolution.[22]
Neutralist hypotheses emphasize the importance of mutation, purifying
selection, and random genetic drift.[23] The introduction of the neutral theory
by Kimura,[24] quickly followed by King and Jukes' own findings,[25] led to a
fierce debate about the relevance of neodarwinism at the molecular level. The
Neutral_theory_of_molecular_evolution proposes that most mutations in DNA are
at locations not important to function or fitness. These neutral changes drift
towards fixation within a population. Positive changes will be very rare, and
so will not greatly contribute to DNA polymorphisms.[26] Deleterious mutations
do not contribute much to DNA diversity because they negatively affect fitness
and so are removed from the gene pool before long.[27] This theory provides a
framework for the molecular clock.[26] The fate of neutral mutations are
governed by genetic_drift, and contribute to both nucleotide polymorphism and
fixed differences between species.[28][29]
In the strictest sense, the neutral theory is not accurate.[30] Subtle changes
in DNA very often have effects, but sometimes these effects are too small for
natural selection to act on.[30] Even synonymous mutations are not necessarily
neutral [30] because there is not a uniform amount of each codon. The nearly
neutral theory expanded the neutralist perspective, suggesting that several
mutations are nearly neutral, which means both random drift and natural
selection is relevant to their dynamics.[30] The main difference between the
neutral theory and nearly neutral theory is that the latter focuses on weak
selection, not strictly neutral.[27]
Mutationists hypotheses emphasize random drift and biases in mutation patterns.
[31] Sueoka was the first to propose a modern mutationist view. He proposed
that the variation in GC_content was not the result of positive selection, but
a consequence of the GC mutational pressure.[32]
***** Protein evolution[edit] *****
This chart compares the sequence identity of different lipase proteins
throughout the human body. It demonstrates how proteins evolve, keeping some
regions conserved while others change dramatically.
Evolution of proteins is studied by comparing the sequences and structures of
proteins from many organisms representing distinct evolutionary clades. If the
sequences/structures of two proteins are similar indicating that the proteins
diverged from a common origin, these proteins are called as homologous
proteins. More specifically, homologous proteins that exist in two distinct
species are called as orthologs. Whereas, homologous proteins encoded by the
genome of a single species are called paralogs.
The phylogenetic relationships of proteins are examined by multiple sequence
comparisons. Phylogenetic trees of proteins can be established by the
comparison of sequence identities among proteins. Such phylogenetic trees have
established that the sequence similarities among proteins reflect closely the
evolutionary relationships among organisms.[33][34]
Protein evolution describes the changes over time in protein shape, function,
and composition. Through quantitative analysis and experimentation, scientists
have strived to understand the rate and causes of protein evolution. Using the
amino acid sequences of hemoglobin and cytochrome c from multiple species,
scientists were able to derive estimations of protein evolution rates. What
they found was that the rates were not the same among proteins.[27] Each
protein has its own rate, and that rate is constant across phylogenies (i.e.,
hemoglobin does not evolve at the same rate as cytochrome c, but hemoglobins
from humans, mice, etc. do have comparable rates of evolution.). Not all
regions within a protein mutate at the same rate; functionally important areas
mutate more slowly and amino acid substitutions involving similar amino acids
occurs more often than dissimilar substitutions.[27] Overall, the level of
polymorphisms in proteins seems to be fairly constant. Several species
(including humans, fruit flies, and mice) have similar levels of protein
polymorphism.[26]
**** Relation to nucleic acid evolution[edit] ****
Protein evolution is inescapably tied to changes and selection of DNA
polymorphisms and mutations because protein sequences change in response to
alterations in the DNA sequence. Amino acid sequences and nucleic acid
sequences do not mutate at the same rate. Due to the degenerate nature of DNA,
bases can change without affecting the amino acid sequence. For example, there
are six codons that code for leucine. Thus, despite the difference in mutation
rates, it is essential to incorporate nucleic acid evolution into the
discussion of protein evolution. At the end of the 1960s, two groups of
scientistsâKimura (1968) and King and Jukes (1969)âindependently proposed
that a majority of the evolutionary changes observed in proteins were neutral.
[26][27] Since then, the neutral theory has been expanded upon and debated.[27]
***** Discordance with morphological evolution[edit] *****
There are sometimes discordances between molecular and morphological evolution,
which are reflected in molecular and morphological systematic studies,
especially of bacteria, archaea and eukaryotic microbes. These discordances can
be categorized as two types: (i) one morphology, multiple lineages (e.g.
morphological_convergence, cryptic_species) and (ii) one lineage, multiple
morphologies (e.g. phenotypic_plasticity, multiple life-cycle stages). Neutral
evolution possibly could explain the incongruences in some cases.[35]
***** Journals and societies[edit] *****
The Society_for_Molecular_Biology_and_Evolution publishes the journals
"Molecular Biology and Evolution" and "Genome Biology and Evolution" and holds
an annual international meeting. Other journals dedicated to molecular
evolution include Journal of Molecular Evolution and Molecular Phylogenetics
and Evolution. Research in molecular evolution is also published in journals of
genetics, molecular_biology, genomics, systematics, and evolutionary_biology.
***** See also[edit] *****
    * [icon]Evolutionary_biology_portal
    * Molecular_and_cellular_biology_portal
    * Abiogenesis
    * Adaptor_protein_evolution
    * Comparative_phylogenetics
    * Evolution
    * E._coli_long-term_evolution_experiment
    * Evolutionary_physiology
    * Evolution_of_dietary_antioxidants
    * Genomic_organization
    * Genetic_drift
    * Genome_evolution
    * Heterotachy
    * History_of_molecular_evolution
    * Horizontal_gene_transfer
    * Human_evolution
    * Molecular_clock
    * Molecular_paleontology
    * Neutral_theory_of_molecular_evolution
    * Nucleotide_diversity
    * Parsimony
    * Population_genetics
    * Selection
***** References[edit] *****
   1. ^ a b cDietrich, Michael R. (1998). "Paradox and Persuasion: Negotiating
      the Place of Molecular Evolution within Evolutionary Biology". Journal of
      the History of Biology. 31 (1): 85â111. doi:10.1023/A:1004257523100.
      PMID 11619919.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Hagen, Joel B. (1999). "Naturalists, Molecular Biologists, and the
      Challenge of Molecular Evolution". Journal of the History of Biology. 32
      (2): 321â341. doi:10.1023/A:1004660202226. PMID 11624208.
   4. ^King, Jack L.; Jukes, Thomas (1969). "Non-Darwinian Evolution". Science.
      164 (3881): 788â798. Bibcode:1969Sci...164..788L. doi:10.1126/
      science.164.3881.788. PMID 5767777.
   5. ^"Transitions_vs_transversions".
   6. ^Lynch,_M. (2007). The Origins of Genome Architecture. Sinauer. ISBN 978-
      0-87893-484-3.
   7. ^Organ, C. L.; Shedlock, A. M.; Meade, A.; Pagel, M.; Edwards, S. V.
      (2007). "Origin of avian genome size and structure in nonavian
      dinosaurs". Nature. 446 (7132): 180â184. Bibcode:2007Natur.446..180O.
      doi:10.1038/nature05621. PMID 17344851.
   8. ^Crosland MW, Crozier RH (1986). "Myrmecia pilosula, an ant with only one
      pair of chromosomes". Science. 231 (4743): 1278. Bibcode:
      1986Sci...231.1278C. doi:10.1126/science.231.4743.1278. PMID 17839565.
   9. ^Gerardus J. H. Grubben (2004). Vegetables. PROTA. p. 404. ISBN 978-90-
      5782-147-9. Retrieved 10 March 2013.
  10. ^Nikolai P. Kandul; Vladimir A. Lukhtanov; Naomi E. Pierce (2007),
      "KARYOTYPIC DIVERSITY AND SPECIATION IN AGRODIAETUS BUTTERFLIES",
      Evolution, 61 (3): 546â559, doi:10.1111/j.1558-5646.2007.00046.x,
      PMID 17348919
  11. ^McLysaght, Aoife; Guerzoni, Daniele (31 August 2015). "New_genes_from
      non-coding_sequence:_the_role_of_de_novo_protein-coding_genes_in
      eukaryotic_evolutionary_innovation". Philosophical Transactions of the
      Royal Society B: Biological Sciences. 370 (1678): 20140332. doi:10.1098/
      rstb.2014.0332. PMC 4571571. PMID 26323763.
  12. ^Levine MT, Jones CD, Kern AD, et al. (2006). "Novel_genes_derived_from
      noncoding_DNA_in_Drosophila_melanogaster_are_frequently_X-linked_and
      exhibit_testis-biased_expression". Proc Natl Acad Sci USA. 103 (26):
      9935â9939. Bibcode:2006PNAS..103.9935L. doi:10.1073/pnas.0509809103.
      PMC 1502557. PMID 16777968.
  13. ^Zhou Q, Zhang G, Zhang Y, et al. (2008). "On_the_origin_of_new_genes_in
      Drosophila". Genome Res. 18 (9): 1446â1455. doi:10.1101/gr.076588.108.
      PMC 2527705. PMID 18550802.
  14. ^Cai J, Zhao R, Jiang H, et al. (2008). "De_novo_origination_of_a_new
      protein-coding_gene_in_Saccharomyces_cerevisiae". Genetics. 179 (1):
      487â496. doi:10.1534/genetics.107.084491. PMC 2390625. PMID 18493065.
  15. ^Xiao W, Liu H, Li Y, et al. (2009). El-Shemy HA (ed.). "A_rice_gene_of
      de_novo_origin_negatively_regulates_pathogen-_induced_defense_response".
      PLoS ONE. 4 (2): e4603. Bibcode:2009PLoSO...4.4603X. doi:10.1371/
      journal.pone.0004603. PMC 2643483. PMID 19240804.
  16. ^Knowles DG, McLysaght A (2009). "Recent_de_novo_origin_of_human_protein-
      coding_genes". Genome Res. 19 (10): 1752â1759. doi:10.1101/
      gr.095026.109. PMC 2765279. PMID 19726446.
  17. ^Wilson, Ben A.; Joanna Masel (2011). "Putatively_Noncoding_Transcripts
      Show_Extensive_Association_with_Ribosomes". Genome Biology and Evolution.
      3: 1245â1252. doi:10.1093/gbe/evr099. PMC 3209793. PMID 21948395.
  18. ^Ramisetty, Bhaskar Chandra Mohan; Sudhakari, Pavithra Anantharaman
      (2019). "Bacterial_'Grounded'_Prophages:_Hotspots_for_Genetic_Renovation
      and_Innovation". Frontiers in Genetics. 10: 65. doi:10.3389/
      fgene.2019.00065. ISSN 1664-8021. PMC 6379469. PMID 30809245.
  19. ^ a bDonnelly, Ann E.; Murphy, Grant S.; Digianantonio, Katherine M.;
      Hecht, Michael H. (March 2018). "A de novo enzyme catalyzes a life-
      sustaining reaction in Escherichia coli". Nature Chemical Biology. 14
      (3): 253â255. doi:10.1038/nchembio.2550. ISSN 1552-4469. PMID 29334382.
  20. ^Graur, D. & Li, W.-H. (2000). Fundamentals of molecular evolution.
      Sinauer. ISBN 0-87893-266-6.
  21. ^Casillas, SÃ²nia; Barbadilla, Antonio (2017). "Molecular_Population
      Genetics". Genetics. 205 (3): 1003â1035. doi:10.1534/
      genetics.116.196493. PMC 5340319. PMID 28270526.
  22. ^Hahn, Matthew W. (February 2008). "Toward A Selection Theory Of
      Molecular Evolution". Evolution. 62 (2): 255â265. doi:10.1111/j.1558-
      5646.2007.00308.x. PMID 18302709.
  23. ^Hershberg, Ruth; Petrov, Dmitri A. (December 2008). "Selection on Codon
      Bias". Annual Review of Genetics. 42 (1): 287â299. doi:10.1146/
      annurev.genet.42.110807.091442. PMID 18983258.
  24. ^Kimura,_M. (1983). The Neutral Theory of Molecular Evolution. Cambridge
      University_Press, Cambridge. ISBN 0-521-23109-4.
  25. ^Kimura, Motoo (1968). "Evolutionary_rate_at_the_molecular_level" (PDF).
      Nature. 217 (5129): 624â626. Bibcode:1968Natur.217..624K. doi:10.1038/
      217624a0. PMID 5637732.
  26. ^King, J.L. & Jukes, T.H. (1969). "Non-Darwinian_Evolution" (PDF).
      Science. 164 (3881): 788â798. Bibcode:1969Sci...164..788L. doi:10.1126/
      science.164.3881.788. PMID 5767777.
  27. ^ a b c dAkashi, H (2012). "Weak_Selection_and_Protein_Evolution".
      Genetics. 192 (1): 15â31. doi:10.1534/genetics.112.140178. PMC 3430532.
      PMID 22964835.
  28. ^ a b c d e fFay, JC, Wu, CI (2003). "Sequence divergence, functional
      constraint, and selection in protein evolution". Annu. Rev. Genom. Hum.
      Genet. 4: 213â35. doi:10.1146/annurev.genom.4.020303.162528.
      PMID 14527302.
  29. ^Nachman M. (2006). C.W. Fox; J.B. Wolf (eds.). ""Detecting selection at
      the molecular level" in: Evolutionary Genetics: concepts and case
      studies": 103â118.
  30. ^ The nearly neutral theory expanded the neutralist perspective,
      suggesting that several mutations are nearly neutral, which means both
      random drift and natural selection is relevant to their dynamics.
  31. ^ a b c dOhta, T (1992). "The Nearly Neutral Theory of Molecular
      Evolution". Annual Review of Ecology and Systematics. 23 (1): 263â286.
      doi:10.1146/annurev.es.23.110192.001403. ISSN 0066-4162.
  32. ^Nei,_M. (2005). "Selectionism_and_Neutralism_in_Molecular_Evolution".
      Molecular Biology and Evolution. 22 (12): 2318â2342. doi:10.1093/
      molbev/msi242. PMC 1513187. PMID 16120807.
  33. ^Sueoka, N. (1964). "On the evolution of informational macromolecules".
      In Bryson, V.; Vogel, H.J. (eds.). Evolving genes and proteins. Academic
      Press, New-York. pp. 479â496.
  34. ^Hanukoglu I (2017). "ASIC and ENaC type sodium channels: Conformational
      states and the structures of the ion selectivity filters". FEBS Journal.
      284 (4): 525â545. doi:10.1111/febs.13840. PMID 27580245.
  35. ^Hanukoglu I, Hanukoglu A (Jan 2016). "Epithelial_sodium_channel_(ENaC)
      family:_Phylogeny,_structure-function,_tissue_distribution,_and
      associated_inherited_diseases". Gene. 579 (2): 95â132. doi:10.1016/
      j.gene.2015.12.061. PMC 4756657. PMID 26772908.
  36. ^Lahr, D. J.; Laughinghouse, H. D.; Oliverio, A. M.; Gao, F.; Katz, L. A.
      (2014). "How_discordant_morphological_and_molecular_evolution_among
      microorganisms_can_revise_our_notions_of_biodiversity_on_Earth".
      BioEssays. 36 (10): 950â959. doi:10.1002/bies.201400056. PMC 4288574.
      PMID 25156897.
***** Further reading[edit] *****
    * Li,_W.-H. (2006). Molecular Evolution. Sinauer. ISBN 0-87893-480-4.
Lynch,_M. (2007). The Origins of Genome Architecture. Sinauer. ISBN 978-0-
87893-484-3.
A. Meyer (Editor), Y. van de Peer, "Genome Evolution: Gene and Genome
Duplications and the Origin of Novel Gene Functions", 2003,
ISBN 978-1-4020-1021-7
T. Ryan Gregory, "The Evolution of the Genome", 2004, YSBN 978-0123014634
    * v
    * t
    * e
Evolutionary_biology
    * Introduction
    * Outline
    * Timeline_of_evolution
    * Evolutionary_history_of_life
    * Index
                        * Abiogenesis
                        * Adaptation
                        * Adaptive_radiation
                        * Cladistics
                        * Coevolution
                        * Common_descent
                        * Convergence
                        * Divergence
                        * Earliest_known_life
                          forms
                        * Evidence_of_evolution
Evolution               * Extinction
                              o Event
                        * Gene-centered_view
                        * Homology
                        * Last_universal_common
                          ancestor
                        * Macroevolution
                        * Microevolution
                        * Origin_of_life
                        * Panspermia
                        * Parallel_evolution
                        * Speciation
                        * Taxonomy
                        * Biodiversity
                        * Gene_flow
Population_genetics     * Genetic_drift
                        * Mutation
                        * Natural_selection
                        * Variation
                        * Canalisation
                        * Evolutionary
Development               developmental_biology
                        * Inversion
                        * Modularity
                        * Phenotypic_plasticity
                        * Birds
                              o origin
                        * Brachiopods
                        * Molluscs
                              o Cephalopods
                        * Dinosaurs
                        * Fish
                        * Fungi
                        * Insects
                              o butterflies
                        * Life
                        * Mammals
                              o cats
                              o canids
                                    # wolves
Of taxa                             # dogs
                              o hyenas
                              o dolphins_and
                                whales
                              o horses
                              o Kangaroos
                              o primates
                                    # humans
                                    # lemurs
                              o sea_cows
                        * Plants
                        * Reptiles
                        * Spiders
                        * Tetrapods
                        * Viruses
                              o influenza
                        * Cell
                        * DNA
                        * Flagella
                        * Eukaryotes
                              o symbiogenesis
                              o chromosome
                              o endomembrane
                                system
Of organs                     o mitochondria
                              o nucleus
                              o plastids
                        * In animals
                              o eye
                              o hair
                              o auditory_ossicle
                              o nervous_system    [Common-Avian-Infection-
                              o brain             Plagued-the-Tyrant-Dinosaurs-
                        * Aging                   pone.0007288.g004.jpg]
                              o Death                 * [icon]Evolutionary
                              o Programmed_cell         biology_portal
                                death
                        * Avian_flight
                        * Biological_complexity
                        * Cooperation
                        * Color_vision
                              o in_primates
                        * Emotion
                        * Empathy
                        * Ethics
                        * Eusociality
                        * Immune_system
Of processes            * Metabolism
                        * Monogamy
                        * Morality
                        * Mosaic_evolution
                        * Multicellularity
                        * Sexual_reproduction
                              o Gamete
                                differentiation/
                                sexes
                              o Life_cycles/
                                nuclear_phases
                              o Mating_types
                              o Meiosis
                              o Sex-determination
                        * Snake_venom
                        * Gradualism/Punctuated
                          equilibrium/
                          Saltationism
Tempo_and_modes         * Micromutation/
                          Macromutation
                        * Uniformitarianism/
                          Catastrophism
                        * Allopatric
                        * Anagenesis
                        * Catagenesis
                        * Cladogenesis
                        * Cospeciation
Speciation              * Ecological
                        * Hybrid
                        * Parapatric
                        * Peripatric
                        * Reinforcement
                        * Sympatric
                        * Renaissance_and
                          Enlightenment
                        * Transmutation_of
                          species
                        * Charles_Darwin
                              o On_the_Origin_of
                                Species
                        * History_of_paleontology
History                 * Transitional_fossil
                        * Blending_inheritance
                        * Mendelian_inheritance
                        * The_eclipse_of
                          Darwinism
                        * Modern_synthesis
                        * History_of_molecular
                          evolution
                        * Extended_evolutionary
                          synthesis
                        * Darwinism
                        * Alternatives
                              o Catastrophism
                              o Lamarckism
                              o Orthogenesis
Philosophy                    o Mutationism
                              o Saltationism
                              o Structuralism
                                    # Spandrel
                              o Theistic
                              o Vitalism
                        * Teleology_in_biology
                        * Biogeography
                        * Ecological_genetics
                        * Molecular evolution
Related                 * Phylogenetics
                              o Tree
                        * Polymorphism
                        * Protocell
                        * Systematics
    * [Category] Category
    * [Portal] Portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Molecular_evolution&oldid=907209073"
Categories:
    * Molecular_evolution
Hidden categories:
    * Articles_with_short_description
    * Articles_needing_additional_references_from_March_2018
    * All_articles_needing_additional_references
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * Deutsch
    * EspaÃ±ol
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * Magyar
    * æ¥æ¬èª
    * Polski
    * PortuguÃªs
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Simple_English
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Suomi
    * Svenska
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * ä¸­æ
Edit_links
    * This page was last edited on 21 July 2019, at 08:45 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
