The following text has been accessed from https://en.wikipedia.org/wiki/Biosynthesis at Thu Aug 8 23:57:20 IST 2019
Creative_Commons_Attribution-ShareAlike_License




















****** Biosynthesis ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Biosynthesis is a multi-step, enzyme-catalyzed process where substrates are
converted into more complex products in living organisms. In biosynthesis,
simple compounds are modified, converted into other compounds, or joined
together to form macromolecules. This process often consists of metabolic
pathways. Some of these biosynthetic pathways are located within a single
cellular organelle, while others involve enzymes that are located within
multiple cellular organelles. Examples of these biosynthetic pathways include
the production of lipid_membrane components and nucleotides. Biosynthesis is
usually synonymous with anabolism.
The prerequisite elements for biosynthesis include: precursor compounds,
chemical_energy (e.g. ATP), and catalytic enzymes which may require coenzymes
(e.g.NADH, NADPH). These elements create monomers, the building blocks for
macromolecules. Some important biological macromolecules include: proteins,
which are composed of amino_acid monomers joined via peptide_bonds, and DNA
molecules, which are composed of nucleotides joined via phosphodiester_bonds.
⁰
***** Contents *****
    * 1_Properties_of_chemical_reactions
    * 2_Lipid
          o 2.1_Phospholipids
          o 2.2_Sphingolipids
          o 2.3_Cholesterol
    * 3_Nucleotides
          o 3.1_Purine_nucleotides
          o 3.2_Pyrimidine_nucleotides
    * 4_DNA
    * 5_Amino_acids
          o 5.1_Amino_acid_basic_structure
          o 5.2_Nitrogen_source
          o 5.3_The_glutamate_family_of_amino_acids
          o 5.4_The_serine_family_of_amino_acids
          o 5.5_The_aspartate_family_of_amino_acids
    * 6_Proteins
          o 6.1_Additional_background
          o 6.2_Translation_in_steps
                # 6.2.1_Step_1:_Initiation
                # 6.2.2_Step_2:_Elongation
                # 6.2.3_Step_3:_Termination
    * 7_Diseases_associated_with_macromolecule_deficiency
    * 8_See_also
    * 9_References
***** Properties of chemical reactions[edit] *****
Biosynthesis occurs due to a series of chemical reactions. For these reactions
to take place, the following elements are necessary:[1]
    * Precursor_compounds: these compounds are the starting molecules or
      substrates in a reaction. These may also be viewed as the reactants in a
      given chemical process.
    * Chemical_energy: chemical energy can be found in the form of high energy
      molecules. These molecules are required for energetically unfavorable
      reactions. Furthermore, the hydrolysis of these compounds drives a
      reaction forward. High energy molecules, such as ATP, have three
      phosphates. Often, the terminal phosphate is split off during hydrolysis
      and transferred to another molecule.
    * Catalytic_enzymes: these molecules are special proteins that catalyze a
      reaction by increasing the rate_of_the_reaction and lowering the
      activation_energy.
    * Coenzymes or cofactors: cofactors are molecules that assist in chemical
      reactions. These may be metal_ions, vitamin derivatives such as NADH and
      acetyl_CoA, or non-vitamin derivatives such as ATP. In the case of NADH,
      the molecule transfers a hydrogen, whereas acetyl CoA transfers an acetyl
      group, and ATP transfers a phosphate.
In the simplest sense, the reactions that occur in biosynthesis have the
following format:[2]
                Reactant   &#x2192;  e n z y m e     Product    {\displaystyle
            {\ce {Reactant ->[][enzyme] Product}}}  [{\displaystyle {\ce
            {Reactant ->[][enzyme] Product}}}]
Some variations of this basic equation which will be discussed later in more
detail are:[3]
   1. Simple compounds which are converted into other compounds, usually as
      part of a multiple step reaction pathway. Two examples of this type of
      reaction occur during the formation of nucleic_acids and the charging of
      tRNA prior to translation. For some of these steps, chemical energy is
      required:
                       Precursor &#xA0; molecule  + ATP       &#x21BD;
                  &#x2212;         &#x2212;     &#x21C0;        product &#xA0;
                  AMP  +  PP  i         {\displaystyle {\ce {
                  {Precursor~molecule}+ATP<=>{product~AMP}+PP_{i}}}}  [
                  {\displaystyle {\ce {{Precursor~molecule}+ATP<=>
                  {product~AMP}+PP_{i}}}}]
   2. Simple compounds that are converted into other compounds with the
      assistance of cofactors. For example, the synthesis of phospholipids
      requires acetyl CoA, while the synthesis of another membrane component,
      sphingolipids, requires NADH and FADH for the formation the sphingosine
      backbone. The general equation for these examples is:
                       Precursor &#xA0; molecule  + Cofactor   &#x2192;  e n z
                  y m e     macromolecule    {\displaystyle {\ce {
                  {Precursor~molecule}+Cofactor->[][enzyme]macromolecule}}}  [
                  {\displaystyle {\ce {{Precursor~molecule}+Cofactor->[]
                  [enzyme]macromolecule}}}]
   3. Simple compounds that join together to create a macromolecule. For
      example, fatty_acids join together to form phospholipids. In turn,
      phospholipids and cholesterol interact noncovalently in order to form the
      lipid_bilayer. This reaction may be depicted as follows:
                       Molecule &#xA0; 1   + Molecule &#xA0; 2  &#x27F6;
                  macromolecule    {\displaystyle {\ce {
                  {Molecule~1}+Molecule~2->macromolecule}}}  [{\displaystyle
                  {\ce {{Molecule~1}+Molecule~2->macromolecule}}}]
***** Lipid[edit] *****
Lipid membrane bilayer
Many intricate macromolecules are synthesized in a pattern of simple, repeated
structures.[4] For example, the simplest structures of lipids are fatty_acids.
Fatty acids are hydrocarbon derivatives; they contain a carboxyl_group "head"
and a hydrocarbon chain "tail".[4] These fatty acids create larger components,
which in turn incorporate noncovalent interactions to form the lipid bilayer.
[4] Fatty acid chains are found in two major components of membrane lipids:
phospholipids and sphingolipids. A third major membrane component, cholesterol,
does not contain these fatty acid units.[5]
**** Phospholipids[edit] ****
The foundation of all biomembranes consists of a bilayer structure of
phospholipids.[6] The phospholipid molecule is amphipathic; it contains a
hydrophilic polar head and a hydrophobic nonpolar tail.[4] The phospholipid
heads interact with each other and aqueous media, while the hydrocarbon tails
orient themselves in the center, away from water.[7] These latter interactions
drive the bilayer structure that acts as a barrier for ions and molecules.[8]
There are various types of phospholipids; consequently, their synthesis
pathways differ. However, the first step in phospholipid synthesis involves the
formation of phosphatidate or diacylglycerol_3-phosphate at the endoplasmic
reticulum and outer_mitochondrial_membrane.[7] The synthesis pathway is found
below:
[Phosphatidic_acid_synthesis]
The pathway starts with glycerol 3-phosphate, which gets converted to
lysophosphatidate via the addition of a fatty acid chain provided by acyl
coenzyme_A.[9] Then, lysophosphatidate is converted to phosphatidate via the
addition of another fatty acid chain contributed by a second acyl CoA; all of
these steps are catalyzed by the glycerol phosphate acyltransferase enzyme.[9]
Phospholipid synthesis continues in the endoplasmic reticulum, and the
biosynthesis pathway diverges depending on the components of the particular
phospholipid.[9]
**** Sphingolipids[edit] ****
Like phospholipids, these fatty acid derivatives have a polar head and nonpolar
tails.[5] Unlike phospholipids, sphingolipids have a sphingosine backbone.[10]
Sphingolipids exist in eukaryotic cells and are particularly abundant in the
central_nervous_system.[7] For example, sphingomyelin is part of the myelin
sheath of nerve fibers.[11]
Sphingolipids are formed from ceramides that consist of a fatty acid chain
attached to the amino group of a sphingosine backbone. These ceramides are
synthesized from the acylation of sphingosine.[11] The biosynthetic pathway for
sphingosine is found below:
[Sphingosine_synthesis]
As the image denotes, during sphingosine synthesis, palmitoyl CoA and serine
undergo a condensation_reaction which results in the formation of
dehydrosphingosine.[7] This product is then reduced to form dihydrospingosine,
which is converted to sphingosine via the oxidation_reaction by FAD.[7]
**** Cholesterol[edit] ****
This lipid belongs to a class of molecules called sterols.[5] Sterols have four
fused rings and a hydroxyl_group.[5] Cholesterol is a particularly important
molecule. Not only does it serve as a component of lipid membranes, it is also
a precursor to several steroid hormones, including cortisol, testosterone, and
estrogen.[12]
Cholesterol is synthesized from acetyl_CoA.[12] The pathway is shown below:
[Cholesterol_synthesis_pathway]
More generally, this synthesis occurs in three stages, with the first stage
taking place in the cytoplasm and the second and third stages occurring in the
endoplasmic reticulum.[9] The stages are as follows:[12]
            1. The synthesis of isopentenyl_pyrophosphate, the "building block"
            of cholesterol
            2. The formation of squalene via the condensation of six molecules
            of isopentenyl phosphate
            3. The conversion of squalene into cholesterol via several
            enzymatic reactions
***** Nucleotides[edit] *****
The biosynthesis of nucleotides involves enzyme-catalyzed reactions that
convert substrates into more complex products.[1] Nucleotides are the building
blocks of DNA and RNA. Nucleotides are composed of a five-membered ring formed
from ribose sugar in RNA, and deoxyribose sugar in DNA; these sugars are linked
to a purine or pyrimidine base with a glycosidic_bond and a phosphate group at
the 5'_location of the sugar.[13]
**** Purine nucleotides[edit] ****
The synthesis of IMP.
The DNA nucleotides adenosine and guanosine consist of a purine base attached
to a ribose sugar with a glycosidic bond. In the case of RNA nucleotides
deoxyadenosine and deoxyguanosine, the purine bases are attached to a
deoxyribose sugar with a glycosidic bond. The purine bases on DNA and RNA
nucleotides are synthesized in a twelve-step reaction mechanism present in most
single-celled organisms. Higher eukaryotes employ a similar reaction_mechanism
in ten reaction steps. Purine bases are synthesized by converting
phosphoribosyl_pyrophosphate (PRPP) to inosine_monophosphate (IMP), which is
the first key intermediate in purine base biosynthesis.[14] Further enzymatic
modification of IMP produces the adenosine and guanosine bases of nucleotides.
   1. The first step in purine biosynthesis is a condensation_reaction,
      performed by glutamine-PRPP_amidotransferase. This enzyme transfers the
      amino_group from glutamine to PRPP, forming 5-phosphoribosylamine. The
      following step requires the activation of glycine by the addition of a
      phosphate group from ATP.
   2. GAR synthetase[15] performs the condensation of activated glycine onto
      PRPP, forming glycineamide_ribonucleotide (GAR).
   3. GAR_transformylase adds a formyl_group onto the amino group of GAR,
      forming formylglycinamide ribonucleotide (FGAR).
   4. FGAR amidotransferase[16] catalyzes the addition of a nitrogen group to
      FGAR, forming formylglycinamidine ribonucleotide (FGAM).
   5. FGAM_cyclase catalyzes ring closure, which involves removal of a water
      molecule, forming the 5-membered imidazole ring 5-aminoimidazole
      ribonucleotide (AIR).
   6. N5-CAIR synthetase transfers a carboxyl group, forming the intermediate
      N5-carboxyaminoimidazole ribonucleotide (N5-CAIR).[17]
   7. N5-CAIR_mutase rearranges the carboxyl functional group and transfers it
      onto the imidazole ring, forming carboxyamino-_imidazole_ribonucleotide
      (CAIR). The two step mechanism of CAIR formation from AIR is mostly found
      in single celled organisms. Higher eukaryotes contain the enzyme AIR
      carboxylase,[18] which transfers a carboxyl group directly to AIR
      imidazole ring, forming CAIR.
   8. SAICAR_synthetase forms a peptide_bond between aspartate and the added
      carboxyl group of the imidazole ring, forming N-succinyl-5-
      aminoimidazole-4-carboxamide_ribonucleotide (SAICAR).
   9. SAICAR_lyase removes the carbon skeleton of the added aspartate, leaving
      the amino group and forming 5-aminoimidazole-4-carboxamide_ribonucleotide
      (AICAR).
  10. AICAR_transformylase transfers a carbonyl group to AICAR, forming N-
      formylaminoimidazole-_4-carboxamide_ribonucleotide (FAICAR).
  11. The final step involves the enzyme IMP_synthase, which performs the
      purine ring closure and forms the inosine monophosphate intermediate.[5]
**** Pyrimidine nucleotides[edit] ****
Uridine monophosphate (UMP) biosynthesis
Other DNA and RNA nucleotide bases that are linked to the ribose sugar via a
glycosidic bond are thymine, cytosine and uracil (which is only found in RNA).
Uridine_monophosphate biosynthesis involves an enzyme that is located in the
mitochondrial_inner_membrane and multifunctional enzymes that are located in
the cytosol.[19]
   1. The first step involves the enzyme carbamoyl_phosphate_synthase combining
      glutamine with CO2 in an ATP dependent reaction to form carbamoyl
      phosphate.
   2. Aspartate_carbamoyltransferase condenses carbamoyl phosphate with
      aspartate to form uridosuccinate.
   3. Dihydroorotase performs ring_closure, a reaction that loses water, to
      form dihydroorotate.
   4. Dihydroorotate_dehydrogenase, located within the mitochondrial inner
      membrane,[19] oxidizes dihydroorotate to orotate.
   5. Orotate phosphoribosyl hydrolase (OMP pyrophosphorylase) condenses
      orotate with PRPP to form orotidine-5'-phosphate.
   6. OMP_decarboxylase catalyzes the conversion of orotidine-5'-phosphate to
      UMP.[20]
After the uridine nucleotide base is synthesized, the other bases, cytosine and
thymine are synthesized. Cytosine biosynthesis is a two-step reaction which
involves the conversion of UMP to UTP. Phosphate addition to UMP is catalyzed
by a kinase enzyme. The enzyme CTP_synthase catalyzes the next reaction step:
the conversion of UTP to CTP by transferring an amino_group from glutamine to
uridine; this forms the cytosine base of CTP.[21] The mechanism, which depicts
the reaction UTP + ATP + glutamine â CTP + ADP + glutamate, is below:
['Thymidylate_synthase_reaction:_dUMP_+_5,10-methylenetetrahydrofolate_â_dTMP
+_dihydrofolate]
[Ctp_synthase_mechanism:_UTP_+_ATP_+_glutamine_â_CTP_+_ADP_+_glutamate]
Cytosine is a nucleotide that is present in both DNA and RNA. However, uracil
is only found in RNA. Therefore, after UTP is synthesized, it is must be
converted into a deoxy form to be incorporated into DNA. This conversion
involves the enzyme ribonucleoside_triphosphate_reductase. This reaction that
removes the 2'-OH of the ribose sugar to generate deoxyribose is not affected
by the bases attached to the sugar. This non-specificity allows ribonucleoside
triphosphate reductase to convert all nucleotide_triphosphates to
deoxyribonucleotide by a similar mechanism.[21]
In contrast to uracil, thymine bases are found mostly in DNA, not RNA. Cells do
not normally contain thymine bases that are linked to ribose sugars in RNA,
thus indicating that cells only synthesize deoxyribose-linked thymine. The
enzyme thymidylate_synthetase is responsible for synthesizing thymine residues
from dUMP to dTMP. This reaction transfers a methyl group onto the uracil base
of dUMP to generate dTMP.[21] The thymidylate synthase reaction, dUMP + 5,10-
methylenetetrahydrofolate â dTMP + dihydrofolate, is shown to the right.
***** DNA[edit] *****
As DNA polymerase moves in a 3' to 5' direction along the template strand, it
synthesizes a new strand in the 5' to 3' direction
Although there are differences between eukaryotic and prokaryotic DNA
synthesis, the following section denotes key characteristics of DNA replication
shared by both organisms.
DNA is composed of nucleotides that are joined by phosphodiester_bonds.[4] DNA
synthesis, which takes place in the nucleus, is a semiconservative process,
which means that the resulting DNA molecule contains an original strand from
the parent structure and a new strand.[22] DNA synthesis is catalyzed by a
family of DNA_polymerases that require four deoxynucleoside triphosphates, a
template_strand, and a primer with a free 3'OH in which to incorporate
nucleotides.[23]
In order for DNA replication to occur, a replication_fork is created by enzymes
called helicases which unwind the DNA helix.[23] Topoisomerases at the
replication fork remove supercoils caused by DNA unwinding, and single-stranded
DNA_binding_proteins maintain the two single-stranded DNA templates stabilized
prior to replication.[13]
DNA synthesis is initiated by the RNA_polymerase primase, which makes an RNA
primer with a free 3'OH.[23] This primer is attached to the single-stranded DNA
template, and DNA polymerase elongates the chain by incorporating nucleotides;
DNA polymerase also proofreads the newly synthesized DNA strand.[23]
During the polymerization reaction catalyzed by DNA polymerase, a nucleophilic
attack occurs by the 3'OH of the growing chain on the innermost phosphorus atom
of a deoxynucleoside triphosphate; this yields the formation of a
phosphodiester_bridge that attaches a new nucleotide and releases
pyrophosphate.[9]
Two types of strands are created simultaneously during replication: the leading
strand, which is synthesized continuously and grows towards the replication
fork, and the lagging_strand, which is made discontinuously in Okazaki
fragments and grows away from the replication fork.[22] Okazaki fragments are
covalently joined by DNA_ligase to form a continuous strand.[22] Then, to
complete DNA replication, RNA primers are removed, and the resulting gaps are
replaced with DNA and joined via DNA ligase.[22]
***** Amino acids[edit] *****
A protein is a polymer that is composed from amino_acids that are linked by
peptide_bonds. There are more than 300_amino_acids found in nature of which
only twenty, known as the standard_amino_acids, are the building blocks for
protein.[24] Only green_plants and most microbes are able to synthesize all of
the 20 standard amino acids that are needed by all living species. Mammals can
only synthesize ten of the twenty standard amino acids. The other amino acids,
valine, methionine, leucine, isoleucine, phenylalanine, lysine, threonine and
tryptophan for adults and histidine, and arginine for babies are obtained
through diet.[25]
**** Amino acid basic structure[edit] ****
L-amino acid
The general structure of the standard amino acids includes a primary_amino
group, a carboxyl_group and the functional_group attached to the Î±-carbon. The
different amino acids are identified by the functional group. As a result of
the three different groups attached to the Î±-carbon, amino acids are
asymmetrical_molecules. For all standard amino acids, except glycine, the Î±-
carbon is a chiral_center. In the case of glycine, the Î±-carbon has two
hydrogen atoms, thus adding symmetry to this molecule. With the exception of
proline, all of the amino acids found in life have the L-isoform conformation.
Proline has a functional group on the Î±-carbon that forms a ring with the
amino group.[24]
[Glutamine_oxoglutarate_aminotransferase_and_glutamine_synthetase]
**** Nitrogen source[edit] ****
One major step in amino acid biosynthesis involves incorporating a nitrogen
group onto the Î±-carbon. In cells, there are two major pathways of
incorporating nitrogen groups. One pathway involves the enzyme glutamine
oxoglutarate_aminotransferase (GOGAT) which removes the amide amino group of
glutamine and transfers it onto 2-oxoglutarate, producing two glutamate
molecules. In this catalysis reaction, glutamine serves as the nitrogen source.
An image illustrating this reaction is found to the right.
The other pathway for incorporating nitrogen onto the Î±-carbon of amino acids
involves the enzyme glutamate_dehydrogenase (GDH). GDH is able to transfer
ammonia onto 2-oxoglutarate and form glutamate. Furthermore, the enzyme
glutamine_synthetase (GS) is able to transfer ammonia onto glutamate and
synthesize glutamine, replenishing glutamine.[26]
**** The glutamate family of amino acids[edit] ****
The glutamate family of amino acids includes the amino acids that derive from
the amino acid glutamate. This family includes: glutamate, glutamine, proline,
and arginine. This family also includes the amino acid lysine, which is derived
from Î±-ketoglutarate.[27]
The biosynthesis of glutamate and glutamine is a key step in the nitrogen
assimilation discussed above. The enzymes GOGAT and GDH catalyze the nitrogen
assimilation reactions.
In bacteria, the enzyme glutamate_5-kinase initiates the biosynthesis of
proline by transferring a phosphate group from ATP onto glutamate. The next
reaction is catalyzed by the enzyme pyrroline-5-carboxylate_synthase (P5CS),
which catalyzes the reduction of the Ï-carboxyl group of L-glutamate 5-
phosphate. This results in the formation of glutamate semialdehyde, which
spontaneously cyclizes to pyrroline-5-carboxylate. Pyrroline-5-carboxylate is
further reduced by the enzyme pyrroline-5-carboxylate reductase (P5CR) to yield
a proline amino acid.[28]
In the first step of arginine biosynthesis in bacteria, glutamate is acetylated
by transferring the acetyl group from acetyl-CoA at the N-Î± position; this
prevents spontaneous cyclization. The enzyme N-acetylglutamate_synthase
(glutamate N-acetyltransferase) is responsible for catalyzing the acetylation
step. Subsequent steps are catalyzed by the enzymes N-acetylglutamate_kinase,
N-acetyl-gamma-glutamyl-phosphate_reductase, and acetylornithine/
succinyldiamino_pimelate_aminotransferase and yield the N-acetyl-L-ornithine.
The acetyl group of acetylornithine is removed by the enzyme acetylornithinase
(AO) or ornithine_acetyltransferase (OAT), and this yields ornithine. Then, the
enzymes citrulline and argininosuccinate convert ornithine to arginine.[29]
The diaminopimelic acid pathway
There are two distinct lysine biosynthetic pathways: the diaminopimelic acid
pathway and the Î±-aminoadipate_pathway. The most common of the two synthetic
pathways is the diaminopimelic acid pathway; it consists of several enzymatic
reactions that add carbon groups to aspartate to yield lysine:[30]
   1. Aspartate_kinase initiates the diaminopimelic acid pathway by
      phosphorylating aspartate and producing aspartyl phosphate.
   2. Aspartate_semialdehyde_dehydrogenase catalyzes the NADPH-dependent
      reduction of aspartyl phosphate to yield aspartate semialdehyde.
   3. 4-hydroxy-tetrahydrodipicolinate_synthase adds a pyruvate group to the
      Î²-aspartyl-4-semialdehyde, and a water molecule is removed. This causes
      cyclization and gives rise to (2S,4S)-4-hydroxy-2,3,4,5-
      tetrahydrodipicolinate.
   4. 4-hydroxy-tetrahydrodipicolinate_reductase catalyzes the reduction of
      (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate by NADPH to yield Î'-
      piperideine-2,6-dicarboxylate (2,3,4,5-tetrahydrodipicolinate) and H2O.
   5. Tetrahydrodipicolinate_acyltransferase catalyzes the acetylation reaction
      that results in ring opening and yields N-acetyl Î±-amino-Îµ-
      ketopimelate.
   6. N-succinyl-Î±-amino-Îµ-ketopimelate-glutamate_aminotransaminase catalyzes
      the transamination reaction that removes the keto group of N-acetyl Î±-
      amino-Îµ-ketopimelate and replaces it with an amino group to yield N-
      succinyl-L-diaminopimelate.[31]
   7. N-acyldiaminopimelate_deacylase catalyzes the deacylation of N-succinyl-
      L-diaminopimelate to yield L,L-diaminopimelate.[32]
   8. DAP_epimerase catalyzes the conversion of L,L-diaminopimelate to the meso
      form of L,L-diaminopimelate.[33]
   9. DAP_decarboxylase catalyzes the removal of the carboxyl group, yielding
      L-lysine.
**** The serine family of amino acids[edit] ****
The serine family of amino acid includes: serine, cysteine, and glycine. Most
microorganisms and plants obtain the sulfur for synthesizing methionine from
the amino acid cysteine. Furthermore, the conversion of serine to glycine
provides the carbons needed for the biosynthesis of the methionine and
histidine.[27]
During serine biosynthesis,[34] the enzyme phosphoglycerate_dehydrogenase
catalyzes the initial reaction that oxidizes 3-phospho-D-glycerate to yield 3-
phosphonooxypyruvate.[35] The following reaction is catalyzed by the enzyme
phosphoserine_aminotransferase, which transfers an amino group from glutamate
onto 3-phosphonooxypyruvate to yield L-phosphoserine.[36] The final step is
catalyzed by the enzyme phosphoserine_phosphatase, which dephosphorylates L-
phosphoserine to yield L-serine.[37]
There are two known pathways for the biosynthesis of glycine. Organisms that
use ethanol and acetate as the major carbon source utilize the glyconeogenic
pathway to synthesize glycine. The other pathway of glycine biosynthesis is
known as the glycolytic pathway. This pathway converts serine synthesized from
the intermediates of glycolysis to glycine. In the glycolytic pathway, the
enzyme serine_hydroxymethyltransferase catalyzes the cleavage of serine to
yield glycine and transfers the cleaved carbon group of serine onto
tetrahydrofolate, forming 5,10-methylene-tetrahydrofolate.[38]
Cysteine biosynthesis is a two-step reaction that involves the incorporation of
inorganic sulfur. In microorganisms and plants, the enzyme serine
acetyltransferase catalyzes the transfer of acetyl group from acetyl-CoA onto
L-serine to yield O-acetyl-L-serine.[39] The following reaction step, catalyzed
by the enzyme O-acetyl_serine_(thiol)_lyase, replaces the acetyl group of O-
acetyl-L-serine with sulfide to yield cysteine.[40]
**** The aspartate family of amino acids[edit] ****
The aspartate family of amino acids includes: threonine, lysine, methionine,
isoleucine, and aspartate. Lysine and isoleucine are considered part of the
aspartate family even though part of their carbon skeleton is derived from
pyruvate. In the case of methionine, the methyl carbon is derived from serine
and the sulfur group, but in most organisms, it is derived from cysteine.[27]
The biosynthesis of aspartate is a one step reaction that is catalyzed by a
single enzyme. The enzyme aspartate_aminotransferase catalyzes the transfer of
an amino group from aspartate onto Î±-ketoglutarate to yield glutamate and
oxaloacetate.[41] Asparagine is synthesized by an ATP-dependent addition of an
amino group onto aspartate; asparagine_synthetase catalyzes the addition of
nitrogen from glutamine or soluble ammonia to aspartate to yield asparagine.
[42]
The diaminopimelic acid lysine biosynthetic pathway
The diaminopimelic acid biosynthetic pathway of lysine belongs to the aspartate
family of amino acids. This pathway involves nine enzyme-catalyzed reactions
that convert aspartate to lysine.[43]
   1. Aspartate_kinase catalyzes the initial step in the diaminopimelic acid
      pathway by transferring a phosphoryl from ATP onto the carboxylate group
      of aspartate, which yields aspartyl-Î²-phosphate.[44]
   2. Aspartate-semialdehyde_dehydrogenase catalyzes the reduction reaction by
      dephosphorylation of aspartyl-Î²-phosphate to yield aspartate-Î²-
      semialdehyde.[45]
   3. Dihydrodipicolinate_synthase catalyzes the condensation reaction of
      aspartate-Î²-semialdehyde with pyruvate to yield dihydrodipicolinic acid.
      [46]
   4. 4-hydroxy-tetrahydrodipicolinate_reductase catalyzes the reduction of
      dihydrodipicolinic acid to yield tetrahydrodipicolinic acid.[47]
   5. Tetrahydrodipicolinate_N-succinyltransferase catalyzes the transfer of a
      succinyl group from succinyl-CoA on to tetrahydrodipicolinic acid to
      yield N-succinyl-L-2,6-diaminoheptanedioate.[48]
   6. N-succinyldiaminopimelate aminotransferase catalyzes the transfer of an
      amino group from glutamate onto N-succinyl-L-2,6-diaminoheptanedioate to
      yield N-succinyl-L,L-diaminopimelic acid.[49]
   7. Succinyl-diaminopimelate_desuccinylase catalyzes the removal of acyl
      group from N-succinyl-L,L-diaminopimelic acid to yield L,L-diaminopimelic
      acid.[50]
   8. Diaminopimelate_epimerase catalyzes the inversion of the Î±-carbon of
      L,L-diaminopimelic acid to yield meso-diaminopimelic_acid.[51]
   9. Siaminopimelate decarboxylase catalyzes the final step in lysine
      biosynthesis that removes the carbon dioxide group from meso-
      diaminopimelic acid to yield L-lysine.[52]
***** Proteins[edit] *****
The tRNA anticodon interacts with the mRNA codon in order to bind an amino acid
to growing polypeptide chain.
The process of tRNA charging
Protein synthesis occurs via a process called translation.[53] During
translation, genetic material called mRNA is read by ribosomes to generate a
protein polypeptide chain.[53] This process requires transfer_RNA (tRNA) which
serves as an adaptor by binding amino_acids on one end and interacting with
mRNA at the other end; the latter pairing between the tRNA and mRNA ensures
that the correct amino acid is added to the chain.[53] Protein synthesis occurs
in three phases: initiation, elongation, and termination.[13] Prokaryotic
translation differs from eukaryotic_translation; however, this section will
mostly focus on the commonalities between the two organisms.
**** Additional background[edit] ****
Before translation can begin, the process of binding a specific amino acid to
its corresponding tRNA must occur. This reaction, called tRNA charging, is
catalyzed by aminoacyl_tRNA_synthetase.[54] A specific tRNA synthetase is
responsible for recognizing and charging a particular amino acid.[54]
Furthermore, this enzyme has special discriminator regions to ensure the
correct binding between tRNA and its cognate amino acid.[54] The first step for
joining an amino acid to its corresponding tRNA is the formation of aminoacyl-
AMP:[54]
           Amino &#xA0; acid  + ATP       &#x21BD;     &#x2212;
      &#x2212;     &#x21C0;        aminoacyl  &#x2212;  AMP  +  PP  i
      {\displaystyle {\ce {{Amino~acid}+ATP<=>{aminoacyl-AMP}+PP_{i}}}}  [
      {\displaystyle {\ce {{Amino~acid}+ATP<=>{aminoacyl-AMP}+PP_{i}}}}]
This is followed by the transfer of the aminoacyl group from aminoacyl-AMP to a
tRNA molecule. The resulting molecule is aminoacyl-tRNA:[54]
           Aminoacyl  &#x2212;  AMP  + tRNA       &#x21BD;     &#x2212;
      &#x2212;     &#x21C0;        aminoacyl  &#x2212;  tRNA  + AMP
      {\displaystyle {\ce {{Aminoacyl-AMP}+ tRNA <=> {aminoacyl-tRNA}+ AMP}}}
      [{\displaystyle {\ce {{Aminoacyl-AMP}+ tRNA <=> {aminoacyl-tRNA}+ AMP}}}]
The combination of these two steps, both of which are catalyzed by aminoacyl
tRNA synthetase, produces a charged tRNA that is ready to add amino acids to
the growing polypeptide chain.
In addition to binding an amino acid, tRNA has a three nucleotide unit called
an anticodon that base_pairs with specific nucleotide triplets on the mRNA
called codons; codons encode a specific amino acid.[55] This interaction is
possible thanks to the ribosome, which serves as the site for protein
synthesis. The ribosome possesses three tRNA binding sites: the aminoacyl site
(A site), the peptidyl site (P site), and the exit site (E site).[56]
There are numerous codons within an mRNA transcript, and it is very common for
an amino acid to be specified by more than one codon; this phenomenon is called
degeneracy.[57] In all, there are 64 codons, 61 of each code for one of the 20
amino acids, while the remaining codons specify chain termination.[57]
**** Translation in steps[edit] ****
As previously mentioned, translation occurs in three phases: initiation,
elongation, and termination.
Translation
*** Step 1: Initiation[edit] ***
The completion of the initiation phase is dependent on the following three
events:[13]
1. The recruitment of the ribosome to mRNA
2. The binding of a charged initiator tRNA into the P site of the ribosome
3. The proper alignment of the ribosome with mRNA's start codon
*** Step 2: Elongation[edit] ***
Following initiation, the polypeptide chain is extended via anticodon:codon
interactions, with the ribosome adding amino acids to the polypeptide chain one
at a time. The following steps must occur to ensure the correct addition of
amino acids:[58]
1. The binding of the correct tRNA into the A site of the ribosome
2. The formation of a peptide_bond between the tRNA in the A site and the
polypeptide chain attached to the tRNA in the P site
3. Translocation or advancement of the tRNA-mRNA complex by three nucleotides
Translocation "kicks off" the tRNA at the E site and shifts the tRNA from the A
site into the P site, leaving the A site free for an incoming tRNA to add
another amino acid.
*** Step 3: Termination[edit] ***
The last stage of translation occurs when a stop_codon enters the A site.[1]
Then, the following steps occur:
1. The recognition of codons by release_factors, which causes the hydrolysis of
the polypeptide chain from the tRNA located in the P site[1]
2. The release of the polypeptide chain[57]
3. The dissociation and "recycling" of the ribosome for future translation
processes[57]
A summary table of the key players in translation is found below:
Key players in Translation Translation Stage       Purpose
tRNA synthetase            before initiation       Responsible for tRNA
                                                   charging
                                                   Template for protein
mRNA                       initiation, elongation, synthesis; contains regions
                           termination             named codons which encode
                                                   amino acids
                                                   Binds ribosomes sites A, P,
                                                   E; anticodon base pairs with
tRNA                       initiation, elongation, mRNA codon to ensure that
                           termination             the correct amino acid is
                                                   incorporated into the
                                                   growing polypeptide chain
                           initiation, elongation, Directs protein synthesis
ribosome                   termination             and catalyzes the formation
                                                   of the peptide bond
***** Diseases associated with macromolecule deficiency[edit] *****
Familial hypercholesterolemia causes cholesterol deposits
Errors in biosynthetic pathways can have deleterious consequences including the
malformation of macromolecules or the underproduction of functional molecules.
Below are examples that illustrate the disruptions that occur due to these
inefficiencies.
    * Familial_hypercholesterolemia: this disorder is characterized by the
      absence of functional receptors for LDL.[59] Deficiencies in the
      formation of LDL receptors may cause faulty receptors which disrupt the
      endocytic pathway, inhibiting the entry of LDL into the liver and other
      cells.[59] This causes a buildup of LDL in the blood plasma, which
      results in atherosclerotic_plaques that narrow arteries and increase the
      risk of heart attacks.[59]
    * LeschâNyhan_syndrome: this genetic disease is characterized by self-
      mutilation, mental deficiency, and gout.[60] It is caused by the absence
      of hypoxanthine-guanine_phosphoribosyltransferase, which is a necessary
      enzyme for purine nucleotide formation.[60] The lack of enzyme reduces
      the level of necessary nucleotides and causes the accumulation of
      biosynthesis intermediates, which results in the aforementioned unusual
      behavior.[60]
    * Severe_combined_immunodeficiency_(SCID): SCID is characterized by a loss
      of T_cells.[61] Shortage of these immune system components increases the
      susceptibility to infectious agents because the affected individuals
      cannot develop immunological memory.[61] This immunological disorder
      results from a deficiency in adenosine_deanimase activity, which causes a
      buildup of dATP. These dATP molecules then inhibit ribonucleotide
      reductase, which prevents of DNA synthesis.[61]
    * Huntington's_disease: this neurological disease is caused from errors
      that occur during DNA synthesis.[62] These errors or mutations lead to
      the expression of a mutant huntingtin protein, which contains repetitive
      glutamine residues that are encoded by expanding CAG_trinucleotide
      repeats in the gene.[62] Huntington's disease is characterized by
      neuronal loss and gliosis. Symptoms of the disease include: movement
      disorder, cognitive decline, and behavioral disorder.[63]
***** See also[edit] *****
    * [icon]Biology_portal
    * [icon]Biotechnology_portal
    * Molecular_and_cellular_biology_portal
    * Lipids
    * Phospholipid_bilayer
    * Nucleotides
    * DNA
    * DNA_replication
    * Proteinogenic_amino_acid
    * Codon_table
    * Prostaglandin
***** References[edit] *****
   1. ^ a b c dAlberts, Bruce (2007). Molecular biology of the cell. New York:
      Garland Science. ISBN 978-0815341055.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Zumdahl, Steven S. Zumdahl, Susan A. (2008). Chemistry (8th ed.). CA:
      Cengage Learning. ISBN 978-0547125329.
   4. ^Pratt, Donald Voet, Judith G. Voet, Charlotte W. (2013). Fundamentals of
      biochemistry : life at the molecular level (4th ed.). Hoboken, NJ: Wiley.
      ISBN 978-0470547847.
   5. ^ a b c d eLodish, Harvey; et al. (2007). Molecular_cell_biology (6th
      ed.). New York: W.H. Freeman. ISBN 978-0716743668.
   6. ^ a b c d eCox, David L. Nelson, Michael M. (2008). Lehninger_principles
      of_biochemistry (5th ed.). New York: W.H. Freeman. ISBN 9780716771081.
   7. ^Hanin, Israel (2013). Phospholipids: Biochemical, Pharmaceutical, and
      Analytical Considerations. Springer. ISBN 978-1475713664.
   8. ^ a b c d eVance, Dennis E.; Vance, Jean E. (2008). Biochemistry of
      lipids, lipoproteins and membranes (5th ed.). Amsterdam: Elsevier.
      ISBN 978-0444532190.
   9. ^Katsaras, J.; et al. (2001). Lipid bilayers : structure and
      interactions ; with 6 tables. Berlin [u.a.]: Springer. ISBN 978-
      3540675556.
  10. ^ a b c d eStryer, Jeremy M. Berg; John L. Tymoczko; Lubert (2007).
      Biochemistry (6. ed., 3. print. ed.). New York: Freeman. ISBN 978-
      0716787242.
  11. ^Gault, CR; LM Obeid; YA Hannun (2010). An Overview of sphingolipid
      metabolism: from synthesis to breakdown. Adv Exp Med Biol. Advances in
      Experimental Medicine and Biology. 688. pp. 1â23. doi:10.1007/978-1-
      4419-6741-1_1. ISBN 978-1-4419-6740-4. PMC 3069696. PMID 20919643.
  12. ^ a bSiegel, George J. (1999). Basic neurochemistry : molecular, cellular
      and medical aspects (6. ed.). Philadelphia, Pa. [u.a.]: Lippincott
      Williams & Wilkins. ISBN 978-0397518203.
  13. ^ a b cHarris, J. Robin (2010). Cholesterol binding and cholesterol
      transport proteins : structure and function in health and disease.
      Dordrecht: Springer. ISBN 978-9048186211.
  14. ^ a b c dWatson, James D.; et al. (2007). Molecular biology of the gene
      (6th ed.). San Francisco, Calif.: Benjamin Cummings. ISBN 978-0805395921.
  15. ^Kappock, TJ; Ealick, SE; Stubbe, J (October 2000). "Modular evolution of
      the purine biosynthetic pathway". Current Opinion in Chemical Biology. 4
      (5): 567â72. doi:10.1016/s1367-5931(00)00133-2. PMID 11006546.
  16. ^Sampei, G; Baba, S; Kanagawa, M; Yanai, H; Ishii, T; Kawai, H; Fukai, Y;
      Ebihara, A; Nakagawa, N; Kawai, G (October 2010). "Crystal structures of
      glycinamide ribonucleotide synthetase, PurD, from thermophilic
      eubacteria". Journal of Biochemistry. 148 (4): 429â38. doi:10.1093/jb/
      mvq088. PMID 20716513.
  17. ^Hoskins, AA; Anand, R; Ealick, SE; Stubbe, J (Aug 17, 2004). "The
      formylglycinamide ribonucleotide amidotransferase complex from Bacillus
      subtilis: metabolite-mediated complex formation". Biochemistry. 43 (32):
      10314â27. doi:10.1021/bi049127h. PMID 15301530.
  18. ^Mueller, EJ; Meyer, E; Rudolph, J; Davisson, VJ; Stubbe, J (Mar 1,
      1994). "N5-carboxyaminoimidazole ribonucleotide: evidence for a new
      intermediate and two new enzymatic activities in the de novo purine
      biosynthetic pathway of Escherichia coli". Biochemistry. 33 (8):
      2269â78. doi:10.1021/bi00174a038. PMID 8117684.
  19. ^Firestine, SM; Poon, SW; Mueller, EJ; Stubbe, J; Davisson, VJ (Oct 4,
      1994). "Reactions catalyzed by 5-aminoimidazole ribonucleotide
      carboxylases from Escherichia coli and Gallus gallus: a case for
      divergent catalytic mechanisms". Biochemistry. 33 (39): 11927â34. doi:
      10.1021/bi00205a031. PMID 7918411.
  20. ^ a bSrere, PA (1987). "Complexes of sequential metabolic enzymes".
      Annual Review of Biochemistry. 56 (1): 89â124. doi:10.1146/
      annurev.bi.56.070187.000513. PMID 2441660.
  21. ^Broach, edited by Jeffrey N. Strathern, Elizabeth W. Jones, James R.
      (1981). The Molecular biology of the yeast Saccharomyces. Cold Spring
      Harbor, N.Y.: Cold Spring Harbor Laboratory. ISBN 978-0879691394.CS1
      maint: Extra text: authors list (link)
  22. ^ a b cO'Donovan, GA; Neuhard, J (September 1970). "Pyrimidine_metabolism
      in_microorganisms". Bacteriological Reviews. 34 (3): 278â343.
      PMC 378357. PMID 4919542.
  23. ^ a b c dGeer, Gerald Karp ; responsible for the revision of chapter 15
      Peter van der (2004). Cell and molecular biology : concepts and
      experiments (4th ed., Wiley International ed.). New York: J. Wiley &
      Sons. ISBN 978-0471656654.
  24. ^ a b c dGriffiths, Anthony J. F. (1999). Modern_genetic_analysis (2.
      print. ed.). New York: Freeman. ISBN 978-0716731184.
  25. ^ a bWu, G (May 2009). "Amino acids: metabolism, functions, and
      nutrition". Amino Acids. 37 (1): 1â17. doi:10.1007/s00726-009-0269-0.
      PMID 19301095.
  26. ^Mousdale, D. M.; Coggins, J. R. (1991). Amino Acid Synthesis. Target
      Sites for Herbicide Action. pp. 29â56. doi:10.1007/978-1-4899-2433-9_2.
      ISBN 978-1-4899-2435-3.
  27. ^Miflin, B. J.; Lea, P. J. (1977). "Amino Acid Metabolism". Annual Review
      of Plant Physiology. 28: 299â329. doi:10.1146/
      annurev.pp.28.060177.001503.
  28. ^ a b cUmbarger, HE (1978). "Amino acid biosynthesis and its regulation".
      Annual Review of Biochemistry. 47 (1): 532â606. doi:10.1146/
      annurev.bi.47.070178.002533. PMID 354503.
  29. ^PÃ©rez-Arellano, I; Carmona-Alvarez, F; MartÃ­nez, AI; RodrÃ­guez-DÃ­az,
      J; Cervera, J (March 2010). "Pyrroline-5-carboxylate_synthase_and_proline
      biosynthesis:_from_osmotolerance_to_rare_metabolic_disease". Protein
      Science. 19 (3): 372â82. doi:10.1002/pro.340. PMC 2866264.
      PMID 20091669.
  30. ^Xu, Y; Labedan, B; Glansdorff, N (March 2007). "Surprising_arginine
      biosynthesis:_a_reappraisal_of_the_enzymology_and_evolution_of_the
      pathway_in_microorganisms". Microbiology and Molecular Biology Reviews.
      71 (1): 36â47. doi:10.1128/MMBR.00032-06. PMC 1847373. PMID 17347518.
  31. ^"MetaCyc:_L-lysine_biosynthesis_I".
  32. ^PETERKOFSKY, B; GILVARG, C (May 1961). "N-Succinyl-L-diaminopimelic-
      glutamic transaminase". The Journal of Biological Chemistry. 236:
      1432â8. PMID 13734750.
  33. ^KINDLER, SH; GILVARG, C (December 1960). "N-Succinyl-L-2,6-
      diaminopimelic acid deacylase". The Journal of Biological Chemistry. 235:
      3532â5. PMID 13756049.
  34. ^Born, TL; Blanchard, JS (October 1999). "Structure/function studies on
      enzymes in the diaminopimelate pathway of bacterial cell wall
      biosynthesis". Current Opinion in Chemical Biology. 3 (5): 607â13. doi:
      10.1016/s1367-5931(99)00016-2. PMID 10508663.
  35. ^"Escherichia_coli_K-12_substr._MG1655". serine biosynthesis. SRI
      International. Retrieved 12 December 2013.
  36. ^Bell, JK; Grant, GA; Banaszak, LJ (Mar 30, 2004). "Multiconformational
      states in phosphoglycerate dehydrogenase". Biochemistry. 43 (12):
      3450â8. doi:10.1021/bi035462e. PMID 15035616.
  37. ^Dubnovitsky, AP; Kapetaniou, EG; Papageorgiou, AC (January 2005).
      "Enzyme_adaptation_to_alkaline_pH:_atomic_resolution_(1.08_A)_structure
      of_phosphoserine_aminotransferase_from_Bacillus_alcalophilus". Protein
      Science. 14 (1): 97â110. doi:10.1110/ps.041029805. PMC 2253317.
      PMID 15608117.
  38. ^Wang, W; Kim, R; Jancarik, J; Yokota, H; Kim, SH (Jan 10, 2001).
      "Crystal structure of phosphoserine phosphatase from Methanococcus
      jannaschii, a hyperthermophile, at 1.8 A resolution". Structure. 9 (1):
      65â71. doi:10.1016/s0969-2126(00)00558-x. PMID 11342136.
  39. ^Monschau, N; Stahmann, KP; Sahm, H; McNeil, JB; Bognar, AL (May 1,
      1997). "Identification of Saccharomyces cerevisiae GLY1 as a threonine
      aldolase: a key enzyme in glycine biosynthesis". FEMS Microbiology
      Letters. 150 (1): 55â60. doi:10.1111/j.1574-6968.1997.tb10349.x.
      PMID 9163906.
  40. ^Pye, VE; Tingey, AP; Robson, RL; Moody, PC (Sep 24, 2004). "The
      structure and mechanism of serine acetyltransferase from Escherichia
      coli". The Journal of Biological Chemistry. 279 (39): 40729â36. doi:
      10.1074/jbc.M403751200. PMID 15231846.
  41. ^Huang, B; Vetting, MW; Roderick, SL (May 2005). "The_active_site_of_O-
      acetylserine_sulfhydrylase_is_the_anchor_point_for_bienzyme_complex
      formation_with_serine_acetyltransferase". Journal of Bacteriology. 187
      (9): 3201â5. doi:10.1128/JB.187.9.3201-3205.2005. PMC 1082839.
      PMID 15838047.
  42. ^McPhalen, CA; Vincent, MG; Picot, D; Jansonius, JN; Lesk, AM; Chothia, C
      (Sep 5, 1992). "Domain closure in mitochondrial aspartate
      aminotransferase". Journal of Molecular Biology. 227 (1): 197â213. doi:
      10.1016/0022-2836(92)90691-C. PMID 1522585.
  43. ^Larsen, TM; Boehlein, SK; Schuster, SM; Richards, NG; Thoden, JB;
      Holden, HM; Rayment, I (Dec 7, 1999). "Three-dimensional structure of
      Escherichia coli asparagine synthetase B: a short journey from substrate
      to product". Biochemistry. 38 (49): 16146â57.
      CiteSeerX 10.1.1.453.5998. doi:10.1021/bi9915768. PMID 10587437.
  44. ^Velasco, AM; Leguina, JI; Lazcano, A (October 2002). "Molecular
      evolution of the lysine biosynthetic pathways". Journal of Molecular
      Evolution. 55 (4): 445â59. doi:10.1007/s00239-002-2340-2.
      PMID 12355264.
  45. ^Kotaka, M; Ren, J; Lockyer, M; Hawkins, AR; Stammers, DK (Oct 20, 2006).
      "Structures of R- and T-state Escherichia coli aspartokinase III.
      Mechanisms of the allosteric transition and inhibition by lysine". The
      Journal of Biological Chemistry. 281 (42): 31544â52. doi:10.1074/
      jbc.M605886200. PMID 16905770.
  46. ^Hadfield, A; Kryger, G; Ouyang, J; Petsko, GA; Ringe, D; Viola, R (Jun
      18, 1999). "Structure of aspartate-beta-semialdehyde dehydrogenase from
      Escherichia coli, a key enzyme in the aspartate family of amino acid
      biosynthesis". Journal of Molecular Biology. 289 (4): 991â1002. doi:
      10.1006/jmbi.1999.2828. PMID 10369777.
  47. ^Mirwaldt, C; KorndÃ¶rfer, I; Huber, R (Feb 10, 1995). "The crystal
      structure of dihydrodipicolinate synthase from Escherichia coli at 2.5 A
      resolution". Journal of Molecular Biology. 246 (1): 227â39. doi:
      10.1006/jmbi.1994.0078. PMID 7853400.
  48. ^Cirilli, M; Zheng, R; Scapin, G; Blanchard, JS (Sep 16, 2003). "The
      three-dimensional structures of the Mycobacterium tuberculosis
      dihydrodipicolinate reductase-NADH-2,6-PDC and -NADPH-2,6-PDC complexes.
      Structural and mutagenic analysis of relaxed nucleotide specificity".
      Biochemistry. 42 (36): 10644â50. doi:10.1021/bi030044v. PMID 12962488.
  49. ^Beaman, TW; Binder, DA; Blanchard, JS; Roderick, SL (Jan 21, 1997).
      "Three-dimensional structure of tetrahydrodipicolinate N-
      succinyltransferase". Biochemistry. 36 (3): 489â94. doi:10.1021/
      bi962522q. PMID 9012664.
  50. ^Weyand, S; Kefala, G; Weiss, MS (Mar 30, 2007). "The three-dimensional
      structure of N-succinyldiaminopimelate aminotransferase from
      Mycobacterium tuberculosis". Journal of Molecular Biology. 367 (3):
      825â38. doi:10.1016/j.jmb.2007.01.023. PMID 17292400.
  51. ^Nocek, BP; Gillner, DM; Fan, Y; Holz, RC; Joachimiak, A (Apr 2, 2010).
      "Structural_basis_for_catalysis_by_the_mono-_and_dimetalated_forms_of_the
      dapE-encoded_N-succinyl-L,L-diaminopimelic_acid_desuccinylase". Journal
      of Molecular Biology. 397 (3): 617â26. doi:10.1016/j.jmb.2010.01.062.
      PMC 2885003. PMID 20138056.
  52. ^Pillai, B; Cherney, M; Diaper, CM; Sutherland, A; Blanchard, JS;
      Vederas, JC; James, MN (Nov 23, 2007). "Dynamics of catalysis revealed
      from the crystal structures of mutants of diaminopimelate epimerase".
      Biochemical and Biophysical Research Communications. 363 (3): 547â53.
      doi:10.1016/j.bbrc.2007.09.012. PMID 17889830.
  53. ^Gokulan, K; Rupp, B; Pavelka MS, Jr; Jacobs WR, Jr; Sacchettini, JC (May
      16, 2003). "Crystal structure of Mycobacterium tuberculosis
      diaminopimelate decarboxylase, an essential enzyme in bacterial lysine
      biosynthesis". The Journal of Biological Chemistry. 278 (20): 18588â96.
      doi:10.1074/jbc.M301549200. PMID 12637582.
  54. ^ a b cWeaver, Robert F. (2005). Molecular_biology (3rd ed.). Boston:
      McGraw-Hill Higher Education. ISBN 978-0-07-284611-9.
  55. ^ a b c d eCooper, Geoffrey M. (2000). The cell : a molecular approach
      (2nd ed.). Washington (DC): ASM Press. ISBN 978-0878931064.
  56. ^Jackson, R.J.; et al. (February 2010). "The_mechanism_of_eukaryotic
      translation_initiation_and_principles_of_its_regulation". Molecular Cell
      Biology. 10: 113â127. PMC 4461372.
  57. ^Green, Rachel; Harry F. Noller; et al. (1997). "Ribosomes and
      Translation". Annu. Rev. Biochem. 66: 679â716. doi:10.1146/
      annurev.biochem.66.1.679. PMID 9242921.
  58. ^ a b c dPestka (editors), Herbert Weissbach, Sidney (1977). Molecular
      Mechanisms of protein biosynthesis. New York: Academic Press. ISBN 978-
      0127442501.CS1 maint: Extra text: authors list (link)
  59. ^Frank, J; Haixiao Gao; et al. (September 2007). "The_process_of
      mRNAâtRNA_translocation". PNAS. 104 (50): 19671â19678. doi:10.1073/
      pnas.0708517104. PMC 2148355. PMID 18003906.
  60. ^ a b cBandeali, Salman J.; Daye, Jad; Virani, Salim S. (30 November
      2013). "Novel Therapies for Treating Familial Hypercholesterolemia".
      Current Atherosclerosis Reports. 16 (1): 382. doi:10.1007/s11883-013-
      0382-0. PMID 24293346.
  61. ^ a b cKang, Tae Hyuk; Park, Yongjin; Bader, Joel S.; Friedmann,
      Theodore; Cooney, Austin John (9 October 2013). "The_Housekeeping_Gene
      Hypoxanthine_Guanine_Phosphoribosyltransferase_(HPRT)_Regulates_Multiple
      Developmental_and_Metabolic_Pathways_of_Murine_Embryonic_Stem_Cell
      Neuronal_Differentiation". PLoS ONE. 8 (10): e74967. doi:10.1371/
      journal.pone.0074967. PMC 3794013. PMID 24130677.
  62. ^ a b cWalport, Ken Murphy, Paul Travers, Mark (2011). Janeway's
      Immunobiology (8. ed.). Oxford: Taylor & Francis. ISBN 978-0815342434.
  63. ^ a bHughes, edited by Donald C. Lo, Robert E. (2010). Neurobiology of
      Huntington's disease : applications to drug discovery (2nd ed.). Boca
      Raton: CRC Press/Taylor & Francis Group. ISBN 978-0849390005.CS1 maint:
      Extra text: authors list (link)
  64. ^Biglan, Kevin M.; Ross, Christopher A.; Langbehn, Douglas R.; Aylward,
      Elizabeth H.; Stout, Julie C.; Queller, Sarah; Carlozzi, Noelle E.; Duff,
      Kevin; Beglinger, Leigh J.; Paulsen, Jane S. (26 June 2009). "Motor
      abnormalities_in_premanifest_persons_with_Huntington's_disease:_The
      PREDICT-HD_study". Movement Disorders. 24 (12): 1763â1772. doi:10.1002/
      mds.22601. PMC 3048804. PMID 19562761.
    * v
    * t
    * e
Chemical_synthesis
          * Artificial_gene_synthesis
          * Biomimetic_synthesis
          * Bioretrosynthesis
          * Biosynthesis
          * Chemosynthesis
          * Convergent_synthesis
          * Custom_peptide_synthesis
          * Direct_process
          * Divergent_synthesis
          * Electrosynthesis
          * Enantioselective_synthesis
          * Fully_automated_synthesis
Types     * Hydrothermal_synthesis
          * LASiS
          * Mechanosynthesis
          * One-pot_synthesis
          * Organic_synthesis
          * Peptide_synthesis
          * Radiosynthesis
          * Retrosynthesis
          * Semisynthesis
          * Solid-phase_synthesis
          * Solvothermal_synthesis
          * Total_synthesis
          * Volume_combustion_synthesis
    * [Category]Category
    * [Commons page] Commons
    * [Portal]Portal
Authority_control [Edit_this_at_Wikidata]     * GND: 4006902-3
                                              * NDL: 00570296
    * [ATP-3D-vdW.png]Metabolism_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Biosynthesis&oldid=909528724"
Categories:
    * Metabolism
    * Biosynthesis
Hidden categories:
    * CS1_maint:_Extra_text:_authors_list
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * AzÉrbaycanca
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * íêµ­ì´
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Bahasa_Indonesia
    * Italiano
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * Tagalog
    * à¹à¸à¸¢
    * Ð¢Ð¾Ò·Ð¸ÐºÓ£
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * ä¸­æ
Edit_links
    * This page was last edited on 6 August 2019, at 00:36 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
