The following text has been accessed from https://en.wikipedia.org/wiki/Complement_system at Fri Aug 9 03:35:56 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Complement system ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
This article is about an aspect of the immune system. For other uses, see
Complement.
Scheme of the complement system
The complement system is a part of the immune_system that enhances
(complements) the ability of antibodies and phagocytic cells to clear microbes
and damaged cells from an organism, promote inflammation, and attack the
pathogen's cell_membrane. It is part of the innate_immune_system,[1] which is
not adaptable and does not change during an individual's lifetime. The
complement system can, however, be recruited and brought into action by
antibodies generated by the adaptive_immune_system.
The complement system consists of a number of small proteins that are
synthesized by the liver, and circulate in the blood as inactive precursors.
When stimulated by one of several triggers, proteases in the system cleave
specific proteins to release cytokines and initiate an amplifying cascade of
further cleavages. The end result of this complement activation or complement
fixation cascade is stimulation of phagocytes to clear foreign and damaged
material, inflammation to attract additional phagocytes, and activation of the
cell-killing membrane_attack_complex. Over 30 proteins and protein fragments
make up the complement system, including serum_proteins, and cell_membrane
receptors. They account for about 10% of the globulin fraction of blood serum.
[citation_needed]
Three biochemical pathways activate the complement system: the classical
complement_pathway, the alternative_complement_pathway, and the lectin_pathway.
[2]
⁰
***** Contents *****
    * 1_History
    * 2_Functions
    * 3_Overview
          o 3.1_Classical_pathway
          o 3.2_Alternative_pathway
          o 3.3_Lectin_pathway
          o 3.4_Complement_protein_fragment_nomenclature
          o 3.5_Viral_inhibition
          o 3.6_Review
    * 4_Activation_of_complements_by_antigen-associated_antibody
    * 5_Regulation
    * 6_Role_in_disease
          o 6.1_Complement_deficiency
          o 6.2_Deficiencies_in_complement_regulators
          o 6.3_Diagnostic_tools
    * 7_Modulation_by_infections
    * 8_References
    * 9_External_links
***** History[edit] *****
In 1888, George_Nuttall found that sheep blood serum had mild killing activity
against the bacterium that causes anthrax. The killing activity disappeared
when he heated the blood.[3] In 1891, Hans_Ernst_August_Buchner, noting the
same property of blood in his experiments, named the killing property "alexin",
which means "to ward off" in Greek.[4] By 1884, several laboratories had
demonstrated that serum from guinea pigs that had recovered from cholera killed
the cholera bacterium in vitro. Heating the serum destroyed its killing
activity. Nevertheless, the heat-inactivated serum, when injected into guinea
pigs exposed to the cholera bacteria, maintained its ability to protect the
animals from illness. Jules_Bordet, a young Belgian scientist in Paris at the
Pasteur Institute, concluded that this principle has two components, one that
maintained a "sensitizing" effect after being heated and one (alexin) whose
toxic effect was lost after being heated. The heat-stable component was
responsible for immunity against specific microorganisms, whereas the heat-
sensitive component was responsible for the non-specific antimicrobial activity
conferred by all normal sera. In 1899, Paul_Ehrlich renamed the heat-sensitive
component "complement."[3]
Ehrlich introduced the term "complement" as part of his larger theory of the
immune system. According to this theory, the immune system consists of cells
that have specific receptors on their surface to recognize antigens. Upon
immunisation with an antigen, more of these receptors are formed, and they are
then shed from the cells to circulate in the blood. Those receptors, which we
now call "antibodies", were called by Ehrlich "amboceptors" to emphasise their
bifunctional binding capacity: They recognise and bind to a specific antigen,
but they also recognise and bind to the heat-labile antimicrobial component of
fresh serum. Ehrlich, therefore, named this heat-labile component "complement",
because it is something in the blood that "complements" the cells of the immune
system. Ehrlich believed that each antigen-specific amboceptor has its own
specific complement, whereas Bordet believed that there is only one type of
complement. In the early 20th century, this controversy was resolved when it
became understood that complement can act in combination with specific
antibodies, or on its own in a non-specific way.[citation_needed]
***** Functions[edit] *****
Membrane Attack Complex (Terminal Complement Complex C5b-9)
Complement triggers the following immune functions:[5]
   1. Phagocytosis â by opsonizing antigens. C3b has most important
      opsonizing activity
   2. Inflammation â by attracting macrophages and neutrophils
   3. Membrane_attack â by rupturing cell wall of bacteria
***** Overview[edit] *****
Most of the proteins and glycoproteins that constitute the complement system
are synthesized by hepatocytes. But significant amounts are also produced by
tissue macrophages, blood monocytes, and epithelial_cells of the genitourinary
system and gastrointestinal_tract. The three pathways of activation all
generate homologous variants of the protease C3-convertase. The classical
complement pathway typically requires antigen-antibody_complexes for activation
(specific immune response), whereas the alternative pathway can be activated by
spontaneous complement_component_3 (C3) hydrolysis, foreign material,
pathogens, or damaged cells. The mannose-binding lectin pathway can be
activated by C3 hydrolysis or antigens without the presence of antibodies (non-
specific immune response). In all three pathways, C3-convertase cleaves and
activates component C3, creating C3a and C3b, and causes a cascade of further
cleavage and activation events. C3b binds to the surface of pathogens, leading
to greater internalization by phagocytic_cells by opsonization.
In the alternative pathway, C3b binds to Factor B. Factor D releases Factor Ba
from Factor B bound to C3b. The complex of C3b(2)Bb is a protease which cleaves
C5 into C5b and C5a. C5 convertase is also formed by the classical pathway when
C3b binds C4b and C2b. C5a is an important chemotactic_protein, helping recruit
inflammatory cells. C3a is the precursor of an important cytokine (adipokine)
named ASP (although this is not universally accepted [6]) and is usually
rapidly cleaved by carboxypeptidase_B. Both C3a and C5a have anaphylatoxin
activity, directly triggering degranulation of mast_cells as well as increasing
vascular permeability and smooth_muscle contraction.[6] C5b initiates the
membrane_attack_pathway, which results in the membrane_attack_complex (MAC),
consisting of C5b, C6, C7, C8, and polymeric C9.[7] MAC is the cytolytic
endproduct of the complement cascade; it forms a transmembrane channel, which
causes osmotic lysis of the target cell. Kupffer_cells and other macrophage
cell types help clear complement-coated pathogens. As part of the innate immune
system, elements of the complement cascade can be found in species earlier than
vertebrates; most recently in the protostome horseshoe_crab species, putting
the origins of the system back further than was previously thought.
Reaction Cascade of the Complement System: Classical, Alternative and Lectin
Pathway, Amplification Loop, Terminal Pathway, and Membrane Attack Complex.
**** Classical pathway[edit] ****
Main article: Classical_complement_pathway
The classical and alternative complement pathways
The classical_pathway is triggered by activation of the C1-complex. The C1-
complex is composed of 1 molecule of C1q, 2 molecules of C1r and 2 molecules of
C1s, or C1qr2s2. This occurs when C1q binds to IgM or IgG complexed with
antigens. A single pentameric IgM can initiate the pathway, while several,
ideally six, IgGs are needed. This also occurs when C1q binds directly to the
surface of the pathogen. Such binding leads to conformational changes in the
C1q molecule, which leads to the activation of two C1r molecules. C1r is a
serine protease. They then cleave C1s (another serine protease). The C1r2s2
component now splits C4 and then C2, producing C4a, C4b, C2a, and C2b
(historically, the larger fragment of C2 was called C2a but is now referred to
as C2b). C4b and C2b bind to form the classical pathway C3-convertase (C4b2b
complex), which promotes cleavage of C3 into C3a and C3b. C3b later joins with
C4b2b to make C5 convertase (C4b2b3b complex).
**** Alternative pathway[edit] ****
Main article: Alternative_complement_pathway
The alternative_pathway is continuously activated at a low level, analogous to
a car engine at idle, as a result of spontaneous C3 hydrolysis due to the
breakdown of the internal thioester bond (C3 is mildly unstable in aqueous
environment). The alternative pathway does not rely on pathogen-binding
antibodies like the other pathways.[2] C3b that is generated from C3 by a C3
convertase enzyme complex in the fluid phase is rapidly inactivated by factor_H
and factor_I, as is the C3b-like C3 that is the product of spontaneous cleavage
of the internal thioester. In contrast, when the internal thioester of C3
reacts with a hydroxyl or amino group of a molecule on the surface of a cell or
pathogen, the C3b that is now covalently bound to the surface is protected from
factor H-mediated inactivation. The surface-bound C3b may now bind factor_B to
form C3bB. This complex in the presence of factor_D will be cleaved into Ba and
Bb. Bb will remain associated with C3b to form C3bBb, which is the alternative
pathway C3 convertase.[citation_needed]
The C3bBb complex is stabilized by binding oligomers of factor_P (properdin).
The stabilized C3 convertase, C3bBbP, then acts enzymatically to cleave much
more C3, some of which becomes covalently attached to the same surface as C3b.
This newly bound C3b recruits more B, D and P activity and greatly amplifies
the complement activation. When complement is activated on a cell surface, the
activation is limited by endogenous complement regulatory proteins, which
include CD35, CD46, CD55 and CD59, depending on the cell. Pathogens, in
general, don't have complement regulatory proteins (there are many exceptions,
which reflect adaptation of microbial pathogens to vertebrate immune defenses).
Thus, the alternative complement pathway is able to distinguish self from non-
self on the basis of the surface expression of complement regulatory proteins.
Host cells don't accumulate cell surface C3b (and the proteolytic fragment of
C3b called iC3b) because this is prevented by the complement regulatory
proteins, while foreign cells, pathogens and abnormal surfaces may be heavily
decorated with C3b and iC3b. Accordingly, the alternative complement pathway is
one element of innate_immunity.[citation_needed]
Once the alternative C3 convertase enzyme is formed on a pathogen or cell
surface, it may bind covalently another C3b, to form C3bBbC3bP, the C5
convertase. This enzyme then cleaves C5 to C5a, a potent anaphylatoxin, and
C5b. The C5b then recruits and assembles C6, C7, C8 and multiple C9 molecules
to assemble the membrane_attack_complex. This creates a hole or pore in the
membrane that can kill or damage the pathogen or cell.[citation_needed]
**** Lectin pathway[edit] ****
Main article: Lectin_pathway
The lectin pathway is homologous to the classical pathway, but with the
opsonin, mannose-binding_lectin (MBL), and ficolins, instead of C1q. This
pathway is activated by binding of MBL to mannose residues on the pathogen
surface, which activates the MBL-associated serine proteases, MASP-1, and MASP-
2 (very similar to C1r and C1s, respectively), which can then split C4 into C4a
and C4b and C2 into C2a and C2b. C4b and C2b then bind together to form the
classical C3-convertase, as in the classical pathway. Ficolins are homologous
to MBL and function via MASP in a similar way. Several single-nucleotide
polymorphisms have been described in M-ficolin in humans, with effect on
ligand-binding ability and serum levels. Historically, the larger fragment of
C2 was named C2a, but it is now referred to as C2b.[8] In invertebrates without
an adaptive immune system, ficolins are expanded and their binding
specificities diversified to compensate for the lack of pathogen-specific
recognition molecules.
**** Complement protein fragment nomenclature[edit] ****
Immunology textbooks have used different naming assignments for the smaller and
larger fragments of C2 as C2a and C2b. The preferred assignment appears to be
that the smaller fragment be designated as C2a: as early as 1994, a well known
textbook recommended that the larger fragment of C2 should be designated C2b.
[9] However, this was amplified in their 1999 4th edition, to say that:[10] "It
is also useful to be aware that the larger active fragment of C2 was originally
designated C2a, and is still called that in some texts and research papers.
Here, for consistency, we shall call all large fragments of complement b, so
the larger fragment of C2 will be designated C2b. In the classical and lectin
pathways the C3 convertase enzyme is formed from membrane-bound C4b with C2b."
[10]
This nomenclature is used in another literature:[11] "(Note that, in older
texts, the smaller fragment is often called C2b, and the larger one is called
C2a for historical reason.)"[12] The assignment is mixed in the latter
literature, though. Some sources designate the larger and smaller fragments as
C2a and C2b respectively[13][14][15][16][17][18][19][20][21] while other
sources apply the converse.[9][10][22][23][24] However, due to the widely
established convention, C2b here is the larger fragment, which, in the
classical pathway, forms C4b2b (classically C4b2a). It may be noteworthy that,
in a series of editions of Janeway's book, 1st to 7th, in the latest edition
[20] they withdraw the stance to indicate the larger fragment of C2 as C2b.
**** Viral inhibition[edit] ****
Fixation of the MBL protein on viral surfaces has also been shown to enhance
neutralization of viral pathogens.[25]
**** Review[edit] ****
Activation pathway Classic         Alternative            Lectin
Activator          AgâAb Complspontaneous hydrolysis MBL-Mannose Complex
                                   of C3
C3-convertase      C4b2b           C3bBb                  C4b2b
C5-convertase      C4b2b3b         C3bC3bBb               C4b2b3b
MAC development    C5b+C6+C7+C8+C9
***** Activation of complements by antigen-associated antibody[edit] *****
In the classical pathway, C1 binds with its C1q subunits to Fc fragments (made
of CH2 region) of IgG or IgM, which has formed a complex with antigens. C4b and
C3b are also able to bind to antigen-associated IgG or IgM, to its Fc portion.
[11][17][20]
Such immunoglobulin-mediated binding of the complement may be interpreted as
that the complement uses the ability of the immunoglobulin to detect and bind
to non-self antigens as its guiding stick. The complement itself can bind non-
self pathogens after detecting their pathogen-associated_molecular_patterns
(PAMPs),[20] however, utilizing specificity of the antibody, complements can
detect non-self enemies much more specifically.
Some components have a variety of binding sites. In the classical pathway, C4
binds to Ig-associated C1q and C1r2s2 enzyme cleaves C4 to C4b and 4a. C4b
binds to C1q, antigen-associated Ig (specifically to its Fc portion), and even
to the microbe surface. C3b binds to antigen-associated Ig and to the microbe
surface. Ability of C3b to bind to antigen-associated Ig would work effectively
against antigen-antibody complexes to make them soluble.[citation_needed]
***** Regulation[edit] *****
The complement system has the potential to be extremely damaging to host
tissues, meaning its activation must be tightly regulated. The complement
system is regulated by complement_control_proteins, which are present at a
higher concentration in the blood plasma than the complement proteins
themselves. Some complement control proteins are present on the membranes of
self-cells preventing them from being targeted by complement. One example is
CD59, also known as protectin, which inhibits C9 polymerisation during the
formation of the membrane_attack_complex. The classical pathway is inhibited by
C1-inhibitor, which binds to C1 to prevent its activation.[citation_needed]
C3-convertase can be inhibited by Decay_accelerating_factor (DAF), which is
bound to erythrocyte plasma membranes via a GPI anchor.[citation_needed]
***** Role in disease[edit] *****
**** Complement deficiency[edit] ****
Main article: Complement_deficiency
It is thought that the complement system might play a role in many diseases
with an immune component, such as BarraquerâSimons_Syndrome, asthma, lupus
erythematosus, glomerulonephritis, various forms of arthritis, autoimmune_heart
disease, multiple_sclerosis, inflammatory_bowel_disease, paroxysmal_nocturnal
hemoglobinuria, atypical_hemolytic_uremic_syndrome and ischemia-reperfusion
injuries,[26][27] and rejection of transplanted organs.[28]
The complement system is also becoming increasingly implicated in diseases of
the central nervous system such as Alzheimer's_disease and other
neurodegenerative conditions such as spinal cord injuries.[29][30][31]
Deficiencies of the terminal pathway predispose to both autoimmune_disease and
infections (particularly Neisseria_meningitidis, due to the role that the
membrane_attack_complex ("MAC") plays in attacking Gram-negative bacteria).[32]
Infections with N. meningitidis and N._gonorrhoeae are the only conditions
known to be associated with deficiencies in the MAC components of complement.
[33] 40â50% of those with MAC deficiencies experience recurrent infections
with N. meningitidis.[34]
**** Deficiencies in complement regulators[edit] ****
Mutations in the complement regulators factor_H and membrane_cofactor_protein
have been associated with atypical hemolytic_uremic_syndrome.[35][36] Moreover,
a common single_nucleotide_polymorphism in factor H (Y402H) has been associated
with the common eye disease age-related_macular_degeneration.[37] Polymorphisms
of complement_component_3, complement_factor_B, and complement_factor_I, as
well as deletion of complement factor H-related 3 and complement factor H-
related 1 also affect a person's risk of developing age-related_macular
degeneration.[38] Both of these disorders are currently thought to be due to
aberrant complement activation on the surface of host cells.
Mutations in the C1 inhibitor gene can cause hereditary_angioedema, a genetic
condition resulting from reduced regulation of bradykinin by C1-INH.[citation
needed]
Paroxysmal_nocturnal_hemoglobinuria is caused by complement breakdown of RBCs
due to an inability to make GPI. Thus the RBCs are not protected by GPI
anchored proteins such as DAF.[39]
**** Diagnostic tools[edit] ****
Diagnostic tools to measure complement activity include the total_complement
activity test.[40]
The presence or absence of complement fixation upon a challenge can indicate
whether particular antigens or antibodies are present in the blood. This is the
principle of the complement_fixation_test.
***** Modulation by infections[edit] *****
Recent research has suggested that the complement system is manipulated during
HIV/AIDS, in a way that further damages the body.[41]
***** References[edit] *****
   1. ^Janeway, CA Jr; Travers P; Walport M; et al. (2001). "The_complement
      system_and_innate_immunity". Immunobiology: The Immune System in Health
      and Disease. New York: Garland Science. Retrieved 25 February 2013.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a bAbbas AK, Lichtman AH, Pillai S (2010). Cellular and Molecular
      Immunology (6th ed.). Elsevier. pp. 272â288. ISBN 978-1-4160-3123-9.
   4. ^ a bChaplin H (2005). "Review: the burgeoning history of the complement
      system 1888â2005". Immunohematology / American Red Cross. 21 (3):
      85â93. PMID 16178664.
   5. ^Nesargikar PN, Spiller B, Chavez R (2012). "The_complement_system:
      history,_pathways,_cascade_and_inhibitors". European Journal of
      Microbiology & Immunology. 2 (2): 103â11. doi:10.1556/EuJMI.2.2012.2.2.
      PMC 3956958. PMID 24672678.
   6. ^Murphy, Kenneth; Weaver, Casey (2017). "Innate Immunity: the First Lines
      of Defense". Janeway's Immunobiology (9th ed.). Garland Science. p. 49.
      ISBN 978-0-8153-4505-3.
   7. ^ a bKlos, A.; Wende, E.; Wareham, K. J.; Monk, P. N. (2013).
      "International Union of Pharmacology. LXXXVII. Complement Peptide C5a,
      C4a, and C3a Receptors". Pharmacological Reviews. 65 (1): 500â43. doi:
      10.1124/pr.111.005223. PMID 23383423.
   8. ^Goldman AS, Prabhakar BS (1996). "The_Complement_System". In Baron S, et
      al. (eds.). Baron's Medical Microbiology (4th ed.). Univ of Texas Medical
      Branch. ISBN 978-0-9631172-1-2.
   9. ^AmmitzbÃ¸ll, Christian Gytz; KjÃ¦r, Troels RÃ¸nn; Steffensen, Rudi;
      Stengaard-Pedersen, Kristian; Nielsen, Hans JÃ¸rgen; Thiel, Steffen;
      BÃ¸gsted, Martin; Jensenius, Jens Christian (2012). "Non-Synonymous
      Polymorphisms_in_the_FCN1_Gene_Determine_Ligand-Binding_Ability_and_Serum
      Levels_of_M-Ficolin". PLoS ONE. 7 (11): e50585. Bibcode:
      2012PLoSO...750585A. doi:10.1371/journal.pone.0050585. PMC 3509001.
      PMID 23209787.
  10. ^ a b Janeway C, Travers P (1994). Immunobiology : The Immune System in
      Health and Disease. London; San Francisco; New York: Current Biology
      Limited; Garland Pub. Inc.,
  11. ISBN 0-8153-1691-7.[page needed]
  12. ^ a b c Janeway CA, Travers P, Walport M, Capra JD (1999). Immunobiology:
      The Immune System in Health and Disease (4th ed.), 635p. New York:
      Garland Pub,
  13. ISBN 0-8153-3217-3.[page needed]
  14. ^ a b Abbas AK, Lichtman AH (2003). Cellular and Molecular Immunology
      (5th ed.), 563p. Philadelphia: Saunders,
  15. ISBN 0-7216-0008-5.[page needed]
  16. ^ p. 332[full_citation_needed]
  17. ^ Peakman M, Vergani D (1997). Basic and Clinical Immunology. New York:
      Churchill Livingstone,
  18. ISBN 0-443-04672-7.[page needed]
  19. ^ Paul WE (ed.) (1999). Fundamental Immunology (4th ed.), 1589p.
      Philadelphia: Lippincott-Raven,
  20. ISBN 0-7817-1412-5.[page needed]
  21. ^ Sims PJ, Wiedmer T (2000). "Complement biology," In Hoffman R, Benz EJ,
      Shattil SJ, Furie B, Cohen HJ, Silberstein LE, McGlave P, (eds.) (2000).
      Hematology: Basic Principles and Practice, 3rd ed. pp. 651â667, New
      York; Edinburgh: Churchill-Livingstone,
  22. ISBN 0-443-07954-4.
  23. ^ Frank K, Atkinson JP (2001) Complement system. In Austen KF, Frank K,
      Atkinson JP, Cantor H. (ed.) Samter's Immunologic Diseases (6th ed.) Vol.
      1, p. 281â298, Philadelphia: Lippincott Williams & Wilkins,
  24. ISBN 0-7817-2120-2.
  25. ^ a b Roitt I, Brostoff J, Male D (2001). Immunology (6th ed.), 480p. St.
      Louis: Mosby,
  26. ISBN 0-7234-3189-2.[page needed]
  27. ^ Anderson DM (2003) Dorland's Illustrated Medical Dictionary (30th ed.),
      Philadelphia: W.B. Saunders,
  28. ISBN 0-7216-0146-4.[page needed]
  29. ^ Parham P (2005). The Immune System. New York: Garland,
  30. ISBN 0-8153-4093-1.[page needed]
  31. ^ a b c d Murphy K, Travers P, Walport M, with contributions by
      Ehrenstein M et al. (2008). Janeway's Immunobiology (7th ed.), New York:
      Garland Science,
  32. ISBN 0-8153-4123-7.[page needed]
  33. ^ Atkinson JP (2009). "Complement system," In Firestein GS, Budd RC,
      Harris ED Jr, McInnes IB, Ruddy S, Sergent JS, eds. (2009). Kelley's
      Textbook of Rheumatology, pp. 323â336, Philadelphia, PA: Saunders/
      Elsevier,
  34. ISBN 978-1-4160-3285-4.
  35. ^Janeway CA Jr.; Travers P; Walport M; Shlomchik MJ (2001). "The
      complement_system_and_innate_immunity". Immunobiology (5th ed.). Garland
      Publishing. ISBN 978-0-8153-3642-6.
  36. ^ Doan T, Melvold R, Viselli S, Waltenbaugh C (2007). Lippincott's
      Illustrated Reviews: Immunology, 320p. Lippincott Williams & Wilkins
      [page needed]
  37. ^ DeFranco AL, Locksley RM, Robertson M (2007). Immunity : The Immune
      Response in Infectious and Inflammatory Disease. London; Sunderland, MA:
      New Science Press; Sinauer Associates,
  38. ISBN 978-0-9539181-0-2.[page needed]
  39. ^Stoermer, Kristina A.; Morrison, Thomas E. (15 March 2011). "Complement
      and_Viral_Pathogenesis". Virology. 411 (2): 362â373 Central. doi:
      10.1016/j.virol.2010.12.045. PMC 3073741. PMID 21292294.
  40. ^Arumugam, Thiruma V; Shiels, Ian A; Woodruff, Trent M; Granger, D Neil;
      Taylor, Stephen M (2004). "The Role of the Complement System in Ischemia-
      Reperfusion Injury". Shock. 21 (5): 401â9. doi:10.1097/00024382-
      200405000-00002. PMID 15087815.
  41. ^Naesens, M.; Li, L.; Ying, L.; Sansanwal, P.; Sigdel, T. K.; Hsieh, S.-
      C.; Kambham, N.; Lerut, E.; Salvatierra, O.; Butte, A. J.; Sarwal, M. M.
      (2009). "Expression_of_Complement_Components_Differs_Between_Kidney
      Allografts_from_Living_and_Deceased_Donors". Journal of the American
      Society of Nephrology. 20 (8): 1839â51. doi:10.1681/ASN.2008111145.
      PMC 2723986. PMID 19443638.
  42. ^Sacks, Steven H; Chowdhury, Paramit; Zhou, Wuding (2003). "Role of the
      complement system in rejection". Current Opinion in Immunology. 15 (5):
      487â92. doi:10.1016/S0952-7915(03)00100-6. PMID 14499254.
  43. ^Galvan, M. D.; Luchetti, S.; Burgos, A. M.; Nguyen, H. X.; Hooshmand, M.
      J.; Hamers, F. P. T.; Anderson, A. J. (2008). "Deficiency_in_Complement
      C1q_Improves_Histological_and_Functional_Locomotor_Outcome_after_Spinal
      Cord_Injury". Journal of Neuroscience. 28 (51): 13876â88. doi:10.1523/
      JNEUROSCI.2823-08.2008. PMC 2680920. PMID 19091977.
  44. ^Nguyen, Hal X; Galvan, Manuel D; Anderson, Aileen J (2008).
      "Characterization_of_early_and_terminal_complement_proteins_associated
      with_polymorphonuclear_leukocytes_in_vitro_and_in_vivo_after_spinal_cord
      injury". Journal of Neuroinflammation. 5: 26. doi:10.1186/1742-2094-5-26.
      PMC 2443364. PMID 18578885.
  45. ^Beck, K. D.; Nguyen, H. X.; Galvan, M. D.; Salazar, D. L.; Woodruff, T.
      M.; Anderson, A. J. (2010). "Quantitative_analysis_of_cellular
      inflammation_after_traumatic_spinal_cord_injury:_Evidence_for_a
      multiphasic_inflammatory_response_in_the_acute_to_chronic_environment".
      Brain. 133 (2): 433â47. doi:10.1093/brain/awp322. PMC 2858013.
      PMID 20085927.
  46. ^Brown, E. J. (1985). "Interaction of Gram-Positive Microorganisms with
      Complement". Bacteria and Complement. Current Topics in Microbiology and
      Immunology. 121. Springer, Berlin, Heidelberg. pp. 159â187. doi:
      10.1007/978-3-642-45604-6_8. ISBN 9783642456060.
  47. ^Ram S, Lewis LA, Rice PA (2010). "Infections_of_people_with_complement
      deficiencies_and_patients_who_have_undergone_splenectomy". Clinical
      Microbiology Reviews. 23 (4): 740â80. doi:10.1128/CMR.00048-09.
      PMC 2952982. PMID 20930072.
  48. ^Lewis LA, Ram S (2014). "Meningococcal_disease_and_the_complement
      system". Virulence. 5 (1): 98â126. doi:10.4161/viru.26515. PMC 3916388.
      PMID 24104403.
  49. ^Dragon-Durey, Marie-AgnÃ¨s; FrÃ©meaux-Bacchi, VÃ©ronique (2005).
      "Atypical haemolytic uraemic syndrome and mutations in complement
      regulator genes". Springer Seminars in Immunopathology. 27 (3): 359â74.
      doi:10.1007/s00281-005-0003-2. PMID 16189652.
  50. ^Zipfel, Peter; Misselwitz, Joachim; Licht, Christoph; Skerka, Christine
      (2006). "The Role of Defective Complement Control in Hemolytic Uremic
      Syndrome". Seminars in Thrombosis and Hemostasis. 32 (2): 146â54. doi:
      10.1055/s-2006-939770. PMID 16575689.
  51. ^Mooijaart, Simon P.; Koeijvoets, Kristel M.C.; Sijbrands, Eric J.G.;
      Daha, Mohamed R.; Westendorp, Rudi G.J. (2007). "Complement Factor H
      polymorphism Y402H associates with inflammation, visual acuity, and
      cardiovascular mortality in the elderly population at large".
      Experimental Gerontology. 42 (11): 1116â22. doi:10.1016/
      j.exger.2007.08.001. PMID 17869048.
  52. ^Bradley, D T; Zipfel, P F; Hughes, A E (2011). "Complement_in_age-
      related_macular_degeneration:_A_focus_on_function". Eye. 25 (6):
      683â93. doi:10.1038/eye.2011.37. PMC 3178140. PMID 21394116.
  53. ^Parker, C.; Omine, M.; Richards, S.; et al. (2005). "Diagnosis_and
      Management_of_Paroxysmal_Nocturnal_Hemoglobinuria". Blood. 106 (12):
      3699â3709. doi:10.1182/blood-2005-04-1717. PMC 1895106. PMID 16051736.
  54. ^"Complement_Deficiencies_Workup:_Laboratory_Studies,_Imaging_Studies,
      Other_Tests". emedicine.medscape.com. Retrieved 2018-04-26.
  55. ^Datta, P.K.; Rappaport, J. (2006). "HIV and complement: Hijacking an
      immune defense". Biomedicine & Pharmacotherapy. 60 (9): 561â8. doi:
      10.1016/j.biopha.2006.07.087. PMID 16978830.
***** External links[edit] *****
    *  Media related to Complement_system at Wikimedia Commons
    * v
    * t
    * e
Complement system
                         * C
Pathways                 * L
                         * A
                                * C: C1
                                      o C1q
                                      o C1r
                                      o C1s
                                * C4
                                      o C4a
                     Early            o C4b
                                * C2
                                * L: MASP1/MASP2
                                * MBL
                                * A: Factor_B
                                * Factor_D
                                * Factor_P/Properdin
Activators/enzymes              * C3
                                      o C3a
                                      o C3b/iC3b
                     Middle     * C5
                                      o C5a
                                      o C5b
                                * C3-convertase
                                * C5-convertase
                                * MAC
                                      o C5b
                     Late             o C6
                                      o C7
                                      o C8
                                      o C9
                         * CLA: C1-inhibitor
                         * Decay-accelerating_factor/CD59
Inhibitors               * Factor_I
                         * CL: C4BP
                         * A: Factor_H
                         * CR1
                         * CR2
                         * CR3
Complement_receptors     * CR4
                         * CD11b/CD11c/CD18
                         * Anaphylatoxin
                               o C3a
                               o C5a
                         * Cytotoxicity(by MAC)
Function                 * immune_adherence
                         * Inducing inflammation
                         * Opsonization
    * v
    * t
    * e
Immunology: lymphocytic adaptive_immune_system and complement
                               * Antigen
                                     o Superantigen
                                     o Allergen
                                     o Antigenic_variation
                               * Hapten
                               * Epitope
            Antigens                 o Linear
                                     o Conformational
                               * Mimotope
                               * Antigen_presentation/Professional_APCs:
                                 Dendritic_cell
                               * Macrophage
                               * B_cell
                               * Immunogen
                               * Antibody
                                     o Monoclonal_antibodies
                                     o Polyclonal_antibodies
                                     o Autoantibody
                                     o Microantibody
            Antibodies         * Polyclonal_B_cell_response
                               * Allotype
                               * Isotype
                               * Idiotype
Lymphoid                       * Immune_complex
                               * Paratope
                               * action: Immunity
                               * Autoimmunity
                               * Alloimmunity
                               * Allergy
                               * Hypersensitivity
                               * Inflammation
            Immunity vs.       * Cross-reactivity
            tolerance          * inaction: Tolerance
                                     o Central
                                     o Peripheral
                                     o Clonal_anergy
                                     o Clonal_deletion
                                     o Tolerance_in_pregnancy
                               * Immunodeficiency
                               * Immune_privilege
                               * Affinity_maturation
                                     o Somatic_hypermutation
                                     o Clonal_selection
            Immunogenetics     * V(D)J_recombination
                               * Junctional_diversity
                               * Immunoglobulin_class_switching
                               * MHC/HLA
                * Cellular
                      o T_cell
Lymphocytes     * Humoral
                      o B_cell
                * NK_cell
                * Cytokines
Substances      * Opsonin
                * Cytolysin
Authority_control [Edit_this_at_Wikidata]     * GND: 4120589-3
                                              * NDL: 00563548

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Complement_system&oldid=908301041"
Categories:
    * Complement_system
    * Immune_system
Hidden categories:
    * Wikipedia_articles_needing_page_number_citations_from_May_2015
    * Articles_with_incomplete_citations_from_May_2015
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_May_2015
    * Commons_category_link_from_Wikidata
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * AzÉrbaycanca
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * á¥áá áá£áá
    * KurdÃ®
    * Magyar
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ç²µèª
    * ä¸­æ
Edit_links
    * This page was last edited on 28 July 2019, at 20:59 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
