The following text has been accessed from https://en.wikipedia.org/wiki/DNA_replication at Fri Aug 9 03:35:04 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** DNA replication ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
DNA replication: The double_helix is un'zipped' and unwound, then each
separated strand (turquoise) acts as a template for replicating a new partner
strand (green). Nucleotides (bases) are matched to synthesize the new partner
strands into two new double helices.
In molecular_biology, DNA replication is the biological_process of producing
two identical replicas of DNA from one original DNA molecule. DNA replication
occurs in all living_organisms acting as the basis for biological_inheritance.
The cell possesses the distinctive property of division, which makes
replication of DNA essential.
DNA is made up of a double_helix of two complementary strands. During
replication, these strands are separated. Each strand of the original DNA
molecule then serves as a template for the production of its counterpart, a
process referred to as semiconservative_replication. As a result of semi-
conservative replication, the new helix will be composed of an original DNA
strand as well as a newly synthesized strand.[1] Cellular proofreading and
error-checking mechanisms ensure near perfect fidelity for DNA replication.[2]
[3]
In a cell, DNA replication begins at specific locations, or origins_of
replication, in the genome.[4] Unwinding of DNA at the origin and synthesis of
new strands, accommodated by an enzyme known as helicase, results in
replication_forks growing bi-directionally from the origin. A number of
proteins are associated with the replication fork to help in the initiation and
continuation of DNA_synthesis. Most prominently, DNA_polymerase synthesizes the
new strands by adding nucleotides that complement each (template) strand. DNA
replication occurs during the S-stage of interphase.
DNA replication (DNA amplification) can also be performed in_vitro
(artificially, outside a cell). DNA polymerases isolated from cells and
artificial DNA primers can be used to start DNA synthesis at known sequences in
a template DNA molecule. Polymerase_chain_reaction (PCR), ligase_chain_reaction
(LCR), and transcription-mediated_amplification (TMA) are examples.
⁰
***** Contents *****
    * 1_DNA_structure
    * 2_DNA_polymerase
    * 3_Replication_process
          o 3.1_Initiation
          o 3.2_Elongation
          o 3.3_Replication_fork
                # 3.3.1_Leading_strand
                # 3.3.2_Lagging_strand
                # 3.3.3_Dynamics_at_the_replication_fork
          o 3.4_DNA_replication_proteins
          o 3.5_Replication_machinery
          o 3.6_Termination
    * 4_Regulation
          o 4.1_Eukaryotes
                # 4.1.1_Replication_focus
          o 4.2_Bacteria
    * 5_Polymerase_chain_reaction
    * 6_See_also
    * 7_Notes
    * 8_References
***** DNA structure[edit] *****
DNA exists as a double-stranded structure, with both strands coiled together to
form the characteristic double-helix. Each single strand of DNA is a chain of
four types of nucleotides. Nucleotides in DNA contain a deoxyribose sugar, a
phosphate, and a nucleobase. The four types of nucleotide correspond to the
four nucleobases adenine, cytosine, guanine, and thymine, commonly abbreviated
as A, C, G and T. Adenine and guanine are purine bases, while cytosine and
thymine are pyrimidines. These nucleotides form phosphodiester_bonds, creating
the phosphate-deoxyribose backbone of the DNA double helix with the nucleobases
pointing inward (i.e., toward the opposing strand). Nucleobases are matched
between strands through hydrogen_bonds to form base_pairs. Adenine pairs with
thymine (two hydrogen bonds), and guanine pairs with cytosine (three hydrogen
bonds).
DNA_strands_have_a_directionality, and the different ends of a single strand
are called the "3â² (three-prime) end" and the "5â² (five-prime) end". By
convention, if the base sequence of a single strand of DNA is given, the left
end of the sequence is the 5â² end, while the right end of the sequence is the
3â² end. The strands of the double helix are anti-parallel with one being 5â²
to 3â², and the opposite strand 3â² to 5â². These terms refer to the carbon
atom in deoxyribose to which the next phosphate in the chain attaches.
Directionality has consequences in DNA synthesis, because DNA polymerase can
synthesize DNA in only one direction by adding nucleotides to the 3â² end of a
DNA strand.
The pairing of complementary bases in DNA (through hydrogen_bonding) means that
the information contained within each strand is redundant. Phosphodiester
(intra-strand) bonds are stronger than hydrogen (inter-strand) bonds. This
allows the strands to be separated from one another. The nucleotides on a
single strand can therefore be used to reconstruct nucleotides on a newly
synthesized partner strand.[5]
***** DNA polymerase[edit] *****
Main article: DNA_polymerase
DNA polymerases adds nucleotides to the 3â² end of a strand of DNA.[6] If a
mismatch is accidentally incorporated, the polymerase is inhibited from further
extension. Proofreading removes the mismatched nucleotide and extension
continues.
DNA_polymerases are a family of enzymes that carry out all forms of DNA
replication.[7] DNA polymerases in general cannot initiate synthesis of new
strands, but can only extend an existing DNA or RNA strand paired with a
template strand. To begin synthesis, a short fragment of RNA, called a primer,
must be created and paired with the template DNA strand.
DNA polymerase adds a new strand of DNA by extending the 3â² end of an
existing nucleotide chain, adding new nucleotides matched to the template
strand one at a time via the creation of phosphodiester_bonds. The energy for
this process of DNA polymerization comes from hydrolysis of the high-energy
phosphate (phosphoanhydride) bonds between the three phosphates attached to
each unincorporated base. Free bases with their attached phosphate groups are
called nucleotides; in particular, bases with three attached phosphate groups
are called nucleoside_triphosphates. When a nucleotide is being added to a
growing DNA strand, the formation of a phosphodiester bond between the proximal
phosphate of the nucleotide to the growing chain is accompanied by hydrolysis
of a high-energy phosphate bond with release of the two distal phosphates as a
pyrophosphate. Enzymatic hydrolysis of the resulting pyrophosphate into
inorganic phosphate consumes a second high-energy phosphate bond and renders
the reaction effectively irreversible.[Note_1]
In general, DNA polymerases are highly accurate, with an intrinsic error rate
of less than one mistake for every 107 nucleotides added.[8] In addition, some
DNA polymerases also have proofreading ability; they can remove nucleotides
from the end of a growing strand in order to correct mismatched bases. Finally,
post-replication mismatch repair mechanisms monitor the DNA for errors, being
capable of distinguishing mismatches in the newly synthesized DNA strand from
the original strand sequence. Together, these three discrimination steps enable
replication fidelity of less than one mistake for every 109 nucleotides added.
[8]
The rate of DNA replication in a living cell was first measured as the rate of
phage T4 DNA elongation in phage-infected E. coli.[9] During the period of
exponential DNA increase at 37 Â°C, the rate was 749 nucleotides per second.
The mutation rate per base pair per replication during phage T4 DNA synthesis
is 1.7 per 108.[10]
***** Replication process[edit] *****
Main articles: Prokaryotic_DNA_replication and Eukaryotic_DNA_replication
DNA replication, like all biological polymerization processes, proceeds in
three enzymatically catalyzed and coordinated steps: initiation, elongation and
termination.
**** Initiation[edit] ****
Role of initiators for initiation of DNA replication.
Formation of pre-replication complex.
For a cell_to_divide, it must first replicate its DNA.[11] This process is
initiated at particular points in the DNA, known as "origins", which are
targeted by initiator_proteins.[4] In E._coli this protein is DnaA; in yeast,
this is the origin_recognition_complex.[12] Sequences used by initiator
proteins tend to be "AT-rich" (rich in adenine and thymine bases), because A-
T base pairs have two hydrogen bonds (rather than the three formed in a C-
G pair) and thus are easier to strand-separate.[13] Once the origin has been
located, these initiators recruit other proteins and form the pre-replication
complex, which unwinds the double-stranded DNA.
**** Elongation[edit] ****
DNA polymerase has 5â²â3â² activity. All known DNA replication systems
require a free 3â² hydroxyl group before synthesis can be initiated (note: the
DNA template is read in 3â² to 5â² direction whereas a new strand is
synthesized in the 5â² to 3â² directionâthis is often confused). Four
distinct mechanisms for DNA synthesis are recognized:
   1. All cellular life forms and many DNA viruses, phages and plasmids use a
      primase to synthesize a short RNA primer with a free 3â² OH group which
      is subsequently elongated by a DNA polymerase.
   2. The retroelements (including retroviruses) employ a transfer RNA that
      primes DNA replication by providing a free 3â² OH that is used for
      elongation by the reverse_transcriptase.
   3. In the adenoviruses and the Ï29 family of bacteriophages, the 3â² OH
      group is provided by the side chain of an amino acid of the genome
      attached protein (the terminal protein) to which nucleotides are added by
      the DNA polymerase to form a new strand.
   4. In the single stranded DNA virusesâa group that includes the
      circoviruses, the geminiviruses, the parvoviruses and othersâand also
      the many phages and plasmids that use the rolling circle replication
      (RCR) mechanism, the RCR endonuclease creates a nick in the genome strand
      (single stranded viruses) or one of the DNA strands (plasmids). The 5â²
      end of the nicked strand is transferred to a tyrosine residue on the
      nuclease and the free 3â² OH group is then used by the DNA polymerase to
      synthesize the new strand.
The first is the best known of these mechanisms and is used by the cellular
organisms. In this mechanism, once the two strands are separated, primase adds
RNA primers to the template strands. The leading strand receives one RNA primer
while the lagging strand receives several. The leading strand is continuously
extended from the primer by a DNA polymerase with high processivity, while the
lagging strand is extended discontinuously from each primer forming Okazaki
fragments. RNase removes the primer RNA fragments, and a low processivity DNA
polymerase distinct from the replicative polymerase enters to fill the gaps.
When this is complete, a single nick on the leading strand and several nicks on
the lagging strand can be found. Ligase works to fill these nicks in, thus
completing the newly replicated DNA molecule.
The primase used in this process differs significantly between bacteria and
archaea/eukaryotes. Bacteria use a primase belonging to the DnaG protein
superfamily which contains a catalytic domain of the TOPRIM fold type.[14] The
TOPRIM fold contains an Î±/Î² core with four conserved strands in a Rossmann-
like topology. This structure is also found in the catalytic domains of
topoisomerase Ia, topoisomerase II, the OLD-family nucleases and DNA repair
proteins related to the RecR protein.
The primase used by archaea and eukaryotes, in contrast, contains a highly
derived version of the RNA_recognition_motif (RRM). This primase is
structurally similar to many viral RNA-dependent RNA polymerases, reverse
transcriptases, cyclic nucleotide generating cyclases and DNA polymerases of
the A/B/Y families that are involved in DNA replication and repair. In
eukaryotic replication, the primase forms a complex with Pol Î±.[15]
Multiple DNA polymerases take on different roles in the DNA replication
process. In E._coli, DNA_Pol_III is the polymerase enzyme primarily responsible
for DNA replication. It assembles into a replication complex at the replication
fork that exhibits extremely high processivity, remaining intact for the entire
replication cycle. In contrast, DNA_Pol_I is the enzyme responsible for
replacing RNA primers with DNA. DNA Pol I has a 5â² to 3â² exonuclease
activity in addition to its polymerase activity, and uses its exonuclease
activity to degrade the RNA primers ahead of it as it extends the DNA strand
behind it, in a process called nick_translation. Pol I is much less processive
than Pol III because its primary function in DNA replication is to create many
short DNA regions rather than a few very long regions.
In eukaryotes, the low-processivity enzyme, Pol Î±, helps to initiate
replication because it forms a complex with primase.[16] In eukaryotes, leading
strand synthesis is thought to be conducted by Pol Îµ; however, this view has
recently been challenged, suggesting a role for Pol Î´.[17] Primer removal is
completed Pol Î´[18] while repair of DNA during replication is completed by Pol
Îµ.
As DNA synthesis continues, the original DNA strands continue to unwind on each
side of the bubble, forming a replication_fork with two prongs. In bacteria,
which have a single origin of replication on their circular chromosome, this
process creates a "theta_structure" (resembling the Greek letter theta: Î¸). In
contrast, eukaryotes have longer linear chromosomes and initiate replication at
multiple origins within these.[19]
**** Replication fork[edit] ****
Scheme of the replication fork.
a: template, b: leading strand, c: lagging strand, d: replication fork, e:
primer, f: Okazaki_fragments
Many enzymes are involved in the DNA replication fork.
The replication fork is a structure that forms within the long helical DNA
during DNA replication. It is created by helicases, which break the hydrogen
bonds holding the two DNA strands together in the helix. The resulting
structure has two branching "prongs", each one made up of a single strand of
DNA. These two strands serve as the template for the leading and lagging
strands, which will be created as DNA polymerase matches complementary
nucleotides to the templates; the templates may be properly referred to as the
leading strand template and the lagging strand template.
DNA is always synthesized in the 5â² to 3â² direction. Since the leading and
lagging strand templates are oriented in opposite directions at the replication
fork, a major issue is how to achieve synthesis of nascent (new) lagging strand
DNA, whose direction of synthesis is opposite to the direction of the growing
replication fork.
*** Leading strand[edit] ***
The leading strand is the strand of nascent DNA which is synthesized in the
same direction as the growing replication fork. This sort of DNA replication is
continuous.
*** Lagging strand[edit] ***
The lagging strand is the strand of nascent DNA whose direction of synthesis is
opposite to the direction of the growing replication fork. Because of its
orientation, replication of the lagging strand is more complicated as compared
to that of the leading strand. As a consequence, the DNA polymerase on this
strand is seen to "lag behind" the other strand.
The lagging strand is synthesized in short, separated segments. On the lagging
strand template, a primase "reads" the template DNA and initiates synthesis of
a short complementary RNA primer. A DNA polymerase extends the primed segments,
forming Okazaki_fragments. The RNA primers are then removed and replaced with
DNA, and the fragments of DNA are joined together by DNA_ligase.
*** Dynamics at the replication fork[edit] ***
The assembled human DNA clamp, a trimer of the protein PCNA.
In all cases the helicase is composed of six polypeptides that wrap around only
one strand of the DNA being replicated. The two polymerases are bound to the
helicase heximer. In eukaryotes the helicase wraps around the leading strand,
and in prokaryotes it wraps around the lagging strand.[20]
As helicase unwinds DNA at the replication fork, the DNA ahead is forced to
rotate. This process results in a build-up of twists in the DNA ahead.[21] This
build-up forms a torsional resistance that would eventually halt the progress
of the replication fork. Topoisomerases are enzymes that temporarily break the
strands of DNA, relieving the tension caused by unwinding the two strands of
the DNA helix; topoisomerases (including DNA_gyrase) achieve this by adding
negative supercoils to the DNA helix.[22]
Bare single-stranded DNA tends to fold back on itself forming secondary
structures; these structures can interfere with the movement of DNA polymerase.
To prevent this, single-strand_binding_proteins bind to the DNA until a second
strand is synthesized, preventing secondary structure formation.[23]
Clamp_proteins form a sliding clamp around DNA, helping the DNA polymerase
maintain contact with its template, thereby assisting with processivity. The
inner face of the clamp enables DNA to be threaded through it. Once the
polymerase reaches the end of the template or detects double-stranded DNA, the
sliding clamp undergoes a conformational change that releases the DNA
polymerase. Clamp-loading proteins are used to initially load the clamp,
recognizing the junction between template and RNA primers.[3]:274-5
**** DNA replication proteins[edit] ****
At the replication fork, many replication enzymes assemble on the DNA into a
complex molecular machine called the replisome. The following is a list of
major DNA replication enzymes that participate in the replisome:[24]
Enzyme        Function in DNA replication
DNA_Helicase  Also known as helix destabilizing enzyme. Helicase separates the two strands of DNA
              at the Replication_Fork behind the topoisomerase.
              The enzyme responsible for catalyzing the addition of nucleotide substrates to DNA
DNA           in the 5â² to 3â² direction during DNA replication. Also performs proof-reading
Polymerase    and error correction. There exist many different types of DNA Polymerase, each of
              which perform different functions in different types of cells.
DNA_clamp     A protein which prevents elongating DNA polymerases from dissociating from the DNA
              parent strand.
Single-Strand Bind to ssDNA and prevent the DNA double helix from re-annealing after DNA helicase
Binding_(SSB) unwinds it, thus maintaining the strand separation, and facilitating the synthesis
Proteins      of the nascent strand.
Topoisomerase Relaxes the DNA from its super-coiled nature.
DNA_Gyrase    Relieves strain of unwinding by DNA helicase; this is a specific type of
              topoisomerase
DNA_Ligase    Re-anneals the semi-conservative strands and joins Okazaki_Fragments of the lagging
              strand.
Primase       Provides a starting point of RNA (or DNA) for DNA polymerase to begin synthesis of
              the new DNA strand.
              Lengthens telomeric DNA by adding repetitive nucleotide sequences to the ends of
Telomerase    eukaryotic_chromosomes. This allows germ cells and stem cells to avoid the Hayflick
              limit on cell division.[25]
**** Replication machinery[edit] ****
Replication machineries consist of factors involved in DNA replication and
appearing on template ssDNAs. Replication machineries include primosotors are
replication enzymes; DNA polymerase, DNA helicases, DNA clamps and DNA
topoisomerases, and replication proteins; e.g. single-stranded DNA binding
proteins (SSB). In the replication machineries these components coordinate. In
most of the bacteria, all of the factors involved in DNA replication are
located on replication forks and the complexes stay on the forks during DNA
replication. These replication machineries are called replisomes or DNA
replicase systems. These terms are generic terms for proteins located on
replication forks. In eukaryotic and some bacterial cells the replisomes are
not formed.
Since replication machineries do not move relatively to template DNAs such as
factories, they are called a replication factory.[26] In an alternative figure,
DNA factories are similar to projectors and DNAs are like as cinematic films
passing constantly into the projectors. In the replication factory model, after
both DNA helicases for leading strands and lagging strands are loaded on the
template DNAs, the helicases run along the DNAs into each other. The helicases
remain associated for the remainder of replication process. Peter Meister et
al. observed directly replication sites in budding_yeast by monitoring green
fluorescent_protein(GFP)-tagged DNA polymerases Î±. They detected DNA
replication of pairs of the tagged loci spaced apart symmetrically from a
replication origin and found that the distance between the pairs decreased
markedly by time.[27] This finding suggests that the mechanism of DNA
replication goes with DNA factories. That is, couples of replication factories
are loaded on replication origins and the factories associated with each other.
Also, template DNAs move into the factories, which bring extrusion of the
template ssDNAs and nascent DNAs. Meisterâs finding is the first direct
evidence of replication factory model. Subsequent research has shown that DNA
helicases form dimers in many eukaryotic cells and bacterial replication
machineries stay in single intranuclear location during DNA synthesis.[26]
The replication factories perform disentanglement of sister chromatids. The
disentanglement is essential for distributing the chromatids into daughter
cells after DNA replication. Because sister chromatids after DNA replication
hold each other by Cohesin rings, there is the only chance for the
disentanglement in DNA replication. Fixing of replication machineries as
replication factories can improve the success rate of DNA replication. If
replication forks move freely in chromosomes, catenation of nuclei is
aggravated and impedes mitotic segregation.[27]
**** Termination[edit] ****
Eukaryotes initiate DNA replication at multiple points in the chromosome, so
replication forks meet and terminate at many points in the chromosome. Because
eukaryotes have linear chromosomes, DNA replication is unable to reach the very
end of the chromosomes. Due to this problem, DNA is lost in each replication
cycle from the end of the chromosome. Telomeres are regions of repetitive DNA
close to the ends and help prevent loss of genes due to this shortening.
Shortening of the telomeres is a normal process in somatic_cells. This shortens
the telomeres of the daughter DNA chromosome. As a result, cells can only
divide a certain number of times before the DNA loss prevents further division.
(This is known as the Hayflick_limit.) Within the germ_cell line, which passes
DNA to the next generation, telomerase extends the repetitive sequences of the
telomere region to prevent degradation. Telomerase can become mistakenly active
in somatic cells, sometimes leading to cancer formation. Increased telomerase
activity is one of the hallmarks of cancer.
Termination requires that the progress of the DNA replication fork must stop or
be blocked. Termination at a specific locus, when it occurs, involves the
interaction between two components: (1) a termination site sequence in the DNA,
and (2) a protein which binds to this sequence to physically stop DNA
replication. In various bacterial species, this is named the DNA replication
terminus site-binding protein, or Ter_protein.
Because bacteria have circular chromosomes, termination of replication occurs
when the two replication forks meet each other on the opposite end of the
parental chromosome. E. coli regulates this process through the use of
termination sequences that, when bound by the Tus_protein, enable only one
direction of replication fork to pass through. As a result, the replication
forks are constrained to always meet within the termination region of the
chromosome.[28]
***** Regulation[edit] *****
Main articles: Cell_division and Cell_cycle
The cell cycle of eukaryotic cells.
**** Eukaryotes[edit] ****
Within eukaryotes, DNA replication is controlled within the context of the cell
cycle. As the cell grows and divides, it progresses through stages in the cell
cycle; DNA replication takes place during the S phase (synthesis phase). The
progress of the eukaryotic cell through the cycle is controlled by cell_cycle
checkpoints. Progression through checkpoints is controlled through complex
interactions between various proteins, including cyclins and cyclin-dependent
kinases.[29] Unlike bacteria, eukaryotic DNA replicates in the confines of the
nucleus.[30]
The G1/S checkpoint (or restriction checkpoint) regulates whether eukaryotic
cells enter the process of DNA replication and subsequent division. Cells that
do not proceed through this checkpoint remain in the G0 stage and do not
replicate their DNA.
Replication of chloroplast and mitochondrial genomes occurs independently of
the cell cycle, through the process of D-loop_replication.
*** Replication focus[edit] ***
In vertebrate cells, replication sites concentrate into positions called
replication foci.[27] Replication sites can be detected by immunostaining
daughter strands and replication enzymes and monitoring GFP-tagged replication
factors. By these methods it is found that replication foci of varying size and
positions appear in S phase of cell division and their number per nucleus is
far smaller than the number of genomic replication forks.
P. Heun et al.(2001) tracked GFP-tagged replication foci in budding yeast cells
and revealed that replication origins move constantly in G1 and S phase and the
dynamics decreased significantly in S phase.[27] Traditionally, replication
sites were fixed on spatial structure of chromosomes by nuclear_matrix or
lamins. The Heunâs results denied the traditional concepts, budding yeasts
don't have lamins, and support that replication origins self-assemble and form
replication foci.
By firing of replication origins, controlled spatially and temporally, the
formation of replication foci is regulated. D. A. Jackson et al.(1998) revealed
that neighboring origins fire simultaneously in mammalian cells.[27] Spatial
juxtaposition of replication sites brings clustering of replication forks. The
clustering do rescue of stalled replication forks and favors normal progress of
replication forks. Progress of replication forks is inhibited by many factors;
collision with proteins or with complexes binding strongly on DNA, deficiency
of dNTPs, nicks on template DNAs and so on. If replication forks stall and the
remaining sequences from the stalled forks are not replicated, the daughter
strands have nick obtained un-replicated sites. The un-replicated sites on one
parent's strand hold the other strand together but not daughter strands.
Therefore, the resulting sister chromatids cannot separate from each other and
cannot divide into 2 daughter cells. When neighboring origins fire and a fork
from one origin is stalled, fork from other origin access on an opposite
direction of the stalled fork and duplicate the un-replicated sites. As other
mechanism of the rescue there is application of dormant replication origins
that excess origins don't fire in normal DNA replication.
**** Bacteria[edit] ****
Dam methylates adenine of GATC sites after replication.
Most bacteria do not go through a well-defined cell cycle but instead
continuously copy their DNA; during rapid growth, this can result in the
concurrent occurrence of multiple rounds of replication.[31] In E. coli, the
best-characterized bacteria, DNA replication is regulated through several
mechanisms, including: the hemimethylation and sequestering of the origin
sequence, the ratio of adenosine_triphosphate_(ATP) to adenosine_diphosphate_
(ADP), and the levels of protein DnaA. All these control the binding of
initiator proteins to the origin sequences.
Because E. coli methylates GATC DNA sequences, DNA synthesis results in
hemimethylated sequences. This hemimethylated DNA is recognized by the protein
SeqA, which binds and sequesters the origin sequence; in addition, DnaA
(required for initiation of replication) binds less well to hemimethylated DNA.
As a result, newly replicated origins are prevented from immediately initiating
another round of DNA replication.[32]
ATP builds up when the cell is in a rich medium, triggering DNA replication
once the cell has reached a specific size. ATP competes with ADP to bind to
DnaA, and the DnaA-ATP complex is able to initiate replication. A certain
number of DnaA proteins are also required for DNA replication â each time the
origin is copied, the number of binding sites for DnaA doubles, requiring the
synthesis of more DnaA to enable another initiation of replication.
In fast-growing bacteria, such as E. coli, chromosome replication takes more
time than dividing the cell. The bacteria solve this by initiating a new round
of replication before the previous one has been terminated.[33] The new round
of replication will form the chromosome of the cell that is born two
generations after the dividing cell. This mechanism creates overlapping
replication cycles.
See also: Plasmid, Plasmid_copy_number, and Plasmid_partition_system
***** Polymerase chain reaction[edit] *****
Main article: Polymerase_chain_reaction
Researchers commonly replicate DNA in vitro using the polymerase_chain_reaction
(PCR). PCR uses a pair of primers to span a target region in template DNA, and
then polymerizes partner strands in each direction from these primers using a
thermostable DNA_polymerase. Repeating this process through multiple cycles
amplifies the targeted DNA region. At the start of each cycle, the mixture of
template and primers is heated, separating the newly synthesized molecule and
template. Then, as the mixture cools, both of these become templates for
annealing of new primers, and the polymerase extends from these. As a result,
the number of copies of the target region doubles each round, increasing
exponentially.[34]
***** See also[edit] *****
 Wikimedia Commons has media related to DNA_replication.
 Wikiversity has learning resources about DNA#DNA_Replication
    * Autopoiesis
    * Chromosome_segregation
    * Data_storage_device
    * Self-replication
***** Notes[edit] *****
   1. ^ The energetics of this process may also help explain the directionality
      of synthesisâif DNA were synthesized in the 3â² to 5â² direction, the
      energy for the process would come from the 5â² end of the growing strand
      rather than from free nucleotides. The problem is that if the high energy
      triphosphates were on the growing strand and not on the free nucleotides,
      proof-reading by removing a mismatched terminal nucleotide would be
      problematic: Once a nucleotide is added, the triphosphate is lost and a
      single phosphate remains on the backbone between the new nucleotide and
      the rest of the strand. If the added nucleotide were mismatched, removal
      would result in a DNA strand terminated by a monophosphate at the end of
      the "growing strand" rather than a high energy triphosphate. So strand
      would be stuck and wouldn't be able to grow anymore. In actuality, the
      high energy triphosphates hydrolyzed at each step originate from the free
      nucleotides, not the polymerized strand, so this issue does not exist.
***** References[edit] *****
   1. ^Pray, Leslie A. "Semi-Conservative_DNA_Replication;_Meselson_and_Stahl".
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ Imperfect DNA replication results in mutations.Berg JM, Tymoczko JL,
      Stryer L, Clarke ND (2002). "Chapter_27:_DNA_Replication,_Recombination,
      and_Repair". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.
   4. [dead_link]
   5. ^ a bAlberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002).
      "5DNA_Replication,_Repair,_and_Recombination". Molecular Biology of the
      Cell. Garland Science. ISBN 0-8153-3218-1.
   6. [dead_link]
   7. ^ a bBerg JM, Tymoczko JL, Stryer L, Clarke ND (2002). "Chapter_27,
      Section_4:_DNA_Replication_of_Both_Strands_Proceeds_Rapidly_from_Specific
      Start_Sites". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.
   8. [dead_link]
   9. ^ Alberts, B., et al., Molecular Biology of the Cell, Garland Science,
      4th ed., 2002, pp. 238â240
  10. ISBN 0-8153-3218-1
  11. ^ Allison, Lizabeth A. Fundamental Molecular Biology. Blackwell
      Publishing. 2007. p.112
  12. ISBN 978-1-4051-0379-4
  13. ^Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). Biochemistry. W.H.
      Freeman and Company. ISBN 0-7167-3051-0.
  14.  Chapter_27,_Section_2:_DNA_Polymerases_Require_a_Template_and_a_Primer
  15. ^ a bMcCulloch, Scott D; Kunkel, Thomas A (January 2008). "The_fidelity
      of_DNA_synthesis_by_eukaryotic_replicative_and_translesion_synthesis
      polymerases". Cell Research. 18 (1): 148â161. doi:10.1038/cr.2008.4.
      PMC 3639319. PMID 18166979.
  16. ^McCarthy, David; Minner, Charles; Bernstein, Harris; Bernstein, Carol
      (October 1976). "DNA elongation rates and growing point distributions of
      wild-type phage T4 and a DNA-delay amber mutant". Journal of Molecular
      Biology. 106 (4): 963â981. doi:10.1016/0022-2836(76)90346-6.
      PMID 789903.
  17. ^ Drake JW (1970) The Molecular Basis of Mutation. Holden-Day, San
      Francisco
  18. ISBN 0816224501
  19. ISBN 978-0816224500
  20. ^Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002).
      Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.
  21.  Chapter_5:_DNA_Replication_Mechanisms
  22. ^Weigel C, Schmidt A, RÃ¼ckert B, Lurz R, Messer W (November 1997). "DnaA
      protein_binding_to_individual_DnaA_boxes_in_the_Escherichia_coli
      replication_origin,_oriC". The EMBO Journal. 16 (21): 6574â83. doi:
      10.1093/emboj/16.21.6574. PMC 1170261. PMID 9351837.
  23. ^Lodish H, Berk A, Zipursky LS, Matsudaira P, Baltimore D, Darnell J
      (2000). Molecular Cell Biology. W. H. Freeman and Company. ISBN 0-7167-
      3136-3.
  24. 12.1._General_Features_of_Chromosomal_Replication:_Three_Common_Features
      of_Replication_Origins
  25. ^Aravind, L.; Leipe, D. D.; Koonin, E. V. (1998). "Toprimâa_conserved
      catalytic_domain_in_type_IA_and_II_topoisomerases,_DnaG-type_primases,
      OLD_family_nucleases_and_RecR_proteins". Nucleic Acids Research. 26 (18):
      4205â4213. doi:10.1093/nar/26.18.4205. PMC 147817. PMID 9722641.
  26. ^Frick, David; Richardson, Charles (July 2001). "DNA Primases". Annual
      Review of Biochemistry. 70: 39â80. doi:10.1146/annurev.biochem.70.1.39.
      PMID 11395402.
  27. ^Barry, Elizabeth R.; Bell, Stephen D. (8 December 2006). "DNA
      Replication_in_the_Archaea". Microbiology and Molecular Biology Reviews.
      70 (4): 876â887. doi:10.1128/MMBR.00029-06. PMC 1698513. PMID 17158702.
  28. ^Stillman, Bruce (July 2015). "Reconsidering_DNA_Polymerases_at_the
      Replication_Fork_in_Eukaryotes". Molecular Cell. 59 (2): 139â141. doi:
      10.1016/j.molcel.2015.07.004. PMC 4636199. PMID 26186286.
  29. ^ Distinguishing_the_pathways_of_primer_removal_during_Eukaryotic_Okazaki
      fragment_maturation Contributor Author Rossi, Marie Louise. Date
      Accessioned: 2009-02-23T17:05:09Z. Date Available: 2009-02-23T17:05:09Z.
      Date Issued: 2009-02-23T17:05:09Z. Identifier Uri: http://hdl.handle.net/
      1802/6537. Description: Dr. Robert A. Bambara, Faculty Advisor. Thesis
      (PhD) â School of Medicine and Dentistry, University of Rochester. UR
      only until January 2010. UR only until January 2010.
  30. ^Huberman, Joel A.; Riggs, Arthur D. (March 1968). "On the mechanism of
      DNA replication in mammalian chromosomes". Journal of Molecular Biology.
      32 (2): 327â341. doi:10.1016/0022-2836(68)90013-2. PMID 5689363.
  31. ^Gao, Yang; et al. (22 February 2019). "Structures and operating
      principles of the replisome". Science. 363 (6429): 835. doi:10.1126/
      science.aav7003. PMID 30679383.
  32. ^Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002).
      Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.
  33.  DNA_Replication_Mechanisms:_DNA_Topoisomerases_Prevent_DNA_Tangling
      During_Replication
  34. ^Reece, Richard J.; Maxwell, Anthony; Wang, James C. (26 September 2008).
      "DNA Gyrase: Structure and Function". Critical Reviews in Biochemistry
      and Molecular Biology. 26 (3â4): 335â375. doi:10.3109/
      10409239109114072. PMID 1657531.
  35. ^Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002).
      Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.
  36.  DNA_Replication_Mechanisms:_Special_Proteins_Help_to_Open_Up_the_DNA
      Double_Helix_in_Front_of_the_Replication_Fork
  37. ^Griffiths A.J.F.; Wessler S.R.; Lewontin R.C.; Carroll S.B. (2008).
      Introduction to Genetic Analysis. W. H. Freeman and Company. ISBN 978-0-
      7167-6887-6.
  38. [Chapter 7: DNA: Structure and Replication. pg 283â290]
  39. ^"Will_the_Hayflick_limit_keep_us_from_living_forever?". Howstuffworks.
      2009-05-11. Retrieved January 20, 2015.
  40. ^ a b James D. Watson et al. (2008), "Molecular Biology of the gene",
      Pearson Education: 237
  41. ^ a b c d e Peter Meister, Angela Taddei1, Susan M. Gasser(June 2006),
      "In and out of the Replication Factory", Cell 125 (7): 1233â1235
  42. ^TA Brown (2002). Genomes. BIOS_Scientific_Publishers. ISBN 1-85996-228-
      9.
  43. 13.2.3._Termination_of_replication
  44. ^Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002).
      Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.
  45.  Intracellular_Control_of_Cell-Cycle_Events:_S-Phase_Cyclin-Cdk_Complexes
      (S-Cdks)_Initiate_DNA_Replication_Once_Per_Cycle
  46. ^Brown, TA (2002). "13". Genomes (2nd ed.). Oxford: Wiley-Liss.
  47. ^Tobiason DM, Seifert HS (2006). "The_Obligate_Human_Pathogen,_Neisseria
      gonorrhoeae,_Is_Polyploid". PLoS Biology. 4 (6): e185. doi:10.1371/
      journal.pbio.0040185. PMC 1470461. PMID 16719561.
  48.  [open_access]
  49. ^Slater, Steven; Wold, Sture; Lu, Min; Boye, Erik; Skarstad, Kirsten;
      Kleckner, Nancy (September 1995). "E. coli SeqA protein binds oriC in two
      different methyl-modulated reactions appropriate to its roles in DNA
      replication initiation and origin sequestration". Cell. 82 (6):
      927â936. doi:10.1016/0092-8674(95)90272-4. PMID 7553853.
  50. ^Cooper, Stephen; Helmstetter, Charles E. (February 1968). "Chromosome
      replication and the division cycle of Escherichia coli". Journal of
      Molecular Biology. 31 (3): 519â540. doi:10.1016/0022-2836(68)90425-7.
  51. ^Saiki, Randall; Gelfand, David H.; Stoffel, Susanne; Scharf, Stephen J.;
      Higuchi, Russell; Horn, Glenn T.; Mullis, Kary B.; Erlich, Henry A. (29
      January 1988). "Primer-directed_enzymatic_amplification_of_DNA_with_a
      thermostable_DNA_polymerase". Science. 239 (4839): 487â491. Bibcode:
      1988Sci...239..487S. doi:10.1126/science.2448875. PMID 2448875. Retrieved
      7 April 2016.
    * v
    * t
    * e
DNA replication (comparing Prokaryotic to Eukaryotic)
                                * Pre-replication_complex
                                * dnaC
                                      o Cdc6
            Prokaryotic         * Helicase
            (initiation)              o dnaA
                                      o dnaB
                                      o T7
                                * Primase
                                      o dnaG
                                * Pre-replication_complex
                                * Origin_recognition_complex
                                      o ORC1
                                      o ORC2
                                      o ORC3
                                      o ORC4
                                      o ORC5
                                      o ORC6
                                * Cdc6
                                * Cdt1
                                * Minichromosome_maintenance
                                      o MCM2
Initiation                            o MCM3
            Eukaryotic                o MCM4
            (preparation_in           o MCM5
            G1_phase)                 o MCM6
                                      o MCM7
                                * Licensing_factor
                                * Autonomously_replicating_sequence
                                * Single-strand_binding_protein
                                      o SSBP2
                                      o SSBP3
                                      o SSBP4
                                * RNase_H
                                      o RNASEH1
                                      o RNASEH2A
                                * Helicase: HFM1
                                * Primase: PRIM1
                                * PRIM2
                                * Origin_of_replication/Ori/Replicon
                                * Replication_fork
            Both                      o Lagging and leading_strands
                                * Okazaki_fragments
                                * Primer
                              * DNA_polymerase_III_holoenzyme
                                    o dnaC
                                    o dnaE
                                    o dnaH
                                    o dnaN
                                    o dnaQ
                                    o dnaT
                                    o dnaX
                                    o holA
            Prokaryotic             o holB
            (elongation)            o holC
                                    o holD
                                    o holE
                              * Replisome
                              * DNA_ligase
                              * DNA_clamp
                              * Topoisomerase
                                    o DNA_gyrase
                              * Prokaryotic_DNA_polymerase: DNA_polymerase_I
                                    o Klenow_fragment
                              * Replication_factor_C
                                    o RFC1
                              * Flap_endonuclease
Replication                         o FEN1
                              * Topoisomerase
                              * Replication_protein_A
                                    o RPA1
                              * Eukaryotic_DNA_polymerase:
                              * alpha
                                    o POLA1
                                    o POLA2
            Eukaryotic              o PRIM1
            (synthesis_in           o PRIM2
            S_phase)          * delta
                                    o POLD1
                                    o POLD2
                                    o POLD3
                                    o POLD4
                              * epsilon
                                    o POLE
                                    o POLE2
                                    o POLE3
                                    o POLE4
                              * DNA_clamp
                                    o PCNA
                              * Control_of_chromosome_duplication
            Both              * Movement: Processivity
                              * DNA_ligase
                * Telomere: Telomerase
Termination           o TERT
                      o TERC
                      o DKC1
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal
Authority_control [Edit_this_at_Wikidata]     * GND: 4153174-7

Retrieved from "https://en.wikipedia.org/w/
index.php?title=DNA_replication&oldid=906933145"
Categories:
    * DNA_replication
    * Senescence
Hidden categories:
    * All_articles_with_dead_external_links
    * Articles_with_dead_external_links_from_March_2019
    * Commons_category_link_is_on_Wikidata
    * Wikipedia_articles_with_GND_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * à¦¬à¦¾à¦à¦²à¦¾
    * ÐÐ°ÑÒ¡Ð¾ÑÑÑÐ°
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * KreyÃ²l_ayisyen
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * LietuviÅ³
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 19 July 2019, at 09:14 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
