The following text has been accessed from https://en.wikipedia.org/wiki/Cellular_differentiation at Fri Aug 9 00:06:23 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Cellular differentiation ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
"Cell differentiation" redirects here. For the journal, see Cell
Differentiation_(journal).
Not to be confused with Cell_division.
Stem cell differentiation into various tissue types.
Cell-count distribution featuring cellular differentiation for three types of
cells (progenitor     z   {\displaystyle z}  [z], osteoblast     y
{\displaystyle y}  [y], and chondrocyte     x   {\displaystyle x}  [x]) exposed
to pro-osteoblast stimulus.[1]
Cellular differentiation is the process where a cell changes from one cell_type
to another.[2][3] Usually, the cell changes to a more specialized type.
Differentiation occurs numerous times during the development of a multicellular
organism as it changes from a simple zygote to a complex system of tissues and
cell types. Differentiation continues in adulthood as adult_stem_cells divide
and create fully differentiated daughter_cells during tissue repair and during
normal cell turnover. Some differentiation occurs in response to antigen
exposure. Differentiation dramatically changes a cell's size, shape, membrane
potential, metabolic_activity, and responsiveness to signals. These changes are
largely due to highly controlled modifications in gene_expression and are the
study of epigenetics. With a few exceptions, cellular differentiation almost
never involves a change in the DNA sequence itself. Thus, different cells can
have very different physical characteristics despite having the same genome.
A specialized type of differentiation, known as 'terminal differentiation', is
of importance in some tissues, for example vertebrate nervous system, striated
muscle, epidermis and gut. During terminal differentiation, a precursor cell
formerly capable of cell division, permanently leaves the cell cycle,
dismantles the cell cycle machinery and often expresses a range of genes
characteristic of the cell's final function (e.g. myosin and actin for a muscle
cell). Differentiation may continue to occur after terminal differentiation if
the capacity and functions of the cell undergo further changes.
Among dividing cells, there are multiple levels of cell_potency, the cell's
ability to differentiate into other cell types. A greater potency indicates a
larger number of cell types that can be derived. A cell that can differentiate
into all cell types, including the placental tissue, is known as totipotent. In
mammals, only the zygote and subsequent blastomeres are totipotent, while in
plants, many differentiated cells can become totipotent with simple laboratory
techniques. A cell that can differentiate into all cell types of the adult
organism is known as pluripotent. Such cells are called meristematic_cells in
higher plants and embryonic_stem_cells in animals, though some groups report
the presence of adult pluripotent cells. Virally induced expression of four
transcription factors Oct4, Sox2, c-Myc, and Klf4 (Yamanaka_factors) is
sufficient to create pluripotent (iPS) cells from adult fibroblasts.[4] A
multipotent cell is one that can differentiate into multiple different, but
closely related cell types.[5] Oligopotent_cells are more restricted than
multipotent, but can still differentiate into a few closely related cell types.
[5] Finally, unipotent cells can differentiate into only one cell type, but are
capable of self-renewal.[5] In cytopathology, the level of cellular
differentiation is used as a measure of cancer progression. "Grade" is a marker
of how differentiated a cell in a tumor is.[6]
⁰
***** Contents *****
    * 1_Mammalian_cell_types
    * 2_Dedifferentiation
    * 3_Mechanisms
    * 4_Epigenetic_control
          o 4.1_Importance_of_epigenetic_control
          o 4.2_Mechanisms_of_epigenetic_regulation
                # 4.2.1_Pioneer_factor|Pioneering_factors_(Oct4,_Sox2,_Nanog)
                # 4.2.2_Polycomb_repressive_complex_(PRC2)
                # 4.2.3_Trithorax_group_proteins_(TrxG)
                # 4.2.4_DNA_methylation
                # 4.2.5_Nucleosome_positioning
                      # 4.2.5.1_Histone_acetylation_and_methylation
                      # 4.2.5.2_In_stem_cells
          o 4.3_Role_of_signaling_in_epigenetic_control
          o 4.4_Effect_of_matrix_elasticity
    * 5_See_also
    * 6_References
***** Mammalian cell types[edit] *****
See also: List_of_distinct_cell_types_in_the_adult_human_body
Three basic categories of cells make up the mammalian body: germ_cells, somatic
cells, and stem_cells. Each of the approximately 37.2 trillion (3.72x1013)
cells in an adult human has its own copy or copies of the genome except certain
cell types, such as red_blood_cells, that lack nuclei in their fully
differentiated state. Most cells are diploid; they have two copies of each
chromosome. Such cells, called somatic cells, make up most of the human body,
such as skin and muscle cells. Cells differentiate to specialize for different
functions.[7]
Germ line cells are any line of cells that give rise to gametes—eggs and
sperm—and thus are continuous through the generations. Stem cells, on the other
hand, have the ability to divide for indefinite periods and to give rise to
specialized cells. They are best described in the context of normal human
development.[citation_needed]
Development begins when a sperm fertilizes an egg and creates a single cell
that has the potential to form an entire organism. In the first hours after
fertilization, this cell divides into identical cells. In humans, approximately
four days after fertilization and after several cycles of cell division, these
cells begin to specialize, forming a hollow sphere of cells, called a
blastocyst.[8] The blastocyst has an outer layer of cells, and inside this
hollow sphere, there is a cluster of cells called the inner_cell_mass. The
cells of the inner cell mass go on to form virtually all of the tissues of the
human body. Although the cells of the inner cell mass can form virtually every
type of cell found in the human body, they cannot form an organism. These cells
are referred to as pluripotent.[9]
Pluripotent stem cells undergo further specialization into multipotent
progenitor_cells that then give rise to functional cells. Examples of stem and
progenitor cells include:[citation_needed]
    * Radial_glial_cells (embryonic neural stem cells) that give rise to
      excitatory neurons in the fetal brain through the process of
      neurogenesis.[10][11][12]
    * Hematopoietic_stem_cells (adult stem cells) from the bone_marrow that
      give rise to red_blood_cells, white_blood_cells, and platelets
    * Mesenchymal_stem_cells (adult stem cells) from the bone_marrow that give
      rise to stromal cells, fat cells, and types of bone cells
    * Epithelial stem cells (progenitor cells) that give rise to the various
      types of skin cells
    * Muscle satellite_cells (progenitor cells) that contribute to
      differentiated muscle_tissue.
A pathway that is guided by the cell adhesion molecules consisting of four
amino acids, arginine, glycine, asparagine, and serine, is created as the
cellular blastomere differentiates from the single-layered blastula to the
three primary layers_of_germ_cells in mammals, namely the ectoderm, mesoderm
and endoderm (listed from most distal (exterior) to proximal (interior)). The
ectoderm ends up forming the skin and the nervous system, the mesoderm forms
the bones and muscular tissue, and the endoderm forms the internal organ
tissues.
***** Dedifferentiation[edit] *****
Micrograph of a liposarcoma with some dedifferentiation, that is not
identifiable as a liposarcoma, (left edge of image) and a differentiated
component (with lipoblasts and increased vascularity (right of image)). Fully
differentiated (morphologically benign) adipose_tissue (center of the image)
has few blood vessels. H&E_stain.
Dedifferentiation, or integration is a cellular process often seen in more
basal life forms such as worms and amphibians in which a partially or
terminally differentiated cell reverts to an earlier developmental stage,
usually as part of a regenerative process.[13][14] Dedifferentiation also
occurs in plants.[15] Cells in cell_culture can lose properties they originally
had, such as protein expression, or change shape. This process is also termed
dedifferentiation.[16]
Some believe dedifferentiation is an aberration of the normal development cycle
that results in cancer,[17] whereas others believe it to be a natural part of
the immune response lost by humans at some point as a result of evolution.
A small molecule dubbed reversine, a purine analog, has been discovered that
has proven to induce dedifferentiation in myotubes. These dedifferentiated
cells could then redifferentiate into osteoblasts and adipocytes.[18]
Diagram exposing several methods used to revert adult somatic cells to
totipotency or pluripotency.
***** Mechanisms[edit] *****
See also: Embryonic_differentiation_waves
Mechanisms of cellular differentiation.
Each specialized cell_type in an organism expresses a subset of all the genes
that constitute the genome of that species. Each cell type is defined by its
particular pattern of regulated_gene_expression. Cell differentiation is thus a
transition of a cell from one cell type to another and it involves a switch
from one pattern of gene expression to another. Cellular differentiation during
development can be understood as the result of a gene_regulatory_network. A
regulatory gene and its cis-regulatory modules are nodes in a gene regulatory
network; they receive input and create output elsewhere in the network.[19] The
systems_biology approach to developmental biology emphasizes the importance of
investigating how developmental mechanisms interact to produce predictable
patterns (morphogenesis). However, an alternative view has been proposed
recently[when?][by_whom?]. Based on stochastic gene expression, cellular
differentiation is the result of a Darwinian selective process occurring among
cells. In this frame, protein and gene networks are the result of cellular
processes and not their cause.[citation_needed]
An overview of major signal transduction pathways.
A few evolutionarily conserved types of molecular processes are often involved
in the cellular mechanisms that control these switches. The major types of
molecular processes that control cellular differentiation involve cell
signaling. Many of the signal molecules that convey information from cell to
cell during the control of cellular differentiation are called growth_factors.
Although the details of specific signal_transduction pathways vary, these
pathways often share the following general steps. A ligand produced by one cell
binds to a receptor in the extracellular region of another cell, inducing a
conformational change in the receptor. The shape of the cytoplasmic domain of
the receptor changes, and the receptor acquires enzymatic activity. The
receptor then catalyzes reactions that phosphorylate other proteins, activating
them. A cascade of phosphorylation reactions eventually activates a dormant
transcription factor or cytoskeletal protein, thus contributing to the
differentiation process in the target cell.[20] Cells and tissues can vary in
competence, their ability to respond to external signals.[21]
Signal induction refers to cascades of signaling events, during which a cell or
tissue signals to another cell or tissue to influence its developmental fate.
[21] Yamamoto and Jeffery[22] investigated the role of the lens in eye
formation in cave- and surface-dwelling fish, a striking example of induction.
[21] Through reciprocal transplants, Yamamoto and Jeffery[22] found that the
lens vesicle of surface fish can induce other parts of the eye to develop in
cave- and surface-dwelling fish, while the lens vesicle of the cave-dwelling
fish cannot.[21]
Other important mechanisms fall under the category of asymmetric_cell
divisions, divisions that give rise to daughter cells with distinct
developmental fates. Asymmetric cell divisions can occur because of
asymmetrically expressed maternal cytoplasmic determinants or because of
signaling.[21] In the former mechanism, distinct daughter cells are created
during cytokinesis because of an uneven distribution of regulatory molecules in
the parent cell; the distinct cytoplasm that each daughter cell inherits
results in a distinct pattern of differentiation for each daughter cell. A
well-studied example of pattern formation by asymmetric divisions is body_axis
patterning_in_Drosophila. RNA molecules are an important type of intracellular
differentiation control signal. The molecular and genetic basis of asymmetric
cell divisions has also been studied in green algae of the genus Volvox, a
model system for studying how unicellular organisms can evolve into
multicellular organisms.[21] In Volvox carteri, the 16 cells in the anterior
hemisphere of a 32-cell embryo divide asymmetrically, each producing one large
and one small daughter cell. The size of the cell at the end of all cell
divisions determines whether it becomes a specialized germ or somatic cell.[21]
[23]
***** Epigenetic control[edit] *****
Main article: Epigenetics_in_stem_cell_differentiation
Since each cell, regardless of cell type, possesses the same genome,
determination of cell type must occur at the level of gene expression. While
the regulation_of_gene_expression can occur through cis- and trans-regulatory
elements including a gene's promoter and enhancers, the problem arises as to
how this expression pattern is maintained over numerous generations of cell
division. As it turns out, epigenetic processes play a crucial role in
regulating the decision to adopt a stem, progenitor, or mature cell fate. This
section will focus primarily on mammalian stem_cells.
In systems biology and mathematical modeling of gene regulatory networks, cell-
fate determination is predicted to exhibit certain dynamics, such as attractor-
convergence (the attractor can be an equilibrium point, limit cycle or strange
attractor) or oscillatory.[24]
**** Importance of epigenetic control[edit] ****
The first question that can be asked is the extent and complexity of the role
of epigenetic processes in the determination of cell fate. A clear answer to
this question can be seen in the 2011 paper by Lister R, et al. [25] on
aberrant epigenomic programming in human induced_pluripotent_stem_cells. As
induced pluripotent stem cells (iPSCs) are thought to mimic embryonic_stem
cells in their pluripotent properties, few epigenetic differences should exist
between them. To test this prediction, the authors conducted whole-genome
profiling of DNA_methylation patterns in several human embryonic stem cell
(ESC), iPSC, and progenitor cell lines.
Female adipose cells, lung fibroblasts, and foreskin fibroblasts were
reprogrammed into induced pluripotent state with the OCT4, SOX2, KLF4, and MYC
genes. Patterns of DNA methylation in ESCs, iPSCs, somatic cells were compared.
Lister R, et al. observed significant resemblance in methylation levels between
embryonic and induced pluripotent cells. Around 80% of CG_dinucleotides in ESCs
and iPSCs were methylated, the same was true of only 60% of CG dinucleotides in
somatic cells. In addition, somatic cells possessed minimal levels of cytosine
methylation in non-CG dinucleotides, while induced pluripotent cells possessed
similar levels of methylation as embryonic stem cells, between 0.5 and 1.5%.
Thus, consistent with their respective transcriptional activities,[25] DNA
methylation patterns, at least on the genomic level, are similar between ESCs
and iPSCs.
However, upon examining methylation patterns more closely, the authors
discovered 1175 regions of differential CG dinucleotide methylation between at
least one ES or iPS cell line. By comparing these regions of differential
methylation with regions of cytosine methylation in the original somatic cells,
44-49% of differentially methylated regions reflected methylation patterns of
the respective progenitor somatic cells, while 51-56% of these regions were
dissimilar to both the progenitor and embryonic cell lines. In_vitro-induced
differentiation of iPSC lines saw transmission of 88% and 46% of hyper and
hypo-methylated differentially methylated regions, respectively.
Two conclusions are readily apparent from this study. First, epigenetic
processes are heavily involved in cell fate determination, as seen from the
similar levels of cytosine methylation between induced pluripotent and
embryonic stem cells, consistent with their respective patterns of
transcription. Second, the mechanisms of de-differentiation (and by extension,
differentiation) are very complex and cannot be easily duplicated, as seen by
the significant number of differentially methylated regions between ES and iPS
cell lines. Now that these two points have been established, we can examine
some of the epigenetic mechanisms that are thought to regulate cellular
differentiation.
**** Mechanisms of epigenetic regulation[edit] ****
*** Pioneer factor|Pioneering factors (Oct4, Sox2, Nanog)[edit] ***
Three transcription factors, OCT4, SOX2, and NANOG â the first two of which
are used in induced pluripotent stem cell (iPSC) reprogramming, along with Klf4
and c-Myc â are highly expressed in undifferentiated embryonic stem cells and
are necessary for the maintenance of their pluripotency.[26] It is thought that
they achieve this through alterations in chromatin structure, such as histone
modification and DNA methylation, to restrict or permit the transcription of
target genes. While highly expressed, their levels require a precise balance to
maintain pluripotency, perturbation of which will promote differentiation
towards different lineages based on how the gene expression levels change.
Differential regulation of Oct-4 and SOX2 levels have been shown to precede
germ layer fate selection.[27] Increased levels of Oct4 and decreased levels of
Sox2 promote a mesendodermal fate, with Oct4 actively suppressing genes
associated with a neural ectodermal fate. Similarly, Increased levels of Sox2
and decreased levels of Oct4 promote differentiation towards a neural
ectodermal fate, with Sox2 inhibiting differentiation towards a mesendodermal
fate. Regardless of the lineage cells differentiate down, suppression of NANOG
has been identified as a necessary prerequisite for differentiation.[27]
*** Polycomb repressive complex (PRC2)[edit] ***
In the realm of gene_silencing, Polycomb_repressive_complex_2, one of two
classes of the Polycomb_group (PcG) family of proteins, catalyzes the di- and
tri-methylation of histone H3 lysine 27 (H3K27me2/me3).[26][28][29] By binding
to the H3K27me2/3-tagged nucleosome, PRC1 (also a complex of PcG family
proteins) catalyzes the mono-ubiquitinylation of histone H2A at lysine 119
(H2AK119Ub1), blocking RNA_polymerase_II activity and resulting in
transcriptional suppression.[26] PcG knockout ES cells do not differentiate
efficiently into the three germ layers, and deletion of the PRC1 and PRC2 genes
leads to increased expression of lineage-affiliated genes and unscheduled
differentiation.[26] Presumably, PcG complexes are responsible for
transcriptionally repressing differentiation and development-promoting genes.
*** Trithorax group proteins (TrxG)[edit] ***
Alternately, upon receiving differentiation signals, PcG proteins are recruited
to promoters of pluripotency transcription factors. PcG-deficient ES cells can
begin differentiation but cannot maintain the differentiated phenotype.[26]
Simultaneously, differentiation and development-promoting genes are activated
by Trithorax group (TrxG) chromatin regulators and lose their repression.[26]
[29] TrxG proteins are recruited at regions of high transcriptional activity,
where they catalyze the trimethylation of histone H3 lysine 4 (H3K4me3) and
promote gene activation through histone acetylation.[29] PcG and TrxG complexes
engage in direct competition and are thought to be functionally antagonistic,
creating at differentiation and development-promoting loci what is termed a
"bivalent domain" and rendering these genes sensitive to rapid induction or
repression.[30]
*** DNA methylation[edit] ***
Regulation of gene expression is further achieved through DNA methylation, in
which the DNA_methyltransferase-mediated methylation of cytosine residues in
CpG dinucleotides maintains heritable repression by controlling DNA
accessibility.[30] The majority of CpG sites in embryonic stem cells are
unmethylated and appear to be associated with H3K4me3-carrying nucleosomes.[26]
Upon differentiation, a small number of genes, including OCT4 and NANOG,[30]
are methylated and their promoters repressed to prevent their further
expression. Consistently, DNA methylation-deficient embryonic stem cells
rapidly enter apoptosis upon in vitro differentiation.[26]
*** Nucleosome positioning[edit] ***
While the DNA_sequence of most cells of an organism is the same, the binding
patterns of transcription factors and the corresponding gene expression
patterns are different. To a large extent, differences in transcription factor
binding are determined by the chromatin accessibility of their binding sites
through histone_modification and/or pioneer_factors. In particular, it is
important to know whether a nucleosome is covering a given genomic binding site
or not. This can be determined using a chromatin_immunoprecipitation (ChIP)
assay.[31]
** Histone acetylation and methylation[edit] **
DNA-nucleosome interactions are characterized by two states: either tightly
bound by nucleosomes and transcriptionally inactive, called heterochromatin, or
loosely bound and usually, but not always, transcriptionally active, called
euchromatin. The epigenetic processes of histone methylation and acetylation,
and their inverses demethylation and deacetylation primarily account for these
changes. The effects of acetylation and deacetylation are more predictable. An
acetyl group is either added to or removed from the positively charged Lysine
residues in histones by enzymes called histone_acetyltransferases or histone
deacteylases, respectively. The acetyl group prevents Lysine's association with
the negatively charged DNA backbone. Methylation is not as straightforward, as
neither methylation nor demethylation consistently correlate with either gene
activation or repression. However, certain methylations have been repeatedly
shown to either activate or repress genes. The trimethylation of lysine 4 on
histone 3 (H3K4Me3) is associated with gene activation, whereas trimethylation
of lysine 27 on histone 3 represses genes[32][33][34]
** In stem cells[edit] **
During differentiation, stem_cells change their gene expression profiles.
Recent studies have implicated a role for nucleosome positioning and histone
modifications during this process.[35] There are two components of this
process: turning off the expression of embryonic stem cell (ESC) genes, and the
activation of cell fate genes. Lysine specific demethylase 1 (KDM1A) is thought
to prevent the use of enhancer regions of pluripotency genes, thereby
inhibiting their transcription.[36] It interacts with Mi-2/NuRD_complex
(nucleosome remodelling and histone deacetylase) complex,[36] giving an
instance where methylation and acetylation are not discrete and mutually
exclusive, but intertwined processes.
**** Role of signaling in epigenetic control[edit] ****
A final question to ask concerns the role of cell_signaling in influencing the
epigenetic processes governing differentiation. Such a role should exist, as it
would be reasonable to think that extrinsic signaling can lead to epigenetic
remodeling, just as it can lead to changes in gene expression through the
activation or repression of different transcription factors. Little direct data
is available concerning the specific signals that influence the epigenome, and
the majority of current knowledge about the subject consists of speculations on
plausible candidate regulators of epigenetic remodeling.[37] We will first
discuss several major candidates thought to be involved in the induction and
maintenance of both embryonic stem cells and their differentiated progeny, and
then turn to one example of specific signaling pathways in which more direct
evidence exists for its role in epigenetic change.
The first major candidate is Wnt_signaling_pathway. The Wnt pathway is involved
in all stages of differentiation, and the ligand Wnt3a can substitute for the
overexpression of c-Myc in the generation of induced pluripotent stem cells.
[37] On the other hand, disruption of Ã-catenin, a component of the Wnt
signaling pathway, leads to decreased proliferation of neural progenitors.
Growth_factors comprise the second major set of candidates of epigenetic
regulators of cellular differentiation. These morphogens are crucial for
development, and include bone_morphogenetic_proteins, transforming_growth
factors (TGFs), and fibroblast_growth_factors (FGFs). TGFs and FGFs have been
shown to sustain expression of OCT4, SOX2, and NANOG by downstream signaling to
Smad proteins.[37] Depletion of growth factors promotes the differentiation of
ESCs, while genes with bivalent chromatin can become either more restrictive or
permissive in their transcription.[37]
Several other signaling pathways are also considered to be primary candidates.
Cytokine leukemia_inhibitory_factors are associated with the maintenance of
mouse ESCs in an undifferentiated state. This is achieved through its
activation of the Jak-STAT3 pathway, which has been shown to be necessary and
sufficient towards maintaining mouse ESC pluripotency.[38] Retinoic_acid can
induce differentiation of human and mouse ESCs,[37] and Notch_signaling is
involved in the proliferation and self-renewal of stem cells. Finally, Sonic
hedgehog, in addition to its role as a morphogen, promotes embryonic stem cell
differentiation and the self-renewal of somatic stem cells.[37]
The problem, of course, is that the candidacy of these signaling pathways was
inferred primarily on the basis of their role in development and cellular
differentiation. While epigenetic regulation is necessary for driving cellular
differentiation, they are certainly not sufficient for this process. Direct
modulation of gene expression through modification of transcription factors
plays a key role that must be distinguished from heritable epigenetic changes
that can persist even in the absence of the original environmental signals.
Only a few examples of signaling pathways leading to epigenetic changes that
alter cell fate currently exist, and we will focus on one of them.
Expression of Shh (Sonic hedgehog) upregulates the production of BMI1, a
component of the PcG complex that recognizes H3K27me3. This occurs in a Gli-
dependent manner, as Gli1 and Gli2 are downstream effectors of the Hedgehog
signaling_pathway. In culture, Bmi1 mediates the Hedgehog pathway's ability to
promote human mammary stem cell self-renewal.[39] In both humans and mice,
researchers showed Bmi1 to be highly expressed in proliferating immature
cerebellar granule cell precursors. When Bmi1 was knocked out in mice, impaired
cerebellar development resulted, leading to significant reductions in postnatal
brain mass along with abnormalities in motor control and behavior.[40] A
separate study showed a significant decrease in neural stem cell proliferation
along with increased astrocyte proliferation in Bmi null mice.[41]
A alternative model of cellular differentiation during embryogenesis is that
positional information is based on mechanical signalling by the cytoskeleton
using Embryonic_differentiation_waves. The mechanical signal is then
epigenetically transduced via signal transduction systems (of which specific
molecules such as Wnt are part) to result in differential gene expression.
In summary, the role of signaling in the epigenetic control of cell fate in
mammals is largely unknown, but distinct examples exist that indicate the
likely existence of further such mechanisms.
**** Effect of matrix elasticity[edit] ****
In order to fulfill the purpose of regenerating a variety of tissues, adult
stems are known to migrate from their niches, adhere to new extracellular
matrices (ECM) and differentiate. The ductility of these microenvironments are
unique to different tissue types. The ECM surrounding brain, muscle and bone
tissues range from soft to stiff. The transduction of the stem cells into these
cells types is not directed solely by chemokine cues and cell to cell
signaling. The elasticity of the microenvironment can also affect the
differentiation of mesenchymal stem cells (MSCs which originate in bone
marrow.) When MSCs are placed on substrates of the same stiffness as brain,
muscle and bone ECM, the MSCs take on properties of those respective cell
types.[42] Matrix sensing requires the cell to pull against the matrix at focal
adhesions, which triggers a cellular mechano-transducer to generate a signal to
be informed what force is needed to deform the matrix. To determine the key
players in matrix-elasticity-driven lineage specification in MSCs, different
matrix microenvironments were mimicked. From these experiments, it was
concluded that focal adhesions of the MSCs were the cellular mechano-transducer
sensing the differences of the matrix elasticity. The non-muscle myosin IIa-
c isoforms generates the forces in the cell that lead to signaling of early
commitment markers. Nonmuscle myosin IIa generates the least force increasing
to non-muscle myosin IIc. There are also factors in the cell that inhibit non-
muscle myosin II, such as blebbistatin. This makes the cell effectively blind
to the surrounding matrix.[42] Researchers have obtained some success in
inducing stem cell-like properties in HEK 239 cells by providing a soft matrix
without the use of diffusing factors.[43] The stem-cell properties appear to be
linked to tension in the cells' actin network. One identified mechanism for
matrix-induced differentiation is tension-induced proteins, which remodel
chromatin in response to mechanical stretch.[44] The RhoA pathway is also
implicated in this process.
***** See also[edit] *****
    * Interbilayer_Forces_in_Membrane_Fusion
    * Fusion_mechanism
    * Lipid_bilayer_fusion
    * Cell-cell_fusogens
    * CAF-1
    * List_of_human_cell_types_derived_from_the_germ_layers
***** References[edit] *****
   1. ^Kryven, I.; RÃ¶blitz, S.; SchÃ¼tte, Ch. (2015). "Solution_of_the
      chemical_master_equation_by_radial_basis_functions_approximation_with
      interface_tracking". BMC Systems Biology. 9 (1): 67. doi:10.1186/s12918-
      015-0210-y. PMC 4599742. PMID 26449665.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. [open_access]
   4. ^ Slack, J.M.W. (2013) Essential Developmental Biology. Wiley-Blackwell,
      Oxford.
   5. ^Slack, J.M.W. (2007). "Metaplasia and transdifferentiation: from pure
      biology to the clinic". Nature Reviews Molecular Cell Biology. 8 (5):
      369â378. doi:10.1038/nrm2146. PMID 17377526.
   6. ^Takahashi, K; Yamanaka, S (2006). "Induction of pluripotent stem cells
      from mouse embryonic and adult fibroblast cultures by defined factors".
      Cell. 126 (4): 663â76. doi:10.1016/j.cell.2006.07.024. PMID 16904174.
   7. ^ a b cSchÃ¶ler, Hans R. (2007). "The Potential of Stem Cells: An
      Inventory". In Nikolaus Knoepffler; Dagmar Schipanski; Stefan Lorenz
      Sorgner (eds.). Humanbiotechnology as Social Challenge. Ashgate
      Publishing. p. 28. ISBN 978-0-7546-5755-2.
   8. ^"NCI_Dictionary_of_Cancer_Terms". National Cancer Institute. Retrieved 1
      November 2013.
   9. ^Lodish, Harvey (2000). Molecular_Cell_Biology (4th ed.). New York: W. H.
      Freeman. Section 14.2. ISBN 978-0-7167-3136-8.
  10. ^Kumar, Rani (2008). Textbook_of_Human_Embryology. I.K. International
      Publishing House. p. 22. ISBN 9788190675710.
  11. ^D. Binder, Marc; Hirokawa, Nobutaka; Windhorst, Uwe (2009). Encyclopedia
      of Neuroscience. Springer. ISBN 978-3540237358.
  12. ^Rakic, P (October 2009). "Evolution_of_the_neocortex:_a_perspective_from
      developmental_biology". Nature Reviews. Neuroscience. 10 (10): 724â35.
      doi:10.1038/nrn2719. PMC 2913577. PMID 19763105.
  13. ^Lui, JH; Hansen, DV; Kriegstein, AR (8 July 2011). "Development_and
      evolution_of_the_human_neocortex". Cell. 146 (1): 18â36. doi:10.1016/
      j.cell.2011.06.030. PMC 3610574. PMID 21729779.
  14. ^Rash, BG; Ackman, JB; Rakic, P (February 2016). "Bidirectional_radial_Ca
      (2+)_activity_regulates_neurogenesis_and_migration_during_early_cortical
      column_formation". Science Advances. 2 (2): e1501733. Bibcode:
      2016SciA....2E1733R. doi:10.1126/sciadv.1501733. PMC 4771444.
      PMID 26933693.
  15. ^Stocum DL (2004). "Amphibian regeneration and stem cells". Curr. Top.
      Microbiol. Immunol. Current Topics in Microbiology and Immunology. 280:
      1â70. doi:10.1007/978-3-642-18846-6_1. ISBN 978-3-540-02238-1.
      PMID 14594207.
  16. ^Casimir CM, Gates PB, Patient RK, Brockes JP (1988-12-01). "Evidence_for
      dedifferentiation_and_metaplasia_in_amphibian_limb_regeneration_from
      inheritance_of_DNA_methylation". Development. 104 (4): 657â668.
      PMID 3268408.
  17. ^Giles KL (1971). "Dedifferentiation_and_Regeneration_in_Bryophytes:_A
      Selective_Review". New Zealand Journal of Botany. 9 (4): 689â94. doi:
      10.1080/0028825x.1971.10430231. Archived from the_original on 2008-12-04.
      Retrieved 2008-01-01.
  18. ^Schnabel M, Marlovits S, Eckhoff G, et al. (January 2002).
      "Dedifferentiation-associated changes in morphology and gene expression
      in primary human articular chondrocytes in cell culture". Osteoarthr.
      Cartil. 10 (1): 62â70. doi:10.1053/joca.2001.0482. PMID 11795984.
  19. ^Sell S (December 1993). "Cellular_origin_of_cancer:_dedifferentiation_or
      stem_cell_maturation_arrest?". Environ. Health Perspect. 101 (Suppl 5):
      15â26. doi:10.2307/3431838. JSTOR 3431838. PMC 1519468. PMID 7516873.
  20. ^Tsonis PA (April 2004). "Stem_cells_from_differentiated_cells". Mol.
      Interv. 4 (2): 81â3. doi:10.1124/mi.4.2.4. PMID 15087480. Archived from
      the_original on 2016-05-23. Retrieved 2010-12-26.
  21. ^Ben-Tabou de-Leon S, Davidson EH (2007). "Gene regulation: gene control
      network in development". Annu Rev Biophys Biomol Struct. 36 (191):
      191â212. doi:10.1146/annurev.biophys.35.040405.102002. PMID 17291181.
  22. ^Knisely, Karen; Gilbert, Scott F. (2009). Developmental Biology (8th
      ed.). Sunderland, Mass: Sinauer Associates. p. 147. ISBN 978-0-87893-371-
      6.
  23. ^ a b c d e f g Rudel and Sommer; The evolution of developmental
      mechanisms. Developmental Biology 264, 15-37, 2003Rudel, D.; Sommer, R.
      J. (2003). "The evolution of developmental mechanisms". Developmental
      Biology. 264 (1): 15â37. doi:10.1016/S0012-1606(03)00353-1.
      PMID 14623229.
  24. ^ a b Yamamoto Y and WR Jeffery; Central role for the lens in cave fish
      eye degeneration. Science 289 (5479), 631-633, 2000Yamamoto, Y.; Jeffery,
      W. R. (2000). "Central Role for the Lens in Cave Fish Eye Degeneration".
      Science. 289 (5479): 631â633. Bibcode:2000Sci...289..631Y. doi:10.1126/
      science.289.5479.631. PMID 10915628.
  25. ^ Kirk MM, A Ransick, SE Mcrae, DL Kirk; The relationship between cell
      size and cell fate in Volvox carteri. Journal of Cell Biology 123, 191-
      208, 1993Kirk, M. M.; Ransick, A.; McRae, S. E.; Kirk, D. L. (1993). "The
      relationship_between_cell_size_and_cell_fate_in_Volvox_carteri". Journal
      of_Cell_Biology. 123 (1): 191â208. doi:10.1083/jcb.123.1.191.
      PMC 2119814. PMID 8408198.
  26. ^Rabajante JF, Babierra AL (January 30, 2015). "Branching and
      oscillations in the epigenetic landscape of cell-fate determination".
      Progress in Biophysics and Molecular Biology. 117 (2â3): 240â9. doi:
      10.1016/j.pbiomolbio.2015.01.006. PMID 25641423.
  27. ^ a bLister R; et al. (2011). "Hotspots_of_aberrant_epigenomic
      reprogramming_in_human_induced_pluripotent_stem_cells". Nature. 471
      (7336): 68â73. Bibcode:2011Natur.471...68L. doi:10.1038/nature09798.
      PMC 3100360. PMID 21289626.
  28. ^ a b c d e f g hChristophersen NS, Helin K (2010). "Epigenetic_control
      of_embryonic_stem_cell_fate". J Exp Med. 207 (11): 2287â95. doi:
      10.1084/jem.20101438. PMC 2964577. PMID 20975044.
  29. ^ a bThomson, M; Liu, S. J.; Zou, L. N.; Smith, Z; Meissner, A;
      Ramanathan, S (2011). "Pluripotency_factors_in_embryonic_stem_cells
      regulate_differentiation_into_germ_layers". Cell. 145 (6): 875â89. doi:
      10.1016/j.cell.2011.05.017. PMC 5603300. PMID 21663792.
  30. ^Zhu, J.; et al. (2013). "Genome-wide_chromatin_state_transitions
      associated_with_developmental_and_environmental_cues". Cell. 152 (3):
      642â654. doi:10.1016/j.cell.2012.12.033. PMC 3563935. PMID 23333102.
  31. ^ a b cGuenther MG, Young RA (2010). "Repressive_Transcription" (PDF).
      Science. 329 (5988): 150â1. Bibcode:2010Sci...329..150G. doi:10.1126/
      science.1193995. PMC 3006433. PMID 20616255.
  32. ^ a b cMeissner A (2010). "Epigenetic modifications in pluripotent and
      differentiated cells". Nat Biotechnol. 28 (10): 1079â88. doi:10.1038/
      nbt.1684. PMID 20944600.
  33. ^ ChIP_Overview
  34. ^Krogan NJ, Dover J, Wood A, Schneider J, Heidt J, Boateng MA, Dean K,
      Ryan OW, Golshani A, Johnston M, Greenblatt JF, Shilatifard A (Mar 2003).
      "The Paf1 complex is required for histone H3 methylation by COMPASS and
      Dot1p: linking transcriptional elongation to histone methylation".
      Molecular Cell. 11 (3): 721â9. doi:10.1016/S1097-2765(03)00091-1.
      PMID 12667454.
  35. ^Ng HH, Robert F, Young RA, Struhl K (Mar 2003). "Targeted recruitment of
      Set1 histone methylase by elongating Pol II provides a localized mark and
      memory of recent transcriptional activity". Molecular Cell. 11 (3):
      709â19. doi:10.1016/S1097-2765(03)00092-3. PMID 12667453.
  36. ^Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, Huebert
      DJ, McMahon S, Karlsson EK, Kulbokas EJ, Gingeras TR, Schreiber SL,
      Lander ES (Jan 2005). "Genomic maps and comparative analysis of histone
      modifications in human and mouse". Cell. 120 (2): 169â81. doi:10.1016/
      j.cell.2005.01.001. PMID 15680324.
  37. ^Teif VB, Vainshtein Y, Caudron-Herger M, Mallm JP, Marth C, HÃ¶fer T,
      Rippe K (2012). "Genome-wide nucleosome positioning during embryonic stem
      cell development". Nat Struct Mol Biol. 19 (11): 1185â92. doi:10.1038/
      nsmb.2419. PMID 23085715.
  38. ^ a bWhyte, W. A.; Bilodeau, S; Orlando, D. A.; Hoke, H. A.; Frampton, G.
      M.; Foster, C. T.; Cowley, S. M.; Young, R. A. (2012). "Enhancer
      decommissioning_by_LSD1_during_embryonic_stem_cell_differentiation".
      Nature. 482 (7384): 221â5. Bibcode:2012Natur.482..221W. doi:10.1038/
      nature10805. PMC 4144424. PMID 22297846.
  39. ^ a b c d e fMohammad HP, Baylin SB (2010). "Linking cell signaling and
      the epigenetic machinery". Nat Biotechnol. 28 (10): 1033â8. doi:
      10.1038/nbt1010-1033. PMID 20944593.
  40. ^Niwa H, Burdon T, Chambers I, Smith A (1998). "Self-renewal_of
      pluripotent_embryonic_stem_cells_is_mediated_via_activation_of_STAT3".
      Genes Dev. 12 (13): 2048â60. doi:10.1101/gad.12.13.2048. PMC 316954.
      PMID 9649508.
  41. ^Liu S; et al. (2006). "Hedgehog_Signaling_and_Bmi-1_Regulate_Self-
      renewal_of_Normal_and_Malignant_Human_Mammary_Stem_Cells". Cancer Res. 66
      (12): 6063â71. doi:10.1158/0008-5472.CAN-06-0054. PMC 4386278.
      PMID 16778178.
  42. ^Leung C; et al. (2004). "Bmi1 is essential for cerebellar development
      and is overexpressed in human medulloblastomas". Nature. 428 (6980):
      337â41. Bibcode:2004Natur.428..337L. doi:10.1038/nature02385.
      PMID 15029199.
  43. ^Zencak D; et al. (2005). "Bmi1 loss produces an increase in astroglial
      cells and a decrease in neural stem cell population and proliferation". J
      Neurosci. 25 (24): 5774â83. doi:10.1523/JNEUROSCI.3452-04.2005.
      PMID 15958744.
  44. ^ a bEngler, AJ; Sen, S; Sweeney, HL; Discher, DE (August 2006). "Matrix
      Elasticity Directs Stem Cell Lineage Specification". Cell. 126 (4):
      677â689. doi:10.1016/j.cell.2006.06.044. PMID 16923388.
  45. ^Guo, Jun; Wang, Yuexiu; Sachs, Frederick; Meng, Fanjie (2014-12-09).
      "Actin_stress_in_cell_reprogramming". Proceedings of the National Academy
      of Sciences. 111 (49): E5252âE5261. Bibcode:2014PNAS..111E5252G. doi:
      10.1073/pnas.1411683111. ISSN 0027-8424. PMC 4267376. PMID 25422450.
  46. ^Guilak, Farshid; Cohen, Daniel M.; Estes, Bradley T.; Gimble, Jeffrey
      M.; Liedtke, Wolfgang; Chen, Christopher S. (2009-07-02). "Control_of
      Stem_Cell_Fate_by_Physical_Interactions_with_the_Extracellular_Matrix".
      Cell Stem Cell. 5 (1): 17â26. doi:10.1016/j.stem.2009.06.016.
      PMC 2768283. PMID 19570510.
    * v
    * t
    * e
Stem_cells
                     * Embryonic_stem_cells
                     * Adult_stem_cells
Sources/types        * Cancer_stem_cells
                     * Induced_pluripotent_stem_cells
                     * Induced_stem_cells
                     * Totipotent
                           o Zygote
                           o Spore
                           o Morula
                     * Pluripotent
                           o Embryonic_stem_cell
Cell_potency               o Callus
                     * Multipotent
                           o Progenitor_cell: Endothelial_stem_cell
                           o Hematopoietic_stem_cell
                           o Mesenchymal_stem_cell
                           o Neural_stem_cell
                     * Unipotent
                           o Precursor_cell
                     * Cellular differentiation
                     * Stem_cell_therapy
                     * Stem_cell_controversy
Related articles     * Stem_cell_line
                     * Stem_cell_laws
                     * Stem_cell_laws_and_policy_in_the_United_States
                     * Epigenetics_in_stem_cell_differentiation
    * [Category]Category
    * [Portal] Portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Cellular_differentiation&oldid=908362556"
Categories:
    * Cellular_processes
    * Developmental_biology
    * Induced_stem_cells
Hidden categories:
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_January_2011
    * All_articles_with_vague_or_ambiguous_time
    * Vague_or_ambiguous_time_from_June_2019
    * Articles_with_specifically_marked_weasel-worded_phrases_from_June_2019
    * Articles_with_unsourced_statements_from_June_2019
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * á¥áá áá£áá
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * PortuguÃªs
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Simple_English
    * SlovenÄina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 29 July 2019, at 07:23 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
