The following text has been accessed from https://en.wikipedia.org/wiki/RNA_virus at Fri Aug 9 01:44:20 IST 2019
Creative_Commons_Attribution-ShareAlike_License




















****** RNA virus ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
An RNA virus is a virus that has RNA (ribonucleic acid) as its genetic
material.[1] This nucleic_acid is usually single-stranded RNA (ssRNA) but may
be double-stranded RNA (dsRNA).[2] Notable human diseases caused by RNA viruses
include Ebola_virus_disease, SARS, rabies, common_cold, influenza, hepatitis_C,
hepatitis_E, West_Nile_fever, polio and measles.
The International_Committee_on_Taxonomy_of_Viruses (ICTV) classifies RNA
viruses as those that belong to Group III, Group IV or Group V of the Baltimore
classification system of classifying viruses and does not consider viruses with
DNA intermediates in their life cycle as RNA viruses.[3] Viruses with RNA as
their genetic material which also include DNA intermediates in their
replication cycle are called retroviruses, and comprise Group VI of the
Baltimore classification. Notable human retroviruses include HIV-1 and HIV-2,
the cause of the disease AIDS.
Another term for RNA viruses that explicitly excludes retroviruses is
ribovirus.[4]
⁰
***** Contents *****
    * 1_Characteristics
          o 1.1_Single-stranded_RNA_viruses_and_RNA_Sense
          o 1.2_Double-stranded_RNA_viruses
          o 1.3_Mutation_rates
    * 2_Replication
    * 3_Classification
          o 3.1_Positive_strand_RNA_viruses
          o 3.2_Double-stranded_RNA_viruses
          o 3.3_Negative_strand_RNA_viruses
          o 3.4_Satellite_viruses
    * 4_Group_IIIâdsRNA_viruses
    * 5_Group_IVâpositive-sense_ssRNA_viruses
    * 6_Group_Vânegative-sense_ssRNA_viruses
    * 7_Gallery
    * 8_Notes
    * 9_See_also
    * 10_References
    * 11_External_links
***** Characteristics[edit] *****
**** Single-stranded RNA viruses and RNA Sense[edit] ****
RNA viruses can be further classified according to the sense or polarity of
their RNA into negative-sense and positive-sense, or ambisense RNA viruses.
Positive-sense viral RNA is similar to mRNA and thus can be immediately
translated by the host cell. Negative-sense viral RNA is complementary to mRNA
and thus must be converted to positive-sense RNA by an RNA-dependent_RNA
polymerase before translation. As such, purified RNA of a positive-sense virus
can directly cause infection though it may be less infectious than the whole
virus particle. Purified RNA of a negative-sense virus is not infectious by
itself as it needs to be transcribed into positive-sense RNA; each virion can
be transcribed to several positive-sense RNAs. Ambisense RNA viruses resemble
negative-sense RNA viruses, except they also translate genes from the negative
strand.[5]
**** Double-stranded RNA viruses[edit] ****
Further information: Double-stranded_RNA_viruses
Structure of the reovirus virion
The double-stranded_(ds)RNA_viruses represent a diverse group of viruses that
vary widely in host range (humans, animals, plants, fungi, and bacteria),
genome segment number (one to twelve), and virion organization (Triangulation
number, capsid layers, spikes, turrets, etc.). Members of this group include
the rotaviruses, renowned globally as the most common cause of gastroenteritis
in young children, and picobirnaviruses, renowned worldwide as the most
commonly occurring virus in fecal samples of both humans and animals with or
without signs of diarrhea. Bluetongue_virus is an economically important
pathogen of cattle and sheep. In recent years, remarkable progress has been
made in determining, at atomic and subnanometeric levels, the structures of a
number of key viral proteins and of the virion capsids of several dsRNA
viruses, highlighting the significant parallels in the structure and
replicative processes of many of these viruses.[2]
**** Mutation rates[edit] ****
RNA viruses generally have very high mutation rates compared to DNA_viruses,[6]
because viral RNA polymerases lack the proofreading ability of DNA_polymerases.
[7] This is one reason why it is difficult to make effective vaccines to
prevent diseases caused by RNA viruses.[8] Retroviruses also have a high
mutation rate even though their DNA intermediate integrates into the host
genome (and is thus subject to host DNA proofreading once integrated), because
errors during reverse transcription are embedded into both strands of DNA
before integration.[9] Some genes of RNA virus are important to the viral
replication cycles and mutations are not tolerated. For example, the region of
the hepatitis_C_virus genome that encodes the core protein is highly_conserved,
[10] because it contains an RNA structure involved in an internal_ribosome
entry_site.[11]
***** Replication[edit] *****
Animal RNA viruses are classified by the ICTV. There are three distinct groups
of RNA viruses depending on their genome and mode of replication:
    * Double-stranded RNA viruses (Group III) contain from one to a dozen
      different RNA molecules, each coding for one or more viral proteins.
    * Positive-sense ssRNA viruses (Group IV) have their genome directly
      utilized as mRNA, with host ribosomes translating it into a single
      protein that is modified by host and viral proteins to form the various
      proteins needed for replication. One of these includes RNA-dependent RNA
      polymerase (RNA replicase), which copies the viral RNA to form a double-
      stranded replicative form. In turn this dsRNA directs the formation of
      new viral RNA.
    * Negative-sense ssRNA viruses (Group V) must have their genome copied by
      an RNA replicase to form positive-sense RNA. This means that the virus
      must bring along with it the RNA replicase enzyme. The positive-sense RNA
      molecule then acts as viral mRNA, which is translated into proteins by
      the host ribosomes.
Retroviruses (Group VI) have a single-stranded RNA genome but, in general, are
not considered RNA viruses because they use DNA intermediates to replicate.
Reverse_transcriptase, a viral enzyme that comes from the virus itself after it
is uncoated, converts the viral RNA into a complementary strand of DNA, which
is copied to produce a double-stranded molecule of viral DNA. After this DNA is
integrated into the host genome using the viral enzyme integrase, expression of
the encoded genes may lead to the formation of new virions.
***** Classification[edit] *****
Classification of the RNA viruses has proven to be a difficult problem. This is
in part due to the high mutation rates these genomes undergo. Classification is
based principally on the type of genome (double-stranded, negative- or
positive-single-strand) and gene number and organisation. Currently there are 5
orders and 47 families of RNA viruses recognised. There are also many
unassigned species and genera.
Related to but distinct from the RNA viruses are the viroids and the RNA
satellite_viruses. These are not currently classified as RNA viruses and are
described on their own pages.
A study of several thousand RNA viruses has shown the presence of at least five
main taxa: a levivirus and relatives group; a picornavirus supergroup; an
alphavirus supergroup plus a flavivirus supergroup; the dsRNA viruses; and the
-ve strand viruses.[12] The lentivirus group appears to be basal to all the
remaining RNA viruses. The next major division lies between the
picornasupragroup and the remaining viruses. The dsRNA viruses appear to have
evolved from a +ve RNA ancestor and the -ve RNA viruses from within the dsRNA
viruses. The closest reltion to the -ve stranded RNA viruses are the
Reoviridae.
**** Positive strand RNA viruses[edit] ****
This is the single largest group of RNA viruses[13] with 30 families. Attempts
have been made to group these families in higher orders. These proposals were
based on an analysis of the RNA polymerases and are still under consideration.
To date, the suggestions proposed have not been broadly accepted because of
doubts over the suitability of a single gene to determine the taxonomy of the
clade.
The proposed classification of positive-strand RNA viruses is based on the RNA-
dependent RNA polymerase. Three groups have been recognised:[14]
   1. Bymoviruses, comoviruses, nepoviruses, nodaviruses, picornaviruses,
      potyviruses, sobemoviruses and a subset of luteoviruses (beet western
      yellows virus and potato leafroll virus)âthe picorna like group
      (Picornavirata).
   2. Carmoviruses, dianthoviruses, flaviviruses, pestiviruses, statoviruses,
      tombusviruses, single-stranded RNA bacteriophages, hepatitis C virus and
      a subset of luteoviruses (barley yellow dwarf virus)âthe flavi like
      group (Flavivirata).
   3. Alphaviruses, carlaviruses, furoviruses, hordeiviruses, potexviruses,
      rubiviruses, tobraviruses, tricornaviruses, tymoviruses, apple chlorotic
      leaf spot virus, beet yellows virus and hepatitis E virusâthe alpha
      like group (Rubivirata).
A division of the alpha-like (Sindbis-like) supergroup on the basis of a novel
domain located near the N termini of the proteins involved in viral replication
has been proposed.[15] The two groups proposed are: the 'altovirus' group
(alphaviruses, furoviruses, hepatitis E virus, hordeiviruses, tobamoviruses,
tobraviruses, tricornaviruses and probably rubiviruses); and the 'typovirus'
group (apple chlorotic leaf spot virus, carlaviruses, potexviruses and
tymoviruses).
The alpha like supergroup can be further divided into three clades: the rubi-
like, tobamo-like, and tymo-like viruses.[16]
Additional work has identified five groups of positive-stranded RNA viruses
containing four, three, three, three, and one order(s), respectively.[17] These
fourteen orders contain 31 virus families (including 17 families of plant
viruses) and 48 genera (including 30 genera of plant viruses). This analysis
suggests that alphaviruses and flaviviruses can be separated into two
familiesâthe Togaviridae and Flaviridae, respectivelyâbut suggests that
other taxonomic assignments, such as the pestiviruses, hepatitis C virus,
rubiviruses, hepatitis E virus, and arteriviruses, may be incorrect. The
coronaviruses and toroviruses appear to be distinct families in distinct orders
and not distinct genera of the same family as currently classified. The
luteoviruses appear to be two families rather than one, and apple chlorotic
leaf spot virus appears not to be a closterovirus but a new genus of the
Potexviridae.
  Evolution
The evolution of the picornaviruses based on an analysis of their RNA
polymerases and helicases appears to date to the divergence of the eukaryotes.
[18] Their putative ancestors include the bacterial group II retroelements, the
family of HtrA proteases and DNA bacteriophages.
Partitiviruses are related to and may have evolved from a totivirus ancestor.
[19]
Hypoviruses and barnaviruses appear to share an ancestry with the potyvirus and
sobemovirus lineages respectively.[19]
**** Double-stranded RNA viruses[edit] ****
This analysis also suggests that the dsRNA viruses are not closely related to
each other but instead belong to four additional classesâBirnaviridae,
Cystoviridae, Partitiviridae, and Reoviridae â and one additional order
(Totiviridae) of one of the classes of positive ssRNA viruses in the same
subphylum as the positive-strand RNA viruses.
One study has suggested that there are two large clades: One includes the
Caliciviridae, Flaviviridae, and Picornaviridae families and a second that
includes the Alphatetraviridae, Birnaviridae and Cystoviridae, Nodaviridae, and
Permutotretraviridae families.[20]
**** Negative strand RNA viruses[edit] ****
These viruses have multiple types of genome ranging from a single RNA molecule
up to eight segments. Despite their diversity it appears that they may have
originated in arthropods and to have diversified from there.[21]
**** Satellite viruses[edit] ****
A number of satellite viruses - viruses that require the assistance of another
virus to complete their life cycle - are also known. Their taxonomy has yet to
be settled. The following four genera have been proposed for positive sense
single stranded RNA satellite viruses that infect plants - Albetovirus,
Aumaivirus, Papanivirus and Virtovirus.[22] A family - Sarthroviridae which
includes the genus Macronovirus - has been proposed for the positive sense
single stranded RNA satellite viruses that infect arthropods.
***** Group IIIâdsRNA viruses[edit] *****
Main article: Double-stranded_RNA_viruses
There are twelve families and a number of unassigned genera and species
recognised in this group.[7]
    * Family Amalgaviridae
    * Family Birnaviridae
    * Family Chrysoviridae
    * Family Cystoviridae
    * Family Endornaviridae
    * Family Hypoviridae
    * Family Megabirnaviridae
    * Family Partitiviridae
    * Family Picobirnaviridae
    * Family Reoviridaeâincludes Rotavirus
    * Family Totiviridae
    * Family Quadriviridae
    * Genus Botybirnavirus
    * Unassigned species
          o Botrytis_porri_RNA_virus_1
          o Circulifer_tenellus_virus_1
          o Colletotrichum_camelliae_filamentous_virus_1
          o Cucurbit_yellows_associated_virus
          o Sclerotinia_sclerotiorum_debilitation-associated_virus
          o Spissistilus_festinus_virus_1
***** Group IVâpositive-sense ssRNA viruses[edit] *****
Main article: Positive-sense_single-stranded_RNA_virus
There are three orders and 34 families recognised in this group. In addition,
there are a number of unclassified species and genera.
    * Order Nidovirales
          o Family Arteriviridae
          o Family Coronaviridae â includes Coronavirus, SARS
          o Family Mesoniviridae
          o Family Roniviridae
    * Order Picornavirales
          o Family Dicistroviridae
          o Family Iflaviridae
          o Family Marnaviridae
          o Family Picornaviridaeâincludes Poliovirus, Rhinovirus (a common
            cold virus), Hepatitis_A virus
          o Family Secoviridae includes subfamily Comovirinae
          o Genus Bacillariornavirus
          o Genus Dicipivirus
          o Genus Labyrnavirus
          o Genus Sequiviridae
          o Species Kelp_fly_virus
    * Order Tymovirales
          o Family Alphaflexiviridae
          o Family Betaflexiviridae
          o Family Gammaflexiviridae
          o Family Tymoviridae
    * Unassigned
          o Family Alphatetraviridae
          o Family Alvernaviridae
          o Family Astroviridae
          o Family Barnaviridae
          o Family Benyviridae
          o Family Bromoviridae
          o Family Caliciviridae â includes Norwalk_virus
          o Family Carmotetraviridae
          o Family Closteroviridae
          o Family Flaviviridae â includes Yellow_fever virus, West_Nile
            virus, Hepatitis_C_virus, Dengue_fever virus, Zika_virus
          o Family Fusariviridae
          o Family Hepeviridae
          o Family Hypoviridae
          o Family Leviviridae
          o Family Luteoviridae â includes Barley_yellow_dwarf_virus
          o Family Polycipiviridae
          o Family Narnaviridae
          o Family Nodaviridae
          o Family Permutotetraviridae
          o Family Potyviridae
          o Family Sarthroviridae
          o Family Statovirus
          o Family Togaviridae â includes Rubella virus, Ross_River_virus,
            Sindbis_virus, Chikungunya_virus
          o Family Tombusviridae
          o Family Virgaviridae[23]
          o Unassigned genera
                # Genus Blunervirus
                # Genus Cilevirus
                # Genus Higrevirus
                # Genus Idaeovirus
                # Genus Negevirus
                # Genus Ourmiavirus
                # Genus Polemovirus
                # Genus Sinaivirus
                # Genus Sobemovirus
          o Unassigned species
                # Acyrthosiphon_pisum_virus
                # Bastrovirus
                # Blackford_virus
                # Blueberry_necrotic_ring_blotch_virus
                # Cadicistrovirus
                # Chara_australis_virus
                # Extra_small_virus
                # Goji_berry_chlorosis_virus
                # Hepelivirus
                # Jingmen_tick_virus
                # Le_Blanc_virus
                # Nedicistrovirus
                # Nesidiocoris_tenuis_virus_1
                # Niflavirus
                # Nylanderia_fulva_virus_1
                # Orsay_virus
                # Osedax_japonicus_RNA_virus_1
                # Picalivirus
                # Plasmopara_halstedii_virus
                # Rosellinia_necatrix_fusarivirus_1
                # Santeuil_virus
                # Secalivirus
                # Solenopsis_invicta_virus_3
                # Wuhan_large_pig_roundworm_virus
Satellite viruses
    * Family Sarthroviridae
    * Genus Albetovirus
    * Genus Aumaivirus
    * Genus Papanivirus
    * Genus Virtovirus
    * Chronic_bee_paralysis_virus
An unclassified astrovirus/hepevirus-like virus has also been described.[24]
***** Group Vânegative-sense ssRNA viruses[edit] *****
Main article: Negative-sense_single-stranded_RNA_virus
With the exception of the Hepatitis_D_virus, this group of viruses have been
placed into a single phylum - Negarnaviricota. This phylum has been divided
into two subphyla - Haploviricotina and Polyploviricotina. Within the subphylum
Haploviricotina four classes are currently recognised: Chunqiuviricetes,
Milneviricetes, Monjiviricetes and Yunchangviricetes. In the subphylum
Polyploviricotina two classes are recognised: Ellioviricetes and
Insthoviricetes.
Six classes, seven orders and twenty four families are currently recognised in
this group. A number of unassigned species and genera are yet to be classified.
[7]
    * Phylum Negarnaviricota[25]
          o Subphylum Haploviricotina
                # Class Chunqiuviricetes
                      # Order Muvirales
                            # Family Qinviridae
                # Class Milneviricetes
                      # Order Serpentovirales
                            # Family Aspiviridae
                # Class Monjiviricetes
                      # Order Jingchuvirales
                            # Family Chuviridae
                      # Order Mononegavirales
                            # Family Bornaviridae â Borna_disease_virus
                            # Family Filoviridae â includes Ebola virus,
                              Marburg_virus
                            # Family Mymonaviridae
                            # Family Nyamiviridae[26]
                            # Family Paramyxoviridae â includes Measles
                              virus, Mumps_virus, Nipah_virus, Hendra_virus,
                              and NDV
                            # Family Pneumoviridae â includes RSV and
                              Metapneumovirus
                            # Family Rhabdoviridae â includes Rabies virus
                            # Family Sunviridae
                            # Genus Anphevirus
                            # Genus Arlivirus
                            # Genus Chengtivirus
                            # Genus Crustavirus
                            # Genus Wastrivirus
                # Class Yunchangviricetes
                      # Order Goujianvirales
                            # Family Yueviridae
          o Subphylum Polyploviricotina
                # Class Ellioviricetes
                      # Order Bunyavirales
                            # Family Arenaviridae â includes Lassa_virus
                            # Family Cruliviridae
                            # Family Feraviridae
                            # Family Fimoviridae
                            # Family Hantaviridae
                            # Family Jonviridae
                            # Family Nairoviridae
                            # Family Peribunyaviridae
                            # Family Phasmaviridae
                            # Family Phenuiviridae
                            # Family Tospoviridae
                            # Genus Tilapineviridae
                # Class Insthoviricetes
                      # Order Articulavirales
                            # Family Amnoonviridae â includes Taastrup_virus
                            # Family Orthomyxoviridae â includes Influenza
                              viruses
    * Unassigned genera:
          o Genus Deltavirus â includes Hepatitis_D virus
***** Gallery[edit] *****
    * Lassa_virus (Arenaviridae)
    * Lymphocytic_choriomeningitis_virus (Arenaviridae)
    * Hantavirus (Bunyaviridae)
    * Marburg_Virus (Filoviridae)
    * Ebola_virus (Filoviridae)
    * Influenza (Orthomyxoviridae)
    * Measles (Paramyxoviridae)
    * Mumps_virus (Paramyxoviridae)
    * Human_respiratory_syncytial_virus (Paramyxoviridae)
    * Parainfluenza (Paramyxoviridae)
    * Rabies (Rhabdoviridae)
    * Vesicular_stomatitis_virus (Rhabdoviridae)
***** Notes[edit] *****
The majority of fungal viruses are double-stranded RNA viruses. A small number
of positive-strand RNA viruses have been described. One report has suggested
the possibility of a negative stranded virus.[27]
***** See also[edit] *****
    * [icon]Viruses_portal
    * Virus_classification
    * List_of_viruses
    * Viral_replication
    * Positive/negative-sense
    * Animal_viruses
    * Double-stranded_RNA_viruses
    * Retrovirus
    * DNA_viruses
    * Norovirus_cis-acting_replication_element
    * Viroid
***** References[edit] *****
   1. ^ MeSH, retrieved on 12 April 2008.
   2. ^ a bPatton JT (editor). (2008). Segmented_Double-stranded_RNA_Viruses:
      Structure_and_Molecular_Biology. Caister Academic Press. ISBN 978-1-
      904455-21-9.
   3. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   4. ^"Listing_in_Taxonomic_OrderâIndex_to_ICTV_Species_Lists". Retrieved 11
      April 2008.
   5. ^Drake JW, Holland JJ (November 1999). "Mutation_rates_among_RNA
      viruses". Proc. Natl. Acad. Sci. U.S.A. 96 (24): 13910â3. doi:10.1073/
      pnas.96.24.13910. PMC 24164. PMID 10570172.
   6. ^Nguyen M, Haenni AL (2003). "Expression strategies of ambisense
      viruses". Virus Res. 93 (2): 141â50. doi:10.1016/S0168-1702(03)00094-7.
      PMID 12782362.
   7. ^Sanjuan, R.; Nebot, M. R.; Chirico, N.; Mansky, L. M.; Belshaw, R.
      (2010). "Viral_Mutation_Rates". Journal of Virology. 84 (19):
      9733â9748. doi:10.1128/JVI.00694-10. ISSN 0022-538X. PMC 2937809.
      PMID 20660197.
   8. ^ a b cKlein, Donald W.; Prescott, Lansing M.; Harley, John (1993).
      Microbiology. Dubuque, Iowa: Wm. C. Brown. ISBN 978-0-697-01372-9.
   9. ^Steinhauer DA, Holland JJ (1987). "Rapid evolution of RNA viruses".
      Annu._Rev._Microbiol. 41: 409â33. doi:10.1146/
      annurev.mi.41.100187.002205. PMID 3318675.
  10. ^Boutwell CL, Rolland MM, Herbeck JT, Mullins JI, Allen TM (October
      2010). "Viral_Evolution_and_Escape_during_Acute_HIV-1_Infection". J.
      Infect._Dis. 202 (Suppl 2): S309â14. doi:10.1086/655653. PMC 2945609.
      PMID 20846038.
  11. ^Bukh J, Purcell RH, Miller RH (August 1994). "Sequence_analysis_of_the
      core_gene_of_14_hepatitis_C_virus_genotypes". Proc. Natl. Acad. Sci.
      U.S.A. 91 (17): 8239â43. doi:10.1073/pnas.91.17.8239. PMC 44581.
      PMID 8058787.
  12. ^Tuplin A, Evans DJ, Simmonds P (October 2004). "Detailed mapping of RNA
      secondary structures in core and NS5B-encoding region sequences of
      hepatitis C virus by RNase cleavage and novel bioinformatic prediction
      methods". J. Gen. Virol. 85 (Pt 10): 3037â47. doi:10.1099/vir.0.80141-
      0. PMID 15448367.
  13. ^ Wolf YI, Kazlauskas D, Iranzo J, LucÃ­a-Sanz A, Kuhn JH, Krupovic M,
      Dolja VV, Koonin EV (2018) Origins and Evolution of the Global RNA
      Virome. MBio 9(6) pii: e02329-18
  14. ^Francki, R.I.B; Fauquet, C. M.; Knudson, D. L.; Brown, F. (1991).
      Classification and nomenclature of viruses. Fifth report of the
      International Committee on Taxonomy of Viruses, Archives of Virology
      (Suppl. 2). ISBN 978-3-7091-9163-7.
  15. ^Koonin EV (September 1991). "The_phylogeny_of_RNA-dependent_RNA
      polymerases_of_positive-strand_RNA_viruses". J. Gen. Virol. 72 (Pt 9):
      2197â206. doi:10.1099/0022-1317-72-9-2197. PMID 1895057.
  16. ^Rozanov MN, Koonin EV, Gorbalenya AE (August 1992). "Conservation_of_the
      putative_methyltransferase_domain:_a_hallmark_of_the_'Sindbis-like'
      supergroup_of_positive-strand_RNA_viruses". J. Gen. Virol. 73 (Pt 8):
      2129â34. CiteSeerX 10.1.1.532.7367. doi:10.1099/0022-1317-73-8-2129.
      PMID 1645151.
  17. ^Koonin EV, Dolja VV (1993). "Evolution and taxonomy of positive-strand
      RNA viruses: implications of comparative analysis of amino acid
      sequences". Crit. Rev. Biochem. Mol. Biol. 28 (5): 375â430. doi:
      10.3109/10409239309078440. PMID 8269709.
  18. ^Ward CW (1993). "Progress towards a higher taxonomy of viruses". Res
      Virol. 144 (6): 419â453. doi:10.1016/S0923-2516(06)80059-2.
      PMID 8140287.
  19. ^Koonin EV, Wolf YI, Nagasaki K, Dolja VV (December 2008). "The Big Bang
      of picorna-like virus evolution antedates the radiation of eukaryotic
      supergroups". Nat. Rev. Microbiol. 6 (12): 925â39. doi:10.1038/
      nrmicro2030. PMID 18997823.
  20. ^ a b Ghabrial_SA (1998) Origin, adaptation and evolutionary pathways of
      fungal viruses. Virus Genes 16(1):119-131
  21. ^Gibrat JF, Mariadassou M, Boudinot P, Delmas B (2013). "Analyses_of_the
      radiation_of_birnaviruses_from_diverse_host_phyla_and_of_their
      evolutionary_affinities_with_other_double-stranded_RNA_and_positive
      strand_RNA_viruses_using_robust_structure-based_multiple_sequence
      alignments_and_advanced_phylogenetic_methods". BMC Evol. Biol. 13: 154.
      doi:10.1186/1471-2148-13-154. PMC 3724706. PMID 23865988.
  22. ^ Li CX, Shi M, Tian JH, Lin XD, Kang YJ, Chen LJ, Qin XC, Xu J, Holmes
      EC, Zhang YZ (2015) Unprecedented genomic diversity of RNA viruses in
      arthropods reveals the ancestry of negative-sense RNA viruses. Elife 4.
      doi: 10.7554/eLife.05378.
  23. ^ Krupovic M, Kuhn JH, Fischer MG (2016) A classification system for
      virophages and satellite viruses. Arch Virol 161(1):233-247 doi: 10.1007/
      s00705-015-2622-9
  24. ^Adams MJ, Antoniw JF, Kreuze J (2009). "Virgaviridae: a new family of
      rod-shaped plant viruses". Arch Virol. 154 (12): 1967â72. doi:10.1007/
      s00705-009-0506-6. PMID 19862474.
  25. ^ Pankovics P, Boros Ã, Kiss T, Engelmann P, Reuter G (2019) Genetically
      highly divergent RNA virus with astrovirus-like (5'-end) and hepevirus-
      like (3'-end) genome organization in carnivorous birds, European roller
      (Coracias garrulus). Infect Genet Evol
  26. ^"Virus_Taxonomy:_2018_Release". International_Committee_on_Taxonomy_of
      Viruses. Retrieved 13 November 2018.
  27. ^Mihindukulasuriya, K. A.; Nguyen, N. L.; Wu, G.; Huang, H. V.; Travassos
      da Rosa, A. P.; Popov, V. L.; Tesh, R. B.; Wang, D. (2009). "Nyamanini
      and_Midway_viruses_define_a_novel_taxon_of_RNA_viruses_in_the_order
      Mononegavirales". J. Virol. 83 (10): 5109â16. doi:10.1128/JVI.02667-08.
      PMC 2682064. PMID 19279111.
  28. ^Kondo, H.; Chiba, S.; Toyoda, K.; Suzuki, N. (2012). "Evidence for
      negative-strand RNA virus infection in fungi". Virology. 435 (2):
      201â9. doi:10.1016/j.virol.2012.10.002. PMID 23099204.
***** External links[edit] *****
    * RNA_Viruses at the US National Library of Medicine Medical_Subject
      Headings (MeSH)
    * Animal_viruses
    * v
    * t
    * e
Microbiology: Virus
    * Introduction
    * History
    * Social_history_of_viruses
                     * Viral_envelope
Components           * Capsid
                     * Viral_protein
                     * Viral_entry
                     * Viral_replication
Viral_life_cycle     * Viral_shedding
                     * Virus_latency
                     * Viroplasm
                     * Reassortment
Genetics             * Antigenic_shift
                     * Antigenic_drift
                     * Phenotype_mixing
                     * Bacteriophage
                     * Virophage
                     * Mycovirus
By host              * Plant_virus (Plant_to   [Rotavirus_Reconstruction.jpg]
                       Human)
                     * Animal_virus
                     * Human_virome
                     * Viral_disease
                     * Helper_virus
                     * Laboratory_diagnosis_of
                       viral_infections
                     * Viral_load
Other                * Virus-like_particle
                     * Virus_quantification
                     * Antiviral_drug
                     * Neurotropic_virus
                     * Oncovirus
                     * Satellites
                     * Giant_virus
    * [Category] Category
    * [Portal] Portal
    * [Commons page] common
    * [WikiProject] WikiProject
    * v
    * t
    * e
Baltimore (virus_classification)
    I: dsDNA_viruses
                       * Myoviridae
    Caudovirales       * Podoviridae
                       * Siphoviridae
                       * Alloherpesviridae
    Herpesvirales      * Herpesviridae
                       * Malacoherpesviridae
    Ligamenvirales     * Lipothrixviridae
                       * Rudiviridae
                       * NLCDV: Ascoviridae
                       * Asfarviridae
                       * Iridoviridae
                       * Marseilleviridae
                       * Mimiviridae
                       * Phycodnaviridae
                       * Poxviridae
                       * genera: Dinodnavirus
                       * Pithovirus
                       * Pandoravirus
                       * nonenveloped:
                         Adenoviridae
                       * Papillomaviridae
                       * Papovaviridae
                         (obsolete)
                       * Polyomaviridae
                       * genera: Rhizidiovirus
    Unassigned         * Ampullaviridae
DNA                    * Baculoviridae
                       * Bicaudaviridae
                       * Clavaviridae
                       * Corticoviridae
                       * Fuselloviridae
                       * Globuloviridae
                       * Guttaviridae
                       * Hytrosaviridae
                       * Nimaviridae
                       * Nudiviridae
                       * Plasmaviridae
                       * Polydnaviridae
                       * Sphaerolipoviridae
                       * Tectiviridae
                       * Turriviridae
                       * genera:
                         Salterprovirus
    II: ssDNA_viruses
        * Anelloviridae
        * Bidnaviridae
        * Circoviridae
        * Geminiviridae
        * Inoviridae
        * Microviridae
        * Nanoviridae
        * Parvoviridae
        * Spiraviridae
        * genera: Bacilladnavirus
    III: dsRNA_viruses
        * Amalgaviridae
        * Birnaviridae
        * Chrysoviridae
        * Cystoviridae
        * Endornaviridae
        * Hypoviridae
        * Megabirnaviridae
        * Partitiviridae
        * Picobirnaviridae
        * Quadriviridae
        * Reoviridae
        * Totiviridae
    IV: (+)ssRNA_viruses (primarily
    icosahedral)
                       * Arteriviridae
    Nidovirales        * Coronaviridae
                       * Mesoniviridae
                       * Roniviridae
                       * Comoviridae (obsolete)
                       * Dicistroviridae
                       * Iflaviridae
                       * Marnaviridae
                       * Picornaviridae
    Picornavirales     * Secoviridae
                       * Sequiviridae
                         (obsolete)
                       * genera:
                         Bacillarnavirus
                       * Labyrnavirus
                       * Alphaflexiviridae
    Tymovirales        * Betaflexiviridae
                       * Gammaflexiviridae
                       * Tymoviridae
                       * Alphatetraviridae
                       * Alvernaviridae
                       * Astroviridae
                       * Barnaviridae
                       * Benyviridae
                       * Bromoviridae
                       * Caliciviridae
RNA                    * Carmotetraviridae
                       * Closteroviridae
                       * Flaviviridae
                       * Flexiviridae
                         (obsolete)
                       * Hepeviridae
                       * Leviviridae
    Unassigned         * Luteoviridae
                       * Narnaviridae
                       * Nodaviridae
                       * Permutotetraviridae
                       * Potyviridae
                       * Tetraviridae
                         (obsolete)
                       * Togaviridae
                       * Tombusviridae
                       * Virgaviridae
                       * genera: Cilevirus
                       * Higrevirus
                       * Idaeovirus
                       * Ourmiavirus
                       * Polemovirus
                       * Sobemovirus
    V: (â)ssRNA_viruses (primarily helical)
                        * Bornaviridae
                        * Filoviridae
    Mononegavirales     * Nyamiviridae
                        * Paramyxoviridae
                        * Rhabdoviridae
                        * Hantaviridae
                        * Nairoviridae
                        * Peribunyaviridae
                        * Phenuiviridae
    Bunyavirales        * Tospoviridae
                        * Feraviridae
                        * Fimoviridae
                        * Jonviridae
                        * Phasmaviridae
                        * Arenaviridae
    Unassigned          * Ophioviridae
                        * Orthomyxoviridae
                        * genera:Deltavirus
    VI: ssRNA-RT_viruses
        * Metaviridae
        * Pseudoviridae
RT      * Retroviridae
    VII: dsDNA-RT
    viruses
        * Caulimoviridae
        * Hepadnaviridae

Retrieved from "https://en.wikipedia.org/w/
index.php?title=RNA_virus&oldid=906397051"
Categories:
    * RNA_viruses
    * RNA
Hidden categories:
    * Use_dmy_dates_from_September_2011
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
    * Wikispecies
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Afrikaans
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Italiano
    * Magyar
    * à´®à´²à´¯à´¾à´³à´
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Norsk_nynorsk
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Simple_English
    * SlovenÅ¡Äina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Svenska
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * ä¸­æ
Edit_links
    * This page was last edited on 15 July 2019, at 15:47 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
