The following text has been accessed from https://en.wikipedia.org/wiki/Salmonella at Fri Aug 9 01:34:35 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Salmonella ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
This article is about the bacteria. For the disease caused by such bacteria,
see Salmonellosis.
Genus of prokaryotes
Salmonella
[SalmonellaNIAID.jpg]
Color-enhanced scanning_electron_micrograph showing Salmonella Typhimurium
(red) invading cultured human cells
Scientific_classification [e]
Domain: Bacteria
Phylum: Proteobacteria
Class:  Gammaproteobacteria
Order:  Enterobacteriales
Family: Enterobacteriaceae
Genus:  Salmonella
        LigniÃ¨res 1900
Species and subspecies[1]
    * Salmonella_bongori
    * Salmonella_enterica
          o Salmonella_enterica_subsp._arizonae
          o Salmonella_enterica_subsp._diarizonae
          o Salmonella_enterica_subsp._enterica
          o Salmonella_enterica_subsp._houtenae
          o Salmonella_enterica_subsp._indica
          o Salmonella_enterica_subsp._salamae
Salmonella is a genus of rod-shaped (bacillus) Gram-negative bacteria of the
family Enterobacteriaceae. The two species of Salmonella are Salmonella
enterica and Salmonella_bongori. S. enterica is the type species and is further
divided into six subspecies[2] that include over 2,600 serotypes.[3] Salmonella
was named after Daniel_Elmer_Salmon (1850â1914), an American veterinary
surgeon. [4]
Salmonella species are non-spore-forming, predominantly motile enterobacteria
with cell diameters between about 0.7 and 1.5 Âµm, lengths from 2 to 5 Âµm, and
peritrichous flagella (all around the cell body).[5] They are chemotrophs,
obtaining their energy from oxidation_and_reduction_reactions using organic
sources. They are also facultative_aerobes, capable of generating ATP with
oxygen ("aerobically") when it is available, or when oxygen is not available,
using other electron acceptors or fermentation ("anaerobically").[5] S.
enterica subspecies are found worldwide in all warm-blooded animals and in the
environment. S. bongori is restricted to cold-blooded_animals, particularly
reptiles.[6]
Salmonella species are intracellular_pathogens;[7] certain serotypes causing
illness. Nontyphoidal serotypes can be transferred from animal-to-human and
from human-to-human. They usually invade only the gastrointestinal tract and
cause salmonellosis, the symptoms of which can be resolved without antibiotics.
However, in sub-Saharan_Africa, nontyphoidal Salmonella can be invasive and
cause paratyphoid_fever, which requires immediate treatment with antibiotics.
Typhoidal serotypes can only be transferred from human-to-human, and can cause
food-borne infection, typhoid_fever, and paratyphoid fever.[8] Typhoid fever is
caused by Salmonella invading the bloodstream (the typhoidal form), or in
addition spreads throughout the body, invades organs, and secretes endotoxins
(the septic form). This can lead to life-threatening hypovolemic_shock and
septic_shock, and requires intensive_care including antibiotics.
The collapse of the Aztec society in Mesoamerica is linked to a catastrophic
Salmonella outbreak, one of humanity's deadliest, that occurred after the
Spanish_conquest.[9]
⁰
***** Contents *****
    * 1_Taxonomy
    * 2_History
    * 3_Serotyping
    * 4_Detection,_culture,_and_growth_conditions
    * 5_Nomenclature
    * 6_Pathogenicity
    * 7_Nontyphoidal_Salmonella
          o 7.1_Non-invasive
          o 7.2_Invasive
    * 8_Typhoidal_Salmonella
    * 9_Global_monitoring
    * 10_Molecular_mechanisms_of_infection
          o 10.1_Selective_Immune_Knockout
    * 11_Resistance_to_oxidative_burst
    * 12_Host_adaptation
    * 13_Genetics
    * 14_See_also
    * 15_References
    * 16_External_links
***** Taxonomy[edit] *****
The genus Salmonella is part of the family of Enterobacteriaceae. Its taxonomy
has been revised and has the potential to confuse. The genus comprises two
species, S. bongori and S. enterica, the latter of which is divided into six
subspecies: S. e. enterica, S. e. salamae, S. e. arizonae, S. e. diarizonae, S.
e. houtenae, and S. e. indica.[10][11] The taxonomic group contains more than
2500 serotypes (also serovars) defined on the basis of the somatic O
(lipopolysaccharide) and flagellar H antigens (the KauffmanâWhite
classification). The full name of a serotype is given as, for example,
Salmonella enterica subsp. enterica serotype Typhimurium, but can be
abbreviated to Salmonella Typhimurium. Further differentiation of strains to
assist clinical and epidemiological investigation may be achieved by antibiotic
sensitivity_testing and by other molecular_biology techniques such as pulsed-
field_gel_electrophoresis, multilocus_sequence_typing, and, increasingly, whole
genome_sequencing. Historically, salmonellae have been clinically categorized
as invasive (typhoidal) or noninvasive (nontyphoidal salmonellae) based on host
preference and disease manifestations in humans.[12]
***** History[edit] *****
Salmonella was first visualized in 1880 by Karl_Eberth in the Peyer's_patches
and spleens of typhoid patients.[13] Four years later, Georg_Theodor_Gaffky was
able to successfully grow the pathogen in pure culture.[14] A year after that,
medical research scientist Theobald_Smith discovered what would be later known
as Salmonella_enterica (var. Choleraesuis). At the time, Smith was working as a
research laboratory assistant in the Veterinary Division of the United_States
Department_of_Agriculture. The department was under the administration of
Daniel_Elmer_Salmon, a veterinary pathologist.[15] Initially, Salmonella
Choleraesuis was thought to be the causative agent of hog_cholera, so Salmon
and Smith named it "Hog-cholerabacillus". The name Salmonella was not used
until 1900, when Joseph Leon LigniÃ¨res proposed that the pathogen discovered
by Salmon's group be called Salmonella in his honor.[16]:16
***** Serotyping[edit] *****
Serotyping is done by mixing cells with antibodies for a particular antigen. It
can give some idea about risk. For example, in 2014, a study showed that S.
Reading is very common among young turkey samples, but it is not a significant
contributor to human salmonellosis.[17] Serotyping can assist identify the
source of contamination. For example, if we have a case of food illness. There
are two ways to figure out the source of contamination. One of them is
serotyping the Salmonella in infected people, and serotyping Salmonella found
at possible source of contamination (e.g. where food was prepared), and then
determining if the serotypes match. The other way is to serotype the Salmonella
in infected people, see the result and if the result matches with a particular
kind of serotype of Salmonella that can only found in a particular species of
animals or at particular place, then we can know our source of contamination.
[18]  Moreover, serotyping may suggest adequate treatment by giving us the idea
about the antibiotic resistance of salmonella species. For example,
sulfonamides are antibiotics which are appropriate for treatment of Salmonella
enterica serotype Typhi (normally it is just shortened to S.Typhi).[19]
Serotyping is a faster method than antibiotic sensitivity testing culture
methods.
***** Detection, culture, and growth conditions[edit] *****
US Food_and_Drug_Administration scientist tests for presence of Salmonella
Most subspecies of Salmonella produce hydrogen_sulfide,[20] which can readily
be detected by growing them on media containing ferrous_sulfate, such as is
used in the triple_sugar_iron test. Most isolates exist in two phases, a motile
phase I and a nonmotile phase II. Cultures that are nonmotile upon primary
culture may be switched to the motile phase using a Craigie_tube or ditch
plate.[21] RVS_broth can be used to enrich for Salmonella species for detection
in a clinical sample.[22]
Salmonella can also be detected and subtyped using multiplex[23] or real-time
polymerase_chain_reactions (PCR)[24] from extracted Salmonella DNA.
Mathematical models of Salmonella growth kinetics have been developed for
chicken, pork, tomatoes, and melons.[25][26][27][28][29] Salmonella reproduce
asexually with a cell division interval of 40 minutes.[16]:16
Salmonella species lead predominantly host-associated lifestyles, but the
bacteria were found to be able to persist in a bathroom setting for weeks
following contamination, and are frequently isolated from water sources, which
act as bacterial reservoirs and may help to facilitate transmission between
hosts.[30] Salmonella is notorious for its ability to survive desiccation and
can persist for years in dry environments and foods.[31]
The bacteria are not destroyed by freezing,[32][33] but UV_light and heat
accelerate their destruction. They perish after being heated to 55 Â°C
(131 Â°F) for 90 min, or to 60 Â°C (140 Â°F) for 12 min.[34] To protect against
Salmonella infection, heating food to an internal temperature of 75 Â°C
(167 Â°F) is recommended.[35][36]
Salmonella species can be found in the digestive tracts of humans and animals,
especially reptiles. Salmonella on the skin of reptiles or amphibians can be
passed to people who handle the animals.[37] Food and water can also be
contaminated with the bacteria if they come in contact with the feces of
infected people or animals.[38]
***** Nomenclature[edit] *****
Initially, each Salmonella "species" was named according to clinical
considerations,[39] for example Salmonella typhi-murium (mouse typhoid fever),
S. cholerae-suis. After host specificity was recognized to not exist for many
species, new strains received species names according to the location at which
the new strain was isolated. Later, molecular findings led to the hypothesis
that Salmonella consisted of only one species,[40] S. enterica, and the
serotypes were classified into six groups,[41] two of which are medically
relevant. As this now-formalized nomenclature[42][43] is not in harmony with
the traditional usage familiar to specialists in microbiology and
infectologists, the traditional nomenclature is still common. Currently, the
two recognized species are S._enterica, and S._bongori. In 2005, a third
species, Salmonella subterranean, was proposed, but according to the World
Health Organization, the bacterium reported does not belong in the genus
Salmonella.[44] The six main recognised subspecies are: enterica (serotype I),
salamae (serotype II), arizonae (IIIa), diarizonae (IIIb), houtenae (IV), and
indica (VI).[45] The former serotype (V) was bongori, which is now considered
its own species.
The serotype or serovar, is a classification of Salmonella into subspecies
based on antigens that the organism presents. It is based on the Kauffman-White
classification scheme that differentiates serological varieties from each
other. Serotypes are usually put into subspecies groups after the genus and
species, with the serotypes/serovars capitalized, but not italicized: An
example is Salmonella enterica serovar Typhimurium. More modern approaches for
typing and subtyping Salmonella include DNA-based methods such as pulsed_field
gel_electrophoresis, multiple-loci_VNTR_analysis, multilocus_sequence_typing,
and multiplex-PCR-based methods.[46][47]
***** Pathogenicity[edit] *****
Salmonella species are facultative intracellular_pathogens.[7] A facultative
organism uses oxygen to make ATP; when it is not available, it "exercises its
option"âthe literal meaning of the termâand makes ATP by fermentation, or
by substituting one or more of four less efficient electron acceptors as oxygen
at the end of the electron transport chain: sulfate, nitrate, sulfur, or
fumarate.
Most infections are due to ingestion of food contaminated by animal feces, or
by human feces, such as by a food-service worker at a commercial eatery.
Salmonella serotypes can be divided into two main groupsâtyphoidal and
nontyphoidal. Nontyphoidal serotypes are more common, and usually cause self-
limiting gastrointestinal disease. They can infect a range of animals, and are
zoonotic, meaning they can be transferred between humans and other animals.
Typhoidal serotypes include Salmonella Typhi and Salmonella Paratyphi A, which
are adapted to humans and do not occur in other animals.
***** Nontyphoidal Salmonella[edit] *****
See also: Salmonellosis
**** Non-invasive[edit] ****
Infection with nontyphoidal serotypes of Salmonella generally results in food
poisoning. Infection usually occurs when a person ingests foods that contain a
high concentration of the bacteria. Infants and young children are much more
susceptible to infection, easily achieved by ingesting a small number of
bacteria. In infants, infection through inhalation of bacteria-laden dust is
possible.
The organisms enter through the digestive tract and must be ingested in large
numbers to cause disease in healthy adults. An infection can only begin after
living salmonellae (not merely Salmonella-produced toxins) reach the
gastrointestinal tract. Some of the microorganisms are killed in the stomach,
while the surviving ones enter the small intestine and multiply in tissues.
Gastric acidity is responsible for the destruction of the majority of ingested
bacteria, but Salmonella has evolved a degree of tolerance to acidic
environments that allows a subset of ingested bacteria to survive.[48]
Bacterial colonies may also become trapped in mucus produced in the esophagus.
By the end of the incubation period, the nearby host cells are poisoned by
endotoxins released from the dead salmonellae. The local response to the
endotoxins is enteritis and gastrointestinal disorder.
About 2,000 serotypes of nontyphoidal Salmonella are known, which may be
responsible for as many as 1.4 million illnesses in the United States each
year. People who are at risk for severe illness include infants, elderly,
organ-transplant recipients, and the immunocompromised.[38]
**** Invasive[edit] ****
While in developed countries, nontyphoidal serotypes present mostly as
gastrointestinal disease, in sub-Saharan Africa, these serotypes can create a
major problem in bloodstream infections, and are the most commonly isolated
bacteria from the blood of those presenting with fever. Bloodstream infections
caused by nontyphoidal salmonellae in Africa were reported in 2012 to have a
case_fatality_rate of 20â25%. Most cases of invasive nontyphoidal salmonella
infection (iNTS) are caused by S. typhimurium or S. enteritidis. A new form of
Salmonella typhimurium (ST313) emerged in the southeast of the African
continent 75 years ago, followed by a second wave which came out of central
Africa 18 years later. This second wave of iNTS possibly originated in the
Congo Basin, and early in the event picked up a gene that made it resistant to
the antibiotic chloramphenicol. This created the need to use expensive
antimicrobial drugs in areas of Africa that were very poor, making treatment
difficult. The increased prevalence of iNTS in sub-Saharan Africa compared to
other regions is thought to be due to the large proportion of the African
population with some degree of immune suppression or impairment due to the
burden of HIV, malaria, and malnutrition, especially in children. The genetic
makeup of iNTS is evolving into a more typhoid-like bacterium, able to
efficiently spread around the human body. Symptoms are reported to be diverse,
including fever, hepatosplenomegaly, and respiratory symptoms, often with an
absence of gastrointestinal symptoms.[49]
***** Typhoidal Salmonella[edit] *****
See also: Typhoid_fever and Paratyphoid_fever
Typhoid fever is caused by Salmonella serotypes which are strictly adapted to
humans or higher primatesâthese include Salmonella_typhi, Paratyphi A,
Paratyphi B, and Paratyphi C. In the systemic form of the disease, salmonellae
pass through the lymphatic system of the intestine into the blood of the
patients (typhoid form) and are carried to various organs (liver, spleen,
kidneys) to form secondary foci (septic form). Endotoxins first act on the
vascular and nervous apparatus, resulting in increased permeability and
decreased tone of the vessels, upset of thermal regulation, and vomiting and
diarrhoea. In severe forms of the disease, enough liquid and electrolytes are
lost to upset the water-salt metabolism, decrease the circulating blood volume
and arterial pressure, and cause hypovolemic_shock. Septic_shock may also
develop. Shock of mixed character (with signs of both hypovolemic and septic
shock) is more common in severe salmonellosis. Oliguria and azotemia may
develop in severe cases as a result of renal involvement due to hypoxia and
toxemia.
***** Global monitoring[edit] *****
In Germany, food-borne infections must be reported.[50] From 1990 to 2016, the
number of officially recorded cases decreased from about 200,000 to about
13,000 cases.[51] In the United States, about 1,200,000 cases of Salmonella
infection are estimated to occur each year.[52] A World Health Organization
study estimated that 21,650,974 cases of typhoid fever occurred in 2000,
216,510 of which resulted in death, along with 5,412,744 cases of paratyphoid
fever.[53]
***** Molecular mechanisms of infection[edit] *****
Mechanisms of infection differ between typhoidal and nontyphoidal serotypes,
owing to their different targets in the body and the different symptoms that
they cause. Both groups must enter by crossing the barrier created by the
intestinal cell wall, but once they have passed this barrier, they use
different strategies to cause infection.
Nontyphoidal serotypes preferentially enter M_cells on the intestinal wall by
bacterial-mediated endocytosis, a process associated with intestinal
inflammation and diarrhoea. They are also able to disrupt tight_junctions
between the cells of the intestinal wall, impairing the cells' ability to stop
the flow of ions, water, and immune cells into and out of the intestine. The
combination of the inflammation caused by bacterial-mediated endocytosis and
the disruption of tight junctions is thought to contribute significantly to the
induction of diarrhoea.[54]
Salmonellae are also able to breach the intestinal barrier via phagocytosis and
trafficking by CD18-positive immune cells, which may be a mechanism key to
typhoidal Salmonella infection. This is thought to be a more stealthy way of
passing the intestinal barrier, and may, therefore, contribute to the fact that
lower numbers of typhoidal Salmonella are required for infection than
nontyphoidal Salmonella.[54] Salmonella cells are able to enter macrophages via
macropinocytosis.[55] Typhoidal serotypes can use this to achieve dissemination
throughout the body via the mononuclear_phagocyte_system, a network of
connective tissue that contains immune cells, and surrounds tissue associated
with the immune system throughout the body.[54]
Much of the success of Salmonella in causing infection is attributed to two
type_III_secretion_systems which function at different times during an
infection. One is required for the invasion of nonphagocytic_cells,
colonization of the intestine, and induction of intestinal inflammatory
responses and diarrhea. The other is important for survival in macrophages and
establishment of systemic disease.[54] These systems contain many genes which
must work co-operatively to achieve infection.
The AvrA toxin injected by the SPI1 type III secretion system of S. Typhimurium
works to inhibit the innate_immune_system by virtue of its serine/threonine
acetyltransferase activity, and requires binding to eukaryotic target cell
phytic_acid (IP6).[56] This leaves the host more susceptible to infection.
Salmonellosis is known to be able to cause back_pain or spondylosis. It can
manifest as five clinical patterns: gastrointestinal tract infection, enteric
fever, bacteremia, local infection, and the chronic reservoir state. The
initial symptoms are nonspecific fever, weakness, and myalgia among others. In
the bacteremia state, it can spread to any parts of the body and this induces
localized infection or it forms abscesses. The forms of localized Salmonella
infections are arthritis, urinary tract infection, infection of the central
nervous system, bone infection, soft tissue infection, etc.[57] Infection may
remain as the latent form for a long time, and when the function of reticular
endothelial_cells is deteriorated, it may become activated and consequently, it
may secondarily induce spreading infection in the bone several months or
several years after acute salmonellosis.[57]
**** Selective Immune Knockout[edit] ****
A 2018 Imperial_College_London study shows how salmonella disrupt specific arms
of the immune_system (e.g. 3 of 5 NF-kappaB proteins) using a family of zinc
metalloproteinase effectors, leaving others untouched.[58]
***** Resistance to oxidative burst[edit] *****
A hallmark of Salmonella pathogenesis is the ability of the bacterium to
survive and proliferate within phagocytes. Phagocytes produce DNA-damaging
agents such as nitric_oxide and oxygen radicals as a defense against pathogens.
Thus, Salmonella species must face attack by molecules that challenge genome
integrity. Buchmeieret al.[59] showed that mutants of S. enterica lacking RecA
or RecBC protein function are highly sensitive to oxidative compounds
synthesized by macrophages, and furthermore these findings indicate that
successful systemic infection by S. enterica requires RecA- and RecBC-mediated
recombinational repair of DNA damage.[59][60]
***** Host adaptation[edit] *****
S. enterica, through some of its serotypes such as Typhimurium and Enteriditis,
shows signs of the ability to infect several different mammalian host species,
while other serotypes such as Typhi seem to be restricted to only a few hosts.
[61] Some of the ways that Salmonella serotypes have adapted to their hosts
include loss of genetic material and mutation. In more complex mammalian
species, immune_systems, which include pathogen specific immune responses,
target serovars of Salmonella through binding of antibodies to structures such
as flagella. Through the loss of the genetic material that codes for a
flagellum to form, Salmonella can evade a host's immune_system.[62] mgtC leader
RNA from bacteria virulence gene (mgtCBR operon) decreases flagellin production
during infection by directly base pairing with mRNAs of the fljB gene encoding
flagellin and promotes degradation.[63] In the study by Kisela et al., more
pathogenic serovars of S. enterica were found to have certain adhesins in
common that have developed out of convergent evolution.[64] This means that, as
these strains of Salmonella have been exposed to similar conditions such as
immune systems, similar structures evolved separately to negate these similar,
more advanced defenses in hosts. Still, many questions remain about the way
that Salmonella has evolved into so many different types, but Salmonella may
have evolved through several phases. As Baumler et al. have suggested,
Salmonella most likely evolved through horizontal_gene_transfer, formation of
new serovars due to additional pathogenicity_islands. and an approximation of
its ancestry.[65] So, Salmonella could have evolved into its many different
serotypes through gaining genetic information from different pathogenic
bacteria. The presence of several pathogenicity_islands in the genome of
different serotypes has lent credence to this theory.[65]
Salmonella sv. Newport has signs of adaptation to a plant colonization
lifestyle, which may play a role in its disproportionate association with
foodborne illness linked to produce. A variety of functions selected for during
sv. Newport persistence in tomatoes have been reported to be similar to those
selected for in sv. Typhimurium from animal hosts.[66] The papA gene, which is
unique to sv. Newport, contributes to the strain's fitness in tomatoes, and has
homologs in genomes of other Enterobacteriaceae that are able to colonize plant
and animal hosts.[66]
***** Genetics[edit] *****
In addition to its importance as a pathogen, S. enterica serovar Typhimurium
has been instrumental in the development of genetic tools that led to an
understanding of fundamental bacterial physiology. These developments were
enabled by the discovery of the first generalized transducing phage, P22,[67]
in Typhimurium that allowed quick and easy genetic exchange that allowed fine
structure genetic analysis. The large number of mutants led to a revision of
genetic nomenclature for bacteria.[68] Many of the uses of transposons as
genetic tools, including transposon delivery, mutagenesis, construction of
chromosome rearrangements, were also developed in Typhimurium. These genetic
tools also led to a simple test for carcinogens, the Ames test.[69]
***** See also[edit] *****
    * Host-pathogen_interface
    * 1984_Rajneeshee_bioterror_attack
    * 2008_United_States_salmonellosis_outbreak
    * 2008â2009_peanut-borne_salmonellosis
    * Bismuth_sulfite_agar
    * Food_testing_strips
    * List_of_foodborne_illness_outbreaks
    * Wright_County_Egg
    * Rappaport_Vassiliadis_soya_peptone_broth
    * XLD_agar
    * American_Public_Health_Association_v._Butz
***** References[edit] *****
   1. ^"Salmonella". NCBI taxonomy. Bethesda, MD: National Center for
      Biotechnology Information. Retrieved 27 January 2019.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Su LH, Chiu CH (2007). "Salmonella: clinical importance and evolution of
      nomenclature". Chang Gung Medical Journal. 30 (3): 210â9.
      PMID 17760271.
   4. ^Gal-Mor O, Boyle EC, Grassl GA (2014). "Same_species,_different
      diseases:_how_and_why_typhoidal_and_non-typhoidal_Salmonella_enterica
      serovars_differ". Frontiers in Microbiology. 5: 391. doi:10.3389/
      fmicb.2014.00391. PMC 4120697. PMID 25136336.
   5. ^ https://www.dictionary.com/browse/salmonella
   6. ^ a bFÃ brega A, Vila J (April 2013). "Salmonella_enterica_serovar
      Typhimurium_skills_to_succeed_in_the_host:_virulence_and_regulation".
      Clinical Microbiology Reviews. 26 (2): 308â41. doi:10.1128/CMR.00066-
      12. PMC 3623383. PMID 23554419.
   7. ^Tortora GA (2008). Microbiology:_An_Introduction] (9th ed.). Pearson.
      pp. 323â324. ISBN 978-8131722329.
   8. ^ a bJantsch J, Chikkaballi D, Hensel M (March 2011). "Cellular aspects
      of immunity to intracellular Salmonella enterica". Immunological Reviews.
      240 (1): 185â95. doi:10.1111/j.1600-065X.2010.00981.x. PMID 21349094.
   9. ^Ryan I KJ, Ray CG, eds. (2004). Sherris Medical Microbiology (4th ed.).
      McGraw Hill. pp. 362â8. ISBN 978-0-8385-8529-0.
  10. ^ Collapse_of_Aztec_society_linked_to_catastrophic_salmonella_outbreak
  11. ^Brenner FW, Villar RG, Angulo FJ, Tauxe R, Swaminathan B (July 2000).
      "Salmonella_nomenclature". Journal of Clinical Microbiology. 38 (7):
      2465â7. PMC 86943. PMID 10878026.
  12. ^editors, Gillespie SH, Hawkey PM (2006). Principles and practice of
      clinical bacteriology (2nd ed.). Hoboken, NJ: John Wiley & Sons.
      ISBN 9780470017968.
  13. ^Okoro CK, Kingsley RA, Connor TR, Harris SR, Parry CM, Al-Mashhadani MN,
      Kariuki S, Msefula CL, Gordon MA, de Pinna E, Wain J, Heyderman RS, Obaro
      S, Alonso PL, Mandomando I, MacLennan CA, Tapia MD, Levine MM, Tennant
      SM, Parkhill J, Dougan G (November 2012). "Intracontinental_spread_of
      human_invasive_Salmonella_Typhimurium_pathovariants_in_sub-Saharan
      Africa". Nature Genetics. 44 (11): 1215â21. doi:10.1038/ng.2423.
      PMC 3491877. PMID 23023330.
  14. ^Eberth, Prof C. J. (1880-07-01). "Die Organismen in den Organen bei
      Typhus abdominalis". Archiv fÃ¼r Pathologische Anatomie und Physiologie
      und fÃ¼r Klinische Medizin (in German). 81 (1): 58â74. doi:10.1007/
      BF01995472. ISSN 0720-8723.
  15. ^Hardy A (August 1999). "Food, hygiene, and the laboratory. A short
      history of food poisoning in Britain, circa 1850-1950". Social History of
      Medicine. 12 (2): 293â311. doi:10.1093/shm/12.2.293. PMID 11623930.
  16. ^"FDA/CFSANâFood_Safety_A_to_Z_Reference_GuideâSalmonella".
      FDAâCenter for Food Safety and Applied Nutrition. 2008-07-03. Archived
      from the_original on 2009-03-02. Retrieved 2009-02-14.
  17. ^ a bHeymann DA, Alcamo IE, Heymann DL (2006). Salmonella. Philadelphia:
      Chelsea House Publishers. ISBN 978-0-7910-8500-4. Retrieved 31 July 2015.
  18. ^"Serotypes_Profile_of_Salmonella_Isolates_from_Meat_and_Poultry
      Products,_January_1998_through_December_2014".
  19. ^"Steps_in_a_Foodborne_Outbreak_Investigation". 2018-11-09.
  20. ^Yoon KB, Song BJ, Shin MY, Lim HC, Yoon YH, Jeon DY, Ha H, Yang SI, Kim
      JB (2017). "Antibiotic_Resistance_Patterns_and_Serotypes_of_Salmonella
      spp._Isolated_at_Jeollanam-do_in_Korea". Osong Public Health Res
      Perspect. 8 (3): 211â219. doi:10.24171/j.phrp.2017.8.3.08. PMC 5525558.
      PMID 28781944.CS1 maint: Multiple names: authors list (link)
  21. ^Clark MA, Barret EL (June 1987). "The_phs_gene_and_hydrogen_sulfide
      production_by_Salmonella_typhimurium". Journal of Bacteriology. 169 (6):
      2391â2397. doi:10.1128/jb.169.6.2391-2397.1987. PMC 212072.
      PMID 3108233.
  22. ^"UK_Standards_for_Microbiology_Investigations:_Changing_the_Phase_of
      Salmonella" (PDF). UK Standards for Microbiology Investigations: 8â10.
      8 January 2015. Retrieved 2 August 2015.
  23. ^Snyder JW, Atlas RM (2006). Handbook_of_Media_for_Clinical_Microbiology.
      CRC Press. p. 374. ISBN 978-0849337956.
  24. ^Alvarez J, Sota M, Vivanco AB, Perales I, Cisterna R, Rementeria A,
      Garaizar J (April 2004). "Development_of_a_multiplex_PCR_technique_for
      detection_and_epidemiological_typing_of_salmonella_in_human_clinical
      samples". Journal of Clinical Microbiology. 42 (4): 1734â8. doi:
      10.1128/JCM.42.4.1734-1738.2004. PMC 387595. PMID 15071035.
  25. ^Hoorfar J, Ahrens P, RÃ¥dstrÃ¶m P (September 2000). "Automated_5'
      nuclease_PCR_assay_for_identification_of_Salmonella_enterica". Journal of
      Clinical Microbiology. 38 (9): 3429â35. PMC 87399. PMID 10970396.
  26. ^Dominguez, Silvia A. "Modelling_the_Growth_of_Salmonella_in_Raw_Poultry
      Stored_under_Aerobic_Conditions".
  27. ^Pin C, AvendaÃ±o-Perez G, Cosciani-Cunico E, GÃ³mez N, Gounadakic A,
      Nychas GJ, Skandamis P, Barker G (March 2011). "Modelling Salmonella
      concentration throughout the pork supply chain by considering growth and
      survival in fluctuating conditions of temperature, pH and a(w)".
      International Journal of Food Microbiology. 145 Suppl 1: S96â102. doi:
      10.1016/j.ijfoodmicro.2010.09.025. PMID 20951457.
  28. ^Pan W. "Modelling_the_Growth_of_Salmonella_in_Cut_Red_Round_Tomatoes_as
      a_Function_of_Temperature".
  29. ^Li D, Friedrich LM, Danyluk MD, Harris LJ, Schaffner DW (June 2013).
      "Development_and_validation_of_a_mathematical_model_for_growth_of
      pathogens_in_cut_melons". Journal of Food Protection. 76 (6): 953â8.
      doi:10.4315/0362-028X.JFP-12-398. PMID 23726189.
  30. ^Li D. "Development_and_validation_of_a_mathematical_model_for_growth_of
      salmonella_in_cantaloupe".
  31. ^Winfield MD, Groisman EA (July 2003). "Role_of_nonhost_environments_in
      the_lifestyles_of_Salmonella_and_Escherichia_coli". Applied and
      Environmental Microbiology. 69 (7): 3687â94. doi:10.1128/aem.69.7.3687-
      3694.2003. PMC 165204. PMID 12839733.
  32. ^Mandal RK, Kwon YM (8 September 2017). "Salmonella_entericaSerovar
      Typhimurium_Genes_for_Desiccation_Survival". Frontiers in Microbiology. 8
      (1723): 1723. doi:10.3389/fmicb.2017.01723. PMC 5596212. PMID 28943871.
  33. ^Sorrells KM, Speck ML, Warren JA (January 1970). "Pathogenicity_of
      Salmonella_gallinarum_after_metabolic_injury_by_freezing". Applied
      Microbiology. 19 (1): 39â43. PMC 376605. PMID 5461164. Mortality
      differences between wholly uninjured and predominantly injured
      populations were small and consistent (5% level) with a hypothesis of no
      difference.
  34. ^Beuchat LR, Heaton EK (June 1975). "Salmonella_survival_on_pecans_as
      influenced_by_processing_and_storage_conditions". Applied Microbiology.
      29 (6): 795â801. PMC 187082. PMID 1098573. Little decrease in viable
      population of the three species was noted on inoculated pecan halves
      stored at -18, -7, and 5Â°C for 32 weeks.
  35. ^Goodfellow SJ, Brown WL (August 1978). "Fate of Salmonella Inoculated
      into Beef for Cooking". Journal of Food Protection. 41 (8): 598â605.
      doi:10.4315/0362-028x-41.8.598. PMID 30795117.
  36. ^ Partnership_for_Food_Safety_Education (PFSE) Fight_BAC!_Basic_Brochure
      Archived 2013-08-31 at the Wayback_Machine.
  37. ^ USDA Internal_Cooking_Temperatures_Chart Archived 2012-05-03 at the
      Wayback_Machine. The USDA has other resources available at their Safe
      Food_Handling Archived 2013-06-05 at the Wayback_Machine fact-sheet page.
      See also the National_Center_for_Home_Food_Preservation.
  38. ^"Reptiles,_Amphibians,_and_Salmonella". Centers for Disease Control and
      Prevention. U.S. Department of Health & Human Services. 25 November 2013.
      Retrieved 3 August 2013.
  39. ^ a bGoldrick, Barbara (2003). "Foodborne_Diseases:_More_efforts_needed
      to_meet_the_Healthy_People_2010_objectives". The American Journal of
      Nursing. 103 (3): 105â106. doi:10.1097/00000446-200303000-00043.
      Retrieved 6 December 2014.
  40. ^ F. Kauffmann: Die Bakteriologie der Salmonella-Gruppe. Munksgaard,
      Kopenhagen, 1941
  41. ^Le ML, Popoff MY (1987). "Request for an Opinion. Designation of
      Salmonella enterica. sp. nov., nom. rev., as the type and only species of
      the genus Salmonella". Int. J. Syst. Bacteriol. 37 (4): 465â468. doi:
      10.1099/00207713-37-4-465.
  42. ^Reeves MW, Evins GM, Heiba AA, Plikaytis BD, Farmer JJ (February 1989).
      "Clonal_nature_of_Salmonella_typhi_and_its_genetic_relatedness_to_other
      salmonellae_as_shown_by_multilocus_enzyme_electrophoresis,_and_proposal
      of_Salmonella_bongori_comb._nov". Journal of Clinical Microbiology. 27
      (2): 313â20. PMC 267299. PMID 2915026.
  43. ^Judicial Commission of the International Committee on Systematics of
      Prokaryotes (January 2005). "The type species of the genus Salmonella
      Lignieres 1900 is Salmonella enterica (ex Kauffmann and Edwards 1952) Le
      Minor and Popoff 1987, with the type strain LT2T, and conservation of the
      epithet enterica in Salmonella enterica over all earlier epithets that
      may be applied to this species. Opinion 80". International Journal of
      Systematic and Evolutionary Microbiology. 55 (Pt 1): 519â20. doi:
      10.1099/ijs.0.63579-0. PMID 15653929.
  44. ^Tindall BJ, Grimont PA, Garrity GM, EuzÃ©by JP (January 2005).
      "Nomenclature and taxonomy of the genus Salmonella". International
      Journal of Systematic and Evolutionary Microbiology. 55 (Pt 1): 521â4.
      doi:10.1099/ijs.0.63580-0. PMID 15653930.
  45. ^Grimont PA, Xavier Weill F (2007). Antigenic_Formulae_of_the_Salmonella
      Serovars (PDF) (9th ed.). Institut Pasteur, Paris, France: WHO
      Collaborating Centre for Reference and Research on Salmonella. p. 7.
      Retrieved 26 August 2015.
  46. ^ Janda JM, Abbott SL (2006). "The Enterobacteria", ASM Press.
  47. ^Porwollik S, ed. (2011). Salmonella: From Genome to Function. Caister
      Academic_Press. ISBN 978-1-904455-73-8.
  48. ^Achtman_M, Wain J, Weill FX, Nair S, Zhou Z, Sangal V, Krauland MG, Hale
      JL, Harbottle H, Uesbeck A, Dougan_G, Harrison LH, Brisse S, S. Enterica
      MLST Study Group (2012). "Multilocus_sequence_typing_as_a_replacement_for
      serotyping_in_Salmonella_enterica". PLoS Pathogens. 8 (6): e1002776. doi:
      10.1371/journal.ppat.1002776. PMC 3380943. PMID 22737074.
  49.  [open_access]
  50. ^Garcia-del Portillo F, Foster JW, Finlay BB (October 1993). "Role_of
      acid_tolerance_response_genes_in_Salmonella_typhimurium_virulence".
      Infection and Immunity. 61 (10): 4489â92. PMC 281185. PMID 8406841.
  51. ^Feasey NA, Dougan G, Kingsley RA, Heyderman RS, Gordon MA (June 2012).
      "Invasive_non-typhoidal_salmonella_disease:_an_emerging_and_neglected
      tropical_disease_in_Africa". Lancet. 379 (9835): 2489â99. doi:10.1016/
      S0140-6736(11)61752-2. PMC 3402672. PMID 22587967.
  52. ^ Â§ 6 and Â§ 7 of the German law on infectious disease prevention,
      Infektionsschutzgesetz
  53. ^"Anzahl_der_jÃ¤hrlich_registrierten_Salmonellose-Erkrankungen_in
      Deutschland_bis_2016_|_Statistik".
  54. ^ Centers_for_Disease_Control_and_Prevention
  55. ^Crump JA, Luby SP, Mintz ED (May 2004). "The_global_burden_of_typhoid
      fever". Bulletin of the World Health Organization. 82 (5): 346â53.
      PMC 2622843. PMID 15298225.
  56. ^ a b c dHaraga A, Ohlson MB, Miller SI (January 2008). "Salmonellae
      interplay with host cells". Nature Reviews. Microbiology. 6 (1): 53â66.
      doi:10.1038/nrmicro1788. PMID 18026123.
  57. ^Kerr MC, Wang JT, Castro NA, Hamilton NA, Town L, Brown DL, Meunier FA,
      Brown NF, Stow JL, Teasdale RD (April 2010). "Inhibition_of_the_PtdIns(5)
      kinase_PIKfyve_disrupts_intracellular_replication_of_Salmonella". The
      EMBO Journal. 29 (8): 1331â47. doi:10.1038/emboj.2010.28. PMC 2868569.
      PMID 20300065.
  58. ^Mittal R, Peak-Chew SY, Sade RS, Vallis Y, McMahon HT (June 2010). "The
      acetyltransferase_activity_of_the_bacterial_toxin_YopJ_of_Yersinia_is
      activated_by_eukaryotic_host_cell_inositol_hexakisphosphate". The Journal
      of Biological Chemistry. 285 (26): 19927â34. doi:10.1074/
      jbc.M110.126581. PMC 2888404. PMID 20430892.
  59. ^ a bChoi YS, Cho WJ, Yun SH, Lee SY, Park SH, Park JC, Jang EH, Shin HY
      (December 2010). "A_case_of_back_pain_caused_by_Salmonella_spondylitis_-
      A_case_report-". Korean Journal of Anesthesiology. 59 Suppl: S233â7.
      doi:10.4097/kjae.2010.59.S.S233. PMC 3030045. PMID 21286449.
  60. ^"Bacterial_protein_mimics_DNA_to_sabotage_cells'_defenses:_Study_reveals
      details_of_Salmonella_infections".
  61. ^ a bBuchmeier NA, Lipps CJ, So MY, Heffron F (March 1993).
      "Recombination-deficient mutants of Salmonella typhimurium are avirulent
      and sensitive to the oxidative burst of macrophages". Molecular
      Microbiology. 7 (6): 933â6. doi:10.1111/j.1365-2958.1993.tb01184.x.
      PMID 8387147.
  62. ^Cano DA, Pucciarelli MG, GarcÃ­a-del Portillo F, CasadesÃºs J (January
      2002). "Role_of_the_RecBCD_recombination_pathway_in_Salmonella
      virulence". Journal of Bacteriology. 184 (2): 592â5. doi:10.1128/
      jb.184.2.592-595.2002. PMC 139588. PMID 11751841.
  63. ^Thomson NR, Clayton DJ, Windhorst D, Vernikos G, Davidson S, Churcher C,
      Quail MA, Stevens M, Jones MA, Watson M, Barron A, Layton A, Pickard D,
      Kingsley RA, Bignell A, Clark L, Harris B, Ormond D, Abdellah Z, Brooks
      K, Cherevach I, Chillingworth T, Woodward J, Norberczak H, Lord A,
      Arrowsmith C, Jagels K, Moule S, Mungall K, Sanders M, Whitehead S,
      Chabalgoity JA, Maskell D, Humphrey T, Roberts M, Barrow PA, Dougan G,
      Parkhill J (October 2008). "Comparative_genome_analysis_of_Salmonella
      Enteritidis_PT4_and_Salmonella_Gallinarum_287/91_provides_insights_into
      evolutionary_and_host_adaptation_pathways". Genome Research. 18 (10):
      1624â37. doi:10.1101/gr.077404.108. PMC 2556274. PMID 18583645.
  64. ^den Bakker HC, Moreno Switt AI, Govoni G, Cummings CA, Ranieri ML,
      Degoricija L, Hoelzer K, Rodriguez-Rivera LD, Brown S, Bolchacova E,
      Furtado MR, Wiedmann M (August 2011). "Genome_sequencing_reveals
      diversification_of_virulence_factor_content_and_possible_host_adaptation
      in_distinct_subpopulations_of_Salmonella_enterica". BMC Genomics. 12:
      425. doi:10.1186/1471-2164-12-425. PMC 3176500. PMID 21859443.
  65. ^Choi E, Han Y, Cho YJ, Nam D, Lee EJ (September 2017).
      "Salmonellavirulence_gene". Proceedings of the National Academy of
      Sciences of the United States of America. 114 (38): 10232â10237. doi:
      10.1073/pnas.1705437114. PMC 5617274. PMID 28874555.
  66. ^Kisiela DI, Chattopadhyay S, Libby SJ, Karlinsey JE, Fang FC,
      Tchesnokova V, Kramer JJ, Beskhlebnaya V, Samadpour M, Grzymajlo K,
      Ugorski M, Lankau EW, Mackie RI, Clegg S, Sokurenko EV (2012). "Evolution
      of_Salmonella_enterica_virulence_via_point_mutations_in_the_fimbrial
      adhesin". PLoS Pathogens. 8 (6): e1002733. doi:10.1371/
      journal.ppat.1002733. PMC 3369946. PMID 22685400.
  67. ^ a bBÃ¤umler AJ, Tsolis RM, Ficht TA, Adams LG (October 1998).
      "Evolution_of_host_adaptation_in_Salmonella_enterica". Infection and
      Immunity. 66 (10): 4579â87. PMC 108564. PMID 9746553.
  68. ^ a bde Moraes MH, Soto EB, Salas GonzÃ¡lez I, Desai P, Chu W, Porwollik
      S, McClelland M, Teplitski M (2018). "Salmonella_sv._Newport_to_a_Plant
      Colonization_Lifestyle". Frontiers in Microbiology. 9: 877. doi:10.3389/
      fmicb.2018.00877. PMC 5968271. PMID 29867794.
  69. ^Zinder ND, Lederberg J (November 1952). "Genetic_exchange_in_Salmonella"
      (PDF). Journal of Bacteriology. 64 (5): 679â99. PMC 169409.
      PMID 12999698.
  70. ^Demerec M, Adelberg EA, Clark AJ, Hartman PE (July 1966). "A_proposal
      for_a_uniform_nomenclature_in_bacterial_genetics" (PDF). Genetics. 54
      (1): 61â76. PMC 1211113. PMID 5961488.
  71. ^Ames BN, Mccann J, Yamasaki E (December 1975). "Methods for detecting
      carcinogens and mutagens with the Salmonella/mammalian-microsome
      mutagenicity test". Mutation Research. 31 (6): 347â64. doi:10.1016/
      0165-1161(75)90046-1. PMID 768755.
***** External links[edit] *****
 Wikimedia Commons has media related to Salmonella.
 Wikispecies has information related to Salmonella
    * Background_on_Salmonella from the Food_Safety_and_Inspection_Service of
      the United_States_Department_of_Agriculture
    * Salmonella genomes and related information at PATRIC, a Bioinformatics
      Resource Center funded by NIAID
    * Questions_and_Answers_about_commercial_and_institutional_sanitizing
      methods
    * Symptoms_of_Salmonella_Poisoning
    * Salmonella_as_an_emerging_pathogen from IFAS
    * Notes_on_Salmonella_nomenclature
    * Salmonella_motility video
    * Avian_Salmonella
    * Overview_of_Salmonellosis â The Merck Veterinary Manual
    * v
    * t
    * e
Prokaryotes: Bacteria classification (phyla and orders)
      Plant
      Hacrobia
      Heterokont
      Alveolata
      Rhizaria
      Excavata
      Amoebozoa
      Opisthokonta
            Animal
            Fungi)
                                                 * DeinococcusâThermus
                                                       o Deinococcales
                                                       o Thermales
                                                 * Chloroflexi
                                                       o Anaerolineales
                                                       o Caldilineales
                         Eobacteria                    o Chloroflexales
         Terra-/_                                      o Herpetosiphonales
         Glidobacteria                                 o Dehalococcoidales
         (BV1)                                         o Ktedonobacterales
                                                       o Thermogemmatisporales
                                                       o Thermomicrobiales
                                                       o Sphaerobacterales
                                                 * Thermodesulfobacteria
                                                 * thermophiles
                         other glidobacteria           o Aquificae
                                                       o Thermotogae
                                                 * Cyanobacteria
                                     * Caulobacterales
                                     * Kiloniellales
                                     * Kordiimonadales
                                     * Magnetococcales
                                     * Parvularculales
                         Alpha       * Rhizobiales
                                     * Rhodobacterales
                                     * Rhodospirillales
                                     * Rickettsiales
                                     * Sneathiellales
                                     * Sphingomonadales
                                     * Burkholderiales
                                     * Hydrogenophilales
                                     * Methylophilales
                         Beta        * Neisseriales
                                     * Nitrosomonadales
                                     * Procabacteriales
                                     * Rhodocyclales
                                     * Acidithiobacillales
                                     * Aeromonadales
                                     * Alteromonadales
                                     * Cardiobacteriales
         Proteobacteria              * Chromatiales
         (BV2)                       * Enterobacteriales
                                     * Legionellales
                         Gamma       * Methylococcales
                                     * Oceanospirillales
                                     * Orbales
                                     * Pasteurellales
                                     * Pseudomonadales
                                     * Salinisphaerales
                                     * Thiotrichales
                                     * Vibrionales
                                     * Xanthomonadales
G-/                                  * Bdellovibrionales
OM                                   * Desulfarculales
                                     * Desulfobacterales
                                     * Desulfovibrionales
                         Delta       * Desulfurellales
                                     * Desulfuromonadales
                                     * Myxococcales
                                     * Syntrophobacterales
                                     * Syntrophorhabdales
                         Epsilon     * Campylobacterales
                                     * Nautiliales
                         Zeta        * Mariprofundales
                         Spirochaetes          * Spirochaetes
                                               * Fibrobacteres
                                               * Chlorobi
                                                     o Chlorobiales
                         Sphingobacteria             o Ignavibacteriales
                         (FCB group)           * Bacteroidetes
                                                     o Bacteroidales
                                                     o Cytophagales
                                                     o Flavobacteriales
                                                     o Sphingobacteriales
                                               * Chlamydiae
                                               * Lentisphaerae
                                                     o Lentisphaerales
                                                     o Oligosphaerales
                                                     o Victivallales
                                               * Planctomycetes
                         Planctobacteria/            o Phycisphaerales
                         (PVC group)                 o Planctomycetales
                                               * Verrucomicrobia
                                                     o Puniceicoccales
         BV4                                         o Opitutales
                                                     o Chthoniobacterales
                                                     o Verrucomicrobiales
                                               * "Poribacteria"
                                               * Acidobacteria
                                                     o Acidobacteriales
                                                     o Acanthopleuribacterales
                                                     o Holophagales
                                                     o Solibacterales
                                               * Armatimonadetes
                                                     o Armatimonadales
                                                     o Chthonomonadales
                         Other GN                    o Fimbriimonadales
                                               * Caldiserica
                                               * Chrysiogenetes
                                               * Deferribacteres
                                               * Dictyoglomi
                                               * Elusimicrobia
                                               * Fusobacteria
                                               * Gemmatimonadetes
                                               * Nitrospirae
                                               * Synergistetes
                                                * Bacillales
                                                * Lactobacillales
                                                * Aphragmobacteria
                                                      o Erysipelotrichia
                                                            # Erysipelotrichiales
                        Bacilli                       o Mollicutes
                                                            # Mycoplasmatales
                                                            # Entomoplasmatales
         Firmicutes                                         # Anaeroplasmatales
         (BV3)                                              # Acholeplasmatales
                                                            # Haloplasmatales
                                                * Clostridiales
                                                * Halanaerobiales
G+/                     Clostridia              * Thermoanaerobacteriales
no OM                                           * Natranaerobiales
                                                * Negativicutes
                                                      o Selenomonadales
                        Thermolithobacteria     * Thermolithobacterales
                        Actinobacteria      * Actinomycetales
                                            * Bifidobacteriales
                        Acidimicrobiia      * Acidimicrobiales
         Actinobacteria Coriobacteriia      * Coriobacteriales
                        Nitriliruptoria     * Euzebyales
         (BV5)                              * Nitriliruptorales
                                            * Gaiellales
                        Rubrobacteria       * Rubrobacterales
                                            * Thermoleophilales
                                            * Solirubrobacterales
Incertae     * â Archaeosphaeroides
sedis        * â Eobacterium
             * â Leptotrichites
Source: Bergey's_Manual (2001â2012). Alternative views: Wikispecies.
    * v
    * t
    * e
Chicken
    * Rooster
    * Hen
    * Chick
                        * Chicken_as_food
                              o List_of_chicken_dishes
                        * Breeds
As_poultry              * Capon
                        * Poularde
                        * Poussin
                        * Pullet
                        * Broiler
                        * Battery_cage
                        * Free_range
                        * Furnished_cages
                        * Yarding
                        * Chicken_tractor
                        * Poultry_farming
Husbandry               * Broiler_industry
                        * Beak_trimming
                        * Hatchery
                        * Chick_sexing
                        * Chick_culling
                        * Candling
                        * Abnormal_behaviour_of_birds_in_captivity
                        * European_Union_Council_Directive_1999/74/EC
                        * Cockatrice
Religion, mythology     * Cockfighting
and culture             * Kapparos
                        * Sarimanok
                        * Rooster_Flag
                        * Aspergillosis
                        * Avian_influenza
                        * Avian_sarcoma_leukosis_virus
                        * Histomoniasis_(blackhead_disease)
                        * Botulism
                        * Campylobacteriosis
                        * Candidiasis
                        * Coccidiosis
                        * Colds
                        * Dermanyssus_gallinae
                        * Egg_binding
                        * Erysipelas
                        * Fatty_liver_hemorrhagic_syndrome
Diseases                * Fowlpox
                        * Gallid_herpesvirus_1
                        * Gapeworm
                        * Infectious_bursal_disease
                        * Infectious_coryza_in_chickens
                        * Marek's_disease
                        * Mycoplasmas
                        * Newcastle_disease
                        * Omphalitis
                        * Psittacosis
                        * Pullorum
                        * Scaly_leg
                        * Squamous_cell_carcinoma
                        * Tibial_dyschondroplasia
                        * Toxoplasmosis
    * v
    * t
    * e
Food_safety
                           * 3-MCPD
                           * Aldicarb
                           * Cyanide
Adulterants, food          * Formaldehyde
contaminants               * Lead_poisoning
                           * Melamine
                           * Mercury_in_fish
                           * Sudan_I
                           * Monosodium_glutamate_(MSG)
Flavorings                 * Salt
                           * Sugar
                                 o High-fructose_corn_syrup
                           * Botulism
                           * Campylobacter_jejuni
                           * Clostridium_perfringens
                           * Escherichia_coli_O104:H4
                           * Escherichia_coli_O157:H7
Microorganisms             * Hepatitis_A
                           * Hepatitis_E
                           * Listeria
                           * Norovirus
                           * Rotavirus
                           * Salmonella
                           * Amoebiasis
                           * Anisakiasis
                           * Cryptosporidiosis
                           * Cyclosporiasis
                           * Diphyllobothriasis
                           * Enterobiasis
Parasitic_infections       * Fasciolopsiasis
through_food               * Fasciolosis
                           * Giardiasis
                           * Gnathostomiasis
                           * Paragonimiasis
                           * Toxoplasmosis
                           * Trichinosis
                           * Trichuriasis
                           * Chlorpyrifos
                           * DDT
Pesticides                 * Lindane
                           * Malathion
                           * Methamidophos
                           * Benzoic_acid
Preservatives              * Ethylenediaminetetraacetic_acid_(EDTA)
                           * Sodium_benzoate
                           * Acesulfame_potassium
                           * Aspartame
Sugar_substitutes          * Saccharin
                           * Sodium_cyclamate
                           * Sorbitol
                           * Sucralose
                           * Aflatoxin
                           * Arsenic_contamination_of_groundwater
                           * Benzene_in_soft_drinks
                           * Bisphenol_A
Toxins, poisons,           * Dieldrin
environment_pollution      * Diethylstilbestrol
                           * Dioxin
                           * Mycotoxins
                           * Nonylphenol
                           * Shellfish_poisoning
                           * Devon_colic
                           * Swill_milk_scandal
                           * Esing_Bakery_incident
                           * 1858_Bradford_sweets_poisoning
                           * 1900_English_beer_poisoning
                           * Morinaga_Milk_arsenic_poisoning_incident
                           * Minamata_disease
                           * 1971_Iraq_poison_grain_disaster
                           * Toxic_oil_syndrome
                           * 1993_Jack_in_the_Box_E._coli_outbreak
                           * 1996_Odwalla_E._coli_outbreak
                           * 2006_North_American_E._coli_outbreaks
                           * ICA_meat_repackaging_controversy
                           * 2008_Canada_listeriosis_outbreak
                           * 2008_Chinese_milk_scandal
Food_contamination         * 2008_Irish_pork_crisis
incidents                  * 2008_United_States_salmonellosis_outbreak
                           * 2011_Germany_E._coli_outbreak
                           * 2011_United_States_listeriosis_outbreak
                           * 2013_Bihar_school_meal_poisoning
                           * 2013_horse_meat_scandal
                           * 2017_Brazil_weak_meat_scandal
                           * 2017â18_South_African_listeriosis_outbreak
                           * 2018_Australian_rockmelon_listeriosis_outbreak
                           * 2018_Australian_strawberry_contamination
                           * Food_safety_incidents_in_China
                           * Food_safety_incidents_in_Taiwan
                           * Food_safety_in_Australia
                           * Foodborne_illness
                                 o outbreaks
                                 o death_toll
                                 o United_States
                           * Acceptable_daily_intake
                           * E_number
Regulation, standards,     * Food_labeling_regulations
watchdogs                  * Food_libel_laws
                           * International_Food_Safety_Network
                           * ISO_22000
                           * Quality_Assurance_International
                           * Centre_for_Food_Safety
                           * European_Food_Safety_Authority
Institutions               * Institute_for_Food_Safety_and_Health
                           * International_Food_Safety_Network
                           * Ministry_of_Food_and_Drug_Safety
    * [Issoria_lathonia.jpg]Biology_portal
                      * Wikidata: Q150839
                      * Wikispecies: Salmonella
                      * EoL: 83137
                      * EPPO: 1SALLG
                      * GBIF: 3221815
                      * iNaturalist: 437986
Taxon_identifiers     * IRMNG: 1293891
                      * ISC: 48237
                      * ITIS: 302
                      * LPSN: salmonella.html
                      * NCBI: 590
                      * NZOR: 58cb9353-0d98-468c-82e3-0b68076fc0d7
                      * WoRMS: 571268

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Salmonella&oldid=908598895"
Categories:
    * Salmonella
    * Enterobacteriaceae
    * Gram-negative_bacteria
    * Pathogenic_bacteria
    * Neglected_diseases
    * Tropical_diseases
    * Zoonoses
    * Rodent-carried_diseases
    * Bacteria_genera
Hidden categories:
    * CS1:_JulianâGregorian_uncertainty
    * CS1_German-language_sources_(de)
    * CS1_maint:_Multiple_names:_authors_list
    * CS1:_long_volume_value
    * Webarchive_template_wayback_links
    * Articles_with_short_description
    * Articles_with_'species'_microformats
    * Commons_category_link_is_on_Wikidata
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
    * Wikispecies
    * Wikinews
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * CatalÃ 
    * ÄeÅ¡tina
    * Cymraeg
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FÃ¸royskt
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * Latina
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Norsk_nynorsk
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Shqip
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Suomi
    * Svenska
    * Tagalog
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 30 July 2019, at 19:00 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
