The following text has been accessed from https://en.wikipedia.org/wiki/Mutation#Classification_of_mutation_types at Fri Aug 9 01:44:26 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Mutation ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
For other uses, see Mutation_(disambiguation).
A permanent change of the nucleotide sequence of the genome of an organism
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
Part of a_series on
Genetics
[Hybridogenesis_in_water_frogs_gametes.svg]
 
Key components
    * Chromosome
    * DNA
    * RNA
    * Genome
    * Heredity
    * Mutation
    * Nucleotide
    * Variation
    * Outline
    * Index
History and topics
    * Introduction
    * History
    * Evolution (molecular)
    * Population_genetics
    * Mendelian_inheritance
    * Quantitative_genetics
    * Molecular_genetics
Research
    * DNA_sequencing
    * Genetic_engineering
    * Genomics ([Template] template)
    * Medical_genetics
    * Branches_of_genetics
Personalized medicine
Personalized_medicine
    * Biology_portal
    * Molecular_and_cellular_biology_portal
    * v
    * t
    * e
A tulip flower exhibiting a partially yellow petal because of a mutation in its
genes.
In biology, a mutation is the alteration of the nucleotide_sequence of the
genome of an organism, virus, or extrachromosomal_DNA.[1]
Mutations result from errors during DNA_replication (especially during meiosis)
or other types of damage to DNA (such as may be caused by exposure to radiation
or carcinogens), which then may undergo error-prone repair (especially
microhomology-mediated_end_joining[2]), or cause an error during other forms of
repair,[3][4] or else may cause an error during replication (translesion
synthesis). Mutations may also result from insertion or deletion of segments of
DNA due to mobile_genetic_elements.[5][6][7] Mutations may or may not produce
discernible changes in the observable characteristics (phenotype) of an
organism. Mutations play a part in both normal and abnormal biological
processes including: evolution, cancer, and the development of the immune
system, including junctional_diversity.
The genomes of RNA_viruses are based on RNA rather than DNA. The RNA viral
genome can be double-stranded (as in DNA) or single-stranded. In some of these
viruses (such as the single-stranded human_immunodeficiency_virus) replication
occurs quickly and there are no mechanisms to check the genome for accuracy.
This error-prone process often results in mutations.
Mutation can result in many different types of change in sequences. Mutations
in genes can either have no effect, alter the product_of_a_gene, or prevent the
gene from functioning properly or completely. Mutations can also occur in
nongenic_regions. One study on genetic_variations between different species of
Drosophila suggests that, if a mutation changes a protein produced by a gene,
the result is likely to be harmful, with an estimated 70 percent of amino_acid
polymorphisms that have damaging effects, and the remainder being either
neutral or marginally beneficial.[8] Due to the damaging effects that mutations
can have on genes, organisms have mechanisms such as DNA_repair to prevent or
correct mutations by reverting the mutated sequence back to its original state.
[5]
⁰
***** Contents *****
    * 1_Description
    * 2_History
    * 3_Causes
          o 3.1_Spontaneous_mutation
          o 3.2_Error-prone_replication_bypass
          o 3.3_Errors_introduced_during_DNA_repair
          o 3.4_Induced_mutation
    * 4_Classification_of_types
          o 4.1_By_effect_on_structure
                # 4.1.1_Small-scale_mutations
                # 4.1.2_Large-scale_mutations
          o 4.2_By_effect_on_function
          o 4.3_By_effect_on_fitness
                # 4.3.1_Distribution_of_fitness_effects
          o 4.4_By_impact_on_protein_sequence
          o 4.5_By_inheritance
          o 4.6_Special_classes
          o 4.7_Nomenclature
    * 5_Mutation_rates
    * 6_Harmful_mutations
    * 7_Beneficial_mutations
    * 8_Prion_mutations
    * 9_Somatic_mutation
    * 10_Amorphic_mutations
    * 11_Hypomorphic_and_hypermorphic_mutations
    * 12_See_also
    * 13_References
    * 14_External_links
***** Description[edit] *****
Mutations can involve the duplication of large sections of DNA, usually through
genetic_recombination.[9] These duplications are a major source of raw material
for evolving new genes, with tens to hundreds of genes duplicated in animal
genomes every million years.[10] Most genes belong to larger gene_families of
shared ancestry, detectable by their sequence_homology.[11] Novel genes are
produced by several methods, commonly through the duplication and mutation of
an ancestral gene, or by recombining parts of different genes to form new
combinations with new functions.[12][13]
Here, protein_domains act as modules, each with a particular and independent
function, that can be mixed together to produce genes encoding new proteins
with novel properties.[14] For example, the human eye uses four genes to make
structures that sense light: three for cone_cell or color_vision and one for
rod_cell or night vision; all four arose from a single ancestral gene.[15]
Another advantage of duplicating a gene (or even an entire genome) is that this
increases engineering_redundancy; this allows one gene in the pair to acquire a
new function while the other copy performs the original function.[16][17] Other
types of mutation occasionally create new genes from previously noncoding_DNA.
[18][19]
Changes in chromosome number may involve even larger mutations, where segments
of the DNA within chromosomes break and then rearrange. For example, in the
Homininae, two chromosomes fused to produce human chromosome_2; this fusion did
not occur in the lineage of the other apes, and they retain these separate
chromosomes.[20] In evolution, the most important role of such chromosomal
rearrangements may be to accelerate the divergence of a population into new
species by making populations less likely to interbreed, thereby preserving
genetic differences between these populations.[21]
Sequences of DNA that can move about the genome, such as transposons, make up a
major fraction of the genetic material of plants and animals, and may have been
important in the evolution of genomes.[22] For example, more than a million
copies of the Alu_sequence are present in the human_genome, and these sequences
have now been recruited to perform functions such as regulating gene
expression.[23] Another effect of these mobile DNA sequences is that when they
move within a genome, they can mutate or delete existing genes and thereby
produce genetic diversity.[6]
Nonlethal mutations accumulate within the gene_pool and increase the amount of
genetic variation.[24] The abundance of some genetic changes within the gene
pool can be reduced by natural_selection, while other "more favorable"
mutations may accumulate and result in adaptive changes.
Prodryas_persephone, a Late Eocene butterfly
For example, a butterfly may produce offspring with new mutations. The majority
of these mutations will have no effect; but one might change the color of one
of the butterfly's offspring, making it harder (or easier) for predators to
see. If this color change is advantageous, the chances of this butterfly's
surviving and producing its own offspring are a little better, and over time
the number of butterflies with this mutation may form a larger percentage of
the population.
Neutral_mutations are defined as mutations whose effects do not influence the
fitness of an individual. These can increase in frequency over time due to
genetic_drift. It is believed that the overwhelming majority of mutations have
no significant effect on an organism's fitness.[25][26][better source needed]
Also, DNA repair mechanisms are able to mend most changes before they become
permanent mutations, and many organisms have mechanisms for eliminating
otherwise-permanently mutated somatic_cells.
Beneficial mutations can improve reproductive success.[27][28]
***** History[edit] *****
Main article: Mutationism
Hugo_de_Vries, making a painting of an evening_primrose, the plant which had
apparently produced_new_forms_by_large_mutations in his experiments, by
ThÃ©rÃ¨se_Schwartze, 1918
Mutationism is one of several alternatives_to_evolution_by_natural_selection
that have existed both before and after the publication of Charles_Darwin's
1859 book, On_the_Origin_of_Species. In the theory, mutation was the source of
novelty, creating new forms and new_species, potentially instantaneously,[29]
in a sudden jump.[30] This was envisaged as driving evolution, which was
limited by the supply of mutations.
Before Darwin, biologists commonly believed in saltationism, the possibility of
large evolutionary jumps, including immediate speciation. For example, in 1822
Ãtienne_Geoffroy_Saint-Hilaire argued that species could be formed by sudden
transformations, or what would later be called macromutation.[31] Darwin
opposed saltation, insisting on gradualism in evolution as in_geology. In 1864,
Albert_von_KÃ¶lliker revived Geoffroy's theory.[32] In 1901 the geneticist Hugo
de_Vries gave the name "mutation" to seemingly new forms that suddenly arose in
his experiments on the evening primrose Oenothera_lamarckiana, and in the first
decade of the 20th century, mutationism, or as de Vries named it
mutationstheorie,[33][29] became a rival to Darwinism supported for a while by
geneticists including William_Bateson,[34] Thomas_Hunt_Morgan, and Reginald
Punnett.[35][29]
Understanding of mutationism is clouded by the mid-20th century portrayal of
the early mutationists by supporters of the modern_synthesis as opponents of
Darwinian evolution and rivals of the biometrics school who argued that
selection operated on continuous variation. In this portrayal, mutationism was
defeated by a synthesis of genetics and natural selection that supposedly
started later, around 1918, with work by the mathematician Ronald_Fisher.[36]
[37][38][39] However, the alignment of Mendelian genetics and natural selection
began as early as 1902 with a paper by Udny_Yule,[40] and built up with
theoretical and experimental work in Europe and America. Despite the
controversy, the early mutationists had by 1918 already accepted natural
selection and explained continuous variation as the result of multiple genes
acting on the same characteristic, such as height.[37][38]
Mutationism, along with other alternatives to Darwinism like Lamarckism and
orthogenesis, was discarded by most biologists as they came to see that
Mendelian genetics and natural selection could readily work together; mutation
took its place as a source of the genetic variation essential for natural
selection to work on. However, mutationism did not entirely vanish. In 1940,
Richard_Goldschmidt again argued for single-step speciation by macromutation,
describing the organisms thus produced as "hopeful monsters", earning
widespread ridicule.[41][42] In 1987, Masatoshi_Nei argued controversially that
evolution was often mutation-limited.[43] Modern biologists such as Douglas_J.
Futuyma conclude that essentially all claims of evolution driven by large
mutations can be explained by Darwinian evolution.[44]
***** Causes[edit] *****
Main article: Mutagenesis
Four classes of mutations are (1) spontaneous mutations (molecular decay), (2)
mutations due to error-prone replication bypass of naturally_occurring_DNA
damage (also called error-prone translesion synthesis), (3) errors introduced
during DNA repair, and (4) induced mutations caused by mutagens. Scientists may
also deliberately introduce mutant sequences through DNA manipulation for the
sake of scientific experimentation.
One 2017 study claimed that 66% of cancer-causing mutations are random, 29% are
due to the environment (the studied population spanned 69 countries), and 5%
are inherited.[45]
Humans on average pass 60 new mutations to their children but fathers pass more
mutations depending on their age with every year adding two new mutations to a
child.[46]
**** Spontaneous mutation[edit] ****
Spontaneous mutations occur with non-zero probability even given a healthy,
uncontaminated cell. They can be characterized by the specific change:[47]
    * Tautomerism â A base is changed by the repositioning of a hydrogen
      atom, altering the hydrogen bonding pattern of that base, resulting in
      incorrect base_pairing during replication.
    * Depurination â Loss of a purine base (A or G) to form an apurinic site
      (AP_site).
    * Deamination â Hydrolysis changes a normal base to an atypical base
      containing a keto group in place of the original amine group. Examples
      include C â U and A â HX (hypoxanthine), which can be corrected by
      DNA repair mechanisms; and 5MeC (5-methylcytosine) â T, which is less
      likely to be detected as a mutation because thymine is a normal DNA base.
    * Slipped_strand_mispairing â Denaturation of the new strand from the
      template during replication, followed by renaturation in a different spot
      ("slipping"). This can lead to insertions or deletions.
    * Replication_slippage
**** Error-prone replication bypass[edit] ****
There is increasing evidence that the majority of spontaneously arising
mutations are due to error-prone replication (translesion synthesis) past DNA
damage in the template strand. Naturally occurring oxidative DNA damages arise
at least 10,000 times per cell per day in humans and 50,000 times or more per
cell per day in rats.[48] In mice, the majority of mutations are caused by
translesion synthesis.[49] Likewise, in yeast, Kunz et al.[50] found that more
than 60% of the spontaneous single base pair substitutions and deletions were
caused by translation synthesis.
**** Errors introduced during DNA repair[edit] ****
See also: DNA_damage_(naturally_occurring) and DNA_repair
Although naturally occurring double-strand breaks occur at a relatively low
frequency in DNA, their repair often causes mutation. Non-homologous_end
joining (NHEJ) is a major pathway for repairing double-strand breaks. NHEJ
involves removal of a few nucleotides to allow somewhat inaccurate alignment of
the two ends for rejoining followed by addition of nucleotides to fill in gaps.
As a consequence, NHEJ often introduces mutations.[51]
A covalent adduct between the metabolite of benzo[a]pyrene, the major mutagen
in tobacco_smoke, and DNA[52]
**** Induced mutation[edit] ****
Induced mutations are alterations in the gene after it has come in contact with
mutagens and environmental causes.
Induced mutations on the molecular level can be caused by:
    * Chemicals
          o Hydroxylamine
          o Base_analogs (e.g., Bromodeoxyuridine (BrdU))
          o Alkylating_agents (e.g., N-ethyl-N-nitrosourea (ENU)). These agents
            can mutate both replicating and non-replicating DNA. In contrast, a
            base analog can mutate the DNA only when the analog is incorporated
            in replicating the DNA. Each of these classes of chemical mutagens
            has certain effects that then lead to transitions, transversions,
            or deletions.
          o Agents that form DNA_adducts (e.g., ochratoxin_A)[53]
          o DNA intercalating agents (e.g., ethidium_bromide)
          o DNA_crosslinkers
          o This figure depicts the following processes of transcription,
            splicing, and translation of a eukaryotic gene.
            Oxidative_damage
          o Nitrous_acid converts amine groups on A and C to diazo groups,
            altering their hydrogen bonding patterns, which leads to incorrect
            base pairing during replication.
    * Radiation
          o Ultraviolet light (UV) (non-ionizing_radiation). Two nucleotide
            bases in DNAâcytosine and thymineâare most vulnerable to
            radiation that can change their properties. UV light can induce
            adjacent pyrimidine bases in a DNA strand to become covalently
            joined as a pyrimidine_dimer. UV radiation, in particular longer-
            wave UVA, can also cause oxidative_damage_to_DNA.[54]
          o Ionizing_radiation. Exposure to ionizing radiation, such as gamma
            radiation, can result in mutation, possibly resulting in cancer or
            death.
Whereas in former times mutations were assumed to occur by chance, or induced
by mutagens, molecular mechanisms of mutation have been discovered in bacteria
and across the tree of life. As S. Rosenberg states, "These mechanisms reveal a
picture of highly regulated mutagenesis, up-regulated temporally by stress
responses and activated when cells/organisms are maladapted to their
environmentsâwhen stressedâpotentially accelerating adaptation." [55] Since
they are self-induced mutagenic mechanisms that increase the adaptation rate of
organisms, they have some times been named as adaptive mutagenesis mechanisms,
and include the SOS response in bacteria,[56] ectopic intrachromosomal
recombination [57] and other chromosomal events such as duplications.[55]
***** Classification of types[edit] *****
See also: Chromosome_abnormality
**** By effect on structure[edit] ****
Five types of chromosomal mutations.
Selection of disease-causing mutations, in a standard table of the genetic_code
of amino_acids.[58]
The sequence of a gene can be altered in a number of ways.[59] Gene mutations
have varying effects on health depending on where they occur and whether they
alter the function of essential proteins. Mutations in the structure of genes
can be classified into several types.
*** Small-scale mutations[edit] ***
Small-scale mutations affect a gene in one or a few nucleotides. (If only a
single nucleotide is affected, they are called point_mutations.) Small-scale
mutations include:
    * Insertions add one or more extra nucleotides into the DNA. They are
      usually caused by transposable_elements, or errors during replication of
      repeating elements. Insertions in the coding region of a gene may alter
      splicing of the mRNA (splice_site_mutation), or cause a shift in the
      reading_frame (frameshift), both of which can significantly alter the
      gene_product. Insertions can be reversed by excision of the transposable
      element.
    * Deletions remove one or more nucleotides from the DNA. Like insertions,
      these mutations can alter the reading frame of the gene. In general, they
      are irreversible: Though exactly the same sequence might, in theory, be
      restored by an insertion, transposable elements able to revert a very
      short deletion (say 1â2 bases) in any location either are highly
      unlikely to exist or do not exist at all.
    * Substitution_mutations, often caused by chemicals or malfunction of DNA
      replication, exchange a single nucleotide for another.[60] These changes
      are classified as transitions or transversions.[61] Most common is the
      transition that exchanges a purine for a purine (A â G) or a pyrimidine
      for a pyrimidine, (C â T). A transition can be caused by nitrous acid,
      base mispairing, or mutagenic base analogs such as BrdU. Less common is a
      transversion, which exchanges a purine for a pyrimidine or a pyrimidine
      for a purine (C/T â A/G). An example of a transversion is the
      conversion of adenine (A) into a cytosine (C). Point mutations are
      modifications of single base pairs of DNA or other small base pairs
      within a gene. A point mutation can be reversed by another point
      mutation, in which the nucleotide is changed back to its original state
      (true reversion) or by second-site reversion (a complementary mutation
      elsewhere that results in regained gene functionality). As discussed
      below, point mutations that occur within the protein coding_region of a
      gene may be classified as synonymous or nonsynonymous_substitutions, the
      latter of which in turn can be divided into missense or nonsense
      mutations.
*** Large-scale mutations[edit] ***
Large-scale mutations in chromosomal structure include:
    * Amplifications (or gene_duplications) leading to multiple copies of all
      chromosomal regions, increasing the dosage of the genes located within
      them.
    * Deletions of large chromosomal regions, leading to loss of the genes
      within those regions.
    * Mutations whose effect is to juxtapose previously separate pieces of DNA,
      potentially bringing together separate genes to form functionally
      distinct fusion_genes (e.g., bcr-abl).
    * Large scale changes to the structure of chromosomes called chromosomal
      rearrangement that can lead to a decrease of fitness but also to
      speciation in isolated, inbred populations. These include:
          o Chromosomal_translocations: interchange of genetic parts from
            nonhomologous chromosomes.
          o Chromosomal_inversions: reversing the orientation of a chromosomal
            segment.
          o Non-homologous chromosomal_crossover.
          o Interstitial deletions: an intra-chromosomal deletion that removes
            a segment of DNA from a single chromosome, thereby apposing
            previously distant genes. For example, cells isolated from a human
            astrocytoma, a type of brain tumor, were found to have a
            chromosomal deletion removing sequences between the Fused in
            Glioblastoma (FIG) gene and the receptor tyrosine kinase (ROS),
            producing a fusion protein (FIG-ROS). The abnormal FIG-ROS fusion
            protein has constitutively active kinase activity that causes
            oncogenic transformation (a transformation from normal cells to
            cancer cells).
    * Loss_of_heterozygosity: loss of one allele, either by a deletion or a
      genetic recombination event, in an organism that previously had two
      different alleles.
**** By effect on function[edit] ****
    * Loss-of-function mutations, also called inactivating mutations, result in
      the gene product having less or no function (being partially or wholly
      inactivated). When the allele has a complete loss of function (null
      allele), it is often called an amorph or amorphic mutation in the
      Muller's_morphs schema. Phenotypes associated with such mutations are
      most often recessive. Exceptions are when the organism is haploid, or
      when the reduced dosage of a normal gene product is not enough for a
      normal phenotype (this is called haploinsufficiency).
    * Gain-of-function mutations, also called activating mutations, change the
      gene product such that its effect gets stronger (enhanced activation) or
      even is superseded by a different and abnormal function. When the new
      allele is created, a heterozygote containing the newly created allele as
      well as the original will express the new allele; genetically this
      defines the mutations as dominant phenotypes. Several of Muller's morphs
      correspond to gain of function, including hypermorph and neomorph. In
      December 2017, the U.S. government lifted a temporary ban implemented in
      2014 that banned federal funding for any new "gain-of-function"
      experiments that enhance pathogens "such as Avian influenza, SARS and the
      Middle East Respiratory Syndrome or MERS viruses."[62]
    * Dominant negative mutations (also called antimorphic mutations) have an
      altered gene product that acts antagonistically to the wild-type allele.
      These mutations usually result in an altered molecular function (often
      inactive) and are characterized by a dominant or semi-dominant phenotype.
      In humans, dominant negative mutations have been implicated in cancer
      (e.g., mutations in genes p53,[63] ATM,[64] CEBPA[65] and PPARgamma[66]).
      Marfan_syndrome is caused by mutations in the FBN1 gene, located on
      chromosome_15, which encodes fibrillin-1, a glycoprotein component of the
      extracellular_matrix.[67] Marfan syndrome is also an example of dominant
      negative mutation and haploinsufficiency.[68][69]
    * Hypomorphs, after Mullerian classification, are characterized by altered
      gene products that acts with decreased gene_expression compared to the
      wild_type allele.
    * Neomorphs are characterized by the control of new protein product
      synthesis.
    * Lethal_mutations are mutations that lead to the death of the organisms
      that carry the mutations.
    * A back mutation or reversion is a point mutation that restores the
      original sequence and hence the original phenotype.[70]
**** By effect on fitness[edit] ****
See also: Fitness_(biology)
In applied genetics, it is usual to speak of mutations as either harmful or
beneficial.
    * A harmful, or deleterious, mutation decreases the fitness of the
      organism.
    * A beneficial, or advantageous mutation increases the fitness of the
      organism.
    * A neutral mutation has no harmful or beneficial effect on the organism.
      Such mutations occur at a steady rate, forming the basis for the
      molecular_clock. In the neutral_theory_of_molecular_evolution, neutral
      mutations provide genetic drift as the basis for most variation at the
      molecular level.
    * A nearly neutral mutation is a mutation that may be slightly deleterious
      or advantageous, although most nearly neutral mutations are slightly
      deleterious.
*** Distribution of fitness effects[edit] ***
Attempts have been made to infer the distribution of fitness effects (DFE)
using mutagenesis experiments and theoretical models applied to molecular
sequence data. DFE, as used to determine the relative abundance of different
types of mutations (i.e., strongly deleterious, nearly neutral or
advantageous), is relevant to many evolutionary questions, such as the
maintenance of genetic_variation,[71] the rate of genomic_decay,[72] the
maintenance of outcrossing sexual_reproduction as opposed to inbreeding[73] and
the evolution of sex and genetic_recombination.[74] In summary, the DFE plays
an important role in predicting evolutionary_dynamics.[75][76] A variety of
approaches have been used to study the DFE, including theoretical, experimental
and analytical methods.
    * Mutagenesis experiment: The direct method to investigate the DFE is to
      induce mutations and then measure the mutational fitness effects, which
      has already been done in viruses, bacteria, yeast, and Drosophila. For
      example, most studies of the DFE in viruses used site-directed
      mutagenesis to create point mutations and measure relative fitness of
      each mutant.[77][78][79][80] In Escherichia_coli, one study used
      transposon_mutagenesis to directly measure the fitness of a random
      insertion of a derivative of Tn10.[81] In yeast, a combined mutagenesis
      and deep_sequencing approach has been developed to generate high-quality
      systematic mutant libraries and measure fitness in high throughput.[82]
      However, given that many mutations have effects too small to be detected
      [83] and that mutagenesis experiments can detect only mutations of
      moderately large effect; DNA sequence_data_analysis can provide valuable
      information about these mutations.
The distribution of fitness effects (DFE) of mutations in vesicular_stomatitis
virus. In this experiment, random mutations were introduced into the virus by
site-directed mutagenesis, and the fitness of each mutant was compared with the
ancestral type. A fitness of zero, less than one, one, more than one,
respectively, indicates that mutations are lethal, deleterious, neutral, and
advantageous.[77]
    * Molecular sequence analysis: With rapid development of DNA_sequencing
      technology, an enormous amount of DNA sequence data is available and even
      more is forthcoming in the future. Various methods have been developed to
      infer the DFE from DNA sequence data.[84][85][86][87] By examining DNA
      sequence differences within and between species, we are able to infer
      various characteristics of the DFE for neutral, deleterious and
      advantageous mutations.[24] To be specific, the DNA sequence analysis
      approach allows us to estimate the effects of mutations with very small
      effects, which are hardly detectable through mutagenesis experiments.
One of the earliest theoretical studies of the distribution of fitness effects
was done by Motoo_Kimura, an influential theoretical population geneticist. His
neutral theory of molecular evolution proposes that most novel mutations will
be highly deleterious, with a small fraction being neutral.[88][25] Hiroshi
Akashi more recently proposed a bimodal model for the DFE, with modes centered
around highly deleterious and neutral mutations.[89] Both theories agree that
the vast majority of novel mutations are neutral or deleterious and that
advantageous mutations are rare, which has been supported by experimental
results. One example is a study done on the DFE of random mutations in
vesicular_stomatitis_virus.[77] Out of all mutations, 39.6% were lethal, 31.2%
were non-lethal deleterious, and 27.1% were neutral. Another example comes from
a high throughput mutagenesis experiment with yeast.[82] In this experiment it
was shown that the overall DFE is bimodal, with a cluster of neutral mutations,
and a broad distribution of deleterious mutations.
Though relatively few mutations are advantageous, those that are play an
important role in evolutionary changes.[90] Like neutral mutations, weakly
selected advantageous mutations can be lost due to random genetic drift, but
strongly selected advantageous mutations are more likely to be fixed. Knowing
the DFE of advantageous mutations may lead to increased ability to predict the
evolutionary dynamics. Theoretical work on the DFE for advantageous mutations
has been done by John_H._Gillespie[91] and H._Allen_Orr.[92] They proposed that
the distribution for advantageous mutations should be exponential under a wide
range of conditions, which, in general, has been supported by experimental
studies, at least for strongly selected advantageous mutations.[93][94][95]
In general, it is accepted that the majority of mutations are neutral or
deleterious, with advantageous mutations being rare; however, the proportion of
types of mutations varies between species. This indicates two important points:
first, the proportion of effectively neutral mutations is likely to vary
between species, resulting from dependence on effective_population_size;
second, the average effect of deleterious mutations varies dramatically between
species.[24] In addition, the DFE also differs between coding regions and
noncoding_regions, with the DFE of noncoding DNA containing more weakly
selected mutations.[24]
**** By impact on protein sequence[edit] ****
    * A frameshift_mutation is a mutation caused by insertion or deletion of a
      number of nucleotides that is not evenly divisible by three from a DNA
      sequence. Due to the triplet nature of gene expression by codons, the
      insertion or deletion can disrupt the reading frame, or the grouping of
      the codons, resulting in a completely different translation from the
      original.[96] The earlier in the sequence the deletion or insertion
      occurs, the more altered the protein produced is. (For example, the code
      CCU GAC UAC CUA codes for the amino acids proline, aspartic acid,
      tyrosine, and leucine. If the U in CCU was deleted, the resulting
      sequence would be CCG ACU ACC UAx, which would instead code for proline,
      threonine, threonine, and part of another amino acid or perhaps a stop
      codon (where the x stands for the following nucleotide).) By contrast,
      any insertion or deletion that is evenly divisible by three is termed an
      in-frame mutation.
    * A point substitution mutation results in a change in a single nucleotide
      and can be either synonymous or nonsynonymous.
          o A synonymous_substitution replaces a codon with another codon that
            codes for the same amino acid, so that the produced amino acid
            sequence is not modified. Synonymous mutations occur due to the
            degenerate nature of the genetic_code. If this mutation does not
            result in any phenotypic effects, then it is called silent, but not
            all synonymous substitutions are silent. (There can also be silent
            mutations in nucleotides outside of the coding regions, such as the
            introns, because the exact nucleotide sequence is not as crucial as
            it is in the coding regions, but these are not considered
            synonymous substitutions.)
          o A nonsynonymous_substitution replaces a codon with another codon
            that codes for a different amino acid, so that the produced amino
            acid sequence is modified. Nonsynonymous substitutions can be
            classified as nonsense or missense mutations:
                # A missense_mutation changes a nucleotide to cause
                  substitution of a different amino acid. This in turn can
                  render the resulting protein nonfunctional. Such mutations
                  are responsible for diseases such as Epidermolysis_bullosa,
                  sickle-cell_disease, and SOD1-mediated ALS.[97] On the other
                  hand, if a missense mutation occurs in an amino acid codon
                  that results in the use of a different, but chemically
                  similar, amino acid, then sometimes little or no change is
                  rendered in the protein. For example, a change from AAA to
                  AGA will encode arginine, a chemically similar molecule to
                  the intended lysine. In this latter case the mutation will
                  have little or no effect on phenotype and therefore be
                  neutral.
                # A nonsense_mutation is a point mutation in a sequence of DNA
                  that results in a premature stop codon, or a nonsense codon
                  in the transcribed mRNA, and possibly a truncated, and often
                  nonfunctional protein product. This sort of mutation has been
                  linked to different mutations, such as congenital_adrenal
                  hyperplasia. (See Stop_codon.)
**** By inheritance[edit] ****
A mutation has caused this moss_rose plant to produce flowers of different
colors. This is a somatic mutation that may also be passed on in the germline.
In multicellular_organisms with dedicated reproductive_cells, mutations can be
subdivided into germline_mutations, which can be passed on to descendants
through their reproductive cells, and somatic mutations (also called acquired
mutations),[98] which involve cells outside the dedicated reproductive group
and which are not usually transmitted to descendants.
A germline mutation gives rise to a constitutional mutation in the offspring,
that is, a mutation that is present in every cell. A constitutional mutation
can also occur very soon after fertilisation, or continue from a previous
constitutional mutation in a parent.[99]
The distinction between germline and somatic mutations is important in animals
that have a dedicated germline to produce reproductive cells. However, it is of
little value in understanding the effects of mutations in plants, which lack a
dedicated germline. The distinction is also blurred in those animals that
reproduce_asexually through mechanisms such as budding, because the cells that
give rise to the daughter organisms also give rise to that organism's germline.
A new germline mutation not inherited from either parent is called a de_novo
mutation.
Diploid organisms (e.g., humans) contain two copies of each geneâa paternal
and a maternal allele. Based on the occurrence of mutation on each chromosome,
we may classify mutations into three types.
    * A heterozygous mutation is a mutation of only one allele.
    * A homozygous mutation is an identical mutation of both the paternal and
      maternal alleles.
    * Compound_heterozygous mutations or a genetic compound consists of two
      different mutations in the paternal and maternal alleles.[100]
A wild_type or homozygous non-mutated organism is one in which neither allele
is mutated.
**** Special classes[edit] ****
    * Conditional mutation is a mutation that has wild-type (or less severe)
      phenotype under certain "permissive" environmental conditions and a
      mutant phenotype under certain "restrictive" conditions. For example, a
      temperature-sensitive mutation can cause cell death at high temperature
      (restrictive condition), but might have no deleterious consequences at a
      lower temperature (permissive condition).[101] These mutations are non-
      autonomous, as their manifestation depends upon presence of certain
      conditions, as opposed to other mutations which appear autonomously.[102]
      The permissive conditions may be temperature,[103] certain chemicals,
      [104] light[104] or mutations in other parts of the genome.[102] In_vivo
      mechanisms like transcriptional switches can create conditional
      mutations. For instance, association of Steroid Binding Domain can create
      a transcriptional switch that can change the expression of a gene based
      on the presence of a steroid ligand.[105] Conditional mutations have
      applications in research as they allow control over gene expression. This
      is especially useful studying diseases in adults by allowing expression
      after a certain period of growth, thus eliminating the deleterious effect
      of gene expression seen during stages of development in model organisms.
      [104] DNA Recombinase systems like Cre-Lox_Recombination used in
      association with promoters that are activated under certain conditions
      can generate conditional mutations. Dual Recombinase technology can be
      used to induce multiple conditional mutations to study the diseases which
      manifest as a result of simultaneous mutations in multiple genes.[104]
      Certain inteins have been identified which splice only at certain
      permissive temperatures, leading to improper protein synthesis and thus,
      loss of function mutations at other temperatures.[106] Conditional
      mutations may also be used in genetic studies associated with ageing, as
      the expression can be changed after a certain time period in the
      organism's lifespan.[103]
    * Replication_timing_quantitative_trait_loci affects DNA replication.
**** Nomenclature[edit] ****
In order to categorize a mutation as such, the "normal" sequence must be
obtained from the DNA of a "normal" or "healthy" organism (as opposed to a
"mutant" or "sick" one), it should be identified and reported; ideally, it
should be made publicly available for a straightforward nucleotide-by-
nucleotide comparison, and agreed upon by the scientific community or by a
group of expert geneticists and biologists, who have the responsibility of
establishing the standard or so-called "consensus" sequence. This step requires
a tremendous scientific effort. Once the consensus sequence is known, the
mutations in a genome can be pinpointed, described, and classified. The
committee of the Human Genome Variation Society (HGVS) has developed the
standard human sequence variant nomenclature,[107] which should be used by
researchers and DNA_diagnostic centers to generate unambiguous mutation
descriptions. In principle, this nomenclature can also be used to describe
mutations in other organisms. The nomenclature specifies the type of mutation
and base or amino acid changes.
    * Nucleotide substitution (e.g., 76A>T) â The number is the position of
      the nucleotide from the 5' end; the first letter represents the wild-type
      nucleotide, and the second letter represents the nucleotide that replaced
      the wild type. In the given example, the adenine at the 76th position was
      replaced by a thymine.
          o If it becomes necessary to differentiate between mutations in
            genomic_DNA, mitochondrial_DNA, and RNA, a simple convention is
            used. For example, if the 100th base of a nucleotide sequence
            mutated from G to C, then it would be written as g.100G>C if the
            mutation occurred in genomic DNA, m.100G>C if the mutation occurred
            in mitochondrial DNA, or r.100g>c if the mutation occurred in RNA.
            Note that, for mutations in RNA, the nucleotide code is written in
            lower case.
    * Amino acid substitution (e.g., D111E) â The first letter is the one
      letter code of the wild-type amino acid, the number is the position of
      the amino acid from the N-terminus, and the second letter is the one
      letter code of the amino acid present in the mutation. Nonsense mutations
      are represented with an X for the second amino acid (e.g. D111X).
    * Amino acid deletion (e.g., ÎF508) â The Greek letter Î (delta)
      indicates a deletion. The letter refers to the amino acid present in the
      wild type and the number is the position from the N terminus of the amino
      acid were it to be present as in the wild type.
***** Mutation rates[edit] *****
Further information: Mutation_rate
Mutation_rates vary substantially across species, and the evolutionary forces
that generally determine mutation are the subject of ongoing investigation.
***** Harmful mutations[edit] *****
Changes in DNA caused by mutation can cause errors in protein sequence,
creating partially or completely non-functional proteins. Each cell, in order
to function correctly, depends on thousands of proteins to function in the
right places at the right times. When a mutation alters a protein that plays a
critical role in the body, a medical condition can result. Some mutations alter
a gene's DNA base sequence but do not change the function of the protein made
by the gene. One study on the comparison of genes between different species of
Drosophila suggests that if a mutation does change a protein, this will
probably be harmful, with an estimated 70 percent of amino acid polymorphisms
having damaging effects, and the remainder being either neutral or weakly
beneficial.[8] Studies have shown that only 7% of point mutations in noncoding
DNA of yeast are deleterious and 12% in coding DNA are deleterious. The rest of
the mutations are either neutral or slightly beneficial.[108]
If a mutation is present in a germ_cell, it can give rise to offspring that
carries the mutation in all of its cells. This is the case in hereditary
diseases. In particular, if there is a mutation in a DNA repair gene within a
germ cell, humans carrying such germline mutations may have an increased risk
of cancer. A list of 34 such germline mutations is given in the article DNA
repair-deficiency_disorder. An example of one is albinism, a mutation that
occurs in the OCA1 or OCA2 gene. Individuals with this disorder are more prone
to many types of cancers, other disorders and have impaired vision. On the
other hand, a mutation may occur in a somatic cell of an organism. Such
mutations will be present in all descendants of this cell within the same
organism, and certain mutations can cause the cell to become malignant, and,
thus, cause cancer.[109]
A DNA damage can cause an error when the DNA is replicated, and this error of
replication can cause a gene mutation that, in turn, could cause a genetic
disorder. DNA damages are repaired by the DNA repair system of the cell. Each
cell has a number of pathways through which enzymes recognize and repair
damages in DNA. Because DNA can be damaged in many ways, the process of DNA
repair is an important way in which the body protects itself from disease. Once
DNA damage has given rise to a mutation, the mutation cannot be repaired.
***** Beneficial mutations[edit] *****
Although mutations that cause changes in protein sequences can be harmful to an
organism, on occasions the effect may be positive in a given environment. In
this case, the mutation may enable the mutant organism to withstand particular
environmental stresses better than wild-type organisms, or reproduce more
quickly. In these cases a mutation will tend to become more common in a
population through natural selection. Examples include the following:
HIV resistance: a specific 32 base pair deletion in human CCR5 (CCR5-Î32)
confers HIV resistance to homozygotes and delays AIDS onset in heterozygotes.
[110] One possible explanation of the etiology of the relatively high frequency
of CCR5-Î32 in the European population is that it conferred resistance to the
bubonic_plague in mid-14th century Europe. People with this mutation were more
likely to survive infection; thus its frequency in the population increased.
[111] This theory could explain why this mutation is not found in Southern
Africa, which remained untouched by bubonic plague. A newer theory suggests
that the selective_pressure on the CCR5 Delta 32 mutation was caused by
smallpox instead of the bubonic plague.[112]
Malaria resistance: An example of a harmful mutation is sickle-cell_disease, a
blood disorder in which the body produces an abnormal type of the oxygen-
carrying substance hemoglobin in the red_blood_cells. One-third of all
indigenous inhabitants of Sub-Saharan_Africa carry the allele, because, in
areas where malaria is common, there is a survival_value in carrying only a
single sickle-cell allele (sickle_cell_trait).[113] Those with only one of the
two alleles of the sickle-cell disease are more resistant to malaria, since the
infestation of the malaria Plasmodium is halted by the sickling of the cells
that it infests.
Antibiotic resistance: Practically all bacteria develop antibiotic resistance
when exposed to antibiotics. In fact, bacterial populations already have such
mutations that get selected under antibiotic selection.[114] Obviously, such
mutations are only beneficial for the bacteria but not for those infected.
Lactase_persistence. A mutation allowed humans to express the enzyme lactase
after they are naturally weaned from breast milk, allowing adults to digest
lactose, which is probably one of the most beneficial mutations in recent human
evolution.[115]
***** Prion mutations[edit] *****
Prions are proteins and do not contain genetic material. However, prion
replication has been shown to be subject to mutation and natural selection just
like other forms of replication.[116] The human gene PRNP codes for the major
prion protein, PrP, and is subject to mutations that can give rise to disease-
causing prions.
***** Somatic mutation[edit] *****
See also: Carcinogenesis and Loss_of_heterozygosity
A change in the genetic structure that is not inherited from a parent, and also
not passed to offspring, is called a somatic mutation.[98] Somatic mutations
are not inherited because they do not affect the germline. These types of
mutations are usually prompted by environmental causes, such as ultraviolet
radiation or any exposure to certain harmful chemicals, and can cause diseases
including cancer.[117]
With plants, some somatic mutations can be propagated without the need for seed
production, for example, by grafting and stem cuttings. These type of mutation
have led to new types of fruits, such as the "Delicious" apple and the
"Washington" navel orange.[118]
Human and mouse somatic_cells have a mutation rate more than ten times higher
than the germline mutation rate for both species; mice have a higher rate of
both somatic and germline mutations per cell_division than humans. The
disparity in mutation rate between the germline and somatic tissues likely
reflects the greater importance of genome maintenance in the germline than in
the soma.[119]
***** Amorphic mutations[edit] *****
An amorph, a term utilized by Muller in 1932, is a mutated allele, which has
lost the ability of the parent (whether wild type or any other type) allele to
encode any functional protein. An amorphic mutation may be caused by the
replacement of an amino acid that deactivates an enzyme or by the deletion of
part of a gene that produces the enzyme.
Cells with heterozygous mutations (one good copy of gene and one mutated copy)
may function normally with the unmutated copy until the good copy has been
spontaneously somatically mutated. This kind of mutation happens all the time
in living organisms, but it is difficult to measure the rate. Measuring this
rate is important in predicting the rate at which people may develop cancer.
[120]
Point mutations may arise from spontaneous mutations that occur during DNA
replication. The rate of mutation may be increased by mutagens. Mutagens can be
physical, such as radiation from UV_rays, X-rays or extreme heat, or chemical
(molecules that misplace base pairs or disrupt the helical shape of DNA).
Mutagens associated with cancers are often studied to learn about cancer and
its prevention.
***** Hypomorphic and hypermorphic mutations[edit] *****
See also: Muller's_morphs
A hypomorphic mutation is a mutation which results in lowered gene expression.
Usually, hypomorphic mutations are recessive, but haploinsufficiency causes
some alleles to be dominant.
A hypermorphic mutation results in increased gene expression.
***** See also[edit] *****
    * Aneuploidy
    * Antioxidant
    * Budgerigar_colour_genetics
    * Ecogenetics
    * Embryology
    * Homeobox
    * Polyploidy
    * Robertsonian_translocation
    * Signature-tagged_mutagenesis
    * Somatic_hypermutation
    * TILLING_(molecular_biology)
    * Trinucleotide_repeat_expansion
***** References[edit] *****
   1. ^"mutation_|_Learn_Science_at_Scitable". www.nature.com. Retrieved 24
      September 2018.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Sharma S, Javadekar SM, Pandey M, Srivastava M, Kumari R, Raghavan SC
      (March 2015). "Homology_and_enzymatic_requirements_of_microhomology-
      dependent_alternative_end_joining". Cell Death & Disease. 6 (3): e1697.
      doi:10.1038/cddis.2015.58. PMC 4385936. PMID 25789972.
   4. ^Chen J, Miller BF, Furano AV (April 2014). "Repair_of_naturally
      occurring_mismatches_can_induce_mutations_in_flanking_DNA". eLife. 3:
      e02001. doi:10.7554/elife.02001. PMC 3999860. PMID 24843013.
   5. ^Rodgers K, McVey M (January 2016). "Error-Prone_Repair_of_DNA_Double-
      Strand_Breaks". Journal of Cellular Physiology. 231 (1): 15â24. doi:
      10.1002/jcp.25053. PMC 4586358. PMID 26033759.
   6. ^ a bBertram JS (December 2000). "The molecular biology of cancer".
      Molecular Aspects of Medicine. 21 (6): 167â223. doi:10.1016/S0098-2997
      (00)00007-8. PMID 11173079.
   7. ^ a bAminetzach YT, Macpherson JM, Petrov DA (July 2005). "Pesticide
      resistance via transposition-mediated adaptive gene truncation in
      Drosophila". Science. 309 (5735): 764â7. Bibcode:2005Sci...309..764A.
      doi:10.1126/science.1112699. PMID 16051794.
   8. ^Burrus V, Waldor MK (June 2004). "Shaping bacterial genomes with
      integrative and conjugative elements". Research in Microbiology. 155 (5):
      376â86. doi:10.1016/j.resmic.2004.01.012. PMID 15207870.
   9. ^ a bSawyer SA, Parsch J, Zhang Z, Hartl DL (April 2007). "Prevalence_of
      positive_selection_among_nearly_neutral_amino_acid_replacements_in
      Drosophila". Proceedings of the National Academy of Sciences of the
      United States of America. 104 (16): 6504â10. Bibcode:
      2007PNAS..104.6504S. doi:10.1073/pnas.0701572104. PMC 1871816.
      PMID 17409186.
  10. ^Hastings PJ, Lupski_JR, Rosenberg SM, Ira G (August 2009). "Mechanisms
      of_change_in_gene_copy_number". Nature Reviews. Genetics. 10 (8):
      551â64. doi:10.1038/nrg2593. PMC 2864001. PMID 19597530.
  11. ^Carroll_SB, Grenier JK, Weatherbee SD (2005). From DNA to Diversity:
      Molecular Genetics and the Evolution of Animal Design (2nd ed.). Malden,
      MA: Blackwell_Publishing. ISBN 978-1-4051-1950-4. LCCN 2003027991.
      OCLC 53972564.
  12. ^Harrison PM, Gerstein_M (May 2002). "Studying genomes through the aeons:
      protein families, pseudogenes and proteome evolution". Journal of
      Molecular Biology. 318 (5): 1155â74. doi:10.1016/S0022-2836(02)00109-2.
      PMID 12083509.
  13. ^Orengo CA, Thornton_JM (July 2005). "Protein families and their
      evolution-a structural perspective". Annual Review of Biochemistry. 74:
      867â900. doi:10.1146/annurev.biochem.74.082803.133029. PMID 15954844.
  14. ^Long M, BetrÃ¡n E, Thornton K, Wang W (November 2003). "The origin of
      new genes: glimpses from the young and old". Nature Reviews. Genetics. 4
      (11): 865â75. doi:10.1038/nrg1204. PMID 14634634.
  15. ^Wang M, Caetano-AnollÃ©s_G (January 2009). "The evolutionary mechanics
      of domain organization in proteomes and the rise of modularity in the
      protein world". Structure. 17 (1): 66â78. doi:10.1016/
      j.str.2008.11.008. PMID 19141283.
  16. ^Bowmaker JK (May 1998). "Evolution of colour vision in vertebrates".
      Eye. 12 (Pt 3b): 541â7. doi:10.1038/eye.1998.143. PMID 9775215.
  17. ^Gregory_TR, Hebert_PD (April 1999). "The modulation of DNA content:
      proximate causes and ultimate consequences". Genome Research. 9 (4):
      317â24. doi:10.1101/gr.9.4.317 (inactive 11 March 2019). PMID 10207154.
  18. ^Hurles M (July 2004). "Gene_duplication:_the_genomic_trade_in_spare
      parts". PLoS Biology. 2 (7): E206. doi:10.1371/journal.pbio.0020206.
      PMC 449868. PMID 15252449.
  19. ^Liu N, Okamura K, Tyler DM, Phillips MD, Chung WJ, Lai EC (October
      2008). "The_evolution_and_functional_diversification_of_animal_microRNA
      genes". Cell Research. 18 (10): 985â96. doi:10.1038/cr.2008.278.
      PMC 2712117. PMID 18711447.
  20. ^Siepel_A (October 2009). "Darwinian_alchemy:_Human_genes_from_noncoding
      DNA". Genome Research. 19 (10): 1693â5. doi:10.1101/gr.098376.109.
      PMC 2765273. PMID 19797681.
  21. ^Zhang J, Wang X, Podlaha O (May 2004). "Testing_the_chromosomal
      speciation_hypothesis_for_humans_and_chimpanzees". Genome Research. 14
      (5): 845â51. doi:10.1101/gr.1891104. PMC 479111. PMID 15123584.
  22. ^Ayala_FJ, Coluzzi M (May 2005). "Chromosome_speciation:_humans,
      Drosophila,_and_mosquitoes". Proceedings of the National Academy of
      Sciences of the United States of America. 102 Suppl 1 (Suppl 1):
      6535â42. Bibcode:2005PNAS..102.6535A. doi:10.1073/pnas.0501847102.
      PMC 1131864. PMID 15851677.
  23. ^Hurst GD, Werren JH (August 2001). "The role of selfish genetic elements
      in eukaryotic evolution". Nature Reviews Genetics. 2 (8): 597â606. doi:
      10.1038/35084545. PMID 11483984.
  24. ^HÃ¤sler J, Strub K (November 2006). "Alu_elements_as_regulators_of_gene
      expression". Nucleic Acids Research. 34 (19): 5491â7. doi:10.1093/nar/
      gkl706. PMC 1636486. PMID 17020921.
  25. ^ a b c dEyre-Walker A, Keightley_PD (August 2007). "The_distribution_of
      fitness_effects_of_new_mutations" (PDF). Nature Reviews Genetics. 8 (8):
      610â8. doi:10.1038/nrg2146. PMID 17637733. Archived (PDF) from the
      original on 4 March 2016.
  26. ^ a bKimura_M (1983). The_Neutral_Theory_of_Molecular_Evolution.
      Cambridge, UK; New York: Cambridge_University_Press. ISBN 978-0-521-
      23109-1. LCCN 82022225. OCLC 9081989.
  27. ^Bohidar HB (January 2015). Fundamentals_of_Polymer_Physics_and_Molecular
      Biophysics. Cambridge University Press. ISBN 978-1-316-09302-3.
  28. ^Dover GA, Darwin C (2000). Dear_Mr._Darwin:_Letters_on_the_Evolution_of
      Life_and_Human_Nature. University of California Press.
      ISBN 9780520227903.
  29. ^Tibayrenc, Michel (12 January 2017). Genetics_and_Evolution_of
      Infectious_Diseases. Elsevier. ISBN 9780128001530.
  30. ^ a b cBowler,_Peter_J. (1992) [1983]. The Eclipse of Darwinism. p. 198.
  31. ^Smocovitis, Vassiliki Betty (1996). Unifying Biology: The Evolutionary
      Synthesis and Evolutionary Biology. Princeton, NJ: Princeton University
      Press. p. 56. ISBN 978-0-691-03343-3. LCCN 96005605. OCLC 34411399.
  32. ^HallgrÃ­msson, Benedikt; Hall, Brian K. (2011). Variation: A Central
      Concept in Biology. Academic Press. p. 18.
  33. ^ Sewall_Wright. (1984). Evolution and the Genetics of Populations:
      Genetics and Biometric Foundations Volume 1. University of Chicago Press.
      p. 10
  34. ^De_Vries,_Hugo (1905). Species and Varieties: Their Origin by Mutation.
  35. ^Bateson,_William (1894). Materials for the Study of Variation, Treated
      with Especial Regard to Discontinuity in the Origin of Species.
  36. ^Punnett,_Reginald_C. (1915). Mimicry in Butterflies. Cambridge
      University Press.
  37. ^Mayr,_Ernst (2007). What Makes Biology Unique?: Considerations on the
      Autonomy of a Scientific Discipline. Cambridge University Press.
  38. ^ a bProvine, W. B. (2001). The Origins of Theoretical Population
      Genetics, with a new afterword. University of Chicago Press, Chicago.
      pp. 56â107.
  39. ^ a bStoltzfus, A.; Cable, K. (2014). "Mendelian-Mutationism: The
      Forgotten Evolutionary Synthesis". J Hist Biol. 47 (4): 501â546. doi:
      10.1007/s10739-014-9383-2. PMID 24811736.
  40. ^Hull, D. L. (1985). "Darwinism as an historical entity: A
      historiographic proposal". In Kohn, D. (ed.). The Darwinian Heritage.
      Princeton University Press. pp. 773â812.
  41. ^Yule,_G._Udny (1902). "Mendel's Laws and their probable relations to
      inter-racial heredity". New Phytologist. 1 (10): 226â227. doi:10.1111/
      j.1469-8137.1902.tb07336.x.
  42. ^ Gould, S. J. (1982). "The uses of heresey; an introduction to Richard
      Goldschmidt's The Material Basis of Evolution." pp._xiii-xlii. Yale
      University Press.
  43. ^Ruse,_Michael (1996). Monad to man: the Concept of Progress in
      Evolutionary Biology. Harvard University Press. pp. 412â413. ISBN 978-
      0-674-03248-4.
  44. ^A. Stoltzfus (2014). "In search of mutation-driven evolution". Evolution
      & Development. 16: 57â59. doi:10.1111/ede.12062.
  45. ^Futuyma,_Douglas_J. (2015). Serrelli, E.; Gontier, N. (eds.). Can_Modern
      Evolutionary_Theory_Explain_Macroevolution? (PDF). Macroevolution.
      Springer. pp. 29â85.
  46. ^"Cancer_Is_Partly_Caused_By_Bad_Luck,_Study_Finds". Archived from the
      original on 13 July 2017.
  47. ^Jha, Alok (22 August 2012). "Older_fathers_pass_on_more_genetic
      mutations,_study_shows". the Guardian.
  48. ^Montelone BA (1998). "Mutation,_Mutagens,_and_DNA_Repair". www-
      personal.ksu.edu. Archived from the original on 26 September 2015.
      Retrieved 2 October 2015.
  49. ^Bernstein C, Prasad AR, Nfonsam V, Bernstein H (2013). DNA_Damage,_DNA
      Repair_and_Cancer,_New_Research_Directions_in_DNA_Repair,_Prof._Clark
      Chen_(Ed.). intechopen.com. ISBN 978-953-51-1114-6. Archived from the
      original on 24 September 2015.
  50. ^Stuart GR, Oda Y, de Boer JG, Glickman BW (March 2000). "Mutation
      frequency_and_specificity_with_age_in_liver,_bladder_and_brain_of_lacI
      transgenic_mice". Genetics. 154 (3): 1291â300. PMC 1460990.
      PMID 10757770.
  51. ^Kunz BA, Ramachandran K, Vonarx EJ (April 1998). "DNA_sequence_analysis
      of_spontaneous_mutagenesis_in_Saccharomyces_cerevisiae". Genetics. 148
      (4): 1491â505. PMC 1460101. PMID 9560369.
  52. ^Lieber MR (July 2010). "The_mechanism_of_double-strand_DNA_break_repair
      by_the_nonhomologous_DNA_end-joining_pathway". Annual Review of
      Biochemistry. 79: 181â211. doi:10.1146/annurev.biochem.052308.093131.
      PMC 3079308. PMID 20192759.
  53. ^ Created from PDB_1JDG Archived 31 December 2015 at the Wayback_Machine
  54. ^Pfohl-Leszkowicz A, Manderville RA (January 2007). "Ochratoxin A: An
      overview on toxicity and carcinogenicity in animals and humans".
      Molecular Nutrition & Food Research. 51 (1): 61â99. doi:10.1002/
      mnfr.200600137. PMID 17195275.
  55. ^Kozmin S, Slezak G, Reynaud-Angelin A, Elie C, de Rycke Y, Boiteux S,
      Sage E (September 2005). "UVA_radiation_is_highly_mutagenic_in_cells_that
      are_unable_to_repair_7,8-dihydro-8-oxoguanine_in_Saccharomyces
      cerevisiae". Proceedings of the National Academy of Sciences of the
      United States of America. 102 (38): 13538â43. Bibcode:
      2005PNAS..10213538K. doi:10.1073/pnas.0504497102. PMC 1224634.
      PMID 16157879.
  56. ^ a bFitzgerald DM, Rosenberg SM (April 2019). "What_is_mutation?_A
      chapter_in_the_series:_How_microbes_"jeopardize"_the_modern_synthesis".
      PLoS Genetics. 15 (4): e1007995. doi:10.1371/journal.pgen.1007995.
      PMC 6443146. PMID 30933985.
  57. ^Galhardo RS, Hastings PJ, Rosenberg SM (1 January 2007). "Mutation_as_a
      stress_response_and_the_regulation_of_evolvability". Critical Reviews in
      Biochemistry and Molecular Biology. 42 (5): 399â435. doi:10.1080/
      10409230701648502. PMC 3319127. PMID 17917874.
  58. ^Quinto-Alemany D, Canerina-Amaro A, HernÃ¡ndez-Abad LG, MachÃ­n F,
      Romesberg FE, Gil-Lamaignere C (31 July 2012). Sturtevant J (ed.).
      "Yeasts_acquire_resistance_secondary_to_antifungal_drug_treatment_by
      adaptive_mutagenesis". PLOS One. 7 (7): e42279. doi:10.1371/
      journal.pone.0042279. PMC 3409178. PMID 22860105.
  59. ^ References for the image are found in Wikimedia Commons page at:
      Commons:File:Notable_mutations.svg#References.
  60. ^Rahman, Nazneen. "The_clinical_impact_of_DNA_sequence_changes".
      Transforming Genetic Medicine Initiative. Archived from the original on 4
      August 2017. Retrieved 27 June 2017.
  61. ^Freese_E (April 1959). "THE_DIFFERENCE_BETWEEN_SPONTANEOUS_AND_BASE-
      ANALOGUE_INDUCED_MUTATIONS_OF_PHAGE_T4". Proceedings of the National
      Academy of Sciences of the United States of America. 45 (4): 622â33.
      Bibcode:1959PNAS...45..622F. doi:10.1073/pnas.45.4.622. PMC 222607.
      PMID 16590424.
  62. ^Freese E (June 1959). "The specific mutagenic effect of base analogues
      on Phage T4". Journal of Molecular Biology. 1 (2): 87â105. doi:10.1016/
      S0022-2836(59)80038-3.
  63. ^Steenhuysen, Julie (19 December 2017). "U.S._Lifts_Funding_Ban_on
      Studies_That_Enhance_Dangerous_Germs". U.S. News & World Report.
      Retrieved 15 January 2018.
  64. ^Goh AM, Coffill CR, Lane_DP (January 2011). "The role of mutant p53 in
      human cancer". The Journal of Pathology. 223 (2): 116â26. doi:10.1002/
      path.2784. PMID 21125670.
  65. ^Chenevix-Trench_G, Spurdle AB, Gatei M, Kelly H, Marsh A, Chen X, Donn
      K, Cummings M, Nyholt D, Jenkins MA, Scott C, Pupo GM, DÃ¶rk T, Bendix R,
      Kirk J, Tucker K, McCredie MR, Hopper JL, Sambrook J, Mann GJ, Khanna KK
      (February 2002). "Dominant negative ATM mutations in breast cancer
      families". Journal of the National Cancer Institute. 94 (3): 205â15.
      CiteSeerX 10.1.1.557.6394. doi:10.1093/jnci/94.3.205. PMID 11830610.
  66. ^Paz-Priel I, Friedman A (2011). "C/EBPÎ±_dysregulation_in_AML_and_ALL".
      Critical Reviews in Oncogenesis. 16 (1â2): 93â102. doi:10.1615/
      critrevoncog.v16.i1-2.90. PMC 3243939. PMID 22150310.
  67. ^Capaccio D, Ciccodicola A, Sabatino L, Casamassimi A, Pancione M, Fucci
      A, Febbraro A, Merlino A, Graziano G, Colantuoni V (June 2010). "A novel
      germline mutation in peroxisome proliferator-activated receptor gamma
      gene associated with large intestine polyp formation and dyslipidemia".
      Biochimica et Biophysica Acta. 1802 (6): 572â81. doi:10.1016/
      j.bbadis.2010.01.012. PMID 20123124.
  68. ^McKusick_VA (July 1991). "The defect in Marfan syndrome". Nature. 352
      (6333): 279â81. Bibcode:1991Natur.352..279M. doi:10.1038/352279a0.
      PMID 1852198.
  69. ^Judge DP, Biery NJ, Keene DR, Geubtner J, Myers L, Huso DL, Sakai LY,
      Dietz HC (July 2004). "Evidence_for_a_critical_contribution_of
      haploinsufficiency_in_the_complex_pathogenesis_of_Marfan_syndrome". The
      Journal of Clinical Investigation. 114 (2): 172â81. doi:10.1172/
      JCI20641. PMC 449744. PMID 15254584.
  70. ^Judge DP, Dietz HC (December 2005). "Marfan's_syndrome". Lancet. 366
      (9501): 1965â76. doi:10.1016/S0140-6736(05)67789-6. PMC 1513064.
      PMID 16325700.
  71. ^Ellis NA, Ciocci S, German J (February 2001). "Back mutation can produce
      phenotype reversion in Bloom syndrome somatic cells". Human Genetics. 108
      (2): 167â73. doi:10.1007/s004390000447. PMID 11281456.
  72. ^Charlesworth_D, Charlesworth_B, Morgan MT (December 1995). "The_pattern
      of_neutral_molecular_variation_under_the_background_selection_model".
      Genetics. 141 (4): 1619â32. PMC 1206892. PMID 8601499.
  73. ^Loewe L (April 2006). "Quantifying the genomic decay paradox due to
      Muller's ratchet in human mitochondrial DNA". Genetical Research. 87 (2):
      133â59. doi:10.1017/S0016672306008123. PMID 16709275.
  74. ^Bernstein H, Hopf FA, Michod RE (1987). The molecular basis of the
      evolution of sex. Advances in Genetics. 24. pp. 323â70. doi:10.1016/
      s0065-2660(08)60012-7. ISBN 9780120176243. PMID 3324702.
  75. ^Peck JR, Barreau G, Heath SC (April 1997). "Imperfect_genes,_Fisherian
      mutation_and_the_evolution_of_sex". Genetics. 145 (4): 1171â99.
      PMC 1207886. PMID 9093868.
  76. ^Keightley PD, Lynch_M (March 2003). "Toward a realistic model of
      mutations affecting fitness". Evolution; International Journal of Organic
      Evolution. 57 (3): 683â5, discussion 686â9. doi:10.1554/0014-3820
      (2003)057[0683:tarmom]2.0.co;2. JSTOR 3094781. PMID 12703958.
  77. ^Barton_NH, Keightley PD (January 2002). "Understanding quantitative
      genetic variation". Nature Reviews Genetics. 3 (1): 11â21. doi:10.1038/
      nrg700. PMID 11823787.
  78. ^ a b cSanjuÃ¡n R, Moya A, Elena SF (June 2004). "The_distribution_of
      fitness_effects_caused_by_single-nucleotide_substitutions_in_an_RNA
      virus". Proceedings of the National Academy of Sciences of the United
      States of America. 101 (22): 8396â401. Bibcode:2004PNAS..101.8396S.
      doi:10.1073/pnas.0400146101. PMC 420405. PMID 15159545.
  79. ^Carrasco P, de la Iglesia F, Elena SF (December 2007). "Distribution_of
      fitness_and_virulence_effects_caused_by_single-nucleotide_substitutions
      in_Tobacco_Etch_virus". Journal of Virology. 81 (23): 12979â84. doi:
      10.1128/JVI.00524-07. PMC 2169111. PMID 17898073.
  80. ^SanjuÃ¡n R (June 2010). "Mutational_fitness_effects_in_RNA_and_single-
      stranded_DNA_viruses:_common_patterns_revealed_by_site-directed
      mutagenesis_studies". Philosophical Transactions of the Royal Society of
      London. Series B, Biological Sciences. 365 (1548): 1975â82. doi:
      10.1098/rstb.2010.0063. PMC 2880115. PMID 20478892.
  81. ^Peris JB, Davis P, Cuevas JM, Nebot MR, SanjuÃ¡n R (June 2010).
      "Distribution_of_fitness_effects_caused_by_single-nucleotide
      substitutions_in_bacteriophage_f1". Genetics. 185 (2): 603â9. doi:
      10.1534/genetics.110.115162. PMC 2881140. PMID 20382832.
  82. ^Elena SF, Ekunwe L, Hajela N, Oden SA, Lenski_RE (March 1998).
      "Distribution of fitness effects caused by random insertion mutations in
      Escherichia coli". Genetica. 102â103 (1â6): 349â58. doi:10.1023/A:
      1017031008316. PMID 9720287.
  83. ^ a bHietpas RT, Jensen JD, Bolon DN (May 2011). "Experimental
      illumination_of_a_fitness_landscape". Proceedings of the National Academy
      of Sciences of the United States of America. 108 (19): 7896â901.
      Bibcode:2011PNAS..108.7896H. doi:10.1073/pnas.1016024108. PMC 3093508.
      PMID 21464309.
  84. ^Davies EK, Peters AD, Keightley PD (September 1999). "High frequency of
      cryptic deleterious mutations in Caenorhabditis elegans". Science. 285
      (5434): 1748â51. doi:10.1126/science.285.5434.1748. PMID 10481013.
  85. ^Loewe L, Charlesworth B (September 2006). "Inferring_the_distribution_of
      mutational_effects_on_fitness_in_Drosophila". Biology Letters. 2 (3):
      426â30. doi:10.1098/rsbl.2006.0481. PMC 1686194. PMID 17148422.
  86. ^Eyre-Walker A, Woolfit M, Phelps T (June 2006). "The_distribution_of
      fitness_effects_of_new_deleterious_amino_acid_mutations_in_humans".
      Genetics. 173 (2): 891â900. doi:10.1534/genetics.106.057570.
      PMC 1526495. PMID 16547091.
  87. ^Sawyer SA, Kulathinal RJ, Bustamante_CD, Hartl DL (August 2003).
      "Bayesian analysis suggests that most amino acid replacements in
      Drosophila are driven by positive selection". Journal of Molecular
      Evolution. 57 Suppl 1 (1): S154â64. Bibcode:2003JMolE..57S.154S.
      CiteSeerX 10.1.1.78.65. doi:10.1007/s00239-003-0022-3. PMID 15008412.
  88. ^Piganeau G, Eyre-Walker A (September 2003). "Estimating_the_distribution
      of_fitness_effects_from_DNA_sequence_data:_implications_for_the_molecular
      clock". Proceedings of the National Academy of Sciences of the United
      States of America. 100 (18): 10335â40. Bibcode:2003PNAS..10010335P.
      doi:10.1073/pnas.1833064100. PMC 193562. PMID 12925735.
  89. ^Kimura_M (February 1968). "Evolutionary rate at the molecular level".
      Nature. 217 (5129): 624â6. Bibcode:1968Natur.217..624K. doi:10.1038/
      217624a0. PMID 5637732.
  90. ^Akashi H (September 1999). "Within- and between-species DNA sequence
      variation and the 'footprint' of natural selection". Gene. 238 (1):
      39â51. doi:10.1016/S0378-1119(99)00294-2. PMID 10570982.
  91. ^Eyre-Walker A (October 2006). "The genomic rate of adaptive evolution".
      Trends in Ecology & Evolution. 21 (10): 569â75. doi:10.1016/
      j.tree.2006.06.015. PMID 16820244.
  92. ^Gillespie_JH (September 1984). "Molecular Evolution Over the Mutational
      Landscape". Evolution. 38 (5): 1116â1129. doi:10.2307/2408444.
      JSTOR 2408444. PMID 28555784.
  93. ^Orr_HA (April 2003). "The_distribution_of_fitness_effects_among
      beneficial_mutations". Genetics. 163 (4): 1519â26. PMC 1462510.
      PMID 12702694.
  94. ^Kassen R, Bataillon T (April 2006). "Distribution of fitness effects
      among beneficial mutations before selection in experimental populations
      of bacteria". Nature Genetics. 38 (4): 484â8. doi:10.1038/ng1751.
      PMID 16550173.
  95. ^Rokyta DR, Joyce P, Caudle SB, Wichman HA (April 2005). "An empirical
      test of the mutational landscape model of adaptation using a single-
      stranded DNA virus". Nature Genetics. 37 (4): 441â4. doi:10.1038/
      ng1535. PMID 15778707.
  96. ^Imhof M, Schlotterer C (January 2001). "Fitness_effects_of_advantageous
      mutations_in_evolving_Escherichia_coli_populations". Proceedings of the
      National Academy of Sciences of the United States of America. 98 (3):
      1113â7. Bibcode:2001PNAS...98.1113I. doi:10.1073/pnas.98.3.1113.
      PMC 14717. PMID 11158603.
  97. ^Hogan, C. Michael (12 October 2010). "Mutation". In Monosson, Emily
      (ed.). Encyclopedia_of_Earth. Washington, D.C.: Environmental Information
      Coalition, National_Council_for_Science_and_the_Environment.
      OCLC 72808636. Archived from the original on 14 November 2015. Retrieved
      8 October 2015.
  98. ^BoillÃ©e S, Vande Velde C, Cleveland DW (October 2006). "ALS: a disease
      of motor neurons and their nonneuronal neighbors". Neuron. 52 (1):
      39â59. CiteSeerX 10.1.1.325.7514. doi:10.1016/j.neuron.2006.09.018.
      PMID 17015226.
  99. ^ a b"Somatic_cell_genetic_mutation". Genome Dictionary. Athens, Greece:
      Information Technology Associates. 30 June 2007. Archived from the
      original on 24 February 2010. Retrieved 6 June 2010.
 100. ^"RB1_Genetics". Daisy's Eye Cancer Fund. Oxford, UK. Archived from the
      original on 26 November 2011. Retrieved 9 October 2015.
 101. ^"Compound_heterozygote". MedTerms. New York: WebMD. 14 June 2012.
      Archived from the original on 4 March 2016. Retrieved 9 October 2015.
 102. ^Alberts (2014). Molecular Biology of the Cell (6 ed.). Garland Science.
      p. 487. ISBN 9780815344322.
 103. ^ a bChadov BF, Fedorova NB, Chadova EV (1 July 2015). "Conditional
      mutations in Drosophila melanogaster: On the occasion of the 150th
      anniversary of G. Mendel's report in BrÃ¼nn". Mutation Research/Reviews
      in Mutation Research. 765: 40â55. doi:10.1016/j.mrrev.2015.06.001.
      PMID 26281767.
 104. ^ a bLandis G, Bhole D, Lu L, Tower J (July 2001). "High-frequency
      generation_of_conditional_mutations_affecting_Drosophila_melanogaster
      development_and_life_span". Genetics. 158 (3): 1167â76. PMC 1461716.
      PMID 11454765. Archived from the original on 22 March 2017.
 105. ^ a b c dGierut JJ, Jacks TE, Haigis KM (April 2014). "Strategies_to
      achieve_conditional_gene_mutation_in_mice". Cold Spring Harbor Protocols.
      2014 (4): 339â49. doi:10.1101/pdb.top069807. PMC 4142476.
      PMID 24692485.
 106. ^Spencer DM (May 1996). "Creating conditional mutations in mammals".
      Trends in Genetics. 12 (5): 181â7. doi:10.1016/0168-9525(96)10013-5.
      PMID 8984733.
 107. ^Tan G, Chen M, Foote C, Tan C (September 2009). "Temperature-sensitive
      mutations_made_easy:_generating_conditional_mutations_by_using
      temperature-sensitive_inteins_that_function_within_different_temperature
      ranges". Genetics. 183 (1): 13â22. doi:10.1534/genetics.109.104794.
      PMC 2746138. PMID 19596904.
 108. ^den Dunnen JT, Antonarakis_SE (January 2000). "Mutation nomenclature
      extensions and suggestions to describe complex mutations: a discussion".
      Human Mutation. 15 (1): 7â12. doi:10.1002/(SICI)1098-1004(200001)15:
      1<7::AID-HUMU4>3.0.CO;2-N. PMID 10612815.
 109. ^Doniger SW, Kim HS, Swain D, Corcuera D, Williams M, Yang SP, Fay JC
      (August 2008). Pritchard_JK (ed.). "A_catalog_of_neutral_and_deleterious
      polymorphism_in_yeast". PLoS Genetics. 4 (8): e1000183. doi:10.1371/
      journal.pgen.1000183. PMC 2515631. PMID 18769710.
 110. ^Ionov Y, Peinado MA, Malkhosyan S, Shibata D, Perucho M (June 1993).
      "Ubiquitous somatic mutations in simple repeated sequences reveal a new
      mechanism for colonic carcinogenesis". Nature. 363 (6429): 558â61.
      Bibcode:1993Natur.363..558I. doi:10.1038/363558a0. PMID 8505985.
 111. ^Sullivan AD, Wigginton J, Kirschner D (August 2001). "The_coreceptor
      mutation_CCR5Delta32_influences_the_dynamics_of_HIV_epidemics_and_is
      selected_for_by_HIV". Proceedings of the National Academy of Sciences of
      the United States of America. 98 (18): 10214â9. Bibcode:
      2001PNAS...9810214S. doi:10.1073/pnas.181325198. PMC 56941.
      PMID 11517319.
 112. ^"Mystery_of_the_Black_Death". Secrets_of_the_Dead. Season 3. Episode 2.
      30 October 2002. PBS. Archived from the original on 12 October 2015.
      Retrieved 10 October 2015.
 113.  Episode background.
 114. ^Galvani AP, Slatkin_M (December 2003). "Evaluating_plague_and_smallpox
      as_historical_selective_pressures_for_the_CCR5-Delta_32_HIV-resistance
      allele". Proceedings of the National Academy of Sciences of the United
      States of America. 100 (25): 15276â9. Bibcode:2003PNAS..10015276G. doi:
      10.1073/pnas.2435085100. PMC 299980. PMID 14645720.
 115. ^Konotey-Ahulu, Felix. "Frequently_Asked_Questions_[FAQ's]".
      sicklecell.md. Archived from the original on 30 April 2011.
 116. ^Hughes D, Andersson DI (September 2017). "Evolutionary Trajectories to
      Antibiotic Resistance". Annual Review of Microbiology. 71: 579â596.
      doi:10.1146/annurev-micro-090816-093813. PMID 28697667.
 117. ^SÃ©gurel L, Bon C (August 2017). "On the Evolution of Lactase
      Persistence in Humans". Annual Review of Genomics and Human Genetics. 18:
      297â319. doi:10.1146/annurev-genom-091416-035340. PMID 28426286.
 118. ^"'Lifeless'_prion_proteins_are_'capable_of_evolution'". Health. BBC_News
      Online. London. 1 January 2010. Archived from the original on 25
      September 2015. Retrieved 10 October 2015.
 119. ^"somatic_mutation_|_genetics". Encyclopedia Britannica. Archived from
      the original on 31 March 2017. Retrieved 31 March 2017.
 120. ^Hartl, Jones, Daniel L., Elizabeth W. (1998). Genetics Principles and
      Analysis. Sudbury, Massachusetts: Jones and Bartlett Publishers. p. 556.
      ISBN 978-0-7637-0489-6.
 121. ^Milholland B, Dong X, Zhang L, Hao X, Suh Y, Vijg J (2017). "Differences
      between_germline_and_somatic_mutation_rates_in_humans_and_mice". Nat
      Commun. 8: 15183. Bibcode:2017NatCo...815183M. doi:10.1038/ncomms15183.
      PMC 5436103. PMID 28485371.
 122. ^Araten DJ, Golde DW, Zhang RH, Thaler HT, Gargiulo L, Notaro R, Luzzatto
      L (September 2005). "A quantitative measurement of the human somatic
      mutation rate". Cancer Research. 65 (18): 8111â7. doi:10.1158/0008-
      5472.CAN-04-1198. PMID 16166284.
***** External links[edit] *****
 Wikimedia Commons has media related to Mutations.
    * Jones_S, Woolfson A, Partridge_L (6 December 2007). "Genetic_Mutation".
      In_Our_Time. BBC_Radio_4. Retrieved 18 October 2015.
Liou, Stephanie (5 February 2011). "All_About_Mutations". HOPES. Huntington's
Disease_Outreach_Project_for_Education_at_Stanford. Retrieved 18 October 2015.
"Locus_Specific_Mutation_Databases". Leiden, the Netherlands: Leiden_University
Medical_Center. Retrieved 18 October 2015.
"Welcome_to_the_Mutalyzer_website". Leiden, the Netherlands: Leiden University
Medical Center. Retrieved 18 October 2015.
 â The Mutalyzer website.
    * v
    * t
    * e
Mutation
                                             * Insertion
                                             * Deletion
Mechanisms of mutation                       * Substitution
                                                   o Transversion
                                                   o Transition
                                                            * Nonsense
                                                              mutation
                                                            * Missense
                                                              mutation
                                                            * Conservative
                                         Point_mutation       mutation
                                                            * Silent
Mutation with respect to structure                            mutation
                                                            * Frameshift
                                                              mutation
                                                            * Dynamic
                                                              mutation
                                                            * Chromosomal
                                         Large-scale          translocations
                                         mutation           * Chromosomal
                                                              inversions
                                             * Deleterious_mutation
Mutation with respect to overall fitness     * Advantageous_mutation
                                             * Neutral_mutation
                                             * Nearly_neutral_mutation
    * v
    * t
    * e
Chromosome_abnormalities (Q90âQ99, 758)
                               * Down_syndrome
                                     o 21
                               * Edwards_syndrome
                                     o 18
                               * Patau_syndrome
               Trisomies             o 13
                               * Trisomy_9
                               * Warkany_syndrome_2
                                     o 8
                               * Cat_eye_syndrome/Trisomy_22
                                     o 22
                               * Trisomy_16
                               * 1q21.1_deletion_syndrome/1q21.1_duplication_syndrome/
                                 TAR_syndrome
                                     o 1
                               * WolfâHirschhorn_syndrome
Autosomal                            o 4
                               * Cri_du_chat/Chromosome_5q_deletion_syndrome
                                     o 5
                               * Williams_syndrome
                                     o 7
                               * Jacobsen_syndrome
               Monosomies/           o 11
               deletions       * MillerâDieker_syndrome/SmithâMagenis_syndrome
                                     o 17
                               * DiGeorge_syndrome
                                     o 22
                               * 22q11.2_distal_deletion_syndrome
                                     o 22
                               * 22q13_deletion_syndrome
                                     o 22
                               * genomic_imprinting
                                     o Angelman_syndrome/PraderâWilli_syndrome (15)
                               * Distal_18q-/Proximal_18q-
               Monosomy                     * Turner_syndrome_(45,X)
                                            * Klinefelter_syndrome_(47,XXY)
                                            * XXYY_syndrome_(48,XXYY)
                                            * XXXY_syndrome_(48,XXXY)
                                            * 49,XXXYY
                                            * 49,XXXXY
X/Y linked     Trisomy/tetrasomy,           * Triple_X_syndrome_(47,XXX)
               other_karyotypes/mosaics     * Tetrasomy_X_(48,XXXX)
                                            * 49,XXXXX
                                            * Jacobs_syndrome_(47,XYY)
                                            * 48,XYYY
                                            * 49,XYYYY
                                            * 45,X/46,XY
                                              * Burkitt's_lymphoma t(8 MYC;14 IGH)
                                              * Follicular_lymphoma t(14 IGH;18 BCL2)
                                              * Mantle_cell_lymphoma/Multiple_myeloma t
                                 Lymphoid       (11 CCND1:14 IGH)
                                              * Anaplastic_large-cell_lymphoma t(2 ALK;5
                                                NPM1)
               Leukemia/lymphoma              * Acute_lymphoblastic_leukemia
                                              * Philadelphia_chromosome t(9 ABL; 22 BCR)
                                              * Acute_myeloblastic_leukemia_with
                                                maturation t(8 RUNX1T1;21 RUNX1)
                                 Myeloid      * Acute_promyelocytic_leukemia t(15 PML,17
Translocations                                  RARA)
                                              * Acute_megakaryoblastic_leukemia t(1
                                                RBM15;22 MKL1)
                                     * Ewing's_sarcoma t(11 FLI1; 22 EWS)
                                     * Synovial_sarcoma t(x SYT;18 SSX)
                                     * Dermatofibrosarcoma_protuberans t(17 COL1A1;22
                                       PDGFB)
               Other                 * Myxoid_liposarcoma t(12 DDIT3; 16 FUS)
                                     * Desmoplastic_small-round-cell_tumor t(11 WT1; 22
                                       EWS)
                                     * Alveolar_rhabdomyosarcoma t(2 PAX3; 13 FOXO1) t
                                       (1 PAX7; 13 FOXO1)
                   * Fragile_X_syndrome
                   * Uniparental_disomy
Other              * XX_male_syndrome/46,XX_testicular_disorders_of_sex_development
                   * Marker_chromosome
                   * Ring_chromosome
                         o 6; 9; 14; 15; 18; 20; 21, 22
    * v
    * t
    * e
Evolutionary_biology
    * Introduction
    * Outline
    * Timeline_of_evolution
    * Evolutionary_history_of_life
    * Index
                        * Abiogenesis
                        * Adaptation
                        * Adaptive_radiation
                        * Cladistics
                        * Coevolution
                        * Common_descent
                        * Convergence
                        * Divergence
                        * Earliest_known_life
                          forms
                        * Evidence_of_evolution
Evolution               * Extinction
                              o Event
                        * Gene-centered_view
                        * Homology
                        * Last_universal_common
                          ancestor
                        * Macroevolution
                        * Microevolution
                        * Origin_of_life
                        * Panspermia
                        * Parallel_evolution
                        * Speciation
                        * Taxonomy
                        * Biodiversity
                        * Gene_flow
Population_genetics     * Genetic_drift
                        * Mutation
                        * Natural_selection
                        * Variation
                        * Canalisation
                        * Evolutionary
Development               developmental_biology
                        * Inversion
                        * Modularity
                        * Phenotypic_plasticity
                        * Birds
                              o origin
                        * Brachiopods
                        * Molluscs
                              o Cephalopods
                        * Dinosaurs
                        * Fish
                        * Fungi
                        * Insects
                              o butterflies
                        * Life
                        * Mammals
                              o cats
                              o canids
                                    # wolves
Of taxa                             # dogs
                              o hyenas
                              o dolphins_and
                                whales
                              o horses
                              o Kangaroos
                              o primates
                                    # humans
                                    # lemurs
                              o sea_cows
                        * Plants
                        * Reptiles
                        * Spiders
                        * Tetrapods
                        * Viruses
                              o influenza
                        * Cell
                        * DNA
                        * Flagella
                        * Eukaryotes
                              o symbiogenesis
                              o chromosome
                              o endomembrane
                                system
Of organs                     o mitochondria
                              o nucleus
                              o plastids
                        * In animals
                              o eye
                              o hair
                              o auditory_ossicle
                              o nervous_system    [Common-Avian-Infection-
                              o brain             Plagued-the-Tyrant-Dinosaurs-
                        * Aging                   pone.0007288.g004.jpg]
                              o Death                 * [icon]Evolutionary
                              o Programmed_cell         biology_portal
                                death
                        * Avian_flight
                        * Biological_complexity
                        * Cooperation
                        * Color_vision
                              o in_primates
                        * Emotion
                        * Empathy
                        * Ethics
                        * Eusociality
                        * Immune_system
Of processes            * Metabolism
                        * Monogamy
                        * Morality
                        * Mosaic_evolution
                        * Multicellularity
                        * Sexual_reproduction
                              o Gamete
                                differentiation/
                                sexes
                              o Life_cycles/
                                nuclear_phases
                              o Mating_types
                              o Meiosis
                              o Sex-determination
                        * Snake_venom
                        * Gradualism/Punctuated
                          equilibrium/
                          Saltationism
Tempo_and_modes         * Micromutation/
                          Macromutation
                        * Uniformitarianism/
                          Catastrophism
                        * Allopatric
                        * Anagenesis
                        * Catagenesis
                        * Cladogenesis
                        * Cospeciation
Speciation              * Ecological
                        * Hybrid
                        * Parapatric
                        * Peripatric
                        * Reinforcement
                        * Sympatric
                        * Renaissance_and
                          Enlightenment
                        * Transmutation_of
                          species
                        * Charles_Darwin
                              o On_the_Origin_of
                                Species
                        * History_of_paleontology
History                 * Transitional_fossil
                        * Blending_inheritance
                        * Mendelian_inheritance
                        * The_eclipse_of
                          Darwinism
                        * Modern_synthesis
                        * History_of_molecular
                          evolution
                        * Extended_evolutionary
                          synthesis
                        * Darwinism
                        * Alternatives
                              o Catastrophism
                              o Lamarckism
                              o Orthogenesis
Philosophy                    o Mutationism
                              o Saltationism
                              o Structuralism
                                    # Spandrel
                              o Theistic
                              o Vitalism
                        * Teleology_in_biology
                        * Biogeography
                        * Ecological_genetics
                        * Molecular_evolution
Related                 * Phylogenetics
                              o Tree
                        * Polymorphism
                        * Protocell
                        * Systematics
    * [Category] Category
    * [Portal] Portal
    * [Issoria_lathonia.jpg]Biology_portal
    * [WHO_Rod.svg]Medicine_portal
    * [Tree_of_life.svg]Evolutionary_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Mutation&oldid=909707958"
Categories:
    * Mutation
    * Evolutionary_biology
    * Radiation_health_effects
    * Molecular_evolution
Hidden categories:
    * Pages_with_DOIs_inactive_as_of_2019
    * CS1:_long_volume_value
    * Webarchive_template_wayback_links
    * Articles_with_short_description
    * All_articles_lacking_reliable_references
    * Articles_lacking_reliable_references_from_January_2017
    * Commons_category_link_is_on_Wikidata
    * Use_dmy_dates_from_July_2011
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Afrikaans
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * AzÉrbaycanca
    * à¦¬à¦¾à¦à¦²à¦¾
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ_(ÑÐ°ÑÐ°ÑÐºÐµÐ²ÑÑÐ°)â
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Ido
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * à²à²¨à³à²¨à²¡
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * KreyÃ²l_ayisyen
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * LatvieÅ¡u
    * LietuviÅ³
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * à¤¨à¥à¤ªà¤¾à¤²à¥
    * æ¥æ¬èª
    * Nordfriisk
    * Norsk
    * Norsk_nynorsk
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * à¨ªà©°à¨à¨¾à¨¬à©
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Shqip
    * Sicilianu
    * à·à·à¶à·à¶½
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * Winaray
    * å´è¯­
    * ××Ö´×××©
    * ç²µèª
    * Zazaki
    * ä¸­æ
Edit_links
    * This page was last edited on 7 August 2019, at 02:45 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
