The following text has been accessed from https://en.wikipedia.org/wiki/Genetic_linkage at Thu Aug 8 23:59:23 IST 2019
Creative_Commons_Attribution-ShareAlike_License




















****** Genetic linkage ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
The tendency of DNA sequences that are close together on a chromosome to be
inherited together
"Genetic map" redirects here. It is not to be confused with Gene_map.
Genetic linkage is the tendency of DNA_sequences that are close together on a
chromosome to be inherited together during the meiosis phase of sexual
reproduction. Two genetic_markers that are physically near to each other are
unlikely to be separated onto different chromatids during chromosomal
crossover, and are therefore said to be more linked than markers that are far
apart. In other words, the nearer two genes are on a chromosome, the lower the
chance of recombination between them, and the more likely they are to be
inherited together. Markers on different chromosomes are perfectly unlinked.
Genetic linkage is the most prominent exception to Gregor_Mendel's Law_of
Independent_Assortment. The first experiment to demonstrate linkage was carried
out in 1905. At the time, the reason why certain traits tend to be inherited
together was unknown. Later work revealed that genes are physical structures
related by physical distance.
The typical unit of genetic linkage is the centimorgan (cM). A distance of 1 cM
between two markers means that the markers are separated to different
chromosomes on average once per 100 meiotic product, thus once per 50 meioses.
⁰
***** Contents *****
    * 1_Discovery
    * 2_Linkage_map
    * 3_Linkage_analysis
          o 3.1_Parametric_linkage_analysis
          o 3.2_Limitations
    * 4_Recombination_frequency
    * 5_Variation_of_recombination_frequency
    * 6_Meiosis_indicators
    * 7_See_also
    * 8_References
***** Discovery[edit] *****
Gregor_Mendel's Law_of_Independent_Assortment states that every trait is
inherited independently of every other trait. But shortly after Mendel's_work
was_rediscovered, exceptions to this rule were found. In 1905, the British
geneticists William_Bateson, Edith_Rebecca_Saunders and Reginald_Punnett cross-
bred pea plants in experiments similar to Mendel's.[1][2] They were interested
in trait inheritance in the sweet pea and were studying two genesâthe gene
for flower colour (P, purple, and p, red) and the gene affecting the shape of
pollen grains (L, long, and l, round). They crossed the pure lines PPLL and
ppll and then self-crossed the resulting PpLl lines.
According to Mendelian_genetics, the expected phenotypes would occur in a 9:3:
3:1 ratio of PL:Pl:pL:pl. To their surprise, they observed an increased
frequency of PL and pl and a decreased frequency of Pl and pL:
         Bateson, Saunders, and Punnett experiment
Phenotype and genotype Observed Expected from 9:3:3:1 ratio
Purple, long (P_L_)    284      216
Purple, round (P_ll)   21       72
Red, long (ppL_)       21       72
Red, round (ppll)      55       24
Their experiment revealed linkage between the P and L alleles and the p and l
alleles. The frequency of P occurring together with L and p occurring together
with l is greater than that of the recombinant Pl and pL. The recombination
frequency is more difficult to compute in an F2 cross than a backcross,[3] but
the lack of fit between observed and expected numbers of progeny in the above
table indicate it is less than 50%. This indicated that two factors interacted
in some way to create this difference by masking the appearance of the other
two phenotypes. This led to the conclusion that some traits are related to each
other because of their near proximity to each other on a chromosome. This
provided the grounds to determine the difference between independent and
codependent alleles.[citation_needed]
The understanding of linkage was expanded by the work of Thomas_Hunt_Morgan.
Morgan's observation that the amount of crossing_over between linked genes
differs led to the idea that crossover frequency might indicate the distance
separating genes on the chromosome. The centimorgan, which expresses the
frequency of crossing over, is named in his honour.
***** Linkage map[edit] *****
Thomas_Hunt_Morgan's Drosophila_melanogaster genetic linkage map. This was the
first successful gene_mapping work and provides important evidence for the
chromosome_theory_of_inheritance. The map shows the relative positions of
alleles on the second Drosophila chromosome. The distances between the genes
(centimorgans) are equal to the percentages of chromosomal_crossover events
that occur between different alleles.[4]
A linkage map (also known as a genetic map) is a table for a species or
experimental population that shows the position of its known genes or genetic
markers relative to each other in terms of recombination frequency, rather than
a specific physical distance along each chromosome. Linkage maps were first
developed by Alfred_Sturtevant, a student of Thomas_Hunt_Morgan.
A linkage map is a map based on the frequencies of recombination between
markers during crossover of homologous_chromosomes. The greater the frequency
of recombination (segregation) between two genetic markers, the further apart
they are assumed to be. Conversely, the lower the frequency of recombination
between the markers, the smaller the physical distance between them.
Historically, the markers originally used were detectable phenotypes (enzyme
production, eye colour) derived from coding_DNA sequences; eventually,
confirmed or assumed noncoding_DNA sequences such as microsatellites or those
generating restriction fragment length polymorphisms (RFLPs) have been used.
Linkage maps help researchers to locate other markers, such as other genes by
testing for genetic linkage of the already known markers. In the early stages
of developing a linkage map, the data are used to assemble linkage groups, a
set of genes which are known to be linked. As knowledge advances, more markers
can be added to a group, until the group covers an entire chromosome.[5] For
well-studied organisms the linkage groups correspond one-to-one with the
chromosomes.
A linkage map is not a physical map (such as a radiation_reduced_hybrid map) or
gene_map.
***** Linkage analysis[edit] *****
Linkage analysis is a genetic method that searches for chromosomal segments
that cosegregate with the ailment phenotype through families and is the
analysis technique that has been used to determine the bulk of lipodystrophy
genes.[6][7] It can be used to map genes for both binary and quantitative
traits.[7] Linkage analysis may be either parametric (if we know the
relationship between phenotypic and genetic similarity) or non-parametric.
Parametric linkage analysis is the traditional approach, whereby the
probability that a gene important for a disease is linked to a genetic marker
is studied through the LOD score, which assesses the probability that a given
pedigree, where the disease and the marker are cosegregating, is due to the
existence of linkage (with a given linkage value) or to chance. Non-parametric
linkage analysis, in turn, studies the probability of an allele being identical
by_descent with itself.
**** Parametric linkage analysis[edit] ****
The LOD score (logarithm (base 10) of odds), developed by Newton_Morton,[8] is
a statistical test often used for linkage analysis in human, animal, and plant
populations. The LOD score compares the likelihood of obtaining the test data
if the two loci are indeed linked, to the likelihood of observing the same data
purely by chance. Positive LOD scores favour the presence of linkage, whereas
negative LOD scores indicate that linkage is less likely. Computerised LOD
score analysis is a simple way to analyse complex family pedigrees in order to
determine the linkage between Mendelian traits (or between a trait and a
marker, or two markers).
The method is described in greater detail by Strachan and Read.[1] Briefly, it
works as follows:
   1. Establish a pedigree
   2. Make a number of estimates of recombination frequency
   3. Calculate a LOD score for each estimate
   4. The estimate with the highest LOD score will be considered the best
      estimate
The LOD score is calculated as follows:
       L O D = Z    =  log  10   &#x2061;    probability of birth sequence with
a given linkage value   probability of birth sequence with no linkage    =  log
10   &#x2061;    ( 1 &#x2212; &#x03B8;  )  N R   &#x00D7;  &#x03B8;  R     0.5
( N R + R )           {\displaystyle {\begin{aligned}LOD=Z&=\log _{10}{\frac
{\mbox{probability of birth sequence with a given linkage value}}{\mbox
{probability of birth sequence with no linkage}}}=\log _{10}{\frac {(1-\theta
)^{NR}\times \theta ^{R}}{0.5^{(NR+R)}}}\end{aligned}}}  [{\displaystyle
{\begin{aligned}LOD=Z&=\log _{10}{\frac {\mbox{probability of birth sequence
with a given linkage value}}{\mbox{probability of birth sequence with no
linkage}}}=\log _{10}{\frac {(1-\theta )^{NR}\times \theta ^{R}}{0.5^{
(NR+R)}}}\end{aligned}}}]
NR denotes the number of non-recombinant offspring, and R denotes the number of
recombinant offspring. The reason 0.5 is used in the denominator is that any
alleles that are completely unlinked (e.g. alleles on separate chromosomes)
have a 50% chance of recombination, due to independent assortment. 'Î¸' is the
recombinant fraction, i.e. the fraction of births in which recombination has
happened between the studied genetic marker and the putative gene associated
with the disease. Thus, it is equal to R / (NR + R)
By convention, a LOD score greater than 3.0 is considered evidence for linkage,
as it indicates 1000 to 1 odds that the linkage being observed did not occur by
chance. On the other hand, a LOD score less than -2.0 is considered evidence to
exclude linkage. Although it is very unlikely that a LOD score of 3 would be
obtained from a single pedigree, the mathematical properties of the test allow
data from a number of pedigrees to be combined by summing their LOD scores. A
LOD score of 3 translates to a p-value of approximately 0.05,[9] and no
multiple_testing_correction (e.g. Bonferroni_correction) is required.[10]
**** Limitations[edit] ****
Linkage analysis has a number of methodological and theoretical limitations
that can significantly increase the type-1_error rate and reduce the power to
map human quantitative trait loci (QTL).[11] While linkage analysis was
successfully used to identify genetic variants that contribute to rare
disorders such as Huntington_disease, it didnât perform that well when
applied to more common disorders such as heart_disease or different forms of
cancer.[12] An explanation for this is that the genetic mechanisms affecting
common disorders are different from those causing rare disorders.[13]
***** Recombination frequency[edit] *****
Recombination frequency is a measure of genetic linkage and is used in the
creation of a genetic linkage map. Recombination frequency (Î¸) is the
frequency with which a single chromosomal_crossover will take place between two
genes during meiosis. A centimorgan (cM) is a unit that describes a
recombination frequency of 1%. In this way we can measure the genetic distance
between two loci, based upon their recombination frequency. This is a good
estimate of the real distance. Double crossovers would turn into no
recombination. In this case we cannot tell if crossovers took place. If the
loci we're analysing are very close (less than 7 cM) a double crossover is very
unlikely. When distances become higher, the likelihood of a double crossover
increases. As the likelihood of a double crossover increases we systematically
underestimate the genetic distance between two loci.
During meiosis, chromosomes assort randomly into gametes, such that the
segregation of alleles of one gene is independent of alleles of another gene.
This is stated in Mendel's_Second_Law and is known as the law of independent
assortment. The law of independent assortment always holds true for genes that
are located on different chromosomes, but for genes that are on the same
chromosome, it does not always hold true.
As an example of independent assortment, consider the crossing of the pure-bred
homozygote parental strain with genotype AABB with a different pure-bred strain
with genotype aabb. A and a and B and b represent the alleles of genes A and B.
Crossing these homozygous parental strains will result in F1 generation
offspring that are double heterozygotes with genotype AaBb. The F1 offspring
AaBb produces gametes that are AB, Ab, aB, and ab with equal frequencies (25%)
because the alleles of gene A assort independently of the alleles for gene B
during meiosis. Note that 2 of the 4 gametes (50%)âAb and aBâwere not
present in the parental generation. These gametes represent recombinant
gametes. Recombinant gametes are those gametes that differ from both of the
haploid gametes that made up the original diploid cell. In this example, the
recombination frequency is 50% since 2 of the 4 gametes were recombinant
gametes.
The recombination frequency will be 50% when two genes are located on different
chromosomes or when they are widely separated on the same chromosome. This is a
consequence of independent assortment.
When two genes are close together on the same chromosome, they do not assort
independently and are said to be linked. Whereas genes located on different
chromosomes assort independently and have a recombination frequency of 50%,
linked genes have a recombination frequency that is less than 50%.
As an example of linkage, consider the classic experiment by William_Bateson
and Reginald_Punnett.[citation_needed] They were interested in trait
inheritance in the sweet pea and were studying two genesâthe gene for flower
colour (P, purple, and p, red) and the gene affecting the shape of pollen
grains (L, long, and l, round). They crossed the pure lines PPLL and ppll and
then self-crossed the resulting PpLl lines. According to Mendelian_genetics,
the expected phenotypes would occur in a 9:3:3:1 ratio of PL:Pl:pL:pl. To their
surprise, they observed an increased frequency of PL and pl and a decreased
frequency of Pl and pL (see table below).
              Bateson and Punnett experiment
Phenotype and genotype Observed Expected from 9:3:3:1 ratio
Purple, long (P_L_)    284      216
Purple, round (P_ll)   21       72
Red, long (ppL_)       21       72
Red, round (ppll)      55       24
Their experiment revealed linkage between the P and L alleles and the p and l
alleles. The frequency of P occurring together with L and with p occurring
together with l is greater than that of the recombinant Pl and pL. The
recombination frequency is more difficult to compute in an F2 cross than a
backcross,[3] but the lack of fit between observed and expected numbers of
progeny in the above table indicate it is less than 50%.
The progeny in this case received two dominant alleles linked on one chromosome
(referred to as coupling or cis arrangement). However, after crossover, some
progeny could have received one parental chromosome with a dominant allele for
one trait (e.g. Purple) linked to a recessive allele for a second trait (e.g.
round) with the opposite being true for the other parental chromosome (e.g. red
and Long). This is referred to as repulsion or a trans arrangement. The
phenotype here would still be purple and long but a test cross of this
individual with the recessive parent would produce progeny with much greater
proportion of the two crossover phenotypes. While such a problem may not seem
likely from this example, unfavourable repulsion linkages do appear when
breeding for disease resistance in some crops.
The two possible arrangements, cis and trans, of alleles in a double
heterozygote are referred to as gametic_phases, and phasing is the process of
determining which of the two is present in a given individual.
When two genes are located on the same chromosome, the chance of a crossover
producing recombination between the genes is related to the distance between
the two genes. Thus, the use of recombination frequencies has been used to
develop linkage maps or genetic maps.
However, it is important to note that recombination frequency tends to
underestimate the distance between two linked genes. This is because as the two
genes are located farther apart, the chance of double or even number of
crossovers between them also increases. Double or even number of crossovers
between the two genes results in them being cosegregated to the same gamete,
yielding a parental progeny instead of the expected recombinant progeny. As
mentioned above, the Kosambi and Haldane transformations attempt to correct for
multiple crossovers.[14][15]
***** Variation of recombination frequency[edit] *****
While recombination of chromosomes is an essential process during meiosis,
there is a large range of frequency of cross overs across organisms and within
species. Sexually dimorphic rates of recombination are termed heterochiasmy,
and are observed more often than a common rate between male and females. In
mammals, females often have a higher rate of recombination compared to males.
It is theorised that there are unique selections acting or meiotic drivers
which influence the difference in rates. The difference in rates may also
reflect the vastly different environments and conditions of meiosis in
oogenesis and spermatogenesis.[citation_needed]
***** Meiosis indicators[edit] *****
With very large pedigrees or with very dense genetic marker data, such as from
whole-genome sequencing, it is possible to precisely locate recombinations.
With this type of genetic analysis, a meiosis indicator is assigned to each
position of the genome for each meiosis in a pedigree. The indicator indicates
which copy of the parental chromosome contributes to the transmitted gamete at
that position. For example, if the allele from the 'first' copy of the parental
chromosome is transmitted, a '0' might be assigned to that meiosis. If the
allele from the 'second' copy of the parental chromosome is transmitted, a '1'
would be assigned to that meiosis. The two alleles in the parent came, one
each, from two grandparents. These indicators are then used to determine
identical-by-descent (IBD) states or inheritance states, which are in turn used
to identify genes responsible for diseases.
***** See also[edit] *****
    * Centimorgan
    * Genetic_association
    * Genetic_epidemiology
    * Genome-wide_association_study
    * Identity_by_descent
    * LanderâGreen_algorithm
    * Linkage_disequilibrium
    * Structural_motif
***** References[edit] *****
   1. ^Lobo, Ingrid; Shaw, Kenna. "Discovery_and_Types_of_Genetic_Linkage".
      Scitable. Nature_Education. Retrieved 21 January 2017.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Bateson,_W; Saunders,_ER; Punnett,_RC (18 May 1904). Reports_to_the
      Evolution_committee_of_the_Royal_Society. London: Harrison and Sons,
      Printers. Retrieved 21 January 2017.
   4. ^ a bFisher,_RA; Balmukand, B (July 1928). "The estimation of linkage
      from the offspring of selfed heterozygotes". Journal_of_Genetics. 20 (1):
      79â92. doi:10.1007/BF02983317.
   5. ^Mader, Sylvia (2007). Biology Ninth Edition. New York: McGraw-Hill.
      p. 209. ISBN 978-0-07-325839-3.
   6. ^Griffiths, AJF (2000). An_Introduction_to_Genetic_Analysis (7th ed.). W.
      H. Freeman.
   7. ^Lanktree, Matthew B.; Johansen, Christopher T.; Joy, Tisha R.; Hegele,
      Robert A. (2010), "A Translational View of the Genetics of Lipodystrophy
      and Ectopic Fat Deposition", in Bouchard, Claude (ed.), Progress in
      Molecular Biology and Translational Science, Genes and Obesity, 94,
      Academic Press, pp. 159â196, doi:10.1016/b978-0-12-375003-7.00006-6,
      ISBN 9780123750037, PMID 21036325
   8. ^ a bCantor, Rita M. (2013), "Analysis of Genetic Linkage", in Rimoin,
      David; Pyeritz, Reed; Korf, Bruce (eds.), Emery and Rimoin's Principles
      and Practice of Medical Genetics (6th ed.), Academic Press, pp. 1â9,
      doi:10.1016/b978-0-12-383834-6.00010-0, ISBN 9780123838346
   9. ^Morton NE (1955). "Sequential_tests_for_the_detection_of_linkage".
      American Journal of Human Genetics. 7 (3): 277â318. PMC 1716611.
      PMID 13258560.
  10. ^Nyholt, Dale R (August 2000). "All_LODs_Are_Not_Created_Equal". American
      Journal_of_Human_Genetics. 67 (2): 282â288. doi:10.1086/303029.
      PMC 1287176. PMID 10884360.
  11. ^Risch, Neil (June 1991). "A_Note_on_Multiple_Testing_Procedures_in
      Linkage_Analysis". American_Journal_of_Human_Genetics. 48 (6):
      1058â1064. PMC 1683115. PMID 2035526.
  12. ^Ferreira, Manuel A. R. (2004-10-01). "Linkage Analysis: Principles and
      Methods for the Analysis of Human Quantitative Traits". Twin Research and
      Human Genetics. 7 (5): 513â530. doi:10.1375/twin.7.5.513. ISSN 2053-
      6003.
  13. ^Gusella, James F.; Frontali, Marina; Wasmuth, John J.; Collins, Francis
      S.; Lehrach, Hans; Myers, Richard; Altherr, Michael; Allitto, Bernice;
      Taylor, Sherry (1992-05-01). "The Huntington's disease candidate region
      exhibits many different haplotypes". Nature Genetics. 1 (2): 99â103.
      doi:10.1038/ng0592-99. ISSN 1546-1718. PMID 1302016.
  14. ^Mark J. Daly; Hirschhorn, Joel N. (2005-02-01). "Genome-wide association
      studies for common diseases and complex traits". Nature Reviews Genetics.
      6 (2): 95â108. doi:10.1038/nrg1521. ISSN 1471-0064. PMID 15716906.
  15. ^Griffiths, AJF; Miller, JH; Suzuki, DT (2000). "Accurate_calculation_of
      large_map_distances,_Derivation_of_a_mapping_function". An Introduction
      to Genetic Analysis (7th ed.). New York: W._H._Freeman. ISBN 978-0-7167-
      3520-5.
  16. ^Griffiths, AJF; Miller, JH; Suzuki, DT (2000). "Accurate_calculation_of
      large_map_distances,_Figure_6-4". An Introduction to Genetic Analysis
      (7th ed.). New York: W._H._Freeman. ISBN 978-0-7167-3520-5.
  17.  Graph of mapping function from compared to idealised 1-1 equivalence of
      recombination frequency percentage (RF%) to map units.
Library_resources about
Gene mapping
===============================================================================
    * Resources_in_your_library
    * Resources_in_other_libraries
    * Griffiths AJF; Miller JH; Suzuki DT; Lewontin RC; et al. (1993). "Chapter
      5". An Introduction to Genetic Analysis (5th ed.). New York: W.H. Freeman
      and Company. ISBN 978-0-7167-2285-4.
Poehlman JM; Sleper DA (1995). "Chapter 3". Breeding Field Crops (4th ed.).
Iowa: Iowa State Press. ISBN 978-0-8138-2427-7.
    * v
    * t
    * e
Personal_genomics
Data collection         * Biobank
                        * Biological_database
                        * Biological_specimen
                        * De-identification
                        * Human_genetic_variation
Field concepts          * Genetic linkage
                        * Single-nucleotide_polymorphisms
                        * Identity_by_descent
                        * Genetic_disorder
                        * Personalized_medicine
Applications            * Predictive_medicine
                        * Genetic_epidemiology
                        * Pharmacogenomics
                        * Whole_genome_sequencing
Analysis techniques     * Genome-wide_association_study
                        * SNP_array
                        * Genetic_testing
                        * Human_Genome_Project
Major projects          * International_HapMap_Project
                        * 1000_Genomes_Project
                        * Human_Genome_Diversity_Project

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Genetic_linkage&oldid=908285295"
Categories:
    * Classical_genetics
Hidden categories:
    * CS1:_JulianâGregorian_uncertainty
    * Articles_with_short_description
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_June_2018
    * Articles_with_unsourced_statements_from_March_2014
    * Articles_with_unsourced_statements_from_June_2015
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * Hrvatski
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * LietuviÅ³
    * æ¥æ¬èª
    * Norsk
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Simple_English
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 28 July 2019, at 18:57 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
