The following text has been accessed from https://en.wikipedia.org/wiki/Phylogenetic_tree at Fri Aug 9 02:18:44 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Phylogenetic tree ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Diagrammatic hypothesis about the evolutionary relationships of a group of
organisms
A speculatively rooted tree for rRNA genes, showing the three_life_domains:
bacteria, archaea, and eukaryota. The black trunk at the bottom of the tree
links the three branches of living organisms to the last_universal_common
ancestor.
A rooted phylogenetic tree, illustrating how Eukaryota and Archaea are more
closely related to each other than to Bacteria (based on Cavalier-Smith's
theory of bacterial evolution). Neomura is a clade composed of two life
domains, Archaea and Eukaryota.
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
A phylogenetic tree or evolutionary tree is a branching diagram or "tree"
showing the evolutionary relationships among various biological species or
other entitiesâtheir phylogeny (/faÉªËlÉdÊÉni/)âbased upon similarities
and differences in their physical or genetic characteristics. All life on Earth
is part of a single phylogenetic tree, indicating common_ancestry.
In a rooted phylogenetic tree, each node with descendants represents the
inferred most_recent_common_ancestor of those descendants, and the edge lengths
in some trees may be interpreted as time estimates. Each node is called a
taxonomic unit. Internal nodes are generally called hypothetical taxonomic
units, as they cannot be directly observed. Trees are useful in fields of
biology such as bioinformatics, systematics, and phylogenetics. Unrooted trees
illustrate only the relatedness of the leaf_nodes and do not require the
ancestral root to be known or inferred.
⁰
***** Contents *****
    * 1_History
    * 2_Types
          o 2.1_Rooted_tree
          o 2.2_Unrooted_tree
          o 2.3_Bifurcating_tree
          o 2.4_Coral_of_life
          o 2.5_Special_tree_types
    * 3_Construction
    * 4_Limitations_of_phylogenetic_analysis
    * 5_See_also
    * 6_References
    * 7_Further_reading
    * 8_External_links
          o 8.1_Images
          o 8.2_General
***** History[edit] *****
The idea of a "tree_of_life" arose from ancient notions of a ladder-like
progression from lower into higher forms of life (such as in the Great_Chain_of
Being). Early representations of "branching" phylogenetic trees include a
"paleontological chart" showing the geological relationships among plants and
animals in the book Elementary Geology, by Edward Hitchcock (first edition:
1840).
Charles_Darwin (1859) also produced one of the first illustrations and
crucially popularized the notion of an evolutionary_"tree" in his seminal book
The_Origin_of_Species. Over a century later, evolutionary_biologists still use
tree_diagrams to depict evolution because such diagrams effectively convey the
concept that speciation occurs through the adaptive and semirandom splitting of
lineages. Over time, species classification has become less static and more
dynamic.
The term phylogenetic, or phylogeny, derives from the two ancient_greek words
Ïá¿¦Î»Î¿Î½ (phÃ»lon), meaning "race, lineage", and Î³Î­Î½ÎµÏÎ¹Ï (gÃ©nesis),
meaning "origin, source".[1][2]
***** Types[edit] *****
**** Rooted tree[edit] ****
A rooted phylogenetic tree (see two graphics at top) is a directed tree with a
unique node â the root â corresponding to the (usually imputed) most recent
common ancestor of all the entities at the leaves of the tree. The root node
does not have a parent node, but serves as the parent of all other nodes in the
tree. The root is therefore a node of degree 2 while other internal nodes have
a minimum degree of 3 (where "degree" here refers to the total number of
incoming and outgoing edges).
The most common method for rooting trees is the use of an uncontroversial
outgroupâclose enough to allow inference from trait data or molecular
sequencing, but far enough to be a clear outgroup.
**** Unrooted tree[edit] ****
An unrooted phylogenetic tree for myosin, a superfamily of proteins.[3]
Unrooted trees illustrate the relatedness of the leaf nodes without making
assumptions about ancestry. They do not require the ancestral root to be known
or inferred.[4] Unrooted trees can always be generated from rooted ones by
simply omitting the root. By contrast, inferring the root of an unrooted tree
requires some means of identifying ancestry. This is normally done by including
an outgroup in the input data so that the root is necessarily between the
outgroup and the rest of the taxa in the tree, or by introducing additional
assumptions about the relative rates of evolution on each branch, such as an
application of the molecular_clock hypothesis.[5]
**** Bifurcating tree[edit] ****
Both rooted and unrooted phylogenetic trees can be either bifurcating or
multifurcating, and either labeled or unlabeled. A rooted bifurcating tree has
exactly two descendants arising from each interior_node (that is, it forms a
binary_tree), and an unrooted bifurcating tree takes the form of an unrooted
binary_tree, a free_tree with exactly three neighbors at each internal node. In
contrast, a rooted multifurcating tree may have more than two children at some
nodes and an unrooted multifurcating tree may have more than three neighbors at
some nodes. A labeled tree has specific values assigned to its leaves, while an
unlabeled tree, sometimes called a tree shape, defines a topology only. The
number of possible trees for a given number of leaf nodes depends on the
specific type of tree, but there are always more multifurcating than
bifurcating trees, more labeled than unlabeled trees, and more rooted than
unrooted trees. The last distinction is the most biologically relevant; it
arises because there are many places on an unrooted tree to put the root. For
labeled bifurcating trees, there are:
         ( 2 n &#x2212; 3 ) ! ! =    ( 2 n &#x2212; 3 ) !    2  n &#x2212; 2
      ( n &#x2212; 2 ) !       for   n &#x2265; 2   {\displaystyle (2n-3)!!=
      {\frac {(2n-3)!}{2^{n-2}(n-2)!}}\,\,{\text{for}}\,n\geq 2}  [
      {\displaystyle (2n-3)!!={\frac {(2n-3)!}{2^{n-2}(n-2)!}}\,\,{\text
      {for}}\,n\geq 2}]
total rooted trees and
         ( 2 n &#x2212; 5 ) ! ! =    ( 2 n &#x2212; 5 ) !    2  n &#x2212; 3
      ( n &#x2212; 3 ) !       for   n &#x2265; 3   {\displaystyle (2n-5)!!=
      {\frac {(2n-5)!}{2^{n-3}(n-3)!}}\,\,{\text{for}}\,n\geq 3}  [
      {\displaystyle (2n-5)!!={\frac {(2n-5)!}{2^{n-3}(n-3)!}}\,\,{\text
      {for}}\,n\geq 3}]
total unrooted trees, where     n   {\displaystyle n}  [n] represents the
number of leaf nodes. Among labeled bifurcating trees, the number of unrooted
trees with     n   {\displaystyle n}  [n] leaves is equal to the number of
rooted trees with     n &#x2212; 1   {\displaystyle n-1}  [n-1] leaves.[6]
**** Coral of life[edit] ****
The Coral of Life
Darwin[7] also mentioned that the coral may be a more suitable metaphor than
the tree. Indeed, phylogenetic_corals are useful for portraying past and
present life, and they have some advantages over trees (anastomoses allowed,
etc).[8]
**** Special tree types[edit] ****
 This section needs additional citations for verification. Please help improve_this
 article by adding_citations_to_reliable_sources. Unsourced material may be
 challenged and removed.
 Find sources: "Phylogenetic_tree" â news Â· newspapers Â· books Â· scholar Â·
 JSTOR (January 2019)(Learn_how_and_when_to_remove_this_template_message)
A spindle diagram, showing the evolution of the vertebrates at class level,
width of spindles indicating number of families. Spindle diagrams are often
used in evolutionary_taxonomy.
A highly resolved, automatically generated tree_of_life, based on completely
sequenced genomes.[9][10]
    * A dendrogram is a general name for a tree, whether phylogenetic or not,
      and hence also for the diagrammatic representation of a phylogenetic
      tree.[11]
    * A cladogram only represents a branching pattern; i.e., its branch spans
      do not represent time or relative amount of character change, and its
      internal nodes do not represent ancestors.[12]
    * A phylogram is a phylogenetic tree that has branch spans proportional to
      the amount of character change.[13]
    * A chronogram is a phylogenetic tree that explicitly represents time
      through its branch spans.[14]
    * A spindle diagram (often called a Romerogram after the American
      palaeontologist Alfred_Romer) is the representation of the evolution and
      abundance of the various taxa through time, but is not an evolutionary
      tree.
    * A Dahlgrenogram is a diagram representing a cross section of a
      phylogenetic tree
    * A phylogenetic_network is not strictly speaking a tree, but rather a more
      general graph, or a directed_acyclic_graph in the case of rooted
      networks. They are used to overcome some of the limitations inherent to
      trees.
***** Construction[edit] *****
Main article: Computational_phylogenetics
Phylogenetic trees composed with a nontrivial number of input sequences are
constructed using computational_phylogenetics methods. Distance-matrix methods
such as neighbor-joining or UPGMA, which calculate genetic_distance from
multiple_sequence_alignments, are simplest to implement, but do not invoke an
evolutionary model. Many sequence alignment methods such as ClustalW also
create trees by using the simpler algorithms (i.e. those based on distance) of
tree construction. Maximum_parsimony is another simple method of estimating
phylogenetic trees, but implies an implicit model of evolution (i.e.
parsimony). More advanced methods use the optimality_criterion of maximum
likelihood, often within a Bayesian_framework, and apply an explicit model of
evolution to phylogenetic tree estimation.[6] Identifying the optimal tree
using many of these techniques is NP-hard,[6] so heuristic search and
optimization methods are used in combination with tree-scoring functions to
identify a reasonably good tree that fits the data.
Tree-building methods can be assessed on the basis of several criteria:[15]
    * efficiency (how long does it take to compute the answer, how much memory
      does it need?)
    * power (does it make good use of the data, or is information being
      wasted?)
    * consistency (will it converge on the same answer repeatedly, if each time
      given different data for the same model problem?)
    * robustness (does it cope well with violations of the assumptions of the
      underlying model?)
    * falsifiability (does it alert us when it is not good to use, i.e. when
      assumptions are violated?)
Tree-building techniques have also gained the attention of mathematicians.
Trees can also be built using T-theory.[16]
***** Limitations of phylogenetic analysis[edit] *****
 This section needs additional citations for verification. Please help improve_this
 article by adding_citations_to_reliable_sources. Unsourced material may be
 challenged and removed.
 Find sources: "Phylogenetic_tree" â news Â· newspapers Â· books Â· scholar Â·
 JSTOR (October 2012)(Learn_how_and_when_to_remove_this_template_message)
Although phylogenetic trees produced on the basis of sequenced genes or genomic
data in different species can provide evolutionary insight, these analyses have
important limitations. Most importantly, the trees that they generate are not
necessarily correct â they do not necessarily accurately represent the
evolutionary history of the included taxa. As with any scientific result, they
are subject to falsification by further study (e.g., gathering of additional
data, analyzing the existing data with improved methods). The data on which
they are based may be noisy;[17] the analysis can be confounded by genetic
recombination,[18] horizontal_gene_transfer,[19] hybridisation between species
that were not nearest neighbors on the tree before hybridisation takes place,
convergent_evolution, and conserved_sequences.
Also, there are problems in basing an analysis on a single type of character,
such as a single gene or protein or only on morphological analysis, because
such trees constructed from another unrelated data source often differ from the
first, and therefore great care is needed in inferring phylogenetic
relationships among species. This is most true of genetic material that is
subject to lateral gene transfer and recombination, where different haplotype
blocks can have different histories. In these types of analysis, the output
tree of a phylogenetic analysis of a single gene is an estimate of the gene's
phylogeny (i.e. a gene tree) and not the phylogeny of the taxa (i.e. species
tree) from which these characters were sampled, though ideally, both should be
very close. For this reason, serious phylogenetic studies generally use a
combination of genes that come from different genomic sources (e.g., from
mitochondrial or plastid vs. nuclear genomes)[20], or genes that would be
expected to evolve under different selective regimes, so that homoplasy (false
homology) would be unlikely to result from natural selection.
When extinct species are included as terminal_nodes in an analysis (rather
than, for example, to constrain internal nodes), they are considered not to
represent direct ancestors of any extant species. Extinct species do not
typically contain high-quality DNA.
The range of useful DNA materials has expanded with advances in extraction and
sequencing technologies. Development of technologies able to infer sequences
from smaller fragments, or from spatial patterns of DNA degradation products,
would further expand the range of DNA considered useful.
Phylogenetic trees can also be inferred from a range of other data types,
including morphology, the presence or absence of particular types of genes,
insertion and deletion events â and any other observation thought to contain
an evolutionary signal.
Phylogenetic_networks are used when bifurcating trees are not suitable, due to
these complications which suggest a more reticulate evolutionary history of the
organisms sampled.
***** See also[edit] *****
    * [icon]Evolutionary_biology_portal
    * Cladistics
    * Comparative_phylogenetics
    * Computational_phylogenetics
    * Evolutionary_taxonomy
    * Evolutionary_biology
    * Generalized_tree_alignment
    * List_of_phylogenetics_software
    * List_of_phylogenetic_tree_visualization_software
    * Phylogenetic_comparative_methods
***** References[edit] *****
   1. ^Bailly, Anatole (1981-01-01). AbrÃ©gÃ© du dictionnaire grec franÃ§ais.
      Paris: Hachette. ISBN 978-2010035289. OCLC 461974285.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Bailly, Anatole. "Greek-french_dictionary_online". www.tabularium.be.
      Archived from the original on April 21, 2014. Retrieved March 2, 2018.
   4. ^Hodge T, Cope M (1 October 2000). "A_myosin_family_tree". J Cell Sci.
      113 (19): 3353â4. PMID 10984423. Archived from the original on 30
      September 2007.
   5. ^""Tree"_Facts:_Rooted_versus_Unrooted_Trees". Archived from the original
      on 2014-04-14. Retrieved 2014-05-26.
   6. ^Maher BA (2002). "Uprooting_the_Tree_of_Life". The Scientist. 16: 18.
      Archived from the original on 2003-10-02.
   7. ^ a b c Felsenstein J. (2004). Inferring Phylogenies Sinauer Associates:
      Sunderland, MA.
   8. ^Darwin, Charles (1837). Notebook B. p. 25.
   9. ^  Podani J. (2019). The Coral of Life. Evolutionary Biology 46: 123-144.
      https://doi.org/10.1007/s11692-019-09474-w
  10. ^Letunic, Ivica; Bork,_Peer (1 January 2007). "Interactive_Tree_Of_Life_
      (iTOL):_an_online_tool_for_phylogenetic_tree_display_and_annotation"
      (PDF). Bioinformatics. 23 (1): 127â128. doi:10.1093/bioinformatics/
      btl529. ISSN 1367-4803. PMID 17050570. Archived (PDF) from the original
      on November 29, 2015. Retrieved 2015-07-21.
  11. ^Ciccarelli, F. D.; Doerks, T.; Von Mering, C.; Creevey, C. J.; Snel, B.;
      Bork, P. (2006). "Toward_automatic_reconstruction_of_a_highly_resolved
      tree_of_life" (PDF). Science. 311 (5765): 1283â1287. Bibcode:
      2006Sci...311.1283C. CiteSeerX 10.1.1.381.9514. doi:10.1126/
      science.1123061. PMID 16513982.
  12. ^Fox, Emily. "The_dendrogram". coursea. coursea. Archived from the
      original on 28 September 2017. Retrieved 28 September 2017.
  13. ^ Mayr, Ernst (2009)"Cladistic analysis or cladistic classification?".
      Journal of Zoological Systematics and Evolutionary Research. 12:
      94â128. doi:10.1111/j.1439-0469.1974.tb00160.x..
  14. ^ Antonio Soares, Ricardo RÃ¢belo, Alexandre Delbem, Optimization based
      on phylogram analysis, In Expert Systems with Applications, Volume 78,
      2017, Pages 32-50, ISSN 0957-4174, https://doi.org/10.1016/
      j.eswa.2017.02.012.
  15. ^Santamaria, R.; Theron, R. (2009-05-26). "Treevolution: visual analysis
      of phylogenetic trees". Bioinformatics. 25 (15): 1970â1971. doi:
      10.1093/bioinformatics/btp333. PMID 19470585.
  16. ^Penny, D.; Hendy, M. D.; Steel,_M._A. (1992). "Progress with methods for
      constructing evolutionary trees". Trends in Ecology and Evolution. 7 (3):
      73â79. doi:10.1016/0169-5347(92)90244-6. PMID 21235960.
  17. ^ A. Dress, K. T. Huber, and V. Moulton. 2001. Metric Spaces in Pure and
      Applied Mathematics. Documenta Mathematica LSU 2001: 121-139
  18. ^Townsend JP, Su Z, Tekle Y (2012). "Phylogenetic Signal and Noise:
      Predicting the Power of a Data Set to Resolve Phylogeny". Genetics. 61
      (5): 835â849. doi:10.1093/sysbio/sys036. PMID 22389443.
  19. ^Arenas M, Posada D (2010). "The_effect_of_recombination_on_the
      reconstruction_of_ancestral_sequences". Genetics. 184 (4): 1133â1139.
      doi:10.1534/genetics.109.113423. PMC 2865913. PMID 20124027.
  20. ^Woese C (2002). "On_the_evolution_of_cells". Proc Natl Acad Sci USA. 99
      (13): 8742â7. Bibcode:2002PNAS...99.8742W. doi:10.1073/pnas.132266999.
      PMC 124369. PMID 12077305.
  21. ^Parhi J., Tripathy P.S., Priyadarshi, H., Mandal S.C., Pandey P.K.
      (2019). "Diagnosis of mitogenome for robust phylogeny: A case of
      Cypriniformes fish group". Gene. 713: 143967. doi:10.1016/
      j.gene.2019.143967.CS1 maint: Multiple names: authors list (link)
***** Further reading[edit] *****
    * Schuh, R. T. and A. V. Z. Brower. 2009. Biological Systematics:
      principles and applications (2nd edn.)
ISBN 978-0-8014-4799-0
Manuel_Lima, The Book of Trees: Visualizing Branches of Knowledge, 2014,
Princeton Architectural Press, New York.
MEGA, a free software to draw phylogenetic trees.
***** External links[edit] *****
 Wikimedia Commons has media related to Phylogenetic_tree.
**** Images[edit] ****
    * Human_Y-Chromosome_2002_Phylogenetic_Tree
    * iTOL:_Interactive_Tree_Of_Life
    * Phylogenetic_Tree_of_Artificial_Organisms_Evolved_on_Computers
    * Miyamoto_and_Goodman's_Phylogram_of_Eutherian_Mammals
**** General[edit] ****
    * An overview of different methods of tree visualization is available
      atPage, R. D. M. (2011). "Space, time, form: Viewing the Tree of Life".
      Trends in Ecology & Evolution. 27 (2): 113â120. doi:10.1016/
      j.tree.2011.12.002. PMID 22209094.
Discover_Life An interactive tree based on the U.S. National Science
Foundation's Assembling the Tree of Life Project
PhyloCode
A_Multiple_Alignment_of_139_Myosin_Sequences_and_a_Phylogenetic_Tree
Tree_of_Life_Web_Project
Phylogenetic_inferring_on_the_T-REX_server
NCBI's_Taxonomy_Database[1]
ETE:_A_Python_Environment_for_Tree_Exploration This is a programming library to
analyze, manipulate and visualize phylogenetic trees. Ref.
A_daily-updated_tree_of_(sequenced)_lifeFang, H.; Oates, M. E.; Pethica, R. B.;
Greenwood, J. M.; Sardar, A. J.; Rackham, O. J. L.; Donoghue, P. C. J.;
Stamatakis, A.; De Lima Morais, D. A.; Gough, J. (2013). "A_daily-updated_tree
of_(sequenced)_life_as_a_reference_for_genome_research". Scientific Reports. 3:
2015. Bibcode:2013NatSR...3E2015F. doi:10.1038/srep02015. PMC 6504836.
PMID 23778980.
    * v
    * t
    * e
Phylogenetics
                      * Computational
                        phylogenetics
                      * Molecular_phylogenetics
Relevant fields       * Cladistics
                      * Taxonomy
                      * Evolutionary_taxonomy
                      * Systematics
                      * Phylogenesis
                            o Cladogenesis
                      * Phylogenetic tree
                            o Cladogram
Basic concepts        * Phylogenetic_network
                      * Long_branch_attraction
                      * Clade vs Grade
                      * Lineage
                            o Ghost_lineage
                      * Ghost_population
                      * Maximum_parsimony
                      * Probabilistic methods
                            o Maximum_likelihood
                            o Bayesian_inference
Inference methods     * Distance-matrix_methods
                            o Neighbor-joining
                            o UPGMA
                            o Least_squares           * [icon]Evolutionary
                      * Three-taxon_analysis            biology_portal
                      * PhyloCode
                      * DNA_barcoding
                      * Molecular_phylogenetics
                      * Phylogenetic_comparative
Current topics          methods
                      * Phylogenetic_niche
                        conservatism
                      * Phylogenetics_software
                      * Phylogenomics
                      * Phylogeography
                      * Primitive
                            o Plesiomorphy
                            o Symplesiomorphy
Group traits          * Derived
                            o Apomorphy
                            o Synapomorphy
                            o Autapomorphy
                      * Monophyly
Group types           * Paraphyly
                      * Polyphyly
                      * Phylogenetic_nomenclature
                      * Crown_group
Nomenclature          * Sister_group
                      * Basal
                      * Supertree
    * [Category] Category
    * [Commons page] Commons
    * v
    * t
    * e
Origin_of_life
               * Common_descent
               * Earliest_known_life_forms
Concepts       * Last_universal_ancestor
               * Protocell
               * Symbiogenesis
               * Initial origin (Abiogenesis):
                 Clay_hypothesis
               * Ironâsulfur_world
                     o Primordial_sandwich
               * Oparin-Haldane_hypothesis
                     o Coacervation
                     o MillerâUrey_experiment     * [icon]Evolutionary
Hypotheses           o Primordial_soup                   biology_portal
               * PAH_world
               * Panspermia
                     o Cosmic_ancestry
               * Peptide-RNA_world
               * Quasispecies_model
               * RNA_world
               * Ongoing origin: Biogenesis
               * Spontaneous_generation
               * Astrobiology
Research       * Evolutionary_biology
               * Paleobiology
    * [Tree_of_life.svg]Evolutionary_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Phylogenetic_tree&oldid=908097590"
Categories:
    * Phylogenetics
    * Tree_of_life_(biology)
    * Trees_(data_structures)
Hidden categories:
    * CS1_maint:_Multiple_names:_authors_list
    * Articles_with_short_description
    * Articles_needing_additional_references_from_January_2019
    * All_articles_needing_additional_references
    * Articles_needing_additional_references_from_October_2012
    * Commons_category_link_is_on_Wikidata
    * Commons_category_link_is_on_Wikidata_using_P373
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * EspaÃ±ol
    * Esperanto
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * Lingua_Franca_Nova
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Tagalog
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 27 July 2019, at 12:07 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
