The following text has been accessed from https://en.wikipedia.org/wiki/Gene%E2%80%93environment_interaction at Fri Aug 9 04:03:06 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Geneâenvironment interaction ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Response to the same environmental variation differently by different genotypes
"GXE" redirects here. For the aircraft, see Waco_GXE.
Geneâenvironment interaction (or genotypeâenvironment interaction or GxE or
GÃE) is when two different genotypes respond to environmental variation in
different ways. A norm_of_reaction is a graph that shows the relationship
between genes and environmental_factors when phenotypic differences are
continuous.[1] They can help illustrate GxE interactions. When the norm of
reaction is not parallel, as shown in the figure below, there is a gene by
environment interaction. This indicates that each genotype responds to
environmental variation in a different way. Environmental variation can be
physical, chemical, biological, behavior patterns or life events.[2]
This norm of reaction shows lines that are not parallel indicating a gene by
environment interaction. Each genotype is responding to environmental variation
in a different way.
Geneâenvironment interactions are studied to gain a better understanding of
various phenomena. In genetic_epidemiology, geneâenvironment interactions are
useful for understanding some diseases. Sometimes, sensitivity to environmental
risk_factors for a disease are inherited rather than the disease itself being
inherited. Individuals with different genotypes are affected differently by
exposure to the same environmental factors, and thus geneâenvironment
interactions can result in different disease phenotypes. For example, sunlight
exposure has a stronger influence on skin_cancer risk in fair-skinned_humans
than_in_individuals_with_darker_skin.[3]
These interactions are of particular interest to genetic_epidemiologists for
predicting disease rates and methods of prevention with respect to public
health.[2] The term is also used amongst developmental_psychobiologists to
better understand individual and evolutionary development.[4]
Nature_versus_nurture debates assume that variation in a trait is primarily due
to either genetic differences or environmental differences. However, the
current scientific opinion holds that neither genetic differences nor
environmental differences are solely responsible for producing phenotypic
variation, and that virtually all traits are influenced by both genetic and
environmental differences.[5][6][7]
Statistical_analysis of the genetic and environmental differences contributing
to the phenotype would have to be used to confirm these as geneâenvironment
interactions. In developmental genetics, a causal interaction is enough to
confirm geneâenvironment interactions.[8]
⁰
***** Contents *****
    * 1_History_of_the_definition
    * 2_Definitions
    * 3_Epidemiological_models_of_GxE
    * 4_Methods_of_analysis
          o 4.1_Traditional_genetic_designs
                # 4.1.1_Adoption_studies
                # 4.1.2_Twin_studies
                # 4.1.3_Family_studies
          o 4.2_Molecular_analyses
                # 4.2.1_Interaction_with_single_genes
                # 4.2.2_Interaction_with_multiple_genes
                # 4.2.3_Genome-wide_association_studies_and_genome_wide
                  interaction_studies
    * 5_Controversies
          o 5.1_Lack_of_replication
          o 5.2_Additive_vs_multiplicative_model
    * 6_Gene_Ã_environment_Ã_environment_interactions
    * 7_Medical_significance
    * 8_Examples
    * 9_See_also
    * 10_References
***** History of the definition[edit] *****
The history of defining geneâenvironment interaction dates back to the 1930s
and remains a topic of debate today. The first instance of debate occurred
between Ronald_Fisher and Lancelot_Hogben. Galat h Fisher sought to eliminate
interaction from statistical studies as it was a phenomenon that could be
removed using a variation in scale. Hogben believed that the interaction should
be investigated instead of eliminated as it provided information on the
causation of certain elements of development.
A similar argument faced multiple scientists in the 1970s. Arthur_Jensen
published the study âHow_much_can_we_boost_IQ_and_scholastic_achievement?â,
which amongst much criticism also faced contention by scientists Richard
Lewontin and David_Layzer. Lewontin and Layzer argued that in order to conclude
causal mechanisms, the geneâenvironment interaction could not be ignored in
the context of the study while Jensen defended that interaction was purely a
statistical phenomenon and not related to development.[9]
Around the same time, Kenneth J. Rothman supported the use of a statistical
definition for interaction while researchers Kupper and Hogan believed the
definition and existence of interaction was dependent on the model being used.
[10]
The most recent criticisms were spurred by Moffitt and Caspi's studies on 5-
HTTLPR and stress and its influence on depression. In contrast to previous
debates, Moffitt and Caspi were now using the statistical analysis to prove
that interaction existed and could be used to uncover the mechanisms of a
vulnerability trait. Contention came from Zammit, Owen and Lewis who reiterated
the concerns of Fisher in that the statistical effect was not related to the
developmental process and would not be replicable with a difference of scale.
[9]
***** Definitions[edit] *****
There are two different conceptions of geneâenvironment interaction today.
Tabery[11] has labeled them biometric and developmental interaction, while
Sesardic[12] uses the terms statistical and commonsense interaction.
The biometric (or statistical) conception has its origins in research programs
that seek to measure the relative proportions of genetic and environmental
contributions to phenotypic variation within populations. Biometric
geneâenvironment interaction has particular currency in population_genetics
and behavioral_genetics.[11] Any interaction results in the breakdown of the
additivity of the main_effects of heredity and environment, but whether such
interaction is present in particular settings is an empirical question.
Biometric interaction is relevant in the context of research on individual
differences rather than in the context of the development of a particular
organism.[4]
Developmental geneâenvironment interaction is a concept more commonly used by
developmental_geneticists and developmental_psychobiologists. Developmental
interaction is not seen merely as a statistical phenomenon. Whether statistical
interaction is present or not, developmental interaction is in any case
manifested in the causal interaction of genes and environments in producing an
individual's phenotype.[4]
***** Epidemiological models of GxE[edit] *****
In epidemiology, the following models can be used to group the different
interactions between gene and environment.
Model A describes a genotype that increases the level of expression of a risk
factor but does not cause the disease itself. For example, the PKU gene results
in higher levels of phenylalanine than normal which in turn causes mental
retardation.
The risk factor in Model B in contrast has a direct effect on disease
susceptibility which is amplified by the genetic susceptibility. Model C
depicts the inverse, where the genetic susceptibility directly effects disease
while the risk factor amplifies this effect. In each independent situation, the
factor directly effecting the disease can cause disease by itself.
Model D differs as neither factor in this situation can effect disease risk,
however, when both genetic susceptibility and risk factor are present the risk
is increased. For example, the G6PD deficiency gene when combined with fava
bean consumption results in hemolytic anemia. This disease does not arise in
individuals that eat fava beans and lack G6PD deficiency nor in G6PD-deficient
people who do not eat fava beans.
Lastly, Model E depicts a scenario where the environmental risk factor and
genetic susceptibility can individually both influence disease risk. When
combined, however, the effect on disease risk differs.
The models are limited by the fact that the variables are binary and so do not
consider polygenic or continuous scale variable scenarios.[2]
***** Methods of analysis[edit] *****
**** Traditional genetic designs[edit] ****
*** Adoption studies[edit] ***
Adoption studies have been used to investigate how similar individuals that
have been adopted are to their biological parents with whom they did not share
the same environment with. Additionally, adopted individuals are compared to
their adoptive family due to the difference in genes but shared environment.
For example, an adoption study showed that Swedish men with disadvantaged
adoptive environments and a genetic predisposition were more likely to abuse
alcohol.[13]
*** Twin studies[edit] ***
Using monozygotic_twins, the effects of different environments on identical
genotypes could be observed. Later studies leverage biometrical modelling
techniques to include the comparisons of dizygotic twins to ultimately
determine the different levels of gene expression in different environments.
[13]
*** Family studies[edit] ***
Family-based research focuses on the comparison of low-risk controls to high
risk children to determine the environmental effect on subjects with different
levels of genetic risk. For example, a Danish study on high-risk children with
schizophrenic mothers depicted that children without a stable caregiver were
associated with an increased risk of schizophrenia.[13]
**** Molecular analyses[edit] ****
*** Interaction with single genes[edit] ***
The often used method to detect geneâenvironment interactions is by studying
the effect a single gene variation (candidate_gene) has with respect to a
particular environment. Single nucleotide polymorphisms (SNPâs) are compared
with single binary exposure factors to determine any effects.
Candidate studies such as these require strong biological hypotheses which are
currently difficult to select given the little understanding of biological
mechanisms that lead to higher risk.
These studies are also often difficult to replicate commonly due to small
sample sizes which typically results in disputed results.
The polygenic nature of complex phenotypes suggests single candidate studies
could be ineffective in determining the various smaller scale effects from the
large number of influencing gene variants.[14]
*** Interaction with multiple genes[edit] ***
Since the same environmental factor could interact with multiple genes, a
polygenic approach can be taken to analyze GxE interactions. A polygenic_score
is generated using the alleles associated with a trait and their respective
weights based on effect and examined in combination with environmental
exposure. Though this method of research is still early, it is consistent with
psychiatric disorders. As a result of the overlap of endophenotypes amongst
disorders this suggests that the outcomes of geneâenvironment interactions
are applicable across various diagnoses.[14]
*** Genome-wide association studies and genome wide interaction studies[edit]
***
A genome wide interaction scan (GEWIS) approach examines the interaction
between the environment and a large number of independent SNPâs. An effective
approach to this all-encompassing study occurs in two-steps where the genome is
first filtered using gene-level tests and pathway based gene set analyses. The
second step uses the SNPâs with GâE association and tests for interaction.
[15]
The differential susceptibility hypothesis has been reaffirmed through genome
wide approaches.[16]
***** Controversies[edit] *****
**** Lack of replication[edit] ****
A particular concern with geneâenvironment interaction studies is the lack of
reproducibility. Specifically complex_traits studies have come under scrutiny
for producing results that cannot be replicated. For example, studies of the 5-
HTTLPR gene and stress resulting in modified risk of depression have had
conflicting results.[17][15]
A possible explanation behind the inconsistent results is the heavy use of
multiple testing. Studies are suggested to produce inaccurate results due to
the investigation of multiple phenotypes and environmental factors in
individual experiments.[15]
**** Additive vs multiplicative model[edit] ****
There are two different models for the scale of measurement that helps
determine if geneâenvironment interaction exists in a statistical context.
There is disagreement on which scale should be used. Under these analyses, if
the combined variables fit either model then there is no interaction. The
combined effects must either be greater for synergistic or less than for an
antagonistic outcome. The additive model measures risk differences while the
multiplicative model uses ratios to measure effects. The additive model has
been suggested to be a better fit for predicting disease risk in a population
while a multiplicative model is more appropriate for disease etiology.[2]
Epigenetics is an example of an underlying mechanism of geneâenvironment
effects, however, it does not conclude whether environment effects are
additive, multiplicative or interactive.[13]
***** Gene Ã environment Ã environment interactions[edit] *****
New studies have also revealed the interactive effect of multiple environment
factors. For example, a child with a poor quality environment would be more
sensitive to a poor environment as an adult which ultimately led to higher
psychological distress scores. This depicts a three way interaction Gene x
Environment x Environment. The same study suggests taking a life course
approach to determining genetic sensitivity to environmental influences within
the scope of mental illnesses.[18]
***** Medical significance[edit] *****
Doctors are interested in knowing whether disease can be prevented by reducing
exposure to environmental risks. Some people carry genetic factors that confer
susceptibility or resistance to a certain disorder in a particular environment.
The interaction between the genetic factors and environmental stimulus is what
results in the disease phenotype.[19] There may be significant public_health
benefits in using gene by environment interactions to prevent or cure disease.
[20]
An individualâs response to a drug can result from various gene by
environment interactions.[19] Therefore, the clinical importance of
pharmacogenetics and gene by environment interactions comes from the
possibility that genomic, along with environmental information, will allow more
accurate predictions of an individualâs drug response. This would allow
doctors to more precisely select a certain drug and dosage to achieve
therapeutic response in a patient while minimizing side effects and adverse
drug_reactions.[21] This information could also help to prevent the health care
costs associated with adverse drug reactions and inconveniently prescribing
drugs to patients who likely wonât respond to them.[19]
In a similar manner, an individual can respond to other environmental stimuli,
factors or challenges differently according to specific genetic differences or
alleles. These other factors include the diet and specific nutrients within the
diet, physical activity, alcohol and tobacco use, sleep (bed time, duration),
and any of a number of exposures (or exposome), including toxins, pollutants,
sunlight (latitude north/south of the equator), among any number of others. The
diet, for example, is modifiable and has significant impact on a host of
cardiometabolic diseases, including cardiovascular disease, coronary artery
disease, coronary heart disease, type_2_diabetes, hypertension, stroke,
myocardial_infarction, and non-alcoholic fatty liver disease. In the clinic,
typically assessed risks of these conditions include blood lipids
(triglyceride, and HDL, LDL and total cholesterol), glycemic traits (plasma
glucose and insulin, HOMA-IR, beta cell function as HOMA-BC), obesity
anthropometrics (BMI/obesity, adiposity, body weight, waist circumference,
waist-to-hip ratio), vascular measures (diastolic and systolic blood pressure),
and biomarkers of inflammation. Geneâenvironment interactions can modulate
the adverse effects of an allele that confers increased risk of disease, or can
exacerbate the genotypeâphenotype relationship and increase risk, in a manner
often referred to as nutrigenetics.[22] A catalog of genetic variants that
associate with these and related cardiometabolic phenotypes and modified by
common environmental factors is available.[23]
Conversely, a disease study using breast cancer, type 2 diabetes, and
rheumatoid arthritis shows that including GxE interactions in a risk prediction
model does not improve risk identification.[24]
***** Examples[edit] *****
Mean bristle number by Â°C
   1. In Drosophila: A classic example of geneâenvironment interaction was
      performed on Drosophila by Gupta and Lewontin in 1981. In their
      experiment they demonstrated that the mean bristle number on Drosophila
      could vary with changing temperatures. As seen in the graph to the right,
      different genotypes reacted differently to the changing environment. Each
      line represents a given genotype, and the slope of the line reflects the
      changing phenotype (bristle number) with changing temperature. Some
      individuals had an increase in bristle number with increasing temperature
      while others had a sharp decrease in bristle number with increasing
      temperature. This showed that the norms of reaction were not parallel for
      these flies, proving that geneâenvironment interactions exist.[25]
   2. In plants: One very interesting approach about genotype by environment
      interaction strategies is its use in the selection of sugarcane cultivars
      adapted to different environments[26]. In this article, they analyzed
      twenty sugarcane genotypes grown in eight different locations over two
      crop cycles to identify mega-environments related to higher cane yield,
      measured in tons of cane per hectare (TCH) and percentage of sucrose
      (Pol% cane) using biplot multivariate GEI models. The authors then
      created a novel strategy to study both yield variables in a two-way
      coupled strategy even though the results showed a mean negative
      correlation. Through coinertia analysis, it was possible to determine the
      best-fitted genotypes for both yield variables in all environments[27].
      The use of these novel strategies like coinertia in GEI, proved to be a
      great complement analysis to AMMI and GGE, especially when the yield
      improvement implies multiple yield variables. Seven genetically distinct
      yarrow plants were collected and three cuttings taken from each plant.
      One cutting of each genotype was planted at low, medium, and high
      elevations, respectively. When the plants matured, no one genotype grew
      best at all altitudes, and at each altitude the seven genotypes fared
      differently. For example, one genotype grew the tallest at the medium
      elevation but attained only middling height at the other two elevations.
      The best growers at low and high elevation grew poorly at medium
      elevation. The medium altitude produced the worst overall results, but
      still yielded one tall and two medium-tall samples. Altitude had an
      effect on each genotype, but not to the same degree nor in the same way.
      [28] A sorghum bi-parental population was repeatedly grown in seven
      diverse geographic locations across years. A group of genotypes requires
      similar growing_degree-day (GDD) to flower across all environments, while
      another group of genotypes need less GDD in certain environments, but
      higher GDD in different environments to flower. The complex flowering
      time patterns is attributed to the interaction of major flowering time
      genes (Ma1,[29] Ma6,[30] FT, ELF3) and an explicit environmental factor,
      photothermal_time (PTT) capturing the interaction between temperature and
      photoperiod.[31]
   3. Phenylketonuria (PKU) is a human genetic_condition caused by mutations to
      a gene coding for a particular liver enzyme. In the absence of this
      enzyme, an amino_acid known as phenylalanine does not get converted into
      the next amino acid in a biochemical_pathway, and therefore too much
      phenylalanine passes into the blood and other tissues. This disturbs
      brain_development leading to mental_retardation and other problems. PKU
      affects approximately 1 out of every 15,000 infants in the U.S. However,
      most affected infants do not grow up impaired because of a standard
      screening program used in the U.S. and other industrialized societies.
      Newborns found to have high levels of phenylalanine in their blood can be
      put on a special, phenylalanine-free diet. If they are put on this diet
      right away and stay on it, these children avoid the severe effects of
      PKU.[32] This example shows that a change in environment (lowering
      Phenylalanine consumption) can affect the phenotype of a particular
      trait, demonstrating a geneâenvironment interaction.
   4. A single_nucleotide_polymorphism rs1800566 in NAD(P)H Quinone
      Dehydrogenase 1 (NQO1) alters the risk of asthma and general lung injury
      upon interaction with NOx pollutants, in individuals with this mutation.
      [33][34]
   5. A functional polymorphism in the monoamine_oxidase_A (MAOA) gene promoter
      can moderate the association between early life trauma and increased risk
      for violence and antisocial_behavior. Low MAOA activity is a significant
      risk factor for aggressive and antisocial behavior in adults who report
      victimization as children. Persons who were abused as children but have a
      genotype conferring high levels of MAOA expression are less likely to
      develop symptoms of antisocial behavior.[35] These findings must be
      interpreted with caution, however, because gene association studies on
      complex traits are notorious for being very difficult to confirm.[36]
   6. In Drosophila eggs:
      Egg Development Time by Temperature
      Contrary to the aforementioned examples, length of egg development in
      Drosophila as a function of temperature demonstrates the lack of
      geneâenvironment interactions. The attached graph shows parallel
      reaction norms for a variety of individual Drosophila flies, showing that
      there is not a geneâenvironment interaction present between the two
      variables. In other words, each genotype responds similarly to the
      changing environment producing similar phenotypes. For all individual
      genotypes, average egg development time decreases with increasing
      temperature. The environment is influencing each of the genotypes in the
      same predictable manner.[25]
***** See also[edit] *****
    * Biopsychosocial_model
    * Diathesisâstress_model
    * Differential_susceptibility
    * Envirome
    * Epidemiology
    * Epigenetics
    * Evolutionary_developmental_psychology
    * Exposome
    * Geneâenvironment_correlation
    * Genetic_epidemiology
    * Genomics
    * Molecular_epidemiology
    * Molecular_pathological_epidemiology
    * Molecular_pathology
***** References[edit] *****
   1. ^Krebs JR (April 2, 2012). An Introduction to Behavioural Ecology.
      Oxford: Wiley-Blackwell. ISBN 978-1405114165.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a b c dOttman R (1996). "Geneâenvironment_interaction:_definitions
      and_study_designs". Preventive Medicine. 25 (6): 764â70. doi:10.1006/
      pmed.1996.0117. PMC 2823480. PMID 8936580.
   4. ^Green A, Trichopoulos D (2002). "Skin cancer". In Adami H, Hunter D,
      Trichopoulos D (eds.). Textbook of Cancer Epidemiology. Oxford: Oxford
      University Press. pp. 281â300.
   5. ^ a b cTabery J, Griffiths PE (2010). "Historical and Philosophical
      Perspectives on Behavioral Genetics and Developmental Science". In Hood
      KE, Halpern CT, Greenberg G, Lerner RM (eds.). Handbook of Developmental
      Science, Behavior, and Genetics. Wiley-Blackwell. pp. 41â60.
   6. ^ Ridley,_M. (2003) Nature_via_Nurture:_Genes,_Experience,_&_What_Makes
      Us_Human. Harper Collins.
   7. ISBN 0-00-200663-4
   8. ^ Rutter, Michael. (2006) Genes and Behavior: NatureâNurture Interplay
      Explained Oxford, UK: Blackwell Publishers
   9. ^Cuhna F, Heckman JJ (2010). "Chapter 18: Cost-effective Early Childhood
      Programs in the First Decade: A Human Capital Integration". In Reynolds
      AJ, Rolnick A, Englund MM, Temple J (eds.). Investing in Our Young
      People. New York: Cambridge University Press. pp. 381â414.
  10. ^Tabery J, Griffiths PE (2010). Hood KE, Halpern CT, Greenberg G, Lerner
      RM (eds.). Handbook of Developmental Science, Behavior, and Genetics.
      Wiley-Blackwell. pp. 39â60. doi:10.1002/9781444327632.ch3.
      ISBN 9781444327632.
  11. ^ a bTabery J (August 2015). "Debating interaction: the history, and an
      explanation". International Journal of Epidemiology. 44 (4): 1117â23.
      doi:10.1093/ije/dyv053. PMID 25855719.
  12. ^Rothman KJ, Greenland S, Walker AM (October 1980). "Concepts of
      interaction". American Journal of Epidemiology. 112 (4): 467â70. doi:
      10.1093/oxfordjournals.aje.a113015. PMID 7424895.
  13. ^ a bTabery J (2007). "Biometric and developmental geneâenvironment
      interactions: looking back, moving forward". Development and
      Psychopathology. 19 (4): 961â76. doi:10.1017/s0954579407000478.
      PMID 17931428.
  14. ^ Sesardic, N. (2005). Making sense of heritability. Cambridge: Cambridge
      University Press, p. 48.
  15. ^ a b c dDick DM (2011). "Geneâenvironment_interaction_in_psychological
      traits_and_disorders". Annual Review of Clinical Psychology. 7:
      383â409. doi:10.1146/annurev-clinpsy-032210-104518. PMC 3647367.
      PMID 21219196.
  16. ^ a bAssary E, Vincent JP, Keers R, Pluess M (May 2018).
      "Geneâenvironment interaction and psychiatric disorders: Review and
      future directions". Seminars in Cell & Developmental Biology. 77:
      133â143. doi:10.1016/j.semcdb.2017.10.016. PMID 29051054.
  17. ^ a b cWinham SJ, Biernacka JM (October 2013). "Geneâenvironment
      interactions_in_genome-wide_association_studies:_current_approaches_and
      new_directions". Journal of Child Psychology and Psychiatry, and Allied
      Disciplines. 54 (10): 1120â34. doi:10.1111/jcpp.12114. PMC 3829379.
      PMID 23808649.
  18. ^Keers R, Coleman JR, Lester KJ, Roberts S, Breen G, Thastum M, BÃ¶gels
      S, Schneider S, Heiervang E, Meiser-Stedman R, Nauta M, Creswell C,
      Thirlwall K, Rapee RM, Hudson JL, Lewis C, Plomin R, Eley TC (2016). "A
      Genome-Wide_Test_of_the_Differential_Susceptibility_Hypothesis_Reveals_a
      Genetic_Predictor_of_Differential_Response_to_Psychological_Treatments
      for_Child_Anxiety_Disorders". Psychotherapy and Psychosomatics. 85 (3):
      146â58. doi:10.1159/000444023. PMC 5079103. PMID 27043157.
  19. ^Caspi A, Hariri AR, Holmes A, Uher R, Moffitt TE (May 2010). "Genetic
      sensitivity_to_the_environment:_the_case_of_the_serotonin_transporter
      gene_and_its_implications_for_studying_complex_diseases_and_traits". The
      American Journal of Psychiatry. 167 (5): 509â27. doi:10.1176/
      appi.ajp.2010.09101452. PMC 2943341. PMID 20231323.
  20. ^Grabe HJ, Schwahn C, Mahler J, Schulz A, Spitzer C, Fenske K, Appel K,
      Barnow S, Nauck M, Schomerus G, Biffar R, Rosskopf D, John U, VÃ¶lzke H,
      Freyberger HJ (April 2012). "Moderation of adult depression by the
      serotonin transporter promoter variant (5-HTTLPR), childhood abuse and
      adult traumatic events in a general population sample". American Journal
      of Medical Genetics. Part B, Neuropsychiatric Genetics. 159B (3):
      298â309. doi:10.1002/ajmg.b.32027. PMID 22328412.
  21. ^ a b cHaga SB, Burke W (June 2004). "Using pharmacogenetics to improve
      drug safety and efficacy". JAMA. 291 (23): 2869â71. doi:10.1001/
      jama.291.23.2869. PMID 15199039.
  22. ^Khoury MJ, Davis R, Gwinn M, Lindegren ML, Yoon P (May 2005). "Do we
      need genomic research for the prevention of common diseases with
      environmental causes?". American Journal of Epidemiology. 161 (9):
      799â805. doi:10.1093/aje/kwi113. PMID 15840611.
  23. ^Eichelbaum M, Ingelman-Sundberg M, Evans WE (2006). "Pharmacogenomics
      and individualized drug therapy". Annual Review of Medicine. 57:
      119â37. doi:10.1146/annurev.med.56.082103.104724. PMID 16409140.
  24. ^Ordovas JM (December 2008). "Genotypeâphenotype_associations:
      modulation_by_diet_and_obesity". Obesity. 16 Suppl 3: S40â6. doi:
      10.1038/oby.2008.515. PMC 2771769. PMID 19037211.
  25. ^Parnell LD, Blokker BA, Dashti HS, Nesbeth PD, Cooper BE, Ma Y, Lee YC,
      Hou R, Lai CQ, Richardson K, OrdovÃ¡s JM (2014). "CardioGxE,_a_catalog_of
      geneâenvironment_interactions_for_cardiometabolic_traits". BioData
      Mining. 7: 21. doi:10.1186/1756-0381-7-21. PMC 4217104. PMID 25368670.
  26. ^Aschard H, Chen J, Cornelis MC, Chibnik LB, Karlson EW, Kraft P (June
      2012). "Inclusion_of_geneâgene_and_geneâenvironment_interactions
      unlikely_to_dramatically_improve_risk_prediction_for_complex_diseases".
      American Journal of Human Genetics. 90 (6): 962â72. doi:10.1016/
      j.ajhg.2012.04.017. PMC 3370279. PMID 22633398.
  27. ^ a bGupta AP, Lewontin RC (September 1982). "A Study of Reaction Norms
      in Natural Populations of Drosophila pseudoobscura". Evolution;
      International Journal of Organic Evolution. 36 (5): 934â948. doi:
      10.1111/j.1558-5646.1982.tb05464.x. PMID 28567833.
  28. ^ Rea, R., De Sousa-Vieira, O., DÃ­az, A., RamÃ³n, M., BriceÃ±o, R.,
      George, J., ... & Balzano-Nogueira, L. (2016). GenotypeâEnvironment
      Interaction, Megaenvironments and Two-Table Coupling Methods for
      Sugarcane Yield Studies in Venezuela. Sugar Tech, 18(4), 354-364. https:/
      /link.springer.com/article/10.1007/s12355-015-0407-9.
  29. ^ Dray, S., Chessel, D., & Thioulouse, J. (2003). Coâinertia analysis
      and the linking of ecological data tables. Ecology, 84(11), 3078-3089.
      https://esajournals.onlinelibrary.wiley.com/doi/full/10.1890/03-0178.
  30. ^Clausen J, Keck D, Hiesey WM (1948). "Experimental studies on the nature
      of species. III. Environmental responses of climatic races of Achillea,
      Carnegie Inst Washington Publ 581": 1â129.
  31. ^Murphy RL, Klein RR, Morishige DT, Brady JA, Rooney WL, Miller FR, Dugas
      DV, Klein PE, Mullet JE (September 2011). "Coincident_light_and_clock
      regulation_of_pseudoresponse_regulator_protein_37_(PRR37)_controls
      photoperiodic_flowering_in_sorghum". Proceedings of the National Academy
      of Sciences of the United States of America. 108 (39): 16469â74. doi:
      10.1073/pnas.1106212108. PMC 3182727. PMID 21930910.
  32. ^Murphy RL, Morishige DT, Brady JA, Rooney WL, Yang S, Klein PE, Mullet
      JE (2014-07-01). "Ghd7 ( Ma 6 ) Represses Sorghum Flowering in Long Days:
      Ghd7 Alleles Enhance Biomass Accumulation and Grain Production". The
      Plant Genome. 7 (2): 0. doi:10.3835/plantgenome2013.11.0040. ISSN 1940-
      3372.
  33. ^Li X, Guo T, Mu Q, Li X, Yu J (June 2018). "Genomic_and_environmental
      determinants_and_their_interplay_underlying_phenotypic_plasticity".
      Proceedings of the National Academy of Sciences of the United States of
      America. 115 (26): 6679â6684. doi:10.1073/pnas.1718326115. PMC 6042117.
      PMID 29891664.
  34. ^Baker C (2004). "Chapter_3._Environment_Illustrated". Behavioral
      Genetics. AAAS. ISBN 978-0871686978.
  35. ^Castro-Giner F, KÃ¼nzli N, Jacquemin B, Forsberg B, de Cid R, Sunyer J,
      et al. (December 2009). "Traffic-related_air_pollution,_oxidative_stress
      genes,_and_asthma_(ECHRS)". Environmental Health Perspectives. 117 (12):
      1919â24. doi:10.1289/ehp.0900589. PMC 2799467. PMID 20049212.
  36. ^Basharat Z, Messaoudi A, Ruba S, Yasmin A (October 2016). "NQO1
      rs1800566 polymorph is more prone to NOx induced lung injury: Endorsing
      deleterious functionality through informatics approach". Gene. 591 (1):
      14â20. doi:10.1016/j.gene.2016.06.048. PMID 27349566.
  37. ^Caspi A, McClay J, Moffitt TE, Mill J, Martin J, Craig IW, Taylor A,
      Poulton R (August 2002). "Role of genotype in the cycle of violence in
      maltreated children". Science. 297 (5582): 851â4. doi:10.1126/
      science.1072290. PMID 12161658.
  38. ^MunafÃ² MR, Durrant C, Lewis G, Flint J (February 2009). "Gene X
      environment interactions at the serotonin transporter locus". Biological
      Psychiatry. 65 (3): 211â9. doi:10.1016/j.biopsych.2008.06.009.
      PMID 18691701.
    * v
    * t
    * e
The development of phenotype
                                 * Genotypeâphenotype_distinction
                                 * Norms_of_reaction
                                 * Geneâenvironment interaction
                                 * Geneâenvironment_correlation
Key concepts                     * Operon
                                 * Heritability
                                 * Quantitative_genetics
                                 * Heterochrony
                                       o Neoteny
                                 * Heterotopy
                                 * Canalisation
                                 * Genetic_assimilation
                                 * Dominance
                                 * Epistasis
Genetic_architecture             * Fitness_landscape/evolutionary_landscape
                                 * Pleiotropy
                                 * Plasticity
                                 * Polygenic_inheritance
                                 * Transgressive_segregation
                                 * Sequence_space
                                 * Epigenetics
                                 * Maternal_effect
Non-genetic influences           * Genomic_imprinting
                                 * Dual_inheritance_theory
                                 * Polyphenism
                                 * Developmental_biology
                                 * Morphogenesis
                                       o Eyespot
Developmental architecture             o Pattern_formation
                                 * Segmentation
                                 * Metamerism
                                 * Modularity
                                 * Evolvability
Evolution of genetic systems     * Mutational_robustness
                                 * Neutral_networks
                                 * Evolution_of_sexual_reproduction
                                          * Regulation_of_gene_expression
                                          * Gene_regulatory_network
                                          * Developmental-genetic_toolkit
                             Systems      * Evolutionary_developmental_biology
                                          * Homeobox
                                          * Hedgehog_signaling_pathway
                                          * Notch_signaling_pathway
                                          * Homeotic_gene
Control of development                    * Hox_gene
                                          * Pax_genes
                                                o eyeless_gene
                                          * Distal-less
                             Elements     * Engrailed
                                          * cis-regulatory_element
                                          * Ligand
                                          * Morphogen
                                          * Cell_surface_receptor
                                          * Transcription_factor
                                 * C._H._Waddington
                                 * Richard_Lewontin
                                 * FranÃ§ois_Jacob + Jacques_Monod
                                       o Lac_operon
Influential figures              * Eric_F._Wieschaus
                                 * Christiane_NÃ¼sslein-Volhard
                                 * William_McGinnis
                                 * Mike_Levine
                                 * Sean_B._Carroll
                                       o Endless_Forms_Most_Beautiful
Debates                          * Nature_versus_nurture
                                 * Morphogenetic_field
Index_of_evolutionary_biology_articles

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Geneâenvironment_interaction&oldid=908256167"
Categories:
    * Psychological_theories
    * Human_genetics
Hidden categories:
    * CS1:_long_volume_value
    * Articles_with_short_description
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Dansk
    * Polski
Edit_links
    * This page was last edited on 28 July 2019, at 14:49 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
