The following text has been accessed from https://en.wikipedia.org/wiki/Population_genetics at Fri Aug 9 00:08:37 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Population genetics ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Study of genetic differences within and between populations including the study
of adaptation, speciation, and population structure
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
Part of a_series on
Genetic_genealogy
Concepts
    * Population genetics
    * Haplogroup / Haplotype / Subclade
    * Most_recent_common_ancestor
    * DNA_haplogroups
      (Human_mitochondrial,
      Human_Y-chromosome)
    * Genomics
Related topics
    * Y-chromosome_haplogroups_by_population
    * Genealogical_DNA_test
    * Surname_DNA_project
    * Personal_genomics
    * Genographic_Project
    * ISOGG
    * v
    * t
    * e
Population genetics is a subfield of genetics that deals with genetic
differences within and between populations, and is a part of evolutionary
biology. Studies in this branch of biology examine such phenomena as
adaptation, speciation, and population_structure.[1]
Population genetics was a vital ingredient in the emergence of the modern
evolutionary_synthesis. Its primary founders were Sewall_Wright, J._B._S.
Haldane and Ronald_Fisher, who also laid the foundations for the related
discipline of quantitative_genetics. Traditionally a highly mathematical
discipline, modern population genetics encompasses theoretical, lab, and field
work. Population genetic models are used both for statistical_inference from
DNA sequence data and for proof/disproof of concept.[2]
What sets population genetics apart today from newer, more phenotypic
approaches to modelling evolution, such as evolutionary_game_theory and
adaptive_dynamics, is its emphasis on genetic phenomena as dominance,
epistasis, and the degree to which genetic_recombination breaks up linkage
disequilibrium. This makes it appropriate for comparison to population_genomics
data.
⁰
***** Contents *****
    * 1_History
          o 1.1_Modern_synthesis
          o 1.2_Neutral_theory_and_origin-fixation_dynamics
    * 2_Four_processes
          o 2.1_Selection
                # 2.1.1_Dominance
                # 2.1.2_Epistasis
          o 2.2_Mutation
          o 2.3_Genetic_drift
          o 2.4_Gene_flow
                # 2.4.1_Horizontal_gene_transfer
    * 3_Linkage
    * 4_Applications
          o 4.1_Explaining_levels_of_genetic_variation
          o 4.2_Detecting_selection
          o 4.3_Demographic_inference
          o 4.4_Evolution_of_genetic_systems
    * 5_See_also
    * 6_References
    * 7_External_links
***** History[edit] *****
Population genetics began as a reconciliation of Mendelian_inheritance and
biostatistics models. Natural_selection will only cause evolution if there is
enough genetic_variation in a population. Before the discovery of Mendelian
genetics, one common hypothesis was blending_inheritance. But with blending
inheritance, genetic_variance would be rapidly lost, making evolution by
natural or sexual selection implausible. The HardyâWeinberg_principle
provides the solution to how variation is maintained in a population with
Mendelian_inheritance. According to this principle, the frequencies of alleles
(variations in a gene) will remain constant in the absence of selection,
mutation, migration and genetic drift.[3]
The typical white-bodied form of the peppered_moth.
Industrial_melanism: the black-bodied form of the peppered moth appeared in
polluted areas.
The next key step was the work of the British biologist and statistician Ronald
Fisher. In a series of papers starting in 1918 and culminating in his 1930 book
The_Genetical_Theory_of_Natural_Selection, Fisher showed that the continuous
variation measured by the biometricians could be produced by the combined
action of many discrete genes, and that natural selection could change allele
frequencies in a population, resulting in evolution. In a series of papers
beginning in 1924, another British geneticist, J.B.S._Haldane, worked out the
mathematics of allele frequency change at a single gene locus under a broad
range of conditions. Haldane also applied statistical analysis to real-world
examples of natural selection, such as peppered_moth_evolution and industrial
melanism, and showed that selection_coefficients could be larger than Fisher
assumed, leading to more rapid adaptive evolution as a camouflage strategy
following increased pollution.[4][5]
The American biologist Sewall_Wright, who had a background in animal_breeding
experiments, focused on combinations of interacting genes, and the effects of
inbreeding on small, relatively isolated populations that exhibited genetic
drift. In 1932 Wright introduced the concept of an adaptive_landscape and
argued that genetic drift and inbreeding could drive a small, isolated sub-
population away from an adaptive peak, allowing natural selection to drive it
towards different adaptive peaks.
The work of Fisher, Haldane and Wright founded the discipline of population
genetics. This integrated natural selection with Mendelian genetics, which was
the critical first step in developing a unified theory of how evolution worked.
[4][5] John_Maynard_Smith was Haldane's pupil, whilst W.D._Hamilton was heavily
influenced by the writings of Fisher. The American George_R._Price worked with
both Hamilton and Maynard Smith. American Richard_Lewontin and Japanese Motoo
Kimura were heavily influenced by Wright.
**** Modern synthesis[edit] ****
Main article: Modern_synthesis_(20th_century)
The mathematics of population genetics were originally developed as the
beginning of the modern_synthesis. Authors such as Beatty[6] have asserted that
population genetics defines the core of the modern synthesis. For the first few
decades of the 20th century, most field naturalists continued to believe that
Lamarckism and orthogenesis provided the best explanation for the complexity
they observed in the living world.[7] During the modern synthesis, these ideas
were purged, and only evolutionary causes that could be expressed in the
mathematical framework of population genetics were retained.[8] Consensus was
reached as to which evolutionary factors might influence evolution, but not as
to the relative importance of the various factors.[8]
Theodosius_Dobzhansky, a postdoctoral worker in T. H. Morgan's lab, had been
influenced by the work on genetic_diversity by Russian geneticists such as
Sergei_Chetverikov. He helped to bridge the divide between the foundations of
microevolution developed by the population geneticists and the patterns of
macroevolution observed by field biologists, with his 1937 book Genetics_and
the_Origin_of_Species. Dobzhansky examined the genetic diversity of wild
populations and showed that, contrary to the assumptions of the population
geneticists, these populations had large amounts of genetic diversity, with
marked differences between sub-populations. The book also took the highly
mathematical work of the population geneticists and put it into a more
accessible form. Many more biologists were influenced by population genetics
via Dobzhansky than were able to read the highly mathematical works in the
original.[9]
In Great Britain E.B._Ford, the pioneer of ecological_genetics, continued
throughout the 1930s and 1940s to empirically demonstrate the power of
selection due to ecological factors including the ability to maintain genetic
diversity through genetic polymorphisms such as human blood_types. Ford's work,
in collaboration with Fisher, contributed to a shift in emphasis during the
course of the modern synthesis towards natural selection as the dominant force.
[4][5][10][11]
**** Neutral theory and origin-fixation dynamics[edit] ****
The original, modern synthesis view of population genetics assumes that
mutations provide ample raw material, and focuses only on the change in
frequency_of_alleles within populations.[12] The main processes influencing
allele frequencies are natural_selection, genetic_drift, gene_flow and
recurrent mutation. Fisher and Wright had some fundamental disagreements about
the relative roles of selection and drift.[13]
The availability of molecular data on all genetic differences led to the
neutral_theory_of_molecular_evolution. In this view, many mutations are
deleterious and so never observed, and most of the remainder are neutral, i.e.
are not under selection. With the fate of each neutral mutation left to chance
(genetic drift), the direction of evolutionary change is driven by which
mutations occur, and so cannot be captured by models of change in the frequency
of (existing) alleles alone.[12][14]
The origin-fixation view of population genetics generalizes this approach
beyond strictly neutral mutations, and sees the rate at which a particular
change happens as the product of the mutation rate and the fixation
probability.[12]
***** Four processes[edit] *****
**** Selection[edit] ****
Natural_selection, which includes sexual_selection, is the fact that some
traits make it more likely for an organism to survive and reproduce. Population
genetics describes natural selection by defining fitness as a propensity_or
probability of survival and reproduction in a particular environment. The
fitness is normally given by the symbol w=1-s where s is the selection
coefficient. Natural selection acts on phenotypes, so population genetic models
assume relatively simple relationships to predict the phenotype and hence
fitness from the allele at one or a small number of loci. In this way, natural
selection converts differences in the fitness of individuals with different
phenotypes into changes in allele frequency in a population over successive
generations.
Before the advent of population genetics, many biologists doubted that small
differences in fitness were sufficient to make a large difference to evolution.
[9] Population geneticists addressed this concern in part by comparing
selection to genetic_drift. Selection can overcome genetic drift when s is
greater than 1 divided by the effective_population_size. When this criterion is
met, the probability that a new advantageous mutant becomes fixed is
approximately equal to 2s.[15][16] The time until fixation of such an allele
depends little on genetic drift, and is approximately proportional to log(sN)/
s.[17]
*** Dominance[edit] ***
Dominance means that the phenotypic and/or fitness effect of one allele at a
locus depends on which allele is present in the second copy for that locus.
Consider three genotypes at one locus, with the following fitness values[18]
- Genotype: A1A1 A1A2 A2A2 - Relative fitness: 1 1-hs 1-s
s is the selection_coefficient and h is the dominance coefficient. The value of
h yields the following information:
      A1              A2
- h=0 dominant, - h=1 dominant, - 0<h<1 incomplete - h<0 overdominance - h>1 Underdominance
      A2              A1                dominance
      recessive       recessive
*** Epistasis[edit] ***
The logarithm of fitness as a function of the number of deleterious mutations.
Synergistic epistasis is represented by the red line - each subsequent
deleterious mutation has a larger proportionate effect on the organism's
fitness. Antagonistic epistasis is in blue. The black line shows the non-
epistatic case, where fitness is the product of the contributions from each of
its loci.
Epistasis means that the phenotypic and/or fitness effect of an allele at one
locus depends on which alleles are present at other loci. Selection does not
act on a single locus, but on a phenotype that arises through development from
a complete genotype.[19] However, many population genetics models of sexual
species are "single locus" models, where the fitness of an individual is
calculated as the product of the contributions from each of its
lociâeffectively assuming no epistasis.
In fact, the genotype_to_fitness_landscape is more complex. Population genetics
must either model this complexity in detail, or capture it by some simpler
average rule. Empirically, beneficial mutations tend to have a smaller fitness
benefit when added to a genetic background that already has high fitness: this
is known as diminishing returns epistasis.[20] When deleterious mutations also
have a smaller fitness effect on high fitness backgrounds, this is known as
"synergistic epistasis". However, the effect of deleterious mutations tends on
average to be very close to multiplicative, or can even show the opposite
pattern, known as "antagonistic epistasis".[21]
Synergistic epistasis is central to some theories of the purging of mutation
load[22] and to the evolution_of_sexual_reproduction.
**** Mutation[edit] ****
Main article: Mutation
Drosophila melanogaster
Mutation is the ultimate source of genetic_variation in the form of new
alleles. In addition, mutation may influence the direction of evolution when
there is mutation bias, i.e. different probabilities for different mutations to
occur. For example, recurrent mutation that tends to be in the opposite
direction to selection can lead to mutation-selection_balance. At the molecular
level, if mutation from G to A happens more often than mutation from A to G,
then genotypes with A will tend to evolve.[23] Different insertion vs. deletion
mutation biases in different taxa can lead to the evolution of different genome
sizes.[24][25] Developmental or mutational biases have also been observed in
morphological evolution.[26][27] For example, according to the phenotype-first
theory_of_evolution, mutations can eventually cause the genetic_assimilation of
traits that were previously induced_by_the_environment.[28][29]
Mutation bias effects are superimposed on other processes. If selection would
favor either one out of two mutations, but there is no extra advantage to
having both, then the mutation that occurs the most frequently is the one that
is most likely to become fixed in a population.[30][31]
Mutation can have no effect, alter the product_of_a_gene, or prevent the gene
from functioning. Studies in the fly Drosophila_melanogaster suggest that if a
mutation changes a protein produced by a gene, this will probably be harmful,
with about 70 percent of these mutations having damaging effects, and the
remainder being either neutral or weakly beneficial.[32] Most loss of function
mutations are selected against. But when selection is weak, mutation bias
towards loss of function can affect evolution.[33] For example, pigments are no
longer useful when animals live in the darkness of caves, and tend to be lost.
[34] This kind of loss of function can occur because of mutation bias, and/or
because the function had a cost, and once the benefit of the function
disappeared, natural selection leads to the loss. Loss of sporulation ability
in a bacterium during laboratory evolution appears to have been caused by
mutation bias, rather than natural selection against the cost of maintaining
sporulation ability.[35] When there is no selection for loss of function, the
speed at which loss evolves depends more on the mutation rate than it does on
the effective_population_size,[36] indicating that it is driven more by
mutation bias than by genetic drift.
Mutations can involve large sections of DNA becoming duplicated, usually
through genetic_recombination.[37] This leads to copy-number_variation within a
population. Duplications are a major source of raw material for evolving new
genes.[38] Other types of mutation occasionally create new genes from
previously noncoding DNA.[39][40]
**** Genetic drift[edit] ****
Main article: Genetic_drift
Genetic drift is a change in allele_frequencies caused by random_sampling.[41]
That is, the alleles in the offspring are a random sample of those in the
parents.[42] Genetic drift may cause gene variants to disappear completely, and
thereby reduce genetic variability. In contrast to natural selection, which
makes gene variants more common or less common depending on their reproductive
success,[43] the changes due to genetic drift are not driven by environmental
or adaptive pressures, and are equally likely to make an allele more common as
less common.
The effect of genetic drift is larger for alleles present in few copies than
when an allele is present in many copies. The population genetics of genetic
drift are described using either branching_processes or a diffusion_equation
describing changes in allele frequency.[44] These approaches are usually
applied to the Wright-Fisher and Moran models of population genetics. Assuming
genetic drift is the only evolutionary force acting on an allele, after t
generations in many replicated populations, starting with allele frequencies of
p and q, the variance in allele frequency across those populations is
          V  t   &#x2248; p q  (  1 &#x2212; exp &#x2061;  {  &#x2212;   t  2
      N  e       }   )  .   {\displaystyle V_{t}\approx pq\left(1-\exp \left\{-
      {\frac {t}{2N_{e}}}\right\}\right).}  [V_{t}\approx pq\left(1-\exp \left\
      {-{\frac {t}{2N_{e}}}\right\}\right).][45]
Ronald_Fisher held the view that genetic drift plays at the most a minor role
in evolution, and this remained the dominant view for several decades. No
population genetics perspective have ever given genetic drift a central role by
itself, but some have made genetic drift important in combination with another
non-selective force. The shifting_balance_theory of Sewall_Wright held that the
combination of population structure and genetic drift was important. Motoo
Kimura's neutral_theory_of_molecular_evolution claims that most genetic
differences within and between populations are caused by the combination of
neutral mutations and genetic drift.[46]
The role of genetic drift by means of sampling_error in evolution has been
criticized by John_H_Gillespie[47] and Will_Provine,[48] who argue that
selection on linked sites is a more important stochastic force, doing the work
traditionally ascribed to genetic drift by means of sampling error. The
mathematical properties of genetic draft are different from those of genetic
drift.[49] The direction of the random change in allele frequency is
autocorrelated across generations.[41]
**** Gene flow[edit] ****
Gene flow is the transfer of alleles from one population to another population
through immigration of individuals. In this example, one of the birds from
population A immigrates to population B, which has fewer of the dominant
alleles, and through mating incorporates its alleles into the other population.
The Great_Wall_of_China is an obstacle to gene flow of some terrestrial
species.
Main article: Gene_flow
Because of physical barriers to migration, along with the limited tendency for
individuals to move or spread (vagility), and tendency to remain or come back
to natal place (philopatry), natural populations rarely all interbreed as may
be assumed in theoretical random models (panmixy).[50] There is usually a
geographic range within which individuals are more closely related to one
another than those randomly selected from the general population. This is
described as the extent to which a population is genetically structured.[51]
Genetic structuring can be caused by migration due to historical climate
change, species range_expansion or current availability of habitat. Gene flow
is hindered by mountain ranges, oceans and deserts or even man-made structures
such as the Great_Wall_of_China, which has hindered the flow of plant genes.
[52]
Gene_flow is the exchange of genes between populations or species, breaking
down the structure. Examples of gene flow within a species include the
migration and then breeding of organisms, or the exchange of pollen. Gene
transfer between species includes the formation of hybrid organisms and
horizontal_gene_transfer. Population genetic models can be used to identify
which populations show significant genetic isolation from one another, and to
reconstruct their history.[53]
Subjecting a population to isolation leads to inbreeding_depression. Migration
into a population can introduce new genetic variants,[54] potentially
contributing to evolutionary_rescue. If a significant proportion of individuals
or gametes migrate, it can also change allele frequencies, e.g. giving rise to
migration_load.[55]
In the presence of gene flow, other barriers_to_hybridization between two
diverging populations of an outcrossing species are required for the
populations to become_new_species.
*** Horizontal gene transfer[edit] ***
Current tree of life showing vertical and horizontal gene transfers.
Main article: Horizontal_gene_transfer
Horizontal gene transfer is the transfer of genetic material from one organism
to another organism that is not its offspring; this is most common among
prokaryotes.[56] In medicine, this contributes to the spread of antibiotic
resistance, as when one bacteria acquires resistance genes it can rapidly
transfer them to other species.[57] Horizontal transfer of genes from bacteria
to eukaryotes such as the yeast Saccharomyces_cerevisiae and the adzuki bean
beetle Callosobruchus_chinensis may also have occurred.[58][59] An example of
larger-scale transfers are the eukaryotic bdelloid_rotifers, which appear to
have received a range of genes from bacteria, fungi, and plants.[60] Viruses
can also carry DNA between organisms, allowing transfer of genes even across
biological_domains.[61] Large-scale gene transfer has also occurred between the
ancestors of eukaryotic_cells and prokaryotes, during the acquisition of
chloroplasts and mitochondria.[62]
***** Linkage[edit] *****
If all genes are in linkage_equilibrium, the effect of an allele at one locus
can be averaged across the gene_pool at other loci. In reality, one allele is
frequently found in linkage_disequilibrium with genes at other loci, especially
with genes located nearby on the same chromosome. Recombination breaks up this
linkage disequilibrium too slowly to avoid genetic_hitchhiking, where an allele
at one locus rises to high frequency because it is linked to an allele under
selection at a nearby locus. Linkage also slows down the rate of adaptation,
even in sexual populations.[63][64][65] The effect of linkage disequilibrium in
slowing down the rate of adaptive evolution arises from a combination of the
HillâRobertson_effect (delays in bringing beneficial mutations together) and
background_selection (delays in separating beneficial mutations from
deleterious hitchhikers).
Linkage is a problem for population genetic models that treat one gene locus at
a time. It can, however, be exploited as a method for detecting the action of
natural_selection via selective_sweeps.
In the extreme case of an asexual_population, linkage is complete, and
population genetic equations can be derived and solved in terms of a travelling
wave of genotype frequencies along a simple fitness_landscape.[66] Most
microbes, such as bacteria, are asexual. The population genetics of their
adaptation have two contrasting regimes. When the product of the beneficial
mutation rate and population size is small, asexual populations follow a
"successional regime" of origin-fixation dynamics, with adaptation rate
strongly dependent on this product. When the product is much larger, asexual
populations follow a "concurrent mutations" regime with adaptation rate less
dependent on the product, characterized by clonal_interference and the
appearance of a new beneficial mutation before the last one has fixed.
***** Applications[edit] *****
**** Explaining levels of genetic variation[edit] ****
Neutral_theory predicts that the level of nucleotide_diversity in a population
will be proportional to the product of the population size and the neutral
mutation rate. The fact that levels of genetic diversity vary much less than
population sizes do is known as the "paradox of variation".[67] While high
levels of genetic diversity were one of the original arguments in favor of
neutral theory, the paradox of variation has been one of the strongest
arguments against neutral theory.
It is clear that levels of genetic diversity vary greatly within a species as a
function of local recombination rate, due to both genetic_hitchhiking and
background_selection. Most current solutions to the paradox of variation invoke
some level of selection at linked sites.[68] For example, one analysis suggests
that larger populations have more selective sweeps, which remove more neutral
genetic diversity.[69] A negative correlation between mutation rate and
population size may also contribute.[70]
Life history affects genetic diversity more than population history does, e.g.
r-strategists have more genetic diversity.[68]
**** Detecting selection[edit] ****
Population genetics models are used to infer which genes are undergoing
selection. One common approach is to look for regions of high linkage
disequilibrium and low genetic variance along the chromosome, to detect recent
selective_sweeps.
A second common approach is the McDonaldâKreitman_test. The
McDonaldâKreitman test compares the amount of variation within a species
(polymorphism) to the divergence between species (substitutions) at two types
of sites, one assumed to be neutral. Typically, synonymous sites are assumed to
be neutral.[71] Genes undergoing positive selection have an excess of divergent
sites relative to polymorphic sites. The test can also be use to obtain a
genome-wide estimate of the proportion of substitutions that are fixed by
positive selection, Î±.[72][73] According to the neutral_theory_of_molecular
evolution, this number should be near zero. High numbers have therefore been
interpreted as a genome-wide falsification of neutral theory.[74]
**** Demographic inference[edit] ****
The simplest test for population structure in a sexually reproducing, diploid
species, is to see whether genotype frequencies follow Hardy-Weinberg
proportions as a function of allele frequencies. For example, in the simplest
case of a single locus with two alleles denoted A and a at frequencies p and q,
random mating predicts freq(AA) = p2 for the AA homozygotes, freq(aa) = q2 for
the aa homozygotes, and freq(Aa) = 2pq for the heterozygotes. In the absence of
population structure, Hardy-Weinberg proportions are reached within 1-
2 generations of random mating. More typically, there is an excess of
homozygotes, indicative of population structure. The extent of this excess can
be quantified as the inbreeding_coefficient,_F.
Individuals can be clustered into K subpopulations.[75][76] The degree of
population structure can then be calculated using FST, which is a measure of
the proportion of genetic variance that can be explained by population
structure. Genetic population structure can then be related to geographic
structure, and genetic_admixture can be detected.
Coalescent_theory relates genetic diversity in a sample to demographic history
of the population from which it was taken. It normally assumes neutrality, and
so sequences from more neutrally-evolving portions of genomes are therefore
selected for such analyses. It can be used to infer the relationships between
species (phylogenetics), as well as the population structure, demographic
history (e.g. population_bottlenecks, population_growth), biological_dispersal,
source-sink_dynamics[77] and introgression within a species.
Another approach to demographic inference relies on the allele_frequency
spectrum.[78]
**** Evolution of genetic systems[edit] ****
By assuming that there are loci that control the genetic system itself,
population genetic models are created to describe the evolution_of_dominance
and other forms of robustness, the evolution_of_sexual_reproduction and
recombination rates, the evolution of mutation_rates, the evolution of
evolutionary_capacitors, the evolution of costly_signalling_traits, the
evolution_of_ageing, and the evolution of co-operation. For example, most
mutations are deleterious, so the optimal mutation_rate for a species may be a
trade-off between the damage from a high deleterious mutation rate and the
metabolic costs of maintaining systems to reduce the mutation rate, such as DNA
repair enzymes.[79]
One important aspect of such models is that selection is only strong enough to
purge deleterious mutations and hence overpower mutational bias towards
degradation if the selection coefficient s is greater than the inverse of the
effective_population_size. This is known as the drift barrier and is related to
the nearly_neutral_theory_of_molecular_evolution. Drift barrier theory predicts
that species with large effective population sizes will have highly
streamlined, efficient genetic systems, while those with small population sizes
will have bloated and complex genomes containing for example introns and
transposable_elements.[80] However, somewhat paradoxically, species with large
population sizes might be so tolerant to the consequences of certain types of
errors that they evolve higher error rates, e.g. in transcription and
translation, than small populations.[81]
***** See also[edit] *****
    * Genotype-phenotype_distinction
    * Haldane's_dilemma
    * Human_genetic_variation
    * Laboratory_experiments_of_speciation
    * List_of_population_genetics_projects
    * Muller's_ratchet
    * Viral_quasispecies
***** References[edit] *****
   1. ^"Population_genetics_-_Latest_research_and_news_|_Nature".
      www.nature.com. Retrieved 2018-01-29.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Servedio,_Maria_R.; Brandvain, Yaniv; Dhole, Sumit; Fitzpatrick,
      Courtney L.; Goldberg, Emma E.; Stern, Caitlin A.; Van Cleve, Jeremy;
      Yeh, D. Justin (9 December 2014). "Not_Just_a_TheoryâThe_Utility_of
      Mathematical_Models_in_Evolutionary_Biology". PLoS Biology. 12 (12):
      e1002017. doi:10.1371/journal.pbio.1002017. PMC 4260780. PMID 25489940.
   4. ^Ewens W.J. (2004). Mathematical Population Genetics (2nd Edition).
      Springer-Verlag, New York. ISBN 978-0-387-20191-7.
   5. ^ a b cBowler, Peter J. (2003). Evolution : the history of an idea (3rd
      ed.). Berkeley: University of California Press. pp. 325â339. ISBN 978-
      0-520-23693-6.
   6. ^ a b cLarson, Edward J. (2004). Evolution : the remarkable history of a
      scientific theory (Modern Library ed.). New York: Modern Library.
      pp. 221â243. ISBN 978-0-679-64288-6.
   7. ^Beatty, John (1986). "The Synthesis and the Synthetic Theory".
      Integrating Scientific Disciplines. Science and Philosophy. 2. Springer
      Netherlands. pp. 125â135. doi:10.1007/978-94-010-9435-1_7.
      ISBN 9789024733422.
   8. ^Mayr, edited by Ernst; Mayer, William B. Provine ; with a new preface by
      Ernst (1998). The Evolutionary synthesis : perspectives on the
      unification of biology ([New ed]. ed.). Cambridge, Massachusetts: Harvard
      University Press. pp. 295â298. ISBN 9780674272262.CS1 maint: Extra
      text: authors list (link)
   9. ^ a bProvine, W. B. (1988). "Progress in evolution and meaning in life".
      Evolutionary progress. University of Chicago Press. pp. 49â79.
  10. ^ a bProvine, William B. (1978). "The role of mathematical population
      geneticists in the evolutionary synthesis of the 1930s and 1940s".
      Studies of the History of Biology. 2: 167â192.
  11. ^Mayr,_Ernst (1988). Toward_a_New_Philosophy_of_Biology:_Observations_of
      an_Evolutionist. Cambridge, Massachusetts: Belknap_Press_of_Harvard
      University_Press. p. 402. ISBN 978-0-674-89665-9.
  12. ^Mayr, edited by Ernst; Mayer, William B. Provine ; with a new preface by
      Ernst (1998). The Evolutionary synthesis : perspectives on the
      unification of biology ([New ed]. ed.). Cambridge, Massachusetts: Harvard
      University Press. pp. 338â341. ISBN 9780674272262.CS1 maint: Extra
      text: authors list (link)
  13. ^ a b cMcCandlish, David M.; Stoltzfus, Arlin (September 2014). "Modeling
      Evolution Using the Probability of Fixation: History and Implications".
      The Quarterly Review of Biology. 89 (3): 225â252. doi:10.1086/677571.
  14. ^ Wright_and_Fisher_on_Inbreeding_and_Random_Drift_by_James_F._Crow,
      published_inGenetics Published 2010
  15. ^Casillas, SÃ²nia; Barbadilla, Antonio (2017). "Molecular_Population
      Genetics". Genetics. 205 (3): 1003â1035. doi:10.1534/
      genetics.116.196493. PMC 5340319. PMID 28270526.
  16. ^JBS Haldane (1927). "A Mathematical Theory of Natural and Artificial
      Selection, Part V: Selection and Mutation". Mathematical Proceedings of
      the Cambridge Philosophical Society. 23 (7): 838â844. Bibcode:
      1927PCPS...23..838H. doi:10.1017/S0305004100015644.
  17. ^Orr, H. A. (2010). "The_population_genetics_of_beneficial_mutations".
      Philosophical Transactions of the Royal Society B: Biological Sciences.
      365 (1544): 1195â1201. doi:10.1098/rstb.2009.0282. PMC 2871816.
      PMID 20308094.
  18. ^Hermisson J, Pennings PS; Pennings (2005). "Soft_sweeps:_molecular
      population_genetics_of_adaptation_from_standing_genetic_variation".
      Genetics. 169 (4): 2335â2352. doi:10.1534/genetics.104.036947.
      PMC 1449620. PMID 15716498.
  19. ^Gillespie,_John (2004). Population Genetics: A Concise Guide (2nd ed.).
      Johns_Hopkins_University_Press. ISBN 978-0-8018-8008-7.
  20. ^Miko, I. (2008). "Epistasis:_Gene_interaction_and_phenotype_effects".
      Nature Education. 1 (1): 197.
  21. ^Berger, D.; Postma, E. (13 October 2014). "Biased_Estimates_of
      Diminishing-Returns_Epistasis?_Empirical_Evidence_Revisited". Genetics.
      198 (4): 1417â1420. doi:10.1534/genetics.114.169870. PMC 4256761.
      PMID 25313131.
  22. ^Kouyos, Roger D.; Silander, Olin K.; Bonhoeffer, Sebastian (June 2007).
      "Epistasis between deleterious mutations and the evolution of
      recombination". Trends in Ecology & Evolution. 22 (6): 308â315. doi:
      10.1016/j.tree.2007.02.014. PMID 17337087.
  23. ^Crow, JF (5 August 1997). "The_high_spontaneous_mutation_rate:_is_it_a
      health_risk?". Proceedings of the National Academy of Sciences of the
      United States of America. 94 (16): 8380â8386. Bibcode:
      1997PNAS...94.8380C. doi:10.1073/pnas.94.16.8380. PMC 33757.
      PMID 9237985.
  24. ^Smith N.G.C., Webster M.T., Ellegren, H.; Webster; Ellegren (2002).
      "Deterministic_Mutation_Rate_Variation_in_the_Human_Genome". Genome
      Research. 12 (9): 1350â1356. doi:10.1101/gr.220502. PMC 186654.
      PMID 12213772.CS1 maint: Multiple names: authors list (link)
  25. ^Petrov DA, Sangster TA, Johnston JS, Hartl DL, Shaw KL; Sangster;
      Johnston; Hartl; Shaw (2000). "Evidence for DNA loss as a determinant of
      genome size". Science. 287 (5455): 1060â1062. Bibcode:
      2000Sci...287.1060P. doi:10.1126/science.287.5455.1060. PMID 10669421.CS1
      maint: Multiple names: authors list (link)
  26. ^Petrov DA (2002). "DNA loss and evolution of genome size in Drosophila".
      Genetica. 115 (1): 81â91. doi:10.1023/A:1016076215168. PMID 12188050.
  27. ^Kiontke K, Barriere A, Kolotuev I, Podbilewicz B, Sommer R, Fitch DHA,
      Felix MA; BarriÃ¨re; Kolotuev; Podbilewicz; Sommer; Fitch; FÃ©lix (2007).
      "Trends, stasis, and drift in the evolution of nematode vulva
      development". Current Biology. 17 (22): 1925â1937. doi:10.1016/
      j.cub.2007.10.061. PMID 18024125.CS1 maint: Multiple names: authors list
      (link)
  28. ^Braendle C, Baer CF, Felix MA; Baer; FÃ©lix (2010). Barsh, Gregory S
      (ed.). "Bias_and_Evolution_of_the_Mutationally_Accessible_Phenotypic
      Space_in_a_Developmental_System". PLoS Genetics. 6 (3): e1000877. doi:
      10.1371/journal.pgen.1000877. PMC 2837400. PMID 20300655.CS1 maint:
      Multiple names: authors list (link)
  29. ^Palmer, RA (2004). "Symmetry breaking and the evolution of development".
      Science. 306 (5697): 828â833. Bibcode:2004Sci...306..828P.
      CiteSeerX 10.1.1.631.4256. doi:10.1126/science.1103707. PMID 15514148.
  30. ^West-Eberhard, M-J. (2003). Developmental plasticity and evolution. New
      York: Oxford University Press. ISBN 978-0-19-512235-0.
  31. ^Stoltzfus, A & Yampolsky, L.Y. (2009). "Climbing Mount Probable:
      Mutation as a Cause of Nonrandomness in Evolution". J Hered. 100 (5):
      637â647. doi:10.1093/jhered/esp048. PMID 19625453.
  32. ^Yampolsky, L.Y. & Stoltzfus, A (2001). "Bias in the introduction of
      variation as an orienting factor in evolution". Evol Dev. 3 (2): 73â83.
      doi:10.1046/j.1525-142x.2001.003002073.x. PMID 11341676.
  33. ^Sawyer SA, Parsch J, Zhang Z, Hartl DL; Parsch; Zhang; Hartl (2007).
      "Prevalence_of_positive_selection_among_nearly_neutral_amino_acid
      replacements_in_Drosophila". Proc. Natl. Acad. Sci. U.S.A. 104 (16):
      6504â10. Bibcode:2007PNAS..104.6504S. doi:10.1073/pnas.0701572104.
      PMC 1871816. PMID 17409186.CS1 maint: Multiple names: authors list (link)
  34. ^Haldane, JBS (1933). "The Part Played by Recurrent Mutation in
      Evolution". American Naturalist. 67 (708): 5â19. doi:10.1086/280465.
      JSTOR 2457127.
  35. ^Protas, Meredith; Conrad, M; Gross, JB; Tabin, C; Borowsky, R (2007).
      "Regressive_evolution_in_the_Mexican_cave_tetra,_Astyanax_mexicanus".
      Current Biology. 17 (5): 452â454. doi:10.1016/j.cub.2007.01.051.
      PMC 2570642. PMID 17306543.
  36. ^Maughan H, Masel J, Birky WC, Nicholson WL; Masel; Birky Jr; Nicholson
      (2007). "The_roles_of_mutation_accumulation_and_selection_in_loss_of
      sporulation_in_experimental_populations_of_Bacillus_subtilis". Genetics.
      177 (2): 937â948. doi:10.1534/genetics.107.075663. PMC 2034656.
      PMID 17720926.CS1 maint: Multiple names: authors list (link)
  37. ^Masel J, King OD, Maughan H; King; Maughan (2007). "The_loss_of_adaptive
      plasticity_during_long_periods_of_environmental_stasis". American
      Naturalist. 169 (1): 38â46. doi:10.1086/510212. PMC 1766558.
      PMID 17206583.CS1 maint: Multiple names: authors list (link)
  38. ^Hastings, P J; Lupski, JR; Rosenberg, SM; Ira, G (2009). "Mechanisms_of
      change_in_gene_copy_number". Nature Reviews Genetics. 10 (8): 551â564.
      doi:10.1038/nrg2593. PMC 2864001. PMID 19597530.
  39. ^Long M, BetrÃ¡n E, Thornton K, Wang W; BetrÃ¡n; Thornton; Wang (November
      2003). "The origin of new genes: glimpses from the young and old". Nat.
      Rev. Genet. 4 (11): 865â75. doi:10.1038/nrg1204. PMID 14634634.CS1
      maint: Multiple names: authors list (link)
  40. ^Liu N, Okamura K, Tyler DM; Okamura; Tyler; Phillips; Chung; Lai (2008).
      "The_evolution_and_functional_diversification_of_animal_microRNA_genes".
      Cell Res. 18 (10): 985â96. doi:10.1038/cr.2008.278. PMC 2712117.
      PMID 18711447.CS1 maint: Multiple names: authors list (link)
  41. ^McLysaght, Aoife; Hurst, Laurence D. (25 July 2016). "Open questions in
      the study of de novo genes: what, how and why". Nature Reviews Genetics.
      17 (9): 567â578. doi:10.1038/nrg.2016.78. PMID 27452112.
  42. ^ a bMasel, J. (2011). "Genetic drift". Current Biology. 21 (20):
      R837âR838. doi:10.1016/j.cub.2011.08.007. PMID 22032182.
  43. ^Futuyma, Douglas (1998). Evolutionary Biology. Sinauer_Associates.
      p. Glossary. ISBN 978-0-87893-189-7.
  44. ^Avers, Charlotte (1989). "Process and Pattern in Evolution". Oxford
      University Press.
  45. ^Wahl L.M. (2011). "Fixation_when_N_and_s_Vary:_Classic_Approaches_Give
      Elegant_New_Results". Genetics. 188 (4): 783â785. doi:10.1534/
      genetics.111.131748. PMC 3176088. PMID 21828279.
  46. ^Nicholas H. Barton; Derek E. G. Briggs; Jonathan A. Eisen; David B.
      Goldstein; Nipam H. Patel (2007). Evolution. Cold Spring Harbor
      Laboratory Press. p. 417. ISBN 978-0-87969-684-9.
  47. ^Futuyma, Douglas (1998). Evolutionary Biology. Sinauer_Associates.
      p. 320. ISBN 978-0-87893-189-7.
  48. ^Gillespie, J.H. (2000). "Genetic_Drift_in_an_Infinite_Population:_The
      Pseudohitchhiking_Model". Genetics. 155 (2): 909â919. PMC 1461093.
      PMID 10835409.
  49. ^Provine, William B. The "Random Genetic Drift" Fallacy. CreateSpace.
  50. ^Neher, Richard A.; Shraiman, Boris I. (August 2011). "Genetic_Draft_and
      Quasi-Neutrality_in_Large_Facultatively_Sexual_Populations". Genetics.
      188 (4): 975â996. doi:10.1534/genetics.111.128876. ISSN 0016-6731.
      PMC 3176096. PMID 21625002.
  51. ^Buston, P. M.; Pilkington, J. G.; et al. (2007). "Are clownfish groups
      composed of close relatives? An analysis of microsatellite DNA vraiation
      in Amphiprion percula". Molecular Ecology. 12 (3): 733â742. doi:
      10.1046/j.1365-294X.2003.01762.x. PMID 12675828.
  52. ^Repaci, V.; Stow, A.J.; Briscoe, D.A. (2007). "Fine-scale genetic
      structure, co-founding and multiple mating in the Australian allodapine
      bee (Ramphocinclus brachyurus)". Journal of Zoology. 270 (4): 687â691.
      doi:10.1111/j.1469-7998.2006.00191.x.
  53. ^Su, H.; Qu, L.; He, K., Zhang, Z.; Wang, J.; Chen, Z.; Gu, H.; Qu; He;
      Zhang; Wang; Chen; Gu (2003). "The Great Wall of China: a physical
      barrier to gene flow?". Heredity. 90 (3): 212â9. doi:10.1038/
      sj.hdy.6800237. PMID 12634804.CS1 maint: Multiple names: authors list
      (link)
  54. ^Gravel, S., S. (2012). "Population_Genetics_Models_of_Local_Ancestry".
      Genetics. 1202 (2): 607â619. arXiv:1202.4811. Bibcode:
      2012arXiv1202.4811G. doi:10.1534/genetics.112.139808. PMC 3374321.
      PMID 22491189.
  55. ^Morjan, C.; Rieseberg, L.; Rieseberg (2004). "How_species_evolve
      collectively:_implications_of_gene_flow_and_selection_for_the_spread_of
      advantageous_alleles". Mol. Ecol. 13 (6): 1341â56. doi:10.1111/j.1365-
      294X.2004.02164.x. PMC 2600545. PMID 15140081.CS1 maint: Multiple names:
      authors list (link)
  56. ^Bolnick, Daniel I.; Nosil, Patrik (September 2007). "Natural Selection
      in Populations Subject to a Migration Load". Evolution. 61 (9):
      2229â2243. doi:10.1111/j.1558-5646.2007.00179.x. PMID 17767592.
  57. ^Boucher, Y.; Douady, C.J.; Papke, R.T.; Walsh, D.A.; Boudreau, M.E.;
      Nesbo, C.L.; Case, R.J.; Doolittle, W.F.; Douady; Papke; Walsh; Boudreau;
      NesbÃ¸; Case; Doolittle (2003). "Lateral gene transfer and the origins of
      prokaryotic groups". Annu Rev Genet. 37: 283â328. doi:10.1146/
      annurev.genet.37.050503.084247. PMID 14616063.CS1 maint: Multiple names:
      authors list (link)
  58. ^Walsh T (2006). "Combinatorial genetic evolution of multiresistance".
      Curr. Opin. Microbiol. 9 (5): 476â82. doi:10.1016/j.mib.2006.08.009.
      PMID 16942901.
  59. ^Kondo N, Nikoh N, Ijichi N, Shimada M, Fukatsu T; Nikoh; Ijichi;
      Shimada; Fukatsu (2002). "Genome_fragment_of_Wolbachia_endosymbiont
      transferred_to_X_chromosome_of_host_insect". PNAS. 99 (22): 14280â5.
      Bibcode:2002PNAS...9914280K. doi:10.1073/pnas.222228199. PMC 137875.
      PMID 12386340.CS1 maint: Multiple names: authors list (link)
  60. ^Sprague G (1991). "Genetic exchange between kingdoms". Curr. Opin.
      Genet. Dev. 1 (4): 530â3. doi:10.1016/S0959-437X(05)80203-5.
      PMID 1822285.
  61. ^Gladyshev EA, Meselson M, Arkhipova IR; Meselson; Arkhipova (May 2008).
      "Massive_horizontal_gene_transfer_in_bdelloid_rotifers". Science. 320
      (5880): 1210â3. Bibcode:2008Sci...320.1210G. doi:10.1126/
      science.1156407. PMID 18511688.CS1 maint: Multiple names: authors list
      (link)
  62. ^Baldo A, McClure M; McClure (1 September 1999). "Evolution_and
      horizontal_transfer_of_dUTPase-encoding_genes_in_viruses_and_their
      hosts". J. Virol. 73 (9): 7710â21. PMC 104298. PMID 10438861.
  63. ^Poole A, Penny D; Penny (2007). "Evaluating hypotheses for the origin of
      eukaryotes". BioEssays. 29 (1): 74â84. doi:10.1002/bies.20516.
      PMID 17187354.
  64. ^Weissman, D. B.; Hallatschek, O. (15 January 2014). "The_Rate_of
      Adaptation_in_Large_Sexual_Populations_with_Linear_Chromosomes".
      Genetics. 196 (4): 1167â1183. doi:10.1534/genetics.113.160705.
      PMC 3982688. PMID 24429280.
  65. ^Weissman, Daniel B.; Barton, Nicholas H.; McVean, Gil (7 June 2012).
      "Limits_to_the_Rate_of_Adaptive_Substitution_in_Sexual_Populations". PLoS
      Genetics. 8 (6): e1002740. doi:10.1371/journal.pgen.1002740. PMC 3369949.
      PMID 22685419.
  66. ^Neher, R. A.; Shraiman, B. I.; Fisher, D. S. (30 November 2009). "Rate
      of_Adaptation_in_Large_Sexual_Populations". Genetics. 184 (2): 467â481.
      arXiv:1108.3464. doi:10.1534/genetics.109.109009. PMC 2828726.
      PMID 19948891.
  67. ^Michael M. Desai, Daniel S. Fisher; Fisher (2007). "Beneficial_Mutation
      Selection_Balance_and_the_Effect_of_Linkage_on_Positive_Selection".
      Genetics. 176 (3): 1759â1798. doi:10.1534/genetics.106.067678.
      PMC 1931526. PMID 17483432.
  68. ^Lewontin, [by] R. C. (1973). The genetic basis of evolutionary change (
      [4th printing.] ed.). New York: Columbia University Press. ISBN 978-
      0231033923.
  69. ^ a bEllegren, Hans; Galtier, Nicolas (6 June 2016). "Determinants_of
      genetic_diversity". Nature Reviews Genetics. 17 (7): 422â433. doi:
      10.1038/nrg.2016.58. PMID 27265362.
  70. ^Corbett-Detig, Russell B.; Hartl, Daniel L.; Sackton, Timothy B.;
      Barton, Nick H. (10 April 2015). "Natural_Selection_Constrains_Neutral
      Diversity_across_A_Wide_Range_of_Species". PLOS Biology. 13 (4):
      e1002112. doi:10.1371/journal.pbio.1002112. PMC 4393120. PMID 25859758.
  71. ^Sung, W.; Ackerman, M. S.; Miller, S. F.; Doak, T. G.; Lynch, M. (17
      October 2012). "Drift-barrier_hypothesis_and_mutation-rate_evolution"
      (PDF). Proceedings of the National Academy of Sciences. 109 (45):
      18488â18492. Bibcode:2012PNAS..10918488S. doi:10.1073/pnas.1216223109.
      PMC 3494944. PMID 23077252.
  72. ^Charlesworth, J. Eyre-Walker (2008). "The McDonaldâKreitman Test and
      Slightly Deleterious Mutations". Molecular Biology and Evolution. 25 (6):
      1007â1015. doi:10.1093/molbev/msn005. PMID 18195052.
  73. ^Eyre-Walker, A (2006). "The_genomic_rate_of_adaptive_evolution" (PDF).
      Trends in Ecology and Evolution. 21 (10): 569â575. doi:10.1016/
      j.tree.2006.06.015. PMID 16820244.
  74. ^Smith, N. G. C.; Eyre-Walker, A. (2002). "Adaptive protein evolution in
      Drosophila". Nature. 415 (6875): 1022â1024. doi:10.1038/4151022a.
      PMID 11875568.
  75. ^Hahn, M.W. (2008). "Toward a selection theory of molecular evolution".
      Evolution. 62 (2): 255â265. doi:10.1111/j.1558-5646.2007.00308.x.
      PMID 18302709.
  76. ^Pritchard, J K; Stephens, M; Donnelly, P (June 2000). "Inference_of
      population_structure_using_multilocus_genotype_data". Genetics. 155 (2):
      945â959. ISSN 0016-6731. PMC 1461096. PMID 10835412.
  77. ^Verity, Robert; Nichols, Richard A. (August 2016). "Estimating_the
      Number_of_Subpopulations_(K)_in_Structured_Populations". Genetics. 203
      (4): 1827â1839. doi:10.1534/genetics.115.180992. ISSN 0016-6731.
      PMC 4981280. PMID 27317680.
  78. ^Manlik, Oliver; Chabanne, Delphine; Daniel, Claire; Bejder, Lars; Allen,
      Simon J.; Sherwin, William B. (13 November 2018). "Demography and
      genetics suggest reversal of dolphin source-sink dynamics, with
      implications for conservation". Marine Mammal Science. 35 (3): 732â759.
      doi:10.1111/mms.12555.
  79. ^Gutenkunst, Ryan N.; Hernandez, Ryan D.; Williamson, Scott H.;
      Bustamante, Carlos D.; McVean, Gil (23 October 2009). "Inferring_the
      Joint_Demographic_History_of_Multiple_Populations_from_Multidimensional
      SNP_Frequency_Data". PLoS Genetics. 5 (10): e1000695. doi:10.1371/
      journal.pgen.1000695. PMC 2760211. PMID 19851460.
  80. ^Sniegowski P, Gerrish P, Johnson T, Shaver A; Gerrish; Johnson; Shaver
      (2000). "The evolution of mutation rates: separating causes from
      consequences". BioEssays. 22 (12): 1057â1066. doi:10.1002/1521-1878
      (200012)22:12<1057::AID-BIES3>3.0.CO;2-W. PMID 11084621.CS1 maint:
      Multiple names: authors list (link)
  81. ^Lynch, Michael, John S. Conery; Conery (2003). "The origins of genome
      complexity". Science. 302 (5649): 1401â1404. Bibcode:
      2003Sci...302.1401L. CiteSeerX 10.1.1.135.974. doi:10.1126/
      science.1089370. PMID 14631042.CS1 maint: Multiple names: authors list
      (link)
  82. ^Rajon, E.; Masel, J. (3 January 2011). "Evolution_of_molecular_error
      rates_and_the_consequences_for_evolvability". Proceedings of the National
      Academy of Sciences. 108 (3): 1082â1087. Bibcode:2011PNAS..108.1082R.
      doi:10.1073/pnas.1012918108. PMC 3024668. PMID 21199946.
***** External links[edit] *****
 Wikimedia Commons has media related to Population_genetics.
    * Population_Genetics_Tutorials
    * Molecular_population_genetics
    * The_ALlele_FREquency_Database at Yale_University
    * EHSTRAFD.org_-_Earth_Human_STR_Allele_Frequencies_Database
    * History_of_population_genetics
    * How_Selection_Changes_the_Genetic_Composition_of_Population, video of
      lecture by Stephen_C._Stearns (Yale_University)
    * National_Geographic: Atlas_of_the_Human_Journey (Haplogroup-based human
      migration maps)
    * v
    * t
    * e
Population genetics
                         * Hardy-Weinberg_law
                         * Genetic_linkage
                         * Identity_by_descent
                         * Linkage_disequilibrium
                         * Fisher's_fundamental_theorem
Key concepts             * Neutral_theory
                         * Shifting_balance_theory
                         * Price_equation
                         * Coefficient_of_inbreeding and relationship
                         * Fitness
                         * Heritability
                         * Natural
Selection                * Sexual
                         * Artificial
                         * Ecological
Effects of selection     * Genetic_hitchhiking
on genomic variation     * Background_selection
                         * Small_population_size
                         * Population_bottleneck
Genetic_drift            * Founder_effect
                         * Coalescence
                         * BaldingâNichols_model
                         * R._A._Fisher
Founders                 * J._B._S._Haldane
                         * Sewall_Wright
                         * Evolution
                         * Microevolution
Related topics           * Evolutionary_game_theory
                         * Fitness_landscape
                         * Genetic_genealogy
                         * Quantitative_genetics
Index_of_evolutionary_biology_articles
    * v
    * t
    * e
Human_genetics
                                 * Human_genome
                                 * Evolutionary_genetics
                                       o Human-chimp_MRCA
                                       o Neanderthal_genetics
                                       o Timeline
                                 * Genetic_variation
Sub-topics                             o Blood_type_distribution_by_country
                                       o Genealogical_DNA_test
                                       o Genetic_genealogy
                                       o Race_and_genetics
                                       o Recent_evolution
                                       o Surname_DNA_project
                                 * Genetic_engineering
                                 * Sub-Saharan_African
                                 * Australo-Melanesian
                                 * South_Asian
                                 * North_African
                                 * Near_Eastern
                                       o Early_Anatolian_Farmers
                                 * Caucasus
                                       o Caucasian_Hunter-Gatherer
                                 * Europe
Genetic_history by region              o Western_Hunter-Gatherer
                                       o British_Isles
                                       o Iberia
                                       o Italy
                                 * Central_Asian
                                       o Ancient_North_Eurasian
                                 * East_Asian
                                 * Southeast_Asian
                                 * American
                                       o Ancient_Beringian
                                 * Europe
                                       o Albanians
                                       o Basques
                                       o Bosniaks
                                       o Bulgarians
                                       o Croats
                                       o Romanians
                                       o Russians
                                       o Sami
                                       o Serbs
                                 * Jews
                                 * MENA
                                       o Arabs
Population genetics by group           o Azerbaijanis
                                       o Egyptians
                                       o Moroccans
                                       o Turks
                                 * South Asia
                                       o Gujaratis
                                       o Sinhalese
                                       o Tamils_(Sri_Lankan)
                                 * East Asia
                                       o Han_Chinese
                                       o Japanese
                                 * Sub-Saharan Africa
                                       o Hutu/Tutsi
                                       o Khoisan
                                       o Pygmies
    * v
    * t
    * e
Genetics
    * Introduction
    * Outline
    * History
    * Index
                       * Chromosome
                       * DNA
Key components         * RNA
                       * Nucleotide
                       * Genome
                       * Classical
                       * Conservation
                       * Ecological
Fields                 * Immunogenetics
                       * Molecular
                       * Population
                       * Quantitative
                       * the_Americas
                       * the_British_Isles
Archaeogenetics of     * Europe
                       * Italy
                       * the_Near_East
                       * South_Asia
                       * Behavioural_genetics
                       * Epigenetics
                       * Geneticist
                       * Genome_editing
                       * Genomics
                       * Genetic_code
                       * Medical_genetics
Related topics         * Molecular_evolution
                       * Reverse_genetics
                       * Genetic_engineering
                       * Genetic_diversity
                       * Heredity
                       * Genetic_monitoring
                       * Genetic_genealogy
                       * Lulu_and_Nana_controversy
    * [List-Class article] List_of_genetics_research_organizations
    * [Category] Category
    * v
    * t
    * e
Evolutionary_biology
    * Introduction
    * Outline
    * Timeline_of_evolution
    * Evolutionary_history_of_life
    * Index
                        * Abiogenesis
                        * Adaptation
                        * Adaptive_radiation
                        * Cladistics
                        * Coevolution
                        * Common_descent
                        * Convergence
                        * Divergence
                        * Earliest_known_life
                          forms
                        * Evidence_of_evolution
Evolution               * Extinction
                              o Event
                        * Gene-centered_view
                        * Homology
                        * Last_universal_common
                          ancestor
                        * Macroevolution
                        * Microevolution
                        * Origin_of_life
                        * Panspermia
                        * Parallel_evolution
                        * Speciation
                        * Taxonomy
                        * Biodiversity
                        * Gene_flow
Population genetics     * Genetic_drift
                        * Mutation
                        * Natural_selection
                        * Variation
                        * Canalisation
                        * Evolutionary
Development               developmental_biology
                        * Inversion
                        * Modularity
                        * Phenotypic_plasticity
                        * Birds
                              o origin
                        * Brachiopods
                        * Molluscs
                              o Cephalopods
                        * Dinosaurs
                        * Fish
                        * Fungi
                        * Insects
                              o butterflies
                        * Life
                        * Mammals
                              o cats
                              o canids
                                    # wolves
Of taxa                             # dogs
                              o hyenas
                              o dolphins_and
                                whales
                              o horses
                              o Kangaroos
                              o primates
                                    # humans
                                    # lemurs
                              o sea_cows
                        * Plants
                        * Reptiles
                        * Spiders
                        * Tetrapods
                        * Viruses
                              o influenza
                        * Cell
                        * DNA
                        * Flagella
                        * Eukaryotes
                              o symbiogenesis
                              o chromosome
                              o endomembrane
                                system
Of organs                     o mitochondria
                              o nucleus
                              o plastids
                        * In animals
                              o eye
                              o hair
                              o auditory_ossicle
                              o nervous_system    [Common-Avian-Infection-
                              o brain             Plagued-the-Tyrant-Dinosaurs-
                        * Aging                   pone.0007288.g004.jpg]
                              o Death                 * [icon]Evolutionary
                              o Programmed_cell         biology_portal
                                death
                        * Avian_flight
                        * Biological_complexity
                        * Cooperation
                        * Color_vision
                              o in_primates
                        * Emotion
                        * Empathy
                        * Ethics
                        * Eusociality
                        * Immune_system
Of processes            * Metabolism
                        * Monogamy
                        * Morality
                        * Mosaic_evolution
                        * Multicellularity
                        * Sexual_reproduction
                              o Gamete
                                differentiation/
                                sexes
                              o Life_cycles/
                                nuclear_phases
                              o Mating_types
                              o Meiosis
                              o Sex-determination
                        * Snake_venom
                        * Gradualism/Punctuated
                          equilibrium/
                          Saltationism
Tempo_and_modes         * Micromutation/
                          Macromutation
                        * Uniformitarianism/
                          Catastrophism
                        * Allopatric
                        * Anagenesis
                        * Catagenesis
                        * Cladogenesis
                        * Cospeciation
Speciation              * Ecological
                        * Hybrid
                        * Parapatric
                        * Peripatric
                        * Reinforcement
                        * Sympatric
                        * Renaissance_and
                          Enlightenment
                        * Transmutation_of
                          species
                        * Charles_Darwin
                              o On_the_Origin_of
                                Species
                        * History_of_paleontology
History                 * Transitional_fossil
                        * Blending_inheritance
                        * Mendelian_inheritance
                        * The_eclipse_of
                          Darwinism
                        * Modern_synthesis
                        * History_of_molecular
                          evolution
                        * Extended_evolutionary
                          synthesis
                        * Darwinism
                        * Alternatives
                              o Catastrophism
                              o Lamarckism
                              o Orthogenesis
Philosophy                    o Mutationism
                              o Saltationism
                              o Structuralism
                                    # Spandrel
                              o Theistic
                              o Vitalism
                        * Teleology_in_biology
                        * Biogeography
                        * Ecological_genetics
                        * Molecular_evolution
Related                 * Phylogenetics
                              o Tree
                        * Polymorphism
                        * Protocell
                        * Systematics
    * [Category] Category
    * [Portal] Portal
Authority_control [Edit_this_at_Wikidata]     * GND: 4046804-5
                                              * NDL: 00572347
    * [Tree_of_life.svg]Evolutionary_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Population_genetics&oldid=909636210"
Categories:
    * Population_genetics
    * Evolutionary_biology
    * Statistical_genetics
Hidden categories:
    * CS1_maint:_Extra_text:_authors_list
    * CS1_maint:_Multiple_names:_authors_list
    * Articles_with_short_description
    * Commons_category_link_from_Wikidata
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * AzÉrbaycanca
    * à¦¬à¦¾à¦à¦²à¦¾
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Latina
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Novial
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Simple_English
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * Tagalog
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 6 August 2019, at 17:04 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
