The following text has been accessed from https://en.wikipedia.org/wiki/Lysozyme https://en.wikipedia.org/wiki/File:JBSlysozymemechanism_copy2.jpg at Fri Aug 9 03:34:43 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Lysozyme ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Not to be confused with Lysosome or Lysosomal_enzymes.
Lysozyme
[Lysozymecrystals1.png]
Lysozyme crystals stained with methylene_blue.
Identifiers
EC_number      3.2.1.17
CAS_number     9001-63-2
Databases
IntEnz         IntEnz_view
BRENDA         BRENDA_entry
ExPASy         NiceZyme_view
KEGG           KEGG_entry
MetaCyc        metabolic_pathway
PRIAM          profile
PDB structures RCSB_PDB PDBe PDBsum
Gene_Ontology  AmiGO / QuickGO
Search
PMC    articles
PubMed articles
NCBI   proteins
Lysozyme, also known as muramidase or N-acetylmuramide glycanhydrolase, is an
antimicrobial enzyme produced by animals that forms part of the innate_immune
system. Lysozyme is a glycoside_hydrolase that catalyzes the hydrolysis of 1,4-
beta-linkages between N-acetylmuramic_acid and N-acetyl-D-glucosamine residues
in peptidoglycan, which is the major component of gram-positive_bacterial cell
wall.[1] This hydrolysis in turn compromises the integrity of bacterial cell
walls causing lysis of the bacteria.
Lysozyme is abundant in secretions including tears, saliva, human_milk, and
mucus. It is also present in cytoplasmic granules of the macrophages and the
polymorphonuclear_neutrophils (PMNs). Large amounts of lysozyme can be found in
egg white. C-type lysozymes are closely related to alpha-lactalbumin in
sequence and structure, making them part of the same glycoside_hydrolase_family
22.[2] In humans, the C-type lysozyme enzyme is encoded by the LYZ gene.[3][4]
Lysozyme is thermally stable, with a melting_point reaching up to 72 â at pH
5.0.[5] However, in human milk it loses activity very quickly at that
temperature.[6] Its isoelectric_point is 11.35. Lysozyme can survive in a large
range of pH (6-9).[7]
⁰
***** Contents *****
    * 1_Function_and_mechanism
          o 1.1_Mechanism
                # 1.1.1_Phillips
                # 1.1.2_Koshland
          o 1.2_Inhibition
          o 1.3_Non-enzymatic_action
          o 1.4_Enzyme_conformation_changes
    * 2_Role_in_disease_and_therapy
    * 3_Chemical_synthesis
    * 4_Other_applications
    * 5_History
    * 6_See_also
    * 7_References
    * 8_External_links
***** Function and mechanism[edit] *****
The enzyme functions by attacking, hydrolyzing, and breaking glycosidic bonds
in peptidoglycans. The enzyme can also break glycosidic_bonds in chitin,
although not as effectively as true chitinases.[8]
Overview of the reaction catalysed by lysozyme
Lysozymes active site binds the peptidoglycan molecule in the prominent cleft
between its two domains. It attacks peptidoglycans (found in the cell walls of
bacteria, especially Gram-positive_bacteria), its natural substrate, between N-
acetylmuramic_acid (NAM) and the fourth carbon atom of N-acetylglucosamine
(NAG).
Shorter saccharides like tetrasaccharide have also shown to be viable
substrates but via an intermediate with a longer chain.[9] Chitin has also been
shown to be a viable lysozyme substrate. Artificial substrates have also been
developed and used in lysozyme.[10]
**** Mechanism[edit] ****
*** Phillips[edit] ***
The Phillips Mechanism proposed that the enzyme's catalytic power came from
both steric strain on the bound substrate and electrostatic stabilization of an
oxo-carbenium intermediate. From X-ray crystallographic data, Phillips proposed
the active site of the enzyme, where a hexasaccharide binds. The lysozyme
distorts the fourth sugar (in the D or -1 subsite) in the hexasaccharide into a
half-chair conformation. In this stressed state, the glycosidic bond is more
easily broken.[11] An ionic intermediate containing an oxo-carbenium is created
as a result of the glycosidic bond breaking.[12] Thus distortion causing the
substrate molecule to adopt a strained conformation similar to that of the
transition state will lower the energy barrier of the reaction.[13]
The proposed oxo-carbonium intermediate was speculated to be electrostatically
stabilized by aspartate and glutamate residues in the active site by Arieh
Warshel in 1978. The electrostatic stabilization argument was based on
comparison to bulk water, the reorientation of water dipoles can cancel out the
stabilizing energy of charge interaction. In Warshel's model, the enzyme acts
as a super-sovlent, which fixes the orientation of ion pairs and provides
super-solvation (very good stabilization of ion pairs), and especially lower
the energy when to ions are close to each other.[14]
The rate-determining_step(RDS) in this mechanism is related to formation of the
oxo-carbenium intermediate. There were some contradictory results to indicate
the exact RDS. By tracing the formation of product (p-nitrophenol), it was
discovered that the RDS can change over different temperatures, which was a
reason for those contradictory results. At a higher temperature the RDS is
formation of glycosyl enzyme intermediate and at a lower temperature the break
down of that intermediate.[15]
Covalent intermediate of lysozyme enzyme, with covalent bond in black and
experimental evidence as blue mesh. [16]
*** Koshland[edit] ***
Substrates in Vocadlo's experiment
In an early debate in 1969, Dahlquist proposed a covalent mechanism for
lysozyme based on kinetic_isotope_effect,[17] but for a long time the ionic
mechanism was more accepted. In 2001, a revised mechanism was proposed by
Vocadlo via a covalent but not ionic intermediate. Evidence from ESI-MS
analysis indicated a covalent intermediate. A 2-fluoro substituted substrate
was used to lower the reaction rate and accumulate an intermediate for
characterization.[18] The amino acid side-chains glutamic acid 35 (Glu35) and
aspartate 52 (Asp52) have been found to be critical to the activity of this
enzyme. Glu35 acts as a proton donor to the glycosidic bond, cleaving the C-
O bond in the substrate, whereas Asp52 acts as a nucleophile to generate a
glycosyl enzyme intermediate. The Glu35 reacts with water to form hydroxyl ion,
a stronger nucleophile than water, which then attacks the glycosyl enzyme
intermediate, to give the product of hydrolysis and leaving the enzyme
unchanged.[19] This covalent mechanism was named after Koshland, who first
proposed this type of mechanism.[20]
More recently, quantum mechanics/ molecular mechanics (QM/MM) molecular
dynamics simulations have been using the crystal of HEWL and predict the
existence of a covalent intermediate.[21] Evidence for the ESI-MS and X-ray
structures indicate the existence of covalent intermediate, but primarily rely
on using a less active mutant or non-native substrate. Thus, QM/MM molecular
dynamics provides the unique ability to directly investigate the mechanism of
wild-type HEWL and native substrate. The calculations revealed that the
covalent intermediate from the Koshland mechanism is ~30 kcal/mol more stable
than the ionic intermediate from the Phillips mechanism.[21] These calculation
demonstrate that the ionic intermediate is extremely energetically unfavorable
and the covalent intermediates observed from experiments using less active
mutant or non-native substrates provide useful insight into the mechanism of
wild-type HEWL.
Two Possible Mechanisms of Lysozyme
**** Inhibition[edit] ****
Imidazole derivatives can form a charge-transfer_complex with some residues (in
or outside active center) to achieve a competitive inhibition of lysozyme.[22]
In Gram-negative_bacteria, the lipopolysaccharide acts as a non-competitive
inhibitior by highly-favored binding with lysozyme.[23]
Further information: Glycoside_hydrolase
**** Non-enzymatic action[edit] ****
Despite that the muramidase activity of lysozyme has been supposed to play the
key role for its antibacterial properties, evidence of its non-enzymatic action
was also reported. For example, blocking the catalytic activity of lysozyme by
mutation of critical amino acid in the active site (52-Asp -> 52-Ser) does not
eliminate its antimicrobial activity.[24] The lectin-like ability of lysozyme
to recognize bacterial carbohydrate antigen without lytic activity was reported
for tetrasaccharide related to lipopolysaccharide of Klebsiella_pneumoniae.[25]
Also, lysozyme interacts with antibodies and T-cell_receptors.[26]
**** Enzyme conformation changes[edit] ****
Lysozyme exhibits two conformations: an open active state and a closed inactive
state. The catalytic relevance was examined with single walled carbon_nanotubes
(SWCN) field effect transitors (FETs), where a singular lysozyme was bound to
the SWCN FET.[27] Electronically monitoring the lysozyme showed two
conformations, an open active site and a closed inactive site. In its active
state lysozyme is able to processively hydrolyze its substrate, breaking on
average 100 bonds at a rate of 15 per second. In order to bind a new substrate
and move from the closed inactive state to the open active state requires two
conformation step changes, while inactivation requires one step.
***** Role in disease and therapy[edit] *****
LYZ
[Lysozyme.png]
Available structures
    Ortholog search: PDBe RCSB
    List of PDB id codes
    133L, 134L, 1B5U, 1B5V, 1B5W, 1B5X, 1B5Y, 1B5Z, 1B7L, 1B7M,
    1B7N, 1B7O, 1B7P, 1B7Q, 1B7R, 1B7S, 1BB3, 1BB4, 1BB5, 1C43,
    1C45, 1C46, 1C7P, 1CJ6, 1CJ7, 1CJ8, 1CJ9, 1CKC, 1CKD, 1CKF,
    1CKG, 1CKH, 1D6P, 1D6Q, 1DI3, 1DI4, 1DI5, 1EQ4, 1EQ5, 1EQE,
    1GAY, 1GAZ, 1GB0, 1GB2, 1GB3, 1GB5, 1GB6, 1GB7, 1GB8, 1GB9,
    1GBO, 1GBW, 1GBX, 1GBY, 1GBZ, 1GDW, 1GDX, 1GE0, 1GE1, 1GE2,
    1GE3, 1GE4, 1GEV, 1GEZ, 1GF0, 1GF3, 1GF4, 1GF5, 1GF6, 1GF7,
    1GF8, 1GF9, 1GFA, 1GFE, 1GFG, 1GFH, 1GFJ, 1GFK, 1GFR, 1GFT,
    1GFU, 1GFV, 1HNL, 1I1Z, 1I20, 1I22, 1INU, 1IOC, 1IP1, 1IP2,
PDB 1IP3, 1IP4, 1IP5, 1IP6, 1IP7, 1IWT, 1IWU, 1IWV, 1IWW, 1IWX,
    1IWY, 1IWZ, 1IX0, 1IY3, 1IY4, 1JKA, 1JKB, 1JKC, 1JKD, 1JSF,
    1JWR, 1LAA, 1LHH, 1LHI, 1LHJ, 1LHK, 1LHL, 1LHM, 1LMT, 1LOZ,
    1LYY, 1LZ1, 1LZ4, 1LZ5, 1LZ6, 1LZR, 1LZS, 1OP9, 1OUA, 1OUB,
    1OUC, 1OUD, 1OUE, 1OUF, 1OUG, 1OUH, 1OUI, 1OUJ, 1QSW, 1RE2,
    1REM, 1REX, 1REY, 1REZ, 1TAY, 1TBY, 1TCY, 1TDY, 1UBZ, 1W08,
    1WQM, 1WQN, 1WQO, 1WQP, 1WQQ, 1WQR, 1YAM, 1YAN, 1YAO, 1YAP,
    1YAQ, 207L, 208L, 2BQA, 2BQB, 2BQC, 2BQD, 2BQE, 2BQF, 2BQG,
    2BQH, 2BQI, 2BQJ, 2BQK, 2BQL, 2BQM, 2BQN, 2BQO, 2HEA, 2HEB,
    2HEC, 2HED, 2HEE, 2HEF, 2LHM, 2MEA, 2MEB, 2MEC, 2MED, 2MEE,
    2MEF, 2MEG, 2MEH, 2MEI, 2NWD, 2ZIJ, 2ZIK, 2ZIL, 2ZWB, 3EBA,
    3FE0, 3LHM, 3LN2, 4I0C, 4ML7, 4R0P
Identifiers
Aliases          LYZ, LZM, LYZF1, lysozyme
External IDs     OMIM: 153450 MGI: 96902 HomoloGene: 121490 GeneCards: LYZ
Gene location (Human)
[Chromosome_12_(human)]
Chr. Chromosome_12_(human)[28]
            [Chromosome_12_(human)]
           [Genomic_location_for_LYZ]
           [Genomic_location_for_LYZ]
Band 12q15 Start 69,348,341 bp[28]
           End   69,354,234 bp[28]
Gene location (Mouse)
[Chromosome_10_(mouse)]
Chr. Chromosome 10 (mouse)[29]
            [Chromosome_10_(mouse)]
           [Genomic_location_for_LYZ]
           [Genomic_location_for_LYZ]
Band 10|10 D2 Start 117,287,797 bp[29]
              End   117,292,868 bp[29]
RNA_expression pattern
[PBB_GE_LYZ_213975_s_at_fs.png]
More_reference_expression_data
Gene_ontology
                   â¢ catalytic_activity
                   â¢ lysozyme_activity
Molecular function â¢ hydrolase_activity
                   â¢ hydrolase_activity,_acting_on_glycosyl_bonds
                   â¢ identical_protein_binding
                   â¢ extracellular_region
                   â¢ extracellular_exosome
Cellular component â¢ extracellular_space
                   â¢ azurophil_granule_lumen
                   â¢ specific_granule_lumen
                   â¢ tertiary_granule_lumen
                   â¢ cellular_protein_metabolic_process
                   â¢ metabolism
                   â¢ retina_homeostasis
                   â¢ inflammatory_response
                   â¢ cytolysis
Biological process â¢ defense_response_to_bacterium
                   â¢ antimicrobial_humoral_response
                   â¢ neutrophil_degranulation
                   â¢ cell_wall_macromolecule_catabolic_process
                   â¢ defense_response_to_Gram-negative_bacterium
                   â¢ defense_response_to_Gram-positive_bacterium
Sources:Amigo / QuickGO
Orthologs
Species          Human                      Mouse
Entrez           4069                       17110
Ensembl          ENSG00000090382            ENSMUSG00000069515
UniProt          P61626                     P17897
RefSeq (mRNA)    NM_000239                  NM_013590
RefSeq (protein) NP_000230                  NP_038618
Location (UCSC)  Chr_12:_69.35_â_69.35_Chr_10:_117.29_â_117.29_Mb
PubMed search    [30]                       [31]
Wikidata
View/Edit_Human View/Edit_Mouse
Lysozyme is part of the innate immune system. Reduced lysozyme levels have been
associated with bronchopulmonary_dysplasia in newborns.[32] Piglets fed with
human lysozyme milk can recover from diarrheal disease caused by E. coli
faster. The concentration of lysozyme in human milk is 1,600 to 3,000 times
greater than the concentration in livestock milk. Human lysozyme is more active
than hen egg white lysozyme. A transgenic line of goats (with a founder named
"Artemis") were developed to produce milk with human lysozyme to protect
children from diarrhea if they can't get the benefits of human breastfeeding.
[33][34]
Since lysozyme is a natural form of protection from Gram-positive pathogens
like Bacillus and Streptococcus,[35] it plays an important role in immunology
of infants in human milk feeding.[36] Whereas the skin is a protective barrier
due to its dryness and acidity, the conjunctiva (membrane covering the eye) is,
instead, protected by secreted enzymes, mainly lysozyme and defensin. However,
when these protective barriers fail, conjunctivitis results.
In certain cancers (especially myelomonocytic leukemia) excessive production of
lysozyme by cancer cells can lead to toxic levels of lysozyme in the blood.
High lysozyme blood levels can lead to kidney failure and low blood potassium,
conditions that may improve or resolve with treatment of the primary
malignancy.
Serum lysozyme is much less specific for diagnosis of sarcoidosis than serum
angiotensin converting enzyme; however, since it is more sensitive, it is used
as a marker of sarcoidosis disease activity and is suitable for disease
monitoring in proven cases.[37]
***** Chemical synthesis[edit] *****
The first chemical synthesis of a lysozyme protein was attempted by Prof.
George W. Kenner and his group at the University of Liverpool in England.[38]
This was finally achieved in 2007 by Steve Kent at the University of Chicago
who made a synthetic functional lysozyme molecule.[39]
***** Other applications[edit] *****
Lysozyme crystals have been used to grow other functional materials for
catalysis and biomedical applications.[40][41][42] Lysozyme is a commonly used
enzyme for lysing gram positive bacteria.[43] Due to the unique function of
lysozyme in which it can digest the cell wall and causes osmotic_shock (burst
the cell by suddenly changing solute concentration around the cell and thus the
osmotic_pressure), lysozyme is commonly used in lab setting to release proteins
from bacterium periplasm while the inner membrane remains sealed as vesicles
called the spheroplast.[44][45]
For example, E. coli can be lysed using lysozyme to free the contents of the
periplasmic space. It is especially useful in lab setting for trying to collect
the contents of the periplasm.[1] Lysozyme treatment is optimal at particular
temperatures, pH ranges, and salt concentrations. Lysozyme activity increases
with increasing temperatures, up to 60 degrees Celsius, with a pH range of 6.0-
7.0. The salts present also affect lysozyme treatment, where some assert
inhibitory effects, and others promote lysis via lysozyme treatment. Sodium
chloride induces lysis, but at high concentrations, it is an active inhibitor
of lysis. Similar observations have been seen with the use of potassium salts.
Slight variations are present due to differences in bacterial strains.[46]
***** History[edit] *****
The antibacterial property of hen egg white, due to the lysozyme it contains,
was first observed by Laschtschenko in 1909,[47] although it was not until 1922
that the name 'lysozyme' was coined, by Alexander_Fleming, the second scientist
to discover penicillin.[48] [49] But according to one publication, it was
Fleming "who first clearly showed that an enzymic substance present in a wide
variety of secretions is capable of rapidly lysing (ie.., dissolving) certain
bacteria, particularly a yellow "coccus" that he studied". [50] Fleming first
observed the antibacterial action of lysozyme when he treated bacterial
cultures with nasal mucus from a patient suffering from a head_cold.[49]
Lysozyme was first crystallised by Edward_Abraham in 1937 enabling the three-
dimensional structure of hen egg white lysozyme to be described by David
Chilton_Phillips in 1965, when he obtained the first 2-Ã¥ngstrÃ¶m (200 pm)
resolution model via X-ray_crystallography.[51][52] The structure was publicly
presented at a Royal_Institution lecture in 1965.[53] Lysozyme was the second
protein structure and the first enzyme structure to be solved via X-ray
diffraction methods, and the first enzyme to be fully sequenced that contains
all twenty common amino acids.[54] As a result of Phillips' elucidation of the
structure of lysozyme, it was also the first enzyme to have a detailed,
specific mechanism suggested for its method of catalytic action.[55][56][57]
This work led Phillips to provide an explanation for how enzymes speed up a
chemical reaction in terms of its physical structures. The original mechanism
proposed by Phillips was more recently revised.[18]
***** See also[edit] *****
    * Egg_allergy
***** References[edit] *****
   1. ^ a bManchenko GP (1994). "Lysozyme". Handbook of Detection of Enzymes on
      Electrophoretic Gels. Boca Raton, Fla.: CRC Press. p. 223. ISBN 978-0-
      8493-8935-1.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Williams S, Vocadlo D. "Glycoside_hydrolase_family_22". Cazypedia.
      Retrieved 11 April 2017.
   4. ^Yoshimura K, Toibana A, Nakahama K (January 1988). "Human lysozyme:
      sequencing of a cDNA, and expression and secretion by Saccharomyces
      cerevisiae". Biochemical and Biophysical Research Communications. 150
      (2): 794â801. doi:10.1016/0006-291X(88)90461-5. PMID 2829884.
   5. ^Peters CW, Kruse U, Pollwein R, Grzeschik KH, Sippel AE (July 1989).
      "The human lysozyme gene. Sequence organization and chromosomal
      localization". European Journal of Biochemistry. 182 (3): 507â16. doi:
      10.1111/j.1432-1033.1989.tb14857.x. PMID 2546758.
   6. ^Venkataramani S, Truntzer J, Coleman DR (April 2013). "Thermal_stability
      of_high_concentration_lysozyme_across_varying_pH:_A_Fourier_Transform
      Infrared_study". Journal of Pharmacy & Bioallied Sciences. 5 (2):
      148â53. doi:10.4103/0975-7406.111821. PMC 3697194. PMID 23833521.
   7. ^Chandan RC, Shahani KM, Holly RG (October 1964). "Lysozyme Content of
      Human Milk". Nature. 204 (4953): 76â7. doi:10.1038/204076a0.
      PMID 14240122.
   8. ^"Lysozyme,_Product_information" (PDF). Sigma-Aldrich.
   9. ^SkujiÅÅ J, PuÄ·ite A, McLaren AD (December 1973). "Adsorption and
      reactions of chitinase and lysozyme on chitin". Molecular and Cellular
      Biochemistry. 2 (2): 221â8. doi:10.1007/BF01795475. PMID 4359167.
  10. ^Sharon N (April 1967). "The chemical structure of lysozyme substrates
      and their cleavage by the enzyme". Proceedings of the Royal Society of
      London. Series B, Biological Sciences. 167 (1009): 402â15. doi:10.1098/
      rspb.1967.0037. PMID 4382803.
  11. ^HÃ¶ltje JV (1 January 1996). "Lysozyme substrates". Exs. 75: 105â10.
      PMID 8765297.
  12. ^Blake CC, Johnson LN, Mair GA, North AC, Phillips DC, Sarma VR (April
      1967). "Crystallographic studies of the activity of hen egg-white
      lysozyme". Proceedings of the Royal Society of London. Series B,
      Biological Sciences. 167 (1009): 378â88. doi:10.1098/rspb.1967.0035.
      PMID 4382801.
  13. ^Dahlquist FW, Rand-Meir T, Raftery MA (October 1969). "Application of
      secondary alpha-deuterium kinetic isotope effects to studies of enzyme
      catalysis. Glycoside hydrolysis by lysozyme and beta-glucosidase".
      Biochemistry. 8 (10): 4214â21. doi:10.1021/bi00838a045. PMID 5388150.
  14. ^McKenzie HA, White FH (1991). "Lysozyme_and_alpha-lactalbumin:
      structure,_function,_and_interrelationships". Advances in Protein
      Chemistry. 41: 173â315. doi:10.1016/s0065-3233(08)60198-9.
      ISBN 9780120342419. PMID 2069076.
  15. ^Warshel A (November 1978). "Energetics_of_enzyme_catalysis". Proceedings
      of the National Academy of Sciences of the United States of America. 75
      (11): 5250â4. doi:10.1073/pnas.75.11.5250. PMC 392938. PMID 281676.
  16. ^Weber JP, Fink AL (October 1980). "Temperature-dependent change in the
      rate-limiting step of beta-glucosidase catalysis". The Journal of
      Biological Chemistry. 255 (19): 9030â2. PMID 6773958.
  17. ^ http://proteopedia.org/wiki/index.php/
      Lysozyme#Covalent_intermediate_and_product_complex
  18. ^Dahlquist FW, Rand-Meir T, Raftery MA (October 1969). "Application of
      secondary alpha-deuterium kinetic isotope effects to studies of enzyme
      catalysis. Glycoside hydrolysis by lysozyme and beta-glucosidase".
      Biochemistry. 8 (10): 4214â21. doi:10.1021/bi00838a045. PMID 5388150.
  19. ^ a bVocadlo DJ, Davies GJ, Laine R, Withers SG (August 2001). "Catalysis
      by_hen_egg-white_lysozyme_proceeds_via_a_covalent_intermediate" (PDF).
      Nature. 412 (6849): 835â8. doi:10.1038/35090602. PMID 11518970.
  20. ^Grisham CM, Garrett RH (2007). "Chapter_14:_Mechanism_of_enzyme_action".
      Biochemistry. Australia: Thomson Brooks/Cole. pp. 467â9. ISBN 978-0-
      495-11912-8.
  21. ^Koshland DE (November 1953). "Stereochemistry and the Mechanism of
      Enzymatic Reactions". Biological Reviews. 28 (4): 416â436. doi:10.1111/
      j.1469-185X.1953.tb01386.x.
  22. ^ a bBowman AL, Grant IM, Mulholland AJ (October 2008). "QM/MM
      simulations predict a covalent intermediate in the hen egg white lysozyme
      reaction with its natural substrate". Chemical Communications (37):
      4425â7. doi:10.1039/b810099c. PMID 18802578.
  23. ^Swan ID (March 1972). "The inhibition of hen egg-white lysozyme by
      imidazole and indole derivatives". Journal of Molecular Biology. 65 (1):
      59â62. doi:10.1016/0022-2836(72)90491-3. PMID 5063023.
  24. ^Ohno N, Morrison DC (March 1989). "Lipopolysaccharide interaction with
      lysozyme. Binding of lipopolysaccharide to lysozyme and inhibition of
      lysozyme enzymatic activity". The Journal of Biological Chemistry. 264
      (8): 4434â41. PMID 2647736.
  25. ^Ibrahim HR, Matsuzaki T, Aoki T (2001). "Genetic evidence that
      antibacterial activity of lysozyme is independent of its catalytic
      function". FEBS Letters. 506 (1): 27â32. doi:10.1016/S0014-5793
      (01)02872-1. PMID 11591365.
  26. ^Zhang R, Wu L, Eckert T, Burg-Roderfeld M, Rojas-Macias MA, LÃ¼tteke T
      (2017). "Lysozyme's lectin-like characteristics facilitates its immune
      defense function". Quarterly Reviews of Biophysics. 50: e9. doi:10.1017/
      S0033583517000075. PMID 29233221.
  27. ^Grivel JC, Smith-Gill SJ (1996). Lysozyme: Antigenic structure as
      defined by antibody and T cell responses. CRC Press. pp. 91â144.
      ISBN 978-0-8493-9225-2.
  28. ^Choi Y, Moody IS, Sims PC, Hunt SR, Corso BL, Perez I, Weiss GA, Collins
      PG (January 2012). "Single-molecule_lysozyme_dynamics_monitored_by_an
      electronic_circuit". Science. 335 (6066): 319â24. doi:10.1126/
      science.1214824. PMC 3914775. PMID 22267809.
  29. ^ a b c GRCh38:_Ensembl_release_89:_ENSG00000090382 - Ensembl, May 2017
  30. ^ a b c GRCm38:_Ensembl_release_89:_ENSMUSG00000069515 - Ensembl, May
      2017
  31. ^"Human_PubMed_Reference:".
  32. ^"Mouse_PubMed_Reference:".
  33. ^Revenis ME, Kaliner MA (August 1992). "Lactoferrin_and_lysozyme
      deficiency_in_airway_secretions:_association_with_the_development_of
      bronchopulmonary_dysplasia". The Journal of Pediatrics. 121 (2):
      262â70. doi:10.1016/S0022-3476(05)81201-6. PMID 1640295.
  34. ^Cooper CA, Garas Klobas LC, Maga EA, Murray JD (2013). "Consuming
      transgenic_goats'_milk_containing_the_antimicrobial_protein_lysozyme
      helps_resolve_diarrhea_in_young_pigs". PLOS ONE. 8 (3): e58409. doi:
      10.1371/journal.pone.0058409. PMC 3596375. PMID 23516474.
  35. ^Molteni M (30 June 2016). "Spilled_Milk". Case Studies: News Features.
      Undark: Truth, Beauty, Science. Retrieved 12 January 2017.
  36. ^Nester EW, Anderson DG, Roberts CE, Nester MT (2007). Microbiology: A
      Human Perspective (5th ed.). Boston, Mass.: McGraw-Hill Higher Education.
      ISBN 978-0-07-110706-8.
  37. ^Chandra RK (September 1978). "Immunological aspects of human milk".
      Nutrition Reviews. 36 (9): 265â72. doi:10.1111/j.1753-
      4887.1978.tb07393.x. PMID 362248.
  38. ^Tomita H, Sato S, Matsuda R, Sugiura Y, Kawaguchi H, Niimi T, Yoshida S,
      Morishita M (1999). "Serum lysozyme levels and clinical features of
      sarcoidosis". Lung. 177 (3): 161â7. doi:10.1007/pl00007637.
      PMID 10192763.
  39. ^Kenner GW (June 1977). "The Bakerian lecture. Towards synthesis of
      proteins". Proceedings of the Royal Society of London. Series B,
      Biological Sciences. 197 (1128): 237â53. doi:10.1098/rspb.1977.0068.
      PMID 19745.
  40. ^Durek T, Torbeev VY, Kent SB (March 2007). "Convergent_chemical
      synthesis_and_high-resolution_x-ray_structure_of_human_lysozyme".
      Proceedings of the National Academy of Sciences of the United States of
      America. 104 (12): 4846â51. doi:10.1073/pnas.0610630104. PMC 1829227.
      PMID 17360367.
  41. ^Wei H, Wang Z, Zhang J, House S, Gao YG, Yang L, Robinson H, Tan LH,
      Xing H, Hou C, Robertson IM, Zuo JM, Lu Y (February 2011). "Time-
      dependent, protein-directed growth of gold nanoparticles within a single
      crystal of lysozyme". Nature Nanotechnology. 6 (2): 93â7. doi:10.1038/
      nnano.2010.280. PMID 21278750.
  42. ^Sanghamitra NJ, Ueno T (May 2013). "Expanding coordination chemistry
      from protein to protein assembly". Chemical Communications. 49 (39):
      4114â26. doi:10.1039/C2CC36935D. PMID 23211931.
  43. ^Ueno T (July 2013). "Porous protein crystals as reaction vessels".
      Chemistry. 19 (28): 9096â102. doi:10.1002/chem.201300250.
      PMID 23813903.
  44. ^Repaske R (October 1956). "Lysis of gram-negative bacteria by lysozyme".
      Biochimica et Biophysica Acta. 22 (1): 189â91. doi:10.1016/0006-3002
      (56)90240-2. PMID 13373865.
  45. ^Gunton J, Shiryayev A, Pagan DL (2007). Protein Condensation : Kinetic
      Pathways to Crystallization and Disease. Cambridge: Cambridge University
      Press. pp. 156â158. ISBN 9780511535321.
  46. ^Ninfa A, Ballou D, Benore M (2010). Fundamental Laboratory Approaches
      for Biochemistry and Biotechnology. John Wiley. ISBN 978-0470087664.
  47. ^Salton MR (June 1957). "The_properties_of_lysozyme_and_its_action_on
      microorganisms". Bacteriological Reviews. 21 (2): 82â100. PMC 180888.
      PMID 13436356.
  48. ^Laschtschenko P (1909). "Ãber die keimtÃ¶tende und entwicklungshemmende
      Wirkung HÃ¼hnereiweiÃ" [On the germ-killing and growth-inhibiting effect
      chicken egg albumin]. Z. Hyg. InfektKrankh. (in German). 64: 419â427.
      doi:10.1007/BF02216170.
  49. ^"Duckett_S._Ernest_Duchesne_and_the_concept_of_fungal_antibiotic
      therapy._The_Lancet_Department_Of_Medical_History_354(9195):2068-2071,
      December_11,_1999".
  50. ^ a bFleming A (May 1922). "On a remarkable bacteriolytic element found
      in tissues and secretions". Proceedings_of_the_Royal_Society_B. 93 (653):
      306â317. doi:10.1098/rspb.1922.0023. JSTOR 80959.
  51. ^Advances_in_Protein_Chemistry. Academic Press. 13 June 1991. pp. 176â.
      ISBN 978-0-08-058214-6.
  52. ^Blake CC, Koenig DF, Mair GA, North AC, Phillips DC, Sarma VR (May
      1965). "Structure of hen egg-white lysozyme. A three-dimensional Fourier
      synthesis at 2 Angstrom resolution". Nature. 206 (4986): 757â61. doi:
      10.1038/206757a0. PMID 5891407.
  53. ^Johnson LN, Phillips DC (May 1965). "Structure of some crystalline
      lysozyme-inhibitor complexes determined by X-ray analysis at 6 Angstrom
      resolution". Nature. 206 (4986): 761â3. doi:10.1038/206761a0.
      PMID 5840126.
  54. ^Johnson LN (November 1998). "The early history of lysozyme". Nature
      Structural Biology. 5 (11): 942â4. doi:10.1038/2917. PMID 9808036.
  55. ^Canfield RE (August 1963). "The_Amino_Acid_Sequence_of_Egg_White
      Lysozyme". The Journal of Biological Chemistry. 238 (8): 2698â707.
      PMID 14063294.
  56. ^Vernon CA (April 1967). "The mechanisms of hydrolysis of glycosides and
      their revelance to enzyme-catalysed reactions". Proceedings of the Royal
      Society of London. Series B, Biological Sciences. 167 (1009): 389â401.
      doi:10.1098/rspb.1967.0036. JSTOR 75680. PMID 4382802.
  57. ^Rupley JA (April 1967). "The binding and cleavage by lysozyme of N-
      acetylglucosamine oligosaccharides". Proceedings of the Royal Society of
      London. Series B, Biological Sciences. 167 (1009): 416â28. doi:10.1098/
      rspb.1967.0038. JSTOR 75682. PMID 4382804.
  58. ^Sharon N (April 1967). "The chemical structure of lysozyme substrates
      and their cleavage by the enzyme". Proceedings of the Royal Society of
      London. Series B, Biological Sciences. 167 (1009): 402â15. doi:10.1098/
      rspb.1967.0037. JSTOR 75681. PMID 4382803.
***** External links[edit] *****
    * Muramidase at the US National Library of Medicine Medical_Subject
      Headings (MeSH)
    * Proteopedia.org_HEW_Lysozyme
    * PDBe-KB provides an overview of all the structure information available
      in the PDB for Human Lysozyme C.
    * PDBe-KB provides an overview of all the structure information available
      in the PDB for Hen egg white Lysozyme C.

    * v
    * t
    * e
PDB gallery
    * 133l: ROLE OF ARG 115 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME. X-RAY
      STRUCTURE OF HIS 115 AND GLU 115 MUTANTS
    * 134l: ROLE OF ARG 115 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME. X-RAY
      STRUCTURE OF HIS 115 AND GLU 115 MUTANTS
    * 1b5u: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME: CALORIMETRY AND X-RAY ANALYSIS OF SIX SER->ALA MUTANT
    * 1b5v: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME: CALORIMETRY AND X-RAY ANALYSIS OF SIX SER->ALA MUTANTS
    * 1b5w: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME: CALORIMETRY AND X-RAY ANALYSIS OF SIX SER->ALA MUTANTS
    * 1b5x: Contribution of hydrogen bonds to the conformational stability of
      human lysozyme: calorimetry and x-ray analysis of six ser->ala mutants
    * 1b5y: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME: CALORIMETRY AND X-RAY ANALYSIS OF SIX SER->ALA MUTANTS
    * 1b5z: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME: CALORIMETRY AND X-RAY ANALYSIS OF SIX SER->ALA MUTANTS
    * 1b7l: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7m: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7n: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7o: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7p: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7q: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7r: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1b7s: VERIFICATION OF SPMP USING MUTANT HUMAN LYSOZYMES
    * 1bb3: HUMAN LYSOZYME MUTANT A96L
    * 1bb4: HUMAN LYSOZYME DOUBLE MUTANT A96L, W109H
    * 1bb5: HUMAN LYSOZYME MUTANT A96L COMPLEXED WITH CHITOTRIOSE
    * 1c43: MUTANT HUMAN LYSOZYME WITH FOREIGN N-TERMINAL RESIDUES
    * 1c45: MUTANT HUMAN LYSOZYME WITH FOREIGN N-TERMINAL RESIDUES
    * 1c46: MUTANT HUMAN LYSOZYME WITH FOREIGN N-TERMINAL RESIDUES
    * 1c7p: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME WITH FOUR EXTRA
      RESIDUES (EAEA) AT THE N-TERMINAL
    * 1cj6: T11A MUTANT HUMAN LYSOZYME
    * 1cj7: T11V MUTANT HUMAN LYSOZYME
    * 1cj8: T40A MUTANT HUMAN LYSOZYME
    * 1cj9: T40V MUTANT HUMAN LYSOZYME
    * 1ckc: T43A MUTANT HUMAN LYSOZYME
    * 1ckd: T43V MUTANT HUMAN LYSOZYME
    * 1ckf: T52A MUTANT HUMAN LYSOZYME
    * 1ckg: T52V MUTANT HUMAN LYSOZYME
    * 1ckh: T70V MUTANT HUMAN LYSOZYME
    * 1d6p: HUMAN LYSOZYME L63 MUTANT LABELLED WITH 2',3'-EPOXYPROPYL N,N'-
      DIACETYLCHITOBIOSE
    * 1d6q: HUMAN LYSOZYME E102 MUTANT LABELLED WITH 2',3'-EPOXYPROPYL
      GLYCOSIDE OF N-ACETYLLACTOSAMINE
    * 1di3: ROLE OF AMINO ACID RESIDUES AT TURNS IN THE CONFORMATIONAL
      STABILITY AND FOLDING OF HUMAN LYSOZYME
    * 1di4: ROLE OF AMINO ACID RESIDUES AT TURNS IN THE CONFORMATIONAL
      STABILITY AND FOLDING OF HUMAN LYSOZYME
    * 1di5: ROLE OF AMINO ACID RESIDUES AT TURNS IN THE CONFORMATIONAL
      STABILITY AND FOLDING OF HUMAN LYSOZYME
    * 1eq4: CRYSTAL STRUCTURES OF SALT BRIDGE MUTANTS OF HUMAN LYSOZYME
    * 1eq5: CRYSTAL STRUCTURES OF SALT BRIDGE MUTANTS OF HUMAN LYSOZYME
    * 1eqe: CRYSTAL STRUCTURES OF SALT BRIDGE MUTANTS OF HUMAN LYSOZYME
    * 1gay: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gaz: Crystal Structure of Mutant Human Lysozyme Substituted at the
      Surface Positions
    * 1gb0: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb2: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb3: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb5: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb6: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb7: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb8: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gb9: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gbo: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gbw: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gbx: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gby: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gbz: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gdw: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1gdx: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1ge0: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1ge1: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1ge2: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1ge3: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1ge4: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT LEFT-
      HANDED HELICAL POSITIONS
    * 1gev: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gez: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf0: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf3: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf4: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf5: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf6: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf7: BURIED POLAR MUTANT HUMAN LYSOZYME
    * 1gf8: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gf9: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfa: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfe: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfg: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfh: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfj: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfk: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfr: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gft: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfu: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1gfv: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1hnl: CRYSTAL STRUCTURE OF A GLUTATHIONYLATED HUMAN LYSOZYME: A FOLDING
      INTERMEDIATE MIMIC IN THE FORMATION OF A DISULFIDE BOND
    * 1i1z: MUTANT HUMAN LYSOZYME (Q86D)
    * 1i20: MUTANT HUMAN LYSOZYME (A92D)
    * 1i22: MUTANT HUMAN LYSOZYME (A83K/Q86D/A92D)
    * 1inu: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME SUBSTITUTED AT THE
      SURFACE POSITIONS
    * 1ioc: CRYSTAL STRUCTURE OF MUTANT HUMAN LYSOZYME, EAEA-I56T
    * 1ip1: G37A HUMAN LYSOZYME
    * 1ip2: G48A HUMAN LYSOZYME
    * 1ip3: G68A HUMAN LYSOZYME
    * 1ip4: G72A HUMAN LYSOZYME
    * 1ip5: G105A HUMAN LYSOZYME
    * 1ip6: G127A HUMAN LYSOZYME
    * 1ip7: G129A HUMAN LYSOZYME
    * 1iwt: Crystal Structure Analysis of Human lysozyme at 113K.
    * 1iwu: Crystal Structure Analysis of Human lysozyme at 127K.
    * 1iwv: Crystal Structure Analysis of Human lysozyme at 147K.
    * 1iww: Crystal Structure Analysis of Human lysozyme at 152K.
    * 1iwx: Crystal Structure Analysis of Human lysozyme at 161K.
    * 1iwy: Crystal Structure Analysis of Human lysozyme at 170K.
    * 1iwz: Crystal Structure Analysis of Human lysozyme at 178K.
    * 1ix0: I59A-3SS human lysozyme
    * 1iy3: Solution Structure of the Human lysozyme at 4 degree C
    * 1iy4: Solution structure of the human lysozyme at 35 degree C
    * 1jka: HUMAN LYSOZYME MUTANT WITH GLU 35 REPLACED BY ASP
    * 1jkb: HUMAN LYSOZYME MUTANT WITH GLU 35 REPLACED BY ALA
    * 1jkc: HUMAN LYSOZYME MUTANT WITH TRP 109 REPLACED BY PHE
    * 1jkd: HUMAN LYSOZYME MUTANT WITH TRP 109 REPLACED BY ALA
    * 1jsf: FULL-MATRIX LEAST-SQUARES REFINEMENT OF HUMAN LYSOZYME
    * 1jwr: Crystal structure of human lysozyme at 100 K
    * 1laa: X-RAY STRUCTURE OF GLU 53 HUMAN LYSOZYME
    * 1lhh: ROLE OF PROLINE RESIDUES IN HUMAN LYSOZYME STABILITY: A SCANNING
      CALORIMETRIC STUDY COMBINED WITH X-RAY STRUCTURE ANALYSIS OF PROLINE
      MUTANTS
    * 1lhi: ROLE OF PROLINE RESIDUES IN HUMAN LYSOZYME STABILITY: A SCANNING
      CALORIMETRIC STUDY COMBINED WITH X-RAY STRUCTURE ANALYSIS OF PROLINE
      MUTANTS
    * 1lhj: ROLE OF PROLINE RESIDUES IN HUMAN LYSOZYME STABILITY: A SCANNING
      CALORIMETRIC STUDY COMBINED WITH X-RAY STRUCTURE ANALYSIS OF PROLINE
      MUTANTS
    * 1lhk: ROLE OF PROLINE RESIDUES IN HUMAN LYSOZYME STABILITY: A SCANNING
      CALORIMETRIC STUDY COMBINED WITH X-RAY STRUCTURE ANALYSIS OF PROLINE
      MUTANTS
    * 1lhl: ROLE OF PROLINE RESIDUES IN HUMAN LYSOZYME STABILITY: A SCANNING
      CALORIMETRIC STUDY COMBINED WITH X-RAY STRUCTURE ANALYSIS OF PROLINE
      MUTANTS
    * 1lhm: THE CRYSTAL STRUCTURE OF A MUTANT LYSOZYME C77(SLASH)95A WITH
      INCREASED SECRETION EFFICIENCY IN YEAST
    * 1lmt: STRUCTURE OF A CONFORMATIONALLY CONSTRAINED ARG-GLY-ASP SEQUENCE
      INSERTED INTO HUMAN LYSOZYME
    * 1loz: AMYLOIDOGENIC VARIANT (I56T) VARIANT OF HUMAN LYSOZYME
    * 1lyy: AMYLOIDOGENIC VARIANT (ASP67HIS) OF HUMAN LYSOZYME
    * 1lz1: REFINEMENT OF HUMAN LYSOZYME AT 1.5 ANGSTROMS RESOLUTION. ANALYSIS
      OF NON-BONDED AND HYDROGEN-BOND INTERACTIONS
    * 1lz4: ENTHALPIC DESTABILIZATION OF A MUTANT HUMAN LYSOZYME LACKING A
      DISULFIDE BRIDGE BETWEEN CYSTEINE-77 AND CYSTEINE-95
    * 1lz5: STRUCTURAL AND FUNCTIONAL ANALYSES OF THE ARG-GLY-ASP SEQUENCE
      INTRODUCED INTO HUMAN LYSOZYME
    * 1lz6: STRUCTURAL AND FUNCTIONAL ANALYSES OF THE ARG-GLY-ASP SEQUENCE
      INTRODUCED INTO HUMAN LYSOZYME
    * 1lzr: STRUCTURAL CHANGES OF THE ACTIVE SITE CLEFT AND DIFFERENT
      SACCHARIDE BINDING MODES IN HUMAN LYSOZYME CO-CRYSTALLIZED WITH HEXA-N-
      ACETYL-CHITOHEXAOSE AT PH 4.0
    * 1lzs: STRUCTURAL CHANGES OF THE ACTIVE SITE CLEFT AND DIFFERENT
      SACCHARIDE BINDING MODES IN HUMAN LYSOZYME CO-CRYSTALLIZED WITH HEXA-N-
      ACETYL-CHITOHEXAOSE AT PH 4.0
    * 1op9: Complex of human lysozyme with camelid VHH HL6 antibody fragment
    * 1oua: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE I56T MUTANT
    * 1oub: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V100A MUTANT
    * 1ouc: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V110A MUTANT
    * 1oud: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V121A MUTANT
    * 1oue: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V125A MUTANT
    * 1ouf: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V130A MUTANT
    * 1oug: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V2A MUTANT
    * 1ouh: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V74A MUTANT
    * 1oui: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V93A MUTANT
    * 1ouj: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: X-RAY STRUCTURE OF THE V99A MUTANT
    * 1qsw: CRYSTAL STRUCTURE ANALYSIS OF A HUMAN LYSOZYME MUTANT W64C C65A
    * 1re2: HUMAN LYSOZYME LABELLED WITH TWO 2',3'-EPOXYPROPYL BETA-GLYCOSIDE
      OF N-ACETYLLACTOSAMINE
    * 1rem: HUMAN LYSOZYME WITH MAN-B1,4-GLCNAC COVALENTLY ATTACHED TO ASP53
    * 1rex: NATIVE HUMAN LYSOZYME
    * 1rey: HUMAN LYSOZYME-N,N'-DIACETYLCHITOBIOSE COMPLEX
    * 1rez: HUMAN LYSOZYME-N-ACETYLLACTOSAMINE COMPLEX
    * 1tay: DISSECTION OF THE FUNCTIONAL ROLE OF STRUCTURAL ELEMENTS OF
      TYROSINE-63 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME
    * 1tby: DISSECTION OF THE FUNCTIONAL ROLE OF STRUCTURAL ELEMENTS OF
      TYROSINE-63 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME
    * 1tcy: DISSECTION OF THE FUNCTIONAL ROLE OF STRUCTURAL ELEMENTS OF
      TYROSINE-63 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME
    * 1tdy: DISSECTION OF THE FUNCTIONAL ROLE OF STRUCTURAL ELEMENTS OF
      TYROSINE-63 IN THE CATALYTIC ACTION OF HUMAN LYSOZYME
    * 1ubz: Crystal structure of Glu102-mutant human lysozyme doubly labeled
      with 2',3'-epoxypropyl beta-glycoside of N-acetyllactosamine
    * 1w08: STRUCTURE OF T70N HUMAN LYSOZYME
    * 1wqm: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1wqn: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1wqo: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1wqp: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1wqq: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1wqr: CONTRIBUTION OF HYDROGEN BONDS TO THE CONFORMATIONAL STABILITY OF
      HUMAN LYSOZYME
    * 1yam: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: CALORIMETRIC STUDIES AND X-RAY STRUCTURAL ANALYSIS OF THE FIVE
      ISOLEUCINE TO VALINE MUTANTS
    * 1yan: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: CALORIMETRIC STUDIES AND X-RAY STRUCTURAL ANALYSIS OF THE FIVE
      ISOLEUCINE TO VALINE MUTANTS
    * 1yao: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: CALORIMETRIC STUDIES AND X-RAY STRUCTURAL ANALYSIS OF THE FIVE
      ISOLEUCINE TO VALINE MUTANTS
    * 1yap: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: CALORIMETRIC STUDIES AND X-RAY STRUCTURAL ANALYSIS OF THE FIVE
      ISOLEUCINE TO VALINE MUTANTS
    * 1yaq: CONTRIBUTION OF HYDROPHOBIC RESIDUES TO THE STABILITY OF HUMAN
      LYSOZYME: CALORIMETRIC STUDIES AND X-RAY STRUCTURAL ANALYSIS OF THE FIVE
      ISOLEUCINE TO VALINE MUTANTS
    * 207l: MUTANT HUMAN LYSOZYME C77A
    * 208l: MUTANT HUMAN LYSOZYME C77A
    * 2bqa: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqb: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqc: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqd: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqe: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqf: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqg: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqh: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqi: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqj: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqk: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bql: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqm: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqn: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2bqo: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2hea: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2heb: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2hec: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2hed: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2hee: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2hef: CONTRIBUTION OF WATER MOLECULES IN THE INTERIOR OF A PROTEIN TO
      THE CONFORMATIONAL STABILITY
    * 2lhm: CRYSTAL STRUCTURES OF THE APO-AND HOLOMUTANT HUMAN LYSOZYMES WITH
      AN INTRODUCED CA2+ BINDING SITE
    * 2mea: CHANGES IN CONFORMATIONAL STABILITY OF A SERIES OF MUTANT HUMAN
      LYSOZYMES AT CONSTANT POSITIONS
    * 2meb: CHANGES IN CONFORMATIONAL STABILITY OF A SERIES OF MUTANT HUMAN
      LYSOZYMES AT CONSTANT POSITIONS
    * 2mec: CHANGES IN CONFORMATIONAL STABILITY OF A SERIES OF MUTANT HUMAN
      LYSOZYMES AT CONSTANT POSITIONS
    * 2med: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2mee: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2mef: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2meg: CHANGES IN CONFORMATIONAL STABILITY OF A SERIES OF MUTANT HUMAN
      LYSOZYMES AT CONSTANT POSITIONS.
    * 2meh: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2mei: CONTRIBUTION OF HYDROPHOBIC EFFECT TO THE CONFORMATIONAL STABILITY
      OF HUMAN LYSOZYME
    * 2nwd: Structure of chemically synthesized human lysozyme at 1 Angstrom
      resolution
    * 3lhm: CRYSTAL STRUCTURES OF THE APO-AND HOLOMUTANT HUMAN LYSOZYMES WITH
      AN INTRODUCED CA2+ BINDING SITE
    * v
    * t
    * e
Antimicrobial_peptides: Granulocyte granule contents
                               * Myeloperoxidase
                               * Defensins
Azurophilic_granules (1Â°    * neutral serine_proteases (Proteinase_3)
                               * Lysozyme
                               * Bactericidal/permeability-increasing_protein
                               * Collagenase
                                          * Alkaline_phosphatase
                                          * Lactoferrin
                           Neutrophil     * Lysozyme
                                          * NADPH_oxidase
                                          * Collagenase
                                          * Cathelicidin
Specific_granules (2Â°)                 * Cathepsin
                                          * Major_basic_protein
                           Eosinophil     * Eosinophil_cationic_protein
                                          * Eosinophil_peroxidase
                                          * Eosinophil-derived_neurotoxin
                           Basophil       * Heparin
                                          * Histamine
    * see also platelet_alpha-granule, dense_granule
    * v
    * t
    * e
Antibiotics active on the cell_wall and envelope (J01C-J01D)
                  * Inhibit peptidoglycan subunit synthesis and transport: NAM synthesis inhibition
Intracellular       (Fosfomycin)
                  * DADAL/AR inhibitors (Cycloserine)
                  * bactoprenol inhibitors (Bacitracin)
                  * Inhibit PG chain elongation: Vancomycin# (Oritavancin
Glycopeptide      * Telavancin)
                  * Teicoplanin (Dalbavancin)
                  * Ramoplanin
                                                                   * Benzylpenicillin_(G)#
                                                                   * Benzathine_benzylpenicillin#
                                                                   * Procaine_benzylpenicillin#
                                      Î²-lactamase_sensitive     * Phenoxymethylpenicillin_(V)#
                                      (1st generation)             * Propicillinâ¡
                                                                   * Pheneticillinâ¡
                             Narrow                                * Azidocillinâ¡
                             spectrum                              * Clometocillinâ¡
                                                                   * Penamecillinâ¡
                                                                   * Cloxacillin# (Dicloxacillin
                                      Î²-lactamase_resistant     * Flucloxacillin)
                                      (2nd generation)             * Oxacillin
                                                                   * Nafcillin
              Penicillins                                          * Methicillinâ¡
              (Penams)                                       * Amoxicillin#
                                                             * Ampicillin# (Pivampicillin
                                      Aminopenicillins       * Hetacillinâ¡
                                      (3rd generation)       * Bacampicillinâ¡
                                                             * Metampicillinâ¡
                                                             * Talampicillinâ¡)
                             Extended                        * Epicillinâ¡
                             spectrum Carboxypenicillins     * Ticarcillin
                                      (4th generation)       * Carbenicillinâ¡ / Carindacillinâ¡
                                                             * Temocillinâ¡
                                      Ureidopenicillins      * Piperacillin
                                      (4th generation)       * Azlocillinâ¡
                                                             * Mezlocillinâ¡
                                      Other                  * Mecillinamâ¡ (Pivmecillinamâ¡)
                                                             * Sulbenicillinâ¡
              Penems             * Faropenemâ¡
                                 * RitipenemÂ§
                                 * Ertapenem
                                 * Antipseudomonal (Doripenem
              Carbapenems        * Imipenem
                                 * Meropenem)
                                 * Biapenemâ¡
                                 * Panipenemâ¡
                                                * Cefazolin#
                                                * Cefalexin #
                                                * Cefadroxil
                                                * Cefapirin
                                                * Cefazedoneâ¡
                                                * Cefazaflurâ¡
                                                * Cefradineâ¡
                             1st_generation     * Cefroxadineâ¡
                                                * Ceftezoleâ¡
                                                * Cefaloglycinâ¡
                                                * Cefacetrileâ¡
                                                * Cefaloniumâ¡
                                                * Cefaloridineâ¡
                                                * Cefalotinâ¡
                                                * Cefatrizineâ¡
                                                * Cefaclor
                                                * Cefotetan
                                                * Cephamycin (Cefoxitin
                                                * Cefprozil
                                                * Cefuroxime
                                                * Cefuroxime_axetil
Î²-lactams/                2nd_generation     * Cefamandoleâ¡
(inhibit PBP                                    * Cefminoxâ¡
cross-links)                                    * Cefonicidâ¡
                                                * Ceforanideâ¡
                                                * Cefbuperazoneâ¡
                                                * Cefuzonamâ¡
                                                * Cefmetazoleâ¡)
                                                * Carbacephemâ¡ (Loracarbefâ¡)
              Cephalosporins                    * Cefixime#
              / Cephamycins                     * Ceftriaxone#
              (Cephems)                         * Antipseudomonal (Ceftazidime#
                                                * Cefoperazoneâ¡)
                                                * Cefdinir
                                                * Cefcapene
                                                * Cefdaloxime
                                                * Ceftizoxime
                                                * Cefmenoxime
                                                * Cefotaxime#
                                                * Cefotiamâ¡
                             3rd_generation     * Cefpiramide
                                                * Cefpodoxime
                                                * Ceftibuten
                                                * Cefditoren
                                                * Cefetametâ¡
                                                * Cefodizimeâ¡
                                                * Cefpimizoleâ¡
                                                * Cefsulodinâ¡
                                                * Cefteramâ¡
                                                * Ceftioleneâ¡
                                                * Oxacephem (Flomoxefâ¡
                                                * Latamoxefâ¡)
                                                * Cefepime
                             4th_generation     * Cefozopranâ¡
                                                * Cefpiromeâ¡
                                                * Cefquinomeâ¡
                                                * Ceftaroline_fosamil
                             5th_generation     * Ceftolozane
                                                * Ceftobiprole
                                                * Ceftiofur
                             Veterinary         * Cefquinome
                                                * Cefovecin
                                 * Aztreonam#
              Monobactams        * Tigemonamâ¡
                                 * Carumonamâ¡
                                 * Nocardicin_Aâ¡
                                 * Penam (Sulbactam
                                 * Tazobactam)
              Î²-lactamase     * Clavam (Clavulanic_acid)
              inhibitors         * Avibactam
                                 * Relebactam
                                 * Sulbactam
                                 * Vaborbactam
                                 * Amoxicillin/clavulanic_acid#
                                 * Imipenem/cilastatin#
                                 * Imipenem/cilastatin/relebactam
                                 * Ampicillin/flucloxacillin
              Combinations       * Ampicillin/sulbactam (Sultamicillin)
                                 * Ceftazidime/avibactam
                                 * Piperacillin/tazobactam
                                 * Ceftolozane/tazobactam
                                 * Cefoperazone/sulbactam
                                 * Meropenem/vaborbactam
                  * polymyxins/detergent
                        o Colistin
                        o Polymyxin_B
                  * depolarizing
                        o Daptomycin
Other             * Hydrolyze NAM-NAG
                        o lysozyme
                  * Tyrothricin
                        o Gramicidin
                        o Tyrocidine
                  * Isoniazid#
                  * Teixobactin
    * #WHO-EM
    * â¡Withdrawn from market
    * Clinical_trials:
          o â Phase_III
          o Â§Never to phase III
    * v
    * t
    * e
Hydrolase: sugar hydrolases (EC 3.2)
                                               * Sucrase/Sucrase-isomaltase/
                                                 Invertase
                            Disaccharidase     * Maltase
                                               * Trehalase
                                               * Lactase
                                               * Cellulase
                                               * Alpha-glucosidase
                                                     o Acid
                            Glucosidases             o Neutral_AB
                                                     o Neutral_C
                                               * Beta-glucosidase
                                                     o cytosolic
                                               * Debranching_enzyme
                                               * Amylase
                                                     o Alpha-amylase
                                               * Chitinase
                                               * Lysozyme
                                               * Neuraminidase
                                                     o NEU1
                                                     o NEU2
                                                     o NEU3
                                                     o NEU4
                                                     o Bacterial_neuraminidase
3.2.1: Glycoside_hydrolases                          o Viral_neuraminidase
                                               * Galactosidases
                                                     o Alpha
                                                     o Beta
                                               * alpha-Mannosidase
                                               * Glucuronidase
                                                     o Klotho
                            Other              * Hyaluronidase
                                               * Pullulanase
                                               * Glucosylceramidase
                                                     o lysosomal
                                                     o non-lysosomal
                                               * Galactosylceramidase
                                               * Alpha-N-
                                                 acetylgalactosaminidase
                                                     o NAGA
                                               * Alpha-N-acetylglucosaminidase
                                               * Fucosidase
                                               * Hexosaminidase
                                                     o HEXA
                                                     o HEXB
                                               * Iduronidase
                                               * Maltase-glucoamylase
                                               * Heparanase
                                                     o HPSE2
3.2.2: Hydrolysing              * DNA_glycosylases: Oxoguanine_glycosylase
N-Glycosyl compounds
    * v
    * t
    * e
Enzymes
                   * Active_site
                   * Binding_site
                   * Catalytic_triad
                   * Oxyanion_hole
Activity           * Enzyme_promiscuity
                   * Catalytically_perfect_enzyme
                   * Coenzyme
                   * Cofactor
                   * Enzyme_catalysis
                   * Allosteric_regulation
Regulation         * Cooperativity
                   * Enzyme_inhibitor
                   * Enzyme_activator
                   * EC_number
Classification     * Enzyme_superfamily
                   * Enzyme_family
                   * List_of_enzymes
                   * Enzyme_kinetics
                   * EadieâHofstee_diagram
Kinetics           * HanesâWoolf_plot
                   * LineweaverâBurk_plot
                   * MichaelisâMenten_kinetics
                   * EC1 Oxidoreductases (list)
                   * EC2 Transferases (list)
                   * EC3 Hydrolases (list)
Types              * EC4 Lyases (list)
                   * EC5 Isomerases (list)
                   * EC6 Ligases (list)
                   * EC7 Translocases (list)
    * [TPI1_structure.png]Molecular_and_Cellular_Biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Lysozyme&oldid=907463799"
Categories:
    * Genes_on_human_chromosome_12
    * EC_3.2.1
    * Immunology
    * E-number_additives
Hidden categories:
    * CS1_German-language_sources_(de)
    * Use_dmy_dates_from_August_2011
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Cymraeg
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Occitan
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * à¨ªà©°à¨à¨¾à¨¬à©
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Simple_English
    * SlovenÅ¡Äina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 23 July 2019, at 02:42 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]



















[This_is_a_file_from_the_Wikimedia_Commons]
****** File:JBSlysozymemechanism copy2.jpg ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
    * File
    * File_history
    * File_usage
    * Metadata
[File:JBSlysozymemechanism_copy2.jpg]
Size of this preview: 477_Ã_599_pixels. Other resolutions: 191_Ã_240_pixels |
382_Ã_480_pixels | 477_Ã_600_pixels | 611_Ã_768_pixels | 1,028_Ã_1,292
pixels.
Original_file ‎(1,028 Ã 1,292 pixels, file size: 108 KB, MIME type: image/
jpeg)
 This is a file from the Wikimedia_Commons. Information from its description
 page_there is shown below.
 Commons is a freely licensed media file repository. You_can_help.
***** Summary *****
DescriptionJBSlysozymemechanism copy2.jpg English: JBSlysozymemechanism copy2
Date                                      20 February 2017
Source                                    Own work
Author                                    Jsolomon580
***** Licensing *****
I, the copyright holder of this work, hereby publish it under the following
license:
              This file is licensed under the Creative_Commons Attribution-Share_Alike_4.0
              International license.
                    You are free:
                        * to share â to copy, distribute and transmit the work
                        * to remix â to adapt the work
[w:en:              Under the following conditions:
Creative                * attribution â You must give appropriate credit, provide a link to
Commons]                  the license, and indicate if changes were made. You may do so in
[attribution]             any reasonable manner, but not in any way that suggests the
[share alike]             licensor endorses you or your use.
                        * share alike â If you remix, transform, or build upon the
                          material, you must distribute your contributions under the same_or
                          compatible_license as the original.
              https://creativecommons.org/licenses/by-sa/4.0 CC BY-SA 4.0 Creative Commons
              Attribution-Share Alike 4.0 truetrue
                                This media file is uncategorized.
                                Please help improve this media file by adding
                                it to one or more categories, so it may be
                                associated with related media files (how?), and
[Gnome-applications-office.svg] so that it can be more easily found.
                                ===============================================
                                Please notify the uploader with
                                {{subst:Please link images|File:
                                JBSlysozymemechanism copy2.jpg}} ~~~~

***** File history *****
Click on a date/time to view the file as it appeared at that time.
        Date/Time Thumbnail    Dimensions            User        Comment
                  [Thumbnail
        19:05,_28 for_version  1,028 Ã 1,292 (108             User created
current February  as_of_19:05, KB)                   Jsolomon580 page with
        2017      28_February                                    UploadWizard
                  2017]
***** File usage *****
The following pages on the English Wikipedia use this file (pages on other
projects are not listed):
    * Lysozyme
    * User:Gbzwg/sandbox
    * User:Jsolomon580/sandbox
***** Metadata *****
This file contains additional information, probably added from the digital
camera or scanner used to create or digitize it.
If the file has been modified from its original state, some details may not
fully reflect the modified file.
Horizontal resolution 144 dpi
Vertical resolution   144 dpi
Retrieved from "https://en.wikipedia.org/wiki/File:
JBSlysozymemechanism_copy2.jpg"
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * File
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * View_on_Commons
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Upload_file
    * Special_pages
    * Page_information
**** Languages ****
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
