The following text has been accessed from https://en.wikipedia.org/wiki/Plasmid at Fri Aug 9 02:22:47 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Plasmid ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Illustration of a bacterium showing chromosomal DNA and plasmids. Not to scale.
A plasmid is a small DNA molecule within a cell that is physically separated
from chromosomal_DNA and can replicate independently. They are most commonly
found as small circular, double-stranded DNA molecules in bacteria; however,
plasmids are sometimes present in archaea and eukaryotic_organisms. In nature,
plasmids often carry genes that benefit the survival of the organism, such as
by providing antibiotic_resistance. While the chromosomes are big and contain
all the essential genetic information for living under normal conditions,
plasmids usually are very small and contain only additional genes that may be
useful in certain situations or conditions. Artificial plasmids are widely used
as vectors in molecular_cloning, serving to drive the replication of
recombinant_DNA sequences within host organisms. In the laboratory, plasmids
may be introduced into a cell via transformation.
Plasmids are considered replicons, units of DNA capable of replicating
autonomously within a suitable host. However, plasmids, like viruses, are not
generally classified as life.[1] Plasmids are transmitted from one bacterium to
another (even of another species) mostly through conjugation.[2] This host-to-
host transfer of genetic material is one mechanism of horizontal_gene_transfer,
and plasmids are considered part of the mobilome. Unlike viruses, which encase
their genetic material in a protective protein coat called a capsid, plasmids
are "naked" DNA and do not encode genes necessary to encase the genetic
material for transfer to a new host. However, some classes of plasmids encode
the conjugative_"sex"_pilus necessary for their own transfer. The size of the
plasmid varies from 1 to over 200 kbp,[3] and the number of identical plasmids
in a single cell can range anywhere from one to thousands under some
circumstances.
The relationship between microbes and plasmid DNA is neither parasitic nor
mutualistic, because each implies the presence of an independent species living
in a detrimental or commensal state with the host organism. Rather, plasmids
provide a mechanism for horizontal gene transfer within a population of
microbes and typically provide a selective advantage under a given
environmental state. Plasmids may carry genes that provide resistance to
naturally occurring antibiotics in a competitive environmental_niche, or the
proteins produced may act as toxins under similar circumstances, or allow the
organism to utilize particular organic compounds that would be advantageous
when nutrients are scarce.[4]
‚Å∞
***** Contents *****
    * 1_History
    * 2_Properties_and_characteristics
    * 3_Classifications_and_types
    * 4_Vectors
          o 4.1_Cloning
          o 4.2_Protein_production
          o 4.3_Gene_therapy
          o 4.4_Disease_models
    * 5_Episomes
    * 6_Plasmid_maintenance
    * 7_Yeast_plasmids
    * 8_Plasmid_DNA_extraction
    * 9_Conformations
    * 10_Software_for_bioinformatics_and_design
    * 11_Plasmid_collections
    * 12_See_also
    * 13_References
    * 14_Further_reading
          o 14.1_Episomes
    * 15_External_links
***** History[edit] *****
The term plasmid was introduced in 1952 by the American molecular_biologist
Joshua_Lederberg to refer to "any extrachromosomal hereditary determinant."[5]
The term's early usage included any bacterial genetic material that exists
extrachromosomally for at least part of its replication cycle, but because that
description includes bacterial viruses, the notion of plasmid was refined over
time to comprise genetic elements that reproduce autonomously.[6] Later in
1968, it was decided that the term plasmid should be adopted as the term for
extrachromosomal genetic element,[7] and to distinguish it from viruses, the
definition was narrowed to genetic elements that exist exclusively or
predominantly outside of the chromosome and can replicate autonomously.[6]
***** Properties and characteristics[edit] *****
There are two types of plasmid integration into a host bacteria: Non-
integrating plasmids replicate as with the top instance, whereas episomes, the
lower example, can integrate into the host chromosome.
In order for plasmids to replicate independently within a cell, they must
possess a stretch of DNA that can act as an origin_of_replication. The self-
replicating unit, in this case the plasmid, is called a replicon. A typical
bacterial replicon may consist of a number of elements, such as the gene for
plasmid-specific replication initiation protein (Rep), repeating units called
iterons, DnaA boxes, and an adjacent AT-rich region.[6] Smaller plasmids make
use of the host replicative enzymes to make copies of themselves, while larger
plasmids may carry genes specific for the replication of those plasmids. A few
types of plasmids can also insert into the host chromosome, and these
integrative plasmids are sometimes referred to as episomes in prokaryotes.[8]
Plasmids almost always carry at least one gene. Many of the genes carried by a
plasmid are beneficial for the host cells, for example: enabling the host cell
to survive in an environment that would otherwise be lethal or restrictive for
growth. Some of these genes encode traits for antibiotic resistance or
resistance to heavy metal, while others may produce virulence_factors that
enable a bacterium to colonize a host and overcome its defences, or have
specific metabolic functions that allow the bacterium to utilize a particular
nutrient, including the ability to degrade recalcitrant or toxic organic
compounds.[6] Plasmids can also provide bacteria with the ability to fix
nitrogen. Some plasmids, however, have no observable effect on the phenotype of
the host cell or its benefit to the host cells cannot be determined, and these
plasmids are called cryptic plasmids.[9]
Naturally occurring plasmids vary greatly in their physical properties. Their
size can range from very small mini-plasmids of less than a 1 kilobase pairs
(Kbp), to very large megaplasmids of several megabase pairs (Mbp). At the upper
end, little can differentiate between a megaplasmid and a minichromosome.
Plasmids are generally circular, but examples of linear plasmids are also
known. These linear plasmids require specialized mechanisms to replicate their
ends.[6]
Plasmids may be present in an individual cell in varying number, ranging from
one to several hundreds. The normal number of copies of plasmid that may be
found in a single cell is called the Plasmid_copy_number, and is determined by
how the replication initiation is regulated and the size of the molecule.
Larger plasmids tend to have lower copy numbers.[8] Low-copy-number plasmids
that exist only as one or a few copies in each bacterium are, upon cell
division, in danger of being lost in one of the segregating bacteria. Such
single-copy plasmids have systems that attempt to actively distribute a copy to
both daughter cells. These systems, which include the parABS_system and parMRC
system, are often referred to as the partition_system or partition function of
a plasmid.
***** Classifications and types[edit] *****
Overview of bacterial conjugation
Electron_micrograph of a DNA fiber bundle, presumably of a single bacterial
chromosome loop
Electron micrograph of a bacterial DNA plasmid (chromosome fragment)
Plasmids may be classified in a number of ways. Plasmids can be broadly
classified into conjugative plasmids and non-conjugative plasmids. Conjugative
plasmids contain a set of transfer or tra genes which promote sexual
conjugation between different cells.[8] In the complex process of conjugation,
plasmid may be transferred from one bacterium to another via sex pili encoded
by some of the tra genes (see figure).[10] Non-conjugative plasmids are
incapable of initiating conjugation, hence they can be transferred only with
the assistance of conjugative plasmids. An intermediate class of plasmids are
mobilizable, and carry only a subset of the genes required for transfer. They
can parasitize a conjugative plasmid, transferring at high frequency only in
its presence.
Plasmids can also be classified into incompatibility groups. A microbe can
harbour different types of plasmids, but different plasmids can only exist in a
single bacterial cell if they are compatible. If two plasmids are not
compatible, one or the other will be rapidly lost from the cell. Different
plasmids may therefore be assigned to different incompatibility groups
depending on whether they can coexist together. Incompatible plasmids
(belonging to the same incompatibility group) normally share the same
replication or partition mechanisms and can thus not be kept together in a
single cell.[11][12]
Another way to classify plasmids is by function. There are five main classes:
    * Fertility F-plasmids, which contain tra genes. They are capable of
      conjugation and result in the expression of sex pili.
    * Resistance (R) plasmids, which contain genes that provide resistance
      against antibiotics or poisons. Historically known as R-factors, before
      the nature of plasmids was understood.
    * Col plasmids, which contain genes that code for bacteriocins, proteins
      that can kill other bacteria.
    * Degradative plasmids, which enable the digestion of unusual substances,
      e.g. toluene and salicylic_acid.
    * Virulence plasmids, which turn the bacterium into a pathogen.
Plasmids can belong to more than one of these functional groups.
***** Vectors[edit] *****
Further information: Vector_(molecular_biology)
Artificially constructed plasmids may be used as vectors in genetic
engineering. These plasmids serve as important tools in genetics and
biotechnology labs, where they are commonly used to clone and amplify (make
many copies of) or express particular genes.[13] A wide variety of plasmids are
commercially available for such uses. The gene to be replicated is normally
inserted into a plasmid that typically contains a number of features for their
use. These include a gene that confers resistance to particular antibiotics
(ampicillin is most frequently used for bacterial strains), an origin_of
replication to allow the bacterial cells to replicate the plasmid DNA, and a
suitable site for cloning (referred to as a multiple_cloning_site).
DNA structural instability can be defined as a series of spontaneous events
that culminate in an unforeseen rearrangement, loss, or gain of genetic
material. Such events are frequently triggered by the transposition of mobile
elements or by the presence of unstable elements such as non-canonical (non-B)
structures. Accessory regions pertaining to the bacterial backbone may engage
in a wide range of structural instability phenomena. Well-known catalysts of
genetic instability include direct, inverted, and tandem repeats, which are
known to be conspicuous in a large number of commercially available cloning and
expression vectors.[14] Insertion sequences can also severely impact plasmid
function and yield, by leading to deletions and rearrangements, activation,
down-regulation or inactivation of neighboring gene expression.[15] Therefore,
the reduction or complete elimination of extraneous noncoding backbone
sequences would pointedly reduce the propensity for such events to take place,
and consequently, the overall recombinogenic potential of the plasmid.[16][17]
A schematic representation of the pBR322 plasmid, one of the first plasmids to
be used widely as a cloning_vector. Shown on the plasmid diagram are the genes
encoded (amp and tet for ampicillin and tetracycline resistance respectively),
its origin of replication (ori), and various restriction_sites (indicated in
blue).
**** Cloning[edit] ****
Main article: Cloning_vector
Plasmids are the most-commonly used bacterial cloning vectors.[18] These
cloning vectors contain a site that allows DNA fragments to be inserted, for
example a multiple_cloning_site or polylinker which has several commonly used
restriction_sites to which DNA fragments may be ligated. After the gene of
interest is inserted, the plasmids are introduced into bacteria by a process
called transformation. These plasmids contain a selectable_marker, usually an
antibiotic resistance gene, which confers on the bacteria an ability to survive
and proliferate in a selective growth medium containing the particular
antibiotics. The cells after transformation are exposed to the selective media,
and only cells containing the plasmid may survive. In this way, the antibiotics
act as a filter to select only the bacteria containing the plasmid DNA. The
vector may also contain other marker_genes or reporter_genes to facilitate
selection of plasmid with cloned insert. Bacteria containing the plasmid can
then be grown in large amounts, harvested, and the plasmid of interest may then
be isolated using various methods of plasmid_preparation.
A plasmid cloning vector is typically used to clone DNA fragments of up to 15
kbp.[19] To clone longer lengths of DNA, lambda_phage with lysogeny genes
deleted, cosmids, bacterial_artificial_chromosomes, or yeast_artificial
chromosomes are used.
**** Protein production[edit] ****
Main article: Expression_vector
Another major use of plasmids is to make large amounts of proteins. In this
case, researchers grow bacteria containing a plasmid harboring the gene of
interest. Just as the bacterium produces proteins to confer its antibiotic
resistance, it can also be induced to produce large amounts of proteins from
the inserted gene. This is a cheap and easy way of mass-producing the protein
the gene codes for, for example, insulin.
**** Gene therapy[edit] ****
Main article: Vectors_in_gene_therapy
Plasmid may also be used for gene transfer into human cells as potential
treatment in gene_therapy so that it may express the protein that is lacking in
the cells. Some strategies of gene_therapy require the insertion of therapeutic
genes at pre-selected chromosomal target sites within the human genome. Plasmid
vectors are one of many approaches that could be used for this purpose. Zinc
finger_nucleases (ZFNs) offer a way to cause a site-specific double-strand
break to the DNA genome and cause homologous_recombination. Plasmids encoding
ZFN could help deliver a therapeutic gene to a specific site so that cell
damage, cancer-causing mutations, or an immune response is avoided.[20]
**** Disease models[edit] ****
Plasmids were historically used to genetically engineer the embryonic stem
cells of rats to create rat genetic disease models. The limited efficiency of
plasmid-based techniques precluded their use in the creation of more accurate
human cell models. However, developments in Adeno-associated_virus
recombination techniques, and Zinc_finger_nucleases, have enabled the creation
of a new generation of isogenic_human_disease_models.
***** Episomes[edit] *****
"Episome" redirects here. For the album by Bill Laswell, Otomo Yoshihide and
Tatsuya Yoshida, see Episome_(album).
The term episome was introduced by Fran√É¬ßois_Jacob and √É¬âlie_Wollman in 1958 to
refer to extra-chromosomal genetic material that may replicate autonomously or
become integrated into the chromosome.[21][22] Since the term was introduced,
however, its use has shifted, as plasmid has become the preferred term for
autonomously replicating extrachromosomal DNA. At a 1968 symposium in London
some participants suggested that the term episome be abandoned, although others
continued to use the term with a shift in meaning.[23][24]
Today some authors use episome in the context of prokaryotes to refer to a
plasmid that is capable of integrating into the chromosome. The integrative
plasmids may be replicated and stably maintained in a cell through multiple
generations, but always at some stage they exist as an independent plasmid
molecule.[25] In the context of eukaryotes, the term episomes is used to mean a
non-integrated extrachromosomal closed circular DNA molecule that may be
replicated in the nucleus.[26][27] Viruses are the most common examples of
this, such as herpesviruses, adenoviruses, and polyomaviruses, but some are
plasmids. Other examples include aberrant chromosomal fragments, such as double
minute_chromosomes, that can arise during artificial gene amplifications or in
pathologic processes (e.g., cancer cell transformation). Episomes in eukaryotes
behave similarly to plasmids in prokaryotes in that the DNA is stably
maintained and replicated with the host cell. Cytoplasmic viral episomes (as in
poxvirus infections) can also occur. Some episomes, such as herpesviruses,
replicate in a rolling_circle mechanism, similar to bacterial phage viruses.
Others replicate through a bidirectional replication mechanism (Theta type
plasmids). In either case, episomes remain physically separate from host cell
chromosomes. Several cancer viruses, including Epstein-Barr_virus and Kaposi's
sarcoma-associated_herpesvirus, are maintained as latent, chromosomally
distinct episomes in cancer cells, where the viruses express oncogenes that
promote cancer cell proliferation. In cancers, these episomes passively
replicate together with host chromosomes when the cell divides. When these
viral episomes initiate lytic_replication to generate multiple virus particles,
they in general activate cellular innate_immunity defense mechanisms that kill
the host cell.
***** Plasmid maintenance[edit] *****
Main article: Addiction_module
Some plasmids or microbial hosts include an addiction_system or
postsegregational killing system (PSK), such as the hok/sok (host killing/
suppressor of killing) system of plasmid R1 in Escherichia_coli.[28] This
variant produces both a long-lived poison and a short-lived antidote. Several
types of plasmid addiction systems (toxin/ antitoxin, metabolism-based, ORT
systems) were described in the literature[29] and used in biotechnical
(fermentation) or biomedical (vaccine therapy) applications. Daughter cells
that retain a copy of the plasmid survive, while a daughter cell that fails to
inherit the plasmid dies or suffers a reduced growth-rate because of the
lingering poison from the parent cell. Finally, the overall productivity could
be enhanced.
In contrast, virtually all biotechnologically used plasmids (such as pUC18,
pBR322 and derived vectors) do not contain toxin-antitoxin addiction systems
and thus need to be kept under antibiotic pressure to avoid plasmid loss.
***** Yeast plasmids[edit] *****
Yeasts naturally harbour various plasmids. Notable among them are 2¬†√Ç¬µm
plasmids √¢¬Ä¬î small circular plasmids often used for genetic_engineering of
yeast √¢¬Ä¬î and linear pGKL plasmids from Kluyveromyces_lactis, that are
responsible for killer_phenotypes.[30]
Other types of plasmids are often related to yeast cloning vectors that
include:
    * Yeast integrative plasmid (YIp), yeast vectors that rely on integration
      into the host chromosome for survival and replication, and are usually
      used when studying the functionality of a solo gene or when the gene is
      toxic. Also connected with the gene URA3, that codes an enzyme related to
      the biosynthesis of pyrimidine nucleotides (T, C);
    * Yeast Replicative Plasmid (YRp), which transport a sequence of
      chromosomal DNA that includes an origin of replication. These plasmids
      are less stable, as they can get lost during the budding.
***** Plasmid DNA extraction[edit] *****
As alluded to above, plasmids are often used to purify a specific sequence,
since they can easily be purified away from the rest of the genome. For their
use as vectors, and for molecular_cloning, plasmids often need to be isolated.
There are several methods to isolate_plasmid_DNA from bacteria, the archetypes
of which are the miniprep and the maxiprep/bulkprep.[13] The former can be used
to quickly find out whether the plasmid is correct in any of several bacterial
clones. The yield is a small amount of impure plasmid DNA, which is sufficient
for analysis by restriction_digest and for some cloning techniques.
In the latter, much larger volumes of bacterial suspension are grown from which
a maxi-prep can be performed. In essence, this is a scaled-up miniprep followed
by additional purification. This results in relatively large amounts (several
hundreds micrograms) of very pure plasmid DNA.
In recent times, many commercial kits have been created to perform plasmid
extraction at various scales, purity, and levels of automation. Commercial
services can prepare plasmid DNA at quoted prices below $300/mg in milligram
quantities and $15/mg in gram quantities (early 2007[update]).
***** Conformations[edit] *****
Plasmid DNA may appear in one of five conformations, which (for a given size)
run at different speeds in a gel during electrophoresis. The conformations are
listed below in order of electrophoretic mobility (speed for a given applied
voltage) from slowest to fastest:
    * Nicked open-circular DNA has one strand cut.
    * Relaxed circular DNA is fully intact with both strands uncut, but has
      been enzymatically relaxed (supercoils removed). This can be modeled by
      letting a twisted extension cord unwind and relax and then plugging it
      into itself.
    * Linear DNA has free ends, either because both strands have been cut or
      because the DNA was linear in vivo. This can be modeled with an
      electrical extension cord that is not plugged into itself.
    * Supercoiled (or covalently closed-circular) DNA is fully intact with both
      strands uncut, and with an integral twist, resulting in a compact form.
      This can be modeled by twisting an extension_cord and then plugging it
      into itself.
    * Supercoiled denatured DNA is like supercoiled DNA, but has unpaired
      regions that make it slightly less compact; this can result from
      excessive alkalinity during plasmid preparation.
The rate of migration for small linear fragments is directly proportional to
the voltage applied at low voltages. At higher voltages, larger fragments
migrate at continuously increasing yet different rates. Thus, the resolution of
a gel decreases with increased voltage.
At a specified, low voltage, the migration rate of small linear DNA fragments
is a function of their length. Large linear fragments (over 20 kb or so)
migrate at a certain fixed rate regardless of length. This is because the
molecules 'resperate', with the bulk of the molecule following the leading end
through the gel matrix. Restriction_digests are frequently used to analyse
purified plasmids. These enzymes specifically break the DNA at certain short
sequences. The resulting linear fragments form 'bands' after gel
electrophoresis. It is possible to purify certain fragments by cutting the
bands out of the gel and dissolving the gel to release the DNA fragments.
Because of its tight conformation, supercoiled DNA migrates faster through a
gel than linear or open-circular DNA.
***** Software for bioinformatics and design[edit] *****
The use of plasmids as a technique in molecular_biology is supported by
bioinformatics software. These programs record the DNA sequence of plasmid
vectors, help to predict cut sites of restriction_enzymes, and to plan
manipulations. Examples of software packages that handle plasmid maps are ApE,
Clone_Manager, GeneConstructionKit, Geneious, Genome_Compiler, LabGenius,
Lasergene, MacVector, pDraw32, Serial Cloner, VectorFriends, Vector_NTI, and
WebDSV. These software help conduct entire experiments in silico before doing
wet experiments.[31]
***** Plasmid collections[edit] *****
Many plasmids have been created over the years and researchers have given out
plasmids to plasmid databases such as the non-profit organisations Addgene and
BCCM/LMBP. One can find and request plasmids from those databases for research.
Researcher also often upload plasmid sequences to the NCBI_database, from which
sequences of specific plasmids can be retrieved.
***** See also[edit] *****
    * Bacterial_artificial_chromosome
    * Bacteriophage
    * Provirus
    * Segrosome
    * Transposon
    * Triparental_mating
    * Plasmidome
    * DNA_recombination
***** References[edit] *****
   1. ^Sinkovics J, Horvath J, Horak A (1998). "The origin and evolution of
      viruses (a review)". Acta Microbiologica et Immunologica Hungarica. 45
      (3√¢¬Ä¬ì4): 349√¢¬Ä¬ì90. PMID¬†9873943.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Smillie C, Garcill√É¬°n-Barcia MP, Francia MV, Rocha EP, de la Cruz F
      (September 2010). "Mobility_of_plasmids". Microbiology and Molecular
      Biology Reviews. 74 (3): 434√¢¬Ä¬ì52. doi:10.1128/MMBR.00020-10.
      PMC¬†2937521. PMID¬†20805406.
   4. ^Thomas CM, Summers D (2008). Bacterial Plasmids. Encyclopedia of Life
      Sciences. doi:10.1002/9780470015902.a0000468.pub2. ISBN¬†978-0470016176.
   5. ^Wolfgang Schumann (2008). "Chapter 1 √¢¬Ä¬ì Escherichia coli Cloning and
      Expression Vectors". In Georg Lipps (eds.). Plasmids: Current Research
      and Future Trends. Caister Academic Press. pp.¬†1√¢¬Ä¬ì2. ISBN¬†978-1-904455-
      35-6.CS1 maint: Uses editors parameter (link)
   6. ^Lederberg J (October 1952). "Cell genetics and hereditary symbiosis".
      Physiological Reviews. 32 (4): 403√¢¬Ä¬ì30. CiteSeerX¬†10.1.1.458.985. doi:
      10.1152/physrev.1952.32.4.403. PMID¬†13003535.
   7. ^ a b c d eFinbarr Hayes (2003). "Chapter_1_√¢¬Ä¬ì_The_Function_and
      Organization_of_Plasmids". In Nicola Casali, Andrew Presto (eds.). E.
      Coli Plasmid Vectors: Methods and Applications. Methods in Molecular
      Biology. 235. Humana Press. pp.¬†1√¢¬Ä¬ì5. ISBN¬†978-1-58829-151-6.CS1 maint:
      Uses editors parameter (link)
   8. ^Stanley Falkow. "Microbial_Genomics:_Standing_on_the_Shoulders_of
      Giants". Microbiology Society.
   9. ^ a b cT. A. Brown (2010). "Chapter_2_√¢¬Ä¬ì_Vectors_for_Gene_Cloning:
      Plasmids_and_Bacteriophages". Gene Cloning and DNA Analysis: An
      Introduction (6th ed.). Wiley-Blackwell. ISBN¬†978-1405181730.
  10. ^David Summers (1996). "Chapter_1_√¢¬Ä¬ì_The_Function_and_Organization_of
      Plasmids". The Biology of Plasmids. Wiley-Blackwell; First Edition.
      pp.¬†21√¢¬Ä¬ì22. ISBN¬†978-0632034369.
  11. ^David P. Clark; Nanette Jean Pazdernik (2012). Molecular_Biology (2nd
      ed.). Academic Cell. p.¬†795. ISBN¬†978-0123785947.
  12. ^Margaret C. M. Smith and R. Elizabeth Sockett, eds. (1999). Genetic
      Methods_for_Diverse_Prokaryotes. Methods in Microbiology, vol. 29.
      Academic Press. pp.¬†75√¢¬Ä¬ì77. ISBN¬†978-0-12-652340-9.CS1 maint: Uses
      editors parameter (link)
  13. ^Morgan K. "Plasmids_101:_Origin_of_Replication". addgene.org.
  14. ^ a bRussell, David W.; Sambrook, Joseph (2001). Molecular cloning: a
      laboratory manual. Cold Spring Harbor, N.Y: Cold Spring Harbor
      Laboratory.
  15. ^Oliveira PH, Prather KJ, Prazeres DM, Monteiro GA (August 2010).
      "Analysis of DNA repeats in bacterial plasmids reveals the potential for
      recurrent instability events". Applied Microbiology and Biotechnology. 87
      (6): 2157√¢¬Ä¬ì67. doi:10.1007/s00253-010-2671-7. PMID¬†20496146.
  16. ^Gon√É¬ßalves GA, Oliveira PH, Gomes AG, Prather KL, Lewis LA, Prazeres DM,
      Monteiro GA (August 2014). "Evidence that the insertion events of IS2
      transposition are biased towards abrupt compositional shifts in target
      DNA and modulated by a diverse set of culture parameters". Applied
      Microbiology and Biotechnology. 98 (15): 6609√¢¬Ä¬ì19. doi:10.1007/s00253-
      014-5695-6. hdl:1721.1/104375. PMID¬†24769900.
  17. ^Oliveira PH, Mairhofer J (September 2013). "Marker-free_plasmids_for
      biotechnological_applications_-_implications_and_perspectives". Trends in
      Biotechnology. 31 (9): 539√¢¬Ä¬ì47. doi:10.1016/j.tibtech.2013.06.001.
      PMID¬†23830144.
  18. ^Oliveira PH, Prather KJ, Prazeres DM, Monteiro GA (September 2009).
      "Structural_instability_of_plasmid_biopharmaceuticals:_challenges_and
      implications". Trends in Biotechnology. 27 (9): 503√¢¬Ä¬ì11. doi:10.1016/
      j.tibtech.2009.06.004. PMID¬†19656584.
  19. ^Uldis N. Streips, Ronald E. Yasbin, eds. (2002). Modern_Microbial
      Genetics (2nd ed.). Wiley-Blackwell. p.¬†248. ISBN¬†978-0471386650.CS1
      maint: Uses editors parameter (link)
  20. ^Andrew Preston (2003). "Chapter_2_√¢¬Ä¬ì_Choosing_a_Cloning_Vector". In
      Nicola Casali, Andrew Preston (eds.). E. Coli Plasmid Vectors: Methods
      and Applications. Methods in Molecular Biology, Vol. 235. Humana Press.
      pp.¬†19√¢¬Ä¬ì26. ISBN¬†978-1-58829-151-6.CS1 maint: Uses editors parameter
      (link)
  21. ^Kandavelou K, Chandrasegaran S (2008). "Plasmids for Gene Therapy".
      Plasmids: Current Research and Future Trends. Caister Academic Press.
      ISBN¬†978-1-904455-35-6.
  22. ^Morange M (December 2009). "What_history_tells_us_XIX._The_notion_of_the
      episome" (PDF). Journal of Biosciences. 34 (6): 845√¢¬Ä¬ì8. doi:10.1007/
      s12038-009-0098-z. PMID¬†20093737.
  23. ^Jacob F, Wollman EL (1958), "Les √É¬©pisomes, elements g√É¬©n√É¬©tiques
      ajout√É¬©s", Comptes Rendus de l'Acad√É¬©mie des Sciences de Paris, 247 (1):
      154√¢¬Ä¬ì156, PMID¬†13561654
  24. ^Hayes, W (1969). "What_are_episomes_and_plasmids?". In Gordon E. W.
      Wolstenholme; Maeve O'Connor (eds.). Bacterial Episomes and Plasmids
      (CIBA Foundation Symposium). pp.¬†4√¢¬Ä¬ì8. ISBN¬†978-0700014057.
  25. ^Gordon E. W. Wolstenholme; Maeve O'Connor, eds. (1969). Bacterial
      Episomes_and_Plasmids_(CIBA_Foundation_Symposium). pp.¬†244√¢¬Ä¬ì245.
      ISBN¬†978-0700014057.
  26. ^T. A. Brown (2011). Introduction_to_Genetics:_A_Molecular_Approach.
      Garland Science. p.¬†238. ISBN¬†978-0815365099.
  27. ^Van Craenenbroeck K, Vanhoenacker P, Haegeman G (September 2000).
      "Episomal vectors for gene expression in mammalian cells". European
      Journal of Biochemistry. 267 (18): 5665√¢¬Ä¬ì78. doi:10.1046/j.1432-
      1327.2000.01645.x. PMID¬†10971576.
  28. ^Colosimo A, Goncz KK, Holmes AR, Kunzelmann K, Novelli G, Malone RW,
      Bennett MJ, Gruenert DC (August 2000). "Transfer_and_expression_of
      foreign_genes_in_mammalian_cells" (PDF). BioTechniques. 29 (2): 314√¢¬Ä¬ì8,
      320√¢¬Ä¬ì2, 324 passim. doi:10.2144/00292rv01. PMID¬†10948433. Archived from
      the_original (PDF) on 24 July 2011.
  29. ^Gerdes K, Rasmussen PB, Molin S (May 1986). "Unique_type_of_plasmid
      maintenance_function:_postsegregational_killing_of_plasmid-free_cells".
      Proceedings of the National Academy of Sciences of the United States of
      America. 83 (10): 3116√¢¬Ä¬ì20. Bibcode:1986PNAS...83.3116G. doi:10.1073/
      pnas.83.10.3116. PMC¬†323463. PMID¬†3517851.
  30. ^Kroll J, Klinter S, Schneider C, Voss I, Steinb√É¬ºchel A (November 2010).
      "Plasmid_addiction_systems:_perspectives_and_applications_in
      biotechnology". Microbial Biotechnology. 3 (6): 634√¢¬Ä¬ì57. doi:10.1111/
      j.1751-7915.2010.00170.x. PMC¬†3815339. PMID¬†21255361.
  31. ^Gunge N, Murata K, Sakaguchi K (July 1982). "Transformation_of
      Saccharomyces_cerevisiae_with_linear_DNA_killer_plasmids_from
      Kluyveromyces_lactis". Journal of Bacteriology. 151 (1): 462√¢¬Ä¬ì4.
      PMC¬†220260. PMID¬†7045080.
  32. ^"Vector_NTI_feedback_video". The DNA Lab.
***** Further reading[edit] *****
    * Klein DW, Prescott LM, Harley J (1999). Microbiology. Boston: WCB/McGraw-
      Hill.
Smith CU. Elements of Molecular Neurobiology. Wiley. pp.¬†101, 111.
Moat AG, Foster JW, Spector MP (2002). Microbial Physiology. Wiley-Liss.
ISBN¬†978-0-471-39483-9.
**** Episomes[edit] ****
    * Piechaczek C, Fetzer C, Baiker A, Bode J, Lipps HJ (January 1999). "A
      vector_based_on_the_SV40_origin_of_replication_and_chromosomal_S/MARs
      replicates_episomally_in_CHO_cells". Nucleic Acids Research. 27 (2):
      426√¢¬Ä¬ì8. doi:10.1093/nar/27.2.426. PMC¬†148196. PMID¬†9862961.
Bode J; Fetzer CP; Nehlsen K; Scinteie M; Hinrichsen B-H; Baiker A; Piechazcek
C; Benham C; Lipps HJ (2001). "The_Hitchhiking_principle:_Optimizing_episomal
vectors_for_the_use_in_gene_therapy_and_biotechnology" (PDF). Gene Ther Mol
Biol. 6: 33√¢¬Ä¬ì46. Archived from the_original (PDF) on 30 May 2009.
Nehlsen K, Broll S, Bode J (2006). "Replicating_minicircles:_Generation_of
nonviral_episomes_for_the_efficient_modification_of_dividing_cells" (PDF). Gene
Ther Mol Biol. 10: 233√¢¬Ä¬ì244. Archived from the_original (PDF) on 30 May 2009.
Ehrhardt A, Haase R, Schepers A, Deutsch MJ, Lipps HJ, Baiker A (June 2008).
"Episomal_vectors_for_gene_therapy". Current Gene Therapy. 8 (3): 147√¢¬Ä¬ì61.
doi:10.2174/156652308784746440. PMID¬†18537590. Archived from the_original on 26
September 2011.
Argyros O, Wong SP, Niceta M, Waddington SN, Howe SJ, Coutelle C, Miller AD,
Harbottle RP (December 2008). "Persistent episomal transgene expression in
liver following delivery of a scaffold/matrix attachment region containing non-
viral vector". Gene Therapy. 15 (24): 1593√¢¬Ä¬ì605. doi:10.1038/gt.2008.113.
PMID¬†18633447.
Wong SP, Argyros O, Coutelle C, Harbottle RP (August 2009). "Strategies_for_the
episomal_modification_of_cells". Current Opinion in Molecular Therapeutics. 11
(4): 433√¢¬Ä¬ì41. PMID¬†19649988. Archived from the_original on 17 September 2011.
Haase R, Argyros O, Wong SP, Harbottle RP, Lipps HJ, Ogris M, Magnusson T,
Vizoso Pinto MG, Haas J, Baiker A (March 2010). "pEPito:_a_significantly
improved_non-viral_episomal_expression_vector_for_mammalian_cells" (PDF). BMC
Biotechnology. 10: 20. doi:10.1186/1472-6750-10-20. PMC¬†2847955. PMID¬†20230618.
***** External links[edit] *****
    * International_Society_for_Plasmid_Biology_and_other_Mobile_Genetic
      Elements
    * History_of_Plasmids_with_timeline
    * [TPI1_structure.png]Molecular_and_Cellular_Biology_portal
Authority_control [Edit_this_at_Wikidata]     * NDL: 01031244

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Plasmid&oldid=905948510"
Categories:
    * Molecular_biology
    * Mobile_genetic_elements
    * Molecular_biology_techniques
    * Gene_delivery
Hidden categories:
    * CS1_maint:_Uses_editors_parameter
    * Use_dmy_dates_from_February_2013
    * Articles_containing_potentially_dated_statements_from_2007
    * All_articles_containing_potentially_dated_statements
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
‚Å∞
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
‚Å∞
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * √ò¬ß√ô¬Ñ√ò¬π√ò¬±√ò¬®√ô¬ä√ò¬©
    * Asturianu
    * Az√â¬ôrbaycanca
    * √†¬¶¬¨√†¬¶¬æ√†¬¶¬Ç√†¬¶¬≤√†¬¶¬æ
    * √ê¬ë√ë¬ä√ê¬ª√ê¬≥√ê¬∞√ë¬Ä√ë¬Å√ê¬∫√ê¬∏
    * Bosanski
    * Catal√É¬†
    * √Ñ¬åe√Ö¬°tina
    * Dansk
    * Deutsch
    * Eesti
    * √é¬ï√é¬ª√é¬ª√é¬∑√é¬Ω√é¬π√é¬∫√é¬¨
    * Espa√É¬±ol
    * Esperanto
    * Euskara
    * √ô¬Å√ò¬ß√ò¬±√ò¬≥√õ¬å
    * Fran√É¬ßais
    * Galego
    * √≠¬ï¬ú√™¬µ¬≠√¨¬ñ¬¥
    * √ï¬Ä√ï¬°√ï¬µ√ï¬•√ñ¬Ä√ï¬•√ï¬∂
    * √†¬§¬π√†¬§¬ø√†¬§¬®√†¬•¬ç√†¬§¬¶√†¬•¬Ä
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * √ó¬¢√ó¬ë√ó¬®√ó¬ô√ó¬™
    * Jawa
    * √°¬É¬•√°¬É¬ê√°¬É¬†√°¬É¬ó√°¬É¬£√°¬É¬ö√°¬É¬ò
    * √í¬ö√ê¬∞√ê¬∑√ê¬∞√í¬õ√ë¬à√ê¬∞
    * √ê¬ö√ë¬ã√ë¬Ä√ê¬≥√ë¬ã√ê¬∑√ë¬á√ê¬∞
    * Magyar
    * √ê¬ú√ê¬∞√ê¬∫√ê¬µ√ê¬¥√ê¬æ√ê¬Ω√ë¬Å√ê¬∫√ê¬∏
    * √†¬¥¬Æ√†¬¥¬≤√†¬¥¬Ø√†¬¥¬æ√†¬¥¬≥√†¬¥¬Ç
    * Bahasa_Melayu
    * Nederlands
    * √¶¬ó¬•√¶¬ú¬¨√®¬™¬û
    * Norsk
    * Norsk_nynorsk
    * Occitan
    * Polski
    * Portugu√É¬™s
    * Rom√É¬¢n√Ñ¬É
    * √ê¬†√ë¬É√ë¬Å√ë¬Å√ê¬∫√ê¬∏√ê¬π
    * Scots
    * Simple_English
    * Sloven√Ñ¬çina
    * Sloven√Ö¬°√Ñ¬çina
    * √ö¬©√ô¬à√ò¬±√ò¬Ø√õ¬å
    * √ê¬°√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬∏_/_srpski
    * Srpskohrvatski_/_√ë¬Å√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬æ√ë¬Ö√ë¬Ä√ê¬≤√ê¬∞√ë¬Ç√ë¬Å√ê¬∫√ê¬∏
    * Suomi
    * Svenska
    * Tagalog
    * √†¬Æ¬§√†¬Æ¬Æ√†¬Æ¬ø√†¬Æ¬¥√†¬Ø¬ç
    * T√É¬ºrk√É¬ße
    * √ê¬£√ê¬∫√ë¬Ä√ê¬∞√ë¬ó√ê¬Ω√ë¬Å√ë¬å√ê¬∫√ê¬∞
    * Ti√°¬∫¬øng_Vi√°¬ª¬át
    * West-Vlams
    * √§¬∏¬≠√¶¬ñ¬á
Edit_links
    * This page was last edited on 12 July 2019, at 15:16¬†(UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. Wikipedia√Ç¬Æ is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
