The following text has been accessed from https://en.wikipedia.org/wiki/Molecular_phylogenetics at Fri Aug 9 04:02:32 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Molecular phylogenetics ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
The branch of phylogeny that analyzes genetic, hereditary molecular differences
Phylogenetic Tree of Life by Haeckel
Molecular phylogenetics (/mÉËlÉkjÊlÉr_ËfaÉªloÊdÊÉËnÉtÉªks,_mÉ-,
moÊ-/[1][2]) is the branch of phylogeny that analyzes genetic, hereditary
molecular differences, predominately in DNA sequences, to gain information on
an organism's evolutionary relationships. From these analyses, it is possible
to determine the processes by which diversity among species has been achieved.
The result of a molecular phylogenetic analysis is expressed in a phylogenetic
tree. Molecular phylogenetics is one aspect of molecular systematics, a broader
term that also includes the use of molecular data in taxonomy and biogeography.
[3][4][5]
Molecular phylogenetics and molecular_evolution correlate. Molecular evolution
is the process of selective changes (mutations) at a molecular level (genes,
proteins, etc.) throughout various branches in the tree of life (evolution).
Molecular phylogenetics makes inferences of the evolutionary relationships that
arise due to molecular evolution and results in the construction of a
phylogenetic tree. The figure displayed on the right depicts the phylogenetic
tree of life as one of the first detailed trees, according to information known
in the 1870s by Haeckel.[6]
⁰
***** Contents *****
    * 1_History
    * 2_Theoretical_background
    * 3_Techniques_and_applications
    * 4_Molecular_phylogenetic_analysis
    * 5_Limitations
    * 6_See_also
    * 7_Notes_and_references
    * 8_Further_reading
    * 9_External_links
***** History[edit] *****
Further information: History_of_molecular_evolution
The theoretical frameworks for molecular systematics were laid in the 1960s in
the works of Emile_Zuckerkandl, Emanuel_Margoliash, Linus_Pauling, and Walter
M._Fitch.[7] Applications of molecular systematics were pioneered by Charles_G.
Sibley (birds), Herbert_C._Dessauer (herpetology), and Morris_Goodman
(primates), followed by Allan_C._Wilson, Robert_K._Selander, and John_C._Avise
(who studied various groups). Work with protein_electrophoresis began around
1956. Although the results were not quantitative and did not initially improve
on morphological classification, they provided tantalizing hints that long-held
notions of the classifications of birds, for example, needed substantial
revision. In the period of 1974â1986, DNA-DNA_hybridization was the dominant
technique used to measure genetic difference.[8]
***** Theoretical background[edit] *****
Early attempts at molecular systematics were also termed as chemotaxonomy and
made use of proteins, enzymes, carbohydrates, and other molecules that were
separated and characterized using techniques such as chromatography. These have
been replaced in recent times largely by DNA_sequencing, which produces the
exact sequences of nucleotides or bases in either DNA or RNA segments extracted
using different techniques. In general, these are considered superior for
evolutionary studies, since the actions of evolution are ultimately reflected
in the genetic sequences. At present, it is still a long and expensive process
to sequence the entire DNA of an organism (its genome). However, it is quite
feasible to determine the sequence of a defined area of a particular
chromosome. Typical molecular systematic analyses require the sequencing of
around 1000 base_pairs. At any location within such a sequence, the bases found
in a given position may vary between organisms. The particular sequence found
in a given organism is referred to as its haplotype. In principle, since there
are four base types, with 1000 base pairs, we could have 41000 distinct
haplotypes. However, for organisms within a particular species or in a group of
related species, it has been found empirically that only a minority of sites
show any variation at all, and most of the variations that are found are
correlated, so that the number of distinct haplotypes that are found is
relatively small.[9]
In a phylogenetic tree, numerous groupings (clades) exist. A clade may be
defined as a group of organisms having a common ancestor throughout evolution.
This figure illustrates how a clade in a phylogenetic tree may be expressed.
In a molecular systematic analysis, the haplotypes are determined for a defined
area of genetic_material; a substantial sample of individuals of the target
species or other taxon is used; however, many current studies are based on
single individuals. Haplotypes of individuals of closely related, yet
different, taxa are also determined. Finally, haplotypes from a smaller number
of individuals from a definitely different taxon are determined: these are
referred to as an outgroup. The base sequences for the haplotypes are then
compared. In the simplest case, the difference between two haplotypes is
assessed by counting the number of locations where they have different bases:
this is referred to as the number of substitutions (other kinds of differences
between haplotypes can also occur, for example, the insertion of a section of
nucleic_acid in one haplotype that is not present in another). The difference
between organisms is usually re-expressed as a percentage divergence, by
dividing the number of substitutions by the number of base pairs analysed: the
hope is that this measure will be independent of the location and length of the
section of DNA that is sequenced.
An older and superseded approach was to determine the divergences between the
genotypes of individuals by DNA-DNA_hybridization. The advantage claimed for
using hybridization rather than gene sequencing was that it was based on the
entire genotype, rather than on particular sections of DNA. Modern sequence
comparison techniques overcome this objection by the use of multiple sequences.
Once the divergences between all pairs of samples have been determined, the
resulting triangular_matrix of differences is submitted to some form of
statistical cluster_analysis, and the resulting dendrogram is examined in order
to see whether the samples cluster in the way that would be expected from
current ideas about the taxonomy of the group. Any group of haplotypes that are
all more similar to one another than any of them is to any other haplotype may
be said to constitute a clade, which may be visually represented as the figure
displayed on the right demonstrates. Statistical techniques such as
bootstrapping and jackknifing help in providing reliability estimates for the
positions of haplotypes within the evolutionary trees.
***** Techniques and applications[edit] *****
Every living organism contains deoxyribonucleic acid (DNA), ribonucleic acid
(RNA), and proteins. In general, closely related organisms have a high degree
of similarity in the molecular_structure of these substances, while the
molecules of organisms distantly related often show a pattern of dissimilarity.
Conserved sequences, such as mitochondrial DNA, are expected to accumulate
mutations over time, and assuming a constant rate of mutation, provide a
molecular_clock for dating divergence. Molecular phylogeny uses such data to
build a "relationship tree" that shows the probable evolution of various
organisms. With the invention of Sanger_sequencing in 1977, it became possible
to isolate and identify these molecular structures.[10][11] High-throughput
sequencing may also be used to obtain the transcriptome of an organism,
allowing inference_of_phylogenetic_relationships_using_transcriptomic_data.
The most common approach is the comparison of homologous_sequences for genes
using sequence_alignment techniques to identify similarity. Another application
of molecular phylogeny is in DNA_barcoding, wherein the species of an
individual organism is identified using small sections of mitochondrial_DNA or
chloroplast_DNA. Another application of the techniques that make this possible
can be seen in the very limited field of human genetics, such as the ever-more-
popular use of genetic_testing to determine a child's paternity, as well as the
emergence of a new branch of criminal forensics focused on evidence known as
genetic_fingerprinting.
***** Molecular phylogenetic analysis[edit] *****
There are several methods available for performing a molecular phylogenetic
analysis. One method, including a comprehensive step-by-step protocol on
constructing a phylogenetic tree, including DNA/Amino Acid contiguous sequence
assembly, multiple_sequence_alignment, model-test (testing best-fitting
substitution models), and phylogeny reconstruction using Maximum Likelihood and
Bayesian Inference, is available at Nature Protocol.[12]
Another molecular phylogenetic analysis technique has been described by Pevsner
and shall be summarized in the sentences to follow (Pevsner, 2015). A
phylogenetic analysis typically consists of five major steps. The first stage
comprises sequence acquisition. The following step consists of performing a
multiple sequence alignment, which is the fundamental basis of constructing a
phylogenetic tree. The third stage includes different models of DNA and amino
acid substitution. Several models of substitution exist. A few examples include
Hamming_distance, the Jukes and Cantor one-parameter model, and the Kimura two-
parameter model (see Models_of_DNA_evolution). The fourth stage consists of
various methods of tree building, including distance-based and character-based
methods. The normalized Hamming distance and the Jukes-Cantor correction
formulas provide the degree of divergence and the probability that a nucleotide
changes to another, respectively. Common tree-building methods include
unweighted pair group method using arithmetic mean (UPGMA) and Neighbor
joining, which are distance-based methods, Maximum_parsimony, which is a
character-based method, and Maximum_likelihood_estimation and Bayesian
inference, which are character-based/model-based methods. UPGMA is a simple
method; however, it is less accurate than the neighbor-joining approach.
Finally, the last step comprises evaluating the trees. This assessment of
accuracy is composed of consistency, efficiency, and robustness.[13]
Five Stages of Molecular Phylogenetic Analysis
MEGA_(molecular_evolutionary_genetics_analysis) is an analysis software that is
user-friendly and free to download and use. This software is capable of
analyzing both distance-based and character-based tree methodologies. MEGA also
contains several options one may choose to utilize, such as heuristic
approaches and bootstrapping. Bootstrapping is an approach that is commonly
used to measure the robustness of topology in a phylogenetic tree, which
demonstrates the percentage each clade is supported after numerous replicates.
In general, a value greater than 70% is considered significant. The flow chart
displayed on the right visually demonstrates the order of the five stages of
Pevsner's molecular phylogenetic analysis technique that have been described.
[13]
***** Limitations[edit] *****
Molecular systematics is an essentially cladistic approach: it assumes that
classification must correspond to phylogenetic descent, and that all valid taxa
must be monophyletic. This is a limitation when attempting to determine the
optimal tree(s), which often involves bisecting and reconnecting portions of
the phylogenetic tree(s).
The recent discovery of extensive horizontal_gene_transfer among organisms
provides a significant complication to molecular systematics, indicating that
different genes within the same organism can have different phylogenies.
In addition, molecular phylogenies are sensitive to the assumptions and models
that go into making them. They face issues such as long-branch_attraction,
saturation, and taxon sampling problems: This means that strikingly different
results can be obtained by applying different models to the same dataset.[14]
Moreover, as previously mentioned, UPGMA is a simple approach in which the tree
is always rooted. The algorithm assumes a constant molecular clock for
sequences in the tree. This is associated with being a limitation in that if
unequal substitution rates exist, the result may be an incorrect tree.[13]
***** See also[edit] *****
    * Computational_phylogenetics
    * Microbial_phylogenetics
    * Molecular_clock
    * Molecular_evolution
    * PhyloCode
    * Phylogenetic_nomenclature
***** Notes and references[edit] *****
   1. ^Jones,_Daniel (2003) [1917], Peter Roach; James Hartmann; Jane Setter
      (eds.), English Pronouncing Dictionary, Cambridge: Cambridge University
      Press, ISBN 3-12-539683-2
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^"Phylogenetic". Merriam-Webster_Dictionary.
   4. ^ Felsenstein,_J. 2004. Inferring phylogenies. Sinauer Associates
      Incorporated.
   5. ISBN 0-87893-177-5.
   6. ^ Soltis,_P.S., Soltis,_D.E., and Doyle, J.J. (1992) Molecular
      systematics of plants. Chapman & Hall, New York.
   7. ISBN 0-41202-231-1.
   8. ^ Soltis, P.S., Soltis, D.E., and Doyle, J.J. (1998) Molecular
      Systematics of Plants II: DNA Sequencing. Kluwer Academic Publishers
      Boston, Dordrecht, London.
   9. ISBN 0-41211-131-4.
  10. ^ Hillis,_D._M. & Moritz, C. 1996. Molecular systematics. 2nd ed. Sinauer
      Associates Incorporated.
  11. ISBN 0-87893-282-8.
  12. ^SuÃ¡rez-DÃ­az, Edna & Anaya-MuÃ±oz, Victor H. (2008). "History,
      objectivity, and the construction of molecular phylogenies". Stud. Hist.
      Phil. Biol. & Biomed. Sci. 39 (4): 451â468. doi:10.1016/
      j.shpsc.2008.09.002. PMID 19026976.
  13. ^Ahlquist, Jon E. (1999). "Charles_G._Sibley:_A_commentary_on_30_years_of
      collaboration". The Auk. 116 (3): 856â860. doi:10.2307/4089352.
      JSTOR 4089352.
  14. ^Page, Roderic D. M.; Holmes,_Edward_C. (1998). Molecular_evolution :_a
      phylogenetic_approach. Oxford: Blackwell_Science. ISBN 9780865428898.
      OCLC 47011609.
  15. ^Sanger F, Coulson AR (May 1975). "A rapid method for determining
      sequences in DNA by primed synthesis with DNA polymerase". J. Mol. Biol.
      94 (3): 441â8. doi:10.1016/0022-2836(75)90213-2. PMID 1100841.
  16. ^Sanger F, Nicklen S, Coulson AR (December 1977). "DNA_sequencing_with
      chain-terminating_inhibitors". Proc. Natl. Acad. Sci. U.S.A. 74 (12):
      5463â7. Bibcode:1977PNAS...74.5463S. doi:10.1073/pnas.74.12.5463.
      PMC 431765. PMID 271968.
  17. ^Bast, F. (2013). "Sequence Similarity Search, Multiple Sequence
      Alignment, Model Selection, Distance Matrix and Phylogeny
      Reconstruction". Protoc. Exch. doi:10.1038/protex.2013.065.
  18. ^ a b cPevsner, J. (2015). "Chapter 7: Molecular Phylogeny and
      Evolution". Bioinformatics and Functional Genomics (3rd ed.). Wiley-
      Blackwell. pp. 245â295. ISBN 978-1-118-58178-0.
  19. ^Philippe, H.; Brinkmann, H.; Lavrov, D. V.; Littlewood, D. T. J.;
      Manuel, M.; WÃ¶rheide, G.; Baurain, D. (2011). Penny, David (ed.).
      "Resolving_Difficult_Phylogenetic_Questions:_Why_More_Sequences_Are_Not
      Enough". PLoS Biology. 9 (3): e1000602. doi:10.1371/journal.pbio.1000602.
      PMC 3057953. PMID 21423652.
***** Further reading[edit] *****
    * San Mauro, D.; Agorreta, A. (2010). "Molecular_systematics:_a_synthesis
      of_the_common_methods_and_the_state_of_knowledge". Cellular & Molecular
      Biology Letters. 15 (2): 311â341. doi:10.2478/s11658-010-0010-8.
      PMC 6275913.
***** External links[edit] *****
    * NCBI_â_Systematics_and_Molecular_Phylogenetics
    * MEGA_Software
    * The_promise_of_a_DNA_taxonomy_(Mark_L._Blaxter)
    * Molecular_phylogenetics from EncyclopÃ¦dia Britannica.
    * v
    * t
    * e
Phylogenetics
                      * Computational
                        phylogenetics
                      * Molecular phylogenetics
Relevant fields       * Cladistics
                      * Taxonomy
                      * Evolutionary_taxonomy
                      * Systematics
                      * Phylogenesis
                            o Cladogenesis
                      * Phylogenetic_tree
                            o Cladogram
Basic concepts        * Phylogenetic_network
                      * Long_branch_attraction
                      * Clade vs Grade
                      * Lineage
                            o Ghost_lineage
                      * Ghost_population
                      * Maximum_parsimony
                      * Probabilistic methods
                            o Maximum_likelihood
                            o Bayesian_inference
Inference methods     * Distance-matrix_methods
                            o Neighbor-joining
                            o UPGMA
                            o Least_squares           * [icon]Evolutionary
                      * Three-taxon_analysis            biology_portal
                      * PhyloCode
                      * DNA_barcoding
                      * Molecular phylogenetics
                      * Phylogenetic_comparative
Current topics          methods
                      * Phylogenetic_niche
                        conservatism
                      * Phylogenetics_software
                      * Phylogenomics
                      * Phylogeography
                      * Primitive
                            o Plesiomorphy
                            o Symplesiomorphy
Group traits          * Derived
                            o Apomorphy
                            o Synapomorphy
                            o Autapomorphy
                      * Monophyly
Group types           * Paraphyly
                      * Polyphyly
                      * Phylogenetic_nomenclature
                      * Crown_group
Nomenclature          * Sister_group
                      * Basal
                      * Supertree
    * [Category] Category
    * [Commons page] Commons
    * v
    * t
    * e
Bioinformatics
                   * Sequence databases: GenBank, European_Nucleotide_Archive
                     and DNA_Data_Bank_of_Japan
                   * Secondary databases: UniProt, database of protein
                     sequences grouping together Swiss-Prot, TrEMBL and Protein
Databases            Information_Resource
                   * Other databases: Protein_Data_Bank, Ensembl and InterPro
                   * Specialised genomic databases: BOLD, Saccharomyces_Genome
                     Database, FlyBase, VectorBase, WormBase, PHI-base,
                     Arabidopsis_Information_Resource and Zebrafish_Information
                     Network
                   * BLAST
                   * Bowtie
                   * Clustal
Software           * HMMER
                   * MUSCLE
                   * SAMtools
                   * TopHat
Other              * Server: ExPASy
                   * Ontology: Gene_Ontology
                   * European_Bioinformatics_Institute
                   * US_National_Center_for_Biotechnology_Information
                   * Swiss_Institute_of_Bioinformatics
Institutions       * Japanese_Institute_of_Genetics
                   * Philippine_Genome_Center
                   * Broad_Institute
                   * Wellcome_Sanger_Institute
                   * International_Society_for_Computational_Biology (ISCB)
                   * European_Molecular_Biology_network (EMBnet)
Organizations      * African_Society_for_Bioinformatics_and_Computational
                     Biology (ASBCB)
                   * Japanese_Society_for_Bioinformatics (JSBi)
                   * Intelligent_Systems_for_Molecular_Biology (ISMB)
                   * Research_in_Computational_Molecular_Biology (RECOMB)
                   * European_Conference_on_Computational_Biology (ECCB)
Meetings           * Pacific_Symposium_on_Biocomputing (PSB)
                   * ISCB_Africa_ASBCB_Conference_on_Bioinformatics
                   * Basel_Computational_Biology_Conferenceâ ([BC2])
                   * International_Conference_on_Bioinformatics (InCoB)
                   * Computational_biology
                   * List_of_biological_databases
Related topics     * Sequencing
                   * Sequence_database
                   * Sequence_alignment
                   * Molecular phylogenetics
    * [Category] Category
    * [Commons page] Commons

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Molecular_phylogenetics&oldid=907979485"
Categories:
    * Phylogenetics
    * Molecular_evolution
Hidden categories:
    * Articles_with_short_description
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * EspaÃ±ol
    * Esperanto
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * æ¥æ¬èª
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 26 July 2019, at 15:34 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
