The following text has been accessed from https://en.wikipedia.org/wiki/Marker-assisted_selection at Thu Aug 8 23:58:23 IST 2019
Creative_Commons_Attribution-ShareAlike_License




















****** Marker-assisted selection ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
 This article includes a list_of_references, but its sources remain unclear
 because it has insufficient inline_citations. Please help to improve this
 article by introducing more precise citations. (June 2016)(Learn_how_and_when
 to_remove_this_template_message)
Marker assisted selection or marker aided selection (MAS) is an indirect
selection process where a trait of interest is selected based on a marker
(morphological, biochemical or DNA/RNA variation) linked to a trait of interest
(e.g. productivity, disease resistance, abiotic stress tolerance, and quality),
rather than on the trait itself.[1][2][3][4] This process has been extensively
researched and proposed for plant and animal_breeding, nevertheless, as of 2013
"breeding programs based on DNA markers for improving quantitative traits in
plants are rare".[5]
For example, using MAS to select individuals with disease resistance involves
identifying a marker allele that is linked with disease resistance rather than
the level of disease resistance. The assumption is that the marker associates
at high frequency with the gene or quantitative_trait_locus (QTL) of interest,
due to genetic linkage (close proximity, on the chromosome, of the marker locus
and the disease resistance-determining locus). MAS can be useful to select for
traits that are difficult or expensive to measure, exhibit low heritability
and/or are expressed late in development. At certain points in the breeding
process the specimens are examined to ensure that they express the desired
trait.
‚Å∞
***** Contents *****
    * 1_Marker_types
    * 2_Positive_and_negative_selectable_markers
    * 3_Gene_vs_marker
    * 4_Important_properties_of_ideal_markers_for_MAS
    * 5_Drawbacks_of_morphological_markers
    * 6_Selection_for_major_genes_linked_to_markers
    * 7_Situations_that_are_favorable_for_molecular_marker_selection
    * 8_Steps_for_MAS
    * 9_QTL_mapping_techniques
    * 10_Single_step_MAS_and_QTL_mapping
    * 11_High-throughput_genotyping_techniques
    * 12_Use_of_MAS_for_backcross_breeding
    * 13_Marker_assisted_gene_pyramiding
    * 14_See_also
    * 15_References
    * 16_Further_reading
***** Marker types[edit] *****
The majority of MAS work in the present era uses DNA-based markers. However,
the first markers that allowed indirect selection of a trait of interest were
morphological markers. In 1923, Sax[who?] first reported association of a
simply inherited genetic_marker with a quantitative trait in plants when he
observed segregation of seed size associated with segregation for a seed coat
color marker in beans (Phaseolus_vulgaris L. ). In 1935, Rasmusson demonstrated
linkage of flowering time (a quantitative trait) in peas with a simply
inherited gene for flower color.[citation_needed]
Markers may be:
    * Morphological - These markers are often detectable by eye, by simple
      visual inspection. Examples of this type of marker include the presence
      or absence of an awn, leaf sheath coloration, height, grain color, aroma
      of rice etc. In well-characterized crops like maize, tomato, pea, barley
      or wheat, tens or hundreds of genes that determine morphological traits
      have been mapped to specific chromosome locations.
    * Biochemical- A protein that can be extracted and observed; for example,
      isozymes and storage proteins.
    * Cytological - The chromosomal banding produced by different stains; for
      example, G_banding.
    * DNA-based- Including microsatellites (also known as short tandem repeats,
      STRs, or simple sequence repeats, SSRs), restriction_fragment_length
      polymorphism (RFLP), random_amplification_of_polymorphic_DNA (RAPD),
      amplified_fragment_length_polymorphism (AFLP), and single nucleotide
      polymorphisms (SNPs).[6]
***** Positive and negative selectable markers[edit] *****
The following terms are generally less relevant to discussions of MAS in plant
and animal breeding, but are highly relevant in molecular biology research:
    * Positive selectable markers are selectable markers that confer selective
      advantage to the host organism.[7] An example would be antibiotic
      resistance, which allows the host organism to survive antibiotic
      selection.
    * Negative selectable markers are selectable markers that eliminate or
      inhibit growth of the host organism upon selection.[8] An example would
      be thymidine_kinase, which makes the host sensitive to ganciclovir
      selection.
A distinction can be made between selectable markers (which eliminate certain
genotypes from the population) and screenable markers (which cause certain
genotypes to be readily identifiable, at which point the experimenter must
"score" or evaluate the population and act to retain the preferred genotypes).
Most MAS uses screenable markers rather than selectable markers.
***** Gene vs marker[edit] *****
The gene of interest directly causes production of protein(s) or RNA that
produce a desired trait or phenotype, whereas markers (a DNA sequence or the
morphological or biochemical markers produced due to that DNA) are genetically
linked to the gene of interest. The gene of interest and the marker tend to
move together during segregation of gametes due to their proximity on the same
chromosome and concomitant reduction in recombination (chromosome crossover
events) between the marker and gene of interest. For some traits, the gene of
interest has been discovered and the presence of desirable alleles can be
directly assayed with a high level of confidence. However, if the gene of
interest is not known, markers linked to the gene of interest can still be used
to select for individuals with desirable alleles of the gene of interest. When
markers are used there may be some inaccurate results due to inaccurate tests
for the marker. There also can be false positive results when markers are used,
due to recombination between the marker of interest and gene (or QTL). A
perfect marker would elicit no false positive results. The term 'perfect
marker' is sometimes used when tests are performed to detect a SNP or other DNA
polymorphism in the gene of interest, if that SNP or other polymorphism is the
direct cause of the trait of interest. The term 'marker' is still appropriate
to use when directly assaying the gene of interest, because the test of
genotype is an indirect test of the trait or phenotype of interest.[citation
needed]
***** Important properties of ideal markers for MAS[edit] *****
An ideal marker:
    * Easy recognition of all possible phenotypes (homo- and heterozygotes)
      from all different alleles
    * Demonstrates measurable differences in expression between trait types or
      gene of interest alleles, early in the development of the organism
    * Testing for the marker does not have variable success depending on the
      allele at the marker locus or the allele at the target locus (the gene of
      interest that determines the trait of interest).
    * Low or null interaction among the markers allowing the use of many at the
      same time in a segregating population
    * Abundant in number
    * Polymorphic
***** Drawbacks of morphological markers[edit] *****
Morphological markers are associated with several general deficits that reduce
their usefulness including:
    * the delay of marker expression until late into the development of the
      organism
    * dominance
    * deleterious effects
    * pleiotropy
    * confounding effects of genes unrelated to the gene or trait of interest
      but which also affect the morphological marker (epistasis)
    * rare polymorphism
    * frequent confounding effects of environmental factors which affect the
      morphological characteristics of the organism
To avoid problems specific to morphological markers, DNA-based markers have
been developed. They are highly polymorphic, exhibit simple inheritance (often
codominant), are abundant throughout the genome, are easy and fast to detect,
exhibit minimum pleiotropic effects, and detection is not dependent on the
developmental stage of the organism. Numerous markers have been mapped to
different chromosomes in several crops including rice, wheat, maize, soybean
and several others, and in livestock such as cattle, pigs and chickens. Those
markers have been used in diversity analysis, parentage detection, DNA
fingerprinting, and prediction of hybrid performance. Molecular markers are
useful in indirect selection processes, enabling manual selection of
individuals for further propagation.
***** Selection for major genes linked to markers[edit] *****
'Major genes' that are responsible for economically important characteristics
are frequent in the plant kingdom. Such characteristics include disease
resistance, male sterility,[9] self-incompatibility, and others related to
shape, color, and architecture of whole plants and are often of mono- or
oligogenic in nature. The marker loci that are tightly linked to major genes
can be used for selection and are sometimes more efficient than direct
selection for the target gene. Such advantages in efficiency may be due for
example, to higher expression of the marker mRNA in such cases that the marker
is itself a gene. Alternatively, in such cases that the target gene of interest
differs between two alleles by a difficult-to-detect single_nucleotide
polymorphism, an external marker (be it another gene or a polymorphism that is
easier to detect, such as a short_tandem_repeat) may present as the most
realistic option.
***** Situations that are favorable for molecular marker selection[edit] *****
There are several indications for the use of molecular markers in the selection
of a genetic trait.
In such situations that:
    * the selected character is expressed late in plant development, like fruit
      and flower features or adult characters with a juvenile period (so that
      it is not necessary to wait for the organism to become fully developed
      before arrangements can be made for propagation)
    * the expression of the target gene is recessive (so that individuals which
      are heterozygous_positive for the recessive allele can be crossed to
      produce some homozygous offspring with the desired trait)
    * there is requirement for the presence of special conditions in order to
      invoke expression of the target gene(s), as in the case of breeding for
      disease and pest resistance (where inoculation with the disease or
      subjection to pests would otherwise be required). This advantage derives
      from the errors due to unreliable inoculation methods and the fact that
      field inoculation with the pathogen is not allowed in many areas for
      safety reasons. Moreover, problems in the recognition of the
      environmentally unstable genes can be eluded.
    * the phenotype is affected by two or more unlinked genes (epistatis). For
      example, selection for multiple genes which provide resistance against
      diseases or insect pests for gene_pyramiding.
The cost of genotyping (an example of a molecular marker assay) is reducing
while the cost of phenotyping is increasing[citation_needed] particularly in
developed countries thus increasing the attractiveness of MAS as the
development of the technology continues.
***** Steps for MAS[edit] *****
Generally the first step is to map the gene or quantitative_trait_locus (QTL)
of interest first by using different techniques and then using this information
for marker assisted selection. Generally, the markers to be used should be
close to gene of interest (<5 recombination_unit or cM) in order to ensure that
only minor fraction of the selected individuals will be recombinants.
Generally, not only a single marker but rather two markers are used in order to
reduce the chances of an error due to homologous recombination. For example, if
two flanking markers are used at same time with an interval between them of
approximately 20cM, there is higher probability (99%) for recovery of the
target gene.
***** QTL mapping techniques[edit] *****
Main article: Quantitative_trait_locus_√Ç¬ß¬†QTL_mapping
In plants QTL mapping is generally achieved using bi-parental cross
populations; a cross between two parents which have a contrasting phenotype for
the trait of interest are developed. Commonly used populations are near
isogenic lines (NILs), recombinant inbred lines (RILs), doubled_haploids (DH),
back cross and F2. Linkage between the phenotype and markers which have already
been mapped is tested in these populations in order to determine the position
of the QTL. Such techniques are based on linkage and are therefore referred to
as "linkage_mapping".A
***** Single step MAS and QTL mapping[edit] *****
In contrast to two-step QTL mapping and MAS, a single-step method for breeding
typical plant populations has been developed.[10][11]
In such an approach, in the first few breeding cycles, markers linked to the
trait of interest are identified by QTL mapping and later the same information
is used in the same population. In this approach, pedigree structure is created
from families that are created by crossing number of parents (in three-way or
four way crosses). Both phenotyping and genotyping is done using molecular
markers mapped the possible location of QTL of interest. This will identify
markers and their favorable alleles. Once these favorable marker alleles are
identified, the frequency of such alleles will be increased and response to
marker assisted selection is estimated. Marker allele(s) with desirable effect
will be further used in next selection cycle or other experiments.
***** High-throughput genotyping techniques[edit] *****
Recently high-throughput genotyping techniques are developed which allows
marker aided screening of many genotypes. This will help breeders in shifting
traditional breeding to marker aided selection. One example of such automation
is using DNA isolation robots, capillary electrophoresis and pipetting robots.
One recent example of capllilary system is Applied Biosystems 3130 Genetic
Analyzer. This is the latest generation of 4-capillary electrophoresis
instruments for the low to medium throughput laboratories.
***** Use of MAS for backcross breeding[edit] *****
A minimum of five or six-backcross generations are required to transfer a gene
of interest from a donor (may not be adapted) to a recipient (recurrent √¢¬Ä¬ì
adapted cultivar). The recovery of the recurrent genotype can be accelerated
with the use of molecular markers. If the F1 is heterozygous for the marker
locus, individuals with the recurrent parent allele(s) at the marker locus in
first or subsequent backcross generations will also carry a chromosome tagged
by the marker.
***** Marker assisted gene pyramiding[edit] *****
Gene pyramiding has been proposed and applied to enhance resistance to disease
and insects by selecting for two or more than two genes at a time. For example,
in rice such pyramids have been developed against bacterial blight and blast.
The advantage of use of markers in this case allows to select for QTL-allele-
linked markers that have same phenotypic effect.
ABI 3130 genetic analyzer.
MAS has also been proved useful for livestock improvement.[12]
A coordinated effort to implement wheat (Triticum turgidum and Triticum
aestivum) marker assisted selection in the U.S. as well as a resource for
marker assisted selection exists at the Wheat_CAP_(Coordinated_Agricultural
Project)_website.
***** See also[edit] *****
    * Genomics_of_domestication
    * Selection_methods_in_plant_breeding_based_on_mode_of_reproduction
    * Smart_breeding
    * Molecular_breeding
    * QTL_mapping
    * Family_based_QTL_mapping
    * Association_mapping
    * Nested_association_mapping
    * History_of_plant_breeding
***** References[edit] *****
   1. ^ review_MAS_in_plant_breeding
   2. ^ Ribaut, J.-M. et al., Genetic basis of physiological traits. In
      Application of Physiology in Wheat Breeding, CIMMYT, Mexico, 2001.
   3. ^ Ribaut, J.-M. and Hoisington, D. A., Marker assisted selection: new
      tools and strategies. Trends Plant Sci., 1998, 3, 236√¢¬Ä¬ì239.
   4. ^ Rosyara, U.R. 2006. REQUIREMENT OF ROBUST MOLECULAR MARKER TECHNOLOGY
      FOR PLANT BREEDING APPLICATIONS. Journal of Plant Breed. Gr. 1: 67 √¢¬Ä¬ì
      72. click_to_download
   5. ^Ben-Ari, Giora; Lavi, Uri (2012). Plant_Biotechnology_and_Agriculture.
      Chapter_11:_Marker-assisted_selection_in_plant_breeding. Science Direct.
      pp.¬†163√¢¬Ä¬ì184.
   6. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   7. ^Gous Miah, Mohd Y. Rafii, Mohd R. Ismail, Adam B. Puteh, Harun A. Rahim,
      Kh. Nurul Islam, Mohammad Abdul Latif (2013). "A_Review_of_Microsatellite
      Markers_and_Their_Applications_in_Rice_Breeding_Programs_to_Improve_Blast
      Disease_Resistance". Int. J. Mol. Sci. 14 (11): 22499√¢¬Ä¬ì22528. doi:
      10.3390/ijms141122499. PMC¬†3856076. PMID¬†24240810.CS1 maint: Uses authors
      parameter (link)
   8. ^"positive_selection". Scitable. Nature. Retrieved 29 September 2011.
   9. ^"negative_selection". Scitable. Nature. Retrieved 29 September 2011.
  10. ^Nowicki, Marcin; et al. (26 October 2013), More_than_meets_the_eye:_A
      multi-year_expressivity_analyses_of_tomato_sterility_in_ps_and_ps-2_lines
      (PDF), Australian Journal of Crop Science, 7(13):2154-2161; Southern
      Cross Publishing, retrieved 29 October 2013
  11. ^ Rosyara, U. R.; K.L. Maxson-Stein; K.D. Glover; J.M. Stein; J.L.
      Gonzalez-Hernandez. 2007. Family-based mapping of FHB resistance QTLs in
      hexaploid wheat. Proceedings of National Fusarium head blight forum,
      2007, Dec 2-4, Kansas City, MO.
  12. ^ Rosyara U.R., J.L. Gonzalez-Hernandez, K.D. Glover, K.R. Gedye and J.M.
      Stein. 2009. Family-based mapping of quantitative trait loci in plant
      breeding populations with resistance to Fusarium head blight in wheat as
      an illustration Theoretical_Applied_Genetics_118:1617-1631
  13. ^ Dekkers., J. C. M. 2004. Commercial application of marker- and gene-
      assisted selection in livestock: Strategies and lessons. J. Anim. Sci.
      82:E313-E328.
***** Further reading[edit] *****
    * review_application_of_MAS_in_crop_improvement
    * Collard, Bertrand C. Y.; Mackill, David J. (12 February 2008). "Marker-
      assisted_selection:_an_approach_for_precision_plant_breeding_in_the
      twenty-first_century". Philosophical Transactions of the Royal Society B:
      Biological Sciences. 363 (1491): 557√¢¬Ä¬ì572. doi:10.1098/rstb.2007.2170.
      ISSN¬†0962-8436. PMC¬†2610170. PMID¬†17715053.
Gupta, P. K.; Langridge, Peter; Mir, R. R. (11 December 2009). "Marker-assisted
wheat breeding: present status and future possibilities". Molecular Breeding.
26 (2): 145√¢¬Ä¬ì161. doi:10.1007/s11032-009-9359-7. ISSN¬†1380-3743.
Moose, Stephen P.; Mumm, Rita H. (1 July 2008). "Molecular_Plant_Breeding_as
the_Foundation_for_21st_Century_Crop_Improvement". Plant Physiology. 147 (3):
969√¢¬Ä¬ì977. doi:10.1104/pp.108.118232. ISSN¬†1532-2548. PMC¬†2442525.
PMID¬†18612074.
Plant_Breeding_and_Genomics

Retrieved from "https://en.wikipedia.org/w/index.php?title=Marker-
assisted_selection&oldid=902599010"
Categories:
    * Genetics
Hidden categories:
    * CS1_maint:_Uses_authors_parameter
    * Articles_lacking_in-text_citations_from_June_2016
    * All_articles_lacking_in-text_citations
    * All_articles_with_specifically_marked_weasel-worded_phrases
    * Articles_with_specifically_marked_weasel-worded_phrases_from_April_2010
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_January_2008
    * Articles_with_unsourced_statements_from_January_2016
    * Use_dmy_dates_from_April_2017
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
‚Å∞
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
‚Å∞
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * √ò¬ß√ô¬Ñ√ò¬π√ò¬±√ò¬®√ô¬ä√ò¬©
    * Catal√É¬†
    * Deutsch
    * Fran√É¬ßais
    * Bahasa_Indonesia
    * Italiano
    * Suomi
    * T√É¬ºrk√É¬ße
Edit_links
    * This page was last edited on 19 June 2019, at 22:53¬†(UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. Wikipedia√Ç¬Æ is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
