The following text has been accessed from https://en.wikipedia.org/wiki/Bacteria at Fri Aug 9 01:44:57 IST 2019
Creative_Commons_Attribution-ShareAlike_License



















[This_is_a_featured_article._Click_here_for_more_information.]
[Page_semi-protected]
****** Bacteria ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
This article is about the microorganisms. For the genus, see Bacterium_(genus).
For other uses, see Bacteria_(disambiguation).
A domain of prokaryotes √¢¬Ä¬ì single celled organisms without a nucleus
Bacteria
Temporal range: Archean or earlier¬†√¢¬Ä¬ì present
Had'n
Archean
Proterozoic
Pha.
[EscherichiaColi_NIAID.jpg]
Scanning_electron_micrograph of Escherichia_coli rods
Scientific_classification [e]
Domain: Bacteria
        Woese, Kandler & Wheelis, 1990[1]
Phyla
    * Acidobacteria
    * Actinobacteria
    * Aquificae
    * Armatimonadetes
    * Bacteroidetes
    * Caldiserica
    * Chlamydiae
    * Chlorobi
    * Chloroflexi
    * Chrysiogenetes
    * Coprothermobacterota[2]
    * Cyanobacteria
    * Deferribacteres
    * Deinococcus-Thermus
    * Dictyoglomi
    * Elusimicrobia
    * Fibrobacteres
    * Firmicutes
    * Fusobacteria
    * Gemmatimonadetes
    * Lentisphaerae
    * Nitrospirae
    * Planctomycetes
    * Proteobacteria
    * Spirochaetes
    * Synergistetes
    * Tenericutes
    * Thermodesulfobacteria
    * Thermotogae
    * Verrucomicrobia
Synonyms
      Eubacteria Woese & Fox, 1977[3]
Bacteria (/b√É¬¶k√ã¬àt√â¬™√â¬ôri√â¬ô/ ([About_this_sound]listen); common noun bacteria,
singular bacterium) are a type of biological_cell. They constitute a large
domain of prokaryotic microorganisms. Typically a few micrometres in length,
bacteria have a number_of_shapes, ranging from spheres to rods and spirals.
Bacteria were among the first life forms to appear on Earth, and are present in
most of its habitats. Bacteria inhabit soil, water, acidic_hot_springs,
radioactive_waste,[4] and the deep portions of Earth's_crust. Bacteria also
live in symbiotic and parasitic relationships with plants and animals. Most
bacteria have not been characterised, and only about 27 percent of the
bacterial_phyla have species that can be grown in the laboratory (specifically
unculturable phyla, known as candidate phyla, make up 103 out of approximately
142 known phyla). [5] The study of bacteria is known as bacteriology, a branch
of microbiology.
Virtually all animal life on earth is dependent on bacteria for their survival
as only bacteria and some archea possess the genes and enzymes necessary to
synthesize vitamin_B12, also known as cobalamin, and provide it through the
food chain. Vitamin B12 is a water-soluble vitamin that is involved in the
metabolism of every cell of the human body. It is a cofactor in DNA_synthesis,
and in both fatty_acid and amino_acid_metabolism, It is particularly important
in the normal functioning of the nervous_system via its role in the synthesis
of_myelin.[6][7][8][9]
There are typically 40 million bacterial cells in a gram of soil and a million
bacterial cells in a millilitre of fresh_water. There are approximately 5√É¬ó1030
bacteria on Earth,[10] forming a biomass which exceeds that of all plants and
animals.[11] Bacteria are vital in many stages of the nutrient_cycle by
recycling nutrients such as the fixation_of_nitrogen from the atmosphere. The
nutrient cycle includes the decomposition of dead_bodies; bacteria are
responsible for the putrefaction stage in this process.[12] In the biological
communities surrounding hydrothermal_vents and cold_seeps, extremophile
bacteria provide the nutrients needed to sustain life by converting dissolved
compounds, such as hydrogen_sulphide and methane, to energy. Data reported by
researchers in October 2012 and published in March 2013 suggested that bacteria
thrive in the Mariana_Trench, which, with a depth of up to 11 kilometres, is
the deepest known part of the oceans.[13][14] Other researchers reported
related studies that microbes thrive inside rocks up to 580 metres below the
sea floor under 2.6 kilometres of ocean off the coast of the northwestern
United States.[13][15] According to one of the researchers, "You can find
microbes everywhere√¢¬Ä¬îthey're extremely adaptable to conditions, and survive
wherever they are."[13]
The famous notion that bacterial cells in the human body outnumber human cells
by a factor of 10:1 has been debunked. There are approximately 39 trillion
bacterial cells in the human_microbiota as personified by a "reference" 70¬†kg
male 170¬†cm tall, whereas there are 30 trillion human cells in the body. This
means that although they do have the upper hand in actual numbers, it is only
by 30%, and not 900%.[16]
The largest number exist in the gut_flora, and a large number on the skin.[17]
The vast majority of the bacteria in the body are rendered harmless by the
protective effects of the immune_system, though many are beneficial,
particularly in the gut flora. However, several species of bacteria are
pathogenic and cause infectious_diseases, including cholera, syphilis, anthrax,
leprosy, and bubonic_plague. The most common fatal bacterial diseases are
respiratory_infections, with tuberculosis alone killing about 2 million people
per year, mostly in sub-Saharan_Africa.[18] In developed_countries, antibiotics
are used to treat bacterial_infections and are also used in farming, making
antibiotic_resistance a growing problem. In industry, bacteria are important in
sewage_treatment and the breakdown of oil_spills, the production of cheese and
yogurt through fermentation, the recovery of gold, palladium, copper and other
metals in the mining sector,[19] as well as in biotechnology, and the
manufacture of antibiotics and other chemicals.[20]
Once regarded as plants constituting the class Schizomycetes, bacteria are now
classified as prokaryotes. Unlike cells of animals and other eukaryotes,
bacterial cells do not contain a nucleus and rarely harbour membrane-bound
organelles. Although the term bacteria traditionally included all prokaryotes,
the scientific_classification changed after the discovery in the 1990s that
prokaryotes consist of two very different groups of organisms that evolved from
an ancient_common_ancestor. These evolutionary_domains are called Bacteria and
Archaea.[1]
‚Å∞
***** Contents *****
    * 1_Etymology
    * 2_Origin_and_early_evolution
    * 3_Morphology
    * 4_Cellular_structure
          o 4.1_Intracellular_structures
          o 4.2_Extracellular_structures
          o 4.3_Endospores
    * 5_Metabolism
    * 6_Growth_and_reproduction
    * 7_Genetics
    * 8_Behaviour
          o 8.1_Movement
          o 8.2_Communication
    * 9_Classification_and_identification
    * 10_Interactions_with_other_organisms
          o 10.1_Predators
          o 10.2_Mutualists
          o 10.3_Pathogens
    * 11_Significance_in_technology_and_industry
    * 12_History_of_bacteriology
    * 13_Computer-made_form
    * 14_See_also
    * 15_References
    * 16_Further_reading
    * 17_External_links
***** Etymology *****
The word bacteria is the plural of the New_Latin bacterium, which is the
latinisation of the Greek √é¬≤√é¬±√é¬∫√è¬Ñ√é¬Æ√è¬Å√é¬π√é¬ø√é¬Ω (bakterion),[21] the diminutive of
√é¬≤√é¬±√é¬∫√è¬Ñ√é¬∑√è¬Å√é¬Ø√é¬± (bakteria), meaning "staff, cane",[22] because the first ones
to be discovered were rod-shaped.[23][24]
***** Origin and early evolution *****
Further information: Earliest_known_life_forms, Evolutionary_history_of_life,
and Timeline_of_evolution
The ancestors of modern bacteria were unicellular microorganisms that were the
first_forms_of_life to appear on Earth, about 4 billion years ago. For about 3
billion years, most organisms were microscopic, and bacteria and archaea were
the dominant forms of life.[25][26] Although bacterial fossils exist, such as
stromatolites, their lack of distinctive morphology prevents them from being
used to examine the history of bacterial evolution, or to date the time of
origin of a particular bacterial species. However, gene sequences can be used
to reconstruct the bacterial phylogeny, and these studies indicate that
bacteria diverged first from the archaeal/eukaryotic lineage.[27] The most
recent_common_ancestor of bacteria and archaea was probably a hyperthermophile
that lived about 2.5 billion√¢¬Ä¬ì3.2 billion years ago.[28][29]
Bacteria were also involved in the second great evolutionary divergence, that
of the archaea and eukaryotes. Here, eukaryotes resulted from the entering of
ancient bacteria into endosymbiotic associations with the ancestors of
eukaryotic cells, which were themselves possibly related to the Archaea.[30]
[31] This involved the engulfment by proto-eukaryotic cells of
alphaproteobacterial symbionts to form either mitochondria or hydrogenosomes,
which are still found in all known Eukarya (sometimes in highly reduced_form,
e.g. in ancient "amitochondrial" protozoa). Later, some eukaryotes that already
contained mitochondria also engulfed cyanobacteria-like organisms, leading to
the formation of chloroplasts in algae and plants.[32][33] This is known as
primary_endosymbiosis.
In July 2018, scientists reported that the earliest_life on land may have been
bacteria living on land 3.22 billion years ago.[34]
***** Morphology *****
Further information: Bacterial_cell_structure_√Ç¬ß¬†Cell_morphology
[a_diagram_showing_bacteria_morphology]
Bacteria display many cell morphologies and arrangements
Bacteria display a wide diversity of shapes and sizes, called morphologies.
Bacterial cells are about one-tenth the size of eukaryotic cells and are
typically 0.5√¢¬Ä¬ì5.0¬†micrometres in length. However, a few species are visible
to the unaided eye√¢¬Ä¬îfor example, Thiomargarita_namibiensis is up to half a
millimetre long[35] and Epulopiscium_fishelsoni reaches 0.7¬†mm.[36] Among the
smallest bacteria are members of the genus Mycoplasma, which measure only
0.3¬†micrometres, as small as the largest viruses.[37] Some bacteria may be even
smaller, but these ultramicrobacteria are not well-studied.[38]
Most bacterial species are either spherical, called cocci (sing. coccus, from
Greek k√É¬≥kkos, grain, seed), or rod-shaped, called bacilli (sing. bacillus,
from Latin baculus, stick).[39] Some bacteria, called vibrio, are shaped like
slightly curved rods or comma-shaped; others can be spiral-shaped, called
spirilla, or tightly coiled, called spirochaetes. A small number of other
unusual shapes have been described, such as star-shaped bacteria.[40] This wide
variety of shapes is determined by the bacterial cell_wall and cytoskeleton,
and is important because it can influence the ability of bacteria to acquire
nutrients, attach to surfaces, swim through liquids and escape predators.[41]
[42]
The range of sizes shown by prokaryotes, relative to those of other organisms
and biomolecules.
Many bacterial species exist simply as single cells, others associate in
characteristic patterns: Neisseria form diploids (pairs), Streptococcus form
chains, and Staphylococcus group together in "bunch of grapes" clusters.
Bacteria can also group to form larger multicellular structures, such as the
elongated filaments of Actinobacteria, the aggregates of Myxobacteria, and the
complex hyphae of Streptomyces.[43] These multicellular structures are often
only seen in certain conditions. For example, when starved of amino acids,
Myxobacteria detect surrounding cells in a process known as quorum_sensing,
migrate towards each other, and aggregate to form fruiting bodies up to
500¬†micrometres long and containing approximately 100,000 bacterial cells.[44]
In these fruiting bodies, the bacteria perform separate tasks; for example,
about one in ten cells migrate to the top of a fruiting body and differentiate
into a specialised dormant state called a myxospore, which is more resistant to
drying and other adverse environmental conditions.[45]
Bacteria often attach to surfaces and form dense aggregations called biofilms,
and larger formations known as microbial_mats. These biofilms and mats can
range from a few micrometres in thickness to up to half a metre in depth, and
may contain multiple species of bacteria, protists and archaea. Bacteria living
in biofilms display a complex arrangement of cells and extracellular
components, forming secondary structures, such as microcolonies, through which
there are networks of channels to enable better diffusion of nutrients.[46][47]
In natural environments, such as soil or the surfaces of plants, the majority
of bacteria are bound to surfaces in biofilms.[48] Biofilms are also important
in medicine, as these structures are often present during chronic bacterial
infections or in infections of implanted medical_devices, and bacteria
protected within biofilms are much harder to kill than individual isolated
bacteria.[49]
***** Cellular structure *****
Further information: Bacterial_cell_structure
[Prokaryote_cell_with_structure_and_parts]
Structure and contents of a typical gram-positive bacterial cell (seen by the
fact that only one cell membrane is present).
**** Intracellular structures ****
The bacterial cell is surrounded by a cell_membrane which is made primarily of
phospholipids. This membrane encloses the contents of the cell and acts as a
barrier to hold nutrients, proteins and other essential components of the
cytoplasm within the cell.[50] Unlike eukaryotic_cells, bacteria usually lack
large membrane-bound structures in their cytoplasm such as a nucleus,
mitochondria, chloroplasts and the other organelles present in eukaryotic
cells.[51] However, some bacteria have protein-bound organelles in the
cytoplasm which compartmentalize aspects of bacterial metabolism,[52][53] such
as the carboxysome.[54] Additionally, bacteria have a multi-component
cytoskeleton to control the localisation of proteins and nucleic acids within
the cell, and to manage the process of cell_division.[55][56][57]
Many important biochemical reactions, such as energy generation, occur due to
concentration_gradients across membranes, creating a potential difference
analogous to a battery. The general lack of internal membranes in bacteria
means these reactions, such as electron_transport, occur across the cell
membrane between the cytoplasm and the outside of the cell or periplasm.[58]
However, in many photosynthetic bacteria the plasma membrane is highly folded
and fills most of the cell with layers of light-gathering membrane.[59] These
light-gathering complexes may even form lipid-enclosed structures called
chlorosomes in green_sulfur_bacteria.[60]
An electron_micrograph of Halothiobacillus_neapolitanus cells with carboxysomes
inside, with arrows highlighting visible carboxysomes. Scale bars indicate 100
nm.
Bacteria do not have a membrane-bound nucleus, and their genetic material is
typically a single circular_bacterial_chromosome of DNA located in the
cytoplasm in an irregularly shaped body called the nucleoid.[61] The nucleoid
contains the chromosome with its associated proteins and RNA. Like all other
organisms, bacteria contain ribosomes for the production of proteins, but the
structure of the bacterial ribosome is different from that of eukaryotes and
Archaea.[62]
Some bacteria produce intracellular nutrient storage granules, such as
glycogen,[63] polyphosphate,[64] sulfur[65] or polyhydroxyalkanoates.[66]
Certain bacterial species, such as the photosynthetic Cyanobacteria, produce
internal gas_vacuoles which they use to regulate their buoyancy, allowing them
to move up or down into water layers with different light intensities and
nutrient levels.[67]
**** Extracellular structures ****
Further information: Cell_envelope
Around the outside of the cell membrane is the cell_wall. Bacterial cell walls
are made of peptidoglycan (also called murein), which is made from
polysaccharide chains cross-linked by peptides containing D-amino_acids.[68]
Bacterial cell walls are different from the cell walls of plants and fungi,
which are made of cellulose and chitin, respectively.[69] The cell wall of
bacteria is also distinct from that of Archaea, which do not contain
peptidoglycan. The cell wall is essential to the survival of many bacteria, and
the antibiotic penicillin is able to kill bacteria by inhibiting a step in the
synthesis of peptidoglycan.[69]
There are broadly speaking two different types of cell wall in bacteria, that
classify bacteria into gram-positive_bacteria and gram-negative_bacteria. The
names originate from the reaction of cells to the Gram_stain, a long-standing
test for the classification of bacterial species.[70]
Gram-positive bacteria possess a thick cell wall containing many layers of
peptidoglycan and teichoic_acids. In contrast, gram-negative bacteria have a
relatively thin cell wall consisting of a few layers of peptidoglycan
surrounded by a second lipid_membrane containing lipopolysaccharides and
lipoproteins. Most bacteria have the gram-negative cell wall, and only the
Firmicutes and Actinobacteria (previously known as the low G+C and high G+C
gram-positive bacteria, respectively) have the alternative gram-positive
arrangement.[71] These differences in structure can produce differences in
antibiotic susceptibility; for instance, vancomycin can kill only gram-positive
bacteria and is ineffective against gram-negative pathogens, such as
Haemophilus_influenzae or Pseudomonas_aeruginosa.[72] Some bacteria have cell
wall structures that are neither classically gram-positive or gram-negative.
This includes clinically important bacteria such as Mycobacteria which have a
thick peptidoglycan cell wall like a gram-positive bacterium, but also a second
outer layer of lipids.[73]
In many bacteria, an S-layer of rigidly arrayed protein molecules covers the
outside of the cell.[74] This layer provides chemical and physical protection
for the cell surface and can act as a macromolecular diffusion_barrier. S-
layers have diverse but mostly poorly understood functions, but are known to
act as virulence factors in Campylobacter and contain surface enzymes in
Bacillus_stearothermophilus.[75]
[Helicobacter_pylori_electron_micrograph,_showing_multiple_flagella_on_the_cell
surface]
Helicobacter_pylori electron micrograph, showing multiple flagella on the cell
surface
Flagella are rigid protein structures, about 20¬†nanometres in diameter and up
to 20¬†micrometres in length, that are used for motility. Flagella are driven by
the energy released by the transfer of ions down an electrochemical_gradient
across the cell membrane.[76]
Fimbriae (sometimes called "attachment_pili") are fine filaments of protein,
usually 2√¢¬Ä¬ì10¬†nanometres in diameter and up to several micrometres in length.
They are distributed over the surface of the cell, and resemble fine hairs when
seen under the electron_microscope. Fimbriae are believed to be involved in
attachment to solid surfaces or to other cells, and are essential for the
virulence of some bacterial pathogens.[77] Pili (sing. pilus) are cellular
appendages, slightly larger than fimbriae, that can transfer genetic_material
between bacterial cells in a process called conjugation where they are called
conjugation_pili or sex pili (see bacterial genetics, below).[78] They can also
generate movement where they are called type_IV_pili.[79]
Glycocalyx is produced by many bacteria to surround their cells, and varies in
structural complexity: ranging from a disorganised slime_layer of extracellular
polymeric_substances to a highly structured capsule. These structures can
protect cells from engulfment by eukaryotic cells such as macrophages (part of
the human immune_system).[80] They can also act as antigens and be involved in
cell recognition, as well as aiding attachment to surfaces and the formation of
biofilms.[81]
The assembly of these extracellular structures is dependent on bacterial
secretion_systems. These transfer proteins from the cytoplasm into the
periplasm or into the environment around the cell. Many types of secretion
systems are known and these structures are often essential for the virulence of
pathogens, so are intensively studied.[82]
**** Endospores ****
Further information: Endospore
[Anthrax_stained_purple]
Bacillus_anthracis (stained purple) growing in cerebrospinal_fluid
Certain genera of gram-positive bacteria, such as Bacillus, Clostridium,
Sporohalobacter, Anaerobacter, and Heliobacterium, can form highly resistant,
dormant structures called endospores.[83] Endospores develop within the
cytoplasm of the cell; generally a single endospore develops in each cell.[84]
Each endospore contains a core of DNA and ribosomes surrounded by a cortex
layer and protected by a multilayer rigid coat composed of peptidoglycan and a
variety of proteins.[84]
Endospores show no detectable metabolism and can survive extreme physical and
chemical stresses, such as high levels of UV_light, gamma_radiation,
detergents, disinfectants, heat, freezing, pressure, and desiccation.[85] In
this dormant state, these organisms may remain viable for millions of years,
[86][87] and endospores even allow bacteria to survive exposure to the vacuum
and radiation in space.[88] Endospore-forming bacteria can also cause disease:
for example, anthrax can be contracted by the inhalation of Bacillus_anthracis
endospores, and contamination of deep puncture wounds with Clostridium_tetani
endospores causes tetanus.[89]
***** Metabolism *****
Further information: Microbial_metabolism
Bacteria exhibit an extremely wide variety of metabolic types.[90] The
distribution of metabolic traits within a group of bacteria has traditionally
been used to define their taxonomy, but these traits often do not correspond
with modern genetic classifications.[91] Bacterial metabolism is classified
into nutritional_groups on the basis of three major criteria: the source of
energy, the electron_donors used, and the source of carbon used for growth.[92]
Bacteria either derive energy from light using photosynthesis (called
phototrophy), or by breaking down chemical compounds using oxidation (called
chemotrophy).[93] Chemotrophs use chemical compounds as a source of energy by
transferring electrons from a given electron donor to a terminal_electron
acceptor in a redox_reaction. This reaction releases energy that can be used to
drive metabolism. Chemotrophs are further divided by the types of compounds
they use to transfer electrons. Bacteria that use inorganic compounds such as
hydrogren, carbon_monoxide, or ammonia as sources_of_electrons are called
lithotrophs, while those that use organic compounds are called organotrophs.
[93] The compounds used to receive electrons are also used to classify
bacteria: aerobic_organisms use oxygen as the terminal electron acceptor, while
anaerobic_organisms use other compounds such as nitrate, sulfate, or carbon
dioxide.[93]
Many bacteria get their carbon from other organic_carbon, called heterotrophy.
Others such as cyanobacteria and some purple_bacteria are autotrophic, meaning
that they obtain cellular carbon by fixing carbon_dioxide.[94] In unusual
circumstances, the gas methane can be used by methanotrophic bacteria as both a
source of electrons and a substrate for carbon anabolism.[95]
                                    Nutritional types in bacterial metabolism
Nutritional   Source of Source of carbon                                            Examples
type          energy
                        ¬†Organic compounds (photoheterotrophs) or carbon fixation  ¬†Cyanobacteria, Green_sulfur
¬†Phototrophs¬Sunlight  (photoautotrophs)                                           bacteria, Chloroflexi, or
                                                                                    Purple_bacteria¬†
              Inorganic ¬†Organic compounds (lithoheterotrophs) or carbon fixation  ¬†Thermodesulfobacteria,
¬†Lithotrophs compounds (lithoautotrophs)                                           Hydrogenophilaceae, or
                                                                                    Nitrospirae¬†
¬†OrganotrophsOrganic   ¬†Organic compounds (chemoheterotrophs) or carbon fixation  ¬†Bacillus, Clostridium or
              compounds (chemoautotrophs) ¬†                                        Enterobacteriaceae¬†
In many ways, bacterial metabolism provides traits that are useful for
ecological stability and for human society. One example is that some bacteria
have the ability to fix_nitrogen gas using the enzyme nitrogenase. This
environmentally important trait can be found in bacteria of most metabolic
types listed above.[96] This leads to the ecologically important processes of
denitrification, sulfate reduction, and acetogenesis, respectively.[97][98]
Bacterial metabolic processes are also important in biological responses to
pollution; for example, sulfate-reducing_bacteria are largely responsible for
the production of the highly toxic forms of mercury (methyl- and
dimethylmercury) in the environment.[99] Non-respiratory anaerobes use
fermentation to generate energy and reducing power, secreting metabolic by-
products (such as ethanol in brewing) as waste. Facultative_anaerobes can
switch between fermentation and different terminal_electron_acceptors depending
on the environmental conditions in which they find themselves.
***** Growth and reproduction *****
[drawing_of_showing_the_processes_of_binary_fission,_mitosis,_and_meiosis]
Many bacteria reproduce through binary_fission, which is compared to mitosis
and meiosis in this image.
Further information: Bacterial_growth
Unlike in multicellular organisms, increases in cell size (cell_growth) and
reproduction by cell_division are tightly linked in unicellular organisms.
Bacteria grow to a fixed size and then reproduce through binary_fission, a form
of asexual_reproduction.[100] Under optimal conditions, bacteria can grow and
divide extremely rapidly, and bacterial populations can double as quickly as
every 9.8¬†minutes.[101] In cell division, two identical clone daughter cells
are produced. Some bacteria, while still reproducing asexually, form more
complex reproductive structures that help disperse the newly formed daughter
cells. Examples include fruiting body formation by Myxobacteria and aerial
hyphae formation by Streptomyces, or budding. Budding involves a cell forming a
protrusion that breaks away and produces a daughter cell.
[E._coli_colony]
A colony of Escherichia_coli[102]
In the laboratory, bacteria are usually grown using solid or liquid media.
Solid growth_media, such as agar_plates, are used to isolate pure cultures of a
bacterial strain. However, liquid growth media are used when measurement of
growth or large volumes of cells are required. Growth in stirred liquid media
occurs as an even cell suspension, making the cultures easy to divide and
transfer, although isolating single bacteria from liquid media is difficult.
The use of selective media (media with specific nutrients added or deficient,
or with antibiotics added) can help identify specific organisms.[103]
Most laboratory techniques for growing bacteria use high levels of nutrients to
produce large amounts of cells cheaply and quickly. However, in natural
environments, nutrients are limited, meaning that bacteria cannot continue to
reproduce indefinitely. This nutrient limitation has led the evolution of
different growth strategies (see r/K_selection_theory). Some organisms can grow
extremely rapidly when nutrients become available, such as the formation of
algal (and cyanobacterial) blooms that often occur in lakes during the summer.
[104] Other organisms have adaptations to harsh environments, such as the
production of multiple antibiotics by Streptomyces that inhibit the growth of
competing microorganisms.[105] In nature, many organisms live in communities
(e.g., biofilms) that may allow for increased supply of nutrients and
protection from environmental stresses.[48] These relationships can be
essential for growth of a particular organism or group of organisms
(syntrophy).[106]
Bacterial_growth follows four phases. When a population of bacteria first enter
a high-nutrient environment that allows growth, the cells need to adapt to
their new environment. The first phase of growth is the lag_phase, a period of
slow growth when the cells are adapting to the high-nutrient environment and
preparing for fast growth. The lag phase has high biosynthesis rates, as
proteins necessary for rapid growth are produced.[107][108] The second phase of
growth is the logarithmic_phase, also known as the exponential phase. The log
phase is marked by rapid exponential_growth. The rate at which cells grow
during this phase is known as the growth rate (k), and the time it takes the
cells to double is known as the generation time (g). During log phase,
nutrients are metabolised at maximum speed until one of the nutrients is
depleted and starts limiting growth. The third phase of growth is the
stationary_phase and is caused by depleted nutrients. The cells reduce their
metabolic activity and consume non-essential cellular proteins. The stationary
phase is a transition from rapid growth to a stress response state and there is
increased expression_of_genes involved in DNA_repair, antioxidant_metabolism
and nutrient_transport.[109] The final phase is the death_phase where the
bacteria run out of nutrients and die.[110]
***** Genetics *****
Main article: Bacterial_genetics
Most bacteria have a single circular chromosome that can range in size from
only 160,000 base_pairs in the endosymbiotic bacteria Carsonella_ruddii,[111]
to 12,200,000 base pairs (12.2 Mbp) in the soil-dwelling bacteria Sorangium
cellulosum.[112] There are many exceptions to this, for example some
Streptomyces and Borrelia species contain a single linear chromosome,[113][114]
while some Vibrio species contain more than one chromosome.[115] Bacteria can
also contain plasmids, small extra-chromosomal DNAs that may contain genes for
various useful functions such as antibiotic_resistance, metabolic capabilities,
or various virulence_factors.[116]
Bacteria genomes usually encode a few hundred to a few thousand genes. The
genes in bacterial genomes are usually a single continuous stretch of DNA and
although several different types of introns do exist in bacteria, these are
much rarer than in eukaryotes.[117]
Bacteria, as asexual organisms, inherit an identical copy of the parent's
genomes and are clonal. However, all bacteria can evolve by selection on
changes to their genetic material DNA caused by genetic_recombination or
mutations. Mutations come from errors made during the replication of DNA or
from exposure to mutagens. Mutation rates vary widely among different species
of bacteria and even among different clones of a single species of bacteria.
[118] Genetic changes in bacterial genomes come from either random mutation
during replication or "stress-directed mutation", where genes involved in a
particular growth-limiting process have an increased mutation rate.[119]
Some bacteria also transfer genetic material between cells. This can occur in
three main ways. First, bacteria can take up exogenous DNA from their
environment, in a process called transformation.[120] Many bacteria can
naturally take up DNA from the environment, while others must be chemically
altered in order to induce them to take up DNA.[121] The development of
competence in nature is usually associated with stressful environmental
conditions, and seems to be an adaptation for facilitating repair of DNA damage
in recipient cells.[122] The second way bacteria transfer genetic material is
by transduction, when the integration of a bacteriophage introduces foreign DNA
into the chromosome. Many types of bacteriophage exist, some simply infect and
lyse their host bacteria, while others insert into the bacterial chromosome.
[123] Bacteria resist phage infection through restriction_modification_systems
that degrade foreign DNA,[124] and a system that uses CRISPR sequences to
retain fragments of the genomes of phage that the bacteria have come into
contact with in the past, which allows them to block virus replication through
a form of RNA_interference.[125][126] The third method of gene transfer is
conjugation, whereby DNA is transferred through direct cell contact. In
ordinary circumstances, transduction, conjugation, and transformation involve
transfer of DNA between individual bacteria of the same species, but
occasionally transfer may occur between individuals of different bacterial
species and this may have significant consequences, such as the transfer of
antibiotic resistance.[127][128] In such cases, gene acquisition from other
bacteria or the environment is called horizontal_gene_transfer and may be
common under natural conditions.[129]
***** Behaviour *****
**** Movement ****
Further information: Chemotaxis, Flagellum, and Pilus
Transmission electron micrograph of Desulfovibrio_vulgaris showing a single
flagellum at one end of the cell. Scale bar is 0.5 micrometers long.
Many bacteria are motile and can move using a variety of mechanisms. The best
studied of these are flagella, long filaments that are turned by a motor at the
base to generate propeller-like movement.[130] The bacterial flagellum is made
of about 20 proteins, with approximately another 30 proteins required for its
regulation and assembly.[130] The flagellum is a rotating structure driven by a
reversible motor at the base that uses the electrochemical_gradient across the
membrane for power.[131]
The different arrangements of bacterial flagella: A-Monotrichous; B-
Lophotrichous; C-Amphitrichous; D-Peritrichous
Bacteria can use flagella in different ways to generate different kinds of
movement. Many bacteria (such as E._coli) have two distinct modes of movement:
forward movement (swimming) and tumbling. The tumbling allows them to reorient
and makes their movement a three-dimensional random_walk.[132] Bacterial
species differ in the number and arrangement of flagella on their surface; some
have a single flagellum (monotrichous), a flagellum at each end
(amphitrichous), clusters of flagella at the poles of the cell (lophotrichous),
while others have flagella distributed over the entire surface of the cell
(peritrichous). The flagella of a unique group of bacteria, the spirochaetes,
are found between two membranes in the periplasmic space. They have a
distinctive helical body that twists about as it moves.[130]
Two other types of bacterial motion are called twitching_motility that relies
on a structure called the type_IV_pilus,[133] and gliding_motility, that uses
other mechanisms. In twitching motility, the rod-like pilus extends out from
the cell, binds some substrate, and then retracts, pulling the cell forward.
[134]
Motile bacteria are attracted or repelled by certain stimuli in behaviours
called taxes: these include chemotaxis, phototaxis, energy_taxis, and
magnetotaxis.[135][136][137] In one peculiar group, the myxobacteria,
individual bacteria move together to form waves of cells that then
differentiate to form fruiting bodies containing spores.[45] The myxobacteria
move only when on solid surfaces, unlike E. coli, which is motile in liquid or
solid media.
Several Listeria and Shigella species move inside host cells by usurping the
cytoskeleton, which is normally used to move organelles inside the cell. By
promoting actin polymerisation at one pole of their cells, they can form a kind
of tail that pushes them through the host cell's cytoplasm.[138]
**** Communication ****
See also: Prokaryote_√Ç¬ß¬†Sociality
A few bacteria have chemical systems that generate light. This bioluminescence
often occurs in bacteria that live in association with fish, and the light
probably serves to attract fish or other large animals.[139]
Bacteria often function as multicellular aggregates known as biofilms,
exchanging a variety of molecular signals for inter-cell_communication, and
engaging in coordinated multicellular behaviour.[140][141]
The communal benefits of multicellular cooperation include a cellular division
of labour, accessing resources that cannot effectively be used by single cells,
collectively defending against antagonists, and optimising population survival
by differentiating into distinct cell types.[140] For example, bacteria in
biofilms can have more than 500 times increased resistance to antibacterial
agents than individual "planktonic" bacteria of the same species.[141]
One type of inter-cellular communication by a molecular signal is called quorum
sensing, which serves the purpose of determining whether there is a local
population density that is sufficiently high that it is productive to invest in
processes that are only successful if large numbers of similar organisms behave
similarly, as in excreting digestive enzymes or emitting light.
Quorum sensing allows bacteria to coordinate gene_expression, and enables them
to produce, release and detect autoinducers or pheromones which accumulate with
the growth in cell population.[142]
***** Classification and identification *****
Main article: Bacterial_taxonomy
Further information: Scientific_classification, Systematics, Bacterial_phyla,
and Clinical_pathology
[blue_stain_of_Streptococcus_mutans]
Streptococcus mutans visualised with a Gram stain
Phylogenetic_tree showing the diversity of bacteria, compared to other
organisms.[143] Eukaryotes are coloured red, archaea green and bacteria blue.
Classification seeks to describe the diversity of bacterial species by naming
and grouping organisms based on similarities. Bacteria can be classified on the
basis of cell structure, cellular_metabolism or on differences in cell
components, such as DNA, fatty_acids, pigments, antigens and quinones.[103]
While these schemes allowed the identification and classification of bacterial
strains, it was unclear whether these differences represented variation between
distinct species or between strains of the same species. This uncertainty was
due to the lack of distinctive structures in most bacteria, as well as lateral
gene_transfer between unrelated species.[144] Due to lateral gene transfer,
some closely related bacteria can have very different morphologies and
metabolisms. To overcome this uncertainty, modern bacterial classification
emphasises molecular_systematics, using genetic techniques such as guanine
cytosine ratio determination, genome-genome hybridisation, as well as
sequencing genes that have not undergone extensive lateral gene transfer, such
as the rRNA_gene.[145] Classification of bacteria is determined by publication
in the International Journal of Systematic Bacteriology,[146] and Bergey's
Manual of Systematic Bacteriology.[147] The International_Committee_on
Systematic_Bacteriology (ICSB) maintains international rules for the naming of
bacteria and taxonomic categories and for the ranking of them in the
International_Code_of_Nomenclature_of_Bacteria.
The term "bacteria" was traditionally applied to all microscopic, single-cell
prokaryotes. However, molecular systematics showed prokaryotic life to consist
of two separate domains, originally called Eubacteria and Archaebacteria, but
now called Bacteria and Archaea that evolved independently from an ancient
common ancestor.[1] The archaea and eukaryotes are more closely related to each
other than either is to the bacteria. These two domains, along with Eukarya,
are the basis of the three-domain_system, which is currently the most widely
used classification system in microbiology.[148] However, due to the relatively
recent introduction of molecular systematics and a rapid increase in the number
of genome sequences that are available, bacterial classification remains a
changing and expanding field.[149][150] For example, a few biologists argue
that the Archaea and Eukaryotes evolved from gram-positive bacteria.[151]
The identification of bacteria in the laboratory is particularly relevant in
medicine, where the correct treatment is determined by the bacterial species
causing an infection. Consequently, the need to identify human pathogens was a
major impetus for the development of techniques to identify bacteria.
The Gram_stain, developed in 1884 by Hans_Christian_Gram, characterises
bacteria based on the structural characteristics of their cell walls.[70] The
thick layers of peptidoglycan in the "gram-positive" cell wall stain purple,
while the thin "gram-negative" cell wall appears pink. By combining morphology
and Gram-staining, most bacteria can be classified as belonging to one of four
groups (gram-positive cocci, gram-positive bacilli, gram-negative cocci and
gram-negative bacilli). Some organisms are best identified by stains other than
the Gram stain, particularly mycobacteria or Nocardia, which show acid-fastness
on Ziehl√¢¬Ä¬ìNeelsen or similar stains.[152] Other organisms may need to be
identified by their growth in special media, or by other techniques, such as
serology.
Culture techniques are designed to promote the growth and identify particular
bacteria, while restricting the growth of the other bacteria in the sample.
Often these techniques are designed for specific specimens; for example, a
sputum sample will be treated to identify organisms that cause pneumonia, while
stool specimens are cultured on selective_media to identify organisms that
cause diarrhoea, while preventing growth of non-pathogenic bacteria. Specimens
that are normally sterile, such as blood, urine or spinal_fluid, are cultured
under conditions designed to grow all possible organisms.[103][153] Once a
pathogenic organism has been isolated, it can be further characterised by its
morphology, growth patterns (such as aerobic or anaerobic growth), patterns_of
hemolysis, and staining.
As with bacterial classification, identification of bacteria is increasingly
using molecular methods. Diagnostics using DNA-based tools, such as polymerase
chain_reaction, are increasingly popular due to their specificity and speed,
compared to culture-based methods.[154] These methods also allow the detection
and identification of "viable_but_nonculturable" cells that are metabolically
active but non-dividing.[155] However, even using these improved methods, the
total number of bacterial species is not known and cannot even be estimated
with any certainty. Following present classification, there are a little less
than 9,300 known species of prokaryotes, which includes bacteria and archaea;
[156] but attempts to estimate the true number of bacterial diversity have
ranged from 107 to 109 total species√¢¬Ä¬îand even these diverse estimates may be
off by many orders of magnitude.[157][158]
***** Interactions with other organisms *****
[chart_showing_bacterial_infections_upon_various_parts_of_human_body]
Overview of bacterial infections and main species involved.[159][160]
Further information: Microbes_in_human_culture
Despite their apparent simplicity, bacteria can form complex associations with
other organisms. These symbiotic associations can be divided into parasitism,
mutualism and commensalism. Due to their small size, commensal bacteria are
ubiquitous and grow on animals and plants exactly as they will grow on any
other surface. However, their growth can be increased by warmth and sweat, and
large populations of these organisms in humans are the cause of body_odour.
**** Predators ****
Some species of bacteria kill and then consume other microorganisms, these
species are called predatory bacteria.[161] These include organisms such as
Myxococcus_xanthus, which forms swarms of cells that kill and digest any
bacteria they encounter.[162] Other bacterial predators either attach to their
prey in order to digest them and absorb nutrients, such as Vampirovibrio
chlorellavorus,[163] or invade another cell and multiply inside the cytosol,
such as Daptobacter.[164] These predatory bacteria are thought to have evolved
from saprophages that consumed dead microorganisms, through adaptations that
allowed them to entrap and kill other organisms.[165]
**** Mutualists ****
Certain bacteria form close spatial associations that are essential for their
survival. One such mutualistic association, called interspecies hydrogen
transfer, occurs between clusters of anaerobic_bacteria that consume organic
acids, such as butyric_acid or propionic_acid, and produce hydrogen, and
methanogenic Archaea that consume hydrogen.[166] The bacteria in this
association are unable to consume the organic acids as this reaction produces
hydrogen that accumulates in their surroundings. Only the intimate association
with the hydrogen-consuming Archaea keeps the hydrogen concentration low enough
to allow the bacteria to grow.
In soil, microorganisms that reside in the rhizosphere (a zone that includes
the root surface and the soil that adheres to the root after gentle shaking)
carry out nitrogen_fixation, converting nitrogen gas to nitrogenous compounds.
[167] This serves to provide an easily absorbable form of nitrogen for many
plants, which cannot fix nitrogen themselves. Many other bacteria are found as
symbionts in_humans and other organisms. For example, the presence of over
1,000 bacterial species in the normal human gut_flora of the intestines can
contribute to gut immunity, synthesise vitamins, such as folic_acid, vitamin_K
and biotin, convert sugars to lactic_acid (see Lactobacillus), as well as
fermenting complex undigestible carbohydrates.[168][169][170] The presence of
this gut flora also inhibits the growth of potentially pathogenic bacteria
(usually through competitive_exclusion) and these beneficial bacteria are
consequently sold as probiotic dietary_supplements.[171]
**** Pathogens ****
Main article: Pathogenic_bacteria
[Color-enhanced_scanning_electron_micrograph_of_red_Salmonella_typhimurium_in
yellow_human_cells]
Colour-enhanced scanning electron micrograph showing Salmonella_typhimurium
(red) invading cultured human cells
If bacteria form a parasitic association with other organisms, they are classed
as pathogens. Pathogenic bacteria are a major cause of human death and disease
and cause infections such as tetanus, typhoid_fever, diphtheria, syphilis,
cholera, foodborne_illness, leprosy and tuberculosis. A pathogenic cause for a
known medical disease may only be discovered many years after, as was the case
with Helicobacter_pylori and peptic_ulcer_disease. Bacterial diseases are also
important in agriculture, with bacteria causing leaf_spot, fire_blight and
wilts in plants, as well as Johne's_disease, mastitis, salmonella and anthrax
in farm animals.
Each species of pathogen has a characteristic spectrum of interactions with its
human hosts. Some organisms, such as Staphylococcus or Streptococcus, can cause
skin infections, pneumonia, meningitis and even overwhelming sepsis, a systemic
inflammatory_response producing shock, massive vasodilation and death.[172] Yet
these organisms are also part of the normal human flora and usually exist on
the skin or in the nose without causing any disease at all. Other organisms
invariably cause disease in humans, such as the Rickettsia, which are obligate
intracellular_parasites able to grow and reproduce only within the cells of
other organisms. One species of Rickettsia causes typhus, while another causes
Rocky_Mountain_spotted_fever. Chlamydia, another phylum of obligate
intracellular parasites, contains species that can cause pneumonia, or urinary
tract_infection and may be involved in coronary_heart_disease.[173] Finally,
some species, such as Pseudomonas_aeruginosa, Burkholderia_cenocepacia, and
Mycobacterium_avium, are opportunistic_pathogens and cause disease mainly in
people suffering from immunosuppression or cystic_fibrosis.[174][175]
Bacterial infections may be treated with antibiotics, which are classified as
bacteriocidal if they kill bacteria, or bacteriostatic if they just prevent
bacterial growth. There are many types of antibiotics and each class inhibits a
process that is different in the pathogen from that found in the host. An
example of how antibiotics produce selective toxicity are chloramphenicol and
puromycin, which inhibit the bacterial ribosome, but not the structurally
different eukaryotic ribosome.[176] Antibiotics are used both in treating human
disease and in intensive_farming to promote animal growth, where they may be
contributing to the rapid development of antibiotic_resistance in bacterial
populations.[177] Infections can be prevented by antiseptic measures such as
sterilising the skin prior to piercing it with the needle of a syringe, and by
proper care of indwelling catheters. Surgical and dental instruments are also
sterilised to prevent contamination by bacteria. Disinfectants such as bleach
are used to kill bacteria or other pathogens on surfaces to prevent
contamination and further reduce the risk of infection.
***** Significance in technology and industry *****
Further information: Economic_importance_of_bacteria
Bacteria, often lactic_acid_bacteria, such as Lactobacillus and Lactococcus, in
combination with yeasts and moulds, have been used for thousands of years in
the preparation of fermented foods, such as cheese, pickles, soy_sauce,
sauerkraut, vinegar, wine and yogurt.[178][179]
The ability of bacteria to degrade a variety of organic compounds is remarkable
and has been used in waste processing and bioremediation. Bacteria capable of
digesting the hydrocarbons in petroleum are often used to clean up oil_spills.
[180] Fertiliser was added to some of the beaches in Prince_William_Sound in an
attempt to promote the growth of these naturally occurring bacteria after the
1989 Exxon_Valdez_oil_spill. These efforts were effective on beaches that were
not too thickly covered in oil. Bacteria are also used for the bioremediation
of industrial toxic_wastes.[181] In the chemical_industry, bacteria are most
important in the production of enantiomerically pure chemicals for use as
pharmaceuticals or agrichemicals.[182]
Bacteria can also be used in the place of pesticides in the biological_pest
control. This commonly involves Bacillus_thuringiensis (also called BT), a
gram-positive, soil dwelling bacterium. Subspecies of this bacteria are used as
a Lepidopteran-specific insecticides under trade names such as Dipel and
Thuricide.[183] Because of their specificity, these pesticides are regarded as
environmentally_friendly, with little or no effect on humans, wildlife,
pollinators and most other beneficial_insects.[184][185]
Because of their ability to quickly grow and the relative ease with which they
can be manipulated, bacteria are the workhorses for the fields of molecular
biology, genetics and biochemistry. By making mutations in bacterial DNA and
examining the resulting phenotypes, scientists can determine the function of
genes, enzymes and metabolic_pathways in bacteria, then apply this knowledge to
more complex organisms.[186] This aim of understanding the biochemistry of a
cell reaches its most complex expression in the synthesis of huge amounts of
enzyme_kinetic and gene_expression data into mathematical_models of entire
organisms. This is achievable in some well-studied bacteria, with models of
Escherichia coli metabolism now being produced and tested.[187][188] This
understanding of bacterial metabolism and genetics allows the use of
biotechnology to bioengineer bacteria for the production of therapeutic
proteins, such as insulin, growth_factors, or antibodies.[189][190]
Because of their importance for research in general, samples of bacterial
strains are isolated and preserved in Biological_Resource_Centers. This ensures
the availability of the strain to scientists worldwide.
***** History of bacteriology *****
For the history of microbiology, see Microbiology. For the history of bacterial
classification, see Bacterial_taxonomy. For the natural history of Bacteria,
see Last_universal_common_ancestor.
[painting_of_Antonie_van_Leeuwenhoek,_in_robe_and_frilled_shirt,_with_ink_pen
and_paper]
Antonie_van_Leeuwenhoek, the first microbiologist and the first person to
observe bacteria using a microscope.
Bacteria were first observed by the Dutch microscopist Antonie_van_Leeuwenhoek
in 1676, using a single-lens microscope of his own design.[191] He then
published his observations in a series of letters to the Royal_Society_of
London.[192][193][194] Bacteria were Leeuwenhoek's most remarkable microscopic
discovery. They were just at the limit of what his simple lenses could make out
and, in one of the most striking hiatuses in the history of science, no one
else would see them again for over a century.[195] His observations had also
included protozoans which he called animalcules, and his findings were looked
at again in the light of the more recent findings of cell_theory.
Christian_Gottfried_Ehrenberg introduced the word "bacterium" in 1828.[196] In
fact, his Bacterium was a genus that contained non-spore-forming rod-shaped
bacteria,[197] as opposed to Bacillus, a genus of spore-forming rod-shaped
bacteria defined by Ehrenberg in 1835.[198]
Louis_Pasteur demonstrated in 1859 that the growth of microorganisms causes the
fermentation process, and that this growth is not due to spontaneous
generation. (Yeasts and moulds, commonly associated with fermentation, are not
bacteria, but rather fungi.) Along with his contemporary Robert_Koch, Pasteur
was an early advocate of the germ_theory_of_disease.[199]
Robert Koch, a pioneer in medical microbiology, worked on cholera, anthrax and
tuberculosis. In his research into tuberculosis Koch finally proved the germ
theory, for which he received a Nobel_Prize in 1905.[200] In Koch's_postulates,
he set out criteria to test if an organism is the cause of a disease, and these
postulates are still used today.[201]
Ferdinand_Cohn is said to be a founder of bacteriology, studying bacteria from
1870. Cohn was the first to classify bacteria based on their morphology.[202]
[203]
Though it was known in the nineteenth century that bacteria are the cause of
many diseases, no effective antibacterial treatments were available.[204] In
1910, Paul_Ehrlich developed the first antibiotic, by changing dyes that
selectively stained Treponema_pallidum√¢¬Ä¬îthe spirochaete that causes
syphilis√¢¬Ä¬îinto compounds that selectively killed the pathogen.[205] Ehrlich
had been awarded a 1908 Nobel Prize for his work on immunology, and pioneered
the use of stains to detect and identify bacteria, with his work being the
basis of the Gram_stain and the Ziehl√¢¬Ä¬ìNeelsen_stain.[206]
A major step forward in the study of bacteria came in 1977 when Carl_Woese
recognised that archaea have a separate line of evolutionary descent from
bacteria.[3] This new phylogenetic taxonomy depended on the sequencing of 16S
ribosomal_RNA, and divided prokaryotes into two evolutionary domains, as part
of the three-domain_system.[1]
***** Computer-made form *****
In April 2019, scientists at ETH_Zurich reported the creation of the world's
first bacterial_genome, named Caulobacter_ethensis-2.0, made entirely by a
computer, although a related viable form of C. ethensis-2.0 does not yet exist.
[207][208]
***** See also *****
    * Bacteriotherapy
    * Genetically_modified_bacteria
    * List_of_bacterial_orders
    * Panspermia
    * Polysaccharide_encapsulated_bacteria
    * Psychrotrophic_bacteria
***** References *****
   1. ^ a b c dWoese CR, Kandler O, Wheelis ML (June 1990). "Towards_a_natural
      system_of_organisms:_proposal_for_the_domains_Archaea,_Bacteria,_and
      Eucarya". Proceedings of the National Academy of Sciences of the United
      States of America. 87 (12): 4576√¢¬Ä¬ì79. Bibcode:1990PNAS...87.4576W. doi:
      10.1073/pnas.87.12.4576. PMC¬†54159. PMID¬†2112744.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Pavan ME, et al. (May 2018). "Proposal for a new classification of a
      deep branching bacterial phylogenetic lineage: transfer of
      Coprothermobacter proteolyticus and Coprothermobacter platensis to
      Coprothermobacteraceae fam. nov., within Coprothermobacterales ord. nov.,
      Coprothermobacteria classis nov. and Coprothermobacterota phyl. nov. and
      emended description of the family Thermodesulfobiaceae". Int. J. Syst.
      Evol. Microbiol. 68 (5): 1627√¢¬Ä¬ì32. doi:10.1099/ijsem.0.002720.
      PMID¬†29595416.
   4. ^ a bWoese CR, Fox GE (November 1977). "Phylogenetic_structure_of_the
      prokaryotic_domain:_the_primary_kingdoms". Proceedings of the National
      Academy of Sciences of the United States of America. 74 (11): 5088√¢¬Ä¬ì90.
      Bibcode:1977PNAS...74.5088W. doi:10.1073/pnas.74.11.5088. PMC¬†432104.
      PMID¬†270744.
   5. ^Fredrickson JK, Zachara JM, Balkwill DL, Kennedy D, Li SM,
      Kostandarithes HM, Daly MJ, Romine MF, Brockman FJ (July 2004).
      "Geomicrobiology_of_high-level_nuclear_waste-contaminated_vadose
      sediments_at_the_Hanford_site,_Washington_state". Applied and
      Environmental Microbiology. 70 (7): 4230√¢¬Ä¬ì41. doi:10.1128/AEM.70.7.4230-
      4241.2004. PMC¬†444790. PMID¬†15240306.
   6. ^Dudek NK, Sun CL, Burstein D (2017). "Novel Microbial Diversity and
      Functional Potential in the Marine Mammal Oral Microbiome". Current
      Biology. 27 (24): 3752√¢¬Ä¬ì3762. doi:10.1016/j.cub.2017.10.040.
   7. ^Fang H, Kang J, Zhang D (January 2017). "12:_a_review_and_future
      perspectives". Microbial Cell Factories. 16 (1): 15. doi:10.1186/s12934-
      017-0631-y. PMC¬†5282855. PMID¬†28137297.
   8. ^Moore SJ, Warren MJ (June 2012). "The anaerobic biosynthesis of vitamin
      B12". Biochemical Society Transactions. 40 (3): 581√¢¬Ä¬ì6. doi:10.1042/
      BST20120066. PMID¬†22616870.
   9. ^Graham RM, Deery E, Warren MJ (2009). "18: Vitamin B12: Biosynthesis of
      the Corrin Ring". In Warren MJ, Smith_AG (eds.). Tetrapyrroles Birth,
      Life and Death. New York, NY: Springer-Verlag. p.¬†286. doi:10.1007/978-0-
      387-78518-9_18. ISBN¬†978-0-387-78518-9.
  10. ^Miller A, Korem M, Almog R, Galboiz Y (June 2005). "Vitamin B12,
      demyelination, remyelination and repair in multiple sclerosis". Journal
      of the Neurological Sciences. 233 (1√¢¬Ä¬ì2): 93√¢¬Ä¬ì7. doi:10.1016/
      j.jns.2005.03.009. PMID¬†15896807.
  11. ^Whitman WB, Coleman DC, Wiebe WJ (June 1998). "Prokaryotes:_the_unseen
      majority". Proceedings of the National Academy of Sciences of the United
      States of America. 95 (12): 6578√¢¬Ä¬ì83. Bibcode:1998PNAS...95.6578W. doi:
      10.1073/pnas.95.12.6578. PMC¬†33863. PMID¬†9618454.
  12. ^ C.Michael Hogan. 2010. Bacteria._Encyclopedia_of_Earth._eds._Sidney
      Draggan_and_C.J.Cleveland,_National_Council_for_Science_and_the
      Environment,_Washington,_DC Archived 11 May 2011 at the Wayback_Machine
  13. ^Forbes SL (2008). "Decomposition Chemistry in a Burial Environment". In
      Tibbett M, Carter DO (eds.). Soil Analysis in Forensic Taphonomy. CRC
      Press. pp.¬†203√¢¬Ä¬ì223. ISBN¬†978-1-4200-6991-4.
  14. ^ a b cChoi CQ (17 March 2013). "Microbes_Thrive_in_Deepest_Spot_on
      Earth". LiveScience. Archived from the original on 2 April 2013.
      Retrieved 17 March 2013.
  15. ^Glud R, Wenzh√É¬∂fer F, Middelboe M, Oguri K, Turnewitsch R, Canfield DE,
      Kitazato H (2013). "High rates of microbial carbon turnover in sediments
      in the deepest oceanic trench on Earth". Nature_Geoscience. 6 (4):
      284√¢¬Ä¬ì88. Bibcode:2013NatGe...6..284G. doi:10.1038/ngeo1773.
  16. ^Oskin B (14 March 2013). "Intraterrestrials:_Life_Thrives_in_Ocean
      Floor". LiveScience. Archived from the original on 2 April 2013.
      Retrieved 17 March 2013.
  17. ^Sender R, Fuchs S, Milo R (2016). "Revised estimates for the number of
      human and bacteria cells in the body". bioRxiv¬†036103.
  18. ^Sears CL (October 2005). "A dynamic partnership: celebrating our gut
      flora". Anaerobe. 11 (5): 247√¢¬Ä¬ì51. doi:10.1016/j.anaerobe.2005.05.001.
      PMID¬†16701579.
  19. ^"2002_WHO_mortality_data". Archived from the original on 23 October
      2013. Retrieved 20 January 2007.
  20. ^"Metal-Mining_Bacteria_Are_Green_Chemists". Science Daily. 2 September
      2010. Archived from the original on 31 August 2017.
  21. ^Ishige T, Honda K, Shimizu S (April 2005). "Whole organism
      biocatalysis". Current Opinion in Chemical Biology. 9 (2): 174√¢¬Ä¬ì80. doi:
      10.1016/j.cbpa.2005.02.001. PMID¬†15811802.
  22. ^ √é¬≤√é¬±√é¬∫√è¬Ñ√é¬Æ√è¬Å√é¬π√é¬ø√é¬Ω. Liddell,_Henry_George; Scott,_Robert; A
      Greek√¢¬Ä¬ìEnglish_Lexicon at the Perseus_Project.
  23. ^ √é¬≤√é¬±√é¬∫√è¬Ñ√é¬∑√è¬Å√é¬Ø√é¬±¬†in Liddell and Scott.
  24. ^ bacterium Archived 27 January 2011 at the Wayback_Machine, on Oxford
      Dictionaries.
  25. ^Harper, Douglas. "bacteria". Online_Etymology_Dictionary.
  26. ^Schopf JW (July 1994). "Disparate_rates,_differing_fates:_tempo_and_mode
      of_evolution_changed_from_the_Precambrian_to_the_Phanerozoic".
      Proceedings of the National Academy of Sciences of the United States of
      America. 91 (15): 6735√¢¬Ä¬ì42. Bibcode:1994PNAS...91.6735S. doi:10.1073/
      pnas.91.15.6735. PMC¬†44277. PMID¬†8041691.
  27. ^DeLong EF, Pace NR (August 2001). "Environmental diversity of bacteria
      and archaea". Systematic Biology. 50 (4): 470√¢¬Ä¬ì78.
      CiteSeerX¬†10.1.1.321.8828. doi:10.1080/106351501750435040. PMID¬†12116647.
  28. ^Brown JR, Doolittle WF (December 1997). "Archaea_and_the_prokaryote-to-
      eukaryote_transition". Microbiology and Molecular Biology Reviews. 61
      (4): 456√¢¬Ä¬ì502. PMC¬†232621. PMID¬†9409149.
  29. ^Di Giulio M (December 2003). "The universal ancestor and the ancestor of
      bacteria were hyperthermophiles". Journal of Molecular Evolution. 57 (6):
      721√¢¬Ä¬ì30. Bibcode:2003JMolE..57..721D. doi:10.1007/s00239-003-2522-6.
      PMID¬†14745541.
  30. ^Battistuzzi FU, Feijao A, Hedges SB (November 2004). "A_genomic
      timescale_of_prokaryote_evolution:_insights_into_the_origin_of
      methanogenesis,_phototrophy,_and_the_colonization_of_land". BMC
      Evolutionary Biology. 4: 44. doi:10.1186/1471-2148-4-44. PMC¬†533871.
      PMID¬†15535883.
  31. ^Poole AM, Penny D (January 2007). "Evaluating hypotheses for the origin
      of eukaryotes". BioEssays. 29 (1): 74√¢¬Ä¬ì84. doi:10.1002/bies.20516.
      PMID¬†17187354.
  32. ^Dyall SD, Brown MT, Johnson PJ (April 2004). "Ancient invasions: from
      endosymbionts to organelles". Science. 304 (5668): 253√¢¬Ä¬ì7. Bibcode:
      2004Sci...304..253D. doi:10.1126/science.1094884. PMID¬†15073369.
  33. ^Lang BF, Gray MW, Burger G (1999). "Mitochondrial genome evolution and
      the origin of eukaryotes". Annual Review of Genetics. 33: 351√¢¬Ä¬ì97. doi:
      10.1146/annurev.genet.33.1.351. PMID¬†10690412.
  34. ^McFadden GI (December 1999). "Endosymbiosis and evolution of the plant
      cell". Current Opinion in Plant Biology. 2 (6): 513√¢¬Ä¬ì19. doi:10.1016/
      S1369-5266(99)00025-4. PMID¬†10607659.
  35. ^Homann, Martin; et al. (23 July 2018). "Microbial life and
      biogeochemical cycling on land 3,220 million years ago". Nature
      Geoscience. 11 (9): 665√¢¬Ä¬ì671. doi:10.1038/s41561-018-0190-9.
  36. ^Schulz HN, Jorgensen BB (2001). "Big bacteria". Annual Review of
      Microbiology. 55: 105√¢¬Ä¬ì37. doi:10.1146/annurev.micro.55.1.105.
      PMID¬†11544351.
  37. ^Williams C (2011). "Who are you calling simple?". New Scientist. 211
      (2821): 38√¢¬Ä¬ì41. doi:10.1016/S0262-4079(11)61709-0.
  38. ^Robertson J, Gomersall M, Gill P (November 1975). "Mycoplasma_hominis:
      growth,_reproduction,_and_isolation_of_small_viable_cells". Journal of
      Bacteriology. 124 (2): 1007√¢¬Ä¬ì18. PMC¬†235991. PMID¬†1102522.
  39. ^Velimirov B (2001). "Nanobacteria, Ultramicrobacteria and Starvation
      Forms: A Search for the Smallest Metabolizing Bacterium". Microbes and
      Environments. 16 (2): 67√¢¬Ä¬ì77. doi:10.1264/jsme2.2001.67.
  40. ^ Dusenbery, David B (2009). Living at Micro Scale, pp. 20√¢¬Ä¬ì25. Harvard
      University Press, Cambridge, Massachusetts
  41. ISBN¬†978-0-674-03116-6.
  42. ^Yang DC, Blair KM, Salama NR (March 2016). "Staying_in_Shape:_the_Impact
      of_Cell_Shape_on_Bacterial_Survival_in_Diverse_Environments".
      Microbiology and Molecular Biology Reviews. 80 (1): 187√¢¬Ä¬ì203. doi:
      10.1128/MMBR.00031-15. PMC¬†4771367. PMID¬†26864431.
  43. ^Cabeen MT, Jacobs-Wagner C (August 2005). "Bacterial cell shape". Nature
      Reviews. Microbiology. 3 (8): 601√¢¬Ä¬ì10. doi:10.1038/nrmicro1205.
      PMID¬†16012516.
  44. ^Young KD (September 2006). "The_selective_value_of_bacterial_shape".
      Microbiology and Molecular Biology Reviews. 70 (3): 660√¢¬Ä¬ì703. doi:
      10.1128/MMBR.00001-06. PMC¬†1594593. PMID¬†16959965.
  45. ^Claessen D, Rozen DE, Kuipers OP, S√É¬∏gaard-Andersen L, van Wezel GP
      (February 2014). "Bacterial_solutions_to_multicellularity:_a_tale_of
      biofilms,_filaments_and_fruiting_bodies". Nature Reviews. Microbiology.
      12 (2): 115√¢¬Ä¬ì24. doi:10.1038/nrmicro3178. PMID¬†24384602.
  46. ^Shimkets LJ (1999). "Intercellular signaling during fruiting-body
      development of Myxococcus xanthus". Annual Review of Microbiology. 53:
      525√¢¬Ä¬ì49. doi:10.1146/annurev.micro.53.1.525. PMID¬†10547700.
  47. ^ a bKaiser D (2004). "Signaling in myxobacteria". Annual Review of
      Microbiology. 58: 75√¢¬Ä¬ì98. doi:10.1146/annurev.micro.58.030603.123620.
      PMID¬†15487930.
  48. ^Donlan RM (September 2002). "Biofilms:_microbial_life_on_surfaces".
      Emerging Infectious Diseases. 8 (9): 881√¢¬Ä¬ì90. doi:10.3201/
      eid0809.020063. PMC¬†2732559. PMID¬†12194761.
  49. ^Branda SS, Vik S, Friedman L, Kolter R (January 2005). "Biofilms: the
      matrix revisited". Trends in Microbiology. 13 (1): 20√¢¬Ä¬ì26. doi:10.1016/
      j.tim.2004.11.006. PMID¬†15639628.
  50. ^ a bDavey ME, O'toole GA (December 2000). "Microbial_biofilms:_from
      ecology_to_molecular_genetics". Microbiology and Molecular Biology
      Reviews. 64 (4): 847√¢¬Ä¬ì67. doi:10.1128/MMBR.64.4.847-867.2000. PMC¬†99016.
      PMID¬†11104821.
  51. ^Donlan RM, Costerton JW (April 2002). "Biofilms:_survival_mechanisms_of
      clinically_relevant_microorganisms". Clinical Microbiology Reviews. 15
      (2): 167√¢¬Ä¬ì93. doi:10.1128/CMR.15.2.167-193.2002. PMC¬†118068.
      PMID¬†11932229.
  52. ^Slonczewski JL, Foster JW (2013). Microbiology¬†: an Evolving Science
      (Third ed.). New York: W W Norton. p.¬†82. ISBN¬†978-0393123678.
  53. ^Lodish H, Berk A, Kaiser CA, Krieger M, Bretscher A, Ploegh H, Amon A,
      Scott MP (2013). Molecular Cell Biology (7th ed.). WH Freeman. p.¬†13.
      ISBN¬†978-1429234139.
  54. ^Bobik TA (May 2006). "Polyhedral organelles compartmenting bacterial
      metabolic processes". Applied Microbiology and Biotechnology. 70 (5):
      517√¢¬Ä¬ì25. doi:10.1007/s00253-005-0295-0. PMID¬†16525780.
  55. ^Yeates TO, Kerfeld CA, Heinhorst S, Cannon GC, Shively JM (September
      2008). "Protein-based organelles in bacteria: carboxysomes and related
      microcompartments". Nature Reviews. Microbiology. 6 (9): 681√¢¬Ä¬ì91. doi:
      10.1038/nrmicro1913. PMID¬†18679172.
  56. ^Kerfeld CA, Sawaya MR, Tanaka S, Nguyen CV, Phillips M, Beeby M, Yeates
      TO (August 2005). "Protein structures forming the shell of primitive
      bacterial organelles". Science. 309 (5736): 936√¢¬Ä¬ì38. Bibcode:
      2005Sci...309..936K. CiteSeerX¬†10.1.1.1026.896. doi:10.1126/
      science.1113397. PMID¬†16081736.
  57. ^Gitai Z (March 2005). "The new bacterial cell biology: moving parts and
      subcellular architecture". Cell. 120 (5): 577√¢¬Ä¬ì86. doi:10.1016/
      j.cell.2005.02.026. PMID¬†15766522.
  58. ^Shih YL, Rothfield L (September 2006). "The_bacterial_cytoskeleton".
      Microbiology and Molecular Biology Reviews. 70 (3): 729√¢¬Ä¬ì54. doi:
      10.1128/MMBR.00017-06. PMC¬†1594594. PMID¬†16959967.
  59. ^Norris V, den Blaauwen T, Cabin-Flaman A, Doi RH, Harshey R, Janniere L,
      Jimenez-Sanchez A, Jin DJ, Levin PA, Mileykovskaya E, Minsky A, Saier M,
      Skarstad K (March 2007). "Functional_taxonomy_of_bacterial
      hyperstructures". Microbiology and Molecular Biology Reviews. 71 (1):
      230√¢¬Ä¬ì53. doi:10.1128/MMBR.00035-06. PMC¬†1847379. PMID¬†17347523.
  60. ^Harold FM (June 1972). "Conservation_and_transformation_of_energy_by
      bacterial_membranes". Bacteriological Reviews. 36 (2): 172√¢¬Ä¬ì230.
      PMC¬†408323. PMID¬†4261111.
  61. ^Bryant DA, Frigaard NU (November 2006). "Prokaryotic photosynthesis and
      phototrophy illuminated". Trends in Microbiology. 14 (11): 488√¢¬Ä¬ì96. doi:
      10.1016/j.tim.2006.09.001. PMID¬†16997562.
  62. ^Psenc√É¬≠k J, Ikonen TP, Laurinm√É¬§ki P, Merckel MC, Butcher SJ, Serimaa
      RE, Tuma R (August 2004). "Lamellar_organization_of_pigments_in
      chlorosomes,_the_light_harvesting_complexes_of_green_photosynthetic
      bacteria". Biophysical Journal. 87 (2): 1165√¢¬Ä¬ì72. Bibcode:
      2004BpJ....87.1165P. doi:10.1529/biophysj.104.040956. PMC¬†1304455.
      PMID¬†15298919.
  63. ^Thanbichler M, Wang SC, Shapiro L (October 2005). "The bacterial
      nucleoid: a highly organized and dynamic structure". Journal of Cellular
      Biochemistry. 96 (3): 506√¢¬Ä¬ì21. doi:10.1002/jcb.20519. PMID¬†15988757.
  64. ^Poehlsgaard J, Douthwaite S (November 2005). "The bacterial ribosome as
      a target for antibiotics". Nature Reviews. Microbiology. 3 (11):
      870√¢¬Ä¬ì81. doi:10.1038/nrmicro1265. PMID¬†16261170.
  65. ^Yeo M, Chater K (March 2005). "The_interplay_of_glycogen_metabolism_and
      differentiation_provides_an_insight_into_the_developmental_biology_of
      Streptomyces_coelicolor". Microbiology. 151 (Pt 3): 855√¢¬Ä¬ì61. doi:
      10.1099/mic.0.27428-0. PMID¬†15758231. Archived from the original on 29
      September 2007.
  66. ^Shiba T, Tsutsumi K, Ishige K, Noguchi T (March 2000). "Inorganic
      polyphosphate_and_polyphosphate_kinase:_their_novel_biological_functions
      and_applications". Biochemistry. Biokhimiia. 65 (3): 315√¢¬Ä¬ì23.
      PMID¬†10739474. Archived from the original on 25 September 2006.
  67. ^Brune DC (June 1995). "Isolation and characterization of sulfur globule
      proteins from Chromatium vinosum and Thiocapsa roseopersicina". Archives
      of Microbiology. 163 (6): 391√¢¬Ä¬ì99. doi:10.1007/BF00272127. PMID¬†7575095.
  68. ^Kadouri D, Jurkevitch E, Okon Y, Castro-Sowinski S (2005). "Ecological
      and agricultural significance of bacterial polyhydroxyalkanoates".
      Critical Reviews in Microbiology. 31 (2): 55√¢¬Ä¬ì67. doi:10.1080/
      10408410590899228. PMID¬†15986831.
  69. ^Walsby AE (March 1994). "Gas_vesicles". Microbiological Reviews. 58 (1):
      94√¢¬Ä¬ì144. PMC¬†372955. PMID¬†8177173.
  70. ^van Heijenoort J (March 2001). "Formation of the glycan chains in the
      synthesis of bacterial peptidoglycan". Glycobiology. 11 (3): 25R√¢¬Ä¬ì36R.
      doi:10.1093/glycob/11.3.25R. PMID¬†11320055.
  71. ^ a bKoch AL (October 2003). "Bacterial_wall_as_target_for_attack:_past,
      present,_and_future_research". Clinical Microbiology Reviews. 16 (4):
      673√¢¬Ä¬ì87. doi:10.1128/CMR.16.4.673-687.2003. PMC¬†207114. PMID¬†14557293.
  72. ^ a bGram,_HC (1884). "√É¬úber die isolierte F√É¬§rbung der Schizomyceten in
      Schnitt- und Trockenpr√É¬§paraten". Fortschr. Med. 2: 185√¢¬Ä¬ì89.
  73. ^Hugenholtz P (2002). "Exploring_prokaryotic_diversity_in_the_genomic
      era". Genome Biology. 3 (2): REVIEWS0003. doi:10.1186/gb-2002-3-2-
      reviews0003. PMC¬†139013. PMID¬†11864374.
  74. ^Walsh FM, Amyes SG (October 2004). "Microbiology and drug resistance
      mechanisms of fully resistant pathogens". Current Opinion in
      Microbiology. 7 (5): 439√¢¬Ä¬ì44. doi:10.1016/j.mib.2004.08.007.
      PMID¬†15451497.
  75. ^Alderwick LJ, Harrison J, Lloyd GS, Birch HL (March 2015). "The
      Mycobacterial_Cell_Wall_√¢¬Ä¬ì_Peptidoglycan_and_Arabinogalactan". Cold
      Spring Harbor Perspectives in Medicine. 5 (8): a021113. doi:10.1101/
      cshperspect.a021113. PMC¬†4526729. PMID¬†25818664.
  76. ^Engelhardt H, Peters J (December 1998). "Structural research on surface
      layers: a focus on stability, surface layer homology domains, and surface
      layer-cell wall interactions". Journal of Structural Biology. 124
      (2√¢¬Ä¬ì3): 276√¢¬Ä¬ì302. doi:10.1006/jsbi.1998.4070. PMID¬†10049812.
  77. ^Beveridge TJ, Pouwels PH, S√É¬°ra M, Kotiranta A, Lounatmaa K, Kari K,
      Kerosuo E, Haapasalo M, Egelseer EM, Schocher I, Sleytr UB, Morelli L,
      Callegari ML, Nomellini JF, Bingle WH, Smit J, Leibovitz E, Lemaire M,
      Miras I, Salamitou S, B√É¬©guin P, Ohayon H, Gounon P, Matuschek M, Koval
      SF (June 1997). "Functions of S-layers". FEMS Microbiology Reviews. 20
      (1√¢¬Ä¬ì2): 99√¢¬Ä¬ì149. doi:10.1016/S0168-6445(97)00043-0. PMID¬†9276929.
  78. ^Kojima S, Blair DF (2004). The bacterial flagellar motor: structure and
      function of a complex molecular machine. International Review of
      Cytology. 233. pp.¬†93√¢¬Ä¬ì134. doi:10.1016/S0074-7696(04)33003-2. ISBN¬†978-
      0-12-364637-8. PMID¬†15037363.
  79. ^Beachey EH (March 1981). "Bacterial adherence: adhesin-receptor
      interactions mediating the attachment of bacteria to mucosal surface".
      The Journal of Infectious Diseases. 143 (3): 325√¢¬Ä¬ì45. doi:10.1093/
      infdis/143.3.325. PMID¬†7014727.
  80. ^Silverman PM (February 1997). "Towards a structural biology of bacterial
      conjugation". Molecular Microbiology. 23 (3): 423√¢¬Ä¬ì29. doi:10.1046/
      j.1365-2958.1997.2411604.x. PMID¬†9044277.
  81. ^Costa TR, Felisberto-Rodrigues C, Meir A, Prevost MS, Redzej A, Trokter
      M, Waksman G (June 2015). "Secretion systems in Gram-negative bacteria:
      structural and mechanistic insights". Nature Reviews. Microbiology. 13
      (6): 343√¢¬Ä¬ì59. doi:10.1038/nrmicro3456. PMID¬†25978706.
  82. ^Stokes RW, Norris-Jones R, Brooks DE, Beveridge TJ, Doxsee D, Thorson LM
      (October 2004). "The_glycan-rich_outer_layer_of_the_cell_wall_of
      Mycobacterium_tuberculosis_acts_as_an_antiphagocytic_capsule_limiting_the
      association_of_the_bacterium_with_macrophages". Infection and Immunity.
      72 (10): 5676√¢¬Ä¬ì86. doi:10.1128/IAI.72.10.5676-5686.2004. PMC¬†517526.
      PMID¬†15385466.
  83. ^Daff√É¬© M, Etienne G (1999). "The capsule of Mycobacterium tuberculosis
      and its implications for pathogenicity". Tubercle and Lung Disease. 79
      (3): 153√¢¬Ä¬ì69. doi:10.1054/tuld.1998.0200. PMID¬†10656114.
  84. ^Finlay BB, Falkow S (June 1997). "Common_themes_in_microbial
      pathogenicity_revisited". Microbiology and Molecular Biology Reviews. 61
      (2): 136√¢¬Ä¬ì69. PMC¬†232605. PMID¬†9184008.
  85. ^Nicholson WL, Munakata N, Horneck G, Melosh HJ, Setlow P (September
      2000). "Resistance_of_Bacillus_endospores_to_extreme_terrestrial_and
      extraterrestrial_environments". Microbiology and Molecular Biology
      Reviews. 64 (3): 548√¢¬Ä¬ì72. doi:10.1128/MMBR.64.3.548-572.2000. PMC¬†99004.
      PMID¬†10974126.
  86. ^ a bMcKenney PT, Driks A, Eichenberger P (January 2013). "The Bacillus
      subtilis endospore: assembly and functions of the multilayered coat".
      Nature Reviews. Microbiology. 11 (1): 33√¢¬Ä¬ì44. doi:10.1038/nrmicro2921.
      PMID¬†23202530.
  87. ^Nicholson WL, Fajardo-Cavazos P, Rebeil R, Slieman TA, Riesenman PJ, Law
      JF, Xue Y (August 2002). "Bacterial endospores and their significance in
      stress resistance". Antonie van Leeuwenhoek. 81 (1√¢¬Ä¬ì4): 27√¢¬Ä¬ì32. doi:
      10.1023/A:1020561122764. PMID¬†12448702.
  88. ^Vreeland RH, Rosenzweig WD, Powers DW (October 2000). "Isolation of a
      250 million-year-old halotolerant bacterium from a primary salt crystal".
      Nature. 407 (6806): 897√¢¬Ä¬ì900. Bibcode:2000Natur.407..897V. doi:10.1038/
      35038060. PMID¬†11057666.
  89. ^Cano RJ, Borucki MK (May 1995). "Revival and identification of bacterial
      spores in 25- to 40-million-year-old Dominican amber". Science. 268
      (5213): 1060√¢¬Ä¬ì64. Bibcode:1995Sci...268.1060C. doi:10.1126/
      science.7538699. PMID¬†7538699.
  90. ^Nicholson WL, Schuerger AC, Setlow P (April 2005). "The solar UV
      environment and bacterial spore UV resistance: considerations for Earth-
      to-Mars transport by natural processes and human spaceflight". Mutation
      Research. 571 (1√¢¬Ä¬ì2): 249√¢¬Ä¬ì64. doi:10.1016/j.mrfmmm.2004.10.012.
      PMID¬†15748651.
  91. ^Hatheway CL (January 1990). "Toxigenic_clostridia". Clinical
      Microbiology Reviews. 3 (1): 66√¢¬Ä¬ì98. doi:10.1128/CMR.3.1.66. PMC¬†358141.
      PMID¬†2404569.
  92. ^Nealson KH (January 1999). "Post-Viking microbiology: new approaches,
      new data, new insights". Origins of Life and Evolution of the Biosphere.
      29 (1): 73√¢¬Ä¬ì93. doi:10.1023/A:1006515817767. PMID¬†11536899.
  93. ^Xu J (June 2006). "Microbial ecology in the age of genomics and
      metagenomics: concepts, tools, and recent advances". Molecular Ecology.
      15 (7): 1713√¢¬Ä¬ì31. doi:10.1111/j.1365-294X.2006.02882.x. PMID¬†16689892.
  94. ^Zillig W (December 1991). "Comparative biochemistry of Archaea and
      Bacteria". Current Opinion in Genetics & Development. 1 (4): 544√¢¬Ä¬ì51.
      doi:10.1016/S0959-437X(05)80206-0. PMID¬†1822288.
  95. ^ a b cSlonczewski JL, Foster JW. Microbiology: An Evolving Science (3
      ed.). WW Norton & Company. pp.¬†491√¢¬Ä¬ì44.
  96. ^Hellingwerf KJ, Crielaard W, Hoff WD, Matthijs HC, Mur LR, van Rotterdam
      BJ (1994). "Photobiology_of_bacteria". Antonie van Leeuwenhoek (Submitted
      manuscript). 65 (4): 331√¢¬Ä¬ì47. doi:10.1007/BF00872217. PMID¬†7832590.
  97. ^Dalton H (June 2005). "The_Leeuwenhoek_Lecture_2000_the_natural_and
      unnatural_history_of_methane-oxidizing_bacteria". Philosophical
      Transactions of the Royal Society of London. Series B, Biological
      Sciences. 360 (1458): 1207√¢¬Ä¬ì22. doi:10.1098/rstb.2005.1657. PMC¬†1569495.
      PMID¬†16147517.
  98. ^Zehr JP, Jenkins BD, Short SM, Steward GF (July 2003). "Nitrogenase gene
      diversity and microbial community structure: a cross-system comparison".
      Environmental Microbiology. 5 (7): 539√¢¬Ä¬ì54. doi:10.1046/j.1462-
      2920.2003.00451.x. PMID¬†12823187.
  99. ^Zumft WG (December 1997). "Cell_biology_and_molecular_basis_of
      denitrification". Microbiology and Molecular Biology Reviews. 61 (4):
      533√¢¬Ä¬ì616. PMC¬†232623. PMID¬†9409151.
 100. ^Drake HL, Daniel SL, K√É¬ºsel K, Matthies C, Kuhner C, Braus-Stromeyer S
      (1997). "Acetogenic bacteria: what are the in situ consequences of their
      diverse metabolic versatilities?". BioFactors. 6 (1): 13√¢¬Ä¬ì24. doi:
      10.1002/biof.5520060103. PMID¬†9233536.
 101. ^Morel FM, Kraepiel AM, Amyot M (1998). "The chemical cycle and
      bioaccumulation of mercury". Annual Review of Ecology and Systematics.
      29: 543√¢¬Ä¬ì66. doi:10.1146/annurev.ecolsys.29.1.543.
 102. ^Koch AL (2002). "Control of the bacterial cell cycle by cytoplasmic
      growth". Critical Reviews in Microbiology. 28 (1): 61√¢¬Ä¬ì77. doi:10.1080/
      1040-840291046696. PMID¬†12003041.
 103. ^Eagon RG (April 1962). "Pseudomonas_natriegens,_a_marine_bacterium_with
      a_generation_time_of_less_than_10_minutes". Journal of Bacteriology. 83
      (4): 736√¢¬Ä¬ì37. PMC¬†279347. PMID¬†13888946.
 104. ^Stewart EJ, Madden R, Paul G, Taddei F (February 2005). "Aging_and_death
      in_an_organism_that_reproduces_by_morphologically_symmetric_division".
      PLoS Biology. 3 (2): e45. doi:10.1371/journal.pbio.0030045. PMC¬†546039.
      PMID¬†15685293.
 105. ^ a b cThomson RB, Bertram H (December 2001). "Laboratory diagnosis of
      central nervous system infections". Infectious Disease Clinics of North
      America. 15 (4): 1047√¢¬Ä¬ì71. doi:10.1016/S0891-5520(05)70186-0.
      PMID¬†11780267.
 106. ^Paerl HW, Fulton RS, Moisander PH, Dyble J (April 2001). "Harmful
      freshwater_algal_blooms,_with_an_emphasis_on_cyanobacteria".
      TheScientificWorldJournal. 1: 76√¢¬Ä¬ì113. doi:10.1100/tsw.2001.16.
      PMC¬†6083932. PMID¬†12805693.
 107. ^Challis GL, Hopwood DA (November 2003). "Synergy_and_contingency_as
      driving_forces_for_the_evolution_of_multiple_secondary_metabolite
      production_by_Streptomyces_species". Proceedings of the National Academy
      of Sciences of the United States of America. 100 Suppl 2 (90002):
      14555√¢¬Ä¬ì61. Bibcode:2003PNAS..10014555C. doi:10.1073/pnas.1934677100.
      PMC¬†304118. PMID¬†12970466.
 108. ^Kooijman SA, Auger P, Poggiale JC, Kooi BW (August 2003). "Quantitative
      steps in symbiogenesis and the evolution of homeostasis". Biological
      Reviews of the Cambridge Philosophical Society. 78 (3): 435√¢¬Ä¬ì63. doi:
      10.1017/S1464793102006127. PMID¬†14558592.
 109. ^Bertrand RL (2019). "Lag Phase is a Dynamic, Organized, Adaptive, and
      Evolvable Period that Prepares Bacteria for Cell Division". Journal of
      Bacteriology. 201 (7): e00697√¢¬Ä¬ì18. doi:10.1128/JB.00697-18.
      PMID¬†30642990.CS1 maint: Uses authors parameter (link)
 110. ^Prats C, L√É¬≥pez D, Gir√É¬≥ A, Ferrer J, Valls J (August 2006).
      "Individual-based modelling of bacterial cultures to study the
      microscopic causes of the lag phase". Journal of Theoretical Biology. 241
      (4): 939√¢¬Ä¬ì53. doi:10.1016/j.jtbi.2006.01.029. PMID¬†16524598.
 111. ^Hecker M, V√É¬∂lker U (2001). General stress response of Bacillus subtilis
      and other bacteria. Advances in Microbial Physiology. 44. pp.¬†35√¢¬Ä¬ì91.
      doi:10.1016/S0065-2911(01)44011-2. ISBN¬†978-0-12-027744-5. PMID¬†11407115.
 112. ^Slonczewski JL, Foster JW. Microbiology: An Evolving Science (3 ed.). WW
      Norton & Company. p.¬†143.
 113. ^Nakabachi A, Yamashita A, Toh H, Ishikawa H, Dunbar HE, Moran NA,
      Hattori M (October 2006). "The 160-kilobase genome of the bacterial
      endosymbiont Carsonella". Science. 314 (5797): 267. doi:10.1126/
      science.1134196. PMID¬†17038615.
 114. ^Pradella S, Hans A, Spr√É¬∂er C, Reichenbach H, Gerth K, Beyer S (December
      2002). "Characterisation, genome size and genetic manipulation of the
      myxobacterium Sorangium cellulosum So ce56". Archives of Microbiology.
      178 (6): 484√¢¬Ä¬ì92. doi:10.1007/s00203-002-0479-2. PMID¬†12420170.
 115. ^Hinnebusch J, Tilly K (December 1993). "Linear_plasmids_and_chromosomes
      in_bacteria". Molecular Microbiology. 10 (5): 917√¢¬Ä¬ì22. doi:10.1111/
      j.1365-2958.1993.tb00963.x. PMID¬†7934868.
 116. ^Lin YS, Kieser HM, Hopwood DA, Chen CW (December 1993). "The chromosomal
      DNA of Streptomyces lividans 66 is linear". Molecular Microbiology. 10
      (5): 923√¢¬Ä¬ì33. doi:10.1111/j.1365-2958.1993.tb00964.x. PMID¬†7934869.
 117. ^Val ME, Soler-Bistu√É¬© A, Bland MJ, Mazel D (December 2014). "Management
      of_multipartite_genomes:_the_Vibrio_cholerae_model". Current Opinion in
      Microbiology. 22: 120√¢¬Ä¬ì26. doi:10.1016/j.mib.2014.10.003. PMID¬†25460805.
 118. ^Kado CI (October 2014). Historical Events That Spawned the Field of
      Plasmid Biology. Microbiology Spectrum. 2. p.¬†3. doi:10.1128/
      microbiolspec.PLAS-0019-2013. ISBN¬†9781555818975. PMID¬†26104369.
 119. ^Belfort M, Reaban ME, Coetzee T, Dalgaard JZ (July 1995). "Prokaryotic
      introns_and_inteins:_a_panoply_of_form_and_function". Journal of
      Bacteriology. 177 (14): 3897√¢¬Ä¬ì903. doi:10.1128/jb.177.14.3897-3903.1995.
      PMC¬†177115. PMID¬†7608058.
 120. ^Denamur E, Matic I (May 2006). "Evolution of mutation rates in
      bacteria". Molecular Microbiology. 60 (4): 820√¢¬Ä¬ì27. doi:10.1111/j.1365-
      2958.2006.05150.x. PMID¬†16677295.
 121. ^Wright BE (May 2004). "Stress-directed adaptive mutations and
      evolution". Molecular Microbiology. 52 (3): 643√¢¬Ä¬ì50. doi:10.1111/j.1365-
      2958.2004.04012.x. PMID¬†15101972.
 122. ^Chen I, Dubnau D (March 2004). "DNA uptake during bacterial
      transformation". Nature Reviews. Microbiology. 2 (3): 241√¢¬Ä¬ì49. doi:
      10.1038/nrmicro844. PMID¬†15083159.
 123. ^Johnsborg O, Eldholm V, H√É¬•varstein LS (December 2007). "Natural genetic
      transformation: prevalence, mechanisms and function". Research in
      Microbiology. 158 (10): 767√¢¬Ä¬ì78. doi:10.1016/j.resmic.2007.09.004.
      PMID¬†17997281.
 124. ^ Bernstein H, Bernstein C, Michod RE (2012). "DNA repair as the primary
      adaptive function of sex in bacteria and eukaryotes". Chapter 1: pp.
      1√¢¬Ä¬ì49 in: DNA Repair: New Research, Sakura Kimura and Sora Shimizu
      (eds.). Nova Sci. Publ., Hauppauge, NY
 125. ISBN¬†978-1-62100-808-8.
 126. ^Br√É¬ºssow H, Canchaya C, Hardt WD (September 2004). "Phages_and_the
      evolution_of_bacterial_pathogens:_from_genomic_rearrangements_to
      lysogenic_conversion". Microbiology and Molecular Biology Reviews. 68
      (3): 560√¢¬Ä¬ì602, table of contents. doi:10.1128/MMBR.68.3.560-602.2004.
      PMC¬†515249. PMID¬†15353570.
 127. ^Bickle TA, Kr√É¬ºger DH (June 1993). "Biology_of_DNA_restriction".
      Microbiological Reviews. 57 (2): 434√¢¬Ä¬ì50. PMC¬†372918. PMID¬†8336674.
 128. ^Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, Moineau S,
      Romero DA, Horvath P (March 2007). "CRISPR provides acquired resistance
      against viruses in prokaryotes". Science. 315 (5819): 1709√¢¬Ä¬ì12. Bibcode:
      2007Sci...315.1709B. doi:10.1126/science.1138140. PMID¬†17379808.
 129. ^Brouns SJ, Jore MM, Lundgren M, Westra ER, Slijkhuis RJ, Snijders AP,
      Dickman MJ, Makarova KS, Koonin EV, van der Oost J (August 2008). "Small
      CRISPR_RNAs_guide_antiviral_defense_in_prokaryotes". Science. 321 (5891):
      960√¢¬Ä¬ì64. Bibcode:2008Sci...321..960B. doi:10.1126/science.1159689.
      PMC¬†5898235. PMID¬†18703739.
 130. ^Michod RE, Bernstein H, Nedelcu AM (May 2008). "Adaptive_value_of_sex_in
      microbial_pathogens" (PDF). Infection, Genetics and Evolution. 8 (3):
      267√¢¬Ä¬ì85. doi:10.1016/j.meegid.2008.01.002. PMID¬†18295550. Archived (PDF)
      from the original on 30 December 2016.
 131. ^Hastings PJ, Rosenberg SM, Slack A (September 2004). "Antibiotic-induced
      lateral transfer of antibiotic resistance". Trends in Microbiology. 12
      (9): 401√¢¬Ä¬ì14. doi:10.1016/j.tim.2004.07.003. PMID¬†15337159.
 132. ^Davison J (September 1999). "Genetic exchange between bacteria in the
      environment". Plasmid. 42 (2): 73√¢¬Ä¬ì91. doi:10.1006/plas.1999.1421.
      PMID¬†10489325.
 133. ^ a b cBardy SL, Ng SY, Jarrell KF (February 2003). "Prokaryotic motility
      structures". Microbiology. 149 (Pt 2): 295√¢¬Ä¬ì304. doi:10.1099/
      mic.0.25948-0. PMID¬†12624192.
 134. ^Macnab RM (December 1999). "The_bacterial_flagellum:_reversible_rotary
      propellor_and_type_III_export_apparatus". Journal of Bacteriology. 181
      (23): 7149√¢¬Ä¬ì53. PMC¬†103673. PMID¬†10572114.
 135. ^Wu M, Roberts JW, Kim S, Koch DL, DeLisa MP (July 2006). "Collective
      bacterial_dynamics_revealed_using_a_three-dimensional_population-scale
      defocused_particle_tracking_technique". Applied and Environmental
      Microbiology. 72 (7): 4987√¢¬Ä¬ì94. doi:10.1128/AEM.00158-06. PMC¬†1489374.
      PMID¬†16820497.
 136. ^Mattick, John S (2002). "Type IV Pili and Twitching Motility". Annual
      Review of Microbiology. 56: 289√¢¬Ä¬ì314. doi:10.1146/
      annurev.micro.56.012302.160938. PMID¬†12142488.
 137. ^Merz AJ, So M, Sheetz MP (September 2000). "Pilus retraction powers
      bacterial twitching motility". Nature. 407 (6800): 98√¢¬Ä¬ì102. Bibcode:
      2000Natur.407...98M. doi:10.1038/35024105. PMID¬†10993081.
 138. ^Lux R, Shi W (July 2004). "Chemotaxis-guided movements in bacteria".
      Critical Reviews in Oral Biology and Medicine. 15 (4): 207√¢¬Ä¬ì20. doi:
      10.1177/154411130401500404. PMID¬†15284186.
 139. ^Schweinitzer T, Josenhans C (July 2010). "Bacterial_energy_taxis:_a
      global_strategy?". Archives of Microbiology. 192 (7): 507√¢¬Ä¬ì20. doi:
      10.1007/s00203-010-0575-7. PMC¬†2886117. PMID¬†20411245.
 140. ^Frankel RB, Bazylinski DA, Johnson MS, Taylor BL (August 1997).
      "Magneto-aerotaxis_in_marine_coccoid_bacteria". Biophysical Journal. 73
      (2): 994√¢¬Ä¬ì1000. Bibcode:1997BpJ....73..994F. doi:10.1016/S0006-3495
      (97)78132-3. PMC¬†1180996. PMID¬†9251816.
 141. ^Goldberg MB (December 2001). "Actin-based_motility_of_intracellular
      microbial_pathogens". Microbiology and Molecular Biology Reviews. 65 (4):
      595√¢¬Ä¬ì626, table of contents. doi:10.1128/MMBR.65.4.595-626.2001.
      PMC¬†99042. PMID¬†11729265.
 142. ^ Dusenbery, David B (1996). Life at Small Scale. Scientific American
      Library.
 143. ISBN¬†0-7167-5060-0.
 144. ^ a bShapiro JA (1998). "Thinking_about_bacterial_populations_as
      multicellular_organisms" (PDF). Annual Review of Microbiology. 52:
      81√¢¬Ä¬ì104. doi:10.1146/annurev.micro.52.1.81. PMID¬†9891794. Archived from
      the_original (PDF) on 17 July 2011.
 145. ^ a bCosterton JW, Lewandowski Z, Caldwell DE, Korber DR, Lappin-Scott HM
      (1995). "Microbial biofilms". Annual Review of Microbiology. 49:
      711√¢¬Ä¬ì45. doi:10.1146/annurev.mi.49.100195.003431. PMID¬†8561477.
 146. ^Miller MB, Bassler BL (2001). "Quorum sensing in bacteria". Annual
      Review of Microbiology. 55: 165√¢¬Ä¬ì99. doi:10.1146/annurev.micro.55.1.165.
      PMID¬†11544353.
 147. ^Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, Bork P (March
      2006). "Toward automatic reconstruction of a highly resolved tree of
      life". Science. 311 (5765): 1283√¢¬Ä¬ì87. Bibcode:2006Sci...311.1283C.
      CiteSeerX¬†10.1.1.381.9514. doi:10.1126/science.1123061. PMID¬†16513982.
 148. ^Boucher Y, Douady CJ, Papke RT, Walsh DA, Boudreau ME, Nesb√É¬∏ CL, Case
      RJ, Doolittle WF (2003). "Lateral gene transfer and the origins of
      prokaryotic groups". Annual Review of Genetics. 37: 283√¢¬Ä¬ì328. doi:
      10.1146/annurev.genet.37.050503.084247. PMID¬†14616063.
 149. ^Olsen GJ, Woese CR, Overbeek R (January 1994). "The_winds_of_
      (evolutionary)_change:_breathing_new_life_into_microbiology". Journal of
      Bacteriology. 176 (1): 1√¢¬Ä¬ì6. doi:10.2172/205047. PMC¬†205007.
      PMID¬†8282683.
 150. ^"IJSEM_Home". Ijs.sgmjournals.org. 28 October 2011. Archived from the
      original on 19 October 2011. Retrieved 4 November 2011.
 151. ^"Bergey's_Manual_Trust". Bergeys.org. Archived from the original on 7
      November 2011. Retrieved 4 November 2011.
 152. ^Gupta RS (2000). "The natural evolutionary relationships among
      prokaryotes". Critical Reviews in Microbiology. 26 (2): 111√¢¬Ä¬ì31.
      CiteSeerX¬†10.1.1.496.1356. doi:10.1080/10408410091154219. PMID¬†10890353.
 153. ^Rapp√É¬© MS, Giovannoni SJ (2003). "The uncultured microbial majority".
      Annual Review of Microbiology. 57: 369√¢¬Ä¬ì94. doi:10.1146/
      annurev.micro.57.030502.090759. PMID¬†14527284.
 154. ^Doolittle RF (June 2005). "Evolutionary aspects of whole-genome
      biology". Current Opinion in Structural Biology. 15 (3): 248√¢¬Ä¬ì53. doi:
      10.1016/j.sbi.2005.04.001. PMID¬†15963888.
 155. ^Cavalier-Smith T (January 2002). "The neomuran origin of archaebacteria,
      the negibacterial root of the universal tree and bacterial
      megaclassification". International Journal of Systematic and Evolutionary
      Microbiology. 52 (Pt 1): 7√¢¬Ä¬ì76. doi:10.1099/00207713-52-1-7.
      PMID¬†11837318.
 156. ^Woods GL, Walker DH (July 1996). "Detection_of_infection_or_infectious
      agents_by_use_of_cytologic_and_histologic_stains". Clinical Microbiology
      Reviews. 9 (3): 382√¢¬Ä¬ì404. doi:10.1128/CMR.9.3.382. PMC¬†172900.
      PMID¬†8809467.
 157. ^Weinstein MP (March 1994). "Clinical importance of blood cultures".
      Clinics in Laboratory Medicine. 14 (1): 9√¢¬Ä¬ì16. doi:10.1016/S0272-2712
      (18)30390-1. PMID¬†8181237.
 158. ^Louie M, Louie L, Simor AE (August 2000). "The_role_of_DNA_amplification
      technology_in_the_diagnosis_of_infectious_diseases". CMAJ. 163 (3):
      301√¢¬Ä¬ì09. PMC¬†80298. PMID¬†10951731. Archived from the original on 14 June
      2006.
 159. ^Oliver JD (February 2005). "The_viable_but_nonculturable_state_in
      bacteria". Journal of Microbiology. 43 Spec No: 93√¢¬Ä¬ì100. PMID¬†15765062.
      Archived from the_original on 28 September 2007.
 160. ^Euz√É¬©by JP (8 December 2011). "Number_of_published_names". List of
      Prokaryotic names with Standing in Nomenclature. Archived from the
      original on 19 January 2012. Retrieved 10 December 2011.
 161. ^Curtis TP, Sloan WT, Scannell JW (August 2002). "Estimating_prokaryotic
      diversity_and_its_limits". Proceedings of the National Academy of
      Sciences of the United States of America. 99 (16): 10494√¢¬Ä¬ì99. Bibcode:
      2002PNAS...9910494C. doi:10.1073/pnas.142680199. PMC¬†124953.
      PMID¬†12097644.
 162. ^Schloss PD, Handelsman J (December 2004). "Status_of_the_microbial
      census". Microbiology and Molecular Biology Reviews. 68 (4): 686√¢¬Ä¬ì91.
      doi:10.1128/MMBR.68.4.686-691.2004. PMC¬†539005. PMID¬†15590780.
 163. ^Fisher B, Harvey RP, Champe PC (2007). "Chapter 33". Lippincott's
      Illustrated Reviews: Microbiology (Lippincott's Illustrated Reviews
      Series). Hagerstwon, MD: Lippincott Williams & Wilkins. pp.¬†367√¢¬Ä¬ì92.
      ISBN¬†978-0-7817-8215-9.
 164. ^ LEF.org_>_Bacterial_Infections Updated: 19 January 2006. Retrieved on
      11 April 2009
 165. ^Martin MO (September 2002). "Predatory prokaryotes: an emerging research
      opportunity". Journal of Molecular Microbiology and Biotechnology. 4 (5):
      467√¢¬Ä¬ì77. PMID¬†12432957.
 166. ^Velicer GJ, Stredwick KL (August 2002). "Experimental social evolution
      with Myxococcus xanthus". Antonie van Leeuwenhoek. 81 (1√¢¬Ä¬ì4): 155√¢¬Ä¬ì64.
      doi:10.1023/A:1020546130033. PMID¬†12448714.
 167. ^Gromov BV (1972). "Electron Microscope Study of Parasitism by
      Bdellovibrio Chorellavorus Bacteria on Cells of the Green Alga Chorella
      Vulgaris". Tsitologiya. 14 (2): 256√¢¬Ä¬ì60.
 168. ^Guerrero R, Pedros-Alio C, Esteve I, Mas J, Chase D, Margulis L (April
      1986). "Predatory_prokaryotes:_predation_and_primary_consumption_evolved
      in_bacteria". Proceedings of the National Academy of Sciences of the
      United States of America. 83 (7): 2138√¢¬Ä¬ì42. Bibcode:1986PNAS...83.2138G.
      doi:10.1073/pnas.83.7.2138. PMC¬†323246. PMID¬†11542073.
 169. ^Velicer GJ, Mendes-Soares H (January 2009). "Bacterial predators".
      Current Biology. 19 (2): R55√¢¬Ä¬ì56. doi:10.1016/j.cub.2008.10.043.
      PMID¬†19174136.
 170. ^Stams AJ, de Bok FA, Plugge CM, van Eekert MH, Dolfing J, Schraa G
      (March 2006). "Exocellular electron transfer in anaerobic microbial
      communities". Environmental Microbiology. 8 (3): 371√¢¬Ä¬ì82. doi:10.1111/
      j.1462-2920.2006.00989.x. PMID¬†16478444.
 171. ^Barea JM, Pozo MJ, Azc√É¬≥n R, Azc√É¬≥n-Aguilar C (July 2005). "Microbial
      co-operation in the rhizosphere". Journal of Experimental Botany. 56
      (417): 1761√¢¬Ä¬ì78. doi:10.1093/jxb/eri197. PMID¬†15911555.
 172. ^O'Hara AM, Shanahan F (July 2006). "The_gut_flora_as_a_forgotten_organ".
      EMBO Reports. 7 (7): 688√¢¬Ä¬ì93. doi:10.1038/sj.embor.7400731. PMC¬†1500832.
      PMID¬†16819463.
 173. ^Zoetendal EG, Vaughan EE, de Vos WM (March 2006). "A microbial world
      within us". Molecular Microbiology. 59 (6): 1639√¢¬Ä¬ì50. doi:10.1111/
      j.1365-2958.2006.05056.x. PMID¬†16553872.
 174. ^Gorbach_SL (February 1990). "Lactic acid bacteria and human health".
      Annals of Medicine. 22 (1): 37√¢¬Ä¬ì41. doi:10.3109/07853899009147239.
      PMID¬†2109988.
 175. ^Salminen SJ, Gueimonde M, Isolauri E (May 2005). "Probiotics that modify
      disease risk". The Journal of Nutrition. 135 (5): 1294√¢¬Ä¬ì98. doi:10.1093/
      jn/135.5.1294. PMID¬†15867327.
 176. ^Fish DN (February 2002). "Optimal antimicrobial therapy for sepsis".
      American Journal of Health-System Pharmacy. 59 Suppl 1: S13√¢¬Ä¬ì19. doi:
      10.1093/ajhp/59.suppl_1.S13. PMID¬†11885408.
 177. ^Belland RJ, Ouellette SP, Gieffers J, Byrne GI (February 2004).
      "Chlamydia pneumoniae and atherosclerosis". Cellular Microbiology. 6 (2):
      117√¢¬Ä¬ì27. doi:10.1046/j.1462-5822.2003.00352.x. PMID¬†14706098.
 178. ^Heise ER (February 1982). "Diseases_associated_with_immunosuppression".
      Environmental Health Perspectives. 43: 9√¢¬Ä¬ì19. doi:10.2307/3429162.
      JSTOR¬†3429162. PMC¬†1568899. PMID¬†7037390.
 179. ^Saiman L (2004). "Microbiology of early CF lung disease". Paediatric
      Respiratory Reviews. 5 Suppl A: S367√¢¬Ä¬ì69. doi:10.1016/S1526-0542
      (04)90065-6. PMID¬†14980298.
 180. ^Yonath A, Bashan A (2004). "Ribosomal crystallography: initiation,
      peptide bond formation, and amino acid polymerization are hampered by
      antibiotics". Annual Review of Microbiology. 58: 233√¢¬Ä¬ì51. doi:10.1146/
      annurev.micro.58.030603.123822. PMID¬†15487937.
 181. ^Khachatourians GG (November 1998). "Agricultural_use_of_antibiotics_and
      the_evolution_and_transfer_of_antibiotic-resistant_bacteria". CMAJ. 159
      (9): 1129√¢¬Ä¬ì36. PMC¬†1229782. PMID¬†9835883.
 182. ^Johnson ME, Lucey JA (April 2006). "Major technological advances and
      trends in cheese". Journal of Dairy Science. 89 (4): 1174√¢¬Ä¬ì78. doi:
      10.3168/jds.S0022-0302(06)72186-5. PMID¬†16537950.
 183. ^Hagedorn S, Kaphammer B (1994). "Microbial biocatalysis in the
      generation of flavor and fragrance chemicals". Annual Review of
      Microbiology. 48: 773√¢¬Ä¬ì800. doi:10.1146/annurev.mi.48.100194.004013.
      PMID¬†7826026.
 184. ^Cohen Y (December 2002). "Bioremediation of oil by marine microbial
      mats". International Microbiology. 5 (4): 189√¢¬Ä¬ì93. doi:10.1007/s10123-
      002-0089-5. PMID¬†12497184.
 185. ^Neves LC, Miyamura TT, Moraes DA, Penna TC, Converti A (2006).
      "Biofiltration methods for the removal of phenolic residues". Applied
      Biochemistry and Biotechnology. 129√¢¬Ä¬ì132 (1√¢¬Ä¬ì3): 130√¢¬Ä¬ì52. doi:10.1385/
      ABAB:129:1:130. PMID¬†16915636.
 186. ^Liese A, Filho MV (December 1999). "Production of fine chemicals using
      biocatalysis". Current Opinion in Biotechnology. 10 (6): 595√¢¬Ä¬ì603. doi:
      10.1016/S0958-1669(99)00040-3. PMID¬†10600695.
 187. ^Aronson AI, Shai Y (February 2001). "Why Bacillus thuringiensis
      insecticidal toxins are so effective: unique features of their mode of
      action". FEMS Microbiology Letters. 195 (1): 1√¢¬Ä¬ì8. doi:10.1111/j.1574-
      6968.2001.tb10489.x. PMID¬†11166987.
 188. ^Bozsik A (July 2006). "Susceptibility of adult Coccinella septempunctata
      (Coleoptera: Coccinellidae) to insecticides with different modes of
      action". Pest Management Science. 62 (7): 651√¢¬Ä¬ì54. doi:10.1002/ps.1221.
      PMID¬†16649191.
 189. ^Chattopadhyay A, Bhatnagar NB, Bhatnagar R (2004). "Bacterial
      insecticidal toxins". Critical Reviews in Microbiology. 30 (1): 33√¢¬Ä¬ì54.
      doi:10.1080/10408410490270712. PMID¬†15116762.
 190. ^Serres MH, Gopal S, Nahum LA, Liang P, Gaasterland T, Riley M (2001). "A
      functional_update_of_the_Escherichia_coli_K-12_genome". Genome Biology. 2
      (9): RESEARCH0035. doi:10.1186/gb-2001-2-9-research0035. PMC¬†56896.
      PMID¬†11574054.
 191. ^Almaas E, Kov√É¬°cs B, Vicsek T, Oltvai ZN, Barab√É¬°si AL (February 2004).
      "Global organization of metabolic fluxes in the bacterium Escherichia
      coli". Nature. 427 (6977): 839√¢¬Ä¬ì43. arXiv:q-bio/0403001. Bibcode:
      2004Natur.427..839A. doi:10.1038/nature02289. PMID¬†14985762.
 192. ^Reed JL, Vo TD, Schilling CH, Palsson BO (2003). "An_expanded_genome-
      scale_model_of_Escherichia_coli_K-12_(iJR904_GSM/GPR)". Genome Biology. 4
      (9): R54. doi:10.1186/gb-2003-4-9-r54. PMC¬†193654. PMID¬†12952533.
 193. ^Walsh G (April 2005). "Therapeutic insulins and their large-scale
      manufacture". Applied Microbiology and Biotechnology. 67 (2): 151√¢¬Ä¬ì59.
      doi:10.1007/s00253-004-1809-x. PMID¬†15580495.
 194. ^Graumann K, Premstaller A (February 2006). "Manufacturing of recombinant
      therapeutic proteins in microbial systems". Biotechnology Journal. 1 (2):
      164√¢¬Ä¬ì86. doi:10.1002/biot.200500051. PMID¬†16892246.
 195. ^Porter JR (June 1976). "Antony_van_Leeuwenhoek:_tercentenary_of_his
      discovery_of_bacteria". Bacteriological Reviews. 40 (2): 260√¢¬Ä¬ì69.
      PMC¬†413956. PMID¬†786250.
 196. ^van Leeuwenhoek A (1684). "An abstract of a letter from Mr. Anthony
      Leevvenhoek at Delft, dated Sep. 17, 1683, Containing Some Microscopical
      Observations, about Animals in the Scurf of the Teeth, the Substance
      Call'd Worms in the Nose, the Cuticula Consisting of Scales".
      Philosophical Transactions. 14 (155√¢¬Ä¬ì166): 568√¢¬Ä¬ì74. doi:10.1098/
      rstl.1684.0030.
 197. ^van Leeuwenhoek A (1700). "Part of a Letter from Mr Antony van
      Leeuwenhoek, concerning the Worms in Sheeps Livers, Gnats, and Animalcula
      in the Excrements of Frogs". Philosophical Transactions. 22 (260√¢¬Ä¬ì276):
      509√¢¬Ä¬ì18. doi:10.1098/rstl.1700.0013.
 198. ^van Leeuwenhoek A (1702). "Part of a Letter from Mr Antony van
      Leeuwenhoek, F.R.S. concerning Green Weeds Growing in Water, and Some
      Animalcula Found about Them". Philosophical Transactions. 23 (277√¢¬Ä¬ì288):
      1304√¢¬Ä¬ì11. doi:10.1098/rstl.1702.0042.
 199. ^Asimov_I (1982). Asimov's_Biographical_Encyclopedia_of_Science_and
      Technology (2nd ed.). Garden City, NY: Doubleday and Company. p.¬†143.
 200. ^Ehrenberg CG (1828). Symbolae Physioe. Animalia evertebrata. Berlin:
      Decas prima.
 201. ^Breed RS, Conn HJ (May 1936). "The_Status_of_the_Generic_Term_Bacterium
      Ehrenberg_1828". Journal of Bacteriology. 31 (5): 517√¢¬Ä¬ì18. PMC¬†543738.
      PMID¬†16559906.
 202. ^Ehrenberg CG (1835). Dritter Beitrag zur Erkenntniss grosser
      Organisation in der Richtung des kleinsten Raumes [Third contribution to
      the knowledge of great organization in the direction of the smallest
      space] (in German). Berlin: Physikalische Abhandlungen der Koeniglichen
      Akademie der Wissenschaften. pp.¬†143√¢¬Ä¬ì336.
 203. ^"Pasteur's_Papers_on_the_Germ_Theory". LSU Law Center's Medical and
      Public Health Law Site, Historic Public Health Articles. Archived from
      the original on 18 December 2006. Retrieved 23 November 2006.
 204. ^"The_Nobel_Prize_in_Physiology_or_Medicine_1905". Nobelprize.org.
      Archived from the original on 10 December 2006. Retrieved 22 November
      2006.
 205. ^O'Brien SJ, Goedert JJ (October 1996). "HIV_causes_AIDS:_Koch's
      postulates_fulfilled". Current Opinion in Immunology. 8 (5): 613√¢¬Ä¬ì18.
      doi:10.1016/S0952-7915(96)80075-6. PMID¬†8902385.
 206. ^Chung K. "Ferdinand_Julius_Cohn_(1828√¢¬Ä¬ì1898):_Pioneer_of_Bacteriology"
      (PDF). Department of Microbiology and Molecular Cell Sciences, The
      University of Memphis. Archived (PDF) from the original on 27 July 2011.
 207. ^Drews, Gerhart (1999). "Ferdinand_Cohn,_a_founder_of_modern
      microbiology" (PDF). ASM News. 65 (8): 547√¢¬Ä¬ì52. Archived from the
      original (PDF) on 13 July 2017.
 208. ^Thurston AJ (December 2000). "Of blood, inflammation and gunshot wounds:
      the history of the control of sepsis". The Australian and New Zealand
      Journal of Surgery. 70 (12): 855√¢¬Ä¬ì61. doi:10.1046/j.1440-
      1622.2000.01983.x. PMID¬†11167573.
 209. ^Schwartz RS (March 2004). "Paul Ehrlich's magic bullets". The New
      England Journal of Medicine. 350 (11): 1079√¢¬Ä¬ì80. doi:10.1056/
      NEJMp048021. PMID¬†15014180.
 210. ^"Biography_of_Paul_Ehrlich". Nobelprize.org. Archived from the original
      on 28 November 2006. Retrieved 26 November 2006.
 211. ^ETH_Zurich (1 April 2019). "First_bacterial_genome_created_entirely_with
      a_computer". EurekAlert!. Retrieved 2 April 2019.
 212. ^Venetz, Jonathan E.; et al. (1 April 2019). "Chemical_synthesis
      rewriting_of_a_bacterial_genome_to_achieve_design_flexibility_and
      biological_functionality". Proceedings_of_the_National_Academy_of
      Sciences_of_the_United_States_of_America. 116 (16): 8070√¢¬Ä¬ì8079. doi:
      10.1073/pnas.1818259116. PMC¬†6475421.
***** Further reading *****
Library_resources about
Bacteria
===============================================================================
    * Online_books
    * Resources_in_your_library
    * Resources_in_other_libraries
    * Alcamo IE (2001). Fundamentals of microbiology. Boston: Jones and
      Bartlett. ISBN¬†978-0-7637-1067-5.
Atlas RM (1995). Principles of microbiology. St. Louis: Mosby. ISBN¬†978-0-8016-
7790-8.
Martinko JM, Madigan MT (2005). Brock Biology of Microorganisms (11th ed.).
Englewood Cliffs, N.J: Prentice Hall. ISBN¬†978-0-13-144329-7.
Holt JC, Bergey DH (1994). Bergey's manual of determinative bacteriology (9th
ed.). Baltimore: Williams & Wilkins. ISBN¬†978-0-683-00603-2.
Hugenholtz P, Goebel BM, Pace NR (September 1998). "Impact_of_culture-
independent_studies_on_the_emerging_phylogenetic_view_of_bacterial_diversity".
Journal of Bacteriology. 180 (18): 4765√¢¬Ä¬ì74. PMC¬†107498. PMID¬†9733676.
Funke BR, Tortora GJ, Case CL (2004). Microbiology: an introduction (8th ed.).
San Francisco: Benjamin Cummings. ISBN¬†978-0-8053-7614-2.
Ogunseitan OA (2005). Microbial Diversity: Form and Function in Prokaryotes.
Wiley-Blackwell. ISBN¬†978-1-4051-4448-3.
Shively JM (2006). Complex Intracellular Structures in Prokaryotes
(Microbiology Monographs). Berlin: Springer. ISBN¬†978-3-540-32524-6.
***** External links *****
    * MicrobeWiki, an extensive wiki about bacteria and viruses
    * Bacteria_that_affect_crops_and_other_plants
    * Bacterial_Nomenclature_Up-To-Date_from_DSMZ
    * Genera_of_the_domain_Bacteria √¢¬Ä¬ì list of Prokaryotic names with Standing
      in Nomenclature
    * The_largest_bacteria
    * Tree_of_Life:_Eubacteria
    * Videos of bacteria swimming and tumbling, use of optical tweezers and
      other videos.
    * Planet_of_the_Bacteria by Stephen_Jay_Gould
    * On-line_text_book_on_bacteriology
    * Animated_guide_to_bacterial_cell_structure.
    * Bacteria_Make_Major_Evolutionary_Shift_in_the_Lab
    * Online_collaboration_for_bacterial_taxonomy.
    * PATRIC, a Bioinformatics Resource Center for bacterial pathogens, funded
      by NIAID
    * Bacterial_Chemotaxis_Interactive_Simulator √¢¬Ä¬ì A web-app that uses
      several simple algorithms to simulate bacterial chemotaxis.
    * Cell-Cell_Communication_in_Bacteria on-line lecture by Bonnie_Bassler,
      and TED:_Discovering_bacteria's_amazing_communication_system
    * Sulfur-cycling_fossil_bacteria_from_the_1.8-Ga_Duck_Creek_Formation
      provide_promising_evidence_of_evolution's_null_hypothesis, Proceedings of
      the National Academy of Sciences of the USA
          o Summarised in: Scientists_discover_bacteria_that_haven't_evolved_in
            more_than_2_billion_years, LiveScience and BusinessInsider
    * v
    * t
    * e
Prokaryotes: Bacteria classification (phyla and orders)
      Plant
      Hacrobia
      Heterokont
      Alveolata
      Rhizaria
      Excavata
      Amoebozoa
      Opisthokonta
            Animal
            Fungi)
                                                 * Deinococcus√¢¬Ä¬ìThermus
                                                       o Deinococcales
                                                       o Thermales
                                                 * Chloroflexi
                                                       o Anaerolineales
                                                       o Caldilineales
                         Eobacteria                    o Chloroflexales
         Terra-/_                                      o Herpetosiphonales
         Glidobacteria                                 o Dehalococcoidales
         (BV1)                                         o Ktedonobacterales
                                                       o Thermogemmatisporales
                                                       o Thermomicrobiales
                                                       o Sphaerobacterales
                                                 * Thermodesulfobacteria
                                                 * thermophiles
                         other glidobacteria           o Aquificae
                                                       o Thermotogae
                                                 * Cyanobacteria
                                     * Caulobacterales
                                     * Kiloniellales
                                     * Kordiimonadales
                                     * Magnetococcales
                                     * Parvularculales
                         Alpha       * Rhizobiales
                                     * Rhodobacterales
                                     * Rhodospirillales
                                     * Rickettsiales
                                     * Sneathiellales
                                     * Sphingomonadales
                                     * Burkholderiales
                                     * Hydrogenophilales
                                     * Methylophilales
                         Beta        * Neisseriales
                                     * Nitrosomonadales
                                     * Procabacteriales
                                     * Rhodocyclales
                                     * Acidithiobacillales
                                     * Aeromonadales
                                     * Alteromonadales
                                     * Cardiobacteriales
         Proteobacteria              * Chromatiales
         (BV2)                       * Enterobacteriales
                                     * Legionellales
                         Gamma       * Methylococcales
                                     * Oceanospirillales
                                     * Orbales
                                     * Pasteurellales
                                     * Pseudomonadales
                                     * Salinisphaerales
                                     * Thiotrichales
                                     * Vibrionales
                                     * Xanthomonadales
G-/                                  * Bdellovibrionales
OM                                   * Desulfarculales
                                     * Desulfobacterales
                                     * Desulfovibrionales
                         Delta       * Desulfurellales
                                     * Desulfuromonadales
                                     * Myxococcales
                                     * Syntrophobacterales
                                     * Syntrophorhabdales
                         Epsilon     * Campylobacterales
                                     * Nautiliales
                         Zeta        * Mariprofundales
                         Spirochaetes          * Spirochaetes
                                               * Fibrobacteres
                                               * Chlorobi
                                                     o Chlorobiales
                         Sphingobacteria             o Ignavibacteriales
                         (FCB group)           * Bacteroidetes
                                                     o Bacteroidales
                                                     o Cytophagales
                                                     o Flavobacteriales
                                                     o Sphingobacteriales
                                               * Chlamydiae
                                               * Lentisphaerae
                                                     o Lentisphaerales
                                                     o Oligosphaerales
                                                     o Victivallales
                                               * Planctomycetes
                         Planctobacteria/            o Phycisphaerales
                         (PVC group)                 o Planctomycetales
                                               * Verrucomicrobia
                                                     o Puniceicoccales
         BV4                                         o Opitutales
                                                     o Chthoniobacterales
                                                     o Verrucomicrobiales
                                               * "Poribacteria"
                                               * Acidobacteria
                                                     o Acidobacteriales
                                                     o Acanthopleuribacterales
                                                     o Holophagales
                                                     o Solibacterales
                                               * Armatimonadetes
                                                     o Armatimonadales
                                                     o Chthonomonadales
                         Other GN                    o Fimbriimonadales
                                               * Caldiserica
                                               * Chrysiogenetes
                                               * Deferribacteres
                                               * Dictyoglomi
                                               * Elusimicrobia
                                               * Fusobacteria
                                               * Gemmatimonadetes
                                               * Nitrospirae
                                               * Synergistetes
                                                * Bacillales
                                                * Lactobacillales
                                                * Aphragmobacteria
                                                      o Erysipelotrichia
                                                            # Erysipelotrichiales
                        Bacilli                       o Mollicutes
                                                            # Mycoplasmatales
                                                            # Entomoplasmatales
         Firmicutes                                         # Anaeroplasmatales
         (BV3)                                              # Acholeplasmatales
                                                            # Haloplasmatales
                                                * Clostridiales
                                                * Halanaerobiales
G+/                     Clostridia              * Thermoanaerobacteriales
no OM                                           * Natranaerobiales
                                                * Negativicutes
                                                      o Selenomonadales
                        Thermolithobacteria     * Thermolithobacterales
                        Actinobacteria      * Actinomycetales
                                            * Bifidobacteriales
                        Acidimicrobiia      * Acidimicrobiales
         Actinobacteria Coriobacteriia      * Coriobacteriales
                        Nitriliruptoria     * Euzebyales
         (BV5)                              * Nitriliruptorales
                                            * Gaiellales
                        Rubrobacteria       * Rubrobacterales
                                            * Thermoleophilales
                                            * Solirubrobacterales
Incertae     * √¢¬Ä¬†Archaeosphaeroides
sedis        * √¢¬Ä¬†Eobacterium
             * √¢¬Ä¬†Leptotrichites
Source: Bergey's_Manual (2001√¢¬Ä¬ì2012). Alternative views: Wikispecies.
    * v
    * t
    * e
Microbiology: Bacteria
                 * infection
                 * Coley's_toxins
Medical          * Exotoxin
microbiology     * Lysogenic_cycle
                 * Pathogenic_bacteria
                 * resistance
                            * Aerobic
                                  o Obligate
                            * Anaerobic
             Oxygen               o Facultative
             preference           o Obligate
                            * Microaerophile
                            * Nanaerobe
                            * Aerotolerant
                            * Extremophile
                            * Human_flora
                                  o Gut
Biochemistry                      o Lung
and ecology                       o Mouth
                                  o Skin
                                  o Vaginal (In_pregnancy)
                                  o Placental
             Other                o Uterine
                                  o Salivary
                            * Microbial_metabolism
                            * Nitrogen_fixation
                            * Microbial_ecology
                            * Primary_nutritional_groups
                            * Substrate preference
                                  o Lipophilic
                                  o Saccharophilic
                 * Bacterial_cellular_morphologies
                 * Coccus
Shape                  o Diplococcus
                 * Bacillus
                 * Coccobacillus
                 * Spiral
                           * Cell_membrane
                           * Cell_wall: Peptidoglycan
                                 o NAM
                                 o NAG
                                 o DAP
                           * Gram-positive_bacteria only: Teichoic_acid
             Cell          * Lipoteichoic_acid
             envelope      * Endospore
                           * Gram-negative_bacteria only: Bacterial_outer
                             membrane
                                 o Porin
Structure                        o Lipopolysaccharide
                           * Periplasmic_space
                           * Mycobacteria only: Arabinogalactan
                           * Mycolic_acid
                           * Bacterial_capsule
                           * Slime_layer
             Outside       * S-layer
             envelope      * Glycocalyx
                           * Pilus
                           * Fimbria
                           * Non-motile_bacteria
             Composite     * Biofilm
                 * Bacteria_(classifications)
                 * Bacterial_phyla
                 * Former groupings: Schizomycetes
                 * Monera
Taxonomy         * Prokaryota
                       o Gracilicutes
                       o Firmicutes
                       o Mollicutes
                       o Mendosicutes
    * [Wikipedia book] Book
    * [Category] Category
    * [Commons page] Commons
    * [Portal] Portal
    * v
    * t
    * e
Microorganisms
                  * Archaea
                  * Bacterium
                  * Cyanobacteria
                  * Fungi
Groups            * Nanobacterium
                  * Prokaryote
                  * Protist
                  * Protozoa
                  * Virus
                  * Microbial_biogeography
                  * Microbial_genetics
                  * Microbial_intelligence
Microbiology      * Microbial_metabolism
                  * Microbial_phylogenetics
                  * Microbial_population_biology
                  * Mycology
                  * Virology
                  * Microbial_cooperation
                  * Microbial_ecology
                  * Microbial_food_web
Ecology           * Microbial_loop
                  * Microbial_synergy
                  * Host_microbe_interactions_in
                    Caenorhabditis_elegans
                  * Dark_field_microscopy
                  * Impedance_microbiology
Techniques        * Microbial_cytology
                  * Microbiological_culture
                  * Staining
                  * Marine_microorganism               [Streptococcus_pyogenes]
                        o Antarctic
Marine            * Marine_microbial_symbiosis
                  * Microalgae
                  * International_Census_of_Marine
                    Microbes
                  * Microbes_in_human_culture
                  * Food_microbiology
                  * Microbial_oil
                  * Microbial_symbiosis_and_immunity
                  * Nylon-eating
                  * Human_microbiota
                        o asthma
                        o dysbiosis
                        o fecal
Human related           o Human_Microbiome_Project
                        o gut
                        o lung
                        o mouth
                        o skin
                        o vagina
                              # in_pregnancy
                        o placenta
                        o uterus
                  * Protein_production
                  * Lines_on_the_Antiquity_of_Microbes
                  * Microbially_induced_sedimentary
Other               structure
                  * Microbial_dark_matter
                  * Physical_factors_affecting
                    microbial_life
    * [Category] Category
    * [Commons page] Commons
    * v
    * t
    * e
Antonie_van_Leeuwenhoek
                                                    * Bacteria
                                                    * Ciliate
                                                    * Giardia
                          Microscopic_organisms     * Infusoria
Microscopic discoveries¬†                           * Protist
1                                                   * Protozoa
                                                    * Rotifer
                                                    * Volvox
                                                    * Spermatozoa (sperm_cells)
                          Others                    * Red_blood_cells
                                                    * Crystals in gouty_tophi
                              * Animalcule
                              * Bacteriology
                              * Protozoology
                              * Protistology
                              * Spontaneous_generation
                              * Preformationism
                              * Microscopic_discovery_of_microorganisms
                              * Microscopic_scale
                              * History_of_biology
                              * History_of_microbiology
General topics                * Natural_history
                              * Optical_microscopy
                              * History_of_microscopy
                              * History_of_the_microscope
                              * Invention_of_the_optical_microscope
                              * Timeline_of_microscope_technology
                              * Golden_Age_of_Dutch_science_and_technology
                              * Science_and_technology_in_the_Dutch_Republic
                              * Royal_Society
                              * Scientific_Revolution
                              * Age_of_Reason
                              * Clifford_Dobell (Leeuwenhoek scholar)
                              * Brian_J._Ford (Leeuwenhoek scholar)
                              * Galileo_Galilei
                              * Regnier_de_Graaf
Related people                * Robert_Hooke (author of Micrographia)
                              * Nicolaas_Hartsoeker
                              * Nicolas_Steno
                              * Jan_Swammerdam
                              * Johannes_Vermeer
                              * Antoni_van_Leeuwenhoek_Ziekenhuis
                              * Leeuwenhoek_(crater)
                              * Leeuwenhoek_Lecture
                              * Leeuwenhoek_Medal
Recognitions                  * Antonie_van_Leeuwenhoek_(journal)
                              * Leeuwenhoeckia
                              * Levenhookia
                              * Leeuwenhoekiella
                              * List_of_people_considered_father_or_mother_of_a
                                scientific_field
    * 1 First observed, described, and studied by van Leeuwenhoek.
    * [Category] Category
    * v
    * t
    * e
Extant Life phyla/divisions by domain
             * Acidobacteria
             * Actinobacteria
             * Aquificae
             * Bacteroidetes
             * Chlamydiae
             * Chlorobi
             * Chloroflexi
             * Chrysiogenetes
             * Cyanobacteria
             * Deferribacteres
             * Deinococcus√¢¬Ä¬ìThermus
Bacteria     * Dictyoglomi
             * Fibrobacteres
             * Firmicutes
             * Fusobacteria
             * Gemmatimonadetes
             * Nitrospirae
             * Planctomycetes
             * Proteobacteria
             * Spirochaetes
             * Thermodesulfobacteria
             * Thermomicrobia
             * Thermotogae
             * Verrucomicrobia
            * Crenarchaeota
            * Euryarchaeota
Archaea     * Korarchaeota
            * Nanoarchaeota
            * Archaeal_Richmond_Mine_acidophilic
              nanoorganisms
                       * Heterokontophyta
                       * Haptophyta
                       * Cryptophyta
                       * Ciliophora
                       * Apicomplexa
                       * Dinoflagellata
                       * Euglenozoa
          Protista     * Percolozoa
                       * Metamonada
                       * Radiolaria
                       * Foraminifera
                       * Cercozoa
                       * Rhodophyta
                       * Glaucophyta
                       * Amoebozoa
                       * Chytridiomycota
                       * Blastocladiomycota
                       * Neocallimastigomycota
          Fungi        * Glomeromycota
                       * Zygomycota
                       * Ascomycota
                       * Basidiomycota
                       * Chlorophyta
                       * Charophyta
                       * Marchantiophyta
                       * Anthocerotophyta
                       * Moss
          Plant        * Lycopodiophyta
                       * Pteridophyta
                       * Cycadophyta
                       * Ginkgophyta
                       * Pinophyta
                       * Gnetophyta
Eukaryote              * Flowering_plant
                       * Sponge
                       * Ctenophora
                       * Placozoa
                       * Cnidaria
                       * Xenacoelomorpha
                       * Chordata
                       * Hemichordata
                       * Echinodermata
                       * Chaetognatha
                       * Kinorhyncha
                       * Loricifera
                       * Priapulida
                       * Nematoda
                       * Nematomorpha
                       * Onychophora
                       * Tardigrada
          Animal       * Arthropoda
                       * Flatworm
                       * Gastrotricha
                       * Orthonectida
                       * Dicyemida
                       * Rotifera
                       * Acanthocephala
                       * Gnathostomulida
                       * Micrognathozoa
                       * Cycliophora
                       * Nemertea
                       * Phoronida
                       * Bryozoa
                       * Entoprocta
                       * Brachiopoda
                       * Mollusca
                       * Annelida
                      * Wikidata: Q10876
                      * Wikispecies: Bacteria
                      * EoL: 288
                      * EPPO: 1BACTK
Taxon_identifiers     * GBIF: 3
                      * iNaturalist: 67333
                      * ITIS: 50
                      * NCBI: 2
                      * NZOR: 50966b34-a83a-4324-9136-d64696ca3d52
                      * PPE: bacteria
                                              * GND: 4004296-0
Authority_control [Edit_this_at_Wikidata]     * LCCN: sh85010813
                                              * NDL: 00570000

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Bacteria&oldid=909716386"
Categories:
    * Bacteria
    * Bacteriology
    * Domains_(biology)
    * Microscopic_organisms_described_by_Antonie_van_Leeuwenhoek
    * 1670s_in_science
Hidden categories:
    * Webarchive_template_wayback_links
    * Articles_containing_Ancient_Greek-language_text
    * CS1:_long_volume_value
    * CS1_maint:_Uses_authors_parameter
    * CS1_German-language_sources_(de)
    * Wikipedia_indefinitely_semi-protected_pages
    * Wikipedia_indefinitely_move-protected_pages
    * Articles_with_short_description
    * Use_dmy_dates_from_March_2014
    * Featured_articles
    * Use_British_English_from_September_2016
    * Articles_with_'species'_microformats
    * Articles_with_hAudio_microformats
    * Pages_including_recorded_pronunciations_(English)
    * Articles_containing_Latin-language_text
    * Articles_containing_Greek-language_text
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_LCCN_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
‚Å∞
**** Variants ****
**** Views ****
    * Read
    * View_source
    * View_history
‚Å∞
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
    * Wikispecies
    * Wikiquote
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ac√É¬®h
    * Afrikaans
    * Alemannisch
    * √°¬ä¬†√°¬à¬õ√°¬à¬≠√°¬ä¬õ
    * √ò¬ß√ô¬Ñ√ò¬π√ò¬±√ò¬®√ô¬ä√ò¬©
    * Aragon√É¬©s
    * √†¬¶¬Ö√†¬¶¬∏√†¬¶¬Æ√†¬ß¬Ä√†¬¶¬Ø√†¬¶¬º√†¬¶¬æ
    * Asturianu
    * Ava√É¬±e'√°¬∫¬Ω
    * Aymar_aru
    * Az√â¬ôrbaycanca
    * √ò¬™√õ¬Ü√ò¬±√ö¬©√ò¬¨√ô¬á
    * √†¬¶¬¨√†¬¶¬æ√†¬¶¬Ç√†¬¶¬≤√†¬¶¬æ
    * B√É¬¢n-l√É¬¢m-g√É¬∫
    * √ê¬ë√ê¬∞√ë¬à√í¬°√ê¬æ√ë¬Ä√ë¬Ç√ë¬Å√ê¬∞
    * √ê¬ë√ê¬µ√ê¬ª√ê¬∞√ë¬Ä√ë¬É√ë¬Å√ê¬∫√ê¬∞√ë¬è
    * √ê¬ë√ê¬µ√ê¬ª√ê¬∞√ë¬Ä√ë¬É√ë¬Å√ê¬∫√ê¬∞√ë¬è_(√ë¬Ç√ê¬∞√ë¬Ä√ê¬∞√ë¬à√ê¬∫√ê¬µ√ê¬≤√ë¬ñ√ë¬Ü√ê¬∞)√¢¬Ä¬é
    * √ê¬ë√ë¬ä√ê¬ª√ê¬≥√ê¬∞√ë¬Ä√ë¬Å√ê¬∫√ê¬∏
    * Bosanski
    * Brezhoneg
    * √ê¬ë√ë¬É√ë¬Ä√ë¬è√ê¬∞√ê¬¥
    * Catal√É¬†
    * √ê¬ß√ì¬ë√ê¬≤√ê¬∞√ë¬à√ê¬ª√ê¬∞
    * √Ñ¬åe√Ö¬°tina
    * Cymraeg
    * Dansk
    * Deutsch
    * √û¬ã√û¬®√û¬à√û¬¨√û¬Ä√û¬®√û¬Ñ√û¬¶√û¬ê√û¬∞
    * Eesti
    * √é¬ï√é¬ª√é¬ª√é¬∑√é¬Ω√é¬π√é¬∫√é¬¨
    * Espa√É¬±ol
    * Esperanto
    * Estreme√É¬±u
    * Euskara
    * √ô¬Å√ò¬ß√ò¬±√ò¬≥√õ¬å
    * Fiji_Hindi
    * F√É¬∏royskt
    * Fran√É¬ßais
    * Frysk
    * Gaeilge
    * Gaelg
    * Galego
    * G√Ñ¬©k√Ö¬©y√Ö¬©
    * √•¬Æ¬¢√•¬Æ¬∂√®¬™¬û/Hak-k√É¬¢-ng√É¬Æ
    * √≠¬ï¬ú√™¬µ¬≠√¨¬ñ¬¥
    * √ï¬Ä√ï¬°√ï¬µ√ï¬•√ñ¬Ä√ï¬•√ï¬∂
    * √†¬§¬π√†¬§¬ø√†¬§¬®√†¬•¬ç√†¬§¬¶√†¬•¬Ä
    * Hornjoserbsce
    * Hrvatski
    * Ido
    * Ilokano
    * Bahasa_Indonesia
    * Interlingua
    * √É¬çslenska
    * Italiano
    * √ó¬¢√ó¬ë√ó¬®√ó¬ô√ó¬™
    * Jawa
    * √†¬≤¬ï√†¬≤¬®√†¬≥¬ç√†¬≤¬®√†¬≤¬°
    * √°¬É¬•√°¬É¬ê√°¬É¬†√°¬É¬ó√°¬É¬£√°¬É¬ö√°¬É¬ò
    * √í¬ö√ê¬∞√ê¬∑√ê¬∞√í¬õ√ë¬à√ê¬∞
    * Kernowek
    * Kinyarwanda
    * Kiswahili
    * Krey√É¬≤l_ayisyen
    * Kurd√É¬Æ
    * √ê¬ö√ë¬ã√ë¬Ä√ê¬≥√ë¬ã√ê¬∑√ë¬á√ê¬∞
    * Latina
    * Latvie√Ö¬°u
    * L√É¬´tzebuergesch
    * √ê¬õ√ê¬µ√ê¬∑√ê¬≥√ê¬∏
    * Lietuvi√Ö¬≥
    * Ligure
    * Limburgs
    * Lingua_Franca_Nova
    * Luganda
    * Lumbaart
    * Magyar
    * √ê¬ú√ê¬∞√ê¬∫√ê¬µ√ê¬¥√ê¬æ√ê¬Ω√ë¬Å√ê¬∫√ê¬∏
    * √†¬¥¬Æ√†¬¥¬≤√†¬¥¬Ø√†¬¥¬æ√†¬¥¬≥√†¬¥¬Ç
    * √†¬§¬Æ√†¬§¬∞√†¬§¬æ√†¬§¬†√†¬•¬Ä
    * Bahasa_Melayu
    * M√É¬¨ng-d√Ñ¬ï√å¬§ng-ng√°¬π¬≥√å¬Ñ
    * √ê¬ú√ê¬æ√ê¬Ω√ê¬≥√ê¬æ√ê¬ª
    * √°¬Ä¬ô√°¬Ä¬º√°¬Ä¬î√°¬Ä¬∫√°¬Ä¬ô√°¬Ä¬¨√°¬Ä¬ò√°¬Ä¬¨√°¬Ä¬û√°¬Ä¬¨
    * N√Ñ¬Åhuatl
    * Nederlands
    * √†¬§¬®√†¬•¬á√†¬§¬™√†¬§¬æ√†¬§¬≤√†¬•¬Ä
    * √†¬§¬®√†¬•¬á√†¬§¬™√†¬§¬æ√†¬§¬≤_√†¬§¬≠√†¬§¬æ√†¬§¬∑√†¬§¬æ
    * √¶¬ó¬•√¶¬ú¬¨√®¬™¬û
    * Napulitano
    * Nordfriisk
    * Norsk
    * Norsk_nynorsk
    * Nouormand
    * Occitan
    * O√ä¬ªzbekcha/√ë¬û√ê¬∑√ê¬±√ê¬µ√ê¬∫√ë¬á√ê¬∞
    * √†¬®¬™√†¬©¬∞√†¬®¬ú√†¬®¬æ√†¬®¬¨√†¬©¬Ä
    * √ô¬æ√ô¬Ü√ò¬¨√ò¬ß√ò¬®√õ¬å
    * Papiamentu
    * √ô¬æ√ö¬ö√ò¬™√ô¬à
    * Patois
    * Plattd√É¬º√É¬ºtsch
    * Polski
    * Portugu√É¬™s
    * Qaraqalpaqsha
    * Rom√É¬¢n√Ñ¬É
    * Runa_Simi
    * √ê¬†√ë¬É√ë¬Å√ê¬∏√ê¬Ω√ë¬å√ë¬Å√ê¬∫√ë¬ã√ê¬π
    * √ê¬†√ë¬É√ë¬Å√ë¬Å√ê¬∫√ê¬∏√ê¬π
    * √ê¬°√ê¬∞√ë¬Ö√ê¬∞_√ë¬Ç√ë¬ã√ê¬ª√ê¬∞
    * √†¬§¬∏√†¬§¬Ç√†¬§¬∏√†¬•¬ç√†¬§¬ï√†¬•¬É√†¬§¬§√†¬§¬Æ√†¬•¬ç
    * Scots
    * Shqip
    * Sicilianu
    * √†¬∑¬É√†¬∑¬í√†¬∂¬Ç√†¬∑¬Ñ√†¬∂¬Ω
    * Simple_English
    * √ò¬≥√ô¬Ü√ö¬å√ô¬ä
    * Sloven√Ñ¬çina
    * Sloven√Ö¬°√Ñ¬çina
    * Soomaaliga
    * √ö¬©√ô¬à√ò¬±√ò¬Ø√õ¬å
    * √ê¬°√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬∏_/_srpski
    * Srpskohrvatski_/_√ë¬Å√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬æ√ë¬Ö√ë¬Ä√ê¬≤√ê¬∞√ë¬Ç√ë¬Å√ê¬∫√ê¬∏
    * Basa_Sunda
    * Suomi
    * Svenska
    * Tagalog
    * √†¬Æ¬§√†¬Æ¬Æ√†¬Æ¬ø√†¬Æ¬¥√†¬Ø¬ç
    * √ê¬¢√ê¬∞√ë¬Ç√ê¬∞√ë¬Ä√ë¬á√ê¬∞/tatar√É¬ßa
    * √†¬∞¬§√†¬±¬Ü√†¬∞¬≤√†¬±¬Å√†¬∞¬ó√†¬±¬Å
    * √†¬π¬Ñ√†¬∏¬ó√†¬∏¬¢
    * √ê¬¢√ê¬æ√í¬∑√ê¬∏√ê¬∫√ì¬£
    * √°¬è¬£√°¬é¬≥√°¬é¬©
    * T√É¬ºrk√É¬ße
    * √ê¬£√ê¬∫√ë¬Ä√ê¬∞√ë¬ó√ê¬Ω√ë¬Å√ë¬å√ê¬∫√ê¬∞
    * √ò¬ß√ò¬±√ò¬Ø√ô¬à
    * Vahcuengh
    * V√É¬®neto
    * Ti√°¬∫¬øng_Vi√°¬ª¬át
    * Volap√É¬ºk
    * V√É¬µro
    * Walon
    * Winaray
    * √•¬ê¬¥√®¬Ø¬≠
    * √ó¬ô√ó¬ô√ñ¬¥√ó¬ì√ó¬ô√ó¬©
    * Yor√É¬πb√É¬°
    * √ß¬≤¬µ√®¬™¬û
    * √Ö¬Ωemait√Ñ¬ó√Ö¬°ka
    * √§¬∏¬≠√¶¬ñ¬á
Edit_links
    * This page was last edited on 7 August 2019, at 04:21¬†(UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. Wikipedia√Ç¬Æ is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
