The following text has been accessed from https://en.wikipedia.org/wiki/Polymerase_chain_reaction at Fri Aug 9 03:35:17 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Polymerase chain reaction ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
"PCR" redirects here. For other uses, see PCR_(disambiguation).
method for amplifying DNA
 This article may be too technical for most readers to understand. Please help
 improve_it to make_it_understandable_to_non-experts, without removing the
 technical details. (October 2007)(Learn_how_and_when_to_remove_this_template
 message)
A strip of eight PCR tubes, each containing a 100 Î¼l reaction mixture
Polymerase chain reaction (PCR) is a method widely used in molecular_biology to
make many copies of a specific DNA segment. Using PCR, copies of DNA_sequences
are exponentially amplified to generate thousands to millions of more copies of
that particular DNA segment. PCR is now a common and often indispensable
technique used in medical_laboratory and clinical laboratory research for a
broad variety of applications including biomedical research and criminal
forensics.[1][2] PCR was developed by Kary_Mullis[3][4] in 1983 while he was an
employee of the Cetus Corporation. He was awarded the Nobel_Prize_in_Chemistry
in 1993 (along with Michael_Smith) for his work in developing the method.[5]
The vast majority of PCR methods rely on thermal_cycling. Thermal cycling
exposes reactants to repeated cycles of heating and cooling to permit different
temperature-dependent reactionsâspecifically, DNA_melting and enzyme-driven
DNA replication. PCR employs two main reagents â primers (which are short
single strand DNA fragments known as oligonucleotides that are a complementary
sequence to the target DNA region) and a DNA_polymerase. In the first step of
PCR, the two strands of the DNA double helix are physically separated at a high
temperature in a process called DNA_melting. In the second step, the
temperature is lowered and the primers bind to the complementary sequences of
DNA. The two DNA strands then become templates for DNA polymerase to
enzymatically assemble a new DNA strand from free nucleotides, the building
blocks of DNA. As PCR progresses, the DNA generated is itself used as a
template for replication, setting in motion a chain_reaction in which the
original DNA template is exponentially amplified.
Almost all PCR applications employ a heat-stable DNA polymerase, such as Taq
polymerase, an enzyme originally isolated from the thermophilic bacterium
Thermus_aquaticus. If the polymerase used was heat-susceptible, it would
denature under the high temperatures of the denaturation step. Before the use
of Taq polymerase, DNA polymerase had to be manually added every cycle, which
was a tedious and costly process.[6]
Applications of the technique include DNA_cloning for sequencing, gene cloning
and manipulation, gene mutagenesis; construction of DNA-based phylogenies, or
functional analysis of genes; diagnosis and monitoring of hereditary_diseases;
amplification of ancient DNA;[7] analysis of genetic fingerprints for DNA
profiling (for example, in forensic_science and parentage_testing); and
detection of pathogens in nucleic_acid_tests for the diagnosis of infectious
diseases.
Placing a strip of eight PCR tubes into a thermal_cycler
⁰
***** Contents *****
    * 1_Principles
          o 1.1_Procedure
          o 1.2_Stages
    * 2_Optimization
    * 3_Applications
          o 3.1_Selective_DNA_isolation
          o 3.2_Amplification_and_quantification_of_DNA
          o 3.3_Medical_and_diagnostic_applications
          o 3.4_Infectious_disease_applications
          o 3.5_Forensic_applications
          o 3.6_Research_applications
    * 4_Advantages
    * 5_Limitations
    * 6_Variations
    * 7_History
          o 7.1_Patent_disputes
    * 8_See_also
    * 9_References
    * 10_External_links
***** Principles[edit] *****
A thermal_cycler for PCR
An older, three-temperature thermal_cycler for PCR
PCR amplifies a specific region of a DNA strand (the DNA target). Most PCR
methods amplify DNA fragments of between 0.1 and 10 kilo_base_pairs (kbp) in
length, although some techniques allow for amplification of fragments up to 40
kbp.[8] The amount of amplified product is determined by the available
substrates in the reaction, which become limiting as the reaction progresses.
[9]
A basic PCR set-up requires several components and reagents,[10] including a
DNA template that contains the DNA target region to amplify; a DNA_polymerase;
an enzyme that polymerizes new DNA strands; heat-resistant Taq_polymerase is
especially common,[11] as it is more likely to remain intact during the high-
temperature DNA denaturation process; two DNA primers that are complementary to
the 3' (three prime) ends of each of the sense_and_anti-sense strands of the
DNA target (DNA polymerase can only bind to and elongate from a double-stranded
region of DNA; without primers there is no double-stranded initiation site at
which the polymerase can bind);[12] specific primers that are complementary to
the DNA target region are selected beforehand, and are often custom-made in a
laboratory or purchased from commercial biochemical suppliers; deoxynucleoside
triphosphates, or dNTPs (sometimes called "deoxynucleotide triphosphates";
nucleotides containing triphosphate groups), the building blocks from which the
DNA polymerase synthesizes a new DNA strand; a buffer_solution providing a
suitable chemical environment for optimum activity and stability of the DNA
polymerase; bivalent cations, typically magnesium (Mg) or manganese (Mn) ions;
Mg2+ is the most common, but Mn2+ can be used for PCR-mediated_DNA_mutagenesis,
as a higher Mn2+ concentration increases the error rate during DNA synthesis;
[13] and monovalent cations, typically potassium (K) ions
The reaction is commonly carried out in a volume of 10â200 Î¼L in small
reaction tubes (0.2â0.5 mL volumes) in a thermal_cycler. The thermal cycler
heats and cools the reaction tubes to achieve the temperatures required at each
step of the reaction (see below). Many modern thermal cyclers make use of the
Peltier_effect, which permits both heating and cooling of the block holding the
PCR tubes simply by reversing the electric current. Thin-walled reaction tubes
permit favorable thermal conductivity to allow for rapid thermal equilibration.
Most thermal cyclers have heated lids to prevent condensation at the top of the
reaction tube. Older thermal cyclers lacking a heated lid require a layer of
oil on top of the reaction mixture or a ball of wax inside the tube.
**** Procedure[edit] ****
Typically, PCR consists of a series of 20â40 repeated temperature changes,
called thermal cycles, with each cycle commonly consisting of two or three
discrete temperature steps (see figure below). The cycling is often preceded by
a single temperature step at a very high temperature (>90 Â°C (194 Â°F)), and
followed by one hold at the end for final product extension or brief storage.
The temperatures used and the length of time they are applied in each cycle
depend on a variety of parameters, including the enzyme used for DNA synthesis,
the concentration of bivalent ions and dNTPs in the reaction, and the melting
temperature (Tm) of the primers.[14] The individual steps common to most PCR
methods are as follows:
    * Initialization: This step is only required for DNA polymerases that
      require heat activation by hot-start_PCR.[15] It consists of heating the
      reaction chamber to a temperature of 94â96 Â°C (201â205 Â°F), or
      98 Â°C (208 Â°F) if extremely thermostable polymerases are used, which is
      then held for 1â10 minutes.
    * Denaturation: This step is the first regular cycling event and consists
      of heating the reaction chamber to 94â98 Â°C (201â208 Â°F) for
      20â30 seconds. This causes DNA_melting, or denaturation, of the double-
      stranded DNA template by breaking the hydrogen bonds between
      complementary bases, yielding two single-stranded DNA molecules.
    * Annealing: In the next step, the reaction temperature is lowered to
      50â65 Â°C (122â149 Â°F) for 20â40 seconds, allowing annealing of
      the primers to each of the single-stranded DNA templates. Two different
      primers are typically included in the reaction mixture: one for each of
      the two single-stranded complements containing the target region. The
      primers are single-stranded sequences themselves, but are much shorter
      than the length of the target region, complementing only very short
      sequences at the 3' end of each strand.
      It is critical to determine a proper temperature for the annealing step
      because efficiency and specificity are strongly affected by the annealing
      temperature. This temperature must be low enough to allow for
      hybridization of the primer to the strand, but high enough for the
      hybridization to be specific, i.e., the primer should bind only to a
      perfectly complementary part of the strand, and nowhere else. If the
      temperature is too low, the primer may bind imperfectly. If it is too
      high, the primer may not bind at all. A typical annealing temperature is
      about 3â5 Â°C below the Tm of the primers used. Stable hydrogen bonds
      between complementary bases are formed only when the primer sequence very
      closely matches the template sequence. During this step, the polymerase
      binds to the primer-template hybrid and begins DNA formation.
    * Extension/elongation: The temperature at this step depends on the DNA
      polymerase used; the optimum activity temperature for the thermostable
      DNA polymerase of Taq_(Thermus_aquaticus)_polymerase is approximately
      75â80 Â°C (167â176 Â°F),[16][17] though a temperature of 72 Â°C
      (162 Â°F) is commonly used with this enzyme. In this step, the DNA
      polymerase synthesizes a new DNA strand complementary to the DNA template
      strand by adding free dNTPs from the reaction mixture that are
      complementary to the template in the 5'-to-3' direction, condensing the
      5'-phosphate_group of the dNTPs with the 3'-hydroxy_group at the end of
      the nascent (elongating) DNA strand. The precise time required for
      elongation depends both on the DNA polymerase used and on the length of
      the DNA target region to amplify. As a rule of thumb, at their optimal
      temperature, most DNA polymerases polymerize a thousand bases per minute.
      Under optimal conditions (i.e., if there are no limitations due to
      limiting substrates or reagents), at each extension/elongation step, the
      number of DNA target sequences is doubled. With each successive cycle,
      the original template strands plus all newly generated strands become
      template strands for the next round of elongation, leading to exponential
      (geometric) amplification of the specific DNA target region.
      The processes of denaturation, annealing and elongation constitute a
      single cycle. Multiple cycles are required to amplify the DNA target to
      millions of copies. The formula used to calculate the number of DNA
      copies formed after a given number of cycles is 2n, where n is the number
      of cycles. Thus, a reaction set for 30 cycles results in 230, or
      1073741824, copies of the original double-stranded DNA target region.
    * Final elongation: This single step is optional, but is performed at a
      temperature of 70â74 Â°C (158â165 Â°F) (the temperature range
      required for optimal activity of most polymerases used in PCR) for 5â15
      minutes after the last PCR cycle to ensure that any remaining single-
      stranded DNA is fully elongated.
    * Final hold: The final step cools the reaction chamber to 4â15 Â°C
      (39â59 Â°F) for an indefinite time, and may be employed for short-term
      storage of the PCR products.
[Schematic_drawing_of_a_complete_PCR_cycle]
Ethidium_bromide-stained PCR products after gel_electrophoresis. Two sets of
primers were used to amplify a target sequence from three different tissue
samples. No amplification is present in sample #1; DNA bands in sample #2 and
#3 indicate successful amplification of the target sequence. The gel also shows
a positive control, and a DNA ladder containing DNA fragments of defined length
for sizing the bands in the experimental PCRs.
To check whether the PCR successfully generated the anticipated DNA target
region (also sometimes referred to as the amplimer or amplicon), agarose_gel
electrophoresis may be employed for size separation of the PCR products. The
size(s) of PCR products is determined by comparison with a DNA_ladder, a
molecular weight marker which contains DNA fragments of known size run on the
gel alongside the PCR products.
[Tucker_PCR]
**** Stages[edit] ****
As with other chemical reactions, the reaction rate and efficiency of PCR are
affected by limiting factors. Thus, the entire PCR process can further be
divided into three stages based on reaction progress:
    * Exponential amplification: At every cycle, the amount of product is
      doubled (assuming 100% reaction efficiency). After 30 cycles, a single
      copy of DNA can be increased up to 1,000,000,000 (one billion) copies. In
      a sense, then, the replication of a discrete strand of DNA is being
      manipulated in a tube under controlled conditions.[18] The reaction is
      very sensitive: only minute quantities of DNA must be present.
    * Leveling off stage: The reaction slows as the DNA polymerase loses
      activity and as consumption of reagents, such as dNTPs and primers,
      causes them to become more limited.
    * Plateau: No more product accumulates due to exhaustion of reagents and
      enzyme.
***** Optimization[edit] *****
Main article: PCR_optimization
In practice, PCR can fail for various reasons, in part due to its sensitivity
to contamination causing amplification of spurious DNA products. Because of
this, a number of techniques and procedures have been developed for optimizing
PCR conditions.[19][20] Contamination with extraneous DNA is addressed with lab
protocols and procedures that separate pre-PCR mixtures from potential DNA
contaminants.[10] This usually involves spatial separation of PCR-setup areas
from areas for analysis or purification of PCR products, use of disposable
plasticware, and thoroughly cleaning the work surface between reaction setups.
Primer-design techniques are important in improving PCR product yield and in
avoiding the formation of spurious products, and the usage of alternate buffer
components or polymerase enzymes can help with amplification of long or
otherwise problematic regions of DNA. Addition of reagents, such as formamide,
in buffer systems may increase the specificity and yield of PCR.[21] Computer
simulations of theoretical PCR results (Electronic_PCR) may be performed to
assist in primer design.[22]
***** Applications[edit] *****
**** Selective DNA isolation[edit] ****
PCR allows isolation of DNA fragments from genomic DNA by selective
amplification of a specific region of DNA. This use of PCR augments many ways,
such as generating hybridization_probes for Southern or northern hybridization
and DNA_cloning, which require larger amounts of DNA, representing a specific
DNA region. PCR supplies these techniques with high amounts of pure DNA,
enabling analysis of DNA samples even from very small amounts of starting
material.
Other applications of PCR include DNA_sequencing to determine unknown PCR-
amplified sequences in which one of the amplification primers may be used in
Sanger_sequencing, isolation of a DNA sequence to expedite recombinant DNA
technologies involving the insertion of a DNA sequence into a plasmid, phage,
or cosmid (depending on size) or the genetic material of another organism.
Bacterial colonies (such as E._coli) can be rapidly screened by PCR for correct
DNA vector constructs.[23] PCR may also be used for genetic_fingerprinting; a
forensic technique used to identify a person or organism by comparing
experimental DNAs through different PCR-based methods.
Some PCR 'fingerprints' methods have high discriminative power and can be used
to identify genetic relationships between individuals, such as parent-child or
between siblings, and are used in paternity testing (Fig. 4). This technique
may also be used to determine evolutionary relationships among organisms when
certain molecular clocks are used (i.e., the 16S_rRNA and recA genes of
microorganisms).[24]
Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3)
Mother. The child has inherited some, but not all of the fingerprint of each of
its parents, giving it a new, unique fingerprint.
**** Amplification and quantification of DNA[edit] ****
See also: Use_of_DNA_in_forensic_entomology
Because PCR amplifies the regions of DNA that it targets, PCR can be used to
analyze extremely small amounts of sample. This is often critical for forensic
analysis, when only a trace amount of DNA is available as evidence. PCR may
also be used in the analysis of ancient_DNA that is tens of thousands of years
old. These PCR-based techniques have been successfully used on animals, such as
a forty-thousand-year-old mammoth, and also on human DNA, in applications
ranging from the analysis of Egyptian mummies to the identification of a
Russian tsar and the body of English king Richard_III.[25]
Quantitative_PCR or Real Time PCR (qPCR[26], not to be confused with RT-PCR)
methods allow the estimation of the amount of a given sequence present in a
sampleâa technique often applied to quantitatively determine levels of gene
expression. Quantitative PCR is an established tool for DNA quantification that
measures the accumulation of DNA product after each round of PCR amplification.
qPCR allows the quantification and detection of a specific DNA sequence in real
time since it measures concentration while the synthesis process is taking
place. There are two methods for simultaneous detection and quantification. The
first method consists of using fluorescent dyes that are retained
nonspecifically in between the double strands. The second method involves
probes that code for specific sequences and are fluorescently labeled.
Detection of DNA using these methods can only be seen after the hybridization
of probes with its complementary DNA takes place. An interesting technique
combination is real-time PCR and reverse transcription. This sophisticated
technique, called RT-qPCR, allows for the quantification of a small quantity of
RNA. Through this combined technique, mRNA is converted to cDNA, which is
further quantified using qPCR. This technique lowers the possibility of error
at the end point of PCR,[27] increasing chances for detection of genes
associated with genetic diseases such as cancer.[7] Laboratories use RT-qPCR
for the purpose of sensitively measuring gene regulation.
**** Medical and diagnostic applications[edit] ****
Prospective parents can be tested for being genetic_carriers, or their children
might be tested for actually being affected by a disease.[1] DNA samples for
prenatal_testing can be obtained by amniocentesis, chorionic_villus_sampling,
or even by the analysis of rare fetal cells circulating in the mother's
bloodstream. PCR analysis is also essential to preimplantation_genetic
diagnosis, where individual cells of a developing embryo are tested for
mutations.
    * PCR can also be used as part of a sensitive test for tissue_typing, vital
      to organ_transplantation. As of 2008, there is even a proposal to replace
      the traditional antibody-based tests for blood_type with PCR-based tests.
      [28]
    * Many forms of cancer involve alterations to oncogenes. By using PCR-based
      tests to study these mutations, therapy regimens can sometimes be
      individually customized to a patient. PCR permits early diagnosis of
      malignant diseases such as leukemia and lymphomas, which is currently the
      highest-developed in cancer research and is already being used routinely.
      PCR assays can be performed directly on genomic DNA samples to detect
      translocation-specific malignant cells at a sensitivity that is at least
      10,000 fold higher than that of other methods.[29] PCR is very useful in
      the medical field since it allows for the isolation and amplification of
      tumor suppressors. Quantitative PCR for example, can be used to quantify
      and analyze single cells, as well as recognize DNA, mRNA and protein
      confirmations and combinations.[27]
**** Infectious disease applications[edit] ****
PCR allows for rapid and highly specific diagnosis of infectious diseases,
including those caused by bacteria or viruses.[30] PCR also permits
identification of non-cultivatable or slow-growing microorganisms such as
mycobacteria, anaerobic_bacteria, or viruses from tissue_culture assays and
animal_models. The basis for PCR diagnostic applications in microbiology is the
detection of infectious agents and the discrimination of non-pathogenic from
pathogenic strains by virtue of specific genes.[30][31]
Characterization and detection of infectious disease organisms have been
revolutionized by PCR in the following ways:
    * The human immunodeficiency virus (or HIV), is a difficult target to find
      and eradicate. The earliest tests for infection relied on the presence of
      antibodies to the virus circulating in the bloodstream. However,
      antibodies don't appear until many weeks after infection, maternal
      antibodies mask the infection of a newborn, and therapeutic agents to
      fight the infection don't affect the antibodies. PCR tests have been
      developed that can detect as little as one viral genome among the DNA of
      over 50,000 host cells.[32] Infections can be detected earlier, donated
      blood can be screened directly for the virus, newborns can be immediately
      tested for infection, and the effects of antiviral treatments can be
      quantified.
    * Some disease organisms, such as that for tuberculosis, are difficult to
      sample from patients and slow to be grown in the laboratory. PCR-based
      tests have allowed detection of small numbers of disease organisms (both
      live or dead), in convenient samples. Detailed genetic analysis can also
      be used to detect antibiotic resistance, allowing immediate and effective
      therapy. The effects of therapy can also be immediately evaluated.
    * The spread of a disease organism through populations of domestic or wild
      animals can be monitored by PCR testing. In many cases, the appearance of
      new virulent sub-types can be detected and monitored. The sub-types of an
      organism that were responsible for earlier_epidemics can also be
      determined by PCR analysis.
    * Viral DNA can be detected by PCR. The primers used must be specific to
      the targeted sequences in the DNA of a virus, and PCR can be used for
      diagnostic analyses or DNA sequencing of the viral genome. The high
      sensitivity of PCR permits virus detection soon after infection and even
      before the onset of disease.[30] Such early detection may give physicians
      a significant lead time in treatment. The amount of virus ("viral_load")
      in a patient can also be quantified by PCR-based DNA quantitation
      techniques (see below).
    * Diseases such as pertussis (or whooping cough) are cause by the bacteria
      Bordetella pertussis. This bacteria is marked by a serious acute
      respiratory infection that affects various animals and humans and has led
      to the deaths of many young children. The pertussis toxin is a protein
      exotoxin that binds to cell receptors by two dimers and reacts with
      different cell types such as T lymphocytes which plays a role in cell
      immunity.[33] PCR is an important testing tool that can detect the
      sequences that are within the pertussis toxin gene. This is because PCR
      has a high sensitivity for the toxin and has demonstrated a rapid
      turnaround time. PCR is very efficient for diagnosing pertussis when
      compared to culture.[34]
**** Forensic applications[edit] ****
The development of PCR-based genetic (or DNA) fingerprinting protocols has seen
widespread application in forensics:
    * In its most discriminating form, genetic_fingerprinting can uniquely
      discriminate any one person from the entire population of the world.
      Minute samples of DNA can be isolated from a crime_scene, and compared to
      that from suspects, or from a DNA_database of earlier evidence or
      convicts. Simpler versions of these tests are often used to rapidly rule
      out suspects during a criminal investigation. Evidence from decades-old
      crimes can be tested, confirming or exonerating the people originally
      convicted.
    * Forensic DNA typing has been an effective way of identifying or
      exonerating criminal suspects due to analysis of evidence discovered at a
      crime scene. The human genome has many repetitive regions that can be
      found within gene sequences or in non-coding regions of the genome.
      Specifically, up to 40% of human DNA is repetitive.[7] There are two
      distinct categories for these repetitive, non-coding regions in the
      genome. The first category is called variable number tandem repeats
      (VNTR), which are 10â100 base pairs long and the second category is
      called short tandem repeats (STR) and these consist of repeated 2â10
      base pair sections. PCR is used to amplify several well-known VNTRs and
      STRs using primers that flank each of the repetitive regions. The sizes
      of the fragments obtained from any individual for each of the STRs will
      indicate which alleles are present. By analyzing several STRs for an
      individual, a set of alleles for each person will be found that
      statistically is likely to be unique.[7] Researchers have identified the
      complete sequence of the human genome. This sequence can be easily
      accessed through the NCBI website and is used in many real-life
      applications. For example, the FBI has compiled a set of DNA marker sites
      used for identification, and these are called the Combined DNA Index
      System (CODIS) DNA database.[7] Using this database enables statistical
      analysis to be used to determine the probability that a DNA sample will
      match. PCR is a very powerful and significant analytical tool to use for
      forensic DNA typing because researchers only need a very small amount of
      the target DNA to be used for analysis. For example, a single human hair
      with attached hair follicle has enough DNA to conduct the analysis.
      Similarly, a few sperm, skin samples from under the fingernails, or a
      small amount of blood can provide enough DNA for conclusive analysis.[7]
    * Less discriminating forms of DNA_fingerprinting can help in DNA_paternity
      testing, where an individual is matched with their close relatives. DNA
      from unidentified human remains can be tested, and compared with that
      from possible parents, siblings, or children. Similar testing can be used
      to confirm the biological parents of an adopted (or kidnapped) child. The
      actual biological father of a newborn can also be confirmed (or ruled
      out).
    * The PCR AMGX/AMGY design has been shown to not only facilitating in
      amplifying DNA sequences from a very minuscule amount of genome. However
      it can also be used for real time sex determination from forensic bone
      samples. This provides us with a powerful and effective way to determine
      the sex of not only ancient specimens but also current suspects in
      crimes.[35]
**** Research applications[edit] ****
PCR has been applied to many areas of research in molecular genetics:
    * PCR allows rapid production of short pieces of DNA, even when not more
      than the sequence of the two primers is known. This ability of PCR
      augments many methods, such as generating hybridization probes for
      Southern or northern_blot hybridization. PCR supplies these techniques
      with large amounts of pure DNA, sometimes as a single strand, enabling
      analysis even from very small amounts of starting material.
    * The task of DNA_sequencing can also be assisted by PCR. Known segments of
      DNA can easily be produced from a patient with a genetic disease
      mutation. Modifications to the amplification technique can extract
      segments from a completely unknown genome, or can generate just a single
      strand of an area of interest.
    * PCR has numerous applications to the more traditional process of DNA
      cloning. It can extract segments for insertion into a vector from a
      larger genome, which may be only available in small quantities. Using a
      single set of 'vector primers', it can also analyze or extract fragments
      that have already been inserted into vectors. Some alterations to the PCR
      protocol can generate mutations (general or site-directed) of an inserted
      fragment.
    * Sequence-tagged_sites is a process where PCR is used as an indicator that
      a particular segment of a genome is present in a particular clone. The
      Human_Genome_Project found this application vital to mapping the cosmid
      clones they were sequencing, and to coordinating the results from
      different laboratories.
    * An exciting application of PCR is the phylogenic analysis of DNA from
      ancient_sources, such as that found in the recovered bones of
      Neanderthals, from frozen tissues of mammoths, or from the brain of
      Egyptian mummies. Have been amplified and sequenced.[18] In some cases
      the highly degraded DNA from these sources might be reassembled during
      the early stages of amplification.
    * A common application of PCR is the study of patterns of gene_expression.
      Tissues (or even individual cells) can be analyzed at different stages to
      see which genes have become active, or which have been switched off. This
      application can also use quantitative_PCR to quantitate the actual levels
      of expression
    * The ability of PCR to simultaneously amplify several loci from individual
      sperm[36] has greatly enhanced the more traditional task of genetic
      mapping by studying chromosomal_crossovers after meiosis. Rare crossover
      events between very close loci have been directly observed by analyzing
      thousands of individual sperms. Similarly, unusual deletions, insertions,
      translocations, or inversions can be analyzed, all without having to wait
      (or pay) for the long and laborious processes of fertilization,
      embryogenesis, etc.
    * Site-directed_mutagenesis: PCR can be used to create mutant genes with
      mutations chosen by scientists at will. These mutations can be chosen in
      order to understand how proteins accomplish their functions, and to
      change or improve protein function.
***** Advantages[edit] *****
PCR has a number of advantages. It is fairly simple to understand and to use,
and produces results rapidly. The technique is highly sensitive with the
potential to produce millions to billions of copies of a specific product for
sequencing, cloning, and analysis. qRT-PCR shares the same advantages as the
PCR, with an added advantage of quantification of the synthesized product.
Therefore, it has its uses to analyze alterations of gene expression levels in
tumors, microbes, or other disease states.[37]
PCR is a very powerful and practical research tool. The sequencing of unknown
etiologies of many diseases are being figured out by the PCR. The technique can
help identify the sequence of previously unknown viruses related to those
already known and thus give us a better understanding of the disease itself. If
the procedure can be further simplified and sensitive non radiometric detection
systems can be developed, the PCR will assume a prominent place in the clinical
laboratory for years to come.[18]
***** Limitations[edit] *****
One major limitation of PCR is that prior information about the target sequence
is necessary in order to generate the primers that will allow its selective
amplification.[38] This means that, typically, PCR users must know the precise
sequence(s) upstream of the target region on each of the two single-stranded
templates in order to ensure that the DNA polymerase properly binds to the
primer-template hybrids and subsequently generates the entire target region
during DNA synthesis.
Like all enzymes, DNA polymerases are also prone to error, which in turn causes
mutations in the PCR fragments that are generated.[39]
Another limitation of PCR is that even the smallest amount of contaminating DNA
can be amplified, resulting in misleading or ambiguous results. To minimize the
chance of contamination, investigators should reserve separate rooms for
reagent preparation, the PCR, and analysis of product. Reagents should be
dispensed into single-use aliquots. Pipetters with disposable plungers and
extra-long pipette tips should be routinely used.[18]
***** Variations[edit] *****
Main article: Variants_of_PCR
    * Allele-specific PCR: a diagnostic or cloning technique based on single-
      nucleotide variations (SNVs not to be confused with SNPs) (single-base
      differences in a patient). It requires prior knowledge of a DNA sequence,
      including differences between alleles, and uses primers whose 3' ends
      encompass the SNV (base pair buffer around SNV usually incorporated). PCR
      amplification under stringent conditions is much less efficient in the
      presence of a mismatch between template and primer, so successful
      amplification with an SNP-specific primer signals presence of the
      specific SNP in a sequence.[40] See SNP_genotyping for more information.
    * Assembly_PCR or Polymerase Cycling Assembly (PCA): artificial synthesis
      of long DNA sequences by performing PCR on a pool of long
      oligonucleotides with short overlapping segments. The oligonucleotides
      alternate between sense and antisense directions, and the overlapping
      segments determine the order of the PCR fragments, thereby selectively
      producing the final long DNA product.[41]
    * Asymmetric_PCR: preferentially amplifies one DNA strand in a double-
      stranded DNA template. It is used in sequencing and hybridization probing
      where amplification of only one of the two complementary strands is
      required. PCR is carried out as usual, but with a great excess of the
      primer for the strand targeted for amplification. Because of the slow
      (arithmetic) amplification later in the reaction after the limiting
      primer has been used up, extra cycles of PCR are required.[42] A recent
      modification on this process, known as Linear-After-The-Exponential-PCR
      (LATE-PCR), uses a limiting primer with a higher melting temperature (Tm)
      than the excess primer to maintain reaction efficiency as the limiting
      primer concentration decreases mid-reaction.[43]
    * Convective PCR: a pseudo-isothermal way of performing PCR. Instead of
      repeatedly heating and cooling the PCR mixture, the solution is subjected
      to a thermal gradient. The resulting thermal instability driven
      convective flow automatically shuffles the PCR reagents from the hot and
      cold regions repeatedly enabling PCR.[44] Parameters such as thermal
      boundary conditions and geometry of the PCR enclosure can be optimized to
      yield robust and rapid PCR by harnessing the emergence of chaotic flow
      fields.[45] Such convective flow PCR setup significantly reduces device
      power requirement and operation time.
    * Dial-out PCR: a highly parallel method for retrieving accurate DNA
      molecules for gene synthesis. A complex library of DNA molecules is
      modified with unique flanking tags before massively parallel sequencing.
      Tag-directed primers then enable the retrieval of molecules with desired
      sequences by PCR.[46]
    * Digital_PCR (dPCR): used to measure the quantity of a target DNA sequence
      in a DNA sample. The DNA sample is highly diluted so that after running
      many PCRs in parallel, some of them do not receive a single molecule of
      the target DNA. The target DNA concentration is calculated using the
      proportion of negative outcomes. Hence the name 'digital PCR'.
    * Helicase-dependent_amplification: similar to traditional PCR, but uses a
      constant temperature rather than cycling through denaturation and
      annealing/extension cycles. DNA_helicase, an enzyme that unwinds DNA, is
      used in place of thermal denaturation.[47]
    * Hot_start_PCR: a technique that reduces non-specific amplification during
      the initial set up stages of the PCR. It may be performed manually by
      heating the reaction components to the denaturation temperature (e.g.,
      95 Â°C) before adding the polymerase.[48] Specialized enzyme systems have
      been developed that inhibit the polymerase's activity at ambient
      temperature, either by the binding of an antibody[15][49] or by the
      presence of covalently bound inhibitors that dissociate only after a
      high-temperature activation step. Hot-start/cold-finish PCR is achieved
      with new hybrid polymerases that are inactive at ambient temperature and
      are instantly activated at elongation temperature.
    * In_silico_PCR (digital PCR, virtual PCR, electronic PCR, e-PCR) refers to
      computational tools used to calculate theoretical polymerase chain
      reaction results using a given set of primers (probes) to amplify DNA
      sequences from a sequenced genome or transcriptome. In silico PCR was
      proposed as an educational tool for molecular biology.[50]
    * Intersequence-specific PCR (ISSR): a PCR method for DNA fingerprinting
      that amplifies regions between simple sequence repeats to produce a
      unique fingerprint of amplified fragment lengths.[51]
    * Inverse_PCR: is commonly used to identify the flanking sequences around
      genomic inserts. It involves a series of DNA_digestions and self
      ligation, resulting in known sequences at either end of the unknown
      sequence.[52]
    * Ligation-mediated PCR: uses small DNA linkers ligated to the DNA of
      interest and multiple primers annealing to the DNA linkers; it has been
      used for DNA_sequencing, genome_walking, and DNA_footprinting.[53]
    * Methylation-specific_PCR (MSP): developed by Stephen_Baylin and James_G.
      Herman at the Johns Hopkins School of Medicine,[54] and is used to detect
      methylation of CpG islands in genomic DNA. DNA is first treated with
      sodium bisulfite, which converts unmethylated cytosine bases to uracil,
      which is recognized by PCR primers as thymine. Two PCRs are then carried
      out on the modified DNA, using primer sets identical except at any CpG
      islands within the primer sequences. At these points, one primer set
      recognizes DNA with cytosines to amplify methylated DNA, and one set
      recognizes DNA with uracil or thymine to amplify unmethylated DNA. MSP
      using qPCR can also be performed to obtain quantitative rather than
      qualitative information about methylation.
    * Miniprimer PCR: uses a thermostable polymerase (S-Tbr) that can extend
      from short primers ("smalligos") as short as 9 or 10 nucleotides. This
      method permits PCR targeting to smaller primer binding regions, and is
      used to amplify conserved DNA sequences, such as the 16S (or eukaryotic
      18S) rRNA gene.[55]
    * Multiplex_ligation-dependent_probe_amplification (MLPA): permits
      amplifying multiple targets with a single primer pair, thus avoiding the
      resolution limitations of multiplex PCR (see below).
    * Multiplex-PCR: consists of multiple primer sets within a single PCR
      mixture to produce amplicons of varying sizes that are specific to
      different DNA sequences. By targeting multiple genes at once, additional
      information may be gained from a single test-run that otherwise would
      require several times the reagents and more time to perform. Annealing
      temperatures for each of the primer sets must be optimized to work
      correctly within a single reaction, and amplicon sizes. That is, their
      base pair length should be different enough to form distinct bands when
      visualized by gel_electrophoresis.
    * Nanoparticle-Assisted PCR (nanoPCR): some nanoparticles (NPs) can enhance
      the efficiency of PCR (thus being called nanoPCR), and some can even
      outperform the original PCR enhancers. It was reported that quantum dots
      (QDs) can improve PCR specificity and efficiency. Single-walled carbon
      nanotubes (SWCNTs) and multi-walled carbon nanotubes (MWCNTs) are
      efficient in enhancing the amplification of long PCR. Carbon nanopowder
      (CNP) can improve the efficiency of repeated PCR and long PCR, while zinc
      oxide, titanium_dioxide and Ag NPs were found to increase the PCR yield.
      Previous data indicated that non-metallic NPs retained acceptable
      amplification fidelity. Given that many NPs are capable of enhancing PCR
      efficiency, it is clear that there is likely to be great potential for
      nanoPCR technology improvements and product development.[56][57]
    * Nested_PCR: increases the specificity of DNA amplification, by reducing
      background due to non-specific amplification of DNA. Two sets of primers
      are used in two successive PCRs. In the first reaction, one pair of
      primers is used to generate DNA products, which besides the intended
      target, may still consist of non-specifically amplified DNA fragments.
      The product(s) are then used in a second PCR with a set of primers whose
      binding sites are completely or partially different from and located 3'
      of each of the primers used in the first reaction. Nested PCR is often
      more successful in specifically amplifying long DNA fragments than
      conventional PCR, but it requires more detailed knowledge of the target
      sequences.
    * Overlap-extension_PCR or Splicing by overlap extension (SOEing): a
      genetic_engineering technique that is used to splice together two or more
      DNA fragments that contain complementary sequences. It is used to join
      DNA pieces containing genes, regulatory sequences, or mutations; the
      technique enables creation of specific and long DNA constructs. It can
      also introduce deletions, insertions or point mutations into a DNA
      sequence.[58][59]
    * PAN-AC: uses isothermal conditions for amplification, and may be used in
      living cells.[60][61]
    * quantitative_PCR (qPCR): used to measure the quantity of a target
      sequence (commonly in real-time). It quantitatively measures starting
      amounts of DNA, cDNA, or RNA. quantitative_PCR is commonly used to
      determine whether a DNA sequence is present in a sample and the number of
      its copies in the sample. Quantitative PCR has a very high degree of
      precision. Quantitative PCR methods use fluorescent dyes, such as Sybr
      Green, EvaGreen or fluorophore-containing DNA probes, such as TaqMan, to
      measure the amount of amplified product in real time. It is also
      sometimes abbreviated to RT-PCR (real-time PCR) but this abbreviation
      should be used only for reverse_transcription_PCR. qPCR is the
      appropriate contractions for quantitative_PCR (real-time PCR).
    * Reverse Transcription PCR (RT-PCR): for amplifying DNA from RNA. Reverse
      transcriptase reverse transcribes RNA into cDNA, which is then amplified
      by PCR. RT-PCR is widely used in expression_profiling, to determine the
      expression of a gene or to identify the sequence of an RNA transcript,
      including transcription start and termination sites. If the genomic DNA
      sequence of a gene is known, RT-PCR can be used to map the location of
      exons and introns in the gene. The 5' end of a gene (corresponding to the
      transcription start site) is typically identified by RACE-PCR (Rapid
      Amplification of cDNA Ends).
    * RNase_H-dependent_PCR (rhPCR): a modification of PCR that utilizes
      primers with a 3â extension block that can be removed by a thermostable
      RNase HII enzyme. This system reduces primer-dimers and allows for
      multiplexed reactions to be performed with higher numbers of primers.[62]
    * Single_Specific_Primer-PCR (SSP-PCR): allows the amplification of double-
      stranded DNA even when the sequence information is available at one end
      only. This method permits amplification of genes for which only a partial
      sequence information is available, and allows unidirectional genome
      walking from known into unknown regions of the chromosome.[63]
    * Solid Phase PCR: encompasses multiple meanings, including Polony
      Amplification (where PCR colonies are derived in a gel matrix, for
      example), Bridge PCR[64] (primers are covalently linked to a solid-
      support surface), conventional Solid Phase PCR (where Asymmetric PCR is
      applied in the presence of solid support bearing primer with sequence
      matching one of the aqueous primers) and Enhanced Solid Phase PCR[65]
      (where conventional Solid Phase PCR can be improved by employing high Tm
      and nested solid support primer with optional application of a thermal
      'step' to favour solid support priming).
    * Suicide PCR: typically used in paleogenetics or other studies where
      avoiding false positives and ensuring the specificity of the amplified
      fragment is the highest priority. It was originally described in a study
      to verify the presence of the microbe Yersinia_pestis in dental samples
      obtained from 14th Century graves of people supposedly killed by plague
      during the medieval Black_Death epidemic.[66] The method prescribes the
      use of any primer combination only once in a PCR (hence the term
      "suicide"), which should never have been used in any positive control PCR
      reaction, and the primers should always target a genomic region never
      amplified before in the lab using this or any other set of primers. This
      ensures that no contaminating DNA from previous PCR reactions is present
      in the lab, which could otherwise generate false positives.
    * Thermal asymmetric interlaced PCR (TAIL-PCR): for isolation of an unknown
      sequence flanking a known sequence. Within the known sequence, TAIL-PCR
      uses a nested pair of primers with differing annealing temperatures; a
      degenerate primer is used to amplify in the other direction from the
      unknown sequence.[67]
    * Touchdown_PCR (Step-down PCR): a variant of PCR that aims to reduce
      nonspecific background by gradually lowering the annealing temperature as
      PCR cycling progresses. The annealing temperature at the initial cycles
      is usually a few degrees (3â5 Â°C) above the Tm of the primers used,
      while at the later cycles, it is a few degrees (3â5 Â°C) below the
      primer Tm. The higher temperatures give greater specificity for primer
      binding, and the lower temperatures permit more efficient amplification
      from the specific products formed during the initial cycles.[68]
    * Universal Fast Walking: for genome walking and genetic fingerprinting
      using a more specific 'two-sided' PCR than conventional 'one-sided'
      approaches (using only one gene-specific primer and one general
      primerâwhich can lead to artefactual 'noise')[69] by virtue of a
      mechanism involving lariat structure formation. Streamlined derivatives
      of UFW are LaNe RAGE (lariat-dependent nested PCR for rapid amplification
      of genomic DNA ends),[70] 5'RACE LaNe[71] and 3'RACE LaNe.[72]
***** History[edit] *****
Diagrammatic representation of an example primer pair. The use of primers in an
in vitro assay to allow DNA synthesis was a major innovation that allowed the
development of PCR.
Main article: History_of_polymerase_chain_reaction
A 1971 paper in the Journal_of_Molecular_Biology by Kjell_Kleppe [no] and co-
workers in the laboratory of H._Gobind_Khorana first described a method of
using an enzymatic assay to replicate a short DNA template with primers in
vitro.[73] However, this early manifestation of the basic PCR principle did not
receive much attention at the time and the invention of the polymerase chain
reaction in 1983 is generally credited to Kary_Mullis.[74]
"Baby Blue", a 1986 prototype machine for doing PCR
When Mullis developed the PCR in 1983, he was working in Emeryville, California
for Cetus_Corporation, one of the first biotechnology companies, where he was
responsible for synthesizing short chains of DNA. Mullis has written that he
first conceived the idea for PCR while cruising along the Pacific_Coast_Highway
one night in his car.[75] He was playing in his mind with a new way of
analyzing changes (mutations) in DNA when he realized that he had instead
invented a method of amplifying any DNA region through repeated cycles of
duplication driven by DNA polymerase. In Scientific_American, Mullis summarized
the procedure: "Beginning with a single molecule of the genetic material DNA,
the PCR can generate 100 billion similar molecules in an afternoon. The
reaction is easy to execute. It requires no more than a test tube, a few simple
reagents, and a source of heat."[76] DNA fingerprinting was first used for
paternity_testing in 1988.[77]
Mullis was awarded the Nobel_Prize_in_Chemistry in 1993 for his invention,
seven years after he and his colleagues at Cetus first put his proposal to
practice.[5] Mullisâs 1985 paper with R. K. Saiki and H. A. Erlich,
âEnzymatic Amplification of Î²-globin Genomic Sequences and Restriction Site
Analysis for Diagnosis of Sickle Cell Anemiaââthe polymerase chain reaction
invention (PCR) â was honored by a Citation for Chemical Breakthrough Award
from the Division of History of Chemistry of the American Chemical Society in
2017.[78][79]
Some controversies have remained about the intellectual and practical
contributions of other scientists to Mullis' work, and whether he had been the
sole inventor of the PCR principle (see below).
At the core of the PCR method is the use of a suitable DNA_polymerase able to
withstand the high temperatures of >90 Â°C (194 Â°F) required for separation of
the two DNA strands in the DNA_double_helix after each replication cycle. The
DNA polymerases initially employed for in_vitro experiments presaging PCR were
unable to withstand these high temperatures.[1] So the early procedures for DNA
replication were very inefficient and time-consuming, and required large
amounts of DNA polymerase and continuous handling throughout the process.
The discovery in 1976 of Taq_polymeraseâa DNA polymerase purified from the
thermophilic_bacterium, Thermus_aquaticus, which naturally lives in hot (50 to
80 Â°C (122 to 176 Â°F)) environments[16] such as hot springsâpaved the way
for dramatic improvements of the PCR method. The DNA polymerase isolated from
T. aquaticus is stable at high temperatures remaining active even after DNA
denaturation,[17] thus obviating the need to add new DNA polymerase after each
cycle.[2] This allowed an automated thermocycler-based process for DNA
amplification.
**** Patent disputes[edit] ****
The PCR technique was patented by Kary_Mullis and assigned to Cetus
Corporation, where Mullis worked when he invented the technique in 1983. The
Taq polymerase enzyme was also covered by patents. There have been several
high-profile lawsuits related to the technique, including an unsuccessful
lawsuit brought by DuPont. The pharmaceutical company Hoffmann-La_Roche
purchased the rights to the patents in 1992 and currently holds those that are
still protected.
A related patent battle over the Taq polymerase enzyme is still ongoing in
several jurisdictions around the world between Roche and Promega. The legal
arguments have extended beyond the lives of the original PCR and Taq polymerase
patents, which expired on March 28, 2005.[80]
***** See also[edit] *****
    * Molecular_and_cellular_biology_portal
    * Loop-mediated_isothermal_amplification
    * DNA_spiking
***** References[edit] *****
   1. ^ a b cSaiki, R.; Scharf, S.; Faloona, F.; Mullis, K.; Horn, G.; Erlich,
      H.; Arnheim, N. (1985). "Enzymatic amplification of beta-globin genomic
      sequences and restriction site analysis for diagnosis of sickle cell
      anemia". Science. 230 (4732): 1350â1354. doi:10.1126/science.2999980.
      PMID 2999980.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a bSaiki, R.; Gelfand, D.; Stoffel, S.; Scharf, S.; Higuchi, R.; Horn,
      G.; Mullis, K.; Erlich, H. (1988). "Primer-directed enzymatic
      amplification of DNA with a thermostable DNA polymerase". Science. 239
      (4839): 487â491. Bibcode:1988Sci...239..487S. doi:10.1126/
      science.2448875. PMID 2448875.
   4. ^Bartlett, J. M. S.; Stirling, D. (2003). "A Short History of the
      Polymerase Chain Reaction". PCR Protocols. Methods in Molecular Biology.
      226 (2nd ed.). pp. 3â6. doi:10.1385/1-59259-384-4:3. ISBN 978-1-59259-
      384-2. PMID 12958470.
   5. ^ Mullis, Kary B. et al. (1986) "Process for amplifying, detecting, and/
      or-cloning nucleic acid sequences" U.S._Patent_4,683,195
   6. ^ a b"Kary_B._Mullis_â_Nobel_Lecture:_The_Polymerase_Chain_Reaction".
   7. ^"Determining Annealing Temperatures for Polymerase Chain Reaction". The
      American Biology Teacher. 74 (4): 256â260. 2012. doi:10.1525/
      abt.2012.74.4.9.
   8. ^ a b c d e fNinfa, Alexander; Ballou, David; Benore, Marilee (2009).
      Fundamental Laboratory Approaches for Biochemistry and Biotechnology.
      United States: Wiley. pp. 408â410. ISBN 978-0470087664.
   9. ^Cheng, S.; Fockler, C.; Barnes, W. M.; Higuchi, R. (1994). "Effective
      Amplification_of_Long_Targets_from_Cloned_Inserts_and_Human_Genomic_DNA".
      Proceedings of the National Academy of Sciences. 91 (12): 5695â5699.
      Bibcode:1994PNAS...91.5695C. doi:10.1073/pnas.91.12.5695. PMC 44063.
      PMID 8202550.
  10. ^Carr AC, Moore SD (2012). Lucia A (ed.). "Robust_quantification_of
      polymerase_chain_reactions_using_global_fitting". PLoS ONE. 7 (5):
      e37640. Bibcode:2012PLoSO...737640C. doi:10.1371/journal.pone.0037640.
      PMC 3365123. PMID 22701526.
  11. ^ a bJoseph Sambrook & David W. Russel (2001). Molecular Cloning: A
      Laboratory Manual (3rd ed.). Cold Spring Harbor, N.Y.: Cold Spring Harbor
      Laboratory Press. ISBN 978-0-879-69576-7.
  12.  Chapter 8: In vitro Amplification of DNA by the Polymerase Chain
      Reaction
  13. ^"Polymerase_Chain_Reaction_(PCR)". National Center for Biotechnology
      Information, U.S. National Library of Medicine.
  14. ^"PCR". Genetic Science Learning Center, University_of_Utah.
  15. ^Pavlov, A. R.; Pavlova, N. V.; Kozyavkin, S. A.; Slesarev, A. I. (2004).
      "Recent developments in the optimization of thermostable DNA polymerases
      for efficient applicationsâ". Trends in Biotechnology. 22 (5):
      253â260. doi:10.1016/j.tibtech.2004.02.011. PMID 15109812.
  16. ^Rychlik W, Spencer WJ, Rhoads RE (1990). "Optimization_of_the_annealing
      temperature_for_DNA_amplification_in_vitro". Nucleic Acids Res. 18 (21):
      6409â6412. doi:10.1093/nar/18.21.6409. PMC 332522. PMID 2243783.
  17. ^ a bSharkey, D. J.; Scalice, E. R.; Christy, K. G.; Atwood, S. M.;
      Daiss, J. L. (1994). "Antibodies as Thermolabile Switches: High
      Temperature Triggering for the Polymerase Chain Reaction". Bio/
      Technology. 12 (5): 506â509. doi:10.1038/nbt0594-506.
  18. ^ a bChien A, Edgar DB, Trela JM (1976). "Deoxyribonucleic_acid
      polymerase_from_the_extreme_thermophile_Thermus_aquaticus". J. Bacteriol.
      127 (3): 1550â1557. PMC 232952. PMID 8432.
  19. ^ a bLawyer, F.; Stoffel, S.; Saiki, R.; Chang, S.; Landre, P.; Abramson,
      R.; Gelfand, D. (1993). "High-level expression, purification, and
      enzymatic characterization of full-length Thermus aquaticus DNA
      polymerase and a truncated form deficient in 5' to 3' exonuclease
      activity". PCR Methods and Applications. 2 (4): 275â287. doi:10.1101/
      gr.2.4.275. PMID 8324500.
  20. ^ a b c dSchochetman, Gerald; Ou, Chin-Yih; Jones, Wanda K. (1988).
      "Polymerase Chain Reaction". The Journal of Infectious Diseases. 158 (6):
      1154â1157. doi:10.1093/infdis/158.6.1154. JSTOR 30137034.
  21. ^Borman, Jon; Schuster, David; Li, Wu-bo; Jessee, Joel; Rashtchian, Ayoub
      (2000). "PCR_from_problematic_templates" (PDF). Focus. 22 (1): 10.
  22. ^Bogetto, Prachi and Waidne, Lisa (2000). "Helpful_tips_for_PCR" (PDF).
      Focus. 22 (1): 12.CS1 maint: Multiple names: authors list (link)
  23. ^Sarkar, G.; Kapelner, S.; Sommer, S. (1990). "Formamide_can_dramatically
      improve_the_specificity_of_PCR". Nucleic Acids Research. 18 (24): 7465.
      doi:10.1093/nar/18.24.7465. PMC 332902. PMID 2259646.
  24. ^"Electronic_PCR". NCBI â National Center for Biotechnology
      Information. Retrieved 13 March 2012.
  25. ^Pavlov AR, Pavlova NV, Kozyavkin SA, Slesarev AI (2006). "Thermostable
      DNA Polymerases for a Wide Spectrum of Applications: Comparison of a
      Robust Hybrid TopoTaq to other enzymes". In Kieleczawa J (ed.). DNA
      Sequencing II: Optimizing Preparation and Cleanup. Jones and Bartlett.
      pp. 241â257. ISBN 978-0-7637-3383-4.
  26. ^Pombert JF, Sistek V, Boissinot M, Frenette M (2009). "Evolutionary
      relationships_among_salivarius_streptococci_as_inferred_from_multilocus
      phylogenies_based_on_16S_rRNA-encoding,_recA,_secA,_and_secY_gene
      sequences". BMC Microbiol. 9: 232. doi:10.1186/1471-2180-9-232.
      PMC 2777182. PMID 19878555.
  27. ^"Chemical_Synthesis,_Sequencing,_and_Amplification_of_DNA_(class_notes
      on_MBB/BIO_343)". Arizona State University. Archived from the_original on
      9 October 1997. Retrieved 29 October 2007.
  28. ^Bustin, S. A.; Benes, V.; Garson, J. A.; Hellemans, J.; Huggett, J.;
      Kubista, M.; Mueller, R.; Nolan, T.; Pfaffl, M. W.; Shipley, G. L.;
      Vandesompele, J.; Wittwer, C. T. (2009). "The MIQE Guidelines: Minimum
      Information for Publication of Quantitative Real-Time PCR Experiments".
      Clinical Chemistry. 55 (4): 611â622. doi:10.1373/clinchem.2008.112797.
      PMID 19246619.
  29. ^ a bGaribyan, Avashia (March 2013). "Polymerase_Chain_Reaction". Journal
      of Investigative Dermatology. 133 (3): 1â4. doi:10.1038/jid.2013.1.
      PMC 4102308. PMID 23399825.
  30. ^Quill, E. (2008). "Medicine. Blood-matching goes genetic". Science. 319
      (5869): 1478â9. doi:10.1126/science.319.5869.1478. PMID 18339916.
  31. ^Tomar, Rukam (2010). Molecular Markers and Plant Biotechnology. Pitman
      Pura, New Delhi: New India Publishing Agency. p. 188. ISBN 978-93-80235-
      25-7.
  32. ^ a b cCai, H; Caswell JL; Prescott JF (March 2014). "Nonculture
      Molecular Techniques for Diagnosis of Bacterial Disease in Animals: A
      Diagnostic Laboratory Perspective". Veterinary Pathology. 51 (2):
      341â350. doi:10.1177/0300985813511132. PMID 24569613.
  33. ^Salis AD (2009). "Applications in Clinical Microbiology". Real-Time PCR:
      Current Technology and Applications. Caister Academic Press. ISBN 978-1-
      904455-39-4.
  34. ^Kwok, S.; Mack, D. H.; Mullis, K. B.; Poiesz, B.; Ehrlich, G.; Blair,
      D.; Friedman-Kien, A.; Sninsky, J. J. (1987). "Identification_of_human
      immunodeficiency_virus_sequences_by_using_in_vitro_enzymatic
      amplification_and_oligomer_cleavage_detection". Journal of Virology. 61
      (5): 1690â4. PMC 254157. PMID 2437321.
  35. ^Finger, Horst; von Koenig, Carl Heinz Wirsing (1996). Baron, Samuel
      (ed.). Medical_Microbiology (4th ed.). Galveston (TX): University of
      Texas Medical Branch at Galveston. ISBN 978-0963117212. PMID 21413270.
  36. ^Yeh, Sylvia H.; Mink, ChrisAnna M. (2012). "Netter's Infectious
      Diseases". Netter's Infectious Diseases: 11â14. doi:10.1016/B978-1-
      4377-0126-5.00003-3.
  37. ^Alonso, A (28 January 2004). "Real-time PCR designs to estimate nuclear
      and mitochondrial DNA copy number in forensic and ancient DNA studies".
      Forensic Science International. 139 (2â3): 141â149. doi:10.1016/
      j.forsciint.2003.10.008. PMID 15040907.
  38. ^Boehnke, M.; Arnheim, N.; Li, H.; Collins, F. S. (1989). "Fine-structure
      genetic_mapping_of_human_chromosomes_using_the_polymerase_chain_reaction
      on_single_sperm:_Experimental_design_considerations". American Journal of
      Human Genetics. 45 (1): 21â32. PMC 1683385. PMID 2568090.
  39. ^Garibyan, Lilit; Avashia, Nidhi (1 March 2013). "Polymerase_Chain
      Reaction". Journal of Investigative Dermatology. 133 (3): 1â4. doi:
      10.1038/jid.2013.1. PMC 4102308. PMID 23399825.
  40. ^Garibyan L, Avashia N (2013). "Polymerase_Chain_Reaction". Journal of
      Investigative Dermatology. 133 (3): 1â4. doi:10.1038/jid.2013.1.
      PMC 4102308. PMID 23399825.
  41. ^Zhou, Y H; Zhang, X P; Ebright, R H (11 November 1991). "Random
      mutagenesis_of_gene-sized_DNA_molecules_by_use_of_PCR_with_Taq_DNA
      polymerase". Nucleic Acids Research. 19 (21): 6052. doi:10.1093/nar/
      19.21.6052. PMC 329070. PMID 1658751.
  42. ^Newton CR, Graham A, Heptinstall LE, Powell SJ, Summers C, Kalsheker N,
      Smith JC, Markham AF (1989). "Analysis_of_any_point_mutation_in_DNA._The
      amplification_refractory_mutation_system_(ARMS)". Nucleic Acids Research.
      17 (7): 2503â2516. doi:10.1093/nar/17.7.2503. PMC 317639. PMID 2785681.
  43. ^Stemmer WP, Crameri A, Ha KD, Brennan TM, Heyneker HL (1995). "Single-
      step assembly of a gene and entire plasmid from large numbers of
      oligodeoxyribonucleotides". Gene. 164 (1): 49â53. doi:10.1016/0378-1119
      (95)00511-4. PMID 7590320.
  44. ^Innis MA, Myambo KB, Gelfand DH, Brow MA (1988). "DNA_sequencing_with
      Thermus_aquaticus_DNA_polymerase_and_direct_sequencing_of_polymerase
      chain_reaction-amplified_DNA". Proc Natl Acad Sci USA. 85 (24):
      9436â9440. Bibcode:1988PNAS...85.9436I. doi:10.1073/pnas.85.24.9436.
      PMC 282767. PMID 3200828.
  45. ^Pierce KE & Wangh LJ (2007). Linear-after-the-exponential polymerase
      chain reaction and allied technologies Real-time detection strategies for
      rapid, reliable diagnosis from single cells. Methods in Molecular
      Medicine. 132. pp. 65â85. doi:10.1007/978-1-59745-298-4_7. ISBN 978-1-
      58829-578-1. PMID 17876077.
  46. ^Krishnan, Madhavi; Ugaz, Victor; Burns, Mark (2002). "PCR in a Rayleigh-
      Benard convection cell". Science. 298 (5594): 793. doi:10.1126/
      science.298.5594.793. PMID 12399582.
  47. ^Priye, Aashish; Hassan, Yassin; Ugaz, Victor (2013). "Microscale chaotic
      advection enables robust convective DNA replication". Analytical
      Chemistry. 85 (21): 10536â10541. doi:10.1021/ac402611s. PMID 24083802.
  48. ^Schwartz JJ, Lee C, Shendure J (2012). "Accurate_gene_synthesis_with
      tag-directed_retrieval_of_sequence-verified_DNA_molecules". Nature
      Methods. 9 (9): 913â915. doi:10.1038/nmeth.2137. PMC 3433648.
      PMID 22886093.
  49. ^Vincent M, Xu Y, Kong H (2004). "Helicase-dependent_isothermal_DNA
      amplification". EMBO Reports. 5 (8): 795â800. doi:10.1038/
      sj.embor.7400200. PMC 1249482. PMID 15247927.
  50. ^Chou Q, Russell M, Birch DE, Raymond J, Bloch W (1992). "Prevention_of
      pre-PCR_mis-priming_and_primer_dimerization_improves_low-copy-number
      amplifications". Nucleic Acids Research. 20 (7): 1717â1723. doi:
      10.1093/nar/20.7.1717. PMC 312262. PMID 1579465.
  51. ^Kellogg, DE; Rybalkin, I; Chen, S; Mukhamedova, N; Vlasik, T; Siebert,
      PD; Chenchik, A (1994). "TaqStart Antibody: "hot start" PCR facilitated
      by a neutralizing monoclonal antibody directed against Taq DNA
      polymerase". BioTechniques. 16 (6): 1134â7. PMID 8074881.
  52. ^San Millan RM, Martinez-Ballesteros I, Rementeria A, Garaizar J, Bikandi
      J (2013). "Online_exercise_for_the_design_and_simulation_of_PCR_and_PCR-
      RFLP_experiments". BMC Research Notes. 6: 513. doi:10.1186/1756-0500-6-
      513. PMC 4029544. PMID 24314313.
  53. ^Zietkiewicz, E.; Rafalski, A.; Labuda, D. (1994). "Genome fingerprinting
      by simple sequence repeat (SSR)-anchored polymerase chain reaction
      amplification". Genomics. 20 (2): 176â83. doi:10.1006/geno.1994.1151.
      PMID 8020964.
  54. ^Ochman H, Gerber AS, Hartl DL (1988). "Genetic_Applications_of_an
      Inverse_Polymerase_Chain_Reaction". Genetics. 120 (3): 621â623.
      PMC 1203539. PMID 2852134.
  55. ^Mueller PR, Wold B (1988). "In vivo footprinting of a muscle specific
      enhancer by ligation mediated PCR". Science. 246 (4931): 780â786. doi:
      10.1126/science.2814500. PMID 2814500.
  56. ^Herman_JG, Graff JR, MyÃ¶hÃ¤nen S, Nelkin BD, Baylin SB (1996).
      "Methylation-specific_PCR:_a_novel_PCR_assay_for_methylation_status_of
      CpG_islands". Proc Natl Acad Sci USA. 93 (13): 9821â9826. Bibcode:
      1996PNAS...93.9821H. doi:10.1073/pnas.93.18.9821. PMC 38513.
      PMID 8790415.
  57. ^Isenbarger TA, Finney M, RÃ­os-VelÃ¡zquez C, Handelsman J, Ruvkun G
      (2008). "Miniprimer_PCR,_a_New_Lens_for_Viewing_the_Microbial_World".
      Applied and Environmental Microbiology. 74 (3): 840â9. doi:10.1128/
      AEM.01933-07. PMC 2227730. PMID 18083877.
  58. ^Cenchao Shen; Wenjuan Yang; Qiaoli Ji; Hisaji Maki; Anjie Dong; Zhizhou
      Zhang (2009). "NanoPCR observation: different levels of DNA replication
      fidelity in nanoparticle-enhanced polymerase chain reactions".
      Nanotechnology. 20 (45): 455103. Bibcode:2009Nanot..20S5103S. doi:
      10.1088/0957-4484/20/45/455103. PMID 19822925.
  59. ^Shen, Cenchao (2013). "An Overview of Nanoparticle-Assisted Polymerase
      Chain Reaction Technology". An Overview of NanoparticleâAssisted
      Polymerase Chain Reaction Technology. US: Wiley-Blackwell Publishing Ltd.
      pp. 97â106. doi:10.1002/9781118451915.ch5. ISBN 9781118451915.
  60. ^Horton RM, Hunt HD, Ho SN, Pullen JK, Pease LR (1989). "Engineering
      hybrid genes without the use of restriction enzymes: gene splicing by
      overlap exten-sion". Gene. 77 (1): 61â68. doi:10.1016/0378-1119
      (89)90359-4. PMID 2744488.
  61. ^Moller, Simon (2006). PCR: The Basics. US: Taylor & Francis Group.
      p. 144. ISBN 9780415355476.
  62. ^David F, Turlotte E (1998). "Une mÃ©thode d'amplification gÃ©nique
      isotherme" [An Isothermal Amplification Method]. Comptes Rendus de
      l'AcadÃ©mie des Sciences, SÃ©rie III. 321 (11): 909â914. Bibcode:
      1998CRASG.321..909D. doi:10.1016/S0764-4469(99)80005-5.
  63. ^Fabrice David (September â October 2002). "Utiliser_les_propriÃ©tÃ©s
      topologiques_de_l'ADN:_une_nouvelle_arme_contre_les_agents_pathogÃ¨nes"
      (PDF). Fusion. Archived from the_original (PDF) on 28 November 2007.
  64. (in French)
  65. ^Dobosy JR, Rose SD, Beltz KR, Rupp SM, Powers KM, Behlke MA, Walder JA
      (August 2011). "RNase_H-dependent_PCR_(rhPCR):_improved_specificity_and
      single_nucleotide_polymorphism_detection_using_blocked_cleavable
      primers". BMC Biotechnology. 11: 80. doi:10.1186/1472-6750-11-80.
      PMC 3224242. PMID 21831278.
  66. ^Shyamala, V.; Ferro-Luzzi, Ames G. (1993). Single Specific Primer-
      Polymerase Chain Reaction (SSP-PCR) and Genome Walking. Methods in
      Molecular Biology. 15. pp. 339â48. doi:10.1385/0-89603-244-2:339.
      ISBN 978-0-89603-244-6. PMID 21400290.
  67. ^Bing DH, Boles C, Rehman FN, Audeh M, Belmarsh M, Kelley B, Adams CP
      (1996). "Bridge_amplification:_a_solid_phase_PCR_system_for_the
      amplification_and_detection_of_allelic_differences_in_single_copy_genes".
      Genetic Identity Conference Proceedings, Seventh International Symposium
      on Human Identification. Archived from the_original on 7 May 2001.
  68. ^Khan Z, Poetter K, Park DJ (2008). "Enhanced solid phase PCR: mechanisms
      to increase priming by solid support primers". Analytical Biochemistry.
      375 (2): 391â393. doi:10.1016/j.ab.2008.01.021. PMID 18267099.
  69. ^Raoult, D; G Aboudharam; E Crubezy; G Larrouy; B Ludes; M Drancourt (7
      November 2000). "Molecular_identification_by_"suicide_PCR"_of_Yersinia
      pestis_as_the_agent_of_medieval_black_death". Proc. Natl. Acad. Sci.
      U.S.A. 97 (23): 12800â12803. Bibcode:2000PNAS...9712800R. doi:10.1073/
      pnas.220225197. PMC 18844. PMID 11058154.
  70. ^Y.G. Liu & R. F. Whittier (1995). "Thermal asymmetric interlaced PCR:
      automatable amplification and sequencing of insert end fragments from P1
      and YAC clones for chromosome walking". Genomics. 25 (3): 674â81. doi:
      10.1016/0888-7543(95)80010-J. PMID 7759102.
  71. ^Don RH, Cox PT, Wainwright BJ, Baker K, Mattick JS (1991). "'Touchdown'
      PCR_to_circumvent_spurious_priming_during_gene_amplification". Nucleic
      Acids Res. 19 (14): 4008. doi:10.1093/nar/19.14.4008. PMC 328507.
      PMID 1861999.
  72. ^Myrick KV, Gelbart WM (2002). "Universal Fast Walking for direct and
      versatile determination of flanking sequence". Gene. 284 (1â2):
      125â131. doi:10.1016/S0378-1119(02)00384-0. PMID 11891053.
  73. ^"Full_Text_â_LaNe_RAGE:_a_new_tool_for_genomic_DNA_flanking_sequence
      determination".
  74. ^Park DJ (2005). "A new 5' terminal murine GAPDH exon identified using
      5'RACE LaNe". Molecular Biotechnology. 29 (1): 39â46. doi:10.1385/MB:
      29:1:39. PMID 15668518.
  75. ^Park DJ (2004). "3'RACE LaNe: a simple and rapid fully nested PCR method
      to determine 3'-terminal cDNA sequence". BioTechniques. 36 (4):
      586â588, 590. doi:10.2144/04364BM04. PMID 15088375.
  76. ^Kleppe K, Ohtsuka E, Kleppe R, Molineux I, Khorana HG (1971). "Studies
      on polynucleotides. XCVI. Repair replications of short synthetic DNA's as
      catalyzed by DNA polymerases". J. Mol. Biol. 56 (2): 341â361. doi:
      10.1016/0022-2836(71)90469-4. PMID 4927950.
  77. ^Rabinow,_Paul (1996). Making PCR: A Story of Biotechnology. Chicago:
      University of Chicago Press. ISBN 978-0-226-70146-2.
  78. ^Mullis,_Kary (1998). Dancing Naked in the Mind Field. New York: Pantheon
      Books. ISBN 978-0-679-44255-4.
  79. ^Mullis, Kary (1990). "The unusual origin of the polymerase chain
      reaction". Scientific American. 262 (4): 56â61, 64â5. Bibcode:
      1990SciAm.262d..56M. doi:10.1038/scientificamerican0490-56. PMID 2315679.
  80. ^Patidar, Madhvika; Agrawal, Suraksha; Parveen, Farah; Khare, Parul
      (2015). "Molecular_insights_of_saliva_in_solving_paternity_dispute".
      Journal of Forensic Dental Sciences. 7 (1): 76â79. doi:10.4103/0975-
      1475.150325. PMC 4330625. PMID 25709326.
  81. ^"Citations_for_Chemical_Breakthrough_Awards_2017_Awardees". Division of
      the History of Chemistry. Retrieved 12 March 2018.
  82. ^Saiki, R.; Scharf, S; Faloona, F; Mullis, K.; Horn, G.; Erlich, H.;
      Arnheim, N (1985). "Enzymatic amplification of beta-globin genomic
      sequences and restriction site analysis for diagnosis of sickle cell
      anemia". Science. 230 (4732): 1350â1354. doi:10.1126/science.2999980.
  83. ^"Advice_on_How_to_Survive_the_Taq_Wars". GEN Genetic Engineering News
      â Biobusiness Channel. 26 (9). 1 May 2006.
***** External links[edit] *****
Library_resources about
Polymerase chain reaction
===============================================================================
    * Resources_in_your_library
    * Resources_in_other_libraries
 Wikimedia Commons has media related to Polymerase_chain_reaction.
    * PCR_Animation maxanim.com
    * OpenPCR Open-source PCR thermalcycler project
    * US_Patent_for_PCR
    * OpenWetWare
    * What_is_PCR_plateau_effect? YouTube tutorial video
    * GeneWarrior Online PCR Primer design tool
    * History_of_the_Polymerase_Chain_Reaction from the Smithsonian_Institution
      Archives
    * Computer_exercise._Design_of_PCR_and_PCR-RFLP_experiments
    * v
    * t
    * e
Polymerase chain reaction techniques
                   * Initialization
                   * cycle
                         o Denaturation
Procedure                o Annealing
                         o elongation
                   * Final elongation
                   * Final hold
                   * Klenow
Polymerase         * Taq
                   * Pfu
                   * Quantitative_polymerase_chain_reaction (qPCR)
                   * Reverse_transcription_polymerase_chain_reaction (RT-PCR)
                   * Inverse_polymerase_chain_reaction
optimization       * Nested_polymerase_chain_reaction
ï¼Variants     * Touchdown_polymerase_chain_reaction
                   * Hot_start_PCR
                   * Overlap_extension_polymerase_chain_reaction
                   * Multiplex_polymerase_chain_reaction
                   * Multiplex_ligation-dependent_probe_amplification
history            * Kary_Mullis
ï¼people       * Kjell_Kleppe [no]
                   * Alice_Chien
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal
Authority_control [Edit_this_at_Wikidata]     * GND: 4256726-9

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Polymerase_chain_reaction&oldid=904955852"
Categories:
    * Molecular_biology
    * Polymerase_chain_reaction
    * Laboratory_techniques
    * DNA_profiling_techniques
    * Amplifiers
    * Hoffmann-La_Roche
    * Biotechnology
    * Molecular_biology_techniques
    * American_inventions
Hidden categories:
    * CS1_maint:_Multiple_names:_authors_list
    * Articles_with_short_description
    * Use_dmy_dates_from_February_2013
    * Wikipedia_articles_that_are_too_technical_from_October_2007
    * All_articles_that_are_too_technical
    * Articles_needing_expert_attention_from_October_2007
    * All_articles_needing_expert_attention
    * Interlanguage_link_template_link_number
    * Wikipedia_articles_with_GND_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Afrikaans
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * AzÉrbaycanca
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Gaeilge
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * LietuviÅ³
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * æ¥æ¬èª
    * Nordfriisk
    * Norsk
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Shqip
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * Taqbaylit
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 5 July 2019, at 19:59 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
