The following text has been accessed from https://en.wikipedia.org/wiki/DNA_repair at Fri Aug 9 01:43:47 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_featured_article._Click_here_for_more_information.]
[Listen_to_this_article]
****** DNA repair ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
For the journal, see DNA_Repair_(journal).
Processes by which a cell identifies and corrects damage to the DNA molecules
that encode its genome
DNA damage resulting in multiple broken chromosomes
DNA repair is a collection of processes by which a cell identifies and corrects
damage to the DNA molecules that encode its genome.[1] In human cells, both
normal metabolic activities and environmental factors such as radiation can
cause DNA damage, resulting in as many as 1 million individual molecular
lesions per cell per day.[2] Many of these lesions cause structural damage to
the DNA molecule and can alter or eliminate the cell's ability to transcribe
the gene that the affected DNA encodes. Other lesions induce potentially
harmful mutations in the cell's genome, which affect the survival of its
daughter cells after it undergoes mitosis. As a consequence, the DNA repair
process is constantly active as it responds to damage in the DNA structure.
When normal repair processes fail, and when cellular apoptosis does not occur,
irreparable DNA damage may occur, including double-strand breaks and DNA
crosslinkages (interstrand crosslinks or ICLs).[3][4] This can eventually lead
to malignant tumors, or cancer as per the two_hit_hypothesis.
The rate of DNA repair is dependent on many factors, including the cell type,
the age of the cell, and the extracellular environment. A cell that has
accumulated a large amount of DNA damage, or one that no longer effectively
repairs damage incurred to its DNA, can enter one of three possible states:
   1. an irreversible state of dormancy, known as senescence
   2. cell suicide, also known as apoptosis or programmed_cell_death
   3. unregulated cell division, which can lead to the formation of a tumor
      that is cancerous
The DNA repair ability of a cell is vital to the integrity of its genome and
thus to the normal functionality of that organism. Many genes that were
initially shown to influence life_span have turned out to be involved in DNA
damage repair and protection.[5]
[File:Paul Modrich.webm]Play_media
Paul Modrich talks about himself and his work in DNA repair.
The 2015 Nobel_Prize_in_Chemistry was awarded to Tomas_Lindahl, Paul_Modrich,
and Aziz_Sancar for their work on the molecular mechanisms of DNA repair
processes.[6][7]
⁰
***** Contents *****
    * 1_DNA_damage
          o 1.1_Sources
          o 1.2_Types
          o 1.3_Nuclear_versus_mitochondrial
          o 1.4_Senescence_and_apoptosis
          o 1.5_Mutation
    * 2_Mechanisms
          o 2.1_Direct_reversal
          o 2.2_Single-strand_damage
          o 2.3_Double-strand_breaks
          o 2.4_Translesion_synthesis
    * 3_Global_response_to_DNA_damage
          o 3.1_Initial_steps
          o 3.2_DNA_damage_checkpoints
          o 3.3_The_prokaryotic_SOS_response
          o 3.4_Eukaryotic_transcriptional_responses_to_DNA_damage
    * 4_Aging
          o 4.1_Pathological_effects_of_poor_DNA_repair
          o 4.2_Longevity_and_caloric_restriction
    * 5_Medicine_and_DNA_repair_modulation
          o 5.1_Hereditary_DNA_repair_disorders
    * 6_Cancer
          o 6.1_DNA_repair_defects_in_cancer
          o 6.2_Epigenetic_DNA_repair_defects_in_cancer
          o 6.3_Frequencies_of_epimutations_in_DNA_repair_genes
          o 6.4_Genome-wide_distribution_of_DNA_repair_in_human_somatic_cells
    * 7_Evolution
          o 7.1_Rate_of_evolutionary_change
    * 8_Technology
    * 9_See_also
    * 10_References
    * 11_Further_reading
    * 12_External_links
***** DNA damage[edit] *****
Further information: DNA_damage_(naturally_occurring) and Free_radical_damage
to_DNA
DNA damage, due to environmental factors and normal metabolic processes inside
the cell, occurs at a rate of 10,000 to 1,000,000 molecular lesions per cell
per day.[2] While this constitutes only 0.000165% of the human genome's
approximately 6 billion bases (3 billion base pairs), unrepaired lesions in
critical genes (such as tumor_suppressor_genes) can impede a cell's ability to
carry out its function and appreciably increase the likelihood of tumor
formation and contribute to tumour_heterogeneity.
The vast majority of DNA damage affects the primary_structure of the double
helix; that is, the bases themselves are chemically modified. These
modifications can in turn disrupt the molecules' regular helical structure by
introducing non-native chemical bonds or bulky adducts that do not fit in the
standard double helix. Unlike proteins and RNA, DNA usually lacks tertiary
structure and therefore damage or disturbance does not occur at that level. DNA
is, however, supercoiled and wound around "packaging" proteins called histones
(in eukaryotes), and both superstructures are vulnerable to the effects of DNA
damage.
**** Sources[edit] ****
DNA damage can be subdivided into two main types:
   1. endogenous damage such as attack by reactive_oxygen_species produced from
      normal metabolic byproducts (spontaneous mutation), especially the
      process of oxidative_deamination
         1. also includes replication_errors
   2. exogenous damage caused by external agents such as
         1. ultraviolet [UV 200â400 nm] radiation from the sun
         2. other radiation frequencies, including x-rays and gamma_rays
         3. hydrolysis or thermal disruption
         4. certain plant toxins
         5. human-made mutagenic_chemicals, especially aromatic compounds that
            act as DNA intercalating_agents
         6. viruses[8]
The replication of damaged DNA before cell division can lead to the
incorporation of wrong bases opposite damaged ones. Daughter cells that inherit
these wrong bases carry mutations from which the original DNA sequence is
unrecoverable (except in the rare case of a back_mutation, for example, through
gene_conversion).
**** Types[edit] ****
There are several types of damage to DNA due to endogenous cellular processes:
   1. oxidation of bases [e.g. 8-oxo-7,8-dihydroguanine (8-oxoG)] and
      generation of DNA strand interruptions from reactive oxygen species,
   2. alkylation of bases (usually methylation), such as formation of 7-
      methylguanosine, 1-methyladenine, 6-O-Methylguanine
   3. hydrolysis of bases, such as deamination, depurination, and
      depyrimidination.
   4. "bulky_adduct_formation" (e.g., benzo[a]pyrene diol epoxide-dG adduct,
      aristolactam I-dA adduct)
   5. mismatch of bases, due to errors in DNA_replication, in which the wrong
      DNA base is stitched into place in a newly forming DNA strand, or a DNA
      base is skipped over or mistakenly inserted.
   6. Monoadduct damage cause by change in single nitrogenous base of DNA
   7. Diadduct damage
Damage caused by exogenous agents comes in many forms. Some examples are:
   1. UV-B_light causes crosslinking between adjacent cytosine and thymine
      bases creating pyrimidine_dimers. This is called direct_DNA_damage.
   2. UV-A_light creates mostly free radicals. The damage caused by free
      radicals is called indirect_DNA_damage.
   3. Ionizing_radiation such as that created by radioactive decay or in cosmic
      rays causes breaks in DNA strands. Intermediate-level ionizing radiation
      may induce irreparable DNA damage (leading to replicational and
      transcriptional errors needed for neoplasia or may trigger viral
      interactions) leading to pre-mature aging and cancer.
   4. Thermal disruption at elevated temperature increases the rate of
      depurination (loss of purine bases from the DNA backbone) and single-
      strand breaks. For example, hydrolytic depurination is seen in the
      thermophilic_bacteria, which grow in hot_springs at 40â80 Â°C.[9][10]
      The rate of depurination (300 purine residues per genome per generation)
      is too high in these species to be repaired by normal repair machinery,
      hence a possibility of an adaptive response cannot be ruled out.
   5. Industrial chemicals such as vinyl_chloride and hydrogen_peroxide, and
      environmental chemicals such as polycyclic_aromatic_hydrocarbons found in
      smoke, soot and tar create a huge diversity of DNA adducts- ethenobases,
      oxidized bases, alkylated phosphotriesters and crosslinking_of_DNA, just
      to name a few.
UV damage, alkylation/methylation, X-ray damage and oxidative damage are
examples of induced damage. Spontaneous damage can include the loss of a base,
deamination, sugar ring_puckering and tautomeric shift. Constitutive
(spontaneous) DNA damage caused by endogenous oxidants can be detected as a low
level of histone H2AX phosphorylation in untreated cells.[11]
**** Nuclear versus mitochondrial[edit] ****
In human cells, and eukaryotic cells in general, DNA is found in two cellular
locations â inside the nucleus and inside the mitochondria. Nuclear DNA
(nDNA) exists as chromatin during non-replicative stages of the cell_cycle and
is condensed into aggregate structures known as chromosomes during cell
division. In either state the DNA is highly compacted and wound up around bead-
like proteins called histones. Whenever a cell needs to express the genetic
information encoded in its nDNA the required chromosomal region is unravelled,
genes located therein are expressed, and then the region is condensed back to
its resting conformation. Mitochondrial DNA (mtDNA) is located inside
mitochondria organelles, exists in multiple copies, and is also tightly
associated with a number of proteins to form a complex known as the nucleoid.
Inside mitochondria, reactive_oxygen_species (ROS), or free_radicals,
byproducts of the constant production of adenosine_triphosphate (ATP) via
oxidative_phosphorylation, create a highly oxidative environment that is known
to damage mtDNA. A critical enzyme in counteracting the toxicity of these
species is superoxide_dismutase, which is present in both the mitochondria and
cytoplasm of eukaryotic cells.
**** Senescence and apoptosis[edit] ****
Senescence, an irreversible process in which the cell no longer divides, is a
protective response to the shortening of the chromosome_ends. The telomeres are
long regions of repetitive noncoding_DNA that cap chromosomes and undergo
partial degradation each time a cell undergoes division (see Hayflick_limit).
[12] In contrast, quiescence is a reversible state of cellular dormancy that is
unrelated to genome damage (see cell_cycle). Senescence in cells may serve as a
functional alternative to apoptosis in cases where the physical presence of a
cell for spatial reasons is required by the organism,[13] which serves as a
"last resort" mechanism to prevent a cell with damaged DNA from replicating
inappropriately in the absence of pro-growth cellular_signaling. Unregulated
cell division can lead to the formation of a tumor (see cancer), which is
potentially lethal to an organism. Therefore, the induction of senescence and
apoptosis is considered to be part of a strategy of protection against cancer.
[14]
**** Mutation[edit] ****
It is important to distinguish between DNA damage and mutation, the two major
types of error in DNA. DNA damage and mutation are fundamentally different.
Damage results in physical abnormalities in the DNA, such as single- and
double-strand breaks, 8-hydroxydeoxyguanosine residues, and polycyclic aromatic
hydrocarbon adducts. DNA damage can be recognized by enzymes, and thus can be
correctly repaired if redundant information, such as the undamaged sequence in
the complementary DNA strand or in a homologous chromosome, is available for
copying. If a cell retains DNA damage, transcription of a gene can be
prevented, and thus translation into a protein will also be blocked.
Replication may also be blocked or the cell may die.
In contrast to DNA damage, a mutation is a change in the base sequence of the
DNA. A mutation cannot be recognized by enzymes once the base change is present
in both DNA strands, and thus a mutation cannot be repaired. At the cellular
level, mutations can cause alterations in protein function and regulation.
Mutations are replicated when the cell replicates. In a population of cells,
mutant cells will increase or decrease in frequency according to the effects of
the mutation on the ability of the cell to survive and reproduce.
Although distinctly different from each other, DNA damage and mutation are
related because DNA damage often causes errors of DNA synthesis during
replication or repair; these errors are a major source of mutation.
Given these properties of DNA damage and mutation, it can be seen that DNA
damage is a special problem in non-dividing or slowly-dividing cells, where
unrepaired damage will tend to accumulate over time. On the other hand, in
rapidly-dividing cells, unrepaired DNA damage that does not kill the cell by
blocking replication will tend to cause replication errors and thus mutation.
The great majority of mutations that are not neutral in their effect are
deleterious to a cell's survival. Thus, in a population of cells composing a
tissue with replicating cells, mutant cells will tend to be lost. However,
infrequent mutations that provide a survival advantage will tend to clonally
expand at the expense of neighboring cells in the tissue. This advantage to the
cell is disadvantageous to the whole organism, because such mutant cells can
give rise to cancer. Thus, DNA damage in frequently dividing cells, because it
gives rise to mutations, is a prominent cause of cancer. In contrast, DNA
damage in infrequently-dividing cells is likely a prominent cause of aging.[15]
***** Mechanisms[edit] *****
Cells cannot function if DNA damage corrupts the integrity and accessibility of
essential information in the genome (but cells remain superficially functional
when non-essential genes are missing or damaged). Depending on the type of
damage inflicted on the DNA's double helical structure, a variety of repair
strategies have evolved to restore lost information. If possible, cells use the
unmodified complementary strand of the DNA or the sister chromatid as a
template to recover the original information. Without access to a template,
cells use an error-prone recovery mechanism known as translesion_synthesis as a
last resort.
Damage to DNA alters the spatial configuration of the helix, and such
alterations can be detected by the cell. Once damage is localized, specific DNA
repair molecules bind at or near the site of damage, inducing other molecules
to bind and form a complex that enables the actual repair to take place.
**** Direct reversal[edit] ****
Cells are known to eliminate three types of damage to their DNA by chemically
reversing it. These mechanisms do not require a template, since the types of
damage they counteract can occur in only one of the four bases. Such direct
reversal mechanisms are specific to the type of damage incurred and do not
involve breakage of the phosphodiester backbone. The formation of pyrimidine
dimers upon irradiation with UV light results in an abnormal covalent bond
between adjacent pyrimidine bases. The photoreactivation process directly
reverses this damage by the action of the enzyme photolyase, whose activation
is obligately dependent on energy absorbed from blue/UV_light (300â500 nm
wavelength) to promote catalysis.[16] Photolyase, an old enzyme present in
bacteria, fungi, and most animals no longer functions in humans,[17] who
instead use nucleotide_excision_repair to repair damage from UV irradiation.
Another type of damage, methylation of guanine bases, is directly reversed by
the protein methyl guanine methyl transferase (MGMT), the bacterial equivalent
of which is called ogt. This is an expensive process because each MGMT molecule
can be used only once; that is, the reaction is stoichiometric rather than
catalytic.[18] A generalized response to methylating agents in bacteria is
known as the adaptive_response and confers a level of resistance to alkylating
agents upon sustained exposure by upregulation of alkylation repair enzymes.
[19] The third type of DNA damage reversed by cells is certain methylation of
the bases cytosine and adenine.
**** Single-strand damage[edit] ****
Structure of the base-excision repair enzyme uracil-DNA_glycosylase excising a
hydrolytically-produced uracil residue from DNA. The uracil residue is shown in
yellow.
When only one of the two strands of a double helix has a defect, the other
strand can be used as a template to guide the correction of the damaged strand.
In order to repair damage to one of the two paired molecules of DNA, there
exist a number of excision_repair mechanisms that remove the damaged nucleotide
and replace it with an undamaged nucleotide complementary to that found in the
undamaged DNA strand.[18]
   1. Base_excision_repair (BER) repairs damage to a single nitrogenous_base by
      deploying enzymes called glycosylases.[20] These enzymes remove a single
      nitrogenous base to create an apurinic or apyrimidinic site (AP_site).
      [20] Enzymes called AP_endonucleases nick the damaged DNA backbone at the
      AP site. DNA polymerase then removes the damaged region using its 5â to
      3â exonuclease activity and correctly synthesizes the new strand using
      the complementary strand as a template.[20]
   2. Nucleotide_excision_repair (NER) repairs damaged DNA which commonly
      consists of bulky, helix-distorting damage, such as pyrimidine
      dimerization caused by UV light. Damaged regions are removed in 12â24
      nucleotide-long strands in a three-step process which consists of
      recognition of damage, excision of damaged DNA both upstream and
      downstream of damage by endonucleases, and resynthesis of removed DNA
      region.[21] NER is a highly evolutionarily conserved repair mechanism and
      is used in nearly all eukaryotic and prokaryotic cells.[21] In
      prokaryotes, NER is mediated by Uvr_proteins.[21] In eukaryotes, many
      more proteins are involved, although the general strategy is the same.
      [21]
   3. Mismatch_repair systems are present in essentially all cells to correct
      errors that are not corrected by proofreading. These systems consist of
      at least two proteins. One detects the mismatch, and the other recruits
      an endonuclease that cleaves the newly synthesized DNA strand close to
      the region of damage. In E. coli, the proteins involved are the Mut class
      proteins. This is followed by removal of damaged region by an
      exonuclease, resynthesis by DNA polymerase, and nick sealing by DNA
      ligase.[22]
**** Double-strand breaks[edit] ****
Double-strand breaks, in which both strands in the double helix are severed,
are particularly hazardous to the cell because they can lead to genome
rearrangements. It was noted in some studies that double-strand breaks and a
"cross-linkage joining both strands at the same point is irreparable because
neither strand can then serve as a template for repair. The cell will die in
the next mitosis or in some rare instances, mutate."[3][4] Three mechanisms
exist to repair double-strand breaks (DSBs): non-homologous_end_joining (NHEJ),
microhomology-mediated_end_joining (MMEJ), and homologous_recombination (HR).
[18][23] In an in vitro system, MMEJ occurred in mammalian cells at the levels
of 10â20% of HR when both HR and NHEJ mechanisms were also available.[24]
DNA ligase, shown above repairing chromosomal damage, is an enzyme that joins
broken nucleotides together by catalyzing the formation of an internucleotide
ester bond between the phosphate backbone and the deoxyribose nucleotides.
In NHEJ, DNA_Ligase_IV, a specialized DNA_ligase that forms a complex with the
cofactor XRCC4, directly joins the two ends.[25] To guide accurate repair, NHEJ
relies on short homologous sequences called microhomologies present on the
single-stranded tails of the DNA ends to be joined. If these overhangs are
compatible, repair is usually accurate.[26][27][28][29] NHEJ can also introduce
mutations during repair. Loss of damaged nucleotides at the break site can lead
to deletions, and joining of nonmatching termini forms insertions or
translocations. NHEJ is especially important before the cell has replicated its
DNA, since there is no template available for repair by homologous
recombination. There are "backup" NHEJ pathways in higher eukaryotes.[30]
Besides its role as a genome caretaker, NHEJ is required for joining hairpin-
capped double-strand breaks induced during V(D)J_recombination, the process
that generates diversity in B-cell and T-cell_receptors in the vertebrate
immune_system.[31]
Homologous recombination requires the presence of an identical or nearly
identical sequence to be used as a template for repair of the break. The
enzymatic machinery responsible for this repair process is nearly identical to
the machinery responsible for chromosomal_crossover during meiosis. This
pathway allows a damaged chromosome to be repaired using a sister chromatid
(available in G2 after DNA replication) or a homologous_chromosome as a
template. DSBs caused by the replication machinery attempting to synthesize
across a single-strand break or unrepaired lesion cause collapse of the
replication_fork and are typically repaired by recombination.
MMEJ starts with short-range end resection by MRE11 nuclease on either side of
a double-strand break to reveal microhomology regions.[24] In further steps,
[32] Poly_(ADP-ribose)_polymerase_1 (PARP1) is required and may be an early
step in MMEJ. There is pairing of microhomology regions followed by recruitment
of flap_structure-specific_endonuclease_1 (FEN1) to remove overhanging flaps.
This is followed by recruitment of XRCC1âLIG3 to the site for ligating the
DNA ends, leading to an intact DNA. MMEJ is always accompanied by a deletion,
so that MMEJ is a mutagenic pathway for DNA repair.[33]
The extremophile Deinococcus_radiodurans has a remarkable ability to survive
DNA damage from ionizing_radiation and other sources. At least two copies of
the genome, with random DNA breaks, can form DNA fragments through annealing.
Partially overlapping fragments are then used for synthesis of homologous
regions through a moving D-loop that can continue extension until they find
complementary partner strands. In the final step there is crossover by means of
RecA-dependent homologous_recombination.[34]
Topoisomerases introduce both single- and double-strand breaks in the course of
changing the DNA's state of supercoiling, which is especially common in regions
near an open replication fork. Such breaks are not considered DNA damage
because they are a natural intermediate in the topoisomerase biochemical
mechanism and are immediately repaired by the enzymes that created them.
**** Translesion synthesis[edit] ****
Translesion synthesis (TLS) is a DNA damage tolerance process that allows the
DNA_replication machinery to replicate past DNA lesions such as thymine_dimers
or AP_sites.[35] It involves switching out regular DNA_polymerases for
specialized translesion polymerases (i.e. DNA polymerase IV or V, from the Y
Polymerase family), often with larger active sites that can facilitate the
insertion of bases opposite damaged nucleotides. The polymerase switching is
thought to be mediated by, among other factors, the post-translational
modification of the replication processivity factor PCNA. Translesion synthesis
polymerases often have low fidelity (high propensity to insert wrong bases) on
undamaged templates relative to regular polymerases. However, many are
extremely efficient at inserting correct bases opposite specific types of
damage. For example, Pol_Î· mediates error-free bypass of lesions induced by UV
irradiation, whereas Pol_Î¹ introduces mutations at these sites. Pol Î· is
known to add the first adenine across the T^T_photodimer using Watson-Crick
base_pairing and the second adenine will be added in its syn conformation using
Hoogsteen_base_pairing. From a cellular perspective, risking the introduction
of point_mutations during translesion synthesis may be preferable to resorting
to more drastic mechanisms of DNA repair, which may cause gross chromosomal
aberrations or cell death. In short, the process involves specialized
polymerases either bypassing or repairing lesions at locations of stalled DNA
replication. For example, Human DNA polymerase eta can bypass complex DNA
lesions like guanine-thymine intra-strand crosslink, G[8,5-Me]T, although it
can cause targeted and semi-targeted mutations.[36] Paromita Raychaudhury and
Ashis Basu[37] studied the toxicity and mutagenesis of the same lesion in
Escherichia coli by replicating a G[8,5-Me]T-modified plasmid in E. coli with
specific DNA polymerase knockouts. Viability was very low in a strain lacking
pol II, pol IV, and pol V, the three SOS-inducible DNA polymerases, indicating
that translesion synthesis is conducted primarily by these specialized DNA
polymerases. A bypass platform is provided to these polymerases by
Proliferating_cell_nuclear_antigen (PCNA). Under normal circumstances, PCNA
bound to polymerases replicates the DNA. At a site of lesion, PCNA is
ubiquitinated, or modified, by the RAD6/RAD18 proteins to provide a platform
for the specialized polymerases to bypass the lesion and resume DNA
replication.[38][39] After translesion synthesis, extension is required. This
extension can be carried out by a replicative polymerase if the TLS is error-
free, as in the case of Pol Î·, yet if TLS results in a mismatch, a specialized
polymerase is needed to extend it; Pol_Î¶. Pol Î¶ is unique in that it can
extend terminal mismatches, whereas more processive polymerases cannot. So when
a lesion is encountered, the replication fork will stall, PCNA will switch from
a processive polymerase to a TLS polymerase such as Pol Î¹ to fix the lesion,
then PCNA may switch to Pol Î¶ to extend the mismatch, and last PCNA will
switch to the processive polymerase to continue replication.
***** Global response to DNA damage[edit] *****
Cells exposed to ionizing_radiation, ultraviolet_light or chemicals are prone
to acquire multiple sites of bulky DNA lesions and double-strand breaks.
Moreover, DNA damaging agents can damage other biomolecules such as proteins,
carbohydrates, lipids, and RNA. The accumulation of damage, to be specific,
double-strand breaks or adducts stalling the replication_forks, are among known
stimulation signals for a global response to DNA damage.[40] The global
response to damage is an act directed toward the cells' own preservation and
triggers multiple pathways of macromolecular repair, lesion bypass, tolerance,
or apoptosis. The common features of global response are induction of multiple
genes, cell_cycle arrest, and inhibition of cell_division.
**** Initial steps[edit] ****
The packaging of eukaryotic DNA into chromatin presents a barrier to all DNA-
based processes that require recruitment of enzymes to their sites of action.
To allow DNA repair, the chromatin must be remodeled. In eukaryotes, ATP
dependent chromatin_remodeling complexes and histone-modifying_enzymes are two
predominant factors employed to accomplish this remodeling process.[41]
Chromatin relaxation occurs rapidly at the site of a DNA damage.[42][43] In one
of the earliest steps, the stress-activated protein kinase, c-Jun_N-terminal
kinase_(JNK), phosphorylates SIRT6 on serine 10 in response to double-strand
breaks or other DNA damage.[44] This post-translational_modification
facilitates the mobilization of SIRT6 to DNA damage sites, and is required for
efficient recruitment of poly (ADP-ribose) polymerase 1 (PARP1) to DNA break
sites and for efficient repair of DSBs.[44] PARP1 protein starts to appear at
DNA damage sites in less than a second, with half maximum accumulation within
1.6 seconds after the damage occurs.[45] PARP1 synthesizes polymeric adenosine
diphosphate_ribose (poly (ADP-ribose) or PAR) chains on itself. Next the
chromatin remodeler ALC1 quickly attaches to the product of PARP1 action, a
poly-ADP ribose chain, and ALC1 completes arrival at the DNA damage within 10
seconds of the occurrence of the damage.[43] About half of the maximum
chromatin relaxation, presumably due to action of ALC1, occurs by 10 seconds.
[43] This then allows recruitment of the DNA repair enzyme MRE11, to initiate
DNA repair, within 13 seconds.[45]
Î³H2AX, the phosphorylated form of H2AX is also involved in the early steps
leading to chromatin decondensation after DNA double-strand breaks. The histone
variant H2AX constitutes about 10% of the H2A histones in human chromatin.[46]
Î³H2AX (H2AX phosphorylated on serine 139) can be detected as soon as 20
seconds after irradiation of cells (with DNA double-strand break formation),
and half maximum accumulation of Î³H2AX occurs in one minute.[46] The extent of
chromatin with phosphorylated Î³H2AX is about two million base pairs at the
site of a DNA double-strand break.[46] Î³H2AX does not, itself, cause chromatin
decondensation, but within 30 seconds of irradiation, RNF8 protein can be
detected in association with Î³H2AX.[47] RNF8 mediates extensive chromatin
decondensation, through its subsequent interaction with CHD4,[48] a component
of the nucleosome remodeling and deacetylase complex NuRD.
DDB2 occurs in a heterodimeric complex with DDB1. This complex further
complexes with the ubiquitin_ligase protein CUL4A[49] and with PARP1.[50] This
larger complex rapidly associates with UV-induced damage within chromatin, with
half-maximum association completed in 40 seconds.[49] The PARP1 protein,
attached to both DDB1 and DDB2, then PARylates (creates a poly-ADP ribose
chain) on DDB2 that attracts the DNA remodeling protein ALC1.[50] Action of
ALC1 relaxes the chromatin at the site of UV damage to DNA. This relaxation
allows other proteins in the nucleotide_excision_repair pathway to enter the
chromatin and repair UV-induced cyclobutane_pyrimidine_dimer damages.
After rapid chromatin_remodeling, cell_cycle checkpoints are activated to allow
DNA repair to occur before the cell cycle progresses. First, two kinases, ATM
and ATR are activated within 5 or 6 minutes after DNA is damaged. This is
followed by phosphorylation of the cell cycle checkpoint protein Chk1,
initiating its function, about 10 minutes after DNA is damaged.[51]
**** DNA damage checkpoints[edit] ****
After DNA damage, cell_cycle checkpoints are activated. Checkpoint activation
pauses the cell cycle and gives the cell time to repair the damage before
continuing to divide. DNA damage checkpoints occur at the G1/S and G2/
M boundaries. An intra-S checkpoint also exists. Checkpoint activation is
controlled by two master kinases, ATM and ATR. ATM responds to DNA double-
strand breaks and disruptions in chromatin structure,[52] whereas ATR primarily
responds to stalled replication_forks. These kinases phosphorylate downstream
targets in a signal_transduction cascade, eventually leading to cell cycle
arrest. A class of checkpoint mediator proteins including BRCA1, MDC1, and
53BP1 has also been identified.[53] These proteins seem to be required for
transmitting the checkpoint activation signal to downstream proteins.
DNA damage checkpoint is a signal_transduction_pathway that blocks cell_cycle
progression in G1, G2 and metaphase and slows down the rate of S phase
progression when DNA is damaged. It leads to a pause in cell cycle allowing the
cell time to repair the damage before continuing to divide.
Checkpoint Proteins can be separated into four groups: phosphatidylinositol_3-
kinase (PI3K)-like protein_kinase, proliferating_cell_nuclear_antigen (PCNA)-
like group, two serine/threonine(S/T) kinases and their adaptors. Central to
all DNA damage induced checkpoints responses is a pair of large protein kinases
belonging to the first group of PI3K-like protein kinases-the ATM (Ataxia
telangiectasia_mutated) and ATR (Ataxia- and Rad-related) kinases, whose
sequence and functions have been well conserved in evolution. All DNA damage
response requires either ATM or ATR because they have the ability to bind to
the chromosomes at the site of DNA damage, together with accessory proteins
that are platforms on which DNA damage response components and DNA repair
complexes can be assembled.
An important downstream target of ATM and ATR is p53, as it is required for
inducing apoptosis following DNA damage.[54] The cyclin-dependent_kinase
inhibitor p21 is induced by both p53-dependent and p53-independent mechanisms
and can arrest the cell cycle at the G1/S and G2/M checkpoints by deactivating
cyclin/cyclin-dependent_kinase complexes.[55]
**** The prokaryotic SOS response[edit] ****
The SOS_response is the changes in gene_expression in Escherichia_coli and
other bacteria in response to extensive DNA damage. The prokaryotic SOS system
is regulated by two key proteins: LexA and RecA. The LexA homodimer is a
transcriptional repressor that binds to operator sequences commonly referred to
as SOS boxes. In Escherichia_coli it is known that LexA regulates transcription
of approximately 48 genes including the lexA and recA genes.[56] The SOS
response is known to be widespread in the Bacteria domain, but it is mostly
absent in some bacterial phyla, like the Spirochetes.[57] The most common
cellular signals activating the SOS response are regions of single-stranded DNA
(ssDNA), arising from stalled replication_forks or double-strand breaks, which
are processed by DNA_helicase to separate the two DNA strands.[40] In the
initiation step, RecA protein binds to ssDNA in an ATP_hydrolysis driven
reaction creating RecAâssDNA filaments. RecAâssDNA filaments activate LexA
autoprotease activity, which ultimately leads to cleavage of LexA dimer and
subsequent LexA degradation. The loss of LexA repressor induces transcription
of the SOS genes and allows for further signal induction, inhibition of cell
division and an increase in levels of proteins responsible for damage
processing.
In Escherichia coli, SOS boxes are 20-nucleotide long sequences near promoters
with palindromic structure and a high degree of sequence conservation. In other
classes and phyla, the sequence of SOS boxes varies considerably, with
different length and composition, but it is always highly conserved and one of
the strongest short signals in the genome.[57] The high information content of
SOS boxes permits differential binding of LexA to different promoters and
allows for timing of the SOS response. The lesion repair genes are induced at
the beginning of SOS response. The error-prone translesion polymerases, for
example, UmuCD'2 (also called DNA polymerase V), are induced later on as a last
resort.[58] Once the DNA damage is repaired or bypassed using polymerases or
through recombination, the amount of single-stranded DNA in cells is decreased,
lowering the amounts of RecA filaments decreases cleavage activity of LexA
homodimer, which then binds to the SOS boxes near promoters and restores normal
gene expression.
**** Eukaryotic transcriptional responses to DNA damage[edit] ****
Eukaryotic cells exposed to DNA damaging agents also activate important
defensive pathways by inducing multiple proteins involved in DNA repair, cell
cycle_checkpoint control, protein trafficking and degradation. Such genome wide
transcriptional response is very complex and tightly regulated, thus allowing
coordinated global response to damage. Exposure of yeast Saccharomyces
cerevisiae to DNA damaging agents results in overlapping but distinct
transcriptional profiles. Similarities to environmental shock_response
indicates that a general global stress response pathway exist at the level of
transcriptional activation. In contrast, different human cell types respond to
damage differently indicating an absence of a common global response. The
probable explanation for this difference between yeast and human cells may be
in the heterogeneity of mammalian cells. In an animal different types of cells
are distributed among different organs that have evolved different
sensitivities to DNA damage.[59]
In general global response to DNA damage involves expression of multiple genes
responsible for postreplication_repair, homologous recombination, nucleotide
excision repair, DNA_damage_checkpoint, global transcriptional activation,
genes controlling mRNA decay, and many others. A large amount of damage to a
cell leaves it with an important decision: undergo apoptosis and die, or
survive at the cost of living with a modified genome. An increase in tolerance
to damage can lead to an increased rate of survival that will allow a greater
accumulation of mutations. Yeast Rev1 and human polymerase Î· are members of [Y
family translesion DNA polymerases present during global response to DNA damage
and are responsible for enhanced mutagenesis during a global response to DNA
damage in eukaryotes.[40]
***** Aging[edit] *****
Main article: DNA_damage_theory_of_aging
**** Pathological effects of poor DNA repair[edit] ****
DNA repair rate is an important determinant of cell pathology
Experimental animals with genetic deficiencies in DNA repair often show
decreased life span and increased cancer incidence.[15] For example, mice
deficient in the dominant NHEJ pathway and in telomere maintenance mechanisms
get lymphoma and infections more often, and, as a consequence, have shorter
lifespans than wild-type mice.[60] In similar manner, mice deficient in a key
repair and transcription protein that unwinds DNA helices have premature onset
of aging-related diseases and consequent shortening of lifespan.[61] However,
not every DNA repair deficiency creates exactly the predicted effects; mice
deficient in the NER pathway exhibited shortened life span without
correspondingly higher rates of mutation.[62]
If the rate of DNA damage exceeds the capacity of the cell to repair it, the
accumulation of errors can overwhelm the cell and result in early senescence,
apoptosis, or cancer. Inherited diseases associated with faulty DNA repair
functioning result in premature aging,[15] increased sensitivity to
carcinogens, and correspondingly increased cancer risk (see below). On the
other hand, organisms with enhanced DNA repair systems, such as Deinococcus
radiodurans, the most radiation-resistant known organism, exhibit remarkable
resistance to the double-strand break-inducing effects of radioactivity, likely
due to enhanced efficiency of DNA repair and especially NHEJ.[63]
**** Longevity and caloric restriction[edit] ****
Most life span influencing genes affect the rate of DNA damage
A number of individual genes have been identified as influencing variations in
life span within a population of organisms. The effects of these genes is
strongly dependent on the environment, in particular, on the organism's diet.
Caloric_restriction reproducibly results in extended lifespan in a variety of
organisms, likely via nutrient_sensing pathways and decreased metabolic_rate.
The molecular mechanisms by which such restriction results in lengthened
lifespan are as yet unclear (see[64] for some discussion); however, the
behavior of many genes known to be involved in DNA repair is altered under
conditions of caloric restriction. Several agents reported to have anti-aging
properties have been shown to attenuate constitutive level of mTOR signaling,
an evidence of reduction of metabolic_activity, and concurrently to reduce
constitutive level of DNA_damage induced by endogenously generated reactive
oxygen species.[65]
For example, increasing the gene_dosage of the gene SIR-2, which regulates DNA
packaging in the nematode worm Caenorhabditis_elegans, can significantly extend
lifespan.[66] The mammalian homolog of SIR-2 is known to induce downstream DNA
repair factors involved in NHEJ, an activity that is especially promoted under
conditions of caloric restriction.[67] Caloric restriction has been closely
linked to the rate of base excision repair in the nuclear DNA of rodents,[68]
although similar effects have not been observed in mitochondrial DNA.[69]
The C. elegans gene AGE-1, an upstream effector of DNA repair pathways, confers
dramatically extended life span under free-feeding conditions but leads to a
decrease in reproductive fitness under conditions of caloric restriction.[70]
This observation supports the pleiotropy theory of the biological_origins_of
aging, which suggests that genes conferring a large survival advantage early in
life will be selected for even if they carry a corresponding disadvantage late
in life.
***** Medicine and DNA repair modulation[edit] *****
Main article: DNA_repair-deficiency_disorder
**** Hereditary DNA repair disorders[edit] ****
Defects in the NER mechanism are responsible for several genetic disorders,
including:
    * Xeroderma_pigmentosum: hypersensitivity to sunlight/UV, resulting in
      increased skin cancer incidence and premature aging
    * Cockayne_syndrome: hypersensitivity to UV and chemical agents
    * Trichothiodystrophy: sensitive skin, brittle hair and nails
Mental retardation often accompanies the latter two disorders, suggesting
increased vulnerability of developmental neurons.
Other DNA repair disorders include:
    * Werner's_syndrome: premature aging and retarded growth
    * Bloom's_syndrome: sunlight hypersensitivity, high incidence of
      malignancies (especially leukemias).
    * Ataxia_telangiectasia: sensitivity to ionizing radiation and some
      chemical agents
All of the above diseases are often called "segmental progerias" ("accelerated
aging_diseases") because their victims appear elderly and suffer from aging-
related diseases at an abnormally young age, while not manifesting all the
symptoms of old age.
Other diseases associated with reduced DNA repair function include Fanconi
anemia, hereditary breast_cancer and hereditary colon_cancer.
***** Cancer[edit] *****
Because of inherent limitations in the DNA repair mechanisms, if humans lived
long enough, they would all eventually develop cancer.[71][72] There are at
least 34 Inherited_human_DNA_repair_gene_mutations_that_increase_cancer_risk.
Many of these mutations cause DNA repair to be less effective than normal. In
particular, Hereditary_nonpolyposis_colorectal_cancer (HNPCC) is strongly
associated with specific mutations in the DNA mismatch repair pathway. BRCA1
and BRCA2, two important genes whose mutations confer a hugely increased risk
of breast cancer on carriers,[73] are both associated with a large number of
DNA repair pathways, especially NHEJ and homologous recombination.
Cancer therapy procedures such as chemotherapy and radiotherapy work by
overwhelming the capacity of the cell to repair DNA damage, resulting in cell
death. Cells that are most rapidly dividing â most typically cancer cells â
are preferentially affected. The side-effect is that other non-cancerous but
rapidly dividing cells such as progenitor cells in the gut, skin, and
hematopoietic system are also affected. Modern cancer treatments attempt to
localize the DNA damage to cells and tissues only associated with cancer,
either by physical means (concentrating the therapeutic agent in the region of
the tumor) or by biochemical means (exploiting a feature unique to cancer cells
in the body). In the context of therapies targeting DNA damage response genes,
the latter approach has been termed 'synthetic lethality'.[74]
Perhaps the most well-known of these 'synthetic lethality' drugs is the poly
(ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, which was approved by the
Food and Drug Administration in 2015 for the treatment in women of BRCA-
defective ovarian cancer. Tumor cells with partial loss of DNA damage response
(specifically, homologous_recombination repair) are dependent on another
mechanism â single-strand break repair â which is a mechanism consisting,
in part, of the PARP1 gene product.[75] Olaparib is combined with
chemotherapeutics to inhibit single-strand break repair induced by DNA damage
caused by the co-administered chemotherapy. Tumor cells relying on this
residual DNA repair mechanism are unable to repair the damage and hence are not
able to survive and proliferate, whereas normal cells can repair the damage
with the functioning homologous recombination mechanism.
Many other drugs for use against other residual DNA repair mechanisms commonly
found in cancer are currently under investigation. However, synthetic lethality
therapeutic approaches have been questioned due to emerging evidence of
acquired resistance, achieved through rewiring of DNA damage response pathways
and reversion of previously-inhibited defects.[76]
**** DNA repair defects in cancer[edit] ****
It has become apparent over the past several years that the DNA damage response
acts as a barrier to the malignant transformation of preneoplastic cells.[77]
Previous studies have shown an elevated DNA damage response in cell-culture
models with oncogene activation[78] and preneoplastic colon adenomas.[79] DNA
damage response mechanisms trigger cell-cycle arrest, and attempt to repair DNA
lesions or promote cell death/senescence if repair is not possible. Replication
stress is observed in preneoplastic cells due to increased proliferation
signals from oncogenic mutations. Replication stress is characterized by:
increased replication initiation/origin firing; increased transcription and
collisions of transcription-replication complexes; nucleotide deficiency;
increase in reactive oxygen species (ROS).[80]
Replication stress, along with the selection for inactivating mutations in DNA
damage response genes in the evolution of the tumor,[81] leads to
downregulation and/or loss of some DNA damage response mechanisms, and hence
loss of DNA repair and/or senescence/programmed cell death. In experimental
mouse models, loss of DNA damage response-mediated cell senescence was observed
after using a short_hairpin_RNA (shRNA) to inhibit the double-strand break
response kinase ataxia telangiectasia (ATM), leading to increased tumor size
and invasiveness.[79] Humans born with inherited defects in DNA repair
mechanisms (for example, Li-Fraumeni_syndrome) have a higher cancer risk.[82]
The prevalence of DNA damage response mutations differs across cancer types;
for example, 30% of breast invasive carcinomas have mutations in genes involved
in homologous recombination.[77] In cancer, downregulation is observed across
all DNA damage response mechanisms (base excision repair (BER), nucleotide
excision repair (NER), DNA mismatch repair (MMR), homologous recombination
repair (HR), non-homologous end joining (NHEJ) and translesion DNA synthesis
(TLS).[83] As well as mutations to DNA damage repair genes, mutations also
arise in the genes responsible for arresting the cell_cycle to allow sufficient
time for DNA repair to occur, and some genes are involved in both DNA damage
repair and cell cycle checkpoint control, for example ATM and checkpoint kinase
2 (CHEK2) â a tumor suppressor that is often absent or downregulated in non-
small cell lung cancer.[84]
      HR NHEJ SSA FA BER NER MMR
ATM   x  x    x
ATR   x  x    x
PAXIP x  x
RPA   x       x          x
BRCA1 x           x
BRCA2 x           x
RAD51 x           x
RFC   x              x   x
XRCC1                x   x
PCNA                 x   x   x
PARP1    x           x
ERCC1 x       x   x      x
MSH3  x       x              x
Table: Genes involved in DNA damage response pathways and frequently mutated in
cancer (HR = homologous recombination; NHEJ = non-homologous end joining; SSA =
single-strand annealing; FA = fanconi anemia pathway; BER = base excision
repair; NER = nucleotide excision repair; MMR = mismatch repair)
**** Epigenetic DNA repair defects in cancer[edit] ****
Classically, cancer has been viewed as a set of diseases that are driven by
progressive genetic abnormalities that include mutations in tumour-suppressor
genes and oncogenes, and chromosomal aberrations. However, it has become
apparent that cancer is also driven by epigenetic_alterations.[85]
Epigenetic alterations refer to functionally relevant modifications to the
genome that do not involve a change in the nucleotide sequence. Examples of
such modifications are changes in DNA_methylation (hypermethylation and
hypomethylation) and histone_modification,[86] changes in chromosomal
architecture (caused by inappropriate expression of proteins such as HMGA2 or
HMGA1)[87] and changes caused by microRNAs. Each of these epigenetic
alterations serves to regulate gene expression without altering the underlying
DNA_sequence. These changes usually remain through cell_divisions, last for
multiple cell generations, and can be considered to be epimutations (equivalent
to mutations).
While large numbers of epigenetic alterations are found in cancers, the
epigenetic alterations in DNA repair genes, causing reduced expression of DNA
repair proteins, appear to be particularly important. Such alterations are
thought to occur early in progression to cancer and to be a likely cause of the
genetic instability characteristic of cancers.[88][89][90][91]
Reduced expression of DNA repair genes causes deficient DNA repair. When DNA
repair is deficient DNA damages remain in cells at a higher than usual level
and these excess damages cause increased frequencies of mutation or
epimutation. Mutation rates increase substantially in cells defective in DNA
mismatch_repair[92][93] or in homologous_recombinational repair (HRR).[94]
Chromosomal rearrangements and aneuploidy also increase in HRR defective cells.
[95]
Higher levels of DNA damage not only cause increased mutation, but also cause
increased epimutation. During repair of DNA double strand breaks, or repair of
other DNA damages, incompletely cleared sites of repair can cause epigenetic
gene silencing.[96][97]
Deficient expression of DNA repair proteins due to an inherited mutation can
cause increased risk of cancer. Individuals with an inherited impairment in any
of 34 DNA repair genes (see article DNA_repair-deficiency_disorder) have an
increased risk of cancer, with some defects causing up to a 100% lifetime
chance of cancer (e.g. p53 mutations).[98] However, such germline mutations
(which cause highly penetrant cancer syndromes) are the cause of only about 1
percent of cancers.[99]
**** Frequencies of epimutations in DNA repair genes[edit] ****
A chart of common DNA damaging agents, examples of lesions they cause in DNA,
and pathways used to repair these lesions. Also shown are many of the genes in
these pathways, an indication of which genes are epigenetically regulated to
have reduced (or increased) expression in various cancers. It also shows genes
in the error prone microhomology-mediated end joining pathway with increased
expression in various cancers.
Deficiencies in DNA repair enzymes are occasionally caused by a newly arising
somatic mutation in a DNA repair gene, but are much more frequently caused by
epigenetic alterations that reduce or silence expression of DNA repair genes.
For example, when 113 colorectal cancers were examined in sequence, only four
had a missense_mutation in the DNA repair gene MGMT, while the majority had
reduced MGMT expression due to methylation of the MGMT promoter region (an
epigenetic alteration).[100] Five different studies found that between 40% and
90% of colorectal cancers have reduced MGMT expression due to methylation of
the MGMT promoter region.[101][102][103][104][105]
Similarly, out of 119 cases of mismatch repair-deficient colorectal cancers
that lacked DNA repair gene PMS2 expression, PMS2 was deficient in 6 due to
mutations in the PMS2 gene, while in 103 cases PMS2 expression was deficient
because its pairing partner MLH1 was repressed due to promoter methylation
(PMS2 protein is unstable in the absence of MLH1).[106] In the other 10 cases,
loss of PMS2 expression was likely due to epigenetic overexpression of the
microRNA, miR-155, which down-regulates MLH1.[107]
In further examples (tabulated in Table 4 of this reference[108]), epigenetic
defects were found at frequencies of between 13%â100% for the DNA repair
genes BRCA1, WRN, FANCB, FANCF, MGMT, MLH1, MSH2, MSH4, ERCC1, XPF, NEIL1 and
ATM. These epigenetic defects occurred in various cancers (e.g. breast,
ovarian, colorectal and head and neck). Two or three deficiencies in the
expression of ERCC1, XPF or PMS2 occur simultaneously in the majority of the 49
colon cancers evaluated by Facista et al.[109]
The chart in this section shows some frequent DNA damaging agents, examples of
DNA lesions they cause, and the pathways that deal with these DNA damages. At
least 169 enzymes are either directly employed in DNA repair or influence DNA
repair processes.[110] Of these, 83 are directly employed in repairing the 5
types of DNA damages illustrated in the chart.
Some of the more well studied genes central to these repair processes are shown
in the chart. The gene designations shown in red, gray or cyan indicate genes
frequently epigenetically altered in various types of cancers. Wikipedia
articles on each of the genes highlighted by red, gray or cyan describe the
epigenetic alteration(s) and the cancer(s) in which these epimutations are
found. Four review articles,[108][111][112][113] and two broad experimental
survey articles[114][115] also document most of these epigenetic DNA repair
deficiencies in cancers.
Red-highlighted genes are frequently reduced or silenced by epigenetic
mechanisms in various cancers. When these genes have low or absent expression,
DNA damages can accumulate. Replication errors past these damages (see
translesion_synthesis) can lead to increased mutations and, ultimately, cancer.
Epigenetic repression of DNA repair genes in accurate DNA repair pathways
appear to be central to carcinogenesis.
The two gray-highlighted genes RAD51 and BRCA2, are required for homologous
recombinational repair. They are sometimes epigenetically over-expressed and
sometimes under-expressed in certain cancers. As indicated in the Wikipedia
articles on RAD51 and BRCA2, such cancers ordinarily have epigenetic
deficiencies in other DNA repair genes. These repair deficiencies would likely
cause increased unrepaired DNA damages. The over-expression of RAD51 and BRCA2
seen in these cancers may reflect selective pressures for compensatory RAD51 or
BRCA2 over-expression and increased homologous recombinational repair to at
least partially deal with such excess DNA damages. In those cases where RAD51
or BRCA2 are under-expressed, this would itself lead to increased unrepaired
DNA damages. Replication errors past these damages (see translesion_synthesis)
could cause increased mutations and cancer, so that under-expression of RAD51
or BRCA2 would be carcinogenic in itself.
Cyan-highlighted genes are in the microhomology-mediated_end_joining (MMEJ)
pathway and are up-regulated in cancer. MMEJ is an additional error-prone
inaccurate repair pathway for double-strand breaks. In MMEJ repair of a double-
strand break, an homology of 5â25 complementary base pairs between both
paired strands is sufficient to align the strands, but mismatched ends (flaps)
are usually present. MMEJ removes the extra nucleotides (flaps) where strands
are joined, and then ligates the strands to create an intact DNA double helix.
MMEJ almost always involves at least a small deletion, so that it is a
mutagenic pathway.[116] FEN1, the flap endonuclease in MMEJ, is epigenetically
increased by promoter hypomethylation and is over-expressed in the majority of
cancers of the breast,[117] prostate,[118] stomach,[119][120] neuroblastomas,
[121] pancreas,[122] and lung.[123] PARP1 is also over-expressed when its
promoter region ETS site is epigenetically hypomethylated, and this contributes
to progression to endometrial cancer,[124] BRCA-mutated ovarian cancer,[125]
and BRCA-mutated serous ovarian cancer.[126] Other genes in the MMEJ pathway
are also over-expressed in a number of cancers (see MMEJ for summary), and are
also shown in cyan.
**** Genome-wide distribution of DNA repair in human somatic cells[edit] ****
Differential activity of DNA repair pathways across various regions of the
human genome causes mutations to be very unevenly distributed within tumor
genomes.[127][128] In particular, the gene-rich, early-replicating regions of
the human genome exhibit lower mutation frequencies than the gene-poor, late-
replicating heterochromatin. One mechanism underlying this involves the histone
modification H3K36me3, which can recruit mismatch_repair proteins,[129] thereby
lowering mutation rates in H3K36me3-marked regions.[130] Another important
mechanism concerns nucleotide_excision_repair, which can be recruited by the
transcription machinery, lowering somatic mutation rates in active genes[128]
and other open chromatin regions.[131]
***** Evolution[edit] *****
The basic processes of DNA repair are highly conserved among both prokaryotes
and eukaryotes and even among bacteriophages (viruses which infect bacteria);
however, more complex organisms with more complex genomes have correspondingly
more complex repair mechanisms.[132] The ability of a large number of protein
structural_motifs to catalyze relevant chemical reactions has played a
significant role in the elaboration of repair mechanisms during evolution. For
an extremely detailed review of hypotheses relating to the evolution of DNA
repair, see.[133]
The fossil_record indicates that single-cell life began to proliferate on the
planet at some point during the Precambrian period, although exactly when
recognizably modern life first emerged is unclear. Nucleic_acids became the
sole and universal means of encoding genetic information, requiring DNA repair
mechanisms that in their basic form have been inherited by all extant life
forms from their common ancestor. The emergence of Earth's oxygen-rich
atmosphere (known as the "oxygen_catastrophe") due to photosynthetic organisms,
as well as the presence of potentially damaging free_radicals in the cell due
to oxidative_phosphorylation, necessitated the evolution of DNA repair
mechanisms that act specifically to counter the types of damage induced by
oxidative_stress.
**** Rate of evolutionary change[edit] ****
On some occasions, DNA damage is not repaired, or is repaired by an error-prone
mechanism that results in a change from the original sequence. When this
occurs, mutations may propagate into the genomes of the cell's progeny. Should
such an event occur in a germ_line cell that will eventually produce a gamete,
the mutation has the potential to be passed on to the organism's offspring. The
rate of evolution in a particular species (or, in a particular gene) is a
function of the rate of mutation. As a consequence, the rate and accuracy of
DNA repair mechanisms have an influence over the process of evolutionary
change.[134] DNA damage protection and repair does not influence the rate of
adaptation by gene regulation and by recombination and selection of alleles. On
the other hand, DNA damage repair and protection does influence the rate of
accumulation of irreparable, advantageous, code expanding, inheritable
mutations, and slows down the evolutionary mechanism for expansion of the
genome of organisms with new functionalities. The tension between evolvability
and mutation repair and protection needs further investigation.
***** Technology[edit] *****
A technology named clustered regularly interspaced short palindromic repeat
(shortened to CRISPR-Cas9) was discovered in 2012. The new technology allows
anyone with molecular biology training to alter the genes of any species with
precision.[135]
It is cheaper, more efficient, and more precise than other technologies. With
the help of CRISPRâCas9, parts of a genome can be edited by scientists by
removing, adding, or altering parts in a DNA sequence.
***** See also[edit] *****
    * Molecular_and_Cellular_Biology_portal
    * Accelerated_aging_disease
    * Aging_DNA
    * Cell_cycle
    * DNA_damage_(naturally_occurring)
    * DNA_damage_theory_of_aging
    * DNA_replication
    * Direct_DNA_damage
    * Gene_therapy
    * Human_mitochondrial_genetics
    * Indirect_DNA_damage
    * Life_extension
    * Progeria
    * Senescence
    * SiDNA
    * The scientific journal DNA Repair under Mutation_Research
***** References[edit] *****
   1. ^"Nature_Reviews_Series:_DNA_damage". Nature_Reviews_Molecular_Cell
      Biology. 5 July 2017. Retrieved 7 November 2018.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a bLodish H, Berk A, Matsudaira P, Kaiser CA, Krieger M, Scott MP,
      Zipursky SL, Darnell J (2004). Molecular Biology of the Cell (5th ed.).
      New York: WH Freeman. p. 963.
   4. ^ a bAcharya, PV (1971). "The isolation and partial characterization of
      age-correlated oligo-deoxyribo-ribonucleotides with covalently linked
      aspartyl-glutamyl polypeptides". Johns Hopkins Medical Journal.
      Supplement (1): 254â60. PMID 5055816.
   5. ^ a bBjorksten, J; Acharya, PVN; Ashman, S; Wetlaufer, DB (1971).
      "Gerogenic fractions in the tritiated rat". Journal of the American
      Geriatrics Society. 19 (7): 561â74. doi:10.1111/j.1532-
      5415.1971.tb02577.x. PMID 5106728.
   6. ^Browner, WS; Kahn, AJ; Ziv, E; Reiner, AP; Oshima, J; Cawthon, RM;
      Hsueh, WC; Cummings, SR. (2004). "The genetics of human longevity". Am J
      Med. 117 (11): 851â60. CiteSeerX 10.1.1.556.6874. doi:10.1016/
      j.amjmed.2004.06.033. PMID 15589490.
   7. ^Broad, William J. (7 October 2015). "Nobel_Prize_in_Chemistry_Awarded_to
      Tomas_Lindahl,_Paul_Modrich_and_Aziz_Sancar_for_DNA_Studies". The_New
      York_Times. Retrieved 7 October 2015.
   8. ^Staff (7 October 2015). "The_Nobel_Prize_in_Chemistry_2015_â_DNA
      repair_â_providing_chemical_stability_for_life" (PDF). Nobel_Prize.
      Retrieved 7 October 2015.
   9. ^Roulston A, Marcellus RC, Branton PE (1999). "Viruses and apoptosis".
      Annu. Rev. Microbiol. 53: 577â628. doi:10.1146/annurev.micro.53.1.577.
      PMID 10547702.
  10. ^Madigan MT, Martino JM (2006). Brock Biology of Microorganisms (11th
      ed.). Pearson. p. 136. ISBN 978-0-13-196893-6.
  11. ^Ohta, Toshihiro; Shin-ichi, Tokishita; Mochizuki, Kayo; Kawase, Jun;
      Sakahira, Masahide; Yamagata, Hideo (2006). "UV Sensitivity and
      Mutagenesis of the Extremely Thermophilic Eubacterium Thermus
      thermophilus HB27". Genes and Environment. 28 (2): 56â61. doi:10.3123/
      jemsge.28.56.
  12. ^ Tanaka T, Halicka HD, Huang X, Traganos F, Darzynkiewicz Z. (2006)
      Constitutive histone H2AX phosphorylation and ATM activation, the
      reporters of DNA damage by endogenous oxidants. Cell Cycle, 5:1940-5,
  13. PMID 16940754
  14. ^Braig, M; Schmitt, CA. (2006). "Oncogene-induced senescence: putting the
      brakes on tumor development". Cancer Res. 66 (6): 2881â2884. doi:
      10.1158/0008-5472.CAN-05-4006. PMID 16540631.
  15. ^Lynch, MD. (2006). "How does cellular senescence prevent cancer?". DNA
      Cell Biol. 25 (2): 69â78. doi:10.1089/dna.2006.25.69. PMID 16460230.
  16. ^Campisi J, d'Adda di Fagagna F (2007). "Cellular senescence: when bad
      things happen to good cells". Rev Mol Cell Biol. 8 (9): 729â40. doi:
      10.1038/nrm2233. PMID 17667954.
  17. ^ a b cBest, Benjamin P (2009). "Nuclear_DNA_damage_as_a_direct_cause_of
      aging" (PDF). Rejuvenation Research. 12 (3): 199â208.
      CiteSeerX 10.1.1.318.738. doi:10.1089/rej.2009.0847. PMID 19594328.
  18. ^Sancar, A. (2003). "Structure and function of DNA photolyase and
      cryptochrome blue-light photoreceptors". Chem Rev. 103 (6): 2203â37.
      doi:10.1021/cr0204348. PMID 12797829.
  19. ^Michael Lynch, JosÃ© Ignacio Lucas-LledÃ³; Lynch, M. (19 February 2009).
      "Evolution_of_Mutation_Rates:_Phylogenomic_Analysis_of_the_Photolyase/
      Cryptochrome_Family". Molecular Biology and Evolution. 26 (5):
      1143â1153. doi:10.1093/molbev/msp029. PMC 2668831. PMID 19228922.
  20. ^ a b cWatson JD, Baker TA, Bell SP, Gann A, Levine M, Losick R (2004).
      Molecular Biology of the Gene (5th ed.). Pearson Benjamin Cummings; CSHL
      Press. Ch. 9, 10. OCLC 936762772.
  21. ^Volkert MR (1988). "Adaptive response of Escherichia coli to alkylation
      damage". Environmental and Molecular Mutagenesis. 11 (2): 241â55. doi:
      10.1002/em.2850110210. PMID 3278898.
  22. ^ a b cWilley, J; Sherwood, L; Woolverton, C (2014). Prescott's
      Microbiology. New York, New York: McGraw Hill. p. 381. ISBN 978-0-07-
      3402-40-6.
  23. ^ a b c dReardon, J; Sancar, A (2006). Purification and Characterization
      ofEscherichia coliand Human Nucleotide Excision Repair Enzyme Systems.
      Methods in Enzymology. 408. pp. 189â213. doi:10.1016/S0076-6879
      (06)08012-8. ISBN 9780121828134. PMID 16793370.
  24. ^Berg, M; Tymoczko, J; Stryer, L (2012). Biochemistry 7th edition. New
      York: W.H. Freeman and Company. p. 840. ISBN 9781429229364.
  25. ^Liang L, Deng L, Chen Y, Li GC, Shao C, Tischfield JA (2005).
      "Modulation of DNA end joining by nuclear proteins". J. Biol. Chem. 280
      (36): 31442â9. doi:10.1074/jbc.M503776200. PMID 16012167.
  26. ^ a bTruong LN, Li Y, Shi LZ, Hwang PY, He J, Wang H, Razavian N, Berns
      MW, Wu X (2013). "Microhomology-mediated_End_Joining_and_Homologous
      Recombination_share_the_initial_end_resection_step_to_repair_DNA_double-
      strand_breaks_in_mammalian_cells". Proc. Natl. Acad. Sci. U.S.A. 110
      (19): 7720â5. doi:10.1073/pnas.1213431110. PMC 3651503. PMID 23610439.
  27. ^Wilson TE, Grawunder U, Lieber MR (July 1997). "Yeast DNA ligase IV
      mediates non-homologous DNA end joining". Nature. 388 (6641): 495â8.
      doi:10.1038/41365. PMID 9242411.
  28. ^Moore JK, Haber JE (May 1996). "Cell_cycle_and_genetic_requirements_of
      two_pathways_of_nonhomologous_end-joining_repair_of_double-strand_breaks
      in_Saccharomyces_cerevisiae". Molecular and Cellular Biology. 16 (5):
      2164â73. doi:10.1128/mcb.16.5.2164. PMC 231204. PMID 8628283.
  29. ^Boulton SJ, Jackson SP (September 1996). "Saccharomyces_cerevisiae_Ku70
      potentiates_illegitimate_DNA_double-strand_break_repair_and_serves_as_a
      barrier_to_error-prone_DNA_repair_pathways". The EMBO Journal. 15 (18):
      5093â103. doi:10.1002/j.1460-2075.1996.tb00890.x. PMC 452249.
      PMID 8890183.
  30. ^Wilson TE, Lieber MR (August 1999). "Efficient processing of DNA ends
      during yeast nonhomologous end joining. Evidence for a DNA polymerase
      beta (Pol4)-dependent pathway". The Journal of Biological Chemistry. 274
      (33): 23599â609. doi:10.1074/jbc.274.33.23599. PMID 10438542.
  31. ^Budman J, Chu G (February 2005). "Processing_of_DNA_for_nonhomologous
      end-joining_by_cell-free_extract". The EMBO Journal. 24 (4): 849â60.
      doi:10.1038/sj.emboj.7600563. PMC 549622. PMID 15692565.
  32. ^Wang H, Perrault AR, Takeda Y, Qin W, Wang H, Iliakis G (September
      2003). "Biochemical_evidence_for_Ku-independent_backup_pathways_of_NHEJ".
      Nucleic Acids Research. 31 (18): 5377â88. doi:10.1093/nar/gkg728.
      PMC 203313. PMID 12954774.
  33. ^Jung D, Alt FW (January 2004). "Unraveling V(D)J recombination; insights
      into gene regulation". Cell. 116 (2): 299â311. doi:10.1016/S0092-8674
      (04)00039-X. PMID 14744439.
  34. ^Sharma S, Javadekar SM, Pandey M, Srivastava M, Kumari R, Raghavan SC
      (2015). "Homology_and_enzymatic_requirements_of_microhomology-dependent
      alternative_end_joining". Cell Death Dis. 6 (3): e1697. doi:10.1038/
      cddis.2015.58. PMC 4385936. PMID 25789972.
  35. ^Decottignies A (2013). "Alternative_end-joining_mechanisms:_a_historical
      perspective". Front Genet. 4: 48. doi:10.3389/fgene.2013.00048.
      PMC 3613618. PMID 23565119.
  36. ^Zahradka K, Slade D, Bailone A, et al. (October 2006). "Reassembly of
      shattered chromosomes in Deinococcus radiodurans". Nature. 443 (7111):
      569â73. doi:10.1038/nature05160. PMID 17006450.
  37. ^Waters LS, Minesinger BK, Wiltrout ME, D'Souza S, Woodruff RV, Walker GC
      (March 2009). "Eukaryotic_Translesion_Polymerases_and_Their_Roles_and
      Regulation_in_DNA_Damage_Tolerance". Microbiol. Mol. Biol. Rev. 73 (1):
      134â54. doi:10.1128/MMBR.00034-08. PMC 2650891. PMID 19258535.
  38. ^LC Colis; P Raychaudhury; AK Basu (2008). "Mutational_specificity_of
      gamma-radiation-induced_guanine-thymine_and_thymine-guanine_intrastrand
      cross-links_in_mammalian_cells_and_translesion_synthesis_past_the
      guanine-thymine_lesion_by_human_DNA_polymerase_eta". Biochemistry. 47
      (6): 8070â8079. doi:10.1021/bi800529f. PMC 2646719. PMID 18616294.
  39. ^P Raychaudhury; Basu, Ashis K. (2011). "Genetic_requirement_for
      mutagenesis_of_the_G[8,5-Me]T_cross-link_in_Escherichia_coli:_DNA
      polymerases_IV_and_V_compete_for_error-prone_bypass". Biochemistry. 50
      (12): 2330â2338. doi:10.1021/bi102064z. PMC 3062377. PMID 21302943.
  40. ^"Translesion_Synthesis". Research.chem.psu.edu. Archived from the
      original on 10 March 2012. Retrieved 14 August 2012.
  41. ^Wang, Z (2001). "Translesion synthesis by the UmuC family of DNA
      polymerases". Mutat. Res. 486 (2): 59â70. doi:10.1016/S0921-8777
      (01)00089-1. PMID 11425512.
  42. ^ a b c Friedberg EC, Walker GC, Siede W, Wood RD, Schultz RA,
      Ellenberger T. (2006). DNA Repair and Mutagenesis, part 3. ASM Press. 2nd
      ed.
  43. ^Liu B, Yip RK, Zhou Z (2012). "Chromatin_remodeling,_DNA_damage_repair
      and_aging". Curr. Genomics. 13 (7): 533â47. doi:10.2174/
      138920212803251373. PMC 3468886. PMID 23633913.
  44. ^ Halicka HD, Zhao H, Podhorecka M, Traganos F, Darzynkiewicz, Z. (2009)
      Cytometric detection of chromatin relaxation, an early reporter of DNA
      damage response. Cell Cycle, 8:2233-2237.
  45. PMID 19502789
  46. ^ a b cSellou H, Lebeaupin T, Chapuis C, Smith R, Hegele A, Singh HR,
      Kozlowski M, Bultmann S, Ladurner AG, Timinszky G, Huet S (2016). "The
      poly(ADP-ribose)-dependent_chromatin_remodeler_Alc1_induces_local
      chromatin_relaxation_upon_DNA_damage". Mol. Biol. Cell. 27 (24):
      3791â3799. doi:10.1091/mbc.E16-05-0269. PMC 5170603. PMID 27733626.
  47. ^ a bVan Meter M, Simon M, Tombline G, May A, Morello TD, Hubbard BP,
      Bredbenner K, Park R, Sinclair DA, Bohr VA, Gorbunova V, Seluanov A
      (2016). "JNK_Phosphorylates_SIRT6_to_Stimulate_DNA_Double-Strand_Break
      Repair_in_Response_to_Oxidative_Stress_by_Recruiting_PARP1_to_DNA
      Breaks". Cell Rep. 16 (10): 2641â50. doi:10.1016/j.celrep.2016.08.006.
      PMC 5089070. PMID 27568560.
  48. ^ a bHaince JF, McDonald D, Rodrigue A, DÃ©ry U, Masson JY, Hendzel MJ,
      Poirier GG (2008). "PARP1-dependent kinetics of recruitment of MRE11 and
      NBS1 proteins to multiple DNA damage sites". J. Biol. Chem. 283 (2):
      1197â208. doi:10.1074/jbc.M706734200. PMID 18025084.
  49. ^ a b cRogakou EP, Pilch DR, Orr AH, Ivanova VS, Bonner WM (1998). "DNA
      double-stranded breaks induce histone H2AX phosphorylation on serine
      139". J. Biol. Chem. 273 (10): 5858â68. doi:10.1074/jbc.273.10.5858.
      PMID 9488723.
  50. ^Mailand N, Bekker-Jensen S, Faustrup H, Melander F, Bartek J, Lukas C,
      Lukas J (2007). "RNF8 ubiquitylates histones at DNA double-strand breaks
      and promotes assembly of repair proteins". Cell. 131 (5): 887â900. doi:
      10.1016/j.cell.2007.09.040. PMID 18001824.
  51. ^Luijsterburg MS, Acs K, Ackermann L, Wiegant WW, Bekker-Jensen S, Larsen
      DH, Khanna KK, van Attikum H, Mailand N, Dantuma NP (2012). "A_new_non-
      catalytic_role_for_ubiquitin_ligase_RNF8_in_unfolding_higher-order
      chromatin_structure". EMBO J. 31 (11): 2511â27. doi:10.1038/
      emboj.2012.104. PMC 3365417. PMID 22531782.
  52. ^ a bLuijsterburg MS, Goedhart J, Moser J, Kool H, Geverts B, Houtsmuller
      AB, Mullenders LH, Vermeulen W, van Driel R (August 2007). "Dynamic in
      vivo interaction of DDB2 E3 ubiquitin ligase with UV-damaged DNA is
      independent of damage-recognition protein XPC". Journal of Cell Science.
      120 (Pt 15): 2706â16. doi:10.1242/jcs.008367. PMID 17635991.
  53. ^ a bPines A, Vrouwe MG, Marteijn JA, Typas D, Luijsterburg MS, Cansoy M,
      Hensbergen P, Deelder A, de Groot A, Matsumoto S, Sugasawa K, Thoma N,
      Vermeulen W, Vrieling H, Mullenders L (October 2012). "PARP1_promotes
      nucleotide_excision_repair_through_DDB2_stabilization_and_recruitment_of
      ALC1". The Journal of Cell Biology. 199 (2): 235â49. doi:10.1083/
      jcb.201112132. PMC 3471223. PMID 23045548.
  54. ^Jazayeri A, Falck J, Lukas C, Bartek J, Smith GC, Lukas J, Jackson SP
      (2006). "ATM- and cell cycle-dependent regulation of ATR in response to
      DNA double-strand breaks". Nat. Cell Biol. 8 (1): 37â45. doi:10.1038/
      ncb1337. PMID 16327781.
  55. ^Bakkenist CJ, Kastan MB (January 2003). "DNA damage activates ATM
      through intermolecular autophosphorylation and dimer dissociation".
      Nature. 421 (6922): 499â506. doi:10.1038/nature01368. PMID 12556884.
  56. ^Wei, Qingyi; Lei Li; David Chen (2007). DNA Repair, Genetic Instability,
      and Cancer. World Scientific. ISBN 978-981-270-014-8.
  57. [page needed]
  58. ^Schonthal, Axel H. (2004). Checkpoint Controls and Cancer. Humana Press.
      ISBN 978-1-58829-500-2.
  59. [page needed]
  60. ^Gartel AL, Tyner AL (June 2002). "The_role_of_the_cyclin-dependent
      kinase_inhibitor_p21_in_apoptosis". Molecular Cancer Therapeutics. 1 (8):
      639â49. PMID 12479224.
  61. ^Janion, C. (2001). "Some aspects of the SOS response system-a critical
      survey". Acta Biochim Pol. 48 (3): 599â610. PMID 11833768.
  62. ^ a bErill, I; Campoy, S; BarbÃ©, J (2007). "Aeons of distress: an
      evolutionary perspective on the bacterial SOS response". FEMS Microbiol.
      Rev. 31 (6): 637â656. doi:10.1111/j.1574-6976.2007.00082.x.
      PMID 17883408.
  63. ^Schlacher, K; Pham, P; Cox, MM; Goodman, MF. (2006). "Roles of DNA
      Polymerase V and RecA Protein in SOS Damage-Induced Mutation". Chem. Rev.
      106 (2): 406â419. doi:10.1021/cr0404951. PMID 16464012.
  64. ^Fry, RC; Begley, TJ; Samson, LD (2004). "Genome-wide responses to DNA-
      damaging agents". Annu Rev Microbiol. 59: 357â77. doi:10.1146/
      annurev.micro.59.031805.133658. PMID 16153173.
  65. ^Espejel, S; Martin, M; Klatt, P; Martin-Caballero, J; Flores, JM;
      Blasco, MA. (2004). "Shorter_telomeres,_accelerated_ageing_and_increased
      lymphoma_in_DNA-PKcs-deficient_mice". EMBO Rep. 5 (5): 503â9. doi:
      10.1038/sj.embor.7400127. PMC 1299048. PMID 15105825.
  66. ^De Boer, J; Andressoo, JO; De Wit, J; Huijmans, J; Beems, RB; Van Steeg,
      H; Weeda, G; Van Der Horst, GT; et al. (2002). "Premature aging in mice
      deficient in DNA repair and transcription". Science. 296 (5571):
      1276â9. doi:10.1126/science.1070174. PMID 11950998.
  67. ^DollÃ© ME, Busuttil RA, Garcia AM, Wijnhoven S, van Drunen E,
      Niedernhofer LJ, van der Horst G, Hoeijmakers JH, van Steeg H, Vijg J
      (April 2006). "Increased genomic instability is not a prerequisite for
      shortened lifespan in DNA repair deficient mice". Mutation Research. 596
      (1â2): 22â35. doi:10.1016/j.mrfmmm.2005.11.008. PMID 16472827.
  68. ^Kobayashi, Y; Narumi, I; Satoh, K; Funayama, T; Kikuchi, M; Kitayama, S;
      Watanabe, H. (2004). "Radiation response mechanisms of the extremely
      radioresistant bacterium Deinococcus radiodurans". Biol Sci Space. 18
      (3): 134â5. PMID 15858357.
  69. ^Spindler, SR. (2005). "Rapid and reversible induction of the longevity,
      anticancer and genomic effects of caloric restriction". Mech Ageing Dev.
      126 (9): 960â6. doi:10.1016/j.mad.2005.03.016. PMID 15927235.
  70. ^Halicka, HD; Zhao, H; Li, J; Lee, YS; Hsieh, TC; Wu, JM; Darzynkiewicz,
      Z (2012). "Potential_anti-aging_agents_suppress_the_level_of_constitutive
      mTOR-_and_DNA_damage-_signaling". Aging (Albany NY). 4 (12): 952â65.
      doi:10.18632/aging.100521. PMC 3615161. PMID 23363784.
  71. ^Tissenbaum, HA; Guarente, L. (2001). "Increased dosage of a sir-2 gene
      extends life span in Caenorhabditis elegans". Nature. 410 (6825):
      227â30. doi:10.1038/35065638. PMID 11242085.
  72. ^Cohen HY, Miller C, Bitterman KJ, Wall NR, Hekking B, Kessler B, Howitz
      KT, Gorospe M, de Cabo R, Sinclair DA (July 2004). "Calorie restriction
      promotes mammalian cell survival by inducing the SIRT1 deacetylase".
      Science. 305 (5682): 390â2. doi:10.1126/science.1099196. PMID 15205477.
  73. ^Cabelof, DC; Yanamadala, S; Raffoul, JJ; Guo, Z; Soofi, A; Heydari, AR.
      (2003). "Caloric restriction promotes genomic stability by induction of
      base excision repair and reversal of its age-related decline". DNA Repair
      (Amst.). 2 (3): 295â307. doi:10.1016/S1568-7864(02)00219-7.
      PMID 12547392.
  74. ^Stuart, JA; Karahalil, B; Hogue, BA; Souza-Pinto, NC; Bohr, VA. (2004).
      "Mitochondrial_and_nuclear_DNA_base_excision_repair_are_affected
      differently_by_caloric_restriction". FASEB J. 18 (3): 595â7. doi:
      10.1096/fj.03-0890fje. PMID 14734635.
  75. ^Walker, DW; McColl, G; Jenkins, NL; Harris, J; Lithgow, GJ. (2000).
      "Evolution of lifespan in C. elegans". Nature. 405 (6784): 296â7. doi:
      10.1038/35012693. PMID 10830948.
  76. ^Johnson, George (28 December 2010). "Unearthing_Prehistoric_Tumors,_and
      Debate". The_New_York_Times. If we lived long enough, sooner or later we
      all would get cancer.
  77. ^Alberts, B; Johnson A; Lewis J; et al. (2002). "The_Preventable_Causes
      of_Cancer". Molecular biology of the cell (4th ed.). New York: Garland
      Science. ISBN 978-0-8153-4072-0. A certain irreducible background
      incidence of cancer is to be expected regardless of circumstances:
      mutations can never be absolutely avoided, because they are an
      inescapable consequence of fundamental limitations on the accuracy of DNA
      replication, as discussed in Chapter 5. If a human could live long
      enough, it is inevitable that at least one of his or her cells would
      eventually accumulate a set of mutations sufficient for cancer to
      develop.
  78. ^Friedenson B (August 2007). "The_BRCA1/2_pathway_prevents_hematologic
      cancers_in_addition_to_breast_and_ovarian_cancers". BMC Cancer. 7: 152.
      doi:10.1186/1471-2407-7-152. PMC 1959234. PMID 17683622.
  79. ^Gavande NS, VanderVere-Carozza PS, Hinshaw HD, Jalal SI, Sears CR,
      Pawelczak KS, Turchi JJ (April 2016). "DNA_repair_targeted_therapy:_The
      past_or_future_of_cancer_treatment?". Pharmacol. Ther. 160: 65â83. doi:
      10.1016/j.pharmthera.2016.02.003. PMC 4811676. PMID 26896565.
  80. ^Bryant HE, Schultz N, Thomas HD, Parker KM, Flower D, Lopez E, Kyle S,
      Meuth M, Curtin NJ, Helleday T (April 2005). "Specific killing of BRCA2-
      deficient tumours with inhibitors of poly(ADP-ribose) polymerase".
      Nature. 434 (7035): 913â7. doi:10.1038/nature03443. PMID 15829966.
  81. ^Goldstein M, Kastan MB (2015). "The DNA damage response: implications
      for tumor responses to radiation and chemotherapy". Annu Rev Med. 66:
      129â43. doi:10.1146/annurev-med-081313-121208. PMID 25423595.
  82. ^ a bJeggo PA, Pearl LH, Carr AM (January 2016). "DNA_repair,_genome
      stability_and_cancer:_a_historical_perspective" (PDF). Nat Rev Cancer. 16
      (1): 35â42. doi:10.1038/nrc.2015.4. PMID 26667849.
  83. ^Bartkova J, Horejsi Z, et al. (April 2005). "DNA damage response as a
      candidate anti-cancer barrier in early human tumorigenesis". Nature. 434
      (7035): 864â70. doi:10.1038/nature03482. PMID 15829956.
  84. ^ a bBartkova J, Rezaei N, et al. (November 2006). "Oncogene-induced
      senescence is part of the tumorigenesis barrier imposed by DNA damage
      checkpoints". Nature. 444 (7119): 633â7. doi:10.1038/nature05268.
      PMID 17136093.
  85. ^Gaillard H, Garcia-Muse T, Aguilera A (May 2015). "Replication stress
      and cancer". Nat Rev Cancer. 15 (5): 276â89. doi:10.1038/nrc3916. hdl:
      10261/123721. PMID 25907220.
  86. ^Halazonetis TD, Gorgoulis VG, Bartek J (March 2008). "An oncogene-
      induced DNA damage model for cancer development". Science. 319 (5868):
      1352â5. doi:10.1126/science.1140735. PMID 18323444.
  87. ^de Boer J, Heoijmakers JH (March 2000). "Nucleotide excision repair and
      human syndromes". Carcinogenesis. 21 (3): 453â60. doi:10.1093/carcin/
      21.3.453. PMID 10688865.
  88. ^Broustas CG, Lieberman HB (February 2014). "DNA_damage_response_genes
      and_the_development_of_cancer_metastasis". Radiat Res. 181 (2): 111â30.
      doi:10.1667/RR13515.1. PMC 4064942. PMID 24397478.
  89. ^Zhang P, Wang J, Gao W, Yuan BZ, Rogers J, Reed E (May 2004). "CHK2
      kinase_expression_is_down-regulated_due_to_promoter_methylation_in_non-
      small_cell_lung_cancer". Mol Cancer. 3 (4): 3â14. doi:10.1186/1476-
      4598-3-14. PMC 419366. PMID 15125777.
  90. ^Baylin_SB; Ohm, Joyce E. (February 2006). "Epigenetic gene silencing in
      cancer â a mechanism for early oncogenic pathway addiction?". Nat. Rev.
      Cancer. 6 (2): 107â16. doi:10.1038/nrc1799. PMID 16491070.
  91. ^Kanwal R, Gupta S (April 2012). "Epigenetic_modifications_in_cancer".
      Clinical Genetics. 81 (4): 303â11. doi:10.1111/j.1399-
      0004.2011.01809.x. PMC 3590802. PMID 22082348.
  92. ^Baldassarre G, Battista S, Belletti B, et al. (April 2003). "Negative
      regulation_of_BRCA1_gene_expression_by_HMGA1_proteins_accounts_for_the
      reduced_BRCA1_protein_levels_in_sporadic_breast_carcinoma". Molecular and
      Cellular Biology. 23 (7): 2225â38. doi:10.1128/MCB.23.7.2225-2238.2003.
      PMC 150734. PMID 12640109.
  93. ^Jacinto FV, Esteller M; Esteller, M. (July 2007). "Mutator pathways
      unleashed by epigenetic silencing in human cancer". Mutagenesis. 22 (4):
      247â53. doi:10.1093/mutage/gem009. PMID 17412712.
  94. ^Lahtz C; Pfeifer, G. P. (February 2011). "Epigenetic_changes_of_DNA
      repair_genes_in_cancer". J Mol Cell Biol. 3 (1): 51â8. doi:10.1093/
      jmcb/mjq053. PMC 3030973. PMID 21278452.
  95.  http://jmcb.oxfordjournals.org/content/3/1/51.long
  96. ^Bernstein C, Nfonsam V, Prasad AR, Bernstein H (March 2013). "Epigenetic
      field_defects_in_progression_to_cancer". World J Gastrointest Oncol. 5
      (3): 43â9. doi:10.4251/wjgo.v5.i3.43. PMC 3648662. PMID 23671730.
  97. ^ Bernstein C, Prasad AR, Nfonsam V, Bernstein H. (2013). "Chapter_16:
      DNA_Damage,_DNA_Repair_and_Cancer", New Research Directions in DNA
      Repair, Clark Chen (Ed.),
  98. ISBN 978-953-51-1114-6, InTech.
  99. ^Narayanan L, Fritzell JA, Baker SM, Liskay RM, Glazer PM (April 1997).
      "Elevated_levels_of_mutation_in_multiple_tissues_of_mice_deficient_in_the
      DNA_mismatch_repair_gene_Pms2". Proceedings of the National Academy of
      Sciences of the United States of America. 94 (7): 3122â7. doi:10.1073/
      pnas.94.7.3122. PMC 20332. PMID 9096356.
 100. ^Hegan DC, Narayanan L, Jirik FR, Edelmann W, Liskay RM, Glazer PM
      (December 2006). "Differing_patterns_of_genetic_instability_in_mice
      deficient_in_the_mismatch_repair_genes_Pms2,_Mlh1,_Msh2,_Msh3_and_Msh6".
      Carcinogenesis. 27 (12): 2402â8. doi:10.1093/carcin/bgl079.
      PMC 2612936. PMID 16728433.
 101. ^Tutt AN, van Oostrom CT, Ross GM, van Steeg H, Ashworth A (March 2002).
      "Disruption_of_Brca2_increases_the_spontaneous_mutation_rate_in_vivo:
      synergism_with_ionizing_radiation". EMBO Reports. 3 (3): 255â60. doi:
      10.1093/embo-reports/kvf037. PMC 1084010. PMID 11850397.
 102. ^German J (March 1969). "Bloom's_syndrome._I._Genetical_and_clinical
      observations_in_the_first_twenty-seven_patients". American Journal of
      Human Genetics. 21 (2): 196â227. PMC 1706430. PMID 5770175.
 103. ^O'Hagan HM, Mohammad HP, Baylin SB (2008). "Double_strand_breaks_can
      initiate_gene_silencing_and_SIRT1-dependent_onset_of_DNA_methylation_in
      an_exogenous_promoter_CpG_island". PLoS Genetics. 4 (8): e1000155. doi:
      10.1371/journal.pgen.1000155. PMC 2491723. PMID 18704159.
 104. ^Cuozzo C, Porcellini A, Angrisano T, et al. (July 2007). "DNA_damage,
      homology-directed_repair,_and_DNA_methylation". PLoS Genetics. 3 (7):
      e110. doi:10.1371/journal.pgen.0030110. PMC 1913100. PMID 17616978.
 105. ^Malkin D (April 2011). "Li-fraumeni_syndrome". Genes & Cancer. 2 (4):
      475â84. doi:10.1177/1947601911413466. PMC 3135649. PMID 21779515.
 106. ^Fearon ER (November 1997). "Human cancer syndromes: clues to the origin
      and nature of cancer". Science. 278 (5340): 1043â50. doi:10.1126/
      science.278.5340.1043. PMID 9353177.
 107. ^Halford S, Rowan A, Sawyer E, Talbot I, Tomlinson I (June 2005). "O(6)-
      methylguanine_methyltransferase_in_colorectal_cancers:_detection_of
      mutations,_loss_of_expression,_and_weak_association_with_G:C>A:
      T_transitions". Gut. 54 (6): 797â802. doi:10.1136/gut.2004.059535.
      PMC 1774551. PMID 15888787.
 108. ^Shen L, Kondo Y, Rosner GL, et al. (September 2005). "MGMT promoter
      methylation and field defect in sporadic colorectal cancer". Journal of
      the National Cancer Institute. 97 (18): 1330â8. doi:10.1093/jnci/
      dji275. PMID 16174854.
 109. ^Psofaki V, Kalogera C, Tzambouras N, et al. (July 2010). "Promoter
      methylation_status_of_hMLH1,_MGMT,_and_CDKN2A/p16_in_colorectal
      adenomas". World Journal of Gastroenterology. 16 (28): 3553â60. doi:
      10.3748/wjg.v16.i28.3553. PMC 2909555. PMID 20653064.
 110. ^Lee KH, Lee JS, Nam JH, et al. (October 2011). "Promoter methylation
      status of hMLH1, hMSH2, and MGMT genes in colorectal cancer associated
      with adenoma-carcinoma sequence". Langenbeck's Archives of Surgery. 396
      (7): 1017â26. doi:10.1007/s00423-011-0812-9. PMID 21706233.
 111. ^Amatu A, Sartore-Bianchi A, Moutinho C, et al. (April 2013). "Promoter
      CpG island hypermethylation of the DNA repair enzyme MGMT predicts
      clinical response to dacarbazine in a phase II study for metastatic
      colorectal cancer". Clinical Cancer Research. 19 (8): 2265â72. doi:
      10.1158/1078-0432.CCR-12-3518. PMID 23422094.
 112. ^Mokarram P, Zamani M, Kavousipour S, et al. (May 2013). "Different
      patterns of DNA methylation of the two distinct O6-methylguanine-DNA
      methyltransferase (O6-MGMT) promoter regions in colorectal cancer".
      Molecular Biology Reports. 40 (5): 3851â7. doi:10.1007/s11033-012-2465-
      3. PMID 23271133.
 113. ^Truninger K, Menigatti M, Luz J, et al. (May 2005). "Immunohistochemical
      analysis reveals high frequency of PMS2 defects in colorectal cancer".
      Gastroenterology. 128 (5): 1160â71. doi:10.1053/j.gastro.2005.01.056.
      PMID 15887099.
 114. ^Valeri N, Gasparini P, Fabbri M, et al. (April 2010). "Modulation_of
      mismatch_repair_and_genomic_stability_by_miR-155". Proceedings of the
      National Academy of Sciences of the United States of America. 107 (15):
      6982â7. doi:10.1073/pnas.1002472107. PMC 2872463. PMID 20351277.
 115. ^ a b Carol Bernstein and Harris Bernstein (2015). "Epigenetic_Reduction
      of_DNA_Repair_in_Progression_to_Cancer", Advances in DNA Repair, Prof.
      Clark Chen (ed.),
 116. ISBN 978-953-51-2209-8, InTech
 117. ^Facista A, Nguyen H, Lewis C, et al. (2012). "Deficient_expression_of
      DNA_repair_enzymes_in_early_progression_to_sporadic_colon_cancer". Genome
      Integrity. 3 (1): 3. doi:10.1186/2041-9414-3-3. PMC 3351028.
      PMID 22494821.
 118. ^ Human_DNA_Repair_Genes, 15 April 2014, MD Anderson Cancer Center,
      University of Texas
 119. ^ Xinrong Chen and Tao Chen (2011). "Chapter_18:_Roles_of_MicroRNA_in_DNA
      Damage_and_Repair", DNA Repair, Inna Kruman (ed.),
 120. ISBN 9789533076973, InTech.
 121. ^Gao D, Herman_JG, Guo M (June 2016). "The_clinical_value_of_aberrant
      epigenetic_changes_of_DNA_damage_repair_genes_in_human_cancer".
      Oncotarget. 7 (24): 37331â37346. doi:10.18632/oncotarget.7949.
      PMC 5095080. PMID 26967246.
 122. ^Jin B, Robertson KD (2013). "DNA_methyltransferases,_DNA_damage_repair,
      and_cancer". Adv. Exp. Med. Biol. Advances in Experimental Medicine and
      Biology. 754: 3â29. doi:10.1007/978-1-4419-9967-2_1. ISBN 978-1-4419-
      9966-5. PMC 3707278. PMID 22956494.
 123. ^Krishnan K, Steptoe AL, Martin HC, et al. (February 2013). "MicroRNA-
      182-5p_targets_a_network_of_genes_involved_in_DNA_repair". RNA. 19 (2):
      230â42. doi:10.1261/rna.034926.112. PMC 3543090. PMID 23249749.
 124. ^Chaisaingmongkol J, Popanda O, Warta R, et al. (December 2012).
      "Epigenetic screen of human DNA repair genes identifies aberrant promoter
      methylation of NEIL1 in head and neck squamous cell carcinoma". Oncogene.
      31 (49): 5108â16. doi:10.1038/onc.2011.660. PMID 22286769.
 125. ^Liang L, Deng L, Chen Y, Li GC, Shao C, Tischfield JA (2005).
      "Modulation of DNA end joining by nuclear proteins". J. Biol. Chem. 280
      (36): 31442â9. doi:10.1074/jbc.M503776200. PMID 16012167.
 126. ^Singh P, Yang M, Dai H, Yu D, Huang Q, Tan W, Kernstine KH, Lin D, Shen
      B (2008). "Overexpression_and_hypomethylation_of_flap_endonuclease_1_gene
      in_breast_and_other_cancers". Mol. Cancer Res. 6 (11): 1710â7. doi:
      10.1158/1541-7786.MCR-08-0269 (inactive 19 July 2019). PMC 2948671.
      PMID 19010819.
 127. ^Lam JS, Seligson DB, Yu H, Li A, Eeva M, Pantuck AJ, Zeng G, Horvath S,
      Belldegrun AS (2006). "Flap endonuclease 1 is overexpressed in prostate
      cancer and is associated with a high Gleason score". BJU Int. 98 (2):
      445â51. doi:10.1111/j.1464-410X.2006.06224.x. PMID 16879693.
 128. ^Kim JM, Sohn HY, Yoon SY, Oh JH, Yang JO, Kim JH, Song KS, Rho SM, Yoo
      HS, Yoo HS, Kim YS, Kim JG, Kim NS (2005). "Identification of gastric
      cancer-related genes using a cDNA microarray containing novel expressed
      sequence tags expressed in gastric cancer cells". Clin. Cancer Res. 11 (2
      Pt 1): 473â82. PMID 15701830.
 129. ^Wang K, Xie C, Chen D (2014). "Flap endonuclease 1 is a promising
      candidate biomarker in gastric cancer and is involved in cell
      proliferation and apoptosis". Int. J. Mol. Med. 33 (5): 1268â74. doi:
      10.3892/ijmm.2014.1682. PMID 24590400.
 130. ^Krause A, Combaret V, Iacono I, Lacroix B, Compagnon C, Bergeron C,
      Valsesia-Wittmann S, Leissner P, Mougin B, Puisieux A (2005). "Genome-
      wide analysis of gene expression in neuroblastomas detected by mass
      screening". Cancer Lett. 225 (1): 111â20. doi:10.1016/
      j.canlet.2004.10.035. PMID 15922863.
 131. ^Iacobuzio-Donahue CA, Maitra A, Olsen M, Lowe AW, van Heek NT, Rosty C,
      Walter K, Sato N, Parker A, Ashfaq R, Jaffee E, Ryu B, Jones J, Eshleman
      JR, Yeo CJ, Cameron JL, Kern SE, Hruban RH, Brown PO, Goggins M (2003).
      "Exploration_of_global_gene_expression_patterns_in_pancreatic
      adenocarcinoma_using_cDNA_microarrays". Am. J. Pathol. 162 (4):
      1151â62. doi:10.1016/S0002-9440(10)63911-9. PMC 1851213. PMID 12651607.
 132. ^Sato M, Girard L, Sekine I, Sunaga N, Ramirez RD, Kamibayashi C, Minna
      JD (2003). "Increased expression and no mutation of the Flap endonuclease
      (FEN1) gene in human lung cancer". Oncogene. 22 (46): 7243â6. doi:
      10.1038/sj.onc.1206977. PMID 14562054.
 133. ^Bi FF, Li D, Yang Q (2013). "Hypomethylation_of_ETS_transcription_factor
      binding_sites_and_upregulation_of_PARP1_expression_in_endometrial
      cancer". Biomed Res Int. 2013: 1â5. doi:10.1155/2013/946268.
      PMC 3666359. PMID 23762867.
 134. ^Li D, Bi FF, Cao JM, Cao C, Li CY, Liu B, Yang Q (2014). "Poly_(ADP-
      ribose)_polymerase_1_transcriptional_regulation:_a_novel_crosstalk
      between_histone_modification_H3K9ac_and_ETS1_motif_hypomethylation_in
      BRCA1-mutated_ovarian_cancer". Oncotarget. 5 (1): 291â7. doi:10.18632/
      oncotarget.1549. PMC 3960209. PMID 24448423.
 135. ^Bi FF, Li D, Yang Q (2013). "Promoter_hypomethylation,_especially_around
      the_E26_transformation-specific_motif,_and_increased_expression_of_poly_
      (ADP-ribose)_polymerase_1_in_BRCA-mutated_serous_ovarian_cancer". BMC
      Cancer. 13: 90. doi:10.1186/1471-2407-13-90. PMC 3599366. PMID 23442605.
 136. ^Supek, Fran; Lehner, Ben (7 May 2015). "Differential_DNA_mismatch_repair
      underlies_mutation_rate_variation_across_the_human_genome". Nature. 521
      (7550): 81â84. doi:10.1038/nature14173. ISSN 0028-0836. PMC 4425546.
      PMID 25707793.
 137. ^ a bZheng, Christina L.; Wang, Nicholas J.; Chung, Jongsuk; Moslehi,
      Homayoun; Sanborn, J. Zachary; Hur, Joseph S.; Collisson, Eric A.;
      Vemula, Swapna S.; Naujokas, Agne (20 November 2014). "Transcription
      restores_DNA_repair_to_heterochromatin,_determining_regional_mutation
      rates_in_cancer_genomes". Cell Reports. 9 (4): 1228â1234. doi:10.1016/
      j.celrep.2014.10.031. ISSN 2211-1247. PMC 4254608. PMID 25456125.
 138. ^Li, Feng; Mao, Guogen; Tong, Dan; Huang, Jian; Gu, Liya; Yang, Wei; Li,
      Guo-Min (25 April 2013). "The_histone_mark_H3K36me3_regulates_human_DNA
      mismatch_repair_through_its_interaction_with_MutSÎ±". Cell. 153 (3):
      590â600. doi:10.1016/j.cell.2013.03.025. ISSN 1097-4172. PMC 3641580.
      PMID 23622243.
 139. ^Supek, Fran; Lehner, Ben (27 July 2017). "Clustered Mutation Signatures
      Reveal that Error-Prone DNA Repair Targets Mutations to Active Genes".
      Cell. 170 (3): 534â547.e23. doi:10.1016/j.cell.2017.07.003. hdl:10230/
      35343. ISSN 1097-4172. PMID 28753428.
 140. ^Polak, Paz; Lawrence, Michael S.; Haugen, Eric; Stoletzki, Nina;
      Stojanov, Petar; Thurman, Robert E; Garraway, Levi A.; Mirkin, Sergei;
      Getz, Gad (January 2014). "Reduced_local_mutation_density_in_regulatory
      DNA_of_cancer_genomes_is_linked_to_DNA_repair". Nature Biotechnology. 32
      (1): 71â75. doi:10.1038/nbt.2778. ISSN 1087-0156. PMC 4116484.
      PMID 24336318.
 141. ^Cromie, GA; Connelly, JC; Leach, DR (2001). "Recombination at double-
      strand breaks and DNA ends: conserved mechanisms from phage to humans".
      Mol. Cell. 8 (6): 1163â74. doi:10.1016/S1097-2765(01)00419-1.
      PMID 11779493.
 142. ^O'Brien, PJ. (2006). "Catalytic promiscuity and the divergent evolution
      of DNA repair enzymes". Chem Rev. 106 (2): 720â52. doi:10.1021/
      cr040481v. PMID 16464022.
 143. ^Maresca, B; Schwartz, JH (2006). "Sudden origins: a general mechanism of
      evolution based on stress protein concentration and rapid environmental
      change". Anat Rec B New Anat. 289 (1): 38â46. doi:10.1002/ar.b.20089.
      PMID 16437551.
 144. ^ "CRISPR_gene-editing_tool_has_scientists_thrilled_â_but_nervous"_CBC
      news._Author_Kelly_Crowe._November_30,_2015.
***** Further reading[edit] *****
   1. Mutirangura A. (2019). A_Hypothesis_to_Explain_How_the_DNA_of_Elderly
      People_Is_Prone_to_Damage:_Genome-Wide_Hypomethylation_Drives_Genomic
      Instability_in_the_Elderly_by_Reducing_Youth-Associated_Genome-
      Stabilizing_DNA_Gaps. In Epigenetics. IntechOpen. doi:10.5772/
      intechopen.83372
***** External links[edit] *****
Library_resources about
DNA repair
===============================================================================
    * Online_books
    * Resources_in_your_library
    * Resources_in_other_libraries
Listen to this article (info/dl)

This audio file was created from a revision of the article "DNA repair" dated
2005-06-17, and does not reflect subsequent edits to the article. (Audio_help)
More_spoken_articles
    *  Media related to DNA_repair at Wikimedia Commons
    * Roswell_Park_Cancer_Institute_DNA_Repair_Lectures
    * A_comprehensive_list_of_Human_DNA_Repair_Genes
    * 3D_structures_of_some_DNA_repair_enzymes
    * Human_DNA_repair_diseases
    * DNA_repair_special_interest_group
    * DNA_Repair
    * DNA_Damage_and_DNA_Repair
    * Segmental_Progeria
    * DNA-damage_repair;_the_good,_the_bad,_and_the_ugly
    * v
    * t
    * e
DNA repair
                             * Base_excision_repair/AP_site
                                   o DNA_glycosylase
                                   o Uracil-DNA_glycosylase
                                   o Poly_ADP_ribose_polymerase
                             * Nucleotide_excision_repair/ERCC
                                   o XPA
                                   o XPB
                                   o XPC
                                   o XPD/ERCC2
Excision_repair                    o XPE/DDB1
                                   o XPF/DDB1
                                   o XPG/ERCC5
                                   o ERCC1
                                   o RPA
                                   o RAD23A
                                   o RAD23B
                                   o Excinuclease
                             * DNA_mismatch_repair
                                   o MLH1
                                   o MSH2
                             * Transcription-coupled_repair
                                   o ERCC6
                                   o ERCC8
                             * Homology_directed_repair
                             * Non-homologous_end_joining
Other forms of repair              o Ku
                             * Microhomology-mediated_end_joining
                             * Postreplication_repair
                             * Photolyase
                                   o CRY1
                                   o CRY2
                             * Ogt
                             * PcrA
                             * Proliferating_Cell_Nuclear_Antigen
Other/ungrouped proteins     * Homologous_recombination
                                   o RecA/RAD51
                                   o Sgs1
                                   o Slx4
Regulation                   * SOS_box
                             * SOS_response
                             * 8-Oxoguanine
                             * Adaptive_response
                             * Meiotic_recombination_checkpoint
                             * RecF_pathway
                             * DNA_helicase: BLM
                             * WRN
                             * FANC_proteins: core protein complex
                                   o FANCA
                                   o FANCB
Other/ungrouped                    o FANCC
                                   o FANCE
                                   o FANCF
                                   o FANCG
                                   o FANCL
                                   o FANCM
                             * FANCD1
                             * FANCD2
                             * FANCI
                             * FANCJ
                             * FANCN
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=DNA_repair&oldid=909326944"
Categories:
    * Spoken_articles
    * Cellular_processes
    * Molecular_genetics
    * Mutation
    * DNA_repair
    * Senescence
Hidden categories:
    * Wikipedia_articles_needing_page_number_citations_from_December_2014
    * Pages_with_DOIs_inactive_as_of_2019
    * Wikipedia_indefinitely_move-protected_pages
    * Use_dmy_dates_from_August_2018
    * Articles_with_short_description
    * Articles_with_hAudio_microformats
    * Commons_category_link_from_Wikidata
    * Featured_articles
    * Articles_containing_video_clips
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * LatvieÅ¡u
    * Lumbaart
    * Magyar
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Polski
    * PortuguÃªs
    * Ð ÑÑÑÐºÐ¸Ð¹
    * à·à·à¶à·à¶½
    * Simple_English
    * SlovenÄina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * à®¤à®®à®¿à®´à¯
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 4 August 2019, at 18:06 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
