The following text has been accessed from https://en.wikipedia.org/wiki/Protein_structure at Fri Aug 9 03:34:59 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Protein structure ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
 This article needs additional citations for verification. Please help improve_this
 article by adding_citations_to_reliable_sources. Unsourced material may be
 challenged and removed.
 Find sources: "Protein_structure" â news Â· newspapers Â· books Â· scholar Â·
 JSTOR (May 2018)(Learn_how_and_when_to_remove_this_template_message)
[The image above contains clickable links]
Interactive diagram of protein structure, using PCNA as an example. (PDB:
1AXC&#x200b;)
Protein structure is the three-dimensional_arrangement_of_atoms in an amino
acid-chain molecule. Proteins are polymers – specifically polypeptides – formed
from sequences of amino_acids, the monomers of the polymer. A single amino acid
monomer may also be called a residue indicating a repeating unit of a polymer.
Proteins form by amino acids undergoing condensation_reactions, in which the
amino acids lose one water_molecule per reaction in order to attach to one
another with a peptide_bond. By convention, a chain under 30 amino acids is
often identified as a peptide, rather than a protein.[1] To be able to perform
their biological function, proteins fold into one or more specific spatial
conformations driven by a number of non-covalent_interactions such as hydrogen
bonding, ionic_interactions, Van_der_Waals_forces, and hydrophobic packing. To
understand the functions of proteins at a molecular level, it is often
necessary to determine their three-dimensional structure. This is the topic of
the scientific field of structural_biology, which employs techniques such as X-
ray_crystallography, NMR_spectroscopy, and dual_polarisation_interferometry to
determine the structure of proteins.
Protein structures range in size from tens to several thousand amino acids.[2]
By physical size, proteins are classified as nanoparticles, between 1â100 nm.
Very large aggregates can be formed from protein_subunits. For example, many
thousands of actin molecules assemble into a microfilament.
A protein generally undergoes reversible structural changes in performing its
biological function. The alternative structures of the same protein are
referred to as different conformational_isomers, or simply, conformations, and
transitions between them are called conformational_changes.
⁰
***** Contents *****
    * 1_Levels_of_protein_structure
          o 1.1_Primary_structure
          o 1.2_Secondary_structure
          o 1.3_Tertiary_structure
          o 1.4_Quaternary_structure
    * 2_Domains,_motifs,_and_folds_in_protein_structure
          o 2.1_Structural_domain
          o 2.2_Structural_and_sequence_motif
          o 2.3_Supersecondary_structure
          o 2.4_Protein_fold
          o 2.5_Superdomain
    * 3_Protein_dynamics
    * 4_Protein_folding
    * 5_Protein_stability
    * 6_Protein_structure_determination
          o 6.1_Protein_Sequence_Analysis:_Ensembles
    * 7_Protein_structure_databases
    * 8_Structure_classification
    * 9_Computational_prediction_of_protein_structure
    * 10_See_also
    * 11_References
    * 12_Further_reading
    * 13_External_links
***** Levels of protein structure[edit] *****
There are four distinct levels of protein structure.
**** Primary structure[edit] ****
Main article: Protein_primary_structure
The primary_structure of a protein refers to the sequence of amino_acids in the
polypeptide chain. The primary structure is held together by peptide_bonds that
are made during the process of protein_biosynthesis. The two ends of the
polypeptide_chain are referred to as the carboxyl_terminus (C-terminus) and the
amino_terminus (N-terminus) based on the nature of the free group on each
extremity. Counting of residues always starts at the N-terminal end (NH2-
group), which is the end where the amino group is not involved in a peptide
bond. The primary structure of a protein is determined by the gene
corresponding to the protein. A specific sequence of nucleotides in DNA is
transcribed into mRNA, which is read by the ribosome in a process called
translation. The sequence of amino acids in insulin was discovered by Frederick
Sanger, establishing that proteins have defining amino acid sequences.[3][4]
The sequence of a protein is unique to that protein, and defines the structure
and function of the protein. The sequence of a protein can be determined by
methods such as Edman_degradation or tandem_mass_spectrometry. Often, however,
it is read directly from the sequence of the gene using the genetic_code. It is
strictly recommended to use the words "amino acid residues" when discussing
proteins because when a peptide bond is formed, a water_molecule is lost, and
therefore proteins are made up of amino acid residues. Post-translational
modification such as phosphorylations and glycosylations are usually also
considered a part of the primary structure, and cannot be read from the gene.
For example, insulin is composed of 51 amino acids in 2 chains. One chain has
31 amino acids, and the other has 20 amino acids.
**** Secondary structure[edit] ****
An Î±-helix with hydrogen bonds (yellow dots)
Main article: Protein_secondary_structure
Secondary_structure refers to highly regular local sub-structures on the actual
polypeptide backbone chain. Two main types of secondary structure, the Î±-helix
and the Î²-strand or Î²-sheets, were suggested in 1951 by Linus_Pauling et al.
[5] These secondary structures are defined by patterns of hydrogen_bonds
between the main-chain peptide groups. They have a regular geometry, being
constrained to specific values of the dihedral angles Ï and Ï on the
Ramachandran_plot. Both the Î±-helix and the Î²-sheet represent a way of
saturating all the hydrogen bond donors and acceptors in the peptide backbone.
Some parts of the protein are ordered but do not form any regular structures.
They should not be confused with random_coil, an unfolded polypeptide chain
lacking any fixed three-dimensional structure. Several sequential secondary
structures may form a "supersecondary_unit".[6]
**** Tertiary structure[edit] ****
Main article: Protein_tertiary_structure
Tertiary_structure refers to the three-dimensional structure of monomeric and
multimeric protein molecules. The Î±-helixes and Î²-pleated-sheets are folded
into a compact globular_structure. The folding is driven by the non-specific
hydrophobic_interactions, the burial of hydrophobic_residues from water, but
the structure is stable only when the parts of a protein_domain are locked into
place by specific tertiary interactions, such as salt_bridges, hydrogen bonds,
and the tight packing of side chains and disulfide_bonds. The disulfide bonds
are extremely rare in cytosolic proteins, since the cytosol (intracellular
fluid) is generally a reducing environment.
**** Quaternary structure[edit] ****
Main article: Protein_quaternary_structure
Quaternary structure is the three-dimensional structure consisting of the
aggregation of two or more individual polypeptide chains (subunits) that
operate as a single functional unit (multimer). The resulting multimer is
stabilized by the same non-covalent_interactions and disulfide bonds as in
tertiary structure. There are many possible quaternary structure organisations.
[7] Complexes of two or more polypeptides (i.e. multiple subunits) are called
multimers. Specifically it would be called a dimer if it contains two subunits,
a trimer if it contains three subunits, a tetramer if it contains four
subunits, and a pentamer if it contains five subunits. The subunits are
frequently related to one another by symmetry_operations, such as a 2-fold axis
in a dimer. Multimers made up of identical subunits are referred to with a
prefix of "homo-" (e.g. a homotetramer) and those made up of different subunits
are referred to with a prefix of "hetero-", for example, a heterotetramer, such
as the two alpha and two beta chains of hemoglobin.

***** Domains, motifs, and folds in protein structure[edit] *****
Protein domains. The two shown protein structures share a common domain
(maroon), the PH_domain, which is involved in phosphatidylinositol_(3,4,5)-
trisphosphate binding
Proteins are frequently described as consisting of several structural units.
These units include domains, motifs, and folds. Despite the fact that there are
about 100,000 different proteins expressed in eukaryotic systems, there are
many fewer different domains, structural motifs and folds.
**** Structural domain[edit] ****
A structural_domain is an element of the protein's overall structure that is
self-stabilizing and often folds independently of the rest of the protein
chain. Many domains are not unique to the protein products of one gene or one
gene_family but instead appear in a variety of proteins. Domains often are
named and singled out because they figure prominently in the biological
function of the protein they belong to; for example, the "calcium-binding
domain of calmodulin". Because they are independently stable, domains can be
"swapped" by genetic_engineering between one protein and another to make
chimera proteins.
**** Structural and sequence motif[edit] ****
The structural and sequence motifs refer to short segments of protein three-
dimensional structure or amino acid sequence that were found in a large number
of different proteins.
**** Supersecondary structure[edit] ****
The supersecondary_structure refers to a specific combination of secondary
structure elements, such as Î²-Î±-Î² units or a helix-turn-helix motif. Some of
them may be also referred to as structural motifs.
**** Protein fold[edit] ****
A protein fold refers to the general protein architecture, like a helix_bundle,
Î²-barrel, Rossman_fold or different "folds" provided in the Structural
Classification_of_Proteins_database.[8] A related concept is protein_topology
that refers to the arrangement of contacts within the protein.
**** Superdomain[edit] ****
A superdomain consists of two or more nominally unrelated structural domains
that are inherited as a single unit and occur in different proteins.[9] An
example is provided by the protein_tyrosine_phosphatase domain and C2_domain
pair in PTEN, several tensin proteins, auxilin and proteins in plants and
fungi. The PTP-C2 superdomain evidently came into existence prior to the
divergence of fungi, plants and animals is therefore likely to be about 1.5
billion years old.[citation_needed]
***** Protein dynamics[edit] *****
A ribosome is a biological_machine that utilizes protein_dynamics on nanoscales
Main article: Protein_dynamics
Proteins are not however strictly static objects, but rather populate ensembles
of conformational_states. Transitions between these states typically occur on
nanoscales, and have been linked to functionally relevant phenomena such as
allosteric_signaling[10] and enzyme_catalysis.[11] Protein_dynamics and
conformational_changes allow proteins to function as nanoscale biological
machines within cells, often in the form of multi-protein_complexes.[12]
Examples include motor_proteins, such as myosin, which is responsible for
muscle contraction, kinesin, which moves cargo inside cells away from the
nucleus along microtubules, and dynein, which moves cargo inside cells towards
the nucleus and produces the axonemal beating of motile_cilia and flagella. "
[I]n effect, the [motile cilium] is a nanomachine composed of perhaps over 600
proteins in molecular complexes, many of which also function independently as
nanomachines...Flexible_linkers allow the mobile_protein_domains connected by
them to recruit their binding partners and induce long-range allostery via
protein_domain_dynamics. "[13]
***** Protein folding[edit] *****
[[icon]] This section needs expansion. You can help by adding_to_it. (April
         2019)
Main article: Protein_folding
As it is translated, polypeptides exit the ribosome as a random_coil and folds
into its native_state.[14][15] Since the fold is determined by a network of
interactions between amino acids in the polypeptide, the final structure of the
protein chain is determined by its amino acid sequence (Anfinsen's_dogma).[16]
***** Protein stability[edit] *****
Protein stability depends upon a few factors such as 1) Non-covalent
electrostatic interactions 2) Hydrophobic interactions These interaction
energies are from the order of 20-40 kJ/mol.[citation_needed] Proteins are very
sensitive to changing temperatures and a change in temperature may result in
unfolding or denaturation. Protein denaturation may result in loss of function,
and loss of native state.or it can be primitive state as well..
X-ray crystallography and calorimetry indicates that there is no general
mechanism that describes the effect of temperature change on the functions and
structure of proteins. This is due to the fact that proteins do not represent a
uniform class of chemical entities from an energetic point of view. The
structure and stability of an individual protein depends on the ratio of its
polar and non-polar residues. They contribute to the conformational and the net
enthalpies of local and non-local interactions.
Taking the weak intermolecular interactions responsible for structural
integrity into consideration, it is hard to predict the effects of temperature
because there are too many unknown factors contributing to the hypothetical
free energy balance and its temperature dependence. Internal salt linkages
produce thermal stability, and whether cold temperature results in the
destabilization of these linkages is unknown.
In principle, the free energy of stabilization of soluble globular proteins
does not exceed 50-100 kJ/mol.[citation_needed] The stabilization is based on
the equivalent of few hydrogen bonds, ion pairs, or hydrophobic interactions,
even though numerous intramolecular interactions results in stabilization.
Taking into consideration the large number of hydrogen bonds that take place
for the stabilization of secondary structures, and the stabilization of the
inner core through hydrophobic interactions, the free energy of stabilization
emerges as small difference between large numbers. Therefore, the structure of
a native protein is not optimized for the maximum stability.[17]
***** Protein structure determination[edit] *****
Examples of protein structures from the PDB
Rate of Protein Structure Determination by Method and Year
Around 90% of the protein structures available in the Protein_Data_Bank have
been determined by X-ray_crystallography.[18] This method allows one to measure
the three-dimensional (3-D) density distribution of electrons in the protein,
in the crystallized state, and thereby infer the 3-D coordinates of all the
atoms to be determined to a certain resolution. Roughly 9% of the known protein
structures have been obtained by nuclear_magnetic_resonance (NMR) techniques.
[citation_needed] For larger protein complexes, cryo-electron_microscopy can
determine protein structures. The resolution is typically lower than that of X-
ray crystallography, or NMR, but the maximum resolution is steadily increasing.
This technique is still a particularly valuable for very large protein
complexes such as virus_coat_proteins and amyloid fibers.
General secondary structure composition can be determined via circular
dichroism. Vibrational_spectroscopy can also be used to characterize the
conformation of peptides, polypeptides, and proteins.[19] Two-dimensional
infrared_spectroscopy has become a valuable method to investigate the
structures of flexible peptides and proteins that cannot be studied with other
methods.[20][21] A more qualitative picture of protein structure is often
obtained by proteolysis, which is also useful to screen for more crystallizable
protein samples. Novel implementations of this approach, including fast
parallel_proteolysis_(FASTpp), can probe the structured fraction and its
stability without the need for purification.[22] Once a protein's structure has
been experimentally determined, further detailed studies can be done
computationally, using molecular_dynamic simulations of that structure.[23]
**** Protein Sequence Analysis: Ensembles[edit] ****
Schematic view of the two main ensemble modeling approaches.[24]
Proteins are often thought of as relatively stable structures that have a set
tertiary_structure and experience conformational changes as a result of being
modified by other proteins or as part of enzymatic activity. However proteins
have varying degrees of stability and some of the less stable variants are
intrinsically_disordered_proteins. These proteins exist and function in a
relatively 'disordered' state lacking a stable tertiary_structure. As a result,
they are difficult to describe in a standard protein structure model that was
designed for proteins with a fixed tertiary_structure. Conformational_ensembles
have been devised as a way to provide a more accurate and 'dynamic'
representation of the conformational state of intrinsically_disordered
proteins. Conformational_ensembles function by attempting to represent the
various conformations of intrinsically_disordered_proteins within an ensemble
file (the type found at the Protein_Ensemble_Database).
Protein ensemble files are a representation of a protein that can be considered
to have a flexible structure. Creating these files requires determining which
of the various theoretically possible protein conformations actually exist. One
approach is to apply computational algorithms to the protein data in order to
try to determine the most likely set of conformations for an ensemble file.
There are multiple methods for preparing data for the Protein_Ensemble_Database
that fall into two general methodologies â pool and molecular dynamics (MD)
approaches (diagrammed in the figure). The pool based approach uses the
proteinâs amino acid sequence to create a massive pool of random
conformations. This pool is then subjected to more computational processing
that creates a set of theoretical parameters for each conformation based on the
structure. Conformational subsets from this pool whose average theoretical
parameters closely match known experimental data for this protein are selected.
[24]
The molecular dynamics approach takes multiple random conformations at a time
and subjects all of them to experimental data. Here the experimental data is
serving as limitations to be placed on the conformations (e.g. known distances
between atoms). Only conformations that manage to remain within the limits set
by the experimental data are accepted. This approach often applies large
amounts of experimental data to the conformations which is a very
computationally demanding task.[24]
Protein        Data Type    Protocol   PED ID  References
Sic1/Cdc4      NMR and SAXS Pool-based PED9AAA [25]
p15_PAF        NMR and SAXS Pool-based PED6AAA [26]
MKK7           NMR          Pool-based PED5AAB [27]
Beta-synuclein NMR          MD-based   PED1AAD [28]
P27 KID        NMR          MD-based   PED2AAA [29]
(adapted from image in "Computational approaches for inferring the functions of
intrinsically disordered proteins"[24])
***** Protein structure databases[edit] *****
Protein_structure_database is a database that is modeled around the various
experimentally_determined protein structures. The aim of most protein structure
databases is to organize and annotate the protein structures, providing the
biological community access to the experimental data in a useful way. Data
included in protein structure databases often includes 3D coordinates as well
as experimental information, such as unit cell dimensions and angles for x-ray
crystallography determined structures. Though most instances, in this case
either proteins or a specific structure determinations of a protein, also
contain sequence information and some databases even provide means for
performing sequence based queries, the primary attribute of a structure
database is structural information, whereas sequence_databases focus on
sequence information, and contain no structural information for the majority of
entries. Protein structure databases are critical for many efforts in
computational_biology such as structure_based_drug_design, both in developing
the computational methods used and in providing a large experimental dataset
used by some methods to provide insights about the function of a protein.[30]
***** Structure classification[edit] *****
Protein structures can be grouped based on their similarity or a common
evolutionary origin. The Structural_Classification_of_Proteins_database[31] and
CATH database[32] provide two different structural classifications of proteins.
Shared structure between proteins is considered evidence of evolutionary
relatedness between proteins and is used to group proteins together into
protein_superfamilies.[33]
***** Computational prediction of protein structure[edit] *****
Main article: Protein_structure_prediction
The generation of a protein_sequence is much easier than the determination of a
protein structure. However, the structure of a protein gives much more insight
in the function of the protein than its sequence. Therefore, a number of
methods for the computational prediction of protein structure from its sequence
have been developed.[34] Ab initio prediction methods use just the sequence of
the protein. Threading and homology_modeling methods can build a 3-D model for
a protein of unknown structure from experimental structures of evolutionarily-
related proteins, called a protein_family.
***** See also[edit] *****
    * 3did
    * Biomolecular_structure
    * Gene_structure
    * Nucleic_acid_structure
    * Z-matrix conversion from Torsion angles to Cartesian coordinates
***** References[edit] *****
   1. ^H. Stephen Stoker (1 January 2015). Organic_and_Biological_Chemistry.
      Cengage Learning. p. 371. ISBN 978-1-305-68645-8.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Brocchieri L, Karlin S (10 June 2005). "Protein_length_in_eukaryotic_and
      prokaryotic_proteomes". Nucleic Acids Research. 33 (10): 3390â3400.
      doi:10.1093/nar/gki615. PMC 1150220. PMID 15951512.
   4. ^Sanger, F.; Tuppy, H. (1 September 1951). "The_amino-acid_sequence_in
      the_phenylalanyl_chain_of_insulin._I._The_identification_of_lower
      peptides_from_partial_hydrolysates". The Biochemical Journal. 49 (4):
      463â481. doi:10.1042/bj0490463. ISSN 0264-6021. PMC 1197535.
      PMID 14886310.
   5. ^Sanger, F. (15 May 1959). "Chemistry of Insulin". Science. 129 (3359):
      1340â1344. doi:10.1126/science.129.3359.1340. ISSN 0036-8075.
      PMID 13658959.
   6. ^Pauling L, Corey RB, Branson HR (1951). "The_structure_of_proteins;_two
      hydrogen-bonded_helical_configurations_of_the_polypeptide_chain". Proc
      Natl Acad Sci USA. 37 (4): 205â211. doi:10.1073/pnas.37.4.205.
      PMC 1063337. PMID 14816373.
   7. ^Chiang YS, Gelfand TI, Kister AE, Gelfand IM (2007). "New classification
      of supersecondary structures of sandwich-like proteins uncovers strict
      patterns of strand assemblage". Proteins. 68 (4): 915â921. doi:10.1002/
      prot.21473. PMID 17557333.
   8. ^Moutevelis E, Woolfson DN (January 2009). "A periodic table of coiled-
      coil protein structures". J. Mol. Biol. 385 (3): 726â32. doi:10.1016/
      j.jmb.2008.11.028. ISSN 0022-2836. PMID 19059267.
   9. ^Govindarajan S, Recabarren R, Goldstein RA (17 September 1999).
      "Estimating_the_total_number_of_protein_folds". Proteins. 35 (4):
      408â414. doi:10.1002/(SICI)1097-0134(19990601)35:4<408::AID-
      PROT4>3.0.CO;2-A. PMID 10382668. Archived from the_original on 5 January
      2013.
  10. ^Haynie DT, Xue B (2015). "Superdomain_in_the_protein_structure
      hierarchy:_the_case_of_PTP-C2". Protein Science. 24 (5): 874â82. doi:
      10.1002/pro.2664. PMC 4420535. PMID 25694109.
  11. ^Bu Z, Callaway DJ (2011). "Proteins_MOVE!_Protein_dynamics_and_long-
      range_allostery_in_cell_signaling". Protein Structure and Diseases.
      Advances in Protein Chemistry and Structural Biology. 83. pp. 163â221.
      doi:10.1016/B978-0-12-381262-9.00005-7. ISBN 9780123812629.
      PMID 21570668.
  12. ^ Fraser JS, Clarkson MW, Degnan SC, Erion R, Kern D, Alber T (December
      2009). "Hidden_alternative_structures_of_proline_isomerase_essential_for
      catalysis". Nature. 462 (7273): 669â673. Bibcode:2009Natur.462..669F.
      doi:10.1038/nature08615. PMC 2805857. PMID 19956261.
  13. ^Donald, Voet (2011). Biochemistry. Voet, Judith G. (4th ed.). Hoboken,
      NJ: John Wiley & Sons. ISBN 9780470570951. OCLC 690489261.
  14. ^Satir, Peter; SÃ¸ren T. Christensen (26 March 2008). "Structure_and
      function_of_mammalian_cilia". Histochemistry and Cell Biology. 129 (6):
      687â93. doi:10.1007/s00418-008-0416-9. PMC 2386530. PMID 18365235.
      1432-119X.
  15. ^Zhang, Gong; Ignatova, Zoya (1 February 2011). "Folding at the birth of
      the nascent chain: coordinating translation with co-translational
      folding". Current Opinion in Structural Biology. 21 (1): 25â31. doi:
      10.1016/j.sbi.2010.10.008. ISSN 0959-440X. PMID 21111607.
  16. ^Alberts,_Bruce; Alexander Johnson; Julian Lewis; Martin Raff; Keith
      Roberts; Peter Walters (2002). "The_Shape_and_Structure_of_Proteins".
      Molecular Biology of the Cell; Fourth Edition. New York and London:
      Garland Science. ISBN 978-0-8153-3218-3.
  17. ^Anfinsen,_C. (1972). "The_formation_and_stabilization_of_protein
      structure". Biochem. J. 128 (4): 737â49. doi:10.1042/bj1280737.
      PMC 1173893. PMID 4565129.
  18. ^Jaenicke, R.; Heber, U.; Franks, F.; Chapman, D.; Griffin, Mary C. A.;
      Hvidt, A.; Cowan, D. A. (1990). "Protein Structure and Function at Low
      Temperatures [and Discussion]". Philosophical Transactions of the Royal
      Society of London. Series B, Biological Sciences. 326 (1237): 535â553.
      doi:10.1098/rstb.1990.0030. JSTOR 2398703. PMID 1969647.
  19. ^Kendrew, J.C.; Bodo, G.; Dintzis, H. M.; Parrish, R. G.; Wyckoff, H.;
      Phillips, D.C. (1958). "A Three-Dimensional Model of the Myoglobin
      Molecule Obtained by X-Ray Analysis". Nature. 181 (4610): 662â666. doi:
      10.1038/181662a0. PMID 13517261.
  20. ^Krimm S, Bandekar J (1986). Vibrational spectroscopy and conformation of
      peptides, polypeptides, and proteins. Adv. Protein Chem. Advances in
      Protein Chemistry. 38. pp. 181â364. doi:10.1016/S0065-3233(08)60528-8.
      ISBN 9780120342389. PMID 3541539.
  21. ^Lessing, J.; Roy, S.; Reppert, M.; Baer, M.; Marx, D.; Jansen, T.L.C.;
      Knoester, J.; Tokmakoff, A. (2012). "Identifying Residual Structure in
      Intrinsically Disordered Systems: A 2D IR Spectroscopic Study of the
      GVGXPGVG Peptide". J. Am. Chem. Soc. 134 (11): 5032â5035. doi:10.1021/
      ja2114135. PMID 22356513.
  22. ^Jansen, T.L.C.; Knoester, J. (2008). "Two-dimensional_infrared
      population_transfer_spectroscopy_for_enhancing_structural_markers_of
      proteins". Biophys. J. 94 (5): 1818â1825. doi:10.1529/
      biophysj.107.118851. PMC 2242754. PMID 17981904.
  23. ^Minde DP, Maurice MM, RÃ¼diger SG (2012). "Determining_biophysical
      protein_stability_in_lysates_by_a_fast_proteolysis_assay,_FASTpp". PLoS
      ONE. 7 (10): e46147. doi:10.1371/journal.pone.0046147. PMC 3463568.
      PMID 23056252.
  24. ^Kumari I, Sandhu P, Ahmed M, Akhter Y (August 2017). "Molecular Dynamics
      Simulations, Challenges and Opportunities: A Biologist's Prospective".
      Curr. Protein Pept. Sci. 18 (11): 1163â1179. doi:10.2174/
      1389203718666170622074741. PMID 28637405.
  25. ^ a b c dVaradi, Mihaly; Vranken, Wim; Guharoy, Mainak; Tompa, Peter (1
      January 2015). "Computational_approaches_for_inferring_the_functions_of
      intrinsically_disordered_proteins". Frontiers in Molecular Biosciences.
      2: 45. doi:10.3389/fmolb.2015.00045. PMC 4525029. PMID 26301226.
  26. ^Mittag, Tanja; Marsh, Joseph; Grishaev, Alexander; Orlicky, Stephen;
      Lin, Hong; Sicheri, Frank; Tyers, Mike; Forman-Kay, Julie D. (14 March
      2010). "Structure/function_implications_in_a_dynamic_complex_of_the
      intrinsically_disordered_Sic1_with_the_Cdc4_subunit_of_an_SCF_ubiquitin
      ligase". Structure. 18 (4): 494â506. doi:10.1016/j.str.2010.01.020.
      ISSN 1878-4186. PMC 2924144. PMID 20399186.
  27. ^De Biasio, Alfredo; IbÃ¡Ã±ez de Opakua, Alain; Cordeiro, Tiago N.;
      Villate, Maider; Merino, Nekane; Sibille, Nathalie; Lelli, Moreno;
      Diercks, Tammo; BernadÃ³, Pau (18 February 2014). "p15PAF_is_an
      intrinsically_disordered_protein_with_nonrandom_structural_preferences_at
      sites_of_interaction_with_other_proteins". Biophysical Journal. 106 (4):
      865â874. doi:10.1016/j.bpj.2013.12.046. ISSN 1542-0086. PMC 3944474.
      PMID 24559989.
  28. ^Kragelj, Jaka; Palencia, AndrÃ©s; Nanao, Max H.; Maurin, Damien;
      Bouvignies, Guillaume; Blackledge, Martin; Jensen, Malene RingkjÃ¸bing
      (17 March 2015). "Structure_and_dynamics_of_the_MKK7-JNK_signaling
      complex". Proceedings of the National Academy of Sciences of the United
      States of America. 112 (11): 3409â3414. doi:10.1073/pnas.1419528112.
      ISSN 1091-6490. PMC 4371970. PMID 25737554.
  29. ^Allison, Jane R.; Rivers, Robert C.; Christodoulou, John C.;
      Vendruscolo, Michele; Dobson, Christopher M. (25 November 2014). "A
      relationship_between_the_transient_structure_in_the_monomeric_state_and
      the_aggregation_propensities_of_Î±-synuclein_and_Î²-synuclein".
      Biochemistry. 53 (46): 7170â7183. doi:10.1021/bi5009326. ISSN 1520-
      4995. PMC 4245978. PMID 25389903.
  30. ^Sivakolundu, Sivashankar G.; Bashford, Donald; Kriwacki, Richard W. (11
      November 2005). "Disordered p27Kip1 exhibits intrinsic structure
      resembling the Cdk2/cyclin A-bound conformation". Journal of Molecular
      Biology. 353 (5): 1118â1128. doi:10.1016/j.jmb.2005.08.074. ISSN 0022-
      2836. PMID 16214166.
  31. ^Laskowski, RA (2011). "Protein structure databases". Mol Biotechnol. 48
      (2): 183â98. doi:10.1007/s12033-010-9372-4. PMID 21225378.
  32. ^Murzin, A. G.; Brenner,_S.; Hubbard,_T.; Chothia,_C. (1995). "SCOP:_A
      structural_classification_of_proteins_database_for_the_investigation_of
      sequences_and_structures" (PDF). Journal of Molecular Biology. 247 (4):
      536â540. doi:10.1016/S0022-2836(05)80134-2. PMID 7723011. Archived from
      the_original (PDF) on 26 April 2012.
  33. ^Orengo,_C._A.; Michie, A. D.; Jones, S.; Jones, D. T.; Swindells, M. B.;
      Thornton,_J._M. (1997). "CATH--a hierarchic classification of protein
      domain structures". Structure. 5 (8): 1093â1108. doi:10.1016/S0969-2126
      (97)00260-8. PMID 9309224.
  34. ^Holm, L; RosenstrÃ¶m, P (July 2010). "Dali_server:_conservation_mapping
      in_3D". Nucleic Acids Research. 38 (Web Server issue): W545â9. doi:
      10.1093/nar/gkq366. PMC 2896194. PMID 20457744.
  35. ^Zhang Y (2008). "Progress_and_challenges_in_protein_structure
      prediction". Curr Opin Struct Biol. 18 (3): 342â348. doi:10.1016/
      j.sbi.2008.02.004. PMC 2680823. PMID 18436442.
***** Further reading[edit] *****
    * 50_Years_of_Protein_Structure_Determination_Timeline_-_HTML_Version_-
      National_Institute_of_General_Medical_Sciences at NIH
***** External links[edit] *****
    *  Media related to Protein_structures at Wikimedia Commons
    * v
    * t
    * e
Proteins
               * Protein_biosynthesis
               * Post-translational_modification
Processes      * Protein_folding
               * Protein_targeting
               * Proteome
               * Protein_methods
               * Protein structure
Structures     * Protein_structural_domains
               * Proteasome
               * List_of_types_of_proteins
               * List_of_proteins
               * Membrane_protein
Types          * Globular_protein
                     o Globulin
                     o Edestin
                     o Albumin
               * Fibrous_protein
    * v
    * t
    * e
Proteins: key methods of study
                                * Protein_purification
                                * Green_fluorescent_protein
                                * Western_blot
                                * Protein_immunostaining
                                * Protein_sequencing
                                * Gel_electrophoresis/Protein_electrophoresis
                                * Protein_immunoprecipitation
                                * Peptide_mass_fingerprinting/Protein_mass
Experimental                      spectrometry
                                * Dual-polarization_interferometry
                                * Microscale_thermophoresis
                                * Chromatin_immunoprecipitation
                                * Surface_plasmon_resonance
                                * Isothermal_titration_calorimetry
                                * X-ray_crystallography
                                * Protein_NMR
                                * Cryo-electron_microscopy
                                * Freeze-fracture_electron_microscopy
                                * Protein_structure_prediction
                                * Proteinâprotein_docking
Bioinformatics                  * Protein_structural_alignment
                                * Protein_ontology
                                * Proteinâprotein_interaction_prediction
                                * Enzyme_assay
Assay                           * Protein_assay
                                * Secretion_assay
                                * Bacterial_display
                                * mRNA_display
Display techniques              * Phage_display
                                * Ribosome_display
                                * Yeast_display
Super-resolution_microscopy     * Photoactivated_localization_microscopy
                                * Vertico_SMI
    * v
    * t
    * e
Protein_domains
    * BAR
    * BIR
    * BZIP
    * CARD
    * C1
    * C2
    * DED
    * ENTH
    * FYVE
    * HEAT
    * Kringle
    * LIM
    * LRR
    * NACHT
    * PAS
    * PDZ
    * Pyrin
    * PH
    * PX
    * SH2
    * SH3
    * SUN
    * TRIO
    * WD40
    * zinc_finger
    * v
    * t
    * e
Protein structural analysis
                            * Cryo-electron_microscopy
                            * X-ray_crystallography
High resolution             * NMR
                            * Electron_crystallography
                            * EPR
                            * Fiber_diffraction
Medium resolution           * Mass_spectrometry
                            * SAXS
                            * NMR
                            * Circular_dichroism
Spectroscopic               * Dual-polarization_interferometry
                            * Absorbance
                            * Fluorescence
                            * Fluorescence_anisotropy
                            * Analytical_ultracentrifugation
Translational Diffusion     * Size_exclusion_chromatography
                            * Light_scattering
                            * NMR
                            * Fluorescence_anisotropy
Rotational Diffusion        * Flow_birefringence
                            * Dielectric_relaxation
                            * NMR
                            * Hydrogen-deuterium_exchange
Chemical                    * Site-directed_mutagenesis
                            * Chemical_modification
Thermodynamic               * Equilibrium_unfolding
Computational               * Protein_structure_prediction
                            * Molecular_docking
âTertiary_structure
Quaternary_structureâ
    * v
    * t
    * e
Biomolecular_structure
                           * Primary
                           * Secondary
                           * Tertiary
Protein structure          * Quaternary
                           * Determination
                           * Prediction
                           * Design
                           * Thermodynamics
                           * Primary
                           * Secondary
                           * Tertiary
Nucleic_acid_structure     * Quaternary
                           * Determination
                           * Prediction
                           * Design
                           * Thermodynamics
                           * Protein
                           * Protein_domain
                           * Protein_engineering
See also                   * Nucleic_acid
                           * DNA
                           * RNA
                           * Nucleic_acid_double_helix

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Protein_structure&oldid=909265338"
Categories:
    * Protein_structure
Hidden categories:
    * Use_dmy_dates_from_June_2019
    * Articles_needing_additional_references_from_May_2018
    * All_articles_needing_additional_references
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_April_2019
    * Articles_to_be_expanded_from_April_2019
    * All_articles_to_be_expanded
    * Articles_using_small_message_boxes
    * Articles_with_unsourced_statements_from_August_2018
    * Articles_with_unsourced_statements_from_October_2017
    * Commons_category_link_is_on_Wikidata
    * Articles_prone_to_spam_from_August_2014
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * Deutsch
    * EspaÃ±ol
    * Esperanto
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Italiano
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * æ¥æ¬èª
    * Occitan
    * PortuguÃªs
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Shqip
    * Simple_English
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * à°¤à±à°²à±à°à±
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 4 August 2019, at 08:22 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
