The following text has been accessed from https://en.wikipedia.org/wiki/RNA_interference at Fri Aug 9 02:16:40 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_featured_article._Click_here_for_more_information.]
****** RNA interference ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
A biological process in which RNA molecules inhibit gene expression or
translation, by neutralizing targeted mRNA molecules.
Lentiviral delivery of designed shRNAs and the mechanism of RNA interference in
mammalian cells.
RNA interference (RNAi) is a biological process in which RNA molecules inhibit
gene_expression or translation, by neutralizing targeted mRNA molecules.
Historically, RNAi was known by other names, including co-suppression, post-
transcriptional gene silencing (PTGS), and quelling. The detailed study of each
of these seemingly different processes elucidated that the identity of these
phenomena were all actually RNAi. Andrew_Fire and Craig_C._Mello shared the
2006 Nobel_Prize_in_Physiology_or_Medicine for their work on RNA interference
in the nematode worm Caenorhabditis_elegans, which they published in 1998.
Since the discovery of RNAi and its regulatory potentials, it has become
evident that RNAi has immense potential in suppression of desired genes. RNAi
is now known as precise, efficient, stable and better than antisense technology
for gene suppression.[1] However, antisense RNA produced intracellularly by an
expression vector may be developed and find utility as novel therapeutic
agents.[2]
Two types of small ribonucleic_acid (RNA) molecules¬†√¢¬Ä¬ì microRNA (miRNA) and
small_interfering_RNA (siRNA)¬†√¢¬Ä¬ì are central to RNA interference. RNAs are the
direct products of genes, and these small RNAs can direct enzyme complexes to
degrade messenger_RNA (mRNA) molecules and thus decrease their activity by
preventing translation, via post-transcriptional gene silencing. Moreover,
transcription can be inhibited via the pre-transcriptional silencing mechanism
of RNA interference, through which an enzyme complex catalyzes DNA methylation
at genomic positions complementary to complexed siRNA or miRNA. RNA
interference has an important role in defending cells against parasitic
nucleotide sequences¬†√¢¬Ä¬ì viruses and transposons. It also influences
development.
The RNAi pathway is found in many eukaryotes, including animals, and is
initiated by the enzyme Dicer, which cleaves long double-stranded_RNA (dsRNA)
molecules into short double-stranded fragments of ~21 nucleotide siRNAs. Each
siRNA is unwound into two single-stranded RNAs (ssRNAs), the passenger strand
and the guide strand. The passenger strand is degraded and the guide strand is
incorporated into the RNA-induced_silencing_complex (RISC). The most well-
studied outcome is post-transcriptional gene silencing, which occurs when the
guide strand pairs with a complementary sequence in a messenger RNA molecule
and induces cleavage by Argonaute 2 (Ago2), the catalytic component of the
RISC. In some organisms, this process spreads systemically, despite the
initially limited molar concentrations of siRNA.
RNAi is a valuable research tool, both in cell_culture and in living_organisms,
because synthetic dsRNA introduced into cells can selectively and robustly
induce suppression of specific genes of interest. RNAi may be used for large-
scale screens that systematically shut down each gene in the cell, which can
help to identify the components necessary for a particular cellular process or
an event such as cell_division. The pathway is also used as a practical tool in
biotechnology, medicine and insecticides.[3]
‚Å∞
***** Contents *****
    * 1_Cellular_mechanism
          o 1.1_MicroRNA
          o 1.2_Three_prime_untranslated_regions_and_microRNAs
          o 1.3_RISC_activation_and_catalysis
          o 1.4_Transcriptional_silencing
          o 1.5_Crosstalk_with_RNA_editing
          o 1.6_Variation_among_organisms
          o 1.7_Related_prokaryotic_systems
    * 2_Biological_functions
          o 2.1_Immunity
          o 2.2_Downregulation_of_genes
          o 2.3_Upregulation_of_genes
    * 3_Evolution
    * 4_Applications
          o 4.1_Gene_knockdown
          o 4.2_Functional_genomics
          o 4.3_Medicine
                # 4.3.1_Therapeutic_applications
                      # 4.3.1.1_Viral_infection
                      # 4.3.1.2_Cancer
                      # 4.3.1.3_Neurological_diseases
                # 4.3.2_Difficulties_in_Therapeutic_Application
                # 4.3.3_Safety_and_Uses_in_Cancer_treatment
                # 4.3.4_Stimulation_of_immune_response
                # 4.3.5_Prospects_as_a_Therapeutic_Technique
          o 4.4_Biotechnology
                # 4.4.1_Foods
                # 4.4.2_Other_crops
                # 4.4.3_Insecticide
                      # 4.4.3.1_Transgenic_plants
                      # 4.4.3.2_Topical
          o 4.5_Genome-scale_screening
    * 5_History
    * 6_See_also
    * 7_References
    * 8_External_links
***** Cellular mechanism[edit] *****
The dicer protein from Giardia_intestinalis, which catalyzes the cleavage of
dsRNA to siRNAs. The RNase domains are colored green, the PAZ domain yellow,
the platform domain red, and the connector helix blue.[4]
RNAi is an RNA-dependent gene_silencing process that is controlled by the RNA-
induced silencing complex (RISC) and is initiated by short double-stranded RNA
molecules in a cell's cytoplasm, where they interact with the catalytic RISC
component argonaute.[5] When the dsRNA is exogenous (coming from infection by a
virus with an RNA genome or laboratory manipulations), the RNA is imported
directly into the cytoplasm and cleaved to short fragments by Dicer. The
initiating dsRNA can also be endogenous (originating in the cell), as in pre-
microRNAs expressed from RNA-coding_genes in the genome. The primary
transcripts from such genes are first processed to form the characteristic
stem-loop structure of pre-miRNA in the nucleus, then exported to the
cytoplasm. Thus, the two dsRNA pathways, exogenous and endogenous, converge at
the RISC.[6]
Exogenous dsRNA initiates RNAi by activating the ribonuclease protein Dicer,[7]
which binds and cleaves double-stranded RNAs (dsRNAs) in plants, or short
hairpin RNAs (shRNAs) in humans, to produce double-stranded fragments of
20√¢¬Ä¬ì25 base_pairs with a 2-nucleotide overhang at the 3' end.[8]
Bioinformatics studies on the genomes of multiple organisms suggest this length
maximizes target-gene specificity and minimizes non-specific effects.[9] These
short double-stranded fragments are called small interfering RNAs (siRNAs).
These siRNAs are then separated into single strands and integrated into an
active RISC, by RISC-Loading Complex (RLC). RLC includes Dicer-2 and R2D2, and
is crucial to unite Ago2 and RISC.[10] TATA-binding protein-associated factor
11 (TAF11) assembles the RLC by facilitating Dcr-2-R2D2 tetramerization, which
increases the binding affinity to siRNA by 10-fold. Association with TAF11
would convert the R2-D2-Initiator (RDI) complex into the RLC.[11] R2D2 carries
tandem double-stranded RNA-binding domains to recognize the thermodynamically
stable terminus of siRNA duplexes, whereas Dicer-2 the other less stable
extremity. Loading is asymmetric: the MID domain of Ago2 recognizes the
thermodynamically stable end of the siRNA. Therefore, the "passenger" (sense)
strand whose 5√¢¬Ä¬≤ end is discarded by MID is ejected, while the saved "guide"
(antisense) strand cooperates with AGO to form the RISC.[10]
After integration into the RISC, siRNAs base-pair to their target mRNA and
cleave it, thereby preventing it from being used as a translation template.[12]
Differently from siRNA, a miRNA-loaded RISC complex scans cytoplasmic mRNAs for
potential complementarity. Instead of destructive cleavage (by Ago2), miRNAs
rather target the 3√¢¬Ä¬≤ untranslated region (UTR) regions of mRNAs where they
typically bind with imperfect complementarity, thus blocking the access of
ribosomes for translation.[13]
Exogenous dsRNA is detected and bound by an effector protein, known as RDE-4 in
C._elegans and R2D2 in Drosophila, that stimulates dicer activity.[14] The
mechanism producing this length specificity is unknown and this protein only
binds long dsRNAs.[14]
In C. elegans this initiation response is amplified through the synthesis of a
population of 'secondary' siRNAs during which the dicer-produced initiating or
'primary' siRNAs are used as templates.[15] These 'secondary' siRNAs are
structurally distinct from dicer-produced siRNAs and appear to be produced by
an RNA-dependent_RNA_polymerase (RdRP).[16][17]
**** MicroRNA[edit] ****
The stem-loop secondary_structure of a pre-microRNA from Brassica_oleracea.
Main article: MicroRNA
MicroRNAs (miRNAs) are genomically encoded non-coding_RNAs that help regulate
gene_expression, particularly during development.[18] The phenomenon of RNA
interference, broadly defined, includes the endogenously induced gene silencing
effects of miRNAs as well as silencing triggered by foreign dsRNA. Mature
miRNAs are structurally similar to siRNAs produced from exogenous dsRNA, but
before reaching maturity, miRNAs must first undergo extensive post-
transcriptional_modification. A miRNA is expressed from a much longer RNA-
coding gene as a primary_transcript known as a pri-miRNA which is processed, in
the cell_nucleus, to a 70-nucleotide stem-loop structure called a pre-miRNA by
the microprocessor_complex. This complex consists of an RNase_III enzyme called
Drosha and a dsRNA-binding protein DGCR8. The dsRNA portion of this pre-miRNA
is bound and cleaved by Dicer to produce the mature miRNA molecule that can be
integrated into the RISC complex; thus, miRNA and siRNA share the same
downstream cellular machinery.[19] First, viral encoded miRNA was described in
EBV.[20] Thereafter, an increasing number of microRNAs have been described in
viruses. VIRmiRNA is a comprehensive catalogue covering viral microRNA, their
targets and anti-viral miRNAs [21] (see also VIRmiRNA resource: http://
crdd.osdd.net/servers/virmirna/).
siRNAs derived from long dsRNA precursors differ from miRNAs in that miRNAs,
especially those in animals, typically have incomplete base pairing to a target
and inhibit the translation of many different mRNAs with similar sequences. In
contrast, siRNAs typically base-pair perfectly and induce mRNA cleavage only in
a single, specific target.[22] In Drosophila and C. elegans, miRNA and siRNA
are processed by distinct argonaute proteins and dicer enzymes.[23][24]
**** Three prime untranslated regions and microRNAs[edit] ****
Main article: Three_prime_untranslated_region
Three_prime_untranslated_regions (3'UTRs) of messenger_RNAs (mRNAs) often
contain regulatory sequences that post-transcriptionally cause RNA
interference. Such 3'-UTRs often contain both binding sites for microRNAs
(miRNAs) as well as for regulatory proteins. By binding to specific sites
within the 3'-UTR, miRNAs can decrease gene expression of various mRNAs by
either inhibiting translation or directly causing degradation of the
transcript. The 3'-UTR also may have silencer regions that bind repressor
proteins that inhibit the expression of a mRNA.
The 3'-UTR often contains microRNA_response_elements_(MREs). MREs are sequences
to which miRNAs bind. These are prevalent motifs within 3'-UTRs. Among all
regulatory motifs within the 3'-UTRs (e.g. including silencer regions), MREs
make up about half of the motifs.
As of 2014, the miRBase web site,[25] an archive of miRNA sequences and
annotations, listed 28,645 entries in 233 biologic species. Of these, 1,881
miRNAs were in annotated human miRNA loci. miRNAs were predicted to have an
average of about four hundred target mRNAs (affecting expression of several
hundred genes).[26] Friedman et al.[26] estimate that >45,000 miRNA target
sites within human mRNA 3'UTRs are conserved above background levels, and >60%
of human protein-coding genes have been under selective pressure to maintain
pairing to miRNAs.
Direct experiments show that a single miRNA can reduce the stability of
hundreds of unique mRNAs.[27] Other experiments show that a single miRNA may
repress the production of hundreds of proteins, but that this repression often
is relatively mild (less than 2-fold).[28][29]
The effects of miRNA dysregulation of gene expression seem to be important in
cancer.[30] For instance, in gastrointestinal cancers, nine miRNAs have been
identified as epigenetically altered and effective in down regulating DNA
repair enzymes.[31]
The effects of miRNA dysregulation of gene expression also seem to be important
in neuropsychiatric disorders, such as schizophrenia, bipolar disorder, major
depression, Parkinson's disease, Alzheimer's disease and autism spectrum
disorders.[32][33][34]
**** RISC activation and catalysis[edit] ****
Exogenous dsRNA is detected and bound by an effector protein, known as RDE-4 in
C._elegans and R2D2 in Drosophila, that stimulates dicer activity.[14] This
protein only binds long dsRNAs, but the mechanism producing this length
specificity is unknown.[14] This RNA-binding protein then facilitates the
transfer of cleaved siRNAs to the RISC complex.[35]
In C. elegans this initiation response is amplified through the synthesis of a
population of 'secondary' siRNAs during which the dicer-produced initiating or
'primary' siRNAs are used as templates.[15] These 'secondary' siRNAs are
structurally distinct from dicer-produced siRNAs and appear to be produced by
an RNA-dependent_RNA_polymerase (RdRP).[16][17]
small_RNA Biogenesis: primary miRNAs (pri-miRNAs) are transcribed in the
nucleus and fold back onto themselves as hairpins that are then trimmed in the
nucleus by a microprocessor_complex to form a ~60-70nt hairpin pre-RNA. This
pre-miRNA is transported through the nuclear_pore_complex (NPC) into the
cytoplasm, where Dicer further trims it to a ~20nt miRNA duplex (pre-siRNAs
also enter the pathway at this step). This duplex is then loaded into Ago to
form the √¢¬Ä¬úpre-RISC(RNA induced silencing complex)√¢¬Ä¬ù and the passenger strand
is released to form active RISC.
Left: A full-length argonaute protein from the archaea species Pyrococcus
furiosus. Right: The PIWI_domain of an argonaute protein in complex with
double-stranded_RNA.
The active components of an RNA-induced silencing complex (RISC) are
endonucleases called argonaute proteins, which cleave the target mRNA strand
complementary to their bound siRNA.[5] As the fragments produced by dicer are
double-stranded, they could each in theory produce a functional siRNA. However,
only one of the two strands, which is known as the guide strand, binds the
argonaute protein and directs gene silencing. The other anti-guide strand or
passenger strand is degraded during RISC activation.[36] Although it was first
believed that an ATP-dependent helicase separated these two strands,[37] the
process proved to be ATP-independent and performed directly by the protein
components of RISC.[38][39] However, an in vitro kinetic analysis of RNAi in
the presence and absence of ATP showed that ATP may be required to unwind and
remove the cleaved mRNA strand from the RISC complex after catalysis.[40] The
guide strand tends to be the one whose 5'_end is less stably paired to its
complement,[41] but strand selection is unaffected by the direction in which
dicer cleaves the dsRNA before RISC incorporation.[42] Instead, the R2D2
protein may serve as the differentiating factor by binding the more-stable 5'
end of the passenger strand.[43]
The structural basis for binding of RNA to the argonaute protein was examined
by X-ray_crystallography of the binding domain of an RNA-bound argonaute
protein. Here, the phosphorylated 5' end of the RNA strand enters a conserved
basic surface pocket and makes contacts through a divalent cation (an atom with
two positive charges) such as magnesium and by aromatic stacking (a process
that allows more than one atom to share an electron by passing it back and
forth) between the 5' nucleotide in the siRNA and a conserved tyrosine residue.
This site is thought to form a nucleation site for the binding of the siRNA to
its mRNA target.[44] Analysis of the inhibitory effect of mismatches in either
the 5√¢¬Ä¬ô or 3√¢¬Ä¬ô end of the guide strand has demonstrated that the 5√¢¬Ä¬ô end of
the guide strand is likely responsible for matching and binding the target
mRNA, while the 3√¢¬Ä¬ô end is responsible for physically arranging target mRNA
into a cleavage-favorable RISC region.[40]
It is not understood how the activated RISC complex locates complementary mRNAs
within the cell. Although the cleavage process has been proposed to be linked
to translation, translation of the mRNA target is not essential for RNAi-
mediated degradation.[45] Indeed, RNAi may be more effective against mRNA
targets that are not translated.[46] Argonaute proteins are localized to
specific regions in the cytoplasm called P-bodies (also cytoplasmic bodies or
GW bodies), which are regions with high rates of mRNA decay;[47] miRNA activity
is also clustered in P-bodies.[48] Disruption of P-bodies decreases the
efficiency of RNA interference, suggesting that they are a critical site in the
RNAi process.[49]
**** Transcriptional silencing[edit] ****
The enzyme dicer trims double stranded RNA, to form small_interfering_RNA or
microRNA. These processed RNAs are incorporated into the RNA-induced_silencing
complex (RISC), which targets messenger_RNA to prevent translation.[50]
Components of the RNAi pathway are used in many eukaryotes in the maintenance
of the organization and structure of their genomes. Modification of histones
and associated induction of heterochromatin formation serves to downregulate
genes pre-transcriptionally;[51] this process is referred to as RNA-induced
transcriptional_silencing (RITS), and is carried out by a complex of proteins
called the RITS complex. In fission_yeast this complex contains argonaute, a
chromodomain protein Chp1, and a protein called Tas3 of unknown function.[52]
As a consequence, the induction and spread of heterochromatic regions requires
the argonaute and RdRP proteins.[53] Indeed, deletion of these genes in the
fission yeast S._pombe disrupts histone_methylation and centromere formation,
[54] causing slow or stalled anaphase during cell_division.[55] In some cases,
similar processes associated with histone modification have been observed to
transcriptionally upregulate genes.[56]
The mechanism by which the RITS complex induces heterochromatin formation and
organization is not well understood. Most studies have focused on the mating-
type_region in fission yeast, which may not be representative of activities in
other genomic regions/organisms. In maintenance of existing heterochromatin
regions, RITS forms a complex with siRNAs complementary to the local genes and
stably binds local methylated histones, acting co-transcriptionally to degrade
any nascent pre-mRNA transcripts that are initiated by RNA_polymerase. The
formation of such a heterochromatin region, though not its maintenance, is
dicer-dependent, presumably because dicer is required to generate the initial
complement of siRNAs that target subsequent transcripts.[57] Heterochromatin
maintenance has been suggested to function as a self-reinforcing feedback loop,
as new siRNAs are formed from the occasional nascent transcripts by RdRP for
incorporation into local RITS complexes.[58] The relevance of observations from
fission yeast mating-type regions and centromeres to mammals is not clear, as
heterochromatin maintenance in mammalian cells may be independent of the
components of the RNAi pathway.[59]
**** Crosstalk with RNA editing[edit] ****
The type of RNA_editing that is most prevalent in higher eukaryotes converts
adenosine nucleotides into inosine in dsRNAs via the enzyme adenosine_deaminase
(ADAR).[60] It was originally proposed in 2000 that the RNAi and A√¢¬Ü¬íI RNA
editing pathways might compete for a common dsRNA substrate.[61] Some pre-
miRNAs do undergo A√¢¬Ü¬íI RNA editing[62][63] and this mechanism may regulate the
processing and expression of mature miRNAs.[63] Furthermore, at least one
mammalian ADAR can sequester siRNAs from RNAi pathway components.[64] Further
support for this model comes from studies on ADAR-null C. elegans strains
indicating that A√¢¬Ü¬íI RNA editing may counteract RNAi silencing of endogenous
genes and transgenes.[65]
Illustration of the major differences between plant and animal gene silencing.
Natively expressed microRNA or exogenous small_interfering_RNA is processed by
dicer and integrated into the RISC complex, which mediates gene silencing.[66]
**** Variation among organisms[edit] ****
Organisms vary in their ability to take up foreign dsRNA and use it in the RNAi
pathway. The effects of RNA interference can be both systemic and heritable in
plants and C. elegans, although not in Drosophila or mammals. In plants, RNAi
is thought to propagate by the transfer of siRNAs between cells through
plasmodesmata (channels in the cell walls that enable communication and
transport).[37] Heritability comes from methylation of promoters targeted by
RNAi; the new methylation pattern is copied in each new generation of the cell.
[67] A broad general distinction between plants and animals lies in the
targeting of endogenously produced miRNAs; in plants, miRNAs are usually
perfectly or nearly perfectly complementary to their target genes and induce
direct mRNA cleavage by RISC, while animals' miRNAs tend to be more divergent
in sequence and induce translational repression.[66] This translational effect
may be produced by inhibiting the interactions of translation initiation
factors with the messenger RNA's polyadenine_tail.[68]
Some eukaryotic protozoa such as Leishmania_major and Trypanosoma_cruzi lack
the RNAi pathway entirely.[69][70] Most or all of the components are also
missing in some fungi, most notably the model_organism Saccharomyces
cerevisiae.[71] The presence of RNAi in other budding yeast species such as
Saccharomyces_castellii and Candida_albicans, further demonstrates that
inducing two RNAi-related proteins from S. castellii facilitates RNAi in S.
cerevisiae.[72] That certain ascomycetes and basidiomycetes are missing RNA
interference pathways indicates that proteins required for RNA silencing have
been lost independently from many fungal lineages, possibly due to the
evolution of a novel pathway with similar function, or to the lack of selective
advantage in certain niches.[73]
**** Related prokaryotic systems[edit] ****
Gene expression in prokaryotes is influenced by an RNA-based system similar in
some respects to RNAi. Here, RNA-encoding genes control mRNA abundance or
translation by producing a complementary RNA that anneals to an mRNA. However
these regulatory RNAs are not generally considered to be analogous to miRNAs
because the dicer enzyme is not involved.[74] It has been suggested that CRISPR
interference systems in prokaryotes are analogous to eukaryotic RNA
interference systems, although none of the protein components are orthologous.
[75]
***** Biological functions[edit] *****
**** Immunity[edit] ****
RNA interference is a vital part of the immune_response to viruses and other
foreign genetic_material, especially in plants where it may also prevent the
self-propagation of transposons.[76] Plants such as Arabidopsis_thaliana
express multiple dicer homologs that are specialized to react differently when
the plant is exposed to different viruses.[77] Even before the RNAi pathway was
fully understood, it was known that induced gene silencing in plants could
spread throughout the plant in a systemic effect and could be transferred from
stock to scion plants via grafting.[78] This phenomenon has since been
recognized as a feature of the plant adaptive immune system and allows the
entire plant to respond to a virus after an initial localized encounter.[79] In
response, many plant viruses have evolved elaborate mechanisms to suppress the
RNAi response.[80] These include viral proteins that bind short double-stranded
RNA fragments with single-stranded overhang ends, such as those produced by
dicer.[81] Some plant genomes also express endogenous siRNAs in response to
infection by specific types of bacteria.[82] These effects may be part of a
generalized response to pathogens that downregulates any metabolic process in
the host that aids the infection process.[83]
Although animals generally express fewer variants of the dicer enzyme than
plants, RNAi in some animals produces an antiviral response. In both juvenile
and adult Drosophila, RNA interference is important in antiviral innate
immunity and is active against pathogens such as Drosophila_X_virus.[84][85] A
similar role in immunity may operate in C. elegans, as argonaute proteins are
upregulated in response to viruses and worms that overexpress components of the
RNAi pathway are resistant to viral infection.[86][87]
The role of RNA interference in mammalian innate immunity is poorly understood,
and relatively little data is available. However, the existence of viruses that
encode genes able to suppress the RNAi response in mammalian cells may be
evidence in favour of an RNAi-dependent mammalian immune response,[88][89]
although this hypothesis has been challenged as poorly substantiated.[90]
Evidence for the existence of a functional antiviral RNAi pathway in mammalian
cells has been presented.[91][92]
Other functions for RNAi in mammalian viruses also exist, such as miRNAs
expressed by the herpes_virus that may act as heterochromatin organization
triggers to mediate viral latency.[93]
**** Downregulation of genes[edit] ****
Endogenously expressed miRNAs, including both intronic and intergenic miRNAs,
are most important in translational repression[66] and in the regulation of
development, especially on the timing of morphogenesis and the maintenance of
undifferentiated or incompletely differentiated cell types such as stem_cells.
[94] The role of endogenously expressed miRNA in downregulating gene_expression
was first described in C. elegans in 1993.[95] In plants this function was
discovered when the "JAW microRNA" of Arabidopsis was shown to be involved in
the regulation of several genes that control plant shape.[96] In plants, the
majority of genes regulated by miRNAs are transcription_factors;[97] thus miRNA
activity is particularly wide-ranging and regulates entire gene_networks during
development by modulating the expression of key regulatory genes, including
transcription factors as well as F-box_proteins.[98] In many organisms,
including humans, miRNAs are linked to the formation of tumors and
dysregulation of the cell_cycle. Here, miRNAs can function as both oncogenes
and tumor_suppressors.[99]
**** Upregulation of genes[edit] ****
RNA sequences (siRNA and miRNA) that are complementary to parts of a promoter
can increase gene transcription, a phenomenon dubbed RNA_activation. Part of
the mechanism for how these RNA upregulate genes is known: dicer and argonaute
are involved, possibly via histone demethylation.[100] miRNAs have been
proposed to upregulate their target genes upon cell cycle arrest, via unknown
mechanisms.[101]
***** Evolution[edit] *****
Based on parsimony-based phylogenetic analysis, the most_recent_common_ancestor
of all eukaryotes most likely already possessed an early RNA interference
pathway; the absence of the pathway in certain eukaryotes is thought to be a
derived characteristic.[102] This ancestral RNAi system probably contained at
least one dicer-like protein, one argonaute, one PIWI_protein, and an RNA-
dependent_RNA_polymerase that may also have played other cellular roles. A
large-scale comparative_genomics study likewise indicates that the eukaryotic
crown_group already possessed these components, which may then have had closer
functional associations with generalized RNA degradation systems such as the
exosome.[103] This study also suggests that the RNA-binding argonaute protein
family, which is shared among eukaryotes, most archaea, and at least some
bacteria (such as Aquifex_aeolicus), is homologous to and originally evolved
from components of the translation_initiation system.
The ancestral function of the RNAi system is generally agreed to have been
immune defense against exogenous genetic elements such as transposons and viral
genomes.[102][104] Related functions such as histone modification may have
already been present in the ancestor of modern eukaryotes, although other
functions such as regulation of development by miRNA are thought to have
evolved later.[102]
RNA interference genes, as components of the antiviral innate immune system in
many eukaryotes, are involved in an evolutionary_arms_race with viral genes.
Some viruses have evolved mechanisms for suppressing the RNAi response in their
host cells, particularly for plant viruses.[80] Studies of evolutionary rates
in Drosophila have shown that genes in the RNAi pathway are subject to strong
directional_selection and are among the fastest-evolving genes in the
Drosophila genome.[105]
***** Applications[edit] *****
**** Gene knockdown[edit] ****
The RNA interference pathway is often exploited in experimental_biology to
study the function of genes in cell_culture and in_vivo in model_organisms.[5]
Double-stranded RNA is synthesized with a sequence complementary to a gene of
interest and introduced into a cell or organism, where it is recognized as
exogenous genetic material and activates the RNAi pathway. Using this
mechanism, researchers can cause a drastic decrease in the expression of a
targeted gene. Studying the effects of this decrease can show the physiological
role of the gene product. Since RNAi may not totally abolish expression of the
gene, this technique is sometimes referred as a "knockdown", to distinguish it
from "knockout" procedures in which expression of a gene is entirely
eliminated.[106] In a recent study validation of RNAi silencing efficiency
using gene array data showed 18.5% failure rate across 429 independent
experiments.[107]
Extensive efforts in computational_biology have been directed toward the design
of successful dsRNA reagents that maximize gene knockdown but minimize "off-
target" effects. Off-target effects arise when an introduced RNA has a base
sequence that can pair with and thus reduce the expression of multiple genes.
Such problems occur more frequently when the dsRNA contains repetitive
sequences. It has been estimated from studying the genomes of humans, C.
elegans and S._pombe that about 10% of possible siRNAs have substantial off-
target effects.[9] A multitude of software tools have been developed
implementing algorithms for the design of general[108][109] mammal-specific,
[110] and virus-specific[111] siRNAs that are automatically checked for
possible cross-reactivity.
Depending on the organism and experimental system, the exogenous RNA may be a
long strand designed to be cleaved by dicer, or short RNAs designed to serve as
siRNA substrates. In most mammalian cells, shorter RNAs are used because long
double-stranded RNA molecules induce the mammalian interferon response, a form
of innate_immunity that reacts nonspecifically to foreign genetic material.
[112] Mouse oocytes and cells from early mouse embryos lack this reaction to
exogenous dsRNA and are therefore a common model system for studying mammalian
gene-knockdown effects.[113] Specialized laboratory techniques have also been
developed to improve the utility of RNAi in mammalian systems by avoiding the
direct introduction of siRNA, for example, by stable transfection with a
plasmid encoding the appropriate sequence from which siRNAs can be transcribed,
[114] or by more elaborate lentiviral vector systems allowing the inducible
activation or deactivation of transcription, known as conditional RNAi.[115]
[116]
**** Functional genomics[edit] ****
A normal adult Drosophila fly, a common model organism used in RNAi
experiments.
Most functional_genomics applications of RNAi in animals have used C. elegans
[117] and Drosophila,[118] as these are the common model_organisms in which
RNAi is most effective. C. elegans is particularly useful for RNAi research for
two reasons: firstly, the effects of gene silencing are generally heritable,
and secondly because delivery of the dsRNA is extremely simple. Through a
mechanism whose details are poorly understood, bacteria such as E._coli that
carry the desired dsRNA can be fed to the worms and will transfer their RNA
payload to the worm via the intestinal tract. This "delivery by feeding" is
just as effective at inducing gene silencing as more costly and time-consuming
delivery methods, such as soaking the worms in dsRNA solution and injecting
dsRNA into the gonads.[119] Although delivery is more difficult in most other
organisms, efforts are also underway to undertake large-scale genomic screening
applications in cell culture with mammalian cells.[120]
Approaches to the design of genome-wide RNAi libraries can require more
sophistication than the design of a single siRNA for a defined set of
experimental conditions. Artificial_neural_networks are frequently used to
design siRNA libraries[121] and to predict their likely efficiency at gene
knockdown.[122] Mass genomic screening is widely seen as a promising method for
genome_annotation and has triggered the development of high-throughput
screening methods based on microarrays.[123][124] However, the utility of these
screens and the ability of techniques developed on model organisms to
generalize to even closely related species has been questioned, for example
from C. elegans to related parasitic nematodes.[125][126]
Functional genomics using RNAi is a particularly attractive technique for
genomic mapping and annotation in plants because many plants are polyploid,
which presents substantial challenges for more traditional genetic engineering
methods. For example, RNAi has been successfully used for functional genomics
studies in bread_wheat (which is hexaploid)[127] as well as more common plant
model systems Arabidopsis and maize.[128]
**** Medicine[edit] ****
History of RNAi use in medicine
Timeline of the use of RNAi in medicine between 1996 and 2017
The first instance of RNA silencing in animals was documented in 1996, when Guo
and Kemphues observed that, by introducing sense and antisense_RNA to par-
1 mRNA in Caenorhabditis_elegans caused degradation of the par-1 message.[129]
It was thought that this degradation was triggered by single stranded RNA
(ssRNA), but two years later, in 1998, Fire and Mello discovered that this
ability to silence the par-1 gene expression was actually triggered by double-
stranded RNA (dsRNA).[129] They would eventually share the Nobel_Prize_in
Physiology_or_Medicine for this discovery.[130] Just after Fire and Mello's
ground-breaking discovery, Elbashir et al. discovered, by using synthetically
made small_interfering_RNA (siRNA), it was possible to target the silencing of
specific sequences in a gene, rather than silencing the entire gene.[131] Only
a year later, McCaffrey and colleagues demonstrated that this sequence specific
silencing had therapeutic applications by targeting a sequence from the
Hepatitis_C virus in transgenic mice.[132] Since then, multiple researchers
have been attempting to expand the therapeutic applications of RNAi,
specifically looking to target genes that cause various types of cancer.[133]
[134] Finally, in 2004, this new gene silencing technology entered a Phase_I
clinical_trial in humans for wet age-related macular_degeneration.[131] Six
years later the first-in-human Phase I clinical trial was started, using a
nanoparticle_delivery_system to target solid_tumors.[135] Although most
research is currently looking into the applications of RNAi in cancer
treatment, the list of possible applications is extensive. RNAi could
potentially be used to treat viruses,[136] bacterial_diseases,[137] parasites,
[138] maladaptive genetic mutations,[139] control_drug_consumption,[140]
provide_pain_relief,[141] and even modulate sleep.[142]
*** Therapeutic applications[edit] ***
** Viral infection[edit] **
Antiviral treatment is one of the earliest proposed RNAi-based medical
applications, and two different types have been developed. The first type is to
target viral RNAs. Many studies have shown that targeting viral RNAs can
suppress the replication of numerous viruses, including HIV,[143] HPV,[144]
hepatitis_A,[145] hepatitis_B,[146] Influenza_virus,[147][148][149][150]
Respiratory syncytial virus (RSV),[150] SARS coronavirus (SARS-CoV),[150]
Adenovirus[150] and Measles_virus.[151] The other strategy is to block the
initial viral entries by targeting the host cell genes.[152] For example,
suppression of chemokine receptors (CXCR4 and CCR5)on host cells can prevent
HIV viral entry.[153]
** Cancer[edit] **
While traditional chemotherapy can effectively kill cancer cells, lack of
specificity for discriminating normal cells and cancer cells in these
treatments usually cause severe side effects. Numerous studies have
demonstrated that RNAi can provide a more specific approach to inhibit tumor
growth by targeting cancer-related genes (i.e., oncogene).[154] It has also
been proposed that RNAi can enhance the sensitivity of cancer cells to
chemotherapeutic_agents, providing a combinatorial therapeutic approach with
chemotherapy.[155] Another potential RNAi-based treatment is to inhibit cell
invasion and migration.[156]
** Neurological diseases[edit] **
RNAi strategies also show potential for treating neurodegenerative_diseases.
Studies in cells and in mouse have shown that specifically targeting Amyloid
beta-producing genes (e.g. BACE1 and APP) by RNAi can significantly reduced the
amount of A√é¬≤ peptide which is correlated with the cause of Alzheimer's
disease.[157][158][159] In addition, this silencing-based approaches also
provide promising results in treatment of Parkinson's_disease and Polyglutamine
disease.[160][161][162]
*** Difficulties in Therapeutic Application[edit] ***
To achieve the clinical potential of RNAi, siRNA must be efficiently
transportated to the cells of target tissues. However, there are various
barriers that must be fixed before it can be used clinically. For_example,
"Naked"_siRNA is susceptible to several obstacles that reduce its therapeutic
efficacy.[163] Additionally, once siRNA has entered the bloodstream, naked RNA
can be degraded by serum nucleases and can stimulate the innate immune system.
[163] Due to its size and highly polyanionic (containing negative charges at
several sites) nature, unmodified siRNA molecules cannot readily enter the
cells through the cell membrane. Therefore, artificial or nanoparticle
encapsulated_siRNA must be used. However, transporting siRNA across the cell
membrane still has its own unique challenges. If siRNA is transferred across
the cell membrane, unintended toxicities can occur if therapeutic doses are not
optimized, and siRNAs can exhibit off-target effects (e.g. unintended
downregulation of genes with partial_sequence_complementarity).[164] Even after
entering the cells, repeated dosing is required since their effects are diluted
at each cell division. As previously described, parts of the vector that
transports dsRNA can also have regulatory effects. Therefore, unspecific side
effects should be considered and controlled.[165]
*** Safety and Uses in Cancer treatment[edit] ***
Compared with chemotherapy or other anti-cancer drugs, there are a lot of
advantages of siRNA drug.[166] SiRNA acts on the post-translational stage of
gene expression, so it does not modify or change DNA in a deleterious effect.
[166] SiRNA can also be used to produced a specific response in a certain type
of way, such as by downgrading suppression of gene expression.[166] In a single
cancer cell, siRNA can cause dramatic suppression of gene expression with just
several copies.[166] This happens by silencing cancer-promoting genes with
RNAi, as well as targeting an mRNA sequence.[166]
RNAi drugs treat cancer by silencing certain cancer promoting genes.[166] This
is done by complementing the cancer genes with the RNAi, such as keeping the
mRNA sequences in accordance with the RNAi drug.[166] Ideally, RNAi is should
be injected and/or chemically modified so the RNAi can reach cancer cells more
efficiently.[166] RNAi uptake and regulation is monitored by the kidneys.[166]
*** Stimulation of immune response[edit] ***
The human immune system is divided into two separate branches: the innate
immune system and the adaptive immune system.[167] The innate immune system is
the first defense against infection and responds to pathogens in a generic
fashion.[167] On the other hand, the adaptive immune system, a system that was
evolved later than the innate, is composed mainly of highly specialized B and T
cells that are trained to react to specific portions of pathogenic molecules.
[167]
The challenge between old pathogens and new has helped create a system of
guarded cells and particles that are called safe framework.[167] This framework
has given humans an army systems that search out and destroy invader particles,
such as pathogens, microscopic organisms, parasites, and infections.[167] The
mammalian safe framework has developed to incorporate siRNA as a tool to
indicate viral contamination, which has allowed siRNA is create an intense
innate immune response.[167]
siRNA is controlled by the innate immune system, which can be divided into the
acute inflammatory responses and antiviral responses.[167] The inflammatory
response is created with signals from small signaling molecules, or cytokines.
[167] These include interleukin-1 (IL-1), interleukin-6 (IL-6), interleukin-12
(IL-12) and tumor necrosis factor √é¬± (TNF-√é¬±).[167] The innate immune system
generates inflammation and antiviral responses, which cause the release pattern
recognition receptors (PRRs).[167] These receptors help in labeling which
pathogens are viruses, fungi, or bacteria.[167] Moreover, the importance of
siRNA and the innate immune system is to include more PRRs to help recognize
different RNA structures.[167] This makes it more likely for the siRNA to cause
an immunostimulant response in the event of the pathogen.[167]
*** Prospects as a Therapeutic Technique[edit] ***
Clinical Phase I and II studies of siRNA therapies conducted between 2015 and
2017 have demonstrated potent and durable gene_knockdown in the liver, with
some signs of clinical improvement and without unacceptable toxicity.[164] Two
Phase III studies are in progress to treat familial neurodegenerative and
cardiac syndromes caused by mutations in transthyretin (TTR).[164] Numerous
publications have shown that in vivo delivery systems are very promising and
are diverse in characteristics, allowing numerous applications. The
nanoparticle delivery system shows the most promise yet this method presents
additional challenges in the scale-up of the manufacturing process, such as the
need for tightly controlled mixing processes to achieve consistent quality of
the drug product.[163] The table below shows different drugs using RNA
interference and what their phases and status is in clinical trials.[163]
Drug                 Target       Delivery System   Disease               Phase Status     Company         Identifier
ALN√¢¬Ä¬ìVSP02       KSP and VEGF LNP               Solid tumours         I     Completed  Alnylam         NCT01158079
                                                                                           Pharmaceuticals
siRNA√¢¬Ä¬ìEphA2√¢¬Ä¬EphA2        LNP               Advanced cancers      I     Recruiting MD Anderson     NCT01591356
                                                                                           Cancer Center
Atu027               PKN3         LNP               Solid tumours         I     Completed  Silence         NCT00938574
                                                                                           Therapeutics
TKM√¢¬Ä¬ì080301      PLK1         LNP               Cancer                I     Recruiting Tekmira         NCT01262235
                                                                                           Pharmaceutical
                     VP24, VP35,                                                           Tekmira
TKM√¢¬Ä¬ì100201      Zaire Ebola  LNP               Ebola-virus infection I     Recruiting Pharmaceutical  NCT01518881
                     L-polymerase
ALN√¢¬Ä¬ìRSV01       RSV          Naked siRNA       Respiratory syncytial II    Completed  Alnylam         NCT00658086
                     nucleocapsid                   virus infections                       Pharmaceuticals
PRO-040201           ApoB         LNP               Hypercholesterolaemia I     Terminated Tekmira         NCT00927459
                                                                                           Pharmaceutical
ALN√¢¬Ä¬ìPCS02       PCSK9        LNP               Hypercholesterolaemia I     Completed  Alnylam         NCT01437059
                                                                                           Pharmaceuticals
ALN√¢¬Ä¬ìTTR02       TTR          LNP               Transthyretin-        II    Recruiting Alnylam         NCT01617967
                                                    mediated amyloidosis                   Pharmaceuticals
CALAA-01             RRM2         Cyclodextrin NP   Solid tumours         I     Active     Calando         NCT00689065
                                                                                           Pharmaceuticals
                     K6a (N171K                     Pachyonychia                           Pachyonychia
TD101                mutation)    Naked siRNA       congenita             I     Completed  Congenita       NCT00716014
                                                                                           Project
                                                    Age-related macular
AGN211745            VEGFR1       Naked siRNA       degeneration,         II    Terminated Allergan        NCT00395057
                                                    choroidal
                                                    neovascularization
                                                    Optic atrophy, non-
QPI-1007             CASP2        Naked siRNA       arteritic anterior    I     Completed  Quark           NCT01064505
                                                    ischaemic optic                        Pharmaceuticals
                                                    neuropathy
I5NP                 p53          Naked siRNA       Kidney injury, acute  I     Completed  Quark           NCT00554359
                                                    renal failure                          Pharmaceuticals
                                                    Delayed graft
                                                    function,             I, II Recruiting Quark           NCT00802347
                                                    complications of                       Pharmaceuticals
                                                    kidney transplant
                                                    Choroidal
                     RTP801                         neovascularization,                    Quark
PF-655 (PF-04523655) (Proprietary Naked siRNA       diabetic retinopathy, II    Active     Pharmaceuticals NCT01445899
                     target)                        diabetic macular
                                                    oedema
siG12D LODER         KRAS         LODER polymer     Pancreatic cancer     II    Recruiting Silenseed       NCT01676259
                                                    Diabetic macular
Bevasiranib          VEGF         Naked siRNA       oedema, macular       II    Completed  Opko Health     NCT00306904
                                                    degeneration
SYL1001              TRPV1        Naked siRNA       Ocular pain, dry-eye  I, II Recruiting Sylentis        NCT01776658
                                                    syndrome
SYL040012            ADRB2        Naked siRNA       Ocular hypertension,  II    Recruiting Sylentis        NCT01739244
                                                    open-angle glaucoma
CEQ508               CTNNB1       Escherichia coli- Familial adenomatous  I, II Recruiting Marina Biotech  Unknown
                                  carrying shRNA    polyposis
RXi-109              CTGF         Self-delivering   Cicatrix scar         I     Recruiting RXi             NCT01780077
                                  RNAi compound     prevention                             Pharmaceuticals
ALN√¢¬Ä¬ìTTRsc       TTR          siRNA√¢¬Ä¬ìGalNAc Transthyretin-        I     Recruiting Alnylam         NCT01814839
                                  conjugate         mediated amyloidosis                   Pharmaceuticals
                     Conserved                                                             Arrowhead
ARC-520              regions of   DPC               HBV                   I     Recruiting Research        NCT01872065
                     HBV
**** Biotechnology[edit] ****
RNA interference has been used for applications in biotechnology and is nearing
commercialization in other fields. RNAi has resulted in the invention of novel
crops such as nicotine-free tobacco, decaffeinated coffee, nutrient fortified
vegetation, and hypoallergenic crops. The genetically-engineered Arctic_apples
received FDA approval in 2015.[168] The apples were produced by RNAi
suppression of the PPO (polyphenol oxidase) gene, making apple varieties that
will not undergo browning after being sliced. PPO-silenced apples are unable to
convert chlorogenic acid into the standard quinone product.[1]
There are several opportunities for the applications of RNAi in crop science
for its improvement such as stress tolerance and enhanced nutritional level.
RNAi will prove its potential for inhibition of photorespiration to enhance the
productivity of C3 plants. This knockdown technology may be useful in inducing
early flowering, delayed ripening, delayed senescence, breaking dormancy,
stress-free plants, overcoming self-sterility, etc.[1]
*** Foods[edit] ***
RNAi has been used to genetically engineer plants to produce lower levels of
natural plant toxins. Such techniques take advantage of the stable and
heritable RNAi phenotype in plant stocks. Cotton seeds are rich in dietary
protein but naturally contain the toxic terpenoid product gossypol, making them
unsuitable for human consumption. RNAi has been used to produce cotton stocks
whose seeds contain reduced levels of delta-cadinene_synthase, a key enzyme in
gossypol production, without affecting the enzyme's production in other parts
of the plant, where gossypol is itself important in preventing damage from
plant pests.[169] Similar efforts have been directed toward the reduction of
the cyanogenic natural product linamarin in cassava plants.[170]
No plant products that use RNAi-based genetic_engineering have yet exited the
experimental stage. Development efforts have successfully reduced the levels of
allergens in tomato plants[171] and fortification of plants such as tomatoes
with dietary antioxidants.[172] Previous commercial products, including the
Flavr_Savr tomato and two cultivars of ringspot-resistant papaya, were
originally developed using antisense technology but likely exploited the RNAi
pathway.[173][174] RNAi silencing of alpha-amylase have also been used to
decrease Aspergillus_flavus fungal growth in maize which would have otherwise
contaminated the kernels with dangerous aflatoxins.[175] Silencing lachrymatory
factor synthase in onions have produced tearless onions and RNAi has been used
in BP1 genes in rapeseeds to improve photosynthesis.[176] SBEIIa and SBEIIb
genes in wheat have been targeted in wheat in order to produce higher levels of
amylose in order to improve bowel function.[177]
*** Other crops[edit] ***
Another effort decreased the precursors of likely carcinogens in tobacco
plants.[178] Other plant traits that have been engineered in the laboratory
include the production of non-narcotic natural products by the opium_poppy[179]
and resistance to common plant viruses.[180]
*** Insecticide[edit] ***
RNAi is under development as an insecticide, employing multiple approaches,
including genetic engineering and topical application.[3] Cells in the midgut
of some insects take up the dsRNA molecules in the process referred to as
environmental RNAi.[181] In some insects the effect is systemic as the signal
spreads throughout the insect's body (referred to as systemic RNAi).[182]
Animals exposed to RNAi at doses millions of times higher than anticipated
human exposure levels show no adverse effects.[183]
RNAi has varying effects in different species of Lepidoptera (butterflies and
moths).[184] Possibly because their saliva and gut juice is better at breaking
down RNA, the cotton_bollworm, the beet_armyworm and the Asiatic_rice_borer
have so far not been proven susceptible to RNAi by feeding.[3]
To develop resistance to RNAi, the western corn rootworm would have to change
the genetic sequence of its Snf7 gene at multiple sites. Combining multiple
strategies, such as engineering the protein Cry, derived from a bacterium
called Bacillus_thuringiensis (Bt), and RNAi in one plant delay the onset of
resistance.[3][185]
** Transgenic plants[edit] **
Transgenic_crops have been made to express dsRNA, carefully chosen to silence
crucial genes in target pests. These dsRNAs are designed to affect only insects
that express specific gene sequences. As a proof_of_principle, in 2009 a study
showed RNAs that could kill any one of four fruit fly species while not harming
the other three.[3]
In 2012 Syngenta bought Belgian RNAi firm Devgen for $522 million and Monsanto
paid $29.2 million for the exclusive rights to intellectual_property from
Alnylam_Pharmaceuticals. The International_Potato_Center in Lima,_Peru is
looking for genes to target in the sweet potato weevil, a beetle whose larvae
ravage sweet_potatoes globally. Other researchers are trying to silence genes
in ants, caterpillars and pollen beetles. Monsanto will likely be first to
market, with a transgenic corn seed that expresses dsRNA based on gene Snf7
from the western_corn_rootworm, a beetle whose larvae annually cause one
billion dollars in damage in the United States alone. A 2012 paper showed that
silencing Snf7 stunts larval growth, killing them within days. In 2013 the same
team showed that the RNA affects very few other species.[3]
** Topical[edit] **
Alternatively dsRNA can be supplied without genetic engineering. One approach
is to add them to irrigation water. The molecules are absorbed into the plants'
vascular system and poison insects feeding on them. Another approach involves
spraying dsRNA like a conventional pesticide. This would allow faster
adaptation to resistance. Such approaches would require low cost sources of
dsRNAs that do not currently exist.[3]
**** Genome-scale screening[edit] ****
Genome-scale RNAi research relies on high-throughput_screening (HTS)
technology. RNAi HTS technology allows genome-wide loss-of-function screening
and is broadly used in the identification of genes associated with specific
phenotypes. This technology has been hailed as a potential second genomics
wave, following the first genomics wave of gene_expression microarray and
single_nucleotide_polymorphism discovery platforms.[186] One major advantage of
genome-scale RNAi screening is its ability to simultaneously interrogate
thousands of genes. With the ability to generate a large amount of data per
experiment, genome-scale RNAi screening has led to an explosion of data
generation rates. Exploiting such large data sets is a fundamental challenge,
requiring suitable statistics/bioinformatics methods. The basic process of
cell-based RNAi screening includes the choice of an RNAi library, robust and
stable cell types, transfection with RNAi agents, treatment/incubation, signal
detection, analysis and identification of important genes or therapeutical
targets.[187]
***** History[edit] *****
Example petunia plants in which genes for pigmentation are silenced by RNAi.
The left plant is wild-type; the right plants contain transgenes that induce
suppression of both transgene and endogenous gene expression, giving rise to
the unpigmented white areas of the flower.[188]
The process of RNAi was referred to as "co-suppression" and "quelling" when
observed prior to the knowledge of an RNA-related mechanism. The discovery of
RNAi was preceded first by observations of transcriptional inhibition by
antisense RNA expressed in transgenic plants,[189] and more directly by reports
of unexpected outcomes in experiments performed by plant scientists in the
United_States and the Netherlands in the early 1990s.[190] In an attempt to
alter flower colors in petunias, researchers introduced additional copies of a
gene encoding chalcone_synthase, a key enzyme for flower pigmentation into
petunia plants of normally pink or violet flower color. The overexpressed gene
was expected to result in darker flowers, but instead caused some flowers to
have less visible purple pigment, sometimes in variegated patterns, indicating
that the activity of chalcone synthase had been substantially decreased or
became suppressed in a context-specific manner. This would later be explained
as the result of the transgene being inserted adjacent to promoters in the
opposite direction in various positions throughout the genomes of some
transformants, thus leading to expression of antisense transcripts and gene
silencing when these promoters are active. Another early observation of RNAi
was came from a study of the fungus Neurospora_crassa,[191] although it was not
immediately recognized as related. Further investigation of the phenomenon in
plants indicated that the downregulation was due to post-transcriptional
inhibition of gene expression via an increased rate of mRNA degradation.[192]
This phenomenon was called co-suppression of gene expression, but the molecular
mechanism remained unknown.[193]
Not long after, plant virologists working on improving plant resistance to
viral diseases observed a similar unexpected phenomenon. While it was known
that plants expressing virus-specific proteins showed enhanced tolerance or
resistance to viral infection, it was not expected that plants carrying only
short, non-coding regions of viral RNA sequences would show similar levels of
protection. Researchers believed that viral RNA produced by transgenes could
also inhibit viral replication.[194] The reverse experiment, in which short
sequences of plant genes were introduced into viruses, showed that the targeted
gene was suppressed in an infected plant.[195] This phenomenon was labeled
"virus-induced gene silencing" (VIGS), and the set of such phenomena were
collectively called post transcriptional gene silencing.[196]
After these initial observations in plants, laboratories searched for this
phenomenon in other organisms.[197][198] Craig_C._Mello and Andrew_Fire's 1998
Nature paper reported a potent gene silencing effect after injecting double
stranded RNA into C. elegans.[199] In investigating the regulation of muscle
protein production, they observed that neither mRNA nor antisense_RNA
injections had an effect on protein production, but double-stranded RNA
successfully silenced the targeted gene. As a result of this work, they coined
the term RNAi. This discovery represented the first identification of the
causative agent for the phenomenon. Fire and Mello were awarded the 2006 Nobel
Prize_in_Physiology_or_Medicine.[5]
***** See also[edit] *****
    * [icon]Metabolism_portal
    * Molecular_and_cellular_biology_portal
    * DNA-directed_RNA_interference
***** References[edit] *****
   1. ^ a b cSaurabh S, Vidyarthi AS, Prasad D (March 2014). "RNA interference:
      concept to reality in crop improvement". Planta. 239 (3): 543√¢¬Ä¬ì64. doi:
      10.1007/s00425-013-2019-5. PMID¬†24402564.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Weiss B, Davidkova G, Zhou LW (March 1999). "Antisense RNA gene therapy
      for studying and modulating biological processes". Cellular and Molecular
      Life Sciences. 55 (3): 334√¢¬Ä¬ì58. doi:10.1007/s000180050296.
      PMID¬†10228554.
   4. ^ a b c d e f gKupferschmidt, K. (2013). "A Lethal Dose of RNA". Science.
      341 (6147): 732√¢¬Ä¬ì3. doi:10.1126/science.341.6147.732. PMID¬†23950525.
   5. ^Macrae IJ, Zhou K, Li F, Repic A, Brooks AN, Cande WZ, Adams PD, Doudna
      JA (January 2006). "Structural basis for double-stranded RNA processing
      by Dicer". Science. 311 (5758): 195√¢¬Ä¬ì8. Bibcode:2006Sci...311..195M.
      doi:10.1126/science.1121638. PMID¬†16410517.
   6. ^ a b c dDaneholt B. "Advanced_Information:_RNA_interference". The Nobel
      Prize in Physiology or Medicine 2006. Archived from the_original on 20
      January 2007. Retrieved 25 January 2007.
   7. ^Bagasra O, Prilliman KR (August 2004). "RNA interference: the molecular
      immune system". Journal of Molecular Histology. 35 (6): 545√¢¬Ä¬ì53.
      CiteSeerX¬†10.1.1.456.1701. doi:10.1007/s10735-004-2192-8. PMID¬†15614608.
   8. ^Bernstein E, Caudy AA, Hammond SM, Hannon GJ (January 2001). "Role for a
      bidentate ribonuclease in the initiation step of RNA interference".
      Nature. 409 (6818): 363√¢¬Ä¬ì6. doi:10.1038/35053110. PMID¬†11201747.
   9.  [closed_access]
  10. ^Siomi H, Siomi MC (January 2009). "On the road to reading the RNA-
      interference code". Nature. 457 (7228): 396√¢¬Ä¬ì404. Bibcode:
      2009Natur.457..396S. doi:10.1038/nature07754. PMID¬†19158785.
  11.
      Zamore PD, Tuschl T, Sharp PA, Bartel DP (March 2000). "RNAi: double-
      stranded RNA directs the ATP-dependent cleavage of mRNA at 21 to 23
      nucleotide intervals". Cell. 101 (1): 25√¢¬Ä¬ì33. doi:10.1016/S0092-8674
      (00)80620-0. PMID¬†10778853.
  12.
      Vermeulen A, Behlen L, Reynolds A, Wolfson A, Marshall WS, Karpilow J,
      Khvorova A (May 2005). "The_contributions_of_dsRNA_structure_to_Dicer
      specificity_and_efficiency". RNA. 11 (5): 674√¢¬Ä¬ì82. doi:10.1261/
      rna.7272305. PMC¬†1370754. PMID¬†15811921.
  13.
      Castanotto D, Rossi JJ (January 2009). "The_promises_and_pitfalls_of_RNA-
      interference-based_therapeutics". Nature. 457 (7228): 426√¢¬Ä¬ì33. Bibcode:
      2009Natur.457..426C. doi:10.1038/nature07758. PMC¬†2702667. PMID¬†19158789.
  14. ^ a bQiu S, Adema CM, Lane T (2005). "A_computational_study_of_off-target
      effects_of_RNA_interference". Nucleic Acids Research. 33 (6): 1834√¢¬Ä¬ì47.
      doi:10.1093/nar/gki324. PMC¬†1072799. PMID¬†15800213.
  15. ^ a bNakanishi K (September 2016). "Anatomy_of_RISC:_how_do_small_RNAs
      and_chaperones_activate_Argonaute_proteins?". Wiley Interdisciplinary
      Reviews: RNA. 7 (5): 637√¢¬Ä¬ì60. doi:10.1002/wrna.1356. PMC¬†5084781.
      PMID¬†27184117.
  16. ^Liang C, Wang Y, Murota Y, Liu X, Smith D, Siomi MC, Liu Q (September
      2015). "TAF11_Assembles_the_RISC_Loading_Complex_to_Enhance_RNAi
      Efficiency". Molecular Cell. 59 (5): 807√¢¬Ä¬ì18. doi:10.1016/
      j.molcel.2015.07.006. PMC¬†4560963. PMID¬†26257286.
  17. ^Ahlquist P (2002). "RNA-dependent RNA polymerases, viruses, and RNA
      silencing". Science. 296 (5571): 1270√¢¬Ä¬ì3. Bibcode:2002Sci...296.1270A.
      doi:10.1126/science.1069132. PMID¬†12016304.
  18. ^Roberts, TC (2015). The microRNA Machinery. Advances in Experimental
      Medicine and Biology. 887. pp.¬†15√¢¬Ä¬ì30. doi:10.1007/978-3-319-22380-3_2.
      ISBN¬†978-3-319-22379-7. PMID¬†26662984.
  19. ^ a b c dParker G, Eckert D, Bass B (2006). "RDE-4_preferentially_binds
      long_dsRNA_and_its_dimerization_is_necessary_for_cleavage_of_dsRNA_to
      siRNA". RNA. 12 (5): 807√¢¬Ä¬ì18. doi:10.1261/rna.2338706. PMC¬†1440910.
      PMID¬†16603715.
  20. ^ a bBaulcombe DC (January 2007). "Molecular biology. Amplified
      silencing". Science. 315 (5809): 199√¢¬Ä¬ì200. doi:10.1126/science.1138030.
      PMID¬†17218517.
  21. ^ a bPak J, Fire A (January 2007). "Distinct populations of primary and
      secondary effectors during RNAi in C. elegans". Science. 315 (5809):
      241√¢¬Ä¬ì4. Bibcode:2007Sci...315..241P. doi:10.1126/science.1132839.
      PMID¬†17124291.
  22. ^ a bSijen T, Steiner FA, Thijssen KL, Plasterk RH (January 2007).
      "Secondary siRNAs result from unprimed RNA synthesis and form a distinct
      class". Science. 315 (5809): 244√¢¬Ä¬ì7. Bibcode:2007Sci...315..244S. doi:
      10.1126/science.1136699. PMID¬†17158288.
  23. ^Wang QL, Li ZH (May 2007). "The functions of microRNAs in plants".
      Frontiers in Bioscience. 12: 3975√¢¬Ä¬ì82. doi:10.2741/2364. PMID¬†17485351.
  24.
      Zhao Y, Srivastava D (April 2007). "A developmental view of microRNA
      function". Trends in Biochemical Sciences. 32 (4): 189√¢¬Ä¬ì97. doi:10.1016/
      j.tibs.2007.02.006. PMID¬†17350266.
  25. ^Gregory RI, Chendrimada TP, Shiekhattar R (2006). MicroRNA biogenesis:
      isolation and characterization of the microprocessor complex. Methods in
      Molecular Biology. 342. pp.¬†33√¢¬Ä¬ì47. doi:10.1385/1-59745-123-1:33.
      ISBN¬†978-1-59745-123-9. PMID¬†16957365.
  26. ^Pfeffer S, Zavolan M, Gr√É¬§sser FA, Chien M, Russo JJ, Ju J, John B,
      Enright AJ, Marks D, Sander C, Tuschl T (April 2004). "Identification of
      virus-encoded microRNAs". Science. 304 (5671): 734√¢¬Ä¬ì6. Bibcode:
      2004Sci...304..734P. doi:10.1126/science.1096781. PMID¬†15118162.
  27. ^Qureshi A, Thakur N, Monga I, Thakur A, Kumar M (1 January 2014).
      "VIRmiRNA:_a_comprehensive_resource_for_experimentally_validated_viral
      miRNAs_and_their_targets". Database. 2014: bau103. doi:10.1093/database/
      bau103. PMC¬†4224276. PMID¬†25380780.
  28. ^Pillai RS, Bhattacharyya SN, Filipowicz W (2007). "Repression of protein
      synthesis by miRNAs: how many mechanisms?". Trends Cell Biol. 17 (3):
      118√¢¬Ä¬ì26. doi:10.1016/j.tcb.2006.12.007. PMID¬†17197185.
  29. ^Okamura K, Ishizuka A, Siomi H, Siomi M (2004). "Distinct_roles_for
      Argonaute_proteins_in_small_RNA-directed_RNA_cleavage_pathways". Genes
      Dev. 18 (14): 1655√¢¬Ä¬ì66. doi:10.1101/gad.1210204. PMC¬†478188.
      PMID¬†15231716.
  30. ^Lee Y, Nakahara K, Pham J, Kim K, He Z, Sontheimer E, Carthew R (2004).
      "Distinct roles for Drosophila Dicer-1 and Dicer-2 in the siRNA/miRNA
      silencing pathways". Cell. 117 (1): 69√¢¬Ä¬ì81. doi:10.1016/S0092-8674
      (04)00261-2. PMID¬†15066283.
  31. ^ miRBase.org
  32. ^ a bFriedman RC, Farh KK, Burge CB, Bartel DP (2009). "Most_mammalian
      mRNAs_are_conserved_targets_of_microRNAs". Genome Res. 19 (1): 92√¢¬Ä¬ì105.
      doi:10.1101/gr.082701.108. PMC¬†2612969. PMID¬†18955434.
  33. ^Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, Castle J,
      Bartel DP, Linsley PS, Johnson JM (February 2005). "Microarray analysis
      shows that some microRNAs downregulate large numbers of target mRNAs".
      Nature. 433 (7027): 769√¢¬Ä¬ì73. Bibcode:2005Natur.433..769L. doi:10.1038/
      nature03315. PMID¬†15685193.
  34. ^Selbach M, Schwanh√É¬§usser B, Thierfelder N, Fang Z, Khanin R, Rajewsky N
      (September 2008). "Widespread changes in protein synthesis induced by
      microRNAs". Nature. 455 (7209): 58√¢¬Ä¬ì63. Bibcode:2008Natur.455...58S.
      doi:10.1038/nature07228. PMID¬†18668040.
  35. ^Baek D, Vill√É¬©n J, Shin C, Camargo FD, Gygi SP, Bartel DP (September
      2008). "The_impact_of_microRNAs_on_protein_output". Nature. 455 (7209):
      64√¢¬Ä¬ì71. Bibcode:2008Natur.455...64B. doi:10.1038/nature07242.
      PMC¬†2745094. PMID¬†18668037.
  36. ^Palmero EI, de Campos SG, Campos M, de Souza NC, Guerreiro ID, Carvalho
      AL, Marques MM (July 2011). "Mechanisms_and_role_of_microRNA_deregulation
      in_cancer_onset_and_progression". Genetics and Molecular Biology. 34 (3):
      363√¢¬Ä¬ì70. doi:10.1590/S1415-47572011000300001. PMC¬†3168173.
      PMID¬†21931505.
  37. ^Bernstein C, Bernstein H (May 2015). "Epigenetic_reduction_of_DNA_repair
      in_progression_to_gastrointestinal_cancer". World Journal of
      Gastrointestinal Oncology. 7 (5): 30√¢¬Ä¬ì46. doi:10.4251/wjgo.v7.i5.30.
      PMC¬†4434036. PMID¬†25987950.
  38. ^Maffioletti E, Tardito D, Gennarelli M, Bocchio-Chiavetto L (2014).
      "Micro_spies_from_the_brain_to_the_periphery:_new_clues_from_studies_on
      microRNAs_in_neuropsychiatric_disorders". Frontiers in Cellular
      Neuroscience. 8: 75. doi:10.3389/fncel.2014.00075. PMC¬†3949217.
      PMID¬†24653674.
  39. ^Mellios N, Sur M (2012). "The_Emerging_Role_of_microRNAs_in
      Schizophrenia_and_Autism_Spectrum_Disorders". Frontiers in Psychiatry. 3:
      39. doi:10.3389/fpsyt.2012.00039. PMC¬†3336189. PMID¬†22539927.
  40. ^Geaghan M, Cairns MJ (August 2015). "MicroRNA and Posttranscriptional
      Dysregulation in Psychiatry". Biological Psychiatry. 78 (4): 231√¢¬Ä¬ì9.
      doi:10.1016/j.biopsych.2014.12.009. PMID¬†25636176.
  41. ^Liu Q, Rand TA, Kalidas S, Du F, Kim HE, Smith DP, Wang X (September
      2003). "R2D2, a bridge between the initiation and effector steps of the
      Drosophila RNAi pathway". Science. 301 (5641): 1921√¢¬Ä¬ì5. Bibcode:
      2003Sci...301.1921L. doi:10.1126/science.1088710. PMID¬†14512631.
  42. ^Gregory RI, Chendrimada TP, Cooch N, Shiekhattar R (November 2005).
      "Human RISC couples microRNA biogenesis and posttranscriptional gene
      silencing". Cell. 123 (4): 631√¢¬Ä¬ì40. doi:10.1016/j.cell.2005.10.022.
      PMID¬†16271387.
  43. ^ a bLodish H, Berk A, Matsudaira P, Kaiser CA, Krieger M, Scott MP,
      Zipurksy SL, Darnell J (2004). Molecular_Cell_Biology (5th ed.). WH
      Freeman: New York, NY. ISBN¬†978-0-7167-4366-8.
  44. ^Matranga C, Tomari Y, Shin C, Bartel DP, Zamore PD (November 2005).
      "Passenger-strand cleavage facilitates assembly of siRNA into Ago2-
      containing RNAi enzyme complexes". Cell. 123 (4): 607√¢¬Ä¬ì20. doi:10.1016/
      j.cell.2005.08.044. PMID¬†16271386.
  45. ^Leuschner PJ, Ameres SL, Kueng S, Martinez J (March 2006). "Cleavage_of
      the_siRNA_passenger_strand_during_RISC_assembly_in_human_cells". EMBO
      Reports. 7 (3): 314√¢¬Ä¬ì20. doi:10.1038/sj.embor.7400637. PMC¬†1456892.
      PMID¬†16439995.
  46. ^ a bHaley B, Zamore PD (July 2004). "Kinetic analysis of the RNAi enzyme
      complex". Nature Structural & Molecular Biology. 11 (7): 599√¢¬Ä¬ì606. doi:
      10.1038/nsmb780. PMID¬†15170178.
  47. ^Schwarz DS, Hutv√É¬°gner G, Du T, Xu Z, Aronin N, Zamore PD (October
      2003). "Asymmetry in the assembly of the RNAi enzyme complex". Cell. 115
      (2): 199√¢¬Ä¬ì208. doi:10.1016/S0092-8674(03)00759-1. PMID¬†14567917.
  48. ^Preall JB, He Z, Gorra JM, Sontheimer EJ (March 2006). "Short
      interfering RNA strand selection is independent of dsRNA processing
      polarity during RNAi in Drosophila". Current Biology. 16 (5): 530√¢¬Ä¬ì5.
      doi:10.1016/j.cub.2006.01.061. PMID¬†16527750.
  49. ^Tomari Y, Matranga C, Haley B, Martinez N, Zamore PD (November 2004). "A
      protein sensor for siRNA asymmetry". Science. 306 (5700): 1377√¢¬Ä¬ì80.
      Bibcode:2004Sci...306.1377T. doi:10.1126/science.1102755. PMID¬†15550672.
  50. ^Ma JB, Yuan YR, Meister G, Pei Y, Tuschl T, Patel DJ (March 2005).
      "Structural_basis_for_5'-end-specific_recognition_of_guide_RNA_by_the_A.
      fulgidus_Piwi_protein". Nature. 434 (7033): 666√¢¬Ä¬ì70. Bibcode:
      2005Natur.434..666M. doi:10.1038/nature03514. PMC¬†4694588. PMID¬†15800629.
  51. ^Sen G, Wehrman T, Blau H (2005). "mRNA translation is not a prerequisite
      for small interfering RNA-mediated mRNA cleavage". Differentiation. 73
      (6): 287√¢¬Ä¬ì93. doi:10.1111/j.1432-0436.2005.00029.x. PMID¬†16138829.
  52. ^Gu S, Rossi J (2005). "Uncoupling_of_RNAi_from_active_translation_in
      mammalian_cells". RNA. 11 (1): 38√¢¬Ä¬ì44. doi:10.1261/rna.7158605.
      PMC¬†1370689. PMID¬†15574516.
  53. ^Sen G, Blau H (2005). "Argonaute 2/RISC resides in sites of mammalian
      mRNA decay known as cytoplasmic bodies". Nat Cell Biol. 7 (6): 633√¢¬Ä¬ì6.
      doi:10.1038/ncb1265. PMID¬†15908945.
  54. ^Lian S, Jakymiw A, Eystathioy T, Hamel J, Fritzler M, Chan E (2006). "GW
      bodies, microRNAs and the cell cycle". Cell Cycle. 5 (3): 242√¢¬Ä¬ì5. doi:
      10.4161/cc.5.3.2410. PMID¬†16418578.
  55. ^Jakymiw A, Lian S, Eystathioy T, Li S, Satoh M, Hamel J, Fritzler M,
      Chan E (2005). "Disruption of P bodies impairs mammalian RNA
      interference". Nat Cell Biol. 7 (12): 1267√¢¬Ä¬ì74. doi:10.1038/ncb1334.
      PMID¬†16284622.
  56. ^Hammond S, Bernstein E, Beach D, Hannon_G (2000). "An RNA-directed
      nuclease mediates post-transcriptional gene silencing in Drosophila
      cells". Nature. 404 (6775): 293√¢¬Ä¬ì6. doi:10.1038/35005107. PMID¬†10749213.
  57.  [closed_access]
  58. ^Holmquist GP, Ashley T (2006). "Chromosome organization and chromatin
      modification: influence on genome function and evolution". Cytogenetic
      and Genome Research. 114 (2): 96√¢¬Ä¬ì125. doi:10.1159/000093326.
      PMID¬†16825762.
  59. ^Verdel A, Jia S, Gerber S, Sugiyama T, Gygi S, Grewal SI, Moazed D
      (January 2004). "RNAi-mediated_targeting_of_heterochromatin_by_the_RITS
      complex". Science. 303 (5658): 672√¢¬Ä¬ì6. Bibcode:2004Sci...303..672V. doi:
      10.1126/science.1093686. PMC¬†3244756. PMID¬†14704433.
  60. ^Irvine DV, Zaratiegui M, Tolia NH, Goto DB, Chitwood DH, Vaughn MW,
      Joshua-Tor L, Martienssen RA (August 2006). "Argonaute slicing is
      required for heterochromatic silencing and spreading". Science. 313
      (5790): 1134√¢¬Ä¬ì7. Bibcode:2006Sci...313.1134I. doi:10.1126/
      science.1128813. PMID¬†16931764.
  61. ^Volpe TA, Kidner C, Hall IM, Teng G, Grewal SI, Martienssen RA
      (September 2002). "Regulation of heterochromatic silencing and histone H3
      lysine-9 methylation by RNAi". Science. 297 (5588): 1833√¢¬Ä¬ì7. Bibcode:
      2002Sci...297.1833V. doi:10.1126/science.1074973. PMID¬†12193640.
  62. ^Volpe T, Schramke V, Hamilton GL, White SA, Teng G, Martienssen RA,
      Allshire RC (2003). "RNA interference is required for normal centromere
      function in fission yeast". Chromosome Research. 11 (2): 137√¢¬Ä¬ì46. doi:
      10.1023/A:1022815931524. PMID¬†12733640.
  63. ^Li LC, Okino ST, Zhao H, Pookot D, Place RF, Urakami S, Enokida H,
      Dahiya R (November 2006). "Small_dsRNAs_induce_transcriptional_activation
      in_human_cells". Proceedings of the National Academy of Sciences of the
      United States of America. 103 (46): 17337√¢¬Ä¬ì42. Bibcode:
      2006PNAS..10317337L. doi:10.1073/pnas.0607015103. PMC¬†1859931.
      PMID¬†17085592.
  64. ^Noma K, Sugiyama T, Cam H, Verdel A, Zofall M, Jia S, Moazed D, Grewal
      SI (November 2004). "RITS acts in cis to promote RNA interference-
      mediated transcriptional and post-transcriptional silencing". Nature
      Genetics. 36 (11): 1174√¢¬Ä¬ì80. doi:10.1038/ng1452. PMID¬†15475954.
  65. ^Sugiyama T, Cam H, Verdel A, Moazed D, Grewal SI (January 2005). "RNA-
      dependent_RNA_polymerase_is_an_essential_component_of_a_self-enforcing
      loop_coupling_heterochromatin_assembly_to_siRNA_production". Proceedings
      of the National Academy of Sciences of the United States of America. 102
      (1): 152√¢¬Ä¬ì7. Bibcode:2005PNAS..102..152S. doi:10.1073/pnas.0407641102.
      PMC¬†544066. PMID¬†15615848.
  66. ^Wang F, Koyama N, Nishida H, Haraguchi T, Reith W, Tsukamoto T (June
      2006). "The_assembly_and_maintenance_of_heterochromatin_initiated_by
      transgene_repeats_are_independent_of_the_RNA_interference_pathway_in
      mammalian_cells". Molecular and Cellular Biology. 26 (11): 4028√¢¬Ä¬ì40.
      doi:10.1128/MCB.02189-05. PMC¬†1489094. PMID¬†16705157.
  67. ^Bass BL (2002). "RNA_editing_by_adenosine_deaminases_that_act_on_RNA".
      Annual Review of Biochemistry. 71: 817√¢¬Ä¬ì46. doi:10.1146/
      annurev.biochem.71.110601.135501. PMC¬†1823043. PMID¬†12045112.
  68. ^Bass BL (April 2000). "Double-stranded RNA as a template for gene
      silencing". Cell. 101 (3): 235√¢¬Ä¬ì8. doi:10.1016/S0092-8674(02)71133-1.
      PMID¬†10847677.
  69. ^Luciano DJ, Mirsky H, Vendetti NJ, Maas S (August 2004). "RNA_editing_of
      a_miRNA_precursor". RNA. 10 (8): 1174√¢¬Ä¬ì7. doi:10.1261/rna.7350304.
      PMC¬†1370607. PMID¬†15272117.
  70. ^ a bYang W, Chendrimada TP, Wang Q, Higuchi M, Seeburg PH, Shiekhattar
      R, Nishikura K (January 2006). "Modulation_of_microRNA_processing_and
      expression_through_RNA_editing_by_ADAR_deaminases". Nature Structural &
      Molecular Biology. 13 (1): 13√¢¬Ä¬ì21. doi:10.1038/nsmb1041. PMC¬†2950615.
      PMID¬†16369484.
  71. ^Yang W, Wang Q, Howell KL, Lee JT, Cho DS, Murray JM, Nishikura K
      (February 2005). "ADAR1_RNA_deaminase_limits_short_interfering_RNA
      efficacy_in_mammalian_cells". The Journal of Biological Chemistry. 280
      (5): 3946√¢¬Ä¬ì53. doi:10.1074/jbc.M407876200. PMC¬†2947832. PMID¬†15556947.
  72. ^Nishikura K (December 2006). "Editor_meets_silencer:_crosstalk_between
      RNA_editing_and_RNA_interference". Nature Reviews. Molecular Cell
      Biology. 7 (12): 919√¢¬Ä¬ì31. doi:10.1038/nrm2061. PMC¬†2953463.
      PMID¬†17139332.
  73. ^ a b cSaumet A, Lecellier CH (2006). "Anti-viral_RNA_silencing:_do_we
      look_like_plants¬†?". Retrovirology. 3 (1): 3. doi:10.1186/1742-4690-3-3.
      PMC¬†1363733. PMID¬†16409629.
  74. ^Jones L, Ratcliff F, Baulcombe DC (May 2001). "RNA-directed
      transcriptional_gene_silencing_in_plants_can_be_inherited_independently
      of_the_RNA_trigger_and_requires_Met1_for_maintenance". Current Biology.
      11 (10): 747√¢¬Ä¬ì57. doi:10.1016/S0960-9822(01)00226-3. PMID¬†11378384.
      Archived from the_original on 21 December 2007.
  75. ^Humphreys DT, Westman BJ, Martin DI, Preiss T (November 2005).
      "MicroRNAs_control_translation_initiation_by_inhibiting_eukaryotic
      initiation_factor_4E/cap_and_poly(A)_tail_function". Proceedings of the
      National Academy of Sciences of the United States of America. 102 (47):
      16961√¢¬Ä¬ì6. Bibcode:2005PNAS..10216961H. doi:10.1073/pnas.0506482102.
      PMC¬†1287990. PMID¬†16287976.
  76. ^DaRocha WD, Otsu K, Teixeira SM, Donelson JE (February 2004). "Tests of
      cytoplasmic RNA interference (RNAi) and construction of a tetracycline-
      inducible T7 promoter system in Trypanosoma cruzi". Molecular and
      Biochemical Parasitology. 133 (2): 175√¢¬Ä¬ì86. doi:10.1016/
      j.molbiopara.2003.10.005. PMID¬†14698430.
  77. ^Robinson KA, Beverley SM (May 2003). "Improvements in transfection
      efficiency and tests of RNA interference (RNAi) approaches in the
      protozoan parasite Leishmania". Molecular and Biochemical Parasitology.
      128 (2): 217√¢¬Ä¬ì28. doi:10.1016/S0166-6851(03)00079-3. PMID¬†12742588.
  78. ^Aravind L, Watanabe H, Lipman DJ & Koonin EV (October 2000). "Lineage-
      specific_loss_and_divergence_of_functionally_linked_genes_in_eukaryotes".
      Proceedings of the National Academy of Sciences of the United States of
      America. 97 (21): 11319√¢¬Ä¬ì24. Bibcode:2000PNAS...9711319A. doi:10.1073/
      pnas.200346997. PMC¬†17198. PMID¬†11016957.
  79. ^Drinnenberg IA, Weinberg DE, Xie KT, Mower JP, Wolfe KH, Fink GR, Bartel
      DP (October 2009). "RNAi_in_budding_yeast". Science. 326 (5952):
      544√¢¬Ä¬ì550. Bibcode:2009Sci...326..544D. doi:10.1126/science.1176945.
      PMC¬†3786161. PMID¬†19745116.
  80. ^Nakayashiki H, Kadotani N, Mayama S (July 2006). "Evolution and
      diversification of RNA silencing proteins in fungi". Journal of Molecular
      Evolution. 63 (1): 127√¢¬Ä¬ì35. Bibcode:2006JMolE..63..127N. doi:10.1007/
      s00239-005-0257-2. PMID¬†16786437.
  81. ^Morita T, Mochizuki Y, Aiba H (March 2006). "Translational_repression_is
      sufficient_for_gene_silencing_by_bacterial_small_noncoding_RNAs_in_the
      absence_of_mRNA_destruction". Proceedings of the National Academy of
      Sciences of the United States of America. 103 (13): 4858√¢¬Ä¬ì63. Bibcode:
      2006PNAS..103.4858M. doi:10.1073/pnas.0509638103. PMC¬†1458760.
      PMID¬†16549791.
  82. ^Makarova KS, Grishin NV, Shabalina SA, Wolf YI, Koonin EV (March 2006).
      "A_putative_RNA-interference-based_immune_system_in_prokaryotes:
      computational_analysis_of_the_predicted_enzymatic_machinery,_functional
      analogies_with_eukaryotic_RNAi,_and_hypothetical_mechanisms_of_action".
      Biology Direct. 1: 7. doi:10.1186/1745-6150-1-7. PMC¬†1462988.
      PMID¬†16545108.
  83. ^Stram Y, Kuzntzova L (June 2006). "Inhibition of viruses by RNA
      interference". Virus Genes. 32 (3): 299√¢¬Ä¬ì306. doi:10.1007/s11262-005-
      6914-0. PMID¬†16732482.
  84. ^Blevins T, Rajeswaran R, Shivaprasad PV, Beknazariants D, Si-Ammour A,
      Park HS, Vazquez F, Robertson D, Meins F, Hohn T, Pooggin MM (2006).
      "Four_plant_Dicers_mediate_viral_small_RNA_biogenesis_and_DNA_virus
      induced_silencing". Nucleic Acids Research. 34 (21): 6233√¢¬Ä¬ì46. doi:
      10.1093/nar/gkl886. PMC¬†1669714. PMID¬†17090584.
  85. ^Palauqui JC, Elmayan T, Pollien JM, Vaucheret H (August 1997). "Systemic
      acquired_silencing:_transgene-specific_post-transcriptional_silencing_is
      transmitted_by_grafting_from_silenced_stocks_to_non-silenced_scions". The
      EMBO Journal. 16 (15): 4738√¢¬Ä¬ì45. doi:10.1093/emboj/16.15.4738.
      PMC¬†1170100. PMID¬†9303318.
  86. ^Voinnet O (August 2001). "RNA silencing as a plant immune system against
      viruses". Trends in Genetics. 17 (8): 449√¢¬Ä¬ì59. doi:10.1016/S0168-9525
      (01)02367-8. PMID¬†11485817.
  87. ^ a bLucy AP, Guo HS, Li WX, Ding SW (April 2000). "Suppression_of_post-
      transcriptional_gene_silencing_by_a_plant_viral_protein_localized_in_the
      nucleus". The EMBO Journal. 19 (7): 1672√¢¬Ä¬ì80. doi:10.1093/emboj/
      19.7.1672. PMC¬†310235. PMID¬†10747034.
  88. ^M√É¬©rai Z, Ker√É¬©nyi Z, Kert√É¬©sz S, Magna M, Lakatos L, Silhavy D (June
      2006). "Double-stranded_RNA_binding_may_be_a_general_plant_RNA_viral
      strategy_to_suppress_RNA_silencing". Journal of Virology. 80 (12):
      5747√¢¬Ä¬ì56. doi:10.1128/JVI.01963-05. PMC¬†1472586. PMID¬†16731914.
  89. ^Katiyar-Agarwal S, Morgan R, Dahlbeck D, Borsani O, Villegas A, Zhu JK,
      Staskawicz BJ, Jin H (November 2006). "A_pathogen-inducible_endogenous
      siRNA_in_plant_immunity". Proceedings of the National Academy of Sciences
      of the United States of America. 103 (47): 18002√¢¬Ä¬ì7. Bibcode:
      2006PNAS..10318002K. doi:10.1073/pnas.0608258103. PMC¬†1693862.
      PMID¬†17071740.
  90. ^Fritz JH, Girardin SE, Philpott DJ (June 2006). "Innate immune defense
      through RNA interference". Science's STKE. 2006 (339): pe27. doi:10.1126/
      stke.3392006pe27. PMID¬†16772641.
  91. ^Zambon RA, Vakharia VN, Wu LP (May 2006). "RNAi is an antiviral immune
      response against a dsRNA virus in Drosophila melanogaster". Cellular
      Microbiology. 8 (5): 880√¢¬Ä¬ì9. doi:10.1111/j.1462-5822.2006.00688.x.
      PMID¬†16611236.
  92. ^Wang XH, Aliyari R, Li WX, Li HW, Kim K, Carthew R, Atkinson P, Ding SW
      (April 2006). "RNA_interference_directs_innate_immunity_against_viruses
      in_adult_Drosophila". Science. 312 (5772): 452√¢¬Ä¬ì4. Bibcode:
      2006Sci...312..452W. doi:10.1126/science.1125694. PMC¬†1509097.
      PMID¬†16556799.
  93. ^Lu R, Maduro M, Li F, Li HW, Broitman-Maduro G, Li WX, Ding SW (August
      2005). "Animal_virus_replication_and_RNAi-mediated_antiviral_silencing_in
      Caenorhabditis_elegans". Nature. 436 (7053): 1040√¢¬Ä¬ì1043. Bibcode:
      2005Natur.436.1040L. doi:10.1038/nature03870. PMC¬†1388260. PMID¬†16107851.
  94. ^Wilkins C, Dishongh R, Moore SC, Whitt MA, Chow M, Machaca K (August
      2005). "RNA interference is an antiviral defence mechanism in
      Caenorhabditis elegans". Nature. 436 (7053): 1044√¢¬Ä¬ì7. Bibcode:
      2005Natur.436.1044W. doi:10.1038/nature03957. PMID¬†16107852.
  95. ^Berkhout B, Haasnoot J (May 2006). "The interplay between virus
      infection and the cellular RNA interference machinery". FEBS Letters. 580
      (12): 2896√¢¬Ä¬ì902. doi:10.1016/j.febslet.2006.02.070. PMID¬†16563388.
  96. ^Sch√É¬ºtz S, Sarnow P (January 2006). "Interaction of viruses with the
      mammalian RNA interference pathway". Virology. 344 (1): 151√¢¬Ä¬ì7. doi:
      10.1016/j.virol.2005.09.034. PMID¬†16364746.
  97. ^Cullen BR (June 2006). "Is RNA interference involved in intrinsic
      antiviral immunity in mammals?". Nature Immunology. 7 (6): 563√¢¬Ä¬ì7. doi:
      10.1038/ni1352. PMID¬†16715068.
  98. ^Maillard PV, Ciaudo C, Marchais A, Li Y, Jay F, Ding SW, Voinnet O
      (October 2013). "Antiviral_RNA_interference_in_mammalian_cells". Science.
      342 (6155): 235√¢¬Ä¬ì8. Bibcode:2013Sci...342..235M. doi:10.1126/
      science.1241930. PMC¬†3853215. PMID¬†24115438.
  99. ^Li Y, Lu J, Han Y, Fan X, Ding SW (October 2013). "RNA_interference
      functions_as_an_antiviral_immunity_mechanism_in_mammals". Science. 342
      (6155): 231√¢¬Ä¬ì4. Bibcode:2013Sci...342..231L. doi:10.1126/
      science.1241911. PMC¬†3875315. PMID¬†24115437.
 100. ^Li HW, Ding SW (October 2005). "Antiviral_silencing_in_animals". FEBS
      Letters. 579 (26): 5965√¢¬Ä¬ì73. doi:10.1016/j.febslet.2005.08.034.
      PMC¬†1350842. PMID¬†16154568.
 101. ^Carrington JC, Ambros V (July 2003). "Role of microRNAs in plant and
      animal development". Science. 301 (5631): 336√¢¬Ä¬ì8. Bibcode:
      2003Sci...301..336C. doi:10.1126/science.1085242. PMID¬†12869753.
 102. ^Lee RC, Feinbaum RL, Ambros V (December 1993). "The C. elegans
      heterochronic gene lin-4 encodes small RNAs with antisense
      complementarity to lin-14". Cell. 75 (5): 843√¢¬Ä¬ì54. doi:10.1016/0092-8674
      (93)90529-Y. PMID¬†8252621.
 103. ^Palatnik JF, Allen E, Wu X, Schommer C, Schwab R, Carrington JC, Weigel
      D (September 2003). "Control_of_leaf_morphogenesis_by_microRNAs". Nature.
      425 (6955): 257√¢¬Ä¬ì63. Bibcode:2003Natur.425..257P. doi:10.1038/
      nature01958. PMID¬†12931144.
 104. ^Zhang B, Pan X, Cobb GP, Anderson TA (January 2006). "Plant microRNA: a
      small regulatory molecule with big impact". Developmental Biology. 289
      (1): 3√¢¬Ä¬ì16. doi:10.1016/j.ydbio.2005.10.036. PMID¬†16325172.
 105. ^Jones-Rhoades MW, Bartel DP, Bartel B (2006). "MicroRNAS and their
      regulatory roles in plants". Annual Review of Plant Biology. 57: 19√¢¬Ä¬ì53.
      doi:10.1146/annurev.arplant.57.032905.105218. PMID¬†16669754.
 106. ^Zhang B, Pan X, Cobb GP, Anderson TA (February 2007). "microRNAs as
      oncogenes and tumor suppressors". Developmental Biology. 302 (1): 1√¢¬Ä¬ì12.
      doi:10.1016/j.ydbio.2006.08.028. PMID¬†16989803.
 107. ^Check E (August 2007). "RNA interference: hitting the on switch".
      Nature. 448 (7156): 855√¢¬Ä¬ì8. Bibcode:2007Natur.448..855C. doi:10.1038/
      448855a. PMID¬†17713502.
 108.
      Li LC, Okino ST, Zhao H, Pookot D, Place RF, Urakami S, Enokida H, Dahiya
      R (November 2006). "Small_dsRNAs_induce_transcriptional_activation_in
      human_cells". Proceedings of the National Academy of Sciences of the
      United States of America. 103 (46): 17337√¢¬Ä¬ì42. Bibcode:
      2006PNAS..10317337L. doi:10.1073/pnas.0607015103. PMC¬†1859931.
      PMID¬†17085592.
 109. ^Vasudevan S, Tong Y, Steitz JA (December 2007). "Switching from
      repression to activation: microRNAs can up-regulate translation".
      Science. 318 (5858): 1931√¢¬Ä¬ì4. Bibcode:2007Sci...318.1931V. doi:10.1126/
      science.1149460. PMID¬†18048652.
 110. ^ a b cCerutti H, Casas-Mollano JA (August 2006). "On_the_origin_and
      functions_of_RNA-mediated_silencing:_from_protists_to_man". Current
      Genetics. 50 (2): 81√¢¬Ä¬ì99. doi:10.1007/s00294-006-0078-x. PMC¬†2583075.
      PMID¬†16691418.
 111. ^Anantharaman V, Koonin EV, Aravind L (April 2002). "Comparative_genomics
      and_evolution_of_proteins_involved_in_RNA_metabolism". Nucleic Acids
      Research. 30 (7): 1427√¢¬Ä¬ì64. doi:10.1093/nar/30.7.1427. PMC¬†101826.
      PMID¬†11917006.
 112. ^Buchon N, Vaury C (February 2006). "RNAi: a defensive RNA-silencing
      against viruses and transposable elements". Heredity. 96 (2): 195√¢¬Ä¬ì202.
      doi:10.1038/sj.hdy.6800789. PMID¬†16369574.
 113. ^Obbard DJ, Jiggins FM, Halligan DL, Little TJ (March 2006). "Natural
      selection drives extremely rapid evolution in antiviral RNAi genes".
      Current Biology. 16 (6): 580√¢¬Ä¬ì5. doi:10.1016/j.cub.2006.01.065.
      PMID¬†16546082.
 114. ^Voorhoeve PM, Agami R (January 2003). "Knockdown stands up". Trends in
      Biotechnology. 21 (1): 2√¢¬Ä¬ì4. doi:10.1016/S0167-7799(02)00002-1.
      PMID¬†12480342.
 115. ^Munk√É¬°csy G, Sztupinszki Z, Herman P, B√É¬°n B, P√É¬©nzv√É¬°lt√É¬≥ Z, Szarvas N,
      Gy√Ö¬ërffy B (September 2016). "Validation_of_RNAi_Silencing_Efficiency
      Using_Gene_Array_Data_shows_18.5%_Failure_Rate_across_429_Independent
      Experiments". Molecular Therapy. Nucleic Acids. 5 (9): e366. doi:10.1038/
      mtna.2016.66. PMC¬†5056990. PMID¬†27673562.
 116. ^Naito Y, Yamada T, Matsumiya T, Ui-Tei K, Saigo K, Morishita S (July
      2005). "dsCheck:_highly_sensitive_off-target_search_software_for_double-
      stranded_RNA-mediated_RNA_interference". Nucleic Acids Research. 33 (Web
      Server issue): W589√¢¬Ä¬ì91. doi:10.1093/nar/gki419. PMC¬†1160180.
      PMID¬†15980542.
 117. ^Henschel A, Buchholz F, Habermann B (July 2004). "DEQOR:_a_web-based
      tool_for_the_design_and_quality_control_of_siRNAs". Nucleic Acids
      Research. 32 (Web Server issue): W113√¢¬Ä¬ì20. doi:10.1093/nar/gkh408.
      PMC¬†441546. PMID¬†15215362.
 118. ^Naito Y, Yamada T, Ui-Tei K, Morishita S, Saigo K (July 2004).
      "siDirect:_highly_effective,_target-specific_siRNA_design_software_for
      mammalian_RNA_interference". Nucleic Acids Research. 32 (Web Server
      issue): W124√¢¬Ä¬ì9. doi:10.1093/nar/gkh442. PMC¬†441580. PMID¬†15215364.
 119. ^Naito Y, Ui-Tei K, Nishikawa T, Takebe Y, Saigo K (July 2006). "siVirus:
      web-based_antiviral_siRNA_design_software_for_highly_divergent_viral
      sequences". Nucleic Acids Research. 34 (Web Server issue): W448√¢¬Ä¬ì50.
      doi:10.1093/nar/gkl214. PMC¬†1538817. PMID¬†16845046.
 120. ^Reynolds A, Anderson EM, Vermeulen A, Fedorov Y, Robinson K, Leake D,
      Karpilow J, Marshall WS, Khvorova A (June 2006). "Induction_of_the
      interferon_response_by_siRNA_is_cell_type-_and_duplex_length-dependent".
      RNA. 12 (6): 988√¢¬Ä¬ì93. doi:10.1261/rna.2340906. PMC¬†1464853.
      PMID¬†16611941.
 121. ^Stein P, Zeng F, Pan H, Schultz RM (October 2005). "Absence of non-
      specific effects of RNA interference triggered by long double-stranded
      RNA in mouse oocytes". Developmental Biology. 286 (2): 464√¢¬Ä¬ì71. doi:
      10.1016/j.ydbio.2005.08.015. PMID¬†16154556.
 122. ^Brummelkamp TR, Bernards_R, Agami R (April 2002). "A system for stable
      expression of short interfering RNAs in mammalian cells". Science. 296
      (5567): 550√¢¬Ä¬ì3. Bibcode:2002Sci...296..550B. doi:10.1126/
      science.1068999. hdl:1874/15573. PMID¬†11910072.
 123. ^Tiscornia G, Tergaonkar V, Galimi F, Verma IM (May 2004). "CRE
      recombinase-inducible_RNA_interference_mediated_by_lentiviral_vectors".
      Proceedings of the National Academy of Sciences of the United States of
      America. 101 (19): 7347√¢¬Ä¬ì51. Bibcode:2004PNAS..101.7347T. doi:10.1073/
      pnas.0402107101. PMC¬†409921. PMID¬†15123829.
 124. ^Ventura A, Meissner A, Dillon CP, McManus M, Sharp PA, Van Parijs L,
      Jaenisch R, Jacks T (July 2004). "Cre-lox-regulated_conditional_RNA
      interference_from_transgenes". Proceedings of the National Academy of
      Sciences of the United States of America. 101 (28): 10380√¢¬Ä¬ì5. Bibcode:
      2004PNAS..10110380V. doi:10.1073/pnas.0403954101. PMC¬†478580.
      PMID¬†15240889.
 125. ^Kamath RS, Ahringer J (August 2003). "Genome-wide RNAi screening in
      Caenorhabditis elegans". Methods. 30 (4): 313√¢¬Ä¬ì21. doi:10.1016/S1046-
      2023(03)00050-1. PMID¬†12828945.
 126. ^Boutros M, Kiger AA, Armknecht S, Kerr K, Hild M, Koch B, Haas SA, Paro
      R, Perrimon N (February 2004). "Genome-wide RNAi analysis of growth and
      viability in Drosophila cells". Science. 303 (5659): 832√¢¬Ä¬ì5. Bibcode:
      2004Sci...303..832B. doi:10.1126/science.1091266. PMID¬†14764878.
 127. ^Fortunato A, Fraser AG (2005). "Uncover genetic interactions in
      Caenorhabditis elegans by RNA interference". Bioscience Reports. 25
      (5√¢¬Ä¬ì6): 299√¢¬Ä¬ì307. doi:10.1007/s10540-005-2892-7. PMID¬†16307378.
 128. ^Cullen LM, Arndt GM (June 2005). "Genome-wide screening for gene
      function using RNAi in mammalian cells". Immunology and Cell Biology. 83
      (3): 217√¢¬Ä¬ì23. doi:10.1111/j.1440-1711.2005.01332.x. PMID¬†15877598.
 129. ^Huesken D, Lange J, Mickanin C, Weiler J, Asselbergs F, Warner J, Meloon
      B, Engel S, Rosenberg A, Cohen D, Labow M, Reinhardt M, Natt F, Hall J
      (August 2005). "Design of a genome-wide siRNA library using an artificial
      neural network". Nature Biotechnology. 23 (8): 995√¢¬Ä¬ì1001. doi:10.1038/
      nbt1118. PMID¬†16025102.
 130. ^Ge G, Wong GW, Luo B (October 2005). "Prediction of siRNA knockdown
      efficiency using artificial neural network models". Biochemical and
      Biophysical Research Communications. 336 (2): 723√¢¬Ä¬ì8. doi:10.1016/
      j.bbrc.2005.08.147. PMID¬†16153609.
 131. ^Janitz M, Vanhecke D, Lehrach H (2006). "High-Throughput RNA
      Interference in Functional Genomics". RNA Towards Medicine. Handbook of
      Experimental Pharmacology. 173. pp.¬†97√¢¬Ä¬ì104. doi:10.1007/3-540-27262-
      3_5. ISBN¬†978-3-540-27261-8. PMID¬†16594612.
 132. ^Vanhecke D, Janitz M (February 2005). "Functional_genomics_using_high-
      throughput_RNA_interference". Drug Discovery Today. 10 (3): 205√¢¬Ä¬ì12.
      doi:10.1016/S1359-6446(04)03352-5. PMID¬†15708535.
 133. ^Geldhof P, Murray L, Couthier A, Gilleard JS, McLauchlan G, Knox DP,
      Britton C (June 2006). "Testing the efficacy of RNA interference in
      Haemonchus contortus". International Journal for Parasitology. 36 (7):
      801√¢¬Ä¬ì10. doi:10.1016/j.ijpara.2005.12.004. PMID¬†16469321.
 134. ^Geldhof P, Visser A, Clark D, Saunders G, Britton C, Gilleard J,
      Berriman M, Knox D (May 2007). "RNA interference in parasitic helminths:
      current situation, potential pitfalls and future prospects".
      Parasitology. 134 (Pt 5): 609√¢¬Ä¬ì19. doi:10.1017/S0031182006002071.
      PMID¬†17201997.
 135. ^Travella S, Klimm TE, Keller B (September 2006). "RNA_interference-based
      gene_silencing_as_an_efficient_tool_for_functional_genomics_in_hexaploid
      bread_wheat". Plant Physiology. 142 (1): 6√¢¬Ä¬ì20. doi:10.1104/
      pp.106.084517. PMC¬†1557595. PMID¬†16861570.
 136. ^McGinnis K, Chandler V, Cone K, Kaeppler H, Kaeppler S, Kerschen A,
      Pikaard C, Richards E, Sidorenko L, Smith T, Springer N, Wulan T (2005).
      "Transgene-Induced RNA Interference as a Tool for Plant Functional
      Genomics". RNA Interference. Methods in Enzymology. 392. pp.¬†1√¢¬Ä¬ì24. doi:
      10.1016/S0076-6879(04)92001-0. ISBN¬†978-0-12-182797-7. PMID¬†15644172.
 137. ^ a bSen GL, Blau HM (July 2006). "A brief history of RNAi: the silence
      of the genes". FASEB Journal. 20 (9): 1293√¢¬Ä¬ì9. doi:10.1096/fj.06-
      6014rev. PMID¬†16816104.
 138. ^Daneholt B (2 October 2006). "The_Nobel_Prize_in_Physiology_or_Medicine
      2006". nobelprize.org. Retrieved 30 October 2017.
 139. ^ a bMoffatt S (29 December 2016). "siRNA-based Nanoparticles for Cancer
      Therapy: Hurdles and Hopes". MOJ Proteomics & Bioinformatics. 4 (6). doi:
      10.15406/mojpb.2016.04.00142.
 140. ^McCaffrey AP, Meuse L, Pham TT, Conklin DS, Hannon_GJ, Kay MA (July
      2002). "RNA interference in adult mice". Nature. 418 (6893): 38√¢¬Ä¬ì9.
      Bibcode:2002Natur.418...38M. doi:10.1038/418038a. PMID¬†12097900.
 141.  [closed_access]
 142. ^Devi GR (September 2006). "siRNA-based approaches in cancer therapy".
      Cancer Gene Therapy. 13 (9): 819√¢¬Ä¬ì29. doi:10.1038/sj.cgt.7700931.
      PMID¬†16424918.
 143. ^Wall NR, Shi Y (October 2003). "Small RNA: can RNA interference be
      exploited for therapy?". Lancet. 362 (9393): 1401√¢¬Ä¬ì3. doi:10.1016/s0140-
      6736(03)14637-5. PMID¬†14585643.
 144. ^Davis ME, Zuckerman JE, Choi CH, Seligson D, Tolcher A, Alabi CA, Yen Y,
      Heidel JD, Ribas A (April 2010). "Evidence_of_RNAi_in_humans_from
      systemically_administered_siRNA_via_targeted_nanoparticles". Nature. 464
      (7291): 1067√¢¬Ä¬ì70. Bibcode:2010Natur.464.1067D. doi:10.1038/nature08956.
      PMC¬†2855406. PMID¬†20305636.
 145. ^Escobar MA, Civerolo EL, Summerfelt KR, Dandekar AM (November 2001).
      "RNAi-mediated_oncogene_silencing_confers_resistance_to_crown_gall
      tumorigenesis". Proceedings of the National Academy of Sciences of the
      United States of America. 98 (23): 13437√¢¬Ä¬ì42. Bibcode:
      2001PNAS...9813437E. doi:10.1073/pnas.241276898. PMC¬†60889.
      PMID¬†11687652.
 146. ^Pereira TC, Pascoal VD, Marchesini RB, Maia IG, Magalh√É¬£es LA, Zanotti-
      Magalh√É¬£es EM, Lopes-Cendes I (April 2008). "Schistosoma mansoni:
      evaluation of an RNAi-based treatment targeting HGPRTase gene".
      Experimental Parasitology. 118 (4): 619√¢¬Ä¬ì23. doi:10.1016/
      j.exppara.2007.11.017. PMID¬†18237732.
 147. ^Raoul C, Abbas-Terki T, Bensadoun JC, Guillot S, Haase G, Szulc J,
      Henderson CE, Aebischer P (April 2005). "Lentiviral-mediated silencing of
      SOD1 through RNA interference retards disease onset and progression in a
      mouse model of ALS". Nature Medicine. 11 (4): 423√¢¬Ä¬ì8. doi:10.1038/
      nm1207. PMID¬†15768028.
 148. ^Ptasznik A, Nakata Y, Kalota A, Emerson SG, Gewirtz AM (November 2004).
      "Short interfering RNA (siRNA) targeting the Lyn kinase induces apoptosis
      in primary, and drug-resistant, BCR-ABL1(+) leukemia cells". Nature
      Medicine. 10 (11): 1187√¢¬Ä¬ì9. doi:10.1038/nm1127. PMID¬†15502840.
 149. ^Kim SJ, Lee WI, Lee YS, Kim DH, Chang JW, Kim SW, Lee H (November 2009).
      "Effective_relief_of_neuropathic_pain_by_adeno-associated_virus-mediated
      expression_of_a_small_hairpin_RNA_against_GTP_cyclohydrolase_1".
      Molecular Pain. 5: 1744√¢¬Ä¬ì8069√¢¬Ä¬ì5√¢¬Ä¬ì67. doi:10.1186/1744-8069-5-67.
      PMC¬†2785765. PMID¬†19922668.
 150. ^Chen L, McKenna JT, Bolortuya Y, Winston S, Thakkar MM, Basheer R, Brown
      RE, McCarley RW (November 2010). "Knockdown_of_orexin_type_1_receptor_in
      rat_locus_coeruleus_increases_REM_sleep_during_the_dark_period". The
      European Journal of Neuroscience. 32 (9): 1528√¢¬Ä¬ì36. doi:10.1111/j.1460-
      9568.2010.07401.x. PMC¬†3058252. PMID¬†21089218.
 151. ^Vargason JM, Szittya G, Burgy√É¬°n J, Hall TM (December 2003). "Size
      selective recognition of siRNA by an RNA silencing suppressor". Cell. 115
      (7): 799√¢¬Ä¬ì811. doi:10.1016/s0092-8674(03)00984-x. PMID¬†14697199.
 152. ^Berkhout B (April 2004). "RNA interference as an antiviral approach:
      targeting HIV-1". Current Opinion in Molecular Therapeutics. 6 (2):
      141√¢¬Ä¬ì5. PMID¬†15195925.
 153. ^Jiang M, Milner J (September 2002). "Selective silencing of viral gene
      expression in HPV-positive human cervical carcinoma cells treated with
      siRNA, a primer of RNA interference". Oncogene. 21 (39): 6041√¢¬Ä¬ì8. doi:
      10.1038/sj.onc.1205878. PMID¬†12203116.
 154. ^Kusov Y, Kanda T, Palmenberg A, Sgro JY, Gauss-M√É¬ºller V (June 2006).
      "Silencing_of_hepatitis_A_virus_infection_by_small_interfering_RNAs".
      Journal of Virology. 80 (11): 5599√¢¬Ä¬ì610. doi:10.1128/jvi.01773-05.
      PMC¬†1472172. PMID¬†16699041.
 155. ^Jia F, Zhang YZ, Liu CM (October 2006). "A retrovirus-based system to
      stably silence hepatitis B virus genes by RNA interference".
      Biotechnology Letters. 28 (20): 1679√¢¬Ä¬ì85. doi:10.1007/s10529-006-9138-z.
      PMID¬†16900331.
 156. ^Li YC, Kong LH, Cheng BZ, Li KS (December 2005). "Construction of
      influenza virus siRNA expression vectors and their inhibitory effects on
      multiplication of influenza virus". Avian Diseases. 49 (4): 562√¢¬Ä¬ì73.
      doi:10.1637/7365-041205R2.1. PMID¬†16405000.
 157. ^Khanna M, Saxena L, Rajput R, Kumar B, Prasad R (2015). "Gene silencing:
      a therapeutic approach to combat influenza virus infections". Future
      Microbiology. 10 (1): 131√¢¬Ä¬ì40. doi:10.2217/fmb.14.94. PMID¬†25598342.
 158. ^Rajput R, Khanna M, Kumar P, Kumar B, Sharma S, Gupta N, Saxena L
      (December 2012). "Small interfering RNA targeting the nonstructural gene
      1 transcript inhibits influenza A virus replication in experimental
      mice". Nucleic Acid Therapeutics. 22 (6): 414√¢¬Ä¬ì22. doi:10.1089/
      nat.2012.0359. PMID¬†23062009.
 159. ^ a b c dAsha K, Kumar P, Sanicas M, Meseko CA, Khanna M, Kumar B
      (December 2018). "Advancements_in_Nucleic_Acid_Based_Therapeutics_against
      Respiratory_Viral_Infections". Journal of Clinical Medicine. 8 (1): 6.
      doi:10.3390/jcm8010006. PMC¬†6351902. PMID¬†30577479.
 160. ^Hu L, Wang Z, Hu C, Liu X, Yao L, Li W, Qi Y (2005). "Inhibition of
      Measles virus multiplication in cell culture by RNA interference". Acta
      Virologica. 49 (4): 227√¢¬Ä¬ì34. PMID¬†16402679.
 161. ^Qureshi A, Thakur N, Monga I, Thakur A, Kumar M (2014). "VIRmiRNA:_a
      comprehensive_resource_for_experimentally_validated_viral_miRNAs_and
      their_targets". Database. 2014. doi:10.1093/database/bau103. PMC¬†4224276.
      PMID¬†25380780.
 162. ^Crowe S (2003). "Suppression of chemokine receptor expression by RNA
      interference allows for inhibition of HIV-1 replication, by Mart√É¬≠nez et
      al". AIDS. 17 Suppl 4: S103√¢¬Ä¬ì5. PMID¬†15080188.
 163. ^Fuchs U, Damm-Welk C, Borkhardt A (August 2004). "Silencing_of_disease-
      related_genes_by_small_interfering_RNAs". Current Molecular Medicine. 4
      (5): 507√¢¬Ä¬ì17. doi:10.2174/1566524043360492. PMID¬†15267222.
 164. ^Cioca DP, Aoki Y, Kiyosawa K (February 2003). "RNA interference is a
      functional pathway with therapeutic potential in human myeloid leukemia
      cell lines". Cancer Gene Therapy. 10 (2): 125√¢¬Ä¬ì33. doi:10.1038/
      sj.cgt.7700544. PMID¬†12536201.
 165. ^Lapteva N, Yang AG, Sanders DE, Strube RW, Chen SY (January 2005).
      "CXCR4 knockdown by small interfering RNA abrogates breast tumor growth
      in vivo". Cancer Gene Therapy. 12 (1): 84√¢¬Ä¬ì9. doi:10.1038/
      sj.cgt.7700770. PMID¬†15472715.
 166. ^Singer O, Marr RA, Rockenstein E, Crews L, Coufal NG, Gage FH, Verma IM,
      Masliah E (October 2005). "Targeting BACE1 with siRNAs ameliorates
      Alzheimer disease neuropathology in a transgenic model". Nature
      Neuroscience. 8 (10): 1343√¢¬Ä¬ì9. doi:10.1038/nn1531. PMID¬†16136043.
 167. ^Rodr√É¬≠guez-Lebr√É¬≥n E, Gouvion CM, Moore SA, Davidson BL, Paulson HL
      (September 2009). "Allele-specific_RNAi_mitigates_phenotypic_progression
      in_a_transgenic_model_of_Alzheimer's_disease". Molecular Therapy. 17 (9):
      1563√¢¬Ä¬ì73. doi:10.1038/mt.2009.123. PMC¬†2835271. PMID¬†19532137.
 168. ^Piedrahita D, Hern√É¬°ndez I, L√É¬≥pez-Tob√É¬≥n A, Fedorov D, Obara B,
      Manjunath BS, Boudreau RL, Davidson B, Laferla F, Gallego-G√É¬≥mez JC,
      Kosik KS, Cardona-G√É¬≥mez GP (October 2010). "Silencing_of_CDK5_reduces
      neurofibrillary_tangles_in_transgenic_alzheimer's_mice". The Journal of
      Neuroscience. 30 (42): 13966√¢¬Ä¬ì76. doi:10.1523/jneurosci.3637-10.2010.
      PMC¬†3003593. PMID¬†20962218.
 169. ^Raoul C, Barker SD, Aebischer P (March 2006). "Viral-based modelling and
      correction of neurodegenerative diseases by RNA interference". Gene
      Therapy. 13 (6): 487√¢¬Ä¬ì95. doi:10.1038/sj.gt.3302690. PMID¬†16319945.
 170. ^Harper SQ, Staber PD, He X, Eliason SL, Martins IH, Mao Q, Yang L, Kotin
      RM, Paulson HL, Davidson BL (April 2005). "RNA_interference_improves
      motor_and_neuropathological_abnormalities_in_a_Huntington's_disease_mouse
      model". Proceedings of the National Academy of Sciences of the United
      States of America. 102 (16): 5820√¢¬Ä¬ì5. Bibcode:2005PNAS..102.5820H. doi:
      10.1073/pnas.0501507102. PMC¬†556303. PMID¬†15811941.
 171. ^Boudreau RL, Rodr√É¬≠guez-Lebr√É¬≥n E, Davidson BL (April 2011). "RNAi
      medicine_for_the_brain:_progresses_and_challenges". Human Molecular
      Genetics. 20 (R1): R21√¢¬Ä¬ì7. doi:10.1093/hmg/ddr137. PMC¬†3095054.
      PMID¬†21459775.
 172. ^ a b c dKanasty R, Dorkin JR, Vegas A, Anderson D (November 2013).
      "Delivery materials for siRNA therapeutics". Nature Materials. 12 (11):
      967√¢¬Ä¬ì77. Bibcode:2013NatMa..12..967K. doi:10.1038/nmat3765.
      PMID¬†24150415.
 173. ^ a b cWittrup A, Lieberman J (September 2015). "Knocking_down_disease:_a
      progress_report_on_siRNA_therapeutics". Nature Reviews. Genetics. 16 (9):
      543√¢¬Ä¬ì52. doi:10.1038/nrg3978. PMC¬†4756474. PMID¬†26281785.
 174. ^ Evandro A De-Souza, Henrique Camara, Willian G Salgueiro, Ra√É¬≠ssa P
      Moro, Thiago L Knittel, Guilherme Tonon, Silas Pinto, Ana Paula F Pinca,
      Adam Antebi, Amy E Pasquinelli, Katlin B Massirer, Marcelo A Mori, RNA
      interference may result in unexpected phenotypes in Caenorhabditis
      elegans, Nucleic Acids Research, Volume 47, Issue 8, 07 May 2019, Pages
      3957√¢¬Ä¬ì3969, https://doi.org/10.1093/nar/gkz154
 175. ^ a b c d e f g h iXu C, Wang J (1 February 2015). "Delivery systems for
      siRNA drug development in cancer therapy". Asian Journal of
      Pharmaceutical Sciences. 10 (1): 1√¢¬Ä¬ì12. doi:10.1016/j.ajps.2014.08.011.
 176. ^ a b c d e f g h i j k l mWhitehead KA, Dahlman JE, Langer RS, Anderson
      DG (2011). "Silencing or stimulation? siRNA delivery and the immune
      system". Annual Review of Chemical and Biomolecular Engineering. 2:
      77√¢¬Ä¬ì96. doi:10.1146/annurev-chembioeng-061010-114133. PMID¬†22432611.
 177. ^"FDA_concludes_Arctic_Apples_and_Innate_Potatoes_are_safe_for
      consumption". Retrieved 29 September 2017.
 178. ^Sunilkumar G, Campbell LM, Puckhaber L, Stipanovic RD, Rathore KS
      (November 2006). "Engineering_cottonseed_for_use_in_human_nutrition_by
      tissue-specific_reduction_of_toxic_gossypol". Proceedings of the National
      Academy of Sciences of the United States of America. 103 (48): 18054√¢¬Ä¬ì9.
      doi:10.1073/pnas.0605389103. PMC¬†1838705. PMID¬†17110445.
 179. ^Siritunga D, Sayre RT (July 2003). "Generation of cyanogen-free
      transgenic cassava". Planta. 217 (3): 367√¢¬Ä¬ì73. doi:10.1007/s00425-003-
      1005-8. PMID¬†14520563.
 180. ^Le LQ, Lorenz Y, Scheurer S, F√É¬∂tisch K, Enrique E, Bartra J, Biemelt S,
      Vieths S, Sonnewald U (March 2006). "Design of tomato fruits with reduced
      allergenicity by dsRNAi-mediated inhibition of ns-LTP (Lyc e 3)
      expression". Plant Biotechnology Journal. 4 (2): 231√¢¬Ä¬ì42. doi:10.1111/
      j.1467-7652.2005.00175.x. PMID¬†17177799.
 181. ^Niggeweg R, Michael AJ, Martin C (June 2004). "Engineering plants with
      increased levels of the antioxidant chlorogenic acid". Nature
      Biotechnology. 22 (6): 746√¢¬Ä¬ì54. doi:10.1038/nbt966. PMID¬†15107863.
 182. ^Sanders RA, Hiatt W (March 2005). "Tomato transgene structure and
      silencing". Nature Biotechnology. 23 (3): 287√¢¬Ä¬ì9. doi:10.1038/nbt0305-
      287b. PMID¬†15765076.
 183. ^Chiang CH, Wang JJ, Jan FJ, Yeh SD, Gonsalves D (November 2001).
      "Comparative reactions of recombinant papaya ringspot viruses with
      chimeric coat protein (CP) genes and wild-type viruses on CP-transgenic
      papaya". The Journal of General Virology. 82 (Pt 11): 2827√¢¬Ä¬ì36. doi:
      10.1099/0022-1317-82-11-2827. PMID¬†11602796.
 184. ^Gilbert MK, Majumdar R, Rajasekaran K, Chen ZY, Wei Q, Sickler CM, Lebar
      MD, Cary JW, Frame BR, Wang K (June 2018). "RNA_interference-based
      silencing_of_the_alpha-amylase_(amy1)_gene_in_Aspergillus_flavus
      decreases_fungal_growth_and_aflatoxin_production_in_maize_kernels" (PDF).
      Planta. 247 (6): 1465√¢¬Ä¬ì1473. doi:10.1007/s00425-018-2875-0.
      PMID¬†29541880.
 185. ^Katoch R, Thakur N (March 2013). "RNA interference: a promising
      technique for the improvement of traditional crops". International
      Journal of Food Sciences and Nutrition. 64 (2): 248√¢¬Ä¬ì59. doi:10.3109/
      09637486.2012.713918. PMID¬†22861122.
 186. ^Katoch R, Thakur N (March 2013). "Advances in RNA interference
      technology and its impact on nutritional improvement, disease and insect
      control in plants". Applied Biochemistry and Biotechnology. 169 (5):
      1579√¢¬Ä¬ì605. doi:10.1007/s12010-012-0046-5. PMID¬†23322250.
 187. ^Gavilano LB, Coleman NP, Burnley LE, Bowman ML, Kalengamaliro NE, Hayes
      A, Bush L, Siminszky B (November 2006). "Genetic engineering of Nicotiana
      tabacum for reduced nornicotine content". Journal of Agricultural and
      Food Chemistry. 54 (24): 9071√¢¬Ä¬ì8. doi:10.1021/jf0610458. PMID¬†17117792.
 188. ^Allen RS, Millgate AG, Chitty JA, Thisleton J, Miller JA, Fist AJ,
      Gerlach WL, Larkin PJ (December 2004). "RNAi-mediated replacement of
      morphine with the nonnarcotic alkaloid reticuline in opium poppy". Nature
      Biotechnology. 22 (12): 1559√¢¬Ä¬ì66. doi:10.1038/nbt1033. PMID¬†15543134.
 189. ^Zadeh AH, Foster GD (2004). "Transgenic resistance to tobacco ringspot
      virus". Acta Virologica. 48 (3): 145√¢¬Ä¬ì52. PMID¬†15595207.
 190. ^Ivashuta S, Zhang Y, Wiggins BE, Ramaseshadri P, Segers GC, Johnson S,
      Meyer SE, Kerstetter RA, McNulty BC, Bolognesi R, Heck GR (May 2015).
      "Environmental_RNAi_in_herbivorous_insects". RNA. 21 (5): 840√¢¬Ä¬ì50. doi:
      10.1261/rna.048116.114. PMC¬†4408792. PMID¬†25802407.
 191. ^Miller SC, Miyata K, Brown SJ, Tomoyasu Y (2012). "Dissecting_systemic
      RNA_interference_in_the_red_flour_beetle_Tribolium_castaneum:_parameters
      affecting_the_efficiency_of_RNAi". PLOS One. 7 (10): e47431. doi:10.1371/
      journal.pone.0047431. PMC¬†3484993. PMID¬†23133513.
 192. ^Petrick JS, Friedrich GE, Carleton SM, Kessenich CR, Silvanovich A,
      Zhang Y, Koch MS (November 2016). "Corn rootworm-active RNA DvSnf7:
      Repeat dose oral toxicology assessment in support of human and mammalian
      safety". Regulatory Toxicology and Pharmacology. 81: 57√¢¬Ä¬ì68. doi:
      10.1016/j.yrtph.2016.07.009. PMID¬†27436086.
 193. ^Terenius O, Papanicolaou A, Garbutt JS, Eleftherianos I, Huvenne H,
      Kanginakudru S, et al. (February 2011). "RNA_interference_in_Lepidoptera:
      an_overview_of_successful_and_unsuccessful_studies_and_implications_for
      experimental_design". Journal of Insect Physiology. 57 (2): 231√¢¬Ä¬ì45.
      doi:10.1016/j.jinsphys.2010.11.006. PMID¬†21078327.
 194. ^ The_EPA_Quietly_Approved_Monsanto's_New_Genetic-Engineering_Technology:
      It√¢¬Ä¬ôs_the_first_time_RNA_interference_will_be_used_to_kill_insect_pests.
      Vincent Kessler Reuters Sarah Zhang
 195. ^ Matson RS (2005). Applying genomic and proteomic microarray technology
      in drug discovery. CRC Press. p.¬†6. ISBN¬†978-0-8493-1469-8.
 196. ^Zhang XHD (2011). Optimal_High-Throughput_Screening:_Practical
      Experimental_Design_and_Data_Analysis_for_Genome-scale_RNAi_Research.
      Cambridge University Press. pp.¬†ix√¢¬Ä¬ìxiii. ISBN¬†978-0-521-73444-8.
 197. ^Matzke MA, Matzke AJ (2004). "Planting_the_Seeds_of_a_New_Paradigm".
      PLoS Biol. 2 (5): e133. doi:10.1371/journal.pbio.0020133. PMC¬†406394.
      PMID¬†15138502.
 198. ^Ecker JR, Davis RW (August 1986). "Inhibition_of_gene_expression_in
      plant_cells_by_expression_of_antisense_RNA". Proceedings of the National
      Academy of Sciences of the United States of America. 83 (15): 5372√¢¬Ä¬ì6.
      Bibcode:1986PNAS...83.5372E. doi:10.1073/pnas.83.15.5372. PMC¬†386288.
      PMID¬†16593734.
 199. ^Napoli C, Lemieux C, Jorgensen R (April 1990). "Introduction_of_a
      Chimeric_Chalcone_Synthase_Gene_into_Petunia_Results_in_Reversible_Co-
      Suppression_of_Homologous_Genes_in_trans". The Plant Cell. 2 (4):
      279√¢¬Ä¬ì289. doi:10.1105/tpc.2.4.279. PMC¬†159885. PMID¬†12354959.
 200. ^Romano N, Macino G (November 1992). "Quelling: transient inactivation of
      gene expression in Neurospora crassa by transformation with homologous
      sequences". Molecular Microbiology. 6 (22): 3343√¢¬Ä¬ì53. doi:10.1111/
      j.1365-2958.1992.tb02202.x. PMID¬†1484489.
 201. ^Van Blokland R, Van der Geest N, Mol JN, Kooter JM (1994). "Transgene-
      mediated suppression of chalcone synthase expression in Petunia hybrida
      results from an increase in RNA turnover". Plant J. 6 (6): 861√¢¬Ä¬ì77. doi:
      10.1046/j.1365-313X.1994.6060861.x.
 202. ^Mol JN, van der Krol AR (1991). Antisense nucleic acids and proteins:
      fundamentals and applications. M. Dekker. pp.¬†4, 136. ISBN¬†978-0-8247-
      8516-1.
 203. ^Covey S, Al-Kaff N, L√É¬°ngara A, Turner D (1997). "Plants combat
      infection by gene silencing". Nature. 385 (6619): 781√¢¬Ä¬ì2. Bibcode:
      1997Natur.385..781C. doi:10.1038/385781a0.
 204. ^Kumagai MH, Donson J, della-Cioppa G, Harvey D, Hanley K, Grill LK
      (February 1995). "Cytoplasmic_inhibition_of_carotenoid_biosynthesis_with
      virus-derived_RNA". Proceedings of the National Academy of Sciences of
      the United States of America. 92 (5): 1679√¢¬Ä¬ì83. Bibcode:
      1995PNAS...92.1679K. doi:10.1073/pnas.92.5.1679. PMC¬†42583. PMID¬†7878039.
 205. ^Ratcliff F, Harrison BD, Baulcombe DC (June 1997). "A similarity between
      viral defense and gene silencing in plants". Science. 276 (5318):
      1558√¢¬Ä¬ì60. doi:10.1126/science.276.5318.1558. PMID¬†18610513.
 206. ^Guo S, Kemphues KJ (May 1995). "par-1, a gene required for establishing
      polarity in C. elegans embryos, encodes a putative Ser/Thr kinase that is
      asymmetrically distributed". Cell. 81 (4): 611√¢¬Ä¬ì20. doi:10.1016/0092-
      8674(95)90082-9. PMID¬†7758115.
 207. ^Pal-Bhadra M, Bhadra U, Birchler JA (August 1997). "Cosuppression in
      Drosophila: gene silencing of Alcohol dehydrogenase by white-Adh
      transgenes is Polycomb dependent". Cell. 90 (3): 479√¢¬Ä¬ì90. doi:10.1016/
      S0092-8674(00)80508-5. PMID¬†9267028.
 208. ^Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC (February
      1998). "Potent and specific genetic interference by double-stranded RNA
      in Caenorhabditis elegans". Nature. 391 (6669): 806√¢¬Ä¬ì11. Bibcode:
      1998Natur.391..806F. doi:10.1038/35888. PMID¬†9486653.
***** External links[edit] *****
 Wikimedia Commons has media related to RNA_interference.
 Wikiversity has learning resources about RNA_interference
    * Overview_of_the_RNAi_process, from Cambridge_University's_The_Naked
      Scientists
    * Animation_of_the_RNAi_process, from Nature
    * NOVA_scienceNOW_explains_RNAi¬†√¢¬Ä¬ì A 15-minute video of the Nova broadcast
      that aired on PBS, 26 July 2005
    * Silencing_Genomes RNA interference (RNAi) experiments and bioinformatics
      in C. elegans for education. From the Dolan DNA Learning Center of Cold
      Spring Harbor Laboratory.
    * RNAi_screens_in_C._elegans_in_a_96-well_liquid_format_and_their
      application_to_the_systematic_identification_of_genetic_interactions_(a
      protocol)
    * 2_American_√¢¬Ä¬òWorm_People√¢¬Ä¬ô_Win_Nobel_for_RNA_Work, from NY_Times
    * Molecular_Therapy_web_focus:_"The_development_of_RNAi_as_a_therapeutic
      strategy", a collection of free articles about RNAi as a therapeutic
      strategy.
    * GenomeRNAi: a database of phenotypes from RNA interference screening
      experiments in Drosophila melanogaster and Homo sapians
    * RNAi_tools Pre-designed and custom RNA Interference tools
    * v
    * t
    * e
Breakthrough_of_the_Year
            * 1996: HIV understanding
            * 1997: Dolly_the_sheep
            * 1998: Accelerating_universe
            * 1999: Stem_cell
            * 2000: Whole_genome_sequencing
            * 2001: Nanocircuits or Molecular_circuit
            * 2002: RNA interference
            * 2003: Dark_energy
            * 2004: Spirit_rover
            * 2005: Evolution in action
Science     * 2006: Poincar√É¬©_conjecture proof
journal     * 2007: Human_genetic_variation
            * 2008: Cellular_reprogramming
            * 2009: Ardipithecus_ramidus
            * 2010: First quantum_machine
            * 2011: HPTN_052 clinical trial
            * 2012: Higgs_boson discovery
            * 2013: Cancer_immunotherapy
            * 2014: Rosetta_comet_mission
            * 2015: CRISPR genome-editing method
            * 2016: First_observation of gravitational_waves
            * 2017: GW170817 (neutron_star_merger)
            * 2018: Development_cell_by_cell

Retrieved from "https://en.wikipedia.org/w/
index.php?title=RNA_interference&oldid=908702684"
Categories:
    * RNA
    * RNA_interference
    * Gene_expression
    * Molecular_genetics
Hidden categories:
    * CS1:_long_volume_value
    * Use_dmy_dates_from_August_2018
    * Articles_with_short_description
    * Commons_category_link_is_on_Wikidata
    * Featured_articles
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
‚Å∞
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
‚Å∞
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * √ò¬ß√ô¬Ñ√ò¬π√ò¬±√ò¬®√ô¬ä√ò¬©
    * Aragon√É¬©s
    * B√É¬¢n-l√É¬¢m-g√É¬∫
    * Catal√É¬†
    * √Ñ¬åe√Ö¬°tina
    * Dansk
    * Deutsch
    * Eesti
    * √é¬ï√é¬ª√é¬ª√é¬∑√é¬Ω√é¬π√é¬∫√é¬¨
    * Espa√É¬±ol
    * √ô¬Å√ò¬ß√ò¬±√ò¬≥√õ¬å
    * Fran√É¬ßais
    * Galego
    * √≠¬ï¬ú√™¬µ¬≠√¨¬ñ¬¥
    * √†¬§¬π√†¬§¬ø√†¬§¬®√†¬•¬ç√†¬§¬¶√†¬•¬Ä
    * Bahasa_Indonesia
    * Italiano
    * √ó¬¢√ó¬ë√ó¬®√ó¬ô√ó¬™
    * √†¬≤¬ï√†¬≤¬®√†¬≥¬ç√†¬≤¬®√†¬≤¬°
    * Latvie√Ö¬°u
    * Magyar
    * Nederlands
    * √¶¬ó¬•√¶¬ú¬¨√®¬™¬û
    * Norsk
    * Norsk_nynorsk
    * Polski
    * Portugu√É¬™s
    * √ê¬†√ë¬É√ë¬Å√ë¬Å√ê¬∫√ê¬∏√ê¬π
    * Scots
    * Simple_English
    * √ê¬°√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬∏_/_srpski
    * Suomi
    * Svenska
    * √†¬Æ¬§√†¬Æ¬Æ√†¬Æ¬ø√†¬Æ¬¥√†¬Ø¬ç
    * √†¬π¬Ñ√†¬∏¬ó√†¬∏¬¢
    * T√É¬ºrk√É¬ße
    * √ê¬£√ê¬∫√ë¬Ä√ê¬∞√ë¬ó√ê¬Ω√ë¬Å√ë¬å√ê¬∫√ê¬∞
    * √ò¬ß√ò¬±√ò¬Ø√ô¬à
    * Ti√°¬∫¬øng_Vi√°¬ª¬át
    * √§¬∏¬≠√¶¬ñ¬á
Edit_links
    * This page was last edited on 31 July 2019, at 12:11¬†(UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. Wikipedia√Ç¬Æ is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
