The following text has been accessed from https://en.wikipedia.org/wiki/Oxidative_phosphorylation at Fri Aug 9 03:34:14 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_featured_article._Click_here_for_more_information.]
****** Oxidative phosphorylation ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
The electron_transport_chain in the cell is the site of oxidative
phosphorylation in prokaryotes. The NADH and succinate generated in the citric
acid_cycle are oxidized, releasing energy to power the ATP_synthase.
Oxidative phosphorylation (UK /ÉkËsÉªd.É.tÉªv/, US /ËÉËk.sÉªËdeÉª.tÉªv/
[1] or electron transport-linked phosphorylation) is the metabolic_pathway in
which cells use enzymes to oxidize nutrients, thereby releasing energy which is
used to produce adenosine_triphosphate (ATP). In most eukaryotes, this takes
place inside mitochondria. Almost all aerobic_organisms carry out oxidative
phosphorylation. This pathway is probably so pervasive because it is a highly
efficient way of releasing energy, compared to alternative fermentation
processes such as anaerobic glycolysis.
During oxidative phosphorylation, electrons are transferred from electron
donors to electron_acceptors such as oxygen, in redox_reactions. These redox
reactions release energy, which is used to form ATP. In eukaryotes, these redox
reactions are carried out by a series of protein_complexes within the inner
membrane of the cell's mitochondria, whereas, in prokaryotes, these proteins
are located in the cells' intermembrane space. These linked sets of proteins
are called electron_transport_chains. In eukaryotes, five main protein
complexes are involved, whereas in prokaryotes many different enzymes are
present, using a variety of electron donors and acceptors.
The energy released by electrons flowing through this electron transport chain
is used to transport protons across the inner_mitochondrial_membrane, in a
process called electron_transport. This generates potential_energy in the form
of a pH gradient and an electrical_potential across this membrane. This store
of energy is tapped when protons flow back across the membrane and down the
potential energy gradient, through a large enzyme called ATP_synthase; this
process is known as chemiosmosis. The ATP synthase uses the energy to transform
adenosine_diphosphate (ADP) into adenosine triphosphate, in a phosphorylation
reaction. The reaction is driven by the proton flow, which forces the rotation
of a part of the enzyme; the ATP synthase is a rotary mechanical motor.
Although oxidative phosphorylation is a vital part of metabolism, it produces
reactive_oxygen_species such as superoxide and hydrogen_peroxide, which lead to
propagation of free_radicals, damaging cells and contributing to disease and,
possibly, aging (senescence). The enzymes carrying out this metabolic pathway
are also the target of many drugs and poisons that inhibit their activities.
It is the terminal process of cellular_respiration in eukaryotes and accounts
for high ATP yield.
⁰
***** Contents *****
    * 1_Chemiosmosis
    * 2_Electron_and_proton_transfer_molecules
    * 3_Eukaryotic_electron_transport_chains
          o 3.1_NADH-coenzyme_Q_oxidoreductase_(complex_I)
          o 3.2_Succinate-Q_oxidoreductase_(complex_II)
          o 3.3_Electron_transfer_flavoprotein-Q_oxidoreductase
          o 3.4_Q-cytochrome_c_oxidoreductase_(complex_III)
          o 3.5_Cytochrome_c_oxidase_(complex_IV)
          o 3.6_Alternative_reductases_and_oxidases
          o 3.7_Organization_of_complexes
    * 4_Prokaryotic_electron_transport_chains
    * 5_ATP_synthase_(complex_V)
    * 6_Oxidative_Phosphorylation-_Energetics
    * 7_Reactive_oxygen_species
    * 8_Inhibitors
    * 9_History
    * 10_See_also
    * 11_Notes
    * 12_References
    * 13_Further_reading
          o 13.1_Introductory
          o 13.2_Advanced
          o 13.3_General_resources
          o 13.4_Structural_resources
***** Chemiosmosis[edit] *****
Further information: Chemiosmosis and Bioenergetics
Oxidative phosphorylation works by using energy-releasing chemical reactions to
drive energy-requiring reactions: The two sets of reactions are said to be
coupled. This means one cannot occur without the other. The flow of electrons
through the electron transport chain, from electron donors such as NADH to
electron_acceptors such as oxygen, is an exergonic process â it releases
energy, whereas the synthesis of ATP is an endergonic process, which requires
an input of energy. Both the electron transport chain and the ATP synthase are
embedded in a membrane, and energy is transferred from electron transport chain
to the ATP synthase by movements of protons across this membrane, in a process
called chemiosmosis.[2] In practice, this is like a simple electric_circuit,
with a current of protons being driven from the negative N-side of the membrane
to the positive P-side by the proton-pumping enzymes of the electron transport
chain. These enzymes are like a battery, as they perform work to drive current
through the circuit. The movement of protons creates an electrochemical
gradient across the membrane, which is often called the proton-motive force. It
has two components: a difference in proton concentration (a H+ gradient, ÎpH)
and a difference in electric_potential, with the N-side having a negative
charge.[3]
ATP synthase releases this stored energy by completing the circuit and allowing
protons to flow down the electrochemical gradient, back to the N-side of the
membrane.[4] The electrochemical gradient drives the rotation of part of the
enzyme's structure and couples this motion to the synthesis of ATP.
The two components of the proton-motive force are thermodynamically equivalent:
In mitochondria, the largest part of energy is provided by the potential; in
alkaliphile bacteria the electrical energy even has to compensate for a
counteracting inverse pH difference. Inversely, chloroplasts operate mainly on
ÎpH. However, they also require a small membrane potential for the kinetics of
ATP synthesis. In the case of the fusobacterium Propionigenium_modestum it
drives the counter-rotation of subunits a and c of the FO motor of ATP
synthase.[3]
The amount of energy released by oxidative phosphorylation is high, compared
with the amount produced by anaerobic_fermentation. Glycolysis produces only 2
ATP molecules, but somewhere between 30 and 36 ATPs are produced by the
oxidative phosphorylation of the 10 NADH and 2 succinate molecules made by
converting one molecule of glucose to carbon dioxide and water,[5] while each
cycle of beta_oxidation of a fatty_acid yields about 14 ATPs. These ATP yields
are theoretical maximum values; in practice, some protons leak across the
membrane, lowering the yield of ATP.[6]
***** Electron and proton transfer molecules[edit] *****
Further information: Coenzyme and Cofactor_(biochemistry)
Reduction of coenzyme_Q from its ubiquinone form (Q) to the reduced ubiquinol
form (QH2).
The electron transport chain carries both protons and electrons, passing
electrons from donors to acceptors, and transporting protons across a membrane.
These processes use both soluble and protein-bound transfer molecules. In
mitochondria, electrons are transferred within the intermembrane space by the
water-soluble electron transfer protein cytochrome_c.[7] This carries only
electrons, and these are transferred by the reduction and oxidation of an iron
atom that the protein holds within a heme group in its structure. Cytochrome c
is also found in some bacteria, where it is located within the periplasmic
space.[8]
Within the inner mitochondrial membrane, the lipid-soluble electron carrier
coenzyme_Q10 (Q) carries both electrons and protons by a redox cycle.[9] This
small benzoquinone molecule is very hydrophobic, so it diffuses freely within
the membrane. When Q accepts two electrons and two protons, it becomes reduced
to the ubiquinol form (QH2); when QH2 releases two electrons and two protons,
it becomes oxidized back to the ubiquinone (Q) form. As a result, if two
enzymes are arranged so that Q is reduced on one side of the membrane and QH2
oxidized on the other, ubiquinone will couple these reactions and shuttle
protons across the membrane.[10] Some bacterial electron transport chains use
different quinones, such as menaquinone, in addition to ubiquinone.[11]
Within proteins, electrons are transferred between flavin cofactors,[4][12]
ironâsulfur clusters, and cytochromes. There are several types of
ironâsulfur cluster. The simplest kind found in the electron transfer chain
consists of two iron atoms joined by two atoms of inorganic sulfur; these are
called [2Feâ2S] clusters. The second kind, called [4Feâ4S], contains a cube
of four iron atoms and four sulfur atoms. Each iron atom in these clusters is
coordinated by an additional amino_acid, usually by the sulfur atom of
cysteine. Metal ion cofactors undergo redox reactions without binding or
releasing protons, so in the electron transport chain they serve solely to
transport electrons through proteins. Electrons move quite long distances
through proteins by hopping along chains of these cofactors.[13] This occurs by
quantum_tunnelling, which is rapid over distances of less than 1.4Ã10â9 m.
[14]
***** Eukaryotic electron transport chains[edit] *****
Further information: Electron_transport_chain and Chemiosmosis
Many catabolic biochemical processes, such as glycolysis, the citric_acid
cycle, and beta_oxidation, produce the reduced coenzyme NADH. This coenzyme
contains electrons that have a high transfer_potential; in other words, they
will release a large amount of energy upon oxidation. However, the cell does
not release this energy all at once, as this would be an uncontrollable
reaction. Instead, the electrons are removed from NADH and passed to oxygen
through a series of enzymes that each release a small amount of the energy.
This set of enzymes, consisting of complexes I through IV, is called the
electron transport chain and is found in the inner membrane of the
mitochondrion. Succinate is also oxidized by the electron transport chain, but
feeds into the pathway at a different point.
In eukaryotes, the enzymes in this electron transport system use the energy
released from the oxidation of NADH to pump protons across the inner membrane
of the mitochondrion. This causes protons to build up in the intermembrane
space, and generates an electrochemical_gradient across the membrane. The
energy stored in this potential is then used by ATP synthase to produce ATP.
Oxidative phosphorylation in the eukaryotic mitochondrion is the best-
understood example of this process. The mitochondrion is present in almost all
eukaryotes, with the exception of anaerobic protozoa such as Trichomonas
vaginalis that instead reduce protons to hydrogen in a remnant mitochondrion
called a hydrogenosome.[15]
          Typical respiratory enzymes and substrates in eukaryotes.
Respiratory enzyme      Redox_pair                       Midpoint_potential 
                                                         (Volts)
NADH_dehydrogenase      NAD+ / NADH                      â0.32[16]
Succinate_dehydrogenase FMN or FAD / FMNH2 or FADH2      â0.20[16]
Cytochrome_bc1_complex  Coenzyme_Q10ox / Coenzyme Q10red +0.06[16]
Cytochrome bc1 complex  Cytochrome_box / Cytochrome bred +0.12[16]
Complex_IV              Cytochrome_cox / Cytochrome cred +0.22[16]
Complex IV              Cytochrome_aox / Cytochrome ared +0.29[16]
Complex IV              O2 / HOâ                    +0.82[16]
Conditions: pH = 7[16]
**** NADH-coenzyme Q oxidoreductase (complex I)[edit] ****
Complex I or NADH-Q_oxidoreductase. The abbreviations are discussed in the
text. In all diagrams of respiratory complexes in this article, the matrix is
at the bottom, with the intermembrane space above.
NADH-coenzyme_Q_oxidoreductase, also known as NADH dehydrogenase or complex I,
is the first protein in the electron transport chain.[17] Complex I is a giant
enzyme with the mammalian complex I having 46 subunits and a molecular mass of
about 1,000 kilodaltons (kDa).[18] The structure is known in detail only from a
bacterium;[19][20] in most organisms the complex resembles a boot with a large
"ball" poking out from the membrane into the mitochondrion.[21][22] The genes
that encode the individual proteins are contained in both the cell_nucleus and
the mitochondrial_genome, as is the case for many enzymes present in the
mitochondrion.
The reaction that is catalyzed by this enzyme is the two electron oxidation of
NADH by coenzyme_Q10 or ubiquinone (represented as Q in the equation below), a
lipid-soluble quinone that is found in the mitochondrion membrane:
          NADH + Q + 5   H  matrix   +   &#x27F6;  NAD  +   +  QH  2
      + 4   H  intermembrane   +      {\displaystyle {\ce {NADH + Q +    
      5H+_{matrix}-> NAD+ + QH2 + 4H+_{intermembrane}}}}  [                (1)
      {\displaystyle {\ce {NADH + Q + 5H+_{matrix}-> NAD+ + QH2 +
      4H+_{intermembrane}}}}]
The start of the reaction, and indeed of the entire electron chain, is the
binding of a NADH molecule to complex I and the donation of two electrons. The
electrons enter complex I via a prosthetic_group attached to the complex,
flavin_mononucleotide (FMN). The addition of electrons to FMN converts it to
its reduced form, FMNH2. The electrons are then transferred through a series of
ironâsulfur_clusters: the second kind of prosthetic group present in the
complex.[19] There are both [2Feâ2S] and [4Feâ4S] ironâsulfur clusters in
complex I.
As the electrons pass through this complex, four protons are pumped from the
matrix into the intermembrane space. Exactly how this occurs is unclear, but it
seems to involve conformational_changes in complex I that cause the protein to
bind protons on the N-side of the membrane and release them on the P-side of
the membrane.[23] Finally, the electrons are transferred from the chain of
ironâsulfur clusters to a ubiquinone molecule in the membrane.[17] Reduction
of ubiquinone also contributes to the generation of a proton gradient, as two
protons are taken up from the matrix as it is reduced to ubiquinol (QH2).
**** Succinate-Q oxidoreductase (complex II)[edit] ****
Complex II: Succinate-Q_oxidoreductase.
Succinate-Q_oxidoreductase, also known as complex II or succinate
dehydrogenase, is a second entry point to the electron transport chain.[24] It
is unusual because it is the only enzyme that is part of both the citric acid
cycle and the electron transport chain. Complex II consists of four protein
subunits and contains a bound flavin_adenine_dinucleotide (FAD) cofactor,
ironâsulfur clusters, and a heme group that does not participate in electron
transfer to coenzyme Q, but is believed to be important in decreasing
production of reactive oxygen species.[25][26] It oxidizes succinate to
fumarate and reduces ubiquinone. As this reaction releases less energy than the
oxidation of NADH, complex II does not transport protons across the membrane
and does not contribute to the proton gradient.
           Succinate  + Q &#x27F6;  Fumarate  +  QH  2                
      {\displaystyle {\ce {{Succinate}+ Q -> {Fumarate}+ QH2}}}  [      (2)
      {\displaystyle {\ce {{Succinate}+ Q -> {Fumarate}+ QH2}}}]
In some eukaryotes, such as the parasitic_worm Ascaris_suum, an enzyme similar
to complex II, fumarate reductase (menaquinol:fumarate oxidoreductase, or QFR),
operates in reverse to oxidize ubiquinol and reduce fumarate. This allows the
worm to survive in the anaerobic environment of the large_intestine, carrying
out anaerobic oxidative phosphorylation with fumarate as the electron acceptor.
[27] Another unconventional function of complex II is seen in the malaria
parasite Plasmodium_falciparum. Here, the reversed action of complex II as an
oxidase is important in regenerating ubiquinol, which the parasite uses in an
unusual form of pyrimidine biosynthesis.[28]
**** Electron transfer flavoprotein-Q oxidoreductase[edit] ****
Electron_transfer_flavoprotein-ubiquinone_oxidoreductase (ETF-
Q oxidoreductase), also known as electron transferring-flavoprotein
dehydrogenase, is a third entry point to the electron transport chain. It is an
enzyme that accepts electrons from electron-transferring_flavoprotein in the
mitochondrial matrix, and uses these electrons to reduce ubiquinone.[29] This
enzyme contains a flavin and a [4Feâ4S] cluster, but, unlike the other
respiratory complexes, it attaches to the surface of the membrane and does not
cross the lipid bilayer.[30]
           ETF  red        + Q &#x27F6;  ETF  ox        +  QH  2      
      {\displaystyle {\ce {ETF_{red}{}+ Q -> ETF_{ox}{}+ QH2}}}  [      (3)
      {\displaystyle {\ce {ETF_{red}{}+ Q -> ETF_{ox}{}+ QH2}}}]
In mammals, this metabolic pathway is important in beta_oxidation of fatty
acids and catabolism of amino_acids and choline, as it accepts electrons from
multiple acetyl-CoA dehydrogenases.[31][32] In plants, ETF-Q oxidoreductase is
also important in the metabolic responses that allow survival in extended
periods of darkness.[33]
**** Q-cytochrome c oxidoreductase (complex III)[edit] ****
The two electron transfer steps in complex III: Q-cytochrome_c_oxidoreductase.
After each step, Q (in the upper part of the figure) leaves the enzyme.
Q-cytochrome_c_oxidoreductase is also known as cytochrome c reductase,
cytochrome bc1 complex, or simply complex III.[34][35] In mammals, this enzyme
is a dimer, with each subunit complex containing 11 protein subunits, an [2Fe-
2S] ironâsulfur cluster and three cytochromes: one cytochrome c1 and two b
cytochromes.[36] A cytochrome is a kind of electron-transferring protein that
contains at least one heme group. The iron atoms inside complex III's heme
groups alternate between a reduced ferrous (+2) and oxidized ferric (+3) state
as the electrons are transferred through the protein.
The reaction catalyzed by complex III is the oxidation of one molecule of
ubiquinol and the reduction of two molecules of cytochrome_c, a heme protein
loosely associated with the mitochondrion. Unlike coenzyme Q, which carries two
electrons, cytochrome c carries only one electron.
           QH  2        + 2  Cyt   c  ox        + 2   H  matrix   +
      &#x27F6; Q   + 2  Cyt   c  red        + 4   H  intermembrane
      +      {\displaystyle {\ce {QH2{}+ 2 Cyt\, c_{ox}{}+ 2H+_         (4)
      {matrix}-> Q{}+ 2 Cyt\, c_{red}{}+ 4H+_{intermembrane}}}}  [      
      {\displaystyle {\ce {QH2{}+ 2 Cyt\, c_{ox}{}+ 2H+_{matrix}-> Q
      {}+ 2 Cyt\, c_{red}{}+ 4H+_{intermembrane}}}}]
As only one of the electrons can be transferred from the QH2 donor to a
cytochrome c acceptor at a time, the reaction mechanism of complex III is more
elaborate than those of the other respiratory complexes, and occurs in two
steps called the Q_cycle.[37] In the first step, the enzyme binds three
substrates, first, QH2, which is then oxidized, with one electron being passed
to the second substrate, cytochrome c. The two protons released from QH2 pass
into the intermembrane space. The third substrate is Q, which accepts the
second electron from the QH2 and is reduced to Q.â, which is the
ubisemiquinone free_radical. The first two substrates are released, but this
ubisemiquinone intermediate remains bound. In the second step, a second
molecule of QH2 is bound and again passes its first electron to a cytochrome c
acceptor. The second electron is passed to the bound ubisemiquinone, reducing
it to QH2 as it gains two protons from the mitochondrial matrix. This QH2 is
then released from the enzyme.[38]
As coenzyme Q is reduced to ubiquinol on the inner side of the membrane and
oxidized to ubiquinone on the other, a net transfer of protons across the
membrane occurs, adding to the proton gradient.[4] The rather complex two-step
mechanism by which this occurs is important, as it increases the efficiency of
proton transfer. If, instead of the Q cycle, one molecule of QH2 were used to
directly reduce two molecules of cytochrome c, the efficiency would be halved,
with only one proton transferred per cytochrome c reduced.[4]
**** Cytochrome c oxidase (complex IV)[edit] ****
Further information: cytochrome_c_oxidase
Complex IV: cytochrome_c_oxidase.
Cytochrome_c_oxidase, also known as complex IV, is the final protein complex in
the electron transport chain.[39] The mammalian enzyme has an extremely
complicated structure and contains 13 subunits, two heme groups, as well as
multiple metal ion cofactors â in all, three atoms of copper, one of
magnesium and one of zinc.[40]
This enzyme mediates the final reaction in the electron transport chain and
transfers electrons to oxygen, while pumping protons across the membrane.[41]
The final electron_acceptor oxygen, which is also called the terminal electron
acceptor, is reduced to water in this step. Both the direct pumping of protons
and the consumption of matrix protons in the reduction of oxygen contribute to
the proton gradient. The reaction catalyzed is the oxidation of cytochrome c
and the reduction of oxygen:
          4  Cyt   c  red        +  O  2        + 8   H  matrix   +
      &#x27F6; 4  Cyt   c  ox        + 2   H  2      O   + 4   H
      intermembrane   +      {\displaystyle {\ce {4Cyt\,c_{red}{}+O2     (5)
      {}+8H+_{matrix}->4Cyt\,c_{ox}{}+2H2O{}+4H+_{intermembrane}}}}      
      [{\displaystyle {\ce {4Cyt\,c_{red}{}+O2{}+8H+_{matrix}-
      >4Cyt\,c_{ox}{}+2H2O{}+4H+_{intermembrane}}}}]
**** Alternative reductases and oxidases[edit] ****
Many eukaryotic organisms have electron transport chains that differ from the
much-studied mammalian enzymes described above. For example, plants have
alternative NADH oxidases, which oxidize NADH in the cytosol rather than in the
mitochondrial matrix, and pass these electrons to the ubiquinone pool.[42]
These enzymes do not transport protons, and, therefore, reduce ubiquinone
without altering the electrochemical gradient across the inner membrane.[43]
Another example of a divergent electron transport chain is the alternative
oxidase, which is found in plants, as well as some fungi, protists, and
possibly some animals.[44][45] This enzyme transfers electrons directly from
ubiquinol to oxygen.[46]
The electron transport pathways produced by these alternative NADH and
ubiquinone oxidases have lower ATP yields than the full pathway. The advantages
produced by a shortened pathway are not entirely clear. However, the
alternative oxidase is produced in response to stresses such as cold, reactive
oxygen_species, and infection by pathogens, as well as other factors that
inhibit the full electron transport chain.[47][48] Alternative pathways might,
therefore, enhance an organisms' resistance to injury, by reducing oxidative
stress.[49]
**** Organization of complexes[edit] ****
The original model for how the respiratory chain complexes are organized was
that they diffuse freely and independently in the mitochondrial membrane.[50]
However, recent data suggest that the complexes might form higher-order
structures called supercomplexes or "respirasomes".[51] In this model, the
various complexes exist as organized sets of interacting enzymes.[52] These
associations might allow channeling of substrates between the various enzyme
complexes, increasing the rate and efficiency of electron transfer.[53] Within
such mammalian supercomplexes, some components would be present in higher
amounts than others, with some data suggesting a ratio between complexes I/II/
III/IV and the ATP synthase of approximately 1:1:3:7:4.[54] However, the debate
over this supercomplex hypothesis is not completely resolved, as some data do
not appear to fit with this model.[18][55]
***** Prokaryotic electron transport chains[edit] *****
Further information: Microbial_metabolism
In contrast to the general similarity in structure and function of the electron
transport chains in eukaryotes, bacteria and archaea possess a large variety of
electron-transfer enzymes. These use an equally wide set of chemicals as
substrates.[56] In common with eukaryotes, prokaryotic electron transport uses
the energy released from the oxidation of a substrate to pump ions across a
membrane and generate an electrochemical gradient. In the bacteria, oxidative
phosphorylation in Escherichia_coli is understood in most detail, while
archaeal systems are at present poorly understood.[57]
The main difference between eukaryotic and prokaryotic oxidative
phosphorylation is that bacteria and archaea use many different substances to
donate or accept electrons. This allows prokaryotes to grow under a wide
variety of environmental conditions.[58] In E. coli, for example, oxidative
phosphorylation can be driven by a large number of pairs of reducing agents and
oxidizing agents, which are listed below. The midpoint_potential of a chemical
measures how much energy is released when it is oxidized or reduced, with
reducing agents having negative potentials and oxidizing agents positive
potentials.
              Respiratory enzymes and substrates in E. coli.[59]
Respiratory enzyme           Redox_pair                    Midpoint_potential 
                                                           (Volts)
Formate_dehydrogenase        Bicarbonate / Formate                 â0.43
Hydrogenase                  Proton / Hydrogen                     â0.42
NADH_dehydrogenase           NAD+ / NADH                           â0.32
Glycerol-3-phosphate         DHAP / Gly-3-P                        â0.19
dehydrogenase
Pyruvate_oxidase             Acetate + Carbon_dioxide /                  ?
                             Pyruvate
Lactate_dehydrogenase        Pyruvate / Lactate                    â0.19
D-amino_acid_dehydrogenase   2-oxoacid + ammonia / D-amino               ?
                             acid
Glucose_dehydrogenase        Gluconate / Glucose                   â0.14
Succinate_dehydrogenase      Fumarate / Succinate                     +0.03
Ubiquinol_oxidase            Oxygen / Water                           +0.82
Nitrate_reductase            Nitrate / Nitrite                        +0.42
Nitrite_reductase            Nitrite / Ammonia                        +0.36
Dimethyl_sulfoxide_reductase DMSO / DMS                               +0.16
Trimethylamine_N-oxide       TMAO / TMA                               +0.13
reductase
Fumarate_reductase           Fumarate / Succinate                     +0.03
As shown above, E. coli can grow with reducing agents such as formate,
hydrogen, or lactate as electron donors, and nitrate, DMSO, or oxygen as
acceptors.[58] The larger the difference in midpoint potential between an
oxidizing and reducing agent, the more energy is released when they react. Out
of these compounds, the succinate/fumarate pair is unusual, as its midpoint
potential is close to zero. Succinate can therefore be oxidized to fumarate if
a strong oxidizing agent such as oxygen is available, or fumarate can be
reduced to succinate using a strong reducing agent such as formate. These
alternative reactions are catalyzed by succinate_dehydrogenase and fumarate
reductase, respectively.[60]
Some prokaryotes use redox pairs that have only a small difference in midpoint
potential. For example, nitrifying bacteria such as Nitrobacter oxidize nitrite
to nitrate, donating the electrons to oxygen. The small amount of energy
released in this reaction is enough to pump protons and generate ATP, but not
enough to produce NADH or NADPH directly for use in anabolism.[61] This problem
is solved by using a nitrite_oxidoreductase to produce enough proton-motive
force to run part of the electron transport chain in reverse, causing complex I
to generate NADH.[62][63]
Prokaryotes control their use of these electron donors and acceptors by varying
which enzymes are produced, in response to environmental conditions.[64] This
flexibility is possible because different oxidases and reductases use the same
ubiquinone pool. This allows many combinations of enzymes to function together,
linked by the common ubiquinol intermediate.[59] These respiratory chains
therefore have a modular_design, with easily interchangeable sets of enzyme
systems.
In addition to this metabolic diversity, prokaryotes also possess a range of
isozymes â different enzymes that catalyze the same reaction. For example, in
E. coli, there are two different types of ubiquinol oxidase using oxygen as an
electron acceptor. Under highly aerobic conditions, the cell uses an oxidase
with a low affinity for oxygen that can transport two protons per electron.
However, if levels of oxygen fall, they switch to an oxidase that transfers
only one proton per electron, but has a high affinity for oxygen.[65]
***** ATP synthase (complex V)[edit] *****
Further information: ATP_synthase
ATP synthase, also called complex V, is the final enzyme in the oxidative
phosphorylation pathway. This enzyme is found in all forms of life and
functions in the same way in both prokaryotes and eukaryotes.[66] The enzyme
uses the energy stored in a proton gradient across a membrane to drive the
synthesis of ATP from ADP and phosphate (Pi). Estimates of the number of
protons required to synthesize one ATP have ranged from three to four,[67][68]
with some suggesting cells can vary this ratio, to suit different conditions.
[69]
          ADP + P i + 4   H  intermembrane   +         &#x21BD;
      &#x2212;         &#x2212;     &#x21C0;       ATP +  H  2      O
      + 4   H  matrix   +      {\displaystyle {\ce {ADP + P_i + 4H+_     (6)
      {intermembrane}<=> ATP + H2O + 4H+_{matrix}}}}  [{\displaystyle    
      {\ce {ADP + P_i + 4H+_{intermembrane}<=> ATP + H2O + 4H+_
      {matrix}}}}]
This phosphorylation reaction is an equilibrium, which can be shifted by
altering the proton-motive force. In the absence of a proton-motive force, the
ATP synthase reaction will run from right to left, hydrolyzing ATP and pumping
protons out of the matrix across the membrane. However, when the proton-motive
force is high, the reaction is forced to run in the opposite direction; it
proceeds from left to right, allowing protons to flow down their concentration
gradient and turning ADP into ATP.[66] Indeed, in the closely related vacuolar
type_H+-ATPases, the hydrolysis reaction is used to acidify cellular
compartments, by pumping protons and hydrolysing ATP.[70]
ATP synthase is a massive protein complex with a mushroom-like shape. The
mammalian enzyme complex contains 16 subunits and has a mass of approximately
600 kilodaltons.[71] The portion embedded within the membrane is called FO and
contains a ring of c subunits and the proton channel. The stalk and the ball-
shaped headpiece is called F1 and is the site of ATP synthesis. The ball-shaped
complex at the end of the F1 portion contains six proteins of two different
kinds (three Î± subunits and three Î² subunits), whereas the "stalk" consists
of one protein: the Î³ subunit, with the tip of the stalk extending into the
ball of Î± and Î² subunits.[72] Both the Î± and Î² subunits bind nucleotides,
but only the Î² subunits catalyze the ATP synthesis reaction. Reaching along
the side of the F1 portion and back into the membrane is a long rod-like
subunit that anchors the Î± and Î² subunits into the base of the enzyme.
As protons cross the membrane through the channel in the base of ATP synthase,
the FO proton-driven motor rotates.[73] Rotation might be caused by changes in
the ionization of amino acids in the ring of c subunits causing electrostatic
interactions that propel the ring of c subunits past the proton channel.[74]
This rotating ring in turn drives the rotation of the central axle (the Î³
subunit stalk) within the Î± and Î² subunits. The Î± and Î² subunits are
prevented from rotating themselves by the side-arm, which acts as a stator.
This movement of the tip of the Î³ subunit within the ball of Î± and Î²
subunits provides the energy for the active sites in the Î² subunits to undergo
a cycle of movements that produces and then releases ATP.[75]
Mechanism of ATP_synthase. ATP is shown in red, ADP and phosphate in pink and
the rotating Î³ subunit in black.
This ATP synthesis reaction is called the binding change mechanism and involves
the active site of a Î² subunit cycling between three states.[76] In the "open"
state, ADP and phosphate enter the active site (shown in brown in the diagram).
The protein then closes up around the molecules and binds them loosely â the
"loose" state (shown in red). The enzyme then changes shape again and forces
these molecules together, with the active site in the resulting "tight" state
(shown in pink) binding the newly produced ATP molecule with very high
affinity. Finally, the active site cycles back to the open state, releasing ATP
and binding more ADP and phosphate, ready for the next cycle.
In some bacteria and archaea, ATP synthesis is driven by the movement of sodium
ions through the cell membrane, rather than the movement of protons.[77][78]
Archaea such as Methanococcus also contain the A1Ao synthase, a form of the
enzyme that contains additional proteins with little similarity in sequence to
other bacterial and eukaryotic ATP synthase subunits. It is possible that, in
some species, the A1Ao form of the enzyme is a specialized sodium-driven ATP
synthase,[79] but this might not be true in all cases.[78]
***** Oxidative Phosphorylation- Energetics[edit] *****
The transport of electrons from redox pair NAD+/ NADH to the final redox pair
1/2 O2/ H2O can be summarized as
1/2 O2+ NADH + H+ â H2O + NAD+
The potential difference between this two redox pair is 1.14 Volt which is
equivalent to 52 Cal./ Mol.
When one NADH is oxidized through ETC, three ATP are produced which is equal to
21.9 Cal.(7.3 X 3).
The conservation of the energy can be calculated by following formula
Efficiency = (21.9 X 100) / 52 = 42%
So we can conclude that when NADH is oxidized, about 42% of energy is conserved
in the form of three ATPs and the remaining (58%) energy is lost as heat
(entropy).
***** Reactive oxygen species[edit] *****
Further information: Oxidative_stress and Antioxidant
Molecular oxygen is an ideal terminal electron_acceptor because it is a strong
oxidizing agent. The reduction of oxygen does involve potentially harmful
intermediates.[80] Although the transfer of four electrons and four protons
reduces oxygen to water, which is harmless, transfer of one or two electrons
produces superoxide or peroxide anions, which are dangerously reactive.
           O  2        &#x2192;    e  &#x2212;          O  2
      &#x2219; &#x005F;    Superoxide     &#x2192;    e  &#x2212;
      O  2   2   &#x2212;    Peroxide      {\displaystyle {\ce {O2->[
      {\ce {e^{-}}}]{\underset {Superoxide}{O2^{\underline {\bullet      (7)
      }}}}->[{\ce {e^{-}}}]{\underset {Peroxide}{O2^{2-}}}}}}  [         
      {\displaystyle {\ce {O2->[{\ce {e^{-}}}]{\underset {Superoxide}
      {O2^{\underline {\bullet }}}}->[{\ce {e^{-}}}]{\underset
      {Peroxide}{O2^{2-}}}}}}]
These reactive_oxygen_species and their reaction products, such as the hydroxyl
radical, are very harmful to cells, as they oxidize proteins and cause
mutations in DNA. This cellular damage might contribute to disease and is
proposed as one cause of aging.[81][82]
The cytochrome c oxidase complex is highly efficient at reducing oxygen to
water, and it releases very few partly reduced intermediates; however small
amounts of superoxide anion and peroxide are produced by the electron transport
chain.[83] Particularly important is the reduction of coenzyme_Q in complex
III, as a highly reactive ubisemiquinone free radical is formed as an
intermediate in the Q cycle. This unstable species can lead to electron
"leakage" when electrons transfer directly to oxygen, forming superoxide.[84]
As the production of reactive oxygen species by these proton-pumping complexes
is greatest at high membrane potentials, it has been proposed that mitochondria
regulate their activity to maintain the membrane potential within a narrow
range that balances ATP production against oxidant generation.[85] For
instance, oxidants can activate uncoupling_proteins that reduce membrane
potential.[86]
To counteract these reactive oxygen species, cells contain numerous antioxidant
systems, including antioxidant vitamins such as vitamin_C and vitamin_E, and
antioxidant enzymes such as superoxide_dismutase, catalase, and peroxidases,
[80] which detoxify the reactive species, limiting damage to the cell.
***** Inhibitors[edit] *****
There are several well-known drugs and toxins that inhibit oxidative
phosphorylation. Although any one of these toxins inhibits only one enzyme in
the electron transport chain, inhibition of any step in this process will halt
the rest of the process. For example, if oligomycin inhibits ATP synthase,
protons cannot pass back into the mitochondrion.[87] As a result, the proton
pumps are unable to operate, as the gradient becomes too strong for them to
overcome. NADH is then no longer oxidized and the citric acid cycle ceases to
operate because the concentration of NAD+ falls below the concentration that
these enzymes can use.
Compounds        Use        Site of  Effect on oxidative phosphorylation
                            action
Cyanide                              Inhibit the electron transport chain by binding more strongly than oxygen to
Carbon monoxide   Poisons  Complex  the FeâCu center in cytochrome c oxidase, preventing the reduction of
Azide                       IV       oxygen.[88]
Hydrogen_sulfide
Oligomycin       Antibiotic Complex  Inhibits ATP synthase by blocking the flow of protons through the Fo
                            V        subunit.[87]
CCCP              Poisons,  Inner    Ionophores that disrupt the proton gradient by carrying protons across a
2,4-              weight-   membrane membrane. This ionophore uncouples proton pumping from ATP synthesis because
Dinitrophenol    loss[N_1]           it carries protons across the inner mitochondrial membrane.[89]
Rotenone         Pesticide  Complex  Prevents the transfer of electrons from complex I to ubiquinone by blocking
                            I        the ubiquinone-binding site.[90]
Malonate and       Poisons  Complex  Competitive inhibitors of succinate dehydrogenase (complex II).[91]
oxaloacetate                II
Antimycin_A      Piscicide  Complex  Binds to the Qi site of cytochrome_c_reductase, thereby inhibiting the
                            III      oxidation of ubiquinol.
Not all inhibitors of oxidative phosphorylation are toxins. In brown_adipose
tissue, regulated proton channels called uncoupling_proteins can uncouple
respiration from ATP synthesis.[92] This rapid respiration produces heat, and
is particularly important as a way of maintaining body_temperature for
hibernating animals, although these proteins may also have a more general
function in cells' responses to stress.[93]
***** History[edit] *****
Further information: History_of_biochemistry and History_of_molecular_biology
The field of oxidative phosphorylation began with the report in 1906 by Arthur
Harden of a vital role for phosphate in cellular fermentation, but initially
only sugar_phosphates were known to be involved.[94] However, in the early
1940s, the link between the oxidation of sugars and the generation of ATP was
firmly established by Herman_Kalckar,[95] confirming the central role of ATP in
energy transfer that had been proposed by Fritz_Albert_Lipmann in 1941.[96]
Later, in 1949, Morris Friedkin and Albert_L._Lehninger proved that the
coenzyme NADH linked metabolic pathways such as the citric acid cycle and the
synthesis of ATP.[97] The term oxidative phosphorylation was coined by
Volodymyr_Belitser [uk] in 1939.[98][99]
For another twenty years, the mechanism by which ATP is generated remained
mysterious, with scientists searching for an elusive "high-energy intermediate"
that would link oxidation and phosphorylation reactions.[100] This puzzle was
solved by Peter_D._Mitchell with the publication of the chemiosmotic_theory in
1961.[101] At first, this proposal was highly controversial, but it was slowly
accepted and Mitchell was awarded a Nobel_prize in 1978.[102][103] Subsequent
research concentrated on purifying and characterizing the enzymes involved,
with major contributions being made by David_E._Green on the complexes of the
electron-transport chain, as well as Efraim_Racker on the ATP synthase.[104] A
critical step towards solving the mechanism of the ATP synthase was provided by
Paul_D._Boyer, by his development in 1973 of the "binding change" mechanism,
followed by his radical proposal of rotational catalysis in 1982.[76][105] More
recent work has included structural_studies on the enzymes involved in
oxidative phosphorylation by John_E._Walker, with Walker and Boyer being
awarded a Nobel Prize in 1997.[106]
***** See also[edit] *****
    * Respirometry
    * TIM/TOM_Complex
***** Notes[edit] *****
   1. ^ DNP was extensively used as an anti-obesity_medication in the 1930s but
      was ultimately discontinued due to its dangerous side effects. However,
      illicit use of the drug for this purpose continues today. See 2,4-
      Dinitrophenol#Dieting_aid for more information.
***** References[edit] *****
   1. ^"oxidative_Meaning_in_the_Cambridge_English_Dictionary".
      dictionary.cambridge.org. Archived from the original on 24 January 2018.
      Retrieved 28 April 2018.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Mitchell P, Moyle J (1967). "Chemiosmotic hypothesis of oxidative
      phosphorylation". Nature. 213 (5072): 137â9. Bibcode:
      1967Natur.213..137M. doi:10.1038/213137a0. PMID 4291593.
   4. ^ a bDimroth P, Kaim G, Matthey U (1 January 2000). "Crucial_role_of_the
      membrane_potential_for_ATP_synthesis_by_F(1)F(o)_ATP_synthases". J. Exp.
      Biol. 203 (Pt 1): 51â9. PMID 10600673. Archived from the original on 30
      September 2007.
   5. ^ a b c dSchultz BE, Chan_SI (2001). "Structures and proton-pumping
      strategies of mitochondrial respiratory enzymes". Annu Rev Biophys Biomol
      Struct. 30: 23â65. doi:10.1146/annurev.biophys.30.1.23. PMID 11340051.
   6. ^Rich PR (2003). "The molecular machinery of Keilin's respiratory chain".
      Biochem. Soc. Trans. 31 (Pt 6): 1095â105. doi:10.1042/bst0311095.
      PMID 14641005.
   7. ^Porter RK, Brand MD (1995). "Mitochondrial_proton_conductance_and_H+/
      O_ratio_are_independent_of_electron_transport_rate_in_isolated
      hepatocytes". Biochem. J. 310 (Pt 2): 379â82. doi:10.1042/bj3100379.
      PMC 1135905. PMID 7654171.
   8. ^Mathews FS (1985). "The structure, function and evolution of
      cytochromes". Prog. Biophys. Mol. Biol. 45 (1): 1â56. doi:10.1016/0079-
      6107(85)90004-5. PMID 3881803.
   9. ^Wood PM (1983). "Why do c-type cytochromes exist?". FEBS Lett. 164 (2):
      223â6. doi:10.1016/0014-5793(83)80289-0. PMID 6317447.
  10. ^Crane FL (1 December 2001). "Biochemical functions of coenzyme Q10". J
      Am Coll Nutr. 20 (6): 591â8. doi:10.1080/07315724.2001.10719063.
      PMID 11771674.
  11. ^Mitchell P (1979). "Keilin's respiratory chain concept and its
      chemiosmotic consequences". Science. 206 (4423): 1148â59. Bibcode:
      1979Sci...206.1148M. doi:10.1126/science.388618. PMID 388618.
  12. ^SÃ¸balle B, Poole RK (1999). "Microbial_ubiquinones:_multiple_roles_in
      respiration,_gene_regulation_and_oxidative_stress_management" (PDF).
      Microbiology. 145 (8): 1817â30. doi:10.1099/13500872-145-8-1817.
      PMID 10463148. Archived (PDF) from the original on 2008-05-29.
  13. ^Johnson DC, Dean DR, Smith AD, Johnson MK (2005). "Structure, function,
      and formation of biological iron-sulfur clusters". Annu. Rev. Biochem.
      74: 247â81. doi:10.1146/annurev.biochem.74.082803.133518.
      PMID 15952888.
  14. ^Page CC, Moser CC, Chen X, Dutton PL (1999). "Natural engineering
      principles of electron tunnelling in biological oxidation-reduction".
      Nature. 402 (6757): 47â52. Bibcode:1999Natur.402...47P. doi:10.1038/
      46972. PMID 10573417.
  15. ^Leys D, Scrutton NS (2004). "Electrical circuitry in biology: emerging
      principles from protein structure". Curr. Opin. Struct. Biol. 14 (6):
      642â7. doi:10.1016/j.sbi.2004.10.002. PMID 15582386.
  16. ^Boxma B, de Graaf RM, van der Staay GW, van Alen TA, Ricard G, GabaldÃ³n
      T, van Hoek AH, Moon-van der Staay SY, Koopman WJ, van Hellemond JJ,
      Tielens AG, Friedrich T, Veenhuis M, Huynen MA, Hackstein JH (2005). "An
      anaerobic_mitochondrion_that_produces_hydrogen" (PDF). Nature. 434
      (7029): 74â9. Bibcode:2005Natur.434...74B. doi:10.1038/nature03343.
      PMID 15744302.
  17. ^ a b c d e f g h Medical CHEMISTRY Compendium. By Anders Overgaard
      Pedersen and Henning Nielsen. Aarhus University. 2008
  18. ^ a bHirst J (2005). "Energy transduction by respiratory complex I--an
      evaluation of current knowledge". Biochem. Soc. Trans. 33 (Pt 3):
      525â9. doi:10.1042/BST0330525. PMID 15916556.
  19. ^ a bLenaz G, Fato R, Genova ML, Bergamini C, Bianchi C, Biondi A (2006).
      "Mitochondrial Complex I: structural and functional aspects". Biochim.
      Biophys. Acta. 1757 (9â10): 1406â20. doi:10.1016/
      j.bbabio.2006.05.007. PMID 16828051.
  20. ^ a bSazanov LA, Hinchliffe P (2006). "Structure of the hydrophilic
      domain of respiratory complex I from Thermus thermophilus". Science. 311
      (5766): 1430â6. Bibcode:2006Sci...311.1430S. doi:10.1126/
      science.1123809. PMID 16469879.
  21. ^Efremov, RG; Baradaran, R; Sazanov, LA (2010). "The architecture of
      respiratory complex I". Nature. 465 (7297): 441â5. doi:10.1038/
      nature09066. PMID 20505720.
  22. ^Baranova EA, Holt PJ, Sazanov LA (2007). "Projection structure of the
      membrane domain of Escherichia coli respiratory complex I at 8 A
      resolution". J. Mol. Biol. 366 (1): 140â54. doi:10.1016/
      j.jmb.2006.11.026. PMID 17157874.
  23. ^Friedrich T, BÃ¶ttcher B (2004). "The gross structure of the respiratory
      complex I: a Lego System". Biochim. Biophys. Acta. 1608 (1): 1â9. doi:
      10.1016/j.bbabio.2003.10.002. PMID 14741580.
  24. ^Hirst J (January 2010). "Towards the molecular mechanism of respiratory
      complex I". Biochem. J. 425 (2): 327â39. doi:10.1042/BJ20091382.
      PMID 20025615.
  25. ^Cecchini G (2003). "Function and structure of complex II of the
      respiratory chain". Annu. Rev. Biochem. 72: 77â109. doi:10.1146/
      annurev.biochem.72.121801.161700. PMID 14527321.
  26. ^Yankovskaya V, Horsefield R, TÃ¶rnroth S, Luna-Chavez C, Miyoshi H,
      LÃ©ger C, Byrne B, Cecchini G, Iwata S, et al. (2003). "Architecture of
      succinate dehydrogenase and reactive oxygen species generation". Science.
      299 (5607): 700â4. Bibcode:2003Sci...299..700Y. doi:10.1126/
      science.1079605. PMID 12560550.
  27. ^Horsefield R, Iwata S, Byrne B (2004). "Complex II from a structural
      perspective". Curr. Protein Pept. Sci. 5 (2): 107â18. doi:10.2174/
      1389203043486847. PMID 15078221.
  28. ^Kita K, Hirawake H, Miyadera H, Amino H, Takeo S (2002). "Role of
      complex II in anaerobic respiration of the parasite mitochondria from
      Ascaris suum and Plasmodium falciparum". Biochim. Biophys. Acta. 1553
      (1â2): 123â39. doi:10.1016/S0005-2728(01)00237-7. PMID 11803022.
  29. ^Painter HJ, Morrisey JM, Mather MW, Vaidya AB (2007). "Specific role of
      mitochondrial electron transport in blood-stage Plasmodium falciparum".
      Nature. 446 (7131): 88â91. Bibcode:2007Natur.446...88P. doi:10.1038/
      nature05572. PMID 17330044.
  30. ^Ramsay RR, Steenkamp DJ, Husain M (1987). "Reactions_of_electron-
      transfer_flavoprotein_and_electron-transfer_flavoprotein:_ubiquinone
      oxidoreductase". Biochem. J. 241 (3): 883â92. doi:10.1042/bj2410883.
      PMC 1147643. PMID 3593226.
  31. ^Zhang J, Frerman FE, Kim JJ (2006). "Structure_of_electron_transfer
      flavoprotein-ubiquinone_oxidoreductase_and_electron_transfer_to_the
      mitochondrial_ubiquinone_pool". Proc. Natl. Acad. Sci. U.S.A. 103 (44):
      16212â7. Bibcode:2006PNAS..10316212Z. doi:10.1073/pnas.0604567103.
      PMC 1637562. PMID 17050691.
  32. ^Ikeda Y, Dabrowski C, Tanaka K (25 January 1983). "Separation_and
      properties_of_five_distinct_acyl-CoA_dehydrogenases_from_rat_liver
      mitochondria._Identification_of_a_new_2-methyl_branched_chain_acyl-CoA
      dehydrogenase". J. Biol. Chem. 258 (2): 1066â76. PMID 6401712. Archived
      from the original on 29 September 2007.
  33. ^Ruzicka FJ, Beinert H (1977). "A_new_iron-sulfur_flavoprotein_of_the
      respiratory_chain._A_component_of_the_fatty_acid_beta_oxidation_pathway"
      (PDF). J. Biol. Chem. 252 (23): 8440â5. PMID 925004. Archived (PDF)
      from the original on 2007-09-27.
  34. ^Ishizaki K, Larson TR, Schauer N, Fernie AR, Graham IA, Leaver CJ
      (2005). "The_critical_role_of_Arabidopsis_electron-transfer_flavoprotein:
      ubiquinone_oxidoreductase_during_dark-induced_starvation". Plant Cell. 17
      (9): 2587â600. doi:10.1105/tpc.105.035162. PMC 1197437. PMID 16055629.
  35. ^Berry EA, Guergova-Kuras M, Huang LS, Crofts AR (2000). "Structure_and
      function_of_cytochrome_bc_complexes" (PDF). Annu. Rev. Biochem. 69:
      1005â75. doi:10.1146/annurev.biochem.69.1.1005. PMID 10966481. Archived
      (PDF) from the original on 2015-12-28.
  36. ^Crofts AR (2004). "The cytochrome bc1 complex: function in the context
      of structure". Annu. Rev. Physiol. 66: 689â733. doi:10.1146/
      annurev.physiol.66.032102.150251. PMID 14977419.
  37. ^Iwata S, Lee JW, Okada K, Lee JK, Iwata M, Rasmussen B, Link TA,
      Ramaswamy S, Jap BK (1998). "Complete structure of the 11-subunit bovine
      mitochondrial cytochrome bc1 complex". Science. 281 (5373): 64â71.
      Bibcode:1998Sci...281...64I. doi:10.1126/science.281.5373.64.
      PMID 9651245.
  38. ^Trumpower BL (1990). "The_protonmotive_Q_cycle._Energy_transduction_by
      coupling_of_proton_translocation_to_electron_transfer_by_the_cytochrome
      bc1_complex" (PDF). J. Biol. Chem. 265 (20): 11409â12. PMID 2164001.
      Archived (PDF) from the original on 2007-09-27.
  39. ^Hunte C, Palsdottir H, Trumpower BL (2003). "Protonmotive pathways and
      mechanisms in the cytochrome bc1 complex". FEBS Lett. 545 (1): 39â46.
      doi:10.1016/S0014-5793(03)00391-0. PMID 12788490.
  40. ^Calhoun MW, Thomas JW, Gennis RB (1994). "The cytochrome oxidase
      superfamily of redox-driven proton pumps". Trends Biochem. Sci. 19 (8):
      325â30. doi:10.1016/0968-0004(94)90071-X. PMID 7940677.
  41. ^Tsukihara T, Aoyama H, Yamashita E, Tomizaki T, Yamaguchi H, Shinzawa-
      Itoh K, Nakashima R, Yaono R, Yoshikawa S (1996). "The whole structure of
      the 13-subunit oxidized cytochrome c oxidase at 2.8 A". Science. 272
      (5265): 1136â44. Bibcode:1996Sci...272.1136T. doi:10.1126/
      science.272.5265.1136. PMID 8638158.
  42. ^Yoshikawa S, Muramoto K, Shinzawa-Itoh K, Aoyama H, Tsukihara T,
      Shimokata K, Katayama Y, Shimada H (2006). "Proton pumping mechanism of
      bovine heart cytochrome c oxidase". Biochim. Biophys. Acta. 1757
      (9â10): 1110â6. doi:10.1016/j.bbabio.2006.06.004. PMID 16904626.
  43. ^Rasmusson AG, Soole KL, Elthon TE (2004). "Alternative NAD(P)H
      dehydrogenases of plant mitochondria". Annu Rev Plant Biol. 55: 23â39.
      doi:10.1146/annurev.arplant.55.031903.141720. PMID 15725055.
  44. ^Menz RI, Day DA (1996). "Purification and characterization of a 43-kDa
      rotenone-insensitive NADH dehydrogenase from plant mitochondria". J.
      Biol. Chem. 271 (38): 23117â20. doi:10.1074/jbc.271.38.23117.
      PMID 8798503.
  45. ^McDonald A, Vanlerberghe G (2004). "Branched mitochondrial electron
      transport in the Animalia: presence of alternative oxidase in several
      animal phyla". IUBMB Life. 56 (6): 333â41. doi:10.1080/1521-
      6540400000876. PMID 15370881.
  46. ^Sluse FE, Jarmuszkiewicz W (1998). "Alternative oxidase in the branched
      mitochondrial respiratory network: an overview on structure, function,
      regulation, and role". Braz. J. Med. Biol. Res. 31 (6): 733â47. doi:
      10.1590/S0100-879X1998000600003. PMID 9698817.
  47. ^Moore AL, Siedow JN (1991). "The regulation and nature of the cyanide-
      resistant alternative oxidase of plant mitochondria". Biochim. Biophys.
      Acta. 1059 (2): 121â40. doi:10.1016/S0005-2728(05)80197-5.
      PMID 1883834.
  48. ^Vanlerberghe GC, McIntosh L (1997). "ALTERNATIVE OXIDASE: From Gene to
      Function". Annu. Rev. Plant Physiol. Plant Mol. Biol. 48: 703â734. doi:
      10.1146/annurev.arplant.48.1.703. PMID 15012279.
  49. ^Ito Y, Saisho D, Nakazono M, Tsutsumi N, Hirai A (1997). "Transcript
      levels of tandem-arranged alternative oxidase genes in rice are increased
      by low temperature". Gene. 203 (2): 121â9. doi:10.1016/S0378-1119
      (97)00502-7. PMID 9426242.
  50. ^Maxwell DP, Wang Y, McIntosh L (1999). "The_alternative_oxidase_lowers
      mitochondrial_reactive_oxygen_production_in_plant_cells". Proc. Natl.
      Acad. Sci. U.S.A. 96 (14): 8271â6. Bibcode:1999PNAS...96.8271M. doi:
      10.1073/pnas.96.14.8271. PMC 22224. PMID 10393984.
  51. ^Lenaz G (2001). "A critical appraisal of the mitochondrial coenzyme Q
      pool". FEBS Lett. 509 (2): 151â5. doi:10.1016/S0014-5793(01)03172-6.
      PMID 11741580.
  52. ^Heinemeyer J, Braun HP, Boekema EJ, Kouril R (2007). "A structural model
      of the cytochrome C reductase/oxidase supercomplex from yeast
      mitochondria". J. Biol. Chem. 282 (16): 12240â8. doi:10.1074/
      jbc.M610545200. PMID 17322303.
  53. ^SchÃ¤gger H, Pfeiffer K (2000). "Supercomplexes_in_the_respiratory
      chains_of_yeast_and_mammalian_mitochondria". EMBO J. 19 (8): 1777â83.
      doi:10.1093/emboj/19.8.1777. PMC 302020. PMID 10775262.
  54. ^SchÃ¤gger H (2002). "Respiratory chain supercomplexes of mitochondria
      and bacteria". Biochim. Biophys. Acta. 1555 (1â3): 154â9. doi:
      10.1016/S0005-2728(02)00271-2. PMID 12206908.
  55. ^SchÃ¤gger H, Pfeiffer K (2001). "The_ratio_of_oxidative_phosphorylation
      complexes_I-V_in_bovine_heart_mitochondria_and_the_composition_of
      respiratory_chain_supercomplexes". J. Biol. Chem. 276 (41): 37861â7.
      doi:10.1074/jbc.M106474200. PMID 11483615. Archived from the original on
      2007-09-29.
  56. ^Gupte S, Wu ES, Hoechli L, Hoechli M, Jacobson K, Sowers AE, Hackenbrock
      CR (1984). "Relationship_between_lateral_diffusion,_collision_frequency,
      and_electron_transfer_of_mitochondrial_inner_membrane_oxidation-reduction
      components". Proc. Natl. Acad. Sci. U.S.A. 81 (9): 2606â10. Bibcode:
      1984PNAS...81.2606G. doi:10.1073/pnas.81.9.2606. PMC 345118.
      PMID 6326133.
  57. ^Nealson KH (1999). "Post-Viking microbiology: new approaches, new data,
      new insights". Orig Life Evol Biosph. 29 (1): 73â93. doi:10.1023/A:
      1006515817767. PMID 11536899.
  58. ^SchÃ¤fer G, Engelhard M, MÃ¼ller V (1999). "Bioenergetics_of_the
      Archaea". Microbiol. Mol. Biol. Rev. 63 (3): 570â620. PMC 103747.
      PMID 10477309.
  59. ^ a bIngledew WJ, Poole RK (1984). "The_respiratory_chains_of_Escherichia
      coli". Microbiol. Rev. 48 (3): 222â71. PMC 373010. PMID 6387427.
  60. ^ a bUnden G, Bongaerts J (1997). "Alternative respiratory pathways of
      Escherichia coli: energetics and transcriptional regulation in response
      to electron acceptors". Biochim. Biophys. Acta. 1320 (3): 217â34. doi:
      10.1016/S0005-2728(97)00034-0. PMID 9230919.
  61. ^Cecchini G, SchrÃ¶der I, Gunsalus RP, Maklashina E (2002). "Succinate
      dehydrogenase and fumarate reductase from Escherichia coli". Biochim.
      Biophys. Acta. 1553 (1â2): 140â57. doi:10.1016/S0005-2728(01)00238-9.
      PMID 11803023.
  62. ^Freitag A, Bock E; Bock (1990). "Energy conservation in Nitrobacter".
      FEMS Microbiology Letters. 66 (1â3): 157â62. doi:10.1111/j.1574-
      6968.1990.tb03989.x.
  63. ^Starkenburg SR, Chain PS, Sayavedra-Soto LA, Hauser L, Land ML, Larimer
      FW, Malfatti SA, Klotz MG, Bottomley PJ, Arp DJ, Hickey WJ (2006).
      "Genome_sequence_of_the_chemolithoautotrophic_nitrite-oxidizing_bacterium
      Nitrobacter_winogradskyi_Nb-255". Appl. Environ. Microbiol. 72 (3):
      2050â63. doi:10.1128/AEM.72.3.2050-2063.2006. PMC 1393235.
      PMID 16517654.
  64. ^Yamanaka T, Fukumori Y (1988). "The nitrite oxidizing system of
      Nitrobacter winogradskyi". FEMS Microbiol. Rev. 54 (4): 259â70. doi:
      10.1111/j.1574-6968.1988.tb02746.x. PMID 2856189.
  65. ^Iuchi S, Lin EC (1993). "Adaptation of Escherichia coli to redox
      environments by gene expression". Mol. Microbiol. 9 (1): 9â15. doi:
      10.1111/j.1365-2958.1993.tb01664.x. PMID 8412675.
  66. ^Calhoun MW, Oden KL, Gennis RB, de Mattos MJ, Neijssel OM (1993).
      "Energetic_efficiency_of_Escherichia_coli:_effects_of_mutations_in
      components_of_the_aerobic_respiratory_chain" (PDF). J. Bacteriol. 175
      (10): 3020â5. doi:10.1128/jb.175.10.3020-3025.1993. PMC 204621.
      PMID 8491720. Archived (PDF) from the original on 2007-09-27.
  67. ^ a bBoyer PD (1997). "The ATP synthase--a splendid molecular machine".
      Annu. Rev. Biochem. 66: 717â49. doi:10.1146/annurev.biochem.66.1.717.
      PMID 9242922.
  68. ^Van Walraven HS, Strotmann H, Schwarz O, Rumberg B (1996). "The H+/ATP
      coupling ratio of the ATP synthase from thiol-modulated chloroplasts and
      two cyanobacterial strains is four". FEBS Lett. 379 (3): 309â13. doi:
      10.1016/0014-5793(95)01536-1. PMID 8603713.
  69. ^Yoshida M, Muneyuki E, Hisabori T (2001). "ATP synthase--a marvellous
      rotary engine of the cell". Nat. Rev. Mol. Cell Biol. 2 (9): 669â77.
      doi:10.1038/35089509. PMID 11533724.
  70. ^Schemidt RA, Qu J, Williams JR, Brusilow WS (1998). "Effects_of_carbon
      source_on_expression_of_F0_genes_and_on_the_stoichiometry_of_the_c
      subunit_in_the_F1F0_ATPase_of_Escherichia_coli". J. Bacteriol. 180 (12):
      3205â8. PMC 107823. PMID 9620972.
  71. ^Nelson N, Perzov N, Cohen A, Hagai K, Padler V, Nelson H (1 January
      2000). "The_cellular_biology_of_proton-motive_force_generation_by_V-
      ATPases". J. Exp. Biol. 203 (Pt 1): 89â95. PMID 10600677. Archived from
      the original on 30 September 2007.
  72. ^Rubinstein JL, Walker JE, Henderson R (2003). "Structure_of_the
      mitochondrial_ATP_synthase_by_electron_cryomicroscopy". EMBO J. 22 (23):
      6182â92. doi:10.1093/emboj/cdg608. PMC 291849. PMID 14633978.
  73. ^Leslie AG, Walker JE (2000). "Structural_model_of_F1-ATPase_and_the
      implications_for_rotary_catalysis". Philos. Trans. R. Soc. Lond. B Biol.
      Sci. 355 (1396): 465â71. doi:10.1098/rstb.2000.0588. PMC 1692760.
      PMID 10836500.
  74. ^Noji H, Yoshida M (2001). "The rotary machine in the cell, ATP
      synthase". J. Biol. Chem. 276 (3): 1665â8. doi:10.1074/jbc.R000021200.
      PMID 11080505.
  75. ^Capaldi RA, Aggeler R (2002). "Mechanism of the F(1)F(0)-type ATP
      synthase, a biological rotary motor". Trends Biochem. Sci. 27 (3):
      154â60. doi:10.1016/S0968-0004(01)02051-5. PMID 11893513.
  76. ^Dimroth P, von Ballmoos C, Meier T (2006). "Catalytic_and_mechanical
      cycles_in_F-ATP_synthases._Fourth_in_the_Cycles_Review_Series". EMBO Rep.
      7 (3): 276â82. doi:10.1038/sj.embor.7400646. PMC 1456893.
      PMID 16607397.
  77. ^ a bGresser MJ, Myers JA, Boyer PD (25 October 1982). "Catalytic_site
      cooperativity_of_beef_heart_mitochondrial_F1_adenosine_triphosphatase.
      Correlations_of_initial_velocity,_bound_intermediate,_and_oxygen_exchange
      measurements_with_an_alternating_three-site_model". J. Biol. Chem. 257
      (20): 12030â8. PMID 6214554. Archived from the original on 29 September
      2007.
  78. ^Dimroth P (1994). "Bacterial sodium ion-coupled energetics". Antonie van
      Leeuwenhoek. 65 (4): 381â95. doi:10.1007/BF00872221. PMID 7832594.
  79. ^ a bBecher B, MÃ¼ller V (1994). "Delta_mu_Na+_drives_the_synthesis_of
      ATP_via_an_delta_mu_Na(+)-translocating_F1F0-ATP_synthase_in_membrane
      vesicles_of_the_archaeon_Methanosarcina_mazei_GÃ¶1". J. Bacteriol. 176
      (9): 2543â50. doi:10.1128/jb.176.9.2543-2550.1994. PMC 205391.
      PMID 8169202.
  80. ^MÃ¼ller V (2004). "An exceptional variability in the motor of archael
      A1A0 ATPases: from multimeric to monomeric rotors comprising 6-13 ion
      binding sites". J. Bioenerg. Biomembr. 36 (1): 115â25. doi:10.1023/B:
      JOBB.0000019603.68282.04. PMID 15168615.
  81. ^ a bDavies KJ (1995). "Oxidative stress: the paradox of aerobic life".
      Biochem. Soc. Symp. 61: 1â31. doi:10.1042/bss0610001. PMID 8660387.
  82. ^Rattan SI (2006). "Theories_of_biological_aging:_genes,_proteins,_and
      free_radicals" (PDF). Free Radic. Res. 40 (12): 1230â8. doi:10.1080/
      10715760600911303. PMID 17090411. Archived from the_original (PDF) on
      2014-06-14. Retrieved 2017-10-27.
  83. ^Valko M, Leibfritz D, Moncol J, Cronin MT, Mazur M, Telser J (2007).
      "Free radicals and antioxidants in normal physiological functions and
      human disease". Int. J. Biochem. Cell Biol. 39 (1): 44â84. doi:10.1016/
      j.biocel.2006.07.001. PMID 16978905.
  84. ^Raha S, Robinson BH (2000). "Mitochondria, oxygen free radicals, disease
      and ageing". Trends Biochem. Sci. 25 (10): 502â8. doi:10.1016/S0968-
      0004(00)01674-1. PMID 11050436.
  85. ^Finkel T, Holbrook NJ (2000). "Oxidants, oxidative stress and the
      biology of ageing". Nature. 408 (6809): 239â47. doi:10.1038/35041687.
      PMID 11089981.
  86. ^Kadenbach B, Ramzan R, Wen L, Vogt S (March 2010). "New extension of the
      Mitchell Theory for oxidative phosphorylation in mitochondria of living
      organisms". Biochim. Biophys. Acta. 1800 (3): 205â12. doi:10.1016/
      j.bbagen.2009.04.019. PMID 19409964.
  87. ^Echtay KS, Roussel D, St-Pierre J, Jekabsons MB, Cadenas S, Stuart JA,
      Harper JA, Roebuck SJ, Morrison A, Pickering S, Clapham JC, Brand MD
      (January 2002). "Superoxide activates mitochondrial uncoupling proteins".
      Nature. 415 (6867): 96â9. Bibcode:2002Natur.415...96E. doi:10.1038/
      415096a. PMID 11780125.
  88. ^ a bJoshi S, Huang YG (1991). "ATP synthase complex from bovine heart
      mitochondria: the oligomycin sensitivity conferring protein is essential
      for dicyclohexyl carbodiimide-sensitive ATPase". Biochim. Biophys. Acta.
      1067 (2): 255â8. doi:10.1016/0005-2736(91)90051-9. PMID 1831660.
  89. ^Tsubaki M (1993). "Fourier-transform infrared study of cyanide binding
      to the Fea3-CuB binuclear site of bovine heart cytochrome c oxidase:
      implication of the redox-linked conformational change at the binuclear
      site". Biochemistry. 32 (1): 164â73. doi:10.1021/bi00052a022.
      PMID 8380331.
  90. ^Heytler PG (1979). "Uncouplers of oxidative phosphorylation". Meth.
      Enzymol. Methods in Enzymology. 55: 462â42. doi:10.1016/0076-6879
      (79)55060-5. ISBN 978-0-12-181955-2. PMID 156853.
  91. ^Lambert AJ, Brand MD (2004). "Inhibitors of the quinone-binding site
      allow rapid superoxide production from mitochondrial NADH:ubiquinone
      oxidoreductase (complex I)". J. Biol. Chem. 279 (38): 39414â20. doi:
      10.1074/jbc.M406576200. PMID 15262965.
  92. ^Dervartanian DV, Veeger C (November 1964). "Studies on succinate
      dehydrogenase: I. Spectral properties of the purified enzyme and
      formation of enzyme-competitive inhibitor complexes". Biochim. Biophys.
      Acta. 92 (2): 233â47. doi:10.1016/0926-6569(64)90182-8. PMID 14249115.
  93. ^Ricquier D, Bouillaud F (2000). "The_uncoupling_protein_homologues:
      UCP1,_UCP2,_UCP3,_StUCP_and_AtUCP". Biochem. J. 345 (2): 161â79. doi:
      10.1042/0264-6021:3450161. PMC 1220743. PMID 10620491.
  94. ^BoreckÃ½ J, Vercesi AE (2005). "Plant uncoupling mitochondrial protein
      and alternative oxidase: energy metabolism and stress". Biosci. Rep. 25
      (3â4): 271â86. doi:10.1007/s10540-005-2889-2. PMID 16283557.
  95. ^Harden A, Young WJ.; Young (1906). "The alcoholic ferment of yeast-
      juice". Proceedings of the Royal Society. B (77): 405â20. doi:10.1098/
      rspb.1906.0029.
  96. ^Kalckar HM (1974). "Origins of the concept oxidative phosphorylation".
      Mol. Cell. Biochem. 5 (1â2): 55â63. doi:10.1007/BF01874172.
      PMID 4279328.
  97. ^Lipmann F (1941). "Metabolic generation and utilization of phosphate
      bond energy". Adv Enzymol. 1: 99â162. doi:10.4159/
      harvard.9780674366701.c141. ISBN 9780674366701.
  98. ^Friedkin M, Lehninger AL (1 April 1949). "Esterification_of_inorganic
      phosphate_coupled_to_electron_transport_between_dihydrodiphosphopyridine
      nucleotide_and_oxygen". J. Biol. Chem. 178 (2): 611â44. PMID 18116985.
      Archived from the original on 16 December 2008.
  99. ^Kalckar, H. M. (1991). "50 years of biological research--from oxidative
      phosphorylation to energy requiring transport regulation". Annual_Review
      of_Biochemistry. 60: 1â37. doi:10.1146/annurev.bi.60.070191.000245.
      PMID 1883194.
 100. ^Belitser, V. A., Tsibakova, E. T. (1939). "About phosphorilation
      mechanism coupled with respiration". Biokhimiya. 4: 516â534.CS1 maint:
      Multiple names: authors list (link)
 101. ^Slater EC (1953). "Mechanism of phosphorylation in the respiratory
      chain". Nature. 172 (4387): 975â8. Bibcode:1953Natur.172..975S. doi:
      10.1038/172975a0. PMID 13111237.
 102. ^Mitchell P (1961). "Coupling of phosphorylation to electron and hydrogen
      transfer by a chemi-osmotic type of mechanism". Nature. 191 (4784):
      144â8. Bibcode:1961Natur.191..144M. doi:10.1038/191144a0.
      PMID 13771349.
 103. ^Milton H. Saier Jr. Peter Mitchell and the Vital Force. OCLC 55202414.
 104. ^Mitchell, Peter (1978). "David_Keilin's_Respiratory_Chain_Concept_and
      Its_Chemiosmotic_Consequences" (PDF). Nobel lecture. Nobel Foundation.
      Archived (PDF) from the original on 2007-09-27. Retrieved 2007-07-21.
 105. ^Pullman ME, Penefsky HS, Datta A, Racker E (1 November 1960). "Partial
      resolution_of_the_enzymes_catalyzing_oxidative_phosphorylation._I.
      Purification_and_properties_of_soluble_dinitrophenol-stimulated_adenosine
      triphosphatase". J. Biol. Chem. 235 (11): 3322â9. PMID 13738472.
      Archived from the original on 29 September 2007.
 106. ^Boyer PD, Cross RL, Momsen W (1973). "A_new_concept_for_energy_coupling
      in_oxidative_phosphorylation_based_on_a_molecular_explanation_of_the
      oxygen_exchange_reactions". Proc. Natl. Acad. Sci. U.S.A. 70 (10):
      2837â9. Bibcode:1973PNAS...70.2837B. doi:10.1073/pnas.70.10.2837.
      PMC 427120. PMID 4517936.
 107. ^"The_Nobel_Prize_in_Chemistry_1997". Nobel Foundation. Archived from the
      original on 2017-03-25. Retrieved 2007-07-21.
***** Further reading[edit] *****
**** Introductory[edit] ****
    * Nelson DL; Cox MM (2004). Lehninger Principles of Biochemistry (4th ed.).
      W. H. Freeman. ISBN 0-7167-4339-6.
Schneider ED; Sagan D (2006). Into the Cool: Energy Flow, Thermodynamics and
Life (1st ed.). University of Chicago Press. ISBN 0-226-73937-6.
Lane_N (2006). Power, Sex, Suicide: Mitochondria and the Meaning of Life (1st
ed.). Oxford University Press, USA. ISBN 0-19-920564-7.
**** Advanced[edit] ****
    * Nicholls DG; Ferguson SJ (2002). Bioenergetics 3 (1st ed.). Academic
      Press. ISBN 0-12-518121-3.
Haynie D (2001). Biological Thermodynamics (1st ed.). Cambridge University
Press. ISBN 0-521-79549-4.
Rajan SS (2003). Introduction to Bioenergetics (1st ed.). Anmol. ISBN 81-261-
1364-2.
Wikstrom M (Ed) (2005). Biophysical and Structural Aspects of Bioenergetics
(1st ed.). Royal Society of Chemistry. ISBN 0-85404-346-2.CS1 maint: Extra
text: authors list (link)
**** General resources[edit] ****
    * Animated_diagrams_illustrating_oxidative_phosphorylation Wiley_and_Co
      Concepts in Biochemistry
    * On-line_biophysics_lectures Antony Crofts, University_of_Illinois_at
      UrbanaâChampaign
    * ATP_Synthase Graham Johnson
**** Structural resources[edit] ****
    * PDB molecule of the month:
          o ATP_synthase
          o Cytochrome_c
          o Cytochrome_c_oxidase
    * Interactive molecular models at Universidade_Fernando_Pessoa:
          o NADH_dehydrogenase
          o succinate_dehydrogenase
          o Coenzyme_Q_-_cytochrome_c_reductase
          o cytochrome_c_oxidase
    * v
    * t
    * e
Metabolism: Citric_acid_cycle enzymes
                              * Citrate_synthase
                              * Aconitase
                              * Isocitrate_dehydrogenase
Cycle                         * Oxoglutarate_dehydrogenase
                              * Succinyl_CoA_synthetase
                              * Succinate_dehydrogenase (SDHA)
                              * Fumarase
                              * Malate_dehydrogenase and ETC
                                                      * Pyruvate_dehydrogenase
                                                        complex (E1, E2, E3)
                                                      * (regulated by Pyruvate
                          to acetyl-CoA                 dehydrogenase_kinase
                                                        and Pyruvate
Anaplerotic                                             dehydrogenase
                                                        phosphatase)
                          to Î±-ketoglutaric_aci    * Glutamate_dehydrogenase
                          to succinyl-CoA             * Methylmalonyl-CoA
                                                        mutase
                          to oxaloacetate             * Pyruvate_carboxylase
                                                      * Aspartate_transaminase
                                      * Complex_I/NADH_dehydrogenase
                                      * Complex_II/Succinate_dehydrogenase
                                      * Coenzyme_Q
                                      * Complex_III/Coenzyme_Q_-_cytochrome_c
                                        reductase
                                      * Cytochrome_c
                                      * Complex_IV/Cytochrome_c_oxidase
                                      * Coenzyme_Q10 synthesis: COQ2
                          Primary     * COQ3
Mitochondrial                         * COQ4
electron_transport_chain/             * COQ5
oxidative phosphorylation             * COQ6
                                      * COQ7
                                      * COQ9
                                      * COQ10A
                                      * COQ10B
                                      * PDSS1
                                      * PDSS2
                                      * Alternative_oxidase
                          Other       * Electron-transferring-flavoprotein
                                        dehydrogenase
    * v
    * t
    * e
Metabolism, catabolism, anabolism
               * Metabolic_pathway
General        * Metabolic_network
               * Primary_nutritional_groups
           Aerobic          * Glycolysis â Pyruvate_decarboxylation â Citric_acid_cycle â Oxidative
           respiration        phosphorylation (electron_transport_chain + ATP_synthase)
           Anaerobic        * Electron acceptors are other than oxygen
Energy     respiration
metabolism                  * Glycolysis â Substrate-level_phosphorylation
           Fermentation           o ABE
                                  o Ethanol
                                  o Lactic_acid
           Protein_metabolism           * Protein_synthesis
                                        * Catabolism
                                                 * Glycolysis â Gluconeogenesis
                                                 * Glycogenolysis â Glycogenesis
                                                 * Pentose_phosphate_pathway
                                    Human        * Fructolysis
                                                 * Galactolysis
           Carbohydrate_metabolism               * Glycosylation
           (carbohydrate_catabolism                    o N-linked
           and anabolism)                              o O-linked
                                                 * Photosynthesis
                                                 * Anoxygenic_photosynthesis
                                    Nonhuman     * Chemosynthesis
                                                 * Carbon_fixation
Specific                                         * Xylose_metabolism
paths                                            * Radiotrophism
                                    Fatty_acid_metabolism     * Fatty_acid_degradation (Beta_oxidation)
                                                              * Fatty_acid_synthesis
           Lipid_metabolism                                   * Steroid_metabolism
           (lipolysis, lipogenesis)                           * Sphingolipid_metabolism
                                    Other                     * Eicosanoid_metabolism
                                                              * Ketosis
                                                              * Reverse_cholesterol_transport
           Amino_acid                   * Amino_acid_synthesis
                                        * Urea_cycle
           Nucleotide                   * Purine_metabolism
           metabolism                   * Nucleotide_salvage
                                        * Pyrimidine_metabolism
                                        * Metal_metabolism
           Other                              o Iron_metabolism
                                        * Ethanol_metabolism
    * v
    * t
    * e
Metabolism map
[Metro-style_map_of_major_metabolic_pathways]
Carbon
fixation
Photo-
respiration
Pentose
phosphate
pathway
Citric
acid_cycle
Glyoxylate
cycle
Urea
cycle
Fatty
acid
synthesis
Fatty
acid
elongation
Beta
oxidation
Peroxisomal
beta
oxidation

Glyco-
genolysis
Glyco-
genesis
Glyco-
lysis
Gluconeo-
genesis
Pyruvate
decarb-
oxylation
Fermentation
Keto-
lysis
Keto-
genesis
feeders_to
gluconeo-
genesis
Direct_/_C4_/_CAM
carbon_intake
Light_reaction
Oxidative
phosphorylation
Amino_acid
deamination
Citrate
shuttle
Lipogenesis
Lipolysis
Steroidogenesis
MVA_pathway
MEP_pathway
Shikimate
pathway
Transcription_&
replication
Translation
Proteolysis
Glycosyl-
ation

Sugar
acids
Double/multiple
sugars & glycans
Simple
sugars
Inositol-P
Amino_sugars
& sialic_acids
Nucleotide_sugars
Hexose-P
Triose-P
Glycerol
P-glycerates
Pentose-P
Tetrose-P
Propionyl
-CoA
Succinate
Acetyl
-CoA
Pentose-P
P-glycerates
Glyoxylate
Photosystems
Pyruvate
Lactate
Acetyl
-CoA
Citrate
Oxalo-
acetate
Malate
Succinyl
-CoA
Î±-Keto-
glutarate
Ketone
bodies
Respiratory
chain
Serine_group
Alanine
Branched-chain
amino_acids
Aspartate
group
Homoserine
group
& lysine
Glutamate
group
& proline
Arginine
Creatine
& polyamines
Ketogenic &
glucogenic
amino_acids
Amino_acids
Shikimate
Aromatic_amino
acids & histidine
Ascorbate
(vitamin_C)
Î´-ALA
Bile
pigments
Hemes
Cobalamins (vitamin_B12)
Various
vitamin_Bs
Calciferols
(vitamin_D)
Retinoids
(vitamin_A)
Quinones (vitamin_K)
& carotenoids (vitamin_E)
Cofactors
Vitamins
& minerals
Antioxidants
PRPP
Nucleotides
Nucleic
acids
Proteins
Glycoproteins
& proteoglycans
Chlorophylls
MEP
MVA
Acetyl
-CoA
Polyketides
Terpenoid
backbones
Terpenoids
& carotenoids (vitamin_A)
Cholesterol
Bile_acids
Glycero-
phospholipids
Glycerolipids
Acyl-CoA
Fatty
acids
Glyco-
sphingolipids
Sphingolipids
Waxes
Polyunsaturated
fatty_acids
Neurotransmitters
& thyroid_hormones
Steroids
Endo-
cannabinoids
Eicosanoids
[The image above contains clickable links]
Major metabolic_pathways in metro-style_map. Click any text (name of pathway
or metabolites) to link to the corresponding article.
Single lines: pathways common to most lifeforms. Double lines: pathways not in
humans (occurs in e.g. plants, fungi, prokaryotes). [Metabolic metro
orange.svg] Orange nodes: carbohydrate_metabolism. [Metabolic metro
purple.svg] Violet nodes: photosynthesis. [Metabolic metro red.svg] Red nodes:
cellular_respiration. [Metabolic metro pink.svg] Pink nodes: cell_signaling.
[Metabolic metro blue.svg] Blue nodes: amino_acid_metabolism. [Metabolic metro
grey.svg] Grey nodes: vitamin and cofactor metabolism. [Metabolic metro
brown.svg] Brown nodes: nucleotide and protein metabolism. [Metabolic metro
green.svg] Green nodes: lipid_metabolism.
    * [ATP-3D-vdW.png]Metabolism_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Oxidative_phosphorylation&oldid=909262311"
Categories:
    * Cellular_respiration
    * Integral_membrane_proteins
    * Metabolism
Hidden categories:
    * CS1_maint:_Multiple_names:_authors_list
    * Interlanguage_link_template_link_number
    * CS1_maint:_Extra_text:_authors_list
    * Featured_articles
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * CatalÃ 
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * Hrvatski
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * Latina
    * LatvieÅ¡u
    * LÃ«tzebuergesch
    * Magyar
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Basa_Sunda
    * Suomi
    * Svenska
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * ä¸­æ
Edit_links
    * This page was last edited on 4 August 2019, at 07:46 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
