The following text has been accessed from https://en.wikipedia.org/wiki/Molecular_clock at Fri Aug 9 04:03:03 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Molecular clock ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Technique to deduce the time in prehistory when two or more life forms diverged
Not to be confused with Chemical_clock.
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
The molecular clock is figurative term for a technique that uses the mutation
rate of biomolecules to deduce_the_time in prehistory when two or more life
forms diverged. The biomolecular data used for such calculations are usually
nucleotide sequences for DNA or amino_acid sequences for proteins. The
benchmarks for determining the mutation rate are often fossil or archaeological
dates. The molecular clock was first tested in 1962 on the hemoglobin protein
variants of various animals, and is commonly used in molecular_evolution to
estimate times of speciation or radiation. It is sometimes called a gene clock
or an evolutionary clock.
⁰
***** Contents *****
    * 1_Early_discovery_and_genetic_equidistance
    * 2_Relationship_with_neutral_theory
    * 3_Calibration
          o 3.1_Node_calibration
          o 3.2_Tip_calibration
                # 3.2.1_Total_evidence_dating
          o 3.3_Methods
    * 4_Non-constant_rate_of_molecular_clock
    * 5_Uses
    * 6_See_also
    * 7_References
    * 8_Further_reading
    * 9_External_links
***** Early discovery and genetic equidistance[edit] *****
The notion of the existence of a so-called "molecular clock" was first
attributed to Ãmile_Zuckerkandl and Linus_Pauling who, in 1962, noticed that
the number of amino_acid differences in hemoglobin between different lineages
changes roughly linearly with time, as estimated from fossil evidence.[1] They
generalized this observation to assert that the rate of evolutionary change of
any specified protein was approximately constant over time and over different
lineages (known as the molecular clock hypothesis).
The genetic equidistance phenomenon was first noted in 1963 by Emanuel
Margoliash, who wrote: "It appears that the number of residue differences
between cytochrome_c of any two species is mostly conditioned by the time
elapsed since the lines of evolution leading to these two species originally
diverged. If this is correct, the cytochrome c of all mammals should be equally
different from the cytochrome c of all birds. Since fish diverges from the main
stem of vertebrate evolution earlier than either birds or mammals, the
cytochrome c of both mammals and birds should be equally different from the
cytochrome c of fish. Similarly, all vertebrate cytochrome c should be equally
different from the yeast protein."[2] For example, the difference between the
cytochrome c of a carp and a frog, turtle, chicken, rabbit, and horse is a very
constant 13% to 14%. Similarly, the difference between the cytochrome c of a
bacterium and yeast, wheat, moth, tuna, pigeon, and horse ranges from 64% to
69%. Together with the work of Emile Zuckerkandl and Linus Pauling, the genetic
equidistance result directly led to the formal postulation of the molecular
clock hypothesis in the early 1960s.[3]
Similarly, Vincent_Sarich and Allan_Wilson in 1967 demonstrated that molecular
differences among modern Primates in albumin proteins showed that approximately
constant rates of change had occurred in all the lineages they assessed.[4] The
basic logic of their analysis involved recognizing that if one species lineage
had evolved more quickly than a sister species lineage since their common
ancestor, then the molecular differences between an outgroup (more distantly
related) species and the faster-evolving species should be larger (since more
molecular changes would have accumulated on that lineage) than the molecular
differences between the outgroup species and the slower-evolving species. This
method is known as the relative_rate_test. Sarich and Wilson's paper reported,
for example, that human (Homo_sapiens) and chimpanzee (Pan_troglodytes) albumin
immunological cross-reactions suggested they were about equally different from
Ceboidea (New World Monkey) species (within experimental error). This meant
that they had both accumulated approximately equal changes in albumin since
their shared common ancestor. This pattern was also found for all the primate
comparisons they tested. When calibrated with the few well-documented fossil
branch points (such as no Primate fossils of modern aspect found before the K-
T_boundary), this led Sarich and Wilson to argue that the human-chimp
divergence probably occurred only ~4-6 million years ago.[5]
***** Relationship with neutral theory[edit] *****
The observation of a clock-like rate of molecular change was originally purely
phenomenological. Later, the work of Motoo_Kimura[6] developed the neutral
theory_of_molecular_evolution, which predicted a molecular clock. Let there be
N individuals, and to keep this calculation simple, let the individuals be
haploid (i.e. have one copy of each gene). Let the rate of neutral mutations
(i.e. mutations with no effect on fitness) in a new individual be     &#x03BC;
{\displaystyle \mu }  [\mu ]. The probability that this new mutation will
become fixed in the population is then 1/N, since each copy of the gene is as
good as any other. Every generation, each individual can have new mutations, so
there are     &#x03BC;   {\displaystyle \mu }  [\mu ]N new neutral mutations in
the population as a whole. That means that each generation,     &#x03BC;
{\displaystyle \mu }  [\mu ] new neutral mutations will become fixed. If most
changes seen during molecular_evolution are neutral, then fixations in a
population will accumulate at a clock-rate that is equal to the rate of neutral
mutations in an individual.
***** Calibration[edit] *****
The molecular clock alone can only say that one time period is twice as long as
another: it cannot assign concrete dates. For viral phylogenetics and ancient
DNA studiesâtwo areas of evolutionary biology where it is possible to sample
sequences over an evolutionary timescaleâthe dates of the intermediate
samples can be used to more precisely calibrate the molecular clock. However,
most phylogenies require that the molecular clock be calibrated against
independent evidence about dates, such as the fossil record.[7] There are two
general methods for calibrating the molecular clock using fossil data: node
calibration and tip calibration.[8]
**** Node calibration[edit] ****
Sometimes referred to as node dating, node calibration is a method for
phylogeny calibration that is done by placing fossil constraints at nodes. A
node calibration fossil is the oldest discovered representative of that clade,
which is used to constrain its minimum age. Due to the fragmentary nature of
the fossil record, the true most recent common ancestor of a clade will likely
never be found.[8] In order to account for this in node calibration analyses, a
maximum clade age must be estimated. Determining the maximum clade age is
challenging because it relies on negative_evidenceâthe absence of older
fossils in that clade. There are a number of methods for deriving the maximum
clade age using birth-death models, fossil stratigraphic distribution analyses,
or taphonomic controls.[9] Alternatively, instead of a maximum and a minimum, a
prior_probability of the divergence time can be established and used to
calibrate the clock. There are several prior probability distributions
including normal, lognormal, exponential, gamma, uniform, etc.) that can be
used to express the probability of the true age of divergence relative to the
age of the fossil;[10] however, there are very few methods for estimating the
shape and parameters of the probability distribution empirically.[11] The
placement of calibration nodes on the tree informs the placement of the
unconstrained nodes, giving divergence date estimates across the phylogeny.
Historical methods of clock calibration could only make use of a single fossil
constraint (non-parametric rate smoothing),[12] while modern analyses (BEAST
[13] and r8s[14]) allow for the use of multiple fossils to calibrate the
molecular clock. Simulation studies have shown that increasing the number of
fossil constraints increases the accuracy of divergence time estimation.[15]
**** Tip calibration[edit] ****
Sometimes referred to as tip_dating, tip calibration is a method of molecular
clock calibration in which fossils are treated as taxa and placed on the tips
of the tree. This is achieved by creating a matrix that includes a molecular
dataset for the extant_taxa along with a morphological dataset for both the
extinct and the extant taxa.[9] Unlike node calibration, this method
reconstructs the tree topology and places the fossils simultaneously. Molecular
and morphological models work together simultaneously, allowing morphology to
inform the placement of fossils.[8] Tip calibration makes use of all relevant
fossil taxa during clock calibration, rather than relying on only the oldest
fossil of each clade. This method does not rely on the interpretation of
negative evidence to infer maximum clade ages.[9]
*** Total evidence dating[edit] ***
This approach to tip calibration goes a step further by simultaneously
estimating fossil placement, topology, and the evolutionary timescale. In this
method, the age of a fossil can inform its phylogenetic position in addition to
morphology. By allowing all aspects of tree reconstruction to occur
simultaneously, the risk of biased results is decreased.[8] This approach has
been improved upon by pairing it with different models. One current method of
molecular clock calibration is total evidence dating paired with the fossilized
birth-death (FBD) model and a model of morphological evolution.[16] The FBD
model is novel in that it allows for âsampled ancestors,â which are fossil
taxa that are the direct ancestor of a living taxon or lineage. This allows
fossils to be placed on a branch above an extant organism, rather than being
confined to the tips.[17]
**** Methods[edit] ****
Bayesian methods can provide more appropriate estimates of divergence times,
especially if large datasetsâsuch as those yielded by phylogenomicsâare
employed.[18]
***** Non-constant rate of molecular clock[edit] *****
Sometimes only a single divergence date can be estimated from fossils, with all
other dates inferred from that. Other sets of species have abundant fossils
available, allowing the hypothesis of constant divergence rates to be tested.
DNA sequences experiencing low levels of negative_selection showed divergence
rates of 0.7â0.8% per Myr in bacteria, mammals, invertebrates, and plants.
[19] In the same study, genomic regions experiencing very high negative or
purifying selection (encoding rRNA) were considerably slower (1% per 50 Myr).
In addition to such variation in rate with genomic position, since the early
1990s variation among taxa has proven fertile ground for research too,[20] even
over comparatively short periods of evolutionary time (for example mockingbirds
[21]). Tube-nosed_seabirds have molecular clocks that on average run at half
speed of many other birds,[22] possibly due to long generation times, and many
turtles have a molecular clock running at one-eighth the speed it does in small
mammals, or even slower.[23] Effects of small_population_size are also likely
to confound molecular clock analyses. Researchers such as Francisco Ayala have
more fundamentally challenged the molecular clock hypothesis.[24][25] According
to Ayala's 1999 study, five factors combine to limit the application of
molecular clock models:
    * Changing generation times (If the rate of new mutations depends at least
      partly on the number of generations rather than the number of years)
    * Population size (Genetic_drift is stronger in small populations, and so
      more mutations are effectively neutral)
    * Species-specific differences (due to differing metabolism, ecology,
      evolutionary history, ...)
    * Change in function of the protein studied (can be avoided in closely
      related species by utilizing non-coding_DNA sequences or emphasizing
      silent_mutations)
    * Changes in the intensity of natural selection.
[Phylogram_showing_three_groups,_one_of_which_has_strikingly_longer_branches
than_the_two_others]
Woody bamboos (tribes Arundinarieae and Bambuseae) have long generation times
and lower mutation rates, as expressed by short branches in the phylogenetic
tree, than the fast-evolving herbaceous bamboos (Olyreae).
Molecular clock users have developed workaround solutions using a number of
statistical approaches including maximum_likelihood techniques and later
Bayesian_modeling. In particular, models that take into account rate variation
across lineages have been proposed in order to obtain better estimates of
divergence times. These models are called relaxed molecular clocks[26] because
they represent an intermediate position between the 'strict' molecular clock
hypothesis and Joseph Felsenstein's many-rates model[27] and are made possible
through MCMC techniques that explore a weighted range of tree topologies and
simultaneously estimate parameters of the chosen substitution model. It must be
remembered that divergence dates inferred using a molecular clock are based on
statistical inference and not on direct evidence.
The molecular clock runs into particular challenges at very short and very long
timescales. At long timescales, the problem is saturation. When enough time has
passed, many sites have undergone more than one change, but it is impossible to
detect more than one. This means that the observed number of changes is no
longer linear with time, but instead flattens out. Even at intermediate genetic
distances, with phylogenetic data still sufficient to estimate topology, signal
for the overall scale of the tree can be weak under complex likelihood models,
leading to highly uncertain molecular clock estimates.[28]
At very short time scales, many differences between samples do not represent
fixation of different sequences in the different populations. Instead, they
represent alternative alleles that were both present as part of a polymorphism
in the common ancestor. The inclusion of differences that have not yet become
fixed leads to a potentially dramatic inflation of the apparent rate of the
molecular clock at very short timescales.[29][30]
***** Uses[edit] *****
The molecular clock technique is an important tool in molecular_systematics,
the use of molecular_genetics information to determine the correct scientific
classification of organisms or to study variation in selective forces.
Knowledge of approximately constant rate of molecular evolution in particular
sets of lineages also facilitates establishing the dates of phylogenetic
events, including those not documented by fossils, such as the divergence of
living taxa and the formation of the phylogenetic_tree. In these
casesâespecially over long stretches of timeâthe limitations of the
molecular clock hypothesis (above) must be considered; such estimates may be
off by 50% or more.
***** See also[edit] *****
    * Charles_Darwin
    * Gene_orders
    * Human_mitochondrial_molecular_clock
    * Mitochondrial_Eve and Y-chromosomal_Adam
    * Models_of_DNA_evolution
    * Molecular_evolution
    * Neutral_theory_of_molecular_evolution
***** References[edit] *****
   1. ^Zuckerkandl,_E. and Pauling,_L.B. (1962). "Molecular disease, evolution,
      and genic heterogeneity". In Kasha, M.; Pullman, B (eds.). Horizons in
      Biochemistry. Academic Press, New York. pp. 189â225.CS1 maint: Multiple
      names: authors list (link)
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Margoliash E (October 1963). "PRIMARY_STRUCTURE_AND_EVOLUTION_OF
      CYTOCHROME_C". Proc. Natl. Acad. Sci. U.S.A. 50 (4): 672â9. Bibcode:
      1963PNAS...50..672M. doi:10.1073/pnas.50.4.672. PMC 221244.
      PMID 14077496.
   4. ^Kumar S (August 2005). "Molecular clocks: four decades of evolution".
      Nat. Rev. Genet. 6 (8): 654â62. doi:10.1038/nrg1659. PMID 16136655.
   5. ^Sarich, V M; Wilson, A C (July 1967). "Rates_of_albumin_evolution_in
      primates". Proceedings of the National Academy of Sciences of the United
      States of America. 58 (1): 142â148. doi:10.1073/pnas.58.1.142.
      ISSN 0027-8424. PMC 335609. PMID 4962458.
   6. ^Sarich, Vincent M.; Wilson, Allan C. (1967). "Immunological Time Scale
      for Hominid Evolution". Science. 158 (3805): 1200â1203. doi:10.1126/
      science.158.3805.1200. JSTOR 1722843.
   7. ^Kimura, Motoo (1968). "Evolutionary rate at the molecular level".
      Nature. 217 (5129): 624â626. Bibcode:1968Natur.217..624K. doi:10.1038/
      217624a0. PMID 5637732.
   8. ^Benton, M. J. & Donoghue, P. C. J. (2007). "Paleontological evidence to
      date the Tree of Life". Molecular Biology & Evolution. 24 (1): 26â53.
      doi:10.1093/molbev/msl150. PMID 17047029.
   9. ^ a b c dDonoghue, P.C.J. & Ziheng, Y. (2016). "The_evolution_of_methods
      for_establishing_evolutionary_timescales". Phil. Trans. R. Soc. B. 371
      (1): 20160020. doi:10.1098/rstb.2016.0020. PMC 4920342. PMID 27325838.
  10. ^ a b cO'Reilly, J. E. & Mario D. R. (2015). "Dating Tips for Divergence-
      Time Estimation". Trends in Genetics. 31 (11): 637â650. doi:10.1016/
      j.tig.2015.08.001. PMID 26439502.
  11. ^Drummond, A.; Suchard, M. A. (2012). "Bayesian_phylogenetics_with_BEAUti
      and_the_BEAST_1.7". Molecular Biology and Evolution. 29 (8): 1969â1973.
      doi:10.1093/molbev/mss075. PMC 3408070. PMID 22367748.
  12. ^Claramunt, S.; Cracraft, J. (2015). "A_new_time_tree_reveals_Earth
      history's_imprint_on_the_evolution_of_modern_birds". Sci Adv. 1 (11):
      e1501005. Bibcode:2015SciA....1E1005C. doi:10.1126/sciadv.1501005.
      PMC 4730849. PMID 26824065.
  13. ^Sanderson, M. (1997). "A_nonparametric_approach_to_estimating_divergence
      times_in_the_absence_of_rate_constancy" (PDF). Molecular Biology and
      Evolution. 14 (12): 1218â1231. doi:10.1093/
      oxfordjournals.molbev.a025731.
  14. ^Drummond, A.; Rambaut, A. (2007). "BEAST:_Bayesian_evolutionary_analysis
      by_sampling_trees". BMC Evolutionary Biology. 7: 214. doi:10.1186/1471-
      2148-7-214. PMC 2247476. PMID 17996036.
  15. ^Sanderson, M. (2003). "r8s:_inferring_absolute_rates_of_molecular
      evolution_and_divergence_times_in_the_absence_of_a_molecular_clock"
      (PDF). Bioinformatics. 19 (2): 301â302. doi:10.1093/bioinformatics/
      19.2.301.
  16. ^Zheng Y. & Wiens J. J. (2015). "Do missing data influence the accuracy
      of divergence-time estimation with BEAST?". Molecular Phylogenetics and
      Evolution. 85 (1): 41â49. doi:10.1016/j.ympev.2015.02.002.
      PMID 25681677.
  17. ^Heath, T. A. & Huelsenbeck, J. P. (2014). "The_fossilized_birthâdeath
      process_for_coherent_calibration_of_divergence-time_estimates". PNAS. 111
      (29): E2957âE2966. arXiv:1310.2968. Bibcode:2014PNAS..111E2957H. doi:
      10.1073/pnas.1319091111. PMC 4115571. PMID 25009181.
  18. ^Gavryushkina, A.; Heath, T. A.; Ksepka, D. T.; Stadler, T.; Welch, D. &
      Drummond, A. J. (2016). "Bayesian_Total-Evidence_Dating_Reveals_the
      Recent_Crown_Radiation_of_Penguins". Systematic Biology. 66 (1): 1â17.
      arXiv:1506.04797. doi:10.1093/sysbio/syw060. PMC 5410945. PMID 28173531.
  19. ^Dos Reis, M.; Inoue, J.; Hasegawa, M.; Asher, R. J.; Donoghue, P. C. J.;
      Yang, Z. (2012). "Phylogenomic_datasets_provide_both_precision_and
      accuracy_in_estimating_the_timescale_of_placental_mammal_phylogeny".
      Proceedings of the Royal Society B: Biological Sciences. 279 (1742):
      3491â3500. doi:10.1098/rspb.2012.0683. PMC 3396900. PMID 22628470.
  20. ^Ochman H, Wilson AC (1987). "Evolution in bacteria: evidence for a
      universal substitution rate in cellular genomes". J Mol Evol. 26 (1â2):
      74â86. Bibcode:1987JMolE..26...74O. doi:10.1007/BF02111283.
      PMID 3125340.
  21. ^Douzery, E.J.P., Delsuc, F., Stanhope, M.J. and Huchon, D. (2003).
      "Local molecular clocks in three nuclear genes: divergence times for
      rodents and other mammals, and incompatibility among fossil
      calibrations". Journal of Molecular Evolution. 57: S201âS213. Bibcode:
      2003JMolE..57S.201D. doi:10.1007/s00239-003-0028-x. PMID 15008417.CS1
      maint: Multiple names: authors list (link)
  22. ^Hunt, J.S., Bermingham, E., and Ricklefs, R.E. (2001). "Molecular
      systematics and biogeography of Antillean thrashers, tremblers, and
      mockingbirds (Aves: Mimidae)". Auk. 118 (1): 35â55. doi:10.1642/0004-
      8038(2001)118[0035:MSABOA]2.0.CO;2. ISSN 0004-8038.CS1 maint: Multiple
      names: authors list (link)
  23. ^Rheindt, F. E. & Austin, J. (2005). "Major_analytical_and_conceptual
      shortcomings_in_a_recent_taxonomic_revision_of_the_Procellariiformes_-_A
      reply_to_Penhallurick_and_Wink_(2004)" (PDF). Emu. 105 (2): 181â186.
      doi:10.1071/MU04039.
  24. ^Avise, J.C., Bowen, W., Lamb, T., Meylan, A.B. and Bermingham, E. (1 May
      1992). "Mitochondrial_DNA_Evolution_at_a_Turtle's_Pace:_Evidence_for_Low
      Genetic_Variability_and_Reduced_Microevolutionary_Rate_in_the
      Testudines". Molecular Biology and Evolution. 9 (3): 457â473. doi:
      10.1093/oxfordjournals.molbev.a040735. PMID 1584014.CS1 maint: Multiple
      names: authors list (link)
  25. ^Ayala, F.J. (1999). "Molecular_clock_mirages". BioEssays. 21 (1):
      71â75. doi:10.1002/(SICI)1521-1878(199901)21:1<71::AID-BIES9>3.0.CO;2-
      B. PMID 10070256. Archived from the_original on 16 December 2012.
  26. ^Schwartz, J. H. & Maresca, B. (2006). "Do Molecular Clocks Run at All? A
      Critique of Molecular Systematics". Biological Theory. 1 (4): 357â371.
      CiteSeerX 10.1.1.534.4502. doi:10.1162/biot.2006.1.4.357. Lay_summary –
      Science_Daily.
  27. ^Drummond, A.J., Ho, S.Y.W., Phillips, M.J. and Rambaut A. (2006).
      "Relaxed_Phylogenetics_and_Dating_with_Confidence". PLoS_Biology. 4 (5):
      e88. doi:10.1371/journal.pbio.0040088. PMC 1395354. PMID 16683862.CS1
      maint: Multiple names: authors list (link)
  28.  [open_access]
  29. ^Felsenstein, J (2001). "Taking variation of evolutionary rates between
      sites into account in inferring phylogenies". J Mol Evol. 53 (4â5):
      447â55. Bibcode:2001JMolE..53..447F. doi:10.1007/s002390010234.
      PMID 11675604.
  30. ^ Marshall, D. C., et al. 2016. Inflation of molecular clock rates and
      dates: molecular phylogenetics, biogeography, and diversification of a
      global cicada radiation from Australasia (Hemiptera: Cicadidae:
      Cicadettini). Systematic_Biology_65(1):16â34.
  31. ^Ho SY, Phillips MJ, Cooper A, Drummond AJ (2005). "Time dependency of
      molecular rate estimates and systematic overestimation of recent
      divergence times". Molecular Biology & Evolution. 22 (7): 1561â1568.
      doi:10.1093/molbev/msi145. PMID 15814826.
  32. ^Peterson GI, Masel J (2009). "Quantitative_Prediction_of_Molecular_Clock
      and_Ka/Ks_at_Short_Timescales". Molecular Biology & Evolution. 26 (11):
      2595â2603. doi:10.1093/molbev/msp175. PMC 2912466. PMID 19661199.
***** Further reading[edit] *****
    * Morgan, G.J. (1998). "Emile Zuckerkandl, Linus Pauling, and the Molecular
      Evolutionary Clock, 1959-1965". Journal of the History of Biology. 31
      (2): 155â178. doi:10.1023/A:1004394418084. PMID 11620303.
Zuckerkandl,_E.; Pauling,_L.B. (1965). "Evolutionary divergence and convergence
in proteins". In Bryson, V.; Vogel, H.J. (eds.). Evolving Genes and Proteins.
Academic Press, New York. pp. 97â166.
San Mauro, D.; Agorreta, A. (2010). "Molecular_systematics:_a_synthesis_of_the
common_methods_and_the_state_of_knowledge". Cellular & Molecular Biology
Letters. 15 (2): 311â341. doi:10.2478/s11658-010-0010-8. PMC 6275913.
PMID 20213503.
***** External links[edit] *****
    * Allan_Wilson_and_the_molecular_clock
    * Molecular_clock_explanation_of_the_molecular_equidistance_phenomenon
    * Date-a-Clade_service_for_the_molecular_tree_of_life
    * v
    * t
    * e
Chronology
                       * Archaeology
                       * Astronomy
                       * Geology
Key topics             * History
                             o Big_History
                       * Paleontology
                       * Time
                                     * Human_Era
                                     * Ab_urbe_condita
                                     * Anno_Domini / Common_Era
                                     * Anno_Mundi
                                     * Byzantine_era
                                     * Seleucid_era
                                     * Spanish_era
                                     * Before_Present
                   Calendar_eras     * Hijri
                                     * Egyptian
    * Periods                        * Sothic_cycle
    * Eras                           * Hindu_units_of_time (Yuga)
    * Epochs                         * Mesoamerican
                                           o Long_Count
                                           o Short_Count
                                           o Tzolk'in
                                           o Haab'
                                     * Canon_of_Kings
                   Regnal_year       * Lists_of_kings
                                     * Limmu
                                     * Chinese
                   Era_names         * Japanese
                                     * Korean
                                     * Vietnamese
                                           * Pre-Julian_Roman
                   Pre-Julian / Julian    * Original_Julian
                                           * Proleptic_Julian
                                           * Revised_Julian
                                           * Gregorian
                                           * Proleptic_Gregorian
                   Gregorian               * Old_Style_and_New_Style_dates
                                           * Adoption_of_the_Gregorian_calendar
                                           * Dual_dating
                                           * Lunisolar
                   Astronomical            * Solar
Calendars                                  * Lunar
                                           * Astronomical_year_numbering
                                           * Chinese_sexagenary_cycle
                                           * Geologic_Calendar
                                           * Hebrew
                                           * Iranian
                   Others                  * Islamic
                                           * ISO_week_date
                                           * Mesoamerican
                                                 o Maya
                                                 o Aztec
                                           * Winter_count (Plains Indians)
                       * Cosmic_Calendar
                       * Ephemeris
Astronomic_time        * Galactic_year
                       * Metonic_cycle
                       * Milankovitch_cycles
                                 * Deep_time
                   Concepts      * Geological_history_of_Earth
                                 * Geological_time_units
                                 * Global_Standard_Stratigraphic_Age_(GSSA)
                   Standards     * Global_Boundary_Stratotype_Section_and_Point
Geologic_time                      (GSSP)
                                 * Chronostratigraphy
                                 * Geochronology
                   Methods       * Isotope_geochemistry
                                 * Law_of_superposition
                                 * Luminescence_dating
                                 * Samariumâneodymium_dating
                                       * Amino_acid_racemisation
                                       * Archaeomagnetic_dating
                                       * Dendrochronology
                                       * Ice_core
                                       * Incremental_dating
                                       * Lichenometry
                   Absolute_dating     * Paleomagnetism
                                       * Radiometric_dating
Chronological                                o Radiocarbon
dating                                       o Uraniumâlead
                                             o Potassiumâargon
                                       * Tephrochronology
                                       * Luminescence_dating
                                       * Thermoluminescence_dating
                                       * Fluorine_absorption
                                       * Nitrogen_dating
                   Relative_dating     * Obsidian_hydration
                                       * Seriation
                                       * Stratigraphy
Genetic methods        * Molecular clock
Linguistic methods     * Glottochronology
                       * Chronicle
                       * New_Chronology
                       * Synchronoptic_view
                       * Timeline
Related topics         * Year_zero
                       * Circa
                       * Floruit
                       * Terminus_post_quem
                       * ASPRO_chronology

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Molecular_clock&oldid=909372260"
Categories:
    * Molecular_evolution
    * Molecular_genetics
    * Phylogenetics
Hidden categories:
    * CS1_maint:_Multiple_names:_authors_list
    * Articles_with_short_description
    * Use_dmy_dates_from_August_2010
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikiquote
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * à¦¬à¦¾à¦à¦²à¦¾
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * Bahasa_Indonesia
    * Italiano
    * ×¢××¨××ª
    * Nederlands
    * æ¥æ¬èª
    * Polski
    * PortuguÃªs
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Simple_English
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 5 August 2019, at 00:11 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
