The following text has been accessed from https://en.wikipedia.org/wiki/Cell_cycle at Thu Aug 8 23:54:45 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Cell cycle ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
For the separation of chromosomes that occurs as part of the cell cycle, see
mitosis. For the Academic journal, see Cell_Cycle_(journal).
See also: Cell_division
Life cycle of the cell
Onion (Allium) cells in different phases of the cell cycle. Growth in an
'organism' is carefully controlled by regulating the cell cycle.
The cell cycle, or cell-division cycle, is the series of events that take place
in a cell leading to duplication of its DNA (DNA_replication) and division of
cytoplasm and organelles to produce two daughter cells. In bacteria, which lack
a cell_nucleus, the cell cycle is divided into the B, C, and D periods. The B
period extends from the end of cell division to the beginning of DNA
replication. DNA replication occurs during the C period. The D period refers to
the stage between the end of DNA replication and the splitting of the bacterial
cell into two daughter cells.[1] In cells with a nucleus, as in eukaryotes, the
cell cycle is also divided into two main stages: interphase and the mitotic (M)
phase (including mitosis and cytokinesis). During interphase, the cell grows,
accumulating nutrients needed for mitosis, and undergoes DNA replication
preparing it for cell division. During the mitotic phase, the replicated
chromosomes and cytoplasm separate into two new daughter cells. To ensure the
proper division of the cell, there are control mechanisms known as cell_cycle
checkpoints.
The cell-division cycle is a vital process by which a single-celled fertilized
egg develops into a mature organism, as well as the process by which hair,
skin, blood_cells, and some internal_organs are renewed. After cell division,
each of the daughter cells begin the interphase of a new cycle. Although the
various stages of interphase are not usually morphologically distinguishable,
each phase of the cell cycle has a distinct set of specialized biochemical
processes that prepare the cell for initiation of cell division.
⁰
***** Contents *****
    * 1_Phases
          o 1.1_G0_phase_(quiescence)
          o 1.2_Interphase
                # 1.2.1_G1_phase_(First_growth_phase_or_Post_mitotic_gap_phase)
                # 1.2.2_S_phase_(DNA_replication)
                # 1.2.3_G2_phase_(growth)
          o 1.3_Mitotic_phase_(chromosome_separation)
          o 1.4_Cytokinesis_phase_(separation_of_all_cell_components)
    * 2_Regulation_of_eukaryotic_cell_cycle
          o 2.1_Role_of_cyclins_and_CDKs
                # 2.1.1_General_mechanism_of_cyclin-CDK_interaction
                # 2.1.2_Specific_action_of_cyclin-CDK_complexes
          o 2.2_Inhibitors
          o 2.3_Transcriptional_regulatory_network
          o 2.4_DNA_replication_and_DNA_replication_origin_activity
    * 3_Checkpoints
    * 4_Fluorescence_imaging_of_the_cell_cycle
    * 5_Role_in_tumor_formation
    * 6_See_also
    * 7_References
    * 8_Further_reading
    * 9_External_links
***** Phases[edit] *****
The eukaryotic cell cycle consists of four distinct phases: G1_phase, S_phase
(synthesis), G2_phase (collectively known as interphase) and M_phase (mitosis
and cytokinesis). M phase is itself composed of two tightly coupled processes:
mitosis, in which the cell's nucleus divides, and cytokinesis, in which the
cell's cytoplasm divides forming two daughter cells. Activation of each phase
is dependent on the proper progression and completion of the previous one.
Cells that have temporarily or reversibly stopped dividing are said to have
entered a state of quiescence called G0_phase.
Schematic of the cell cycle. Outer ring: I = Interphase, M = Mitosis; inner
ring: M = Mitosis, G1 = Gap_1, G2 = Gap_2, S = Synthesis; not in ring: G0 = Gap
0/Resting[2]
 _____________________________________________________________________________
|State________|Phase____|Abbreviation|Description_____________________________|
|Resting      |Gap_0    |G0          |A phase where the cell has left the     |
|_____________|_________|____________|cycle_and_has_stopped_dividing._________|
|             |         |            |Cells increase in size in Gap 1. The G1 |
|             |Gap_1    |G1          |checkpoint control mechanism ensures    |
|             |         |            |that everything is ready for DNA        |
|             |_________|____________|synthesis.______________________________|
|             |Synthesis|S           |DNA_replication occurs during this      |
|Interphase   |_________|____________|phase.__________________________________|
|             |         |            |During the gap between DNA synthesis and|
|             |         |            |mitosis, the cell will continue to grow.|
|             |Gap_2    |G2          |The G2_checkpoint control mechanism     |
|             |         |            |ensures that everything is ready to     |
|_____________|_________|____________|enter_the_M_(mitosis)_phase_and_divide._|
|             |         |            |Cell growth stops at this stage and     |
|             |         |            |cellular energy is focused on the       |
|             |         |            |orderly division into two daughter      |
|Cell_division|Mitosis  |M           |cells. A checkpoint in the middle of    |
|             |         |            |mitosis (Metaphase_Checkpoint) ensures  |
|             |         |            |that the cell is ready to complete cell |
|_____________|_________|____________|division._______________________________|
After cell division, each of the daughter cells begin the interphase of a new
cycle. Although the various stages of interphase are not usually
morphologically distinguishable, each phase of the cell cycle has a distinct
set of specialized biochemical processes that prepare the cell for initiation
of cell division.
**** G0 phase (quiescence)[edit] ****
Plant cell cycle
Animal cell cycle
G0 is a resting phase where the cell has left the cycle and has stopped
dividing. The cell cycle starts with this phase. The word "post-mitotic" is
sometimes used to refer to both quiescent and senescent cells. Non-
proliferative (non-dividing) cells in multicellular eukaryotes generally enter
the quiescent G0 state from G1 and may remain quiescent for long periods of
time, possibly indefinitely (as is often the case for neurons). This is very
common for cells that are fully differentiated. Cellular senescence occurs in
response to DNA damage and external stress and usually constitutes an arrest in
G1. Some cells enter the G0 phase semi-permanently and are considered post-
mitotic, e.g., some liver, kidney, and stomach cells. Many cells do not enter
G0 and continue to divide throughout an organism's life, e.g., epithelial
cells.
Cellular senescence is also a state that occurs in response to DNA damage or
degradation that would make a cell's progeny nonviable; it is often a
biochemical alternative to the self-destruction of such a damaged cell by
apoptosis.
**** Interphase[edit] ****
Interphase is a series of changes that takes place in a newly formed cell and
its nucleus before it becomes capable of division again. It is also called
preparatory phase or intermitosis. Typically interphase lasts for at least 91%
of the total time required for the cell cycle.
Interphase proceeds in three stages, G1, S, and G2, followed by the cycle of
mitosis and cytokinesis. The cell's nuclear DNA contents are duplicated during
S phase.
*** G1 phase (First growth phase or Post mitotic gap phase)[edit] ***
The first phase within interphase, from the end of the previous M phase until
the beginning of DNA synthesis, is called G1 (G indicating gap). It is also
called the growth phase. During this phase, the biosynthetic activities of the
cell, which are considerably slowed down during M phase, resume at a high rate.
The duration of G1 is highly variable, even among different cells of the same
species.[3] In this phase, the cell increases its supply of proteins, increases
the number of organelles (such as mitochondria, ribosomes), and grows in size.
In G1 phase, a cell has three options.
    * To continue cell cycle and enter S_phase
    * Stop cell cycle and enter G0 phase for undergoing differentiation.
    * Become arrested in G1 phase hence it may enter G0 phase or re-enter cell
      cycle.
The deciding point is called check_point (Restriction_point). This check_point
is called the restriction point or START and is regulated by G1/S cyclins,
which cause transition from G1 to S phase. Passage through the G1 check point
commits the cell to division.
*** S phase (DNA replication)[edit] ***
The ensuing S_phase starts when DNA_synthesis commences; when it is complete,
all of the chromosomes have been replicated, i.e., each chromosome consists of
two sister chromatids. Thus, during this phase, the amount of DNA in the cell
has doubled, though the ploidy and number of chromosomes are unchanged. Rates
of RNA transcription and protein_synthesis are very low during this phase. An
exception to this is histone production, most of which occurs during the S
phase.[4][5][6]
*** G2 phase (growth)[edit] ***
G2_phase occurs after DNA replication and is a period of protein synthesis and
rapid cell growth to prepare the cell for mitosis. During this phase
microtubules begin to reorganize to form a spindle (preprophase). Before
proceeding to mitotic_phase, cells must be checked at the G2 checkpoint for any
DNA damage within the chromosomes. The G2 checkpoint is mainly regulated by the
tumor protein p53. If the DNA is damaged, p53 will either repair the DNA or
trigger the apoptosis of the cell. If p53 is dysfunctional or mutated, cells
with damaged DNA may continue through the cell cycle, leading to the
development of cancer.
**** Mitotic phase (chromosome separation)[edit] ****
Main article: Mitosis
The relatively brief M phase consists of nuclear division (karyokinesis). It is
a relatively short period of the cell cycle. M phase is complex and highly
regulated. The sequence of events is divided into phases, corresponding to the
completion of one set of activities and the start of the next. These phases are
sequentially known as:
    * prophase,
    * prometaphase,
    * metaphase,
    * anaphase,
    * telophase
[A_diagram_of_the_mitotic_phases]
Mitosis is the process by which a eukaryotic cell separates the chromosomes in
its cell_nucleus into two identical sets in two nuclei.[7] During the process
of mitosis the pairs of chromosomes condense and attach to microtubules that
pull the sister_chromatids to opposite sides of the cell.[8]
Mitosis occurs exclusively in eukaryotic cells, but occurs in different ways in
different species. For example, animal cells undergo an "open" mitosis, where
the nuclear_envelope breaks down before the chromosomes separate, while fungi
such as Aspergillus_nidulans and Saccharomyces_cerevisiae (yeast) undergo a
"closed" mitosis, where chromosomes divide within an intact cell_nucleus.[9]
**** Cytokinesis phase (separation of all cell components)[edit] ****
Main article: Cytokinesis
Mitosis is immediately followed by cytokinesis, which divides the nuclei,
cytoplasm, organelles and cell_membrane into two cells containing roughly equal
shares of these cellular components. Mitosis and cytokinesis together define
the division of the mother cell into two daughter cells, genetically identical
to each other and to their parent cell. This accounts for approximately 10% of
the cell cycle.
Because cytokinesis usually occurs in conjunction with mitosis, "mitosis" is
often used interchangeably with "M phase". However, there are many cells where
mitosis and cytokinesis occur separately, forming single cells with multiple
nuclei in a process called endoreplication. This occurs most notably among the
fungi and slime_molds, but is found in various groups. Even in animals,
cytokinesis and mitosis may occur independently, for instance during certain
stages of fruit_fly embryonic development.[10] Errors in mitosis can result in
cell death through apoptosis or cause mutations that may lead to cancer.
***** Regulation of eukaryotic cell cycle[edit] *****
Regulation of the cell cycle involves processes crucial to the survival of a
cell, including the detection and repair of genetic damage as well as the
prevention of uncontrolled cell division. The molecular events that control the
cell cycle are ordered and directional; that is, each process occurs in a
sequential fashion and it is impossible to "reverse" the cycle.
**** Role of cyclins and CDKs[edit] ****
                                       [Paul_Nurse_portrait.jpg] [Tim_hunt.jpg]
                                            Nobel Laureate       Nobel Laureate
                                              Paul_Nurse            Tim_Hunt
Two key classes of regulatory molecules, cyclins and cyclin-dependent_kinases
(CDKs), determine a cell's progress through the cell cycle.[11] Leland_H.
Hartwell, R._Timothy_Hunt, and Paul_M._Nurse won the 2001 Nobel_Prize_in
Physiology_or_Medicine for their discovery of these central molecules.[12] Many
of the genes encoding cyclins and CDKs are conserved among all eukaryotes, but
in general more complex organisms have more elaborate cell cycle control
systems that incorporate more individual components. Many of the relevant genes
were first identified by studying yeast, especially Saccharomyces_cerevisiae;
[13] genetic nomenclature in yeast dubs many of these genes cdc (for "cell
division cycle") followed by an identifying number, e.g. cdc25 or cdc20.
Cyclins form the regulatory subunits and CDKs the catalytic subunits of an
activated heterodimer; cyclins have no catalytic activity and CDKs are inactive
in the absence of a partner cyclin. When activated by a bound cyclin, CDKs
perform a common biochemical reaction called phosphorylation that activates or
inactivates target proteins to orchestrate coordinated entry into the next
phase of the cell cycle. Different cyclin-CDK combinations determine the
downstream proteins targeted. CDKs are constitutively expressed in cells
whereas cyclins are synthesised at specific stages of the cell cycle, in
response to various molecular signals.[14]
*** General mechanism of cyclin-CDK interaction[edit] ***
Upon receiving a pro-mitotic extracellular signal, G1 cyclin-CDK complexes
become active to prepare the cell for S phase, promoting the expression of
transcription_factors that in turn promote the expression of S cyclins and of
enzymes required for DNA_replication. The G1 cyclin-CDK complexes also promote
the degradation of molecules that function as S phase inhibitors by targeting
them for ubiquitination. Once a protein has been ubiquitinated, it is targeted
for proteolytic degradation by the proteasome. However, results from a recent
study of E2F transcriptional dynamics at the single-cell level argue that the
role of G1 cyclin-CDK activities, in particular cyclin D-CDK4/6, is to tune the
timing rather than the commitment of cell cycle entry.[15]
Active S cyclin-CDK complexes phosphorylate proteins that make up the pre-
replication_complexes assembled during G1 phase on DNA replication_origins. The
phosphorylation serves two purposes: to activate each already-assembled pre-
replication complex, and to prevent new complexes from forming. This ensures
that every portion of the cell's genome will be replicated once and only once.
The reason for prevention of gaps in replication is fairly clear, because
daughter cells that are missing all or part of crucial genes will die. However,
for reasons related to gene_copy_number effects, possession of extra copies of
certain genes is also deleterious to the daughter cells.
Mitotic cyclin-CDK complexes, which are synthesized but inactivated during S
and G2 phases, promote the initiation of mitosis by stimulating downstream
proteins involved in chromosome condensation and mitotic_spindle assembly. A
critical complex activated during this process is a ubiquitin_ligase known as
the anaphase-promoting_complex (APC), which promotes degradation of structural
proteins associated with the chromosomal kinetochore. APC also targets the
mitotic cyclins for degradation, ensuring that telophase and cytokinesis can
proceed.[16]
*** Specific action of cyclin-CDK complexes[edit] ***
Cyclin_D is the first cyclin produced in the cell cycle, in response to
extracellular signals (e.g. growth_factors). Cyclin D binds to existing CDK4,
forming the active cyclin D-CDK4 complex. Cyclin D-CDK4 complex in turn
phosphorylates the retinoblastoma susceptibility protein (Rb). The
hyperphosphorylated Rb dissociates from the E2F/DP1/Rb complex (which was bound
to the E2F responsive genes, effectively "blocking" them from transcription),
activating E2F. Activation of E2F results in transcription of various genes
like cyclin_E, cyclin_A, DNA_polymerase, thymidine_kinase, etc. Cyclin E thus
produced binds to CDK2, forming the cyclin E-CDK2 complex, which pushes the
cell from G1 to S phase (G1/S, which initiates the G2/M transition).[17] Cyclin
B-cdk1 complex activation causes breakdown of nuclear_envelope and initiation
of prophase, and subsequently, its deactivation causes the cell to exit
mitosis.[14] A quantitative study of E2F transcriptional dynamics at the
single-cell level by using engineered fluorescent reporter cells provided a
quantitative framework for understanding the control logic of cell cycle entry,
challenging the canonical textbook model. Genes that regulate the amplitude of
E2F accumulation, such as Myc, determine the commitment in cell cycle and S
phase entry. G1 cyclin-CDK activities are not the driver of cell cycle entry.
Instead, they primarily tune the timing of E2F increase, thereby modulating the
pace of cell cycle progression.[15]
**** Inhibitors[edit] ****
Overview of signal transduction pathways involved in apoptosis, also known as
"programmed cell death"
Two families of genes, the cip/kip (CDK interacting protein/Kinase inhibitory
protein) family and the INK4a/ARF (Inhibitor of Kinase 4/Alternative Reading
Frame) family, prevent the progression of the cell cycle. Because these genes
are instrumental in prevention of tumor formation, they are known as tumor
suppressors.
The cip/kip family includes the genes p21, p27 and p57. They halt cell cycle in
G1 phase, by binding to, and inactivating, cyclin-CDK complexes. p21 is
activated by p53 (which, in turn, is triggered by DNA damage e.g. due to
radiation). p27 is activated by Transforming Growth Factor of Î² (TGF_Î²), a
growth inhibitor.
The INK4a/ARF family includes p16INK4a, which binds to CDK4 and arrests the
cell cycle in G1 phase, and p14ARF which prevents p53 degradation.
Synthetic inhibitors of Cdc25 could also be useful for the arrest of cell cycle
and therefore be useful as antineoplastic and anticancer agents.[18]
**** Transcriptional regulatory network[edit] ****
Current evidence suggests that a semi-autonomous transcriptional network acts
in concert with the CDK-cyclin machinery to regulate the cell cycle. Several
gene expression studies in Saccharomyces_cerevisiae have identified 800â1200
genes that change expression over the course of the cell cycle.[13][19][20]
They are transcribed at high levels at specific points in the cell cycle, and
remain at lower levels throughout the rest of the cycle. While the set of
identified genes differs between studies due to the computational methods and
criteria used to identify them, each study indicates that a large portion of
yeast genes are temporally regulated.[21]
Many periodically expressed genes are driven by transcription_factors that are
also periodically expressed. One screen of single-gene knockouts identified 48
transcription factors (about 20% of all non-essential transcription factors)
that show cell cycle progression defects.[22] Genome-wide studies using high
throughput technologies have identified the transcription factors that bind to
the promoters of yeast genes, and correlating these findings with temporal
expression patterns have allowed the identification of transcription factors
that drive phase-specific gene expression.[19][23] The expression profiles of
these transcription factors are driven by the transcription factors that peak
in the prior phase, and computational models have shown that a CDK-autonomous
network of these transcription factors is sufficient to produce steady-state
oscillations in gene expression).[20][24]
Experimental evidence also suggests that gene expression can oscillate with the
period seen in dividing wild-type cells independently of the CDK machinery.
Orlando et al. used microarrays to measure the expression of a set of 1,271
genes that they identified as periodic in both wild type cells and cells
lacking all S-phase and mitotic cyclins (clb1,2,3,4,5,6). Of the 1,271 genes
assayed, 882 continued to be expressed in the cyclin-deficient cells at the
same time as in the wild type cells, despite the fact that the cyclin-deficient
cells arrest at the border between G1 and S_phase. However, 833 of the genes
assayed changed behavior between the wild type and mutant cells, indicating
that these genes are likely directly or indirectly regulated by the CDK-cyclin
machinery. Some genes that continued to be expressed on time in the mutant
cells were also expressed at different levels in the mutant and wild type
cells. These findings suggest that while the transcriptional network may
oscillate independently of the CDK-cyclin oscillator, they are coupled in a
manner that requires both to ensure the proper timing of cell cycle events.[20]
Other work indicates that phosphorylation, a post-translational modification,
of cell cycle transcription factors by Cdk1 may alter the localization or
activity of the transcription factors in order to tightly control timing of
target genes.[22][25][26]
While oscillatory transcription plays a key role in the progression of the
yeast cell cycle, the CDK-cyclin machinery operates independently in the early
embryonic cell cycle. Before the midblastula_transition, zygotic transcription
does not occur and all needed proteins, such as the B-type cyclins, are
translated from maternally loaded mRNA.[27]
**** DNA replication and DNA replication origin activity[edit] ****
Analyses of synchronized cultures of Saccharomyces cerevisiae under conditions
that prevent DNA replication initiation without delaying cell cycle progression
showed that origin licensing decreases the expression of genes with origins
near their 3' ends, revealing that downstream origins can regulate the
expression of upstream genes.[28] This confirms previous predictions from
mathematical modeling of a global causal coordination between DNA replication
origin activity and mRNA expression,[29][30][31] and shows that mathematical
modeling of DNA microarray data can be used to correctly predict previously
unknown biological modes of regulation.
***** Checkpoints[edit] *****
Main article: Cell_cycle_checkpoint
Cell_cycle_checkpoints are used by the cell to monitor and regulate the
progress of the cell cycle.[32] Checkpoints prevent cell cycle progression at
specific points, allowing verification of necessary phase processes and repair
of DNA_damage. The cell cannot proceed to the next phase until checkpoint
requirements have been met. Checkpoints typically consist of a network of
regulatory proteins that monitor and dictate the progression of the cell
through the different stages of the cell cycle.
There are several checkpoints to ensure that damaged or incomplete DNA is not
passed on to daughter cells. Three main checkpoints exist: the G1/S checkpoint,
the G2/M checkpoint and the metaphase (mitotic) checkpoint.
G1/S transition is a rate-limiting step in the cell cycle and is also known as
restriction_point.[14] This is where the cell checks whether it has enough raw
materials to fully replicate its DNA (nucleotide bases, DNA synthase,
chromatin, etc.). An unhealthy or malnourished cell will get stuck at this
checkpoint.
The G2/M checkpoint is where the cell ensures that it has enough cytoplasm and
phospholipids for two daughter cells. But sometimes more importantly, it checks
to see if it is the right time to replicate. There are some situations where
many cells need to all replicate simultaneously (for example, a growing embryo
should have a symmetric cell distribution until it reaches the mid-blastula
transition). This is done by controlling the G2/M checkpoint.
The metaphase checkpoint is a fairly minor checkpoint, in that once a cell is
in metaphase, it has committed to undergoing mitosis. However that's not to say
it isn't important. In this checkpoint, the cell checks to ensure that the
spindle has formed and that all of the chromosomes are aligned at the spindle
equator before anaphase begins.[33]
While these are the three "main" checkpoints, not all cells have to pass
through each of these checkpoints in this order to replicate. Many types of
cancer are caused by mutations that allow the cells to speed through the
various checkpoints or even skip them altogether. Going from S to M to S phase
almost consecutively. Because these cells have lost their checkpoints, any DNA
mutations that may have occurred are disregarded and passed on to the daughter
cells. This is one reason why cancer cells have a tendency to exponentially
accrue mutations. Aside from cancer cells, many fully differentiated cell types
no longer replicate so they leave the cell cycle and stay in G0 until their
death. Thus removing the need for cellular checkpoints. An alternative model of
the cell cycle response to DNA damage has also been proposed, known as the
postreplication_checkpoint.
Checkpoint regulation plays an important role in an organism's development. In
sexual reproduction, when egg fertilization occurs, when the sperm binds to the
egg, it releases signalling factors that notify the egg that it has been
fertilized. Among other things, this induces the now fertilized oocyte to
return from its previously dormant, G0, state back into the cell cycle and on
to mitotic replication and division.
p53 plays an important role in triggering the control mechanisms at both G1/
S and G2/M checkpoints. In addition to p53, checkpoint regulators are being
heavily researched for their roles in cancer growth and proliferation.
***** Fluorescence imaging of the cell cycle[edit] *****
Fluorescent proteins visualize the cell cycle progression. IFP2.0-hGem(1/110)
fluorescence is shown in green and highlights the S/G2/M phases. smURFP-hCdtI
(30/120) fluorescence is shown in red and highlights the G0/G1 phases.
Pioneering work by Atsushi Miyawaki and coworkers developed the fluorescent
ubiquitination-based cell cycle indicator (FUCCI), which enables fluorescence
imaging of the cell cycle. Originally, a green_fluorescent_protein, mAG, was
fused to hGem(1/110) and an orange fluorescent_protein (mKO2) was fused to
hCdt1(30/120). Note, these fusions are fragments that contain a nuclear
localization_signal and ubiquitination sites for degradation, but are not
functional proteins. The green_fluorescent_protein is made during the S, G2, or
M phase and degraded during the G0 or G1 phase, while the orange fluorescent
protein is made during the G0 or G1 phase and destroyed during the S, G2, or M
phase.[34] A far-red and near-infrared FUCCI was developed using a
cyanobacteria-derived fluorescent_protein (smURFP) and a bacteriophytochrome-
derived fluorescent_protein (movie_found_at_this_link).[35]
***** Role in tumor formation[edit] *****
A disregulation of the cell cycle components may lead to tumor formation.[36]
As mentioned above, when some genes like the cell cycle inhibitors, RB, p53
etc. mutate, they may cause the cell to multiply uncontrollably, forming a
tumor. Although the duration of cell cycle in tumor cells is equal to or longer
than that of normal cell cycle, the proportion of cells that are in active cell
division (versus quiescent cells in G0 phase) in tumors is much higher than
that in normal tissue.[citation_needed] Thus there is a net increase in cell
number as the number of cells that die by apoptosis or senescence remains the
same.
The cells which are actively undergoing cell cycle are targeted in cancer
therapy as the DNA is relatively exposed during cell division and hence
susceptible to damage by drugs or radiation. This fact is made use of in cancer
treatment; by a process known as debulking, a significant mass of the tumor is
removed which pushes a significant number of the remaining tumor cells from G0
to G1 phase (due to increased availability of nutrients, oxygen, growth factors
etc.). Radiation or chemotherapy following the debulking procedure kills these
cells which have newly entered the cell cycle.[14]
The fastest cycling mammalian cells in culture, crypt cells in the intestinal
epithelium, have a cycle time as short as 9 to 10 hours. Stem cells in resting
mouse skin may have a cycle time of more than 200 hours. Most of this
difference is due to the varying length of G1, the most variable phase of the
cycle. M and S do not vary much.
In general, cells are most radiosensitive in late M and G2 phases and most
resistant in late S phase.
For cells with a longer cell cycle time and a significantly long G1 phase,
there is a second peak of resistance late in G1.
The pattern of resistance and sensitivity correlates with the level of
sulfhydryl compounds in the cell. Sulfhydryls are natural substances that
protect cells from radiation damage and tend to be at their highest levels in S
and at their lowest near mitosis.
Homologous_recombination (HR) is an accurate process for repairing_DNA double-
strand breaks. HR is nearly absent in G1_phase, is most active in S_phase, and
declines in G2/M.[37] Non-homologous_end_joining, a less accurate and more
mutagenic process for repairing double strand breaks, is active throughout the
cell cycle.
***** See also[edit] *****
    * Cellular_model
    * Synchronous_culture â synchronization of cell cultures
***** References[edit] *****
   1. ^Wang JD, Levin PA (November 2009). "Metabolism,_cell_growth_and_the
      bacterial_cell_cycle". Nature Reviews. Microbiology. 7 (11): 822â7.
      doi:10.1038/nrmicro2202. PMC 2887316. PMID 19806155.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Cooper GM (2000). "Chapter_14:_The_Eukaryotic_Cell_Cycle". The cell: a
      molecular approach (2nd ed.). Washington, D.C: ASM Press. ISBN 978-0-
      87893-106-4.
   4. ^Smith JA, Martin L (April 1973). "Do_cells_cycle?". Proc. Natl. Acad.
      Sci. U.S.A. 70 (4): 1263â7. Bibcode:1973PNAS...70.1263S. doi:10.1073/
      pnas.70.4.1263. PMC 433472. PMID 4515625.
   5. ^Wu RS, Bonner WM (December 1981). "Separation of basal histone synthesis
      from S-phase histone synthesis in dividing cells". Cell. 27 (2 Pt 1):
      321â30. doi:10.1016/0092-8674(81)90415-3. PMID 7199388.
   6. ^Nelson DM, Ye X, Hall C, Santos H, Ma T, Kao GD, Yen TJ, Harper JW,
      Adams PD (November 2002). "Coupling_of_DNA_synthesis_and_histone
      synthesis_in_S_phase_independent_of_cyclin/cdk2_activity". Mol. Cell.
      Biol. 22 (21): 7459â72. doi:10.1128/MCB.22.21.7459-7472.2002.
      PMC 135676. PMID 12370293.
   7. ^Cameron IL, Greulich RC (July 1963). "Evidence_for_an_essentially
      constant_duration_of_DNA_synthesis_in_renewing_epithelia_of_the_adult
      mouse". J. Cell Biol. 18: 31â40. doi:10.1083/jcb.18.1.31. PMC 2106275.
      PMID 14018040.
   8. ^ Rubenstein, Irwin, and Susan M. Wick. "Cell." World Book Online
      Reference Center. 2008. 12 January 2008 <"Archived_copy". Archived from
      the_original on 30 May 2011. Retrieved 2009-07-10.CS1 maint: Archived
      copy as title (link)
   9. >
  10. ^Maton A, Lahart D, Hopkins J, Warner MQ, Johnson S, Wright JD (1997).
      Cells: Building Blocks of Life. New Jersey: Prentice Hall. pp. 70â4.
      ISBN 978-0-13-423476-2.
  11. ^De Souza CP, Osmani SA (September 2007). "Mitosis,_not_just_open_or
      closed". Eukaryotic Cell. 6 (9): 1521â7. doi:10.1128/EC.00178-07.
      PMC 2043359. PMID 17660363.
  12. ^Lilly MA, Duronio RJ (April 2005). "New insights into cell cycle control
      from the Drosophila endocycle". Oncogene. 24 (17): 2765â75. doi:
      10.1038/sj.onc.1208610. PMID 15838513.
  13. ^Nigg EA (June 1995). "Cyclin-dependent protein kinases: key regulators
      of the eukaryotic cell cycle". BioEssays. 17 (6): 471â80. doi:10.1002/
      bies.950170603. PMID 7575488.
  14. ^"Press_release". Nobelprize.org.
  15. ^ a bSpellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, Eisen MB,
      Brown PO, Botstein D, Futcher B (December 1998). "Comprehensive
      identification_of_cell_cycle-regulated_genes_of_the_yeast_Saccharomyces
      cerevisiae_by_microarray_hybridization". Molecular Biology of the Cell. 9
      (12): 3273â97. doi:10.1091/mbc.9.12.3273. PMC 25624. PMID 9843569.
  16. ^ a b c dRobbins SL, Cotran RS (2004). Kumar V, Abbas AK, Fausto N
      (eds.). Pathological Basis of Disease. Elsevier. ISBN 978-81-8147-528-2.
  17. ^ a bDong P, Maddali MV, Srimani JK, ThÃ©lot F, Nevins JR, Mathey-Prevot
      B, You L (September 2014). "Division_of_labour_between_Myc_and_G1_cyclins
      in_cell_cycle_commitment_and_pace_control". Nature Communications. 5:
      4750. Bibcode:2014NatCo...5E4750D. doi:10.1038/ncomms5750. PMC 4164785.
      PMID 25175461.
  18. ^Mahmoudi M, Azadmanesh K, Shokrgozar MA, Journeay WS, Laurent S (May
      2011). "Effect of nanoparticles on the cell life cycle". Chemical
      Reviews. 111 (5): 3407â32. doi:10.1021/cr1003166. PMID 21401073.
  19. ^Norbury C (1995). "Cdk2 protein kinase (vertebrates)". In Hardie DG,
      Hanks S (eds.). Protein kinase factsBook. Boston: Academic Press. p. 184.
      ISBN 978-0-12-324719-3.
  20. ^"Presentation_on_CDC25_PHOSPHATASES:_A_Potential_Target_for_Novel
      Anticancer_Agents". Archived from the_original on 3 March 2016. Retrieved
      11 March 2010.
  21. ^ a bPramila T, Wu W, Miles S, Noble WS, Breeden LL (August 2006). "The
      Forkhead_transcription_factor_Hcm1_regulates_chromosome_segregation_genes
      and_fills_the_S-phase_gap_in_the_transcriptional_circuitry_of_the_cell
      cycle". Genes & Development. 20 (16): 2266â78. doi:10.1101/gad.1450606.
      PMC 1553209. PMID 16912276.
  22. ^ a b cOrlando DA, Lin CY, Bernard A, Wang JY, Socolar JE, Iversen ES,
      Hartemink AJ, Haase SB (June 2008). "Global_control_of_cell-cycle
      transcription_by_coupled_CDK_and_network_oscillators". Nature. 453
      (7197): 944â7. Bibcode:2008Natur.453..944O. doi:10.1038/nature06955.
      PMC 2736871. PMID 18463633.
  23. ^de Lichtenberg U, Jensen LJ, FausbÃ¸ll A, Jensen TS, Bork P, Brunak S
      (April 2005). "Comparison of computational methods for the identification
      of cell cycle-regulated genes". Bioinformatics. 21 (7): 1164â71. doi:
      10.1093/bioinformatics/bti093. PMID 15513999.
  24. ^ a bWhite MA, Riles L, Cohen BA (February 2009). "A_systematic_screen
      for_transcriptional_regulators_of_the_yeast_cell_cycle". Genetics. 181
      (2): 435â46. doi:10.1534/genetics.108.098145. PMC 2644938.
      PMID 19033152.
  25. ^Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, Gerber GK, Hannett
      NM, Harbison CT, Thompson CM, Simon I, Zeitlinger J, Jennings EG, Murray
      HL, Gordon DB, Ren B, Wyrick JJ, Tagne JB, Volkert TL, Fraenkel E,
      Gifford DK, Young RA (October 2002). "Transcriptional_regulatory_networks
      in_Saccharomyces_cerevisiae". Science. 298 (5594): 799â804. Bibcode:
      2002Sci...298..799L. doi:10.1126/science.1075090. PMID 12399584.
  26. ^Simon I, Barnett J, Hannett N, Harbison CT, Rinaldi NJ, Volkert TL,
      Wyrick JJ, Zeitlinger J, Gifford DK, Jaakkola TS, Young RA (September
      2001). "Serial regulation of transcriptional regulators in the yeast cell
      cycle". Cell. 106 (6): 697â708. doi:10.1016/S0092-8674(01)00494-9.
      PMID 11572776.
  27. ^Sidorova JM, Mikesell GE, Breeden LL (December 1995). "Cell_cycle-
      regulated_phosphorylation_of_Swi6_controls_its_nuclear_localization".
      Molecular Biology of the Cell. 6 (12): 1641â58. doi:10.1091/
      mbc.6.12.1641. PMC 301322. PMID 8590795.
  28. ^Ubersax JA, Woodbury EL, Quang PN, Paraz M, Blethrow JD, Shah K, Shokat
      KM, Morgan DO (October 2003). "Targets of the cyclin-dependent kinase
      Cdk1". Nature. 425 (6960): 859â64. Bibcode:2003Natur.425..859U. doi:
      10.1038/nature02062. PMID 14574415.
  29. ^Morgan DO (2007). "2â3". The Cell Cycle: Principles of Control.
      London: New Science Press. p. 18. ISBN 978-0-9539181-2-6.
  30. ^Omberg L, Meyerson JR, Kobayashi K, Drury LS, Diffley JF, Alter O
      (October 2009). "Global_effects_of_DNA_replication_and_DNA_replication
      origin_activity_on_eukaryotic_gene_expression". Molecular Systems
      Biology. 5: 312. doi:10.1038/msb.2009.70. PMC 2779084. PMID 19888207.
  31. ^Alter O, Golub GH, Brown PO, Botstein D (2004). Deutscher MP, Black S,
      Boehmer PE, D'Urso G, Fletcher TM, Huijing F, Marshall A, Pulverer B,
      Renault B, Rosenblatt JD, Slingerland JM, Whelan WJ (eds.). "Novel
      Genome-Scale_Correlation_between_DNA_Replication_and_RNA_Transcription
      During_the_Cell_Cycle_in_Yeast_is_Predicted_by_Data-Driven_Models" (PDF).
      Miami Nature Biotechnology Winter Symposium: Cell Cycle, Chromosomes and
      Cancer. Miami Beach, FL: University of Miami School of Medicine, vol. 15
      (January 31 â February 4, 2004).
  32. ^Alter O, Golub GH (November 2004). "Integrative_analysis_of_genome-scale
      data_by_using_pseudoinverse_projection_predicts_novel_correlation_between
      DNA_replication_and_RNA_transcription". Proceedings of the National
      Academy of Sciences of the United States of America. 101 (47):
      16577â82. Bibcode:2004PNAS..10116577A. doi:10.1073/pnas.0406767101.
      PMC 534520. PMID 15545604.
  33. ^Omberg L, Golub GH, Alter O (November 2007). "A_tensor_higher-order
      singular_value_decomposition_for_integrative_analysis_of_DNA_microarray
      data_from_different_studies". Proceedings of the National Academy of
      Sciences of the United States of America. 104 (47): 18371â6. Bibcode:
      2007PNAS..10418371O. doi:10.1073/pnas.0709146104. PMC 2147680.
      PMID 18003902.
  34. ^Elledge SJ (December 1996). "Cell cycle checkpoints: preventing an
      identity crisis". Science. 274 (5293): 1664â72. Bibcode:
      1996Sci...274.1664E. doi:10.1126/science.274.5293.1664. PMID 8939848.
  35. ^LeMaire-Adkins R, Radke K, Hunt PA (December 1997). "Lack_of_checkpoint
      control_at_the_metaphase/anaphase_transition:_a_mechanism_of_meiotic
      nondisjunction_in_mammalian_females". The Journal of Cell Biology. 139
      (7): 1611â9. doi:10.1083/jcb.139.7.1611. PMC 2132649. PMID 9412457.
  36. ^Sakaue-Sawano A, Kurokawa H, Morimura T, Hanyu A, Hama H, Osawa H,
      Kashiwagi S, Fukami K, Miyata T, Miyoshi H, Imamura T, Ogawa M, Masai H,
      Miyawaki A (February 2008). "Visualizing spatiotemporal dynamics of
      multicellular cell-cycle progression". Cell. 132 (3): 487â98. doi:
      10.1016/j.cell.2007.12.033. PMID 18267078.
  37. ^Rodriguez EA, Tran GN, Gross LA, Crisp JL, Shu X, Lin JY, Tsien RY
      (September 2016). "A_far-red_fluorescent_protein_evolved_from_a
      cyanobacterial_phycobiliprotein". Nature Methods. 13 (9): 763â9. doi:
      10.1038/nmeth.3935. PMC 5007177. PMID 27479328.
  38. ^Champeris Tsaniras S, Kanellakis N, Symeonidou IE, Nikolopoulou P,
      Lygerou Z, Taraviras S (June 2014). "Licensing of DNA replication,
      cancer, pluripotency and differentiation: an interlinked world?".
      Seminars in Cell & Developmental Biology. 30: 174â80. doi:10.1016/
      j.semcdb.2014.03.013. PMID 24641889.
  39. ^Mao Z, Bozzella M, Seluanov A, Gorbunova V (September 2008). "DNA_repair
      by_nonhomologous_end_joining_and_homologous_recombination_during_cell
      cycle_in_human_cells". Cell Cycle. 7 (18): 2902â6. doi:10.4161/
      cc.7.18.6679. PMC 2754209. PMID 18769152.
***** Further reading[edit] *****
    * Morgan DO (2007). The Cell Cycle: Principles of Control. London:
      Published by New Science Press in association with Oxford University
      Press. ISBN 978-0-87893-508-6.
Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2008). "Chapter
17". Molecular Biology of the Cell (5th ed.). New York: Garland Science.
ISBN 978-0-8153-4111-6.
Krieger M, Scott MP, Matsudaira PT, Lodish HF, Darnell JE, Zipursky L, Kaiser
C, Berk A (2004). Molecular cell biology. New York: W.H. Freeman and CO.
ISBN 978-0-7167-4366-8.
Watson JD, Baker TA, Bell SP, Gann A, Levine M, Losick R (2004). "Chapter 7".
Molecular biology of the gene (5th ed.). San Francisco: Pearson/Benjamin
Cummings. ISBN 978-0-8053-4642-8.
***** External links[edit] *****
 Wikimedia Commons has media related to Cell_cycle.
    *  This article incorporates public_domain_material from the NCBI document
      "Science_Primer".
    * David_Morgan's_Seminar:_Controlling_the_Cell_Cycle
    * The_cell_cycle_&_Cell_death
    * Transcriptional_program_of_the_cell_cycle:_high-resolution_timing
    * Cell_cycle_and_metabolic_cycle_regulated_transcription_in_yeast
    * Cell_Cycle_Animation 1Lec.com
    * Cell_Cycle
    * Fucci:Using_GFP_to_visualize_the_cell-cycle
    * Science_Creative_Quarterly's_overview_of_the_cell_cycle
    * KEGG_â_Human_Cell_Cycle
    * v
    * t
    * e
Cell cycle proteins
                       * A (A1, A2)
Cyclin                 * B (B1, B2, B3)
                       * D (D1, D2, D3)
                       * E (E1, E2)
                       * 1
                       * 2
                       * 3
                       * 4
                       * 5
CDK                    * 6
                       * 7
                       * 8
                       * 9
                       * 10
                       * CDK-activating_kinase
CDK_inhibitor          * INK4a/ARF (p14arf/p16, p15, p18, p19)
                       * cip/kip (p21, p27, p57)
                       * p53
P53_p63_p73_family     * p63
                       * p73
                       * Cdc2
                       * Cdc25
                       * Cdc42
Other                  * Cellular_apoptosis_susceptibility_protein
                       * E2F
                       * Maturation_promoting_factor
                       * Wee
                       * Cullin (CUL7)
                                              * G1_phase
                   Interphase                 * S_phase
                                              * G2_phase
                                              * Mitosis (Preprophase
                                              * Prophase
                                              * Prometaphase
                   M_phase                    * Metaphase
Phases and                                    * Anaphase
checkpoints                                   * Telophase)
                                              * Cytokinesis
                                              * Restriction_point
                   Cell_cycle_checkpoints     * Spindle_checkpoint
                                              * Postreplication_checkpoint
                                              * Apoptosis
                   Other cellular phases      * G0_phase
                                              * Meiosis
Authority_control [Edit_this_at_Wikidata]     * NDL: 00576703
    * [Tree_of_life.svg]Evolutionary_biology_portal
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal
    * [Nuvola_apps_kalzium.svg]Science_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Cell_cycle&oldid=908082979"
Categories:
    * Cell_cycle
    * Cellular_senescence
Hidden categories:
    * CS1_maint:_Archived_copy_as_title
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_February_2016
    * Commons_category_link_from_Wikidata
    * Wikipedia_articles_incorporating_text_from_the_United_States_National
      Library_of_Medicine
    * Use_dmy_dates_from_July_2011
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * ÐÐ°ÑÒ¡Ð¾ÑÑÑÐ°
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * Brezhoneg
    * CatalÃ 
    * ÄeÅ¡tina
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * KreyÃ²l_ayisyen
    * Latina
    * Magyar
    * Bahasa_Melayu
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Norsk_nynorsk
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * à¹à¸à¸¢
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 27 July 2019, at 09:18 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
