The following text has been accessed from https://en.wikipedia.org/wiki/Protein at Thu Aug 8 23:57:08 IST 2019
Creative_Commons_Attribution-ShareAlike_License



















[This_is_a_good_article._Follow_the_link_for_more_information.]
[Page_semi-protected]
****** Protein ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
This article is about a class of molecules. For protein as a nutrient, see
protein_(nutrient). For other uses, see protein_(disambiguation).
Biological molecule consisting of chains of amino acid residues
A representation of the 3D structure of the protein myoglobin showing turquoise
Î±-helices. This protein was the first to have its structure solved by X-ray
crystallography. Towards the right-center among the coils, a prosthetic_group
called a heme_group (shown in gray) with a bound oxygen molecule (red).
Proteins are large biomolecules, or macromolecules, consisting of one or more
long chains of amino_acid residues. Proteins perform a vast array of functions
within organisms, including catalysing_metabolic_reactions, DNA_replication,
responding_to_stimuli, providing structure_to_cells and organisms, and
transporting_molecules from one location to another. Proteins differ from one
another primarily in their sequence of amino acids, which is dictated by the
nucleotide_sequence of their genes, and which usually results in protein
folding into a specific three-dimensional_structure that determines its
activity.
A linear chain of amino acid residues is called a polypeptide. A protein
contains at least one long polypeptide. Short polypeptides, containing less
than 20â30 residues, are rarely considered to be proteins and are commonly
called peptides, or sometimes oligopeptides. The individual amino acid residues
are bonded together by peptide_bonds and adjacent amino acid residues. The
sequence of amino acid residues in a protein is defined by the sequence of a
gene, which is encoded in the genetic_code. In general, the genetic code
specifies 20 standard amino acids; however, in certain organisms the genetic
code can include selenocysteine andâin certain archaeaâpyrrolysine. Shortly
after or even during synthesis, the residues in a protein are often chemically
modified by post-translational_modification, which alters the physical and
chemical properties, folding, stability, activity, and ultimately, the function
of the proteins. Sometimes proteins have non-peptide groups attached, which can
be called prosthetic_groups or cofactors. Proteins can also work together to
achieve a particular function, and they often associate to form stable protein
complexes.
Once formed, proteins only exist for a certain period and are then degraded and
recycled by the cell's machinery through the process of protein_turnover. A
protein's lifespan is measured in terms of its half-life and covers a wide
range. They can exist for minutes or years with an average lifespan of 1â2
days in mammalian cells. Abnormal or misfolded proteins are degraded more
rapidly either due to being targeted for destruction or due to being unstable.
Like other biological macromolecules such as polysaccharides and nucleic_acids,
proteins are essential parts of organisms and participate in virtually every
process within cells. Many proteins are enzymes that catalyse biochemical
reactions and are vital to metabolism. Proteins also have structural or
mechanical functions, such as actin and myosin in muscle and the proteins in
the cytoskeleton, which form a system of scaffolding that maintains cell shape.
Other proteins are important in cell_signaling, immune_responses, cell
adhesion, and the cell_cycle. In animals, proteins are needed in the diet to
provide the essential_amino_acids that cannot be synthesized. Digestion breaks
the proteins down for use in the metabolism.
Proteins may be purified from other cellular components using a variety of
techniques such as ultracentrifugation, precipitation, electrophoresis, and
chromatography; the advent of genetic_engineering has made possible a number of
methods to facilitate purification. Methods commonly used to study protein
structure and function include immunohistochemistry, site-directed_mutagenesis,
X-ray_crystallography, nuclear_magnetic_resonance and mass_spectrometry.
⁰
***** Contents *****
    * 1_Biochemistry
          o 1.1_Interactions
          o 1.2_Abundance_in_cells
    * 2_Synthesis
          o 2.1_Biosynthesis
          o 2.2_Chemical_synthesis
    * 3_Structure
          o 3.1_Protein_domains
          o 3.2_Sequence_motif
    * 4_Cellular_functions
          o 4.1_Enzymes
          o 4.2_Cell_signaling_and_ligand_binding
          o 4.3_Structural_proteins
    * 5_Methods_of_study
          o 5.1_Protein_purification
          o 5.2_Cellular_localization
          o 5.3_Proteomics
          o 5.4_Bioinformatics
          o 5.5_Structure_determination
          o 5.6_Structure_prediction_and_simulation
                # 5.6.1_Protein_disorder_and_unstructure_prediction
    * 6_Nutrition
    * 7_History_and_etymology
    * 8_See_also
    * 9_References
    * 10_Textbooks
    * 11_External_links
          o 11.1_Databases_and_projects
          o 11.2_Tutorials_and_educational_websites
***** Biochemistry *****
Chemical structure of the peptide bond (bottom) and the three-dimensional
structure of a peptide bond between an alanine and an adjacent amino acid (top/
inset). The bond itself is made of the CHON elements.
Resonance structures of the peptide_bond that links individual amino acids to
form a protein polymer
Main articles: Biochemistry, Amino_acid, and Peptide_bond
Most proteins consist of linear polymers built from series of up to 20
different L-Î±- amino_acids. All proteinogenic_amino_acids possess common
structural features, including an Î±-carbon to which an amino group, a carboxyl
group, and a variable side_chain are bonded. Only proline differs from this
basic structure as it contains an unusual ring to the N-end amine group, which
forces the COâNH amide moiety into a fixed conformation.[1] The side chains
of the standard amino acids, detailed in the list_of_standard_amino_acids, have
a great variety of chemical structures and properties; it is the combined
effect of all of the amino acid side chains in a protein that ultimately
determines its three-dimensional structure and its chemical reactivity.[2] The
amino_acids in a polypeptide chain are linked by peptide_bonds. Once linked in
the protein chain, an individual amino acid is called a residue, and the linked
series of carbon, nitrogen, and oxygen atoms are known as the main chain or
protein backbone.[3]
The peptide bond has two resonance forms that contribute some double-bond
character and inhibit rotation around its axis, so that the alpha carbons are
roughly coplanar. The other two dihedral_angles in the peptide bond determine
the local shape assumed by the protein backbone.[4] The end with a free amino
group is known as the N-terminus or amino terminus, whereas the end of the
protein with a free carboxyl group is known as the C-terminus or carboxy
terminus (the sequence of the protein is written from N-terminus to C-terminus,
from left to right).
The words protein, polypeptide, and peptide are a little ambiguous and can
overlap in meaning. Protein is generally used to refer to the complete
biological molecule in a stable conformation, whereas peptide is generally
reserved for a short amino acid oligomers often lacking a stable three-
dimensional structure. However, the boundary between the two is not well
defined and usually lies near 20â30 residues.[5] Polypeptide can refer to any
single linear chain of amino acids, usually regardless of length, but often
implies an absence of a defined conformation.
**** Interactions ****
Proteins can interact with many types of molecules, including with_other
proteins, with_lipids, with_carboyhydrates, and with_DNA.[6][7][8][9]
**** Abundance in cells ****
It has been estimated that average-sized bacteria contain about 2 million
proteins per cell (e.g. E._coli and Staphylococcus_aureus). Smaller bacteria,
such as Mycoplasma or spirochetes contain fewer molecules, on the order of
50,000 to 1 million. By contrast, eukaryotic cells are larger and thus contain
much more protein. For instance, yeast cells have been estimated to contain
about 50 million proteins and human cells on the order of 1 to 3 billion.[10]
The concentration of individual protein copies ranges from a few molecules per
cell up to 20 million.[11] Not all genes coding proteins are expressed in most
cells and their number depends on, for example, cell type and external stimuli.
For instance, of the 20,000 or so proteins encoded by the human genome, only
6,000 are detected in lymphoblastoid cells.[12] Moreover, the number of
proteins the genome encodes correlates well with the organism complexity.
Eukaryotes have 15,000, bacteria have 3,200, archaea have 2,400, and viruses
have 42 proteins on average coded in their respective genomes.[13]
***** Synthesis *****
**** Biosynthesis ****
A ribosome produces a protein using mRNA as template
The DNA sequence of a gene encodes the amino_acid sequence of a protein
Main article: Protein_biosynthesis
Proteins are assembled from amino acids using information encoded in genes.
Each protein has its own unique amino acid sequence that is specified by the
nucleotide sequence of the gene encoding this protein. The genetic_code is a
set of three-nucleotide sets called codons and each three-nucleotide
combination designates an amino acid, for example AUG (adenine-uracil-guanine)
is the code for methionine. Because DNA contains four nucleotides, the total
number of possible codons is 64; hence, there is some redundancy in the genetic
code, with some amino acids specified by more than one codon.[14] Genes encoded
in DNA are first transcribed into pre-messenger_RNA (mRNA) by proteins such as
RNA_polymerase. Most organisms then process the pre-mRNA (also known as a
primary transcript) using various forms of Post-transcriptional_modification to
form the mature mRNA, which is then used as a template for protein synthesis by
the ribosome. In prokaryotes the mRNA may either be used as soon as it is
produced, or be bound by a ribosome after having moved away from the nucleoid.
In contrast, eukaryotes make mRNA in the cell_nucleus and then translocate it
across the nuclear_membrane into the cytoplasm, where protein_synthesis then
takes place. The rate of protein synthesis is higher in prokaryotes than
eukaryotes and can reach up to 20 amino acids per second.[15]
The process of synthesizing a protein from an mRNA template is known as
translation. The mRNA is loaded onto the ribosome and is read three nucleotides
at a time by matching each codon to its base_pairing anticodon located on a
transfer_RNA molecule, which carries the amino acid corresponding to the codon
it recognizes. The enzyme aminoacyl_tRNA_synthetase "charges" the tRNA
molecules with the correct amino acids. The growing polypeptide is often termed
the nascent chain. Proteins are always biosynthesized from N-terminus to C-
terminus.[14]
The size of a synthesized protein can be measured by the number of amino acids
it contains and by its total molecular_mass, which is normally reported in
units of daltons (synonymous with atomic_mass_units), or the derivative unit
kilodalton (kDa). The average size of a protein increases from Archaea to
Bacteria to Eukaryote (283, 311, 438 residues and 31, 34, 49 kDa respecitvely)
due to a bigger number of protein_domains constituting proteins in higher
organisms.[13] For instance, yeast proteins are on average 466 amino acids long
and 53 kDa in mass.[5] The largest known proteins are the titins, a component
of the muscle sarcomere, with a molecular mass of almost 3,000 kDa and a total
length of almost 27,000 amino acids.[16]
**** Chemical synthesis ****
Main article: Peptide_synthesis
Short proteins can also be synthesized chemically by a family of methods known
as peptide_synthesis, which rely on organic_synthesis techniques such as
chemical_ligation to produce peptides in high yield.[17] Chemical synthesis
allows for the introduction of non-natural amino acids into polypeptide chains,
such as attachment of fluorescent probes to amino acid side chains.[18] These
methods are useful in laboratory biochemistry and cell_biology, though
generally not for commercial applications. Chemical synthesis is inefficient
for polypeptides longer than about 300 amino acids, and the synthesized
proteins may not readily assume their native tertiary_structure. Most chemical
synthesis methods proceed from C-terminus to N-terminus, opposite the
biological reaction.[19]
***** Structure *****
The crystal structure of the chaperonin, a huge protein complex. A single
protein subunit is highlighted. Chaperonins assist protein folding.
Three possible representations of the three-dimensional structure of the
protein triose_phosphate_isomerase. Left: All-atom representation colored by
atom type. Middle: Simplified representation illustrating the backbone
conformation, colored by secondary structure. Right: Solvent-accessible surface
representation colored by residue type (acidic residues red, basic residues
blue, polar residues green, nonpolar residues white).
Main article: Protein_structure
Further information: Protein_structure_prediction
Most proteins fold into unique 3-dimensional structures. The shape into which a
protein naturally folds is known as its native_conformation.[20] Although many
proteins can fold unassisted, simply through the chemical properties of their
amino acids, others require the aid of molecular chaperones to fold into their
native states.[21] Biochemists often refer to four distinct aspects of a
protein's structure:[22]
    * Primary_structure: the amino_acid_sequence. A protein is a polyamide.
    * Secondary_structure: regularly repeating local structures stabilized by
      hydrogen_bonds. The most common examples are the Î±-helix, Î²-sheet and
      turns. Because secondary structures are local, many regions of different
      secondary structure can be present in the same protein molecule.
    * Tertiary_structure: the overall shape of a single protein molecule; the
      spatial relationship of the secondary structures to one another. Tertiary
      structure is generally stabilized by nonlocal interactions, most commonly
      the formation of a hydrophobic_core, but also through salt_bridges,
      hydrogen bonds, disulfide_bonds, and even posttranslational
      modifications. The term "tertiary structure" is often used as synonymous
      with the term fold. The tertiary structure is what controls the basic
      function of the protein.
    * Quaternary_structure: the structure formed by several protein molecules
      (polypeptide chains), usually called protein_subunits in this context,
      which function as a single protein_complex.
Proteins are not entirely rigid molecules. In addition to these levels of
structure, proteins may shift between several related structures while they
perform their functions. In the context of these functional rearrangements,
these tertiary or quaternary structures are usually referred to as
"conformations", and transitions between them are called conformational
changes. Such changes are often induced by the binding of a substrate molecule
to an enzyme's active_site, or the physical region of the protein that
participates in chemical catalysis. In solution proteins also undergo variation
in structure through thermal vibration and the collision with other molecules.
[23]
Molecular surface of several proteins showing their comparative sizes. From
left to right are: immunoglobulin_G (IgG, an antibody), hemoglobin, insulin (a
hormone), adenylate_kinase (an enzyme), and glutamine_synthetase (an enzyme).
Proteins can be informally divided into three main classes, which correlate
with typical tertiary structures: globular_proteins, fibrous_proteins, and
membrane_proteins. Almost all globular proteins are soluble and many are
enzymes. Fibrous proteins are often structural, such as collagen, the major
component of connective tissue, or keratin, the protein component of hair and
nails. Membrane proteins often serve as receptors or provide channels for polar
or charged molecules to pass through the cell_membrane.[24]
A special case of intramolecular hydrogen bonds within proteins, poorly
shielded from water attack and hence promoting their own dehydration, are
called dehydrons.[25]
**** Protein domains ****
Main article: Protein_domain
Many proteins are composed of several protein_domains, i.e. segments of a
protein that fold into distinct structural units. Domains usually also have
specific functions, such as enzymatic activities (e.g. kinase) or they serve as
binding modules (e.g. the SH3_domain binds to proline-rich sequences in other
proteins).
**** Sequence motif ****
Short amino acid sequences within proteins often act as recognition sites for
other proteins.[26] For instance, SH3_domains typically bind to short PxxP
motifs (i.e. 2 prolines [P], separated by 2 unspecified amino_acids [x],
although the surrounding amino acids may determine the exact binding
specificity). A large number of such motifs has been collected in the
Eukaryotic_Linear_Motif (ELM) database.
***** Cellular functions *****
Proteins are the chief actors within the cell, said to be carrying out the
duties specified by the information encoded in genes.[5] With the exception of
certain types of RNA, most other biological molecules are relatively inert
elements upon which proteins act. Proteins make up half the dry weight of an
Escherichia_coli cell, whereas other macromolecules such as DNA and RNA make up
only 3% and 20%, respectively.[27] The set of proteins expressed in a
particular cell or cell type is known as its proteome.
The enzyme hexokinase is shown as a conventional ball-and-stick molecular
model. To scale in the top right-hand corner are two of its substrates, ATP and
glucose.
The chief characteristic of proteins that also allows their diverse set of
functions is their ability to bind other molecules specifically and tightly.
The region of the protein responsible for binding another molecule is known as
the binding_site and is often a depression or "pocket" on the molecular
surface. This binding ability is mediated by the tertiary structure of the
protein, which defines the binding site pocket, and by the chemical properties
of the surrounding amino acids' side chains. Protein binding can be
extraordinarily tight and specific; for example, the ribonuclease_inhibitor
protein binds to human angiogenin with a sub-femtomolar dissociation_constant
(<10â15 M) but does not bind at all to its amphibian homolog onconase (>1 M).
Extremely minor chemical changes such as the addition of a single methyl group
to a binding partner can sometimes suffice to nearly eliminate binding; for
example, the aminoacyl_tRNA_synthetase specific to the amino acid valine
discriminates against the very similar side chain of the amino acid isoleucine.
[28]
Proteins can bind to other proteins as well as to small-molecule substrates.
When proteins bind specifically to other copies of the same molecule, they can
oligomerize to form fibrils; this process occurs often in structural proteins
that consist of globular monomers that self-associate to form rigid fibers.
Proteinâprotein_interactions also regulate enzymatic activity, control
progression through the cell_cycle, and allow the assembly of large protein
complexes that carry out many closely related reactions with a common
biological function. Proteins can also bind to, or even be integrated into,
cell membranes. The ability of binding partners to induce conformational
changes in proteins allows the construction of enormously complex signaling
networks.[29] As interactions between proteins are reversible, and depend
heavily on the availability of different groups of partner proteins to form
aggregates that are capable to carry out discrete sets of function, study of
the interactions between specific proteins is a key to understand important
aspects of cellular function, and ultimately the properties that distinguish
particular cell types.[30][31]
**** Enzymes ****
Main article: Enzyme
The best-known role of proteins in the cell is as enzymes, which catalyse
chemical reactions. Enzymes are usually highly specific and accelerate only one
or a few chemical reactions. Enzymes carry out most of the reactions involved
in metabolism, as well as manipulating DNA in processes such as DNA
replication, DNA_repair, and transcription. Some enzymes act on other proteins
to add or remove chemical groups in a process known as posttranslational
modification. About 4,000 reactions are known to be catalysed by enzymes.[32]
The rate acceleration conferred by enzymatic catalysis is often enormousâas
much as 1017-fold increase in rate over the uncatalysed reaction in the case of
orotate_decarboxylase (78 million years without the enzyme, 18 milliseconds
with the enzyme).[33]
The molecules bound and acted upon by enzymes are called substrates. Although
enzymes can consist of hundreds of amino acids, it is usually only a small
fraction of the residues that come in contact with the substrate, and an even
smaller fractionâthree to four residues on averageâthat are directly
involved in catalysis.[34] The region of the enzyme that binds the substrate
and contains the catalytic residues is known as the active_site.
Dirigent_proteins are members of a class of proteins that dictate the
stereochemistry of a compound synthesized by other enzymes.[35]
**** Cell signaling and ligand binding ****
Ribbon_diagram of a mouse antibody against cholera that binds a carbohydrate
antigen
Many proteins are involved in the process of cell_signaling and signal
transduction. Some proteins, such as insulin, are extracellular proteins that
transmit a signal from the cell in which they were synthesized to other cells
in distant tissues. Others are membrane_proteins that act as receptors whose
main function is to bind a signaling molecule and induce a biochemical response
in the cell. Many receptors have a binding site exposed on the cell surface and
an effector domain within the cell, which may have enzymatic activity or may
undergo a conformational_change detected by other proteins within the cell.[36]
Antibodies are protein components of an adaptive_immune_system whose main
function is to bind antigens, or foreign substances in the body, and target
them for destruction. Antibodies can be secreted into the extracellular
environment or anchored in the membranes of specialized B_cells known as plasma
cells. Whereas enzymes are limited in their binding affinity for their
substrates by the necessity of conducting their reaction, antibodies have no
such constraints. An antibody's binding affinity to its target is
extraordinarily high.[37]
Many ligand transport proteins bind particular small_biomolecules and transport
them to other locations in the body of a multicellular organism. These proteins
must have a high binding affinity when their ligand is present in high
concentrations, but must also release the ligand when it is present at low
concentrations in the target tissues. The canonical example of a ligand-binding
protein is haemoglobin, which transports oxygen from the lungs to other organs
and tissues in all vertebrates and has close homologs in every biological
kingdom.[38] Lectins are sugar-binding proteins which are highly specific for
their sugar moieties. Lectins typically play a role in biological recognition
phenomena involving cells and proteins.[39] Receptors and hormones are highly
specific binding proteins.
Transmembrane_proteins can also serve as ligand transport proteins that alter
the permeability of the cell membrane to small_molecules and ions. The membrane
alone has a hydrophobic core through which polar or charged molecules cannot
diffuse. Membrane proteins contain internal channels that allow such molecules
to enter and exit the cell. Many ion_channel proteins are specialized to select
for only a particular ion; for example, potassium and sodium channels often
discriminate for only one of the two ions.[40]
**** Structural proteins ****
Structural proteins confer stiffness and rigidity to otherwise-fluid biological
components. Most structural proteins are fibrous_proteins; for example,
collagen and elastin are critical components of connective_tissue such as
cartilage, and keratin is found in hard or filamentous structures such as hair,
nails, feathers, hooves, and some animal_shells.[41] Some globular_proteins can
also play structural functions, for example, actin and tubulin are globular and
soluble as monomers, but polymerize to form long, stiff fibers that make up the
cytoskeleton, which allows the cell to maintain its shape and size.
Other proteins that serve structural functions are motor_proteins such as
myosin, kinesin, and dynein, which are capable of generating mechanical forces.
These proteins are crucial for cellular motility of single celled organisms and
the sperm of many multicellular organisms which reproduce sexually. They also
generate the forces exerted by contracting muscles[42] and play essential roles
in intracellular transport.
***** Methods of study *****
Main article: Protein_methods
The activities and structures of proteins may be examined in_vitro, in_vivo,
and in_silico. In vitro studies of purified proteins in controlled environments
are useful for learning how a protein carries out its function: for example,
enzyme_kinetics studies explore the chemical_mechanism of an enzyme's catalytic
activity and its relative affinity for various possible substrate molecules. By
contrast, in vivo experiments can provide information about the physiological
role of a protein in the context of a cell or even a whole organism. In silico
studies use computational methods to study proteins.
**** Protein purification ****
Main article: Protein_purification
To perform in_vitro analysis, a protein must be purified away from other
cellular components. This process usually begins with cell_lysis, in which a
cell's membrane is disrupted and its internal contents released into a solution
known as a crude_lysate. The resulting mixture can be purified using
ultracentrifugation, which fractionates the various cellular components into
fractions containing soluble proteins; membrane lipids and proteins; cellular
organelles, and nucleic_acids. Precipitation by a method known as salting_out
can concentrate the proteins from this lysate. Various types of chromatography
are then used to isolate the protein or proteins of interest based on
properties such as molecular weight, net charge and binding affinity.[43] The
level of purification can be monitored using various types of gel
electrophoresis if the desired protein's molecular weight and isoelectric_point
are known, by spectroscopy if the protein has distinguishable spectroscopic
features, or by enzyme_assays if the protein has enzymatic activity.
Additionally, proteins can be isolated according to their charge using
electrofocusing.[44]
For natural proteins, a series of purification steps may be necessary to obtain
protein sufficiently pure for laboratory applications. To simplify this
process, genetic_engineering is often used to add chemical features to proteins
that make them easier to purify without affecting their structure or activity.
Here, a "tag" consisting of a specific amino acid sequence, often a series of
histidine residues (a "His-tag"), is attached to one terminus of the protein.
As a result, when the lysate is passed over a chromatography column containing
nickel, the histidine residues ligate the nickel and attach to the column while
the untagged components of the lysate pass unimpeded. A number of different
tags have been developed to help researchers purify specific proteins from
complex mixtures.[45]
**** Cellular localization ****
Proteins in different cellular_compartments and structures tagged with green
fluorescent_protein (here, white)
The study of proteins in vivo is often concerned with the synthesis and
localization of the protein within the cell. Although many intracellular
proteins are synthesized in the cytoplasm and membrane-bound or secreted
proteins in the endoplasmic_reticulum, the specifics of how proteins are
targeted to specific organelles or cellular structures is often unclear. A
useful technique for assessing cellular localization uses genetic engineering
to express in a cell a fusion_protein or chimera consisting of the natural
protein of interest linked to a "reporter" such as green_fluorescent_protein
(GFP).[46] The fused protein's position within the cell can be cleanly and
efficiently visualized using microscopy,[47] as shown in the figure opposite.
Other methods for elucidating the cellular location of proteins requires the
use of known compartmental markers for regions such as the ER, the Golgi,
lysosomes or vacuoles, mitochondria, chloroplasts, plasma membrane, etc. With
the use of fluorescently tagged versions of these markers or of antibodies to
known markers, it becomes much simpler to identify the localization of a
protein of interest. For example, indirect_immunofluorescence will allow for
fluorescence colocalization and demonstration of location. Fluorescent dyes are
used to label cellular compartments for a similar purpose.[48]
Other possibilities exist, as well. For example, immunohistochemistry usually
utilizes an antibody to one or more proteins of interest that are conjugated to
enzymes yielding either luminescent or chromogenic signals that can be compared
between samples, allowing for localization information. Another applicable
technique is cofractionation in sucrose (or other material) gradients using
isopycnic_centrifugation.[49] While this technique does not prove
colocalization of a compartment of known density and the protein of interest,
it does increase the likelihood, and is more amenable to large-scale studies.
Finally, the gold-standard method of cellular localization is immunoelectron
microscopy. This technique also uses an antibody to the protein of interest,
along with classical electron microscopy techniques. The sample is prepared for
normal electron microscopic examination, and then treated with an antibody to
the protein of interest that is conjugated to an extremely electro-dense
material, usually gold. This allows for the localization of both
ultrastructural details as well as the protein of interest.[50]
Through another genetic engineering application known as site-directed
mutagenesis, researchers can alter the protein sequence and hence its
structure, cellular localization, and susceptibility to regulation. This
technique even allows the incorporation of unnatural amino acids into proteins,
using modified tRNAs,[51] and may allow the rational design of new proteins
with novel properties.[52]
**** Proteomics ****
Main article: Proteomics
The total complement of proteins present at a time in a cell or cell type is
known as its proteome, and the study of such large-scale data sets defines the
field of proteomics, named by analogy to the related field of genomics. Key
experimental techniques in proteomics include 2D_electrophoresis,[53] which
allows the separation of a large number of proteins, mass_spectrometry,[54]
which allows rapid high-throughput identification of proteins and sequencing of
peptides (most often after in-gel_digestion), protein_microarrays, which allow
the detection of the relative levels of a large number of proteins present in a
cell, and two-hybrid_screening, which allows the systematic exploration of
proteinâprotein_interactions.[55] The total complement of biologically
possible such interactions is known as the interactome.[56] A systematic
attempt to determine the structures of proteins representing every possible
fold is known as structural_genomics.[57]
**** Bioinformatics ****
Main article: Bioinformatics
A vast array of computational methods have been developed to analyze the
structure, function, and evolution of proteins.
The development of such tools has been driven by the large amount of genomic
and proteomic data available for a variety of organisms, including the human
genome. It is simply impossible to study all proteins experimentally, hence
only a few are subjected to laboratory experiments while computational tools
are used to extrapolate to similar proteins. Such homologous_proteins can be
efficiently identified in distantly related organisms by sequence_alignment.
Genome and gene sequences can be searched by a variety of tools for certain
properties. Sequence_profiling_tools can find restriction_enzyme sites, open
reading_frames in nucleotide sequences, and predict secondary_structures.
Phylogenetic_trees can be constructed and evolutionary hypotheses developed
using special software like ClustalW regarding the ancestry of modern organisms
and the genes they express. The field of bioinformatics is now indispensable
for the analysis of genes and proteins.
**** Structure determination ****
Discovering the tertiary structure of a protein, or the quaternary structure of
its complexes, can provide important clues about how the protein performs its
function and how it can be affected, i.e. in drug_design. As proteins are too
small_to_be_seen under a light_microscope, other methods have to be employed to
determine their structure. Common experimental methods include X-ray
crystallography and NMR_spectroscopy, both of which can produce structural
information at atomic resolution. However, NMR experiments are able to provide
information from which a subset of distances between pairs of atoms can be
estimated, and the final possible conformations for a protein are determined by
solving a distance_geometry problem. Dual_polarisation_interferometry is a
quantitative analytical method for measuring the overall protein_conformation
and conformational_changes due to interactions or other stimulus. Circular
dichroism is another laboratory technique for determining internal Î²-sheet /
Î±-helical composition of proteins. Cryoelectron_microscopy is used to produce
lower-resolution structural information about very large protein complexes,
including assembled viruses;[58] a variant known as electron_crystallography
can also produce high-resolution information in some cases, especially for two-
dimensional crystals of membrane proteins.[59] Solved structures are usually
deposited in the Protein_Data_Bank (PDB), a freely available resource from
which structural data about thousands of proteins can be obtained in the form
of Cartesian_coordinates for each atom in the protein.[60]
Many more gene sequences are known than protein structures. Further, the set of
solved structures is biased toward proteins that can be easily subjected to the
conditions required in X-ray_crystallography, one of the major structure
determination methods. In particular, globular proteins are comparatively easy
to crystallize in preparation for X-ray crystallography. Membrane proteins and
large protein complexes, by contrast, are difficult to crystallize and are
underrepresented in the PDB.[61] Structural_genomics initiatives have attempted
to remedy these deficiencies by systematically solving representative
structures of major fold classes. Protein_structure_prediction methods attempt
to provide a means of generating a plausible structure for proteins whose
structures have not been experimentally determined.[62]
**** Structure prediction and simulation ****
Constituent amino-acids can be analyzed to predict secondary, tertiary and
quaternary protein structure, in this case hemoglobin containing heme units
Main articles: Protein_structure_prediction and List_of_protein_structure
prediction_software
Complementary to the field of structural genomics, protein structure prediction
develops efficient mathematical_models of proteins to computationally predict
the molecular formations in theory, instead of detecting structures with
laboratory observation.[63] The most successful type of structure prediction,
known as homology_modeling, relies on the existence of a "template" structure
with sequence similarity to the protein being modeled; structural genomics'
goal is to provide sufficient representation in solved structures to model most
of those that remain.[64] Although producing accurate models remains a
challenge when only distantly related template structures are available, it has
been suggested that sequence_alignment is the bottleneck in this process, as
quite accurate models can be produced if a "perfect" sequence alignment is
known.[65] Many structure prediction methods have served to inform the emerging
field of protein_engineering, in which novel protein folds have already been
designed.[66] A more complex computational problem is the prediction of
intermolecular interactions, such as in molecular_docking and proteinâprotein
interaction_prediction.[67]
Mathematical models to simulate dynamic processes of protein_folding and
binding involve molecular_mechanics, in particular, molecular_dynamics. Monte
Carlo techniques facilitate the computations, which exploit advances in
parallel and distributed_computing (for example, the Folding@home project[68]
which performs molecular_modeling on GPUs). In_silico simulations discovered
the folding of small Î±-helical protein_domains such as the villin headpiece
[69] and the HIV accessory protein.[70] Hybrid methods combining standard
molecular dynamics with quantum_mechanical mathematics explored the electronic
states of rhodopsins.[71]
*** Protein disorder and unstructure prediction ***
Many proteins (in Eucaryota ~33%) contain large unstructured but biologically
functional segments and can be classified as intrinsically_disordered_proteins.
[72] Predicting and analysing protein disorder is, therefore, an important part
of protein structure characterisation.[73]
***** Nutrition *****
Further information: Protein_(nutrient) and Protein_quality
Most microorganisms and plants can biosynthesize all 20 standard amino_acids,
while animals (including humans) must obtain some of the amino acids from the
diet.[27] The amino acids that an organism cannot synthesize on its own are
referred to as essential_amino_acids. Key enzymes that synthesize certain amino
acids are not present in animalsâsuch as aspartokinase, which catalyses the
first step in the synthesis of lysine, methionine, and threonine from
aspartate. If amino acids are present in the environment, microorganisms can
conserve energy by taking up the amino acids from their surroundings and
downregulating their biosynthetic pathways.
In animals, amino acids are obtained through the consumption of foods
containing protein. Ingested proteins are then broken down into amino acids
through digestion, which typically involves denaturation of the protein through
exposure to acid and hydrolysis by enzymes called proteases. Some ingested
amino acids are used for protein biosynthesis, while others are converted to
glucose through gluconeogenesis, or fed into the citric_acid_cycle. This use of
protein as a fuel is particularly important under starvation conditions as it
allows the body's own proteins to be used to support life, particularly those
found in muscle.[74]
In animals such as dogs and cats, protein maintains the health and quality of
the skin by promoting hair follicle growth and keratinization, and thus
reducing the likelihood of skin problems producing malodours.[75] Poor-quality
proteins also have a role regarding gastrointestinal health, increasing the
potential for flatulence and odorous compounds in dogs because when proteins
reach the colon in an undigested state, they are fermented producing hydrogen
sulfide gas, indole, and skatole.[76] Dogs and cats digest animal proteins
better than those from plants but products of low-quality animal origin are
poorly digested, including skin, feathers, and connective tissue.[76]
***** History and etymology *****
Further information: History_of_molecular_biology
Proteins were recognized as a distinct class of biological molecules in the
eighteenth century by Antoine_Fourcroy and others, distinguished by the
molecules' ability to coagulate or flocculate under treatments with heat or
acid.[77] Noted examples at the time included albumin from egg_whites, blood
serum_albumin, fibrin, and wheat gluten.
Proteins were first described by the Dutch chemist Gerardus_Johannes_Mulder and
named by the Swedish chemist JÃ¶ns_Jacob_Berzelius in 1838.[78][79] Mulder
carried out elemental_analysis of common proteins and found that nearly all
proteins had the same empirical_formula, C400H620N100O120P1S1.[80] He came to
the erroneous conclusion that they might be composed of a single type of (very
large) molecule. The term "protein" to describe these molecules was proposed by
Mulder's associate Berzelius; protein is derived from the Greek word
ÏÏÏÏÎµÎ¹Î¿Ï (proteios), meaning "primary",[81] "in the lead", or "standing
in front",[82] + -in. Mulder went on to identify the products of protein
degradation such as the amino_acid leucine for which he found a (nearly
correct) molecular weight of 131 Da.[80] Prior to "protein", other names were
used, like "albumins" or "albuminous materials" (EiweisskÃ¶rper, in German).
[83]
Early nutritional scientists such as the German Carl_von_Voit believed that
protein was the most important nutrient for maintaining the structure of the
body, because it was generally believed that "flesh makes flesh."[84] Karl
Heinrich_Ritthausen extended known protein forms with the identification of
glutamic_acid. At the Connecticut_Agricultural_Experiment_Station a detailed
review of the vegetable proteins was compiled by Thomas_Burr_Osborne. Working
with Lafayette_Mendel and applying Liebig's_law_of_the_minimum in feeding
laboratory_rats, the nutritionally essential_amino_acids were established. The
work was continued and communicated by William_Cumming_Rose. The understanding
of proteins as polypeptides came through the work of Franz_Hofmeister and
Hermann_Emil_Fischer in 1902.[85][86] The central role of proteins as enzymes
in living organisms was not fully appreciated until 1926, when James_B._Sumner
showed that the enzyme urease was in fact a protein.[87]
The difficulty in purifying proteins in large quantities made them very
difficult for early protein biochemists to study. Hence, early studies focused
on proteins that could be purified in large quantities, e.g., those of blood,
egg_white, various toxins, and digestive/metabolic enzymes obtained from
slaughterhouses. In the 1950s, the Armour_Hot_Dog_Co. purified 1 kg of pure
bovine pancreatic ribonuclease_A and made it freely available to scientists;
this gesture helped ribonuclease A become a major target for biochemical study
for the following decades.[80]
John_Kendrew with model of myoglobin in progress
Linus_Pauling is credited with the successful prediction of regular protein
secondary_structures based on hydrogen_bonding, an idea first put forth by
William_Astbury in 1933.[88] Later work by Walter_Kauzmann on denaturation,[89]
[90] based partly on previous studies by Kaj_LinderstrÃ¸m-Lang,[91] contributed
an understanding of protein_folding and structure mediated by hydrophobic
interactions.
The first protein to be sequenced was insulin, by Frederick_Sanger, in 1949.
Sanger correctly determined the amino acid sequence of insulin, thus
conclusively demonstrating that proteins consisted of linear polymers of amino
acids rather than branched chains, colloids, or cyclols.[92] He won the Nobel
Prize for this achievement in 1958.[93]
The first protein_structures to be solved were hemoglobin and myoglobin, by Max
Perutz and Sir_John_Cowdery_Kendrew, respectively, in 1958.[94][95] As of 2017
[update], the Protein_Data_Bank has over 126,060 atomic-resolution structures
of proteins.[96] In more recent times, cryo-electron_microscopy of large
macromolecular_assemblies[97] and computational protein_structure_prediction of
small protein domains[98] are two methods approaching atomic resolution.
***** See also *****
    * [icon]Biology_portal
    * [icon]Biotechnology_portal
    * Molecular_and_cellular_biology_portal
    * [icon]Medicine_portal
    * Chemistry_portal
    * [icon]Food_portal
    * [icon]Sharks_portal
    * [icon]Ecology_portal
    * [icon]Environment_portal
    * [icon]Metabolism_portal
    * [icon]Dentistry_portal
    * Technology_portal
    * [icon]Science_portal
    * [flag]Oregon_portal
    * [icon]Evolutionary_biology_portal
    * [icon]Animals_portal
    * Deproteination
    * DNA-binding_protein
    * Macromolecule
    * Intein
    * List_of_proteins
    * Proteopathy
    * Proteopedia
    * Proteolysis
    * Protein_sequence_space
    * Protein_superfamily
***** References *****
   1. ^Nelson DL, Cox MM (2005). Lehninger's Principles of Biochemistry (4th
      ed.). New York, New York: W. H. Freeman and Company.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Gutteridge A, Thornton JM (November 2005). "Understanding nature's
      catalytic toolkit". Trends in Biochemical Sciences. 30 (11): 622â29.
      doi:10.1016/j.tibs.2005.09.006. PMID 16214343.
   4. ^ Murray et al., p. 19.
   5. ^ Murray et al., p. 31.
   6. ^ a b cLodish H, Berk A, Matsudaira P, Kaiser CA, Krieger M, Scott MP,
      Zipurksy SL, Darnell J (2004). Molecular Cell Biology (5th ed.). New
      York, New York: WH Freeman and Company.
   7. ^Ardejani, Maziar S.; Powers, Evan T.; Kelly, Jeffery W. (2017). "Using
      Cooperatively_Folded_Peptides_To_Measure_Interaction_Energies_and
      Conformational_Propensities". Accounts of Chemical Research. 50 (8):
      1875â82. doi:10.1021/acs.accounts.7b00195. ISSN 0001-4842. PMC 5584629.
      PMID 28723063.
   8. ^Branden C, Tooze J (1999). Introduction to Protein Structure. New York:
      Garland Pub. ISBN 978-0-8153-2305-1.
   9. ^Murray RF, Harper HW, Granner DK, Mayes PA, Rodwell VW (2006). Harper's
      Illustrated Biochemistry. New York: Lange Medical Books/McGraw-Hill.
      ISBN 978-0-07-146197-9.
  10. ^Van Holde KE, Mathews CK (1996). Biochemistry. Menlo Park, California:
      Benjamin/Cummings Pub. Co., Inc. ISBN 978-0-8053-3931-4.
  11. ^Milo R (December 2013). "What_is_the_total_number_of_protein_molecules
      per_cell_volume?_A_call_to_rethink_some_published_values". BioEssays. 35
      (12): 1050â55. doi:10.1002/bies.201300066. PMC 3910158. PMID 24114984.
  12. ^Beck M, Schmidt A, Malmstroem J, Claassen M, Ori A, Szymborska A, Herzog
      F, Rinner O, Ellenberg J, Aebersold R (November 2011). "The_quantitative
      proteome_of_a_human_cell_line". Molecular Systems Biology. 7: 549. doi:
      10.1038/msb.2011.82. PMC 3261713. PMID 22068332.
  13. ^Wu L, Candille SI, Choi Y, Xie D, Jiang L, Li-Pook-Than J, Tang H,
      Snyder M (July 2013). "Variation_and_genetic_control_of_protein_abundance
      in_humans". Nature. 499 (7456): 79â82. Bibcode:2013Natur.499...79W.
      doi:10.1038/nature12223. PMC 3789121. PMID 23676674.
  14. ^ a bKozlowski LP (January 2017). "Proteome-pI:_proteome_isoelectric
      point_database". Nucleic Acids Research. 45 (D1): D1112âD1116. doi:
      10.1093/nar/gkw978. PMC 5210655. PMID 27789699.
  15. ^ a b van Holde and Mathews, pp. 1002â42.
  16. ^Dobson CM (2000). "The nature and significance of protein folding". In
      Pain RH (ed.). Mechanisms of Protein Folding. Oxford, Oxfordshire: Oxford
      University Press. pp. 1â28. ISBN 978-0-19-963789-8.
  17. ^Fulton AB, Isaacs WB (April 1991). "Titin, a huge, elastic sarcomeric
      protein with a probable role in morphogenesis". BioEssays. 13 (4):
      157â61. doi:10.1002/bies.950130403. PMID 1859393.
  18. ^Bruckdorfer T, Marder O, Albericio F (February 2004). "From production
      of peptides in milligram amounts for research to multi-tons quantities
      for drugs of the future". Current Pharmaceutical Biotechnology. 5 (1):
      29â43. doi:10.2174/1389201043489620. PMID 14965208.
  19. ^Schwarzer D, Cole PA (December 2005). "Protein semisynthesis and
      expressed protein ligation: chasing a protein's tail". Current Opinion in
      Chemical Biology. 9 (6): 561â69. doi:10.1016/j.cbpa.2005.09.018.
      PMID 16226484.
  20. ^Kent SB (February 2009). "Total chemical synthesis of proteins".
      Chemical Society Reviews. 38 (2): 338â51. doi:10.1039/b700141j.
      PMID 19169452.
  21. ^ Murray et al., p. 36.
  22. ^ Murray et al., p. 37.
  23. ^ Murray et al., pp. 30â34.
  24. ^ van Holde and Mathews, pp. 368â75.
  25. ^ van Holde and Mathews, pp. 165â85.
  26. ^FernÃ¡ndez A, Scott R (September 2003). "Dehydron:_a_structurally
      encoded_signal_for_protein_interaction". Biophysical Journal. 85 (3):
      1914â28. Bibcode:2003BpJ....85.1914F. doi:10.1016/S0006-3495(03)74619-
      0. PMC 1303363. PMID 12944304.
  27. ^Davey NE, Van Roey K, Weatheritt RJ, Toedt G, Uyar B, Altenberg B, Budd
      A, Diella F, Dinkel H, Gibson TJ (January 2012). "Attributes of short
      linear motifs". Molecular BioSystems. 8 (1): 268â81. doi:10.1039/
      c1mb05231d. PMID 21909575.
  28. ^ a b Voet D, Voet JG. (2004). Biochemistry Vol 1 3rd ed. Wiley: Hoboken,
      NJ.
  29. ^Sankaranarayanan R, Moras D (2001). "The fidelity of the translation of
      the genetic code". Acta Biochimica Polonica. 48 (2): 323â35.
      PMID 11732604.
  30. ^ van Holde and Mathews, pp. 830â49.
  31. ^Copland JA, Sheffield-Moore M, Koldzic-Zivanovic N, Gentry S, Lamprou G,
      Tzortzatou-Stathopoulou F, Zoumpourlis V, Urban RJ, Vlahopoulos SA (June
      2009). "Sex steroid receptors in skeletal differentiation and epithelial
      neoplasia: is tissue-specific intervention possible?". BioEssays. 31 (6):
      629â41. doi:10.1002/bies.200800138. PMID 19382224.
  32. ^Samarin S, Nusrat A (January 2009). "Regulation of epithelial apical
      junctional complex by Rho family GTPases". Frontiers in Bioscience. 14
      (14): 1129â42. doi:10.2741/3298. PMID 19273120.
  33. ^Bairoch A (January 2000). "The_ENZYME_database_in_2000" (PDF). Nucleic
      Acids Research. 28 (1): 304â05. doi:10.1093/nar/28.1.304. PMC 102465.
      PMID 10592255. Archived from the_original (PDF) on June 1, 2011.
  34. ^Radzicka A, Wolfenden R (January 1995). "A proficient enzyme". Science.
      267 (5194): 90â3. Bibcode:1995Sci...267...90R. doi:10.1126/
      science.7809611. PMID 7809611.
  35. ^EBI External Services (2010-01-20). "The_Catalytic_Site_Atlas_at_The
      European_Bioinformatics_Institute". Ebi.ac.uk. Retrieved 2011-01-16.
  36. ^Pickel B, Schaller A (October 2013). "Dirigent proteins: molecular
      characteristics and potential biotechnological applications". Applied
      Microbiology and Biotechnology. 97 (19): 8427â38. doi:10.1007/s00253-
      013-5167-4. PMID 23989917.
  37. ^ Branden and Tooze, pp. 251â81.
  38. ^ van Holde and Mathews, pp. 247â50.
  39. ^ van Holde and Mathews, pp. 220â29.
  40. ^RÃ¼diger H, Siebert HC, SolÃ­s D, JimÃ©nez-Barbero J, Romero A, von der
      Lieth CW, Diaz-MariÃ±o T, Gabius HJ (April 2000). "Medicinal chemistry
      based on the sugar code: fundamentals of lectinology and experimental
      strategies with lectins as targets". Current Medicinal Chemistry. 7 (4):
      389â416. doi:10.2174/0929867003375164. PMID 10702616.
  41. ^ Branden and Tooze, pp. 232â34.
  42. ^ van Holde and Mathews, pp. 178â81.
  43. ^ van Holde and Mathews, pp. 258â64, 272.
  44. ^ Murray et al., pp. 21â24.
  45. ^Hey J, Posch A, Cohen A, Liu N, Harbers A (2008). Fractionation of
      complex protein mixtures by liquid-phase isoelectric focusing. Methods in
      Molecular Biology. Methods in Molecular Biologyâ¢. 424. pp. 225â39.
      doi:10.1007/978-1-60327-064-9_19. ISBN 978-1-58829-722-8. PMID 18369866.
  46. ^Terpe K (January 2003). "Overview of tag protein fusions: from molecular
      and biochemical fundamentals to commercial systems". Applied Microbiology
      and Biotechnology. 60 (5): 523â33. doi:10.1007/s00253-002-1158-6.
      PMID 12536251.
  47. ^Stepanenko OV, Verkhusha VV, Kuznetsova IM, Uversky VN, Turoverov KK
      (August 2008). "Fluorescent_proteins_as_biomarkers_and_biosensors:
      throwing_color_lights_on_molecular_and_cellular_processes". Current
      Protein & Peptide Science. 9 (4): 338â69. doi:10.2174/
      138920308785132668. PMC 2904242. PMID 18691124.
  48. ^Yuste R (December 2005). "Fluorescence microscopy today". Nature
      Methods. 2 (12): 902â4. doi:10.1038/nmeth1205-902. PMID 16299474.
  49. ^Margolin W (January 2000). "Green fluorescent protein as a reporter for
      macromolecular localization in bacterial cells". Methods. 20 (1):
      62â72. doi:10.1006/meth.1999.0906. PMID 10610805.
  50. ^Walker JH, Wilson K (2000). Principles and Techniques of Practical
      Biochemistry. Cambridge, UK: Cambridge University Press. pp. 287â89.
      ISBN 978-0-521-65873-7.
  51. ^Mayhew TM, Lucocq JM (August 2008). "Developments_in_cell_biology_for
      quantitative_immunoelectron_microscopy_based_on_thin_sections:_a_review".
      Histochemistry and Cell Biology. 130 (2): 299â313. doi:10.1007/s00418-
      008-0451-6. PMC 2491712. PMID 18553098.
  52. ^Hohsaka T, Sisido M (December 2002). "Incorporation of non-natural amino
      acids into proteins". Current Opinion in Chemical Biology. 6 (6):
      809â15. doi:10.1016/S1367-5931(02)00376-9. PMID 12470735.
  53. ^Cedrone F, MÃ©nez A, QuÃ©mÃ©neur E (August 2000). "Tailoring new enzyme
      functions by rational redesign". Current Opinion in Structural Biology.
      10 (4): 405â10. doi:10.1016/S0959-440X(00)00106-8. PMID 10981626.
  54. ^GÃ¶rg A, Weiss W, Dunn MJ (December 2004). "Current two-dimensional
      electrophoresis technology for proteomics". Proteomics. 4 (12):
      3665â85. doi:10.1002/pmic.200401031. PMID 15543535.
  55. ^Conrotto P, Souchelnytskyi S (September 2008). "Proteomic approaches in
      biological and medical sciences: principles and applications".
      Experimental Oncology. 30 (3): 171â80. PMID 18806738.
  56. ^Koegl M, Uetz P (December 2007). "Improving_yeast_two-hybrid_screening
      systems". Briefings in Functional Genomics & Proteomics. 6 (4): 302â12.
      doi:10.1093/bfgp/elm035. PMID 18218650.
  57. ^PlewczyÅski D, Ginalski K (2009). "The_interactome:_predicting_the
      protein-protein_interactions_in_cells". Cellular & Molecular Biology
      Letters. 14 (1): 1â22. doi:10.2478/s11658-008-0024-7. PMC 6275871.
      PMID 18839074.
  58. ^Zhang C, Kim SH (February 2003). "Overview_of_structural_genomics:_from
      structure_to_function". Current Opinion in Chemical Biology. 7 (1):
      28â32. doi:10.1016/S1367-5931(02)00015-7. PMID 12547423.
  59. ^ Branden and Tooze, pp. 340â41.
  60. ^Gonen T, Cheng Y, Sliz P, Hiroaki Y, Fujiyoshi Y, Harrison SC, Walz T
      (December 2005). "Lipid-protein_interactions_in_double-layered_two-
      dimensional_AQP0_crystals". Nature. 438 (7068): 633â38. Bibcode:
      2005Natur.438..633G. doi:10.1038/nature04321. PMC 1350984. PMID 16319884.
  61. ^Standley DM, Kinjo AR, Kinoshita K, Nakamura H (July 2008). "Protein
      structure_databases_with_new_web_services_for_structural_biology_and
      biomedical_research". Briefings in Bioinformatics. 9 (4): 276â85. doi:
      10.1093/bib/bbn015. PMID 18430752.
  62. ^Walian P, Cross TA, Jap BK (2004). "Structural_genomics_of_membrane
      proteins". Genome Biology. 5 (4): 215. doi:10.1186/gb-2004-5-4-215.
      PMC 395774. PMID 15059248.
  63. ^Sleator RD (2012). "Prediction of protein functions". Functional
      Genomics. Methods in Molecular Biology. 815. pp. 15â24. doi:10.1007/
      978-1-61779-424-7_2. ISBN 978-1-61779-423-0. PMID 22130980.
  64. ^Zhang Y (June 2008). "Progress_and_challenges_in_protein_structure
      prediction". Current Opinion in Structural Biology. 18 (3): 342â48.
      doi:10.1016/j.sbi.2008.02.004. PMC 2680823. PMID 18436442.
  65. ^Xiang Z (June 2006). "Advances_in_homology_protein_structure_modeling".
      Current Protein & Peptide Science. 7 (3): 217â27. doi:10.2174/
      138920306777452312. PMC 1839925. PMID 16787261.
  66. ^Zhang Y, Skolnick J (January 2005). "The_protein_structure_prediction
      problem_could_be_solved_using_the_current_PDB_library". Proceedings of
      the National Academy of Sciences of the United States of America. 102
      (4): 1029â34. Bibcode:2005PNAS..102.1029Z. doi:10.1073/pnas.0407152101.
      PMC 545829. PMID 15653774.
  67. ^Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, Baker D (November
      2003). "Design of a novel globular protein fold with atomic-level
      accuracy". Science. 302 (5649): 1364â68. Bibcode:2003Sci...302.1364K.
      doi:10.1126/science.1089427. PMID 14631033.
  68. ^Ritchie DW (February 2008). "Recent progress and future directions in
      protein-protein docking". Current Protein & Peptide Science. 9 (1):
      1â15. CiteSeerX 10.1.1.211.4946. doi:10.2174/138920308783565741.
      PMID 18336319.
  69. ^Scheraga HA, Khalili M, Liwo A (2007). "Protein-folding dynamics:
      overview of molecular simulation techniques". Annual Review of Physical
      Chemistry. 58: 57â83. Bibcode:2007ARPC...58...57S. doi:10.1146/
      annurev.physchem.58.032806.104614. PMID 17034338.
  70. ^Zagrovic B, Snow CD, Shirts MR, Pande VS (November 2002). "Simulation of
      folding of a small alpha-helical protein in atomistic detail using
      worldwide-distributed computing". Journal of Molecular Biology. 323 (5):
      927â37. CiteSeerX 10.1.1.142.8664. doi:10.1016/S0022-2836(02)00997-X.
      PMID 12417204.
  71. ^Herges T, Wenzel W (January 2005). "In silico folding of a three helix
      protein and characterization of its free-energy landscape in an all-atom
      force field". Physical Review Letters. 94 (1): 018101. arXiv:physics/
      0310146. Bibcode:2005PhRvL..94a8101H. doi:10.1103/PhysRevLett.94.018101.
      PMID 15698135.
  72. ^Hoffmann M, Wanko M, Strodel P, KÃ¶nig PH, Frauenheim T, Schulten K,
      Thiel W, Tajkhorshid E, Elstner M (August 2006). "Color tuning in
      rhodopsins: the mechanism for the spectral shift between
      bacteriorhodopsin and sensory rhodopsin II". Journal of the American
      Chemical Society. 128 (33): 10808â18. doi:10.1021/ja062082i.
      PMID 16910676.
  73. ^Ward JJ, Sodhi JS, McGuffin LJ, Buxton BF, Jones DT (March 2004).
      "Prediction and functional analysis of native disorder in proteins from
      the three kingdoms of life". Journal of Molecular Biology. 337 (3):
      635â45. CiteSeerX 10.1.1.120.5605. doi:10.1016/j.jmb.2004.02.002.
      PMID 15019783.
  74. ^Tompa P, Fersht A (18 November 2009). Structure_and_Function_of
      Intrinsically_Disordered_Proteins. CRC Press. ISBN 978-1-4200-7893-0.
  75. ^Brosnan JT (June 2003). "Interorgan_amino_acid_transport_and_its
      regulation". The Journal of Nutrition. 133 (6 Suppl 1): 2068Sâ72S. doi:
      10.1093/jn/133.6.2068S. PMID 12771367.
  76. ^Watson TD (1998). "Diet_and_skin_disease_in_dogs_and_cats". The Journal
      of Nutrition. 128 (12 Suppl): 2783Sâ89S. doi:10.1093/jn/128.12.2783S.
      PMID 9868266.
  77. ^ a bCase LP, Daristotle L, Hayek MG, Raasch MF (2010). Canine and Feline
      Nutrition-E-Book: A Resource for Companion Animal Professionals. Elsevier
      Health Sciences.
  78. ^ Thomas_Burr_Osborne (1909): The_Vegetable_Proteins, History pp 1 to 6,
      from archive.org
  79. ^Mulder GJ (1838). "Sur_la_composition_de_quelques_substances_animales".
      Bulletin des Sciences Physiques et Naturelles en NÃ©erlande: 104.
  80. ^Harold H (1951). "Origin of the Word 'Protein.'". Nature. 168 (4267):
      244. Bibcode:1951Natur.168..244H. doi:10.1038/168244a0. PMID 14875059.
  81. ^ a b cPerrett D (August 2007). "From 'protein' to the beginnings of
      clinical proteomics". Proteomics: Clinical Applications. 1 (8): 720â38.
      doi:10.1002/prca.200700525. PMID 21136729.
  82. ^ New Oxford Dictionary of English
  83. ^Reynolds JA, Tanford C (2003). Nature's Robots: A History of Proteins
      (Oxford Paperbacks). New York, New York: Oxford University Press. p. 15.
      ISBN 978-0-19-860694-9.
  84. ^ Reynolds and Tanford (2003).
  85. ^Bischoff TL, Voit C (1860). Die Gesetze der Ernaehrung des
      Pflanzenfressers durch neue Untersuchungen festgestellt (in German).
      Leipzig, Heidelberg.
  86. ^"Hofmeister,_Franz". encyclopedia.com. Retrieved 4 April 2017.
  87. ^"Protein,_section:_Classification_of_protein". britannica.com. Retrieved
      4 April 2017.
  88. ^Sumner JB (1926). "The_isolation_and_crystallization_of_the_enzyme
      urease._Preliminary_paper" (PDF). Journal of Biological Chemistry. 69
      (2): 435â41.
  89. ^Pauling L, Corey RB (May 1951). "Atomic_coordinates_and_structure
      factors_for_two_helical_configurations_of_polypeptide_chains" (PDF).
      Proceedings of the National Academy of Sciences of the United States of
      America. 37 (5): 235â40. Bibcode:1951PNAS...37..235P. doi:10.1073/
      pnas.37.5.235. PMC 1063348. PMID 14834145.
  90. ^Kauzmann W (May 1956). "Structural factors in protein denaturation".
      Journal of Cellular Physiology. 47 (Suppl 1): 113â31. doi:10.1002/
      jcp.1030470410. PMID 13332017.
  91. ^Kauzmann W (1959). "Some factors in the interpretation of protein
      denaturation". Advances in Protein Chemistry Volume 14. Advances in
      Protein Chemistry. 14. pp. 1â63. doi:10.1016/S0065-3233(08)60608-7.
      ISBN 978-0-12-034214-3. PMID 14404936.
  92. ^Kalman SM, LinderstrÃ¸m-Lang K, Ottesen M, Richards FM (February 1955).
      "Degradation of ribonuclease by subtilisin". Biochimica et Biophysica
      Acta. 16 (2): 297â99. doi:10.1016/0006-3002(55)90224-9. PMID 14363272.
  93. ^Sanger F (1949). "The_terminal_peptides_of_insulin". The Biochemical
      Journal. 45 (5): 563â74. doi:10.1042/bj0450563. PMC 1275055.
      PMID 15396627.
  94. ^Sanger F. (1958), Nobel_lecture:_The_chemistry_of_insulin (PDF),
      Nobelprize.org
  95. ^Muirhead H, Perutz MF (August 1963). "Structure of hemoglobin. A three-
      dimensional fourier synthesis of reduced human hemoglobin at 5.5 Ã
      resolution". Nature. 199 (4894): 633â38. Bibcode:1963Natur.199..633M.
      doi:10.1038/199633a0. PMID 14074546.
  96. ^Kendrew JC, Bodo G, Dintzis HM, Parrish RG, Wyckoff H, Phillips DC
      (March 1958). "A three-dimensional model of the myoglobin molecule
      obtained by x-ray analysis". Nature. 181 (4610): 662â66. Bibcode:
      1958Natur.181..662K. doi:10.1038/181662a0. PMID 13517261.
  97. ^"RCSB_Protein_Data_Bank". Archived from the_original on 2015-04-18.
      Retrieved 2017-01-19.
  98. ^Zhou ZH (April 2008). "Towards_atomic_resolution_structural
      determination_by_single-particle_cryo-electron_microscopy". Current
      Opinion in Structural Biology. 18 (2): 218â28. doi:10.1016/
      j.sbi.2008.03.004. PMC 2714865. PMID 18403197.
  99. ^Keskin O, Tuncbag N, Gursoy A (April 2008). "Characterization and
      prediction of protein interfaces to infer protein-protein interaction
      networks". Current Pharmaceutical Biotechnology. 9 (2): 67â76. doi:
      10.2174/138920108783955191. PMID 18393863.
***** Textbooks *****
    * Branden C, Tooze J (1999). Introduction to Protein Structure. New York:
      Garland Pub. ISBN 978-0-8153-2305-1.
Murray RF, Harper HW, Granner DK, Mayes PA, Rodwell VW (2006). Harper's
Illustrated Biochemistry. New York: Lange Medical Books/McGraw-Hill. ISBN 978-
0-07-146197-9.
Van Holde KE, Mathews CK (1996). Biochemistry. Menlo Park, California:
Benjamin/Cummings Pub. Co., Inc. ISBN 978-0-8053-3931-4.
***** External links *****
 Wikimedia Commons has media related to Proteins.
 Look up protein in Wiktionary, the free dictionary.
**** Databases and projects ****
    * NCBI_Entrez_Protein_database
    * NCBI_Protein_Structure_database
    * Human_Protein_Reference_Database
    * Human_Proteinpedia
    * Folding@Home_(Stanford_University)
    * Protein_Databank_in_Europe (see also PDBeQuips, short articles and
      tutorials on interesting PDB structures)
    * Research_Collaboratory_for_Structural_Bioinformatics (see also Molecule
      of_the_Month, presenting short accounts on selected proteins from the
      PDB)
    * Proteopedia â_Life_in_3D: rotatable, zoomable 3D model with wiki
      annotations for every known protein molecular structure.
    * UniProt_the_Universal_Protein_Resource
**** Tutorials and educational websites ****
    * "An_Introduction_to_Proteins" from HOPES (Huntington's Disease Outreach
      Project for Education at Stanford)
    * Proteins:_Biogenesis_to_Degradation â_The_Virtual_Library_of
      Biochemistry_and_Cell_Biology
    * Protein_at_britannica.com
    * v
    * t
    * e
Gene_expression
                       * Genetic_code
                       * Central_dogma
Introduction                 o DNA â RNA â Protein
to_genetics            * Special_transfers
                             o RNAâRNA
                             o RNAâDNA
                             o ProteinâProtein
                   Types                  * Bacterial
                                          * Eukaryotic
                                          * Transcription_factor
                   Key elements           * RNA_polymerase
                                          * Promoter
Transcription                             * Precursor_mRNA_(pre-mRNA /_hnRNA)
                                          * 5'_capping
                   Post-transcription     * Splicing
                                          * Polyadenylation
                                          * Histone_acetylation_and
                                            deacetylation
                   Types            * Prokaryotic
                                    * Eukaryotic
Translation                         * Ribosome
                   Key elements     * Transfer_RNA_(tRNA)
                                    * Ribosome-nascent_chain_complex_(RNC)
                                    * Post-translational_modification
                       * Epigenetic
                             o imprinting
                       * Transcriptional
                             o Gene_regulatory_network
                             o cis-regulatory_element
                       * lac_operon
Regulation             * Post-transcriptional
                             o sequestration_(P-bodies)
                             o alternative_splicing
                             o microRNA
                       * Translational
                       * Post-translational
                             o reversible
                             o irreversible
Influential people     * FranÃ§ois_Jacob
                       * Jacques_Monod
    * [Portal] Portal
    * v
    * t
    * e
Proteins
               * Protein_biosynthesis
               * Post-translational_modification
Processes      * Protein_folding
               * Protein_targeting
               * Proteome
               * Protein_methods
               * Protein_structure
Structures     * Protein_structural_domains
               * Proteasome
               * List_of_types_of_proteins
               * List_of_proteins
               * Membrane_protein
Types          * Globular_protein
                     o Globulin
                     o Edestin
                     o Albumin
               * Fibrous_protein
    * v
    * t
    * e
Proteins: key methods of study
                                * Protein_purification
                                * Green_fluorescent_protein
                                * Western_blot
                                * Protein_immunostaining
                                * Protein_sequencing
                                * Gel_electrophoresis/Protein_electrophoresis
                                * Protein_immunoprecipitation
                                * Peptide_mass_fingerprinting/Protein_mass
Experimental                      spectrometry
                                * Dual-polarization_interferometry
                                * Microscale_thermophoresis
                                * Chromatin_immunoprecipitation
                                * Surface_plasmon_resonance
                                * Isothermal_titration_calorimetry
                                * X-ray_crystallography
                                * Protein_NMR
                                * Cryo-electron_microscopy
                                * Freeze-fracture_electron_microscopy
                                * Protein_structure_prediction
                                * Proteinâprotein_docking
Bioinformatics                  * Protein_structural_alignment
                                * Protein_ontology
                                * Proteinâprotein_interaction_prediction
                                * Enzyme_assay
Assay                           * Protein_assay
                                * Secretion_assay
                                * Bacterial_display
                                * mRNA_display
Display techniques              * Phage_display
                                * Ribosome_display
                                * Yeast_display
Super-resolution_microscopy     * Photoactivated_localization_microscopy
                                * Vertico_SMI
    * v
    * t
    * e
Proteins: key methods of study
                                * Protein_purification
                                * Green_fluorescent_protein
                                * Western_blot
                                * Protein_immunostaining
                                * Protein_sequencing
                                * Gel_electrophoresis/Protein_electrophoresis
                                * Protein_immunoprecipitation
                                * Peptide_mass_fingerprinting/Protein_mass
Experimental                      spectrometry
                                * Dual-polarization_interferometry
                                * Microscale_thermophoresis
                                * Chromatin_immunoprecipitation
                                * Surface_plasmon_resonance
                                * Isothermal_titration_calorimetry
                                * X-ray_crystallography
                                * Protein_NMR
                                * Cryo-electron_microscopy
                                * Freeze-fracture_electron_microscopy
                                * Protein_structure_prediction
                                * Proteinâprotein_docking
Bioinformatics                  * Protein_structural_alignment
                                * Protein_ontology
                                * Proteinâprotein_interaction_prediction
                                * Enzyme_assay
Assay                           * Protein_assay
                                * Secretion_assay
                                * Bacterial_display
                                * mRNA_display
Display techniques              * Phage_display
                                * Ribosome_display
                                * Yeast_display
Super-resolution_microscopy     * Photoactivated_localization_microscopy
                                * Vertico_SMI
    * v
    * t
    * e
Food_chemistry
    * Additives
    * Carbohydrates
    * Coloring
    * Enzymes
    * Essential_fatty_acids
    * Flavors
    * Fortification
    * Lipids
    * "Minerals"_(Chemical_elements)
    * Proteins
    * Vitamins
    * Water
    * v
    * t
    * e
Metabolism, catabolism, anabolism
               * Metabolic_pathway
General        * Metabolic_network
               * Primary_nutritional_groups
           Aerobic          * Glycolysis â Pyruvate_decarboxylation â Citric_acid_cycle â Oxidative
           respiration        phosphorylation (electron_transport_chain + ATP_synthase)
           Anaerobic        * Electron acceptors are other than oxygen
Energy     respiration
metabolism                  * Glycolysis â Substrate-level_phosphorylation
           Fermentation           o ABE
                                  o Ethanol
                                  o Lactic_acid
           Protein_metabolism           * Protein_synthesis
                                        * Catabolism
                                                 * Glycolysis â Gluconeogenesis
                                                 * Glycogenolysis â Glycogenesis
                                                 * Pentose_phosphate_pathway
                                    Human        * Fructolysis
                                                 * Galactolysis
           Carbohydrate_metabolism               * Glycosylation
           (carbohydrate_catabolism                    o N-linked
           and anabolism)                              o O-linked
                                                 * Photosynthesis
                                                 * Anoxygenic_photosynthesis
                                    Nonhuman     * Chemosynthesis
                                                 * Carbon_fixation
Specific                                         * Xylose_metabolism
paths                                            * Radiotrophism
                                    Fatty_acid_metabolism     * Fatty_acid_degradation (Beta_oxidation)
                                                              * Fatty_acid_synthesis
           Lipid_metabolism                                   * Steroid_metabolism
           (lipolysis, lipogenesis)                           * Sphingolipid_metabolism
                                    Other                     * Eicosanoid_metabolism
                                                              * Ketosis
                                                              * Reverse_cholesterol_transport
           Amino_acid                   * Amino_acid_synthesis
                                        * Urea_cycle
           Nucleotide                   * Purine_metabolism
           metabolism                   * Nucleotide_salvage
                                        * Pyrimidine_metabolism
                                        * Metal_metabolism
           Other                              o Iron_metabolism
                                        * Ethanol_metabolism
                                              * BNF: cb11936447p (data)
Authority_control [Edit_this_at_Wikidata]     * GND: 4076388-2
                                              * LCCN: sh85107666
                                              * NDL: 00572676

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Protein&oldid=909636077"
Categories:
    * Molecular_biology
    * Proteins
    * Proteomics
Hidden categories:
    * CS1_German-language_sources_(de)
    * Wikipedia_indefinitely_semi-protected_pages
    * Articles_with_short_description
    * Articles_containing_potentially_dated_statements_from_2017
    * All_articles_containing_potentially_dated_statements
    * Commons_category_link_is_on_Wikidata
    * Good_articles
    * Wikipedia_articles_with_BNF_identifiers
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_LCCN_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * View_source
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Afrikaans
    * Alemannisch
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * AragonÃ©s
    * à¦à¦¸à¦®à§à¦¯à¦¼à¦¾
    * Asturianu
    * AzÉrbaycanca
    * ØªÛØ±Ú©Ø¬Ù
    * à¦¬à¦¾à¦à¦²à¦¾
    * Bahasa_Banjar
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÐ°ÑÒ¡Ð¾ÑÑÑÐ°
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ_(ÑÐ°ÑÐ°ÑÐºÐµÐ²ÑÑÐ°)â
    * à¤­à¥à¤à¤ªà¥à¤°à¥
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * Brezhoneg
    * ÐÑÑÑÐ°Ð´
    * CatalÃ 
    * Ð§ÓÐ²Ð°ÑÐ»Ð°
    * ÄeÅ¡tina
    * Cymraeg
    * Dansk
    * Deutsch
    * ÞÞ¨ÞÞ¬ÞÞ¨ÞÞ¦ÞÞ°
    * à¤¡à¥à¤à¥à¤²à¥
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * Ð­ÑÐ·ÑÐ½Ñ
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * Fiji_Hindi
    * FÃ¸royskt
    * FranÃ§ais
    * Frysk
    * Gaeilge
    * Gaelg
    * Galego
    * å®¢å®¶èª/Hak-kÃ¢-ngÃ®
    * Ð¥Ð°Ð»ÑÐ¼Ð³
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Ido
    * Ilokano
    * Bahasa_Indonesia
    * Interlingua
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * à²à²¨à³à²¨à²¡
    * Kapampangan
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * Kiswahili
    * KreyÃ²l_ayisyen
    * KurdÃ®
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * àº¥àº²àº§
    * Latina
    * LatvieÅ¡u
    * LÃ«tzebuergesch
    * ÐÐµÐ·Ð³Ð¸
    * LietuviÅ³
    * Limburgs
    * Lumbaart
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * à¤®à¤°à¤¾à¤ à¥
    * ááá áááá£á á
    * ÙØµØ±Ù
    * Bahasa_Melayu
    * MÃ¬ng-dÄÌ¤ng-ngá¹³Ì
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * áá¼ááºáá¬áá¬áá¬
    * Nederlands
    * à¤¨à¥à¤ªà¤¾à¤²à¥
    * à¤¨à¥à¤ªà¤¾à¤²_à¤­à¤¾à¤·à¤¾
    * æ¥æ¬èª
    * Nordfriisk
    * Norsk
    * Norsk_nynorsk
    * Novial
    * Occitan
    * à¬à¬¡à¬¼à¬¿à¬
    * Oromoo
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * à¨ªà©°à¨à¨¾à¨¬à©
    * Ù¾ÙØ¬Ø§Ø¨Û
    * PlattdÃ¼Ã¼tsch
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Runa_Simi
    * Ð ÑÑÐ¸Ð½ÑÑÐºÑÐ¹
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡Ð°ÑÐ°_ÑÑÐ»Ð°
    * Scots
    * Seeltersk
    * Shqip
    * Sicilianu
    * à·à·à¶à·à¶½
    * Simple_English
    * Ø³ÙÚÙ
    * SlovenÄina
    * SlovenÅ¡Äina
    * Soomaaliga
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Basa_Sunda
    * Suomi
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * Ð¢Ð°ÑÐ°ÑÑÐ°/tatarÃ§a
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * Ð¢Ð¾Ò·Ð¸ÐºÓ£
    * á£á³á©
    * TÃ¼rkÃ§e
    * Ð¢ÑÐ²Ð°_Ð´ÑÐ»
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * VÃ¨neto
    * Tiáº¿ng_Viá»t
    * Walon
    * æè¨
    * West-Vlams
    * Winaray
    * å´è¯­
    * ××Ö´×××©
    * ç²µèª
    * Å½emaitÄÅ¡ka
    * ä¸­æ
Edit_links
    * This page was last edited on 6 August 2019, at 17:03 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
