The following text has been accessed from https://en.wikipedia.org/wiki/Bioinformatics at Fri Aug 9 02:18:17 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[Listen_to_this_article]
****** Bioinformatics ******
From Wikipedia, the free encyclopedia
This is the latest_accepted_revision, reviewed on 22 July 2019.
Jump_to_navigation Jump_to_search
Software tools for understanding biological data
For the journal, see Bioinformatics_(journal).
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic_drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
Part of a series on
Biochemistry
[Myoglobin.png]
Key components
    * Biomolecules
    * Metabolism
    * Index
    * Outline
History and topics
    * History
    * Biochemistry
    * Cell_biology
    * Bioinformatics
    * Enzymology
    * Genetics
    * Immunology
    * Molecular_biology
    * Plant_biochemistry
    * Structural_biology
    * Branches_of_biochemistry
    * List_of_biochemists
Glossaries
    * Glossary_of_biology
    * Glossary_of_chemistry
Portals: Biochemistry, Biology, MCB
    * v
    * t
    * e
Map of the human X chromosome (from the National_Center_for_Biotechnology
Information website).
Bioinformatics /ËbaÉª.oÊËÉªnfÉrËmÃ¦tÉªks/ ([About_this_sound]listen) is an
interdisciplinary field that develops methods and software_tools for
understanding biological data. As an interdisciplinary field of science,
bioinformatics combines biology, computer_science, information_engineering,
mathematics and statistics to analyze and interpret biological data.
Bioinformatics has been used for in_silico analyses of biological queries using
mathematical and statistical techniques.
Bioinformatics is both an umbrella_term for the body of biological studies that
use computer_programming as part of their methodology, as well as a reference
to specific analysis "pipelines" that are repeatedly used, particularly in the
field of genomics. Common uses of bioinformatics include the identification of
candidates genes and single nucleotide polymorphisms (SNPs). Often, such
identification is made with the aim of better understanding the genetic basis
of disease, unique adaptations, desirable properties (esp. in agricultural
species), or differences between populations. In a less formal way,
bioinformatics also tries to understand the organisational principles within
nucleic_acid and protein sequences, called proteomics.[1]
⁰
***** Contents *****
    * 1_Introduction
          o 1.1_History
                # 1.1.1_Sequences
          o 1.2_Goals
          o 1.3_Relation_to_other_fields
    * 2_Sequence_analysis
          o 2.1_DNA_sequencing
          o 2.2_Sequence_assembly
          o 2.3_Genome_annotation
          o 2.4_Computational_evolutionary_biology
          o 2.5_Comparative_genomics
          o 2.6_Pan_genomics
          o 2.7_Genetics_of_disease
          o 2.8_Analysis_of_mutations_in_cancer
    * 3_Gene_and_protein_expression
          o 3.1_Analysis_of_gene_expression
          o 3.2_Analysis_of_protein_expression
          o 3.3_Analysis_of_regulation
    * 4_Analysis_of_cellular_organization
          o 4.1_Microscopy_and_image_analysis
          o 4.2_Protein_localization
          o 4.3_Nuclear_organization_of_chromatin
    * 5_Structural_bioinformatics
    * 6_Network_and_systems_biology
          o 6.1_Molecular_interaction_networks
    * 7_Others
          o 7.1_Literature_analysis
          o 7.2_High-throughput_image_analysis
          o 7.3_High-throughput_single_cell_data_analysis
          o 7.4_Biodiversity_informatics
          o 7.5_Ontologies_and_data_integration
    * 8_Databases
    * 9_Software_and_tools
          o 9.1_Open-source_bioinformatics_software
          o 9.2_Web_services_in_bioinformatics
          o 9.3_Bioinformatics_workflow_management_systems
          o 9.4_BioCompute_and_BioCompute_Objects
    * 10_Education_platforms
    * 11_Conferences
    * 12_See_also
    * 13_References
    * 14_Further_reading
    * 15_External_links
***** Introduction[edit] *****
Bioinformatics has become an important part of many areas of biology. In
experimental molecular_biology, bioinformatics techniques such as image and
signal_processing allow extraction of useful results from large amounts of raw
data. In the field of genetics, it aids in sequencing and annotating genomes
and their observed mutations. It plays a role in the text_mining of biological
literature and the development of biological and gene ontologies to organize
and query biological data. It also plays a role in the analysis of gene and
protein expression and regulation. Bioinformatics tools aid in the comparison
of genetic and genomic data and more generally in the understanding of
evolutionary aspects of molecular biology. At a more integrative level, it
helps analyze and catalogue the biological pathways and networks that are an
important part of systems_biology. In structural_biology, it aids in the
simulation and modeling of DNA,[2] RNA,[2][3] proteins[4] as well as
biomolecular interactions.[5][6][7][8]
**** History[edit] ****
Historically, the term bioinformatics did not mean what it means today. Paulien
Hogeweg and Ben_Hesper coined it in 1970 to refer to the study of information
processes in biotic systems.[9][10][11] This definition placed bioinformatics
as a field parallel to biochemistry (the study of chemical processes in
biological systems).[9]
*** Sequences[edit] ***
Sequences of genetic material are frequently used in bioinformatics and are
easier to manage using computers than manually.
Computers became essential in molecular biology when protein_sequences became
available after Frederick_Sanger determined the sequence of insulin in the
early 1950s. Comparing multiple sequences manually turned out to be
impractical. A pioneer in the field was Margaret_Oakley_Dayhoff.[12] She
compiled one of the first protein sequence databases, initially published as
books[13] and pioneered methods of sequence alignment and molecular evolution.
[14] Another early contributor to bioinformatics was Elvin_A._Kabat, who
pioneered biological sequence analysis in 1970 with his comprehensive volumes
of antibody sequences released with Tai Te Wu between 1980 and 1991.[15]
**** Goals[edit] ****
To study how normal cellular activities are altered in different disease
states, the biological data must be combined to form a comprehensive picture of
these activities. Therefore, the field of bioinformatics has evolved such that
the most pressing task now involves the analysis and interpretation of various
types of data. This includes nucleotide and amino_acid_sequences, protein
domains, and protein_structures.[16] The actual process of analyzing and
interpreting data is referred to as computational_biology. Important sub-
disciplines within bioinformatics and computational biology include:
    * Development and implementation of computer programs that enable efficient
      access to, use and management of, various types of information
    * Development of new algorithms (mathematical formulas) and statistical
      measures that assess relationships among members of large data sets. For
      example, there are methods to locate a gene within a sequence, to predict
      protein structure and/or function, and to cluster protein sequences into
      families of related sequences.
The primary goal of bioinformatics is to increase the understanding of
biological processes. What sets it apart from other approaches, however, is its
focus on developing and applying computationally intensive techniques to
achieve this goal. Examples include: pattern_recognition, data_mining, machine
learning algorithms, and visualization. Major research efforts in the field
include sequence_alignment, gene_finding, genome_assembly, drug_design, drug
discovery, protein_structure_alignment, protein_structure_prediction,
prediction of gene_expression and proteinâprotein_interactions, genome-wide
association_studies, the modeling of evolution and cell_division/mitosis.
Bioinformatics now entails the creation and advancement of databases,
algorithms, computational and statistical techniques, and theory to solve
formal and practical problems arising from the management and analysis of
biological data.
Over the past few decades, rapid developments in genomic and other molecular
research technologies and developments in information technologies have
combined to produce a tremendous amount of information related to molecular
biology. Bioinformatics is the name given to these mathematical and computing
approaches used to glean understanding of biological processes.
Common activities in bioinformatics include mapping and analyzing DNA and
protein sequences, aligning DNA and protein sequences to compare them, and
creating and viewing 3-D models of protein structures.
**** Relation to other fields[edit] ****
Bioinformatics is a science field that is similar to but distinct from
biological_computation, while it is often considered synonymous to
computational_biology. Biological computation uses bioengineering and biology
to build biological computers, whereas bioinformatics uses computation to
better understand biology. Bioinformatics and computational biology involve the
analysis of biological data, particularly DNA, RNA, and protein sequences. The
field of bioinformatics experienced explosive growth starting in the mid-1990s,
driven largely by the Human_Genome_Project and by rapid advances in DNA
sequencing technology.
Analyzing biological data to produce meaningful information involves writing
and running software programs that use algorithms from graph_theory, artificial
intelligence, soft_computing, data_mining, image_processing, and computer
simulation. The algorithms in turn depend on theoretical foundations such as
discrete_mathematics, control_theory, system_theory, information_theory, and
statistics.
***** Sequence analysis[edit] *****
Main articles: Sequence_alignment, Sequence_database, and Alignment-free
sequence_analysis
The sequences of different genes or proteins may be aligned side-by-side to
measure their similarity. This alignment compares protein sequences and genomic
sequences containing WPP_domains.
Since the Phage_Î¦-X174 was sequenced in 1977,[17] the DNA_sequences of
thousands of organisms have been decoded and stored in databases. This sequence
information is analyzed to determine genes that encode proteins, RNA genes,
regulatory sequences, structural motifs, and repetitive sequences. A comparison
of genes within a species or between different species can show similarities
between protein functions, or relations between species (the use of molecular
systematics to construct phylogenetic_trees). With the growing amount of data,
it long ago became impractical to analyze DNA sequences manually. Today[when?],
computer_programs such as BLAST are used daily to search sequences from more
than 260 000 organisms, containing over 190 billion nucleotides.[18] These
programs can compensate for mutations (exchanged, deleted or inserted bases) in
the DNA sequence, to identify sequences that are related, but not identical. A
variant of this sequence_alignment is used in the sequencing process itself.
For the special task of taxonomic classification of sequence snippets, modern
k-mer based software like Kraken achieves throughput unreachable by alignment
methods.
**** DNA sequencing[edit] ****
Main article: DNA_sequencing
Before sequences can be analyzed they have to be obtained. DNA_sequencing is
still a non-trivial problem as the raw data may be noisy or afflicted by weak
signals. Algorithms have been developed for base_calling for the various
experimental approaches to DNA sequencing.
**** Sequence assembly[edit] ****
Main article: Sequence_assembly
Most DNA sequencing techniques produce short fragments of sequence that need to
be assembled to obtain complete gene or genome sequences. The so-called shotgun
sequencing technique (which was used, for example, by The_Institute_for_Genomic
Research (TIGR) to sequence the first bacterial genome, Haemophilus_influenzae)
[19] generates the sequences of many thousands of small DNA fragments (ranging
from 35 to 900 nucleotides long, depending on the sequencing technology). The
ends of these fragments overlap and, when aligned properly by a genome assembly
program, can be used to reconstruct the complete genome. Shotgun sequencing
yields sequence data quickly, but the task of assembling the fragments can be
quite complicated for larger genomes. For a genome as large as the human
genome, it may take many days of CPU time on large-memory, multiprocessor
computers to assemble the fragments, and the resulting assembly usually
contains numerous gaps that must be filled in later. Shotgun sequencing is the
method of choice for virtually all genomes sequenced today[when?], and genome
assembly algorithms are a critical area of bioinformatics research.
See also: sequence_analysis, sequence_mining, sequence_profiling_tool, and
sequence_motif
**** Genome annotation[edit] ****
Main article: Gene_prediction
In the context of genomics, annotation is the process of marking the genes and
other biological features in a DNA sequence. This process needs to be automated
because most genomes are too large to annotate by hand, not to mention the
desire to annotate as many genomes as possible, as the rate of sequencing has
ceased to pose a bottleneck. Annotation is made possible by the fact that genes
have recognisable start and stop regions, although the exact sequence found in
these regions can vary between genes.
The first description of a comprehensive genome annotation system was published
in 1995 [19] by the team at The_Institute_for_Genomic_Research that performed
the first complete sequencing and analysis of the genome of a free-living
organism, the bacterium Haemophilus_influenzae.[19] Owen_White designed and
built a software system to identify the genes encoding all proteins, transfer
RNAs, ribosomal RNAs (and other sites) and to make initial functional
assignments. Most current genome annotation systems work similarly, but the
programs available for analysis of genomic DNA, such as the GeneMark program
trained and used to find protein-coding genes in Haemophilus_influenzae, are
constantly changing and improving.
Following the goals that the Human Genome Project left to achieve after its
closure in 2003, a new project developed by the National Human Genome Research
Institute in the U.S appeared. The so-called ENCODE project is a collaborative
data collection of the functional elements of the human genome that uses next-
generation DNA-sequencing technologies and genomic tiling arrays, technologies
able to automatically generate large amounts of data at a dramatically reduced
per-base cost but with the same accuracy (base call error) and fidelity
(assembly error).
**** Computational evolutionary biology[edit] ****
Further information: Computational_phylogenetics
Evolutionary_biology is the study of the origin and descent of species, as well
as their change over time. Informatics has assisted evolutionary biologists by
enabling researchers to:
    * trace the evolution of a large number of organisms by measuring changes
      in their DNA, rather than through physical taxonomy or physiological
      observations alone,
    * compare entire genomes, which permits the study of more complex
      evolutionary events, such as gene_duplication, horizontal_gene_transfer,
      and the prediction of factors important in bacterial speciation,
    * build complex computational population_genetics models to predict the
      outcome of the system over time[20]
    * track and share information on an increasingly large number of species
      and organisms
Future work endeavours to reconstruct the now more complex tree_of_life.
The area of research within computer_science that uses genetic_algorithms is
sometimes confused with computational evolutionary biology, but the two areas
are not necessarily related.
**** Comparative genomics[edit] ****
Main article: Comparative_genomics
The core of comparative genome analysis is the establishment of the
correspondence between genes (orthology analysis) or other genomic features in
different organisms. It is these intergenomic maps that make it possible to
trace the evolutionary processes responsible for the divergence of two genomes.
A multitude of evolutionary events acting at various organizational levels
shape genome evolution. At the lowest level, point mutations affect individual
nucleotides. At a higher level, large chromosomal segments undergo duplication,
lateral transfer, inversion, transposition, deletion and insertion.[21]
Ultimately, whole genomes are involved in processes of hybridization,
polyploidization and endosymbiosis, often leading to rapid speciation. The
complexity of genome evolution poses many exciting challenges to developers of
mathematical models and algorithms, who have recourse to a spectrum of
algorithmic, statistical and mathematical techniques, ranging from exact,
heuristics, fixed parameter and approximation_algorithms for problems based on
parsimony models to Markov_chain_Monte_Carlo algorithms for Bayesian_analysis
of problems based on probabilistic models.
Many of these studies are based on the detection of sequence_homology to assign
sequences to protein_families.[22]
**** Pan genomics[edit] ****
Main article: Pan-genome
Pan genomics is a concept introduced in 2005 by Tettelin and Medini which
eventually took root in bioinformatics. Pan genome is the complete gene
repertoire of a particular taxonomic group: although initially applied to
closely related strains of a species, it can be applied to a larger context
like genus, phylum etc. It is divided in two parts- The Core genome: Set of
genes common to all the genomes under study (These are often housekeeping genes
vital for survival) and The Dispensable/Flexible Genome: Set of genes not
present in all but one or some genomes under study. A bioinformatics tool BPGA
can be used to characterize the Pan Genome of bacterial species.[23]
**** Genetics of disease[edit] ****
Main article: Genome-wide_association_studies
With the advent of next-generation sequencing we are obtaining enough sequence
data to map the genes of complex diseases infertility,[24] breast_cancer[25] or
Alzheimer's_disease.[26] Genome-wide association studies are a useful approach
to pinpoint the mutations responsible for such complex diseases.[27] Through
these studies, thousands of DNA variants have been identified that are
associated with similar diseases and traits.[28] Furthermore, the possibility
for genes to be used at prognosis, diagnosis or treatment is one of the most
essential applications. Many studies are discussing both the promising ways to
choose the genes to be used and the problems and pitfalls of using genes to
predict disease presence or prognosis.[29]
**** Analysis of mutations in cancer[edit] ****
Main article: Oncogenomics
In cancer, the genomes of affected cells are rearranged in complex or even
unpredictable ways. Massive sequencing efforts are used to identify previously
unknown point_mutations in a variety of genes in cancer. Bioinformaticians
continue to produce specialized automated systems to manage the sheer volume of
sequence data produced, and they create new algorithms and software to compare
the sequencing results to the growing collection of human_genome sequences and
germline polymorphisms. New physical detection technologies are employed, such
as oligonucleotide microarrays to identify chromosomal gains and losses (called
comparative_genomic_hybridization), and single-nucleotide_polymorphism arrays
to detect known point mutations. These detection methods simultaneously measure
several hundred thousand sites throughout the genome, and when used in high-
throughput to measure thousands of samples, generate terabytes of data per
experiment. Again the massive amounts and new types of data generate new
opportunities for bioinformaticians. The data is often found to contain
considerable variability, or noise, and thus Hidden_Markov_model and change-
point analysis methods are being developed to infer real copy_number changes.
Two important principles can be used in the analysis of cancer genomes
bioinformatically pertaining to the identification of mutations in the exome.
First, cancer is a disease of accumulated somatic mutations in genes. Second
cancer contains driver mutations which need to be distinguished from
passengers.[30]
With the breakthroughs that this next-generation sequencing technology is
providing to the field of Bioinformatics, cancer genomics could drastically
change. These new methods and software allow bioinformaticians to sequence many
cancer genomes quickly and affordably. This could create a more flexible
process for classifying types of cancer by analysis of cancer driven mutations
in the genome. Furthermore, tracking of patients while the disease progresses
may be possible in the future with the sequence of cancer samples.[31]
Another type of data that requires novel informatics development is the
analysis of lesions found to be recurrent among many tumors.
***** Gene and protein expression[edit] *****
**** Analysis of gene expression[edit] ****
The expression of many genes can be determined by measuring mRNA levels with
multiple techniques including microarrays, expressed_cDNA_sequence_tag (EST)
sequencing, serial_analysis_of_gene_expression (SAGE) tag sequencing, massively
parallel_signature_sequencing (MPSS), RNA-Seq, also known as "Whole
Transcriptome Shotgun Sequencing" (WTSS), or various applications of
multiplexed in-situ hybridization. All of these techniques are extremely noise-
prone and/or subject to bias in the biological measurement, and a major
research area in computational biology involves developing statistical tools to
separate signal from noise in high-throughput gene expression studies.[32] Such
studies are often used to determine the genes implicated in a disorder: one
might compare microarray data from cancerous epithelial cells to data from non-
cancerous cells to determine the transcripts that are up-regulated and down-
regulated in a particular population of cancer cells.
**** Analysis of protein expression[edit] ****
Protein_microarrays and high throughput (HT) mass_spectrometry (MS) can provide
a snapshot of the proteins present in a biological sample. Bioinformatics is
very much involved in making sense of protein microarray and HT MS data; the
former approach faces similar problems as with microarrays targeted at mRNA,
the latter involves the problem of matching large amounts of mass data against
predicted masses from protein sequence databases, and the complicated
statistical analysis of samples where multiple, but incomplete peptides from
each protein are detected. Cellular protein localization in a tissue context
can be achieved through affinity proteomics displayed as spatial data based on
immunohistochemistry and tissue_microarrays.[33]
**** Analysis of regulation[edit] ****
Gene_regulation is the complex orchestration of events by which a signal,
potentially an extracellular signal such as a hormone, eventually leads to an
increase or decrease in the activity of one or more proteins. Bioinformatics
techniques have been applied to explore various steps in this process.
For example, gene expression can be regulated by nearby elements in the genome.
Promoter analysis involves the identification and study of sequence_motifs in
the DNA surrounding the coding region of a gene. These motifs influence the
extent to which that region is transcribed into mRNA. Enhancer elements far
away from the promoter can also regulate gene expression, through three-
dimensional looping interactions. These interactions can be determined by
bioinformatic analysis of chromosome_conformation_capture experiments.
Expression data can be used to infer gene regulation: one might compare
microarray data from a wide variety of states of an organism to form hypotheses
about the genes involved in each state. In a single-cell organism, one might
compare stages of the cell_cycle, along with various stress conditions (heat
shock, starvation, etc.). One can then apply clustering_algorithms to that
expression data to determine which genes are co-expressed. For example, the
upstream regions (promoters) of co-expressed genes can be searched for over-
represented regulatory_elements. Examples of clustering algorithms applied in
gene clustering are k-means_clustering, self-organizing_maps (SOMs),
hierarchical_clustering, and consensus_clustering methods.
***** Analysis of cellular organization[edit] *****
Several approaches have been developed to analyze the location of organelles,
genes, proteins, and other components within cells. This is relevant as the
location of these components affects the events within a cell and thus helps us
to predict the behavior of biological systems. A gene_ontology category,
cellular compartment, has been devised to capture subcellular localization in
many biological_databases.
**** Microscopy and image analysis[edit] ****
Microscopic pictures allow us to locate both organelles as well as molecules.
It may also help us to distinguish between normal and abnormal cells, e.g. in
cancer.
**** Protein localization[edit] ****
The localization of proteins helps us to evaluate the role of a protein. For
instance, if a protein is found in the nucleus it may be involved in gene
regulation or splicing. By contrast, if a protein is found in mitochondria, it
may be involved in respiration or other metabolic_processes. Protein
localization is thus an important component of protein_function_prediction.
There are well developed protein_subcellular_localization_prediction resources
available, including protein subcellular location databases, and prediction
tools.[34][35]
**** Nuclear organization of chromatin[edit] ****
Main article: Nuclear_organization
Data from high-throughput chromosome_conformation_capture experiments, such as
Hi-C_(experiment) and ChIA-PET, can provide information on the spatial
proximity of DNA loci. Analysis of these experiments can determine the three-
dimensional structure and nuclear_organization of chromatin. Bioinformatic
challenges in this field include partitioning the genome into domains, such as
Topologically_Associating_Domains (TADs), that are organised together in three-
dimensional space.[36]
***** Structural bioinformatics[edit] *****
Main articles: Structural_bioinformatics and Protein_structure_prediction
See also: Structural_motif and Structural_domain
3-dimensional protein structures such as this one are common subjects in
bioinformatic analyses.
Protein structure prediction is another important application of
bioinformatics. The amino_acid sequence of a protein, the so-called primary
structure, can be easily determined from the sequence on the gene that codes
for it. In the vast majority of cases, this primary structure uniquely
determines a structure in its native environment. (Of course, there are
exceptions, such as the bovine_spongiform_encephalopathy â a.k.a. Mad_Cow
Disease â prion.) Knowledge of this structure is vital in understanding the
function of the protein. Structural information is usually classified as one of
secondary, tertiary and quaternary structure. A viable general solution to such
predictions remains an open problem. Most efforts have so far been directed
towards heuristics that work most of the time.[citation_needed]
One of the key ideas in bioinformatics is the notion of homology. In the
genomic branch of bioinformatics, homology is used to predict the function of a
gene: if the sequence of gene A, whose function is known, is homologous to the
sequence of gene B, whose function is unknown, one could infer that B may share
A's function. In the structural branch of bioinformatics, homology is used to
determine which parts of a protein are important in structure formation and
interaction with other proteins. In a technique called homology_modeling, this
information is used to predict the structure of a protein once the structure of
a homologous protein is known. This currently remains the only way to predict
protein structures reliably.
One example of this is hemoglobin in humans and the hemoglobin in legumes
(leghemoglobin), which are distant relatives from the same protein_superfamily.
Both serve the same purpose of transporting oxygen in the organism. Although
both of these proteins have completely different amino acid sequences, their
protein structures are virtually identical, which reflects their near identical
purposes and shared ancestor.[37]
Other techniques for predicting protein structure include protein threading and
de novo (from scratch) physics-based modeling.
Another aspect of structural bioinformatics include the use of protein
structures for Virtual_Screening models such as Quantitative_Structure-Activity
Relationship models and proteochemometric models (PCM). Furthermore, a
protein's crystal structure can be used in simulation of for example ligand-
binding studies and in silico mutagenesis studies.
***** Network and systems biology[edit] *****
Main articles: Computational_systems_biology, Biological_network, and
Interactome
Network analysis seeks to understand the relationships within biological
networks such as metabolic or proteinâprotein_interaction_networks. Although
biological networks can be constructed from a single type of molecule or entity
(such as genes), network biology often attempts to integrate many different
data types, such as proteins, small molecules, gene expression data, and
others, which are all connected physically, functionally, or both.
Systems biology involves the use of computer_simulations of cellular subsystems
(such as the networks_of_metabolites and enzymes that comprise metabolism,
signal_transduction pathways and gene_regulatory_networks) to both analyze and
visualize the complex connections of these cellular processes. Artificial_life
or virtual evolution attempts to understand evolutionary processes via the
computer simulation of simple (artificial) life forms.
**** Molecular interaction networks[edit] ****
Interactions between proteins are frequently visualized and analyzed using
networks. This network is made up of proteinâprotein interactions from
Treponema_pallidum, the causative agent of syphilis and other diseases.
Main articles: Proteinâprotein_interaction_prediction and interactome
Tens of thousands of three-dimensional protein structures have been determined
by X-ray_crystallography and protein_nuclear_magnetic_resonance_spectroscopy
(protein NMR) and a central question in structural bioinformatics is whether it
is practical to predict possible proteinâprotein interactions only based on
these 3D shapes, without performing proteinâprotein_interaction experiments.
A variety of methods have been developed to tackle the proteinâprotein
docking problem, though it seems that there is still much work to be done in
this field.
Other interactions encountered in the field include Proteinâligand (including
drug) and proteinâpeptide. Molecular dynamic simulation of movement of atoms
about rotatable bonds is the fundamental principle behind computational
algorithms, termed docking algorithms, for studying molecular_interactions.
***** Others[edit] *****
**** Literature analysis[edit] ****
Main articles: Text_mining and Biomedical_text_mining
The growth in the number of published literature makes it virtually impossible
to read every paper, resulting in disjointed sub-fields of research. Literature
analysis aims to employ computational and statistical linguistics to mine this
growing library of text resources. For example:
    * Abbreviation recognition â identify the long-form and abbreviation of
      biological terms
    * Named entity recognition â recognizing biological terms such as gene
      names
    * Proteinâprotein interaction â identify which proteins interact with
      which proteins from text
The area of research draws from statistics and computational_linguistics.
**** High-throughput image analysis[edit] ****
Computational technologies are used to accelerate or fully automate the
processing, quantification and analysis of large amounts of high-information-
content biomedical_imagery. Modern image_analysis systems augment an observer's
ability to make measurements from a large or complex set of images, by
improving accuracy, objectivity, or speed. A fully developed analysis system
may completely replace the observer. Although these systems are not unique to
biomedical imagery, biomedical imaging is becoming more important for both
diagnostics and research. Some examples are:
    * high-throughput and high-fidelity quantification and sub-cellular
      localization (high-content_screening, cytohistopathology, Bioimage
      informatics)
    * morphometrics
    * clinical image analysis and visualization
    * determining the real-time air-flow patterns in breathing lungs of living
      animals
    * quantifying occlusion size in real-time imagery from the development of
      and recovery during arterial injury
    * making behavioral observations from extended video recordings of
      laboratory animals
    * infrared measurements for metabolic activity determination
    * inferring clone overlaps in DNA_mapping, e.g. the Sulston_score
**** High-throughput single cell data analysis[edit] ****
Main article: Flow_cytometry_bioinformatics
Computational techniques are used to analyse high-throughput, low-measurement
single cell data, such as that obtained from flow_cytometry. These methods
typically involve finding populations of cells that are relevant to a
particular disease state or experimental condition.
**** Biodiversity informatics[edit] ****
Main article: Biodiversity_informatics
Biodiversity informatics deals with the collection and analysis of biodiversity
data, such as taxonomic_databases, or microbiome data. Examples of such
analyses include phylogenetics, niche_modelling, species_richness mapping, DNA
barcoding, or species identification tools.
**** Ontologies and data integration[edit] ****
Biological ontologies are directed acyclic graphs of controlled vocabularies.
They are designed to capture biological concepts and descriptions in a way that
can be easily categorised and analysed with computers. When categorised in this
way, it is possible to gain added value from holistic and integrated analysis.
The OBO_Foundry was an effort to standardise certain ontologies. One of the
most widespread is the Gene_ontology which describes gene function. There are
also ontologies which describe phenotypes.
***** Databases[edit] *****
Main articles: List_of_biological_databases and Biological_database
Databases are essential for bioinformatics research and applications. Many
databases exist, covering various information types: for example, DNA and
protein sequences, molecular structures, phenotypes and biodiversity. Databases
may contain empirical data (obtained directly from experiments), predicted data
(obtained from analysis), or, most commonly, both. They may be specific to a
particular organism, pathway or molecule of interest. Alternatively, they can
incorporate data compiled from multiple other databases. These databases vary
in their format, access mechanism, and whether they are public or not.
Some of the most commonly used databases are listed below. For a more
comprehensive list, please check the link at the beginning of the subsection.
    * Used in biological sequence analysis: Genbank, UniProt
    * Used in structure analysis: Protein_Data_Bank (PDB)
    * Used in finding Protein Families and Motif Finding: InterPro, Pfam
    * Used for Next Generation Sequencing: Sequence_Read_Archive
    * Used in Network Analysis: Metabolic Pathway Databases (KEGG, BioCyc),
      Interaction Analysis Databases, Functional Networks
    * Used in design of synthetic genetic circuits: GenoCAD
***** Software and tools[edit] *****
Software_tools_for_bioinformatics range from simple command-line tools, to more
complex graphical programs and standalone web-services available from various
bioinformatics_companies or public institutions.
**** Open-source bioinformatics software[edit] ****
Many free_and_open-source_software tools have existed and continued to grow
since the 1980s.[38] The combination of a continued need for new algorithms for
the analysis of emerging types of biological readouts, the potential for
innovative in_silico experiments, and freely available open_code bases have
helped to create opportunities for all research groups to contribute to both
bioinformatics and the range of open-source software available, regardless of
their funding arrangements. The open source tools often act as incubators of
ideas, or community-supported plug-ins in commercial applications. They may
also provide de_facto standards and shared object models for assisting with the
challenge of bioinformation integration.
The range_of_open-source_software_packages includes titles such as
Bioconductor, BioPerl, Biopython, BioJava, BioJS, BioRuby, Bioclipse, EMBOSS,
.NET_Bio, Orange with its bioinformatics add-on, Apache_Taverna, UGENE and
GenoCAD. To maintain this tradition and create further opportunities, the non-
profit Open_Bioinformatics_Foundation[38] have supported the annual
Bioinformatics_Open_Source_Conference (BOSC) since 2000.[39]
An alternative method to build public bioinformatics databases is to use the
MediaWiki engine with the WikiOpener extension. This system allows the database
to be accessed and updated by all experts in the field.[40]
**** Web services in bioinformatics[edit] ****
SOAP- and REST-based interfaces have been developed for a wide variety of
bioinformatics applications allowing an application running on one computer in
one part of the world to use algorithms, data and computing resources on
servers in other parts of the world. The main advantages derive from the fact
that end users do not have to deal with software and database maintenance
overheads.
Basic bioinformatics services are classified by the EBI into three categories:
SSS (Sequence Search Services), MSA (Multiple Sequence Alignment), and BSA
(Biological Sequence Analysis).[41] The availability of these service-oriented
bioinformatics resources demonstrate the applicability of web-based
bioinformatics solutions, and range from a collection of standalone tools with
a common data format under a single, standalone or web-based interface, to
integrative, distributed and extensible bioinformatics_workflow_management
systems.
**** Bioinformatics workflow management systems[edit] ****
Main article: Bioinformatics_workflow_management_systems
A bioinformatics_workflow_management_system is a specialized form of a workflow
management_system designed specifically to compose and execute a series of
computational or data manipulation steps, or a workflow, in a Bioinformatics
application. Such systems are designed to
    * provide an easy-to-use environment for individual application scientists
      themselves to create their own workflows,
    * provide interactive tools for the scientists enabling them to execute
      their workflows and view their results in real-time,
    * simplify the process of sharing and reusing workflows between the
      scientists, and
    * enable scientists to track the provenance of the workflow execution
      results and the workflow creation steps.
Some of the platforms giving this service: Galaxy, Kepler, Taverna, UGENE,
Anduril, HIVE.
**** BioCompute and BioCompute Objects[edit] ****
In 2014, the US_Food_and_Drug_Administration sponsored a conference held at the
National_Institutes_of_Health Bethesda Campus to discuss reproducibility in
bioinformatics.[42] Over the next three years, a consortium of stakeholders met
regularly to discuss what would become BioCompute paradigm.[43] These
stakeholders included representatives from government, industry, and academic
entities. Session leaders represented numerous branches of the FDA and NIH
Institutes and Centers, non-profit entities including the Human_Variome_Project
and the European_Federation_for_Medical_Informatics, and research institutions
including Stanford, the New_York_Genome_Center, and the George_Washington
University.
It was decided that the BioCompute paradigm would be in the form of digital
âlab notebooksâ which allow for the reproducibility, replication, review,
and reuse, of bioinformatics protocols. This was proposed to enable greater
continuity within a research group over the course of normal personnel flux
while furthering the exchange of ideas between groups. The US FDA funded this
work so that information on pipelines would be more transparent and accessible
to their regulatory staff.[44]
In 2016, the group reconvened at the NIH in Bethesda and discussed the
potential for a BioCompute_Object, an instance of the BioCompute paradigm. This
work was copied as both a âstandard trial useâ document and a preprint
paper uploaded to bioRxiv. The BioCompute object allows for the JSON-ized
record to be shared among employees, collaborators, and regulators.[45][46]
***** Education platforms[edit] *****
Software platforms designed to teach bioinformatics concepts and methods
include Rosalind and online courses offered through the Swiss_Institute_of
Bioinformatics Training Portal. The Canadian_Bioinformatics_Workshops provides
videos and slides from training workshops on their website under a Creative
Commons license. The 4273Ï project or 4273pi project[47] also offers open
source educational materials for free. The course runs on low cost Raspberry_Pi
computers and has been used to teach adults and school pupils.[48][49] 4273Ï
is actively developed by a consortium of academics and research staff who have
run research level bioinformatics using Raspberry Pi computers and the 4273Ï
operating system.[50][51]
MOOC platforms also provide online certifications in bioinformatics and related
disciplines, including Coursera's Bioinformatics Specialization (UC_San_Diego)
and Genomic Data Science Specialization (Johns_Hopkins) as well as EdX's Data
Analysis for Life Sciences XSeries (Harvard). University of Southern California
offers a Masters_In_Translational_Bioinformatics focusing on biomedical
applications.
***** Conferences[edit] *****
There are several large conferences that are concerned with bioinformatics.
Some of the most notable examples are Intelligent_Systems_for_Molecular_Biology
(ISMB), European_Conference_on_Computational_Biology (ECCB), and Research_in
Computational_Molecular_Biology (RECOMB).
***** See also[edit] *****
    * Biodiversity_informatics
    * Bioinformatics_companies
    * Computational_biology
    * Computational_biomodeling
    * Computational_genomics
    * Functional_genomics
    * Health_informatics
    * International_Society_for_Computational_Biology
    * Jumping_library
    * List_of_bioinformatics_institutions
    * List_of_open-source_bioinformatics_software
    * List_of_bioinformatics_journals
    * Metabolomics
    * Nucleic_acid_sequence
    * Phylogenetics
    * Proteomics
    * Structural_bioinformatics
    * Gene_Disease_Database
***** References[edit] *****
   1. ^Lesk, A. M. (26 July 2013). "Bioinformatics". Encyclopaedia Britannica.
      Retrieved 17 April 2017.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a bSim, A. Y. L.; Minary, P.; Levitt, M. (2012). "Modeling_nucleic
      acids". Current_Opinion_in_Structural_Biology. 22 (3): 273â278. doi:
      10.1016/j.sbi.2012.03.012. PMC 4028509. PMID 22538125.
   4. ^Dawson, W. K.; Maciejczyk, M.; Jankowska, E. J.; Bujnicki, J. M. (2016).
      "Coarse-grained modeling of RNA 3D structure". Methods. 103: 138â156.
      doi:10.1016/j.ymeth.2016.04.026. PMID 27125734.
   5. ^Kmiecik, S.; Gront, D.; Kolinski, M.; Wieteska, L.; Dawid, A. E.;
      Kolinski, A. (2016). "Coarse-Grained Protein Models and Their
      Applications". Chemical_Reviews. 116 (14): 7898â936. doi:10.1021/
      acs.chemrev.6b00163. PMID 27333362.
   6. ^Wong, K. C. (2016). Computational Biology and Bioinformatics: Gene
      Regulation. CRC Press/Taylor & Francis Group. ISBN 9781498724975.
   7. ^Joyce, A. P.; Zhang, C.; Bradley, P.; Havranek, J. J. (2015).
      "Structure-based_modeling_of_protein:_DNA_specificity". Briefings_in
      Functional_Genomics. 14 (1): 39â49. doi:10.1093/bfgp/elu044.
      PMC 4366589. PMID 25414269.
   8. ^Spiga, E.; Degiacomi, M. T.; Dal Peraro, M. (2014). "New Strategies for
      Integrative Dynamic Modeling of Macromolecular Assembly". In
      Karabencheva-Christova, T. (ed.). Biomolecular Modelling and Simulations.
      Advances in Protein Chemistry and Structural Biology. 96. Academic Press.
      pp. 77â111. doi:10.1016/bs.apcsb.2014.06.008. ISBN 9780128000137.
      PMID 25443955.
   9. ^Ciemny, Maciej; Kurcinski, Mateusz; Kamel, Karol; Kolinski, Andrzej;
      Alam, Nawsad; Schueler-Furman, Ora; Kmiecik, Sebastian (4 May 2018).
      "Proteinâpeptide docking: opportunities and challenges". Drug Discovery
      Today. 23 (8): 1530â1537. doi:10.1016/j.drudis.2018.05.006. ISSN 1359-
      6446. PMID 29733895.
  10. ^ a bHogeweg P (2011). Searls, David B. (ed.). "The_Roots_of
      Bioinformatics_in_Theoretical_Biology". PLoS Computational Biology. 7
      (3): e1002021. Bibcode:2011PLSCB...7E2021H. doi:10.1371/
      journal.pcbi.1002021. PMC 3068925. PMID 21483479.
  11. ^Hesper B, Hogeweg P (1970). "Bioinformatica: een werkconcept". 1 (6).
      Kameleon: 28â29.
  12. ^Hogeweg P (1978). "Simulating the growth of cellular forms". Simulation.
      31 (3): 90â96. doi:10.1177/003754977803100305.
  13. ^Moody, Glyn (2004). Digital Code of Life: How Bioinformatics is
      Revolutionizing Science, Medicine, and Business. ISBN 978-0-471-32788-2.
  14. ^ Dayhoff, M.O. (1966) Atlas of protein sequence and structure. National
      Biomedical Research Foundation, 215 pp.
  15. ^Eck RV, Dayhoff MO (1966). "Evolution of the structure of ferredoxin
      based on living relics of primitive amino Acid sequences". Science. 152
      (3720): 363â6. Bibcode:1966Sci...152..363E. doi:10.1126/
      science.152.3720.363. PMID 17775169.
  16. ^Johnson G, Wu TT (January 2000). "Kabat_Database_and_its_applications:
      30_years_after_the_first_variability_plot". Nucleic Acids Res. 28 (1):
      214â218. doi:10.1093/nar/28.1.214. PMC 102431. PMID 10592229.
  17. ^Attwood TK, Gisel A, Eriksson NE, Bongcam-Rudloff E (2011). "Concepts,
      Historical_Milestones_and_the_Central_Place_of_Bioinformatics_in_Modern
      Biology:_A_European_Perspective". Bioinformatics - Trends and
      Methodologies. Bioinformatics â Trends and Methodologies. InTech. doi:
      10.5772/23535. ISBN 978-953-307-282-1. Retrieved 8 January 2012.
  18. ^Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA, Hutchison
      CA, Slocombe PM, Smith M (February 1977). "Nucleotide sequence of
      bacteriophage phi X174 DNA". Nature. 265 (5596): 687â95. Bibcode:
      1977Natur.265..687S. doi:10.1038/265687a0. PMID 870828.
  19. ^Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (January
      2008). "GenBank". Nucleic Acids Res. 36 (Database issue): D25â30. doi:
      10.1093/nar/gkm929. PMC 2238942. PMID 18073190.
  20. ^ a b cFleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF,
      Kerlavage AR, Bult CJ, Tomb JF, Dougherty BA, Merrick JM (July 1995).
      "Whole-genome random sequencing and assembly of Haemophilus influenzae
      Rd". Science. 269 (5223): 496â512. Bibcode:1995Sci...269..496F. doi:
      10.1126/science.7542800. PMID 7542800.
  21. ^Carvajal-RodrÃ­guez A (2012). "Simulation_of_Genes_and_Genomes_Forward
      in_Time". Current Genomics. 11 (1): 58â61. doi:10.2174/
      138920210790218007. PMC 2851118. PMID 20808525.
  22. ^Brown, TA (2002). "Mutation, Repair and Recombination". Genomes (2nd
      ed.). Manchester (UK): Oxford.
  23. ^Carter, N. P.; Fiegler, H.; Piper, J. (2002). "Comparative analysis of
      comparative genomic hybridization microarray technologies: Report of a
      workshop sponsored by the Wellcome trust". Cytometry Part A. 49 (2):
      43â8. doi:10.1002/cyto.10153. PMID 12357458.
  24. ^Chaudhari Narendrakumar M., Kumar Gupta Vinod, Dutta Chitra (2016).
      "BPGA-an_ultra-fast_pan-genome_analysis_pipeline". Scientific Reports. 6:
      24373. Bibcode:2016NatSR...624373C. doi:10.1038/srep24373. PMC 4829868.
      PMID 27071527.CS1 maint: Multiple names: authors list (link)
  25. ^Aston KI (2014). "Genetic susceptibility to male infertility: News from
      genome-wide association studies". Andrology. 2 (3): 315â21. doi:
      10.1111/j.2047-2927.2014.00188.x. PMID 24574159.
  26. ^VÃ©ron A, Blein S, Cox DG (2014). "Genome-wide association studies and
      the clinic: A focus on breast cancer". Biomarkers_in_Medicine. 8 (2):
      287â96. doi:10.2217/bmm.13.121. PMID 24521025.
  27. ^Tosto G, Reitz C (2013). "Genome-wide_association_studies_in_Alzheimer's
      disease:_A_review". Current Neurology and Neuroscience Reports. 13 (10):
      381. doi:10.1007/s11910-013-0381-0. PMC 3809844. PMID 23954969.
  28. ^Londin E, Yadav P, Surrey S, Kricka LJ, Fortina P (2013). Use of Linkage
      Analysis, Genome-Wide Association Studies, and Next-Generation Sequencing
      in the Identification of Disease-Causing Mutations. Pharmacogenomics.
      Methods in Molecular Biology. 1015. pp. 127â46. doi:10.1007/978-1-
      62703-435-7_8. ISBN 978-1-62703-434-0. PMID 23824853.
  29. ^Hindorff, L.A.; et al. (2009). "Potential_etiologic_and_functional
      implications_of_genome-wide_association_loci_for_human_diseases_and
      traits". Proc. Natl. Acad. Sci. USA. 106 (23): 9362â9367. Bibcode:
      2009PNAS..106.9362H. doi:10.1073/pnas.0903103106. PMC 2687147.
      PMID 19474294.
  30. ^Hall, L.O. (2010). Finding the right genes for disease and prognosis
      prediction. System Science and Engineering (ICSSE),2010 International
      Conference. pp. 1â2. doi:10.1109/ICSSE.2010.5551766. ISBN 978-1-4244-
      6472-2.
  31. ^Vazquez, Miguel; Torre, Victor de la; Valencia, Alfonso (27 December
      2012). "Chapter_14:_Cancer_Genome_Analysis". PLOS Computational Biology.
      8 (12): e1002824. Bibcode:2012PLSCB...8E2824V. doi:10.1371/
      journal.pcbi.1002824. ISSN 1553-7358. PMC 3531315. PMID 23300415.
  32. ^Hye-Jung, E.C.; Jaswinder, K.; Martin, K.; Samuel, A.A; Marco, A.M
      (2014). "Second-Generation Sequencing for Cancer Genome Analysis". In
      Dellaire, Graham; Berman, Jason N.; Arceci, Robert J. (eds.). Cancer
      Genomics. Boston (US): Academic Press. pp. 13â30. doi:10.1016/B978-0-
      12-396967-5.00002-5. ISBN 9780123969675.
  33. ^Grau, J.; Ben-Gal, I.; Posch, S.; Grosse, I. (1 July 2006). "VOMBAT:
      prediction_of_transcription_factor_binding_sites_using_variable_order
      Bayesian_trees" (PDF). Nucleic Acids Research. 34 (Web Server):
      W529âW533. doi:10.1093/nar/gkl212. PMC 1538886. PMID 16845064.
  34. ^"The_Human_Protein_Atlas". www.proteinatlas.org. Retrieved 2 October
      2017.
  35. ^"The_human_cell". www.proteinatlas.org. Retrieved 2 October 2017.
  36. ^Thul, Peter J.; Ãkesson, Lovisa; Wiking, Mikaela; Mahdessian, Diana;
      Geladaki, Aikaterini; Blal, Hammou Ait; Alm, Tove; Asplund, Anna; BjÃ¶rk,
      Lars (26 May 2017). "A subcellular map of the human proteome". Science.
      356 (6340): eaal3321. doi:10.1126/science.aal3321. PMID 28495876.
  37. ^Ay, Ferhat; Noble, William S. (2 September 2015). "Analysis_methods_for
      studying_the_3D_architecture_of_the_genome". Genome Biology. 16 (1): 183.
      doi:10.1186/s13059-015-0745-7. PMC 4556012. PMID 26328929.
  38. ^Hoy, JA; Robinson, H; Trent JT, 3rd; Kakar, S; Smagghe, BJ; Hargrove, MS
      (3 August 2007). "Plant hemoglobins: a molecular fossil record for the
      evolution of oxygen transport". Journal of Molecular Biology. 371 (1):
      168â79. doi:10.1016/j.jmb.2007.05.029. PMID 17560601.
  39. ^ a b"Open_Bioinformatics_Foundation:_About_us". Official website. Open
      Bioinformatics_Foundation. Retrieved 10 May 2011.
  40. ^"Open_Bioinformatics_Foundation:_BOSC". Official website. Open
      Bioinformatics_Foundation. Retrieved 10 May 2011.
  41. ^BrohÃ©e, Sylvain; Barriot, Roland; Moreau, Yves (2010). "Biological
      knowledge_bases_using_Wikis:_combining_the_flexibility_of_Wikis_with_the
      structure_of_databases". Bioinformatics. 26 (17): 2210â2211. doi:
      10.1093/bioinformatics/btq348. PMID 20591906. Retrieved 5 May 2015.
  42. ^Nisbet, Robert (14 May 2009). "BIOINFORMATICS". Handbook of Statistical
      Analysis and Data Mining Applications. John Elder IV, Gary Miner.
      Academic Press. p. 328. ISBN 9780080912035. Retrieved 9 May 2014.
  43. ^Commissioner, Office of the. "Advancing_Regulatory_Science_-_Sept._24-
      25,_2014_Public_Workshop:_Next_Generation_Sequencing_Standards".
      www.fda.gov. Retrieved 30 November 2017.
  44. ^Simonyan, Vahan; Goecks, Jeremy; Mazumder, Raja (2017). "Biocompute
      ObjectsâA_Step_towards_Evaluation_and_Validation_of_Biomedical
      Scientific_Computations". PDA Journal of Pharmaceutical Science and
      Technology. 71 (2): 136â146. doi:10.5731/pdajpst.2016.006734.
      ISSN 1079-7440. PMC 5510742. PMID 27974626.
  45. ^Commissioner, Office of the. "Advancing_Regulatory_Science_-_Community-
      based_development_of_HTS_standards_for_validating_data_and_computation
      and_encouraging_interoperability". www.fda.gov. Retrieved 30 November
      2017.
  46. ^Alterovitz, Gil; Dean, Dennis A.; Goble, Carole; Crusoe, Michael R.;
      Soiland-Reyes, Stian; Bell, Amanda; Hayes, Anais; King, Charles Hadley
      S.; Johanson, Elaine (4 October 2017). "Enabling Precision Medicine via
      standard communication of NGS provenance, analysis, and results".
      bioRxiv 191783.
  47. ^BioCompute_Object_(BCO)_project_is_a_collaborative_and_community-driven
      framework_to_standardize_HTS_computational_data._1._BCO_Specification
      Document:_user_manual_for_understanding_and_creating_B., biocompute-
      objects, 3 September 2017, retrieved 30 November 2017
  48. ^Barker, D; Ferrier, D.E.K.; Holland, P.W; Mitchell, J.B.O; Plaisier, H;
      Ritchie, M.G; Smart, S.D. (2013). "4273Ï :_bioinformatics_education_on
      low_cost_ARM_hardware". BMC Bioinformatics. 14: 243. doi:10.1186/1471-
      2105-14-243. PMC 3751261. PMID 23937194.
  49. ^Barker, D; Alderson, R.G; McDonagh, J.L; Plaisier, H; Comrie, M.M;
      Duncan, L; Muirhead, G.T.P; Sweeny, S.D. (2015). "University-level
      practical activities in bioinformatics benefit voluntary groups of pupils
      in the last 2 years of school". International Journal of STEM Education.
      2 (17). doi:10.1186/s40594-015-0030-z.
  50. ^McDonagh, J.L; Barker, D; Alderson, R.G. (2016). "Bringing_computational
      science_to_the_public". SpringerPlus. 5 (259): 259. doi:10.1186/s40064-
      016-1856-7. PMC 4775721. PMID 27006868.
  51. ^Robson, J.F.; Barker, D (2015). "Comparison_of_the_protein-coding_gene
      content_of_Chlamydia_trachomatis_and_Protochlamydia_amoebophila_using_a
      Raspberry_Pi_computer". BMC Research Notes. 8 (561): 561. doi:10.1186/
      s13104-015-1476-2. PMC 4604092. PMID 26462790.
  52. ^Wregglesworth, K.M; Barker, D (2015). "A_comparison_of_the_protein-
      coding_genomes_of_two_green_sulphur_bacteria,_Chlorobium_tepidum_TLS_and
      Pelodictyon_phaeoclathratiforme_BU-1". BMC Research Notes. 8 (565): 565.
      doi:10.1186/s13104-015-1535-8. PMC 4606965. PMID 26467441.
***** Further reading[edit] *****
    * Sehgal et al. : Structural, phylogenetic and docking studies of D-amino
      acid oxidase activator(DAOA ), a candidate schizophrenia gene.
      Theoretical Biology and Medical Modelling 2013 10 :3.
    * Raul Isea The_Present-Day_Meaning_Of_The_Word_Bioinformatics, Global
      Journal of Advanced Research, 2015
    * Ilzins, O., Isea, R. and Hoebeke, J. Can_Bioinformatics_Be_Considered_as
      an_Experimental_Biological_Science 2015
    * Achuthsankar S Nair Computational_Biology_&_Bioinformatics_â_A_gentle
      Overview, Communications of Computer Society of India, January 2007
    * Aluru,_Srinivas, ed. Handbook of Computational Molecular Biology. Chapman
      & Hall/Crc, 2006.
ISBN 1-58488-406-1 (Chapman & Hall/Crc Computer and Information Science Series)
Baldi, P and Brunak, S, Bioinformatics: The Machine Learning Approach, 2nd
edition. MIT Press, 2001.
ISBN 0-262-02506-X
Barnes, M.R. and Gray, I.C., eds., Bioinformatics for Geneticists, first
edition. Wiley, 2003.
ISBN 0-470-84394-2
Baxevanis, A.D. and Ouellette, B.F.F., eds., Bioinformatics: A Practical Guide
to the Analysis of Genes and Proteins, third edition. Wiley, 2005.
ISBN 0-471-47878-4
Baxevanis, A.D., Petsko, G.A., Stein, L.D., and Stormo, G.D., eds., Current
Protocols in Bioinformatics. Wiley, 2007.
ISBN 0-471-25093-7
Cristianini, N. and Hahn, M. Introduction_to_Computational_Genomics, Cambridge
University Press, 2006. (
ISBN 9780521671910 |
ISBN 0-521-67191-4)
Durbin, R., S. Eddy, A. Krogh and G. Mitchison, Biological sequence analysis.
Cambridge University Press, 1998.
ISBN 0-521-62971-3
Gilbert D (2004). "Bioinformatics_software_resources". Briefings in
Bioinformatics. 5 (3): 300â304. doi:10.1093/bib/5.3.300. PMID 15383216.
Keedwell, E., Intelligent Bioinformatics: The Application of Artificial
Intelligence Techniques to Bioinformatics Problems. Wiley, 2005.
ISBN 0-470-02175-6
Kohane, et al. Microarrays for an Integrative Genomics. The MIT Press, 2002.
ISBN 0-262-11271-X
Lund, O. et al. Immunological Bioinformatics. The MIT Press, 2005.
ISBN 0-262-12280-4
Pachter,_Lior and Sturmfels,_Bernd. "Algebraic Statistics for Computational
Biology" Cambridge University Press, 2005.
ISBN 0-521-85700-7
Pevzner, Pavel A. Computational Molecular Biology: An Algorithmic Approach The
MIT Press, 2000.
ISBN 0-262-16197-4
Soinov, L. Bioinformatics_and_Pattern_Recognition_Come_Together Journal of
Pattern Recognition Research (JPRR), Vol 1 (1) 2006 p. 37â41
Stevens, Hallam, Life Out of Sequence: A Data-Driven History of Bioinformatics,
Chicago: The University of Chicago Press, 2013,
ISBN 9780226080208
Tisdall, James. "Beginning Perl for Bioinformatics" O'Reilly, 2001.
ISBN 0-596-00080-4
Catalyzing_Inquiry_at_the_Interface_of_Computing_and_Biology_(2005)_CSTB_report
Calculating_the_Secrets_of_Life:_Contributions_of_the_Mathematical_Sciences_and
computing_to_Molecular_Biology_(1995)
Foundations_of_Computational_and_Systems_Biology_MIT_Course
Computational_Biology:_Genomes,_Networks,_Evolution_Free_MIT_Course
***** External links[edit] *****
Listen to this article (info/dl)

This audio file was created from a revision of the article "Bioinformatics"
dated 2013-09-20, and does not reflect subsequent edits to the article. (Audio
help)
More_spoken_articles
    *  The dictionary definition of bioinformatics at Wiktionary
    *  Learning materials related to Bioinformatics at Wikiversity
    *  Media related to Bioinformatics at Wikimedia Commons
    * Bioinformatics_Resource_Portal_(SIB)
    * v
    * t
    * e
Bioinformatics
                   * Sequence databases: GenBank, European_Nucleotide_Archive
                     and DNA_Data_Bank_of_Japan
                   * Secondary databases: UniProt, database of protein
                     sequences grouping together Swiss-Prot, TrEMBL and Protein
Databases            Information_Resource
                   * Other databases: Protein_Data_Bank, Ensembl and InterPro
                   * Specialised genomic databases: BOLD, Saccharomyces_Genome
                     Database, FlyBase, VectorBase, WormBase, PHI-base,
                     Arabidopsis_Information_Resource and Zebrafish_Information
                     Network
                   * BLAST
                   * Bowtie
                   * Clustal
Software           * HMMER
                   * MUSCLE
                   * SAMtools
                   * TopHat
Other              * Server: ExPASy
                   * Ontology: Gene_Ontology
                   * European_Bioinformatics_Institute
                   * US_National_Center_for_Biotechnology_Information
                   * Swiss_Institute_of_Bioinformatics
Institutions       * Japanese_Institute_of_Genetics
                   * Philippine_Genome_Center
                   * Broad_Institute
                   * Wellcome_Sanger_Institute
                   * International_Society_for_Computational_Biology (ISCB)
                   * European_Molecular_Biology_network (EMBnet)
Organizations      * African_Society_for_Bioinformatics_and_Computational
                     Biology (ASBCB)
                   * Japanese_Society_for_Bioinformatics (JSBi)
                   * Intelligent_Systems_for_Molecular_Biology (ISMB)
                   * Research_in_Computational_Molecular_Biology (RECOMB)
                   * European_Conference_on_Computational_Biology (ECCB)
Meetings           * Pacific_Symposium_on_Biocomputing (PSB)
                   * ISCB_Africa_ASBCB_Conference_on_Bioinformatics
                   * Basel_Computational_Biology_Conferenceâ ([BC2])
                   * International_Conference_on_Bioinformatics (InCoB)
                   * Computational_biology
                   * List_of_biological_databases
Related topics     * Sequencing
                   * Sequence_database
                   * Sequence_alignment
                   * Molecular_phylogenetics
    * [Category] Category
    * [Commons page] Commons
    * v
    * t
    * e
Informatics
              * Data_warehouse
              * Geoportal
              * Informatics_engineering
Design        * Information_management
              * Information_processing
              * Information_systems
              * Information_technology
              * Web_portal
              * Archival_informatics
              * Astroinformatics
              * Biodiversity_informatics
              * Bioimage_informatics
              * Bioinformatics
              * Business_informatics
              * Cheminformatics
              * Community_informatics
              * Computational_biology
              * Computational_informatics
              * Consumer_health_informatics
              * Development_informatics
              * Disease_informatics
              * Education_informatics
              * Energy_informatics
              * Engineering_Informatics
              * Environmental_informatics
Subfields     * Forest_informatics
              * Geoinformatics
              * Health_informatics
              * Hydroinformatics
              * Imaging_informatics
              * Irrigation_informatics
              * Laboratory_informatics
              * Legal_informatics
              * Materials_informatics
              * Medical_informatics
              * Music_informatics
              * Neuroinformatics
              * Pervasive_Informatics
              * Public_health_informatics
              * Social_informatics
              * Technical_informatics
              * Translational_research_informatics
              * Urban_informatics
    * v
    * t
    * e
Omics
                       * Cognitive_genomics
                       * Computational_genomics
                       * Comparative_genomics
                       * Functional_genomics
                       * Genome_project
Genomics                     o Human_Genome_Project
                       * Metagenomics
                       * Pangenomics
                       * Personal_genomics
                       * Social_genomics
                       * Structural_genomics
                       * Biochip
                       * Cheminformatics
                       * Chemogenomics
                       * Connectomics
                             o Human_Connectome_Project
                       * Glycomics
                       * Immunomics
                       * Lipidomics
Bioinformatics         * Metabolomics
                       * Microbiomics
                       * Nutrigenomics
                       * Paleopolyploidy
                       * Pharmacogenetics
                       * Pharmacogenomics
                       * Systems_biology
                       * Toxicogenomics
                       * Transcriptomics
                       * Proteomics
                             o Human_Proteome_Project
Structural_biology     * Call-map_proteomics
                       * Structure-based_drug_design
                       * Expression_proteomics
                       * 2-D_electrophoresis
                       * Mass_spectrometer
                       * Electrospray_ionization
Research tools         * Matrix-assisted_laser_desorption_ionization
                       * Matrix-assisted_laser_desorption_ionization-time_of
                         flight_mass_spectrometer
                       * Microfluidic-based_tools
                       * Isotope_affinity_tags
                       * National_Institutes_of_Health (USA)
Organizations          * DNA_Data_Bank_of_Japan (JP)
                       * European_Molecular_Biology_Laboratory (EU)
                       * Sanger_Centre (EN)
    * [Category] Category
    * v
    * t
    * e
Branches_of_biology
    * Anatomy
    * Astrobiology
    * Biochemistry
    * Biogeography
    * Biohistory
    * Biomechanics
    * Biophysics
    * Bioinformatics
    * Biostatistics
    * Botany
    * Cell_biology
    * Cellular_microbiology
    * Chemical_biology
    * Chronobiology
    * Computational_biology
    * Conservation_biology
    * Cryobiology
    * Cytogenetics
    * Developmental_biology
    * Ecology
    * Embryology
    * Epidemiology
    * Epigenetics
    * Evolutionary_biology
    * Freshwater_biology
    * Geobiology
    * Genetics
    * Genomics
    * Herpetology
    * Histology
    * Human_biology
    * Immunology
    * Marine_biology
    * Mathematical_biology
    * Microbiology
    * Molecular_biology
    * Mycology
    * Neontology
    * Neuroscience
    * Nutrition
    * Origin_of_life
    * Paleontology
    * Parasitology
    * Pathology
    * Pharmacology
    * Phylogenetics
    * Physiology
    * Quantum_biology
    * Sociobiology
    * Structural_biology
    * Systematics
    * Synthetic_biology
          o Xenobiology
    * Systems_biology
    * Taxonomy
    * Teratology
    * Toxicology
    * Virology
    * Virophysics
    * Zoology
             * History_of_biology
See also     * Nobel_Prize_in_Physiology_or_Medicine
             * Timeline_of_biology_and_organic_chemistry
    * v
    * t
    * e
Computer_science
Note: This template roughly follows the 2012 ACM_Computing_Classification
System.
                          * Printed_circuit_board
                          * Peripheral
                          * Integrated_circuit
Hardware                  * Very_Large_Scale_Integration
                          * Systems_on_Chip_(SoCs)
                          * Energy_consumption_(Green_computing)
                          * Electronic_design_automation
                          * Hardware_acceleration
                          * Computer_architecture
Computer systems          * Embedded_system
organization              * Real-time_computing
                          * Dependability
                          * Network_architecture
                          * Network_protocol
Networks                  * Network_components
                          * Network_scheduler
                          * Network_performance_evaluation
                          * Network_service
                          * Interpreter
                          * Middleware
Software organization     * Virtual_machine
                          * Operating_system
                          * Software_quality
                          * Programming_paradigm
                          * Programming_language
                          * Compiler
                          * Domain-specific_language
Software_notations        * Modeling_language
and tools                 * Software_framework
                          * Integrated_development_environment
                          * Software_configuration_management
                          * Software_library
                          * Software_repository
                          * Software_development_process
                          * Requirements_analysis
                          * Software_design
Software_development      * Software_construction
                          * Software_deployment
                          * Software_maintenance
                          * Programming_team
                          * Open-source_model
                          * Model_of_computation
                          * Formal_language
Theory_of_computation     * Automata_theory
                          * Computational_complexity_theory
                          * Logic
                          * Semantics
                          * Algorithm_design
                          * Analysis_of_algorithms
Algorithms                * Algorithmic_efficiency
                          * Randomized_algorithm
                          * Computational_geometry
                          * Discrete_mathematics
                          * Probability
Mathematics               * Statistics
of computing              * Mathematical_software
                          * Information_theory
                          * Mathematical_analysis
                          * Numerical_analysis
                          * Database_management_system
                          * Information_storage_systems
                          * Enterprise_information_system
                          * Social_information_systems
                          * Geographic_information_system
                          * Decision_support_system
Information               * Process_control_system
systems                   * Multimedia_information_system
                          * Data_mining
                          * Digital_library
                          * Computing_platform
                          * Digital_marketing
                          * World_Wide_Web
                          * Information_retrieval
                          * Cryptography
                          * Formal_methods
                          * Security_services
Security                  * Intrusion_detection_system
                          * Hardware_security
                          * Network_security
                          * Information_security
                          * Application_security
                          * Interaction_design
Humanâcomputer       * Social_computing
interaction               * Ubiquitous_computing
                          * Visualization
                          * Accessibility
                          * Concurrent_computing
                          * Parallel_computing
Concurrency               * Distributed_computing
                          * Multithreading
                          * Multiprocessing
                          * Natural_language_processing
                          * Knowledge_representation_and_reasoning
                          * Computer_vision
Artificial                * Automated_planning_and_scheduling
intelligence              * Search_methodology
                          * Control_method
                          * Philosophy_of_artificial_intelligence
                          * Distributed_artificial_intelligence
                          * Supervised_learning
                          * Unsupervised_learning
Machine_learning          * Reinforcement_learning
                          * Multi-task_learning
                          * Cross-validation
                          * Animation
                          * Rendering
                          * Image_manipulation
Graphics                  * Graphics_processing_unit
                          * Mixed_reality
                          * Virtual_reality
                          * Image_compression
                          * Solid_modeling
                          * E-commerce
                          * Enterprise_software
                          * Computational_mathematics
                          * Computational_physics
                          * Computational_chemistry
                          * Computational_biology
                          * Computational_social_science
                          * Computational_engineering
Applied                   * Computational_healthcare
computing                 * Digital_art
                          * Electronic_publishing
                          * Cyberwarfare
                          * Electronic_voting
                          * Video_games
                          * Word_processing
                          * Operations_research
                          * Educational_technology
                          * Document_management
    * [Wikipedia book] Book
    * [Category] Category
    * [Portal] Portal
    * [Outline] Outline
    * [WikiProject]WikiProject
    * [Commons page] Commons
    * v
    * t
    * e
Health_informatics
                                  * Clinic_management_system
                                  * Computerized_physician_order_entry
                                  * DICOM
Health_information_management     * Electronic_health_record
                                  * Health_information_technology
                                  * Hospital_information_system
                                  * Personal_health_record
                                  * Regional_Health_Information_Organization
                                  * Bioinformatics
                                  * Computational_biology
Subdisciplines                    * Consumer_health_informatics
                                  * Public_health_informatics
                                  * Translational_medicine
                                  * Translational_bioinformatics
                                  * Continuity_of_Care_Record
                                  * HRHIS
Medical_classification            * ICD
                                  * ISO_27799
                                  * LOINC
                                  * American_Association
                                  * Australasian_College
Professional organizations        * Brazilian_Society
                                  * European_Federation
                                  * Indian_Association
                                  * International_Association
                                  * Health_Level_7
                                  * Personalized_medicine / precision_medicine
Other concepts                    * List_of_medical_and_health_informatics
                                    journals
                                  * openEHR
    * [Issoria_lathonia.jpg]Biology_portal
    * [Tree_of_life.svg]Evolutionary_biology_portal
Authority_control [Edit_this_at_Wikidata]     * GND: 4611085-9
                                              * NDL: 00870420

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Bioinformatics&oldid=907326934"
Categories:
    * Spoken_articles
    * Bioinformatics
Hidden categories:
    * CS1_maint:_Multiple_names:_authors_list
    * Articles_with_short_description
    * Use_dmy_dates_from_July_2012
    * Articles_with_hAudio_microformats
    * Pages_including_recorded_pronunciations_(English)
    * All_articles_with_vague_or_ambiguous_time
    * Vague_or_ambiguous_time_from_September_2018
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_July_2015
    * Commons_category_link_from_Wikidata
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
    * Wikibooks
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * à¦¬à¦¾à¦à¦²à¦¾
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * GÄ©kÅ©yÅ©
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Ido
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * à²à²¨à³à²¨à²¡
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * Latina
    * LatvieÅ¡u
    * LÃ«tzebuergesch
    * Limburgs
    * Lingua_Franca_Nova
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Norsk_nynorsk
    * Novial
    * Occitan
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Shqip
    * Simple_English
    * SlovenÄina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * Winaray
    * ××Ö´×××©
    * ä¸­æ
Edit_links
    * This page was last edited on 22 July 2019, at 04:33 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
