The following text has been accessed from https://en.wikipedia.org/wiki/Epistasis at Fri Aug 9 04:03:13 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** Epistasis ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Not to be confused with epistaxis (bleeding from the nose), epitaxis
(deposition of a crystalline overlayer on a crystalline substrate), or epitasis
(in classical drama, the main action of a play).
The gene for total_baldness is epistatic to those for blond_hair or red_hair.
The hair-colour genes are hypostatic to the baldness gene. The baldness
phenotype supersedes genes for hair colour and so the effects are non-additive.
[citation_needed]
Epistasis is the phenomenon where the effect of one gene (locus) is dependent
on the presence of one or more 'modifier genes', i.e. the genetic background.
[1] Originally the term meant that the phenotypic effect of one gene is masked
by a different gene (locus).[2] Thus, epistatic mutations have different
effects in combination than individually. It was originally a concept from
genetics but is now used in biochemistry, computational_biology and
evolutionary_biology. It arises due to interactions, either between genes, or
within them, leading to non-linear effects. Epistasis has a large influence on
the shape of evolutionary_landscapes, which leads to profound consequences for
evolution and evolvability of phenotypic traits.
‚Å∞
***** Contents *****
    * 1_History
    * 2_Classification
          o 2.1_Additivity
          o 2.2_Magnitude_epistasis
          o 2.3_Sign_epistasis
          o 2.4_Haploid_organisms
          o 2.5_Diploid_organisms
    * 3_Genetic_and_molecular_causes
          o 3.1_Additivity
          o 3.2_Epistasis_between_genes
          o 3.3_Epistasis_within_genes
                # 3.3.1_Heterozygotic_epistasis
    * 4_Evolutionary_consequences
          o 4.1_Fitness_landscapes_and_evolvability
          o 4.2_Evolution_of_sex
    * 5_Methods_and_model_systems
          o 5.1_Regression_analysis
          o 5.2_Double_mutant_cycles
          o 5.3_Statistical_coupling_analysis
          o 5.4_Computational_prediction
    * 6_See_also
    * 7_References
    * 8_External_links
***** History[edit] *****
Understanding of epistasis has changed considerably through the history_of
genetics and so too has the use of the term. In early models of natural
selection devised in the early 20th century, each gene was considered to make
its own characteristic contribution to fitness, against an average background
of other genes. Some introductory courses still teach population_genetics this
way. Because of the way that the science of population_genetics was developed,
evolutionary_geneticists have tended to think of epistasis as the exception.
However, in general, the expression of any one allele depends in a complicated
way on many other alleles.
In classical_genetics, if genes A and B are mutated, and each mutation by
itself produces a unique phenotype but the two mutations together show the same
phenotype as the gene A mutation, then gene A is epistatic and gene B is
hypostatic. For example, the gene for total_baldness is epistatic to the gene
for brown_hair. In this sense, epistasis can be contrasted with genetic
dominance, which is an interaction between alleles at the same gene locus. As
the study of genetics developed, and with the advent of molecular_biology,
epistasis started to be studied in relation to Quantitative_Trait_Loci (QTL)
and polygenic_inheritance.
The effects of genes are now commonly quantifiable by assaying the magnitude of
a phenotype (e.g. height, pigmentation or growth_rate) or by biochemically
assaying protein activity (e.g. binding or catalysis). Increasingly
sophisticated computational and evolutionary_biology models aim to describe the
effects of epistasis on a genome-wide scale and the consequences of this for
evolution.[3][4][5] Since identification of epistatic pairs is challenging both
computationally and statistically, some studies try to prioritize epistatic
pairs.[6][7]
***** Classification[edit] *****
Quantitative_trait values after two mutations either alone (Ab and aB) or in
combination (AB). Bars contained in the grey box indicate the combined trait
value under different circumstances of epistasis. Upper panel indicates
epistasis between beneficial mutations (blue).[8][9] Lower panel indicates
epistasis between deleterious mutations (red).[10][11]
Since, on average, mutations are deleterious, random mutations to an organism
cause a decline in fitness. If all mutations are additive, fitness will fall
proportionally to mutation number (black line). When deleterious mutations
display negative (synergistic) epistasis, they are more deleterious in
combination than individually and so fitness falls with the number of mutations
at an increasing rate (upper, red line). When mutations display positive
(antagonistic) epistasis, effects of mutations are less severe in combination
than individually and so fitness falls at a decreasing rate (lower, blue line).
[10][11][12][13]
Terminology about epistasis can vary between scientific fields. Geneticists
often refer to wild_type and mutant alleles where the mutation is implicitly
deleterious and may talk in terms of genetic enhancement, synthetic_lethality
and genetic suppressors. Conversely, a biochemist may more frequently focus on
beneficial mutations and so explicitly state the effect of a mutation and use
terms such as reciprocal sign epistasis and compensatory mutation.[14]
Additionally, there are differences when looking at epistasis within a single
gene (biochemistry) and epistasis within a haploid or diploid genome
(genetics). In general, epistasis is used to denote the departure from
'independence' of the effects of different genetic loci. Confusion often arises
due to the varied interpretation of 'independence' among different branches of
biology.[15] The classifications below attempt to cover the various terms and
how they relate to one another.
**** Additivity[edit] ****
Two mutations are considered to be purely additive if the effect of the double
mutation is the sum of the effects of the single mutations. This occurs when
genes do not interact with each other, for example by acting through different
metabolic_pathways. Simple, additive traits were studied early on in the
history_of_genetics, however they are relatively rare, with most genes
exhibiting at least some level of epistatic interaction.[16][17]
**** Magnitude epistasis[edit] ****
When the double mutation has a fitter phenotype than expected from the effects
of the two single mutations, it is referred to as positive epistasis. Positive
epistasis between beneficial mutations generates greater improvements in
function than expected.[8][9] Positive epistasis between deleterious mutations
protects against the negative effects to cause a less severe fitness drop.[11]
Conversely, when two mutations together lead to a less fit phenotype than
expected from their effects when alone, it is called negative epistasis.[18]
[19] Negative epistasis between beneficial mutations causes smaller than
expected fitness improvements, whereas negative epistasis between deleterious
mutations causes greater-than-additive fitness drops.[10]
Independently, when the effect on fitness of two mutations is more radical than
expected from their effects when alone, it is referred to as synergistic
epistasis. The opposite situation, when the fitness difference of the double
mutant from the wild type is smaller than expected from the effects of the two
single mutations, it is called antagonistic epistasis.[13] Therefore, for
deleterious mutations, negative epistasis is also synergistic, while positive
epistasis is antagonistic; conversely, for advantageous mutations, positive
epistasis is synergistic, while negative epistasis is antagonistic.
The term genetic enhancement is sometimes used when a double (deleterious)
mutant has a more severe phenotype than the additive effects of the single
mutants. Strong positive epistasis is sometimes referred to by creationists as
irreducible_complexity (although most_examples_are_misidentified).
**** Sign epistasis[edit] ****
Sign epistasis[20] occurs when one mutation has the opposite effect when in the
presence of another mutation. This occurs when a mutation that is deleterious
on its own can enhance the effect of a particular beneficial mutation.[15] For
example, a large and complex brain is a waste of energy without a range of
sense_organs, but sense organs are made more useful by a large and complex
brain that can better process the information.
At its most extreme, reciprocal sign epistasis[21] occurs when two deleterious
genes are beneficial when together. For example, producing a toxin alone can
kill a bacterium, and producing a toxin_exporter alone can waste energy, but
producing both can improve fitness by killing competing_organisms.
Reciprocal sign epistasis also leads to genetic suppression whereby two
deleterious mutations are less harmful together than either one on its own,
i.e. one compensates for the other. This term can also apply sign epistasis
where the double mutant has a phenotype intermediate between those of the
single mutants, in which case the more severe single mutant phenotype is
suppressed by the other mutation or genetic condition. For example, in a
diploid organism, a hypomorphic (or partial loss-of-function) mutant phenotype
can be suppressed by knocking out one copy of a gene that acts oppositely in
the same pathway. In this case, the second gene is described as a "dominant
suppressor" of the hypomorphic mutant; "dominant" because the effect is seen
when one wild-type copy of the suppressor gene is present (i.e. even in a
heterozygote). For most genes, the phenotype of the heterozygous suppressor
mutation by itself would be wild type (because most genes are not haplo-
insufficient), so that the double mutant (suppressed) phenotype is intermediate
between those of the single mutants.
In non reciprocal sign epistasis, fitness of the mutant lies in the middle of
that of the extreme effects seen in reciprocal sign epistasis.
When two mutations are viable alone but lethal in combination, it is called
Synthetic_lethality or unlinked non-complementation.[22]
**** Haploid organisms[edit] ****
In a haploid organism with genotypes (at two loci) ab, Ab, aB or AB, we can
think of different forms of epistasis as affecting the magnitude of a phenotype
upon mutation individually (Ab and aB) or in combination (AB).
Interaction type                  ab Ab aB AB
No epistasis (additive)¬†         0  1  1  2  AB = Ab + aB + ab¬†
Positive (synergistic) epistasis  0  1  1  3  AB > Ab + aB + ab¬†
Negative (antagonistic) epistasis 0  1  1  1  AB < Ab + aB + ab¬†
Sign epistasis                    0  1  -1 2  AB has opposite sign to Ab or aB
Reciprocal sign epistasis         0  -1 -1 2  AB has opposite sign to Ab and aB
**** Diploid organisms[edit] ****
Epistasis in diploid organisms is further complicated by the presence of two
copies of each gene. Epistasis can occur between loci, but additionally,
interactions can occur between the two copies of each locus in heterozygotes.
For a two locus, two allele system, there are eight independent types of gene
interaction.[23]
Additive A locus   Additive B locus    Dominance A locus  Dominance B locus
   aa   aA AA         aa   aA   AA        aa   aA   AA       aa   aA   AA
bb 1    0  √¢¬Ä¬ì1 bb 1    1    1      bb √¢¬Ä¬11   √¢¬Ä¬ìbb √¢¬Ä¬√¢¬Ä¬√¢¬Ä¬ì1
bB 1    0  √¢¬Ä¬ì1 bB 0    0    0      bB √¢¬Ä¬11   √¢¬Ä¬ìbB 1    1    1
BB 1    0  √¢¬Ä¬ì1 BB √¢¬Ä¬√¢¬Ä¬√¢¬Ä¬ì1BB √¢¬Ä¬11   √¢¬Ä¬ìBB √¢¬Ä¬√¢¬Ä¬√¢¬Ä¬ì1
Additive by        Additive by         Dominance by       Dominance by
Additive Epistasis Dominance Epistasis Additive Epistasis Dominance Epistasis
   aa   aA AA         aa   aA   AA        aa   aA   AA       aa   aA   AA
bb 1    0  √¢¬Ä¬ì1 bb 1    0    √¢¬Ä¬ì1bb 1    √¢¬Ä¬11    bb √¢¬Ä¬11   √¢¬Ä¬ì1
bB 0    0  0       bB √¢¬Ä¬01   1      bB 0    0    0     bB 1    √¢¬Ä¬11
BB √¢¬Ä¬01 1       BB 1    0    √¢¬Ä¬ì1BB √¢¬Ä¬11   √¢¬Ä¬ìBB √¢¬Ä¬11   √¢¬Ä¬ì1
***** Genetic and molecular causes[edit] *****
**** Additivity[edit] ****
This can be the case when multiple genes act in parallel to achieve the same
effect. For example, when an organism is in need of phosphorus, multiple
enzymes that break down different phosphorylated components from the
environment may act additively to increase the amount of phosphorus available
to the organism. However, there inevitably comes a point where phosphorus is no
longer the limiting factor for growth and reproduction and so further
improvements in phosphorus metabolism have smaller or no effect (negative
epistasis). Some sets of mutations within genes have also been specifically
found to be additive.[24] It is now considered that strict additivity is the
exception, rather than the rule, since most genes interact with hundreds or
thousands of other genes.[16][17]
**** Epistasis between genes[edit] ****
Epistasis within the genomes of organisms occurs due to interactions between
the genes within the genome. This interaction may be direct if the genes encode
proteins that, for example, are separate components of a multi-component
protein (such as the ribosome), inhibit each other's activity, or if the
protein encoded by one gene modifies the other (such as by phosphorylation).
Alternatively the interaction may be indirect, where the genes encode
components of a metabolic_pathway or network, developmental_pathway, signalling
pathway or transcription_factor network. For example, the gene encoding the
enzyme that synthesizes penicillin is of no use to a fungus without the enzymes
that synthesize the necessary precursors in the metabolic pathway.
**** Epistasis within genes[edit] ****
Just as mutations in two separate genes can be non-additive if those genes
interact, mutations in two codons within a gene can be non-additive. In
genetics this is sometimes called intragenic complementation when one
deleterious mutation can be compensated for by a second mutation within that
gene. This occurs when the amino_acids within a protein interact. Due to the
complexity of protein folding and activity, additive mutations are rare.
Proteins are held in their tertiary_structure by a distributed, internal
network of cooperative interactions (hydrophobic, polar and covalent).[25]
Epistatic interactions occur whenever one mutation alters the local environment
of another residue (either by directly contacting it, or by inducing changes in
the protein structure).[26] For example, in a disulphide_bridge, a single
cysteine has no effect on protein_stability until a second is present at the
correct location at which point the two cysteines form a chemical_bond which
enhances the stability of the protein.[27] This would be observed as positive
epistasis where the double-cysteine variant had a much higher stability than
either of the single-cysteine variants. Conversely, when deleterious mutations
are introduced, proteins often exhibit mutational_robustness whereby as
stabilising interactions are destroyed the protein still functions until it
reaches some stability threshold at which point further destabilising mutations
have large, detrimental effects as the protein can no longer fold. This leads
to negative epistasis whereby mutations that have little effect alone have a
large, deleterious effect together.[28][29]
In enzymes, the protein structure orients a few, key amino_acids into precise
geometries to form an active_site to perform chemistry.[30] Since these active
site networks frequently require the cooperation of multiple components,
mutating any one of these components massively compromises activity, and so
mutating a second component has a relatively minor effect on the already
inactivated enzyme. For example, removing any member of the catalytic_triad of
many enzymes will reduce activity to levels low enough that the organism is no
longer viable.[31][32][33]
*** Heterozygotic epistasis[edit] ***
Diploid organisms contain two copies of each gene. If these are different
(heterozygous / heteroallelic), the two different copies of the allele may
interact with each other to cause epistasis. This is sometimes called allelic
complementation, or interallelic complementation. It may be caused by several
mechanisms, for example transvection, where an enhancer from one allele acts in
trans to activate transcription from the promoter of the second allele.
Alternately, trans-splicing of two non-functional RNA molecules may produce a
single, functional RNA. Similarly, at the protein level, proteins that function
as dimers may form a heterodimer composed of one protein from each alternate
gene and may display different properties to the homodimer of one or both
variants.
***** Evolutionary consequences[edit] *****
**** Fitness landscapes and evolvability[edit] ****
The top row indicates interactions between two genes that are either additive
(a), show positive epistasis (b) or reciprocal sign epistasis (c). Below are
fitness_landscapes which display greater and greater levels of global epistasis
between large numbers of genes. Purely additive interactions lead to a single
smooth peak (d), as increasing numbers of genes exhibit epistasis, the
landscape becomes more rugged (e) and when all genes interact epistatically the
landscape becomes so rugged that mutations have seemingly random effects (f).
See also: fitness_landscape and evolvability
In evolutionary_genetics, the sign of epistasis is usually more significant
than the magnitude of epistasis. This is because magnitude epistasis (positive
and negative) simply affects how beneficial mutations are together, however
sign epistasis affects whether mutation combinations are beneficial or
deleterious.[8]
A fitness_landscape is a representation of the fitness where all genotypes are
arranged in 2D space and the fitness of each genotype is represented by height
on a surface. It is frequently used as a visual metaphor for understanding
evolution as the process of moving uphill from one genotype to the next,
nearby, fitter genotype.[16]
If all mutations are additive, they can be acquired in any order and still give
a continuous uphill trajectory. The landscape is perfectly smooth, with only
one peak (global_maximum) and all sequences can evolve uphill to it by the
accumulation of beneficial mutations in any order. Conversely, if mutations
interact with one another by epistasis, the fitness landscape becomes rugged as
the effect of a mutation depends on the genetic background of other mutations.
[34] At its most extreme, interactions are so complex that the fitness is
√¢¬Ä¬òuncorrelated√¢¬Ä¬ô with gene sequence and the topology of the landscape is
random. This is referred to as a rugged fitness landscape and has profound
implications for the evolutionary_optimisation of organisms. If mutations are
deleterious in one combination but beneficial in another, the fittest genotypes
can only be accessed by accumulating mutations in one specific order. This
makes it more likely that organisms will get stuck at local_maxima in the
fitness landscape having acquired mutations in the 'wrong' order.[29][35] For
example, a variant of TEM1_√é¬≤-lactamase with 5 mutations is able to cleave
cefotaxime (a third generation antibiotic).[36] However, of the 120 possible
pathways to this 5-mutant variant, only 7% are accessible to evolution as the
remainder passed through fitness valleys where the combination of mutations
reduces activity. In contrast, changes in environment (and therefore the shape
of the fitness landscape) have been shown to provide escape from local maxima.
[29] In this example, selection in changing antibiotic environments resulted in
a "gateway mutation" which epistatically interacted in a positive manner with
other mutations along an evolutionary pathway, effectively crossing a fitness
valley. This gateway mutation alleviated the negative epistatic interactions of
other individually beneficial mutations, allowing them to better function in
concert. Complex environments or selections may therefore bypass local maxima
found in models assuming simple positive selection.
High epistasis is usually considered a constraining factor on evolution, and
improvements in a highly epistatic trait are considered to have lower
evolvability. This is because, in any given genetic background, very few
mutations will be beneficial, even though many mutations may need to occur to
eventually improve the trait. The lack of a smooth landscape makes it harder
for evolution to access fitness peaks. In highly rugged landscapes, fitness
valleys block access to some genes, and even if ridges exist that allow access,
these may be rare or prohibitively long.[37] Moreover, adaptation can move
proteins into more precarious or rugged regions of the fitness landscape.[38]
These shifting "fitness territories" may act to decelerate evolution and could
represent tradeoffs for adaptive traits.
The frustration of adaptive evolution by rugged fitness landscapes was
recognized as a potential force for the evolution of evolvability. Michael
Conrad in 1972 was the first to propose a mechanism for the evolution of
evolvability by noting that a mutation which smoothed the fitness landscape at
other loci could facilitate the production of advantageous mutations and
hitchhike along with them[39][40]. Rupert_Riedl in 1975 proposed that new genes
which produced the same phenotypic effects with a single mutation as other loci
with reciprocal sign epistasis would be a new means to attain a phenotype
otherwise too unlikely to occur by mutation.[41][42]
Rugged, epistatic fitness landscapes also affect the trajectories of evolution.
When a mutation has a large number of epistatic effects, each accumulated
mutation drastically changes the set of available beneficial_mutations.
Therefore, the evolutionary trajectory followed depends highly on which early
mutations were accepted. Thus, repeats of evolution from the same starting
point tend to diverge to different local maxima rather than converge on a
single global maximum as they would in a smooth, additive landscape.[43][44]
**** Evolution of sex[edit] ****
Main article: evolution_of_sexual_reproduction
Negative epistasis and sex are thought to be intimately correlated.
Experimentally, this idea has been tested in using digital simulations of
asexual and sexual populations. Over time, sexual populations move towards more
negative epistasis, or the lowering of fitness by two interacting alleles. It
is thought that negative epistasis allows individuals carrying the interacting
deleterious mutations to be removed from the populations efficiently. This
removes those alleles from the population, resulting in an overall more fit
population. This hypothesis was proposed by Alexey_Kondrashov, and is sometimes
known as the deterministic mutation hypothesis[45] and has also been tested
using artificial gene_networks.[18]
However, the evidence for this hypothesis has not always been straightforward
and the model proposed by Kondrashov has been criticized for assuming mutation
parameters far from real world observations.[46] In addition, in those tests
which used artificial gene networks, negative epistasis is only found in more
densely connected networks,[18] whereas empirical evidence indicates that
natural gene networks are sparsely connected,[47] and theory shows that
selection for robustness will favor more sparsely connected and minimally
complex networks.[47]
***** Methods and model systems[edit] *****
**** Regression analysis[edit] ****
Quantitative genetics focuses on genetic_variance due to genetic interactions.
Any two locus interactions at a particular gene frequency can be decomposed
into eight independent genetic effects using a weighted_regression. In this
regression, the observed two locus genetic effects are treated as dependent
variables and the "pure" genetic effects are used as the independent variables.
Because the regression is weighted, the partitioning among the variance
components will change as a function of gene frequency. By analogy it is
possible to expand this system to three or more loci, or to cytonuclear
interactions[48]
**** Double mutant cycles[edit] ****
When assaying epistasis within a gene, site-directed_mutagenesis can be used to
generate the different genes, and their protein_products can be assayed (e.g.
for stability or catalytic activity). This is sometimes called a double mutant
cycle and involves producing and assaying the wild type protein, the two single
mutants and the double mutant. Epistasis is measured as the difference between
the effects of the mutations together versus the sum of their individual
effects.[49] This can be expressed as a free energy of interaction. The same
methodology can be used to investigate the interactions between larger sets of
mutations but all combinations have to be produced and assayed. For example,
there are 120 different combinations of 5 mutations, some or all of which may
show epistasis...
**** Statistical coupling analysis[edit] ****
[[icon]] This section is empty. You can help by adding_to_it. (May 2014)
**** Computational prediction[edit] ****
Numerous computational methods have been developed for the detection and
characterization of epistasis. Many of these rely on machine_learning to detect
non-additive effects that might be missed by statistical approaches such as
linear regression. For example, multifactor_dimensionality_reduction (MDR) was
designed specifically for nonparametric and model-free detection of
combinations of genetic variants that are predictive of a phenotype such as
disease status in human populations.[50] [51]. Several of these approaches have
been broadly reviewed in the literature[52]. Even more recently, methods have
been developed that utilize insights from theoretical computer science (the
Hadamard_transform[53]), or have used methods (maximum-likelihood inference.
[54]) and others to distinguish epistatic effects from overall non-linearity in
genotype-phenotype map structure[55]
***** See also[edit] *****
    * Co-adaptation
    * Epistasis_and_functional_genomics
    * Mutation
    * Synthetic_viability
    * Synthetic_Lethality
    * Quantitative_trait_locus
    * Interactome (Genetic interaction network)
    * Fitness_landscape
    * Evolvability
    * Pleiotropy
    * Evolution_of_sexual_reproduction
***** References[edit] *****
   1. ^Gros PA, Le Nagard H, Tenaillon O (May 2009). "The_evolution_of
      epistasis_and_its_links_with_genetic_robustness,_complexity_and_drift_in
      a_phenotypic_model_of_adaptation". Genetics. 182 (1): 277√¢¬Ä¬ì93. doi:
      10.1534/genetics.108.099127. PMC¬†2674823. PMID¬†19279327.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Rieger R, Michaelis A, Green MM (1968), A glossary of genetics and
      cytogenetics: Classical and molecular, New York: Springer-Verlag,
      ISBN¬†9780387076683
   4. ^Szendro IG, Martijn F S, Franke J, Krug J, de Visser J, Arjan GM (16
      January 2013). "Quantitative analyses of empirical fitness landscapes".
      Journal of Statistical Mechanics: Theory and Experiment. 2013 (1):
      P01005. arXiv:1202.4378. Bibcode:2013JSMTE..01..005S. doi:10.1088/1742-
      5468/2013/01/P01005.
   5. ^Edlund JA, Adami C (Spring 2004). "Evolution_of_robustness_in_digital
      organisms" (PDF). Artificial Life. 10 (2): 167√¢¬Ä¬ì79.
      CiteSeerX¬†10.1.1.556.2318. doi:10.1162/106454604773563595. PMID¬†15107229.
   6. ^Chattopadhyay S (Spring 2019). "Genome-wide_interaction_and_pathway-
      based_identification_of_key_regulators_in_multiple_myeloma".
      Communications Biology. 4 (2): 89√¢¬Ä¬ì96. doi:10.1038/s42003-019-0329-2.
      PMC¬†6399257. PMID¬†30854481.
   7. ^Ayati M, Koyut√É¬ºrk M (2014-01-01). Prioritization of Genomic Locus Pairs
      for Testing Epistasis. Proceedings of the 5th ACM Conference on
      Bioinformatics, Computational Biology, and Health Informatics. BCB '14.
      New York, NY, USA. pp.¬†240√¢¬Ä¬ì248. doi:10.1145/2649387.2649449. ISBN¬†978-
      1-4503-2894-4.
   8. ^Piriyapongsa J, Ngamphiw C, Intarapanich A, Kulawonganunchai S,
      Assawamakin A, Bootchai C, Shaw PJ, Tongsima S (2012-12-13). "iLOCi:_a
      SNP_interaction_prioritization_technique_for_detecting_epistasis_in
      genome-wide_association_studies". BMC Genomics. 13 Suppl 7 (Suppl 7): S2.
      doi:10.1186/1471-2164-13-S7-S2. PMC¬†3521387. PMID¬†23281813.
   9. ^ a b cPhillips PC (November 2008). "Epistasis--the_essential_role_of
      gene_interactions_in_the_structure_and_evolution_of_genetic_systems".
      Nature Reviews. Genetics. 9 (11): 855√¢¬Ä¬ì67. doi:10.1038/nrg2452.
      PMC¬†2689140. PMID¬†18852697.
  10. ^ a bDomingo E, Sheldon J, Perales C (June 2012). "Viral_quasispecies
      evolution". Microbiology and Molecular Biology Reviews. 76 (2):
      159√¢¬Ä¬ì216. doi:10.1128/mmbr.05023-11. PMC¬†3372249. PMID¬†22688811.
  11. ^ a b cTokuriki N, Tawfik DS (October 2009). "Stability effects of
      mutations and protein evolvability". Current Opinion in Structural
      Biology. 19 (5): 596√¢¬Ä¬ì604. doi:10.1016/j.sbi.2009.08.003. PMID¬†19765975.
  12. ^ a b cHe X, Qian W, Wang Z, Li Y, Zhang J (March 2010). "Prevalent
      positive_epistasis_in_Escherichia_coli_and_Saccharomyces_cerevisiae
      metabolic_networks". Nature Genetics. 42 (3): 272√¢¬Ä¬ì6. doi:10.1038/
      ng.524. PMC¬†2837480. PMID¬†20101242.
  13. ^Ridley M (2004). Evolution (3rd ed.). Blackwell Publishing.
  14. ^ a bCharlesworth B, Charlesworth D (2010). Elements of Evolutionary
      Genetics. Roberts and Company Publishers.
  15. ^Ortlund EA, Bridgham JT, Redinbo MR, Thornton JW (September 2007).
      "Crystal_structure_of_an_ancient_protein:_evolution_by_conformational
      epistasis". Science. 317 (5844): 1544√¢¬Ä¬ì8. Bibcode:2007Sci...317.1544O.
      doi:10.1126/science.1142819. PMC¬†2519897. PMID¬†17702911.
  16. ^ a bCordell HJ (October 2002). "Epistasis: what it means, what it
      doesn't mean, and statistical methods to detect it in humans". Human
      Molecular Genetics. 11 (20): 2463√¢¬Ä¬ì8. CiteSeerX¬†10.1.1.719.4634. doi:
      10.1093/hmg/11.20.2463. PMID¬†12351582.
  17. ^ a b cKauffman SA (1993). The origins of order¬†: self-organization and
      selection in evolution ([Repr.]. ed.). New York: Oxford University Press.
      ISBN¬†978-0195079517.
  18. ^ a bBornscheuer UT, Huisman GW, Kazlauskas RJ, Lutz S, Moore JC, Robins
      K (May 2012). "Engineering the third wave of biocatalysis". Nature. 485
      (7397): 185√¢¬Ä¬ì94. Bibcode:2012Natur.485..185B. doi:10.1038/nature11117.
      PMID¬†22575958.
  19. ^ a b cAzevedo RB, Lohaus R, Srinivasan S, Dang KK, Burch CL (March
      2006). "Sexual reproduction selects for robustness and negative epistasis
      in artificial gene networks". Nature. 440 (7080): 87√¢¬Ä¬ì90. Bibcode:
      2006Natur.440...87A. doi:10.1038/nature04488. PMID¬†16511495.
  20. ^Bonhoeffer S, Chappey C, Parkin NT, Whitcomb JM, Petropoulos CJ
      (November 2004). "Evidence for positive epistasis in HIV-1". Science. 306
      (5701): 1547√¢¬Ä¬ì50. Bibcode:2004Sci...306.1547B. doi:10.1126/
      science.1101786. PMID¬†15567861.
  21. ^Weinreich DM, Watson RA, Chao L (June 2005). "Perspective: Sign
      epistasis and genetic constraint on evolutionary trajectories".
      Evolution; International Journal of Organic Evolution. 59 (6): 1165√¢¬Ä¬ì74.
      doi:10.1111/j.0014-3820.2005.tb01768.x. JSTOR¬†3448895. PMID¬†16050094.
  22. ^Poelwijk FJ, Kiviet DJ, Weinreich DM, Tans SJ (January 2007). "Empirical
      fitness landscapes reveal accessible evolutionary paths". Nature. 445
      (7126): 383√¢¬Ä¬ì6. Bibcode:2007Natur.445..383P. doi:10.1038/nature05451.
      PMID¬†17251971.
  23. ^French-Mischo S (July 2002). "Synthetic_Lethal_Mutations". Department of
      Microbiology, University of Illinois, Urbana. Archived from the_original
      on 2016-08-23. Retrieved 2017-08-03.
  24. ^Kempthorne O (1969). An_introduction_to_genetic_statistics. Iowa State
      University Press.
  25. ^Lunzer M, Miller SP, Felsheim R, Dean AM (October 2005). "The
      biochemical architecture of an ancient adaptive landscape". Science. 310
      (5747): 499√¢¬Ä¬ì501. Bibcode:2005Sci...310..499L. doi:10.1126/
      science.1115649. PMID¬†16239478.
  26. ^Shakhnovich BE, Deeds E, Delisi C, Shakhnovich E (March 2005). "Protein
      structure_and_evolutionary_history_determine_sequence_space_topology".
      Genome Research. 15 (3): 385√¢¬Ä¬ì92. arXiv:q-bio/0404040. doi:10.1101/
      gr.3133605. PMC¬†551565. PMID¬†15741509.
  27. ^Harms MJ, Thornton JW (August 2013). "Evolutionary_biochemistry:
      revealing_the_historical_and_physical_causes_of_protein_properties".
      Nature Reviews. Genetics. 14 (8): 559√¢¬Ä¬ì71. doi:10.1038/nrg3540.
      PMC¬†4418793. PMID¬†23864121.
  28. ^Witt D (2008). "Recent developments in disulfide bond formation".
      Synthesis. 2008 (16): 2491√¢¬Ä¬ì2509. doi:10.1055/s-2008-1067188.
  29. ^Bershtein S, Segal M, Bekerman R, Tokuriki N, Tawfik DS (December 2006).
      "Robustness-epistasis link shapes the fitness landscape of a randomly
      drifting protein". Nature. 444 (7121): 929√¢¬Ä¬ì32. Bibcode:
      2006Natur.444..929B. doi:10.1038/nature05385. PMID¬†17122770.
  30. ^ a b cSteinberg B, Ostermeier M (January 2016). "Environmental_changes
      bridge_evolutionary_valleys". Science Advances. 2 (1): e1500921. Bibcode:
      2016SciA....2E0921S. doi:10.1126/sciadv.1500921. PMC¬†4737206.
      PMID¬†26844293.
  31. ^Halabi N, Rivoire O, Leibler S, Ranganathan R (August 2009). "Protein
      sectors:_evolutionary_units_of_three-dimensional_structure". Cell. 138
      (4): 774√¢¬Ä¬ì86. doi:10.1016/j.cell.2009.07.038. PMC¬†3210731.
      PMID¬†19703402.
  32. ^Neet KE, Koshland DE (November 1966). "The_conversion_of_serine_at_the
      active_site_of_subtilisin_to_cysteine:_a_"chemical_mutation"".
      Proceedings of the National Academy of Sciences of the United States of
      America. 56 (5): 1606√¢¬Ä¬ì11. Bibcode:1966PNAS...56.1606N. doi:10.1073/
      pnas.56.5.1606. PMC¬†220044. PMID¬†5230319.
  33. ^Beveridge AJ (July 1996). "A_theoretical_study_of_the_active_sites_of
      papain_and_S195C_rat_trypsin:_implications_for_the_low_reactivity_of
      mutant_serine_proteinases". Protein Science. 5 (7): 1355√¢¬Ä¬ì65. doi:
      10.1002/pro.5560050714. PMC¬†2143470. PMID¬†8819168.
  34. ^Sigal IS, Harwood BG, Arentzen R (December 1982). "Thiol-beta-lactamase:
      replacement_of_the_active-site_serine_of_RTEM_beta-lactamase_by_a
      cysteine_residue". Proceedings of the National Academy of Sciences of the
      United States of America. 79 (23): 7157√¢¬Ä¬ì60. Bibcode:
      1982PNAS...79.7157S. doi:10.1073/pnas.79.23.7157. PMC¬†347297.
      PMID¬†6818541.
  35. ^Poelwijk FJ, T√Ñ¬Énase-Nicola S, Kiviet DJ, Tans SJ (March 2011).
      "Reciprocal_sign_epistasis_is_a_necessary_condition_for_multi-peaked
      fitness_landscapes" (PDF). Journal of Theoretical Biology. 272 (1):
      141√¢¬Ä¬ì4. doi:10.1016/j.jtbi.2010.12.015. PMID¬†21167837.
  36. ^Reetz MT, Sanchis J (September 2008). "Constructing and analyzing the
      fitness landscape of an experimental evolutionary process". ChemBioChem.
      9 (14): 2260√¢¬Ä¬ì7. doi:10.1002/cbic.200800371. PMID¬†18712749.
  37. ^Weinreich DM, Delaney NF, Depristo MA, Hartl DL (April 2006). "Darwinian
      evolution can follow only very few mutational paths to fitter proteins".
      Science. 312 (5770): 111√¢¬Ä¬ì4. Bibcode:2006Sci...312..111W. doi:10.1126/
      science.1123539. PMID¬†16601193.
  38. ^Gong LI, Suchard MA, Bloom JD (May 2013). "Stability-mediated_epistasis
      constrains_the_evolution_of_an_influenza_protein". eLife. 2: e00631. doi:
      10.7554/eLife.00631. PMC¬†3654441. PMID¬†23682315.
  39. ^Steinberg B, Ostermeier M (July 2016). "Shifting Fitness and Epistatic
      Landscapes Reflect Trade-offs along an Evolutionary Pathway". Journal of
      Molecular Biology. 428 (13): 2730√¢¬Ä¬ì43. doi:10.1016/j.jmb.2016.04.033.
      PMID¬†27173379.
  40. ^Conrad, Michael (1972). "The importance of molecular hierarchy in
      information processing". Towards a Theoretical Biology. Edinburgh
      University Press Edinburgh (4): 222.CS1 maint: Uses authors parameter
      (link)
  41. ^Conrad, Michael (1979). "Bootstrapping on the adaptive landscape".
      BioSystems. 11 (2√¢¬Ä¬ì3): 167√¢¬Ä¬ì182. doi:10.1016/0303-2647(79)90009-1. hdl:
      2027.42/23514.
  42. ^Riedl, Rupert J. (1975). Die Ordnung des Lebendigen: Systembedingungen
      der Evolution. Hamburg and Berlin: Parey.
  43. ^Riedl, Rupert J. (1977). "A systems-analytical approach to
      macroevolutionary phenomena". Quarterly Review of Biology. 52 (4):
      351√¢¬Ä¬ì370. doi:10.1086/410123.
  44. ^Lobkovsky AE, Wolf YI, Koonin EV (December 2011). "Predictability_of
      evolutionary_trajectories_in_fitness_landscapes". PLoS Computational
      Biology. 7 (12): e1002302. arXiv:1108.3590. Bibcode:2011PLSCB...7E2302L.
      doi:10.1371/journal.pcbi.1002302. PMC¬†3240586. PMID¬†22194675.
  45. ^Bridgham JT, Ortlund EA, Thornton JW (September 2009). "An_epistatic
      ratchet_constrains_the_direction_of_glucocorticoid_receptor_evolution".
      Nature. 461 (7263): 515√¢¬Ä¬ì9. Bibcode:2009Natur.461..515B. doi:10.1038/
      nature08249. PMC¬†6141187. PMID¬†19779450.
  46. ^Kondrashov AS (December 1988). "Deleterious mutations and the evolution
      of sexual reproduction". Nature. 336 (6198): 435√¢¬Ä¬ì40. Bibcode:
      1988Natur.336..435K. doi:10.1038/336435a0. PMID¬†3057385.
  47. ^MacCarthy T, Bergman A (July 2007). "Coevolution_of_robustness,
      epistasis,_and_recombination_favors_asexual_reproduction". Proceedings of
      the National Academy of Sciences of the United States of America. 104
      (31): 12801√¢¬Ä¬ì6. Bibcode:2007PNAS..10412801M. doi:10.1073/
      pnas.0705455104. PMC¬†1931480. PMID¬†17646644.
  48. ^ a bLeclerc RD (August 2008). "Survival_of_the_sparsest:_robust_gene
      networks_are_parsimonious". Molecular Systems Biology. 4 (213): 213. doi:
      10.1038/msb.2008.52. PMC¬†2538912. PMID¬†18682703.
  49. ^Wade MJ, Goodnight CJ (April 2006). "Cyto-nuclear epistasis: two-locus
      random genetic drift in hermaphroditic and dioecious species". Evolution;
      International Journal of Organic Evolution. 60 (4): 643√¢¬Ä¬ì59. doi:
      10.1554/05-019.1. PMID¬†16739448.
  50. ^Horovitz A (1996). "Double-mutant cycles: a powerful tool for analyzing
      protein structure and function". Folding & Design. 1 (6): R121√¢¬Ä¬ì6. doi:
      10.1016/s1359-0278(96)00056-9. PMID¬†9080186.
  51. ^Moore JH, Andrews PC (2015-01-01). "Epistasis analysis using multifactor
      dimensionality reduction". Epistasis. Methods in Molecular Biology. 1253.
      pp.¬†301√¢¬Ä¬ì14. doi:10.1007/978-1-4939-2155-3_16. ISBN¬†978-1-4939-2154-6.
      PMID¬†25403539.
  52. ^Moore JH, Williams SM, eds. (2015). Epistasis:_Methods_and_Protocols.
      Methods in Molecular Biology. Springer. ISBN¬†978-1-4939-2154-6.
  53. ^Cordell HJ (June 2009). "Detecting_gene-gene_interactions_that_underlie
      human_diseases". Nature Reviews. Genetics. 10 (6): 392√¢¬Ä¬ì404. doi:
      10.1038/nrg2579. PMC¬†2872761. PMID¬†19434077.
  54. ^Weinreich DM, Lan Y, Wylie CS, Heckendorn RB (December 2013). "Should
      evolutionary_geneticists_worry_about_higher-order_epistasis?". Current
      Opinion in Genetics & Development. Genetics of system biology. 23 (6):
      700√¢¬Ä¬ì7. doi:10.1016/j.gde.2013.10.007. PMC¬†4313208. PMID¬†24290990.
  55. ^Otwinowski J, McCandlish DM, Plotkin JB (August 2018). "Inferring_the
      shape_of_global_epistasis". Proceedings of the National Academy of
      Sciences of the United States of America. 115 (32): E7550√¢¬Ä¬ìE7558. doi:
      10.1073/pnas.1804015115. PMC¬†6094095. PMID¬†30037990.
  56. ^Sailer ZR, Harms MJ (March 2017). "Detecting_High-Order_Epistasis_in
      Nonlinear_Genotype-Phenotype_Maps". Genetics. 205 (3): 1079√¢¬Ä¬ì1088. doi:
      10.1534/genetics.116.195214. PMC¬†5340324. PMID¬†28100592.
***** External links[edit] *****
 Wikimedia Commons has media related to Epistasis.
    * INTERSNP - a software for genome-wide interaction analysis (GWIA) of
      case-control and case-only SNP data, including analysis of quantitative
      traits.
    * Science_Aid:_Epistasis High school (GCSE, Alevel) resource.
    * GeneticInteractions.org
    * Epistasis.org
    * v
    * t
    * e
The development of phenotype
                                 * Genotype√¢¬Ä¬ìphenotype_distinction
                                 * Norms_of_reaction
                                 * Gene√¢¬Ä¬ìenvironment_interaction
                                 * Gene√¢¬Ä¬ìenvironment_correlation
Key concepts                     * Operon
                                 * Heritability
                                 * Quantitative_genetics
                                 * Heterochrony
                                       o Neoteny
                                 * Heterotopy
                                 * Canalisation
                                 * Genetic_assimilation
                                 * Dominance
                                 * Epistasis
Genetic_architecture             * Fitness_landscape/evolutionary_landscape
                                 * Pleiotropy
                                 * Plasticity
                                 * Polygenic_inheritance
                                 * Transgressive_segregation
                                 * Sequence_space
                                 * Epigenetics
                                 * Maternal_effect
Non-genetic influences           * Genomic_imprinting
                                 * Dual_inheritance_theory
                                 * Polyphenism
                                 * Developmental_biology
                                 * Morphogenesis
                                       o Eyespot
Developmental architecture             o Pattern_formation
                                 * Segmentation
                                 * Metamerism
                                 * Modularity
                                 * Evolvability
Evolution of genetic systems     * Mutational_robustness
                                 * Neutral_networks
                                 * Evolution_of_sexual_reproduction
                                          * Regulation_of_gene_expression
                                          * Gene_regulatory_network
                                          * Developmental-genetic_toolkit
                             Systems      * Evolutionary_developmental_biology
                                          * Homeobox
                                          * Hedgehog_signaling_pathway
                                          * Notch_signaling_pathway
                                          * Homeotic_gene
Control of development                    * Hox_gene
                                          * Pax_genes
                                                o eyeless_gene
                                          * Distal-less
                             Elements     * Engrailed
                                          * cis-regulatory_element
                                          * Ligand
                                          * Morphogen
                                          * Cell_surface_receptor
                                          * Transcription_factor
                                 * C._H._Waddington
                                 * Richard_Lewontin
                                 * Fran√É¬ßois_Jacob + Jacques_Monod
                                       o Lac_operon
Influential figures              * Eric_F._Wieschaus
                                 * Christiane_N√É¬ºsslein-Volhard
                                 * William_McGinnis
                                 * Mike_Levine
                                 * Sean_B._Carroll
                                       o Endless_Forms_Most_Beautiful
Debates                          * Nature_versus_nurture
                                 * Morphogenetic_field
Index_of_evolutionary_biology_articles

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Epistasis&oldid=909933984"
Categories:
    * Classical_genetics
Hidden categories:
    * CS1:_long_volume_value
    * CS1_maint:_Uses_authors_parameter
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_November_2017
    * Articles_to_be_expanded_from_May_2014
    * All_articles_to_be_expanded
    * Articles_with_empty_sections_from_May_2014
    * All_articles_with_empty_sections
    * Articles_using_small_message_boxes
    * Commons_category_link_is_on_Wikidata
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
‚Å∞
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
‚Å∞
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Bosanski
    * Catal√É¬†
    * √Ñ¬åe√Ö¬°tina
    * Dansk
    * Deutsch
    * √é¬ï√é¬ª√é¬ª√é¬∑√é¬Ω√é¬π√é¬∫√é¬¨
    * Espa√É¬±ol
    * Euskara
    * √ô¬Å√ò¬ß√ò¬±√ò¬≥√õ¬å
    * Fran√É¬ßais
    * Galego
    * √≠¬ï¬ú√™¬µ¬≠√¨¬ñ¬¥
    * √†¬§¬π√†¬§¬ø√†¬§¬®√†¬•¬ç√†¬§¬¶√†¬•¬Ä
    * Bahasa_Indonesia
    * Italiano
    * √ó¬¢√ó¬ë√ó¬®√ó¬ô√ó¬™
    * √ê¬ö√ë¬ã√ë¬Ä√ê¬≥√ë¬ã√ê¬∑√ë¬á√ê¬∞
    * Latvie√Ö¬°u
    * Magyar
    * Nederlands
    * √¶¬ó¬•√¶¬ú¬¨√®¬™¬û
    * O√ä¬ªzbekcha/√ë¬û√ê¬∑√ê¬±√ê¬µ√ê¬∫√ë¬á√ê¬∞
    * Polski
    * Portugu√É¬™s
    * Rom√É¬¢n√Ñ¬É
    * √ê¬†√ë¬É√ë¬Å√ë¬Å√ê¬∫√ê¬∏√ê¬π
    * Simple_English
    * √ê¬°√ë¬Ä√ê¬ø√ë¬Å√ê¬∫√ê¬∏_/_srpski
    * Suomi
    * √†¬π¬Ñ√†¬∏¬ó√†¬∏¬¢
    * √ê¬£√ê¬∫√ë¬Ä√ê¬∞√ë¬ó√ê¬Ω√ë¬Å√ë¬å√ê¬∫√ê¬∞
    * Ti√°¬∫¬øng_Vi√°¬ª¬át
    * √ß¬≤¬µ√®¬™¬û
    * √§¬∏¬≠√¶¬ñ¬á
Edit_links
    * This page was last edited on 8 August 2019, at 15:40¬†(UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. Wikipedia√Ç¬Æ is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
