<!DOCTYPE html>
<html class="client-nojs" lang="en" dir="ltr">
<head>
<meta charset="UTF-8"/>
<title>Enzyme kinetics - Wikipedia</title>
<script>document.documentElement.className=document.documentElement.className.replace(/(^|\s)client-nojs(\s|$)/,"$1client-js$2");RLCONF={"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":!1,"wgNamespaceNumber":0,"wgPageName":"Enzyme_kinetics","wgTitle":"Enzyme kinetics","wgCurRevisionId":908666171,"wgRevisionId":908666171,"wgArticleId":3043886,"wgIsArticle":!0,"wgIsRedirect":!1,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1: long volume value","CS1 German-language sources (de)","Wikipedia indefinitely move-protected pages","Use dmy dates from August 2018","Featured articles","Pages that use a deprecated format of the chem tags","Enzyme kinetics","Catalysis"],"wgBreakFrames":!1,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November",
"December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Enzyme_kinetics","wgRelevantArticleId":3043886,"wgRequestId":"XUEquwpAICEAAGZ8lkkAAAAY","wgCSPNonce":!1,"wgIsProbablyEditable":!0,"wgRelevantPageIsProbablyEditable":!0,"wgRestrictionEdit":[],"wgRestrictionMove":["sysop"],"wgMediaViewerOnClick":!0,"wgMediaViewerEnabledByDefault":!0,"wgPopupsReferencePreviews":!1,"wgPopupsConflictsWithNavPopupGadget":!1,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en"},"wgMFDisplayWikibaseDescriptions":{"search":!0,"nearby":!0,"watchlist":!0,"tagline":!1},"wgWMESchemaEditAttemptStepOversample":!1,"wgPoweredByHHVM":!0,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgWikibaseItemId":"Q883112","wgCentralAuthMobileDomain":!1,"wgEditSubmitButtonLabelPublish":!0};RLSTATE={"ext.gadget.charinsert-styles":"ready",
"ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.math.styles":"ready","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"};RLPAGEMODULES=["ext.math.scripts","ext.cite.ux-enhancements","site","mediawiki.page.startup","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar",
"ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.compactlinks","ext.uls.interface","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});});</script>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.math.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;raw=1&amp;skin=vector"></script>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector"/>
<link rel="stylesheet" href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector"/>
<meta name="generator" content="MediaWiki 1.34.0-wmf.15"/>
<meta name="referrer" content="origin"/>
<meta name="referrer" content="origin-when-crossorigin"/>
<meta name="referrer" content="origin-when-cross-origin"/>
<meta property="og:image" content="https://upload.wikimedia.org/wikipedia/commons/6/6d/EcDHFR_raytraced.png"/>
<link rel="alternate" href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Enzyme_kinetics"/>
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=Enzyme_kinetics&amp;action=edit"/>
<link rel="edit" title="Edit this page" href="/w/index.php?title=Enzyme_kinetics&amp;action=edit"/>
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png"/>
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico"/>
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)"/>
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd"/>
<link rel="license" href="//creativecommons.org/licenses/by-sa/3.0/"/>
<link rel="canonical" href="https://en.wikipedia.org/wiki/Enzyme_kinetics"/>
<link rel="dns-prefetch" href="//login.wikimedia.org"/>
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 9]><script src="/w/resources/lib/html5shiv/html5shiv.js"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-Enzyme_kinetics rootpage-Enzyme_kinetics skin-vector action-view">
<div id="mw-page-base" class="noprint"></div>
<div id="mw-head-base" class="noprint"></div>
<div id="content" class="mw-body" role="main">
	<a id="top"></a>
	<div id="siteNotice" class="mw-body-content"><!-- CentralNotice --></div>
	<div class="mw-indicators mw-body-content">
<div id="mw-indicator-featured-star" class="mw-indicator"><a href="/wiki/Wikipedia:Featured_articles" title="This is a featured article. Click here for more information."><img alt="This is a featured article. Click here for more information." src="//upload.wikimedia.org/wikipedia/en/thumb/e/e7/Cscr-featured.svg/20px-Cscr-featured.svg.png" decoding="async" width="20" height="19" srcset="//upload.wikimedia.org/wikipedia/en/thumb/e/e7/Cscr-featured.svg/30px-Cscr-featured.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/e/e7/Cscr-featured.svg/40px-Cscr-featured.svg.png 2x" data-file-width="462" data-file-height="438" /></a></div>
</div>

	<h1 id="firstHeading" class="firstHeading" lang="en">Enzyme kinetics</h1>
	
	<div id="bodyContent" class="mw-body-content">
		<div id="siteSub" class="noprint">From Wikipedia, the free encyclopedia</div>
		<div id="contentSub"></div>
		
		
		
		<div id="jump-to-nav"></div>
		<a class="mw-jump-link" href="#mw-head">Jump to navigation</a>
		<a class="mw-jump-link" href="#p-search">Jump to search</a>
		<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><p class="mw-empty-elt">
</p>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:EcDHFR_raytraced.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/6d/EcDHFR_raytraced.png/300px-EcDHFR_raytraced.png" decoding="async" width="300" height="400" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/6d/EcDHFR_raytraced.png/450px-EcDHFR_raytraced.png 1.5x, //upload.wikimedia.org/wikipedia/commons/6/6d/EcDHFR_raytraced.png 2x" data-file-width="600" data-file-height="800" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:EcDHFR_raytraced.png" class="internal" title="Enlarge"></a></div><a href="/wiki/Dihydrofolate_reductase" title="Dihydrofolate reductase">Dihydrofolate reductase</a> from <i><a href="/wiki/Escherichia_coli" title="Escherichia coli">E. coli</a></i> with its two substrates <a href="/wiki/Dihydrofolic_acid" title="Dihydrofolic acid">dihydrofolate</a> (right) and <a href="/wiki/Nicotinamide_adenine_dinucleotide_phosphate" title="Nicotinamide adenine dinucleotide phosphate">NADPH</a> (left), bound in the active site. The protein is shown as a <a href="/wiki/Ribbon_diagram" title="Ribbon diagram">ribbon diagram</a>, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from <a rel="nofollow" class="external text" href="http://www.rcsb.org/pdb/explore.do?structureId=7DFR">7DFR</a>.</div></div></div>
<p><b>Enzyme kinetics</b> is the study of the <a href="/wiki/Chemical_reaction" title="Chemical reaction">chemical reactions</a> that are <a href="/wiki/Catalyst" class="mw-redirect" title="Catalyst">catalysed</a> by <a href="/wiki/Enzymes" class="mw-redirect" title="Enzymes">enzymes</a>. In enzyme kinetics, the <a href="/wiki/Reaction_rate" title="Reaction rate">reaction rate</a> is measured and the effects of varying the conditions of the reaction are investigated. Studying an enzyme's <a href="/wiki/Chemical_kinetics" title="Chemical kinetics">kinetics</a> in this way can reveal the catalytic mechanism of this enzyme, its role in <a href="/wiki/Metabolism" title="Metabolism">metabolism</a>, how its activity is controlled, and how a <a href="/wiki/Drug" title="Drug">drug</a> or an agonist might <a href="/wiki/Enzyme_inhibitor" title="Enzyme inhibitor">inhibit</a> the enzyme.
</p><p>Enzymes are usually <a href="/wiki/Protein" title="Protein">protein</a> <a href="/wiki/Molecule" title="Molecule">molecules</a> that manipulate other molecules—the enzymes' <a href="/wiki/Substrate_(biochemistry)" class="mw-redirect" title="Substrate (biochemistry)">substrates</a>. These target molecules bind to an enzyme's <a href="/wiki/Active_site" title="Active site">active site</a> and are transformed into <a href="/wiki/Product_(biology)" class="mw-redirect" title="Product (biology)">products</a> through a series of steps known as the <a href="/wiki/Enzyme_catalysis" title="Enzyme catalysis">enzymatic mechanism</a>
</p>
<dl><dd>E + S ⇄ ES ⇄ ES* ⇄ EP ⇄ E + P</dd></dl>
<p>These mechanisms can be divided into single-substrate and multiple-substrate mechanisms. Kinetic studies on enzymes that only bind one substrate, such as <a href="/wiki/Triosephosphateisomerase" class="mw-redirect" title="Triosephosphateisomerase">triosephosphate isomerase</a>, aim to measure the <a href="/wiki/Dissociation_constant" title="Dissociation constant">affinity</a> with which the enzyme binds this substrate and the turnover rate. Some other examples of enzymes are phosphofructokinase and hexokinase, both of which are important for cellular respiration (glycolysis).
</p><p>When enzymes bind multiple substrates, such as <a href="/wiki/Dihydrofolate_reductase" title="Dihydrofolate reductase">dihydrofolate reductase</a> (shown right), enzyme kinetics can also show the sequence in which these substrates bind and the sequence in which products are released.  An example of enzymes that bind a single substrate and release multiple products are <a href="/wiki/Protease" title="Protease">proteases</a>, which cleave one protein substrate into two polypeptide products. Others join two substrates together, such as <a href="/wiki/DNA_polymerase" title="DNA polymerase">DNA polymerase</a> linking a <a href="/wiki/Nucleotide" title="Nucleotide">nucleotide</a> to <a href="/wiki/DNA" title="DNA">DNA</a>. Although these mechanisms are often a complex series of steps, there is typically one <i>rate-determining step</i> that determines the overall kinetics.  This <a href="/wiki/Rate-determining_step" title="Rate-determining step">rate-determining step</a> may be a chemical reaction or a <a href="/wiki/Conformational_isomerism" title="Conformational isomerism">conformational</a> change of the enzyme or substrates, such as those involved in the release of product(s) from the enzyme.
</p><p>Knowledge of the <a href="/wiki/Protein_structure" title="Protein structure">enzyme's structure</a> is helpful in interpreting kinetic data.  For example, the structure can suggest how substrates and products bind during catalysis; what changes occur during the reaction; and even the role of particular <a href="/wiki/Amino_acid" title="Amino acid">amino acid</a> residues in the mechanism.  Some enzymes change shape significantly during the mechanism; in such cases, it is helpful to determine the enzyme structure with and without bound substrate analogues that do not undergo the enzymatic reaction.
</p><p>Not all biological catalysts are protein enzymes; <a href="/wiki/RNA" title="RNA">RNA</a>-based catalysts such as <a href="/wiki/Ribozymes" class="mw-redirect" title="Ribozymes">ribozymes</a> and <a href="/wiki/Ribosomes" class="mw-redirect" title="Ribosomes">ribosomes</a> are essential to many cellular functions, such as <a href="/wiki/Splicing_(genetics)" class="mw-redirect" title="Splicing (genetics)">RNA splicing</a> and <a href="/wiki/Translation_(biology)" title="Translation (biology)">translation</a>. The main difference between ribozymes and enzymes is that RNA catalysts are composed of nucleotides, whereas enzymes are composed of amino acids.  Ribozymes also perform a more limited set of reactions, although their <a href="/wiki/Reaction_mechanism" title="Reaction mechanism">reaction mechanisms</a> and kinetics can be analysed and classified by the same methods.
</p>
<div id="toc" class="toc"><input type="checkbox" role="button" id="toctogglecheckbox" class="toctogglecheckbox" style="display:none" /><div class="toctitle" lang="en" dir="ltr"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#General_principles"><span class="tocnumber">1</span> <span class="toctext">General principles</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Enzyme_assays"><span class="tocnumber">2</span> <span class="toctext">Enzyme assays</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#Single-substrate_reactions"><span class="tocnumber">3</span> <span class="toctext">Single-substrate reactions</span></a>
<ul>
<li class="toclevel-2 tocsection-4"><a href="#Michaelis–Menten_kinetics"><span class="tocnumber">3.1</span> <span class="toctext">Michaelis–Menten kinetics</span></a></li>
<li class="toclevel-2 tocsection-5"><a href="#Direct_use_of_the_Michaelis–Menten_equation_for_time_course_kinetic_analysis"><span class="tocnumber">3.2</span> <span class="toctext">Direct use of the Michaelis–Menten equation for time course kinetic analysis</span></a></li>
<li class="toclevel-2 tocsection-6"><a href="#Linear_plots_of_the_Michaelis–Menten_equation"><span class="tocnumber">3.3</span> <span class="toctext">Linear plots of the Michaelis–Menten equation</span></a></li>
<li class="toclevel-2 tocsection-7"><a href="#Practical_significance_of_kinetic_constants"><span class="tocnumber">3.4</span> <span class="toctext">Practical significance of kinetic constants</span></a></li>
<li class="toclevel-2 tocsection-8"><a href="#Michaelis–Menten_kinetics_with_intermediate"><span class="tocnumber">3.5</span> <span class="toctext">Michaelis–Menten kinetics with intermediate</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-9"><a href="#Multi-substrate_reactions"><span class="tocnumber">4</span> <span class="toctext">Multi-substrate reactions</span></a>
<ul>
<li class="toclevel-2 tocsection-10"><a href="#Ternary-complex_mechanisms"><span class="tocnumber">4.1</span> <span class="toctext">Ternary-complex mechanisms</span></a></li>
<li class="toclevel-2 tocsection-11"><a href="#Ping–pong_mechanisms"><span class="tocnumber">4.2</span> <span class="toctext">Ping–pong mechanisms</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-12"><a href="#Reversible_catalysis_and_the_Haldane_equation"><span class="tocnumber">5</span> <span class="toctext">Reversible catalysis and the Haldane equation</span></a></li>
<li class="toclevel-1 tocsection-13"><a href="#Non-Michaelis–Menten_kinetics"><span class="tocnumber">6</span> <span class="toctext">Non-Michaelis–Menten kinetics</span></a></li>
<li class="toclevel-1 tocsection-14"><a href="#Pre-steady-state_kinetics"><span class="tocnumber">7</span> <span class="toctext">Pre-steady-state kinetics</span></a></li>
<li class="toclevel-1 tocsection-15"><a href="#Chemical_mechanism"><span class="tocnumber">8</span> <span class="toctext">Chemical mechanism</span></a></li>
<li class="toclevel-1 tocsection-16"><a href="#Enzyme_inhibition_and_activation"><span class="tocnumber">9</span> <span class="toctext">Enzyme inhibition and activation</span></a>
<ul>
<li class="toclevel-2 tocsection-17"><a href="#Reversible_inhibitors"><span class="tocnumber">9.1</span> <span class="toctext">Reversible inhibitors</span></a></li>
<li class="toclevel-2 tocsection-18"><a href="#Irreversible_inhibitors"><span class="tocnumber">9.2</span> <span class="toctext">Irreversible inhibitors</span></a></li>
<li class="toclevel-2 tocsection-19"><a href="#Philosophical_discourse_on_reversibility_and_irreversibility_of_inhibition"><span class="tocnumber">9.3</span> <span class="toctext">Philosophical discourse on reversibility and irreversibility of inhibition</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-20"><a href="#Mechanisms_of_catalysis"><span class="tocnumber">10</span> <span class="toctext">Mechanisms of catalysis</span></a></li>
<li class="toclevel-1 tocsection-21"><a href="#History"><span class="tocnumber">11</span> <span class="toctext">History</span></a></li>
<li class="toclevel-1 tocsection-22"><a href="#Software"><span class="tocnumber">12</span> <span class="toctext">Software</span></a>
<ul>
<li class="toclevel-2 tocsection-23"><a href="#ENZO"><span class="tocnumber">12.1</span> <span class="toctext">ENZO</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-24"><a href="#See_also"><span class="tocnumber">13</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-25"><a href="#Footnotes"><span class="tocnumber">14</span> <span class="toctext">Footnotes</span></a></li>
<li class="toclevel-1 tocsection-26"><a href="#References"><span class="tocnumber">15</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1 tocsection-27"><a href="#Further_reading"><span class="tocnumber">16</span> <span class="toctext">Further reading</span></a></li>
<li class="toclevel-1 tocsection-28"><a href="#External_links"><span class="tocnumber">17</span> <span class="toctext">External links</span></a></li>
</ul>
</div>

<h2><span class="mw-headline" id="General_principles">General principles</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=1" title="Edit section: General principles">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:KinEnzymo(en).svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/09/KinEnzymo%28en%29.svg/300px-KinEnzymo%28en%29.svg.png" decoding="async" width="300" height="167" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/09/KinEnzymo%28en%29.svg/450px-KinEnzymo%28en%29.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/09/KinEnzymo%28en%29.svg/600px-KinEnzymo%28en%29.svg.png 2x" data-file-width="990" data-file-height="550" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:KinEnzymo(en).svg" class="internal" title="Enlarge"></a></div>As larger amounts of <a href="/wiki/Substrate_(biochemistry)" class="mw-redirect" title="Substrate (biochemistry)">substrate</a> are added to a reaction, the available enzyme <a href="/wiki/Binding_site" title="Binding site">binding sites</a> become filled to the limit of <a href="/wiki/Michaelis-Menten_kinetics" class="mw-redirect" title="Michaelis-Menten kinetics"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\max }}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\max }}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/ee5a8472bc1a42aac13cd279661c00b5a2d2ec84" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:4.646ex; height:2.509ex;" alt="V_{\max }"/></span></a>.  Beyond this limit the enzyme is saturated with substrate and the reaction rate ceases to increase.</div></div></div>
<p>The reaction catalysed by an enzyme uses exactly the same reactants and produces exactly the same products as the uncatalysed reaction. Like other <a href="/wiki/Catalysts" class="mw-redirect" title="Catalysts">catalysts</a>, enzymes do not alter the position of <a href="/wiki/Chemical_equilibrium" title="Chemical equilibrium">equilibrium</a> between substrates and products.<sup id="cite_ref-1" class="reference"><a href="#cite_note-1">&#91;1&#93;</a></sup> However, unlike uncatalysed chemical reactions, enzyme-catalysed reactions display saturation kinetics.  For a given enzyme concentration and for relatively low substrate concentrations, the reaction rate increases linearly with substrate concentration; the enzyme molecules are largely free to catalyse the reaction, and increasing substrate concentration means an increasing rate at which the enzyme and substrate molecules encounter one another.  However, at relatively high substrate concentrations, the reaction rate <a href="/wiki/Asymptote" title="Asymptote">asymptotically</a> approaches the theoretical maximum; the enzyme active sites are almost all occupied by substrates resulting in saturation, and the reaction rate is determined by the intrinsic turnover rate of the enzyme.<sup id="cite_ref-Fromm_H.J._2012_2-0" class="reference"><a href="#cite_note-Fromm_H.J._2012-2">&#91;2&#93;</a></sup> The substrate concentration midway between these two limiting cases is denoted by <i>K</i><sub>M</sub>. Thus, <i>K</i><sub>M</sub> is the substrate concentration at which the reaction velocity is half of the maximum velocity.<sup id="cite_ref-Fromm_H.J._2012_2-1" class="reference"><a href="#cite_note-Fromm_H.J._2012-2">&#91;2&#93;</a></sup>
</p><p>The two most important kinetic properties of an enzyme are how easily the enzyme becomes saturated with a particular substrate, and the maximum rate it can achieve. Knowing these properties suggests what an enzyme might do in the cell and can show how the enzyme will respond to changes in these conditions.
</p>
<h2><span class="mw-headline" id="Enzyme_assays">Enzyme assays</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=2" title="Edit section: Enzyme assays">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Enzyme_assay" title="Enzyme assay">Enzyme assay</a></div>
<div class="thumb tleft"><div class="thumbinner" style="width:252px;"><a href="/wiki/File:Enzyme_progress_curve.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/8/86/Enzyme_progress_curve.svg/250px-Enzyme_progress_curve.svg.png" decoding="async" width="250" height="195" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/8/86/Enzyme_progress_curve.svg/375px-Enzyme_progress_curve.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/8/86/Enzyme_progress_curve.svg/500px-Enzyme_progress_curve.svg.png 2x" data-file-width="467" data-file-height="364" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Enzyme_progress_curve.svg" class="internal" title="Enlarge"></a></div>Progress curve for an enzyme reaction. The slope in the initial rate period is the <b>initial rate of reaction</b> <i>v</i>.  The <a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Michaelis–Menten equation</a> describes how this slope varies with the concentration of substrate.</div></div></div>
<p><a href="/wiki/Enzyme_assay" title="Enzyme assay">Enzyme assays</a> are laboratory procedures that measure the rate of enzyme reactions. Since enzymes are not consumed by the reactions they catalyse, enzyme assays usually follow changes in the concentration of either substrates or products to measure the rate of reaction. There are many methods of measurement. <a href="/wiki/Ultraviolet-visible_spectroscopy" class="mw-redirect" title="Ultraviolet-visible spectroscopy">Spectrophotometric</a> assays observe change in the <a href="/wiki/Absorbance" title="Absorbance">absorbance</a> of light between products and reactants; radiometric assays involve the incorporation or release of <a href="/wiki/Radioactivity" class="mw-redirect" title="Radioactivity">radioactivity</a> to measure the amount of product made over time. Spectrophotometric assays are most convenient since they allow the rate of the reaction to be measured continuously. Although radiometric assays require the removal and counting of samples (i.e., they are discontinuous assays) they are usually extremely sensitive and can measure very low levels of enzyme activity.<sup id="cite_ref-3" class="reference"><a href="#cite_note-3">&#91;3&#93;</a></sup>  An analogous approach is to use <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a> to monitor the incorporation or release of <a href="/wiki/Stable_isotope" class="mw-redirect" title="Stable isotope">stable isotopes</a> as substrate is converted into product.
</p><p>The most sensitive enzyme assays use <a href="/wiki/Laser" title="Laser">lasers</a> focused through a <a href="/wiki/Microscope" title="Microscope">microscope</a> to observe changes in single enzyme molecules as they catalyse their reactions. These measurements either use changes in the <a href="/wiki/Fluorescence" title="Fluorescence">fluorescence</a> of <a href="/wiki/Cofactor_(biochemistry)" title="Cofactor (biochemistry)">cofactors</a> during an enzyme's reaction mechanism, or of <a href="/wiki/Fluorescent_dyes" class="mw-redirect" title="Fluorescent dyes">fluorescent dyes</a> added onto specific sites of the <a href="/wiki/Protein" title="Protein">protein</a> to report movements that occur during catalysis.<sup id="cite_ref-4" class="reference"><a href="#cite_note-4">&#91;4&#93;</a></sup> These studies are providing a new view of the kinetics and dynamics of single enzymes, as opposed to traditional enzyme kinetics, which observes the average behaviour of populations of millions of enzyme molecules.<sup id="cite_ref-5" class="reference"><a href="#cite_note-5">&#91;5&#93;</a></sup><sup id="cite_ref-6" class="reference"><a href="#cite_note-6">&#91;6&#93;</a></sup>
</p><p>An example progress curve for an enzyme assay is shown above. The enzyme produces product at an initial rate that is approximately linear for a short period after the start of the reaction.  As the reaction proceeds and substrate is consumed, the rate continuously slows (so long as substrate is not still at saturating levels). To measure the initial (and maximal) rate, enzyme assays are typically carried out while the reaction has progressed only a few percent towards total completion.  The length of the initial rate period depends on the assay conditions and can range from milliseconds to hours.  However, equipment for rapidly mixing liquids allows fast kinetic measurements on initial rates of less than one second.<sup id="cite_ref-7" class="reference"><a href="#cite_note-7">&#91;7&#93;</a></sup> These very rapid assays are essential for measuring pre-steady-state kinetics, which are discussed below.
</p><p>Most enzyme kinetics studies concentrate on this initial, approximately linear part of enzyme reactions. However, it is also possible to measure the complete reaction curve and fit this data to a non-linear <a href="/wiki/Rate_equation" title="Rate equation">rate equation</a>. This way of measuring enzyme reactions is called progress-curve analysis.<sup id="cite_ref-8" class="reference"><a href="#cite_note-8">&#91;8&#93;</a></sup> This approach is useful as an alternative to <a href="/w/index.php?title=Rapid_kinetics&amp;action=edit&amp;redlink=1" class="new" title="Rapid kinetics (page does not exist)">rapid kinetics</a> when the initial rate is too fast to measure accurately.
</p>
<h2><span class="mw-headline" id="Single-substrate_reactions">Single-substrate reactions</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=3" title="Edit section: Single-substrate reactions">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Enzymes with single-substrate mechanisms include <a href="/wiki/Isomerase" title="Isomerase">isomerases</a> such as <a href="/wiki/Triosephosphateisomerase" class="mw-redirect" title="Triosephosphateisomerase">triosephosphateisomerase</a> or <a href="/wiki/Bisphosphoglycerate_mutase" title="Bisphosphoglycerate mutase">bisphosphoglycerate mutase</a>, intramolecular <a href="/wiki/Lyase" title="Lyase">lyases</a> such as <a href="/wiki/Adenylate_cyclase" class="mw-redirect" title="Adenylate cyclase">adenylate cyclase</a> and the <a href="/wiki/Ribozyme" title="Ribozyme">hammerhead ribozyme</a>, an RNA lyase.<sup id="cite_ref-9" class="reference"><a href="#cite_note-9">&#91;9&#93;</a></sup> However, some enzymes that only have a single substrate do not fall into this category of mechanisms. <a href="/wiki/Catalase" title="Catalase">Catalase</a> is an example of this, as the enzyme reacts with a first molecule of <a href="/wiki/Hydrogen_peroxide" title="Hydrogen peroxide">hydrogen peroxide</a> substrate, becomes oxidised and is then reduced by a second molecule of substrate. Although a single substrate is involved, the existence of a modified enzyme intermediate means that the mechanism of catalase is actually a ping–pong mechanism, a type of mechanism that is discussed in the <i>Multi-substrate reactions</i> section below.
</p>
<h3><span id="Michaelis.E2.80.93Menten_kinetics"></span><span class="mw-headline" id="Michaelis–Menten_kinetics">Michaelis–Menten kinetics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=4" title="Edit section: Michaelis–Menten kinetics">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<style data-mw-deduplicate="TemplateStyles:r889293853/mw-parser-output/.tmulti">.mw-parser-output .tmulti .thumbinner{display:flex;flex-direction:column}.mw-parser-output .tmulti .trow{display:flex;flex-direction:row;clear:left;flex-wrap:wrap;width:100%;box-sizing:border-box}.mw-parser-output .tmulti .tsingle{margin:1px;float:left}.mw-parser-output .tmulti .theader{clear:both;font-weight:bold;text-align:center;align-self:center;background-color:transparent;width:100%}.mw-parser-output .tmulti .thumbcaption{text-align:left;background-color:transparent}.mw-parser-output .tmulti .text-align-left{text-align:left}.mw-parser-output .tmulti .text-align-right{text-align:right}.mw-parser-output .tmulti .text-align-center{text-align:center}@media all and (max-width:720px){.mw-parser-output .tmulti .thumbinner{width:100%!important;box-sizing:border-box;max-width:none!important;align-items:center}.mw-parser-output .tmulti .trow{justify-content:center}.mw-parser-output .tmulti .tsingle{float:none!important;max-width:100%!important;box-sizing:border-box;text-align:center}.mw-parser-output .tmulti .thumbcaption{text-align:center}}</style><div class="thumb tmulti tright"><div class="thumbinner" style="width:329px;max-width:329px"><div class="trow"><div class="tsingle" style="width:327px;max-width:327px"><div class="thumbimage"><a href="/wiki/File:Enzyme_mechanism_2.svg" class="image"><img alt="Schematic reaction diagrams for uncatalzyed (Substrate to Product) and catalyzed (Enzyme + Substrate to Enzyme/Substrate complex to Enzyme + Product)" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/69/Enzyme_mechanism_2.svg/325px-Enzyme_mechanism_2.svg.png" decoding="async" width="325" height="102" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/69/Enzyme_mechanism_2.svg/488px-Enzyme_mechanism_2.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/69/Enzyme_mechanism_2.svg/650px-Enzyme_mechanism_2.svg.png 2x" data-file-width="400" data-file-height="125" /></a></div><div class="thumbcaption">A chemical reaction mechanism with or without <a href="/wiki/Enzyme_catalysis" title="Enzyme catalysis">enzyme catalysis</a>. The enzyme (E) binds <a href="/wiki/Substrate_(chemistry)" title="Substrate (chemistry)">substrate</a> (S) to produce <a href="/wiki/Product_(chemistry)" title="Product (chemistry)">product</a> (P).</div></div></div><div class="trow"><div class="tsingle" style="width:327px;max-width:327px"><div class="thumbimage"><a href="/wiki/File:Michaelis_Menten_curve_2.svg" class="image"><img alt="A two dimensional plot of substrate concentration (x axis) vs. reaction rate (y axis). The shape of the curve is hyperbolic. The rate of the reaction is zero at zero concentration of substrate and the rate asymptotically reaches a maximum at high substrate concentration." src="//upload.wikimedia.org/wikipedia/commons/thumb/8/83/Michaelis_Menten_curve_2.svg/325px-Michaelis_Menten_curve_2.svg.png" decoding="async" width="325" height="139" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/8/83/Michaelis_Menten_curve_2.svg/488px-Michaelis_Menten_curve_2.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/8/83/Michaelis_Menten_curve_2.svg/650px-Michaelis_Menten_curve_2.svg.png 2x" data-file-width="1200" data-file-height="512" /></a></div><div class="thumbcaption"><a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Saturation curve</a> for an enzyme reaction showing the relation between the substrate concentration and reaction rate.</div></div></div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Michaelis–Menten kinetics</a></div>
<p>Kinetic model to explain single substrate enzyme catalyzed reactions was first proposed by Henri (1903) and Michaelis and Menten (1913). Henri developed the theoretical part of the model while Michaelis and Menten did a experimental work to prove the validity of the model. 
</p><p>Majority of the enzymes are proteins and they speeds up the rate of biochemical reactions by decreasing the activation energy. During the process enzymes combines with the substrate and convert it into product. Enzymes may have single or multiple substrate binding sites (catalytic site). As substrate concentration goes on increasing, catalytic sites may get filled progressively. In the initial period velocity of the enzyme catalyzed reaction is directly proportional to substrate concentration. Michaelis and Menten specifically focused on the initial period of the reaction (initial velocity)  to develop this model.
</p><p>As enzyme-catalysed reactions are saturable, their rate of catalysis does not show a linear response to increasing substrate. If the initial rate of the reaction is measured over a range of substrate concentrations (denoted as [S]), the initial reaction rate (<span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle v_{0}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/60faad24775635f4722ccc438093dbbfe05f34ae" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:2.182ex; height:2.009ex;" alt="v_{0}"/></span>) increases as [S] increases, as shown on the right. However, as [S] gets higher, the enzyme becomes saturated with substrate and the initial rate reaches <i>V</i><sub>max</sub>, the enzyme's maximum rate.
</p><p>The <a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Michaelis–Menten kinetic model of a single-substrate reaction</a> is shown on the right. There is an initial <a href="/wiki/Chemical_kinetics" title="Chemical kinetics">bimolecular reaction</a> between the enzyme E and substrate S to form the enzyme–substrate complex ES. The rate of enzymatic reaction increases with the increase of the substrate concentration up to a certain level called V<sub>max</sub>; at V<sub>max</sub>, increase in substrate concentration does not cause any increase in reaction rate as there is no more enzyme (E) available for reacting with substrate (S). Here, the rate of reaction becomes dependent on the ES complex and the reaction becomes a <a href="/wiki/Chemical_kinetics" title="Chemical kinetics">unimolecular reaction</a> with an order of zero. Though the enzymatic mechanism for the <a href="/wiki/Chemical_kinetics" title="Chemical kinetics">unimolecular reaction</a> <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {ES -&gt;[k_{cat}] E + P}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mtext>ES</mtext>
          <mrow class="MJX-TeXAtom-REL">
            <mover>
              <mo>&#x2192;</mo>
              <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>c</mi>
                    <mi>a</mi>
                    <mi>t</mi>
                  </mrow>
                </msub>
              </mpadded>
            </mover>
          </mrow>
          <mtext>E</mtext>
          <mo>+</mo>
          <mtext>P</mtext>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {ES -&gt;[k_{cat}] E + P}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/299f3433b9ca64a864deef13f572a0127a2d14e0" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.505ex; margin-top: -0.396ex; width:14.282ex; height:4.009ex;" alt="{\displaystyle {\ce {ES -&gt;[k_{cat}] E + P}}}"/></span> can be quite complex, there is typically one rate-determining enzymatic step that allows this reaction to be modelled as a single catalytic step with an apparent unimolecular rate constant <i>k</i><sub>cat</sub>.
If the reaction path proceeds over one or several intermediates, <i>k</i><sub>cat</sub> will be a function of several elementary rate constants, whereas in the simplest case of a single elementary reaction (e.g. no intermediates) it will be identical to the elementary unimolecular rate constant <i>k</i><sub>2</sub>. The apparent unimolecular rate constant <i>k</i><sub>cat</sub> is also called <i>turnover number</i> and denotes the maximum number of enzymatic reactions catalysed per second.
</p><p>The <a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Michaelis–Menten equation</a><sup id="cite_ref-10" class="reference"><a href="#cite_note-10">&#91;10&#93;</a></sup> describes how the (initial) reaction rate <i>v</i><sub>0</sub> depends on the position of the substrate-binding <a href="/wiki/Chemical_equilibrium" title="Chemical equilibrium">equilibrium</a> and the rate constant <i>k</i><sub>2</sub>.
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle v_{0}={\frac {V_{\max }[{{\ce {S}}}]}{K_{M}+[{{\ce {S}}}]}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <msub>
                <mi>V</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo movablelimits="true" form="prefix">max</mo>
                </mrow>
              </msub>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mtext>S</mtext>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
            </mrow>
            <mrow>
              <msub>
                <mi>K</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>M</mi>
                </mrow>
              </msub>
              <mo>+</mo>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mtext>S</mtext>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}={\frac {V_{\max }[{{\ce {S}}}]}{K_{M}+[{{\ce {S}}}]}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/8f2f1d1e9d417b925f380340d6d3581d4006672f" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.671ex; width:15.475ex; height:6.509ex;" alt="{\displaystyle v_{0}={\frac {V_{\max }[{\ce {S}}]}{K_{M}+[{\ce {S}}]}}}"/></span>&#160;&#160;&#160;&#160;<i>(Michaelis–Menten equation)</i></dd></dl>
<p>with the constants
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\begin{aligned}K_{M}\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{2}+k_{-1}}{k_{1}}}\approx K_{D}\\V_{\max }\ &amp;{\stackrel {\mathrm {def} }{=}}\ k_{cat}{{\ce {[E]}}}_{tot}\end{aligned}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mtable columnalign="right left right left right left right left right left right left" rowspacing="3pt" columnspacing="0em 2em 0em 2em 0em 2em 0em 2em 0em 2em 0em" displaystyle="true">
            <mtr>
              <mtd>
                <msub>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>M</mi>
                  </mrow>
                </msub>
                <mtext>&#xA0;</mtext>
              </mtd>
              <mtd>
                <mi></mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-REL">
                    <mover>
                      <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                        <mo>=</mo>
                      </mrow>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mi mathvariant="normal">d</mi>
                          <mi mathvariant="normal">e</mi>
                          <mi mathvariant="normal">f</mi>
                        </mrow>
                      </mrow>
                    </mover>
                  </mrow>
                </mrow>
                <mtext>&#xA0;</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <mrow>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>2</mn>
                        </mrow>
                      </msub>
                      <mo>+</mo>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                          <mn>1</mn>
                        </mrow>
                      </msub>
                    </mrow>
                    <msub>
                      <mi>k</mi>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mn>1</mn>
                      </mrow>
                    </msub>
                  </mfrac>
                </mrow>
                <mo>&#x2248;<!-- ≈ --></mo>
                <msub>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>D</mi>
                  </mrow>
                </msub>
              </mtd>
            </mtr>
            <mtr>
              <mtd>
                <msub>
                  <mi>V</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mo movablelimits="true" form="prefix">max</mo>
                  </mrow>
                </msub>
                <mtext>&#xA0;</mtext>
              </mtd>
              <mtd>
                <mi></mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-REL">
                    <mover>
                      <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                        <mo>=</mo>
                      </mrow>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mi mathvariant="normal">d</mi>
                          <mi mathvariant="normal">e</mi>
                          <mi mathvariant="normal">f</mi>
                        </mrow>
                      </mrow>
                    </mover>
                  </mrow>
                </mrow>
                <mtext>&#xA0;</mtext>
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>c</mi>
                    <mi>a</mi>
                    <mi>t</mi>
                  </mrow>
                </msub>
                <msub>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mrow class="MJX-TeXAtom-ORD">
                        <mo stretchy="false">[</mo>
                        <mtext>E</mtext>
                        <mo stretchy="false">]</mo>
                      </mrow>
                    </mrow>
                  </mrow>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>t</mi>
                    <mi>o</mi>
                    <mi>t</mi>
                  </mrow>
                </msub>
              </mtd>
            </mtr>
          </mtable>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\begin{aligned}K_{M}\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{2}+k_{-1}}{k_{1}}}\approx K_{D}\\V_{\max }\ &amp;{\stackrel {\mathrm {def} }{=}}\ k_{cat}{{\ce {[E]}}}_{tot}\end{aligned}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1714252e04d803899ef6ad5b75c074d0f9ebc50c" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -4.123ex; margin-bottom: -0.215ex; width:24.965ex; height:9.843ex;" alt="{\displaystyle {\begin{aligned}K_{M}\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{2}+k_{-1}}{k_{1}}}\approx K_{D}\\V_{\max }\ &amp;{\stackrel {\mathrm {def} }{=}}\ k_{cat}{\ce {[E]}}_{tot}\end{aligned}}}"/></span></dd></dl>
<p>This Michaelis–Menten equation is the basis for most single-substrate enzyme kinetics. Two  crucial assumptions underlie this equation (apart from the general assumption about the mechanism only involving no intermediate or product inhibition, and there is no <a href="/wiki/Allosteric_regulation" title="Allosteric regulation">allostericity</a> or <a href="/wiki/Cooperative_binding" title="Cooperative binding">cooperativity</a>). The first assumption is  the so-called 
<a href="/wiki/Steady_state_(chemistry)" title="Steady state (chemistry)">quasi-steady-state assumption</a> (or pseudo-steady-state hypothesis), namely that the concentration of the substrate-bound enzyme (and hence also the unbound enzyme) changes much more slowly than those of the product and substrate and thus the change over time of the complex can be set to zero
<span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle d{{\ce {[ES]}}}/{dt}\;{\overset {!}{=}}\;0}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mi>d</mi>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mo stretchy="false">[</mo>
              <mtext>ES</mtext>
              <mo stretchy="false">]</mo>
            </mrow>
          </mrow>
        </mrow>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <mrow class="MJX-TeXAtom-ORD">
          <mi>d</mi>
          <mi>t</mi>
        </mrow>
        <mspace width="thickmathspace" />
        <mrow class="MJX-TeXAtom-ORD">
          <mover>
            <mo>=</mo>
            <mo>!</mo>
          </mover>
        </mrow>
        <mspace width="thickmathspace" />
        <mn>0</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle d{{\ce {[ES]}}}/{dt}\;{\overset {!}{=}}\;0}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/0eea1ce2a30471bd05b46fe979bb4f12e365b4d5" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:12.864ex; height:3.843ex;" alt="{\displaystyle d{\ce {[ES]}}/{dt}\;{\overset {!}{=}}\;0}"/></span>. The second assumption is that the total enzyme concentration does not change over time, thus <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {[E]}}_{\text{tot}}={\ce {[E]}}+{\ce {[ES]}}\;{\overset {!}{=}}\;{\text{const}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mo stretchy="false">[</mo>
              <mtext>E</mtext>
              <mo stretchy="false">]</mo>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>tot</mtext>
          </mrow>
        </msub>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>E</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
        </mrow>
        <mo>+</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>ES</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
        </mrow>
        <mspace width="thickmathspace" />
        <mrow class="MJX-TeXAtom-ORD">
          <mover>
            <mo>=</mo>
            <mo>!</mo>
          </mover>
        </mrow>
        <mspace width="thickmathspace" />
        <mrow class="MJX-TeXAtom-ORD">
          <mtext>const</mtext>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {[E]}}_{\text{tot}}={\ce {[E]}}+{\ce {[ES]}}\;{\overset {!}{=}}\;{\text{const}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/5c082974766078275f236f456e12426c4ea02fc8" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:26.601ex; height:4.009ex;" alt="{\displaystyle {\ce {[E]}}_{\text{tot}}={\ce {[E]}}+{\ce {[ES]}}\;{\overset {!}{=}}\;{\text{const}}}"/></span>.
A complete derivation can be found <a href="/wiki/Michaelis%E2%80%93Menten_kinetics#Equation" title="Michaelis–Menten kinetics">here</a>.
</p><p>The Michaelis constant <i>K</i><sub>M</sub> is experimentally defined as the concentration at which the rate of the enzyme reaction is half <i>V</i><sub>max</sub>, which can be verified by substituting [S] = <i>K</i><sub>M</sub> into the Michaelis–Menten equation and can also be seen graphically. If the rate-determining enzymatic step is slow compared to substrate dissociation (<span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{2}\ll k_{-1}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mo>&#x226A;<!-- ≪ --></mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{2}\ll k_{-1}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1d9d943519e45cd086baea0024f3ef86b905dc95" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:9.424ex; height:2.509ex;" alt="k_{2}\ll k_{{-1}}"/></span>), the Michaelis constant <i>K</i><sub>M</sub> is roughly the <a href="/wiki/Dissociation_constant" title="Dissociation constant">dissociation constant</a> <i>K</i><sub>D</sub> of the ES complex.
</p><p>If <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {[S]}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>S</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {[S]}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/e5909e9989dfe9306325e8dab287928f3c984ee3" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:2.586ex; height:2.843ex;" alt="{\displaystyle {\ce {[S]}}}"/></span> is small compared to <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/14296e7a6cf6d829e6dd8bbb40421855edc4de67" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:3.932ex; height:2.509ex;" alt="K_M"/></span> then the term <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle [{{\ce {S}}}]/(K_{M}+[{{\ce {S}}}])\approx [{{\ce {S}}}]/K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>S</mtext>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
        <mo>+</mo>
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>S</mtext>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo stretchy="false">)</mo>
        <mo>&#x2248;<!-- ≈ --></mo>
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>S</mtext>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [{{\ce {S}}}]/(K_{M}+[{{\ce {S}}}])\approx [{{\ce {S}}}]/K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1d1befa5e00217f79ed63dc6ba5c6a15d78d5425" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:25.696ex; height:2.843ex;" alt="{\displaystyle [{\ce {S}}]/(K_{M}+[{\ce {S}}])\approx [{\ce {S}}]/K_{M}}"/></span> and also very little ES complex is formed, thus <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {[E]_{\rm {tot}}\approx [E]}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <msubsup>
            <mrow class="MJX-TeXAtom-ORD">
              <mo stretchy="false">[</mo>
              <mtext>E</mtext>
              <mo stretchy="false">]</mo>
            </mrow>
            <mrow class="MJX-TeXAtom-ORD">
              <mrow class="MJX-TeXAtom-ORD">
                <mtext>tot</mtext>
              </mrow>
            </mrow>
            <mrow class="MJX-TeXAtom-ORD">
              <mspace width="0pt" height="0pt" depth=".2em" />
            </mrow>
          </msubsup>
          <mo>&#x2248;<!-- ≈ --></mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>E</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {[E]_{\rm {tot}}\approx [E]}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/f6ccda6e543afca4e4287f635c5f1a4931ca93e1" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:11.185ex; height:2.843ex;" alt="{\displaystyle {\ce {[E]_{\rm {tot}}\approx [E]}}}"/></span>.  Therefore, the rate of product formation is
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle v_{0}\approx {\frac {k_{cat}}{K_{M}}}{\ce {[E][S]}}\qquad \qquad {\text{if }}[{\ce {S}}]\ll K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo>&#x2248;<!-- ≈ --></mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <msub>
              <mi>k</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>c</mi>
                <mi>a</mi>
                <mi>t</mi>
              </mrow>
            </msub>
            <msub>
              <mi>K</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>M</mi>
              </mrow>
            </msub>
          </mfrac>
        </mrow>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>E</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mo stretchy="false">[</mo>
            <mtext>S</mtext>
            <mo stretchy="false">]</mo>
          </mrow>
        </mrow>
        <mspace width="2em" />
        <mspace width="2em" />
        <mrow class="MJX-TeXAtom-ORD">
          <mtext>if&#xA0;</mtext>
        </mrow>
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mtext>S</mtext>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo>&#x226A;<!-- ≪ --></mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}\approx {\frac {k_{cat}}{K_{M}}}{\ce {[E][S]}}\qquad \qquad {\text{if }}[{\ce {S}}]\ll K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/596b2c4659de250ffbd0b65c085402f9fd16735d" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.171ex; width:36.874ex; height:5.676ex;" alt="{\displaystyle v_{0}\approx {\frac {k_{cat}}{K_{M}}}{\ce {[E][S]}}\qquad \qquad {\text{if }}[{\ce {S}}]\ll K_{M}}"/></span></dd></dl>
<p>Thus the product formation rate depends on the enzyme concentration as well as on the substrate concentration, the equation resembles a bimolecular reaction with a corresponding pseudo-second order rate constant <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{2}/K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{2}/K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/5fbb1e6cbe74a78948d774119cead039ca23d146" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:7.36ex; height:2.843ex;" alt="k_{2}/K_{M}"/></span>. This constant is a measure of <a href="/wiki/Catalytic_efficiency" class="mw-redirect" title="Catalytic efficiency">catalytic efficiency</a>. The most efficient enzymes reach a <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{2}/K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{2}/K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/5fbb1e6cbe74a78948d774119cead039ca23d146" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:7.36ex; height:2.843ex;" alt="k_{2}/K_{M}"/></span> in the range of <span class="texhtml">10<sup>8</sup> – 10<sup>10</sup>&#160;<a href="/wiki/Molar_concentration" title="Molar concentration">M</a><sup>−1</sup>&#160;<a href="/wiki/Second" title="Second">s</a><sup>−1</sup></span>. These enzymes are so efficient they effectively catalyse a reaction each time they encounter a substrate molecule and have thus reached an upper theoretical limit for efficiency (<a href="/wiki/Diffusion_limited_enzyme" title="Diffusion limited enzyme">diffusion limit</a>); and are sometimes referred to as <a href="/wiki/Diffusion_limited_enzyme" title="Diffusion limited enzyme">kinetically perfect enzymes</a>.<sup id="cite_ref-11" class="reference"><a href="#cite_note-11">&#91;11&#93;</a></sup> But most enzymes are far from perfect: the average values of <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{2}/K_{\rm {M}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">M</mi>
            </mrow>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{2}/K_{\rm {M}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/ad8370899ee9ff7938234bdcffc4bb1fdbe9f704" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:7.14ex; height:2.843ex;" alt="{\displaystyle k_{2}/K_{\rm {M}}}"/></span> and <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{2}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{2}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/c51b4ba57ee596d8435fc4ed76703ca3a2fc444a" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:2.265ex; height:2.509ex;" alt="k_{{2}}"/></span> are about <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle 10^{5}{\rm {s}}^{-1}{\rm {M}}^{-1}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msup>
          <mn>10</mn>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>5</mn>
          </mrow>
        </msup>
        <msup>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">s</mi>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msup>
        <msup>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">M</mi>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msup>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle 10^{5}{\rm {s}}^{-1}{\rm {M}}^{-1}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/ef512037b4da4d312f89dc4e35a5ca44cee2a187" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.338ex; width:11.092ex; height:2.676ex;" alt="{\displaystyle 10^{5}{\rm {s}}^{-1}{\rm {M}}^{-1}}"/></span> and <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle 10{\rm {s}}^{-1}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mn>10</mn>
        <msup>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">s</mi>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msup>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle 10{\rm {s}}^{-1}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.338ex; width:5.574ex; height:2.676ex;" alt="{\displaystyle 10{\rm {s}}^{-1}}"/></span>, respectively.<sup id="cite_ref-Bar-Even_2011_12-0" class="reference"><a href="#cite_note-Bar-Even_2011-12">&#91;12&#93;</a></sup>
</p>
<h3><span id="Direct_use_of_the_Michaelis.E2.80.93Menten_equation_for_time_course_kinetic_analysis"></span><span class="mw-headline" id="Direct_use_of_the_Michaelis–Menten_equation_for_time_course_kinetic_analysis">Direct use of the Michaelis–Menten equation for time course kinetic analysis</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=5" title="Edit section: Direct use of the Michaelis–Menten equation for time course kinetic analysis">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">Further information: <a href="/wiki/Rate_equation" title="Rate equation">Rate equation</a></div><div role="note" class="hatnote navigation-not-searchable">Further information: <a href="/wiki/Reaction_Progress_Kinetic_Analysis" class="mw-redirect" title="Reaction Progress Kinetic Analysis">Reaction Progress Kinetic Analysis</a></div>
<p>The observed velocities predicted by the Michaelis–Menten equation can be used to directly model the <a href="/wiki/Reaction_progress_kinetic_analysis" title="Reaction progress kinetic analysis">time course disappearance of substrate</a> and the production of product through  incorporation of the Michaelis–Menten equation into the equation for first order chemical kinetics.  This can only be achieved however if one recognises the problem associated with the use of <a href="/wiki/Euler%27s_number" class="mw-redirect" title="Euler&#39;s number">Euler's number</a> in the description of first order chemical kinetics. i.e. <i>e</i><sup>−<i>k</i></sup> is a split constant that introduces a systematic error into calculations and can be rewritten as a single constant which represents the remaining substrate after each time period.<sup id="cite_ref-13" class="reference"><a href="#cite_note-13">&#91;13&#93;</a></sup>
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [S]=[S]_{0}(1-k)^{t}\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <mo stretchy="false">]</mo>
        <mo>=</mo>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo stretchy="false">(</mo>
        <mn>1</mn>
        <mo>&#x2212;<!-- − --></mo>
        <mi>k</mi>
        <msup>
          <mo stretchy="false">)</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>t</mi>
          </mrow>
        </msup>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [S]=[S]_{0}(1-k)^{t}\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/93211e467eb88a4ed3ce4b1b8a64f3645c540709" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:17.975ex; height:3.009ex;" alt="[S]=[S]_{0}(1-k)^{{t}}\,"/></span></dd></dl>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [S]=[S]_{0}(1-v/[S]_{0})^{t}\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <mo stretchy="false">]</mo>
        <mo>=</mo>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo stretchy="false">(</mo>
        <mn>1</mn>
        <mo>&#x2212;<!-- − --></mo>
        <mi>v</mi>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <msup>
          <mo stretchy="false">)</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>t</mi>
          </mrow>
        </msup>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [S]=[S]_{0}(1-v/[S]_{0})^{t}\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/fc767ed4ec3fb17dbb2b342b438ca22f3a0c5e15" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:22.901ex; height:3.009ex;" alt="[S]=[S]_{0}(1-v/[S]_{0})^{{t}}\,"/></span></dd></dl>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [S]=[S]_{0}(1-(V_{\max }[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{t}\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <mo stretchy="false">]</mo>
        <mo>=</mo>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo stretchy="false">(</mo>
        <mn>1</mn>
        <mo>&#x2212;<!-- − --></mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
        <mo>+</mo>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <mo stretchy="false">[</mo>
        <mi>S</mi>
        <msub>
          <mo stretchy="false">]</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <msup>
          <mo stretchy="false">)</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>t</mi>
          </mrow>
        </msup>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [S]=[S]_{0}(1-(V_{\max }[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{t}\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/550cebc162d8baf678f05a64c8435882eba78bfb" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:45.668ex; height:3.009ex;" alt="[S]=[S]_{0}(1-(V_{{\max }}[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{{t}}\,"/></span></dd></dl>
<p>In 1983 Stuart Beal (and also independently <a href="/wiki/Santiago_Schnell" title="Santiago Schnell">Santiago Schnell</a> and Claudio Mendoza in 1997) derived a closed form solution for the time course kinetics analysis of the Michaelis-Menten mechanism.<sup id="cite_ref-14" class="reference"><a href="#cite_note-14">&#91;14&#93;</a></sup><sup id="cite_ref-15" class="reference"><a href="#cite_note-15">&#91;15&#93;</a></sup> The solution, known as the Schnell-Mendoza equation, has the form:
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\frac {[S]}{K_{M}}}=W\left[F(t)\right]\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">[</mo>
              <mi>S</mi>
              <mo stretchy="false">]</mo>
            </mrow>
            <msub>
              <mi>K</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>M</mi>
              </mrow>
            </msub>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mi>W</mi>
        <mrow>
          <mo>[</mo>
          <mrow>
            <mi>F</mi>
            <mo stretchy="false">(</mo>
            <mi>t</mi>
            <mo stretchy="false">)</mo>
          </mrow>
          <mo>]</mo>
        </mrow>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\frac {[S]}{K_{M}}}=W\left[F(t)\right]\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/bbfb88da686a3b0298417f08709f60c89538b35e" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.171ex; width:16.76ex; height:6.009ex;" alt="{\frac  {[S]}{K_{M}}}=W\left[F(t)\right]\,"/></span></dd></dl>
<p>where W[ ] is the <a href="/wiki/Lambert_W_function" title="Lambert W function">Lambert-W function</a>.<sup id="cite_ref-16" class="reference"><a href="#cite_note-16">&#91;16&#93;</a></sup><sup id="cite_ref-17" class="reference"><a href="#cite_note-17">&#91;17&#93;</a></sup> and where F(t) is
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle F(t)={\frac {[S]_{0}}{K_{M}}}\exp \!\left({\frac {[S]_{0}}{K_{M}}}-{\frac {V_{\max }}{K_{M}}}\,t\right)\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mi>F</mi>
        <mo stretchy="false">(</mo>
        <mi>t</mi>
        <mo stretchy="false">)</mo>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">[</mo>
              <mi>S</mi>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>0</mn>
                </mrow>
              </msub>
            </mrow>
            <msub>
              <mi>K</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>M</mi>
              </mrow>
            </msub>
          </mfrac>
        </mrow>
        <mi>exp</mi>
        <mspace width="negativethinmathspace" />
        <mrow>
          <mo>(</mo>
          <mrow>
            <mrow class="MJX-TeXAtom-ORD">
              <mfrac>
                <mrow>
                  <mo stretchy="false">[</mo>
                  <mi>S</mi>
                  <msub>
                    <mo stretchy="false">]</mo>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mn>0</mn>
                    </mrow>
                  </msub>
                </mrow>
                <msub>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>M</mi>
                  </mrow>
                </msub>
              </mfrac>
            </mrow>
            <mo>&#x2212;<!-- − --></mo>
            <mrow class="MJX-TeXAtom-ORD">
              <mfrac>
                <msub>
                  <mi>V</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mo movablelimits="true" form="prefix">max</mo>
                  </mrow>
                </msub>
                <msub>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>M</mi>
                  </mrow>
                </msub>
              </mfrac>
            </mrow>
            <mspace width="thinmathspace" />
            <mi>t</mi>
          </mrow>
          <mo>)</mo>
        </mrow>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle F(t)={\frac {[S]_{0}}{K_{M}}}\exp \!\left({\frac {[S]_{0}}{K_{M}}}-{\frac {V_{\max }}{K_{M}}}\,t\right)\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/39737501b38ca63037f8350456c777481706c602" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.505ex; width:34.322ex; height:6.343ex;" alt="F(t) = \frac{[S]_0}{K_M} \exp\!\left(\frac{[S]_0}{K_M} - \frac{V_\max}{K_M}\,t \right) \,  "/></span></dd></dl>
<p>This equation is encompassed by the equation below, obtained by Berberan-Santos,<sup id="cite_ref-18" class="reference"><a href="#cite_note-18">&#91;18&#93;</a></sup> which is also valid when the initial substrate concentration is close to that of enzyme,
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\frac {[S]}{K_{M}}}=W\left[F(t)\right]-{\frac {V_{\max }}{k_{cat}K_{M}}}\ {\frac {W\left[F(t)\right]}{1+W\left[F(t)\right]}}\,}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">[</mo>
              <mi>S</mi>
              <mo stretchy="false">]</mo>
            </mrow>
            <msub>
              <mi>K</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>M</mi>
              </mrow>
            </msub>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mi>W</mi>
        <mrow>
          <mo>[</mo>
          <mrow>
            <mi>F</mi>
            <mo stretchy="false">(</mo>
            <mi>t</mi>
            <mo stretchy="false">)</mo>
          </mrow>
          <mo>]</mo>
        </mrow>
        <mo>&#x2212;<!-- − --></mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <msub>
              <mi>V</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mo movablelimits="true" form="prefix">max</mo>
              </mrow>
            </msub>
            <mrow>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>c</mi>
                  <mi>a</mi>
                  <mi>t</mi>
                </mrow>
              </msub>
              <msub>
                <mi>K</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>M</mi>
                </mrow>
              </msub>
            </mrow>
          </mfrac>
        </mrow>
        <mtext>&#xA0;</mtext>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mi>W</mi>
              <mrow>
                <mo>[</mo>
                <mrow>
                  <mi>F</mi>
                  <mo stretchy="false">(</mo>
                  <mi>t</mi>
                  <mo stretchy="false">)</mo>
                </mrow>
                <mo>]</mo>
              </mrow>
            </mrow>
            <mrow>
              <mn>1</mn>
              <mo>+</mo>
              <mi>W</mi>
              <mrow>
                <mo>[</mo>
                <mrow>
                  <mi>F</mi>
                  <mo stretchy="false">(</mo>
                  <mi>t</mi>
                  <mo stretchy="false">)</mo>
                </mrow>
                <mo>]</mo>
              </mrow>
            </mrow>
          </mfrac>
        </mrow>
        <mspace width="thinmathspace" />
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\frac {[S]}{K_{M}}}=W\left[F(t)\right]-{\frac {V_{\max }}{k_{cat}K_{M}}}\ {\frac {W\left[F(t)\right]}{1+W\left[F(t)\right]}}\,}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/036e38cefdac7ce988899ea0d4b89f80c0b6e81d" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.671ex; width:41.912ex; height:6.509ex;" alt="{\frac  {[S]}{K_{M}}}=W\left[F(t)\right]-{\frac  {V_{\max }}{k_{{cat}}K_{M}}}\ {\frac  {W\left[F(t)\right]}{1+W\left[F(t)\right]}}\,"/></span></dd></dl>
<p>where W[ ] is again the <a href="/wiki/Lambert_W_function" title="Lambert W function">Lambert-W function</a>.
</p>
<h3><span id="Linear_plots_of_the_Michaelis.E2.80.93Menten_equation"></span><span class="mw-headline" id="Linear_plots_of_the_Michaelis–Menten_equation">Linear plots of the Michaelis–Menten equation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=6" title="Edit section: Linear plots of the Michaelis–Menten equation">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote navigation-not-searchable">See also: <a href="/wiki/Lineweaver%E2%80%93Burk_plot" title="Lineweaver–Burk plot">Lineweaver–Burk plot</a> and <a href="/wiki/Eadie%E2%80%93Hofstee_diagram" title="Eadie–Hofstee diagram">Eadie–Hofstee diagram</a></div>
<div class="thumb tright"><div class="thumbinner" style="width:352px;"><a href="/wiki/File:Lineweaver-Burke_plot.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/70/Lineweaver-Burke_plot.svg/350px-Lineweaver-Burke_plot.svg.png" decoding="async" width="350" height="231" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/70/Lineweaver-Burke_plot.svg/525px-Lineweaver-Burke_plot.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/70/Lineweaver-Burke_plot.svg/700px-Lineweaver-Burke_plot.svg.png 2x" data-file-width="420" data-file-height="277" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Lineweaver-Burke_plot.svg" class="internal" title="Enlarge"></a></div>Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient.</div></div></div>
<p>The plot of <i>v</i> versus [S] above is not linear; although initially linear at low [S], it bends over to saturate at high [S].  Before the modern era of <a href="/wiki/Nonlinear_regression" title="Nonlinear regression">nonlinear curve-fitting</a> on computers, this nonlinearity could make it difficult to estimate <i>K</i><sub>M</sub> and <i>V</i><sub>max</sub> accurately.  Therefore, several researchers developed linearisations of the Michaelis–Menten equation, such as the <a href="/wiki/Lineweaver%E2%80%93Burk_plot" title="Lineweaver–Burk plot">Lineweaver–Burk plot</a>, the <a href="/wiki/Eadie%E2%80%93Hofstee_diagram" title="Eadie–Hofstee diagram">Eadie–Hofstee diagram</a> and the <a href="/wiki/Hanes%E2%80%93Woolf_plot" title="Hanes–Woolf plot">Hanes–Woolf plot</a>.  All of these linear representations can be useful for visualising data, but none should be used to determine kinetic parameters, as computer software is readily available that allows for more accurate determination by <a href="/wiki/Nonlinear_regression" title="Nonlinear regression">nonlinear regression</a> methods.<sup id="cite_ref-19" class="reference"><a href="#cite_note-19">&#91;19&#93;</a></sup>
</p><p>The <a href="/wiki/Lineweaver%E2%80%93Burk_plot" title="Lineweaver–Burk plot">Lineweaver–Burk plot</a> or double reciprocal plot is a common way of illustrating kinetic data. This is produced by taking the <a href="/wiki/Multiplicative_inverse" title="Multiplicative inverse">reciprocal</a> of both sides of the Michaelis–Menten equation. As shown on the right, this is a linear form of the Michaelis–Menten equation and produces a straight line with the equation <i>y</i> = m<i>x</i> + c with a <i>y</i>-intercept equivalent to 1/<i>V</i><sub>max</sub> and an <i>x</i>-intercept of the graph representing −1/<i>K</i><sub>M</sub>.
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\frac {1}{v}}={\frac {K_{M}}{V_{\max }[{\mbox{S}}]}}+{\frac {1}{V_{\max }}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mn>1</mn>
            <mi>v</mi>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <msub>
              <mi>K</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mi>M</mi>
              </mrow>
            </msub>
            <mrow>
              <msub>
                <mi>V</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo movablelimits="true" form="prefix">max</mo>
                </mrow>
              </msub>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mstyle displaystyle="false" scriptlevel="0">
                  <mtext>S</mtext>
                </mstyle>
              </mrow>
              <mo stretchy="false">]</mo>
            </mrow>
          </mfrac>
        </mrow>
        <mo>+</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mn>1</mn>
            <msub>
              <mi>V</mi>
              <mrow class="MJX-TeXAtom-ORD">
                <mo movablelimits="true" form="prefix">max</mo>
              </mrow>
            </msub>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\frac {1}{v}}={\frac {K_{M}}{V_{\max }[{\mbox{S}}]}}+{\frac {1}{V_{\max }}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/f67c173c3e3e8c78da7dc5fa15c3b5ff299e4439" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.671ex; width:21.489ex; height:6.009ex;" alt="{\frac  {1}{v}}={\frac  {K_{{M}}}{V_{{\max }}[{\mbox{S}}]}}+{\frac  {1}{V_{\max }}}"/></span></dd></dl>
<p>Naturally, no experimental values can be taken at negative 1/[S]; the lower limiting value 1/[S] = 0 (the <i>y</i>-intercept) corresponds to an infinite substrate concentration, where <i>1/v=1/V<sub>max</sub></i> as shown at the right; thus, the <i>x</i>-intercept is an <a href="/wiki/Extrapolation" title="Extrapolation">extrapolation</a> of the experimental data taken at positive concentrations.  More generally, the Lineweaver–Burk plot skews the importance of measurements taken at low substrate concentrations and, thus, can yield inaccurate estimates of <i>V</i><sub>max</sub> and <i>K</i><sub>M</sub>.<sup id="cite_ref-Tseng_20-0" class="reference"><a href="#cite_note-Tseng-20">&#91;20&#93;</a></sup>  A more accurate linear plotting method is the <a href="/wiki/Eadie%E2%80%93Hofstee_diagram" title="Eadie–Hofstee diagram">Eadie–Hofstee plot</a>. In this case, <i>v</i> is plotted against <i>v</i>/[S].  In the third common linear representation, the <a href="/wiki/Hanes%E2%80%93Woolf_plot" title="Hanes–Woolf plot">Hanes–Woolf plot</a>, [S]/<i>v</i> is plotted against [S].
In general, data normalisation can help diminish the amount of experimental work and can increase the reliability of the output, and is suitable for both graphical and numerical analysis.<sup id="cite_ref-21" class="reference"><a href="#cite_note-21">&#91;21&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Practical_significance_of_kinetic_constants">Practical significance of kinetic constants</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=7" title="Edit section: Practical significance of kinetic constants">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The study of enzyme kinetics is important for two basic reasons. Firstly, it helps explain how enzymes work, and secondly, it helps predict how enzymes behave in living organisms. The kinetic constants defined above, <i>K</i><sub>M</sub> and <i>V</i><sub>max</sub>, are critical to attempts to understand how enzymes work together to control <a href="/wiki/Metabolism" title="Metabolism">metabolism</a>.
</p><p>Making these predictions is not trivial, even for simple systems. For example, <a href="/wiki/Oxaloacetate" class="mw-redirect" title="Oxaloacetate">oxaloacetate</a> is formed by <a href="/wiki/Malate_dehydrogenase" title="Malate dehydrogenase">malate dehydrogenase</a> within the <a href="/wiki/Mitochondrion" title="Mitochondrion">mitochondrion</a>. Oxaloacetate can then be consumed by <a href="/wiki/Citrate_synthase" title="Citrate synthase">citrate synthase</a>, <a href="/wiki/Phosphoenolpyruvate_carboxykinase" title="Phosphoenolpyruvate carboxykinase">phosphoenolpyruvate carboxykinase</a> or <a href="/wiki/Aspartate_aminotransferase" class="mw-redirect" title="Aspartate aminotransferase">aspartate aminotransferase</a>, feeding into the <a href="/wiki/Citric_acid_cycle" title="Citric acid cycle">citric acid cycle</a>, <a href="/wiki/Gluconeogenesis" title="Gluconeogenesis">gluconeogenesis</a> or <a href="/wiki/Aspartic_acid" title="Aspartic acid">aspartic acid</a> biosynthesis, respectively. Being able to predict how much oxaloacetate goes into which pathway requires knowledge of the concentration of oxaloacetate as well as the concentration and kinetics of each of these enzymes. This aim of predicting the behaviour of metabolic pathways reaches its most complex expression in the synthesis of huge amounts of kinetic and <a href="/wiki/Gene_expression" title="Gene expression">gene expression</a> data into mathematical models of entire organisms. Alternatively, one useful simplification of the metabolic modelling problem is to ignore the underlying enzyme kinetics and only rely on information about the reaction network's stoichiometry, a technique called <a href="/wiki/Flux_balance_analysis" title="Flux balance analysis">flux balance analysis</a>.<sup id="cite_ref-22" class="reference"><a href="#cite_note-22">&#91;22&#93;</a></sup><sup id="cite_ref-23" class="reference"><a href="#cite_note-23">&#91;23&#93;</a></sup>
</p>
<h3><span id="Michaelis.E2.80.93Menten_kinetics_with_intermediate"></span><span class="mw-headline" id="Michaelis–Menten_kinetics_with_intermediate">Michaelis–Menten kinetics with intermediate</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=8" title="Edit section: Michaelis–Menten kinetics with intermediate">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>One could also consider the less simple case
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {{E}+S&lt;=&gt;[k_{1}][k_{-1}]ES-&gt;[k_{2}]EI-&gt;[k_{3}]{E}+P}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>E</mtext>
          </mrow>
          <mo>+</mo>
          <mtext>S</mtext>
          <mrow class="MJX-TeXAtom-REL">
            <munderover>
              <mo>&#x21CC;</mo>
              <mpadded width="+0.667em" lspace="0.278em" voffset="-.24em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mo>&#x2212;<!-- − --></mo>
                    <mn>1</mn>
                  </mrow>
                </msub>
              </mpadded>
              <mpadded width="+0.667em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mn>1</mn>
                  </mrow>
                </msub>
              </mpadded>
            </munderover>
          </mrow>
          <mtext>ES</mtext>
          <mrow class="MJX-TeXAtom-REL">
            <mover>
              <mo>&#x2192;</mo>
              <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mn>2</mn>
                  </mrow>
                </msub>
              </mpadded>
            </mover>
          </mrow>
          <mtext>EI</mtext>
          <mrow class="MJX-TeXAtom-REL">
            <mover>
              <mo>&#x2192;</mo>
              <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mn>3</mn>
                  </mrow>
                </msub>
              </mpadded>
            </mover>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>E</mtext>
          </mrow>
          <mo>+</mo>
          <mtext>P</mtext>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {{E}+S&lt;=&gt;[k_{1}][k_{-1}]ES-&gt;[k_{2}]EI-&gt;[k_{3}]{E}+P}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/5cd1904bee27689d5df0933e40a4b01631243041" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.223ex; margin-top: -0.32ex; margin-bottom: -0.615ex; width:30.942ex; height:6.676ex;" alt="{\displaystyle {\ce {{E}+S&lt;=&gt;[k_{1}][k_{-1}]ES-&gt;[k_{2}]EI-&gt;[k_{3}]{E}+P}}}"/></span></dd></dl>
<p>where a complex with the enzyme and an intermediate exists and the intermediate is converted into product in a second step. In this case we have a very similar equation<sup id="cite_ref-24" class="reference"><a href="#cite_note-24">&#91;24&#93;</a></sup>
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle v_{0}=k_{cat}{\frac {{\ce {[S] [E]_0}}}{K_{M}^{\prime }+{\ce {[S]}}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo>=</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>c</mi>
            <mi>a</mi>
            <mi>t</mi>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow class="MJX-TeXAtom-ORD">
              <mrow class="MJX-TeXAtom-ORD">
                <mo stretchy="false">[</mo>
                <mtext>S</mtext>
                <mo stretchy="false">]</mo>
              </mrow>
              <msubsup>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mtext>E</mtext>
                  <mo stretchy="false">]</mo>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>0</mn>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mspace width="0pt" height="0pt" depth=".2em" />
                </mrow>
              </msubsup>
            </mrow>
            <mrow>
              <msubsup>
                <mi>K</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>M</mi>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi class="MJX-variant" mathvariant="normal">&#x2032;<!-- ′ --></mi>
                </mrow>
              </msubsup>
              <mo>+</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mtext>S</mtext>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mrow>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}=k_{cat}{\frac {{\ce {[S] [E]_0}}}{K_{M}^{\prime }+{\ce {[S]}}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/d90d4c1b92e79705a9ecf0f8615982c0bc91f4a3" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.838ex; width:19.094ex; height:6.843ex;" alt="{\displaystyle v_{0}=k_{cat}{\frac {{\ce {[S] [E]_0}}}{K_{M}^{\prime }+{\ce {[S]}}}}}"/></span></dd></dl>
<p>but the constants are different
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\begin{aligned}K_{M}^{\prime }\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{3}}{k_{2}+k_{3}}}K_{M}={\frac {k_{3}}{k_{2}+k_{3}}}\cdot {\frac {k_{2}+k_{-1}}{k_{1}}}\\k_{cat}\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\dfrac {k_{3}k_{2}}{k_{2}+k_{3}}}\end{aligned}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mtable columnalign="right left right left right left right left right left right left" rowspacing="3pt" columnspacing="0em 2em 0em 2em 0em 2em 0em 2em 0em 2em 0em" displaystyle="true">
            <mtr>
              <mtd>
                <msubsup>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>M</mi>
                  </mrow>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi class="MJX-variant" mathvariant="normal">&#x2032;<!-- ′ --></mi>
                  </mrow>
                </msubsup>
                <mtext>&#xA0;</mtext>
              </mtd>
              <mtd>
                <mi></mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-REL">
                    <mover>
                      <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                        <mo>=</mo>
                      </mrow>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mi mathvariant="normal">d</mi>
                          <mi mathvariant="normal">e</mi>
                          <mi mathvariant="normal">f</mi>
                        </mrow>
                      </mrow>
                    </mover>
                  </mrow>
                </mrow>
                <mtext>&#xA0;</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <msub>
                      <mi>k</mi>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mn>3</mn>
                      </mrow>
                    </msub>
                    <mrow>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>2</mn>
                        </mrow>
                      </msub>
                      <mo>+</mo>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>3</mn>
                        </mrow>
                      </msub>
                    </mrow>
                  </mfrac>
                </mrow>
                <msub>
                  <mi>K</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>M</mi>
                  </mrow>
                </msub>
                <mo>=</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <msub>
                      <mi>k</mi>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mn>3</mn>
                      </mrow>
                    </msub>
                    <mrow>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>2</mn>
                        </mrow>
                      </msub>
                      <mo>+</mo>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>3</mn>
                        </mrow>
                      </msub>
                    </mrow>
                  </mfrac>
                </mrow>
                <mo>&#x22C5;<!-- ⋅ --></mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <mrow>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>2</mn>
                        </mrow>
                      </msub>
                      <mo>+</mo>
                      <msub>
                        <mi>k</mi>
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                          <mn>1</mn>
                        </mrow>
                      </msub>
                    </mrow>
                    <msub>
                      <mi>k</mi>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mn>1</mn>
                      </mrow>
                    </msub>
                  </mfrac>
                </mrow>
              </mtd>
            </mtr>
            <mtr>
              <mtd>
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi>c</mi>
                    <mi>a</mi>
                    <mi>t</mi>
                  </mrow>
                </msub>
                <mtext>&#xA0;</mtext>
              </mtd>
              <mtd>
                <mi></mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-REL">
                    <mover>
                      <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                        <mo>=</mo>
                      </mrow>
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mi mathvariant="normal">d</mi>
                          <mi mathvariant="normal">e</mi>
                          <mi mathvariant="normal">f</mi>
                        </mrow>
                      </mrow>
                    </mover>
                  </mrow>
                </mrow>
                <mtext>&#xA0;</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mstyle displaystyle="true" scriptlevel="0">
                    <mfrac>
                      <mrow>
                        <msub>
                          <mi>k</mi>
                          <mrow class="MJX-TeXAtom-ORD">
                            <mn>3</mn>
                          </mrow>
                        </msub>
                        <msub>
                          <mi>k</mi>
                          <mrow class="MJX-TeXAtom-ORD">
                            <mn>2</mn>
                          </mrow>
                        </msub>
                      </mrow>
                      <mrow>
                        <msub>
                          <mi>k</mi>
                          <mrow class="MJX-TeXAtom-ORD">
                            <mn>2</mn>
                          </mrow>
                        </msub>
                        <mo>+</mo>
                        <msub>
                          <mi>k</mi>
                          <mrow class="MJX-TeXAtom-ORD">
                            <mn>3</mn>
                          </mrow>
                        </msub>
                      </mrow>
                    </mfrac>
                  </mstyle>
                </mrow>
              </mtd>
            </mtr>
          </mtable>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\begin{aligned}K_{M}^{\prime }\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{3}}{k_{2}+k_{3}}}K_{M}={\frac {k_{3}}{k_{2}+k_{3}}}\cdot {\frac {k_{2}+k_{-1}}{k_{1}}}\\k_{cat}\ &amp;{\stackrel {\mathrm {def} }{=}}\ {\dfrac {k_{3}k_{2}}{k_{2}+k_{3}}}\end{aligned}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1a408a6b97544b6a8c95d3ec9974976cacee9cba" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -5.338ex; width:42.711ex; height:11.843ex;" alt="{\begin{aligned}K_{M}^{{\prime }}\ &amp;{\stackrel  {{\mathrm  {def}}}{=}}\ {\frac  {k_{3}}{k_{2}+k_{3}}}K_{M}={\frac  {k_{3}}{k_{2}+k_{3}}}\cdot {\frac  {k_{{2}}+k_{{-1}}}{k_{{1}}}}\\k_{{cat}}\ &amp;{\stackrel  {{\mathrm  {def}}}{=}}\ {\dfrac  {k_{3}k_{2}}{k_{2}+k_{3}}}\end{aligned}}"/></span></dd></dl>
<p>We see that for the limiting case <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{3}\gg k_{2}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>3</mn>
          </mrow>
        </msub>
        <mo>&#x226B;<!-- ≫ --></mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{3}\gg k_{2}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/f482bbb82bd526720c99efea00baa289fdf96187" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:8.145ex; height:2.509ex;" alt="k_{3}\gg k_{2}"/></span>, thus when the last step from <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {EI -&gt; E + P}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mtext>EI</mtext>
          <mo stretchy="false">&#x27F6;<!-- ⟶ --></mo>
          <mtext>E</mtext>
          <mo>+</mo>
          <mtext>P</mtext>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {EI -&gt; E + P}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/ce5d8cd4343c44180a57a649236c4fca85a7993f" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.505ex; width:13.524ex; height:2.343ex;" alt="{\displaystyle {\ce {EI -&gt; E + P}}}"/></span> is much faster than the previous step, we get again the original equation. Mathematically we have then <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{M}^{\prime }\approx K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msubsup>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi class="MJX-variant" mathvariant="normal">&#x2032;<!-- ′ --></mi>
          </mrow>
        </msubsup>
        <mo>&#x2248;<!-- ≈ --></mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{M}^{\prime }\approx K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/866ad8e62e297abcbd3ec5f95d64ac79e66f71eb" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:10.963ex; height:2.843ex;" alt="K_{M}^{{\prime }}\approx K_{M}"/></span> and <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle k_{cat}\approx k_{2}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>c</mi>
            <mi>a</mi>
            <mi>t</mi>
          </mrow>
        </msub>
        <mo>&#x2248;<!-- ≈ --></mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle k_{cat}\approx k_{2}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/3ca27e348f3ae09fc272705a48cdd71d52f72b4e" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:8.983ex; height:2.509ex;" alt="k_{{cat}}\approx k_{2}"/></span>.
</p>
<h2><span class="mw-headline" id="Multi-substrate_reactions">Multi-substrate reactions</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=9" title="Edit section: Multi-substrate reactions">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Multi-substrate reactions follow complex rate equations that describe how the substrates bind and in what sequence. The analysis of these reactions is much simpler if the concentration of substrate A is kept constant and substrate B varied. Under these conditions, the enzyme behaves just like a single-substrate enzyme and a plot of <i>v</i> by [S] gives apparent <i>K</i><sub>M</sub> and <i>V</i><sub>max</sub> constants for substrate B. If a set of these measurements is performed at different fixed concentrations of A, these data can be used to work out what the mechanism of the reaction is. For an enzyme that takes two substrates A and B and turns them into two products P and Q, there are two types of mechanism: ternary complex and ping–pong.
</p>
<h3><span class="mw-headline" id="Ternary-complex_mechanisms">Ternary-complex mechanisms</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=10" title="Edit section: Ternary-complex mechanisms">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:312px;"><a href="/wiki/File:Random_order_ternary_mechanism.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/en/thumb/b/bc/Random_order_ternary_mechanism.svg/310px-Random_order_ternary_mechanism.svg.png" decoding="async" width="310" height="149" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/en/thumb/b/bc/Random_order_ternary_mechanism.svg/465px-Random_order_ternary_mechanism.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/b/bc/Random_order_ternary_mechanism.svg/620px-Random_order_ternary_mechanism.svg.png 2x" data-file-width="450" data-file-height="217" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Random_order_ternary_mechanism.svg" class="internal" title="Enlarge"></a></div>Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line.</div></div></div>
<p>In these enzymes, both substrates bind to the enzyme at the same time to produce an EAB ternary complex. The order of binding can either be random (in a random mechanism) or substrates have to bind in a particular sequence (in an ordered mechanism). When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ternary-complex mechanism are plotted in a <a href="/wiki/Lineweaver%E2%80%93Burk_plot" title="Lineweaver–Burk plot">Lineweaver–Burk plot</a>, the set of lines produced will intersect.
</p><p>Enzymes with ternary-complex mechanisms include <a href="/wiki/Glutathione_S-transferase" title="Glutathione S-transferase">glutathione <i>S</i>-transferase</a>,<sup id="cite_ref-25" class="reference"><a href="#cite_note-25">&#91;25&#93;</a></sup> <a href="/wiki/Dihydrofolate_reductase" title="Dihydrofolate reductase">dihydrofolate reductase</a><sup id="cite_ref-26" class="reference"><a href="#cite_note-26">&#91;26&#93;</a></sup> and <a href="/wiki/DNA_polymerase" title="DNA polymerase">DNA polymerase</a>.<sup id="cite_ref-27" class="reference"><a href="#cite_note-27">&#91;27&#93;</a></sup> The following links show short animations of the ternary-complex mechanisms of the enzymes dihydrofolate reductase<sup class="reference" id="ref_Bnone"><a href="#endnote_Bnone">[β]</a></sup> and DNA polymerase<sup class="reference" id="ref_Cnone"><a href="#endnote_Cnone">[γ]</a></sup>.
</p>
<h3><span id="Ping.E2.80.93pong_mechanisms"></span><span class="mw-headline" id="Ping–pong_mechanisms">Ping–pong mechanisms</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=11" title="Edit section: Ping–pong mechanisms">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:412px"><div class="thumbimage noresize" style="width:410px;"><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {\overset {}{E-&gt;[{\ce {A \atop \downarrow }}]EA&lt;=&gt;E^{\ast }P-&gt;[{\ce {P \atop \uparrow }}]E^{\ast }-&gt;[{\ce {B \atop \downarrow }}]E^{\ast }B&lt;=&gt;EQ-&gt;[{\ce {Q \atop \uparrow }}]E}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mover>
            <mrow>
              <mtext>E</mtext>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mo>&#x2192;</mo>
                  <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mfrac linethickness="0">
                        <mtext>A</mtext>
                        <mo stretchy="false">&#x2193;<!-- ↓ --></mo>
                      </mfrac>
                    </mrow>
                  </mpadded>
                </mover>
              </mrow>
              <mtext>EA</mtext>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mpadded height="0" depth="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">&#x21BD;<!-- ↽ --></mo>
                        </mrow>
                        <mspace width="negativethinmathspace" />
                        <mspace width="negativethinmathspace" />
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                        </mrow>
                      </mpadded>
                    </mrow>
                  </mrow>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mstyle displaystyle="false" scriptlevel="0">
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                        </mrow>
                        <mspace width="negativethinmathspace" />
                        <mspace width="negativethinmathspace" />
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">&#x21C0;<!-- ⇀ --></mo>
                        </mrow>
                      </mrow>
                    </mstyle>
                  </mrow>
                </mover>
              </mrow>
              <msup>
                <mtext>E</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2217;<!-- ∗ --></mo>
                </mrow>
              </msup>
              <mtext>P</mtext>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mo>&#x2192;</mo>
                  <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mfrac linethickness="0">
                        <mtext>P</mtext>
                        <mo stretchy="false">&#x2191;<!-- ↑ --></mo>
                      </mfrac>
                    </mrow>
                  </mpadded>
                </mover>
              </mrow>
              <msup>
                <mtext>E</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2217;<!-- ∗ --></mo>
                </mrow>
              </msup>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mo>&#x2192;</mo>
                  <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mfrac linethickness="0">
                        <mtext>B</mtext>
                        <mo stretchy="false">&#x2193;<!-- ↓ --></mo>
                      </mfrac>
                    </mrow>
                  </mpadded>
                </mover>
              </mrow>
              <msup>
                <mtext>E</mtext>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2217;<!-- ∗ --></mo>
                </mrow>
              </msup>
              <mtext>B</mtext>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mrow class="MJX-TeXAtom-OP MJX-fixedlimits">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mpadded height="0" depth="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">&#x21BD;<!-- ↽ --></mo>
                        </mrow>
                        <mspace width="negativethinmathspace" />
                        <mspace width="negativethinmathspace" />
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                        </mrow>
                      </mpadded>
                    </mrow>
                  </mrow>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mstyle displaystyle="false" scriptlevel="0">
                      <mrow class="MJX-TeXAtom-ORD">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo>&#x2212;<!-- − --></mo>
                        </mrow>
                        <mspace width="negativethinmathspace" />
                        <mspace width="negativethinmathspace" />
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">&#x21C0;<!-- ⇀ --></mo>
                        </mrow>
                      </mrow>
                    </mstyle>
                  </mrow>
                </mover>
              </mrow>
              <mtext>EQ</mtext>
              <mrow class="MJX-TeXAtom-REL">
                <mover>
                  <mo>&#x2192;</mo>
                  <mpadded width="+0.611em" lspace="0.278em" voffset=".15em">
                    <mrow class="MJX-TeXAtom-ORD">
                      <mfrac linethickness="0">
                        <mtext>Q</mtext>
                        <mo stretchy="false">&#x2191;<!-- ↑ --></mo>
                      </mfrac>
                    </mrow>
                  </mpadded>
                </mover>
              </mrow>
              <mtext>E</mtext>
            </mrow>
            <mrow />
          </mover>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {\overset {}{E-&gt;[{\ce {A \atop \downarrow }}]EA&lt;=&gt;E^{\ast }P-&gt;[{\ce {P \atop \uparrow }}]E^{\ast }-&gt;[{\ce {B \atop \downarrow }}]E^{\ast }B&lt;=&gt;EQ-&gt;[{\ce {Q \atop \uparrow }}]E}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/f9b768dbbd547267c22748f29590cb0a639375de" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:46.896ex; height:6.176ex;" alt="{\displaystyle {\ce {\overset {}{E-&gt;[{\ce {A \atop \downarrow }}]EA&lt;=&gt;E^{\ast }P-&gt;[{\ce {P \atop \uparrow }}]E^{\ast }-&gt;[{\ce {B \atop \downarrow }}]E^{\ast }B&lt;=&gt;EQ-&gt;[{\ce {Q \atop \uparrow }}]E}}}}"/></span></div><div class="thumbcaption">Ping–pong mechanism for an enzyme reaction. Intermediates contain substrates A and B or products P and Q.</div></div></div>
<p>As shown on the right, enzymes with a ping-pong mechanism can exist in two states, E and a chemically modified form of the enzyme E*; this modified enzyme is known as an <a href="/wiki/Reactive_intermediate" title="Reactive intermediate">intermediate</a>. In such mechanisms, substrate A binds, changes the enzyme to E* by, for example, transferring a chemical group to the active site, and is then released. Only after the first substrate is released can substrate B bind and react with the modified enzyme, regenerating the unmodified E form. When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ping–pong mechanism are plotted in a Lineweaver–Burk plot, a set of parallel lines will be produced.  This is called a <a href="/wiki/Secondary_plot_(kinetics)" title="Secondary plot (kinetics)">secondary plot</a>.
</p><p>Enzymes with ping–pong mechanisms include some <a href="/wiki/Oxidoreductases" class="mw-redirect" title="Oxidoreductases">oxidoreductases</a> such as <a href="/wiki/Peroxidase" title="Peroxidase">thioredoxin peroxidase</a>,<sup id="cite_ref-28" class="reference"><a href="#cite_note-28">&#91;28&#93;</a></sup> <a href="/wiki/Transferases" class="mw-redirect" title="Transferases">transferases</a> such as acylneuraminate cytidylyltransferase<sup id="cite_ref-29" class="reference"><a href="#cite_note-29">&#91;29&#93;</a></sup> and <a href="/wiki/Serine_protease" title="Serine protease">serine proteases</a> such as <a href="/wiki/Trypsin" title="Trypsin">trypsin</a> and <a href="/wiki/Chymotrypsin" title="Chymotrypsin">chymotrypsin</a>.<sup id="cite_ref-30" class="reference"><a href="#cite_note-30">&#91;30&#93;</a></sup>  Serine proteases are a very common and diverse family of enzymes, including <a href="/wiki/Digestion" title="Digestion">digestive</a> enzymes (trypsin, chymotrypsin, and elastase), several enzymes of the <a href="/wiki/Coagulation" title="Coagulation">blood clotting cascade</a> and many others. In these serine proteases, the E* intermediate is an acyl-enzyme species formed by the attack of an active site <a href="/wiki/Serine" title="Serine">serine</a> residue on a <a href="/wiki/Peptide_bond" title="Peptide bond">peptide bond</a> in a protein substrate. A short animation showing the mechanism of chymotrypsin is linked here.<sup class="reference" id="ref_Dnone"><a href="#endnote_Dnone">[δ]</a></sup>
</p>
<h2><span class="mw-headline" id="Reversible_catalysis_and_the_Haldane_equation">Reversible catalysis and the Haldane equation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=12" title="Edit section: Reversible catalysis and the Haldane equation">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>We consider the case of an enzyme that can catalyse the reaction in both directions (not all enzymes can):
</p><p><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="{\displaystyle {\ce {{E}+{S}&lt;=&gt;[k_{1}][k_{-1}]ES&lt;=&gt;[k_{2}][k_{-2}]{E}+{P}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>E</mtext>
          </mrow>
          <mo>+</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>S</mtext>
          </mrow>
          <mrow class="MJX-TeXAtom-REL">
            <munderover>
              <mo>&#x21CC;</mo>
              <mpadded width="+0.667em" lspace="0.278em" voffset="-.24em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mo>&#x2212;<!-- − --></mo>
                    <mn>1</mn>
                  </mrow>
                </msub>
              </mpadded>
              <mpadded width="+0.667em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mn>1</mn>
                  </mrow>
                </msub>
              </mpadded>
            </munderover>
          </mrow>
          <mtext>ES</mtext>
          <mrow class="MJX-TeXAtom-REL">
            <munderover>
              <mo>&#x21CC;</mo>
              <mpadded width="+0.667em" lspace="0.278em" voffset="-.24em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mo>&#x2212;<!-- − --></mo>
                    <mn>2</mn>
                  </mrow>
                </msub>
              </mpadded>
              <mpadded width="+0.667em" lspace="0.278em" voffset=".15em">
                <msub>
                  <mi>k</mi>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mn>2</mn>
                  </mrow>
                </msub>
              </mpadded>
            </munderover>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>E</mtext>
          </mrow>
          <mo>+</mo>
          <mrow class="MJX-TeXAtom-ORD">
            <mtext>P</mtext>
          </mrow>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\ce {{E}+{S}&lt;=&gt;[k_{1}][k_{-1}]ES&lt;=&gt;[k_{2}][k_{-2}]{E}+{P}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/fe362b85b97b0614f6139dd0be2e8389a2c3b63e" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.223ex; margin-top: -0.32ex; margin-bottom: -0.615ex; width:25.234ex; height:6.676ex;" alt="{\displaystyle {\ce {{E}+{S}&lt;=&gt;[k_{1}][k_{-1}]ES&lt;=&gt;[k_{2}][k_{-2}]{E}+{P}}}}"/></span>
</p><p>The steady-state, initial rate of the reaction is  <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle v_{0}={\frac {d\,[{\rm {P}}]}{dt}}={\frac {(k_{1}k_{2}\,[{\rm {S}}]-k_{-1}k_{-2}[{\rm {P}}])[{\rm {E}}]_{0}}{k_{-1}+k_{2}+k_{1}\,[{\rm {S}}]+k_{-2}\,[{\rm {P}}]}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mi>d</mi>
              <mspace width="thinmathspace" />
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">P</mi>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
            </mrow>
            <mrow>
              <mi>d</mi>
              <mi>t</mi>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">(</mo>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>1</mn>
                </mrow>
              </msub>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>2</mn>
                </mrow>
              </msub>
              <mspace width="thinmathspace" />
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">S</mi>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
              <mo>&#x2212;<!-- − --></mo>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>1</mn>
                </mrow>
              </msub>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>2</mn>
                </mrow>
              </msub>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">P</mi>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
              <mo stretchy="false">)</mo>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">E</mi>
                </mrow>
              </mrow>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>0</mn>
                </mrow>
              </msub>
            </mrow>
            <mrow>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>1</mn>
                </mrow>
              </msub>
              <mo>+</mo>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>2</mn>
                </mrow>
              </msub>
              <mo>+</mo>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>1</mn>
                </mrow>
              </msub>
              <mspace width="thinmathspace" />
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">S</mi>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
              <mo>+</mo>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>2</mn>
                </mrow>
              </msub>
              <mspace width="thinmathspace" />
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">P</mi>
                </mrow>
              </mrow>
              <mo stretchy="false">]</mo>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}={\frac {d\,[{\rm {P}}]}{dt}}={\frac {(k_{1}k_{2}\,[{\rm {S}}]-k_{-1}k_{-2}[{\rm {P}}])[{\rm {E}}]_{0}}{k_{-1}+k_{2}+k_{1}\,[{\rm {S}}]+k_{-2}\,[{\rm {P}}]}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/01297c24d81598b7438d064174294408dacc5e60" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.671ex; width:40.908ex; height:6.509ex;" alt="{\displaystyle v_{0}={\frac {d\,[{\rm {P}}]}{dt}}={\frac {(k_{1}k_{2}\,[{\rm {S}}]-k_{-1}k_{-2}[{\rm {P}}])[{\rm {E}}]_{0}}{k_{-1}+k_{2}+k_{1}\,[{\rm {S}}]+k_{-2}\,[{\rm {P}}]}}}"/></span>
</p><p><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle v_{0}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>v</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>0</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v_{0}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/60faad24775635f4722ccc438093dbbfe05f34ae" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:2.182ex; height:2.009ex;" alt="v_{0}"/></span> is positive if the reaction proceed in the forward direction (<span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle S\rightarrow P}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mi>S</mi>
        <mo stretchy="false">&#x2192;<!-- → --></mo>
        <mi>P</mi>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle S\rightarrow P}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/bc699757db699063e802dbc1c9f11f8728894f1f" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.338ex; width:6.859ex; height:2.176ex;" alt="{\displaystyle S\rightarrow P}"/></span>) and negative otherwise.
</p><p><a href="/wiki/Equilibrium_constant" title="Equilibrium constant">Equilibrium</a> requires that <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle v=0}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mi>v</mi>
        <mo>=</mo>
        <mn>0</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle v=0}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/ba3d414a23bf4ecfa36cdd039241efc60a5bd9e0" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.338ex; width:5.389ex; height:2.176ex;" alt="v=0"/></span>, which occurs when <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {k_{1}k_{2}}{k_{-1}k_{-2}}}=K_{\rm {eq}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">P</mi>
                </mrow>
              </mrow>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">e</mi>
                    <mi mathvariant="normal">q</mi>
                  </mrow>
                </mrow>
              </msub>
            </mrow>
            <mrow>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">S</mi>
                </mrow>
              </mrow>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">e</mi>
                    <mi mathvariant="normal">q</mi>
                  </mrow>
                </mrow>
              </msub>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>1</mn>
                </mrow>
              </msub>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>2</mn>
                </mrow>
              </msub>
            </mrow>
            <mrow>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>1</mn>
                </mrow>
              </msub>
              <msub>
                <mi>k</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mo>&#x2212;<!-- − --></mo>
                  <mn>2</mn>
                </mrow>
              </msub>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">e</mi>
              <mi mathvariant="normal">q</mi>
            </mrow>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {k_{1}k_{2}}{k_{-1}k_{-2}}}=K_{\rm {eq}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/4f0ca30bfb4f852fcc02a12f729bcbd3adc500d6" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -2.671ex; width:23.49ex; height:6.509ex;" alt="{\displaystyle {\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {k_{1}k_{2}}{k_{-1}k_{-2}}}=K_{\rm {eq}}}"/></span>. This shows that <a href="/wiki/Equilibrium_constant" title="Equilibrium constant">thermodynamics</a> forces a relation between the values of the 4 rate constants.
</p><p>The values of the forward and backward <i>maximal</i> rates, obtained for <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [{\rm {S}}]\rightarrow \infty }">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mi mathvariant="normal">S</mi>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo stretchy="false">&#x2192;<!-- → --></mo>
        <mi mathvariant="normal">&#x221E;<!-- ∞ --></mi>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [{\rm {S}}]\rightarrow \infty }</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/c3e69387ec816688807461db5902b440a6a65ed0" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:8.524ex; height:2.843ex;" alt="{\displaystyle [{\rm {S}}]\rightarrow \infty }"/></span>,  <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [{\rm {P}}]=0}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mi mathvariant="normal">P</mi>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo>=</mo>
        <mn>0</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [{\rm {P}}]=0}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/93c498e9bfae3b2d0f1378c8a1a15031f4002279" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:7.137ex; height:2.843ex;" alt="{\displaystyle [{\rm {P}}]=0}"/></span>, and  <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [{\rm {S}}]=0}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mi mathvariant="normal">S</mi>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo>=</mo>
        <mn>0</mn>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [{\rm {S}}]=0}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/53ac89660d3f5c0ad06eac244190defc17cd7a61" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:6.847ex; height:2.843ex;" alt="{\displaystyle [{\rm {S}}]=0}"/></span>,  <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle [{\rm {P}}]\rightarrow \infty }">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mo stretchy="false">[</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mrow class="MJX-TeXAtom-ORD">
            <mi mathvariant="normal">P</mi>
          </mrow>
        </mrow>
        <mo stretchy="false">]</mo>
        <mo stretchy="false">&#x2192;<!-- → --></mo>
        <mi mathvariant="normal">&#x221E;<!-- ∞ --></mi>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle [{\rm {P}}]\rightarrow \infty }</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/020c2fe953abd3c508b7bd3a6d97bb29225faeeb" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:8.814ex; height:2.843ex;" alt="{\displaystyle [{\rm {P}}]\rightarrow \infty }"/></span>, respectively, are <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\rm {max}}^{f}=k_{2}{\rm {[E]}}_{tot}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msubsup>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">m</mi>
              <mi mathvariant="normal">a</mi>
              <mi mathvariant="normal">x</mi>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>f</mi>
          </mrow>
        </msubsup>
        <mo>=</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <msub>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mo stretchy="false">[</mo>
              <mi mathvariant="normal">E</mi>
              <mo stretchy="false">]</mo>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>t</mi>
            <mi>o</mi>
            <mi>t</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\rm {max}}^{f}=k_{2}{\rm {[E]}}_{tot}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1834ca7ac4c486ffc3e1c89742f31c266797da98" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:15.104ex; height:3.509ex;" alt="{\displaystyle V_{\rm {max}}^{f}=k_{2}{\rm {[E]}}_{tot}}"/></span> and <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\rm {max}}^{b}=-k_{-1}{\rm {[E]}}_{tot}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msubsup>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">m</mi>
              <mi mathvariant="normal">a</mi>
              <mi mathvariant="normal">x</mi>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>b</mi>
          </mrow>
        </msubsup>
        <mo>=</mo>
        <mo>&#x2212;<!-- − --></mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <msub>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mo stretchy="false">[</mo>
              <mi mathvariant="normal">E</mi>
              <mo stretchy="false">]</mo>
            </mrow>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>t</mi>
            <mi>o</mi>
            <mi>t</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\rm {max}}^{b}=-k_{-1}{\rm {[E]}}_{tot}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/a09363446309c8c26702430b70bfec463a3b0134" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:18.19ex; height:3.176ex;" alt="{\displaystyle V_{\rm {max}}^{b}=-k_{-1}{\rm {[E]}}_{tot}}"/></span>, respectively. Their ratio is not equal to the equilibrium constant, which implies that <a href="/wiki/Equilibrium_constant" title="Equilibrium constant">thermodynamics</a> does not constrain the ratio of the maximal rates. This explains that enzymes can be much "better catalysts" (<i>in terms of maximal rates</i>) in one particular direction of the reaction.<sup id="cite_ref-31" class="reference"><a href="#cite_note-31">&#91;31&#93;</a></sup>
</p><p>On can also derive the two Michaelis constants <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{M}^{S}=(k_{-1}+k_{2})/k_{1}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msubsup>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>S</mi>
          </mrow>
        </msubsup>
        <mo>=</mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>+</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>1</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{M}^{S}=(k_{-1}+k_{2})/k_{1}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/f6e2bd31e5b448c7b1565012566755f4f8739681" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:20.918ex; height:3.176ex;" alt="{\displaystyle K_{M}^{S}=(k_{-1}+k_{2})/k_{1}}"/></span>and  <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{M}^{P}=(k_{-1}+k_{2})/k_{-2}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msubsup>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>P</mi>
          </mrow>
        </msubsup>
        <mo>=</mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>+</mo>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mn>2</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>k</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo>&#x2212;<!-- − --></mo>
            <mn>2</mn>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{M}^{P}=(k_{-1}+k_{2})/k_{-2}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/3f95c1d68a0d6c677873f46790d0d07d8925190d" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -1.005ex; width:22.196ex; height:3.176ex;" alt="{\displaystyle K_{M}^{P}=(k_{-1}+k_{2})/k_{-2}}"/></span>. The Haldane equation is the relation <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{\rm {eq}}={\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {V_{\rm {max}}^{f}/K_{M}^{S}}{V_{\rm {max}}^{b}/K_{M}^{P}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">e</mi>
              <mi mathvariant="normal">q</mi>
            </mrow>
          </mrow>
        </msub>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">P</mi>
                </mrow>
              </mrow>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">e</mi>
                    <mi mathvariant="normal">q</mi>
                  </mrow>
                </mrow>
              </msub>
            </mrow>
            <mrow>
              <mo stretchy="false">[</mo>
              <mrow class="MJX-TeXAtom-ORD">
                <mrow class="MJX-TeXAtom-ORD">
                  <mi mathvariant="normal">S</mi>
                </mrow>
              </mrow>
              <msub>
                <mo stretchy="false">]</mo>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">e</mi>
                    <mi mathvariant="normal">q</mi>
                  </mrow>
                </mrow>
              </msub>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <msubsup>
                <mi>V</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">m</mi>
                    <mi mathvariant="normal">a</mi>
                    <mi mathvariant="normal">x</mi>
                  </mrow>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>f</mi>
                </mrow>
              </msubsup>
              <mrow class="MJX-TeXAtom-ORD">
                <mo>/</mo>
              </mrow>
              <msubsup>
                <mi>K</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>M</mi>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>S</mi>
                </mrow>
              </msubsup>
            </mrow>
            <mrow>
              <msubsup>
                <mi>V</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mrow class="MJX-TeXAtom-ORD">
                    <mi mathvariant="normal">m</mi>
                    <mi mathvariant="normal">a</mi>
                    <mi mathvariant="normal">x</mi>
                  </mrow>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>b</mi>
                </mrow>
              </msubsup>
              <mrow class="MJX-TeXAtom-ORD">
                <mo>/</mo>
              </mrow>
              <msubsup>
                <mi>K</mi>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>M</mi>
                </mrow>
                <mrow class="MJX-TeXAtom-ORD">
                  <mi>P</mi>
                </mrow>
              </msubsup>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{\rm {eq}}={\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {V_{\rm {max}}^{f}/K_{M}^{S}}{V_{\rm {max}}^{b}/K_{M}^{P}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/621e1b38683241916244c9e90f5625f14f1f0088" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -3.005ex; width:26.143ex; height:7.509ex;" alt="{\displaystyle K_{\rm {eq}}={\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {V_{\rm {max}}^{f}/K_{M}^{S}}{V_{\rm {max}}^{b}/K_{M}^{P}}}}"/></span>.
</p><p>Therefore, <a href="/wiki/Equilibrium_constant" title="Equilibrium constant">thermodynamics</a> constrains the ratio between the forward and backward <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\rm {max}}/K_{M}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">m</mi>
              <mi mathvariant="normal">a</mi>
              <mi mathvariant="normal">x</mi>
            </mrow>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mo>/</mo>
        </mrow>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>M</mi>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\rm {max}}/K_{M}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/c86cc6d73f03f4478bfce77ff4c555aeaa61a05d" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.838ex; width:9.741ex; height:2.843ex;" alt="{\displaystyle V_{\rm {max}}/K_{M}}"/></span> values, not the ratio of <span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\rm {max}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mrow class="MJX-TeXAtom-ORD">
              <mi mathvariant="normal">m</mi>
              <mi mathvariant="normal">a</mi>
              <mi mathvariant="normal">x</mi>
            </mrow>
          </mrow>
        </msub>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\rm {max}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/fa70ca539750e758de455ff485259924561d1700" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -0.671ex; width:4.646ex; height:2.509ex;" alt="{\displaystyle V_{\rm {max}}}"/></span>values.
</p>
<h2><span id="Non-Michaelis.E2.80.93Menten_kinetics"></span><span class="mw-headline" id="Non-Michaelis–Menten_kinetics">Non-Michaelis–Menten kinetics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=13" title="Edit section: Non-Michaelis–Menten kinetics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Allosteric_regulation" title="Allosteric regulation">Allosteric regulation</a></div>
<div class="thumb tleft"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:Allosteric_v_by_S_curve.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/4/44/Allosteric_v_by_S_curve.svg/300px-Allosteric_v_by_S_curve.svg.png" decoding="async" width="300" height="231" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/4/44/Allosteric_v_by_S_curve.svg/450px-Allosteric_v_by_S_curve.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/44/Allosteric_v_by_S_curve.svg/600px-Allosteric_v_by_S_curve.svg.png 2x" data-file-width="565" data-file-height="435" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Allosteric_v_by_S_curve.svg" class="internal" title="Enlarge"></a></div>Saturation curve for an enzyme reaction showing sigmoid kinetics.</div></div></div>
<p>Some enzymes produce a <a href="/wiki/Sigmoid_function" title="Sigmoid function">sigmoid</a> <i>v</i> by [S] plot, which often indicates <a href="/wiki/Cooperative_binding" title="Cooperative binding">cooperative binding</a> of substrate to the active site. This means that the binding of one substrate molecule affects the binding of subsequent substrate molecules. This behavior is most common in <a href="/wiki/Protein_structure" title="Protein structure">multimeric</a> enzymes with several interacting active sites.<sup id="cite_ref-32" class="reference"><a href="#cite_note-32">&#91;32&#93;</a></sup> Here, the mechanism of cooperation is similar to that of <a href="/wiki/Hemoglobin" title="Hemoglobin">hemoglobin</a>, with binding of substrate to one active site altering the affinity of the other active sites for substrate molecules. Positive cooperativity occurs when binding of the first substrate molecule <i>increases</i> the affinity of the other active sites for substrate. Negative cooperativity occurs when binding of the first substrate <i>decreases</i> the affinity of the enzyme for other substrate molecules.
</p><p>Allosteric enzymes include mammalian tyrosyl tRNA-synthetase, which shows negative cooperativity,<sup id="cite_ref-33" class="reference"><a href="#cite_note-33">&#91;33&#93;</a></sup> and bacterial <a href="/wiki/Aspartate_transcarbamoylase" class="mw-redirect" title="Aspartate transcarbamoylase">aspartate transcarbamoylase</a><sup id="cite_ref-34" class="reference"><a href="#cite_note-34">&#91;34&#93;</a></sup> and <a href="/wiki/Phosphofructokinase" title="Phosphofructokinase">phosphofructokinase</a>,<sup id="cite_ref-35" class="reference"><a href="#cite_note-35">&#91;35&#93;</a></sup> which show positive cooperativity.
</p><p>Cooperativity is surprisingly common and can help regulate the responses of enzymes to changes in the concentrations of their substrates. Positive cooperativity makes enzymes much more sensitive to [S] and their activities can show large changes over a narrow range of substrate concentration. Conversely, negative cooperativity makes enzymes insensitive to small changes in [S].
</p><p>The <a href="/wiki/Hill_equation_(biochemistry)" title="Hill equation (biochemistry)">Hill equation (biochemistry)</a><sup id="cite_ref-36" class="reference"><a href="#cite_note-36">&#91;36&#93;</a></sup> is often used to describe the degree of cooperativity quantitatively in non-Michaelis–Menten kinetics.  The derived Hill coefficient <i>n</i> measures how much the binding of substrate to one active site affects the binding of substrate to the other active sites. A Hill coefficient of &lt;1 indicates negative cooperativity and a coefficient of &gt;1 indicates positive <a href="/wiki/Cooperativity" title="Cooperativity">cooperativity</a>.
</p>
<h2><span class="mw-headline" id="Pre-steady-state_kinetics">Pre-steady-state kinetics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=14" title="Edit section: Pre-steady-state kinetics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:Burst_phase.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Burst_phase.svg/300px-Burst_phase.svg.png" decoding="async" width="300" height="208" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Burst_phase.svg/450px-Burst_phase.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Burst_phase.svg/600px-Burst_phase.svg.png 2x" data-file-width="704" data-file-height="487" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Burst_phase.svg" class="internal" title="Enlarge"></a></div>Pre-steady state progress curve, showing the burst phase of an enzyme reaction.</div></div></div>
<p>In the first moment after an enzyme is mixed with substrate, no product has been formed and no <a href="/wiki/Reactive_intermediate" title="Reactive intermediate">intermediates</a> exist. The study of the next few milliseconds of the reaction is called pre-steady-state kinetics. Pre-steady-state kinetics is therefore concerned with the formation and consumption of enzyme–substrate intermediates (such as ES or E*) until their <a href="/wiki/Steady_state_(chemistry)" title="Steady state (chemistry)">steady-state concentrations</a> are reached.
</p><p>This approach was first applied to the hydrolysis reaction catalysed by <a href="/wiki/Chymotrypsin" title="Chymotrypsin">chymotrypsin</a>.<sup id="cite_ref-37" class="reference"><a href="#cite_note-37">&#91;37&#93;</a></sup> Often, the detection of an intermediate is a vital piece of evidence in investigations of what mechanism an enzyme follows. For example, in the ping–pong mechanisms that are shown above, rapid kinetic measurements can follow the release of product P and measure the formation of the modified enzyme intermediate E*.<sup id="cite_ref-Fersht_38-0" class="reference"><a href="#cite_note-Fersht-38">&#91;38&#93;</a></sup> In the case of chymotrypsin, this intermediate is formed by an attack on the substrate by the <a href="/wiki/Nucleophile" title="Nucleophile">nucleophilic</a> serine in the active site and the formation of the acyl-enzyme intermediate.
</p><p>In the figure to the right, the enzyme produces E* rapidly in the first few seconds of the reaction. The rate then slows as steady state is reached. This rapid burst phase of the reaction measures a single turnover of the enzyme. Consequently, the amount of product released in this burst, shown as the intercept on the <i>y</i>-axis of the graph, also gives the amount of functional enzyme which is present in the assay.<sup id="cite_ref-39" class="reference"><a href="#cite_note-39">&#91;39&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Chemical_mechanism">Chemical mechanism</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=15" title="Edit section: Chemical mechanism">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>An important goal of measuring enzyme kinetics is to determine the chemical mechanism of an enzyme reaction, i.e., the sequence of chemical steps that transform substrate into product. The kinetic approaches discussed above will show at what rates <a href="/wiki/Reactive_intermediate" title="Reactive intermediate">intermediates</a> are formed and inter-converted, but they cannot identify exactly what these intermediates are.
</p><p>Kinetic measurements taken under various solution conditions or on slightly modified enzymes or substrates often shed light on this chemical mechanism, as they reveal the rate-determining step or intermediates in the reaction. For example, the breaking of a <a href="/wiki/Covalent_bond" title="Covalent bond">covalent bond</a> to a <a href="/wiki/Hydrogen" title="Hydrogen">hydrogen</a> <a href="/wiki/Atom" title="Atom">atom</a> is a common rate-determining step. Which of the possible hydrogen transfers is rate determining can be shown by measuring the kinetic effects of substituting each hydrogen by <a href="/wiki/Deuterium" title="Deuterium">deuterium</a>, its stable <a href="/wiki/Isotope" title="Isotope">isotope</a>. The rate will change when the critical hydrogen is replaced, due to a primary <a href="/wiki/Kinetic_isotope_effect" title="Kinetic isotope effect">kinetic isotope effect</a>, which occurs because bonds to deuterium are harder to break than bonds to hydrogen.<sup id="cite_ref-40" class="reference"><a href="#cite_note-40">&#91;40&#93;</a></sup> It is also possible to measure similar effects with other isotope substitutions, such as <sup>13</sup>C/<sup>12</sup>C and <sup>18</sup>O/<sup>16</sup>O, but these effects are more subtle.<sup id="cite_ref-41" class="reference"><a href="#cite_note-41">&#91;41&#93;</a></sup>
</p><p>Isotopes can also be used to reveal the fate of various parts of the substrate molecules in the final products. For example, it is sometimes difficult to discern the origin of an <a href="/wiki/Oxygen" title="Oxygen">oxygen</a> atom in the final product; since it may have come from water or from part of the substrate. This may be determined by systematically substituting oxygen's stable isotope <sup>18</sup>O into the various molecules that participate in the reaction and checking for the isotope in the product.<sup id="cite_ref-42" class="reference"><a href="#cite_note-42">&#91;42&#93;</a></sup> The chemical mechanism can also be elucidated by examining the kinetics and isotope effects under different pH conditions,<sup id="cite_ref-43" class="reference"><a href="#cite_note-43">&#91;43&#93;</a></sup> by altering the metal ions or other bound <a href="/wiki/Cofactor_(biochemistry)" title="Cofactor (biochemistry)">cofactors</a>,<sup id="cite_ref-44" class="reference"><a href="#cite_note-44">&#91;44&#93;</a></sup>  by <a href="/wiki/Site-directed_mutagenesis" title="Site-directed mutagenesis">site-directed mutagenesis</a> of conserved amino acid residues, or by studying the behaviour of the enzyme in the presence of analogues of the substrate(s).<sup id="cite_ref-45" class="reference"><a href="#cite_note-45">&#91;45&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Enzyme_inhibition_and_activation">Enzyme inhibition and activation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=16" title="Edit section: Enzyme inhibition and activation">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Enzyme_inhibitor" title="Enzyme inhibitor">Enzyme inhibitor</a></div>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:Reversible_inhibition.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Reversible_inhibition.svg/300px-Reversible_inhibition.svg.png" decoding="async" width="300" height="142" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Reversible_inhibition.svg/450px-Reversible_inhibition.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Reversible_inhibition.svg/600px-Reversible_inhibition.svg.png 2x" data-file-width="761" data-file-height="361" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Reversible_inhibition.svg" class="internal" title="Enlarge"></a></div>Kinetic scheme for reversible enzyme inhibitors.</div></div></div>
<p>Enzyme inhibitors are molecules that reduce or abolish enzyme activity, while enzyme activators are molecules that increase the catalytic rate of enzymes. These interactions can be either <i>reversible</i> (i.e., removal of the inhibitor restores enzyme activity) or <i>irreversible</i> (i.e., the inhibitor permanently inactivates the enzyme).
</p>
<h3><span class="mw-headline" id="Reversible_inhibitors">Reversible inhibitors</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=17" title="Edit section: Reversible inhibitors">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Traditionally reversible enzyme inhibitors have been classified as competitive, uncompetitive, or non-competitive, according to their effects on <i>K</i><sub>M</sub> and <i>V</i><sub>max</sub>. These different effects result from the inhibitor binding to the enzyme E, to the enzyme–substrate complex ES, or to both, respectively.  The division of these classes arises from a problem in their derivation and results in the need to use two different binding constants for one binding event.  The binding of an inhibitor and its effect on the enzymatic activity are two distinctly different things, another problem the traditional equations fail to acknowledge. In noncompetitive inhibition the binding of the inhibitor results in 100% inhibition of the enzyme only, and fails to consider the possibility of anything in between.<sup id="cite_ref-46" class="reference"><a href="#cite_note-46">&#91;46&#93;</a></sup>  In noncompetitive inhibition, the inhibitor will bind to an enzyme at its allosteric site; therefore, the binding affinity, or inverse of <i>K</i><sub>M</sub>, of the substrate with the enzyme will remain the same. On the other hand, the Vmax will decrease relative to an uninhibited enzyme. On a Lineweaver-Burk plot, the presence of a noncompetitive inhibitor is illustrated by a change in the y-intercept, defined as 1/Vmax. The x-intercept, defined as −1/<i>K</i><sub>M</sub>, will remain the same. In competitive inhibition, the inhibitor will bind to an enzyme at the active site, competing with the substrate. As a result, the <i>K</i><sub>M</sub> will increase and the Vmax will remain the same.<sup id="cite_ref-47" class="reference"><a href="#cite_note-47">&#91;47&#93;</a></sup> The common form of the inhibitory term also obscures the relationship between the inhibitor binding to the enzyme and its relationship to any other binding term be it the Michaelis–Menten equation or a dose response curve associated with ligand receptor binding. To demonstrate the relationship the following rearrangement can be made:
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\cfrac {V_{\max }}{1+{\cfrac {[I]}{K_{i}}}}}={\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{K_{i}}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <msub>
                    <mi>V</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mo movablelimits="true" form="prefix">max</mo>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mn>1</mn>
                  <mo>+</mo>
                  <mrow class="MJX-TeXAtom-ORD">
                    <mfrac>
                      <mrow>
                        <mpadded width="0" height="8.6pt" depth="3pt">
                          <mrow />
                        </mpadded>
                        <mstyle displaystyle="false" scriptlevel="0">
                          <mrow class="MJX-TeXAtom-ORD">
                            <mo stretchy="false">[</mo>
                            <mi>I</mi>
                            <mo stretchy="false">]</mo>
                          </mrow>
                        </mstyle>
                      </mrow>
                      <mrow>
                        <mpadded width="0" height="8.6pt" depth="3pt">
                          <mrow />
                        </mpadded>
                        <mstyle displaystyle="false" scriptlevel="0">
                          <mrow class="MJX-TeXAtom-ORD">
                            <msub>
                              <mi>K</mi>
                              <mrow class="MJX-TeXAtom-ORD">
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow>
                        </mstyle>
                      </mrow>
                    </mfrac>
                  </mrow>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <msub>
                    <mi>V</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mo movablelimits="true" form="prefix">max</mo>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">[</mo>
                          <mi>I</mi>
                          <mo stretchy="false">]</mo>
                          <mo>+</mo>
                          <msub>
                            <mi>K</mi>
                            <mrow class="MJX-TeXAtom-ORD">
                              <mi>i</mi>
                            </mrow>
                          </msub>
                        </mrow>
                      </mstyle>
                    </mrow>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <msub>
                            <mi>K</mi>
                            <mrow class="MJX-TeXAtom-ORD">
                              <mi>i</mi>
                            </mrow>
                          </msub>
                        </mrow>
                      </mstyle>
                    </mrow>
                  </mfrac>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\cfrac {V_{\max }}{1+{\cfrac {[I]}{K_{i}}}}}={\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{K_{i}}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/1c58cef49731511f5011822f1e92d4da22814891" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -6.671ex; width:21.297ex; height:10.843ex;" alt="{\displaystyle {\cfrac {V_{\max }}{1+{\cfrac {[I]}{K_{i}}}}}={\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{K_{i}}}}}"/></span></dd></dl>
<p>Adding zero to the bottom ([I]-[I])
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{[I]+K_{i}-[I]}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <msub>
                    <mi>V</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mo movablelimits="true" form="prefix">max</mo>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">[</mo>
                          <mi>I</mi>
                          <mo stretchy="false">]</mo>
                          <mo>+</mo>
                          <msub>
                            <mi>K</mi>
                            <mrow class="MJX-TeXAtom-ORD">
                              <mi>i</mi>
                            </mrow>
                          </msub>
                        </mrow>
                      </mstyle>
                    </mrow>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mo stretchy="false">[</mo>
                          <mi>I</mi>
                          <mo stretchy="false">]</mo>
                          <mo>+</mo>
                          <msub>
                            <mi>K</mi>
                            <mrow class="MJX-TeXAtom-ORD">
                              <mi>i</mi>
                            </mrow>
                          </msub>
                          <mo>&#x2212;<!-- − --></mo>
                          <mo stretchy="false">[</mo>
                          <mi>I</mi>
                          <mo stretchy="false">]</mo>
                        </mrow>
                      </mstyle>
                    </mrow>
                  </mfrac>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{[I]+K_{i}-[I]}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/42e34b0927bf9484b8a664e022d3fd6ba0ad2326" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -6.671ex; width:15.057ex; height:10.843ex;" alt="{\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{[I]+K_{i}-[I]}}}"/></span></dd></dl>
<p>Dividing by [I]+K<sub>i</sub>
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle {\cfrac {V_{\max }}{\cfrac {1}{1-{\cfrac {[I]}{[I]+K_{i}}}}}}=V_{\max }-V_{\max }{\cfrac {[I]}{[I]+K_{i}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <msub>
                    <mi>V</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mo movablelimits="true" form="prefix">max</mo>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mfrac>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>1</mn>
                        </mrow>
                      </mstyle>
                    </mrow>
                    <mrow>
                      <mpadded width="0" height="8.6pt" depth="3pt">
                        <mrow />
                      </mpadded>
                      <mstyle displaystyle="false" scriptlevel="0">
                        <mrow class="MJX-TeXAtom-ORD">
                          <mn>1</mn>
                          <mo>&#x2212;<!-- − --></mo>
                          <mrow class="MJX-TeXAtom-ORD">
                            <mfrac>
                              <mrow>
                                <mpadded width="0" height="8.6pt" depth="3pt">
                                  <mrow />
                                </mpadded>
                                <mstyle displaystyle="false" scriptlevel="0">
                                  <mrow class="MJX-TeXAtom-ORD">
                                    <mo stretchy="false">[</mo>
                                    <mi>I</mi>
                                    <mo stretchy="false">]</mo>
                                  </mrow>
                                </mstyle>
                              </mrow>
                              <mrow>
                                <mpadded width="0" height="8.6pt" depth="3pt">
                                  <mrow />
                                </mpadded>
                                <mstyle displaystyle="false" scriptlevel="0">
                                  <mrow class="MJX-TeXAtom-ORD">
                                    <mo stretchy="false">[</mo>
                                    <mi>I</mi>
                                    <mo stretchy="false">]</mo>
                                    <mo>+</mo>
                                    <msub>
                                      <mi>K</mi>
                                      <mrow class="MJX-TeXAtom-ORD">
                                        <mi>i</mi>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </mstyle>
                              </mrow>
                            </mfrac>
                          </mrow>
                        </mrow>
                      </mstyle>
                    </mrow>
                  </mfrac>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
        <mo>=</mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                  <mo>+</mo>
                  <msub>
                    <mi>K</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mi>i</mi>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle {\cfrac {V_{\max }}{\cfrac {1}{1-{\cfrac {[I]}{[I]+K_{i}}}}}}=V_{\max }-V_{\max }{\cfrac {[I]}{[I]+K_{i}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/4132fbd61b7474f8e3ef391f82d1d196a2a325ff" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -10.171ex; width:38.736ex; height:14.343ex;" alt="{\displaystyle {\cfrac {V_{\max }}{\cfrac {1}{1-{\cfrac {[I]}{[I]+K_{i}}}}}}=V_{\max }-V_{\max }{\cfrac {[I]}{[I]+K_{i}}}}"/></span></dd></dl>
<p>This notation demonstrates that similar to the Michaelis–Menten equation, where the rate of reaction depends on the percent of the enzyme population interacting with substrate, the effect of the inhibitor is a result of the percent of the enzyme population interacting with inhibitor.  The only problem with this equation in its present form is that it assumes absolute inhibition of the enzyme with inhibitor binding, when in fact there can be a wide range of effects anywhere from 100% inhibition of substrate turn over to just &gt;0%.  To account for this the equation can be easily modified to allow for different degrees of inhibition by including a delta <i>V</i><sub>max</sub> term.
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\max }-\Delta V_{\max }{\cfrac {[I]}{[I]+K_{i}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <mi mathvariant="normal">&#x0394;<!-- Δ --></mi>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
          </mrow>
        </msub>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                  <mo>+</mo>
                  <msub>
                    <mi>K</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mi>i</mi>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\max }-\Delta V_{\max }{\cfrac {[I]}{[I]+K_{i}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/90f5601fefd8114c165ac3dfb739e0642e62610c" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -3.005ex; width:22.984ex; height:7.176ex;" alt="V_{\max }-\Delta V_{\max }{\cfrac {[I]}{[I]+K_{i}}}"/></span></dd></dl>
<p>or
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[I]}{[I]+K_{i}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>2</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>I</mi>
                  <mo stretchy="false">]</mo>
                  <mo>+</mo>
                  <msub>
                    <mi>K</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mi>i</mi>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[I]}{[I]+K_{i}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/8c31b5f56aaeed122d1f8a67491c56d272686b6e" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -3.005ex; width:33.971ex; height:7.176ex;" alt="{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[I]}{[I]+K_{i}}}}"/></span></dd></dl>
<p>This term can then define the residual enzymatic activity present when the inhibitor is interacting with individual enzymes in the population.  However the inclusion of this term has the added value of allowing for the possibility of activation if the secondary <i>V</i><sub>max</sub> term turns out to be higher than the initial term. To account for the possibly of activation as well the notation can then be rewritten replacing the inhibitor "I" with a modifier term denoted here as "X".
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[X]}{[X]+K_{x}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <msub>
          <mi>V</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mo movablelimits="true" form="prefix">max</mo>
            <mn>2</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>X</mi>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>X</mi>
                  <mo stretchy="false">]</mo>
                  <mo>+</mo>
                  <msub>
                    <mi>K</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mi>x</mi>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[X]}{[X]+K_{x}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/2bb687fca77a00dba879a00fb32b7ca1bc867973" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -3.005ex; width:35.152ex; height:7.176ex;" alt="{\displaystyle V_{\max 1}-(V_{\max 1}-V_{\max 2}){\cfrac {[X]}{[X]+K_{x}}}}"/></span></dd></dl>
<p>While this terminology results in a simplified way of dealing with kinetic effects relating to the maximum velocity of the Michaelis–Menten equation, it highlights potential problems with the term used to describe effects relating to the <i>K</i><sub>M</sub>.  The <i>K</i><sub>M</sub> relating to the affinity of the enzyme for the substrate should in most cases relate to potential changes in the binding site of the enzyme which would directly result from enzyme inhibitor interactions.  As such a term similar to the one proposed above to modulate <i>V</i><sub>max</sub> should be appropriate in most situations:<sup id="cite_ref-48" class="reference"><a href="#cite_note-48">&#91;48&#93;</a></sup><sup id="cite_ref-49" class="reference"><a href="#cite_note-49">&#91;49&#93;</a></sup>
</p>
<dl><dd><span class="mwe-math-element"><span class="mwe-math-mathml-inline mwe-math-mathml-a11y" style="display: none;"><math xmlns="http://www.w3.org/1998/Math/MathML"  alttext="{\displaystyle K_{m1}-(K_{m1}-K_{m2}){\cfrac {[X]}{[X]+K_{x}}}}">
  <semantics>
    <mrow class="MJX-TeXAtom-ORD">
      <mstyle displaystyle="true" scriptlevel="0">
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>m</mi>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <mo stretchy="false">(</mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>m</mi>
            <mn>1</mn>
          </mrow>
        </msub>
        <mo>&#x2212;<!-- − --></mo>
        <msub>
          <mi>K</mi>
          <mrow class="MJX-TeXAtom-ORD">
            <mi>m</mi>
            <mn>2</mn>
          </mrow>
        </msub>
        <mo stretchy="false">)</mo>
        <mrow class="MJX-TeXAtom-ORD">
          <mfrac>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>X</mi>
                  <mo stretchy="false">]</mo>
                </mrow>
              </mstyle>
            </mrow>
            <mrow>
              <mpadded width="0" height="8.6pt" depth="3pt">
                <mrow />
              </mpadded>
              <mstyle displaystyle="false" scriptlevel="0">
                <mrow class="MJX-TeXAtom-ORD">
                  <mo stretchy="false">[</mo>
                  <mi>X</mi>
                  <mo stretchy="false">]</mo>
                  <mo>+</mo>
                  <msub>
                    <mi>K</mi>
                    <mrow class="MJX-TeXAtom-ORD">
                      <mi>x</mi>
                    </mrow>
                  </msub>
                </mrow>
              </mstyle>
            </mrow>
          </mfrac>
        </mrow>
      </mstyle>
    </mrow>
    <annotation encoding="application/x-tex">{\displaystyle K_{m1}-(K_{m1}-K_{m2}){\cfrac {[X]}{[X]+K_{x}}}}</annotation>
  </semantics>
</math></span><img src="https://wikimedia.org/api/rest_v1/media/math/render/svg/55a298dbb441e88b96a618cc31df3aaebac17d85" class="mwe-math-fallback-image-inline" aria-hidden="true" style="vertical-align: -3.005ex; width:30.996ex; height:7.176ex;" alt="{\displaystyle K_{m1}-(K_{m1}-K_{m2}){\cfrac {[X]}{[X]+K_{x}}}}"/></span></dd></dl>
<p>A few examples of reversible inhibition belonging to the competitive and uncompetitive models have been discussed in the following papers.<sup id="cite_ref-:0_50-0" class="reference"><a href="#cite_note-:0-50">&#91;50&#93;</a></sup><sup id="cite_ref-:1_51-0" class="reference"><a href="#cite_note-:1-51">&#91;51&#93;</a></sup><sup id="cite_ref-52" class="reference"><a href="#cite_note-52">&#91;52&#93;</a></sup>
</p>
<h3><span class="mw-headline" id="Irreversible_inhibitors">Irreversible inhibitors</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=18" title="Edit section: Irreversible inhibitors">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Enzyme inhibitors can also irreversibly inactivate enzymes, usually by covalently modifying active site residues. These reactions, which may be called suicide substrates, follow <a href="/wiki/Exponential_decay" title="Exponential decay">exponential decay</a> functions and are usually saturable. Below saturation, they follow <a href="/wiki/Reaction_rate" title="Reaction rate">first order</a> kinetics with respect to inhibitor. Irreversible inhibition could be classified into two distinct types. Affinity labelling is a type of irreversible inhibition where a functional group that is highly reactive modifies a catalytically critical residue on the protein of interest to bring about inhibition.  Mechanism-based inhibition, on the other hand, involves binding of the inhibitor followed by enzyme mediated alterations that transform the latter into a reactive group that irreversibly modifies the enzyme.
</p>
<h3><span class="mw-headline" id="Philosophical_discourse_on_reversibility_and_irreversibility_of_inhibition">Philosophical discourse on reversibility and irreversibility of inhibition</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=19" title="Edit section: Philosophical discourse on reversibility and irreversibility of inhibition">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Having discussed reversible inhibition and irreversible inhibition in the above two headings, it would have to be pointed out that the concept of reversibility ( or irreversibility) is a purely theoretical construct exclusively dependent on the time-frame of the assay, i.e., a reversible assay involving association and dissociation of the inhibitor molecule in the minute timescales would seem irreversible if an assay assess the outcome in the seconds and vice versa. There is a continuum of inhibitor behaviors spanning reversibility and irreversibility at a given non-arbitrary assay time frame.  There are inhibitors that show slow-onset behavior<sup id="cite_ref-:0_50-1" class="reference"><a href="#cite_note-:0-50">&#91;50&#93;</a></sup> and most of these inhibitors, invariably, also show tight-binding to the protein target of interest.<sup id="cite_ref-:0_50-2" class="reference"><a href="#cite_note-:0-50">&#91;50&#93;</a></sup><sup id="cite_ref-:1_51-1" class="reference"><a href="#cite_note-:1-51">&#91;51&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Mechanisms_of_catalysis">Mechanisms of catalysis</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=20" title="Edit section: Mechanisms of catalysis">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: <a href="/wiki/Enzyme_catalysis" title="Enzyme catalysis">Enzyme catalysis</a></div>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/File:Activation2_updated.svg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/e/e3/Activation2_updated.svg/300px-Activation2_updated.svg.png" decoding="async" width="300" height="235" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e3/Activation2_updated.svg/450px-Activation2_updated.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e3/Activation2_updated.svg/600px-Activation2_updated.svg.png 2x" data-file-width="504" data-file-height="395" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/File:Activation2_updated.svg" class="internal" title="Enlarge"></a></div>The energy variation as a function of <a href="/wiki/Reaction_coordinate" title="Reaction coordinate">reaction coordinate</a> shows the stabilisation of the transition state by an enzyme.</div></div></div>
<p>The favoured model for the enzyme–substrate interaction is the induced fit model.<sup id="cite_ref-53" class="reference"><a href="#cite_note-53">&#91;53&#93;</a></sup> This model proposes that the initial interaction between enzyme and substrate is relatively weak, but that these weak interactions rapidly induce <a href="/wiki/Conformational_change" title="Conformational change">conformational changes</a> in the enzyme that strengthen binding. These <a href="/wiki/Tertiary_structure" class="mw-redirect" title="Tertiary structure">conformational</a> changes also bring catalytic residues in the active site close to the chemical bonds in the substrate that will be altered in the reaction.<sup id="cite_ref-54" class="reference"><a href="#cite_note-54">&#91;54&#93;</a></sup> Conformational changes can be measured using <a href="/wiki/Circular_dichroism" title="Circular dichroism">circular dichroism</a> or <a href="/wiki/Dual_polarisation_interferometry" class="mw-redirect" title="Dual polarisation interferometry">dual polarisation interferometry</a>. After binding takes place, one or more mechanisms of catalysis lower the energy of the reaction's <a href="/wiki/Transition_state" title="Transition state">transition state</a> by providing an alternative chemical pathway for the reaction. Mechanisms of catalysis include catalysis by bond strain; by proximity and orientation; by active-site proton donors or acceptors; covalent catalysis and <a href="/wiki/Quantum_tunnelling" title="Quantum tunnelling">quantum tunnelling</a>.<sup id="cite_ref-Fersht_38-1" class="reference"><a href="#cite_note-Fersht-38">&#91;38&#93;</a></sup><sup id="cite_ref-55" class="reference"><a href="#cite_note-55">&#91;55&#93;</a></sup>
</p><p>Enzyme kinetics cannot prove which modes of catalysis are used by an enzyme. However, some kinetic data can suggest possibilities to be examined by other techniques. For example, a ping–pong mechanism with burst-phase pre-steady-state kinetics would suggest covalent catalysis might be important in this enzyme's mechanism. Alternatively, the observation of a strong pH effect on <i>V</i><sub>max</sub> but not <i>K</i><sub>M</sub> might indicate that a residue in the active site needs to be in a particular <a href="/wiki/Ionization" title="Ionization">ionisation</a> state for catalysis to occur.
</p>
<h2><span class="mw-headline" id="History">History</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=21" title="Edit section: History">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>In 1902 <a href="/wiki/Victor_Henri" title="Victor Henri">Victor Henri</a> proposed a quantitative theory of enzyme kinetics,<sup id="cite_ref-56" class="reference"><a href="#cite_note-56">&#91;56&#93;</a></sup> but at the time the experimental significance of the <a href="/wiki/PH" title="PH">hydrogen ion concentration</a> was not yet recognized. After <a href="/wiki/S._P._L._S%C3%B8rensen" title="S. P. L. Sørensen">Peter Lauritz Sørensen</a> had defined the logarithmic pH-scale and introduced the concept of <a href="/wiki/Buffer_solution" title="Buffer solution">buffering</a> in 1909<sup id="cite_ref-57" class="reference"><a href="#cite_note-57">&#91;57&#93;</a></sup> the German chemist <a href="/wiki/Leonor_Michaelis" title="Leonor Michaelis">Leonor Michaelis</a> and Dr. <a href="/wiki/Maud_Leonora_Menten" class="mw-redirect" title="Maud Leonora Menten">Maud Leonora Menten</a> (a postdoctoral researcher in Michaelis's lab at the time)repeated Henri's experiments and confirmed his equation, which is now generally referred to as <a href="/wiki/Michaelis-Menten_kinetics" class="mw-redirect" title="Michaelis-Menten kinetics">Michaelis-Menten kinetics</a> (sometimes also <i>Henri-Michaelis-Menten kinetics</i>).<sup id="cite_ref-58" class="reference"><a href="#cite_note-58">&#91;58&#93;</a></sup> Their work was further developed by <a href="/wiki/George_Edward_Briggs" title="George Edward Briggs">G. E. Briggs</a> and <a href="/wiki/J._B._S._Haldane" title="J. B. S. Haldane">J. B. S. Haldane</a>, who derived kinetic equations that are still widely considered today a starting point in modeling enzymatic activity.<sup id="cite_ref-59" class="reference"><a href="#cite_note-59">&#91;59&#93;</a></sup>
</p><p>The major contribution of the Henri-Michaelis-Menten approach was to think of enzyme reactions in two stages. In the first, the substrate binds reversibly to the enzyme, forming the enzyme-substrate complex. This is sometimes called the Michaelis complex. The enzyme then catalyzes the chemical step in the reaction and releases the product. The kinetics of many enzymes is adequately described by the simple Michaelis-Menten model, but all enzymes have <a href="/wiki/Protein_dynamics" title="Protein dynamics">internal motions</a> that are not accounted for in the model and can have significant contributions to the overall reaction kinetics. This can be modeled by introducing several Michaelis-Menten pathways that are connected with fluctuating rates,<sup id="cite_ref-OF_2005_60-0" class="reference"><a href="#cite_note-OF_2005-60">&#91;60&#93;</a></sup><sup id="cite_ref-English_2006_61-0" class="reference"><a href="#cite_note-English_2006-61">&#91;61&#93;</a></sup><sup id="cite_ref-Lu_1998_62-0" class="reference"><a href="#cite_note-Lu_1998-62">&#91;62&#93;</a></sup> which is a mathematical extension of the basic Michaelis Menten mechanism.<sup id="cite_ref-XX_2006_63-0" class="reference"><a href="#cite_note-XX_2006-63">&#91;63&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="Software">Software</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=22" title="Edit section: Software">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<h3><span class="mw-headline" id="ENZO">ENZO</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=23" title="Edit section: ENZO">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><b>ENZO</b> (Enzyme Kinetics) is a graphical interface tool for building kinetic models of enzyme catalyzed reactions. ENZO automatically generates the corresponding differential equations from a stipulated enzyme reaction scheme. These differential equations are processed by a numerical solver and a regression algorithm which fits the coefficients of differential equations to experimentally observed time course curves. ENZO allows rapid evaluation of rival reaction schemes and can be used for routine tests in enzyme kinetics.<sup id="cite_ref-64" class="reference"><a href="#cite_note-64">&#91;64&#93;</a></sup>
</p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=24" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul><li><a href="/wiki/Protein_dynamics" title="Protein dynamics">Protein dynamics</a></li>
<li><a href="/wiki/Diffusion_limited_enzyme" title="Diffusion limited enzyme">Diffusion limited enzyme</a></li>
<li><a href="/wiki/Langmuir_adsorption_model" title="Langmuir adsorption model">Langmuir adsorption model</a></li></ul>
<h2><span class="mw-headline" id="Footnotes">Footnotes</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=25" title="Edit section: Footnotes">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<dl><dd><b>α.</b> <span class="citation wikicite" id="endnote_Anone"><a href="#ref_Anone"><b><sup>^</sup></b></a></span> <a rel="nofollow" class="external text" href="https://web.archive.org/web/20120616044431/http://cti.itc.virginia.edu/~cmg/Demo/scriptFrame.html">Link: Interactive Michaelis–Menten kinetics tutorial (Java required)</a></dd></dl>
<dl><dd><b>β.</b> <span class="citation wikicite" id="endnote_Bnone"><a href="#ref_Bnone"><b><sup>^</sup></b></a></span> <a rel="nofollow" class="external text" href="https://web.archive.org/web/20060820062258/http://chem-faculty.ucsd.edu/kraut/dhfr.html">Link: dihydrofolate reductase mechanism (Gif)</a></dd></dl>
<dl><dd><b>γ.</b> <span class="citation wikicite" id="endnote_Cnone"><a href="#ref_Cnone"><b><sup>^</sup></b></a></span> <a rel="nofollow" class="external text" href="https://web.archive.org/web/20060514070741/http://chem-faculty.ucsd.edu/kraut/dNTP.html">Link: DNA polymerase mechanism (Gif)</a></dd></dl>
<dl><dd><b>δ.</b> <span class="citation wikicite" id="endnote_Dnone"><a href="#ref_Dnone"><b><sup>^</sup></b></a></span> <a rel="nofollow" class="external text" href="https://web.archive.org/web/20070319235224/http://courses.cm.utexas.edu/jrobertus/ch339k/overheads-2/06_21_chymotrypsin.html">Link: Chymotrypsin mechanism (Flash required)</a></dd></dl>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=26" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-width" style="-moz-column-width: 35em; -webkit-column-width: 35em; column-width: 35em; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-1"><span class="mw-cite-backlink"><b><a href="#cite_ref-1">^</a></b></span> <span class="reference-text"><cite class="citation book">Wrighton MS, Ebbing DD (1993). <i>General chemistry</i> (4th ed.). Boston: Houghton Mifflin. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-395-63696-1" title="Special:BookSources/978-0-395-63696-1"><bdi>978-0-395-63696-1</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=General+chemistry&amp;rft.place=Boston&amp;rft.edition=4th&amp;rft.pub=Houghton+Mifflin&amp;rft.date=1993&amp;rft.isbn=978-0-395-63696-1&amp;rft.aulast=Wrighton&amp;rft.aufirst=MS&amp;rft.au=Ebbing%2C+DD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><style data-mw-deduplicate="TemplateStyles:r886058088">.mw-parser-output cite.citation{font-style:inherit}.mw-parser-output .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .citation .cs1-lock-subscription a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-parser-output .cs1-registration span{border-bottom:1px dotted;cursor:help}.mw-parser-output .cs1-ws-icon a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output code.cs1-code{color:inherit;background:inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error{font-size:100%}.mw-parser-output .cs1-maint{display:none;color:#33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-wl-right{padding-right:0.2em}</style></span>
</li>
<li id="cite_note-Fromm_H.J._2012-2"><span class="mw-cite-backlink">^ <a href="#cite_ref-Fromm_H.J._2012_2-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Fromm_H.J._2012_2-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">Fromm H.J., Hargrove M.S. (2012) Enzyme Kinetics. In: Essentials of Biochemistry. Springer, Berlin, Heidelberg</span>
</li>
<li id="cite_note-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-3">^</a></b></span> <span class="reference-text"><cite class="citation book">Danson M, Eisenthal R (2002). <i>Enzyme assays: a practical approach</i>. Oxford [Oxfordshire]: Oxford University Press. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-19-963820-8" title="Special:BookSources/978-0-19-963820-8"><bdi>978-0-19-963820-8</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Enzyme+assays%3A+a+practical+approach&amp;rft.place=Oxford+%5BOxfordshire%5D&amp;rft.pub=Oxford+University+Press&amp;rft.date=2002&amp;rft.isbn=978-0-19-963820-8&amp;rft.aulast=Danson&amp;rft.aufirst=M&amp;rft.au=Eisenthal%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-4"><span class="mw-cite-backlink"><b><a href="#cite_ref-4">^</a></b></span> <span class="reference-text"><cite class="citation journal">Xie XS, Lu HP (June 1999). "Single-molecule enzymology". <i>The Journal of Biological Chemistry</i>. <b>274</b> (23): 15967–70. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1074%2Fjbc.274.23.15967">10.1074/jbc.274.23.15967</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10347141">10347141</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Journal+of+Biological+Chemistry&amp;rft.atitle=Single-molecule+enzymology&amp;rft.volume=274&amp;rft.issue=23&amp;rft.pages=15967-70&amp;rft.date=1999-06&amp;rft_id=info%3Adoi%2F10.1074%2Fjbc.274.23.15967&amp;rft_id=info%3Apmid%2F10347141&amp;rft.aulast=Xie&amp;rft.aufirst=XS&amp;rft.au=Lu%2C+HP&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-5">^</a></b></span> <span class="reference-text"><cite class="citation journal">Lu HP (June 2004). "Single-molecule spectroscopy studies of conformational change dynamics in enzymatic reactions". <i>Current Pharmaceutical Biotechnology</i>. <b>5</b> (3): 261–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.2174%2F1389201043376887">10.2174/1389201043376887</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15180547">15180547</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Current+Pharmaceutical+Biotechnology&amp;rft.atitle=Single-molecule+spectroscopy+studies+of+conformational+change+dynamics+in+enzymatic+reactions&amp;rft.volume=5&amp;rft.issue=3&amp;rft.pages=261-9&amp;rft.date=2004-06&amp;rft_id=info%3Adoi%2F10.2174%2F1389201043376887&amp;rft_id=info%3Apmid%2F15180547&amp;rft.aulast=Lu&amp;rft.aufirst=HP&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-6">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schnell JR, <a href="/wiki/Jane_Dyson" title="Jane Dyson">Dyson HJ</a>, Wright PE (2004). "Structure, dynamics, and catalytic function of dihydrofolate reductase". <i>Annual Review of Biophysics and Biomolecular Structure</i>. <b>33</b>: 119–40. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1146%2Fannurev.biophys.33.110502.133613">10.1146/annurev.biophys.33.110502.133613</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15139807">15139807</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Biophysics+and+Biomolecular+Structure&amp;rft.atitle=Structure%2C+dynamics%2C+and+catalytic+function+of+dihydrofolate+reductase&amp;rft.volume=33&amp;rft.pages=119-40&amp;rft.date=2004&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.biophys.33.110502.133613&amp;rft_id=info%3Apmid%2F15139807&amp;rft.aulast=Schnell&amp;rft.aufirst=JR&amp;rft.au=Dyson%2C+HJ&amp;rft.au=Wright%2C+PE&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-7">^</a></b></span> <span class="reference-text"><cite class="citation book">Gibson QH (1969). "&#91;6&#93; Rapid mixing: Stopped flow". <i>Rapid mixing: Stopped flow</i>. Methods in Enzymology. <b>16</b>. pp.&#160;187–228. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0076-6879%2869%2916009-7">10.1016/S0076-6879(69)16009-7</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-12-181873-9" title="Special:BookSources/978-0-12-181873-9"><bdi>978-0-12-181873-9</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=%26%2391%3B6%26%2393%3B+Rapid+mixing%3A+Stopped+flow&amp;rft.btitle=Rapid+mixing%3A+Stopped+flow&amp;rft.series=Methods+in+Enzymology&amp;rft.pages=187-228&amp;rft.date=1969&amp;rft_id=info%3Adoi%2F10.1016%2FS0076-6879%2869%2916009-7&amp;rft.isbn=978-0-12-181873-9&amp;rft.au=Gibson+QH&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-8"><span class="mw-cite-backlink"><b><a href="#cite_ref-8">^</a></b></span> <span class="reference-text"><cite class="citation book">Duggleby RG (1995). "&#91;3&#93; Analysis of enzyme progress curves by nonlinear regression". <i>Analysis of enzyme progress curves by non-linear regression</i>. Methods in Enzymology. <b>249</b>. pp.&#160;61–90. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2F0076-6879%2895%2949031-0">10.1016/0076-6879(95)49031-0</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-12-182150-0" title="Special:BookSources/978-0-12-182150-0"><bdi>978-0-12-182150-0</bdi></a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/7791628">7791628</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=%26%2391%3B3%26%2393%3B+Analysis+of+enzyme+progress+curves+by+nonlinear+regression&amp;rft.btitle=Analysis+of+enzyme+progress+curves+by+non-linear+regression&amp;rft.series=Methods+in+Enzymology&amp;rft.pages=61-90&amp;rft.date=1995&amp;rft_id=info%3Apmid%2F7791628&amp;rft_id=info%3Adoi%2F10.1016%2F0076-6879%2895%2949031-0&amp;rft.isbn=978-0-12-182150-0&amp;rft.au=Duggleby+RG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text"><cite class="citation journal">Murray JB, Dunham CM, Scott WG (January 2002). "A pH-dependent conformational change, rather than the chemical step, appears to be rate-limiting in the hammerhead ribozyme cleavage reaction". <i>Journal of Molecular Biology</i>. <b>315</b> (2): 121–30. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1006%2Fjmbi.2001.5145">10.1006/jmbi.2001.5145</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11779233">11779233</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Molecular+Biology&amp;rft.atitle=A+pH-dependent+conformational+change%2C+rather+than+the+chemical+step%2C+appears+to+be+rate-limiting+in+the+hammerhead+ribozyme+cleavage+reaction&amp;rft.volume=315&amp;rft.issue=2&amp;rft.pages=121-30&amp;rft.date=2002-01&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2001.5145&amp;rft_id=info%3Apmid%2F11779233&amp;rft.aulast=Murray&amp;rft.aufirst=JB&amp;rft.au=Dunham%2C+CM&amp;rft.au=Scott%2C+WG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-10">^</a></b></span> <span class="reference-text">Michaelis L. and Menten M.L. <i>Kinetik der Invertinwirkung</i> Biochem. Z. 1913; 49:333–369 <a rel="nofollow" class="external text" href="http://web.lemoyne.edu/~giunta/menten.html">English translation</a> Accessed 6 April 2007</span>
</li>
<li id="cite_note-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-11">^</a></b></span> <span class="reference-text"><cite class="citation journal">Stroppolo ME, Falconi M, Caccuri AM, Desideri A (September 2001). "Superefficient enzymes". <i>Cellular and Molecular Life Sciences</i>. <b>58</b> (10): 1451–60. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1007%2FPL00000788">10.1007/PL00000788</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11693526">11693526</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cellular+and+Molecular+Life+Sciences&amp;rft.atitle=Superefficient+enzymes&amp;rft.volume=58&amp;rft.issue=10&amp;rft.pages=1451-60&amp;rft.date=2001-09&amp;rft_id=info%3Adoi%2F10.1007%2FPL00000788&amp;rft_id=info%3Apmid%2F11693526&amp;rft.aulast=Stroppolo&amp;rft.aufirst=ME&amp;rft.au=Falconi%2C+M&amp;rft.au=Caccuri%2C+AM&amp;rft.au=Desideri%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-Bar-Even_2011-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-Bar-Even_2011_12-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bar-Even A, Noor E, Savir Y, Liebermeister W, Davidi D, Tawfik DS, Milo R (May 2011). "The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters". <i>Biochemistry</i>. <b>50</b> (21): 4402–10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fbi2002289">10.1021/bi2002289</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21506553">21506553</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry&amp;rft.atitle=The+moderately+efficient+enzyme%3A+evolutionary+and+physicochemical+trends+shaping+enzyme+parameters&amp;rft.volume=50&amp;rft.issue=21&amp;rft.pages=4402-10&amp;rft.date=2011-05&amp;rft_id=info%3Adoi%2F10.1021%2Fbi2002289&amp;rft_id=info%3Apmid%2F21506553&amp;rft.aulast=Bar-Even&amp;rft.aufirst=A&amp;rft.au=Noor%2C+E&amp;rft.au=Savir%2C+Y&amp;rft.au=Liebermeister%2C+W&amp;rft.au=Davidi%2C+D&amp;rft.au=Tawfik%2C+DS&amp;rft.au=Milo%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-13">^</a></b></span> <span class="reference-text"><cite class="citation journal">Walsh R, Martin E, Darvesh S (January 2010). "A method to describe enzyme-catalyzed reactions by combining steady state and time course enzyme kinetic parameters". <i>Biochimica et Biophysica Acta</i>. <b>1800</b> (1): 1–5. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.bbagen.2009.10.007">10.1016/j.bbagen.2009.10.007</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19840832">19840832</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochimica+et+Biophysica+Acta&amp;rft.atitle=A+method+to+describe+enzyme-catalyzed+reactions+by+combining+steady+state+and+time+course+enzyme+kinetic+parameters&amp;rft.volume=1800&amp;rft.issue=1&amp;rft.pages=1-5&amp;rft.date=2010-01&amp;rft_id=info%3Adoi%2F10.1016%2Fj.bbagen.2009.10.007&amp;rft_id=info%3Apmid%2F19840832&amp;rft.aulast=Walsh&amp;rft.aufirst=R&amp;rft.au=Martin%2C+E&amp;rft.au=Darvesh%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-14">^</a></b></span> <span class="reference-text"><cite class="citation journal">Beal SL (December 1983). "Computation of the explicit solution to the Michaelis-Menten equation". <i>Journal of Pharmacokinetics and Biopharmaceutics</i>. <b>11</b> (6): 641–57. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1007%2FBF01059062">10.1007/BF01059062</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/6689584">6689584</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Pharmacokinetics+and+Biopharmaceutics&amp;rft.atitle=Computation+of+the+explicit+solution+to+the+Michaelis-Menten+equation&amp;rft.volume=11&amp;rft.issue=6&amp;rft.pages=641-57&amp;rft.date=1983-12&amp;rft_id=info%3Adoi%2F10.1007%2FBF01059062&amp;rft_id=info%3Apmid%2F6689584&amp;rft.aulast=Beal&amp;rft.aufirst=SL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-15">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schnell S, Mendoza C (1997). "Closed Form Solution for Time-dependent Enzyme Kinetics". <i>Journal of Theoretical Biology</i>. <b>187</b> (2): 207–212. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1006%2Fjtbi.1997.0425">10.1006/jtbi.1997.0425</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Theoretical+Biology&amp;rft.atitle=Closed+Form+Solution+for+Time-dependent+Enzyme+Kinetics&amp;rft.volume=187&amp;rft.issue=2&amp;rft.pages=207-212&amp;rft.date=1997&amp;rft_id=info%3Adoi%2F10.1006%2Fjtbi.1997.0425&amp;rft.aulast=Schnell&amp;rft.aufirst=S&amp;rft.au=Mendoza%2C+C&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-16">^</a></b></span> <span class="reference-text"><cite class="citation journal">Goudar CT, Sonnad JR, Duggleby RG (January 1999). <a rel="nofollow" class="external text" href="https://www.webcitation.org/6cuXxfqIj?url=http://rdbiotech.drivehq.com/Biblio/Pubs/Pub103.pdf">"Parameter estimation using a direct solution of the integrated Michaelis-Menten equation"</a> <span class="cs1-format">(PDF)</span>. <i>Biochimica et Biophysica Acta</i>. <b>1429</b> (2): 377–83. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fs0167-4838%2898%2900247-7">10.1016/s0167-4838(98)00247-7</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9989222">9989222</a>. Archived from <a rel="nofollow" class="external text" href="http://rdbiotech.drivehq.com/Biblio/Pubs/Pub103.pdf">the original</a> <span class="cs1-format">(PDF)</span> on 9 November 2015.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochimica+et+Biophysica+Acta&amp;rft.atitle=Parameter+estimation+using+a+direct+solution+of+the+integrated+Michaelis-Menten+equation&amp;rft.volume=1429&amp;rft.issue=2&amp;rft.pages=377-83&amp;rft.date=1999-01&amp;rft_id=info%3Adoi%2F10.1016%2Fs0167-4838%2898%2900247-7&amp;rft_id=info%3Apmid%2F9989222&amp;rft.aulast=Goudar&amp;rft.aufirst=CT&amp;rft.au=Sonnad%2C+JR&amp;rft.au=Duggleby%2C+RG&amp;rft_id=http%3A%2F%2Frdbiotech.drivehq.com%2FBiblio%2FPubs%2FPub103.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-17">^</a></b></span> <span class="reference-text"><cite class="citation journal">Goudar CT, Harris SK, McInerney MJ, Suflita JM (December 2004). "Progress curve analysis for enzyme and microbial kinetic reactions using explicit solutions based on the Lambert W function". <i>Journal of Microbiological Methods</i>. <b>59</b> (3): 317–26. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.mimet.2004.06.013">10.1016/j.mimet.2004.06.013</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15488275">15488275</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Microbiological+Methods&amp;rft.atitle=Progress+curve+analysis+for+enzyme+and+microbial+kinetic+reactions+using+explicit+solutions+based+on+the+Lambert+W+function&amp;rft.volume=59&amp;rft.issue=3&amp;rft.pages=317-26&amp;rft.date=2004-12&amp;rft_id=info%3Adoi%2F10.1016%2Fj.mimet.2004.06.013&amp;rft_id=info%3Apmid%2F15488275&amp;rft.aulast=Goudar&amp;rft.aufirst=CT&amp;rft.au=Harris%2C+SK&amp;rft.au=McInerney%2C+MJ&amp;rft.au=Suflita%2C+JM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-18">^</a></b></span> <span class="reference-text"><cite class="citation journal">Berberan-Santos MN (2010). <a rel="nofollow" class="external text" href="http://match.pmf.kg.ac.rs/electronic_versions/Match63/n2/match63n2_283-318.pdf">"A General Treatment of Henri Michaelis Menten Enzyme Kinetics: Exact Series Solution and Approximate Analytical Solutions"</a> <span class="cs1-format">(PDF)</span>. <i>MATCH Communications in Mathematical and in Computer Chemistry</i>. <b>63</b>: 283.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=MATCH+Communications+in+Mathematical+and+in+Computer+Chemistry&amp;rft.atitle=A+General+Treatment+of+Henri+Michaelis+Menten+Enzyme+Kinetics%3A+Exact+Series+Solution+and+Approximate+Analytical+Solutions&amp;rft.volume=63&amp;rft.pages=283&amp;rft.date=2010&amp;rft.aulast=Berberan-Santos&amp;rft.aufirst=MN&amp;rft_id=http%3A%2F%2Fmatch.pmf.kg.ac.rs%2Felectronic_versions%2FMatch63%2Fn2%2Fmatch63n2_283-318.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-19">^</a></b></span> <span class="reference-text"><cite class="citation journal">Jones ME (December 1992). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1132043">"Analysis of algebraic weighted least-squares estimators for enzyme parameters"</a>. <i>The Biochemical Journal</i>. 288 ( Pt 2) (Pt 2): 533–8. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1042%2Fbj2880533">10.1042/bj2880533</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1132043">1132043</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/1463456">1463456</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Biochemical+Journal&amp;rft.atitle=Analysis+of+algebraic+weighted+least-squares+estimators+for+enzyme+parameters&amp;rft.volume=288+%28+Pt+2%29&amp;rft.issue=Pt+2&amp;rft.pages=533-8&amp;rft.date=1992-12&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1132043&amp;rft_id=info%3Apmid%2F1463456&amp;rft_id=info%3Adoi%2F10.1042%2Fbj2880533&amp;rft.aulast=Jones&amp;rft.aufirst=ME&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1132043&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-Tseng-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-Tseng_20-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tseng SJ, Hsu JP (August 1990). "A comparison of the parameter estimating procedures for the Michaelis-Menten model". <i>Journal of Theoretical Biology</i>. <b>145</b> (4): 457–64. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0022-5193%2805%2980481-3">10.1016/S0022-5193(05)80481-3</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/2246896">2246896</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Theoretical+Biology&amp;rft.atitle=A+comparison+of+the+parameter+estimating+procedures+for+the+Michaelis-Menten+model&amp;rft.volume=145&amp;rft.issue=4&amp;rft.pages=457-64&amp;rft.date=1990-08&amp;rft_id=info%3Adoi%2F10.1016%2FS0022-5193%2805%2980481-3&amp;rft_id=info%3Apmid%2F2246896&amp;rft.aulast=Tseng&amp;rft.aufirst=SJ&amp;rft.au=Hsu%2C+JP&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-21"><span class="mw-cite-backlink"><b><a href="#cite_ref-21">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bravo IG, Busto F, De Arriaga D, Ferrero MA, Rodríguez-Aparicio LB, Martínez-Blanco H, Reglero A (September 2001). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1222113">"A normalized plot as a novel and time-saving tool in complex enzyme kinetic analysis"</a>. <i>The Biochemical Journal</i>. <b>358</b> (Pt 3): 573–83. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1042%2Fbj3580573">10.1042/bj3580573</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1222113">1222113</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11577687">11577687</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Biochemical+Journal&amp;rft.atitle=A+normalized+plot+as+a+novel+and+time-saving+tool+in+complex+enzyme+kinetic+analysis&amp;rft.volume=358&amp;rft.issue=Pt+3&amp;rft.pages=573-83&amp;rft.date=2001-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1222113&amp;rft_id=info%3Apmid%2F11577687&amp;rft_id=info%3Adoi%2F10.1042%2Fbj3580573&amp;rft.aulast=Bravo&amp;rft.aufirst=IG&amp;rft.au=Busto%2C+F&amp;rft.au=De+Arriaga%2C+D&amp;rft.au=Ferrero%2C+MA&amp;rft.au=Rodr%C3%ADguez-Aparicio%2C+LB&amp;rft.au=Mart%C3%ADnez-Blanco%2C+H&amp;rft.au=Reglero%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1222113&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-22">^</a></b></span> <span class="reference-text"><cite class="citation journal">Almaas E, Kovács B, Vicsek T, Oltvai ZN, Barabási AL (February 2004). "Global organization of metabolic fluxes in the bacterium Escherichia coli". <i>Nature</i>. <b>427</b> (6977): 839–43. <a href="/wiki/ArXiv" title="ArXiv">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/q-bio/0403001">q-bio/0403001</a></span>. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2004Natur.427..839A">2004Natur.427..839A</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1038%2Fnature02289">10.1038/nature02289</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/14985762">14985762</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Global+organization+of+metabolic+fluxes+in+the+bacterium+Escherichia+coli&amp;rft.volume=427&amp;rft.issue=6977&amp;rft.pages=839-43&amp;rft.date=2004-02&amp;rft_id=info%3Aarxiv%2Fq-bio%2F0403001&amp;rft_id=info%3Apmid%2F14985762&amp;rft_id=info%3Adoi%2F10.1038%2Fnature02289&amp;rft_id=info%3Abibcode%2F2004Natur.427..839A&amp;rft.aulast=Almaas&amp;rft.aufirst=E&amp;rft.au=Kov%C3%A1cs%2C+B&amp;rft.au=Vicsek%2C+T&amp;rft.au=Oltvai%2C+ZN&amp;rft.au=Barab%C3%A1si%2C+AL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-23">^</a></b></span> <span class="reference-text"><cite class="citation journal">Reed JL, Vo TD, Schilling CH, Palsson BO (2003). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC193654">"An expanded genome-scale model of Escherichia coli K-12 (iJR904 GSM/GPR)"</a>. <i>Genome Biology</i>. <b>4</b> (9): R54. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1186%2Fgb-2003-4-9-r54">10.1186/gb-2003-4-9-r54</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC193654">193654</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12952533">12952533</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Biology&amp;rft.atitle=An+expanded+genome-scale+model+of+Escherichia+coli+K-12+%28iJR904+GSM%2FGPR%29&amp;rft.volume=4&amp;rft.issue=9&amp;rft.pages=R54&amp;rft.date=2003&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC193654&amp;rft_id=info%3Apmid%2F12952533&amp;rft_id=info%3Adoi%2F10.1186%2Fgb-2003-4-9-r54&amp;rft.aulast=Reed&amp;rft.aufirst=JL&amp;rft.au=Vo%2C+TD&amp;rft.au=Schilling%2C+CH&amp;rft.au=Palsson%2C+BO&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC193654&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-24">^</a></b></span> <span class="reference-text">for a complete derivation, see <a href="https://commons.wikimedia.org/wiki/File:Enzyme_Kinetics.pdf" class="extiw" title="commons:File:Enzyme Kinetics.pdf">here</a></span>
</li>
<li id="cite_note-25"><span class="mw-cite-backlink"><b><a href="#cite_ref-25">^</a></b></span> <span class="reference-text"><cite class="citation journal">Dirr H, Reinemer P, Huber R (March 1994). "X-ray crystal structures of cytosolic glutathione S-transferases. Implications for protein architecture, substrate recognition and catalytic function". <i>European Journal of Biochemistry</i>. <b>220</b> (3): 645–61. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1111%2Fj.1432-1033.1994.tb18666.x">10.1111/j.1432-1033.1994.tb18666.x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/8143720">8143720</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=European+Journal+of+Biochemistry&amp;rft.atitle=X-ray+crystal+structures+of+cytosolic+glutathione+S-transferases.+Implications+for+protein+architecture%2C+substrate+recognition+and+catalytic+function&amp;rft.volume=220&amp;rft.issue=3&amp;rft.pages=645-61&amp;rft.date=1994-03&amp;rft_id=info%3Adoi%2F10.1111%2Fj.1432-1033.1994.tb18666.x&amp;rft_id=info%3Apmid%2F8143720&amp;rft.aulast=Dirr&amp;rft.aufirst=H&amp;rft.au=Reinemer%2C+P&amp;rft.au=Huber%2C+R&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-26">^</a></b></span> <span class="reference-text"><cite class="citation journal">Stone SR, Morrison JF (July 1988). "Dihydrofolate reductase from Escherichia coli: the kinetic mechanism with NADPH and reduced acetylpyridine adenine dinucleotide phosphate as substrates". <i>Biochemistry</i>. <b>27</b> (15): 5493–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fbi00415a016">10.1021/bi00415a016</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3052577">3052577</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry&amp;rft.atitle=Dihydrofolate+reductase+from+Escherichia+coli%3A+the+kinetic+mechanism+with+NADPH+and+reduced+acetylpyridine+adenine+dinucleotide+phosphate+as+substrates&amp;rft.volume=27&amp;rft.issue=15&amp;rft.pages=5493-9&amp;rft.date=1988-07&amp;rft_id=info%3Adoi%2F10.1021%2Fbi00415a016&amp;rft_id=info%3Apmid%2F3052577&amp;rft.aulast=Stone&amp;rft.aufirst=SR&amp;rft.au=Morrison%2C+JF&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-27">^</a></b></span> <span class="reference-text"><cite class="citation book">Fisher PA (1994). <i>Enzymologic mechanism of replicative DNA polymerases in higher eukaryotes</i>. Progress in Nucleic Acid Research and Molecular Biology. <b>47</b>. pp.&#160;371–97. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0079-6603%2808%2960257-3">10.1016/S0079-6603(08)60257-3</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-12-540047-3" title="Special:BookSources/978-0-12-540047-3"><bdi>978-0-12-540047-3</bdi></a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/8016325">8016325</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Enzymologic+mechanism+of+replicative+DNA+polymerases+in+higher+eukaryotes&amp;rft.series=Progress+in+Nucleic+Acid+Research+and+Molecular+Biology&amp;rft.pages=371-97&amp;rft.date=1994&amp;rft_id=info%3Apmid%2F8016325&amp;rft_id=info%3Adoi%2F10.1016%2FS0079-6603%2808%2960257-3&amp;rft.isbn=978-0-12-540047-3&amp;rft.au=Fisher+PA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-28">^</a></b></span> <span class="reference-text"><cite class="citation journal">Akerman SE, Müller S (August 2003). "2-Cys peroxiredoxin PfTrx-Px1 is involved in the antioxidant defence of Plasmodium falciparum". <i>Molecular and Biochemical Parasitology</i>. <b>130</b> (2): 75–81. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2FS0166-6851%2803%2900161-0">10.1016/S0166-6851(03)00161-0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12946843">12946843</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Molecular+and+Biochemical+Parasitology&amp;rft.atitle=2-Cys+peroxiredoxin+PfTrx-Px1+is+involved+in+the+antioxidant+defence+of+Plasmodium+falciparum&amp;rft.volume=130&amp;rft.issue=2&amp;rft.pages=75-81&amp;rft.date=2003-08&amp;rft_id=info%3Adoi%2F10.1016%2FS0166-6851%2803%2900161-0&amp;rft_id=info%3Apmid%2F12946843&amp;rft.aulast=Akerman&amp;rft.aufirst=SE&amp;rft.au=M%C3%BCller%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-29">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bravo IG, Barrallo S, Ferrero MA, Rodríguez-Aparicio LB, Martínez-Blanco H, Reglero A (September 2001). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1222114">"Kinetic properties of the acylneuraminate cytidylyltransferase from Pasteurella haemolytica A2"</a>. <i>The Biochemical Journal</i>. <b>358</b> (Pt 3): 585–98. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1042%2Fbj3580585">10.1042/bj3580585</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1222114">1222114</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11577688">11577688</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Biochemical+Journal&amp;rft.atitle=Kinetic+properties+of+the+acylneuraminate+cytidylyltransferase+from+Pasteurella+haemolytica+A2&amp;rft.volume=358&amp;rft.issue=Pt+3&amp;rft.pages=585-98&amp;rft.date=2001-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1222114&amp;rft_id=info%3Apmid%2F11577688&amp;rft_id=info%3Adoi%2F10.1042%2Fbj3580585&amp;rft.aulast=Bravo&amp;rft.aufirst=IG&amp;rft.au=Barrallo%2C+S&amp;rft.au=Ferrero%2C+MA&amp;rft.au=Rodr%C3%ADguez-Aparicio%2C+LB&amp;rft.au=Mart%C3%ADnez-Blanco%2C+H&amp;rft.au=Reglero%2C+A&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1222114&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-30"><span class="mw-cite-backlink"><b><a href="#cite_ref-30">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kraut J (1977). "Serine proteases: structure and mechanism of catalysis". <i>Annual Review of Biochemistry</i>. <b>46</b>: 331–58. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1146%2Fannurev.bi.46.070177.001555">10.1146/annurev.bi.46.070177.001555</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/332063">332063</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Biochemistry&amp;rft.atitle=Serine+proteases%3A+structure+and+mechanism+of+catalysis&amp;rft.volume=46&amp;rft.pages=331-58&amp;rft.date=1977&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.bi.46.070177.001555&amp;rft_id=info%3Apmid%2F332063&amp;rft.aulast=Kraut&amp;rft.aufirst=J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-31">^</a></b></span> <span class="reference-text"><cite class="citation book">Cornish-Bowden A (2004). <i>Fundamentals of Enzyme Kinetics</i>. Portland Press. <q>Some enzymes are much more effective catalysts for one direction than the other. As a striking example, the limiting rates of the forward reaction catalyzed by methionine adenosyltransferase is about 10<sup>5</sup> greater than that for the reverse direction, even though the equilibrium constant is close to unity (page 53).</q></cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Fundamentals+of+Enzyme+Kinetics&amp;rft.pub=Portland+Press&amp;rft.date=2004&amp;rft.aulast=Cornish-Bowden&amp;rft.aufirst=Athel&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-32"><span class="mw-cite-backlink"><b><a href="#cite_ref-32">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ricard J, Cornish-Bowden A (July 1987). "Co-operative and allosteric enzymes: 20 years on". <i>European Journal of Biochemistry</i>. <b>166</b> (2): 255–72. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1111%2Fj.1432-1033.1987.tb13510.x">10.1111/j.1432-1033.1987.tb13510.x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3301336">3301336</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=European+Journal+of+Biochemistry&amp;rft.atitle=Co-operative+and+allosteric+enzymes%3A+20+years+on&amp;rft.volume=166&amp;rft.issue=2&amp;rft.pages=255-72&amp;rft.date=1987-07&amp;rft_id=info%3Adoi%2F10.1111%2Fj.1432-1033.1987.tb13510.x&amp;rft_id=info%3Apmid%2F3301336&amp;rft.aulast=Ricard&amp;rft.aufirst=J&amp;rft.au=Cornish-Bowden%2C+A&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-33"><span class="mw-cite-backlink"><b><a href="#cite_ref-33">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ward WH, Fersht AR (July 1988). "Tyrosyl-tRNA synthetase acts as an asymmetric dimer in charging tRNA. A rationale for half-of-the-sites activity". <i>Biochemistry</i>. <b>27</b> (15): 5525–30. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fbi00415a021">10.1021/bi00415a021</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3179266">3179266</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry&amp;rft.atitle=Tyrosyl-tRNA+synthetase+acts+as+an+asymmetric+dimer+in+charging+tRNA.+A+rationale+for+half-of-the-sites+activity&amp;rft.volume=27&amp;rft.issue=15&amp;rft.pages=5525-30&amp;rft.date=1988-07&amp;rft_id=info%3Adoi%2F10.1021%2Fbi00415a021&amp;rft_id=info%3Apmid%2F3179266&amp;rft.aulast=Ward&amp;rft.aufirst=WH&amp;rft.au=Fersht%2C+AR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-34">^</a></b></span> <span class="reference-text"><cite class="citation journal">Helmstaedt K, Krappmann S, Braus GH (September 2001). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC99034">"Allosteric regulation of catalytic activity: Escherichia coli aspartate transcarbamoylase versus yeast chorismate mutase"</a>. <i>Microbiology and Molecular Biology Reviews</i>. <b>65</b> (3): 404–21, table of contents. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1128%2FMMBR.65.3.404-421.2001">10.1128/MMBR.65.3.404-421.2001</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC99034">99034</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11528003">11528003</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Microbiology+and+Molecular+Biology+Reviews&amp;rft.atitle=Allosteric+regulation+of+catalytic+activity%3A+Escherichia+coli+aspartate+transcarbamoylase+versus+yeast+chorismate+mutase&amp;rft.volume=65&amp;rft.issue=3&amp;rft.pages=404-21%2C+table+of+contents&amp;rft.date=2001-09&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC99034&amp;rft_id=info%3Apmid%2F11528003&amp;rft_id=info%3Adoi%2F10.1128%2FMMBR.65.3.404-421.2001&amp;rft.aulast=Helmstaedt&amp;rft.aufirst=K&amp;rft.au=Krappmann%2C+S&amp;rft.au=Braus%2C+GH&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC99034&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-35">^</a></b></span> <span class="reference-text"><cite class="citation journal">Schirmer T, Evans PR (January 1990). "Structural basis of the allosteric behaviour of phosphofructokinase". <i>Nature</i>. <b>343</b> (6254): 140–5. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1990Natur.343..140S">1990Natur.343..140S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1038%2F343140a0">10.1038/343140a0</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/2136935">2136935</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Structural+basis+of+the+allosteric+behaviour+of+phosphofructokinase&amp;rft.volume=343&amp;rft.issue=6254&amp;rft.pages=140-5&amp;rft.date=1990-01&amp;rft_id=info%3Apmid%2F2136935&amp;rft_id=info%3Adoi%2F10.1038%2F343140a0&amp;rft_id=info%3Abibcode%2F1990Natur.343..140S&amp;rft.aulast=Schirmer&amp;rft.aufirst=T&amp;rft.au=Evans%2C+PR&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-36">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hill AV (1910). "The possible effects of the aggregation of the molecules of haemoglobin on its dissociation curves". <i>J. Physiol</i>. <b>40</b>: iv–vii.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=J.+Physiol.&amp;rft.atitle=The+possible+effects+of+the+aggregation+of+the+molecules+of+haemoglobin+on+its+dissociation+curves.&amp;rft.volume=40&amp;rft.pages=iv-vii&amp;rft.date=1910&amp;rft.aulast=Hill&amp;rft.aufirst=AV&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-37"><span class="mw-cite-backlink"><b><a href="#cite_ref-37">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hartley BS, Kilby BA (February 1954). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1269615">"The reaction of p-nitrophenyl esters with chymotrypsin and insulin"</a>. <i>The Biochemical Journal</i>. <b>56</b> (2): 288–97. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1042%2Fbj0560288">10.1042/bj0560288</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1269615">1269615</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/13140189">13140189</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Biochemical+Journal&amp;rft.atitle=The+reaction+of+p-nitrophenyl+esters+with+chymotrypsin+and+insulin&amp;rft.volume=56&amp;rft.issue=2&amp;rft.pages=288-97&amp;rft.date=1954-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1269615&amp;rft_id=info%3Apmid%2F13140189&amp;rft_id=info%3Adoi%2F10.1042%2Fbj0560288&amp;rft.aulast=Hartley&amp;rft.aufirst=BS&amp;rft.au=Kilby%2C+BA&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1269615&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-Fersht-38"><span class="mw-cite-backlink">^ <a href="#cite_ref-Fersht_38-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Fersht_38-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation book">Fersht, Alan (1999). <i>Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding</i>. San Francisco: W.H. Freeman. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-7167-3268-6" title="Special:BookSources/978-0-7167-3268-6"><bdi>978-0-7167-3268-6</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Structure+and+mechanism+in+protein+science%3A+a+guide+to+enzyme+catalysis+and+protein+folding&amp;rft.place=San+Francisco&amp;rft.pub=W.H.+Freeman&amp;rft.date=1999&amp;rft.isbn=978-0-7167-3268-6&amp;rft.au=Fersht%2C+Alan&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-39"><span class="mw-cite-backlink"><b><a href="#cite_ref-39">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bender ML, Begué-Cantón ML, Blakeley RL, Brubacher LJ, Feder J, Gunter CR, Kézdy FJ, Killheffer JV, Marshall TH, Miller CG, Roeske RW, Stoops JK (December 1966). "The determination of the concentration of hydrolytic enzyme solutions: alpha-chymotrypsin, trypsin, papain, elastase, subtilisin, and acetylcholinesterase". <i>Journal of the American Chemical Society</i>. <b>88</b> (24): 5890–913. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fja00976a034">10.1021/ja00976a034</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/5980876">5980876</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+the+American+Chemical+Society&amp;rft.atitle=The+determination+of+the+concentration+of+hydrolytic+enzyme+solutions%3A+alpha-chymotrypsin%2C+trypsin%2C+papain%2C+elastase%2C+subtilisin%2C+and+acetylcholinesterase&amp;rft.volume=88&amp;rft.issue=24&amp;rft.pages=5890-913&amp;rft.date=1966-12&amp;rft_id=info%3Adoi%2F10.1021%2Fja00976a034&amp;rft_id=info%3Apmid%2F5980876&amp;rft.aulast=Bender&amp;rft.aufirst=ML&amp;rft.au=Begu%C3%A9-Cant%C3%B3n%2C+ML&amp;rft.au=Blakeley%2C+RL&amp;rft.au=Brubacher%2C+LJ&amp;rft.au=Feder%2C+J&amp;rft.au=Gunter%2C+CR&amp;rft.au=K%C3%A9zdy%2C+FJ&amp;rft.au=Killheffer%2C+JV&amp;rft.au=Marshall%2C+TH&amp;rft.au=Miller%2C+CG&amp;rft.au=Roeske%2C+RW&amp;rft.au=Stoops%2C+JK&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-40"><span class="mw-cite-backlink"><b><a href="#cite_ref-40">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cleland WW (January 2005). "The use of isotope effects to determine enzyme mechanisms". <i>Archives of Biochemistry and Biophysics</i>. <b>433</b> (1): 2–12. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.abb.2004.08.027">10.1016/j.abb.2004.08.027</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15581561">15581561</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Archives+of+Biochemistry+and+Biophysics&amp;rft.atitle=The+use+of+isotope+effects+to+determine+enzyme+mechanisms&amp;rft.volume=433&amp;rft.issue=1&amp;rft.pages=2-12&amp;rft.date=2005-01&amp;rft_id=info%3Adoi%2F10.1016%2Fj.abb.2004.08.027&amp;rft_id=info%3Apmid%2F15581561&amp;rft.aulast=Cleland&amp;rft.aufirst=WW&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-41"><span class="mw-cite-backlink"><b><a href="#cite_ref-41">^</a></b></span> <span class="reference-text"><cite class="citation journal">Northrop DB (1981). "The expression of isotope effects on enzyme-catalyzed reactions". <i>Annual Review of Biochemistry</i>. <b>50</b>: 103–31. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1146%2Fannurev.bi.50.070181.000535">10.1146/annurev.bi.50.070181.000535</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/7023356">7023356</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Biochemistry&amp;rft.atitle=The+expression+of+isotope+effects+on+enzyme-catalyzed+reactions&amp;rft.volume=50&amp;rft.pages=103-31&amp;rft.date=1981&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.bi.50.070181.000535&amp;rft_id=info%3Apmid%2F7023356&amp;rft.aulast=Northrop&amp;rft.aufirst=DB&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-42"><span class="mw-cite-backlink"><b><a href="#cite_ref-42">^</a></b></span> <span class="reference-text"><cite class="citation journal">Baillie TA, Rettenmeier AW (1986). "Drug biotransformation: mechanistic studies with stable isotopes". <i>Journal of Clinical Pharmacology</i>. <b>26</b> (6): 448–51. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1002%2Fj.1552-4604.1986.tb03556.x">10.1002/j.1552-4604.1986.tb03556.x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/3734135">3734135</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Clinical+Pharmacology&amp;rft.atitle=Drug+biotransformation%3A+mechanistic+studies+with+stable+isotopes&amp;rft.volume=26&amp;rft.issue=6&amp;rft.pages=448-51&amp;rft.date=1986&amp;rft_id=info%3Adoi%2F10.1002%2Fj.1552-4604.1986.tb03556.x&amp;rft_id=info%3Apmid%2F3734135&amp;rft.aulast=Baillie&amp;rft.aufirst=TA&amp;rft.au=Rettenmeier%2C+AW&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-43"><span class="mw-cite-backlink"><b><a href="#cite_ref-43">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cleland WW (1982). "Use of isotope effects to elucidate enzyme mechanisms". <i>CRC Critical Reviews in Biochemistry</i>. <b>13</b> (4): 385–428. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.3109%2F10409238209108715">10.3109/10409238209108715</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/6759038">6759038</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=CRC+Critical+Reviews+in+Biochemistry&amp;rft.atitle=Use+of+isotope+effects+to+elucidate+enzyme+mechanisms&amp;rft.volume=13&amp;rft.issue=4&amp;rft.pages=385-428&amp;rft.date=1982&amp;rft_id=info%3Adoi%2F10.3109%2F10409238209108715&amp;rft_id=info%3Apmid%2F6759038&amp;rft.aulast=Cleland&amp;rft.aufirst=WW&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-44"><span class="mw-cite-backlink"><b><a href="#cite_ref-44">^</a></b></span> <span class="reference-text"><cite class="citation journal">Christianson DW, Cox JD (1999). "Catalysis by metal-activated hydroxide in zinc and manganese metalloenzymes". <i>Annual Review of Biochemistry</i>. <b>68</b>: 33–57. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1146%2Fannurev.biochem.68.1.33">10.1146/annurev.biochem.68.1.33</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10872443">10872443</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Biochemistry&amp;rft.atitle=Catalysis+by+metal-activated+hydroxide+in+zinc+and+manganese+metalloenzymes&amp;rft.volume=68&amp;rft.pages=33-57&amp;rft.date=1999&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.biochem.68.1.33&amp;rft_id=info%3Apmid%2F10872443&amp;rft.aulast=Christianson&amp;rft.aufirst=DW&amp;rft.au=Cox%2C+JD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-45"><span class="mw-cite-backlink"><b><a href="#cite_ref-45">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kraut DA, Carroll KS, Herschlag D (2003). "Challenges in enzyme mechanism and energetics". <i>Annual Review of Biochemistry</i>. <b>72</b>: 517–71. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1146%2Fannurev.biochem.72.121801.161617">10.1146/annurev.biochem.72.121801.161617</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12704087">12704087</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Annual+Review+of+Biochemistry&amp;rft.atitle=Challenges+in+enzyme+mechanism+and+energetics&amp;rft.volume=72&amp;rft.pages=517-71&amp;rft.date=2003&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev.biochem.72.121801.161617&amp;rft_id=info%3Apmid%2F12704087&amp;rft.aulast=Kraut&amp;rft.aufirst=DA&amp;rft.au=Carroll%2C+KS&amp;rft.au=Herschlag%2C+D&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-46"><span class="mw-cite-backlink"><b><a href="#cite_ref-46">^</a></b></span> <span class="reference-text"><cite class="citation journal">Walsh R, Martin E, Darvesh S (December 2011). "Limitations of conventional inhibitor classifications". <i>Integrative Biology</i>. <b>3</b> (12): 1197–201. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1039%2Fc1ib00053e">10.1039/c1ib00053e</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/22038120">22038120</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Integrative+Biology&amp;rft.atitle=Limitations+of+conventional+inhibitor+classifications&amp;rft.volume=3&amp;rft.issue=12&amp;rft.pages=1197-201&amp;rft.date=2011-12&amp;rft_id=info%3Adoi%2F10.1039%2Fc1ib00053e&amp;rft_id=info%3Apmid%2F22038120&amp;rft.aulast=Walsh&amp;rft.aufirst=R&amp;rft.au=Martin%2C+E&amp;rft.au=Darvesh%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-47"><span class="mw-cite-backlink"><b><a href="#cite_ref-47">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cleland WW (February 1963). "The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection". <i>Biochimica et Biophysica Acta</i>. <b>67</b>: 188–96. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2F0006-3002%2863%2991816-x">10.1016/0006-3002(63)91816-x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/14021669">14021669</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochimica+et+Biophysica+Acta&amp;rft.atitle=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products.+III.+Prediction+of+initial+velocity+and+inhibition+patterns+by+inspection&amp;rft.volume=67&amp;rft.pages=188-96&amp;rft.date=1963-02&amp;rft_id=info%3Adoi%2F10.1016%2F0006-3002%2863%2991816-x&amp;rft_id=info%3Apmid%2F14021669&amp;rft.aulast=Cleland&amp;rft.aufirst=WW&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-48"><span class="mw-cite-backlink"><b><a href="#cite_ref-48">^</a></b></span> <span class="reference-text"><cite class="citation journal">Walsh R, Martin E, Darvesh S (May 2007). "A versatile equation to describe reversible enzyme inhibition and activation kinetics: modeling beta-galactosidase and butyrylcholinesterase". <i>Biochimica et Biophysica Acta</i>. <b>1770</b> (5): 733–46. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.bbagen.2007.01.001">10.1016/j.bbagen.2007.01.001</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17307293">17307293</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochimica+et+Biophysica+Acta&amp;rft.atitle=A+versatile+equation+to+describe+reversible+enzyme+inhibition+and+activation+kinetics%3A+modeling+beta-galactosidase+and+butyrylcholinesterase&amp;rft.volume=1770&amp;rft.issue=5&amp;rft.pages=733-46&amp;rft.date=2007-05&amp;rft_id=info%3Adoi%2F10.1016%2Fj.bbagen.2007.01.001&amp;rft_id=info%3Apmid%2F17307293&amp;rft.aulast=Walsh&amp;rft.aufirst=R&amp;rft.au=Martin%2C+E&amp;rft.au=Darvesh%2C+S&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-49"><span class="mw-cite-backlink"><b><a href="#cite_ref-49">^</a></b></span> <span class="reference-text"><cite class="citation book">Walsh, Ryan (2012). <a rel="nofollow" class="external text" href="http://cdn.intechopen.com/pdfs/36518/InTech-Alternative_perspectives_of_enzyme_kinetic_modeling.pdf">"Ch. 17. Alternative Perspectives of Enzyme Kinetic Modeling"</a> <span class="cs1-format">(PDF)</span>.  In Ekinci, Deniz (ed.). <i>Medicinal Chemistry and Drug Design</i>. InTech. pp.&#160;357–371. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-953-51-0513-8" title="Special:BookSources/978-953-51-0513-8"><bdi>978-953-51-0513-8</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Ch.+17.+Alternative+Perspectives+of+Enzyme+Kinetic+Modeling&amp;rft.btitle=Medicinal+Chemistry+and+Drug+Design&amp;rft.pages=357-371&amp;rft.pub=InTech&amp;rft.date=2012&amp;rft.isbn=978-953-51-0513-8&amp;rft.au=Walsh%2C+Ryan&amp;rft_id=http%3A%2F%2Fcdn.intechopen.com%2Fpdfs%2F36518%2FInTech-Alternative_perspectives_of_enzyme_kinetic_modeling.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-:0-50"><span class="mw-cite-backlink">^ <a href="#cite_ref-:0_50-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:0_50-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-:0_50-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Srinivasan B, Skolnick J (May 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4445455">"Insights into the slow-onset tight-binding inhibition of Escherichia coli dihydrofolate reductase: detailed mechanistic characterization of pyrrolo &#91;3,2-f&#93; quinazoline-1,3-diamine and its derivatives as novel tight-binding inhibitors"</a>. <i>The FEBS Journal</i>. <b>282</b> (10): 1922–38. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1111%2Ffebs.13244">10.1111/febs.13244</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4445455">4445455</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/25703118">25703118</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+FEBS+Journal&amp;rft.atitle=Insights+into+the+slow-onset+tight-binding+inhibition+of+Escherichia+coli+dihydrofolate+reductase%3A+detailed+mechanistic+characterization+of+pyrrolo+%5B3%2C2-f%5D+quinazoline-1%2C3-diamine+and+its+derivatives+as+novel+tight-binding+inhibitors&amp;rft.volume=282&amp;rft.issue=10&amp;rft.pages=1922-38&amp;rft.date=2015-05&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4445455&amp;rft_id=info%3Apmid%2F25703118&amp;rft_id=info%3Adoi%2F10.1111%2Ffebs.13244&amp;rft.aulast=Srinivasan&amp;rft.aufirst=B&amp;rft.au=Skolnick%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4445455&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-:1-51"><span class="mw-cite-backlink">^ <a href="#cite_ref-:1_51-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-:1_51-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Srinivasan B, Tonddast-Navaei S, Skolnick J (October 2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4610388">"Ligand binding studies, preliminary structure-activity relationship and detailed mechanistic characterization of 1-phenyl-6,6-dimethyl-1,3,5-triazine-2,4-diamine derivatives as inhibitors of Escherichia coli dihydrofolate reductase"</a>. <i>European Journal of Medicinal Chemistry</i>. <b>103</b>: 600–14. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1016%2Fj.ejmech.2015.08.021">10.1016/j.ejmech.2015.08.021</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4610388">4610388</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/26414808">26414808</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=European+Journal+of+Medicinal+Chemistry&amp;rft.atitle=Ligand+binding+studies%2C+preliminary+structure-activity+relationship+and+detailed+mechanistic+characterization+of+1-phenyl-6%2C6-dimethyl-1%2C3%2C5-triazine-2%2C4-diamine+derivatives+as+inhibitors+of+Escherichia+coli+dihydrofolate+reductase&amp;rft.volume=103&amp;rft.pages=600-14&amp;rft.date=2015-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4610388&amp;rft_id=info%3Apmid%2F26414808&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ejmech.2015.08.021&amp;rft.aulast=Srinivasan&amp;rft.aufirst=B&amp;rft.au=Tonddast-Navaei%2C+S&amp;rft.au=Skolnick%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4610388&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-52"><span class="mw-cite-backlink"><b><a href="#cite_ref-52">^</a></b></span> <span class="reference-text"><cite class="citation journal">Srinivasan B, Rodrigues JV, Tonddast-Navaei S, Shakhnovich E, Skolnick J (July 2017). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5819740">"Rational Design of Novel Allosteric Dihydrofolate Reductase Inhibitors Showing Antibacterial Effects on Drug-Resistant Escherichia coli Escape Variants"</a>. <i>ACS Chemical Biology</i>. <b>12</b> (7): 1848–1857. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Facschembio.7b00175">10.1021/acschembio.7b00175</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC5819740">5819740</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/28525268">28525268</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=ACS+Chemical+Biology&amp;rft.atitle=Rational+Design+of+Novel+Allosteric+Dihydrofolate+Reductase+Inhibitors+Showing+Antibacterial+Effects+on+Drug-Resistant+Escherichia+coli+Escape+Variants&amp;rft.volume=12&amp;rft.issue=7&amp;rft.pages=1848-1857&amp;rft.date=2017-07&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5819740&amp;rft_id=info%3Apmid%2F28525268&amp;rft_id=info%3Adoi%2F10.1021%2Facschembio.7b00175&amp;rft.aulast=Srinivasan&amp;rft.aufirst=B&amp;rft.au=Rodrigues%2C+JV&amp;rft.au=Tonddast-Navaei%2C+S&amp;rft.au=Shakhnovich%2C+E&amp;rft.au=Skolnick%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5819740&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-53"><span class="mw-cite-backlink"><b><a href="#cite_ref-53">^</a></b></span> <span class="reference-text"><cite class="citation journal">Koshland DE (February 1958). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC335371">"Application of a Theory of Enzyme Specificity to Protein Synthesis"</a>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>. <b>44</b> (2): 98–104. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1958PNAS...44...98K">1958PNAS...44...98K</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1073%2Fpnas.44.2.98">10.1073/pnas.44.2.98</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC335371">335371</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16590179">16590179</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;rft.atitle=Application+of+a+Theory+of+Enzyme+Specificity+to+Protein+Synthesis&amp;rft.volume=44&amp;rft.issue=2&amp;rft.pages=98-104&amp;rft.date=1958-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC335371&amp;rft_id=info%3Apmid%2F16590179&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.44.2.98&amp;rft_id=info%3Abibcode%2F1958PNAS...44...98K&amp;rft.aulast=Koshland&amp;rft.aufirst=DE&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC335371&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-54"><span class="mw-cite-backlink"><b><a href="#cite_ref-54">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hammes GG (July 2002). "Multiple conformational changes in enzyme catalysis". <i>Biochemistry</i>. <b>41</b> (26): 8221–8. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fbi0260839">10.1021/bi0260839</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12081470">12081470</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry&amp;rft.atitle=Multiple+conformational+changes+in+enzyme+catalysis&amp;rft.volume=41&amp;rft.issue=26&amp;rft.pages=8221-8&amp;rft.date=2002-07&amp;rft_id=info%3Adoi%2F10.1021%2Fbi0260839&amp;rft_id=info%3Apmid%2F12081470&amp;rft.aulast=Hammes&amp;rft.aufirst=GG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-55"><span class="mw-cite-backlink"><b><a href="#cite_ref-55">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sutcliffe MJ, Scrutton NS (July 2002). <a rel="nofollow" class="external text" href="http://content.febsjournal.org/cgi/content/full/269/13/3096">"A new conceptual framework for enzyme catalysis. Hydrogen tunnelling coupled to enzyme dynamics in flavoprotein and quinoprotein enzymes"</a>. <i>European Journal of Biochemistry</i>. <b>269</b> (13): 3096–102. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1046%2Fj.1432-1033.2002.03020.x">10.1046/j.1432-1033.2002.03020.x</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12084049">12084049</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=European+Journal+of+Biochemistry&amp;rft.atitle=A+new+conceptual+framework+for+enzyme+catalysis.+Hydrogen+tunnelling+coupled+to+enzyme+dynamics+in+flavoprotein+and+quinoprotein+enzymes&amp;rft.volume=269&amp;rft.issue=13&amp;rft.pages=3096-102&amp;rft.date=2002-07&amp;rft_id=info%3Adoi%2F10.1046%2Fj.1432-1033.2002.03020.x&amp;rft_id=info%3Apmid%2F12084049&amp;rft.aulast=Sutcliffe&amp;rft.aufirst=MJ&amp;rft.au=Scrutton%2C+NS&amp;rft_id=http%3A%2F%2Fcontent.febsjournal.org%2Fcgi%2Fcontent%2Ffull%2F269%2F13%2F3096&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-56"><span class="mw-cite-backlink"><b><a href="#cite_ref-56">^</a></b></span> <span class="reference-text"><cite class="citation journal">Henri V (1902). "Theorie generale de l'action de quelques diastases". <i>Compt. Rend. Acad. Sci. Paris</i>. <b>135</b>: 916–9.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Compt.+Rend.+Acad.+Sci.+Paris&amp;rft.atitle=Theorie+generale+de+l%27action+de+quelques+diastases&amp;rft.volume=135&amp;rft.pages=916-9&amp;rft.date=1902&amp;rft.aulast=Henri&amp;rft.aufirst=V&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-57"><span class="mw-cite-backlink"><b><a href="#cite_ref-57">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sørensen PL (1909). "Enzymstudien {II}. Über die Messung und Bedeutung der Wasserstoffionenkonzentration bei enzymatischen Prozessen" &#91;Enzyme studies III: About the measurement and significance of the hydrogen ion concentration in enzymatic processes&#93;. <i>Biochem. Z.</i> (in German). <b>21</b>: 131–304.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochem.+Z.&amp;rft.atitle=Enzymstudien+%7BII%7D.+%C3%9Cber+die+Messung+und+Bedeutung+der+Wasserstoffionenkonzentration+bei+enzymatischen+Prozessen&amp;rft.volume=21&amp;rft.pages=131-304&amp;rft.date=1909&amp;rft.aulast=S%C3%B8rensen&amp;rft.aufirst=PL&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-58"><span class="mw-cite-backlink"><b><a href="#cite_ref-58">^</a></b></span> <span class="reference-text"><cite class="citation journal">Michaelis L, Menten M (1913). "Die Kinetik der Invertinwirkung" &#91;The Kinetics of Invertase Action&#93;. <i>Biochem. Z.</i> (in German). <b>49</b>: 333–369.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochem.+Z.&amp;rft.atitle=Die+Kinetik+der+Invertinwirkung&amp;rft.volume=49&amp;rft.pages=333-369&amp;rft.date=1913&amp;rft.aulast=Michaelis&amp;rft.aufirst=L&amp;rft.au=Menten%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/>; <cite class="citation journal">Michaelis L, Menten ML, Johnson KA, Goody RS (October 2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3381512">"The original Michaelis constant: translation of the 1913 Michaelis-Menten paper"</a>. <i>Biochemistry</i>. <b>50</b> (39): 8264–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1021%2Fbi201284u">10.1021/bi201284u</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3381512">3381512</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21888353">21888353</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry&amp;rft.atitle=The+original+Michaelis+constant%3A+translation+of+the+1913+Michaelis-Menten+paper&amp;rft.volume=50&amp;rft.issue=39&amp;rft.pages=8264-9&amp;rft.date=2011-10&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3381512&amp;rft_id=info%3Apmid%2F21888353&amp;rft_id=info%3Adoi%2F10.1021%2Fbi201284u&amp;rft.aulast=Michaelis&amp;rft.aufirst=L&amp;rft.au=Menten%2C+ML&amp;rft.au=Johnson%2C+KA&amp;rft.au=Goody%2C+RS&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3381512&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-59"><span class="mw-cite-backlink"><b><a href="#cite_ref-59">^</a></b></span> <span class="reference-text"><cite class="citation journal">Briggs GE, Haldane JB (1925). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1259181">"A Note on the Kinetics of Enzyme Action"</a>. <i>The Biochemical Journal</i>. <b>19</b> (2): 338–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1042%2Fbj0190338">10.1042/bj0190338</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1259181">1259181</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16743508">16743508</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=The+Biochemical+Journal&amp;rft.atitle=A+Note+on+the+Kinetics+of+Enzyme+Action&amp;rft.volume=19&amp;rft.issue=2&amp;rft.pages=338-9&amp;rft.date=1925&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1259181&amp;rft_id=info%3Apmid%2F16743508&amp;rft_id=info%3Adoi%2F10.1042%2Fbj0190338&amp;rft.aulast=Briggs&amp;rft.aufirst=GE&amp;rft.au=Haldane%2C+JB&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1259181&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-OF_2005-60"><span class="mw-cite-backlink"><b><a href="#cite_ref-OF_2005_60-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Flomenbom O, Velonia K, Loos D, Masuo S, Cotlet M, Engelborghs Y, Hofkens J, Rowan AE, Nolte RJ, Van der Auweraer M, de Schryver FC, Klafter J (February 2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC548972">"Stretched exponential decay and correlations in the catalytic activity of fluctuating single lipase molecules"</a>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>. <b>102</b> (7): 2368–72. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2005PNAS..102.2368F">2005PNAS..102.2368F</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1073%2Fpnas.0409039102">10.1073/pnas.0409039102</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC548972">548972</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15695587">15695587</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;rft.atitle=Stretched+exponential+decay+and+correlations+in+the+catalytic+activity+of+fluctuating+single+lipase+molecules&amp;rft.volume=102&amp;rft.issue=7&amp;rft.pages=2368-72&amp;rft.date=2005-02&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC548972&amp;rft_id=info%3Apmid%2F15695587&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0409039102&amp;rft_id=info%3Abibcode%2F2005PNAS..102.2368F&amp;rft.aulast=Flomenbom&amp;rft.aufirst=O&amp;rft.au=Velonia%2C+K&amp;rft.au=Loos%2C+D&amp;rft.au=Masuo%2C+S&amp;rft.au=Cotlet%2C+M&amp;rft.au=Engelborghs%2C+Y&amp;rft.au=Hofkens%2C+J&amp;rft.au=Rowan%2C+AE&amp;rft.au=Nolte%2C+RJ&amp;rft.au=Van+der+Auweraer%2C+M&amp;rft.au=de+Schryver%2C+FC&amp;rft.au=Klafter%2C+J&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC548972&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-English_2006-61"><span class="mw-cite-backlink"><b><a href="#cite_ref-English_2006_61-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">English BP, Min W, van Oijen AM, Lee KT, Luo G, Sun H, Cherayil BJ, Kou SC, Xie XS (February 2006). "Ever-fluctuating single enzyme molecules: Michaelis-Menten equation revisited". <i>Nature Chemical Biology</i>. <b>2</b> (2): 87–94. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1038%2Fnchembio759">10.1038/nchembio759</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16415859">16415859</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Chemical+Biology&amp;rft.atitle=Ever-fluctuating+single+enzyme+molecules%3A+Michaelis-Menten+equation+revisited&amp;rft.volume=2&amp;rft.issue=2&amp;rft.pages=87-94&amp;rft.date=2006-02&amp;rft_id=info%3Adoi%2F10.1038%2Fnchembio759&amp;rft_id=info%3Apmid%2F16415859&amp;rft.aulast=English&amp;rft.aufirst=BP&amp;rft.au=Min%2C+W&amp;rft.au=van+Oijen%2C+AM&amp;rft.au=Lee%2C+KT&amp;rft.au=Luo%2C+G&amp;rft.au=Sun%2C+H&amp;rft.au=Cherayil%2C+BJ&amp;rft.au=Kou%2C+SC&amp;rft.au=Xie%2C+XS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-Lu_1998-62"><span class="mw-cite-backlink"><b><a href="#cite_ref-Lu_1998_62-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Lu HP, Xun L, Xie XS (December 1998). "Single-molecule enzymatic dynamics". <i>Science</i>. <b>282</b> (5395): 1877–82. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1126%2Fscience.282.5395.1877">10.1126/science.282.5395.1877</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9836635">9836635</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Single-molecule+enzymatic+dynamics&amp;rft.volume=282&amp;rft.issue=5395&amp;rft.pages=1877-82&amp;rft.date=1998-12&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.282.5395.1877&amp;rft_id=info%3Apmid%2F9836635&amp;rft.aulast=Lu&amp;rft.aufirst=HP&amp;rft.au=Xun%2C+L&amp;rft.au=Xie%2C+XS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-XX_2006-63"><span class="mw-cite-backlink"><b><a href="#cite_ref-XX_2006_63-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Xue X, Liu F, Ou-Yang ZC (September 2006). "Single molecule Michaelis-Menten equation beyond quasistatic disorder". <i>Physical Review E</i>. <b>74</b> (3 Pt 1): 030902. <a href="/wiki/ArXiv" title="ArXiv">arXiv</a>:<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//arxiv.org/abs/cond-mat/0604364">cond-mat/0604364</a></span>. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2006PhRvE..74c0902X">2006PhRvE..74c0902X</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1103%2FPhysRevE.74.030902">10.1103/PhysRevE.74.030902</a>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17025584">17025584</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Physical+Review+E&amp;rft.atitle=Single+molecule+Michaelis-Menten+equation+beyond+quasistatic+disorder&amp;rft.volume=74&amp;rft.issue=3+Pt+1&amp;rft.pages=030902&amp;rft.date=2006-09&amp;rft_id=info%3Aarxiv%2Fcond-mat%2F0604364&amp;rft_id=info%3Apmid%2F17025584&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevE.74.030902&amp;rft_id=info%3Abibcode%2F2006PhRvE..74c0902X&amp;rft.aulast=Xue&amp;rft.aufirst=X&amp;rft.au=Liu%2C+F&amp;rft.au=Ou-Yang%2C+ZC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></span>
</li>
<li id="cite_note-64"><span class="mw-cite-backlink"><b><a href="#cite_ref-64">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bevc S, Konc J, Stojan J, Hodošček M, Penca M, Praprotnik M, Janežič D (2011). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3139599">"ENZO: a web tool for derivation and evaluation of kinetic models of enzyme catalyzed reactions"</a>. <i>PLOS One</i>. <b>6</b> (7): e22265. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2011PLoSO...622265B">2011PLoSO...622265B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1371%2Fjournal.pone.0022265">10.1371/journal.pone.0022265</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3139599">3139599</a></span>. <a href="/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21818304">21818304</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+One&amp;rft.atitle=ENZO%3A+a+web+tool+for+derivation+and+evaluation+of+kinetic+models+of+enzyme+catalyzed+reactions&amp;rft.volume=6&amp;rft.issue=7&amp;rft.pages=e22265&amp;rft.date=2011&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3139599&amp;rft_id=info%3Apmid%2F21818304&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0022265&amp;rft_id=info%3Abibcode%2F2011PLoSO...622265B&amp;rft.aulast=Bevc&amp;rft.aufirst=S&amp;rft.au=Konc%2C+J&amp;rft.au=Stojan%2C+J&amp;rft.au=Hodo%C5%A1%C4%8Dek%2C+M&amp;rft.au=Penca%2C+M&amp;rft.au=Praprotnik%2C+M&amp;rft.au=Jane%C5%BEi%C4%8D%2C+D&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3139599&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/> 
<a rel="nofollow" class="external text" href="http://enzo.cmm.ki.si/">ENZO server</a></span>
</li>
</ol></div>
<h2><span class="mw-headline" id="Further_reading">Further reading</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=27" title="Edit section: Further reading">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p><b>Introductory</b>
</p>
<ul><li><cite class="citation book">Cornish-Bowden, Athel (2004). <i>Fundamentals of enzyme kinetics</i> (3rd ed.). London: Portland Press. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-85578-158-0" title="Special:BookSources/978-1-85578-158-0"><bdi>978-1-85578-158-0</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Fundamentals+of+enzyme+kinetics&amp;rft.place=London&amp;rft.edition=3rd&amp;rft.pub=Portland+Press&amp;rft.date=2004&amp;rft.isbn=978-1-85578-158-0&amp;rft.au=Cornish-Bowden%2C+Athel&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation book">Stevens L, Price NC (1999). <i>Fundamentals of enzymology: the cell and molecular biology of catalytic proteins</i>. Oxford [Oxfordshire]: Oxford University Press. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-19-850229-6" title="Special:BookSources/978-0-19-850229-6"><bdi>978-0-19-850229-6</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Fundamentals+of+enzymology%3A+the+cell+and+molecular+biology+of+catalytic+proteins&amp;rft.place=Oxford+%5BOxfordshire%5D&amp;rft.pub=Oxford+University+Press&amp;rft.date=1999&amp;rft.isbn=978-0-19-850229-6&amp;rft.aulast=Stevens&amp;rft.aufirst=L&amp;rft.au=Price%2C+NC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation book">Bugg, Tim (2004). <i>Introduction to Enzyme and Coenzyme Chemistry</i>. Cambridge, MA: Blackwell Publishers. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-4051-1452-3" title="Special:BookSources/978-1-4051-1452-3"><bdi>978-1-4051-1452-3</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Introduction+to+Enzyme+and+Coenzyme+Chemistry&amp;rft.place=Cambridge%2C+MA&amp;rft.pub=Blackwell+Publishers&amp;rft.date=2004&amp;rft.isbn=978-1-4051-1452-3&amp;rft.au=Bugg%2C+Tim&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li></ul>
<p><b>Advanced</b>
</p>
<ul><li><cite class="citation book">Segel, Irwin H. (1993). <i>Enzyme kinetics: behavior and analysis of rapid equilibrium and steady state enzyme systems</i> (New ed.). New York: Wiley. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-471-30309-1" title="Special:BookSources/978-0-471-30309-1"><bdi>978-0-471-30309-1</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Enzyme+kinetics%3A+behavior+and+analysis+of+rapid+equilibrium+and+steady+state+enzyme+systems&amp;rft.place=New+York&amp;rft.edition=New&amp;rft.pub=Wiley&amp;rft.date=1993&amp;rft.isbn=978-0-471-30309-1&amp;rft.au=Segel%2C+Irwin+H.&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation book">Fersht, Alan (1999). <i>Structure and mechanism in protein science: a guide to enzyme catalysis and protein folding</i>. San Francisco: W.H. Freeman. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-7167-3268-6" title="Special:BookSources/978-0-7167-3268-6"><bdi>978-0-7167-3268-6</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Structure+and+mechanism+in+protein+science%3A+a+guide+to+enzyme+catalysis+and+protein+folding&amp;rft.place=San+Francisco&amp;rft.pub=W.H.+Freeman&amp;rft.date=1999&amp;rft.isbn=978-0-7167-3268-6&amp;rft.au=Fersht%2C+Alan&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation journal">Schnell S, Maini PK (2004). <a rel="nofollow" class="external text" href="https://web.archive.org/web/20060221045110/http://www.informatics.indiana.edu/schnell/papers/ctb8_169.pdf">"A century of enzyme kinetics: Reliability of the K<sub>M</sub> and v<sub>max</sub> estimates"</a> <span class="cs1-format">(PDF)</span>. <i>Comments on Theoretical Biology</i>. <b>8</b> (2–3): 169–87. <a href="/wiki/CiteSeerX" title="CiteSeerX">CiteSeerX</a>&#160;<span class="cs1-lock-free" title="Freely accessible"><a rel="nofollow" class="external text" href="//citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.493.7178">10.1.1.493.7178</a></span>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//doi.org/10.1080%2F08948550302453">10.1080/08948550302453</a>. Archived from <a rel="nofollow" class="external text" href="http://www.informatics.indiana.edu/schnell/papers/ctb8_169.pdf">the original</a> <span class="cs1-format">(PDF)</span> on 21 February 2006.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Comments+on+Theoretical+Biology&amp;rft.atitle=A+century+of+enzyme+kinetics%3A+Reliability+of+the+K%3Csub%3EM%3C%2Fsub%3E+and+v%3Csub%3Emax%3C%2Fsub%3E+estimates&amp;rft.volume=8&amp;rft.issue=2%E2%80%933&amp;rft.pages=169-87&amp;rft.date=2004&amp;rft_id=%2F%2Fciteseerx.ist.psu.edu%2Fviewdoc%2Fsummary%3Fdoi%3D10.1.1.493.7178&amp;rft_id=info%3Adoi%2F10.1080%2F08948550302453&amp;rft.aulast=Schnell&amp;rft.aufirst=S&amp;rft.au=Maini%2C+PK&amp;rft_id=http%3A%2F%2Fwww.informatics.indiana.edu%2Fschnell%2Fpapers%2Fctb8_169.pdf&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation book">Walsh, Christopher (1979). <i>Enzymatic reaction mechanisms</i>. San Francisco: W. H. Freeman. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-7167-0070-8" title="Special:BookSources/978-0-7167-0070-8"><bdi>978-0-7167-0070-8</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Enzymatic+reaction+mechanisms&amp;rft.place=San+Francisco&amp;rft.pub=W.+H.+Freeman&amp;rft.date=1979&amp;rft.isbn=978-0-7167-0070-8&amp;rft.au=Walsh%2C+Christopher&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li>
<li><cite class="citation book">Cleland WW, Cook P (2007). <i>Enzyme kinetics and mechanism</i>. New York: Garland Science. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-8153-4140-6" title="Special:BookSources/978-0-8153-4140-6"><bdi>978-0-8153-4140-6</bdi></a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Enzyme+kinetics+and+mechanism&amp;rft.place=New+York&amp;rft.pub=Garland+Science&amp;rft.date=2007&amp;rft.isbn=978-0-8153-4140-6&amp;rft.aulast=Cleland&amp;rft.aufirst=WW&amp;rft.au=Cook%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AEnzyme+kinetics" class="Z3988"></span><link rel="mw-deduplicated-inline-style" href="mw-data:TemplateStyles:r886058088"/></li></ul>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit&amp;section=28" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul><li><a rel="nofollow" class="external text" href="http://www.kscience.co.uk/animations/model.swf">Animation of an enzyme assay</a> — Shows effects of manipulating assay conditions</li>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/thornton-srv/databases/MACiE/">MACiE</a> — A database of enzyme reaction mechanisms</li>
<li><a rel="nofollow" class="external text" href="https://web.archive.org/web/20061006064937/http://us.expasy.org/enzyme/">ENZYME</a> — Expasy enzyme nomenclature database</li>
<li><a rel="nofollow" class="external text" href="http://enzo.cmm.ki.si/">ENZO</a> — Web application for easy construction and quick testing of kinetic models of enzyme catalyzed reactions.</li>
<li><a rel="nofollow" class="external text" href="http://ezcatdb.cbrc.jp/EzCatDB/">ExCatDB</a> — A database of enzyme catalytic mechanisms</li>
<li><a rel="nofollow" class="external text" href="http://www.brenda-enzymes.info/">BRENDA</a> — Comprehensive enzyme database, giving substrates, inhibitors and reaction diagrams</li>
<li><a rel="nofollow" class="external text" href="http://sabio.h-its.org/">SABIO-RK</a> — A database of reaction kinetics</li>
<li><a rel="nofollow" class="external text" href="https://web.archive.org/web/20060820062258/http://chem-faculty.ucsd.edu/kraut/dhfr.html">Joseph Kraut's Research Group, University of California San Diego</a> — Animations of several enzyme reaction mechanisms</li>
<li><a rel="nofollow" class="external text" href="https://web.archive.org/web/20060712051732/http://www.chem.qmul.ac.uk/iubmb/kinetics/">Symbolism and Terminology in Enzyme Kinetics</a> — A comprehensive explanation of concepts and terminology in enzyme kinetics</li>
<li><a rel="nofollow" class="external text" href="https://web.archive.org/web/20040612065857/http://orion1.paisley.ac.uk/kinetics/contents.html">An introduction to enzyme kinetics</a> — An accessible set of on-line tutorials on enzyme kinetics</li>
<li><a rel="nofollow" class="external text" href="http://www.wiley.com/college/pratt/0471393878/student/animations/enzyme_kinetics/index.html">Enzyme kinetics animated tutorial</a> — An animated tutorial with audio</li></ul>
<p class="mw-empty-elt">
</p>
<div role="navigation" class="navbox" aria-labelledby="Enzymes" style="padding:3px"><table class="nowraplinks collapsible autocollapse navbox-inner" style="border-spacing:0;background:transparent;color:inherit"><tbody><tr><th scope="col" class="navbox-title" colspan="2"><div class="plainlinks hlist navbar mini"><ul><li class="nv-view"><a href="/wiki/Template:Enzymes" title="Template:Enzymes"><abbr title="View this template" style=";;background:none transparent;border:none;-moz-box-shadow:none;-webkit-box-shadow:none;box-shadow:none; padding:0;">v</abbr></a></li><li class="nv-talk"><a href="/wiki/Template_talk:Enzymes" title="Template talk:Enzymes"><abbr title="Discuss this template" style=";;background:none transparent;border:none;-moz-box-shadow:none;-webkit-box-shadow:none;box-shadow:none; padding:0;">t</abbr></a></li><li class="nv-edit"><a class="external text" href="https://en.wikipedia.org/w/index.php?title=Template:Enzymes&amp;action=edit"><abbr title="Edit this template" style=";;background:none transparent;border:none;-moz-box-shadow:none;-webkit-box-shadow:none;box-shadow:none; padding:0;">e</abbr></a></li></ul></div><div id="Enzymes" style="font-size:114%;margin:0 4em"><a href="/wiki/Enzyme" title="Enzyme">Enzymes</a></div></th></tr><tr><th scope="row" class="navbox-group" style="width:1%">Activity</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><a href="/wiki/Active_site" title="Active site">Active site</a></li>
<li><a href="/wiki/Binding_site" title="Binding site">Binding site</a></li>
<li><a href="/wiki/Catalytic_triad" title="Catalytic triad">Catalytic triad</a></li>
<li><a href="/wiki/Oxyanion_hole" title="Oxyanion hole">Oxyanion hole</a></li>
<li><a href="/wiki/Enzyme_promiscuity" title="Enzyme promiscuity">Enzyme promiscuity</a></li>
<li><a href="/wiki/Catalytically_perfect_enzyme" class="mw-redirect" title="Catalytically perfect enzyme">Catalytically perfect enzyme</a></li>
<li><a href="/wiki/Coenzyme" class="mw-redirect" title="Coenzyme">Coenzyme</a></li>
<li><a href="/wiki/Cofactor_(biochemistry)" title="Cofactor (biochemistry)">Cofactor</a></li>
<li><a href="/wiki/Enzyme_catalysis" title="Enzyme catalysis">Enzyme catalysis</a></li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Regulation</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><a href="/wiki/Allosteric_regulation" title="Allosteric regulation">Allosteric regulation</a></li>
<li><a href="/wiki/Cooperativity" title="Cooperativity">Cooperativity</a></li>
<li><a href="/wiki/Enzyme_inhibitor" title="Enzyme inhibitor">Enzyme inhibitor</a></li>
<li><a href="/wiki/Enzyme_activator" title="Enzyme activator">Enzyme activator</a></li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Classification</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><a href="/wiki/Enzyme_Commission_number" title="Enzyme Commission number">EC number</a></li>
<li><a href="/wiki/Enzyme_superfamily" class="mw-redirect" title="Enzyme superfamily">Enzyme superfamily</a></li>
<li><a href="/wiki/Enzyme_family" class="mw-redirect" title="Enzyme family">Enzyme family</a></li>
<li><a href="/wiki/List_of_enzymes" title="List of enzymes">List of enzymes</a></li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Kinetics</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><a class="mw-selflink selflink">Enzyme kinetics</a></li>
<li><a href="/wiki/Eadie%E2%80%93Hofstee_diagram" title="Eadie–Hofstee diagram">Eadie–Hofstee diagram</a></li>
<li><a href="/wiki/Hanes%E2%80%93Woolf_plot" title="Hanes–Woolf plot">Hanes–Woolf plot</a></li>
<li><a href="/wiki/Lineweaver%E2%80%93Burk_plot" title="Lineweaver–Burk plot">Lineweaver–Burk plot</a></li>
<li><a href="/wiki/Michaelis%E2%80%93Menten_kinetics" title="Michaelis–Menten kinetics">Michaelis–Menten kinetics</a></li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Types</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><b>EC1 <a href="/wiki/Oxidoreductase" title="Oxidoreductase">Oxidoreductases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_1)" title="List of EC numbers (EC 1)">list</a>)</li>
<li><b>EC2 <a href="/wiki/Transferase" title="Transferase">Transferases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_2)" title="List of EC numbers (EC 2)">list</a>)</li>
<li><b>EC3 <a href="/wiki/Hydrolase" title="Hydrolase">Hydrolases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_3)" title="List of EC numbers (EC 3)">list</a>)</li>
<li><b>EC4 <a href="/wiki/Lyase" title="Lyase">Lyases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_4)" title="List of EC numbers (EC 4)">list</a>)</li>
<li><b>EC5 <a href="/wiki/Isomerase" title="Isomerase">Isomerases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_5)" title="List of EC numbers (EC 5)">list</a>)</li>
<li><b>EC6 <a href="/wiki/Ligase" title="Ligase">Ligases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_6)" title="List of EC numbers (EC 6)">list</a>)</li>
<li><b>EC7 <a href="/wiki/Translocase" title="Translocase">Translocases</a></b> (<a href="/wiki/List_of_EC_numbers_(EC_7)" title="List of EC numbers (EC 7)">list</a>)</li></ul>
</div></td></tr></tbody></table></div>
<!-- 
NewPP limit report
Parsed by mw1229
Cached time: 20190731054416
Cache expiry: 2592000
Dynamic content: false
Complications: [vary‐revision‐sha1]
CPU time usage: 1.260 seconds
Real time usage: 1.585 seconds
Preprocessor visited node count: 4008/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 163767/2097152 bytes
Template argument size: 1002/2097152 bytes
Highest expansion depth: 12/40
Expensive parser function count: 7/500
Unstrip recursion depth: 1/20
Unstrip post‐expand size: 226488/5000000 bytes
Number of Wikibase entities loaded: 5/400
Lua time usage: 0.743/10.000 seconds
Lua memory usage: 5.13 MB/50 MB
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00% 1122.229      1 -total
 70.25%  788.412      1 Template:Reflist
 54.39%  610.327     55 Template:Cite_journal
 16.53%  185.540     15 Template:Cite_book
  3.88%   43.488      2 Template:Main_other
  3.80%   42.659      1 Template:Featured_article
  3.57%   40.062      1 Template:Top_icon
  2.69%   30.208      1 Template:Pp-move-indef
  2.19%   24.623      1 Template:Category_handler
  2.18%   24.493      1 Template:Multiple_image
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:3043886-0!canonical!math=5 and timestamp 20190731054414 and revision id 908666171
 -->
</div><noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>
		
		<div class="printfooter">Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=Enzyme_kinetics&amp;oldid=908666171">https://en.wikipedia.org/w/index.php?title=Enzyme_kinetics&amp;oldid=908666171</a>"</div>
		
		<div id="catlinks" class="catlinks" data-mw="interface"><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:Enzyme_kinetics" title="Category:Enzyme kinetics">Enzyme kinetics</a></li><li><a href="/wiki/Category:Catalysis" title="Category:Catalysis">Catalysis</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:CS1:_long_volume_value" title="Category:CS1: long volume value">CS1: long volume value</a></li><li><a href="/wiki/Category:CS1_German-language_sources_(de)" title="Category:CS1 German-language sources (de)">CS1 German-language sources (de)</a></li><li><a href="/wiki/Category:Wikipedia_indefinitely_move-protected_pages" title="Category:Wikipedia indefinitely move-protected pages">Wikipedia indefinitely move-protected pages</a></li><li><a href="/wiki/Category:Use_dmy_dates_from_August_2018" title="Category:Use dmy dates from August 2018">Use dmy dates from August 2018</a></li><li><a href="/wiki/Category:Featured_articles" title="Category:Featured articles">Featured articles</a></li><li><a href="/wiki/Category:Pages_that_use_a_deprecated_format_of_the_chem_tags" title="Category:Pages that use a deprecated format of the chem tags">Pages that use a deprecated format of the chem tags</a></li></ul></div></div>
		<div class="visualClear"></div>
		
	</div>
</div>


		<div id="mw-navigation">
			<h2>Navigation menu</h2>
			<div id="mw-head">
									<div id="p-personal" role="navigation" aria-labelledby="p-personal-label">
						<h3 id="p-personal-label">Personal tools</h3>
						<ul>
							<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:CreateAccount&amp;returnto=Enzyme+kinetics" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Enzyme+kinetics" title="You&#039;re encouraged to log in; however, it&#039;s not mandatory. [o]" accesskey="o">Log in</a></li>						</ul>
					</div>
									<div id="left-navigation">
										<div id="p-namespaces" role="navigation" class="vectorTabs" aria-labelledby="p-namespaces-label">
						<h3 id="p-namespaces-label">Namespaces</h3>
						<ul>
							<li id="ca-nstab-main" class="selected"><span><a href="/wiki/Enzyme_kinetics" title="View the content page [c]" accesskey="c">Article</a></span></li><li id="ca-talk"><span><a href="/wiki/Talk:Enzyme_kinetics" rel="discussion" title="Discussion about the content page [t]" accesskey="t">Talk</a></span></li>						</ul>
					</div>
										<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
												<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-variants-label" />
						<h3 id="p-variants-label">
							<span>Variants</span>
						</h3>
						<ul class="menu">
													</ul>
					</div>
									</div>
				<div id="right-navigation">
										<div id="p-views" role="navigation" class="vectorTabs" aria-labelledby="p-views-label">
						<h3 id="p-views-label">Views</h3>
						<ul>
							<li id="ca-view" class="collapsible selected"><span><a href="/wiki/Enzyme_kinetics">Read</a></span></li><li id="ca-edit" class="collapsible"><span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=edit" title="Edit this page [e]" accesskey="e">Edit</a></span></li><li id="ca-history" class="collapsible"><span><a href="/w/index.php?title=Enzyme_kinetics&amp;action=history" title="Past revisions of this page [h]" accesskey="h">View history</a></span></li>						</ul>
					</div>
										<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
						<input type="checkbox" class="vectorMenuCheckbox" aria-labelledby="p-cactions-label" />
						<h3 id="p-cactions-label"><span>More</span></h3>
						<ul class="menu">
													</ul>
					</div>
										<div id="p-search" role="search">
						<h3>
							<label for="searchInput">Search</label>
						</h3>
						<form action="/w/index.php" id="searchform">
							<div id="simpleSearch">
								<input type="search" name="search" placeholder="Search Wikipedia" title="Search Wikipedia [f]" accesskey="f" id="searchInput"/><input type="hidden" value="Special:Search" name="title"/><input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton"/><input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton"/>							</div>
						</form>
					</div>
									</div>
			</div>
			<div id="mw-panel">
				<div id="p-logo" role="banner"><a class="mw-wiki-logo" href="/wiki/Main_Page" title="Visit the main page"></a></div>
						<div class="portal" role="navigation" id="p-navigation" aria-labelledby="p-navigation-label">
			<h3 id="p-navigation-label">Navigation</h3>
			<div class="body">
								<ul>
					<li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li>				</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-interaction" aria-labelledby="p-interaction-label">
			<h3 id="p-interaction-label">Interaction</h3>
			<div class="body">
								<ul>
					<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li>				</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-tb" aria-labelledby="p-tb-label">
			<h3 id="p-tb-label">Tools</h3>
			<div class="body">
								<ul>
					<li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/Enzyme_kinetics" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/Enzyme_kinetics" rel="nofollow" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=Enzyme_kinetics&amp;oldid=908666171" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=Enzyme_kinetics&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q883112" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=Enzyme_kinetics&amp;id=908666171" title="Information on how to cite this page">Cite this page</a></li>				</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-wikibase-otherprojects" aria-labelledby="p-wikibase-otherprojects-label">
			<h3 id="p-wikibase-otherprojects-label">In other projects</h3>
			<div class="body">
								<ul>
					<li class="wb-otherproject-link wb-otherproject-commons"><a href="https://commons.wikimedia.org/wiki/Category:Enzyme_kinetics" hreflang="en">Wikimedia Commons</a></li>				</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-coll-print_export" aria-labelledby="p-coll-print_export-label">
			<h3 id="p-coll-print_export-label">Print/export</h3>
			<div class="body">
								<ul>
					<li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=Enzyme+kinetics">Create a book</a></li><li id="coll-download-as-rl"><a href="/w/index.php?title=Special:ElectronPdf&amp;page=Enzyme+kinetics&amp;action=show-download-screen">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=Enzyme_kinetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-lang" aria-labelledby="p-lang-label">
			<h3 id="p-lang-label">Languages</h3>
			<div class="body">
								<ul>
					<li class="interlanguage-link interwiki-ar"><a href="https://ar.wikipedia.org/wiki/%D8%AD%D8%B1%D9%83%D9%8A%D8%A7%D8%AA_%D8%A7%D9%84%D8%A5%D9%86%D8%B2%D9%8A%D9%85" title="حركيات الإنزيم – Arabic" lang="ar" hreflang="ar" class="interlanguage-link-target">العربية</a></li><li class="interlanguage-link interwiki-bs"><a href="https://bs.wikipedia.org/wiki/Kinetika_enzima" title="Kinetika enzima – Bosnian" lang="bs" hreflang="bs" class="interlanguage-link-target">Bosanski</a></li><li class="interlanguage-link interwiki-ca"><a href="https://ca.wikipedia.org/wiki/Cin%C3%A8tica_enzim%C3%A0tica" title="Cinètica enzimàtica – Catalan" lang="ca" hreflang="ca" class="interlanguage-link-target">Català</a></li><li class="interlanguage-link interwiki-cs"><a href="https://cs.wikipedia.org/wiki/Enzymatick%C3%A1_kinetika" title="Enzymatická kinetika – Czech" lang="cs" hreflang="cs" class="interlanguage-link-target">Čeština</a></li><li class="interlanguage-link interwiki-da"><a href="https://da.wikipedia.org/wiki/Enzymkinetik" title="Enzymkinetik – Danish" lang="da" hreflang="da" class="interlanguage-link-target">Dansk</a></li><li class="interlanguage-link interwiki-de"><a href="https://de.wikipedia.org/wiki/Enzymkinetik" title="Enzymkinetik – German" lang="de" hreflang="de" class="interlanguage-link-target">Deutsch</a></li><li class="interlanguage-link interwiki-et"><a href="https://et.wikipedia.org/wiki/Ens%C3%BC%C3%BCmikineetika" title="Ensüümikineetika – Estonian" lang="et" hreflang="et" class="interlanguage-link-target">Eesti</a></li><li class="interlanguage-link interwiki-es badge-Q17437796 badge-featuredarticle" title="featured article"><a href="https://es.wikipedia.org/wiki/Cin%C3%A9tica_enzim%C3%A1tica" title="Cinética enzimática – Spanish" lang="es" hreflang="es" class="interlanguage-link-target">Español</a></li><li class="interlanguage-link interwiki-eu"><a href="https://eu.wikipedia.org/wiki/Entzimen_zinetika" title="Entzimen zinetika – Basque" lang="eu" hreflang="eu" class="interlanguage-link-target">Euskara</a></li><li class="interlanguage-link interwiki-fa"><a href="https://fa.wikipedia.org/wiki/%D8%B3%DB%8C%D9%86%D8%AA%DB%8C%DA%A9_%D8%A2%D9%86%D8%B2%DB%8C%D9%85%DB%8C" title="سینتیک آنزیمی – Persian" lang="fa" hreflang="fa" class="interlanguage-link-target">فارسی</a></li><li class="interlanguage-link interwiki-fr"><a href="https://fr.wikipedia.org/wiki/Cin%C3%A9tique_enzymatique" title="Cinétique enzymatique – French" lang="fr" hreflang="fr" class="interlanguage-link-target">Français</a></li><li class="interlanguage-link interwiki-ko"><a href="https://ko.wikipedia.org/wiki/%ED%9A%A8%EC%86%8C%EB%B0%98%EC%9D%91%EC%86%8D%EB%8F%84%EB%A1%A0" title="효소반응속도론 – Korean" lang="ko" hreflang="ko" class="interlanguage-link-target">한국어</a></li><li class="interlanguage-link interwiki-hi"><a href="https://hi.wikipedia.org/wiki/%E0%A4%8F%E0%A4%82%E0%A4%9C%E0%A4%BE%E0%A4%87%E0%A4%AE_%E0%A4%95%E0%A5%88%E0%A4%A8%E0%A5%87%E0%A4%9F%E0%A5%80%E0%A4%95%E0%A5%8D%E0%A4%B8" title="एंजाइम कैनेटीक्स – Hindi" lang="hi" hreflang="hi" class="interlanguage-link-target">हिन्दी</a></li><li class="interlanguage-link interwiki-id"><a href="https://id.wikipedia.org/wiki/Kinetika_enzim" title="Kinetika enzim – Indonesian" lang="id" hreflang="id" class="interlanguage-link-target">Bahasa Indonesia</a></li><li class="interlanguage-link interwiki-he"><a href="https://he.wikipedia.org/wiki/%D7%A7%D7%99%D7%A0%D7%98%D7%99%D7%A7%D7%94_%D7%90%D7%A0%D7%96%D7%99%D7%9E%D7%98%D7%99%D7%AA" title="קינטיקה אנזימטית – Hebrew" lang="he" hreflang="he" class="interlanguage-link-target">עברית</a></li><li class="interlanguage-link interwiki-kn"><a href="https://kn.wikipedia.org/wiki/%E0%B2%95%E0%B2%BF%E0%B2%A3%E0%B3%8D%E0%B2%B5_%E0%B2%9A%E0%B2%B2%E0%B2%A8%E0%B2%B6%E0%B2%BE%E0%B2%B8%E0%B3%8D%E0%B2%A4%E0%B3%8D%E0%B2%B0" title="ಕಿಣ್ವ ಚಲನಶಾಸ್ತ್ರ – Kannada" lang="kn" hreflang="kn" class="interlanguage-link-target">ಕನ್ನಡ</a></li><li class="interlanguage-link interwiki-hu"><a href="https://hu.wikipedia.org/wiki/Enzimkinetika" title="Enzimkinetika – Hungarian" lang="hu" hreflang="hu" class="interlanguage-link-target">Magyar</a></li><li class="interlanguage-link interwiki-nl"><a href="https://nl.wikipedia.org/wiki/Enzymkinetiek" title="Enzymkinetiek – Dutch" lang="nl" hreflang="nl" class="interlanguage-link-target">Nederlands</a></li><li class="interlanguage-link interwiki-ja"><a href="https://ja.wikipedia.org/wiki/%E9%85%B5%E7%B4%A0%E5%8F%8D%E5%BF%9C%E9%80%9F%E5%BA%A6%E8%AB%96" title="酵素反応速度論 – Japanese" lang="ja" hreflang="ja" class="interlanguage-link-target">日本語</a></li><li class="interlanguage-link interwiki-pl"><a href="https://pl.wikipedia.org/wiki/Wikiprojekt:T%C5%82umaczenie_artyku%C5%82%C3%B3w/Kinetyka_enzymatyczna" title="Wikiprojekt:Tłumaczenie artykułów/Kinetyka enzymatyczna – Polish" lang="pl" hreflang="pl" class="interlanguage-link-target">Polski</a></li><li class="interlanguage-link interwiki-pt"><a href="https://pt.wikipedia.org/wiki/Cin%C3%A9tica_enzim%C3%A1tica" title="Cinética enzimática – Portuguese" lang="pt" hreflang="pt" class="interlanguage-link-target">Português</a></li><li class="interlanguage-link interwiki-ro"><a href="https://ro.wikipedia.org/wiki/Cinetic%C4%83_enzimatic%C4%83" title="Cinetică enzimatică – Romanian" lang="ro" hreflang="ro" class="interlanguage-link-target">Română</a></li><li class="interlanguage-link interwiki-ru"><a href="https://ru.wikipedia.org/wiki/%D0%A4%D0%B5%D1%80%D0%BC%D0%B5%D0%BD%D1%82%D0%B0%D1%82%D0%B8%D0%B2%D0%BD%D0%B0%D1%8F_%D0%BA%D0%B8%D0%BD%D0%B5%D1%82%D0%B8%D0%BA%D0%B0" title="Ферментативная кинетика – Russian" lang="ru" hreflang="ru" class="interlanguage-link-target">Русский</a></li><li class="interlanguage-link interwiki-sr badge-Q17437796 badge-featuredarticle" title="featured article"><a href="https://sr.wikipedia.org/wiki/%D0%95%D0%BD%D0%B7%D0%B8%D0%BC%D1%81%D0%BA%D0%B0_%D0%BA%D0%B8%D0%BD%D0%B5%D1%82%D0%B8%D0%BA%D0%B0" title="Ензимска кинетика – Serbian" lang="sr" hreflang="sr" class="interlanguage-link-target">Српски / srpski</a></li><li class="interlanguage-link interwiki-sh"><a href="https://sh.wikipedia.org/wiki/Enzimska_kinetika" title="Enzimska kinetika – Serbo-Croatian" lang="sh" hreflang="sh" class="interlanguage-link-target">Srpskohrvatski / српскохрватски</a></li><li class="interlanguage-link interwiki-th"><a href="https://th.wikipedia.org/wiki/%E0%B8%88%E0%B8%A5%E0%B8%99%E0%B8%A8%E0%B8%B2%E0%B8%AA%E0%B8%95%E0%B8%A3%E0%B9%8C%E0%B8%82%E0%B8%AD%E0%B8%87%E0%B9%80%E0%B8%AD%E0%B8%99%E0%B9%84%E0%B8%8B%E0%B8%A1%E0%B9%8C" title="จลนศาสตร์ของเอนไซม์ – Thai" lang="th" hreflang="th" class="interlanguage-link-target">ไทย</a></li><li class="interlanguage-link interwiki-tr"><a href="https://tr.wikipedia.org/wiki/Enzim_kineti%C4%9Fi" title="Enzim kinetiği – Turkish" lang="tr" hreflang="tr" class="interlanguage-link-target">Türkçe</a></li><li class="interlanguage-link interwiki-zh"><a href="https://zh.wikipedia.org/wiki/%E9%85%B6%E5%8A%A8%E5%8A%9B%E5%AD%A6" title="酶动力学 – Chinese" lang="zh" hreflang="zh" class="interlanguage-link-target">中文</a></li>				</ul>
				<div class="after-portlet after-portlet-lang"><span class="wb-langlinks-edit wb-langlinks-link"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q883112#sitelinks-wikipedia" title="Edit interlanguage links" class="wbc-editpage">Edit links</a></span></div>			</div>
		</div>
				</div>
		</div>
				<div id="footer" role="contentinfo">
						<ul id="footer-info">
								<li id="footer-info-lastmod"> This page was last edited on 31 July 2019, at 05:44<span class="anonymous-show">&#160;(UTC)</span>.</li>
								<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//foundation.wikimedia.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//foundation.wikimedia.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
							</ul>
						<ul id="footer-places">
								<li id="footer-places-privacy"><a href="https://foundation.wikimedia.org/wiki/Privacy_policy" class="extiw" title="wmf:Privacy policy">Privacy policy</a></li>
								<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
								<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
								<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
								<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
								<li id="footer-places-cookiestatement"><a href="https://foundation.wikimedia.org/wiki/Cookie_statement">Cookie statement</a></li>
								<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=Enzyme_kinetics&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
							</ul>
										<ul id="footer-icons" class="noprint">
										<li id="footer-copyrightico">
						<a href="https://wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a>					</li>
										<li id="footer-poweredbyico">
						<a href="https://www.mediawiki.org/"><img src="/static/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/images/poweredby_mediawiki_132x47.png 1.5x, /static/images/poweredby_mediawiki_176x62.png 2x" width="88" height="31"/></a>					</li>
									</ul>
						<div style="clear: both;"></div>
		</div>
		

<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgPageParseReport":{"limitreport":{"cputime":"1.260","walltime":"1.585","ppvisitednodes":{"value":4008,"limit":1000000},"ppgeneratednodes":{"value":0,"limit":1500000},"postexpandincludesize":{"value":163767,"limit":2097152},"templateargumentsize":{"value":1002,"limit":2097152},"expansiondepth":{"value":12,"limit":40},"expensivefunctioncount":{"value":7,"limit":500},"unstrip-depth":{"value":1,"limit":20},"unstrip-size":{"value":226488,"limit":5000000},"entityaccesscount":{"value":5,"limit":400},"timingprofile":["100.00% 1122.229      1 -total"," 70.25%  788.412      1 Template:Reflist"," 54.39%  610.327     55 Template:Cite_journal"," 16.53%  185.540     15 Template:Cite_book","  3.88%   43.488      2 Template:Main_other","  3.80%   42.659      1 Template:Featured_article","  3.57%   40.062      1 Template:Top_icon","  2.69%   30.208      1 Template:Pp-move-indef","  2.19%   24.623      1 Template:Category_handler","  2.18%   24.493      1 Template:Multiple_image"]},"scribunto":{"limitreport-timeusage":{"value":"0.743","limit":"10.000"},"limitreport-memusage":{"value":5383630,"limit":52428800}},"cachereport":{"origin":"mw1229","timestamp":"20190731054416","ttl":2592000,"transientcontent":false}}});});</script>
<script type="application/ld+json">{"@context":"https:\/\/schema.org","@type":"Article","name":"Enzyme kinetics","url":"https:\/\/en.wikipedia.org\/wiki\/Enzyme_kinetics","sameAs":"http:\/\/www.wikidata.org\/entity\/Q883112","mainEntity":"http:\/\/www.wikidata.org\/entity\/Q883112","author":{"@type":"Organization","name":"Contributors to Wikimedia projects"},"publisher":{"@type":"Organization","name":"Wikimedia Foundation, Inc.","logo":{"@type":"ImageObject","url":"https:\/\/www.wikimedia.org\/static\/images\/wmf-hor-googpub.png"}},"datePublished":"2005-10-31T08:55:49Z","dateModified":"2019-07-31T05:44:07Z","image":"https:\/\/upload.wikimedia.org\/wikipedia\/commons\/6\/6d\/EcDHFR_raytraced.png","headline":"the study of biochemical reaction rates catalysed by an enzyme"}</script>
<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgBackendResponseTime":109,"wgHostname":"mw1331"});});</script>
</body>
</html>
