The following text has been accessed from https://en.wikipedia.org/wiki/Enzyme_kinetics at Thu Aug 8 23:56:56 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_featured_article._Click_here_for_more_information.]
****** Enzyme kinetics ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Dihydrofolate_reductase from E._coli with its two substrates dihydrofolate
(right) and NADPH (left), bound in the active site. The protein is shown as a
ribbon_diagram, with alpha helices in red, beta sheets in yellow and loops in
blue. Generated from 7DFR.
Enzyme kinetics is the study of the chemical_reactions that are catalysed by
enzymes. In enzyme kinetics, the reaction_rate is measured and the effects of
varying the conditions of the reaction are investigated. Studying an enzyme's
kinetics in this way can reveal the catalytic mechanism of this enzyme, its
role in metabolism, how its activity is controlled, and how a drug or an
agonist might inhibit the enzyme.
Enzymes are usually protein molecules that manipulate other moleculesâthe
enzymes' substrates. These target molecules bind to an enzyme's active_site and
are transformed into products through a series of steps known as the enzymatic
mechanism
      E + S â ES â ES* â EP â E + P
These mechanisms can be divided into single-substrate and multiple-substrate
mechanisms. Kinetic studies on enzymes that only bind one substrate, such as
triosephosphate_isomerase, aim to measure the affinity with which the enzyme
binds this substrate and the turnover rate. Some other examples of enzymes are
phosphofructokinase and hexokinase, both of which are important for cellular
respiration (glycolysis).
When enzymes bind multiple substrates, such as dihydrofolate_reductase (shown
right), enzyme kinetics can also show the sequence in which these substrates
bind and the sequence in which products are released. An example of enzymes
that bind a single substrate and release multiple products are proteases, which
cleave one protein substrate into two polypeptide products. Others join two
substrates together, such as DNA_polymerase linking a nucleotide to DNA.
Although these mechanisms are often a complex series of steps, there is
typically one rate-determining step that determines the overall kinetics. This
rate-determining_step may be a chemical reaction or a conformational change of
the enzyme or substrates, such as those involved in the release of product(s)
from the enzyme.
Knowledge of the enzyme's_structure is helpful in interpreting kinetic data.
For example, the structure can suggest how substrates and products bind during
catalysis; what changes occur during the reaction; and even the role of
particular amino_acid residues in the mechanism. Some enzymes change shape
significantly during the mechanism; in such cases, it is helpful to determine
the enzyme structure with and without bound substrate analogues that do not
undergo the enzymatic reaction.
Not all biological catalysts are protein enzymes; RNA-based catalysts such as
ribozymes and ribosomes are essential to many cellular functions, such as RNA
splicing and translation. The main difference between ribozymes and enzymes is
that RNA catalysts are composed of nucleotides, whereas enzymes are composed of
amino acids. Ribozymes also perform a more limited set of reactions, although
their reaction_mechanisms and kinetics can be analysed and classified by the
same methods.
⁰
***** Contents *****
    * 1_General_principles
    * 2_Enzyme_assays
    * 3_Single-substrate_reactions
          o 3.1_MichaelisâMenten_kinetics
          o 3.2_Direct_use_of_the_MichaelisâMenten_equation_for_time_course
            kinetic_analysis
          o 3.3_Linear_plots_of_the_MichaelisâMenten_equation
          o 3.4_Practical_significance_of_kinetic_constants
          o 3.5_MichaelisâMenten_kinetics_with_intermediate
    * 4_Multi-substrate_reactions
          o 4.1_Ternary-complex_mechanisms
          o 4.2_Pingâpong_mechanisms
    * 5_Reversible_catalysis_and_the_Haldane_equation
    * 6_Non-MichaelisâMenten_kinetics
    * 7_Pre-steady-state_kinetics
    * 8_Chemical_mechanism
    * 9_Enzyme_inhibition_and_activation
          o 9.1_Reversible_inhibitors
          o 9.2_Irreversible_inhibitors
          o 9.3_Philosophical_discourse_on_reversibility_and_irreversibility_of
            inhibition
    * 10_Mechanisms_of_catalysis
    * 11_History
    * 12_Software
          o 12.1_ENZO
    * 13_See_also
    * 14_Footnotes
    * 15_References
    * 16_Further_reading
    * 17_External_links
***** General principles[edit] *****
As larger amounts of substrate are added to a reaction, the available enzyme
binding_sites become filled to the limit of ____V__max_____{\displaystyle_V_
{\max_}}__[V_{\max_}]. Beyond this limit the enzyme is saturated with substrate
and the reaction rate ceases to increase.
The reaction catalysed by an enzyme uses exactly the same reactants and
produces exactly the same products as the uncatalysed reaction. Like other
catalysts, enzymes do not alter the position of equilibrium between substrates
and products.[1] However, unlike uncatalysed chemical reactions, enzyme-
catalysed reactions display saturation kinetics. For a given enzyme
concentration and for relatively low substrate concentrations, the reaction
rate increases linearly with substrate concentration; the enzyme molecules are
largely free to catalyse the reaction, and increasing substrate concentration
means an increasing rate at which the enzyme and substrate molecules encounter
one another. However, at relatively high substrate concentrations, the reaction
rate asymptotically approaches the theoretical maximum; the enzyme active sites
are almost all occupied by substrates resulting in saturation, and the reaction
rate is determined by the intrinsic turnover rate of the enzyme.[2] The
substrate concentration midway between these two limiting cases is denoted by
KM. Thus, KM is the substrate concentration at which the reaction velocity is
half of the maximum velocity.[2]
The two most important kinetic properties of an enzyme are how easily the
enzyme becomes saturated with a particular substrate, and the maximum rate it
can achieve. Knowing these properties suggests what an enzyme might do in the
cell and can show how the enzyme will respond to changes in these conditions.
***** Enzyme assays[edit] *****
Main article: Enzyme_assay
Progress curve for an enzyme reaction. The slope in the initial rate period is
the initial rate of reaction v. The MichaelisâMenten_equation describes how
this slope varies with the concentration of substrate.
Enzyme_assays are laboratory procedures that measure the rate of enzyme
reactions. Since enzymes are not consumed by the reactions they catalyse,
enzyme assays usually follow changes in the concentration of either substrates
or products to measure the rate of reaction. There are many methods of
measurement. Spectrophotometric assays observe change in the absorbance of
light between products and reactants; radiometric assays involve the
incorporation or release of radioactivity to measure the amount of product made
over time. Spectrophotometric assays are most convenient since they allow the
rate of the reaction to be measured continuously. Although radiometric assays
require the removal and counting of samples (i.e., they are discontinuous
assays) they are usually extremely sensitive and can measure very low levels of
enzyme activity.[3] An analogous approach is to use mass_spectrometry to
monitor the incorporation or release of stable_isotopes as substrate is
converted into product.
The most sensitive enzyme assays use lasers focused through a microscope to
observe changes in single enzyme molecules as they catalyse their reactions.
These measurements either use changes in the fluorescence of cofactors during
an enzyme's reaction mechanism, or of fluorescent_dyes added onto specific
sites of the protein to report movements that occur during catalysis.[4] These
studies are providing a new view of the kinetics and dynamics of single
enzymes, as opposed to traditional enzyme kinetics, which observes the average
behaviour of populations of millions of enzyme molecules.[5][6]
An example progress curve for an enzyme assay is shown above. The enzyme
produces product at an initial rate that is approximately linear for a short
period after the start of the reaction. As the reaction proceeds and substrate
is consumed, the rate continuously slows (so long as substrate is not still at
saturating levels). To measure the initial (and maximal) rate, enzyme assays
are typically carried out while the reaction has progressed only a few percent
towards total completion. The length of the initial rate period depends on the
assay conditions and can range from milliseconds to hours. However, equipment
for rapidly mixing liquids allows fast kinetic measurements on initial rates of
less than one second.[7] These very rapid assays are essential for measuring
pre-steady-state kinetics, which are discussed below.
Most enzyme kinetics studies concentrate on this initial, approximately linear
part of enzyme reactions. However, it is also possible to measure the complete
reaction curve and fit this data to a non-linear rate_equation. This way of
measuring enzyme reactions is called progress-curve analysis.[8] This approach
is useful as an alternative to rapid_kinetics when the initial rate is too fast
to measure accurately.
***** Single-substrate reactions[edit] *****
Enzymes with single-substrate mechanisms include isomerases such as
triosephosphateisomerase or bisphosphoglycerate_mutase, intramolecular lyases
such as adenylate_cyclase and the hammerhead_ribozyme, an RNA lyase.[9]
However, some enzymes that only have a single substrate do not fall into this
category of mechanisms. Catalase is an example of this, as the enzyme reacts
with a first molecule of hydrogen_peroxide substrate, becomes oxidised and is
then reduced by a second molecule of substrate. Although a single substrate is
involved, the existence of a modified enzyme intermediate means that the
mechanism of catalase is actually a pingâpong mechanism, a type of mechanism
that is discussed in the Multi-substrate reactions section below.
**** MichaelisâMenten kinetics[edit] ****
[Schematic_reaction_diagrams_for_uncatalzyed_(Substrate_to_Product)_and
catalyzed_(Enzyme_+_Substrate_to_Enzyme/Substrate_complex_to_Enzyme_+_Product)]
A chemical reaction mechanism with or without enzyme_catalysis. The enzyme (E)
binds substrate (S) to produce product (P).
[A_two_dimensional_plot_of_substrate_concentration_(x_axis)_vs._reaction_rate_
(y_axis)._The_shape_of_the_curve_is_hyperbolic._The_rate_of_the_reaction_is
zero_at_zero_concentration_of_substrate_and_the_rate_asymptotically_reaches_a
maximum_at_high_substrate_concentration.]
Saturation_curve for an enzyme reaction showing the relation between the
substrate concentration and reaction rate.
Main article: MichaelisâMenten_kinetics
Kinetic model to explain single substrate enzyme catalyzed reactions was first
proposed by Henri (1903) and Michaelis and Menten (1913). Henri developed the
theoretical part of the model while Michaelis and Menten did a experimental
work to prove the validity of the model.
Majority of the enzymes are proteins and they speeds up the rate of biochemical
reactions by decreasing the activation energy. During the process enzymes
combines with the substrate and convert it into product. Enzymes may have
single or multiple substrate binding sites (catalytic site). As substrate
concentration goes on increasing, catalytic sites may get filled progressively.
In the initial period velocity of the enzyme catalyzed reaction is directly
proportional to substrate concentration. Michaelis and Menten specifically
focused on the initial period of the reaction (initial velocity) to develop
this model.
As enzyme-catalysed reactions are saturable, their rate of catalysis does not
show a linear response to increasing substrate. If the initial rate of the
reaction is measured over a range of substrate concentrations (denoted as [S]),
the initial reaction rate (     v  0     {\displaystyle v_{0}}  [v_{0}])
increases as [S] increases, as shown on the right. However, as [S] gets higher,
the enzyme becomes saturated with substrate and the initial rate reaches Vmax,
the enzyme's maximum rate.
The MichaelisâMenten_kinetic_model_of_a_single-substrate_reaction is shown on
the right. There is an initial bimolecular_reaction between the enzyme E and
substrate S to form the enzymeâsubstrate complex ES. The rate of enzymatic
reaction increases with the increase of the substrate concentration up to a
certain level called Vmax; at Vmax, increase in substrate concentration does
not cause any increase in reaction rate as there is no more enzyme (E)
available for reacting with substrate (S). Here, the rate of reaction becomes
dependent on the ES complex and the reaction becomes a unimolecular_reaction
with an order of zero. Though the enzymatic mechanism for the unimolecular
reaction      ES   &#x2192;   k  c a t      E + P    {\displaystyle {\ce {ES ->
[k_{cat}] E + P}}}  [{\displaystyle {\ce {ES ->[k_{cat}] E + P}}}] can be quite
complex, there is typically one rate-determining enzymatic step that allows
this reaction to be modelled as a single catalytic step with an apparent
unimolecular rate constant kcat. If the reaction path proceeds over one or
several intermediates, kcat will be a function of several elementary rate
constants, whereas in the simplest case of a single elementary reaction (e.g.
no intermediates) it will be identical to the elementary unimolecular rate
constant k2. The apparent unimolecular rate constant kcat is also called
turnover number and denotes the maximum number of enzymatic reactions catalysed
per second.
The MichaelisâMenten_equation[10] describes how the (initial) reaction rate
v0 depends on the position of the substrate-binding equilibrium and the rate
constant k2.
          v  0   =     V  max   [   S   ]    K  M   + [   S   ]
      {\displaystyle v_{0}={\frac {V_{\max }[{{\ce {S}}}]}{K_{M}+[{{\ce
      {S}}}]}}}  [{\displaystyle v_{0}={\frac {V_{\max }[{\ce {S}}]}{K_{M}+[
      {\ce {S}}]}}}]    (MichaelisâMenten equation)
with the constants
              K  M   &#xA0;        =    d e f      &#xA0;     k  2   +  k
      &#x2212; 1     k  1     &#x2248;  K  D        V  max   &#xA0;        =
      d e f      &#xA0;  k  c a t       [ E ]     t o t         {\displaystyle
      {\begin{aligned}K_{M}\ &{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{2}+k_
      {-1}}{k_{1}}}\approx K_{D}\\V_{\max }\ &{\stackrel {\mathrm {def} }{=}}\
      k_{cat}{{\ce {[E]}}}_{tot}\end{aligned}}}  [{\displaystyle {\begin
      {aligned}K_{M}\ &{\stackrel {\mathrm {def} }{=}}\ {\frac {k_{2}+k_{-1}}
      {k_{1}}}\approx K_{D}\\V_{\max }\ &{\stackrel {\mathrm {def} }{=}}\ k_
      {cat}{\ce {[E]}}_{tot}\end{aligned}}}]
This MichaelisâMenten equation is the basis for most single-substrate enzyme
kinetics. Two crucial assumptions underlie this equation (apart from the
general assumption about the mechanism only involving no intermediate or
product inhibition, and there is no allostericity or cooperativity). The first
assumption is the so-called quasi-steady-state_assumption (or pseudo-steady-
state hypothesis), namely that the concentration of the substrate-bound enzyme
(and hence also the unbound enzyme) changes much more slowly than those of the
product and substrate and thus the change over time of the complex can be set
to zero     d    [ ES ]     /   d t     = !    0   {\displaystyle d{{\ce {
[ES]}}}/{dt}\;{\overset {!}{=}}\;0}  [{\displaystyle d{\ce {[ES]}}/{dt}\;
{\overset {!}{=}}\;0}]. The second assumption is that the total enzyme
concentration does not change over time, thus        [ E ]    tot   =   [ E ]
+   [ ES ]      = !     const    {\displaystyle {\ce {[E]}}_{\text{tot}}={\ce {
[E]}}+{\ce {[ES]}}\;{\overset {!}{=}}\;{\text{const}}}  [{\displaystyle {\ce {
[E]}}_{\text{tot}}={\ce {[E]}}+{\ce {[ES]}}\;{\overset {!}{=}}\;{\text
{const}}}]. A complete derivation can be found here.
The Michaelis constant KM is experimentally defined as the concentration at
which the rate of the enzyme reaction is half Vmax, which can be verified by
substituting [S] = KM into the MichaelisâMenten equation and can also be seen
graphically. If the rate-determining enzymatic step is slow compared to
substrate dissociation (     k  2   &#x226A;  k  &#x2212; 1     {\displaystyle
k_{2}\ll k_{-1}}  [k_{2}\ll k_{{-1}}]), the Michaelis constant KM is roughly
the dissociation_constant KD of the ES complex.
If       [ S ]     {\displaystyle {\ce {[S]}}}  [{\displaystyle {\ce {[S]}}}]
is small compared to      K  M     {\displaystyle K_{M}}  [K_M] then the term
[   S   ]  /  (  K  M   + [   S   ] ) &#x2248; [   S   ]  /   K  M
{\displaystyle [{{\ce {S}}}]/(K_{M}+[{{\ce {S}}}])\approx [{{\ce {S}}}]/K_{M}}
[{\displaystyle [{\ce {S}}]/(K_{M}+[{\ce {S}}])\approx [{\ce {S}}]/K_{M}}] and
also very little ES complex is formed, thus        [ E ]    tot       &#x2248;
[ E ]     {\displaystyle {\ce {[E]_{\rm {tot}}\approx [E]}}}  [{\displaystyle
{\ce {[E]_{\rm {tot}}\approx [E]}}}]. Therefore, the rate of product formation
is
          v  0   &#x2248;    k  c a t    K  M       [ E ]   [ S ]      if&#xA0;
      [  S  ] &#x226A;  K  M     {\displaystyle v_{0}\approx {\frac {k_{cat}}
      {K_{M}}}{\ce {[E][S]}}\qquad \qquad {\text{if }}[{\ce {S}}]\ll K_{M}}  [
      {\displaystyle v_{0}\approx {\frac {k_{cat}}{K_{M}}}{\ce {[E][S]}}\qquad
      \qquad {\text{if }}[{\ce {S}}]\ll K_{M}}]
Thus the product formation rate depends on the enzyme concentration as well as
on the substrate concentration, the equation resembles a bimolecular reaction
with a corresponding pseudo-second order rate constant      k  2    /   K  M
{\displaystyle k_{2}/K_{M}}  [k_{2}/K_{M}]. This constant is a measure of
catalytic_efficiency. The most efficient enzymes reach a      k  2    /   K  M
{\displaystyle k_{2}/K_{M}}  [k_{2}/K_{M}] in the range of 108 â
1010 Mâ1 sâ1. These enzymes are so efficient they effectively catalyse a
reaction each time they encounter a substrate molecule and have thus reached an
upper theoretical limit for efficiency (diffusion_limit); and are sometimes
referred to as kinetically_perfect_enzymes.[11] But most enzymes are far from
perfect: the average values of      k  2    /   K   M      {\displaystyle k_
{2}/K_{\rm {M}}}  [{\displaystyle k_{2}/K_{\rm {M}}}] and      k  2
{\displaystyle k_{2}}  [k_{{2}}] are about      10  5      s    &#x2212; 1
M    &#x2212; 1     {\displaystyle 10^{5}{\rm {s}}^{-1}{\rm {M}}^{-1}}  [
{\displaystyle 10^{5}{\rm {s}}^{-1}{\rm {M}}^{-1}}] and     10    s    &#x2212;
1     {\displaystyle 10{\rm {s}}^{-1}}  [{\displaystyle 10{\rm {s}}^{-1}}],
respectively.[12]
**** Direct use of the MichaelisâMenten equation for time course kinetic
analysis[edit] ****
Further information: Rate_equation
Further information: Reaction_Progress_Kinetic_Analysis
The observed velocities predicted by the MichaelisâMenten equation can be
used to directly model the time_course_disappearance_of_substrate and the
production of product through incorporation of the MichaelisâMenten equation
into the equation for first order chemical kinetics. This can only be achieved
however if one recognises the problem associated with the use of Euler's_number
in the description of first order chemical kinetics. i.e. eâk is a split
constant that introduces a systematic error into calculations and can be
rewritten as a single constant which represents the remaining substrate after
each time period.[13]
         [ S ] = [ S  ]  0   ( 1 &#x2212; k  )  t      {\displaystyle [S]=[S]_
      {0}(1-k)^{t}\,}  [[S]=[S]_{0}(1-k)^{{t}}\,]
         [ S ] = [ S  ]  0   ( 1 &#x2212; v  /  [ S  ]  0    )  t
      {\displaystyle [S]=[S]_{0}(1-v/[S]_{0})^{t}\,}  [[S]=[S]_{0}(1-v/[S]_
      {0})^{{t}}\,]
         [ S ] = [ S  ]  0   ( 1 &#x2212; (  V  max   [ S  ]  0    /  (  K  M
      + [ S  ]  0   )  /  [ S  ]  0   )  )  t      {\displaystyle [S]=[S]_{0}
      (1-(V_{\max }[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{t}\,}  [[S]=[S]_{0}(1-(V_
      {{\max }}[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{{t}}\,]
In 1983 Stuart Beal (and also independently Santiago_Schnell and Claudio
Mendoza in 1997) derived a closed form solution for the time course kinetics
analysis of the Michaelis-Menten mechanism.[14][15] The solution, known as the
Schnell-Mendoza equation, has the form:
            [ S ]   K  M     = W  [  F ( t )  ]     {\displaystyle {\frac {[S]}
      {K_{M}}}=W\left[F(t)\right]\,}  [{\frac  {[S]}{K_{M}}}=W\left[F
      (t)\right]\,]
where W[ ] is the Lambert-W_function.[16][17] and where F(t) is
         F ( t ) =    [ S  ]  0     K  M     exp   (     [ S  ]  0     K  M
      &#x2212;    V  max    K  M      t  )     {\displaystyle F(t)={\frac {[S]_
      {0}}{K_{M}}}\exp \!\left({\frac {[S]_{0}}{K_{M}}}-{\frac {V_{\max }}{K_
      {M}}}\,t\right)\,}  [F(t) = \frac{[S]_0}{K_M} \exp\!\left(\frac{[S]_0}
      {K_M} - \frac{V_\max}{K_M}\,t \right) \,  ]
This equation is encompassed by the equation below, obtained by Berberan-
Santos,[18] which is also valid when the initial substrate concentration is
close to that of enzyme,
            [ S ]   K  M     = W  [  F ( t )  ]  &#x2212;    V  max     k  c a
      t    K  M      &#xA0;    W  [  F ( t )  ]    1 + W  [  F ( t )  ]
      {\displaystyle {\frac {[S]}{K_{M}}}=W\left[F(t)\right]-{\frac {V_{\max }}
      {k_{cat}K_{M}}}\ {\frac {W\left[F(t)\right]}{1+W\left[F(t)\right]}}\,}  [
      {\frac  {[S]}{K_{M}}}=W\left[F(t)\right]-{\frac  {V_{\max }}{k_{{cat}}K_
      {M}}}\ {\frac  {W\left[F(t)\right]}{1+W\left[F(t)\right]}}\,]
where W[ ] is again the Lambert-W_function.
**** Linear plots of the MichaelisâMenten equation[edit] ****
See also: LineweaverâBurk_plot and EadieâHofstee_diagram
LineweaverâBurk or double-reciprocal plot of kinetic data, showing the
significance of the axis intercepts and gradient.
The plot of v versus [S] above is not linear; although initially linear at low
[S], it bends over to saturate at high [S]. Before the modern era of nonlinear
curve-fitting on computers, this nonlinearity could make it difficult to
estimate KM and Vmax accurately. Therefore, several researchers developed
linearisations of the MichaelisâMenten equation, such as the
LineweaverâBurk_plot, the EadieâHofstee_diagram and the HanesâWoolf_plot.
All of these linear representations can be useful for visualising data, but
none should be used to determine kinetic parameters, as computer software is
readily available that allows for more accurate determination by nonlinear
regression methods.[19]
The LineweaverâBurk_plot or double reciprocal plot is a common way of
illustrating kinetic data. This is produced by taking the reciprocal of both
sides of the MichaelisâMenten equation. As shown on the right, this is a
linear form of the MichaelisâMenten equation and produces a straight line
with the equation y = mx + c with a y-intercept equivalent to 1/Vmax and an x-
intercept of the graph representing â1/KM.
           1 v   =    K  M     V  max   [   S   ]    +   1  V  max
      {\displaystyle {\frac {1}{v}}={\frac {K_{M}}{V_{\max }[{\mbox{S}}]}}+
      {\frac {1}{V_{\max }}}}  [{\frac  {1}{v}}={\frac  {K_{{M}}}{V_{{\max }}[
      {\mbox{S}}]}}+{\frac  {1}{V_{\max }}}]
Naturally, no experimental values can be taken at negative 1/[S]; the lower
limiting value 1/[S] = 0 (the y-intercept) corresponds to an infinite substrate
concentration, where 1/v=1/Vmax as shown at the right; thus, the x-intercept is
an extrapolation of the experimental data taken at positive concentrations.
More generally, the LineweaverâBurk plot skews the importance of measurements
taken at low substrate concentrations and, thus, can yield inaccurate estimates
of Vmax and KM.[20] A more accurate linear plotting method is the
EadieâHofstee_plot. In this case, v is plotted against v/[S]. In the third
common linear representation, the HanesâWoolf_plot, [S]/v is plotted against
[S]. In general, data normalisation can help diminish the amount of
experimental work and can increase the reliability of the output, and is
suitable for both graphical and numerical analysis.[21]
**** Practical significance of kinetic constants[edit] ****
The study of enzyme kinetics is important for two basic reasons. Firstly, it
helps explain how enzymes work, and secondly, it helps predict how enzymes
behave in living organisms. The kinetic constants defined above, KM and Vmax,
are critical to attempts to understand how enzymes work together to control
metabolism.
Making these predictions is not trivial, even for simple systems. For example,
oxaloacetate is formed by malate_dehydrogenase within the mitochondrion.
Oxaloacetate can then be consumed by citrate_synthase, phosphoenolpyruvate
carboxykinase or aspartate_aminotransferase, feeding into the citric_acid
cycle, gluconeogenesis or aspartic_acid biosynthesis, respectively. Being able
to predict how much oxaloacetate goes into which pathway requires knowledge of
the concentration of oxaloacetate as well as the concentration and kinetics of
each of these enzymes. This aim of predicting the behaviour of metabolic
pathways reaches its most complex expression in the synthesis of huge amounts
of kinetic and gene_expression data into mathematical models of entire
organisms. Alternatively, one useful simplification of the metabolic modelling
problem is to ignore the underlying enzyme kinetics and only rely on
information about the reaction network's stoichiometry, a technique called flux
balance_analysis.[22][23]
**** MichaelisâMenten kinetics with intermediate[edit] ****
One could also consider the less simple case
           E  + S   &#x21CC;   k  &#x2212; 1      k  1      ES   &#x2192;   k
      2      EI   &#x2192;   k  3       E  + P    {\displaystyle {\ce {{E}+S<=>
      [k_{1}][k_{-1}]ES->[k_{2}]EI->[k_{3}]{E}+P}}}  [{\displaystyle {\ce {
      {E}+S<=>[k_{1}][k_{-1}]ES->[k_{2}]EI->[k_{3}]{E}+P}}}]
where a complex with the enzyme and an intermediate exists and the intermediate
is converted into product in a second step. In this case we have a very similar
equation[24]
          v  0   =  k  c a t       [ S ]    [ E ]   0         K  M   &#x2032;
      +   [ S ]        {\displaystyle v_{0}=k_{cat}{\frac {{\ce {[S] [E]_0}}}
      {K_{M}^{\prime }+{\ce {[S]}}}}}  [{\displaystyle v_{0}=k_{cat}{\frac {
      {\ce {[S] [E]_0}}}{K_{M}^{\prime }+{\ce {[S]}}}}}]
but the constants are different
              K  M   &#x2032;   &#xA0;        =    d e f      &#xA0;    k  3
      k  2   +  k  3       K  M   =    k  3     k  2   +  k  3      &#x22C5;
      k  2   +  k  &#x2212; 1     k  1          k  c a t   &#xA0;        =    d
      e f      &#xA0;      k  3    k  2      k  2   +  k  3
      {\displaystyle {\begin{aligned}K_{M}^{\prime }\ &{\stackrel {\mathrm
      {def} }{=}}\ {\frac {k_{3}}{k_{2}+k_{3}}}K_{M}={\frac {k_{3}}{k_{2}+k_
      {3}}}\cdot {\frac {k_{2}+k_{-1}}{k_{1}}}\\k_{cat}\ &{\stackrel {\mathrm
      {def} }{=}}\ {\dfrac {k_{3}k_{2}}{k_{2}+k_{3}}}\end{aligned}}}  [{\begin
      {aligned}K_{M}^{{\prime }}\ &{\stackrel  {{\mathrm  {def}}}{=}}\ {\frac
      {k_{3}}{k_{2}+k_{3}}}K_{M}={\frac  {k_{3}}{k_{2}+k_{3}}}\cdot {\frac  {k_
      {{2}}+k_{{-1}}}{k_{{1}}}}\\k_{{cat}}\ &{\stackrel  {{\mathrm  {def}}}
      {=}}\ {\dfrac  {k_{3}k_{2}}{k_{2}+k_{3}}}\end{aligned}}]
We see that for the limiting case      k  3   &#x226B;  k  2     {\displaystyle
k_{3}\gg k_{2}}  [k_{3}\gg k_{2}], thus when the last step from      EI
&#x27F6; E + P    {\displaystyle {\ce {EI -> E + P}}}  [{\displaystyle {\ce {EI
-> E + P}}}] is much faster than the previous step, we get again the original
equation. Mathematically we have then      K  M   &#x2032;   &#x2248;  K  M
{\displaystyle K_{M}^{\prime }\approx K_{M}}  [K_{M}^{{\prime }}\approx K_{M}]
and      k  c a t   &#x2248;  k  2     {\displaystyle k_{cat}\approx k_{2}}
[k_{{cat}}\approx k_{2}].
***** Multi-substrate reactions[edit] *****
Multi-substrate reactions follow complex rate equations that describe how the
substrates bind and in what sequence. The analysis of these reactions is much
simpler if the concentration of substrate A is kept constant and substrate B
varied. Under these conditions, the enzyme behaves just like a single-substrate
enzyme and a plot of v by [S] gives apparent KM and Vmax constants for
substrate B. If a set of these measurements is performed at different fixed
concentrations of A, these data can be used to work out what the mechanism of
the reaction is. For an enzyme that takes two substrates A and B and turns them
into two products P and Q, there are two types of mechanism: ternary complex
and pingâpong.
**** Ternary-complex mechanisms[edit] ****
Random-order ternary-complex mechanism for an enzyme reaction. The reaction
path is shown as a line and enzyme intermediates containing substrates A and B
or products P and Q are written below the line.
In these enzymes, both substrates bind to the enzyme at the same time to
produce an EAB ternary complex. The order of binding can either be random (in a
random mechanism) or substrates have to bind in a particular sequence (in an
ordered mechanism). When a set of v by [S] curves (fixed A, varying B) from an
enzyme with a ternary-complex mechanism are plotted in a LineweaverâBurk
plot, the set of lines produced will intersect.
Enzymes with ternary-complex mechanisms include glutathione_S-transferase,[25]
dihydrofolate_reductase[26] and DNA_polymerase.[27] The following links show
short animations of the ternary-complex mechanisms of the enzymes dihydrofolate
reductase[Î²] and DNA polymerase[Î³].
**** Pingâpong mechanisms[edit] ****
      E   &#x2192;    A &#x2193;      EA       &#x21BD;     &#x2212;
&#x2212;     &#x21C0;        E  &#x2217;   P   &#x2192;    P &#x2191;       E
&#x2217;     &#x2192;    B &#x2193;       E  &#x2217;   B       &#x21BD;
&#x2212;         &#x2212;     &#x21C0;       EQ   &#x2192;    Q &#x2191;      E
{\displaystyle {\ce {\overset {}{E->[{\ce {A \atop \downarrow }}]EA<=>E^{\ast
}P->[{\ce {P \atop \uparrow }}]E^{\ast }->[{\ce {B \atop \downarrow }}]E^{\ast
}B<=>EQ->[{\ce {Q \atop \uparrow }}]E}}}}  [{\displaystyle {\ce {\overset {}{E-
>[{\ce {A \atop \downarrow }}]EA<=>E^{\ast }P->[{\ce {P \atop \uparrow }}]E^
{\ast }->[{\ce {B \atop \downarrow }}]E^{\ast }B<=>EQ->[{\ce {Q \atop \uparrow
}}]E}}}}]
Pingâpong mechanism for an enzyme reaction. Intermediates contain substrates
A and B or products P and Q.
As shown on the right, enzymes with a ping-pong mechanism can exist in two
states, E and a chemically modified form of the enzyme E*; this modified enzyme
is known as an intermediate. In such mechanisms, substrate A binds, changes the
enzyme to E* by, for example, transferring a chemical group to the active site,
and is then released. Only after the first substrate is released can substrate
B bind and react with the modified enzyme, regenerating the unmodified E form.
When a set of v by [S] curves (fixed A, varying B) from an enzyme with a
pingâpong mechanism are plotted in a LineweaverâBurk plot, a set of
parallel lines will be produced. This is called a secondary_plot.
Enzymes with pingâpong mechanisms include some oxidoreductases such as
thioredoxin_peroxidase,[28] transferases such as acylneuraminate
cytidylyltransferase[29] and serine_proteases such as trypsin and chymotrypsin.
[30] Serine proteases are a very common and diverse family of enzymes,
including digestive enzymes (trypsin, chymotrypsin, and elastase), several
enzymes of the blood_clotting_cascade and many others. In these serine
proteases, the E* intermediate is an acyl-enzyme species formed by the attack
of an active site serine residue on a peptide_bond in a protein substrate. A
short animation showing the mechanism of chymotrypsin is linked here.[Î´]
***** Reversible catalysis and the Haldane equation[edit] *****
We consider the case of an enzyme that can catalyse the reaction in both
directions (not all enzymes can):
     E  +  S    &#x21CC;   k  &#x2212; 1      k  1      ES   &#x21CC;   k
&#x2212; 2      k  2       E  +  P     {\displaystyle {\ce {{E}+{S}<=>[k_{1}]
[k_{-1}]ES<=>[k_{2}][k_{-2}]{E}+{P}}}}  [{\displaystyle {\ce {{E}+{S}<=>[k_{1}]
[k_{-1}]ES<=>[k_{2}][k_{-2}]{E}+{P}}}}]
The steady-state, initial rate of the reaction is      v  0   =    d  [   P   ]
d t    =    (  k  1    k  2    [   S   ] &#x2212;  k  &#x2212; 1    k  &#x2212;
2   [   P   ] ) [   E    ]  0      k  &#x2212; 1   +  k  2   +  k  1    [   S
] +  k  &#x2212; 2    [   P   ]      {\displaystyle v_{0}={\frac {d\,[{\rm
{P}}]}{dt}}={\frac {(k_{1}k_{2}\,[{\rm {S}}]-k_{-1}k_{-2}[{\rm {P}}])[{\rm
{E}}]_{0}}{k_{-1}+k_{2}+k_{1}\,[{\rm {S}}]+k_{-2}\,[{\rm {P}}]}}}  [
{\displaystyle v_{0}={\frac {d\,[{\rm {P}}]}{dt}}={\frac {(k_{1}k_{2}\,[{\rm
{S}}]-k_{-1}k_{-2}[{\rm {P}}])[{\rm {E}}]_{0}}{k_{-1}+k_{2}+k_{1}\,[{\rm
{S}}]+k_{-2}\,[{\rm {P}}]}}}]
    v  0     {\displaystyle v_{0}}  [v_{0}] is positive if the reaction proceed
in the forward direction (    S &#x2192; P   {\displaystyle S\rightarrow P}  [
{\displaystyle S\rightarrow P}]) and negative otherwise.
Equilibrium requires that     v = 0   {\displaystyle v=0}  [v=0], which occurs
when        [   P    ]   e q      [   S    ]   e q       =     k  1    k  2
k  &#x2212; 1    k  &#x2212; 2      =  K   e q      {\displaystyle {\frac {[
{\rm {P}}]_{\rm {eq}}}{[{\rm {S}}]_{\rm {eq}}}}={\frac {k_{1}k_{2}}{k_{-1}k_{-
2}}}=K_{\rm {eq}}}  [{\displaystyle {\frac {[{\rm {P}}]_{\rm {eq}}}{[{\rm
{S}}]_{\rm {eq}}}}={\frac {k_{1}k_{2}}{k_{-1}k_{-2}}}=K_{\rm {eq}}}]. This
shows that thermodynamics forces a relation between the values of the 4 rate
constants.
The values of the forward and backward maximal rates, obtained for     [   S
] &#x2192; &#x221E;   {\displaystyle [{\rm {S}}]\rightarrow \infty }  [
{\displaystyle [{\rm {S}}]\rightarrow \infty }],     [   P   ] = 0
{\displaystyle [{\rm {P}}]=0}  [{\displaystyle [{\rm {P}}]=0}], and     [   S
] = 0   {\displaystyle [{\rm {S}}]=0}  [{\displaystyle [{\rm {S}}]=0}],
[   P   ] &#x2192; &#x221E;   {\displaystyle [{\rm {P}}]\rightarrow \infty }  [
{\displaystyle [{\rm {P}}]\rightarrow \infty }], respectively, are      V   m a
x    f   =  k  2      [ E ]    t o t     {\displaystyle V_{\rm {max}}^{f}=k_{2}
{\rm {[E]}}_{tot}}  [{\displaystyle V_{\rm {max}}^{f}=k_{2}{\rm {[E]}}_{tot}}]
and      V   m a x    b   = &#x2212;  k  &#x2212; 1      [ E ]    t o t
{\displaystyle V_{\rm {max}}^{b}=-k_{-1}{\rm {[E]}}_{tot}}  [{\displaystyle V_
{\rm {max}}^{b}=-k_{-1}{\rm {[E]}}_{tot}}], respectively. Their ratio is not
equal to the equilibrium constant, which implies that thermodynamics does not
constrain the ratio of the maximal rates. This explains that enzymes can be
much "better catalysts" (in terms of maximal rates) in one particular direction
of the reaction.[31]
On can also derive the two Michaelis constants      K  M   S   = (  k  &#x2212;
1   +  k  2   )  /   k  1     {\displaystyle K_{M}^{S}=(k_{-1}+k_{2})/k_{1}}  [
{\displaystyle K_{M}^{S}=(k_{-1}+k_{2})/k_{1}}]and      K  M   P   = (  k
&#x2212; 1   +  k  2   )  /   k  &#x2212; 2     {\displaystyle K_{M}^{P}=(k_{-
1}+k_{2})/k_{-2}}  [{\displaystyle K_{M}^{P}=(k_{-1}+k_{2})/k_{-2}}]. The
Haldane equation is the relation      K   e q    =    [   P    ]   e q
[   S    ]   e q       =     V   m a x    f    /   K  M   S      V   m a x    b
/   K  M   P        {\displaystyle K_{\rm {eq}}={\frac {[{\rm {P}}]_{\rm {eq}}}
{[{\rm {S}}]_{\rm {eq}}}}={\frac {V_{\rm {max}}^{f}/K_{M}^{S}}{V_{\rm {max}}^
{b}/K_{M}^{P}}}}  [{\displaystyle K_{\rm {eq}}={\frac {[{\rm {P}}]_{\rm {eq}}}{
[{\rm {S}}]_{\rm {eq}}}}={\frac {V_{\rm {max}}^{f}/K_{M}^{S}}{V_{\rm {max}}^
{b}/K_{M}^{P}}}}].
Therefore, thermodynamics constrains the ratio between the forward and backward
V   m a x     /   K  M     {\displaystyle V_{\rm {max}}/K_{M}}  [{\displaystyle
V_{\rm {max}}/K_{M}}] values, not the ratio of      V   m a x
{\displaystyle V_{\rm {max}}}  [{\displaystyle V_{\rm {max}}}]values.
***** Non-MichaelisâMenten kinetics[edit] *****
Main article: Allosteric_regulation
Saturation curve for an enzyme reaction showing sigmoid kinetics.
Some enzymes produce a sigmoid v by [S] plot, which often indicates cooperative
binding of substrate to the active site. This means that the binding of one
substrate molecule affects the binding of subsequent substrate molecules. This
behavior is most common in multimeric enzymes with several interacting active
sites.[32] Here, the mechanism of cooperation is similar to that of hemoglobin,
with binding of substrate to one active site altering the affinity of the other
active sites for substrate molecules. Positive cooperativity occurs when
binding of the first substrate molecule increases the affinity of the other
active sites for substrate. Negative cooperativity occurs when binding of the
first substrate decreases the affinity of the enzyme for other substrate
molecules.
Allosteric enzymes include mammalian tyrosyl tRNA-synthetase, which shows
negative cooperativity,[33] and bacterial aspartate_transcarbamoylase[34] and
phosphofructokinase,[35] which show positive cooperativity.
Cooperativity is surprisingly common and can help regulate the responses of
enzymes to changes in the concentrations of their substrates. Positive
cooperativity makes enzymes much more sensitive to [S] and their activities can
show large changes over a narrow range of substrate concentration. Conversely,
negative cooperativity makes enzymes insensitive to small changes in [S].
The Hill_equation_(biochemistry)[36] is often used to describe the degree of
cooperativity quantitatively in non-MichaelisâMenten kinetics. The derived
Hill coefficient n measures how much the binding of substrate to one active
site affects the binding of substrate to the other active sites. A Hill
coefficient of <1 indicates negative cooperativity and a coefficient of >1
indicates positive cooperativity.
***** Pre-steady-state kinetics[edit] *****
Pre-steady state progress curve, showing the burst phase of an enzyme reaction.
In the first moment after an enzyme is mixed with substrate, no product has
been formed and no intermediates exist. The study of the next few milliseconds
of the reaction is called pre-steady-state kinetics. Pre-steady-state kinetics
is therefore concerned with the formation and consumption of enzymeâsubstrate
intermediates (such as ES or E*) until their steady-state_concentrations are
reached.
This approach was first applied to the hydrolysis reaction catalysed by
chymotrypsin.[37] Often, the detection of an intermediate is a vital piece of
evidence in investigations of what mechanism an enzyme follows. For example, in
the pingâpong mechanisms that are shown above, rapid kinetic measurements can
follow the release of product P and measure the formation of the modified
enzyme intermediate E*.[38] In the case of chymotrypsin, this intermediate is
formed by an attack on the substrate by the nucleophilic serine in the active
site and the formation of the acyl-enzyme intermediate.
In the figure to the right, the enzyme produces E* rapidly in the first few
seconds of the reaction. The rate then slows as steady state is reached. This
rapid burst phase of the reaction measures a single turnover of the enzyme.
Consequently, the amount of product released in this burst, shown as the
intercept on the y-axis of the graph, also gives the amount of functional
enzyme which is present in the assay.[39]
***** Chemical mechanism[edit] *****
An important goal of measuring enzyme kinetics is to determine the chemical
mechanism of an enzyme reaction, i.e., the sequence of chemical steps that
transform substrate into product. The kinetic approaches discussed above will
show at what rates intermediates are formed and inter-converted, but they
cannot identify exactly what these intermediates are.
Kinetic measurements taken under various solution conditions or on slightly
modified enzymes or substrates often shed light on this chemical mechanism, as
they reveal the rate-determining step or intermediates in the reaction. For
example, the breaking of a covalent_bond to a hydrogen atom is a common rate-
determining step. Which of the possible hydrogen transfers is rate determining
can be shown by measuring the kinetic effects of substituting each hydrogen by
deuterium, its stable isotope. The rate will change when the critical hydrogen
is replaced, due to a primary kinetic_isotope_effect, which occurs because
bonds to deuterium are harder to break than bonds to hydrogen.[40] It is also
possible to measure similar effects with other isotope substitutions, such as
13C/12C and 18O/16O, but these effects are more subtle.[41]
Isotopes can also be used to reveal the fate of various parts of the substrate
molecules in the final products. For example, it is sometimes difficult to
discern the origin of an oxygen atom in the final product; since it may have
come from water or from part of the substrate. This may be determined by
systematically substituting oxygen's stable isotope 18O into the various
molecules that participate in the reaction and checking for the isotope in the
product.[42] The chemical mechanism can also be elucidated by examining the
kinetics and isotope effects under different pH conditions,[43] by altering the
metal ions or other bound cofactors,[44] by site-directed_mutagenesis of
conserved amino acid residues, or by studying the behaviour of the enzyme in
the presence of analogues of the substrate(s).[45]
***** Enzyme inhibition and activation[edit] *****
Main article: Enzyme_inhibitor
Kinetic scheme for reversible enzyme inhibitors.
Enzyme inhibitors are molecules that reduce or abolish enzyme activity, while
enzyme activators are molecules that increase the catalytic rate of enzymes.
These interactions can be either reversible (i.e., removal of the inhibitor
restores enzyme activity) or irreversible (i.e., the inhibitor permanently
inactivates the enzyme).
**** Reversible inhibitors[edit] ****
Traditionally reversible enzyme inhibitors have been classified as competitive,
uncompetitive, or non-competitive, according to their effects on KM and Vmax.
These different effects result from the inhibitor binding to the enzyme E, to
the enzymeâsubstrate complex ES, or to both, respectively. The division of
these classes arises from a problem in their derivation and results in the need
to use two different binding constants for one binding event. The binding of an
inhibitor and its effect on the enzymatic activity are two distinctly different
things, another problem the traditional equations fail to acknowledge. In
noncompetitive inhibition the binding of the inhibitor results in 100%
inhibition of the enzyme only, and fails to consider the possibility of
anything in between.[46] In noncompetitive inhibition, the inhibitor will bind
to an enzyme at its allosteric site; therefore, the binding affinity, or
inverse of KM, of the substrate with the enzyme will remain the same. On the
other hand, the Vmax will decrease relative to an uninhibited enzyme. On a
Lineweaver-Burk plot, the presence of a noncompetitive inhibitor is illustrated
by a change in the y-intercept, defined as 1/Vmax. The x-intercept, defined as
â1/KM, will remain the same. In competitive inhibition, the inhibitor will
bind to an enzyme at the active site, competing with the substrate. As a
result, the KM will increase and the Vmax will remain the same.[47] The common
form of the inhibitory term also obscures the relationship between the
inhibitor binding to the enzyme and its relationship to any other binding term
be it the MichaelisâMenten equation or a dose response curve associated with
ligand receptor binding. To demonstrate the relationship the following
rearrangement can be made:
                  V  max            1 +         [ I ]           K  i
      =          V  max                   [ I ] +  K  i             K  i
      {\displaystyle {\cfrac {V_{\max }}{1+{\cfrac {[I]}{K_{i}}}}}={\cfrac {V_
      {\max }}{\cfrac {[I]+K_{i}}{K_{i}}}}}  [{\displaystyle {\cfrac {V_{\max
      }}{1+{\cfrac {[I]}{K_{i}}}}}={\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{K_
      {i}}}}}]
Adding zero to the bottom ([I]-[I])
                  V  max                   [ I ] +  K  i            [ I ] +  K
      i   &#x2212; [ I ]            {\displaystyle {\cfrac {V_{\max }}{\cfrac {
      [I]+K_{i}}{[I]+K_{i}-[I]}}}}  [{\cfrac {V_{\max }}{\cfrac {[I]+K_{i}}{
      [I]+K_{i}-[I]}}}]
Dividing by [I]+Ki
                  V  max                   1          1 &#x2212;         [ I ]
      [ I ] +  K  i                 =  V  max   &#x2212;  V  max           [ I
      ]          [ I ] +  K  i          {\displaystyle {\cfrac {V_{\max }}
      {\cfrac {1}{1-{\cfrac {[I]}{[I]+K_{i}}}}}}=V_{\max }-V_{\max }{\cfrac {
      [I]}{[I]+K_{i}}}}  [{\displaystyle {\cfrac {V_{\max }}{\cfrac {1}{1-
      {\cfrac {[I]}{[I]+K_{i}}}}}}=V_{\max }-V_{\max }{\cfrac {[I]}{[I]+K_
      {i}}}}]
This notation demonstrates that similar to the MichaelisâMenten equation,
where the rate of reaction depends on the percent of the enzyme population
interacting with substrate, the effect of the inhibitor is a result of the
percent of the enzyme population interacting with inhibitor. The only problem
with this equation in its present form is that it assumes absolute inhibition
of the enzyme with inhibitor binding, when in fact there can be a wide range of
effects anywhere from 100% inhibition of substrate turn over to just >0%. To
account for this the equation can be easily modified to allow for different
degrees of inhibition by including a delta Vmax term.
          V  max   &#x2212; &#x0394;  V  max           [ I ]          [ I ] +
      K  i          {\displaystyle V_{\max }-\Delta V_{\max }{\cfrac {[I]}{
      [I]+K_{i}}}}  [V_{\max }-\Delta V_{\max }{\cfrac {[I]}{[I]+K_{i}}}]
or
          V  max 1   &#x2212; (  V  max 1   &#x2212;  V  max 2   )         [ I
      ]          [ I ] +  K  i          {\displaystyle V_{\max 1}-(V_{\max 1}-
      V_{\max 2}){\cfrac {[I]}{[I]+K_{i}}}}  [{\displaystyle V_{\max 1}-(V_
      {\max 1}-V_{\max 2}){\cfrac {[I]}{[I]+K_{i}}}}]
This term can then define the residual enzymatic activity present when the
inhibitor is interacting with individual enzymes in the population. However the
inclusion of this term has the added value of allowing for the possibility of
activation if the secondary Vmax term turns out to be higher than the initial
term. To account for the possibly of activation as well the notation can then
be rewritten replacing the inhibitor "I" with a modifier term denoted here as
"X".
          V  max 1   &#x2212; (  V  max 1   &#x2212;  V  max 2   )         [ X
      ]          [ X ] +  K  x          {\displaystyle V_{\max 1}-(V_{\max 1}-
      V_{\max 2}){\cfrac {[X]}{[X]+K_{x}}}}  [{\displaystyle V_{\max 1}-(V_
      {\max 1}-V_{\max 2}){\cfrac {[X]}{[X]+K_{x}}}}]
While this terminology results in a simplified way of dealing with kinetic
effects relating to the maximum velocity of the MichaelisâMenten equation, it
highlights potential problems with the term used to describe effects relating
to the KM. The KM relating to the affinity of the enzyme for the substrate
should in most cases relate to potential changes in the binding site of the
enzyme which would directly result from enzyme inhibitor interactions. As such
a term similar to the one proposed above to modulate Vmax should be appropriate
in most situations:[48][49]
          K  m 1   &#x2212; (  K  m 1   &#x2212;  K  m 2   )         [ X ]
      [ X ] +  K  x          {\displaystyle K_{m1}-(K_{m1}-K_{m2}){\cfrac {[X]}
      {[X]+K_{x}}}}  [{\displaystyle K_{m1}-(K_{m1}-K_{m2}){\cfrac {[X]}{[X]+K_
      {x}}}}]
A few examples of reversible inhibition belonging to the competitive and
uncompetitive models have been discussed in the following papers.[50][51][52]
**** Irreversible inhibitors[edit] ****
Enzyme inhibitors can also irreversibly inactivate enzymes, usually by
covalently modifying active site residues. These reactions, which may be called
suicide substrates, follow exponential_decay functions and are usually
saturable. Below saturation, they follow first_order kinetics with respect to
inhibitor. Irreversible inhibition could be classified into two distinct types.
Affinity labelling is a type of irreversible inhibition where a functional
group that is highly reactive modifies a catalytically critical residue on the
protein of interest to bring about inhibition. Mechanism-based inhibition, on
the other hand, involves binding of the inhibitor followed by enzyme mediated
alterations that transform the latter into a reactive group that irreversibly
modifies the enzyme.
**** Philosophical discourse on reversibility and irreversibility of inhibition
[edit] ****
Having discussed reversible inhibition and irreversible inhibition in the above
two headings, it would have to be pointed out that the concept of reversibility
( or irreversibility) is a purely theoretical construct exclusively dependent
on the time-frame of the assay, i.e., a reversible assay involving association
and dissociation of the inhibitor molecule in the minute timescales would seem
irreversible if an assay assess the outcome in the seconds and vice versa.
There is a continuum of inhibitor behaviors spanning reversibility and
irreversibility at a given non-arbitrary assay time frame. There are inhibitors
that show slow-onset behavior[50] and most of these inhibitors, invariably,
also show tight-binding to the protein target of interest.[50][51]
***** Mechanisms of catalysis[edit] *****
Main article: Enzyme_catalysis
The energy variation as a function of reaction_coordinate shows the
stabilisation of the transition state by an enzyme.
The favoured model for the enzymeâsubstrate interaction is the induced fit
model.[53] This model proposes that the initial interaction between enzyme and
substrate is relatively weak, but that these weak interactions rapidly induce
conformational_changes in the enzyme that strengthen binding. These
conformational changes also bring catalytic residues in the active site close
to the chemical bonds in the substrate that will be altered in the reaction.
[54] Conformational changes can be measured using circular_dichroism or dual
polarisation_interferometry. After binding takes place, one or more mechanisms
of catalysis lower the energy of the reaction's transition_state by providing
an alternative chemical pathway for the reaction. Mechanisms of catalysis
include catalysis by bond strain; by proximity and orientation; by active-site
proton donors or acceptors; covalent catalysis and quantum_tunnelling.[38][55]
Enzyme kinetics cannot prove which modes of catalysis are used by an enzyme.
However, some kinetic data can suggest possibilities to be examined by other
techniques. For example, a pingâpong mechanism with burst-phase pre-steady-
state kinetics would suggest covalent catalysis might be important in this
enzyme's mechanism. Alternatively, the observation of a strong pH effect on
Vmax but not KM might indicate that a residue in the active site needs to be in
a particular ionisation state for catalysis to occur.
***** History[edit] *****
In 1902 Victor_Henri proposed a quantitative theory of enzyme kinetics,[56] but
at the time the experimental significance of the hydrogen_ion_concentration was
not yet recognized. After Peter_Lauritz_SÃ¸rensen had defined the logarithmic
pH-scale and introduced the concept of buffering in 1909[57] the German chemist
Leonor_Michaelis and Dr. Maud_Leonora_Menten (a postdoctoral researcher in
Michaelis's lab at the time)repeated Henri's experiments and confirmed his
equation, which is now generally referred to as Michaelis-Menten_kinetics
(sometimes also Henri-Michaelis-Menten kinetics).[58] Their work was further
developed by G._E._Briggs and J._B._S._Haldane, who derived kinetic equations
that are still widely considered today a starting point in modeling enzymatic
activity.[59]
The major contribution of the Henri-Michaelis-Menten approach was to think of
enzyme reactions in two stages. In the first, the substrate binds reversibly to
the enzyme, forming the enzyme-substrate complex. This is sometimes called the
Michaelis complex. The enzyme then catalyzes the chemical step in the reaction
and releases the product. The kinetics of many enzymes is adequately described
by the simple Michaelis-Menten model, but all enzymes have internal_motions
that are not accounted for in the model and can have significant contributions
to the overall reaction kinetics. This can be modeled by introducing several
Michaelis-Menten pathways that are connected with fluctuating rates,[60][61]
[62] which is a mathematical extension of the basic Michaelis Menten mechanism.
[63]
***** Software[edit] *****
**** ENZO[edit] ****
ENZO (Enzyme Kinetics) is a graphical interface tool for building kinetic
models of enzyme catalyzed reactions. ENZO automatically generates the
corresponding differential equations from a stipulated enzyme reaction scheme.
These differential equations are processed by a numerical solver and a
regression algorithm which fits the coefficients of differential equations to
experimentally observed time course curves. ENZO allows rapid evaluation of
rival reaction schemes and can be used for routine tests in enzyme kinetics.
[64]
***** See also[edit] *****
    * Protein_dynamics
    * Diffusion_limited_enzyme
    * Langmuir_adsorption_model
***** Footnotes[edit] *****
      Î±. ^ Link:_Interactive_MichaelisâMenten_kinetics_tutorial_(Java
      required)
      Î². ^ Link:_dihydrofolate_reductase_mechanism_(Gif)
      Î³. ^ Link:_DNA_polymerase_mechanism_(Gif)
      Î´. ^ Link:_Chymotrypsin_mechanism_(Flash_required)
***** References[edit] *****
   1. ^Wrighton MS, Ebbing DD (1993). General chemistry (4th ed.). Boston:
      Houghton Mifflin. ISBN 978-0-395-63696-1.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a b Fromm H.J., Hargrove M.S. (2012) Enzyme Kinetics. In: Essentials of
      Biochemistry. Springer, Berlin, Heidelberg
   4. ^Danson M, Eisenthal R (2002). Enzyme assays: a practical approach.
      Oxford [Oxfordshire]: Oxford University Press. ISBN 978-0-19-963820-8.
   5. ^Xie XS, Lu HP (June 1999). "Single-molecule enzymology". The Journal of
      Biological Chemistry. 274 (23): 15967â70. doi:10.1074/jbc.274.23.15967.
      PMID 10347141.
   6. ^Lu HP (June 2004). "Single-molecule spectroscopy studies of
      conformational change dynamics in enzymatic reactions". Current
      Pharmaceutical Biotechnology. 5 (3): 261â9. doi:10.2174/
      1389201043376887. PMID 15180547.
   7. ^Schnell JR, Dyson_HJ, Wright PE (2004). "Structure, dynamics, and
      catalytic function of dihydrofolate reductase". Annual Review of
      Biophysics and Biomolecular Structure. 33: 119â40. doi:10.1146/
      annurev.biophys.33.110502.133613. PMID 15139807.
   8. ^Gibson QH (1969). "[6] Rapid mixing: Stopped flow". Rapid mixing:
      Stopped flow. Methods in Enzymology. 16. pp. 187â228. doi:10.1016/
      S0076-6879(69)16009-7. ISBN 978-0-12-181873-9.
   9. ^Duggleby RG (1995). "[3] Analysis of enzyme progress curves by nonlinear
      regression". Analysis of enzyme progress curves by non-linear regression.
      Methods in Enzymology. 249. pp. 61â90. doi:10.1016/0076-6879(95)49031-
      0. ISBN 978-0-12-182150-0. PMID 7791628.
  10. ^Murray JB, Dunham CM, Scott WG (January 2002). "A pH-dependent
      conformational change, rather than the chemical step, appears to be rate-
      limiting in the hammerhead ribozyme cleavage reaction". Journal of
      Molecular Biology. 315 (2): 121â30. doi:10.1006/jmbi.2001.5145.
      PMID 11779233.
  11. ^ Michaelis L. and Menten M.L. Kinetik der Invertinwirkung Biochem. Z.
      1913; 49:333â369 English_translation Accessed 6 April 2007
  12. ^Stroppolo ME, Falconi M, Caccuri AM, Desideri A (September 2001).
      "Superefficient enzymes". Cellular and Molecular Life Sciences. 58 (10):
      1451â60. doi:10.1007/PL00000788. PMID 11693526.
  13. ^Bar-Even A, Noor E, Savir Y, Liebermeister W, Davidi D, Tawfik DS, Milo
      R (May 2011). "The moderately efficient enzyme: evolutionary and
      physicochemical trends shaping enzyme parameters". Biochemistry. 50 (21):
      4402â10. doi:10.1021/bi2002289. PMID 21506553.
  14. ^Walsh R, Martin E, Darvesh S (January 2010). "A method to describe
      enzyme-catalyzed reactions by combining steady state and time course
      enzyme kinetic parameters". Biochimica et Biophysica Acta. 1800 (1):
      1â5. doi:10.1016/j.bbagen.2009.10.007. PMID 19840832.
  15. ^Beal SL (December 1983). "Computation of the explicit solution to the
      Michaelis-Menten equation". Journal of Pharmacokinetics and
      Biopharmaceutics. 11 (6): 641â57. doi:10.1007/BF01059062. PMID 6689584.
  16. ^Schnell S, Mendoza C (1997). "Closed Form Solution for Time-dependent
      Enzyme Kinetics". Journal of Theoretical Biology. 187 (2): 207â212.
      doi:10.1006/jtbi.1997.0425.
  17. ^Goudar CT, Sonnad JR, Duggleby RG (January 1999). "Parameter_estimation
      using_a_direct_solution_of_the_integrated_Michaelis-Menten_equation"
      (PDF). Biochimica et Biophysica Acta. 1429 (2): 377â83. doi:10.1016/
      s0167-4838(98)00247-7. PMID 9989222. Archived from the_original (PDF) on
      9 November 2015.
  18. ^Goudar CT, Harris SK, McInerney MJ, Suflita JM (December 2004).
      "Progress curve analysis for enzyme and microbial kinetic reactions using
      explicit solutions based on the Lambert W function". Journal of
      Microbiological Methods. 59 (3): 317â26. doi:10.1016/
      j.mimet.2004.06.013. PMID 15488275.
  19. ^Berberan-Santos MN (2010). "A_General_Treatment_of_Henri_Michaelis
      Menten_Enzyme_Kinetics:_Exact_Series_Solution_and_Approximate_Analytical
      Solutions" (PDF). MATCH Communications in Mathematical and in Computer
      Chemistry. 63: 283.
  20. ^Jones ME (December 1992). "Analysis_of_algebraic_weighted_least-squares
      estimators_for_enzyme_parameters". The Biochemical Journal. 288 ( Pt 2)
      (Pt 2): 533â8. doi:10.1042/bj2880533. PMC 1132043. PMID 1463456.
  21. ^Tseng SJ, Hsu JP (August 1990). "A comparison of the parameter
      estimating procedures for the Michaelis-Menten model". Journal of
      Theoretical Biology. 145 (4): 457â64. doi:10.1016/S0022-5193(05)80481-
      3. PMID 2246896.
  22. ^Bravo IG, Busto F, De Arriaga D, Ferrero MA, RodrÃ­guez-Aparicio LB,
      MartÃ­nez-Blanco H, Reglero A (September 2001). "A_normalized_plot_as_a
      novel_and_time-saving_tool_in_complex_enzyme_kinetic_analysis". The
      Biochemical Journal. 358 (Pt 3): 573â83. doi:10.1042/bj3580573.
      PMC 1222113. PMID 11577687.
  23. ^Almaas E, KovÃ¡cs B, Vicsek T, Oltvai ZN, BarabÃ¡si AL (February 2004).
      "Global organization of metabolic fluxes in the bacterium Escherichia
      coli". Nature. 427 (6977): 839â43. arXiv:q-bio/0403001. Bibcode:
      2004Natur.427..839A. doi:10.1038/nature02289. PMID 14985762.
  24. ^Reed JL, Vo TD, Schilling CH, Palsson BO (2003). "An_expanded_genome-
      scale_model_of_Escherichia_coli_K-12_(iJR904_GSM/GPR)". Genome Biology. 4
      (9): R54. doi:10.1186/gb-2003-4-9-r54. PMC 193654. PMID 12952533.
  25. ^ for a complete derivation, see here
  26. ^Dirr H, Reinemer P, Huber R (March 1994). "X-ray crystal structures of
      cytosolic glutathione S-transferases. Implications for protein
      architecture, substrate recognition and catalytic function". European
      Journal of Biochemistry. 220 (3): 645â61. doi:10.1111/j.1432-
      1033.1994.tb18666.x. PMID 8143720.
  27. ^Stone SR, Morrison JF (July 1988). "Dihydrofolate reductase from
      Escherichia coli: the kinetic mechanism with NADPH and reduced
      acetylpyridine adenine dinucleotide phosphate as substrates".
      Biochemistry. 27 (15): 5493â9. doi:10.1021/bi00415a016. PMID 3052577.
  28. ^Fisher PA (1994). Enzymologic mechanism of replicative DNA polymerases
      in higher eukaryotes. Progress in Nucleic Acid Research and Molecular
      Biology. 47. pp. 371â97. doi:10.1016/S0079-6603(08)60257-3. ISBN 978-0-
      12-540047-3. PMID 8016325.
  29. ^Akerman SE, MÃ¼ller S (August 2003). "2-Cys peroxiredoxin PfTrx-Px1 is
      involved in the antioxidant defence of Plasmodium falciparum". Molecular
      and Biochemical Parasitology. 130 (2): 75â81. doi:10.1016/S0166-6851
      (03)00161-0. PMID 12946843.
  30. ^Bravo IG, Barrallo S, Ferrero MA, RodrÃ­guez-Aparicio LB, MartÃ­nez-
      Blanco H, Reglero A (September 2001). "Kinetic_properties_of_the
      acylneuraminate_cytidylyltransferase_from_Pasteurella_haemolytica_A2".
      The Biochemical Journal. 358 (Pt 3): 585â98. doi:10.1042/bj3580585.
      PMC 1222114. PMID 11577688.
  31. ^Kraut J (1977). "Serine proteases: structure and mechanism of
      catalysis". Annual Review of Biochemistry. 46: 331â58. doi:10.1146/
      annurev.bi.46.070177.001555. PMID 332063.
  32. ^Cornish-Bowden A (2004). Fundamentals of Enzyme Kinetics. Portland
      Press. Some enzymes are much more effective catalysts for one direction
      than the other. As a striking example, the limiting rates of the forward
      reaction catalyzed by methionine adenosyltransferase is about 105 greater
      than that for the reverse direction, even though the equilibrium constant
      is close to unity (page 53).
  33. ^Ricard J, Cornish-Bowden A (July 1987). "Co-operative and allosteric
      enzymes: 20 years on". European Journal of Biochemistry. 166 (2):
      255â72. doi:10.1111/j.1432-1033.1987.tb13510.x. PMID 3301336.
  34. ^Ward WH, Fersht AR (July 1988). "Tyrosyl-tRNA synthetase acts as an
      asymmetric dimer in charging tRNA. A rationale for half-of-the-sites
      activity". Biochemistry. 27 (15): 5525â30. doi:10.1021/bi00415a021.
      PMID 3179266.
  35. ^Helmstaedt K, Krappmann S, Braus GH (September 2001). "Allosteric
      regulation_of_catalytic_activity:_Escherichia_coli_aspartate
      transcarbamoylase_versus_yeast_chorismate_mutase". Microbiology and
      Molecular Biology Reviews. 65 (3): 404â21, table of contents. doi:
      10.1128/MMBR.65.3.404-421.2001. PMC 99034. PMID 11528003.
  36. ^Schirmer T, Evans PR (January 1990). "Structural basis of the allosteric
      behaviour of phosphofructokinase". Nature. 343 (6254): 140â5. Bibcode:
      1990Natur.343..140S. doi:10.1038/343140a0. PMID 2136935.
  37. ^Hill AV (1910). "The possible effects of the aggregation of the
      molecules of haemoglobin on its dissociation curves". J. Physiol. 40:
      ivâvii.
  38. ^Hartley BS, Kilby BA (February 1954). "The_reaction_of_p-nitrophenyl
      esters_with_chymotrypsin_and_insulin". The Biochemical Journal. 56 (2):
      288â97. doi:10.1042/bj0560288. PMC 1269615. PMID 13140189.
  39. ^ a bFersht, Alan (1999). Structure and mechanism in protein science: a
      guide to enzyme catalysis and protein folding. San Francisco: W.H.
      Freeman. ISBN 978-0-7167-3268-6.
  40. ^Bender ML, BeguÃ©-CantÃ³n ML, Blakeley RL, Brubacher LJ, Feder J, Gunter
      CR, KÃ©zdy FJ, Killheffer JV, Marshall TH, Miller CG, Roeske RW, Stoops
      JK (December 1966). "The determination of the concentration of hydrolytic
      enzyme solutions: alpha-chymotrypsin, trypsin, papain, elastase,
      subtilisin, and acetylcholinesterase". Journal of the American Chemical
      Society. 88 (24): 5890â913. doi:10.1021/ja00976a034. PMID 5980876.
  41. ^Cleland WW (January 2005). "The use of isotope effects to determine
      enzyme mechanisms". Archives of Biochemistry and Biophysics. 433 (1):
      2â12. doi:10.1016/j.abb.2004.08.027. PMID 15581561.
  42. ^Northrop DB (1981). "The expression of isotope effects on enzyme-
      catalyzed reactions". Annual Review of Biochemistry. 50: 103â31. doi:
      10.1146/annurev.bi.50.070181.000535. PMID 7023356.
  43. ^Baillie TA, Rettenmeier AW (1986). "Drug biotransformation: mechanistic
      studies with stable isotopes". Journal of Clinical Pharmacology. 26 (6):
      448â51. doi:10.1002/j.1552-4604.1986.tb03556.x. PMID 3734135.
  44. ^Cleland WW (1982). "Use of isotope effects to elucidate enzyme
      mechanisms". CRC Critical Reviews in Biochemistry. 13 (4): 385â428.
      doi:10.3109/10409238209108715. PMID 6759038.
  45. ^Christianson DW, Cox JD (1999). "Catalysis by metal-activated hydroxide
      in zinc and manganese metalloenzymes". Annual Review of Biochemistry. 68:
      33â57. doi:10.1146/annurev.biochem.68.1.33. PMID 10872443.
  46. ^Kraut DA, Carroll KS, Herschlag D (2003). "Challenges in enzyme
      mechanism and energetics". Annual Review of Biochemistry. 72: 517â71.
      doi:10.1146/annurev.biochem.72.121801.161617. PMID 12704087.
  47. ^Walsh R, Martin E, Darvesh S (December 2011). "Limitations of
      conventional inhibitor classifications". Integrative Biology. 3 (12):
      1197â201. doi:10.1039/c1ib00053e. PMID 22038120.
  48. ^Cleland WW (February 1963). "The kinetics of enzyme-catalyzed reactions
      with two or more substrates or products. III. Prediction of initial
      velocity and inhibition patterns by inspection". Biochimica et Biophysica
      Acta. 67: 188â96. doi:10.1016/0006-3002(63)91816-x. PMID 14021669.
  49. ^Walsh R, Martin E, Darvesh S (May 2007). "A versatile equation to
      describe reversible enzyme inhibition and activation kinetics: modeling
      beta-galactosidase and butyrylcholinesterase". Biochimica et Biophysica
      Acta. 1770 (5): 733â46. doi:10.1016/j.bbagen.2007.01.001.
      PMID 17307293.
  50. ^Walsh, Ryan (2012). "Ch._17._Alternative_Perspectives_of_Enzyme_Kinetic
      Modeling" (PDF). In Ekinci, Deniz (ed.). Medicinal Chemistry and Drug
      Design. InTech. pp. 357â371. ISBN 978-953-51-0513-8.
  51. ^ a b cSrinivasan B, Skolnick J (May 2015). "Insights_into_the_slow-onset
      tight-binding_inhibition_of_Escherichia_coli_dihydrofolate_reductase:
      detailed_mechanistic_characterization_of_pyrrolo_[3,2-f]_quinazoline-1,3-
      diamine_and_its_derivatives_as_novel_tight-binding_inhibitors". The FEBS
      Journal. 282 (10): 1922â38. doi:10.1111/febs.13244. PMC 4445455.
      PMID 25703118.
  52. ^ a bSrinivasan B, Tonddast-Navaei S, Skolnick J (October 2015). "Ligand
      binding_studies,_preliminary_structure-activity_relationship_and_detailed
      mechanistic_characterization_of_1-phenyl-6,6-dimethyl-1,3,5-triazine-2,4-
      diamine_derivatives_as_inhibitors_of_Escherichia_coli_dihydrofolate
      reductase". European Journal of Medicinal Chemistry. 103: 600â14. doi:
      10.1016/j.ejmech.2015.08.021. PMC 4610388. PMID 26414808.
  53. ^Srinivasan B, Rodrigues JV, Tonddast-Navaei S, Shakhnovich E, Skolnick J
      (July 2017). "Rational_Design_of_Novel_Allosteric_Dihydrofolate_Reductase
      Inhibitors_Showing_Antibacterial_Effects_on_Drug-Resistant_Escherichia
      coli_Escape_Variants". ACS Chemical Biology. 12 (7): 1848â1857. doi:
      10.1021/acschembio.7b00175. PMC 5819740. PMID 28525268.
  54. ^Koshland DE (February 1958). "Application_of_a_Theory_of_Enzyme
      Specificity_to_Protein_Synthesis". Proceedings of the National Academy of
      Sciences of the United States of America. 44 (2): 98â104. Bibcode:
      1958PNAS...44...98K. doi:10.1073/pnas.44.2.98. PMC 335371. PMID 16590179.
  55. ^Hammes GG (July 2002). "Multiple conformational changes in enzyme
      catalysis". Biochemistry. 41 (26): 8221â8. doi:10.1021/bi0260839.
      PMID 12081470.
  56. ^Sutcliffe MJ, Scrutton NS (July 2002). "A_new_conceptual_framework_for
      enzyme_catalysis._Hydrogen_tunnelling_coupled_to_enzyme_dynamics_in
      flavoprotein_and_quinoprotein_enzymes". European Journal of Biochemistry.
      269 (13): 3096â102. doi:10.1046/j.1432-1033.2002.03020.x.
      PMID 12084049.
  57. ^Henri V (1902). "Theorie generale de l'action de quelques diastases".
      Compt. Rend. Acad. Sci. Paris. 135: 916â9.
  58. ^SÃ¸rensen PL (1909). "Enzymstudien {II}. Ãber die Messung und Bedeutung
      der Wasserstoffionenkonzentration bei enzymatischen Prozessen" [Enzyme
      studies III: About the measurement and significance of the hydrogen ion
      concentration in enzymatic processes]. Biochem. Z. (in German). 21:
      131â304.
  59. ^Michaelis L, Menten M (1913). "Die Kinetik der Invertinwirkung" [The
      Kinetics of Invertase Action]. Biochem. Z. (in German). 49: 333â369.
  60. ;Michaelis L, Menten ML, Johnson KA, Goody RS (October 2011). "The
      original_Michaelis_constant:_translation_of_the_1913_Michaelis-Menten
      paper". Biochemistry. 50 (39): 8264â9. doi:10.1021/bi201284u.
      PMC 3381512. PMID 21888353.
  61. ^Briggs GE, Haldane JB (1925). "A_Note_on_the_Kinetics_of_Enzyme_Action".
      The Biochemical Journal. 19 (2): 338â9. doi:10.1042/bj0190338.
      PMC 1259181. PMID 16743508.
  62. ^Flomenbom O, Velonia K, Loos D, Masuo S, Cotlet M, Engelborghs Y,
      Hofkens J, Rowan AE, Nolte RJ, Van der Auweraer M, de Schryver FC,
      Klafter J (February 2005). "Stretched_exponential_decay_and_correlations
      in_the_catalytic_activity_of_fluctuating_single_lipase_molecules".
      Proceedings of the National Academy of Sciences of the United States of
      America. 102 (7): 2368â72. Bibcode:2005PNAS..102.2368F. doi:10.1073/
      pnas.0409039102. PMC 548972. PMID 15695587.
  63. ^English BP, Min W, van Oijen AM, Lee KT, Luo G, Sun H, Cherayil BJ, Kou
      SC, Xie XS (February 2006). "Ever-fluctuating single enzyme molecules:
      Michaelis-Menten equation revisited". Nature Chemical Biology. 2 (2):
      87â94. doi:10.1038/nchembio759. PMID 16415859.
  64. ^Lu HP, Xun L, Xie XS (December 1998). "Single-molecule enzymatic
      dynamics". Science. 282 (5395): 1877â82. doi:10.1126/
      science.282.5395.1877. PMID 9836635.
  65. ^Xue X, Liu F, Ou-Yang ZC (September 2006). "Single molecule Michaelis-
      Menten equation beyond quasistatic disorder". Physical Review E. 74 (3 Pt
      1): 030902. arXiv:cond-mat/0604364. Bibcode:2006PhRvE..74c0902X. doi:
      10.1103/PhysRevE.74.030902. PMID 17025584.
  66. ^Bevc S, Konc J, Stojan J, HodoÅ¡Äek M, Penca M, Praprotnik M, JaneÅ¾iÄ
      D (2011). "ENZO:_a_web_tool_for_derivation_and_evaluation_of_kinetic
      models_of_enzyme_catalyzed_reactions". PLOS One. 6 (7): e22265. Bibcode:
      2011PLoSO...622265B. doi:10.1371/journal.pone.0022265. PMC 3139599.
      PMID 21818304.
  67.  ENZO_server
***** Further reading[edit] *****
Introductory
    * Cornish-Bowden, Athel (2004). Fundamentals of enzyme kinetics (3rd ed.).
      London: Portland Press. ISBN 978-1-85578-158-0.
Stevens L, Price NC (1999). Fundamentals of enzymology: the cell and molecular
biology of catalytic proteins. Oxford [Oxfordshire]: Oxford University Press.
ISBN 978-0-19-850229-6.
Bugg, Tim (2004). Introduction to Enzyme and Coenzyme Chemistry. Cambridge, MA:
Blackwell Publishers. ISBN 978-1-4051-1452-3.
Advanced
    * Segel, Irwin H. (1993). Enzyme kinetics: behavior and analysis of rapid
      equilibrium and steady state enzyme systems (New ed.). New York: Wiley.
      ISBN 978-0-471-30309-1.
Fersht, Alan (1999). Structure and mechanism in protein science: a guide to
enzyme catalysis and protein folding. San Francisco: W.H. Freeman. ISBN 978-0-
7167-3268-6.
Schnell S, Maini PK (2004). "A_century_of_enzyme_kinetics:_Reliability_of_the
KM_and_vmax_estimates" (PDF). Comments on Theoretical Biology. 8 (2â3):
169â87. CiteSeerX 10.1.1.493.7178. doi:10.1080/08948550302453. Archived from
the_original (PDF) on 21 February 2006.
Walsh, Christopher (1979). Enzymatic reaction mechanisms. San Francisco: W. H.
Freeman. ISBN 978-0-7167-0070-8.
Cleland WW, Cook P (2007). Enzyme kinetics and mechanism. New York: Garland
Science. ISBN 978-0-8153-4140-6.
***** External links[edit] *****
    * Animation_of_an_enzyme_assay â Shows effects of manipulating assay
      conditions
    * MACiE â A database of enzyme reaction mechanisms
    * ENZYME â Expasy enzyme nomenclature database
    * ENZO â Web application for easy construction and quick testing of
      kinetic models of enzyme catalyzed reactions.
    * ExCatDB â A database of enzyme catalytic mechanisms
    * BRENDA â Comprehensive enzyme database, giving substrates, inhibitors
      and reaction diagrams
    * SABIO-RK â A database of reaction kinetics
    * Joseph_Kraut's_Research_Group,_University_of_California_San_Diego â
      Animations of several enzyme reaction mechanisms
    * Symbolism_and_Terminology_in_Enzyme_Kinetics â A comprehensive
      explanation of concepts and terminology in enzyme kinetics
    * An_introduction_to_enzyme_kinetics â An accessible set of on-line
      tutorials on enzyme kinetics
    * Enzyme_kinetics_animated_tutorial â An animated tutorial with audio
    * v
    * t
    * e
Enzymes
                   * Active_site
                   * Binding_site
                   * Catalytic_triad
                   * Oxyanion_hole
Activity           * Enzyme_promiscuity
                   * Catalytically_perfect_enzyme
                   * Coenzyme
                   * Cofactor
                   * Enzyme_catalysis
                   * Allosteric_regulation
Regulation         * Cooperativity
                   * Enzyme_inhibitor
                   * Enzyme_activator
                   * EC_number
Classification     * Enzyme_superfamily
                   * Enzyme_family
                   * List_of_enzymes
                   * Enzyme kinetics
                   * EadieâHofstee_diagram
Kinetics           * HanesâWoolf_plot
                   * LineweaverâBurk_plot
                   * MichaelisâMenten_kinetics
                   * EC1 Oxidoreductases (list)
                   * EC2 Transferases (list)
                   * EC3 Hydrolases (list)
Types              * EC4 Lyases (list)
                   * EC5 Isomerases (list)
                   * EC6 Ligases (list)
                   * EC7 Translocases (list)

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Enzyme_kinetics&oldid=908666171"
Categories:
    * Enzyme_kinetics
    * Catalysis
Hidden categories:
    * CS1:_long_volume_value
    * CS1_German-language_sources_(de)
    * Wikipedia_indefinitely_move-protected_pages
    * Use_dmy_dates_from_August_2018
    * Featured_articles
    * Pages_that_use_a_deprecated_format_of_the_chem_tags
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * íêµ­ì´
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Bahasa_Indonesia
    * ×¢××¨××ª
    * à²à²¨à³à²¨à²¡
    * Magyar
    * Nederlands
    * æ¥æ¬èª
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * ä¸­æ
Edit_links
    * This page was last edited on 31 July 2019, at 05:44 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
