The following text has been accessed from https://en.wikipedia.org/wiki/Microbial_genetics at Fri Aug 9 01:43:38 IST 2019
Creative_Commons_Attribution-ShareAlike_License




















****** Microbial genetics ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
Microbial genetics is a subject area within microbiology and genetic
engineering. Microbial genetics studies microorganisms for different purposes.
The microorganisms that are observed are bacteria, and archaea. Some fungi and
protozoa are also subjects used to study in this field. The studies of
microorganisms involve studies of genotype and expression system. Genotypes are
the inherited compositions of an organism. (Austin, "Genotype," n.d.) Genetic
Engineering is a field of work and study within microbial genetics.[1] The
usage of recombinant DNA technology is a process of this work.[1] The process
involves creating recombinant DNA molecules through manipulating a DNA
sequence.[1] That DNA created is then in contact with a host organism. Cloning
is also an example of genetic engineering.[1]
Since the discovery of microorganisms by Robert Hooke and Antoni van
Leeuwenhoek during the period 1665-1885[2] they have been used to study many
processes and have had applications in various areas of study in genetics. For
example: Microorganisms' rapid growth rates and short generation times are used
by scientists to study evolution. Robert Hooke and Antoni van Leeuwenhoek
discoveries involved depictions, observations, and descriptions of
microorganisms.[3] Mucor is the microfungus that Hooke presented and gave a
depiction of.[4] His contribution being, Mucor as the first microorganism to be
illustrated. Antoni van Leeuwenhoekâs contribution to the microscopic
protozoa and microscopic bacteria yielded to scientific observations and
descriptions.[4] These contributions were accomplished by a simple microscope,
which led to the understanding of microbes today and continues to progress
scientists understanding.  [5] Microbial genetics also has applications in
being able to study processes and pathways that are similar to those found in
humans such as drug metabolism.[6]
⁰
***** Contents *****
    * 1_Role_in_understanding_evolution
    * 2_Microorganisms_whose_study_is_encompassed_by_microbial_genetics
          o 2.1_Bacteria
          o 2.2_Archaea
          o 2.3_Fungi
          o 2.4_Protozoa
          o 2.5_Viruses
    * 3_Applications_of_microbial_genetics
    * 4_See_also
    * 5_References
***** Role in understanding evolution[edit] *****
Microbial genetics can focus on Charles Darwin's work and scientists have
continued to study his work and theories by the use of microbes.[7]
Specifically, Darwin's theory of natural selection is a source used. Studying
evolution by using microbial genetics involves scientists looking at
evolutionary balance.[1] An example of how they may accomplish this is studying
natural selection or drift of microbes.[7] Application of this knowledge comes
from looking for the presence or absence in a variety of different ways.[7] The
ways include identifying certain pathways, genes, and functions. Once the
subject is observed, scientist may compare it to a sequence of a conserved
gene.[1] The process of studying microbial evolution in this way lacks the
ability to give a time scale of when the evolution took place.[7] However, by
testing evolution in this way, scientist can learn the rates and outcomes of
evolution. Studying the relationship between microbes and the environment is a
key component to microbial genetics evolution.[8]
***** Microorganisms whose study is encompassed by microbial genetics[edit]
*****
**** Bacteria[edit] ****
Bacteria are classified by their shape.
Bacteria have been on this planet for approximately 3.5 billion years, and are
classified by their shape.[9] Bacterial_genetics studies the mechanisms of
their heritable information, their chromosomes, plasmids, transposons, and
phages.[10]
Gene transfer systems that have been extensively studied in bacteria include
genetic_transformation, conjugation and transduction. Natural_transformation is
a bacterial adaptation for DNA transfer between two cells through the
intervening medium. The uptake of donor DNA and its recombinational
incorporation into the recipient chromosome depends on the expression of
numerous bacterial genes whose products direct this process.[11][12] In
general, transformation is a complex, energy-requiring developmental process
that appears to be an adaptation for repairing DNA damage.[13]
Bacterial_conjugation is the transfer of genetic material between bacterial
cells by direct cell-to-cell contact or by a bridge-like connection between two
cells. Bacterial conjugation has been extensively studied in Escherichia_coli,
but also occurs in other bacteria such as Mycobacterium_smegmatis. Conjugation
requires stable and extended contact between a donor and a recipient strain, is
DNase resistant, and the transferred DNA is incorporated into the recipient
chromosome by homologous_recombination. E. coli conjugation is mediated by
expression of plasmid genes, whereas mycobacterial conjugation is mediated by
genes on the bacterial chromosome.[14]
Transduction is the process by which foreign DNA is introduced into a cell by a
virus or viral_vector. Transduction is a common tool used by molecular
biologists to stably introduce a foreign gene into a host cell's genome.
**** Archaea[edit] ****
Archaea is a domain of organisms that are prokaryotic, single-celled, and are
thought to have developed 4 billion years ago. "They have no cell nucleus or
any other organelles inside their cells."Archaea replicate asexually in a
process known as binary fission. The cell division cycle includes when
chromosomes of daughter cells replicate. Because archea have a singular
structure chromosome, the two daughter cells separate and cell divides. Archaea
have motility include with flagella, which is a tail like structure. Archaeal
chromosomes replicate from different origins of replication, producing two
haploid daughter cells.[15] "[16] They share a common ancestor with bacteria,
but are more closely related to eukaryotes in comparison to bacteria.[17] Some
Archaea are able to survive extreme environments, which leads to many
applications in the field of genetics. One of such applications is the use of
archaeal enzymes, which would be better able to survive harsh conditions in
vitro.[18]
Gene transfer and genetic exchange have been studied in the halophilic archaeon
Halobacterium_volcanii and the hyperthermophilic archaeons Sulfolobus
solfataricus and Sulfolobus_acidocaldarius. H. volcani forms cytoplasmic
bridges between cells that appear to be used for transfer of DNA from one cell
to another in either direction.[19] When S. solfataricus and S. acidocaldarius
are exposed to DNA damaging agents, species-specific cellular aggregation is
induced. Cellular aggregation mediates chromosomal marker exchange and genetic
recombination with high frequency. Cellular aggregation is thought to enhance
species specific DNA transfer between Sulfolobus cells in order to provide
increased repair of damaged DNA by means of homologous_recombination.[20][21]
[22] Archaea are divided into 3 subgroups which are halophiles, methanogens,
and thermoacidophiles. The first group, methanogens, are archaeabacteria that
live in swamps and marshes as well as in the gut of humans. They also play a
major role in decay and decomposition with dead organisms. Methanogens are
anaerobic organisms, which are killed when they are exposed to oxygen. The
second subgroup of archaeabacteria, halophiles are organisms that are present
in areas with high salt concentration like the Great Salt Lake and the Dead
Sea. The third subgroup thermoacidophiles also called thermophiles, are
organisms that live in acidic areas. They are present in area with low pH
levels like hot springs and geyers. Most thermophiles are found in the
Yellowstone National Park.[23]
Archaeal Genetics is the study of genes that consist of single nucleus-free
cells.[24] Archaea have a single, circular chromosomes that contain multiple
origins of replication for initiation of DNA synthesis.[25] DNA replication of
Archaea involves similar processes including initiation, elongation, and
termination. The primase used to synthesize a RNA primer varies than in
eukaryotes. The primase by archaea is highly derived version of RNA recognition
motif(RRM).[25] Archaea come from Gram positive bacteria, which both have a
single lipid bilayer, which are resistant to antibiotics. Archaea are similar
to mitochondria in eukaryotes in that they release energy as adenosine
triphosphate (ATP) through the chemical reaction called metabolism.[25] Some
archaea known as phototrophic archaea use the sunâs energy to produce ATP.
ATP synthase is used as photophosphorylation to convert chemicals into ATP.[15]
Archaea and bacteria are structurally similar even though they are not closely
related in the tree of life. The shapes of both bacteria and archaea cells vary
from a spherical shape known as coccus or a rod-shape known as bacillus. They
are also related with no internal membrane and a cell wall that assists the
cell maintaining its shape. Even though archaeal cells have cells walls, they
do not contain peptidoglycan, which means archaea do not produce cellulose or
chitin. Archaea are most closely related to eukaryotes due to tRNA present in
archaea, but not in bacteria. Archaea have the same ribosomes as eukaryotes
that synthesize into proteins.[26] Aside from the morphology of archaea and
bacteria, there are other differences between these domains. Archaea that live
in extreme and harsh environments with low pH levels such as salt lakes,
oceans, and in the gut of ruminants and humans are also known as extremophiles.
In contrast, bacteria are found in various areas such as plants, animals, soil,
and rocks.[27]
**** Fungi[edit] ****
Fungi can be both multicellular and unicellular organisms, and are
distinguished from other microbes by the way they obtain nutrients. Fungi
secrete enzymes into their surroundings, to break down organic matter.[9]
Fungal genetics uses yeast, and filamentous fungi as model organisms for
eukaryotic genetic research, including cell_cycle regulation, chromatin
structure and gene_regulation.[28]
Studies of the fungus Neurospora_crassa have contributed substantially to
understanding how genes work. N. crassa is a type of red bread mold of the
phylum Ascomycota. It is used as a model_organism because it is easy to grow
and has a haploid life cycle that makes genetic analysis simple since recessive
traits will show up in the offspring. Analysis of genetic recombination is
facilitated by the ordered arrangement of the products of meiosis in
ascospores. In its natural environment, N. crassa lives mainly in tropical and
sub-tropical regions. It often can be found growing on dead plant matter after
fires.
Neurospora was used by Edward_Tatum and George_Beadle in their experiments[29]
for which they won the Nobel_Prize_in_Physiology_or_Medicine in 1958. The
results of these experiments led directly to the one_gene-one_enzyme_hypothesis
that specific genes code for specific proteins. This concept proved to be the
opening gun in what became molecular_genetics and all the developments that
have followed from that.[30]
Saccharomyces_cerevisiae is a yeast of the phylum Ascomycota. During vegetative
growth that ordinarily occurs when nutrients are abundant, S. cerevisiae
reproduces by mitosis as diploid cells. However, when starved, these cells
undergo meiosis to form haploid spores.[31] Mating occurs when haploid cells of
opposite mating_types MATa and MATÎ± come into contact. Ruderfer et al.[32]
pointed out that, in nature, such contacts are frequent between closely related
yeast cells for two reasons. The first is that cells of opposite mating type
are present together in the same acus, the sac that contains the cells directly
produced by a single meiosis, and these cells can mate with each other. The
second reason is that haploid cells of one mating type, upon cell division,
often produce cells of the opposite mating type. An analysis of the ancestry of
natural S. cerevisiae strains concluded that outcrossing occurs very
infrequently (only about once every 50,000 cell divisions).[32] The relative
rarity in nature of meiotic events that result from outcrossing suggests that
the possible long-term benefits of outcrossing (e.g. generation of diversity)
are unlikely to be sufficient for generally maintaining sex from one generation
to the next. Rather, a short term benefit, such as meiotic recombinational
repair of DNA damages caused by stressful conditions (such as starvation)[33]
[34][35] may be the key to the maintenance of sex in S. cerevisiae.
Candida_albicans is a diploid fungus that grows both as a yeast and as a
filament. C. albicans is the most common fungal pathogen in humans. It causes
both debilitating mucosal infections and potentially life-threatening systemic
infections. C. albicans has maintained an elaborate, but largely hidden, mating
apparatus.[36] Johnson[36] suggested that mating strategies may allow C.
albicans to survive in the hostile environment of a mammalian host.
Among the 250 known species of aspergilli, about 33% have an identified sexual
state.[37] Among those Aspergillus species that exhibit a sexual cycle the
overwhelming majority in nature are homothallic (self-fertilizing).[37] Selfing
in the homothallic fungus Aspergillus_nidulans involves activation of the same
mating pathways characteristic of sex in outcrossing species, i.e. self-
fertilization does not bypass required pathways for outcrossing sex but instead
requires activation of these pathways within a single individual.[38] Fusion of
haploid nuclei occurs within reproductive structures termed cleistothecia, in
which the diploid zygote undergoes meiotic divisions to yield haploid
ascospores.
**** Protozoa[edit] ****
Protozoa are unicellular organisms, which have nuclei, and ultramicroscopic
cellular bodies within their cytoplasm.[9] One particular aspect of protozoa
that are of interest to human geneticists are their flagella, which are very
similar to human sperm flagella.
Studies of Paramecium have contributed to our understanding of the function of
meiosis. Like all ciliates, Paramecium has a polyploid macronucleus, and one or
more diploid micronuclei. The macronucleus controls non-reproductive cell
functions, expressing the genes needed for daily functioning. The micronucleus
is the generative, or germline nucleus, containing the genetic material that is
passed along from one generation to the next.[39]
In the asexual fission phase of growth, during which cell divisions occur by
mitosis rather than meiosis, clonal aging occurs leading to a gradual loss of
vitality. In some species, such as the well studied Paramecium tetraurelia, the
asexual line of clonally aging paramecia loses vitality and expires after about
200 fissions if the cells fail to undergo meiosis followed by either autogamy
(self-fertilizaion) or conjugation (outcrossing) (see aging_in_Paramecium). DNA
damage increases dramatically during successive clonal cell divisions and is a
likely cause of clonal aging in P. tetraurelia.[40][41][42]
When clonally aged P. tetraurelia are stimulated to undergo meiosis in
association with either autogamy or conjugation, the progeny are rejuvenated,
and are able to have many more mitotic binary fission divisions. During either
of these processes the micronuclei of the cell(s) undergo meiosis, the old
macronucleus disintegrates and a new macronucleus is formed by replication of
the micronuclear DNA that had recently undergone meiosis. There is apparently
little, if any, DNA damage in the new macronucleus, suggesting that
rejuvenation is associated with the repair of these damages in the micronucleus
during meiosis.[35]
**** Viruses[edit] ****
Viruses are capsid-encoding organisms composed of proteins and nucleic acids
that can self-assemble after replication in a host cell using the host's
replication machinery.[43] There is a disagreement in science about whether
viruses are living due to their lack of ribosomes.[43] Comprehending the viral
genome is important not only for studies in genetics but also for understanding
their pathogenic properties.[44]
Many types of virus are capable of genetic recombination. When two or more
individual viruses of the same type infect a cell, their genomes may recombine
with each other to produce recombinant virus progeny. Both DNA and RNA viruses
can undergo recombination. When two or more viruses, each containing lethal
genomic damage infect the same host cell, the virus genomes often can pair with
each other and undergo homologous recombinational repair to produce viable
progeny.[45][46] This process is known as multiplicity reactivation.[45][47]
Enzymes employed in multiplicity reactivation are functionally homologous to
enzymes employed in bacterial and eukaryotic recombinational repair.
Multiplicity reactivation has been found to occur with pathogenic viruses
including influenza virus, HIV-1, adenovirus simian virus 40, vaccinia virus,
reovirus, poliovirus and herpes simplex virus as well as numerous
bacteriophages.[47]
Any living organism can contract a virus by giving parasites the opportunity to
grow. Parasites feed on the nutrients of another organism which allows the
virus to thrive. Once the human body detects a virus, it then creates fighter
cells that attack the parasite/virus; literally, causing a war within the body.
[48] A virus can affect any part of the body causing a wide range of illnesses
such as the flu, the common cold, and sexually transmitted diseases.[48] The
flu is an airborne virus that travels through tiny droplets and is formally
known as Influenza. Parasites travel through the air and attack the human
respiratory system. People that are initially infected with this virus pass
infection on by normal day to day activity such as talking and sneezing. When a
person comes in contact with the virus, unlike the common cold, the flu virus
affects people almost immediately. Symptoms of this virus are very similar to
the common cold but much worse. Body aches, sore throat, headache, cold sweats,
muscle aches and fatigue are among the many symptoms accompanied by the virus.
[49] A viral infection in the upper respiratory tract results in the common
cold.[50] With symptoms like sore throat, sneezing, small fever, and a cough,
the common cold is usually harmless and tends to clear up within a week or so.
The common cold is also a virus that is spread through the air but can also be
passed through direct contact. This infection takes a few days to develop
symptoms; it is a gradual process unlike the flu.[50]
***** Applications of microbial genetics[edit] *****
Taq polymerase which is used in Polymerase Chain Reaction(PCR)
Microbes are ideally suited for biochemical and genetics studies and have made
huge contributions to these fields of science such as the demonstration that
DNA is the genetic material,[51][52] that the gene has a simple linear
structure,[53] that the genetic code is a triplet code,[54] and that gene
expression is regulated by specific genetic processes.[55] Jacques_Monod and
FranÃ§ois_Jacob used Escherichia_coli, a type of bacteria, in order to develop
the operon model of gene_expression, which lay down the basis of gene
expression and regulation.[56] Furthermore, the hereditary processes of single-
celled eukaryotic microorganisms are similar to those in multi-cellular
organisms allowing researchers to gather information on this process as well.
[57] Another bacterium which has greatly contributed to the field of genetics
is Thermus_aquaticus, which is a bacterium that tolerates high temperatures.
From this microbe scientists isolated the enzyme Taq_polymerase, which is now
used in the powerful experimental technique, Polymerase_chain_reaction(PCR).
[58] Additionally the development of recombinant_DNA_technology through the use
of bacteria has led to the birth of modern genetic_engineering and
biotechnology.[9]
Using microbes, protocols were developed to insert genes into bacterial
plasmids, taking advantage of their fast reproduction, to make biofactories for
the gene of interest. Such genetically engineered bacteria can produce
pharmaceuticals such as insulin, human_growth_hormone, interferons and blood
clotting_factors.[9] These biofactories are typically much cheaper to operate
and maintain than the alternative procedures of producing pharmaceuticals.
They're like millions of tiny pharmaceutical machines that only require basic
raw materials and the right environment to produce a large amount of product.
The utilization of incorporating the human insulin gene alone has had profound
impacts on the medical industry. It's thought that biofactories might be the
ultimate key in reducing the price of expensive life saving pharmaceutical
compounds.
Microbes synthesize a variety of enzymes for industrial applications, such as
fermented foods, laboratory test reagents, dairy products (such as renin), and
even in clothing (such as Trichoderma fungus whose enzyme is used to give jeans
a stone washed appearance).[9]
There is currently potential for microbes to be used as an alternative for
petroleum based surfactants. Microbial surfactants would still have the same
kind of hydrophillic and hydrophobic functional groups as their petroleum based
counterparts, but they have numerous advantages over their competition. In
comparison, microbial amphiphillic compounds have robust a tendency to stay
functional in extreme environments such as areas with high heat or extreme ph.
all while being biodegradable and less toxic to the environment. This efficient
and cheap method of production could be the solution to the ever increasing
global consumption of surfactants. Ironically, the application for bio-based
surfactants with the most demand is the oil industry which uses surfactants in
general production as well as development of specific oil compositions.[59]
Microbes are an abundant source of lipases which have a wide variety of
industrial and consumer applications. Enzymes perform a wide variety of
functions inside the cells of living things, so it only makes sense that we can
use them for similar purposes on a larger scale. Microbial enzymes are
typically preferred for mass production due to the wide variety of functions
available and their ability to be mass produced. Plant and animal enzymes are
typically too expensive to be mass produced, however this is not always the
case. Especially in plants. Industrial applications of lipases generally
include the enzyme as a more efficient and cost effective catalyst in the
production of commercially valuable chemicals from fats and oils, because they
are able to retain their specific properties in mild easy to maintain
conditions and work at an increased rate. Other already successful applications
of lipolytic enzymes include the production of biofuels, polymers, non-
stereoisomeric pharmaceuticals, agricultural compounds, and flavor enhancing
compounds.[60]
In regards to industrial optimization, the benefit of the biofactory method of
production is the ability to direct optimization by means of directed
evolution. The efficiency and specificity of production will increase over time
by imposing artificial selection. This method of improving efficiency is
nothing new in the agriculture, but it's a relatively new concept in industrial
production. It is though that this method will be far superior to conventional
industrial methods because you have optimization on multiple fronts. The first
front being that the microorganisms that make up biofactories can be evolved to
our needs. The second front being the conventional method of optimization
brought about by the integration of advancing technologies. This combination of
conventional and biological advancement is just now becoming utilized and
provides a virtually limitless number of applications.[61]
***** See also[edit] *****
    * Bacterial_genetics
***** References[edit] *****
   1. ^ a b c d e f"Microbes_and_the_Tools_of_Genetic_Engineering_|
      Microbiology". courses.lumenlearning.com. Retrieved 17 November 2018.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^Gest, Hau (22 May 2004). "The_discovery_of_microorganisms_by_Robert
      Hooke_and_Antoni_van_Leeuwenhoek,_Fellows_of_The_Royal_Society". The
      Royal Society. 58 (2): 137â201. doi:10.1098/rsnr.2004.0055.
      PMID 15209075. Retrieved 25 September 2015.
   4. ^"BBC_-_History_-_Historic_Figures:_Antonie_van_Leeuwenhoek_(1632_-
      1723)". Retrieved 17 November 2018.
   5. ^ a b"antonie_van_leeuwenhoek:_Topics_by_Science.gov". www.science.gov.
      Retrieved 17 November 2018.
   6. ^Mortlock, Robert (2013). Microorganisms As Model Systems for Studying
      Evolution. Springer Verlag. p. 2. ISBN 978-1-4684-4846-7.
   7. ^Murphy, Cormac D. (2 September 2014). "Drug metabolism in
      microorganisms". Biotechnology Letters. 37 (1): 19â28. doi:10.1007/
      s10529-014-1653-8. hdl:10197/7674. PMID 25179825.
   8. ^ a b c dBuckley, Merry; Reid, Ann (2011). Microbial_Evolution.
   9. ^Chakraborty, Ranajit; Budowle, Bruce (2011). "Population Genetic
      Considerations in Statistical Interpretation of Microbial Forensic Data
      in Comparison with Human DNA Forensic Standard". Microbial Forensics.
      pp. 561â580. doi:10.1016/B978-0-12-382006-8.00033-5. ISBN 978-0-12-
      382006-8.
  10. ^ a b c d e fWeeks, Benjamin S. (2012). Alcamo's microbes and society
      (3rd ed.). Sudbury, MA: Jones & Bartlett Learning. ISBN 978-0-7637-9064-
      6.
  11. ^"Bacterial_genetics". Nature. Macmillan Publishers Limited. Retrieved 8
      November 2015.
  12. ^Chen I, Dubnau D (2004). "DNA uptake during bacterial transformation".
      Nature Reviews Microbiology. 2 (3): 241â9. doi:10.1038/nrmicro844.
      PMID 15083159.
  13. ^Johnsborg O, Eldholm V, HÃ¥varstein LS (2007). "Natural genetic
      transformation: prevalence, mechanisms and function". Research in
      Microbiology. 158 (10): 767â78. doi:10.1016/j.resmic.2007.09.004.
      PMID 17997281.
  14. ^Michod RE, Bernstein H, Nedelcu AM (2008). "Adaptive value of sex in
      microbial pathogens". Infection, Genetics and Evolution. 8 (3): 267â85.
      doi:10.1016/j.meegid.2008.01.002. PMID 18295550.
  15. ^Gray TA, Krywy JA, Harold J, Palumbo MJ, Derbyshire KM (2013).
      "Distributive_conjugal_transfer_in_mycobacteria_generates_progeny_with
      meiotic-like_genome-wide_mosaicism,_allowing_mapping_of_a_mating_identity
      locus". PLoS Biology. 11 (7): e1001602. doi:10.1371/journal.pbio.1001602.
      PMC 3706393. PMID 23874149.
  16. ^ a bHogan, Michael. "What_are_Archaea?_-_Encyclopedia_of_Life".
      Encyclopedia of Life.
  17. ^"Encyclopedia_of_Life".
  18. ^"Archaea". Microbe World. Microbe World. Retrieved 8 November 2015.
  19. ^Chambers, Cecilia R.; Patrick, Wayne M. (2015). "Archaeal_Nucleic_Acid
      Ligases_and_Their_Potential_in_Biotechnology". Archaea. 2015: 170571.
      doi:10.1155/2015/170571. PMC 4606414. PMID 26494982.
  20. ^Rosenshine I, Tchelet R, Mevarech M (1989). "The mechanism of DNA
      transfer in the mating system of an archaebacterium". Science. 245
      (4924): 1387â9. doi:10.1126/science.2818746. PMID 2818746.
  21. ^FrÃ¶ls S, Ajon M, Wagner M, Teichmann D, Zolghadr B, Folea M, Boekema
      EJ, Driessen AJ, Schleper C, Albers SV (2008). "UV-inducible cellular
      aggregation of the hyperthermophilic archaeon Sulfolobus solfataricus is
      mediated by pili formation". Molecular Microbiology. 70 (4): 938â52.
      doi:10.1111/j.1365-2958.2008.06459.x. PMID 18990182.
  22. ^FrÃ¶ls S, White MF, Schleper C (2009). "Reactions to UV damage in the
      model archaeon Sulfolobus solfataricus". Biochemical Society
      Transactions. 37 (Pt 1): 36â41. doi:10.1042/BST0370036. PMID 19143598.
  23. ^Ajon M, FrÃ¶ls S, van Wolferen M, Stoecker K, Teichmann D, Driessen AJ,
      Grogan DW, Albers SV, Schleper C (2011). "UV-inducible DNA exchange in
      hyperthermophilic archaea mediated by type IV pili". Molecular
      Microbiology. 82 (4): 807â17. doi:10.1111/j.1365-2958.2011.07861.x.
      PMID 21999488.
  24. ^"Archaebacteria_Examples". BiologyWise.
  25. ^"Archaeal_genetics_-_Latest_research_and_news_|_Nature". www.nature.com.
  26. ^ a b c"Archaeal_Genetics_|_Boundless_Microbiology".
      courses.lumenlearning.com.
  27. ^"Morphology_of_the_Archaea". www.ucmp.berkeley.edu.
  28. ^"Archaea_vs_Bacteria_-_Difference_and_Comparison_|_Diffen".
  29. ^"Fungal_Genetics". Nature.com. Macmillan Publishers Limited. Retrieved 9
      November 2015.
  30. ^Beadle GW, Tatum EL (1941). "Genetic_Control_of_Biochemical_Reactions_in
      Neurospora". Proceedings of the National Academy of Sciences of the
      United States of America. 27 (11): 499â506. doi:10.1073/pnas.27.11.499.
      PMC 1078370. PMID 16588492.
  31. ^Horowitz NH, Berg P, Singer M, Lederberg J, Susman M, Doebley J, Crow JF
      (2004). "A_centennial:_George_W._Beadle,_1903-1989". Genetics. 166 (1):
      1â10. doi:10.1534/genetics.166.1.1. PMC 1470705. PMID 15020400.
  32. ^Herskowitz I (1988). "Life_cycle_of_the_budding_yeast_Saccharomyces
      cerevisiae". Microbiological Reviews. 52 (4): 536â53. PMC 373162.
      PMID 3070323.
  33. ^ a bRuderfer DM, Pratt SC, Seidel HS, Kruglyak L (2006). "Population
      genomic analysis of outcrossing and recombination in yeast". Nature
      Genetics. 38 (9): 1077â81. doi:10.1038/ng1859. PMID 16892060.
  34. ^Birdsell, John A.; Wills, Christopher (2003). "The Evolutionary Origin
      and Maintenance of Sexual Recombination: A Review of Contemporary
      Models". Evolutionary Biology. pp. 27â138. doi:10.1007/978-1-4757-5190-
      1_2. ISBN 978-1-4419-3385-0.
  35. ^Hrandl, Elvira (2013). "Meiosis and the Paradox of Sex in Nature".
      Meiosis. doi:10.5772/56542. ISBN 978-953-51-1197-9.
  36. ^ a bBernstein, Harris; Bernstei, Carol (2013). "Evolutionary Origin and
      Adaptive Function of Meiosis". Meiosis. doi:10.5772/56557. ISBN 978-953-
      51-1197-9.
  37. ^ a bJohnson A (2003). "The biology of mating in Candida albicans".
      Nature Reviews Microbiology. 1 (2): 106â16. doi:10.1038/nrmicro752.
      PMID 15035040.
  38. ^ a bDyer PS, O'Gorman CM (2012). "Sexual development and cryptic
      sexuality in fungi: insights from Aspergillus species". FEMS Microbiology
      Reviews. 36 (1): 165â92. doi:10.1111/j.1574-6976.2011.00308.x.
      PMID 22091779.
  39. ^Paoletti M, Seymour FA, Alcocer MJ, Kaur N, Calvo AM, Archer DB, Dyer PS
      (2007). "Mating type and the genetic basis of self-fertility in the model
      fungus Aspergillus nidulans". Current Biology. 17 (16): 1384â9. doi:
      10.1016/j.cub.2007.07.012. PMID 17669651.
  40. ^Prescott DM (1994). "The_DNA_of_ciliated_protozoa". Microbiological
      Reviews. 58 (2): 233â67. PMC 372963. PMID 8078435.
  41. ^Smith-Sonneborn J (1979). "DNA repair and longevity assurance in
      Paramecium tetraurelia". Science. 203 (4385): 1115â7. doi:10.1126/
      science.424739. PMID 424739.
  42. ^Holmes GE, Holmes NR (1986). "Accumulation of DNA damages in aging
      Paramecium tetraurelia". Molecular and General Genetics. 204 (1):
      108â14. doi:10.1007/bf00330196. PMID 3091993.
  43. ^Gilley D, Blackburn EH (1994). "Lack_of_telomere_shortening_during
      senescence_in_Paramecium". Proceedings of the National Academy of
      Sciences of the United States of America. 91 (5): 1955â8. doi:10.1073/
      pnas.91.5.1955. PMC 43283. PMID 8127914.
  44. ^ a bRaoult, Didier; Forterre, Patrick (3 March 2008). "Redefining
      viruses: lessons from Mimivirus". Nature Reviews Microbiology. 6 (4):
      315â319. doi:10.1038/nrmicro1858. PMID 18311164.
  45. ^Seto, Donald (30 November 2010). "Viral_Genomics_and_Bioinformatics".
      Viruses. 2 (12): 2587â2593. doi:10.3390/v2122587. PMC 3185590.
      PMID 21994632.
  46. ^ a bBernstein C (1981). "Deoxyribonucleic_acid_repair_in_bacteriophage".
      Microbiological Reviews. 45 (1): 72â98. PMC 281499. PMID 6261109.
  47. ^Chen D, Bernstein C (1987). "Recombinational repair of hydrogen
      peroxide-induced damages in DNA of phage T4". Mutation Research. 184 (2):
      87â98. doi:10.1016/0167-8817(87)90064-2. PMID 3627145.
  48. ^ a bMichod RE, Bernstein H, Nedelcu AM (May 2008). "Adaptive value of
      sex in microbial pathogens". Infection, Genetics and Evolution. 8 (3):
      267â85. doi:10.1016/j.meegid.2008.01.002. PMID 18295550.
  49. http://www.hummingbirds.arizona.edu/Faculty/Michod/Downloads/
      IGE%20review%20sex.pdf
  50. ^ a bPaula, Tennant (12 March 2018). Viruses : molecular biology, host
      interactions, and applications to biotechnology. Fermin, Gustavo,,
      Foster, Jerome E. San Diego, CA. ISBN 9780128111949. OCLC 1028979396.
  51. ^1956-, Quinn, Tom (2008). Flu : a social history of influenza. London:
      New Holland. ISBN 9781845379414. OCLC 232713128.
  52. ^ a bGreenhough, Beth (6 January 2012). "Where species meet and mingle:
      endemic human-virus relations, embodied communication and more-than-human
      agency at the Common Cold Unit 1946â90". Cultural Geographies. 19 (3):
      281â301. doi:10.1177/1474474011422029. ISSN 1474-4740.
  53. ^Avery OT, MacLeod CM, McCarty M (1979). "Studies_on_the_chemical_nature
      of_the_substance_inducing_transformation_of_pneumococcal_types.
      Inductions_of_transformation_by_a_desoxyribonucleic_acid_fraction
      isolated_from_pneumococcus_type_III". Journal of Experimental Medicine.
      149 (2): 297â326. doi:10.1084/jem.149.2.297. PMC 2184805. PMID 33226.
  54. ^Hershey AD, Chase M (1952). "Independent_functions_of_viral_protein_and
      nucleic_acid_in_growth_of_bacteriophage". Journal of General Physiology.
      36 (1): 39â56. doi:10.1085/jgp.36.1.39. PMC 2147348. PMID 12981234.
  55. ^Benzer S (1959). "On_The_Topology_of_the_Genetic_Fine_Structure".
      Proceedings of the National Academy of Sciences of the United States of
      America. 45 (11): 1607â20. doi:10.1073/pnas.45.11.1607. PMC 222769.
      PMID 16590553.
  56. ^Crick FH, Barnett L, Brenner S, Watts-Tobin RJ (1961). "General nature
      of the genetic code for proteins". Nature. 192 (4809): 1227â32. doi:
      10.1038/1921227a0. PMID 13882203.
  57. ^Jacob F, Monod J (1961). "Genetic regulatory mechanisms in the synthesis
      of proteins". Journal of Molecular Biology. 3 (3): 318â56. doi:10.1016/
      S0022-2836(61)80072-7. PMID 13718526.
  58. ^"Microbial_Genetics". World of Microbiology and Immunology. 2003.
      Retrieved 9 November 2015.
  59. ^Bainbridge, B.W. (1987). Genetics of microbes (2nd ed.). Glasgow:
      Blackie. ISBN 978-0-412-01281-5.
  60. ^Terpe, Kay (1 November 2013). "Overview of thermostable DNA polymerases
      for classical PCR applications: from molecular and biochemical
      fundamentals to commercial systems". Applied Microbiology and
      Biotechnology. 97 (24): 10243â10254. doi:10.1007/s00253-013-5290-2.
      PMID 24177730.
  61. ^Banat, I. M.; Makkar, R. S.; Cameotra, S. S. (15 May 2000). "Potential
      commercial applications of microbial surfactants". Applied Microbiology
      and Biotechnology. 53 (5): 495â508. doi:10.1007/s002530051648.
      ISSN 0175-7598. PMID 10855707.
  62. ^Hasan, Fariha; Shah, Aamer Ali; Hameed, Abdul (26 June 2006).
      "Industrial applications of microbial lipases". Enzyme and Microbial
      Technology. 39 (2): 235â251. doi:10.1016/j.enzmictec.2005.10.016.
      ISSN 0141-0229.
  63. ^Kondo, Akihiko; Ishii, Jun; Hara, Kiyotaka Y.; Hasunuma, Tomohisa;
      Matsuda, Fumio (20 January 2013). "Development of microbial cell
      factories for bio-refinery through synthetic bioengineering". Journal of
      Biotechnology. 163 (2): 204â216. doi:10.1016/j.jbiotec.2012.05.021.
      ISSN 0168-1656. PMID 22728424.
    * v
    * t
    * e
Genetics
    * Introduction
    * Outline
    * History
    * Index
                       * Chromosome
                       * DNA
Key components         * RNA
                       * Nucleotide
                       * Genome
                       * Classical
                       * Conservation
                       * Ecological
Fields                 * Immunogenetics
                       * Molecular
                       * Population
                       * Quantitative
                       * the_Americas
                       * the_British_Isles
Archaeogenetics of     * Europe
                       * Italy
                       * the_Near_East
                       * South_Asia
                       * Behavioural_genetics
                       * Epigenetics
                       * Geneticist
                       * Genome_editing
                       * Genomics
                       * Genetic_code
                       * Medical_genetics
Related topics         * Molecular_evolution
                       * Reverse_genetics
                       * Genetic_engineering
                       * Genetic_diversity
                       * Heredity
                       * Genetic_monitoring
                       * Genetic_genealogy
                       * Lulu_and_Nana_controversy
    * [List-Class article] List_of_genetics_research_organizations
    * [Category] Category

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Microbial_genetics&oldid=908530142"
Categories:
    * Microbiology
    * Genetics_by_type_of_organism
Hidden categories:
    * Use_dmy_dates_from_April_2017
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Bosanski
    * ÙØ§Ø±Ø³Û
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
Edit_links
    * This page was last edited on 30 July 2019, at 09:26 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
