The following text has been accessed from https://en.wikipedia.org/wiki/Amino_acid at Fri Aug 9 03:34:56 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_good_article._Follow_the_link_for_more_information.]
****** Amino acid ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
This article is about the class of chemicals. For the structures and properties
of the standard proteinogenic amino acids, see Proteinogenic_amino_acid.
Organic compounds containing amine and carboxylic groups
The structure of an alpha amino acid in its un-ionized form
Amino acids are organic_compounds containing amine (-NH2) and carboxyl (-COOH)
functional_groups, along with a side_chain (R group) specific to each amino
acid.[1][2][3] The key elements of an amino acid are carbon (C), hydrogen (H),
oxygen (O), and nitrogen (N), although other elements are found in the side
chains of certain amino acids. About 500 naturally occurring amino acids are
known (though only 20 appear in the genetic_code) and can be classified in many
ways.[4] They can be classified according to the core structural functional
groups' locations as alpha-_(Î±-),_beta-_(Î²-),_gamma-_(Î³-)_or_delta-_(Î´-
) amino acids; other categories relate to polarity, pH level, and side chain
group type (aliphatic, acyclic, aromatic, containing hydroxyl or sulfur, etc.).
In the form of proteins, amino acid residues form the second-largest component
(water is the largest) of human muscles and other tissues.[5] Beyond their role
as residues in proteins, amino acids participate in a number of processes such
as neurotransmitter transport and biosynthesis.
In biochemistry, amino acids having both the amine and the carboxylic acid
groups attached to the first_(alpha-)_carbon atom have particular importance.
They are known as 2-, alpha-, or Î±-amino acids (generic formula H2NCHRCOOH in
most cases,[6] where R is an organic substituent known as a "side_chain");[7]
often the term "amino acid" is used to refer specifically to these. They
include the 22 proteinogenic ("protein-building") amino acids,[8][9][10] which
combine into peptide chains ("polypeptides") to form the building-blocks of a
vast array of proteins.[11] These are all L-stereoisomers ("left-handed"
isomers), although a few D-amino acids ("right-handed") occur in bacterial
envelopes, as a neuromodulator (D-serine), and in some antibiotics.[12]
Twenty of the proteinogenic amino acids are encoded directly by triplet codons
in the genetic_code and are known as "standard" amino acids. The other two
("non-standard" or "non-canonical") are selenocysteine (present in many
prokaryotes as well as most eukaryotes, but not coded directly by DNA), and
pyrrolysine (found only in some archaea and one bacterium). Pyrrolysine and
selenocysteine are encoded via variant codons; for example, selenocysteine is
encoded by stop_codon and SECIS_element.[13][14][15] N-formylmethionine (which
is often the initial amino acid of proteins in bacteria, mitochondria, and
chloroplasts) is generally considered as a form of methionine rather than as a
separate proteinogenic amino acid. CodonâtRNA combinations not found in
nature can also be used to "expand"_the_genetic_code and form novel proteins
known as alloproteins incorporating non-proteinogenic_amino_acids.[16][17][18]
Many important proteinogenic and non-proteinogenic amino acids have biological
functions. For example, in the human_brain, glutamate (standard glutamic_acid)
and gamma-amino-butyric_acid ("GABA", non-standard gamma-amino acid) are,
respectively, the main excitatory_and_inhibitory_neurotransmitters.[19]
Hydroxyproline, a major component of the connective_tissue collagen, is
synthesised from proline. Glycine is a biosynthetic precursor to porphyrins
used in red_blood_cells. Carnitine is used in lipid_transport.
Nine proteinogenic amino acids are called "essential" for humans because they
cannot be produced from other compounds by the human body and so must be taken
in as food. Others may be conditionally_essential for certain ages or medical
conditions. Essential amino acids may also differ between species.[20]
Because of their biological significance, amino acids are important in
nutrition and are commonly used in nutritional_supplements, fertilizers, feed,
and food_technology. Industrial uses include the production of drugs,
biodegradable_plastics, and chiral_catalysts.
⁰
***** Contents *****
    * 1_History
    * 2_General_structure
          o 2.1_Isomerism
          o 2.2_Side_chains
          o 2.3_Zwitterions
          o 2.4_Isoelectric_point
    * 3_Occurrence_and_functions_in_biochemistry
          o 3.1_Proteinogenic_amino_acids
          o 3.2_Non-proteinogenic_amino_acids
          o 3.3_D-amino_acid_natural_abundance
          o 3.4_Non-standard_amino_acids
          o 3.5_In_human_nutrition
          o 3.6_Non-protein_functions
    * 4_Uses_in_industry
          o 4.1_Expanded_genetic_code
          o 4.2_Nullomers
          o 4.3_Chemical_building_blocks
          o 4.4_Biodegradable_plastics
    * 5_Synthesis
          o 5.1_Chemical_synthesis
          o 5.2_Biosynthesis
    * 6_Reactions
          o 6.1_Peptide_bond_formation
          o 6.2_Catabolism
          o 6.3_Complexation
    * 7_Physicochemical_properties_of_amino_acids
          o 7.1_Table_of_standard_amino_acid_abbreviations_and_properties
    * 8_See_also
    * 9_References_and_notes
    * 10_Further_reading
    * 11_External_links
***** History[edit] *****
The first few amino acids were discovered in the early 19th century.[21][22] In
1806, French chemists Louis-Nicolas_Vauquelin and Pierre_Jean_Robiquet isolated
a compound in asparagus that was subsequently named asparagine, the first amino
acid to be discovered.[23][24] Cystine was discovered in 1810,[25] although its
monomer, cysteine, remained undiscovered until 1884.[24][26] Glycine and
leucine were discovered in 1820.[27] The last of the 20 common amino acids to
be discovered was threonine in 1935 by William_Cumming_Rose, who also
determined the essential_amino_acids and established the minimum daily
requirements of all amino acids for optimal growth.[28][29]
The unity of the chemical category was recognized by Wurtz in 1865, but he gave
no particular name to it.[30] First use of the term "amino acid" in the English
language dates from 1898,[31] while the German term, AminosÃ¤ure, was used
earlier.[32] Proteins were found to yield amino acids after enzymatic digestion
or acid hydrolysis. In 1902, Emil_Fischer and Franz_Hofmeister independently
proposed that proteins are formed from many amino acids, whereby bonds are
formed between the amino group of one amino acid with the carboxyl group of
another, resulting in a linear structure that Fischer termed "peptide".[33]
***** General structure[edit] *****
The 21 proteinogenic_Î±-amino_acids found in eukaryotes, grouped according to
their side chains' pKa values and charges carried at physiological_pH_(7.4)
In the structure shown at the top of the page, R represents a side_chain
specific to each amino acid. The carbon atom next to the carboxyl_group (which
is therefore numbered 2 in the carbon_chain starting from that functional
group) is called the Î±âcarbon. Amino acids containing an amino_group bonded
directly to the alpha carbon are referred to as alpha amino acids.[34] These
include amino acids such as proline which contain secondary_amines, which used
to be often referred to as "imino acids".[35][36][37]
**** Isomerism[edit] ****
The alpha amino acids are the most common form found in nature, but only when
occurring in the L-isomer. The alpha carbon is a chiral carbon atom, with the
exception of glycine which has two indistinguishable hydrogen atoms on the
alpha carbon.[38] Therefore, all alpha amino acids but glycine can exist in
either of two enantiomers, called L or D amino acids (relative configuration),
which are mirror images of each other (see also Chirality). While L-amino acids
represent all of the amino acids found in proteins during translation in the
ribosome, D-amino acids are found in some proteins produced by enzyme
posttranslational_modifications after translation and translocation to the
endoplasmic_reticulum, as in exotic sea-dwelling organisms such as cone_snails.
[39] They are also abundant components of the peptidoglycan cell_walls of
bacteria,[40] and D-serine may act as a neurotransmitter in the brain.[41] D-
amino acids are used in racemic_crystallography to create centrosymmetric
crystals, which (depending on the protein) may allow for easier and more robust
protein structure determination.[42] The L and D convention for amino acid
configuration refers not to the optical activity of the amino acid itself but
rather to the optical activity of the isomer of glyceraldehyde from which that
amino acid can, in theory, be synthesized (D-glyceraldehyde is dextrorotatory;
L-glyceraldehyde is levorotatory). In alternative fashion, the (S)_and_(R)
designators are used to indicate the absolute configuration. Almost all of the
amino acids in proteins are (S) at the Î± carbon, with cysteine being (R) and
glycine non-chiral.[43] Cysteine has its side chain in the same geometric
position as the other amino acids, but the R/S terminology is reversed because
of the higher atomic number of sulfur compared to the carboxyl oxygen gives the
side chain a higher priority by the CahnâIngoldâPrelog rules, whereas the
atoms in other side chains give them lower priority compared to the carboxyl
group.
**** Side chains[edit] ****
Lysine with carbon atoms labeled by position[clarification_needed]
In amino acids that have a carbon chain attached to the Î±âcarbon (such as
lysine, shown to the right) the carbons are labeled in order as Î±, Î², Î³, Î´,
and so on.[44] In some amino acids, the amine group is attached to the Î² or
Î³-carbon, and these are therefore referred to as beta or gamma amino acids.
Amino acids are usually classified by the properties of their side chain into
four groups. The side chain can make an amino acid a weak_acid or a weak_base,
and a hydrophile if the side chain is polar or a hydrophobe if it is nonpolar.
[38] The chemical_structures of the 22 standard amino acids, along with their
chemical properties, are described more fully in the article on these
proteinogenic_amino_acids.
The phrase "branched-chain_amino_acids" or BCAA refers to the amino acids
having aliphatic side chains that are non-linear; these are leucine,
isoleucine, and valine. Proline is the only proteinogenic amino acid whose
side-group links to the Î±-amino group and, thus, is also the only
proteinogenic amino acid containing a secondary amine at this position.[38] In
chemical terms, proline is, therefore, an imino_acid, since it lacks a primary
amino_group,[45] although it is still classed as an amino acid in the current
biochemical nomenclature,[46] and may also be called an "N-alkylated alpha-
amino acid".[47]
**** Zwitterions[edit] ****
An amino acid in its (1) un-ionized and (2) zwitterionic forms
Main article: Zwitterion
The Î±-carboxylic_acid group of amino acids is a weak_acid, meaning that it
releases a hydron (such as a proton) at moderate pH values. In other words,
carboxylic acid groups (âCO2H) can be deprotonated to become negatively
charged carboxylates (âCO2â ). The negatively charged carboxylate ion
predominates at pH values greater than the pKa of the carboxylic acid group
(typically around 2.2 for the 20 common amino acids; see the table of amino
acid structures above). In a complementary fashion, the Î±-amine of amino acids
is a weak_base, meaning that it accepts a proton at moderate pH values. In
other words, Î±-amino groups (NH2â) can be protonated to become positively
charged Î±-ammonium groups (+H3Nâ). The positively charged Î±-ammonium group
predominates at pH values less than the pKa of the Î±-ammonium group (mean for
the 20 common Î±-amino acids is about 9.4).
Because all amino acids contain amine and carboxylic acid functional groups,
they share amphiprotic properties.[38] Below pH 2.2, the predominant form will
have a neutral carboxylic acid group and a positive Î±-ammonium ion (net charge
+1), and above pH 9.4, a negative carboxylate and neutral Î±-amino group (net
charge â1). At a pH between 2.2 and 9.4, an amino acid typically contains
both a negative carboxylate and a positive Î±-ammonium group, as shown in
structure (2) on the right and exhibits a net zero charge. This molecular state
is known as a zwitterion, from the German Zwitter meaning "hermaphrodite" or
"hybrid".[48] The fully neutral form (structure (1) on the left) is a very
minor species in aqueous solution throughout the pH range (less than 1 part in
107). Altogether, amino acids exist as zwitterions in the solid phase and
crystallize with salt-like properties unlike typical organic acids or amines.
**** Isoelectric point[edit] ****
Composite of titration_curves of twenty proteinogenic amino acids grouped by
side chain category
The variation in titration curves when the amino acids can be grouped by
category.[clarification_needed] With the exception of tyrosine, using titration
to distinguish among hydrophobic amino acids is problematic.
At pH values between the two pKa values, the zwitterion predominates, but
coexists in dynamic_equilibrium with small amounts of net negative and net
positive ions. At the exact midpoint between the two pKa values, the trace
amount of net negative and trace of net positive ions exactly balance, so that
average net charge of all forms present is zero.[49] This pH is known as the
isoelectric_point pI, so pI = Â½(pKa1 + pKa2). The individual amino acids all
have slightly different pKa values and therefore have different isoelectric
points. For amino acids with charged side chains, the pKa of the side chain is
involved. Thus for Asp or Glu with negative side chains, pI = Â½(pKa1 + pKaR),
where pKaR is the side chain pKa. Cysteine also has potentially negative side
chain with pKaR = 8.14, so pI should be calculated as for Asp and Glu, even
though the side chain is not significantly charged at physiological pH. For
His, Lys, and Arg with positive side chains, pI = Â½(pKaR + pKa2). Amino acids
have zero mobility in electrophoresis at their isoelectric point, although this
behaviour is more usually exploited for peptides and proteins than single amino
acids. Zwitterions have minimum solubility at their isoelectric point, and some
amino acids (in particular, with non-polar side chains) can be isolated by
precipitation from water by adjusting the pH to the required isoelectric point.
***** Occurrence and functions in biochemistry[edit] *****
[A_protein_depicted_as_a_long_unbranched_string_of_linked_circles_each
representing_amino_acids]
A polypeptide is an unbranched chain of amino acids
[Diagrammatic_comparison_of_the_structures_of_Î²-alanine_and_Î±-alanine]
Î²-alanine and its Î±-alanine isomer
[A_diagram_showing_the_structure_of_selenocysteine]
The amino acid selenocysteine
**** Proteinogenic amino acids[edit] ****
Main article: Proteinogenic_amino_acids
See also: Protein_primary_structure and Posttranslational_modification
Amino acids are the structural units (monomers) that make up proteins. They
join together to form short polymer chains called peptides or longer chains
called either polypeptides or proteins. These polymers are linear and
unbranched, with each amino acid within the chain attached to two neighboring
amino acids. The process of making proteins encoded by DNA/RNA genetic material
is called translation and involves the step-by-step addition of amino acids to
a growing protein chain by a ribozyme that is called a ribosome.[50] The order
in which the amino acids are added is read through the genetic_code from an
mRNA template, which is an RNA copy of one of the organism's genes.
Twenty-two amino acids are naturally incorporated into polypeptides and are
called proteinogenic or natural amino acids.[38] Of these, 20 are encoded by
the universal genetic_code. The remaining 2, selenocysteine and pyrrolysine,
are incorporated into proteins by unique synthetic mechanisms. Selenocysteine
is incorporated when the mRNA being translated includes a SECIS_element, which
causes the UGA codon to encode selenocysteine instead of a stop_codon.[51]
Pyrrolysine is used by some methanogenic archaea in enzymes that they use to
produce methane. It is coded for with the codon UAG, which is normally a stop
codon in other organisms.[52] This UAG codon is followed by a PYLIS_downstream
sequence.[53]
**** Non-proteinogenic amino acids[edit] ****
Main article: Non-proteinogenic_amino_acids
Aside from the 22 proteinogenic_amino_acids, many non-proteinogenic amino acids
are known. Those either are not found in proteins (for example carnitine, GABA,
levothyroxine) or are not produced directly and in isolation by standard
cellular machinery (for example, hydroxyproline and selenomethionine).
Non-proteinogenic amino acids that are found in proteins are formed by post-
translational_modification, which is modification after translation during
protein synthesis. These modifications are often essential for the function or
regulation of a protein. For example, the carboxylation of glutamate allows for
better binding of calcium_cations,[54] and collagen contains hydroxyproline,
generated by hydroxylation of proline.[55] Another example is the formation of
hypusine in the translation_initiation_factor EIF5A, through modification of a
lysine residue.[56] Such modifications can also determine the localization of
the protein, e.g., the addition of long hydrophobic groups can cause a protein
to bind to a phospholipid membrane.[57]
Some non-proteinogenic amino acids are not found in proteins. Examples include
2-aminoisobutyric_acid and the neurotransmitter gamma-aminobutyric_acid. Non-
proteinogenic amino acids often occur as intermediates in the metabolic
pathways for standard amino acids â for example, ornithine and citrulline
occur in the urea_cycle, part of amino acid catabolism (see below).[58] A rare
exception to the dominance of Î±-amino acids in biology is the Î²-amino acid
beta_alanine (3-aminopropanoic acid), which is used in plants and
microorganisms in the synthesis of pantothenic_acid (vitamin B5), a component
of coenzyme_A.[59]
**** D-amino acid natural abundance[edit] ****
                        This section needs to be updated. Please update this
[Ambox current red.svg] article to reflect recent events or newly available
                        information. (July 2019)
Although D-isomers are uncommon in live organisms, gramicidin is a polypeptide
made up from mixture of D- and L-amino acids.[60] Other compounds containing D-
amino acids are tyrocidine and valinomycin. These compounds disrupt bacterial
cell walls, particularly in Gram-positive bacteria. As of 2011[update], only
837 D-amino acids were found in the Swiss-Prot database out of a total of 187
million amino acids analysed.[61]
**** Non-standard amino acids[edit] ****
The 20 amino acids that are encoded directly by the codons of the universal
genetic_code are called standard or canonical amino acids. A modified form of
methionine (N-formylmethionine) is often incorporated in place of methionine as
the initial amino acid of proteins in bacteria, mitochondria and chloroplasts.
Other amino acids are called non-standard or non-canonical. Most of the non-
standard amino acids are also non-proteinogenic (i.e. they cannot be
incorporated into proteins during translation), but two of them are
proteinogenic, as they can be incorporated translationally into proteins by
exploiting information not encoded in the universal genetic code.
The two non-standard proteinogenic amino acids are selenocysteine (present in
many non-eukaryotes as well as most eukaryotes, but not coded directly by DNA)
and pyrrolysine (found only in some archaea and one bacterium). The
incorporation of these non-standard amino acids is rare. For example, 25 human
proteins include selenocysteine (Sec) in their primary structure,[62] and the
structurally characterized enzymes (selenoenzymes) employ Sec as the catalytic
moiety in their active sites.[63] Pyrrolysine and selenocysteine are encoded
via variant codons. For example, selenocysteine is encoded by stop_codon and
SECIS_element.[13][14][15]
**** In human nutrition[edit] ****
[Diagram_showing_the_relative_occurrence_of_different_amino_acids_in_blood
serum_as_obtained_from_different_diets]
Share of amino acid in different human diets and the resulting mix of amino
acids in human blood serum. Glutamate and glutamine are the most frequent in
food at over 10%, while alanine, glutamine, and glycine are the most common in
blood.
Main article: Essential_amino_acids
Further information: Protein_(nutrient) and Amino_acid_synthesis
When taken up into the human body from the diet, the 20 standard amino acids
either are used to synthesize proteins, other biomolecules, or are oxidized to
urea and carbon_dioxide as a source of energy.[64] The oxidation pathway starts
with the removal of the amino group by a transaminase; the amino group is then
fed into the urea_cycle. The other product of transamidation is a keto_acid
that enters the citric_acid_cycle.[65] Glucogenic_amino_acids can also be
converted into glucose, through gluconeogenesis.[66] Of the 20 standard amino
acids, nine (His, Ile, Leu, Lys, Met, Phe, Thr, Trp and Val) are called
essential_amino_acids because the human_body cannot synthesize them from other
compounds at the level needed for normal growth, so they must be obtained from
food.[67][68][69] In addition, cysteine, taurine, tyrosine, and arginine are
considered semiessential amino-acids in children (though taurine is not
technically an amino acid), because the metabolic pathways that synthesize
these amino acids are not fully developed.[70][71] The amounts required also
depend on the age and health of the individual, so it is hard to make general
statements about the dietary requirement for some amino acids. Dietary exposure
to the non-standard amino acid BMAA has been linked to human neurodegenerative
diseases, including ALS.[72][73]
[Signaling_cascade_diagram]
Diagram of the molecular signaling_cascades that are involved in myofibrillar
muscle protein synthesis and mitochondrial_biogenesis in response to physical
exercise and specific amino acids or their derivatives (primarily L-leucine and
HMB).[74] Many amino acids derived from food protein promote the activation of
mTORC1 and increase protein_synthesis by signaling through Rag_GTPases.[74][75]
Abbreviations and representations:
 • PLD: phospholipase_D
 • PA: phosphatidic_acid
 • mTOR: mechanistic_target_of_rapamycin
 • AMP: adenosine_monophosphate
 • ATP: adenosine_triphosphate
 • AMPK: AMP-activated_protein_kinase
 • PGCâ1Î±: peroxisome_proliferator-activated_receptor_gamma_coactivator-1Î±
 • S6K1: p70S6_kinase
 • 4EBP1: eukaryotic_translation_initiation_factor_4E-binding_protein_1
 • eIF4E: eukaryotic_translation_initiation_factor_4E
 • RPS6: ribosomal_protein_S6
 • eEF2: eukaryotic_elongation_factor_2
 • RE: resistance exercise; EE: endurance exercise
 • Myo: myofibrillar; Mito: mitochondrial
 • AA: amino acids
 • HMB: Î²-hydroxy_Î²-methylbutyric_acid
 • â represents activation
 • Î¤ represents inhibition
[Graph_of_muscle_protein_synthesis_vs_time]
Resistance training stimulates muscle protein synthesis (MPS) for a period of
up to 48 hours following exercise (shown by lighter dotted line).[76] Ingestion
of a protein-rich meal at any point during this period will augment the
exercise-induced increase in muscle protein synthesis (shown by solid lines).
[76]
**** Non-protein functions[edit] ****
Biosynthetic pathways for catecholamines and trace_amines in the human_brain
[77][78][79]
[Graphic_of_catecholamine_and_trace_amine_biosynthesis]
L-Phenylalanine
L-Tyrosine
L-DOPA
Epinephrine
Phenethylamine
p-Tyramine
Dopamine
Norepinephrine
N-Methylphenethylamine
N-Methyltyramine
p-Octopamine
Synephrine
3-Methoxytyramine
AADC
AADC
AADC
primary
pathway
PNMT
PNMT
PNMT
PNMT
AAAH
AAAH
brain
CYP2D6
minor
pathway
COMT
DBH
DBH
[The image above contains clickable links]
Catecholamines and trace_amines are synthesized from phenylalanine and tyrosine
in humans.
Further information: Amino_acid_neurotransmitter
In humans, non-protein amino acids also have important roles as metabolic
intermediates, such as in the biosynthesis of the neurotransmitter gamma-amino-
butyric_acid (GABA). Many amino acids are used to synthesize other molecules,
for example:
    * Tryptophan is a precursor of the neurotransmitter serotonin.[80]
    * Tyrosine (and its precursor phenylalanine) are precursors of the
      catecholamine neurotransmitters dopamine, epinephrine and norepinephrine
      and various trace_amines.
    * Phenylalanine is a precursor of phenethylamine and tyrosine in humans. In
      plants, it is a precursor of various phenylpropanoids, which are
      important in plant metabolism.
    * Glycine is a precursor of porphyrins such as heme.[81]
    * Arginine is a precursor of nitric_oxide.[82]
    * Ornithine and S-adenosylmethionine are precursors of polyamines.[83]
    * Aspartate, glycine, and glutamine are precursors of nucleotides.[84]
      However, not all of the functions of other abundant non-standard amino
      acids are known.
Some non-standard amino acids are used as defenses_against_herbivores in
plants.[85] For example, canavanine is an analogue of arginine that is found in
many legumes,[86] and in particularly large amounts in Canavalia_gladiata
(sword bean).[87] This amino acid protects the plants from predators such as
insects and can cause illness in people if some types of legumes are eaten
without processing.[88] The non-protein amino acid mimosine is found in other
species of legume, in particular Leucaena_leucocephala.[89] This compound is an
analogue of tyrosine and can poison animals that graze on these plants.
***** Uses in industry[edit] *****
Amino acids are used for a variety of applications in industry, but their main
use is as additives to animal_feed. This is necessary, since many of the bulk
components of these feeds, such as soybeans, either have low levels or lack
some of the essential_amino_acids: lysine, methionine, threonine, and
tryptophan are most important in the production of these feeds.[90] In this
industry, amino acids are also used to chelate metal cations in order to
improve the absorption of minerals from supplements, which may be required to
improve the health or production of these animals.[91]
The food_industry is also a major consumer of amino acids, in particular,
glutamic_acid, which is used as a flavor_enhancer,[92] and aspartame (aspartyl-
phenylalanine-1-methyl ester) as a low-calorie artificial_sweetener.[93]
Similar technology to that used for animal nutrition is employed in the human
nutrition industry to alleviate symptoms of mineral deficiencies, such as
anemia, by improving mineral absorption and reducing negative side effects from
inorganic mineral supplementation.[94]
The chelating ability of amino acids has been used in fertilizers for
agriculture to facilitate the delivery of minerals to plants in order to
correct mineral deficiencies, such as iron chlorosis. These fertilizers are
also used to prevent deficiencies from occurring and improving the overall
health of the plants.[95] The remaining production of amino acids is used in
the synthesis of drugs and cosmetics.[90]
Similarly, some amino acids derivatives are used in pharmaceutical industry.
They include 5-HTP (5-hydroxytryptophan) used for experimental treatment of
depression,[96] L-DOPA (L-dihydroxyphenylalanine) for Parkinson's treatment,
[97] and eflornithine drug that inhibits ornithine_decarboxylase and used in
the treatment of sleeping_sickness.[98]
**** Expanded genetic code[edit] ****
Main article: Expanded_genetic_code
Since 2001, 40 non-natural amino acids have been added into protein by creating
a unique codon (recoding) and a corresponding transfer-RNA:aminoacyl â tRNA-
synthetase pair to encode it with diverse physicochemical and biological
properties in order to be used as a tool to exploring protein_structure and
function or to create novel or enhanced proteins.[16][17]
**** Nullomers[edit] ****
Main article: Nullomers
Nullomers are codons that in theory code for an amino acid, however in nature
there is a selective bias against using this codon in favor of another, for
example bacteria prefer to use CGA instead of AGA to code for arginine.[99]
This creates some sequences that do not appear in the genome. This
characteristic can be taken advantage of and used to create new selective
cancer-fighting drugs[100] and to prevent cross-contamination of DNA samples
from crime-scene investigations.[101]
**** Chemical building blocks[edit] ****
Further information: Asymmetric_synthesis
Amino acids are important as low-cost feedstocks. These compounds are used in
chiral_pool_synthesis as enantiomerically_pure building-blocks.[102]
Amino acids have been investigated as precursors chiral_catalysts, e.g., for
asymmetric hydrogenation reactions, although no commercial applications exist.
[103]
**** Biodegradable plastics[edit] ****
Further information: Biodegradable_plastic and Biopolymer
Amino acids are under development as components of a range of biodegradable
polymers. These materials have applications as environmentally_friendly
packaging and in medicine in drug_delivery and the construction of prosthetic
implants. These polymers include polypeptides, polyamides, polyesters,
polysulfides, and polyurethanes with amino acids either forming part of their
main chains or bonded as side chains. These modifications alter the physical
properties and reactivities of the polymers.[104] An interesting example of
such materials is polyaspartate, a water-soluble biodegradable polymer that may
have applications in disposable diapers and agriculture.[105] Due to its
solubility and ability to chelate metal ions, polyaspartate is also being used
as a biodegradeable anti-scaling agent and a corrosion_inhibitor.[106][107] In
addition, the aromatic amino acid tyrosine is being developed as a possible
replacement for toxic phenols such as bisphenol_A in the manufacture of
polycarbonates.[108]
***** Synthesis[edit] *****
Main article: Amino_acid_synthesis
[For_the_steps_in_the_reaction,_see_the_text.]
The Strecker amino acid synthesis
**** Chemical synthesis[edit] ****
The commercial production of amino acids usually relies on mutant bacteria that
overproduce individual amino acids using glucose as a carbon source. Some amino
acids are produced by enzymatic conversions of synthetic intermediates. 2-
Aminothiazoline-4-carboxylic_acid is an intermediate in one industrial
synthesis of L-cysteine for example. Aspartic_acid is produced by the addition
of ammonia to fumarate using a lyase.[109]
**** Biosynthesis[edit] ****
In plants, nitrogen is first assimilated into organic compounds in the form of
glutamate, formed from alpha-ketoglutarate and ammonia in the mitochondrion.
For other amino acids, plants use transaminases to move the amino group from
glutamate to another alpha-keto acids. For example, aspartate aminotransferase
converts glutamate and oxaloacetate to alpha-ketoglutarate and aspartate.[110]
Other organisms use transaminases for amino acid synthesis, too.
Nonstandard amino acids are usually formed through modifications to standard
amino acids. For example, homocysteine is formed through the transsulfuration
pathway or by the demethylation of methionine via the intermediate metabolite
S-adenosyl_methionine,[111] while hydroxyproline is made by a posttranslational
modification of proline.[112]
Microorganisms and plants synthesize many uncommon amino acids. For example,
some microbes make 2-aminoisobutyric_acid and lanthionine, which is a sulfide-
bridged derivative of alanine. Both of these amino acids are found in peptidic
lantibiotics such as alamethicin.[113] However, in plants, 1-aminocyclopropane-
1-carboxylic_acid is a small disubstituted cyclic amino acid that is a key
intermediate in the production of the plant hormone ethylene.[114]
***** Reactions[edit] *****
Amino acids undergo the reactions expected of the constituent functional
groups.[115][116] The types of these reactions are determined by the groups on
these side chains and are, therefore, different between the various types of
amino acid.
**** Peptide bond formation[edit] ****
See also: Peptide_synthesis and Peptide_bond
[Two_amino_acids_are_shown_next_to_each_other._One_loses_a_hydrogen_and_oxygen
from_its_carboxyl_group_(COOH)_and_the_other_loses_a_hydrogen_from_its_amino
group_(NH2)._This_reaction_produces_a_molecule_of_water_(H2O)_and_two_amino
acids_joined_by_a_peptide_bond_(-CO-NH-)._The_two_joined_amino_acids_are_called
a_dipeptide.]
The condensation of two amino acids to form a dipeptide through a peptide_bond
As both the amine and carboxylic acid groups of amino acids can react to form
amide bonds, one amino acid molecule can react with another and become joined
through an amide linkage. This polymerization of amino acids is what creates
proteins. This condensation_reaction yields the newly formed peptide_bond and a
molecule of water. In cells, this reaction does not occur directly; instead,
the amino acid is first activated by attachment to a transfer_RNA molecule
through an ester bond. This aminoacyl-tRNA is produced in an ATP-dependent
reaction carried out by an aminoacyl_tRNA_synthetase.[117] This aminoacyl-tRNA
is then a substrate for the ribosome, which catalyzes the attack of the amino
group of the elongating protein chain on the ester bond.[118] As a result of
this mechanism, all proteins made by ribosomes are synthesized starting at
their N-terminus and moving toward their C-terminus.
However, not all peptide bonds are formed in this way. In a few cases, peptides
are synthesized by specific enzymes. For example, the tripeptide glutathione is
an essential part of the defenses of cells against oxidative stress. This
peptide is synthesized in two steps from free amino acids.[119] In the first
step, gamma-glutamylcysteine_synthetase condenses cysteine and glutamic_acid
through a peptide bond formed between the side chain carboxyl of the glutamate
(the gamma carbon of this side chain) and the amino group of the cysteine. This
dipeptide is then condensed with glycine by glutathione_synthetase to form
glutathione.[120]
In chemistry, peptides are synthesized by a variety of reactions. One of the
most-used in solid-phase_peptide_synthesis uses the aromatic oxime derivatives
of amino acids as activated units. These are added in sequence onto the growing
peptide chain, which is attached to a solid resin support.[121] The ability to
easily synthesize vast numbers of different peptides by varying the types and
order of amino acids (using combinatorial_chemistry) has made peptide synthesis
particularly important in creating libraries of peptides for use in drug
discovery through high-throughput_screening.[122]
The combination of functional groups allow amino acids to be effective
polydentate ligands for metal-amino acid chelates.[123] The multiple side
chains of amino acids can also undergo chemical reactions.
**** Catabolism[edit] ****
Catabolism of proteinogenic amino acids. Amino acids can be classified
according to the properties of their main products as either of the following:
[124]
* Glucogenic, with the products having the ability to form glucose by
gluconeogenesis
* Ketogenic, with the products not having the ability to form glucose. These
products may still be used for ketogenesis or lipid_synthesis.
* Amino acids catabolized into both glucogenic and ketogenic products.
Amino acids must first pass out of organelles and cells into blood circulation
via amino_acid_transporters, since the amine and carboxylic acid groups are
typically ionized. Degradation of an amino acid, occurring in the liver and
kidneys, often involves deamination by moving its amino group to alpha-
ketoglutarate, forming glutamate. This process involves transaminases, often
the same as those used in amination during synthesis. In many vertebrates, the
amino group is then removed through the urea_cycle and is excreted in the form
of urea. However, amino acid degradation can produce uric_acid or ammonia
instead. For example, serine_dehydratase converts serine to pyruvate and
ammonia.[84] After removal of one or more amino groups, the remainder of the
molecule can sometimes be used to synthesize new amino acids, or it can be used
for energy by entering glycolysis or the citric_acid_cycle, as detailed in
image at right.
**** Complexation[edit] ****
Amino acids are bidentate ligands, forming transition_metal_amino_acid
complexes.[125]
      [AAcomplexation.png]
***** Physicochemical properties of amino acids[edit] *****
The 20 amino acids encoded directly by the genetic code can be divided into
several groups based on their properties. Important factors are charge,
hydrophilicity or hydrophobicity, size, and functional groups.[38] These
properties are important for protein_structure and proteinâprotein
interactions. The water-soluble proteins tend to have their hydrophobic
residues (Leu, Ile, Val, Phe, and Trp) buried in the middle of the protein,
whereas hydrophilic side chains are exposed to the aqueous solvent. (Note that
in biochemistry, a residue refers to a specific monomer within the polymeric
chain of a polysaccharide, protein or nucleic_acid.) The integral_membrane
proteins tend to have outer rings of exposed hydrophobic amino acids that
anchor them into the lipid_bilayer. In the case part-way between these two
extremes, some peripheral_membrane_proteins have a patch of hydrophobic amino
acids on their surface that locks onto the membrane. In similar fashion,
proteins that have to bind to positively charged molecules have surfaces rich
with negatively charged amino acids like glutamate and aspartate, while
proteins binding to negatively charged molecules have surfaces rich with
positively charged chains like lysine and arginine. There are different
hydrophobicity_scales of amino acid residues.[126]
Some amino acids have special properties such as cysteine, that can form
covalent disulfide_bonds to other cysteine residues, proline that forms a_cycle
to the polypeptide backbone, and glycine that is more flexible than other amino
acids.
Many proteins undergo a range of posttranslational_modifications, when
additional chemical groups are attached to the amino acids in proteins. Some
modifications can produce hydrophobic lipoproteins,[127] or hydrophilic
glycoproteins.[128] These type of modification allow the reversible targeting
of a protein to a membrane. For example, the addition and removal of the fatty
acid palmitic_acid to cysteine residues in some signaling proteins causes the
proteins to attach and then detach from cell membranes.[129]
**** Table of standard amino acid abbreviations and properties[edit] ****
Main article: Proteinogenic_amino_acid
                                                                                                                                          Coding in
                                                Side                                                                                      the
              3-     1-                Side     chain                                                                          Occurrence Standard
Amino acid    letter letter Side chain chain    charge   Hydropathy Absorbance       Îµ at                          MW       in         Genetic
              [130]  [130]  class      polarity (pH 7.4) index[131] Î»max(nm)[132] Î»max (mMâ1 cmâ1)[132(weight) proteins   Code
                                       [130]    [130]                                                                          (%)[133]   (using
                                                                                                                                          IUPAC
                                                                                                                                          notation)
Alanine       Ala    A      aliphatic  nonpolar neutral  1.8                                                          89.094   8.76       GCN
                                                                                                                                          MGN, CGY
                                                                                                                                          (coding
                                                                                                                                          codons
Arginine      Arg    R      basic      basic    positive â4.5                                                    174.203  5.78       can also
                                       polar                                                                                              be
                                                                                                                                          expressed
                                                                                                                                          by: CGN,
                                                                                                                                          AGR)
Asparagine    Asn    N      amide      polar    neutral  â3.5                                                    132.119  3.93       AAY
Aspartic_acid Asp    D      acid       acidic   negative â3.5                                                    133.104  5.49       GAY
                                       polar
Cysteine      Cys    C      sulfur-    nonpolar neutral  2.5        250              0.3                              121.154  1.38       UGY
                            containing
Glutamine     Gln    Q      amide      polar    neutral  â3.5                                                    146.146  3.9        CAR
Glutamic_acid Glu    E      acid       acidic   negative â3.5                                                    147.131  6.32       GAR
                                       polar
Glycine       Gly    G      aliphatic  nonpolar neutral  â0.4                                                    75.067   7.03       GGN
                                                positive
Histidine     His    H      basic      basic    (10%)    â3.2  211              5.9                              155.156  2.26       CAY
                            aromatic   polar    neutral
                                                (90%)
Isoleucine    Ile    I      aliphatic  nonpolar neutral  4.5                                                          131.175  5.49       AUH
                                                                                                                                          YUR, CUY
                                                                                                                                          (coding
                                                                                                                                          codons
Leucine       Leu    L      aliphatic  nonpolar neutral  3.8                                                          131.175  9.68       can also
                                                                                                                                          be
                                                                                                                                          expressed
                                                                                                                                          by: CUN,
                                                                                                                                          UUR)
Lysine        Lys    K      basic      basic    positive â3.9                                                    146.189  5.19       AAR
                                       polar
Methionine    Met    M      sulfur-    nonpolar neutral  1.9                                                          149.208  2.32       AUG
                            containing
Phenylalanine Phe    F      aromatic   nonpolar neutral  2.8        257, 206, 188    0.2, 9.3, 60.0                   165.192  3.87       UUY
Proline       Pro    P      cyclic     nonpolar neutral  â1.6                                                    115.132  5.02       CCN
Serine        Ser    S      hydroxyl-  polar    neutral  â0.8                                                    105.093  7.14       UCN, AGY
                            containing
Threonine     Thr    T      hydroxyl-  polar    neutral  â0.7                                                    119.119  5.53       ACN
                            containing
Tryptophan    Trp    W      aromatic   nonpolar neutral  â0.9  280, 219         5.6, 47.0                        204.228  1.25       UGG
Tyrosine      Tyr    Y      aromatic   polar    neutral  â1.3  274, 222, 193    1.4, 8.0, 48.0                   181.191  2.91       UAY
Valine        Val    V      aliphatic  nonpolar neutral  4.2                                                          117.148  6.73       GUN
Two additional amino acids are in some species coded for by codons that are
usually interpreted as stop_codons:
21st and 22nd amino acids 3-letter 1-letter MW (weight)
Selenocysteine            Sec      U        168.064
Pyrrolysine               Pyl      O        255.313
In addition to the specific amino acid codes, placeholders are used in cases
where chemical or crystallographic analysis of a peptide or protein cannot
conclusively determine the identity of a residue. They are also used to
summarise conserved_protein_sequence motifs. The use of single letters to
indicate sets of similar residues is similar to the use of abbreviation_codes
for_degenerate_bases.[134][135]
Ambiguous amino    3-letter 1-letter Amino Acids Included Codons Included
acids
Any / unknown      Xaa      X        All                  NNN
Asparagine or      Asx      B        D, N                 RAY
aspartic acid
Glutamine or       Glx      Z        E, Q                 SAR
glutamic acid
                                                          YTR, ATH, CTY (coding
Leucine or                                                codons can also be
Isoleucine         Xle      J        I, L                 expressed by: CTN,
                                                          ATH, TTR; MTY, YTR,
                                                          ATA; MTY, HTA, YTG)
Hydrophobic                 Î¦     V, I, L, F, W, Y, M  NTN, TAY, TGG
                                                          YWY, TTY, TGG (coding
Aromatic                    Î©     F, W, Y, H           codons can also be
                                                          expressed by: TWY,
                                                          CAY, TGG)
                                                          VTN, TTR (coding
Aliphatic (non-             Î¨     V, I, L, M           codons can also be
aromatic)                                                 expressed by: NTR,
                                                          VTY)
Small                       Ï     P, G, A, S           BCN, RGY, GGR
                                                          VAN, WCN, CGN, AGY
Hydrophilic                 Î¶     S, T, H, N, Q, E, D, (coding codons can
                                     K, R                 also be expressed by:
                                                          VAN, WCN, MGY, CGP)
Positively_charged          +        K, R, H              ARR, CRY, CGR
Negatively_charged          â   D, E                 GAN
Unk is sometimes used instead of Xaa, but is less standard.
In addition, many non-standard_amino_acids have a specific code. For example,
several peptide drugs, such as Bortezomib and MG132, are artificially
synthesized and retain their protecting_groups, which have specific codes.
Bortezomib is Pyz-Phe-boroLeu, and MG132 is Z-Leu-Leu-Leu-al. To aid in the
analysis of protein structure, photo-reactive_amino_acid_analogs are available.
These include photoleucine (pLeu) and photomethionine (pMet).[136]
***** See also[edit] *****
    * Molecular_and_Cellular_Biology_portal
    * Chemistry_portal
    * Amino_acid_dating
    * Beta-peptide
    * Degron
    * Erepsin
    * Homochirality
    * Hyperaminoacidemia
    * Leucines
    * MillerâUrey_experiment
    * Nucleic_acid_sequence
    * Proteinogenic_amino_acid
    * Table_of_codons, 3-nucleotide sequences that encode each amino acid
***** References and notes[edit] *****
   1. ^"Amino". Dictionary.com. 2015. Retrieved 3 July 2015.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^"amino_acid". Cambridge Dictionaries Online. Cambridge University Press.
      2015. Retrieved 3 July 2015.
   4. ^"amino". FreeDictionary.com. Farlex. 2015. Retrieved 3 July 2015.
   5. ^Wagner I, Musso H (November 1983). "New Naturally Occurring Amino
      Acids". Angewandte_Chemie_International_Edition_in_English. 22 (11):
      816â28. doi:10.1002/anie.198308161.
   6. [closed_access]
   7. ^Latham MC (1997). "Chapter_8._Body_composition,_the_functions_of_food,
      metabolism_and_energy". Human nutrition in the developing world. Food and
      Nutrition Series â No. 29. Rome: Food and Agriculture Organization of
      the United Nations.
   8. ^ Proline is an exception to this general formula. It lacks the NH2 group
      because of the cyclization of the side chain and is known as an imino
      acid; it falls under the category of special structured amino acids.
   9. ^Clark, Jim (August 2007). "an_introduction_to_amino_acids". chemguide.
      Retrieved 4 July 2015.
  10. ^Jakubke H, Sewald N (2008). "Amino_acids". Peptides from A to Z: A
      Concise Encyclopedia. Germany: Wiley-VCH. p. 20. ISBN 9783527621170 – via
      Google Books.
  11. ^Pollegioni L, Servi S, eds. (2012). Unnatural Amino Acids: Methods and
      Protocols. Methods in Molecular Biology â Volume 794. 794. Humana
      Press. p. v. doi:10.1007/978-1-61779-331-8. ISBN 978-1-61779-331-8.
      OCLC 756512314.
  12. ^Hertweck C (October 2011). "Biosynthesis and Charging of Pyrrolysine,
      the 22nd Genetically Encoded Amino Acid". Angewandte_Chemie_International
      Edition. 50 (41): 9540â1. doi:10.1002/anie.201103769. PMID 21796749.
  13. [closed_access]
  14. ^"Chapter_1:_Proteins_are_the_Body's_Worker_Molecules". The Structures of
      Life. National Institute of General Medical Sciences. 27 October 2011.
      Retrieved 20 May 2008.
  15. ^Michal G, Schomburg D, eds. (2012). Biochemical Pathways: An Atlas of
      Biochemistry and Molecular Biology (2nd ed.). Oxford: Wiley-Blackwell.
      p. 5. ISBN 978-0-470-14684-2.
  16. ^ a b Modeling_Electrostatic_Contributions_to_Protein_Folding_and_Binding
      â Tjong, p.1 footnote
  17. ^ a b Frontiers_in_Drug_Design_and_Discovery ed. Atta-Ur-Rahman & others,
      p.299
  18. ^ a bElzanowski A, Ostell J (7 April 2008). "The_Genetic_Codes". National
      Center for Biotechnology Information (NCBI). Retrieved 10 March 2010.
  19. ^ a bXie J, Schultz PG (December 2005). "Adding amino acids to the
      genetic repertoire". Current Opinion in Chemical Biology. 9 (6):
      548â54. doi:10.1016/j.cbpa.2005.10.011. PMID 16260173.
  20. ^ a bWang Q, Parrish AR, Wang L (March 2009). "Expanding_the_genetic_code
      for_biological_studies". Chemistry & Biology. 16 (3): 323â36. doi:
      10.1016/j.chembiol.2009.03.001. PMC 2696486. PMID 19318213.
  21. ^Simon M (2005). Emergent computation: emphasizing bioinformatics. New
      York: AIP Press/Springer Science+Business Media. pp. 105â106. ISBN 978-
      0-387-22046-8.
  22. ^Petroff OA (December 2002). "GABA_and_glutamate_in_the_human_brain". The
      Neuroscientist. 8 (6): 562â73. doi:10.1177/1073858402238515.
      PMID 12467378.
  23. ^ For example, ruminants such as cows obtain a number of amino acids via
      microbes in the first_two_stomach_chambers.
  24. ^Vickery HB, Schmidt CL (1931). "The history of the discovery of the
      amino acids". Chem. Rev. 9 (2): 169â318. doi:10.1021/cr60033a001.
  25. ^ Hansen, Sabine. Die Entdeckung der proteinogenen AminosÃ¤uren von 1805
      in Paris bis 1935 in Illinois. Berlin, im Mai 2015. link.
  26. ^Vauquelin LN, Robiquet PJ (1806). "The discovery of a new plant
      principle in Asparagus sativus". Annales de Chimie. 57: 88â93.
  27. ^ a bAnfinsen CB, Edsall JT, Richards FM (1972). Advances_in_Protein
      Chemistry. New York: Academic Press. pp. 99,_103. ISBN 978-0-12-034226-6.
  28. ^Wollaston WH (1810). "On cystic oxide, a new species of urinary
      calculus". Philosophical Transactions of the Royal Society. 100:
      223â30. doi:10.1098/rstl.1810.0015.
  29. ^Baumann E (1884). "Ãber_cystin_und_cystein". Z Physiol Chem. 8 (4):
      299â305. Archived from the original on 14 March 2011. Retrieved 28
      March 2011.
  30. ^Braconnot HM (1820). "Sur la conversion des matiÃ¨res animales en
      nouvelles substances par le moyen de l'acide sulfurique". Annales de
      Chimie et de Physique. 2nd Series. 13: 113â25.
  31. ^Simoni RD, Hill RL, Vaughan M (September 2002). "The_discovery_of_the
      amino_acid_threonine:_the_work_of_William_C._Rose_[classical_article]".
      The Journal of Biological Chemistry. 277 (37): E25. PMID 12218068.
  32. ^McCoy RH, Meyer CE, Rose WC (1935). "Feeding Experiments with Mixtures
      of Highly Purified Amino Acids. VIII. Isolation and Identification of a
      New Essential Amino Acid". Journal of Biological Chemistry. 112:
      283â302.
  33. ^ Menten, P. Dictionnaire de chimie: Une approche Ã©tymologique et
      historique. De Boeck, Bruxelles. link.
  34. ^"etymonline.com_entry_for_amino". www.etymonline.com. Retrieved 19 July
      2010.
  35. ^Paal C (1894). "Ueber die Einwirkung von Phenylâiâcyanat auf
      organische AminosÃ¤uren". Berichte der Deutschen Chemischen Gesellschaft.
      27: 974â979. doi:10.1002/cber.189402701205.
  36. ^Fruton JS (1990). "Chapter 5- Emil Fischer and Franz Hofmeister".
      Contrasts in Scientific Style: Research Groups in the Chemical and
      Biochemical Sciences. 191. American Philosophical Society. pp. 163â165.
      ISBN 978-0-87169-191-0.
  37. ^"Alpha_amino_acid_â_Medical_definition". Merriam-Webster dictionary.
  38. ^ Proline at the US National Library of Medicine Medical_Subject_Headings
      (MeSH)
  39. ^"Archived_copy". Archived from the_original on 18 January 2008.
      Retrieved 3 January 2015.CS1 maint: Archived copy as title (link)
  40. ^ IUPAC, Compendium_of_Chemical_Terminology, 2nd ed. (the "Gold Book")
      (1997). Online corrected version:  (2006–) "Imino_acids". doi:10.1351/
      goldbook.I02959
  41. ^ a b c d e fCreighton TH (1993). "Chapter 1". Proteins: structures and
      molecular properties. San Francisco: W. H. Freeman. ISBN 978-0-7167-7030-
      5.
  42. ^Pisarewicz K, Mora D, Pflueger FC, Fields GB, MarÃ­ F (May 2005).
      "Polypeptide chains containing D-gamma-hydroxyvaline". Journal of the
      American Chemical Society. 127 (17): 6207â15. doi:10.1021/ja050088m.
      PMID 15853325.
  43. ^van Heijenoort J (March 2001). "Formation of the glycan chains in the
      synthesis of bacterial peptidoglycan". Glycobiology. 11 (3): 25Râ36R.
      doi:10.1093/glycob/11.3.25R. PMID 11320055.
  44. ^Wolosker H, Dumin E, Balan L, Foltyn VN (July 2008). "D-amino acids in
      the brain: D-serine in neurotransmission and neurodegeneration". The FEBS
      Journal. 275 (14): 3514â26. doi:10.1111/j.1742-4658.2008.06515.x.
      PMID 18564180.
  45. ^Matthews BW (June 2009). "Racemic_crystallographyâeasy_crystals_and
      easy_structures:_What's_not_to_like?". Protein Science. 18 (6): 1135â8.
      doi:10.1002/pro.125. PMC 2774423. PMID 19472321.
  46. ^Hatem, Salama Mohamed Ali (2006). "Gas_chromatographic_determination_of
      Amino_Acid_Enantiomers_in_tobacco_and_bottled_wines". University of
      Giessen. Retrieved 17 November 2008.
  47. ^"Nomenclature_and_Symbolism_for_Amino_Acids_and_Peptides". IUPAC-IUB
      Joint Commission on Biochemical Nomenclature. 1983. Archived from the
      original on 9 October 2008. Retrieved 17 November 2008.
  48. ^Jodidi, S. L. (1 March 1926). "The Formol Titration of Certain Amino
      Acids". Journal of the American Chemical Society. 48 (3): 751â753. doi:
      10.1021/ja01414a033.
  49. ^Liebecq C, ed. (1992). Biochemical Nomenclature and Related Documents
      (2nd ed.). Portland Press. pp. 39â69. ISBN 978-1-85578-005-7.
  50. ^Smith AD (1997). Oxford dictionary of biochemistry and molecular
      biology. Oxford: Oxford University Press. p. 535. ISBN 978-0-19-854768-6.
      OCLC 37616711.
  51. ^Simmons WJ, Meisenberg G (2006). Principles of medical biochemistry.
      Mosby Elsevier. p. 19. ISBN 978-0-323-02942-1.
  52. ^Fennema OR (19 June 1996). Food Chemistry 3rd Ed. CRC Press.
      pp. 327â8. ISBN 978-0-8247-9691-4.
  53. ^Rodnina MV, Beringer M, Wintermeyer W (January 2007). "How ribosomes
      make peptide bonds". Trends in Biochemical Sciences. 32 (1): 20â6. doi:
      10.1016/j.tibs.2006.11.007. PMID 17157507.
  54. ^Driscoll DM, Copeland PR (2003). "Mechanism and regulation of
      selenoprotein synthesis". Annual Review of Nutrition. 23 (1): 17â40.
      doi:10.1146/annurev.nutr.23.011702.073318. PMID 12524431.
  55. ^Krzycki JA (December 2005). "The direct genetic encoding of
      pyrrolysine". Current Opinion in Microbiology. 8 (6): 706â12. doi:
      10.1016/j.mib.2005.10.009. PMID 16256420.
  56. ^ThÃ©obald-Dietrich A, GiegÃ© R, Rudinger-Thirion J (2005). "Evidence for
      the existence in mRNAs of a hairpin element responsible for ribosome
      dependent pyrrolysine insertion into proteins". Biochimie. 87 (9â10):
      813â7. doi:10.1016/j.biochi.2005.03.006. PMID 16164991.
  57. ^Vermeer C (March 1990). "Gamma-carboxyglutamate-containing_proteins_and
      the_vitamin_K-dependent_carboxylase". The Biochemical Journal. 266 (3):
      625â36. doi:10.1042/bj2660625. PMC 1131186. PMID 2183788.
  58. ^Bhattacharjee A, Bansal M (March 2005). "Collagen structure: the Madras
      triple helix and the current scenario". IUBMB Life. 57 (3): 161â72.
      doi:10.1080/15216540500090710. PMID 16036578.
  59. ^Park MH (February 2006). "The_post-translational_synthesis_of_a
      polyamine-derived_amino_acid,_hypusine,_in_the_eukaryotic_translation
      initiation_factor_5A_(eIF5A)". Journal of Biochemistry. 139 (2): 161â9.
      doi:10.1093/jb/mvj034. PMC 2494880. PMID 16452303.
  60. ^Blenis J, Resh MD (December 1993). "Subcellular localization specified
      by protein acylation and phosphorylation". Current Opinion in Cell
      Biology. 5 (6): 984â9. doi:10.1016/0955-0674(93)90081-Z. PMID 8129952.
  61. ^Curis E, Nicolis I, Moinard C, Osowska S, Zerrouk N, BÃ©nazeth S,
      Cynober L (November 2005). "Almost all about citrulline in mammals".
      Amino Acids. 29 (3): 177â205. doi:10.1007/s00726-005-0235-4.
      PMID 16082501.
  62. ^Coxon KM, Chakauya E, Ottenhof HH, Whitney HM, Blundell TL, Abell C,
      Smith AG (August 2005). "Pantothenate biosynthesis in higher plants".
      Biochemical Society Transactions. 33 (Pt 4): 743â6. doi:10.1042/
      BST0330743. PMID 16042590.
  63. ^Ketchem RR, Hu W, Cross TA (September 1993). "High-resolution
      conformation of gramicidin A in a lipid bilayer by solid-state NMR".
      Science. 261 (5127): 1457â60. Bibcode:1993Sci...261.1457K. doi:10.1126/
      science.7690158. PMID 7690158.
  64. ^Khoury GA, Baliban RC, Floudas CA (September 2011). "Proteome-wide_post-
      translational_modification_statistics:_frequency_analysis_and_curation_of
      the_swiss-prot_database". Scientific Reports. 1 (90): 90. Bibcode:
      2011NatSR...1E..90K. doi:10.1038/srep00090. PMC 3201773. PMID 22034591.
  65. ^Kryukov GV, Castellano S, Novoselov SV, Lobanov AV, Zehtab O, GuigÃ³ R,
      Gladyshev VN (May 2003). "Characterization_of_mammalian_selenoproteomes".
      Science. 300 (5624): 1439â43. Bibcode:2003Sci...300.1439K. doi:10.1126/
      science.1083516. PMID 12775843.
  66. ^Gromer S, Urig S, Becker K (January 2004). "The thioredoxin
      systemâfrom science to clinic". Medicinal Research Reviews. 24 (1):
      40â89. doi:10.1002/med.10051. PMID 14595672.
  67. ^Sakami W, Harrington H (1963). "Amino acid metabolism". Annual Review of
      Biochemistry. 32 (1): 355â98. doi:10.1146/annurev.bi.32.070163.002035.
      PMID 14144484.
  68. ^Brosnan JT (April 2000). "Glutamate,_at_the_interface_between_amino_acid
      and_carbohydrate_metabolism". The Journal of Nutrition. 130 (4S Suppl):
      988Sâ90S. doi:10.1093/jn/130.4.988S. PMID 10736367.
  69. ^Young VR, Ajami AM (September 2001). "Glutamine:_the_emperor_or_his
      clothes?". The Journal of Nutrition. 131 (9 Suppl): 2449Sâ59S,
      discussion 2486Sâ7S. doi:10.1093/jn/131.9.2449S. PMID 11533293.
  70. ^Young VR (August 1994). "Adult_amino_acid_requirements:_the_case_for_a
      major_revision_in_current_recommendations". The Journal of Nutrition. 124
      (8 Suppl): 1517Sâ1523S. doi:10.1093/jn/124.suppl_8.1517S. PMID 8064412.
  71. ^FÃ¼rst P, Stehle P (June 2004). "What_are_the_essential_elements_needed
      for_the_determination_of_amino_acid_requirements_in_humans?". The Journal
      of Nutrition. 134 (6 Suppl): 1558Sâ1565S. doi:10.1093/jn/134.6.1558S.
      PMID 15173430.
  72. ^Reeds PJ (July 2000). "Dispensable_and_indispensable_amino_acids_for
      humans". The Journal of Nutrition. 130 (7): 1835Sâ40S. doi:10.1093/jn/
      130.7.1835S. PMID 10867060.
  73. ^Imura K, Okada A (January 1998). "Amino acid metabolism in pediatric
      patients". Nutrition. 14 (1): 143â8. doi:10.1016/S0899-9007(97)00230-X.
      PMID 9437700.
  74. ^LourenÃ§o R, Camilo ME (2002). "Taurine: a conditionally essential amino
      acid in humans? An overview in health and disease". Nutricion
      Hospitalaria. 17 (6): 262â70. PMID 12514918.
  75. ^Holtcamp W (March 2012). "The_emerging_science_of_BMAA:_do_cyanobacteria
      contribute_to_neurodegenerative_disease?". Environmental Health
      Perspectives. 120 (3): A110â6. doi:10.1289/ehp.120-a110. PMC 3295368.
      PMID 22382274.
  76. ^Cox PA, Davis DA, Mash DC, Metcalf JS, Banack SA (January 2016).
      "Dietary_exposure_to_an_environmental_toxin_triggers_neurofibrillary
      tangles_and_amyloid_deposits_in_the_brain". Proceedings: Biological
      Sciences. 283 (1823): 20152397. doi:10.1098/rspb.2015.2397. PMC 4795023.
      PMID 26791617.
  77. ^ a bBrook MS, Wilkinson DJ, Phillips BE, Perez-Schindler J, Philp A,
      Smith K, Atherton PJ (January 2016). "Skeletal_muscle_homeostasis_and
      plasticity_in_youth_and_ageing:_impact_of_nutrition_and_exercise". Acta
      Physiologica. 216 (1): 15â41. doi:10.1111/apha.12532. PMC 4843955.
      PMID 26010896.
  78. ^Lipton JO, Sahin M (October 2014). "The_neurology_of_mTOR". Neuron. 84
      (2): 275â91. doi:10.1016/j.neuron.2014.09.034. PMC 4223653.
      PMID 25374355.
  79.
      Figure_2:_The_mTOR_Signaling_Pathway
  80. ^ a bPhillips SM (May 2014). "A_brief_review_of_critical_processes_in
      exercise-induced_muscular_hypertrophy". Sports Medicine. 44 Suppl 1:
      S71â7. doi:10.1007/s40279-014-0152-3. PMC 4008813. PMID 24791918.
  81. ^Broadley KJ (March 2010). "The vascular effects of trace amines and
      amphetamines". Pharmacol. Ther. 125 (3): 363â375. doi:10.1016/
      j.pharmthera.2009.11.005. PMID 19948186.
  82. ^Lindemann L, Hoener MC (May 2005). "A renaissance in trace amines
      inspired by a novel GPCR family". Trends Pharmacol. Sci. 26 (5):
      274â281. doi:10.1016/j.tips.2005.03.007. PMID 15860375.
  83. ^Wang X, Li J, Dong G, Yue J (February 2014). "The endogenous substrates
      of brain CYP2D". Eur. J. Pharmacol. 724: 211â218. doi:10.1016/
      j.ejphar.2013.12.025. PMID 24374199.
  84. ^Savelieva KV, Zhao S, Pogorelov VM, Rajan I, Yang Q, Cullinan E,
      Lanthorn TH (2008). Bartolomucci A (ed.). "Genetic_disruption_of_both
      tryptophan_hydroxylase_genes_dramatically_reduces_serotonin_and_affects
      behavior_in_models_sensitive_to_antidepressants". PLOS ONE. 3 (10):
      e3301. Bibcode:2008PLoSO...3.3301S. doi:10.1371/journal.pone.0003301.
      PMC 2565062. PMID 18923670.
  85. ^Shemin D, Rittenberg D (December 1946). "The_biological_utilization_of
      glycine_for_the_synthesis_of_the_protoporphyrin_of_hemoglobin". The
      Journal of Biological Chemistry. 166 (2): 621â5. PMID 20276176.
  86. ^Tejero J, Biswas A, Wang ZQ, Page RC, Haque MM, Hemann C, Zweier JL,
      Misra S, Stuehr DJ (November 2008). "Stabilization_and_characterization
      of_a_heme-oxy_reaction_intermediate_in_inducible_nitric-oxide_synthase".
      The Journal of Biological Chemistry. 283 (48): 33498â507. doi:10.1074/
      jbc.M806122200. PMC 2586280. PMID 18815130.
  87. ^RodrÃ­guez-Caso C, MontaÃ±ez R, Cascante M, SÃ¡nchez-JimÃ©nez F, Medina
      MA (August 2006). "Mathematical modeling of polyamine metabolism in
      mammals". The Journal of Biological Chemistry. 281 (31): 21799â812.
      doi:10.1074/jbc.M602756200. PMID 16709566.
  88. ^ a bStryer L, Berg JM, Tymoczko JL (2002). Biochemistry (5th ed.). New
      York: W.H. Freeman. pp. 693â8. ISBN 978-0-7167-4684-3.
  89. ^Hylin, John W. (1969). "Toxic peptides and amino acids in foods and
      feeds". Journal of Agricultural and Food Chemistry. 17 (3): 492â6. doi:
      10.1021/jf60163a003.
  90. ^Turner BL, Harborne JB (1967). "Distribution of canavanine in the plant
      kingdom". Phytochemistry. 6 (6): 863â66. doi:10.1016/S0031-9422
      (00)86033-1.
  91. ^Ekanayake S, Skog K, Asp NG (May 2007). "Canavanine content in sword
      beans (Canavalia gladiata): analysis and effect of processing". Food and
      Chemical Toxicology. 45 (5): 797â803. doi:10.1016/j.fct.2006.10.030.
      PMID 17187914.
  92. ^Rosenthal GA (2001). "L-Canavanine: a higher plant insecticidal
      allelochemical". Amino Acids. 21 (3): 319â30. doi:10.1007/
      s007260170017. PMID 11764412.
  93. ^Hammond AC (May 1995). "Leucaena_toxicosis_and_its_control_in
      ruminants". Journal of Animal Science. 73 (5): 1487â92. doi:10.2527/
      1995.7351487x. PMID 7665380.
  94. [permanent_dead_link]
  95. ^ a bLeuchtenberger W, Huthmacher K, Drauz K (November 2005).
      "Biotechnological production of amino acids and derivatives: current
      status and prospects". Applied Microbiology and Biotechnology. 69 (1):
      1â8. doi:10.1007/s00253-005-0155-y. PMID 16195792.
  96. ^Ashmead HD (1993). The Role of Amino Acid Chelates in Animal Nutrition.
      Westwood: Noyes Publications.
  97. ^Garattini S (April 2000). "Glutamic_acid,_twenty_years_later". The
      Journal of Nutrition. 130 (4S Suppl): 901Sâ9S. doi:10.1093/jn/
      130.4.901S. PMID 10736350.
  98. ^Stegink LD (July 1987). "The_aspartame_story:_a_model_for_the_clinical
      testing_of_a_food_additive". The American Journal of Clinical Nutrition.
      46 (1 Suppl): 204â15. doi:10.1093/ajcn/46.1.204. PMID 3300262.
  99. ^Albion Laboratories, Inc. "Albion_Ferrochel_Website". Retrieved 12 July
      2011.
 100. ^Ashmead HD (1986). Foliar Feeding of Plants with Amino Acid Chelates.
      Park Ridge: Noyes Publications.
 101. ^Turner EH, Loftis JM, Blackwell AD (March 2006). "Serotonin_a_la_carte:
      supplementation_with_the_serotonin_precursor_5-hydroxytryptophan".
      Pharmacology & Therapeutics. 109 (3): 325â38. doi:10.1016/
      j.pharmthera.2005.06.004. PMID 16023217.
 102. ^Kostrzewa RM, Nowak P, Kostrzewa JP, Kostrzewa RA, Brus R (March 2005).
      "Peculiarities of L: -DOPA treatment of Parkinson's disease". Amino
      Acids. 28 (2): 157â64. doi:10.1007/s00726-005-0162-4. PMID 15750845.
 103. ^Heby O, Persson L, Rentala M (August 2007). "Targeting the polyamine
      biosynthetic enzymes: a promising approach to therapy of African sleeping
      sickness, Chagas' disease, and leishmaniasis". Amino Acids. 33 (2):
      359â66. doi:10.1007/s00726-007-0537-9. PMID 17610127.
 104. ^Cruz-Vera LR, Magos-Castro MA, Zamora-Romo E, Guarneros G (2004).
      "Ribosome_stalling_and_peptidyl-tRNA_drop-off_during_translational_delay
      at_AGA_codons". Nucleic Acids Research. 32 (15): 4462â8. doi:10.1093/
      nar/gkh784. PMC 516057. PMID 15317870.
 105. ^Andy C (October 2012). "Molecules_'too_dangerous_for_nature'_kill_cancer
      cells". New Scientist.
 106. ^"Lethal_DNA_tags_could_keep_innocent_people_out_of_jail". New Scientist.
      2 May 2013.
 107. ^Hanessian S (1993). "Reflections on the total synthesis of natural
      products: Art, craft, logic, and the chiron approach". Pure and Applied
      Chemistry. 65 (6): 1189â204. doi:10.1351/pac199365061189.
 108. ^Blaser HU (1992). "The chiral pool as a source of enantioselective
      catalysts and auxiliaries". Chemical Reviews. 92 (5): 935â52. doi:
      10.1021/cr00013a009.
 109. ^Sanda F, Endo T (1999). "Syntheses and functions of polymers based on
      amino acids". Macromolecular Chemistry and Physics. 200 (12): 2651â61.
      doi:10.1002/(SICI)1521-3935(19991201)200:12<2651::AID-MACP2651>3.0.CO;2-
      P.
 110. ^Gross RA, Kalra B (August 2002). "Biodegradable_polymers_for_the
      environment". Science. 297 (5582): 803â7. Bibcode:2002Sci...297..803G.
      doi:10.1126/science.297.5582.803. PMID 12161646.
 111. ^Low KC, Wheeler AP, Koskan LP (1996). Commercial poly(aspartic acid) and
      Its Uses. Advances in Chemistry Series. 248. Washington, D.C.: American
      Chemical_Society.
 112. ^Thombre SM, Sarwade BD (2005). "Synthesis and Biodegradability of
      Polyaspartic Acid: A Critical Review". Journal of Macromolecular Science,
      Part A. 42 (9): 1299â1315. doi:10.1080/10601320500189604.
 113. ^Bourke SL, Kohn J (April 2003). "Polymers derived from the amino acid L-
      tyrosine: polycarbonates, polyarylates and copolymers with poly(ethylene
      glycol)". Advanced Drug Delivery Reviews. 55 (4): 447â66. doi:10.1016/
      S0169-409X(03)00038-3. PMID 12706045.
 114. ^Karlheinz Drauz, Ian Grayson, Axel Kleemann, Hans-Peter Krimmer,
      Wolfgang Leuchtenberger, Christoph Weckbecker (2006). Ullmann's
      Encyclopedia_of_Industrial_Chemistry. Weinheim: Wiley-VCH. doi:10.1002/
      14356007.a02_057.pub2.CS1 maint: Multiple names: authors list (link)
 115. ^Jones RC, Buchanan BB, Gruissem W (2000). Biochemistry_&_molecular
      biology_of_plants. Rockville, Md: American Society of Plant
      Physiologists. pp. 371â2. ISBN 978-0-943088-39-6.
 116. ^Brosnan JT, Brosnan ME (June 2006). "The_sulfur-containing_amino_acids:
      an_overview". The Journal of Nutrition. 136 (6 Suppl): 1636Sâ1640S.
      doi:10.1093/jn/136.6.1636S. PMID 16702333.
 117. ^Kivirikko KI, Pihlajaniemi T (1998). Collagen hydroxylases and the
      protein disulfide isomerase subunit of prolyl 4-hydroxylases. Advances in
      Enzymology and Related Areas of Molecular Biology. Advances in Enzymology
      â and Related Areas of Molecular Biology. 72. pp. 325â98. doi:
      10.1002/9780470123188.ch9. ISBN 9780470123188. PMID 9559057.
 118. ^Whitmore L, Wallace BA (May 2004). "Analysis of peptaibol sequence
      composition: implications for in vivo synthesis and channel formation".
      European Biophysics Journal. 33 (3): 233â7. doi:10.1007/s00249-003-
      0348-1. PMID 14534753.
 119. ^Alexander L, Grierson D (October 2002). "Ethylene biosynthesis and
      action in tomato: a model for climacteric fruit ripening". Journal of
      Experimental Botany. 53 (377): 2039â55. doi:10.1093/jxb/erf072.
      PMID 12324528.
 120. ^Elmore DT, Barrett GC (1998). Amino acids and peptides. Cambridge, UK:
      Cambridge University Press. pp. 48â60. ISBN 978-0-521-46827-5.
 121. ^Gutteridge A, Thornton JM (November 2005). "Understanding nature's
      catalytic toolkit". Trends in Biochemical Sciences. 30 (11): 622â9.
      doi:10.1016/j.tibs.2005.09.006. PMID 16214343.
 122. ^Ibba M, SÃ¶ll D (May 2001). "The_renaissance_of_aminoacyl-tRNA
      synthesis". EMBO Reports. 2 (5): 382â7. doi:10.1093/embo-reports/
      kve095. PMC 1083889. PMID 11375928.
 123. ^Lengyel P, SÃ¶ll D (June 1969). "Mechanism_of_protein_biosynthesis".
      Bacteriological Reviews. 33 (2): 264â301. PMC 378322. PMID 4896351.
 124. ^Wu G, Fang YZ, Yang S, Lupton JR, Turner ND (March 2004). "Glutathione
      metabolism_and_its_implications_for_health". The Journal of Nutrition.
      134 (3): 489â92. doi:10.1093/jn/134.3.489. PMID 14988435.
 125. ^Meister A (November 1988). "Glutathione_metabolism_and_its_selective
      modification". The Journal of Biological Chemistry. 263 (33): 17205â8.
      PMID 3053703.
 126. ^Carpino LA (1992). "1-Hydroxy-7-azabenzotriazole. An efficient peptide
      coupling additive". Journal of the American Chemical Society. 115 (10):
      4397â8. doi:10.1021/ja00063a082.
 127. ^Marasco D, Perretta G, Sabatella M, Ruvo M (October 2008). "Past and
      future perspectives of synthetic peptide libraries". Current Protein &
      Peptide Science. 9 (5): 447â67. doi:10.2174/138920308785915209.
      PMID 18855697.
 128. ^Konara S, Gagnona K, Clearfield A, Thompson C, Hartle J, Ericson C,
      Nelson C (2010). "Structural determination and characterization of copper
      and zinc bis-glycinates with X-ray crystallography and mass
      spectrometry". Journal of Coordination Chemistry. 63 (19): 3335â47.
      doi:10.1080/00958972.2010.514336.
 129. ^ Stipanuk, M. H. (2006). Biochemical, physiological, & molecular aspects
      of human nutrition (2 ed.): Saunders Elsevier.
 130. ^Severin, K.; Bergs, R.; Beck, W. (1998). "Bioorganometallic Chemistry-
      Transition Metal Complexes with Î±-Amino Acids and Peptides". Angew.
      Chem. Int. Ed. 37 (12): 1635â1654. doi:10.1002/(SICI)1521-3773
      (19980703)37:12<1634::AID-ANIE1634>3.0.CO;2-C.CS1 maint: Uses authors
      parameter (link)
 131. ^Urry DW (2004). "The change in Gibbs free energy for hydrophobic
      association: Derivation and evaluation by means of inverse temperature
      transitions". Chemical Physics Letters. 399 (1â3): 177â83. doi:
      10.1016/S0009-2614(04)01565-9.
 132. ^Magee T, Seabra MC (April 2005). "Fatty acylation and prenylation of
      proteins: what's hot in fat". Current Opinion in Cell Biology. 17 (2):
      190â6. doi:10.1016/j.ceb.2005.02.003. PMID 15780596.
 133. ^Pilobello KT, Mahal LK (June 2007). "Deciphering the glycocode: the
      complexity and analytical challenge of glycomics". Current Opinion in
      Chemical Biology. 11 (3): 300â5. doi:10.1016/j.cbpa.2007.05.002.
      PMID 17500024.
 134. ^Smotrys JE, Linder ME (2004). "Palmitoylation of intracellular signaling
      proteins: regulation and function". Annual Review of Biochemistry. 73
      (1): 559â87. doi:10.1146/annurev.biochem.73.011303.073954.
      PMID 15189153.
 135. ^ a b c dHausman RE, Cooper GM (2004). The cell: a molecular approach.
      Washington, D.C: ASM Press. p. 51. ISBN 978-0-87893-214-6.
 136. ^Kyte J, Doolittle RF (May 1982). "A simple method for displaying the
      hydropathic character of a protein". Journal of Molecular Biology. 157
      (1): 105â32. CiteSeerX 10.1.1.458.454. doi:10.1016/0022-2836(82)90515-
      0. PMID 7108955.
 137. ^ a bFreifelder D (1983). Physical Biochemistry (2nd ed.). W. H. Freeman
      and Company. ISBN 978-0-7167-1315-9.
 138. [page needed]
 139. ^Kozlowski LP (January 2017). "Proteome-pI:_proteome_isoelectric_point
      database". Nucleic Acids Research. 45 (D1): D1112âD1116. doi:10.1093/
      nar/gkw978. PMC 5210655. PMID 27789699.
 140. ^Aasland R, Abrams C, Ampe C, Ball LJ, Bedford MT, Cesareni G, Gimona M,
      Hurley JH, Jarchau T, Lehto VP, Lemmon MA, Linding R, Mayer BJ, Nagai M,
      Sudol M, Walter U, Winder SJ (February 2002). "Normalization of
      nomenclature for peptide motifs as ligands of modular protein domains".
      FEBS Letters. 513 (1): 141â4. doi:10.1111/j.1432-1033.1968.tb00350.x.
      PMID 11911894.
 141. ^IUPAC-IUB Commission on Biochemical Nomenclature (1972). "A one-letter
      notation for amino acid sequences". Pure and Applied Chemistry. 31 (4):
      641â5. doi:10.1351/pac197231040639. PMID 5080161.
 142. ^Suchanek M, Radzikowska A, Thiele C (April 2005). "Photo-leucine and
      photo-methionine allow identification of protein-protein interactions in
      living cells". Nature Methods. 2 (4): 261â7. doi:10.1038/nmeth752.
      PMID 15782218.
***** Further reading[edit] *****
    * Tymoczko JL (2012). "Protein Composition and Structure". Biochemistry.
      New York: W. H. Freeman and company. pp. 28â31. ISBN 9781429229364.
Doolittle_RF (1989). "Redundancies in protein sequences". In Fasman, G.D.
(ed.). Predictions of Protein Structure and the Principles of Protein
Conformation. New York: Plenum_Press. pp. 599â623. ISBN 978-0-306-43131-9.
LCCN 89008555.
Nelson DL, Cox MM (2000). Lehninger Principles of Biochemistry (3rd ed.). Worth
Publishers. ISBN 978-1-57259-153-0. LCCN 99049137.
Meierhenrich_U (2008). Amino_acids_and_the_asymmetry_of_life (PDF). Berlin:
Springer_Verlag. ISBN 978-3-540-76885-2. LCCN 2008930865. Archived from the
original on 12 January 2012.CS1 maint: BOT: original-url status unknown (link)
***** External links[edit] *****
    *  Media related to Amino_acid at Wikimedia Commons
    * v
    * t
    * e
Encoded_(proteinogenic)_amino_acids
General        * Protein
topics         * Peptide
               * Genetic_code
                                     * Branched-chain_amino
                                       acids (Valine
                                     * Isoleucine
           Aliphatic                 * Leucine)
                                     * Methionine
                                     * Alanine
                                     * Proline
                                     * Glycine
                                     * Phenylalanine
           Aromatic                  * Tyrosine                  [L-amino_acid
By                                   * Tryptophan                any.png]
properties                           * Histidine
                                     * Asparagine
           Polar, uncharged          * Glutamine
                                     * Serine
                                     * Threonine
                                     * Lysine (â10.8)
           Positive charge (pKa)     * Arginine (â12.5)
                                     * Histidine (â6.1)
                                     * Aspartic_acid (â3.9)
           Negative charge (pKa)     * Glutamic_acid (â4.1)
                                     * Cysteine (â8.3)
                                     * Tyrosine (â10.1)
    * Amino_acids types: Encoded_(proteins)
    * Essential
    * Non-proteinogenic
    * Ketogenic
    * Glucogenic
    * Imino_acids
    * D-amino_acids
    * Dehydroamino_acids
    * v
    * t
    * e
Protein_primary_structure and posttranslational_modifications
                          * Peptide_bond
                          * Protein_biosynthesis
General                   * Proteolysis
                          * Racemization
                          * N-O_acyl_shift
                          * Acetylation
                          * Carbamylation
N_terminus                * Formylation
                          * Glycation
                          * Methylation
                          * Myristoylation (Gly)
                          * Amidation
C_terminus                * Glycosyl_phosphatidylinositol_(GPI)
                          * O-methylation
                          * Detyrosination
                                           * Phosphorylation
                                           * Dephosphorylation
                      Serine/Threonine     * Glycosylation
                                           * ADP-ribosylation
                                           * Methylidene-imidazolone_(MIO)
                                             formation
                                           * Phosphorylation
                                           * Dephosphorylation
                                           * ADP-ribosylation
                                           * Sulfation
                      Tyrosine             * Porphyrin_ring_linkage
                                           * Adenylylation
                                           * Flavin_linkage
                                           * Topaquinone_(TPQ)_formation
                                           * Detyrosination
                      Cysteine             * Palmitoylation
                                           * Prenylation
                      Aspartate            * Succinimide_formation
                                           * ADP-ribosylation
                                           * Carboxylation
                                           * ADP-ribosylation
                      Glutamate            * Methylation
                                           * Polyglutamylation
                                           * Polyglycylation
Single specific AAs   Asparagine           * Deamidation
                                           * Glycosylation
                      Glutamine            * Transglutamination
                                           * Methylation
                                           * Acetylation
                                           * Acylation
                                           * Adenylylation
                                           * Hydroxylation
                                           * Ubiquitination
                                           * Sumoylation
                      Lysine               * ADP-ribosylation
                                           * Deamination
                                           * Oxidative_deamination_to_aldehyde
                                           * O-glycosylation
                                           * Imine_formation
                                           * Glycation
                                           * Carbamylation
                                           * Succinylation
                                           * Citrullination
                      Arginine             * Methylation
                                           * ADP-ribosylation
                      Proline              * Hydroxylation
                      Histidine            * Diphthamide formation
                                           * Adenylylation
                      Tryptophan           * C-mannosylation
                      Cysteine-Cysteine                * Disulfide_bond
                                                       * ADP-ribosylation
                      Methionine-Hydroxylysine         * Sulfilimine_bond
Crosslinks between    Lysine-Tyrosylquinone            * Lysine_tyrosylquinone_
two AAs                                                  (LTQ)_formation
                                                       * Tryptophan
                      Tryptophan-Tryptophylquinone       tryptophylquinone_
                                                         (TTQ)_formation
                                                         * p-
                      SerineâTyrosineâGlyci      Hydroxybenzylidene-
Three consecutive AAs                                      imidazolinone
(chromophore                                               formation
formation)                                               * 4-(p-
                      HistidineâTyrosineâGl      hydroxybenzylidene)-
                                                           5-imidazolinone
                                                           formation
Crosslinks between    Allysine-Allysine-Allysine-Lysine     * Desmosine
four AAs
    * v
    * t
    * e
Metabolism: Protein_metabolism, synthesis and catabolism enzymes
Essential_amino_acids are in Capitals
               LYSINEâ                    * Saccharopine_dehydrogenase
                                               * Glutaryl-CoA_dehydrogenase
                                               * 3-Hydroxybutyryl-CoA_dehydrogenase
                                               * Branched-chain_amino_acid_aminotransferase
                                               * Branched-chain_alpha-keto_acid_dehydrogenase_complex
                                               * Enoyl-CoA_hydratase
                                               * HMG-CoA_lyase
                                               * HMG-CoA_reductase
               LEUCINEâ                   * Isovaleryl_coenzyme_A_dehydrogenase
                                               * Î±-Ketoisocaproate_dioxygenase
Kâacetyl-C                                * Leucine_2,3-aminomutase
                                               * Methylcrotonyl-CoA_carboxylase
                                               * Methylglutaconyl-CoA_hydratase
                                           (See Template:Leucine_metabolism_in_humans â this diagram does not include the pathway for Î²-leucine synthesis via
                                           leucine 2,3-aminomutase)
                                               * Indoleamine_2,3-dioxygenase/Tryptophan_2,3-dioxygenase
                                               * Arylformamidase
               TRYPTOPHANâ                * Kynureninase
                                               * 3-hydroxyanthranilate_oxidase
                                               * Aminocarboxymuconate-semialdehyde_decarboxylase
                                               * Aminomuconate-semialdehyde_dehydrogenase
               PHENYLALANINEâtyrosine    * (see below)
                                                                  * Serine_hydroxymethyltransferase
                                          glycineâserine    * Serine_dehydratase
               Gâpyruvate                                    * glycineâcreatine: Guanidinoacetate_N-methyltransferase
               âcitrate                                      * Creatine_kinase
                                          alanineâ           * Alanine_transaminase
                                          cysteineâ          * D-cysteine_desulfhydrase
                                          threonineâ         * L-threonine_dehydrogenase
                                                                  * Histidine_ammonia-lyase
                                          HISTIDINEâ         * Urocanate_hydratase
                                                                  * Formiminotransferase_cyclodeaminase
                                                                  * Proline_oxidase
                                          prolineâ           * Pyrroline-5-carboxylate_reductase
                                                                  * 1-Pyrroline-5-carboxylate_dehydrogenase/ALDH4A1
                                                                  * PYCR1
               Gâglutamateâ                             * Ornithine_aminotransferase
               Î±-ketoglutarate         arginineâ          * Ornithine_decarboxylase
                                                                  * Agmatinase
                                          âalpha-            * Glutamate_dehydrogenase
                                          ketoglutarateâTCA
                                                                  * cysteine+glutamateâglutathione: Gamma-glutamylcysteine_synthetase
                                                                  * Glutathione_synthetase
                                          Other                   * Gamma-glutamyl_transpeptidase
                                                                  * glutamateâglutamine: Glutamine_synthetase
                                                                  * Glutaminase
                                                                    * Branched-chain_amino_acid_aminotransferase
G                                                                   * Branched-chain_alpha-keto_acid_dehydrogenase_complex
                                          VALINEâ              * Enoyl-CoA_hydratase
                                                                    * 3-hydroxyisobutyryl-CoA_hydrolase
                                                                    * 3-hydroxyisobutyrate_dehydrogenase
                                                                    * Methylmalonate_semialdehyde_dehydrogenase
                                                                    * Branched-chain_amino_acid_aminotransferase
                                          ISOLEUCINEâ          * Branched-chain_alpha-keto_acid_dehydrogenase_complex
                                                                    * 3-hydroxy-2-methylbutyryl-CoA_dehydrogenase
               Gâpropionyl-CoAâ                           * generation_of_homocysteine: Methionine_adenosyltransferase
               succinyl-CoA                                         * Adenosylhomocysteinase
                                                                    * regeneration_of_methionine: Methionine_synthase/Homocysteine
                                          METHIONINEâ            methyltransferase
                                                                    * Betaine-homocysteine_methyltransferase
                                                                    * conversion_to_cysteine: Cystathionine_beta_synthase
                                                                    * Cystathionine_gamma-lyase
                                          THREONINEâ           * Threonine_aldolase
                                                                    * Propionyl-CoA_carboxylase
                                          âsuccinyl-CoAâ    * Methylmalonyl_CoA_epimerase
                                                                    * Methylmalonyl-CoA_mutase
                                                                          * Phenylalanine_hydroxylase
                                                                          * Tyrosine_aminotransferase
               Gâfumarate            PHENYLALANINEâtyrosine    * 4-Hydroxyphenylpyruvate_dioxygenase
                                                                          * Homogentisate_1,2-dioxygenase
                                                                          * Fumarylacetoacetate_hydrolase
                                                                          * tyrosineâmelanin: Tyrosinase
               Gâoxaloacetate        asparagineâaspartate    * Asparaginase/Asparagine_synthetase
                                                                        * Aspartate_transaminase
                                              * GND: 4142205-3
Authority_control [Edit_this_at_Wikidata]     * LCCN: sh85004486
                                              * NDL: 00560236

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Amino_acid&oldid=909810553"
Categories:
    * Amino_acids
    * Nitrogen_cycle
    * Zwitterions
Hidden categories:
    * CS1_maint:_Archived_copy_as_title
    * CS1:_long_volume_value
    * All_articles_with_dead_external_links
    * Articles_with_dead_external_links_from_July_2017
    * Articles_with_permanently_dead_external_links
    * CS1_maint:_Multiple_names:_authors_list
    * CS1_maint:_Uses_authors_parameter
    * Wikipedia_articles_needing_page_number_citations_from_September_2010
    * Articles_with_short_description
    * Wikipedia_indefinitely_move-protected_pages
    * Use_dmy_dates_from_August_2017
    * Good_articles
    * Wikipedia_articles_needing_clarification_from_January_2019
    * Articles_containing_German-language_text
    * Wikipedia_articles_needing_clarification_from_August_2017
    * Pages_using_multiple_image_with_auto_scaled_images
    * Wikipedia_articles_in_need_of_updating_from_July_2019
    * All_Wikipedia_articles_in_need_of_updating
    * Articles_containing_potentially_dated_statements_from_2011
    * All_articles_containing_potentially_dated_statements
    * Pages_using_multiple_image_with_manual_scaled_images
    * CS1_maint:_BOT:_original-url_status_unknown
    * Wikipedia_articles_with_GND_identifiers
    * Wikipedia_articles_with_LCCN_identifiers
    * Wikipedia_articles_with_NDL_identifiers
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Afrikaans
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * à¦à¦¸à¦®à§à¦¯à¦¼à¦¾
    * Asturianu
    * AzÉrbaycanca
    * ØªÛØ±Ú©Ø¬Ù
    * à¦¬à¦¾à¦à¦²à¦¾
    * BÃ¢n-lÃ¢m-gÃº
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ
    * ÐÐµÐ»Ð°ÑÑÑÐºÐ°Ñ_(ÑÐ°ÑÐ°ÑÐºÐµÐ²ÑÑÐ°)â
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Cymraeg
    * Dansk
    * DavvisÃ¡megiella
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FÃ¸royskt
    * FranÃ§ais
    * Gaeilge
    * Galego
    * å®¢å®¶èª/Hak-kÃ¢-ngÃ®
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Ido
    * Bahasa_Indonesia
    * Interlingua
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * à²à²¨à³à²¨à²¡
    * á¥áá áá£áá
    * ÒÐ°Ð·Ð°ÒÑÐ°
    * Kiswahili
    * KreyÃ²l_ayisyen
    * KurdÃ®
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * Latina
    * LatvieÅ¡u
    * LÃ«tzebuergesch
    * LietuviÅ³
    * Limburgs
    * Lumbaart
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * à´®à´²à´¯à´¾à´³à´
    * Bahasa_Melayu
    * ÐÐ¾Ð½Ð³Ð¾Ð»
    * Nederlands
    * æ¥æ¬èª
    * Nordfriisk
    * Norsk
    * Norsk_nynorsk
    * Novial
    * Occitan
    * Oromoo
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * à¨ªà©°à¨à¨¾à¨¬à©
    * Ù¾ÙØ¬Ø§Ø¨Û
    * Ù¾ÚØªÙ
    * áá¶áá¶ááááá
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Seeltersk
    * Shqip
    * à·à·à¶à·à¶½
    * Simple_English
    * SlovenÄina
    * SlovenÅ¡Äina
    * Ú©ÙØ±Ø¯Û
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Basa_Sunda
    * Suomi
    * Svenska
    * Tagalog
    * à®¤à®®à®¿à®´à¯
    * à°¤à±à°²à±à°à±
    * à¹à¸à¸¢
    * Ð¢Ð¾Ò·Ð¸ÐºÓ£
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Ø§Ø±Ø¯Ù
    * Tiáº¿ng_Viá»t
    * West-Vlams
    * Winaray
    * å´è¯­
    * YorÃ¹bÃ¡
    * ç²µèª
    * ä¸­æ
Edit_links
    * This page was last edited on 7 August 2019, at 18:59 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
