The following text has been accessed from https://en.wikipedia.org/wiki/Genetic_drift at Fri Aug 9 04:03:10 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















[This_is_a_good_article._Follow_the_link_for_more_information.]
****** Genetic drift ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
The change in the frequency of an existing gene variant in a population
Not to be confused with Genetic_draft, Antigenic_drift, or Antigenic_shift.
Part of a series on
Evolutionary_biology
[Darwin's_finches_by_Gould.jpg]
Darwin's_finches by John_Gould
Key topics
    * Introduction_to_evolution
    * Evidence_of_evolution
    * Common_descent
    * Evidence_of_common_descent
Processes and outcomes
    * Population_genetics
    * Variation
    * Diversity
    * Mutation
    * Natural_selection
    * Adaptation
    * Polymorphism
    * Genetic drift
    * Gene_flow
    * Speciation
    * Adaptive_radiation
    * Co-operation
    * Coevolution
    * Coextinction
    * Divergence
    * Convergence
    * Parallel_evolution
    * Extinction
Natural history
    * Origin_of_life
    * History_of_life
    * Timeline_of_evolution
    * Human_evolution
    * Phylogeny
    * Biodiversity
    * Biogeography
    * Classification
    * Evolutionary_taxonomy
    * Cladistics
    * Transitional_fossil
    * Extinction_event
History of evolutionary theory
    * Overview
    * Renaissance
    * Before_Darwin
    * Darwin
    * Origin_of_Species
    * Before_synthesis
    * Modern_synthesis
    * Molecular_evolution
    * Evo-devo
    * Current_research
    * History_of_speciation
    * History_of_paleontology (timeline)
Fields and applications
    * Applications_of_evolution
    * Biosocial_criminology
    * Ecological_genetics
    * Evolutionary_aesthetics
    * Evolutionary_anthropology
    * Evolutionary_computation
    * Evolutionary_ecology
    * Evolutionary_economics
    * Evolutionary_epistemology
    * Evolutionary_ethics
    * Evolutionary_game_theory
    * Evolutionary_linguistics
    * Evolutionary_medicine
    * Evolutionary_neuroscience
    * Evolutionary_physiology
    * Evolutionary_psychology
    * Experimental_evolution
    * Phylogenetics
    * Paleontology
    * Selective_breeding
    * Speciation_experiments
    * Sociobiology
    * Systematics
    * Universal_Darwinism
Social implications
    * Evolution_as_fact_and_theory
    * Social_effects
    * Creationâevolution_controversy
    * Objections_to_evolution
    * Level_of_support
    * [Tree_of_life.svg] Evolutionary_biology_portal
    * [Category] Category
    *  Book
    * Related_topics
    * v
    * t
    * e
Genetic drift (also known as allelic drift or the Sewall Wright effect)[1] is
the change in the frequency of an existing gene variant (allele) in a
population due to random sampling of organisms.[2] The alleles in the offspring
are a sample of those in the parents, and chance has a role in determining
whether a given individual survives and reproduces. A population's allele
frequency is the fraction of the copies of one gene that share a particular
form.[3] Genetic drift may cause gene variants to disappear completely and
thereby reduce genetic_variation.[4] It can also cause initially rare alleles
to become much more frequent and even fixed.
When there are few copies of an allele, the effect of genetic drift is larger,
and when there are many copies the effect is smaller. In the early 20th
century, vigorous debates occurred over the relative importance of natural
selection versus neutral processes, including genetic drift. Ronald_Fisher, who
explained natural selection using Mendelian_genetics,[5] held the view that
genetic drift plays at the most a minor role in evolution, and this remained
the dominant view for several decades. In 1968, population geneticist Motoo
Kimura rekindled the debate with his neutral_theory_of_molecular_evolution,
which claims that most instances where a genetic change spreads_across_a
population (although not necessarily changes in phenotypes) are caused by
genetic drift acting on neutral mutations.[6][7]
⁰
***** Contents *****
    * 1_Analogy_with_marbles_in_a_jar
    * 2_Probability_and_allele_frequency
    * 3_Mathematical_models
          o 3.1_WrightâFisher_model
          o 3.2_Moran_model
          o 3.3_Other_models_of_drift
          o 3.4_Random_effects_other_than_sampling_error
    * 4_Drift_and_fixation
          o 4.1_Rate_of_allele_frequency_change_due_to_drift
          o 4.2_Time_to_fixation_or_loss
          o 4.3_Time_to_loss_with_both_drift_and_mutation
    * 5_Versus_natural_selection
    * 6_Population_bottleneck
          o 6.1_Founder_effect
    * 7_History
    * 8_See_also
    * 9_Notes_and_references
    * 10_Bibliography
    * 11_External_links
***** Analogy with marbles in a jar[edit] *****
The process of genetic drift can be illustrated using 20 marbles in a jar to
represent 20 organisms in a population.[8] Consider this jar of marbles as the
starting population. Half of the marbles in the jar are red and half are blue,
with each colour corresponding to a different allele of one gene in the
population. In each new generation the organisms reproduce at random. To
represent this reproduction, randomly select a marble from the original jar and
deposit a new marble with the same colour into a new jar. This is the
"offspring" of the original marble, meaning that the original marble remains in
its jar. Repeat this process until there are 20 new marbles in the second jar.
The second jar will now contain 20 "offspring", or marbles of various colours.
Unless the second jar contains exactly 10 red marbles and 10 blue marbles, a
random shift has occurred in the allele frequencies.
If this process is repeated a number of times, the numbers of red and blue
marbles picked each generation will fluctuate. Sometimes a jar will have more
red marbles than its "parent" jar and sometimes more blue. This fluctuation is
analogous to genetic drift â a change in the population's allele frequency
resulting from a random variation in the distribution of alleles from one
generation to the next.
It is even possible that in any one generation no marbles of a particular
colour are chosen, meaning they have no offspring. In this example, if no red
marbles are selected, the jar representing the new generation contains only
blue offspring. If this happens, the red allele has been lost permanently in
the population, while the remaining blue allele has become fixed: all future
generations are entirely blue. In small populations, fixation can occur in just
a few generations.
In this simulation each black dot on a marble signifies that it has been chosen
for copying (reproduction) one time. There is fixation in the blue "allele"
within five generations.
***** Probability and allele frequency[edit] *****
The mechanisms of genetic drift can be illustrated with a simplified example.
Consider a very large colony of bacteria isolated in a drop of solution. The
bacteria are genetically identical except for a single gene with two alleles
labeled A and B. A and B are neutral alleles meaning that they do not affect
the bacteria's ability to survive and reproduce; all bacteria in this colony
are equally likely to survive and reproduce. Suppose that half the bacteria
have allele A and the other half have allele B. Thus A and B each have allele
frequency 1/2.
The drop of solution then shrinks until it has only enough food to sustain four
bacteria. All other bacteria die without reproducing. Among the four who
survive, there are sixteen possible combinations for the A and B alleles:
(A-A-A-A), (B-A-A-A), (A-B-A-A), (B-B-A-A),
(A-A-B-A), (B-A-B-A), (A-B-B-A), (B-B-B-A),
(A-A-A-B), (B-A-A-B), (A-B-A-B), (B-B-A-B),
(A-A-B-B), (B-A-B-B), (A-B-B-B), (B-B-B-B).
Since all bacteria in the original solution are equally likely to survive when
the solution shrinks, the four survivors are a random sample from the original
colony. The probability that each of the four survivors has a given allele is
1/2, and so the probability that any particular allele combination occurs when
the solution shrinks is
           1 2   &#x22C5;   1 2   &#x22C5;   1 2   &#x22C5;   1 2   =   1 16
      .   {\displaystyle {\frac {1}{2}}\cdot {\frac {1}{2}}\cdot {\frac {1}
      {2}}\cdot {\frac {1}{2}}={\frac {1}{16}}.}  [{\frac {1}{2}}\cdot {\frac
      {1}{2}}\cdot {\frac {1}{2}}\cdot {\frac {1}{2}}={\frac {1}{16}}.]
(The original population size is so large that the sampling effectively happens
without replacement). In other words, each of the sixteen possible allele
combinations is equally likely to occur, with probability 1/16.
Counting the combinations with the same number of A and B, we get the following
table.
A B Combinations Probability
4 0 1            1/16
3 1 4            4/16
2 2 6            6/16
1 3 4            4/16
0 4 1            1/16
As shown in the table, the total number of combinations that have the same
number of A alleles as of B alleles is six, and the probability of this
combination is 6/16. The total number of other combinations is ten, so the
probability of unequal number of A and B alleles is 10/16. Thus, although the
original colony began with an equal number of A and B alleles, it is very
possible that the number of alleles in the remaining population of four members
will not be equal. Equal numbers is actually less likely than unequal numbers.
In the latter case, genetic drift has occurred because the population's allele
frequencies have changed due to random sampling. In this example the population
contracted to just four random survivors, a phenomenon known as population
bottleneck.
The probabilities for the number of copies of allele A (or B) that survive
(given in the last column of the above table) can be calculated directly from
the binomial_distribution where the "success" probability (probability of a
given allele being present) is 1/2 (i.e., the probability that there are k
copies of A (or B) alleles in the combination) is given by
            (   n k   )      (   1 2   )   k     (  1 &#x2212;   1 2    )   n
      &#x2212; k   =    (   n k   )      (   1 2   )   n      {\displaystyle {n
      \choose k}\left({\frac {1}{2}}\right)^{k}\left(1-{\frac {1}{2}}\right)^
      {n-k}={n \choose k}\left({\frac {1}{2}}\right)^{n}\!}  [{n \choose
      k}\left({\frac {1}{2}}\right)^{k}\left(1-{\frac {1}{2}}\right)^{n-k}={n
      \choose k}\left({\frac {1}{2}}\right)^{n}\!]
where n=4 is the number of surviving bacteria.
***** Mathematical models[edit] *****
Mathematical models of genetic drift can be designed using either branching
processes or a diffusion_equation describing changes in allele frequency in an
idealised_population.[9]
**** WrightâFisher model[edit] ****
Consider a gene with two alleles, A or B. In diploid populations consisting of
N individuals there are 2N copies of each gene. An individual can have two
copies of the same allele or two different alleles. We can call the frequency
of one allele p and the frequency of the other q. The WrightâFisher model
(named after Sewall_Wright and Ronald_Fisher) assumes that generations do not
overlap (for example, annual_plants have exactly one generation per year) and
that each copy of the gene found in the new generation is drawn independently
at random from all copies of the gene in the old generation. The formula to
calculate the probability of obtaining k copies of an allele that had frequency
p in the last generation is then[10][11]
            ( 2 N ) !   k ! ( 2 N &#x2212; k ) !     p  k    q  2 N &#x2212; k
      {\displaystyle {\frac {(2N)!}{k!(2N-k)!}}p^{k}q^{2N-k}}  [{\frac {(2N)!}
      {k!(2N-k)!}}p^{k}q^{2N-k}]
where the symbol "!" signifies the factorial function. This expression can also
be formulated using the binomial_coefficient,
            (    2 N  k   )     p  k    q  2 N &#x2212; k     {\displaystyle
      {2N \choose k}p^{k}q^{2N-k}}  [{2N \choose k}p^{k}q^{2N-k}]
**** Moran model[edit] ****
The Moran_model assumes overlapping generations. At each time step, one
individual is chosen to reproduce and one individual is chosen to die. So in
each timestep, the number of copies of a given allele can go up by one, go down
by one, or can stay the same. This means that the transition_matrix is
tridiagonal, which means that mathematical solutions are easier for the Moran
model than for the WrightâFisher model. On the other hand, computer
simulations are usually easier to perform using the WrightâFisher model,
because fewer time steps need to be calculated. In the Moran model, it takes N
timesteps to get through one generation, where N is the effective_population
size. In the WrightâFisher model, it takes just one.[12]
In practice, the Moran and WrightâFisher models give qualitatively similar
results, but genetic drift runs twice as fast in the Moran model.
**** Other models of drift[edit] ****
If the variance in the number of offspring is much greater than that given by
the binomial distribution assumed by the WrightâFisher model, then given the
same overall speed of genetic drift (the variance effective population size),
genetic drift is a less powerful force compared to selection.[13] Even for the
same variance, if higher moments of the offspring number distribution exceed
those of the binomial distribution then again the force of genetic drift is
substantially weakened.[14]
**** Random effects other than sampling error[edit] ****
Random changes in allele frequencies can also be caused by effects other than
sampling_error, for example random changes in selection pressure.[15]
One important alternative source of stochasticity, perhaps more important than
genetic drift, is genetic_draft.[16] Genetic draft is the effect on a locus by
selection on linked loci. The mathematical properties of genetic draft are
different from those of genetic drift.[17] The direction of the random change
in allele frequency is autocorrelated across generations.[2]
***** Drift and fixation[edit] *****
The HardyâWeinberg_principle states that within sufficiently large
populations, the allele frequencies remain constant from one generation to the
next unless the equilibrium is disturbed by migration, genetic mutations, or
selection.[18]
However, in finite populations, no new alleles are gained from the random
sampling of alleles passed to the next generation, but the sampling can cause
an existing allele to disappear. Because random_sampling can remove, but not
replace, an allele, and because random declines or increases in allele
frequency influence expected allele distributions for the next generation,
genetic drift drives a population towards genetic uniformity over time. When an
allele reaches a frequency of 1 (100%) it is said to be "fixed" in the
population and when an allele reaches a frequency of 0 (0%) it is lost. Smaller
populations achieve fixation faster, whereas in the limit of an infinite
population, fixation is not achieved. Once an allele becomes fixed, genetic
drift comes to a halt, and the allele frequency cannot change unless a new
allele is introduced in the population via mutation or gene_flow. Thus even
while genetic drift is a random, directionless process, it acts to eliminate
genetic variation over time.[19]
**** Rate of allele frequency change due to drift[edit] ****
Ten simulations of random genetic drift of a single given allele with an
initial frequency distribution 0.5 measured over the course of 50 generations,
repeated in three reproductively synchronous populations of different sizes. In
these simulations, alleles drift to loss or fixation (frequency of 0.0 or 1.0)
only in the smallest population.
Assuming genetic drift is the only evolutionary force acting on an allele,
after t generations in many replicated populations, starting with allele
frequencies of p and q, the variance in allele frequency across those
populations is
          V  t   &#x2248; p q  (  1 &#x2212; exp &#x2061;  (  &#x2212;   t  2
      N  e       )   )    {\displaystyle V_{t}\approx pq\left(1-\exp \left(-
      {\frac {t}{2N_{e}}}\right)\right)}  [V_{t}\approx pq\left(1-\exp \left(-
      {\frac {t}{2N_{e}}}\right)\right)][20]
**** Time to fixation or loss[edit] ****
Assuming genetic drift is the only evolutionary force acting on an allele, at
any given time the probability that an allele will eventually become fixed in
the population is simply its frequency in the population at that time.[21] For
example, if the frequency p for allele A is 75% and the frequency q for allele
B is 25%, then given unlimited time the probability A will ultimately become
fixed in the population is 75% and the probability that B will become fixed is
25%.
The expected number of generations for fixation to occur is proportional to the
population size, such that fixation is predicted to occur much more rapidly in
smaller populations.[22] Normally the effective population size, which is
smaller than the total population, is used to determine these probabilities.
The effective population (Ne) takes into account factors such as the level of
inbreeding, the stage of the lifecycle in which the population is the smallest,
and the fact that some neutral genes are genetically linked to others that are
under selection.[13] The effective population size may not be the same for
every gene in the same population.[23]
One forward-looking formula used for approximating the expected time before a
neutral allele becomes fixed through genetic drift, according to the
WrightâFisher model, is
             T &#x00AF;     fixed   =    &#x2212; 4  N  e   ( 1 &#x2212; p ) ln
      &#x2061; ( 1 &#x2212; p )  p     {\displaystyle {\bar {T}}_{\text
      {fixed}}={\frac {-4N_{e}(1-p)\ln(1-p)}{p}}}  [{\bar {T}}_{\text{fixed}}=
      {\frac {-4N_{e}(1-p)\ln(1-p)}{p}}]
where T is the number of generations, Ne is the effective population size, and
p is the initial frequency for the given allele. The result is the number of
generations expected to pass before fixation occurs for a given allele in a
population with given size (Ne) and allele frequency (p).[24]
The expected time for the neutral allele to be lost through genetic drift can
be calculated as[10]
             T &#x00AF;     lost   =    &#x2212; 4  N  e   p   1 &#x2212; p
      ln &#x2061; p .   {\displaystyle {\bar {T}}_{\text{lost}}={\frac {-4N_
      {e}p}{1-p}}\ln p.}  [{\bar {T}}_{\text{lost}}={\frac {-4N_{e}p}{1-p}}\ln
      p.]
When a mutation appears only once in a population large enough for the initial
frequency to be negligible, the formulas can be simplified to[25]
             T &#x00AF;     fixed   = 4  N  e     {\displaystyle {\bar {T}}_
      {\text{fixed}}=4N_{e}}  [{\bar {T}}_{\text{fixed}}=4N_{e}]
for average number of generations expected before fixation of a neutral
mutation, and
             T &#x00AF;     lost   = 2  (    N  e   N   )  ln &#x2061; ( 2 N )
      {\displaystyle {\bar {T}}_{\text{lost}}=2\left({\frac {N_{e}}
      {N}}\right)\ln(2N)}  [{\bar {T}}_{\text{lost}}=2\left({\frac {N_{e}}
      {N}}\right)\ln(2N)]
for the average number of generations expected before the loss of a neutral
mutation.[26]
**** Time to loss with both drift and mutation[edit] ****
The formulae above apply to an allele that is already present in a population,
and which is subject to neither mutation nor natural selection. If an allele is
lost by mutation much more often than it is gained by mutation, then mutation,
as well as drift, may influence the time to loss. If the allele prone to
mutational loss begins as fixed in the population, and is lost by mutation at
rate m per replication, then the expected time in generations until its loss in
a haploid population is given by
             T &#x00AF;     lost   &#x2248;   {       1 m    ,  &#xA0;if&#xA0;
      m  N  e   &#x226A; 1         ln &#x2061;  ( m  N  e   )  + &#x03B3;  m
      &#xA0;if&#xA0;  m  N  e   &#x226B; 1         {\displaystyle {\bar {T}}_
      {\text{lost}}\approx {\begin{cases}{\dfrac {1}{m}},{\text{ if }}mN_{e}\ll
      1\\[8pt]{\dfrac {\ln {(mN_{e})}+\gamma }{m}}{\text{ if }}mN_{e}\gg 1\end
      {cases}}}  [{\displaystyle {\bar {T}}_{\text{lost}}\approx {\begin{cases}
      {\dfrac {1}{m}},{\text{ if }}mN_{e}\ll 1\\[8pt]{\dfrac {\ln {(mN_
      {e})}+\gamma }{m}}{\text{ if }}mN_{e}\gg 1\end{cases}}}]
where     &#x03B3;   {\displaystyle \gamma }  [\gamma ] is Euler's_constant.
[27] The first approximation represents the waiting time until the first mutant
destined for loss, with loss then occurring relatively rapidly by genetic
drift, taking time Ne âª 1/m. The second approximation represents the time
needed for deterministic loss by mutation accumulation. In both cases, the time
to fixation is dominated by mutation via the term 1/m, and is less affected by
the effective_population_size.
***** Versus natural selection[edit] *****
In natural populations, genetic drift and natural selection do not act in
isolation; both phenomena are always at play, together with mutation and
migration. Neutral evolution is the product of both mutation and drift, not of
drift alone. Similarly, even when selection overwhelms genetic drift, it can
only act on variation that mutation provides.
While natural selection has a direction, guiding evolution towards heritable
adaptations to the current environment, genetic drift has no direction and is
guided only by the mathematics_of_chance.[28] As a result, drift acts upon the
genotypic_frequencies within a population without regard to their phenotypic
effects. In contrast, selection favors the spread of alleles whose phenotypic
effects increase survival and/or reproduction of their carriers, lowers the
frequencies of alleles that cause unfavorable traits, and ignores those that
are neutral.[29]
The law_of_large_numbers predicts that when the absolute number of copies of
the allele is small (e.g., in_small_populations), the magnitude of drift on
allele frequencies per generation is larger. The magnitude of drift is large
enough to overwhelm selection at any allele frequency when the selection
coefficient is less than 1 divided by the effective population size. Non-
adaptive evolution resulting from the product of mutation and genetic drift is
therefore considered to be a consequential mechanism of evolutionary change
primarily within small, isolated populations.[30] The mathematics of genetic
drift depend on the effective population size, but it is not clear how this is
related to the actual number of individuals in a population.[16] Genetic
linkage to other genes that are under selection can reduce the effective
population size experienced by a neutral allele. With a higher recombination
rate, linkage decreases and with it this local effect on effective population
size.[31][32] This effect is visible in molecular data as a correlation between
local recombination rate and genetic_diversity,[33] and negative correlation
between gene density and diversity at noncoding_DNA regions.[34] Stochasticity
associated with linkage to other genes that are under selection is not the same
as sampling error, and is sometimes known as genetic_draft in order to
distinguish it from genetic drift.[16]
When the allele frequency is very small, drift can also overpower selection
even in large populations. For example, while disadvantageous mutations are
usually eliminated quickly in large populations, new advantageous mutations are
almost as vulnerable to loss through genetic drift as are neutral mutations.
Not until the allele frequency for the advantageous mutation reaches a certain
threshold will genetic drift have no effect.[29]
***** Population bottleneck[edit] *****
Main article: Population_bottleneck
Changes in a population's allele frequency following a population_bottleneck:
the rapid and radical decline in population size has reduced the population's
genetic_variation.
A population bottleneck is when a population contracts to a significantly
smaller size over a short period of time due to some random environmental
event. In a true population bottleneck, the odds for survival of any member of
the population are purely random, and are not improved by any particular
inherent genetic advantage. The bottleneck can result in radical changes in
allele frequencies, completely independent of selection.[35]
The impact of a population bottleneck can be sustained, even when the
bottleneck is caused by a one-time event such as a natural catastrophe. An
interesting example of a bottleneck causing unusual genetic distribution is the
relatively high proportion of individuals with total rod_cell color_blindness
(achromatopsia) on Pingelap_atoll in Micronesia. After a bottleneck, inbreeding
increases. This increases the damage done by recessive deleterious mutations,
in a process known as inbreeding_depression. The worst of these mutations are
selected against, leading to the loss of other alleles that are genetically
linked to them, in a process of background_selection.[2] For recessive harmful
mutations, this selection can be enhanced as a consequence of the bottleneck,
due to genetic_purging. This leads to a further loss of genetic diversity. In
addition, a sustained reduction in population size increases the likelihood of
further allele fluctuations from drift in generations to come.
A population's genetic variation can be greatly reduced by a bottleneck, and
even beneficial adaptations may be permanently eliminated.[36] The loss of
variation leaves the surviving population vulnerable to any new selection
pressures such as disease, climate_change or shift in the available food
source, because adapting in response to environmental changes requires
sufficient genetic variation in the population for natural selection to take
place.[37][38]
There have been many known cases of population bottleneck in the recent past.
Prior to the arrival of Europeans, North_American prairies were habitat for
millions of greater_prairie_chickens. In Illinois alone, their numbers
plummeted from about 100 million birds in 1900 to about 50 birds in the 1990s.
The declines in population resulted from hunting and habitat destruction, but a
consequence has been a loss of most of the species' genetic diversity. DNA
analysis comparing birds from the mid century to birds in the 1990s documents a
steep decline in the genetic variation in just the latter few decades.
Currently the greater prairie chicken is experiencing low reproductive_success.
[39]
However, the genetic loss caused by bottleneck and genetic drift can increase
fitness, as in Ehrlichia.[40]
Over-hunting also caused a severe population bottleneck in the northern
elephant_seal in the 19th century. Their resulting decline in genetic variation
can be deduced by comparing it to that of the southern_elephant_seal, which
were not so aggressively hunted.[41]
**** Founder effect[edit] ****
Main article: Founder_effect
When very few members of a population migrate to form a separate new
population, the founder effect occurs. For a period after the foundation, the
small population experiences intensive drift. In the figure this results in
fixation of the red allele.
The founder effect is a special case of a population bottleneck, occurring when
a small group in a population splinters off from the original population and
forms a new one. The random sample of alleles in the just formed new colony is
expected to grossly misrepresent the original population in at least some
respects.[42] It is even possible that the number of alleles for some genes in
the original population is larger than the number of gene copies in the
founders, making complete representation impossible. When a newly formed colony
is small, its founders can strongly affect the population's genetic make-up far
into the future.
A well-documented example is found in the Amish migration to Pennsylvania in
1744. Two members of the new colony shared the recessive allele for Ellisâvan
Creveld_syndrome. Members of the colony and their descendants tend to be
religious isolates and remain relatively insular. As a result of many
generations of inbreeding, Ellis-van Creveld syndrome is now much more
prevalent among the Amish than in the general population.[29][43]
The difference in gene frequencies between the original population and colony
may also trigger the two groups to diverge significantly over the course of
many generations. As the difference, or genetic_distance, increases, the two
separated populations may become distinct, both genetically and phenetically,
although not only genetic drift but also natural selection, gene flow, and
mutation contribute to this divergence. This potential for relatively rapid
changes in the colony's gene frequency led most scientists to consider the
founder effect (and by extension, genetic drift) a significant driving force in
the evolution of new_species. Sewall Wright was the first to attach this
significance to random drift and small, newly isolated populations with his
shifting_balance_theory of speciation.[44] Following after Wright, Ernst_Mayr
created many persuasive models to show that the decline in genetic variation
and small population size following the founder effect were critically
important for new species to develop.[45] However, there is much less support
for this view today since the hypothesis has been tested repeatedly through
experimental research and the results have been equivocal at best.[46]
***** History[edit] *****
Further information: Modern_synthesis_(20th_century)
The role of random chance in evolution was first outlined by Hagedoorn and
Hagedoorn in 1921 [47]. They highlighted that random survival plays a key role
in the loss of variation from populations. Fisher (1922) responded to this with
the first, albeit marginally incorrect, mathematical treatment of the
'Hagedoorn effect' [48]. Notably, he expected that many natural populations
were too large (an N ~10,000) for the effects of drift to be substantial and
thought drift would have an insignificant effect on the evolutionary process.
The corrected mathematical treatment and term "genetic drift" was later coined
by a founder of population_genetics, Sewall_Wright. His first use of the term
"drift" was in 1929,[49] though at the time he was using it in the sense of a
directed process of change, or natural selection. Random drift by means of
sampling error came to be known as the "SewallâWright effect," though he was
never entirely comfortable to see his name given to it. Wright referred to all
changes in allele frequency as either "steady drift" (e.g., selection) or
"random drift" (e.g., sampling error).[50] "Drift" came to be adopted as a
technical term in the stochastic sense exclusively.[51] Today it is usually
defined still more narrowly, in terms of sampling error,[52] although this
narrow definition is not universal.[53][54] Wright wrote that the "restriction
of "random drift" or even "drift" to only one component, the effects of
accidents of sampling, tends to lead to confusion."[50] Sewall Wright
considered the process of random genetic drift by means of sampling error
equivalent to that by means of inbreeding, but later work has shown them to be
distinct.[55]
In the early days of the modern_evolutionary_synthesis, scientists were
beginning to blend the new science of population genetics with Charles_Darwin's
theory of natural selection. Within this framework, Wright focused on the
effects of inbreeding on small relatively isolated populations. He introduced
the concept of an adaptive_landscape in which phenomena such as cross breeding
and genetic drift in small populations could push them away from adaptive
peaks, which in turn allow natural selection to push them towards new adaptive
peaks.[56] Wright thought smaller populations were more suited for natural
selection because "inbreeding was sufficiently intense to create new
interaction systems through random drift but not intense enough to cause random
nonadaptive fixation of genes."[57]
Wright's views on the role of genetic drift in the evolutionary scheme were
controversial almost from the very beginning. One of the most vociferous and
influential critics was colleague Ronald Fisher. Fisher conceded genetic drift
played some role in evolution, but an insignificant one. Fisher has been
accused of misunderstanding Wright's views because in his criticisms Fisher
seemed to argue Wright had rejected selection almost entirely. To Fisher,
viewing the process of evolution as a long, steady, adaptive progression was
the only way to explain the ever-increasing complexity from simpler forms. But
the debates have continued between the "gradualists" and those who lean more
toward the Wright model of evolution where selection and drift together play an
important role.[58]
In 1968, Motoo Kimura rekindled the debate with his neutral theory of molecular
evolution, which claims that most of the genetic changes are caused by genetic
drift acting on neutral mutations.[6][7]
The role of genetic drift by means of sampling error in evolution has been
criticized by John_H._Gillespie[59] and William_B._Provine, who argue that
selection on linked sites is a more important stochastic force.
***** See also[edit] *****
    * Peripatric_speciation
    * Antigenic_drift
    * Coalescent_theory
    * Gene_pool
    * Meiotic_drive
***** Notes and references[edit] *****
   1. ^Gould_SJ (2002). "Chapter 7, section "Synthesis as Hardening"". The
      Structure_of_Evolutionary_Theory.
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
   3. ^ a b cMasel_J (October 2011). "Genetic drift". Current Biology. Cell
      Press. 21 (20): R837â8. doi:10.1016/j.cub.2011.08.007. PMID 22032182.
   4. ^ Futuyma_1998, Glossary
   5. ^Star B, Spencer HG (May 2013). "Effects_of_genetic_drift_and_gene_flow
      on_the_selective_maintenance_of_genetic_variation". Genetics. 194 (1):
      235â44. doi:10.1534/genetics.113.149781. PMC 3632471. PMID 23457235.
   6. ^ Miller_2000, p. 54
   7. ^ a bKimura_M (February 1968). "Evolutionary rate at the molecular
      level". Nature. Nature Publishing Group. 217 (5129): 624â6. Bibcode:
      1968Natur.217..624K. doi:10.1038/217624a0. PMID 5637732.
   8. ^ a b Futuyma_1998, p. 320
   9. ^"Sampling_Error_and_Evolution". Understanding Evolution. University_of
      California,_Berkeley. Archived from the original on 8 December 2015.
      Retrieved 1 December 2015.
  10. ^Wahl LM (August 2011). "Fixation_when_N_and_s_vary:_classic_approaches
      give_elegant_new_results". Genetics. Genetics_Society_of_America. 188
      (4): 783â5. doi:10.1534/genetics.111.131748. PMC 3176088.
      PMID 21828279.
  11. ^ a b Hartl_&_Clark_2007, p. 112
  12. ^ Tian_2008, p. 11
  13. ^Moran_PA (1958). "Random processes in genetics". Mathematical
      Proceedings_of_the_Cambridge_Philosophical_Society. 54 (1): 60â71.
      Bibcode:1958PCPS...54...60M. doi:10.1017/S0305004100033193.
  14. ^ a bCharlesworth_B (March 2009). "Fundamental concepts in genetics:
      effective population size and patterns of molecular evolution and
      variation". Nature Reviews. Genetics. Nature_Publishing_Group. 10 (3):
      195â205. doi:10.1038/nrg2526. PMID 19204717.
  15. ^Der R, Epstein_CL, Plotkin JB (September 2011). "Generalized population
      models and the nature of genetic drift". Theoretical Population Biology.
      Elsevier. 80 (2): 80â99. doi:10.1016/j.tpb.2011.06.004. PMID 21718713.
  16. ^ Li_&_Graur_1991, p. 28
  17. ^ a b cGillespie_JH (November 2001). "Is the population size of a species
      relevant to its evolution?". Evolution; International Journal of Organic
      Evolution. John_Wiley_&_Sons for the Society_for_the_Study_of_Evolution.
      55 (11): 2161â9. doi:10.1111/j.0014-3820.2001.tb00732.x. PMID 11794777.
  18. ^Neher RA, Shraiman BI (August 2011). "Genetic_draft_and_quasi-neutrality
      in_large_facultatively_sexual_populations". Genetics. Genetics Society of
      America. 188 (4): 975â96. arXiv:1108.1635. doi:10.1534/
      genetics.111.128876. PMC 3176096. PMID 21625002.
  19. ^ Ewens_2004
  20. ^ Li_&_Graur_1991, p. 29
  21. ^ Barton_et_al._2007, p. 417
  22. ^ Futuyma_1998, p. 300
  23. ^Otto_SP, Whitlock MC (June 1997). "The_probability_of_fixation_in
      populations_of_changing_size" (PDF). Genetics. Genetics Society of
      America. 146 (2): 723â33. PMC 1208011. PMID 9178020. Archived (PDF)
      from the original on 19 March 2015.
  24. ^Cutter AD, Choi JY (August 2010). "Natural_selection_shapes_nucleotide
      polymorphism_across_the_genome_of_the_nematode_Caenorhabditis_briggsae".
      Genome Research. Cold_Spring_Harbor_Laboratory_Press. 20 (8): 1103â11.
      doi:10.1101/gr.104331.109. PMC 2909573. PMID 20508143.
  25. ^ Hedrick_2005, p. 315
  26. ^ Li_&_Graur_1991, p. 33
  27. ^ Kimura_&_Ohta_1971
  28. ^Masel_J, King OD, Maughan H (January 2007). "The_loss_of_adaptive
      plasticity_during_long_periods_of_environmental_stasis". The American
      Naturalist. University_of_Chicago_Press on behalf of the American_Society
      of_Naturalists. 169 (1): 38â46. doi:10.1086/510212. PMC 1766558.
      PMID 17206583.
  29. ^"Natural_Selection:_How_Evolution_Works". Actionbioscience. Washington,
      D.C.: American_Institute_of_Biological_Sciences. Archived from the
      original on 6 January 2010. Retrieved 24 November 2009.
  30.  An interview with Douglas_J._Futuyma. See answer to question: Is natural
      selection the only mechanism of evolution?
  31. ^ a b c Cavalli-Sforza,_Menozzi_&_Piazza_1996
  32. ^ Zimmer_2001
  33. ^ Golding_1994, p. 46
  34. ^Charlesworth B, Morgan MT, Charlesworth_D (August 1993). "The_effect_of
      deleterious_mutations_on_neutral_molecular_variation" (PDF). Genetics.
      Genetics Society of America. 134 (4): 1289â303. PMC 1205596.
      PMID 8375663.
  35. ^Presgraves_DC (September 2005). "Recombination enhances protein
      adaptation in Drosophila melanogaster". Current Biology. Cell Press. 15
      (18): 1651â6. doi:10.1016/j.cub.2005.07.065. PMID 16169487.
  36. ^Nordborg M, Hu TT, Ishino Y, Jhaveri J, Toomajian C, Zheng H, Bakker E,
      Calabrese P, Gladstone J, Goyal R, Jakobsson M, Kim S, Morozov Y,
      Padhukasahasram B, Plagnol V, Rosenberg NA, Shah C, Wall JD, Wang J, Zhao
      K, Kalbfleisch T, Schulz V, Kreitman M, Bergelson J (July 2005). "The
      pattern_of_polymorphism_in_Arabidopsis_thaliana". PLoS Biology. Public
      Library_of_Science. 3 (7): e196. doi:10.1371/journal.pbio.0030196.
      PMC 1135296. PMID 15907155.
  37.  [open_access]
  38. ^Robinson R, ed. (2003). "Population_Bottleneck". Genetics. 3. New York:
      Macmillan_Reference_USA. ISBN 0-02-865609-1. LCCN 2002003560.
      OCLC 614996575. Archived from the original on 3 March 2016. Retrieved 14
      December 2015.
  39. ^ Futuyma_1998, pp. 303â304
  40. ^O'Corry-Crowe G (March 2008). "Climate change and the molecular ecology
      of Arctic marine mammals". Ecological Applications. Ecological_Society_of
      America. 18 (2 Suppl): S56â76. doi:10.1890/06-0795.1. PMID 18494363.
  41. ^Cornuet JM, Luikart G (December 1996). "Description_and_power_analysis
      of_two_tests_for_detecting_recent_population_bottlenecks_from_allele
      frequency_data". Genetics. Genetics Society of America. 144 (4):
      2001â14. PMC 1207747. PMID 8978083.
  42. ^ Sadava_et_al._2008, chpts. 1, 21â33, 52â57
  43. ^Dale C, Moran NA (August 2006). "Molecular interactions between
      bacterial symbionts and their hosts". Cell. 126 (3): 453â65. doi:
      10.1016/j.cell.2006.07.014. PMID 16901780.
  44. ^"Bottlenecks_and_founder_effects". Understanding Evolution. University
      of California, Berkeley. Archived from the original on 4 December 2015.
      Retrieved 14 December 2015.
  45. ^ Campbell_1996, p. 423
  46. ^"Genetic_Drift_and_the_Founder_Effect". Evolution Library (Web
      resource). Evolution. Boston, MA: WGBH_Educational_Foundation; Clear Blue
      Sky Productions, Inc. 2001. OCLC 48165595. Archived from the original on
      14 March 2009. Retrieved 7 April 2009.
  47. ^ Wolf,_Brodie_&_Wade_2000
  48. ^ Hey,_Fitch_&_Ayala_2005
  49. ^ Howard_&_Berlocher_1998
  50. ^Hagedoorn, AL; Hagedoorn-Vorstheuvel La Brand, AC (1921). The_Relative
      Value_of_the_Processes_Causing_Evolution. The Hague: Martinus Nijhoff.
  51. ^ Fisher, RA (1922). "On the Dominance Ratio". Proceedings of the Royal
      Society of Edinburgh. 42: 321â341.
  52. ^Wright_S (November â December 1929). "The evolution of dominance". The
      American Naturalist. Chicago, IL: University of Chicago Press on behalf
      of the American Society of Naturalists. 63 (689): 556â561. doi:10.1086/
      280290. ISSN 0003-0147. JSTOR 2456825.
  53. ^ a bWright S (1955). "Classification of the factors of evolution". Cold
      Spring Harbor Symposia on Quantitative Biology. Cold Spring Harbor, NY:
      Cold Spring Harbor Laboratory Press. 20: 16â24. doi:10.1101/
      SQB.1955.020.01.004. ISSN 0091-7451.
  54.  Symposium: "Population Genetics: The Nature and Causes of Genetic
      Variability in Populations".
  55. ^ Stevenson_1991
  56. ^ Freeman_&_Herron_2007
  57. ^Masel J (August 2012). "Rethinking Hardy-Weinberg and genetic drift in
      undergraduate biology". BioEssays. John Wiley & Sons. 34 (8): 701â10.
      doi:10.1002/bies.201100178. PMID 22576789.
  58. ^ Lynch_2007
  59. ^Crow_JF (March 2010). "Wright_and_Fisher_on_inbreeding_and_random
      drift". Genetics. Genetics Society of America. 184 (3): 609â11. doi:
      10.1534/genetics.109.110023. PMC 2845331. PMID 20332416.
  60. ^ Larson_2004, pp. 221â243
  61. ^ Stevenson_1991: Quote attributed to William_B._Provine in The Origins
      of Theoretical Population Genetics (1971), p. 162; Chicago: University of
      Chicago Press.
  62. ^ Avers_1989
  63. ^Gillespie JH (June 2000). "Genetic_drift_in_an_infinite_population._The
      pseudohitchhiking_model". Genetics. Genetics Society of America. 155 (2):
      909â19. PMC 1461093. PMID 10835409.
***** Bibliography[edit] *****
    * Avers CJ (1989). Process_and_Pattern_in_Evolution. New York: Oxford
      University_Press. ISBN 0-19-505275-7. LCCN 88005368. OCLC 17677554.
Barton_NH, Briggs_DE, Eisen_JA, Goldstein_DB, Patel NH (2007). Evolution. Cold
Spring Harbor, NY: Cold_Spring_Harbor_Laboratory_Press. ISBN 978-0-87969-684-9.
LCCN 2007010767. OCLC 86090399.
Campbell_NA (1996). Biology. Benjamin/Cummings Series in the Life Sciences (4th
ed.). Menlo Park, CA: Benjamin/Cummings_Pub._Co. ISBN 0-8053-1940-9.
LCCN 95045572. OCLC 33333455.
Cavalli-Sforza_LL, Menozzi P, Piazza_A (1996). The_History_and_Geography_of
Human_Genes (Abridged paperback ed.). Princeton, N.J.: Princeton_University
Press. ISBN 0-691-02905-9. OCLC 35527063.
Ewens_WJ (2004). Mathematical Population Genetics I. Theoretical Introduction.
Interdisciplinary Applied Mathematics. 27 (2nd ed.). New York: Springer-Verlag.
ISBN 0-387-20191-2. LCCN 2003065728. OCLC 53231891.
Freeman S, Herron JC (2007). Evolutionary Analysis (4th ed.). Upper Saddle
River, NJ: Pearson_Prentice_Hall. ISBN 978-0-13-227584-2. LCCN 2006034384.
OCLC 73502978.
Futuyma_D (1998). Evolutionary Biology (3rd ed.). Sunderland, MA: Sinauer
Associates. ISBN 0-87893-189-9. LCCN 97037947. OCLC 37560100.
Golding B, ed. (1994). Non-Neutral Evolution: Theories and Molecular Data. New
York: Chapman_&_Hall. ISBN 0-412-05391-8. LCCN 93047006. OCLC 29638235.
 "Papers from a workshop sponsored by the Canadian_Institute_for_Advanced
Research."
Hartl DL, Clark_AG (2007). Principles of Population Genetics (4th ed.).
Sunderland, MA: Sinauer Associates. ISBN 978-0-87893-308-2. LCCN 2006036153.
OCLC 75087956.
Hedrick PW (2005). Genetics of Populations (3rd ed.). Boston, MA: Jones_and
Bartlett_Publishers. ISBN 0-7637-4772-6. LCCN 2004056666. OCLC 56194719.
Hey_J, Fitch_WM, Ayala_FJ, eds. (2005). Systematics and the Origin of Species:
On Ernst Mayr's 100th Anniversary. Washington, D.C.: National_Academies_Press.
ISBN 978-0-309-09536-5. LCCN 2005017917. OCLC 70745851.
Howard DJ, Berlocher SH, eds. (1998). Endless Forms: Species and Speciation.
New York: Oxford University Press. ISBN 978-0-19-510901-6. LCCN 97031461.
OCLC 37545522.
Kimura_M, Ohta_T (1971). Theoretical Aspects of Population Genetics. Monographs
in Population Biology. 4. Princeton, NJ: Princeton_University_Press. ISBN 0-
691-08096-8. LCCN 75155963. OCLC 299867647.
Larson_EJ (2004). Evolution:_The_Remarkable_History_of_a_Scientific_Theory.
Modern Library Chronicles. 17. New York: Modern_Library. ISBN 0-679-64288-9.
LCCN 2003064888. OCLC 53483597.
Li_W, Graur_D (1991). Fundamentals_of_Molecular_Evolution. Sunderland, MA:
Sinauer Associates. ISBN 0-87893-452-9. LCCN 90043581. OCLC 22113526.
Lynch_M (2007). The Origins of Genome Architecture. Sunderland, MA: Sinauer
Associates. ISBN 978-0-87893-484-3. LCCN 2007000012. OCLC 77574049.
Miller_G (2000). The_Mating_Mind:_How_Sexual_Choice_Shaped_the_Evolution_of
Human_Nature. New York: Doubleday. ISBN 0-385-49516-1. LCCN 00022673.
OCLC 43648482.
Sadava D, Heller_HC, Orians_GH, Purves WK, Hillis_DM (2008). Life:_The_Science
of_Biology. II: Evolution, Diversity and Ecology (8th ed.). Sunderland, MA;
Gordonsville, VA: Sinauer Associates; W._H._Freeman_and_Company. ISBN 978-0-
7167-7674-1. LCCN 2006031320. OCLC 71632224.
Stevenson JC (1991). Dictionary of Concepts in Physical Anthropology. Reference
Sources for the Social Sciences and Humanities. 10. Westport, CT: Greenwood
Press. ISBN 0-313-24756-0. LCCN 90022815. OCLC 22732327.
Tian JP (2008). Evolution Algebras and their Applications. Lecture Notes in
Mathematics. 1921. Berlin; New York: Springer. doi:10.1007/978-3-540-74284-5.
ISBN 978-3-540-74283-8. LCCN 2007933498. OCLC 173807298. Zbl 1136.17001.
Wolf JB, Brodie ED, Wade_MJ, eds. (2000). Epistasis and the Evolutionary
Process. Oxford, UK; New York: Oxford University Press. ISBN 0-19-512806-0.
LCCN 99046515. OCLC 42603105.
Zimmer_C (2001). Evolution:_The_Triumph_of_an_Idea. Introduction by Stephen_Jay
Gould; foreword by Richard Hutton (1st ed.). New York: HarperCollins. ISBN 0-
06-019906-7. LCCN 2001024077. OCLC 46359440.
***** External links[edit] *****
 Wikimedia Commons has media related to Genetic_drift.
    * Sheehy, Bob. "Population_genetics_simulation_program". Radford, VA:
      Radford_University. Retrieved 21 December 2015.
Grimes, Bill. "Genetic_Drift_Simulation". Tucson, Arizona: The_University_of
Arizona. Retrieved 25 August 2016.
    * v
    * t
    * e
Evolutionary_biology
    * Introduction
    * Outline
    * Timeline_of_evolution
    * Evolutionary_history_of_life
    * Index
                        * Abiogenesis
                        * Adaptation
                        * Adaptive_radiation
                        * Cladistics
                        * Coevolution
                        * Common_descent
                        * Convergence
                        * Divergence
                        * Earliest_known_life
                          forms
                        * Evidence_of_evolution
Evolution               * Extinction
                              o Event
                        * Gene-centered_view
                        * Homology
                        * Last_universal_common
                          ancestor
                        * Macroevolution
                        * Microevolution
                        * Origin_of_life
                        * Panspermia
                        * Parallel_evolution
                        * Speciation
                        * Taxonomy
                        * Biodiversity
                        * Gene_flow
Population_genetics     * Genetic drift
                        * Mutation
                        * Natural_selection
                        * Variation
                        * Canalisation
                        * Evolutionary
Development               developmental_biology
                        * Inversion
                        * Modularity
                        * Phenotypic_plasticity
                        * Birds
                              o origin
                        * Brachiopods
                        * Molluscs
                              o Cephalopods
                        * Dinosaurs
                        * Fish
                        * Fungi
                        * Insects
                              o butterflies
                        * Life
                        * Mammals
                              o cats
                              o canids
                                    # wolves
Of taxa                             # dogs
                              o hyenas
                              o dolphins_and
                                whales
                              o horses
                              o Kangaroos
                              o primates
                                    # humans
                                    # lemurs
                              o sea_cows
                        * Plants
                        * Reptiles
                        * Spiders
                        * Tetrapods
                        * Viruses
                              o influenza
                        * Cell
                        * DNA
                        * Flagella
                        * Eukaryotes
                              o symbiogenesis
                              o chromosome
                              o endomembrane
                                system
Of organs                     o mitochondria
                              o nucleus
                              o plastids
                        * In animals
                              o eye
                              o hair
                              o auditory_ossicle
                              o nervous_system    [Common-Avian-Infection-
                              o brain             Plagued-the-Tyrant-Dinosaurs-
                        * Aging                   pone.0007288.g004.jpg]
                              o Death                 * [icon]Evolutionary
                              o Programmed_cell         biology_portal
                                death
                        * Avian_flight
                        * Biological_complexity
                        * Cooperation
                        * Color_vision
                              o in_primates
                        * Emotion
                        * Empathy
                        * Ethics
                        * Eusociality
                        * Immune_system
Of processes            * Metabolism
                        * Monogamy
                        * Morality
                        * Mosaic_evolution
                        * Multicellularity
                        * Sexual_reproduction
                              o Gamete
                                differentiation/
                                sexes
                              o Life_cycles/
                                nuclear_phases
                              o Mating_types
                              o Meiosis
                              o Sex-determination
                        * Snake_venom
                        * Gradualism/Punctuated
                          equilibrium/
                          Saltationism
Tempo_and_modes         * Micromutation/
                          Macromutation
                        * Uniformitarianism/
                          Catastrophism
                        * Allopatric
                        * Anagenesis
                        * Catagenesis
                        * Cladogenesis
                        * Cospeciation
Speciation              * Ecological
                        * Hybrid
                        * Parapatric
                        * Peripatric
                        * Reinforcement
                        * Sympatric
                        * Renaissance_and
                          Enlightenment
                        * Transmutation_of
                          species
                        * Charles_Darwin
                              o On_the_Origin_of
                                Species
                        * History_of_paleontology
History                 * Transitional_fossil
                        * Blending_inheritance
                        * Mendelian_inheritance
                        * The_eclipse_of
                          Darwinism
                        * Modern_synthesis
                        * History_of_molecular
                          evolution
                        * Extended_evolutionary
                          synthesis
                        * Darwinism
                        * Alternatives
                              o Catastrophism
                              o Lamarckism
                              o Orthogenesis
Philosophy                    o Mutationism
                              o Saltationism
                              o Structuralism
                                    # Spandrel
                              o Theistic
                              o Vitalism
                        * Teleology_in_biology
                        * Biogeography
                        * Ecological_genetics
                        * Molecular_evolution
Related                 * Phylogenetics
                              o Tree
                        * Polymorphism
                        * Protocell
                        * Systematics
    * [Category] Category
    * [Portal] Portal
    * v
    * t
    * e
Population_genetics
                         * Hardy-Weinberg_law
                         * Genetic_linkage
                         * Identity_by_descent
                         * Linkage_disequilibrium
                         * Fisher's_fundamental_theorem
Key concepts             * Neutral_theory
                         * Shifting_balance_theory
                         * Price_equation
                         * Coefficient_of_inbreeding and relationship
                         * Fitness
                         * Heritability
                         * Natural
Selection                * Sexual
                         * Artificial
                         * Ecological
Effects of selection     * Genetic_hitchhiking
on genomic variation     * Background_selection
                         * Small_population_size
                         * Population_bottleneck
Genetic drift            * Founder_effect
                         * Coalescence
                         * BaldingâNichols_model
                         * R._A._Fisher
Founders                 * J._B._S._Haldane
                         * Sewall_Wright
                         * Evolution
                         * Microevolution
Related topics           * Evolutionary_game_theory
                         * Fitness_landscape
                         * Genetic_genealogy
                         * Quantitative_genetics
Index_of_evolutionary_biology_articles
    * [Tree_of_life.svg]Evolutionary_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=Genetic_drift&oldid=909430538"
Categories:
    * Population_genetics
    * Evolutionary_biology
    * Genetic_genealogy
Hidden categories:
    * Articles_with_short_description
    * Use_dmy_dates_from_January_2011
    * Good_articles
    * CS1:_long_volume_value
    * Commons_category_link_is_on_Wikidata
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * Asturianu
    * à¦¬à¦¾à¦à¦²à¦¾
    * ÐÐ°ÑÒ¡Ð¾ÑÑÑÐ°
    * ÐÑÐ»Ð³Ð°ÑÑÐºÐ¸
    * Bosanski
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * ÎÎ»Î»Î·Î½Î¹ÎºÎ¬
    * EspaÃ±ol
    * Esperanto
    * Euskara
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * íêµ­ì´
    * ÕÕ¡ÕµÕ¥ÖÕ¥Õ¶
    * à¤¹à¤¿à¤¨à¥à¤¦à¥
    * Hrvatski
    * Bahasa_Indonesia
    * Interlingua
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * á¥áá áá£áá
    * KreyÃ²l_ayisyen
    * ÐÑÑÐ³ÑÐ·ÑÐ°
    * LietuviÅ³
    * Magyar
    * ÐÐ°ÐºÐµÐ´Ð¾Ð½ÑÐºÐ¸
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Norsk_nynorsk
    * OÊ»zbekcha/ÑÐ·Ð±ÐµÐºÑÐ°
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð ÑÑÑÐºÐ¸Ð¹
    * Scots
    * Simple_English
    * SlovenÅ¡Äina
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * å´è¯­
    * ä¸­æ
Edit_links
    * This page was last edited on 5 August 2019, at 11:19 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
