The following text has been accessed from https://en.wikipedia.org/wiki/DNA_sequencing at Fri Aug 9 02:22:42 IST 2019
Creative_Commons_Attribution-ShareAlike_License





















****** DNA sequencing ******
From Wikipedia, the free encyclopedia
Jump_to_navigation Jump_to_search
process of determining the nucleic acid sequence â the order of nucleotides
in DNA
Part of a_series on
Genetics
[Hybridogenesis_in_water_frogs_gametes.svg]
 
Key components
    * Chromosome
    * DNA
    * RNA
    * Genome
    * Heredity
    * Mutation
    * Nucleotide
    * Variation
    * Outline
    * Index
History and topics
    * Introduction
    * History
    * Evolution (molecular)
    * Population_genetics
    * Mendelian_inheritance
    * Quantitative_genetics
    * Molecular_genetics
Research
    * DNA sequencing
    * Genetic_engineering
    * Genomics ([Template] template)
    * Medical_genetics
    * Branches_of_genetics
Personalized medicine
Personalized_medicine
    * Biology_portal
    * Molecular_and_cellular_biology_portal
    * v
    * t
    * e
DNA sequencing is the process of determining the nucleic_acid_sequence â the
order of nucleotides in DNA. It includes any method or technology that is used
to determine the order of the four bases: adenine, guanine, cytosine, and
thymine. The advent of rapid DNA sequencing methods has greatly accelerated
biological and medical research and discovery.[1][2]
Knowledge of DNA sequences has become indispensable for basic biological
research, and in numerous applied fields such as medical_diagnosis,
biotechnology, forensic_biology, virology and biological systematics. Comparing
healthy and mutated DNA sequences can diagnose different diseases including
various cancers,[3] characterize antibody repertoire,[4] and can be used to
guide patient treatment.[5] Having a quick way to sequence DNA allows for
faster and more individualized medical care to be administered, and for more
organisms to be identified and cataloged.[4]
The rapid speed of sequencing attained with modern DNA sequencing technology
has been instrumental in the sequencing of complete DNA sequences, or genomes,
of numerous types and species of life, including the human_genome and other
complete DNA sequences of many animal, plant, and microbial species.
An example of the results of automated chain-termination DNA sequencing.
The first DNA sequences were obtained in the early 1970s by academic
researchers using laborious methods based on two-dimensional_chromatography.
Following the development of fluorescence-based sequencing methods with a DNA
sequencer,[6] DNA sequencing has become easier and orders of magnitude faster.
[7]
⁰
***** Contents *****
    * 1_Applications
          o 1.1_Molecular_biology
          o 1.2_Evolutionary_biology
          o 1.3_Metagenomics
          o 1.4_Medicine
          o 1.5_Forensics
    * 2_The_four_canonical_bases
    * 3_History
          o 3.1_Discovery_of_DNA_structure_and_function
          o 3.2_RNA_sequencing
          o 3.3_Early_DNA_sequencing_methods
          o 3.4_Sequencing_of_full_genomes
          o 3.5_High-throughput_sequencing_(HTS)_methods
    * 4_Basic_methods
          o 4.1_Maxam-Gilbert_sequencing
          o 4.2_Chain-termination_methods
    * 5_Advanced_methods_and_de_novo_sequencing
          o 5.1_Shotgun_sequencing
          o 5.2_Bridge_PCR
    * 6_High-throughput_methods
          o 6.1_Massively_parallel_signature_sequencing_(MPSS)
          o 6.2_Polony_sequencing
          o 6.3_454_pyrosequencing
          o 6.4_Illumina_(Solexa)_sequencing
          o 6.5_Combinatorial_probe_anchor_synthesis_(cPAS)
          o 6.6_SOLiD_sequencing
          o 6.7_Ion_Torrent_semiconductor_sequencing
          o 6.8_DNA_nanoball_sequencing
          o 6.9_Heliscope_single_molecule_sequencing
          o 6.10_Single_molecule_real_time_(SMRT)_sequencing
          o 6.11_Nanopore_DNA_sequencing
          o 6.12_Microfluidic_Systems
    * 7_Methods_in_development
          o 7.1_Tunnelling_currents_DNA_sequencing
          o 7.2_Sequencing_by_hybridization
          o 7.3_Sequencing_with_mass_spectrometry
          o 7.4_Microfluidic_Sanger_sequencing
          o 7.5_Microscopy-based_techniques
          o 7.6_RNAP_sequencing
          o 7.7_In_vitro_virus_high-throughput_sequencing
          o 7.8_Sample_preparation
          o 7.9_Development_initiatives
          o 7.10_Computational_challenges
                # 7.10.1_Read_trimming
          o 7.11_Ethical_issues
          o 7.12_See_also
          o 7.13_Notes
          o 7.14_References
          o 7.15_External_links
***** Applications[edit] *****
DNA sequencing may be used to determine the sequence of individual genes,
larger genetic regions (i.e. clusters of genes or operons), full chromosomes,
or entire_genomes of any organism. DNA sequencing is also the most efficient
way to indirectly sequence RNA or proteins (via their open_reading_frames). In
fact, DNA sequencing has become a key technology in many areas of biology and
other sciences such as medicine, forensics, and anthropology.
**** Molecular biology[edit] ****
Sequencing is used in molecular_biology to study genomes and the proteins they
encode. Information obtained using sequencing allows researchers to identify
changes in genes, associations with diseases and phenotypes, and identify
potential drug targets.
**** Evolutionary biology[edit] ****
Since DNA is an informative macromolecule in terms of transmission from one
generation to another, DNA sequencing is used in evolutionary_biology to study
how different organisms are related and how they evolved.
**** Metagenomics[edit] ****
Main article: Metagenomics
The field of metagenomics involves identification of organisms present in a
body of water, sewage, dirt, debris filtered from the air, or swab samples from
organisms. Knowing which organisms are present in a particular environment is
critical to research in ecology, epidemiology, microbiology, and other fields.
Sequencing enables researchers to determine which types of microbes may be
present in a microbiome, for example.
**** Medicine[edit] ****
Medical technicians may sequence genes (or, theoretically, full genomes) from
patients to determine if there is risk of genetic diseases. This is a form of
genetic_testing, though some genetic tests may not involve DNA sequencing.
**** Forensics[edit] ****
DNA sequencing may be used along with DNA_profiling methods for forensic
identification[8] and paternity_testing. DNA testing has evolved tremendously
in the last few decades to ultimately link a DNA print to what is under
investigation. The DNA patterns in fingerprint, saliva, hair follicles, etc.
uniquely separate each living organism from another. Testing DNA is a technique
which can detect specific genomes in a DNA strand to produce a unique and
individualized pattern.
***** The four canonical bases[edit] *****
Main article: Nucleotide
The canonical structure of DNA has four bases: thymine (T), adenine (A),
cytosine (C), and guanine (G). DNA sequencing is the determination of the
physical order of these bases in a molecule of DNA. However, there are many
other bases that may be present in a molecule. In some viruses (specifically,
bacteriophage), cytosine may be replaced by hydroxy methyl or hydroxy methyl
glucose cytosine.[9] In mammalian DNA, variant bases with methyl groups or
phosphosulfate may be found.[10][11] Depending on the sequencing technique, a
particular modification, e.g., the 5mC (5_methyl_cytosine) common in humans,
may or may not be detected.[12]
***** History[edit] *****
**** Discovery of DNA structure and function[edit] ****
Deoxyribonucleic acid (DNA) was first discovered and isolated by Friedrich
Miescher in 1869, but it remained understudied for many decades because
proteins, rather than DNA, were thought to hold the genetic blueprint to life.
This situation changed after 1944 as a result of some experiments by Oswald
Avery, Colin_MacLeod, and Maclyn_McCarty demonstrating that purified DNA could
change one strain of bacteria into another. This was the first time that DNA
was shown capable of transforming the properties of cells.
In 1953, James_Watson and Francis_Crick put forward their double-helix model of
DNA, based on crystallized_X-ray structures being studied by Rosalind_Franklin.
According to the model, DNA is composed of two strands of nucleotides coiled
around each other, linked together by hydrogen bonds and running in opposite
directions. Each strand is composed of four complementary nucleotides â
adenine (A), cytosine (C), guanine (G) and thymine (T) â with an A on one
strand always paired with T on the other, and C always paired with G. They
proposed such a structure allowed each strand to be used to reconstruct the
other, an idea central to the passing on of hereditary information between
generations.[13]
Frederick_Sanger, a pioneer of sequencing. Sanger is one of the few scientists
who was awarded two Nobel prizes, one for the sequencing_of_proteins, and the
other for the sequencing of DNA.
The foundation for sequencing proteins was first laid by the work of Frederick
Sanger who by 1955 had completed the sequence of all the amino acids in
insulin, a small protein secreted by the pancreas. This provided the first
conclusive evidence that proteins were chemical entities with a specific
molecular pattern rather than a random mixture of material suspended in fluid.
Sanger's success in sequencing insulin greatly electrified x-ray
crystallographers, including Watson and Crick who by now were trying to
understand how DNA directed the formation of proteins within a cell. Soon after
attending a series of lectures given by Frederick Sanger in October 1954, Crick
began to develop a theory which argued that the arrangement of nucleotides in
DNA determined the sequence of amino acids in proteins which in turn helped
determine the function of a protein. He published this theory in 1958.[14]
**** RNA sequencing[edit] ****
RNA_sequencing was one of the earliest forms of nucleotide sequencing. The
major landmark of RNA sequencing is the sequence of the first complete gene and
the complete genome of Bacteriophage_MS2, identified and published by Walter
Fiers and his coworkers at the University_of_Ghent (Ghent, Belgium), in 1972
[15] and 1976.[16] Traditional RNA sequencing methods require the creation of a
cDNA molecule which must be sequenced.[17]
**** Early DNA sequencing methods[edit] ****
The first method for determining DNA sequences involved a location-specific
primer extension strategy established by Ray_Wu at Cornell_University in 1970.
[18] DNA polymerase catalysis and specific nucleotide labeling, both of which
figure prominently in current sequencing schemes, were used to sequence the
cohesive ends of lambda phage DNA.[19][20][21] Between 1970 and 1973, Wu, R
Padmanabhan and colleagues demonstrated that this method can be employed to
determine any DNA sequence using synthetic location-specific primers.[22][23]
[24] Frederick_Sanger then adopted this primer-extension strategy to develop
more rapid DNA sequencing methods at the MRC_Centre, Cambridge, UK and
published a method for "DNA sequencing with chain-terminating inhibitors" in
1977.[25] Walter_Gilbert and Allan_Maxam at Harvard also developed sequencing
methods, including one for "DNA sequencing by chemical degradation".[26][27] In
1973, Gilbert and Maxam reported the sequence of 24 basepairs using a method
known as wandering-spot analysis.[28] Advancements in sequencing were aided by
the concurrent development of recombinant_DNA technology, allowing DNA samples
to be isolated from sources other than viruses.
**** Sequencing of full genomes[edit] ****
The 5,386 bp genome of bacteriophage_ÏX174. Each coloured block represents a
gene.
The first full DNA genome to be sequenced was that of bacteriophage_ÏX174 in
1977.[29] Medical_Research_Council scientists deciphered the complete DNA
sequence of the Epstein-Barr_virus in 1984, finding it contained 172,282
nucleotides. Completion of the sequence marked a significant turning point in
DNA sequencing because it was achieved with no prior genetic profile knowledge
of the virus.[30]
A non-radioactive method for transferring the DNA molecules of sequencing
reaction mixtures onto an immobilizing matrix during electrophoresis was
developed by Pohl and co-workers in the early 1980s.[31][32] Followed by the
commercialization of the DNA sequencer "Direct-Blotting-Electrophoresis-System
GATC 1500" by GATC_Biotech, which was intensively used in the framework of the
EU genome-sequencing programme, the complete DNA sequence of the yeast
Saccharomyces_cerevisiae chromosome II.[33] Leroy_E._Hood's laboratory at the
California_Institute_of_Technology announced the first semi-automated DNA
sequencing machine in 1986.[34] This was followed by Applied_Biosystems'
marketing of the first fully automated sequencing machine, the ABI 370, in 1987
and by Dupont's Genesis 2000[35] which used a novel fluorescent labeling
technique enabling all four dideoxynucleotides to be identified in a single
lane. By 1990, the U.S. National_Institutes_of_Health (NIH) had begun large-
scale sequencing trials on Mycoplasma_capricolum, Escherichia_coli,
Caenorhabditis_elegans, and Saccharomyces_cerevisiae at a cost of US$0.75 per
base. Meanwhile, sequencing of human cDNA sequences called expressed_sequence
tags began in Craig_Venter's lab, an attempt to capture the coding fraction of
the human_genome.[36] In 1995, Venter, Hamilton_Smith, and colleagues at The
Institute_for_Genomic_Research (TIGR) published the first complete genome of a
free-living organism, the bacterium Haemophilus_influenzae. The circular
chromosome contains 1,830,137 bases and its publication in the journal Science
[37] marked the first published use of whole-genome shotgun sequencing,
eliminating the need for initial mapping efforts.
By 2001, shotgun sequencing methods had been used to produce a draft sequence
of the human genome.[38][39]
**** High-throughput sequencing (HTS) methods[edit] ****
Several new methods for DNA sequencing were developed in the mid to late 1990s
and were implemented in commercial DNA_sequencers by the year 2000. Together
these were called the "next-generation" or "second-generation" sequencing (NGS)
methods, in order to distinguish them from the aforementioned earlier methods,
like Sanger Sequencing. In contrast to the first generation of sequencing, NGS
technology is typically characterized by being highly scalable, allowing the
entire genome to be sequenced at once. Usually, this is accomplished by
fragmenting the genome into small pieces, randomly sampling for a fragment, and
sequencing it using one of a variety of technologies, such as those described
below. An entire genome is possible because multiple fragments are sequenced at
once (giving it the name "massively parallel" sequencing) in an automated
process.
NGS technology has tremendously empowered researchers to look for insights into
health, anthropologists to investigate human origins, and is catalyzing the
"Personalized_Medicine" movement. However, it has also opened the door to more
room for error. There are many software tools to carry out the computational
analysis of NGS data, each with its own algorithm. Even the parameters within
one software package can change the outcome of the analysis. In addition, the
large quantities of data produced by DNA sequencing have also required
development of new methods and programs for sequence analysis. Several efforts
to develop standards in the NGS field have been attempted to address these
challenges, most of which have been small-scale efforts arising from individual
labs. Most recently, a large, organized, FDA-funded effort has culminated in
the BioCompute standard.
On 26 October 1990, Roger_Tsien, Pepi Ross, Margaret Fahnestock and Allan J
Johnston filed a patent describing stepwise ("base-by-base") sequencing with
removable 3' blockers on DNA arrays (blots and single DNA molecules).[40] In
1996, PÃ¥l_NyrÃ©n and his student Mostafa_Ronaghi at the Royal Institute of
Technology in Stockholm published their method of pyrosequencing.[41]
On 1 April 1997, Pascal Mayer and Laurent Farinelli submitted patents to the
World Intellectual Property Organization describing DNA colony sequencing.[42]
The DNA sample preparation and random surface-PCR arraying methods described in
this patent, coupled to Roger Tsien et al.'s "base-by-base" sequencing method,
is now implemented in Illumina's Hi-Seq genome sequencers.
In 1998, Phil Green and Brent Ewing of the University of Washington described
their phred_quality_score for sequencer data analysis,[43] a landmark analysis
technique that gained widespread adoption, and which is still the most common
metric for assessing the accuracy of a sequencing platform.[44]
Lynx Therapeutics published and marketed Massively_parallel_signature
sequencing (MPSS), in 2000. This method incorporated a parallelized, adapter/
ligation-mediated, bead-based sequencing technology and served as the first
commercially available "next-generation" sequencing method, though no DNA
sequencers were sold to independent laboratories.[45]
***** Basic methods[edit] *****
**** Maxam-Gilbert sequencing[edit] ****
Main article: Maxam-Gilbert_sequencing
Allan_Maxam and Walter_Gilbert published a DNA sequencing method in 1977 based
on chemical modification of DNA and subsequent cleavage at specific bases.[26]
Also known as chemical sequencing, this method allowed purified samples of
double-stranded DNA to be used without further cloning. This method's use of
radioactive labeling and its technical complexity discouraged extensive use
after refinements in the Sanger methods had been made.
Maxam-Gilbert sequencing requires radioactive labeling at one 5' end of the DNA
and purification of the DNA fragment to be sequenced. Chemical treatment then
generates breaks at a small proportion of one or two of the four nucleotide
bases in each of four reactions (G, A+G, C, C+T). The concentration of the
modifying chemicals is controlled to introduce on average one modification per
DNA molecule. Thus a series of labeled fragments is generated, from the
radiolabeled end to the first "cut" site in each molecule. The fragments in the
four reactions are electrophoresed side by side in denaturing acrylamide gels
for size separation. To visualize the fragments, the gel is exposed to X-ray
film for autoradiography, yielding a series of dark bands each corresponding to
a radiolabeled DNA fragment, from which the sequence may be inferred.[26]
**** Chain-termination methods[edit] ****
Main article: Sanger_sequencing
The chain-termination_method developed by Frederick_Sanger and coworkers in
1977 soon became the method of choice, owing to its relative ease and
reliability.[25][46] When invented, the chain-terminator method used fewer
toxic chemicals and lower amounts of radioactivity than the Maxam and Gilbert
method. Because of its comparative ease, the Sanger method was soon automated
and was the method used in the first generation of DNA_sequencers.
Sanger sequencing is the method which prevailed from the 1980s until the mid-
2000s. Over that period, great advances were made in the technique, such as
fluorescent labelling, capillary electrophoresis, and general automation. These
developments allowed much more efficient sequencing, leading to lower costs.
The Sanger method, in mass production form, is the technology which produced
the first_human_genome in 2001, ushering in the age of genomics. However, later
in the decade, radically different approaches reached the market, bringing the
cost per genome down from $100 million in 2001 to $10,000 in 2011.[47]
***** Advanced methods and de novo sequencing[edit] *****
Genomic DNA is fragmented into random pieces and cloned as a bacterial library.
DNA from individual bacterial clones is sequenced and the sequence is assembled
by using overlapping DNA regions.(click to expand)
Large-scale sequencing often aims at sequencing very long DNA pieces, such as
whole chromosomes, although large-scale sequencing can also be used to generate
very large numbers of short sequences, such as found in phage_display. For
longer targets such as chromosomes, common approaches consist of cutting (with
restriction_enzymes) or shearing (with mechanical forces) large DNA fragments
into shorter DNA fragments. The fragmented DNA may then be cloned into a DNA
vector and amplified in a bacterial host such as Escherichia_coli. Short DNA
fragments purified from individual bacterial colonies are individually
sequenced and assembled_electronically into one long, contiguous sequence.
Studies have shown that adding a size selection step to collect DNA fragments
of uniform size can improve sequencing efficiency and accuracy of the genome
assembly. In these studies, automated sizing has proven to be more reproducible
and precise than manual gel sizing.[48][49][50]
The term "de novo sequencing" specifically refers to methods used to determine
the sequence of DNA with no previously known sequence. De novo translates from
Latin as "from the beginning". Gaps in the assembled sequence may be filled by
primer_walking. The different strategies have different tradeoffs in speed and
accuracy; shotgun_methods are often used for sequencing large genomes, but its
assembly is complex and difficult, particularly with sequence_repeats often
causing gaps in genome assembly.
Most sequencing approaches use an in vitro cloning step to amplify individual
DNA molecules, because their molecular detection methods are not sensitive
enough for single molecule sequencing. Emulsion PCR[51] isolates individual DNA
molecules along with primer-coated beads in aqueous droplets within an oil
phase. A polymerase_chain_reaction (PCR) then coats each bead with clonal
copies of the DNA molecule followed by immobilization for later sequencing.
Emulsion PCR is used in the methods developed by Marguilis et al.
(commercialized by 454_Life_Sciences), Shendure and Porreca et al. (also known
as "Polony_sequencing") and SOLiD_sequencing, (developed by Agencourt, later
Applied_Biosystems, now Life_Technologies).[52][53][54] Emulsion PCR is also
used in the GemCode and Chromium platforms developed by 10x Genomics.[55]
**** Shotgun sequencing[edit] ****
Main article: Shotgun_sequencing
Shotgun sequencing is a sequencing method designed for analysis of DNA
sequences longer than 1000 base pairs, up to and including entire chromosomes.
This method requires the target DNA to be broken into random fragments. After
sequencing individual fragments, the sequences can be reassembled on the basis
of their overlapping regions.[56]
**** Bridge PCR[edit] ****
Another method for in_vitro clonal amplification is bridge PCR, in which
fragments are amplified upon primers attached to a solid surface[42][57][58]
and form "DNA_colonies" or "DNA clusters". This method is used in the Illumina
Genome Analyzer sequencers. Single-molecule methods, such as that developed by
Stephen_Quake's laboratory (later commercialized by Helicos) are an exception:
they use bright fluorophores and laser excitation to detect base addition
events from individual DNA molecules fixed to a surface, eliminating the need
for molecular amplification.[59]
***** High-throughput methods[edit] *****
Multiple, fragmented sequence reads must be assembled together on the basis of
their overlapping areas.
High-throughput, or next-generation,[nt_1] sequencing applies to genome
sequencing, genome resequencing, transcriptome profiling (RNA-Seq), DNA-protein
interactions (ChIP-sequencing), and epigenome characterization.[60]
Resequencing is necessary, because the genome of a single individual of a
species will not indicate all of the genome variations among other individuals
of the same species.
The high demand for low-cost sequencing has driven the development of high-
throughput sequencing technologies that parallelize the sequencing process,
producing thousands or millions of sequences concurrently.[61][62][63] High-
throughput sequencing technologies are intended to lower the cost of DNA
sequencing beyond what is possible with standard dye-terminator methods.[64] In
ultra-high-throughput sequencing as many as 500,000 sequencing-by-synthesis
operations may be run in parallel.[65][66][67] Such technologies led to the
ability to sequence an entire human genome in as little as one day.[68] As of
2019[update], corporate leaders in the development of high-throughput
sequencing products included Illumina, Qiagen and ThermoFisher_Scientific.[68]
                                                                                  Comparison of high-throughput sequencing methods[69][70]
Method         Read length                       Accuracy (single    Reads per run                                              Time per    Cost per 1 million bases (in US$)                   Advantages    Disadvantages
                                                 read not consensus)                                                            run
Single-                                                                                                                                                                                                       Moderate
molecule real- 30,000 bp (N50);                                                                                                 30 minutes                                                      Fast. Detects throughput.
time           maximum read length >100,000      87% raw-read        500,000 per Sequel SMRT cell, 10â20 gigabases[71][75] to 20 hours $0.05â$0.08                                    4mC, 5mC,     Equipment can
sequencing     bases[71][72][73]                 accuracy[74]        [76]                                                       [71][77]                                                        6mA.[78]      be very
(Pacific                                                                                                                                                                                                      expensive.
Biosciences)
Ion                                                                                                                                                                                             Less
semiconductor  up to 600 bp[79]                  99.6%[80]           up to 80 million                                           2 hours     $1                                                  expensive     Homopolymer
(Ion Torrent                                                                                                                                                                                    equipment.    errors.
sequencing)                                                                                                                                                                                     Fast.
                                                                                                                                                                                                              Runs are
Pyrosequencing 700 bp                            99.9%               1 million                                                  24 hours    $10                                                 Long read     expensive.
(454)                                                                                                                                                                                           size. Fast.   Homopolymer
                                                                                                                                                                                                              errors.
                                                                                                                                1 to 11                                                         Potential for
                                                                                                                                days,                                                           high sequence Equipment can
               MiniSeq, NextSeq: 75â300 bp;                     MiniSeq/MiSeq: 1â25 Million;                          depending                                                       yield,        be very
Sequencing by  MiSeq: 50â600 bp;                                NextSeq: 130-00 Million;                                   upon                                                            depending     expensive.
synthesis      HiSeq 2500: 50â500 bp;       99.9% (Phred30)     HiSeq 2500: 300 million â 2 billion;                  sequencer   $0.05 to $0.15                                      upon          Requires high
(Illumina)     HiSeq 3/4000: 50â300 bp;                         HiSeq 3/4000 2.5 billion;                                  and                                                             sequencer     concentrations
               HiSeq X: 300 bp                                       HiSeq X: 3 billion                                         specified                                                       model and     of DNA.
                                                                                                                                read length                                                     desired
                                                                                                                                [81]                                                            application.
                                                                                                                                1 to 9 days
                                                                                                                                depending
Combinatorial  BGISEQ-50: 35-50bp;                                   BGISEQ-50: 160M;                                           on
probe anchor   MGISEQ 200: 50-200bp;                                 MGISEQ 200: 300M;                                          instrument,
synthesis      BGISEQ-500, MGISEQ-2000: 50-300bp 99.9% (Phred30)     BGISEQ-500: 1300M per flow cell;                           read length $0.035- $0.12
(cPAS- BGI/    [82]                                                  MGISEQ-2000: 375M FCS flow cell, 1500M FCL flow cell per   and number
MGI)                                                                 flow cell.                                                 of flow
                                                                                                                                cells run
                                                                                                                                at a time.
                                                                                                                                                                                                              Slower than
Sequencing by                                                                                                                                                                                                 other methods.
ligation       50+35 or 50+50 bp                 99.9%               1.2 to 1.4 billion                                         1 to 2      $0.13                                               Low cost per  Has issues
(SOLiD                                                                                                                          weeks                                                           base.         sequencing
sequencing)                                                                                                                                                                                                   palindromic
                                                                                                                                                                                                              sequences.[83]
                                                                                                                                                                                                Longest       Lower
                                                                                                                                data                                                            individual    throughput
               Dependent on library preparation,                                                                                streamed in                                                     reads.        than other
Nanopore       not the device, so user chooses   ~92â97% single dependent on read length selected by user                  real time.  $500â999 per Flow Cell, base cost dependent on Accessible    machines,
Sequencing     read length (up to 2,272,580 bp   read                                                                           Choose 1    expt                                                user          Single read
               reported[84]).                                                                                                   min to 48                                                       community.    accuracy in
                                                                                                                                hrs                                                             Portable      90s.
                                                                                                                                                                                                (Palm sized).
                                                                                                                                                                                                              More expensive
                                                                                                                                                                                                              and
                                                                                                                                                                                                              impractical
                                                                                                                                                                                                              for larger
Chain                                                                                                                                                                                                         sequencing
termination                                                                                                                     20 minutes                                                      Useful for    projects. This
(Sanger        400 to 900 bp                     99.9%               N/A                                                        to 3 hours  $2400                                               many          method also
sequencing)                                                                                                                                                                                     applications. requires the
                                                                                                                                                                                                              time-consuming
                                                                                                                                                                                                              step of
                                                                                                                                                                                                              plasmid
                                                                                                                                                                                                              cloning or
                                                                                                                                                                                                              PCR.
**** Massively parallel signature sequencing (MPSS)[edit] ****
The first of the high-throughput sequencing technologies, massively_parallel
signature_sequencing (or MPSS), was developed in the 1990s at Lynx
Therapeutics, a company founded in 1992 by Sydney_Brenner and Sam_Eletr. MPSS
was a bead-based method that used a complex approach of adapter ligation
followed by adapter decoding, reading the sequence in increments of four
nucleotides. This method made it susceptible to sequence-specific bias or loss
of specific sequences. Because the technology was so complex, MPSS was only
performed 'in-house' by Lynx Therapeutics and no DNA sequencing machines were
sold to independent laboratories. Lynx Therapeutics merged with Solexa (later
acquired by Illumina) in 2004, leading to the development of sequencing-by-
synthesis, a simpler approach acquired from Manteia_Predictive_Medicine, which
rendered MPSS obsolete. However, the essential properties of the MPSS output
were typical of later high-throughput data types, including hundreds of
thousands of short DNA sequences. In the case of MPSS, these were typically
used for sequencing cDNA for measurements of gene_expression levels.[45]
**** Polony sequencing[edit] ****
Main article: Polony_sequencing
The Polony_sequencing method, developed in the laboratory of George_M._Church
at Harvard, was among the first high-throughput sequencing systems and was used
to sequence a full E._coli genome in 2005.[85] It combined an in vitro paired-
tag library with emulsion PCR, an automated microscope, and ligation-based
sequencing chemistry to sequence an E. coli genome at an accuracy of >99.9999%
and a cost approximately 1/9 that of Sanger sequencing.[85] The technology was
licensed to Agencourt Biosciences, subsequently spun out into Agencourt
Personal Genomics, and eventually incorporated into the Applied_Biosystems
SOLiD platform. Applied Biosystems was later acquired by Life_Technologies, now
part of Thermo_Fisher_Scientific.
**** 454 pyrosequencing[edit] ****
Main article: 454_Life_Sciences_Â§ Technology
A parallelized version of pyrosequencing was developed by 454_Life_Sciences,
which has since been acquired by Roche_Diagnostics. The method amplifies DNA
inside water droplets in an oil solution (emulsion PCR), with each droplet
containing a single DNA template attached to a single primer-coated bead that
then forms a clonal colony. The sequencing machine contains many picoliter-
volume wells each containing a single bead and sequencing enzymes.
Pyrosequencing uses luciferase to generate light for detection of the
individual nucleotides added to the nascent DNA, and the combined data are used
to generate sequence reads.[52] This technology provides intermediate read
length and price per base compared to Sanger sequencing on one end and Solexa
and SOLiD on the other.[64]
**** Illumina (Solexa) sequencing[edit] ****
Main article: Illumina_dye_sequencing
Solexa, now part of Illumina, was founded by Shankar_Balasubramanian and David
Klenerman in 1998, and developed a sequencing method based on reversible dye-
terminators technology, and engineered polymerases.[86] The reversible
terminated chemistry concept was invented by Bruno Canard and Simon Sarfati at
the Pasteur Institute in Paris.[87][88] It was developed internally at Solexa
by those named on the relevant patents. In 2004, Solexa acquired the company
Manteia_Predictive_Medicine in order to gain a massively parallel sequencing
technology invented in 1997 by Pascal Mayer and Laurent Farinelli.[42] It is
based on "DNA Clusters" or "DNA colonies", which involves the clonal
amplification of DNA on a surface. The cluster technology was co-acquired with
Lynx Therapeutics of California. Solexa Ltd. later merged with Lynx to form
Solexa Inc.
An Illumina HiSeq 2500 sequencer
In this method, DNA molecules and primers are first attached on a slide or flow
cell and amplified with polymerase so that local clonal DNA colonies, later
coined "DNA clusters", are formed. To determine the sequence, four types of
reversible terminator bases (RT-bases) are added and non-incorporated
nucleotides are washed away. A camera takes images of the fluorescently_labeled
nucleotides. Then the dye, along with the terminal 3' blocker, is chemically
removed from the DNA, allowing for the next cycle to begin. Unlike
pyrosequencing, the DNA chains are extended one nucleotide at a time and image
acquisition can be performed at a delayed moment, allowing for very large
arrays of DNA colonies to be captured by sequential images taken from a single
camera.
An Illumina MiSeq sequencer
Decoupling the enzymatic reaction and the image capture allows for optimal
throughput and theoretically unlimited sequencing capacity. With an optimal
configuration, the ultimately reachable instrument throughput is thus dictated
solely by the analog-to-digital conversion rate of the camera, multiplied by
the number of cameras and divided by the number of pixels per DNA colony
required for visualizing them optimally (approximately 10 pixels/colony). In
2012, with cameras operating at more than 10 MHz A/D conversion rates and
available optics, fluidics and enzymatics, throughput can be multiples of 1
million nucleotides/second, corresponding roughly to 1 human genome equivalent
at 1x coverage per hour per instrument, and 1 human genome re-sequenced (at
approx. 30x) per day per instrument (equipped with a single camera).[89]
**** Combinatorial probe anchor synthesis (cPAS)[edit] ****
This method is an upgraded modification to combinatorial probe anchor ligation
technology (cPAL) described by Complete_Genomics[90] which has since become
part of Chinese genomics company BGI in 2013.[91] The two companies have
refined the technology to allow for longer read lengths, reaction time
reductions and faster time to results. In addition, data are now generated as
contiguous full-length reads in the standard FASTQ file format and can be used
as-is in most short-read-based bioinformatics analysis pipelines.[92][citation
needed]
The two technologies that form the basis for this high-throughput sequencing
technology are DNA_nanoballs (DNB) and patterned arrays for nanoball attachment
to a solid surface.[90] DNA nanoballs are simply formed by denaturing double
stranded, adapter ligated libraries and ligating the forward strand only to a
splint oligonucleotide to form a ssDNA circle. Faithful copies of the circles
containing the DNA insert are produced utilizing Rolling Circle Amplification
that generates approximately 300â500 copies. The long strand of ssDNA folds
upon itself to produce a three-dimensional nanoball structure that is
approximately 220 nm in diameter. Making DNBs replaces the need to generate PCR
copies of the library on the flow cell and as such can remove large proportions
of duplicate reads, adapter-adapter ligations and PCR induced errors.[92]
[citation_needed]
A BGI MGISEQ-2000RS sequencer
The patterned array of positively charged spots is fabricated through
photolithography and etching techniques followed by chemical modification to
generate a sequencing flow cell. Each spot on the flow cell is approximately
250 nm in diameter, are separated by 700 nm (centre to centre) and allows easy
attachment of a single negatively charged DNB to the flow cell and thus
reducing under or over-clustering on the flow cell.[90][citation_needed]
Sequencing is then performed by addition of an oligonucleotide probe that
attaches in combination to specific sites within the DNB. The probe acts as an
anchor that then allows one of four single reversibly inactivated, labelled
nucleotides to bind after flowing across the flow cell. Unbound nucleotides are
washed away before laser excitation of the attached labels then emit
fluorescence and signal is captured by cameras that is converted to a digital
output for base calling. The attached base has its terminator and label
chemically cleaved at completion of the cycle. The cycle is repeated with
another flow of free, labelled nucleotides across the flow cell to allow the
next nucleotide to bind and have its signal captured. This process is completed
a number of times (usually 50 to 300 times) to determine the sequence of the
inserted piece of DNA at a rate of approximately 40 million nucleotides per
second as of 2018.[citation_needed]
**** SOLiD sequencing[edit] ****
Library preparation for the SOLiD platform
Main article: 2_base_encoding
Applied_Biosystems' (now a Life_Technologies brand) SOLiD technology employs
sequencing_by_ligation. Here, a pool of all possible oligonucleotides of a
fixed length are labeled according to the sequenced position. Oligonucleotides
are annealed and ligated; the preferential ligation by DNA_ligase for matching
sequences results in a signal informative of the nucleotide at that position.
Before sequencing, the DNA is amplified by emulsion PCR. The resulting beads,
each containing single copies of the same DNA molecule, are deposited on a
glass slide.[93] The result is sequences of quantities and lengths comparable
to Illumina sequencing.[64] This sequencing_by_ligation method has been
reported to have some issue sequencing palindromic sequences.[83]
**** Ion Torrent semiconductor sequencing[edit] ****
Main article: Ion_semiconductor_sequencing
Ion Torrent Systems Inc. (now owned by Life_Technologies) developed a system
based on using standard sequencing chemistry, but with a novel, semiconductor-
based detection system. This method of sequencing is based on the detection of
hydrogen_ions that are released during the polymerisation of DNA, as opposed to
the optical methods used in other sequencing systems. A microwell containing a
template DNA strand to be sequenced is flooded with a single type of
nucleotide. If the introduced nucleotide is complementary to the leading
template nucleotide it is incorporated into the growing complementary strand.
This causes the release of a hydrogen ion that triggers a hypersensitive ion
sensor, which indicates that a reaction has occurred. If homopolymer repeats
are present in the template sequence, multiple nucleotides will be incorporated
in a single cycle. This leads to a corresponding number of released hydrogens
and a proportionally higher electronic signal.[94]
Sequencing of the TAGGCT template with IonTorrent, PacBioRS and GridION
**** DNA nanoball sequencing[edit] ****
Main article: DNA_nanoball_sequencing
DNA_nanoball_sequencing is a type of high throughput sequencing technology used
to determine the entire genomic_sequence of an organism. The company Complete
Genomics uses this technology to sequence samples submitted by independent
researchers. The method uses rolling_circle_replication to amplify small
fragments of genomic DNA into DNA nanoballs. Unchained sequencing by ligation
is then used to determine the nucleotide sequence.[95] This method of DNA
sequencing allows large numbers of DNA nanoballs to be sequenced per run and at
low reagent costs compared to other high-throughput sequencing platforms.[96]
However, only short sequences of DNA are determined from each DNA nanoball
which makes mapping the short reads to a reference_genome difficult.[95] This
technology has been used for multiple genome sequencing projects and is
scheduled to be used for more.[97]
**** Heliscope single molecule sequencing[edit] ****
Main article: Helicos_single_molecule_fluorescent_sequencing
Heliscope sequencing is a method of single-molecule sequencing developed by
Helicos_Biosciences. It uses DNA fragments with added poly-A tail adapters
which are attached to the flow cell surface. The next steps involve extension-
based sequencing with cyclic washes of the flow cell with fluorescently labeled
nucleotides (one nucleotide type at a time, as with the Sanger method). The
reads are performed by the Heliscope sequencer.[98][99] The reads are short,
averaging 35 bp.[100] In 2009 a human genome was sequenced using the Heliscope,
however in 2012 the company went bankrupt.[101]
**** Single molecule real time (SMRT) sequencing[edit] ****
Main article: Single_molecule_real_time_sequencing
SMRT sequencing is based on the sequencing by synthesis approach. The DNA is
synthesized in zero-mode wave-guides (ZMWs) â small well-like containers with
the capturing tools located at the bottom of the well. The sequencing is
performed with use of unmodified polymerase (attached to the ZMW bottom) and
fluorescently labelled nucleotides flowing freely in the solution. The wells
are constructed in a way that only the fluorescence occurring by the bottom of
the well is detected. The fluorescent label is detached from the nucleotide
upon its incorporation into the DNA strand, leaving an unmodified DNA strand.
According to Pacific_Biosciences (PacBio), the SMRT technology developer, this
methodology allows detection of nucleotide modifications (such as cytosine
methylation). This happens through the observation of polymerase kinetics. This
approach allows reads of 20,000 nucleotides or more, with average read lengths
of 5 kilobases.[75][102] In 2015, Pacific Biosciences announced the launch of a
new sequencing instrument called the Sequel System, with 1 million ZMWs
compared to 150,000 ZMWs in the PacBio RS II instrument.[103][104] SMRT
sequencing is referred to as "third-generation" or "long-read" sequencing.
**** Nanopore DNA sequencing[edit] ****
Main article: Nanopore_sequencing
The DNA passing through the nanopore changes its ion current. This change is
dependent on the shape, size and length of the DNA sequence. Each type of the
nucleotide blocks the ion flow through the pore for a different period of time.
The method does not require modified nucleotides and is performed in real time.
Nanopore sequencing is referred to as "third-generation" or "long-read"
sequencing, along with SMRT sequencing.
Early industrial research into this method was based on a technique called
'Exonuclease sequencing', where the readout of electrical signals occurring at
nucleotides passing by alpha(Î±)-hemolysin pores covalently bound with
cyclodextrin.[105] However the subsequently commercial method, 'strand
sequencing' sequencing DNA bases in an intact strand.
Two main areas of nanopore sequencing in development are solid state nanopore
sequencing, and protein based nanopore sequencing. Protein nanopore sequencing
utilizes membrane protein complexes such as Î±-hemolysin, MspA (Mycobacterium
smegmatis Porin A) or CssG, which show great promise given their ability to
distinguish between individual and groups of nucleotides.[106] In contrast,
solid-state nanopore sequencing utilizes synthetic materials such as silicon
nitride and aluminum oxide and it is preferred for its superior mechanical
ability and thermal and chemical stability.[107] The fabrication method is
essential for this type of sequencing given that the nanopore array can contain
hundreds of pores with diameters smaller than eight nanometers.[106]
The concept originated from the idea that single stranded DNA or RNA molecules
can be electrophoretically driven in a strict linear sequence through a
biological pore that can be less than eight nanometers, and can be detected
given that the molecules release an ionic current while moving through the
pore. The pore contains a detection region capable of recognizing different
bases, with each base generating various time specific signals corresponding to
the sequence of bases as they cross the pore which are then evaluated.[107]
Precise control over the DNA transport through the pore is crucial for success.
Various enzymes such as exonucleases and polymerases have been used to moderate
this process by positioning them near the poreâs entrance.[108]
**** Microfluidic Systems[edit] ****
There are two main microfluidic systems that are used to sequence DNA; droplet
based_microfluidics and digital_microfluidics. Microfluidic devices solve many
of the current limitations of current sequencing arrays.
Abate et al. studied the use of droplet-based microfluidic devices for DNA
sequencing.[4] These devices have the ability to form and process picoliter
sized droplets at the rate of thousands per second. The devices were created
from polydimethylsiloxane_(PDMS)and used Forster resonance energy transfer,
FRET_assays to read the sequences of DNA encompassed in the droplets. Each
position on the array tested for a specific 15 base sequence. [4]
Fair et al. used digital microfluidic devices to study DNA pyrosequencing.[109]
Significant advantages include the portability of the device, reagent volume,
speed of analysis, mass manufacturing abilities, and high throughput. This
study provided a proof of concept showing that digital devices can be used for
pyrosequencing; the study included using synthesis, which involves the
extension of the enzymes and addition of labeled nucleotides.[109]
Boles et al. also studied pyrosequencing on digital microfluidic devices.[110]
They used an electro-wetting device to create, mix, and split droplets. The
sequencing uses a three-enzyme protocol and DNA templates anchored with
magnetic beads. The device was tested using two protocols and resulted in 100%
accuracy based on raw pyrogram levels. The advantages of these digital
microfluidic devices include size, cost, and achievable levels of functional
integration.[110]
DNA sequencing research, using microfluidics, also has the ability to be
applied to the sequencing_of_RNA, using similar droplet microfluidic
techniques, such as the method, inDrops.[111] This shows that many of these DNA
sequencing techniques will be able to be applied further and be used to
understand more about genomes and transcriptomes.
****** Methods in development[edit] ******
DNA sequencing methods currently under development include reading the sequence
as a DNA strand transits through nanopores (a method that is now commercial but
subsequent generations such as solid-state nanopores are still in development),
[112][113] and microscopy-based techniques, such as atomic_force_microscopy or
transmission_electron_microscopy that are used to identify the positions of
individual nucleotides within long DNA fragments (>5,000 bp) by nucleotide
labeling with heavier elements (e.g., halogens) for visual detection and
recording.[114][115] Third_generation_technologies aim to increase throughput
and decrease the time to result and cost by eliminating the need for excessive
reagents and harnessing the processivity of DNA polymerase.[116]
**** Tunnelling currents DNA sequencing[edit] ****
Another approach uses measurements of the electrical tunnelling currents across
single-strand DNA as it moves through a channel. Depending on its electronic
structure, each base affects the tunnelling current differently,[117] allowing
differentiation between different bases.[118]
The use of tunnelling currents has the potential to sequence orders of
magnitude faster than ionic current methods and the sequencing of several DNA
oligomers and micro-RNA has already been achieved.[119]
**** Sequencing by hybridization[edit] ****
Sequencing_by_hybridization is a non-enzymatic method that uses a DNA
microarray. A single pool of DNA whose sequence is to be determined is
fluorescently labeled and hybridized to an array containing known sequences.
Strong hybridization signals from a given spot on the array identifies its
sequence in the DNA being sequenced.[120]
This method of sequencing utilizes binding characteristics of a library of
short single stranded DNA molecules (oligonucleotides), also called DNA probes,
to reconstruct a target DNA sequence. Non-specific hybrids are removed by
washing and the target DNA is eluted.[121] Hybrids are re-arranged such that
the DNA sequence can be reconstructed. The benefit of this sequencing type is
its ability to capture a large number of targets with a homogenous coverage.
[122] A large number of chemicals and starting DNA is usually required.
However, with the advent of solution-based hybridization, much less equipment
and chemicals are necessary.[121]
**** Sequencing with mass spectrometry[edit] ****
Mass_spectrometry may be used to determine DNA sequences. Matrix-assisted laser
desorption ionization time-of-flight mass spectrometry, or MALDI-TOF_MS, has
specifically been investigated as an alternative method to gel electrophoresis
for visualizing DNA fragments. With this method, DNA fragments generated by
chain-termination sequencing reactions are compared by mass rather than by
size. The mass of each nucleotide is different from the others and this
difference is detectable by mass spectrometry. Single-nucleotide mutations in a
fragment can be more easily detected with MS than by gel electrophoresis alone.
MALDI-TOF MS can more easily detect differences between RNA fragments, so
researchers may indirectly sequence DNA with MS-based methods by converting it
to RNA first.[123]
The higher resolution of DNA fragments permitted by MS-based methods is of
special interest to researchers in forensic science, as they may wish to find
single-nucleotide_polymorphisms in human DNA samples to identify individuals.
These samples may be highly degraded so forensic researchers often prefer
mitochondrial_DNA for its higher stability and applications for lineage
studies. MS-based sequencing methods have been used to compare the sequences of
human mitochondrial DNA from samples in a Federal_Bureau_of_Investigation
database[124] and from bones found in mass graves of World War I soldiers.[125]
Early chain-termination and TOF MS methods demonstrated read lengths of up to
100 base pairs.[126] Researchers have been unable to exceed this average read
size; like chain-termination sequencing alone, MS-based DNA sequencing may not
be suitable for large de novo sequencing projects. Even so, a recent study did
use the short sequence reads and mass spectroscopy to compare single-nucleotide
polymorphisms in pathogenic Streptococcus strains.[127]
**** Microfluidic Sanger sequencing[edit] ****
Main article: Sanger_sequencing
In microfluidic Sanger sequencing the entire thermocycling amplification of DNA
fragments as well as their separation by electrophoresis is done on a single
glass wafer (approximately 10 cm in diameter) thus reducing the reagent usage
as well as cost.[128] In some instances researchers have shown that they can
increase the throughput of conventional sequencing through the use of
microchips.[129] Research will still need to be done in order to make this use
of technology effective.
**** Microscopy-based techniques[edit] ****
Main article: Transmission_electron_microscopy_DNA_sequencing
This approach directly visualizes the sequence of DNA molecules using electron
microscopy. The first identification of DNA base pairs within intact DNA
molecules by enzymatically incorporating modified bases, which contain atoms of
increased atomic number, direct visualization and identification of
individually labeled bases within a synthetic 3,272 base-pair DNA molecule and
a 7,249 base-pair viral genome has been demonstrated.[130]
**** RNAP sequencing[edit] ****
This method is based on use of RNA_polymerase (RNAP), which is attached to a
polystyrene bead. One end of DNA to be sequenced is attached to another bead,
with both beads being placed in optical traps. RNAP motion during transcription
brings the beads in closer and their relative distance changes, which can then
be recorded at a single nucleotide resolution. The sequence is deduced based on
the four readouts with lowered concentrations of each of the four nucleotide
types, similarly to the Sanger method.[131] A comparison is made between
regions and sequence information is deduced by comparing the known sequence
regions to the unknown sequence regions.[132]
**** In vitro virus high-throughput sequencing[edit] ****
A method has been developed to analyze full sets of protein_interactions using
a combination of 454 pyrosequencing and an in vitro virus mRNA_display method.
Specifically, this method covalently links proteins of interest to the mRNAs
encoding them, then detects the mRNA pieces using reverse transcription PCRs.
The mRNA may then be amplified and sequenced. The combined method was titled
IVV-HiTSeq and can be performed under cell-free conditions, though its results
may not be representative of in vivo conditions.[133]
***** Sample preparation[edit] *****
The success of any DNA sequencing protocol relies upon the DNA or RNA sample
extraction and preparation from the biological material of interest.
    * A successful DNA extraction will yield a DNA sample with long, non-
      degraded strands.
    * A successful RNA extraction will yield a RNA sample that should be
      converted to complementary DNA (cDNA) using reverse transcriptaseâa DNA
      polymerase that synthesizes a complementary DNA based on existing strands
      of RNA in a PCR-like manner.[134] Complementary DNA can then be processed
      the same way as genomic DNA.
According to the sequencing technology to be used, the samples resulting from
either the DNA or the RNA extraction require further preparation. For Sanger
sequencing, either cloning procedures or PCR are required prior to sequencing.
In the case of next-generation sequencing methods, library preparation is
required before processing.[135] Assessing the quality and quantity of nucleic
acids both after extraction and after library preparation identifies degraded,
fragmented, and low-purity samples and yields high-quality sequencing data.
[136]
***** Development initiatives[edit] *****
Total cost of sequencing a human genome over time as calculated by the NHGRI.
In October 2006, the X_Prize_Foundation established an initiative to promote
the development of full_genome_sequencing technologies, called the Archon_X
Prize, intending to award $10 million to "the first Team that can build a
device and use it to sequence 100 human genomes within 10 days or less, with an
accuracy of no more than one error in every 100,000 bases sequenced, with
sequences accurately covering at least 98% of the genome, and at a recurring
cost of no more than $10,000 (US) per genome."[137]
Each year the National_Human_Genome_Research_Institute, or NHGRI, promotes
grants for new research and developments in genomics. 2010 grants and 2011
candidates include continuing work in microfluidic, polony and base-heavy
sequencing methodologies.[138]
***** Computational challenges[edit] *****
The sequencing technologies described here produce raw data that needs to be
assembled into longer sequences such as complete genomes (sequence_assembly).
There are many computational challenges to achieve this, such as the evaluation
of the raw sequence data which is done by programs and algorithms such as Phred
and Phrap. Other challenges have to deal with repetitive sequences that often
prevent complete genome assemblies because they occur in many places of the
genome. As a consequence, many sequences may not be assigned to particular
chromosomes. The production of raw sequence data is only the beginning of its
detailed bioinformatical analysis.[139] Yet new methods for sequencing and
correcting sequencing errors were developed.[140]
**** Read trimming[edit] ****
Sometimes, the raw reads produced by the sequencer are correct and precise only
in a fraction of their length. Using the entire read may introduce artifacts in
the downstream analyses like genome assembly, snp calling, or gene expression
estimation. Two classes of trimming programs have been introduced, based on the
window-based or the running-sum classes of algorithms.[141] This is a partial
list of the trimming algorithms currently available, specifying the algorithm
class they belong to:
                  Read Trimming Algorithms
Name of algorithm     Type of algorithm Link
Cutadapt[142]         Running sum       Cutadapt
ConDeTri[143]         Window based      ConDeTri
ERNE-FILTER[144]      Running sum       ERNE-FILTER
FASTX quality trimmer Window based      FASTX_quality_trimmer
PRINSEQ[145]          Window based      PRINSEQ
Trimmomatic[146]      Window based      Trimmomatic
SolexaQA[147]         Window based      SolexaQA
SolexaQA-BWA          Running sum       SolexaQA-BWA
Sickle                Window based      Sickle
***** Ethical issues[edit] *****
[[icon]] This section needs expansion. You can help by adding_to_it. (May 2015)
Further information: Bioethics
Human genetics have been included within the field of bioethics since the early
1970s[148] and the growth in the use of DNA sequencing (particularly high-
throughput sequencing) has introduced a number of ethical issues. One key issue
is the ownership of an individual's DNA and the data produced when that DNA is
sequenced.[149] Regarding the DNA molecule itself, the leading legal case on
this topic, Moore_v._Regents_of_the_University_of_California (1990) ruled that
individuals have no property rights to discarded cells or any profits made
using these cells (for instance, as a patented cell_line). However, individuals
have a right to informed consent regarding removal and use of cells. Regarding
the data produced through DNA sequencing, Moore gives the individual no rights
to the information derived from their DNA.[149]
As DNA sequencing becomes more widespread, the storage, security and sharing of
genomic data has also become more important.[149][150] For instance, one
concern is that insurers may use an individual's genomic data to modify their
quote, depending on the perceived future health of the individual based on
their DNA.[150][151] In May 2008, the Genetic_Information_Nondiscrimination_Act
(GINA) was signed in the United States, prohibiting discrimination on the basis
of genetic information with respect to health insurance and employment.[152]
[153] In 2012, the US Presidential_Commission_for_the_Study_of_Bioethical
Issues reported that existing privacy legislation for DNA sequencing data such
as GINA and the Health_Insurance_Portability_and_Accountability_Act were
insufficient, noting that whole-genome sequencing data was particularly
sensitive, as it could be used to identify not only the individual from which
the data was created, but also their relatives.[154][155]
Ethical issues have also been raised by the increasing use of genetic variation
screening, both in newborns, and in adults by companies such as 23andMe.[156]
[157] It has been asserted that screening for genetic variations can be
harmful, increasing anxiety in individuals who have been found to have an
increased risk of disease.[158] For example, in one case noted in Time, doctors
screening an ill baby for genetic variants chose not to inform the parents of
an unrelated variant linked to dementia due to the harm it would cause to the
parents.[159] However, a 2011 study in The_New_England_Journal_of_Medicine has
shown that individuals undergoing disease risk profiling did not show increased
levels of anxiety.[158]
***** See also[edit] *****
    * Bioinformatics                  * Multiplex_ligation-dependent_probe
    * Cancer_genome_sequencing          amplification
    * DNA_computing                   * Personalized_medicine
    * DNA_field-effect_transistor     * Protein_sequencing
    * DNA_sequencing_theory           * Sequence_mining
    * DNA_sequencer                   * Sequence_profiling_tool
    * Genome_project                  * Sequencing_by_hybridization
    * Jumping_library                 * Sequencing_by_ligation
    * Nucleic_acid_sequence           * Transmission_electron_microscopy_DNA
                                        sequencing
***** Notes[edit] *****
   1. ^ "Next-generation" remains in broad use as of 2019. For
      instance,Straiton J, Free T, Sawyer A, Martin J (February 2019). "From
      Sanger Sequencing to Genome Databases and Beyond". BioTechniques. 66 (2):
      60â63. doi:10.2144/btn-2019-0011. PMID 30744413. Next-generation
      sequencing (NGS) technologies have revolutionized genomic research.
      (opening sentence of the article)
   2. .mw-parser-output cite.citation{font-style:inherit}.mw-parser-output
      .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-
      free a{background:url("//upload.wikimedia.org/wikipedia/commons/thumb/6/
      65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:
      right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-
      parser-output .citation .cs1-lock-registration a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/
      9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output .citation .cs1-lock-subscription a{background:
      url("//upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-
      2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right
      .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-
      registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-
      parser-output .cs1-registration span{border-bottom:1px dotted;cursor:
      help}.mw-parser-output .cs1-ws-icon a{background:url("//
      upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/
      12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em
      center}.mw-parser-output code.cs1-code{color:inherit;background:
      inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-
      error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error
      {font-size:100%}.mw-parser-output .cs1-maint{display:none;color:
      #33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-
      output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-
      parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-
      left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-
      wl-right{padding-right:0.2em}
***** References[edit] *****
   1. ^"Introducing_'dark_DNA'_â_the_phenomenon_that_could_change_how_we
      think_about_evolution".
   2. ^Behjati S, Tarpey PS (December 2013). "What_is_next_generation
      sequencing?". Archives of Disease in Childhood. Education and Practice
      Edition. 98 (6): 236â8. doi:10.1136/archdischild-2013-304340.
      PMC 3841808. PMID 23986538.
   3. ^Chmielecki J, Meyerson M (14 January 2014). "DNA sequencing of cancer:
      what have we learned?". Annual Review of Medicine. 65 (1): 63â79. doi:
      10.1146/annurev-med-060712-200152. PMID 24274178.
   4. ^ a b c dAbate AR, Hung T, Sperling RA, Mary P, Rotem A, Agresti JJ, et
      al. (December 2013). "DNA_sequence_analysis_with_droplet-based
      microfluidics". Lab on a Chip. 13 (24): 4864â9. doi:10.1039/c3lc50905b.
      PMC 4090915. PMID 24185402.
   5. ^Pekin D, Skhiri Y, Baret JC, Le Corre D, Mazutis L, Salem CB, et al.
      (July 2011). "Quantitative and sensitive detection of rare mutations
      using droplet-based microfluidics". Lab on a Chip. 11 (13): 2156â66.
      doi:10.1039/c1lc20128j. PMID 21594292.
   6. ^Olsvik O, Wahlberg J, Petterson B, UhlÃ©n M, Popovic T, Wachsmuth IK,
      Fields PI (January 1993). "Use_of_automated_sequencing_of_polymerase
      chain_reaction-generated_amplicons_to_identify_three_types_of_cholera
      toxin_subunit_B_in_Vibrio_cholerae_O1_strains". J._Clin._Microbiol. 31
      (1): 22â25. PMC 262614. PMID 7678018.
   7. [open_access]
   8. ^Pettersson E, Lundeberg J, Ahmadian A (February 2009). "Generations of
      sequencing technologies". Genomics. 93 (2): 105â11. doi:10.1016/
      j.ygeno.2008.10.003. PMID 18992322.
   9. ^Curtis C, Hereward J (29 August 2017). "From_the_crime_scene_to_the
      courtroom:_the_journey_of_a_DNA_sample". The Conversation.
  10. ^MorÃ©ra S, LariviÃ¨re L, Kurzeck J, Aschke-Sonnenborn U, Freemont PS,
      Janin J, RÃ¼ger W (August 2001). "High resolution crystal structures of
      T4 phage beta-glucosyltransferase: induced fit and effect of substrate
      and metal binding". Journal of Molecular Biology. 311 (3): 569â77. doi:
      10.1006/jmbi.2001.4905. PMID 11493010.
  11. ^Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA, Gehrke
      C (April 1982). "Amount_and_distribution_of_5-methylcytosine_in_human_DNA
      from_different_types_of_tissues_of_cells". Nucleic Acids Research. 10
      (8): 2709â21. doi:10.1093/nar/10.8.2709. PMC 320645. PMID 7079182.
  12. ^Ehrlich M, Wang RY (June 1981). "5-Methylcytosine in eukaryotic DNA".
      Science. 212 (4501): 1350â7. Bibcode:1981Sci...212.1350E. doi:10.1126/
      science.6262918. PMID 6262918.
  13. ^Song CX, Clark TA, Lu XY, Kislyuk A, Dai Q, Turner SW, et al. (November
      2011). "Sensitive_and_specific_single-molecule_sequencing_of_5-
      hydroxymethylcytosine". Nature Methods. 9 (1): 75â7. doi:10.1038/
      nmeth.1779. PMC 3646335. PMID 22101853.
  14. ^Watson JD, Crick FH (1953). "The structure of DNA". Cold Spring Harb.
      Symp. Quant. Biol. 18: 123â31. doi:10.1101/SQB.1953.018.01.020.
      PMID 13168976.
  15. ^ Marks,_L,_The_path_to_DNA_sequencing:_The_life_and_work_of_Frederick
      Sanger.
  16. ^Min Jou W, Haegeman G, Ysebaert M, Fiers W (May 1972). "Nucleotide
      sequence of the gene coding for the bacteriophage MS2 coat protein".
      Nature. 237 (5350): 82â8. Bibcode:1972Natur.237...82J. doi:10.1038/
      237082a0. PMID 4555447.
  17. ^Fiers W, Contreras R, Duerinck F, Haegeman G, Iserentant D, Merregaert
      J, Min Jou W, Molemans F, Raeymaekers A, Van den Berghe A, Volckaert G,
      Ysebaert M (April 1976). "Complete nucleotide sequence of bacteriophage
      MS2 RNA: primary and secondary structure of the replicase gene". Nature.
      260 (5551): 500â7. Bibcode:1976Natur.260..500F. doi:10.1038/260500a0.
      PMID 1264203.
  18. ^Ozsolak F, Milos PM (February 2011). "RNA_sequencing:_advances,
      challenges_and_opportunities". Nature Reviews. Genetics. 12 (2): 87â98.
      doi:10.1038/nrg2934. PMC 3031867. PMID 21191423.
  19. ^"Ray_Wu_Faculty_Profile". Cornell University. Archived from the_original
      on 4 March 2009.
  20. ^Padmanabhan R, Jay E, Wu R (June 1974). "Chemical_synthesis_of_a_primer
      and_its_use_in_the_sequence_analysis_of_the_lysozyme_gene_of
      bacteriophage_T4". Proceedings of the National Academy of Sciences of the
      United States of America. 71 (6): 2510â4. Bibcode:1974PNAS...71.2510P.
      doi:10.1073/pnas.71.6.2510. PMC 388489. PMID 4526223.
  21. ^Onaga LA (June 2014). "Ray Wu as Fifth Business: Demonstrating
      Collective Memory in the History of DNA Sequencing". Studies in the
      History and Philosophy of Science. Part C. 46: 1â14. doi:10.1016/
      j.shpsc.2013.12.006. PMID 24565976.
  22. ^Wu R (1972). "Nucleotide sequence analysis of DNA". Nature New Biology.
      236 (68): 198â200. doi:10.1038/newbio236198a0. PMID 4553110.
  23. ^Padmanabhan R, Wu R (1972). "Nucleotide sequence analysis of DNA. IX.
      Use of oligonucleotides of defined sequence as primers in DNA sequence
      analysis". Biochem. Biophys. Res. Commun. 48 (5): 1295â302. doi:
      10.1016/0006-291X(72)90852-2. PMID 4560009.
  24. ^Wu R, Tu CD, Padmanabhan R (1973). "Nucleotide sequence analysis of DNA.
      XII. The chemical synthesis and sequence analysis of a
      dodecadeoxynucleotide which binds to the endolysin gene of bacteriophage
      lambda". Biochem. Biophys. Res. Commun. 55 (4): 1092â99. doi:10.1016/
      S0006-291X(73)80007-5. PMID 4358929.
  25. ^Jay E, Bambara R, Padmanabhan R, Wu R (March 1974). "DNA_sequence
      analysis:_a_general,_simple_and_rapid_method_for_sequencing_large
      oligodeoxyribonucleotide_fragments_by_mapping". Nucleic Acids Research. 1
      (3): 331â53. doi:10.1093/nar/1.3.331. PMC 344020. PMID 10793670.
  26. ^ a bSanger F, Nicklen S, Coulson AR (December 1977). "DNA_sequencing
      with_chain-terminating_inhibitors". Proc. Natl. Acad. Sci. USA. 74 (12):
      5463â77. Bibcode:1977PNAS...74.5463S. doi:10.1073/pnas.74.12.5463.
      PMC 431765. PMID 271968.
  27. ^ a b cMaxam AM, Gilbert W (February 1977). "A_new_method_for_sequencing
      DNA". Proc. Natl. Acad. Sci. USA. 74 (2): 560â64. Bibcode:
      1977PNAS...74..560M. doi:10.1073/pnas.74.2.560. PMC 392330. PMID 265521.
  28. ^ Gilbert, W. DNA_sequencing_and_gene_structure. Nobel lecture, 8
      December 1980.
  29. ^Gilbert W, Maxam A (December 1973). "The_Nucleotide_Sequence_of_the_lac
      Operator". Proc. Natl. Acad. Sci. U.S.A. 70 (12): 3581â84. Bibcode:
      1973PNAS...70.3581G. doi:10.1073/pnas.70.12.3581. PMC 427284.
      PMID 4587255.
  30. ^Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA, Hutchison
      CA, Slocombe PM, Smith M (February 1977). "Nucleotide sequence of
      bacteriophage phi X174 DNA". Nature. 265 (5596): 687â95. Bibcode:
      1977Natur.265..687S. doi:10.1038/265687a0. PMID 870828.
  31. ^ "The_Next_Frontier:_Human_Viruses" , whatisbiotechnology.org, Retrieved
      3 May 2017
  32. ^Beck S, Pohl FM (1984). "DNA_sequencing_with_direct_blotting
      electrophoresis". EMBO J. 3 (12): 2905â09. doi:10.1002/j.1460-
      2075.1984.tb02230.x. PMC 557787. PMID 6396083.
  33. ^ United States Patent 4,631,122 (1986)
  34. ^Feldmann H, et al. (1994). "Complete_DNA_sequence_of_yeast_chromosome
      II". EMBO J. 13 (24): 5795â809. doi:10.1002/j.1460-2075.1994.tb06923.x.
      PMC 395553. PMID 7813418.
  35. ^Smith LM, Sanders JZ, Kaiser RJ, Hughes P, Dodd C, Connell CR, Heiner C,
      Kent SB, Hood LE (12 June 1986). "Fluorescence Detection in Automated DNA
      Sequence Analysis". Nature. 321 (6071): 674â79. Bibcode:
      1986Natur.321..674S. doi:10.1038/321674a0. PMID 3713851.
  36. ^Prober JM, Trainor GL, Dam RJ, Hobbs FW, Robertson CW, Zagursky RJ,
      Cocuzza AJ, Jensen MA, Baumeister K (16 October 1987). "A system for
      rapid DNA sequencing with fluorescent chain-terminating
      dideoxynucleotides". Science. 238 (4825): 336â41. Bibcode:
      1987Sci...238..336P. doi:10.1126/science.2443975. PMID 2443975.
  37. ^Adams MD, Kelley JM, Gocayne JD, Dubnick M, Polymeropoulos MH, Xiao H,
      Merril CR, Wu A, Olde B, Moreno RF (June 1991). "Complementary DNA
      sequencing: expressed sequence tags and human genome project". Science.
      252 (5013): 1651â56. Bibcode:1991Sci...252.1651A. doi:10.1126/
      science.2047873. PMID 2047873.
  38. ^Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, Kerlavage
      AR, Bult CJ, Tomb JF, Dougherty BA, Merrick JM (July 1995). "Whole-genome
      random sequencing and assembly of Haemophilus influenzae Rd". Science.
      269 (5223): 496â512. Bibcode:1995Sci...269..496F. doi:10.1126/
      science.7542800. PMID 7542800.
  39. ^Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al. (February
      2001). "Initial_sequencing_and_analysis_of_the_human_genome" (PDF).
      Nature. 409 (6822): 860â921. Bibcode:2001Natur.409..860L. doi:10.1038/
      35057062. PMID 11237011.
  40. ^Venter JC, Adams MD, et al. (February 2001). "The sequence of the human
      genome". Science. 291 (5507): 1304â51. Bibcode:2001Sci...291.1304V.
      doi:10.1126/science.1058040. PMID 11181995.
  41. ^"Espacenet_â_Bibliographic_data". worldwide.espacenet.com.
  42. ^Ronaghi M, Karamohamed S, Pettersson B, UhlÃ©n M, NyrÃ©n P (1996).
      "Real-time DNA sequencing using detection of pyrophosphate release".
      Analytical Biochemistry. 242 (1): 84â89. doi:10.1006/abio.1996.0432.
      PMID 8923969.
  43. ^ a b cKawashima, Eric H.; Laurent Farinelli; Pascal Mayer (12 May 2005).
      "Patent:_Method_of_nucleic_acid_amplification". Archived from the
      original on 22 February 2013. Retrieved 22 December 2012.
  44. ^Ewing B, Green P (March 1998). "Base-calling of automated sequencer
      traces using phred. II. Error probabilities". Genome Res. 8 (3):
      186â94. doi:10.1101/gr.8.3.186. PMID 9521922.
  45. ^"Quality_Scores_for_Next-Generation_Sequencing" (PDF). Illumina. 31
      October 2011. Retrieved 8 May 2018.
  46. ^ a bBrenner S, Johnson M, Bridgham J, Golda G, Lloyd DH, Johnson D, Luo
      S, McCurdy S, Foy M, Ewan M, Roth R, George D, Eletr S, Albrecht G,
      Vermaas E, Williams SR, Moon K, Burcham T, Pallas M, DuBridge RB,
      Kirchner J, Fearon K, Mao J, Corcoran K (2000). "Gene expression analysis
      by massively parallel signature sequencing (MPSS) on microbead arrays".
      Nature Biotechnology. 18 (6): 630â34. doi:10.1038/76469. PMID 10835600.
  47. ^Sanger F, Coulson AR (May 1975). "A rapid method for determining
      sequences in DNA by primed synthesis with DNA polymerase". J. Mol. Biol.
      94 (3): 441â48. doi:10.1016/0022-2836(75)90213-2. PMID 1100841.
  48. ^Wetterstrand, Kris. "DNA_Sequencing_Costs:_Data_from_the_NHGRI_Genome
      Sequencing_Program_(GSP)". National_Human_Genome_Research_Institute.
      Retrieved 30 May 2013.
  49. ^Quail MA, Gu Y, Swerdlow H, Mayho M (2012). "Evaluation and optimisation
      of preparative semi-automated electrophoresis systems for Illumina
      library preparation". Electrophoresis. 33 (23): 3521â28. doi:10.1002/
      elps.201200128. PMID 23147856.
  50. ^Duhaime MB, Deng L, Poulos BT, Sullivan MB (2012). "Towards_quantitative
      metagenomics_of_wild_viruses_and_other_ultra-low_concentration_DNA
      samples:_a_rigorous_assessment_and_optimization_of_the_linker
      amplification_method". Environ. Microbiol. 14 (9): 2526â37. doi:
      10.1111/j.1462-2920.2012.02791.x. PMC 3466414. PMID 22713159.
  51. ^Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE (2012). "Double
      digest_RADseq:_an_inexpensive_method_for_de_novo_SNP_discovery_and
      genotyping_in_model_and_non-model_species". PLoS ONE. 7 (5): e37135.
      Bibcode:2012PLoSO...737135P. doi:10.1371/journal.pone.0037135.
      PMC 3365034. PMID 22675423.
  52. ^Williams R, Peisajovich SG, Miller OJ, Magdassi S, Tawfik DS, Griffiths
      AD (2006). "Amplification of complex gene libraries by emulsion PCR".
      Nature Methods. 3 (7): 545â50. doi:10.1038/nmeth896. PMID 16791213.
  53. ^ a bMargulies M, Egholm M, et al. (September 2005). "Genome_Sequencing
      in_Open_Microfabricated_High_Density_Picoliter_Reactors". Nature. 437
      (7057): 376â80. Bibcode:2005Natur.437..376M. doi:10.1038/nature03959.
      PMC 1464427. PMID 16056220.
  54. ^Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, Rosenbaum AM,
      Wang MD, Zhang K, Mitra RD, Church GM (2005). "Accurate Multiplex Polony
      Sequencing of an Evolved Bacterial Genome". Science. 309 (5741):
      1728â32. Bibcode:2005Sci...309.1728S. doi:10.1126/science.1117389.
      PMID 16081699.
  55. ^"Applied_Biosystems_â_File_Not_Found_(404_Error)". 16 May 2008.
      Archived from the_original on 16 May 2008.
  56. ^Goodwin S, McPherson JD, McCombie WR (May 2016). "Coming of age: ten
      years of next-generation sequencing technologies". Nature Reviews.
      Genetics. 17 (6): 333â51. doi:10.1038/nrg.2016.49. PMID 27184599.
  57. ^Staden R (11 June 1979). "A_strategy_of_DNA_sequencing_employing
      computer_programs". Nucleic Acids Research. 6 (7): 2601â10. doi:
      10.1093/nar/6.7.2601. PMC 327874. PMID 461197.
  58. ^ P. Mayer,L. Farinelli, G. Matton, C. Adessi, G. Turcatti, J. J. Mermod,
      E. Kawashima.DNA_colony_massively_parallel_sequencing_ams98_presentation
  59. ^ U.S._Patent_5,641,658
  60. ^Braslavsky I, Hebert B, Kartalov E, Quake SR (April 2003). "Sequence
      information_can_be_obtained_from_single_DNA_molecules". Proc. Natl. Acad.
      Sci. USA. 100 (7): 3960â64. Bibcode:2003PNAS..100.3960B. doi:10.1073/
      pnas.0230489100. PMC 153030. PMID 12651960.
  61. ^de MagalhÃ£es JP, Finch CE, Janssens G (2010). "Next-generation
      sequencing_in_aging_research:_emerging_applications,_problems,_pitfalls
      and_possible_solutions". Ageing_Research_Reviews. 9 (3): 315â23. doi:
      10.1016/j.arr.2009.10.006. PMC 2878865. PMID 19900591.
  62. ^Grada A (August 2013). "Next-generation sequencing: methodology and
      application". J Invest Dermatol. 133 (8): e11. doi:10.1038/jid.2013.248.
      PMID 23856935.
  63. ^Hall N (May 2007). "Advanced sequencing technologies and their wider
      impact in microbiology". J._Exp._Biol. 210 (Pt 9): 1518â25. doi:
      10.1242/jeb.001370. PMID 17449817.
  64. [open_access]
  65. ^Church_GM (January 2006). "Genomes for all". Sci._Am. 294 (1): 46â54.
      Bibcode:2006SciAm.294a..46C. doi:10.1038/scientificamerican0106-46.
      PMID 16468433.
  66. (subscription required)
  67. ^ a b cSchuster SC (January 2008). "Next-generation sequencing transforms
      today's biology". Nat. Methods. 5 (1): 16â18. doi:10.1038/nmeth1156.
      PMID 18165802.
  68. ^Kalb, Gilbert; Moxley, Robert (1992). Massively Parallel, Optical, and
      Neural Computing in the United States. IOS_Press. ISBN 978-90-5199-097-3.
  69. [page needed]
  70. ^ten Bosch JR, Grody WW (2008). "Keeping_Up_with_the_Next_Generation".
      The Journal of Molecular Diagnostics. 10 (6): 484â92. doi:10.2353/
      jmoldx.2008.080027. PMC 2570630. PMID 18832462.
  71. [open_access]
  72. ^Tucker T, Marra M, Friedman JM (2009). "Massively_Parallel_Sequencing:
      The_Next_Big_Thing_in_Genetic_Medicine". The American Journal of Human
      Genetics. 85 (2): 142â54. doi:10.1016/j.ajhg.2009.06.022. PMC 2725244.
      PMID 19679224.
  73. [open_access]
  74. ^ a bStraiton J, Free T, Sawyer A, Martin J (February 2019). "From Sanger
      sequencing to genome databases and beyond". BioTechniques. Future
      Science. 66 (2): 60â63. doi:10.2144/btn-2019-0011. PMID 30744413.
  75. ^Quail MA, Smith M, Coupland P, Otto TD, Harris SR, Connor TR, Bertoni A,
      Swerdlow HP, Gu Y (1 January 2012). "A_tale_of_three_next_generation
      sequencing_platforms:_comparison_of_Ion_Torrent,_Pacific_Biosciences_and
      illumina_MiSeq_sequencers". BMC_Genomics. 13 (1): 341. doi:10.1186/1471-
      2164-13-341. PMC 3431227. PMID 22827831.
  76. [open_access]
  77. ^Liu L, Li Y, Li S, Hu N, He Y, Pong R, Lin D, Lu L, Law M (1 January
      2012). "Comparison_of_Next-Generation_Sequencing_Systems". Journal of
      Biomedicine and Biotechnology. 2012: 251364. doi:10.1155/2012/251364.
      PMC 3398667. PMID 22829749.
  78. [open_access]
  79. ^ a b c"New_Software,_Polymerase_for_Sequel_System_Boost_Throughput_and
      Affordability_â_PacBio". 7 March 2018.
  80. ^"After_a_Year_of_Testing,_Two_Early_PacBio_Customers_Expect_More_Routine
      Use_of_RS_Sequencer_in_2012". GenomeWeb. 10 January 2012.
  81. (registration_required)
  82. ^Inc., Pacific Biosciences (2013). "Pacific_Biosciences_Introduces_New
      Chemistry_With_Longer_Read_Lengths_to_Detect_Novel_Features_in_DNA
      Sequence_and_Advance_Genome_Studies_of_Large_Organisms".
  83. ^Chin CS, Alexander DH, Marks P, Klammer AA, Drake J, Heiner C, Clum A,
      Copeland A, Huddleston J, Eichler EE, Turner SW, Korlach J (2013).
      "Nonhybrid, finished microbial genome assemblies from long-read SMRT
      sequencing data". Nat. Methods. 10 (6): 563â69. doi:10.1038/nmeth.2474.
      PMID 23644548.
  84. ^ a b"De_novo_bacterial_genome_assembly:_a_solved_problem?". 5 July 2013.
  85. ^Rasko DA, Webster DR, Sahl JW, Bashir A, Boisen N, Scheutz F, Paxinos
      EE, Sebra R, Chin CS, Iliopoulos D, Klammer A, Peluso P, Lee L, Kislyuk
      AO, Bullard J, Kasarskis A, Wang S, Eid J, Rank D, Redman JC, Steyert SR,
      Frimodt-MÃ¸ller J, Struve C, Petersen AM, Krogfelt KA, Nataro JP, Schadt
      EE, Waldor MK (25 August 2011). "Origins_of_the_Strain_Causing_an
      Outbreak_of_HemolyticâUremic_Syndrome_in_Germany". N_Engl_J_Med. 365
      (8): 709â17. doi:10.1056/NEJMoa1106920. PMC 3168948. PMID 21793740.
  86. [open_access]
  87. ^Tran B, Brown AM, Bedard PL, Winquist E, Goss GD, Hotte SJ, Welch SA,
      Hirte HW, Zhang T, Stein_LD, Ferretti V, Watt S, Jiao W, Ng K, Ghai S,
      Shaw P, Petrocelli T, Hudson_TJ, Neel BG, Onetto N, Siu LL, McPherson JD,
      Kamel-Reid S, Dancey JE (1 January 2012). "Feasibility of real time next
      generation sequencing of cancer genes linked to drug response: Results
      from a clinical trial". Int._J._Cancer. 132 (7): 1547â55. doi:10.1002/
      ijc.27817. PMID 22948899.
  88. (subscription required)
  89. ^Murray IA, Clark TA, Morgan RD, Boitano M, Anton BP, Luong K, Fomenkov
      A, Turner SW, Korlach J, Roberts RJ (2 October 2012). "The_methylomes_of
      six_bacteria". Nucleic Acids Research. 40 (22): 11450â62. doi:10.1093/
      nar/gks891. PMC 3526280. PMID 23034806.
  90. ^"Ion_520_&_Ion_530_ExT_Kit-Chef_â_Thermo_Fisher_Scientific".
      www.thermofisher.com.
  91. ^"Archived_copy". Archived from the_original on 30 March 2018. Retrieved
      29 March 2018.CS1 maint: Archived copy as title (link)
  92. ^van Vliet AH (1 January 2010). "Next generation sequencing of microbial
      transcriptomes: challenges and opportunities". FEMS_Microbiology_Letters.
      302 (1): 1â7. doi:10.1111/j.1574-6968.2009.01767.x. PMID 19735299.
  93. [open_access]
  94. ^"BGI_and_MGISEQ". en.mgitech.cn. Retrieved 5 July 2018.
  95. ^ a bHuang YF, Chen SC, Chiang YS, Chen TH, Chiu KP (2012). "Palindromic
      sequence_impedes_sequencing-by-ligation_mechanism". BMC_Systems_Biology.
      6 Suppl 2: S10. doi:10.1186/1752-0509-6-S2-S10. PMC 3521181.
      PMID 23281822.
  96. ^Loose, Matthew; Rakyan, Vardhman; Holmes, Nadine; Payne, Alexander (3
      May 2018). "Whale watching with BulkVis: A graphical viewer for Oxford
      Nanopore bulk fast5 files". bioRxiv 312256.
  97. ^ a bShendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, Rosenbaum
      AM, Wang MD, Zhang K, Mitra RD, Church GM (9 September 2005). "Accurate
      multiplex polony sequencing of an evolved bacterial genome". Science. 309
      (5741): 1728â32. Bibcode:2005Sci...309.1728S. doi:10.1126/
      science.1117389. PMID 16081699.
  98. ^Bentley DR, Balasubramanian S, et al. (2008). "Accurate_whole_human
      genome_sequencing_using_reversible_terminator_chemistry". Nature. 456
      (7218): 53â59. Bibcode:2008Natur.456...53B. doi:10.1038/nature07517.
      PMC 2581791. PMID 18987734.
  99. ^Canard B, Sarfati S (13 October 1994), Novel_derivatives_usable_for_the
      sequencing_of_nucleic_acids, retrieved 9 March 2016
 100. ^Canard B, Sarfati RS (October 1994). "DNA polymerase fluorescent
      substrates with reversible 3'-tags". Gene. 148 (1): 1â6. doi:10.1016/
      0378-1119(94)90226-7. PMID 7523248.
 101. ^Mardis ER (2008). "Next-generation DNA sequencing methods". Annu Rev
      Genom Hum Genet. 9: 387â402. doi:10.1146/annurev.genom.9.081307.164359.
      PMID 18576944.
 102. ^ a b cDrmanac R, Sparks AB, Callow MJ, Halpern AL, Burns NL, Kermani BG,
      et al. (January 2010). "Human genome sequencing using unchained base
      reads on self-assembling DNA nanoarrays". Science. 327 (5961): 78â81.
      Bibcode:2010Sci...327...78D. doi:10.1126/science.1181498. PMID 19892942.
 103. ^brandonvd. "About_Us_-_Complete_Genomics". Complete Genomics. Retrieved
      2 July 2018.
 104. ^ a bHuang J, Liang X, Xuan Y, Geng C, Li Y, Lu H, et al. (May 2017). "A
      reference_human_genome_dataset_of_the_BGISEQ-500_sequencer". GigaScience.
      6 (5): 1â9. doi:10.1093/gigascience/gix024. PMC 5467036. PMID 28379488.
 105. ^Valouev A, Ichikawa J, Tonthat T, Stuart J, Ranade S, Peckham H, Zeng K,
      Malek JA, Costa G, McKernan K, Sidow A, Fire A, Johnson SM (July 2008).
      "A_high-resolution,_nucleosome_position_map_of_C._elegans_reveals_a_lack
      of_universal_sequence-dictated_positioning". Genome Res. 18 (7):
      1051â63. doi:10.1101/gr.076463.108. PMC 2493394. PMID 18477713.
 106. ^Rusk N (2011). "Torrents of sequence". Nat Methods. 8 (1): 44. doi:
      10.1038/nmeth.f.330.
 107. ^ a bDrmanac R, Sparks AB, et al. (2010). "Human Genome Sequencing Using
      Unchained Base Reads in Self-Assembling DNA Nanoarrays". Science. 327
      (5961): 78â81. Bibcode:2010Sci...327...78D. doi:10.1126/
      science.1181498. PMID 19892942.
 108. ^Porreca GJ (2010). "Genome Sequencing on Nanoballs". Nature
      Biotechnology. 28 (1): 43â44. doi:10.1038/nbt0110-43. PMID 20062041.
 109. ^ Complete_Genomics Press release, 2010
 110. ^"HeliScope_Gene_Sequencing_/_Genetic_Analyzer_System :_Helicos
      BioSciences". 2 November 2009. Archived from the_original on 2 November
      2009.
 111. ^Thompson JF, Steinmann KE (October 2010). Single molecule sequencing
      with a HeliScope genetic analysis system. Current Protocols in Molecular
      Biology. Chapter 7. pp. Unit7.10. doi:10.1002/0471142727.mb0710s92.
      ISBN 978-0471142720. PMC 2954431. PMID 20890904.
 112. ^"tSMS_SeqLL_Technical_Explanation". SeqLL. Archived from the_original on
      8 August 2014. Retrieved 9 August 2015.
 113. ^Sara El-Metwally; Osama M. Ouda; Mohamed Helmy (2014). "New Horizons in
      Next-Generation Sequencing". Next Generation Sequencing Technologies and
      Challenges in Sequence Assembly. Next Generation Sequencing Technologies
      and Challenges in Sequence Assembly, Springer Briefs in Systems Biology
      Volume 7. SpringerBriefs in Systems Biology. 7. pp. 51â59. doi:10.1007/
      978-1-4939-0715-1_6. ISBN 978-1-4939-0714-4.
 114. ^"PacBio_Sales_Start_to_Pick_Up_as_Company_Delivers_on_Product
      Enhancements".
 115. ^"Bio-IT_World". www.bio-itworld.com.
 116. ^"PacBio_Launches_Higher-Throughput,_Lower-Cost_Single-Molecule
      Sequencing_System".
 117. ^Clarke J, Wu HC, Jayasinghe L, Patel A, Reid S, Bayley H (April 2009).
      "Continuous base identification for single-molecule nanopore DNA
      sequencing". Nature Nanotechnology. 4 (4): 265â70. Bibcode:
      2009NatNa...4..265C. doi:10.1038/nnano.2009.12. PMID 19350039.
 118. ^ a bdela Torre R, Larkin J, Singer A, Meller A (2012). "Fabrication_and
      characterization_of_solid-state_nanopore_arrays_for_high-throughput_DNA
      sequencing". Nanotechnology. 23 (38): 385308. Bibcode:
      2012Nanot..23L5308D. doi:10.1088/0957-4484/23/38/385308. PMC 3557807.
      PMID 22948520.
 119. ^ a bPathak B, Lofas H, Prasongkit J, Grigoriev A, Ahuja R, Scheicher RH
      (2012). "Double-functionalized_nanopore-embedded_gold_electrodes_for
      rapid_DNA_sequencing". Applied Physics Letters. 100 (2): 023701. Bibcode:
      2012ApPhL.100b3701P. doi:10.1063/1.3673335.
 120. ^Korlach J, Marks PJ, Cicero RL, Gray JJ, Murphy DL, Roitman DB, Pham TT,
      Otto GA, Foquet M, Turner SW (2008). "Selective_aluminum_passivation_for
      targeted_immobilization_of_single_DNA_polymerase_molecules_in_zero-mode
      waveguide_nanostructures". Proceedings of the National Academy of
      Sciences. 105 (4): 1176â81. Bibcode:2008PNAS..105.1176K. doi:10.1073/
      pnas.0710982105. PMC 2234111. PMID 18216253.
 121. ^ a bFair RB, Khlystov A, Tailor TD, Ivanov V, Evans RD, Srinivasan V,
      Pamula VK, Pollack MG, Griffin PB, Zhou J (January 2007). "Chemical and
      Biological Applications of Digital-Microfluidic Devices". IEEE Design &
      Test of Computers. 24 (1): 10â24. doi:10.1109/MDT.2007.8. hdl:10161/
      6987.
 122. ^ a bBoles DJ, Benton JL, Siew GJ, Levy MH, Thwar PK, Sandahl MA, et al.
      (November 2011). "Droplet-based_pyrosequencing_using_digital
      microfluidics". Analytical Chemistry. 83 (22): 8439â47. doi:10.1021/
      ac201416j. PMC 3690483. PMID 21932784.
 123. ^Zilionis R, Nainys J, Veres A, Savova V, Zemmour D, Klein AM, Mazutis L
      (January 2017). "Single-cell barcoding and sequencing using droplet
      microfluidics". Nature Protocols. 12 (1): 44â73. doi:10.1038/
      nprot.2016.154. PMID 27929523.
 124. ^"The_Harvard_Nanopore_Group". Mcb.harvard.edu. Archived from the
      original on 21 February 2002. Retrieved 15 November 2009.
 125. ^"Nanopore_Sequencing_Could_Slash_DNA_Analysis_Costs".
 126. ^ US_patent_20060029957, ZS Genetics, "Systems and methods of analyzing
      nucleic acid polymers and related components", issued 2005-07-14 
 127. ^Xu M, Fujita D, Hanagata N (December 2009). "Perspectives and challenges
      of emerging single-molecule DNA sequencing technologies". Small. 5 (23):
      2638â49. doi:10.1002/smll.200900976. PMID 19904762.
 128. ^Schadt EE, Turner S, Kasarskis A (2010). "A window into third-generation
      sequencing". Human Molecular Genetics. 19 (R2): R227â40. doi:10.1093/
      hmg/ddq416. PMID 20858600.
 129. ^Xu M, Endres RG, Arakawa Y (2007). "The electronic properties of DNA
      bases". Small. 3 (9): 1539â43. doi:10.1002/smll.200600732.
      PMID 17786897.
 130. ^Di Ventra M (2013). "Fast_DNA_sequencing_by_electrical_means_inches
      closer". Nanotechnology. 24 (34): 342501. Bibcode:2013Nanot..24H2501D.
      doi:10.1088/0957-4484/24/34/342501. PMID 23899780.
 131. ^Ohshiro T, Matsubara K, Tsutsui M, Furuhashi M, Taniguchi M, Kawai T
      (2012). "Single-molecule_electrical_random_resequencing_of_DNA_and_RNA".
      Sci Rep. 2: 501. Bibcode:2012NatSR...2E.501O. doi:10.1038/srep00501.
      PMC 3392642. PMID 22787559.
 132. ^Hanna GJ, Johnson VA, Kuritzkes DR, Richman DD, Martinez-Picado J,
      Sutton L, Hazelwood JD, D'Aquila RT (1 July 2000). "Comparison_of
      Sequencing_by_Hybridization_and_Cycle_Sequencing_for_Genotyping_of_Human
      Immunodeficiency_Virus_Type_1_Reverse_Transcriptase". J. Clin. Microbiol.
      38 (7): 2715â21. PMC 87006. PMID 10878069.
 133. ^ a bMorey M, FernÃ¡ndez-Marmiesse A, CastiÃ±eiras D, Fraga JM, Couce ML,
      Cocho JA (2013). "A glimpse into past, present, and future DNA
      sequencing". Molecular Genetics and Metabolism. 110 (1â2): 3â24. doi:
      10.1016/j.ymgme.2013.04.024. PMID 23742747.
 134. ^Qin Y, Schneider TM, Brenner MP (2012). Gibas C (ed.). "Sequencing_by
      Hybridization_of_Long_Targets". PLoS ONE. 7 (5): e35819. Bibcode:
      2012PLoSO...735819Q. doi:10.1371/journal.pone.0035819. PMC 3344849.
      PMID 22574124.
 135. ^Edwards JR, Ruparel H, Ju J (2005). "Mass-spectrometry DNA sequencing".
      Mutation Research. 573 (1â2): 3â12. doi:10.1016/j.mrfmmm.2004.07.021.
      PMID 15829234.
 136. ^Hall TA, Budowle B, Jiang Y, Blyn L, Eshoo M, Sannes-Lowery KA, Sampath
      R, Drader JJ, Hannis JC, Harrell P, Samant V, White N, Ecker DJ,
      Hofstadler SA (2005). "Base composition analysis of human mitochondrial
      DNA using electrospray ionization mass spectrometry: A novel tool for the
      identification and differentiation of humans". Analytical Biochemistry.
      344 (1): 53â69. doi:10.1016/j.ab.2005.05.028. PMID 16054106.
 137. ^Howard R, Encheva V, Thomson J, Bache K, Chan YT, Cowen S, Debenham P,
      Dixon A, Krause JU, Krishan E, Moore D, Moore V, Ojo M, Rodrigues S,
      Stokes P, Walker J, Zimmermann W, Barallon R (15 June 2011). "Comparative
      analysis of human mitochondrial DNA from World War I bone samples by DNA
      sequencing and ESI-TOF mass spectrometry". Forensic Science
      International: Genetics. 7 (1): 1â9. doi:10.1016/j.fsigen.2011.05.009.
      PMID 21683667.
 138. ^Monforte JA, Becker CH (1 March 1997). "High-throughput DNA analysis by
      time-of-flight mass spectrometry". Nature Medicine. 3 (3): 360â62. doi:
      10.1038/nm0397-360. PMID 9055869.
 139. ^Beres SB, Carroll RK, Shea PR, Sitkiewicz I, Martinez-Gutierrez JC, Low
      DE, McGeer A, Willey BM, Green K, Tyrrell GJ, Goldman TD, Feldgarden M,
      Birren BW, Fofanov Y, Boos J, Wheaton WD, Honisch C, Musser JM (8
      February 2010). "Molecular_complexity_of_successive_bacterial_epidemics
      deconvoluted_by_comparative_pathogenomics". Proceedings of the National
      Academy of Sciences. 107 (9): 4371â76. Bibcode:2010PNAS..107.4371B.
      doi:10.1073/pnas.0911295107. PMC 2840111. PMID 20142485.
 140. ^Kan CW, Fredlake CP, Doherty EA, Barron AE (1 November 2004). "DNA
      sequencing and genotyping in miniaturized electrophoresis systems".
      Electrophoresis. 25 (21â22): 3564â88. doi:10.1002/elps.200406161.
      PMID 15565709.
 141. ^Chen YJ, Roller EE, Huang X (2010). "DNA_sequencing_by_denaturation:
      experimental_proof_of_concept_with_an_integrated_fluidic_device". Lab on
      a Chip. 10 (9): 1153â59. doi:10.1039/b921417h. PMC 2881221.
      PMID 20390134.
 142. ^Bell DC, Thomas WK, Murtagh KM, Dionne CA, Graham AC, Anderson JE,
      Glover WR (9 October 2012). "DNA Base Identification by Electron
      Microscopy". Microscopy and Microanalysis. 18 (5): 1049â53. Bibcode:
      2012MiMic..18.1049B. doi:10.1017/S1431927612012615. PMID 23046798.
 143. ^Pareek CS, Smoczynski R, Tretyn A (November 2011). "Sequencing
      technologies_and_genome_sequencing". Journal of Applied Genetics. 52 (4):
      413â35. doi:10.1007/s13353-011-0057-x. PMC 3189340. PMID 21698376.
 144. ^Pareek CS, Smoczynski R, Tretyn A (2011). "Sequencing_technologies_and
      genome_sequencing". Journal of Applied Genetics. 52 (4): 413â35. doi:
      10.1007/s13353-011-0057-x. PMC 3189340. PMID 21698376.
 145. ^Fujimori S, Hirai N, Ohashi H, Masuoka K, Nishikimi A, Fukui Y, Washio
      T, Oshikubo T, Yamashita T, Miyamoto-Sato E (2012). "Next-generation
      sequencing_coupled_with_a_cell-free_display_technology_for_high-
      throughput_production_of_reliable_interactome_data". Scientific Reports.
      2: 691. Bibcode:2012NatSR...2E.691F. doi:10.1038/srep00691. PMC 3466446.
      PMID 23056904.
 146. ^Harbers M (2008). "The Current Status of cDNA Cloning". Genomics. 91
      (3): 232â42. doi:10.1016/j.ygeno.2007.11.004. PMID 18222633.
 147. ^Alberti A, Belser C, Engelen S, Bertrand L, Orvain C, Brinas L, Cruaud
      C, et al. (2014). "Comparison_of_Library_Preparation_Methods_Reveals
      Their_Impact_on_Interpretation_of_Metatranscriptomic_Data". BMC Genomics.
      15: 912â12. doi:10.1186/1471-2164-15-912. PMC 4213505. PMID 25331572.
 148. ^"Scalable_Nucleic_Acid_Quality_Assessments_for_Illumina_Next-Generation
      Sequencing_Library_Prep" (PDF). Retrieved 27 December 2017.
 149. ^"Archon_Genomics_XPRIZE". Archon Genomics XPRIZE.
 150. ^"Grant_Information". National Human Genome Research Institute (NHGRI).
 151. ^Severin J, Lizio M, Harshbarger J, Kawaji H, Daub CO, Hayashizaki Y,
      Bertin N, Forrest AR (2014). "Interactive visualization and analysis of
      large-scale sequencing datasets using ZENBU". Nat. Biotechnol. 32 (3):
      217â19. doi:10.1038/nbt.2840. PMID 24727769.
 152. ^Shmilovici A, Ben-Gal I (2007). "Using_a_VOM_model_for_reconstructing
      potential_coding_regions_in_EST_sequences" (PDF). Computational
      Statistics. 22 (1): 49â69. doi:10.1007/s00180-007-0021-8.
 153. ^Del Fabbro C, Scalabrin S, Morgante M, Giorgi FM (2013). "An_Extensive
      Evaluation_of_Read_Trimming_Effects_on_Illumina_NGS_Data_Analysis". PLoS
      ONE. 8 (12): e85024. Bibcode:2013PLoSO...885024D. doi:10.1371/
      journal.pone.0085024. PMC 3871669. PMID 24376861.
 154. ^Martin, Marcel (2 May 2011). "Cutadapt removes adapter sequences from
      high-throughput sequencing reads". EMBnet.journal. 17 (1): 10. doi:
      10.14806/ej.17.1.200.
 155. ^Smeds L, KÃ¼nstner A (19 October 2011). "ConDeTri--a_content_dependent
      read_trimmer_for_Illumina_data". PLOS ONE. 6 (10): e26314. Bibcode:
      2011PLoSO...626314S. doi:10.1371/journal.pone.0026314. PMC 3198461.
      PMID 22039460.
 156. ^Prezza N, Del Fabbro C, Vezzi F, De Paoli E, Policriti A (2012). ERNE-
      BS5: Aligning BS-treated Sequences by Multiple Hits on a 5-letters
      Alphabet. Proceedings of the ACM Conference on Bioinformatics,
      Computational Biology and Biomedicine. 12. pp. 12â19. doi:10.1145/
      2382936.2382938. ISBN 9781450316705.
 157. ^Schmieder R, Edwards R (March 2011). "Quality_control_and_preprocessing
      of_metagenomic_datasets". Bioinformatics. 27 (6): 863â4. doi:10.1093/
      bioinformatics/btr026. PMC 3051327. PMID 21278185.
 158. ^Bolger AM, Lohse M, Usadel B (August 2014). "Trimmomatic:_a_flexible
      trimmer_for_Illumina_sequence_data". Bioinformatics. 30 (15): 2114â20.
      doi:10.1093/bioinformatics/btu170. PMC 4103590. PMID 24695404.
 159. ^Cox MP, Peterson DA, Biggs PJ (September 2010). "SolexaQA:_At-a-glance
      quality_assessment_of_Illumina_second-generation_sequencing_data". BMC
      Bioinformatics. 11 (1): 485. doi:10.1186/1471-2105-11-485. PMC 2956736.
      PMID 20875133.
 160. ^Murray TH (January 1991). "Ethical issues in human genome research".
      FASEB Journal. 5 (1): 55â60. doi:10.1096/fasebj.5.1.1825074.
      PMID 1825074.
 161. ^ a b cRobertson JA (August 2003). "The $1000 genome: ethical and legal
      issues in whole genome sequencing of individuals". The American Journal
      of Bioethics. 3 (3): WâIF1. doi:10.1162/152651603322874762.
      PMID 14735880.
 162. ^ a bHenderson, Mark (9 September 2013). "Human_genome_sequencing:_the
      real_ethical_dilemmas". The Guardian. Retrieved 20 May 2015.
 163. ^Harmon, Amy (24 February 2008). "Insurance_Fears_Lead_Many_to_Shun_DNA
      Tests". The New York Times. Retrieved 20 May 2015.
 164. ^ Statement_of_Administration_policy, Executive Office of the President,
      Office of Management and Budget, 27 April 2007
 165. ^National Human Genome Research Institute (21 May 2008). "President_Bush
      Signs_the_Genetic_Information_Nondiscrimination_Act_of_2008". Retrieved
      17 February 2014.
 166. ^Baker, Monya. "US_ethics_panel_reports_on_DNA_sequencing_and_privacy".
      Nature New Blog. Retrieved 20 May 2015.
 167. ^"Privacy_and_Progress_in_Whole_Genome_Sequencing" (PDF). Presidential
      Commission for the Study of Bioethical Issues. Archived from the_original
      (PDF) on 12 June 2015. Retrieved 20 May 2015.
 168. ^Goldenberg AJ, Sharp RR (February 2012). "The_ethical_hazards_and
      programmatic_challenges_of_genomic_newborn_screening". JAMA. 307 (5):
      461â2. doi:10.1001/jama.2012.68. PMC 3868436. PMID 22298675.
 169. ^Hughes, Virginia (7 January 2013). "It's_Time_To_Stop_Obsessing_About
      the_Dangers_of_Genetic_Information". Slate Magazine. Retrieved 22 May
      2015.
 170. ^ a bBloss CS, Schork NJ, Topol EJ (February 2011). "Effect_of_direct-to-
      consumer_genomewide_profiling_to_assess_disease_risk". The New England
      Journal of Medicine. 364 (6): 524â34. doi:10.1056/NEJMoa1011893.
      PMC 3786730. PMID 21226570.
 171. ^Rochman, Bonnie (25 October 2012). "What_Your_Doctor_Isn't_Telling_You
      About_Your_DNA". Time.com. Retrieved 22 May 2015.
***** External links[edit] *****
Library_resources about
DNA sequencing
===============================================================================
    * Resources_in_your_library
    * Resources_in_other_libraries
 Wikibooks has a book on the topic of: Next_Generation_Sequencing_(NGS)
    * A wikibook_on_next_generation_sequencing
    * A free_didactic_directory_for_DNA_sequencing_analysis.
    * A The_path_to_DNA_sequencing:_The_life_and_work_of_Fred_Sanger
    * A US_Map_of_New_Generation_Sequencers
    * A Visualization_of_DNA_sequences
    * [TPI1_structure.png]Molecular_and_cellular_biology_portal

Retrieved from "https://en.wikipedia.org/w/
index.php?title=DNA_sequencing&oldid=908764717"
Categories:
    * DNA_sequencing
    * Biotechnology
    * DNA
    * Genetic_mapping
    * Molecular_biology
    * Molecular_biology_techniques
    * 1970_introductions
    * 1970_in_biology
    * 1970_in_biotechnology
    * 1970_in_science
    * 1998_in_technology
Hidden categories:
    * Pages_containing_links_to_subscription-only_content
    * Wikipedia_articles_needing_page_number_citations_from_June_2013
    * Pages_with_login_required_references_or_sources
    * CS1_maint:_Archived_copy_as_title
    * CS1:_long_volume_value
    * Wikipedia_indefinitely_move-protected_pages
    * Articles_with_short_description
    * Use_dmy_dates_from_April_2011
    * Articles_containing_potentially_dated_statements_from_2019
    * All_articles_containing_potentially_dated_statements
    * All_articles_with_unsourced_statements
    * Articles_with_unsourced_statements_from_July_2018
    * Articles_to_be_expanded_from_May_2015
    * All_articles_to_be_expanded
    * Articles_using_small_message_boxes
***** Navigation menu *****
**** Personal tools ****
    * Not logged in
    * Talk
    * Contributions
    * Create_account
    * Log_in
**** Namespaces ****
    * Article
    * Talk
⁰
**** Variants ****
**** Views ****
    * Read
    * Edit
    * View_history
⁰
**** More ****
**** Search ****
[Unknown INPUT type][Search][Go]
**** Navigation ****
    * Main_page
    * Contents
    * Featured_content
    * Current_events
    * Random_article
    * Donate_to_Wikipedia
    * Wikipedia_store
**** Interaction ****
    * Help
    * About_Wikipedia
    * Community_portal
    * Recent_changes
    * Contact_page
**** Tools ****
    * What_links_here
    * Related_changes
    * Upload_file
    * Special_pages
    * Permanent_link
    * Page_information
    * Wikidata_item
    * Cite_this_page
**** In other projects ****
    * Wikimedia_Commons
**** Print/export ****
    * Create_a_book
    * Download_as_PDF
    * Printable_version
**** Languages ****
    * Ø§ÙØ¹Ø±Ø¨ÙØ©
    * CatalÃ 
    * ÄeÅ¡tina
    * Dansk
    * Deutsch
    * Eesti
    * EspaÃ±ol
    * Esperanto
    * ÙØ§Ø±Ø³Û
    * FranÃ§ais
    * Galego
    * Hrvatski
    * Bahasa_Indonesia
    * Ãslenska
    * Italiano
    * ×¢××¨××ª
    * Jawa
    * LietuviÅ³
    * Magyar
    * Bahasa_Melayu
    * Nederlands
    * æ¥æ¬èª
    * Norsk
    * Polski
    * PortuguÃªs
    * RomÃ¢nÄ
    * Ð¡ÑÐ¿ÑÐºÐ¸_/_srpski
    * Srpskohrvatski_/_ÑÑÐ¿ÑÐºÐ¾ÑÑÐ²Ð°ÑÑÐºÐ¸
    * Suomi
    * Svenska
    * à®¤à®®à®¿à®´à¯
    * à¹à¸à¸¢
    * TÃ¼rkÃ§e
    * Ð£ÐºÑÐ°ÑÐ½ÑÑÐºÐ°
    * Tiáº¿ng_Viá»t
    * ä¸­æ
Edit_links
    * This page was last edited on 31 July 2019, at 20:42 (UTC).
    * Text is available under the Creative_Commons_Attribution-ShareAlike
      License; additional terms may apply. By using this site, you agree to the
      Terms_of_Use and Privacy_Policy. WikipediaÂ® is a registered trademark of
      the Wikimedia_Foundation,_Inc., a non-profit organization.
    * Privacy_policy
    * About_Wikipedia
    * Disclaimers
    * Contact_Wikipedia
    * Developers
    * Cookie_statement
    * Mobile_view
    * [Wikimedia_Foundation]
    * [Powered_by_MediaWiki]
